{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append(\"..\")\n",
    "sys.path.append(\"../src\")\n",
    "\n",
    "from dataset import process\n",
    "from dataIO import load_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'a': 1, 'b': 2}"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d = {'a':1, 'b': 2, 'c': 3}\n",
    "d.pop('c')\n",
    "d"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-31 17:53:33,114 - pdf.extract_PDF - INFO - Extracting pdfs for year 2005\n",
      "\n",
      "2025-03-31 17:53:33,127 - pdf.download_PDF - INFO - Downloading pdfs for year 2005\n",
      "\n",
      "2025-03-31 17:53:33,374 - vectorDB.push_qdrant - INFO - Qdrant container already up and running\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "364b988893224d2999ddc2333c7ceb4d",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Downloading 3000 papers from 2005:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-31 17:53:44,375 - pdf.download_PDF - INFO - Closed 0 Chrome processes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/jul/DST/asXiv/data/tmp/download/2005\n",
      "no new data. Will wait 27992.200932741165 more seconds\n",
      "no new data. Will wait 27987.197122573853 more seconds\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "7a1c5c65fe7145d09779e0517175323d",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Extracting 22 new papers:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-31 17:54:00,323 - pdf.extract_PDF - INFO - 1 extractions out of 22 timed out. Terminating the pool.\n",
      "\n",
      "2025-03-31 17:54:00,499 - pdf.extract_PDF - INFO - checking valid pages for 0articles that timed out...\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "8ff385e443b24e9f8744391254d3106d",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Extracting 47 new papers:   0%|          | 0/2 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "e7851914b6de4e20852ea415ba7395ca",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Extracting 155 new papers:   0%|          | 0/4 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "6bc7400771a94266ac656fe183b64014",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Extracting 410 new papers:   0%|          | 0/10 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-31 17:56:04,544 - dataset.embed_and_push - INFO - Will now embed 264 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "d4af1826d9b74a838e02b353c970469b",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Token indices sequence length is longer than the specified maximum sequence length for this model (638 > 512). Running this sequence through the model will result in indexing errors\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "3d21144ff1c048daa8827c6d4c482ff9",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/90 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-31 17:58:30,008 - pdf.extract_PDF - WARNING - Error reading PDF /home/jul/DST/asXiv/data/tmp/download/2005/03803997b650390543c480191ed258fdabd8476e/The FASEB Journal - 2005 - Prinzen - Genomic structure and functional characterization of the human ADAM10 promoter.pdf: Invalid dictionary construct: [/'OP', False, /'op', False, /'Type', /'ExtGState', /'SM', 1.0, /'OPM', 0, /'CA', 1.0, /'SA', /b'fa', /b'lse', /'ca', 1.0]\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "f5e992d121f14e8faa084cf53309f9e8",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Extracting 1146 new papers:   0%|          | 0/28 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 17.09it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 17.85it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 18.51it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 21.12it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 18.43it/s]\n",
      "Uploading: 100%|██████████| 5/5 [00:00<00:00, 24.18it/s]\n",
      "Uploading: 100%|██████████| 7/7 [00:00<00:00, 24.21it/s]\n",
      "Uploading: 100%|██████████| 12/12 [00:00<00:00, 26.28it/s]\n",
      "Uploading: 100%|██████████| 22/22 [00:00<00:00, 24.58it/s]\n",
      "Uploading: 100%|██████████| 34/34 [00:01<00:00, 28.56it/s]\n",
      "Uploading: 100%|██████████| 47/47 [00:02<00:00, 21.07it/s]\n",
      "Uploading: 100%|██████████| 85/85 [00:03<00:00, 23.82it/s]\n",
      "Uploading: 100%|██████████| 45/45 [00:01<00:00, 23.13it/s]\n",
      "\n",
      "2025-03-31 17:59:05,285 - dataset.embed_and_push - INFO - Will now embed 279 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "918542faa9a043448294fdc1f533e8a6",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "3d62454e4dff46c3a028fd9bd60424cd",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/94 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-31 18:00:07,537 - pdf.download_PDF - INFO - Closed 348 Chrome processes\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 24.77it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 26.88it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 30.55it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 26.68it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 27.95it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 20.08it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 26.37it/s]\n",
      "Uploading: 100%|██████████| 4/4 [00:00<00:00, 33.50it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 27.72it/s]\n",
      "Uploading: 100%|██████████| 6/6 [00:00<00:00, 36.00it/s]\n",
      "Uploading: 100%|██████████| 11/11 [00:00<00:00, 33.97it/s]\n",
      "Uploading: 100%|██████████| 16/16 [00:00<00:00, 30.53it/s]\n",
      "Uploading: 100%|██████████| 24/24 [00:01<00:00, 22.70it/s]\n",
      "Uploading: 100%|██████████| 54/54 [00:01<00:00, 31.48it/s]\n",
      "Uploading: 100%|██████████| 69/69 [00:02<00:00, 30.93it/s]\n",
      "Uploading: 100%|██████████| 79/79 [00:02<00:00, 35.24it/s]\n",
      "\n",
      "2025-03-31 18:01:51,718 - dataset.embed_and_push - INFO - Will now embed 277 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "db0afb22cadb42119ed5780d1428733e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-03-31 18:02:08,204 - pdf.extract_PDF - WARNING - Error reading PDF /home/jul/DST/asXiv/data/tmp/download/2005/0731ee67b39b77da79bbba995ac6161fa8d4ed85/Genes to Cells - 2005 - Nonami - The Sprouty‐related protein  Spred‐1  localizes in a lipid raft caveola and inhibits ERK.pdf: Invalid dictionary construct: [/'OP', /b'fa', /b'lse', /'op', False, /'UCR2', /'Default', /'Type', /'ExtGState', /'BG2', /'Default', /'SM', 0.02, /'OPM', 1, /'SA', True]\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "fd218eee47ff4eab8ee2f43620dd00c4",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/95 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "ec763fa92ec749b1a7425ac63769056e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Extracting 246 new papers:   0%|          | 0/6 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 26.29it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 31.91it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 25.93it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 25.60it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 28.75it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 25.51it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 22.19it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 29.74it/s]\n",
      "Uploading: 100%|██████████| 8/8 [00:00<00:00, 36.74it/s]\n",
      "Uploading: 100%|██████████| 10/10 [00:00<00:00, 34.25it/s]\n",
      "Uploading: 100%|██████████| 16/16 [00:00<00:00, 37.60it/s]\n",
      "Uploading: 100%|██████████| 22/22 [00:00<00:00, 38.84it/s]\n",
      "Uploading: 100%|██████████| 25/25 [00:00<00:00, 37.47it/s]\n",
      "Uploading: 100%|██████████| 38/38 [00:01<00:00, 36.05it/s]\n",
      "Uploading: 100%|██████████| 47/47 [00:01<00:00, 36.36it/s]\n",
      "Uploading: 100%|██████████| 62/62 [00:01<00:00, 37.44it/s]\n",
      "Uploading: 100%|██████████| 36/36 [00:01<00:00, 31.94it/s]\n",
      "\n",
      "2025-03-31 18:04:35,228 - dataset.embed_and_push - INFO - Will now embed 279 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "3769ff9e73554586bef762644551b274",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "73b6cc2df27247d891e10e9b22b13df8",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/95 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "no new data. Will wait 54.97905659675598 more seconds\n",
      "no new data. Will wait 49.95940327644348 more seconds\n",
      "no new data. Will wait 44.93690824508667 more seconds\n",
      "no new data. Will wait 39.91395592689514 more seconds\n",
      "no new data. Will wait 34.89076280593872 more seconds\n",
      "no new data. Will wait 29.86532688140869 more seconds\n",
      "no new data. Will wait 24.840577363967896 more seconds\n",
      "no new data. Will wait 19.819511890411377 more seconds\n",
      "no new data. Will wait 14.79427981376648 more seconds\n",
      "no new data. Will wait 9.771430969238281 more seconds\n",
      "no new data. Will wait 4.746606349945068 more seconds\n",
      "no new data. Will wait -0.2763688564300537 more seconds\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 21.35it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 32.21it/s]\n",
      "Uploading: 100%|██████████| 4/4 [00:00<00:00, 32.45it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 28.35it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 27.59it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 22.13it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 31.73it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 31.83it/s]\n",
      "Uploading: 100%|██████████| 6/6 [00:00<00:00, 31.13it/s]\n",
      "Uploading: 100%|██████████| 10/10 [00:00<00:00, 33.02it/s]\n",
      "Uploading: 100%|██████████| 15/15 [00:00<00:00, 32.04it/s]\n",
      "Uploading: 100%|██████████| 24/24 [00:00<00:00, 39.20it/s]\n",
      "Uploading: 100%|██████████| 30/30 [00:00<00:00, 37.66it/s]\n",
      "Uploading: 100%|██████████| 43/43 [00:01<00:00, 33.69it/s]\n",
      "Uploading: 100%|██████████| 62/62 [00:01<00:00, 38.25it/s]\n",
      "Uploading: 100%|██████████| 70/70 [00:01<00:00, 39.57it/s]\n",
      "\n",
      "2025-03-31 18:07:12,887 - dataset.embed_and_push - INFO - Will now embed 280 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "39da918ee0c84335a9c1286699df320e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "6b1a422ae20b4d25b58d415230de7483",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/95 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 50.75it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 32.70it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 33.44it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 24.62it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 26.42it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 21.42it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 17.81it/s]\n",
      "Uploading: 100%|██████████| 5/5 [00:00<00:00, 28.15it/s]\n",
      "Uploading: 100%|██████████| 9/9 [00:00<00:00, 33.64it/s]\n",
      "Uploading: 100%|██████████| 13/13 [00:00<00:00, 35.21it/s]\n",
      "Uploading: 100%|██████████| 21/21 [00:00<00:00, 39.03it/s]\n",
      "Uploading: 100%|██████████| 29/29 [00:00<00:00, 37.46it/s]\n",
      "Uploading: 100%|██████████| 37/37 [00:00<00:00, 41.17it/s]\n",
      "Uploading: 100%|██████████| 46/46 [00:01<00:00, 36.65it/s]\n",
      "Uploading:   0%|          | 0/69 [00:00<?, ?it/s]\n",
      "2025-03-31 18:09:46,686 - dataset.embed_and_push - INFO - Will now embed 40 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "4253d846e64b441b9ee17b1a6ba59fad",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 69/69 [00:01<00:00, 36.99it/s]\n",
      "Uploading: 100%|██████████| 41/41 [00:01<00:00, 26.55it/s]\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "6b02e20e07f24d32967e5b12c5685255",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/14 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 43.43it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 53.52it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 35.08it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 27.35it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 35.03it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 34.40it/s]\n",
      "Uploading: 100%|██████████| 2/2 [00:00<00:00, 32.18it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 26.79it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 33.29it/s]\n",
      "Uploading: 100%|██████████| 9/9 [00:00<00:00, 37.03it/s]\n",
      "Uploading: 100%|██████████| 17/17 [00:00<00:00, 40.60it/s]\n",
      "\n",
      "2025-03-31 18:10:10,652 - dataset.embed_and_push - INFO - Will now embed 279 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "17c405ae6425462fbd0178dd73f342fb",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "d1381f58f2604e7b9b11828399532564",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/95 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 29.10it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 37.24it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 30.89it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 33.47it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 22.93it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 29.05it/s]\n",
      "Uploading: 100%|██████████| 9/9 [00:00<00:00, 37.90it/s]\n",
      "Uploading: 100%|██████████| 12/12 [00:00<00:00, 39.84it/s]\n",
      "Uploading: 100%|██████████| 14/14 [00:00<00:00, 40.71it/s]\n",
      "Uploading: 100%|██████████| 19/19 [00:00<00:00, 39.40it/s]\n",
      "Uploading: 100%|██████████| 25/25 [00:00<00:00, 34.12it/s]\n",
      "Uploading: 100%|██████████| 36/36 [00:00<00:00, 37.48it/s]\n",
      "Uploading: 100%|██████████| 49/49 [00:01<00:00, 37.45it/s]\n",
      "Uploading: 100%|██████████| 68/68 [00:01<00:00, 41.52it/s]\n",
      "Uploading: 100%|██████████| 34/34 [00:00<00:00, 40.63it/s]\n",
      "\n",
      "2025-03-31 18:12:47,247 - dataset.embed_and_push - INFO - Will now embed 277 articles that just got extracted\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "70f2eac244c44dd89367fbda9a556558",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Paper batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "5da354b4ac844e20ba10a6c5c84cd7d7",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "Computing embeddings:   0%|          | 0/93 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 30.99it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 32.00it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 30.16it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 36.14it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 24.13it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 21.49it/s]\n",
      "Uploading: 100%|██████████| 1/1 [00:00<00:00, 22.10it/s]\n",
      "Uploading: 100%|██████████| 3/3 [00:00<00:00, 34.00it/s]\n",
      "Uploading: 100%|██████████| 6/6 [00:00<00:00, 33.21it/s]\n",
      "Uploading: 100%|██████████| 7/7 [00:00<00:00, 36.97it/s]\n",
      "Uploading: 100%|██████████| 12/12 [00:00<00:00, 35.15it/s]\n",
      "Uploading: 100%|██████████| 19/19 [00:00<00:00, 40.89it/s]\n",
      "Uploading: 100%|██████████| 25/25 [00:00<00:00, 43.33it/s]\n",
      "Uploading: 100%|██████████| 30/30 [00:00<00:00, 36.48it/s]\n",
      "Uploading: 100%|██████████| 44/44 [00:01<00:00, 38.52it/s]\n",
      "Uploading: 100%|██████████| 56/56 [00:01<00:00, 38.73it/s]\n",
      "Uploading: 100%|██████████| 60/60 [00:01<00:00, 43.43it/s]\n"
     ]
    }
   ],
   "source": [
    "process(years=2005, download_extract=True, embed_push=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "fieldsOfStudy",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "venue",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorName",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "citationCount",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "influentialCitationCount",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "publicationDate",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "publicationYear",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "referenceIds",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "d7f8b391-04ec-4d1f-a71a-8fd11d8a0173",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "https://aacrjournals.org/clincancerres/article-pdf/11/7/2702/1962059/2702-2712.pdf",
         "Biology,Medicine",
         "Clinical Cancer Research",
         "T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. Nishijo,K. Shibata,Y. Shima,S. Nagayama,T. Katagiri,Yusuke Nakamura,Takashi Nakamura,J. Toguchida",
         "14839966,10239302,2000448727,3201672,7147569,2066435363,31907929,4360342,7367122,144863658,145103616,3630533",
         "88",
         "6.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "http://www.merl.com/publications/docs/TR2005-063.pdf",
         "Computer Science,Mathematics",
         "IEEE Transactions on Communications",
         "Juntan Zhang,M. Fossorier",
         "3193350,48891339",
         "319",
         "26.0",
         "2005-03-07",
         "2005",
         "17d0afe4f814e5abc6252e572e3b039082bf6f07;02464a444975fdfbef09c40e9e6d140b2f10d213;5adb4ba3b3717d86d507e57ec7b04ca91b0daa32;3870f98d6575420d9e3d02a5f299fa483123caaa;610cd4607f39edb48ce8d8dda982b04cb9cb1d75;4f773884c351152c49b1111d98c07446b1543dc2;e28fc01c3f8ca4151abf9b61296b56dc318d3b26;fbf85b75d9d3d8c224de034005d2f2cbc7c4fac2;64e7badd52b7841ebc25d09e6b38e6d7dbea5f38;293620136b122dbfb258fe95fd45ca85c814236c;0bf5d709b14dd074522d96d2ffb4474b3184b984;444e24fd9dff66dfafeb70914b6b3c83e9003853",
         "Low-Density Parity-Check Codes;Turbo decoder architectures for low-density parity-check codes;Shuffled belief propagation decoding;A parallel decoder for low latency decoding of turbo product codes;Parallel versus sequential updating for belief propagation decoding;High throughput low-density parity-check decoder architectures;Good error-correcting codes based on very sparse matrices;Near optimum error correcting coding and decoding: turbo-codes;Multiple turbo codes for deep-space communications;Optimal decoding of linear codes for minimizing symbol error rate (Corresp.);Generic Architecture for LDPC Codes Decoding;Optimal decoding of linear codes for minimizing symbol error rate"
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "https://www.jstage.jst.go.jp/article/bpb/28/8/28_8_1450/_pdf",
         "Medicine",
         "Biological and Pharmaceutical Bulletin",
         "Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,Jian Sun,Q. Xia,K. Bian",
         "11809650,6617669,123635715,11305673,2152146096,144130337,2144753879",
         "24",
         "2.0",
         "2005-08-01",
         "2005",
         "",
         ""
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/dvdy.20220",
         "Biology,Medicine",
         "Developmental Dynamics",
         "Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V. Sukhatme",
         "35690923,2114125286,2113252574,2134773,3658233",
         "53",
         "2.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "http://www.jbc.org/article/S0021925820589394/pdf",
         "Chemistry,Medicine",
         "Journal of Biological Chemistry",
         "M. Westfall,A. M. Lee,D. Robinson",
         "5025983,49249911,48706510",
         "70",
         "3.0",
         "2005-12-16",
         "2005",
         "",
         ""
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "http://dl.acm.org/ft_gateway.cfm?id=1047480&type=pdf",
         "Computer Science",
         "Technical Symposium on Computer Science Education",
         "Susan Bergin,R. Reilly",
         "145985158,2688685",
         "287",
         "26.0",
         "2005-02-23",
         "2005",
         "b8123b4807efdee5d3a408e6f862ff438c642127;afae3da8f9b6e9b644a5dc695b50796e020d4194;cbaddfc496cebf6dc683ed8aad2306a359aa58ce;839b94477a293d0bd10acd2411589fd0cad2512c;259ec7be9d7bbb5b633978c72cf2a93bd86b25f9;78033d2947f98bf7189c5522c78251d4f9bf2eec;499da8c546288d9b12dd5acddc6d1cba3c9a5f56;5453190a0090efe599f136df5a8a2a529ad6ac9a;978a0c742ebfcbb41b9b9b472519512d9759accd;4ea72d8b5f946427398474c32f4bcff74b5f871b;ef3cf9f8bf4aa787afbdb0eaec91b1cc808a0898;3a6651c691c57e1c5badd03bb9bcb2b565832159;500a21438777d026ea95489c5a77d1cb1ac5d5a8;dbae45bc40eeeac237f360c5006f40611360b856;0ae8f4656278b6c683984cd60f3c3a0606da5bc1;f60e8ea65110900e1f4c3816bc720591fcc41f79;fdc5a2d599b5f9c4990f3ab4dc3ce86ae8251627",
         "The impact of prior experience in an information technology programming course sequence;Predictors of success and failure in a CS1 course;A multi-national, multi-institutional study of assessment of programming skills of first-year CS students;The effect of student attributes on success in programming;Contributing to success in an introductory computer science course: a study of twelve factors;Does it help to have some programming experience before beginning a computing degree program?;Factors affecting performance in first-year computing;The effect of a constructivist learning environment for field-dependent/independent students on achievement in introductory computer programming;What best predicts computer proficiency?;Predicting student performance in a beginning computer science class;Predicting the success of freshmen in a computer science major;Predicting student success in an introductory programming course;Predicting success in a first programming course;Investigating the relationship between the development of abstract reasoning and performance in an introductory programming class;Factors affecting course outcomes in introductory programming;An Overview of the Integration of Problem Based Learning into an existing Computer Science Programming Module;Discovering the likely performance of novice programmers on an introductory programming module;Differential Performance by gender and achievement in the Certificate Examination Results 2000/2001: Final Report, Dublin: National Council for Curriculum and Assessment;Examining Gender;Cognitive skills needed in computer programming;speciﬁc cognitive skills: as measured by an in-house cognitive test;previous academic and computer experience: as measured by performance in the Irish Leaving Certiﬁcate (LC) examinations in"
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.ATV.0000168413.29874.d7",
         "Medicine,Biology",
         "Arteriosclerosis, Thrombosis and Vascular Biology",
         "E. Egorina,M. Sovershaev,G. Bjørkøy,F. Gruber,J. Olsen,B. Parhami-Seren,K. Mann,B. Østerud",
         "8887769,4962786,11414707,50148632,33502527,1398546666,3162218,5466077",
         "148",
         "2.0",
         "2005-07-01",
         "2005",
         "65eeaee9e552393e75a743c0344ebe4581a6e6a7;8bf514513122d0af4d0aa2b9789994b4211bd7cd;05d6048fa796174028c92893168a0773e927e454;8be482509b14152ffdb18fe3ae5dc7d1afb009a5;c847dd5549f0c843199c1a50718d0f139aaefcd7;ac3cc136968e89551fc3206b23c3bfb1cc150bbd;4498654bf9d23ded98cee2eb667944f9c27da787;832d3e34e1d44cc5c92702ff3e7e97fa6d43ee44;154f2466119853b7ba397eee95c14f010572275d;41f73ad3c17b7c021670dda4e2e1e0e948d5f90c;e6a10b6e88a706227967bae80080553b1e2b51f5;8527e6d03a0f79a038fc130a4344fa4853c8335a;49e92bb9ad78b8b307e99944834e6d1bc631297f;be3f99f27c1ad7a8cf0a8eca7e8b549ef1b6a6dd;4511c51ef9c78ef929a5c9766af97db701da82af;4ca60e84496caca48756bb5b4f6f24236ca17c8f;d24472339b51de01a39c49b3b201c8e43b16f242;01bdc51fe1630bc406cf0661f7ac67200b449966;0be3116d5897256dd59fef2dcf2ba5401359a10d;0db9a7f606d2539f5e1bd2587ba2c0cb7166d66d;2b4563c2e99ab1208d889c22ad7df414c183dd3e;886b5036def2f1c34f63a621738e9ebfe40332ce;18f594d1e008e6bc4e0eaaf7e771acd7f9b6c6ff;ed3087c11d9f60b01ce54b720df2a2420749e77c;516f18c8244c665e450172bbe183c4286bbaa1d4;25405d6b24240aa2adb1f7b7049b80efeb40ce3e;878ce5ee726c6651d087291f37cc046a9f980d54;4e7a537a06f4b75c72bfb551c9a18f99d9679cef;551e55506bd0f8cfc13f22816c9b20315a51970c;48cc95821fe443d86a1b63d0dbd1486a0b012ce1;e3acf2654d08a2006fd32dce370e4fa4e5653b22",
         "The production and availability of tissue thromboplastin in cellular populations of whole blood exposed to various concentrations of endotoxin. An assay for detection of endotoxin.;Tissue factor activity in whole blood.;Nicotinamide inhibits endotoxin‐induced monocyte tissue factor expression;Cellular activation responses in blood in relation to lipid pattern: healthy men and women in families with myocardial infarction or cancer;Tissue factor activity in human monocytes is regulated by plasma: implications for the high and low responder phenomenon;Role of Platelet P-Selectin and CD40 Ligand in the Induction of Monocytic Tissue Factor Expression;Eicosanoids and Iso-Eicosanoids: Constitutive, Inducible and Transcellular Biosynthesis in Vascular Disease;Cytokine gene polymorphism and heart transplant rejection.;Platelet-derived growth factor-BB and monocyte chemotactic protein-1 induce human peripheral blood monocytes to express tissue factor.;Association of smooth muscle cell tissue factor with caveolae.;Manifestation of cryptic fibroblast tissue factor occurs at detergent concentrations which dissolve the plasma membrane;A novel biological effect of platelet factor 4 (PF4): enhancement of LPS‐induced tissue factor activity in monocytes;Th1 and Th2 T-helper cells exert opposite regulatory effects on procoagulant activity and tissue factor production by human monocytes.;Granulocytes enhance LPS‐induced tissue factor activity in monocytes via an interaction with platelets;Cathepsin G-dependent platelet stimulation by activated polymorphonuclear leukocytes and its inhibition by antiproteinases: role of P-selectin-mediated cell-cell adhesion.;Relations between factor VIIa binding and expression of factor VIIa/tissue factor catalytic activity on cell surfaces.;Platelet activating factor enhancement of lipopolysaccharide‐induced tissue factor activity in monocytes: requirement of platelets and granulocytes;Dysregulation of in vitro cytokine production by monocytes during sepsis.;The role of arachidonic acid release and lipoxygenase pathway in lipopolysaccharide-induced thromboplastin activity in monocytes.;In vitro production of IL 1β, IL 1α, TNF and IL 2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation;Functional tissue factor is entirely cell surface expressed on lipopolysaccharide-stimulated human blood monocytes and a constitutively tissue factor-producing neoplastic cell line;Localization of tissue factor in the normal vessel wall and in the atherosclerotic plaque.;Cytokine-induced procoagulant activity in monocytes and endothelial cells. Further enhancement by cyclosporine.;Tumor necrosis factor induces tissue factor-like activity in human leukemia cell line U937 and peripheral blood monocytes.;Enhancement of mononuclear procoagulant activity by platelet 12-hydroxyeicosatetraenoic acid.;The stimulatory effect of platelets and platelet membranes on the procoagulant activity of leukocytes.;The Role of Protein Synthesis on the Generation of Tissue Factor Activity by Leukocytes;Monocytes.;The high responder phenomenon: enhancement of LPS induced tissue factor activity in monocytes by platelets and granulocytes.;Selective cellular expression of tissue factor in human tissues. Implications for disorders of hemostasis and thrombosis.;In vitro production of IL 1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation.;Mononuclear cell tissue factor: cell of origin and requirements for activation.;After blocking with 5% non-fat skimmed milk, membranes were incubated with mouse anti-human TF monoclonal antibodies;After Lymphoprep isolation, total RNA was extracted from MNC pellets with an;OR) anti-human TF monoclonal antibodies (10 µg/ml) for 30 min on ice. Mouse monoclonal antibodies recognizing extracellular domain of human TF were prepared by the Monoclonal Antibody Facility"
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "https://europepmc.org/articles/pmc549252?pdf=render",
         "Biology,Medicine",
         "Antimicrobial Agents and Chemotherapy",
         "T. Levin,B. Suh,P. Axelrod,A. Truant,T. Fekete",
         "24218070,1941629,144104608,4961128,49307125",
         "179",
         "10.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         "http://www.commsp.ee.ic.ac.uk/~ng1/pdf/gehrigD_CL_Jan05.pdf",
         "Computer Science,Mathematics",
         "IEEE Communications Letters",
         "N. Gehrig,P. Dragotti",
         "3017159,2752909",
         "59",
         "5.0",
         "2005-01-10",
         "2005",
         "a0de588554dc558fc9a7c968f02626444c917679;05c21f03eefa2d756229000ebe133b6bc6d08e1c;5e0eff55c4e3e3158b17e5452e213226fd8f21ac;33e627cb93a476a501308fb500015d282297ac03;d7cfbf051867bf3b35c3a2738a4731a148e06cc5;6e3da810748d174ccf2107591019ed318eaf6e40;f2b32222ec8955c4378fd2804757f4f8449d9241;29c907591b8be92bb535cc4e7453a4423d770238;c41ecfe2e42872cf03ced942f1d5daf918a00b5f;71c969f0a7204259cdd394283ab02d0fc34e5fc2",
         "Design of Slepian-Wolf codes by channel code partitioning;On some new approaches to practical Slepian-Wolf compression inspired by channel coding;Distributed code constructions for the entire Slepian-Wolf rate region for arbitrarily correlated sources;Compression of binary sources with side information at the decoder using LDPC codes;Compression with side information using turbo codes;Compression of correlated binary sources using turbo codes;The capacity of low-density parity-check codes under message-passing decoding;Distributed source coding: symmetric rates and applications to sensor networks;Distributed source coding using syndromes (DISCUS): design and construction;Noiseless coding of correlated information sources"
        ],
        [
         "9",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "http://www.cs.ucr.edu/%7Ekishore/papers/tencon.pdf",
         "Computer Science",
         "IEEE Region 10 Conference",
         "A. Negi,K. P. Kishore",
         "1728262,2341041934",
         "41",
         "3.0",
         "2005-11-01",
         "2005",
         "a6ce59fb59ba15a8833a12bfe6692dfcd436824a;aa03424e6c7e7c9415b554a68643c16085df0187;100ab9c177c6e0abdda1130eada7a3fe8015606c;25be4335206df53c769244cfce81ebc5914a46cf;1feb6fb70407c123de7af972a4271c07472fe3a6;1f0a71167544d4018089c51861b10365993ff151;956615751a21c1071f18ab90193f218d6319013e;6bc43977fb11cceed0b9aa55b23c6dd29dd9a132;9308cfdabf5303534b97d9ce5bfbb2c919a3f9cb;da7a3280ec5e77c007045eee8f6f657f567ab97d;3c520bf9d75f785f4d4935ef8bd9e4b38c7c5db2",
         "Performance Contracts: Predicting and Monitoring Grid Application Behavior;The design, implementation and initial evaluation of an advanced knowledge-based process scheduler;Understanding the Linux Kernel;Predicting Application Run Times Using Historical Information;A Historical Application Profiler for Use by Parallel Schedulers;Context Switching and IPC Performance Comparison between uClinux and Linux on the ARM9 based Processor;Characterizing Process Execution Behaviour Using Machine Learning Techniques;The Pearson Education;Linux Kernel Development , 1st ed;Correlation-based Feature Selection for Machine Learning;A Dynamically Adaptive CPU Scheduler;Linux Kernel 2.4 Internals;Co-rrelation based feature selection for Machine Learning, Master Thesis, Department of Computer Science, University of Waikato;Machine Learning , 1st ed, pp: 52-75, 154-183, 230-244, The;WEKA: The Waikato Environment for Knowledge Analysis;ELF: From The Programmer's Perspective;The Design of the UNIX Operating System;Build the knowledge base of static and dynamic characteristics of the programs from the run traces obtained in step 1 and train them with the C4.5 decision tree algorithm;is a Masters student in the Department of Computer and Informations Sciences;We send this STS information to modi(cid:2)ed scheduler through a system call;The scheduler instructs the CPU such that CPU allocates STS ticks to ’X’;decision tree will classify ’X’ and output the STS. per second.(cid:2)le;Aivazian , Tigran , Linux Kernel 2 . 4 Internals , The Linux Documentation Project , August , 2002"
        ],
        [
         "10",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "https://europepmc.org/articles/pmc1087730?pdf=render",
         "Biology,Medicine",
         "Molecular and Cellular Biology",
         "Takayoshi Bando,K. Sekine,Shizuka Kobayashi,A. M. Watabe,Armin Rump,Minoru Tanaka,Yoshikuni Suda,S. Kato,Y. Morikawa,T. Manabe,A. Miyajima",
         "24043805,2063038091,2110065840,40447710,32435082,2110038433,36341155,2568535,32275049,5658239,4789048",
         "38",
         "3.0",
         "2005-05-01",
         "2005",
         "",
         ""
        ],
        [
         "11",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "https://infoscience.epfl.ch/record/87259/files/Cavallaro2005_1275.pdf",
         "Computer Science",
         "IEEE transactions on circuits and systems for video technology (Print)",
         "A. Cavallaro,Olivier Steiger,T. Ebrahimi",
         "145440152,2359024,1681498",
         "120",
         "7.0",
         "2005-10-01",
         "2005",
         "bbb456acfec69d279b3f653b53dc1182d1b44eb6;cedf72be1fe814ef2ee9d65633dc3226f80f0785;dcdef9d1abf4be052e309058d695f19015868209;168033139bddbd1c02f761dd905375203cc2f73a;c69e28503a607c30f042fbd781de47c2a34b57fb;37eec7c527780a71578ecb32b5a48d563e651c7d;51a5d7d650ea5fdc12ae6e359ed052f67c3482c3;972fc844400ad601ca5c9274912ecb8007457790;d91e1b1ee1f83e2b3dc54ba7811a3c38f77ea701;48c7afd55f620cc665cabb52755968c5c57d9eda;ea3bcd105081726ab403513af553f5d0c32a2c6c;f494794fd30f451365aa69a96227ba8384ca492b;8b2f244b587207e8ee5415d507e432d5220d6163;a741ea5e483061f2ad1767290770ef35e74fcab4;3b9f60928a95709d90cf6b2f0705b16c07fca05b;d497613415c71d3ab544f3531871173749b31226;ce8883dbe8c695d8ef770eefe34e48efe7c0cf64;c1347daf84d6750a4fcb16cd04a1124ea3e7b775;100c788dea25ed53de43b1d5505afd19f4d85e30;8406c229176f56903f7de8e913081326341dcda9;89a05047a487ba4e9c32687ceb7fc00f59a834f2;5e52ecd0f0ab96bcb0b362e4ea4f0207ec996e66;292889c24c0fa339af5927bb4845b25b89a5e57a;86bbc82e2891d5baf900e40668f897e919d9ed87;332da8685270e4d39bdcededc7a00d1631c645d4;efa69b21767d15897e00f11bdf4625a8dcf1c1c4;febae7da65fbd5ec7d1227cf82e82cfbe48532dd;319b9413a46218c2832d6f3571880ac0d55adfbe;ba01ca8a463e294fc6a36473f6391095cbb164f2;e32c87769fef343508bd9bb2f7ee083417354245;dbc8987f77aaecdb058aac10c175124b06b1dbaf;cffb520ae667ed8737bbe3f5a05d86e8f85b9dba;0c646a8923d70ae9a93ee8945c05e47246707d80;cc032af2d72406bff21b34a473fdc3588c2fe920;770429367eb26731adc1abc31ba3693f7126e3a5",
         "Automatic foveation for video compression using a neurobiological model of visual attention;Text information extraction in images and video: a survey;DAVE: a system for quality driven adaptive video delivery;Video transmission adaptation on mobile devices;Visual attention for region of interest coding in JPEG 2000;Semantic segmentation and description for video transcoding;Resource adaptation based on MPEG-21 usage environment descriptions;Video transcoding architectures and techniques: an overview;Metadata-driven multimedia access;Semantic transcoding for live video server;Multiple video object tracking in complex scenes;Complexity-quality analysis of transcoding architectures for reduced spatial resolution;Instantly indexed multimedia databases of real world events;Face detection in color images;MPEG-7 description of generic video objects for scene reconstruction;A new content-based hybrid video transcoding method;Video Processing and Communications;Media conversions to support mobile users;Object-based transcoding for adaptable video content delivery;Overview of fine granularity scalability in MPEG-4 video standard;Video object extraction based on adaptive background and statistical change detection;Video transcoding for universal multimedia access;Heterogeneous Video Transcoding to Lower Spatio-Temporal Resolutions and Different Encoding Formats;Scalable multimedia delivery for pervasive computing;Adapting Multimedia Internet Content for Universal Access;A frequency-domain video transcoder for dynamic bit-rate reduction of MPEG-2 bit streams;Multimedia content description in the InfoPyramid;Objective methods for assessment of video quality : state of the art;On the methods and performances of rational downsizing video transcoding;Shadow detection in image sequences;Video Transcoding Architectures and Techniques : An Overview;Visual radio: should we paint pictures with workds, or pictures?;Dave: A system for qual ity driven adaptive video delivery,”Proceedings of the 5th ACM SIGMM international workshop on Multimedia information retriev;Processing and;Information technology – coding of audio-visual objects – part 2 visual–amendment 2: Streaming video profiles;Statistics, 3 ed;Continuous speech recognit  io;Submitted by;Information technology — Generic coding of moving pictures and associated audio information — Part 2: Video;SUBJECTIVE VIDEO QUALITY ASSESSMENT METHODS FOR MULTIMEDIA APPLICATIONS"
        ],
        [
         "12",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "https://europepmc.org/articles/pmc1081291?pdf=render",
         "Medicine,Biology",
         "Journal of Clinical Microbiology",
         "S. Nunes,R. Sá-Leão,João André Carriço,C. Alves,R. Mato,A. B. Avô,J. Saldanha,J. Almeida,I. Sanches,H. de Lencastre",
         "3688641,1398826424,4908451,2053407517,12533308,10268213,40282052,116697293,2452668,2290923",
         "67",
         "2.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "13",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "https://wwwnc.cdc.gov/eid/article/11/8/pdfs/04-1197.pdf",
         "Medicine",
         "Emerging Infectious Diseases",
         "P. Yagupsky,E. Baron",
         "6618694,39055987",
         "185",
         "2.0",
         "2005-08-01",
         "2005",
         "25f20e694999a6840a06702d74cfd37d95d667f3;1ff74908befd25fee4b2ab003de4f83a41a41e45;66cbab4b36bb6b4952ea76981c60f27ab3275898;bb816e36ad186f5dfde74d71885070d766e2d747;e8d3010e9528076bca3e5fba37538edaa1635eb1;2ae2e7ee84a9137b010e909f76fc936ff775ad6e;af5b6fcc6f7f7a1b4c60f0d7b74ca15e67a14519;7cd9837410cc2ef6f967852c5ffa0cbe336dbdea;256f57d10a7a759d7e388a7cbbaa2fd54613a54b;e59cf90691ee9a5bd3aa34c70030e27b623ffd14;c39da3e000e116a2ebf7e6b115799a90c4d1923f;3fdffab87e6d38e294b3d76c2df8568fcdf8a0c6;0d18a3b83e52b92ee2a0fb372eaa4346feb551ca;a162170411238cb7a93e9c93040982aa25454c19;c41cac3f6e8765bef847ec9b406aaacffe0e4fa1;39eb0675b38ca75201203941d13ba0f4a186e283;594b085d66a958b96aba9d1933972d3b34bb7678;e40f0229ec9bbe688f6356438951dfbb36fc59cb;0ca7f50704a2e0ab7da2896839f913a571b2e777;df8e94c95ce7f4d08c04cb2d562200b1eb0ddc59;4e4469c2a2132167fae10e5133dbd033b50d7471;f8b32e9384596d0545c70b5623e957a3aa7f60f9;1e0d986827359e89c42fdc145007c65b89a15ab5;db1a7e39ea496679bee7a5c6073402b6fd17e8d2;647d1a66e8764bdca4767f352d54f97fcd5252ac;cf4b52becaed752bb036d66d6cba6c0d1785af1d;24fab9ab18c998c12f7e99c14f31ae41c822402f;dd97f7c782c7ff9d1a98d2454f07dba3fc9e2906;29debcd880647e9d5ec1e1033228f5ca91753a8c;a03a55d8331f010683a46d3f2a808c56459961ad;1f7571e5b811f06b77bd16c7f277ed9849cd69e6;31a5d17ec14a53090757e7042ca707f4b8f3a35a;736a26b7dbf151a54974d500bad5c08b0b723ebb;f64fee05bc23ffc94ffec0e1061461c398ddf905;60fedd708bd15ec43d3e4c891b4f389db9c5fecf;e397ef4308fcf8f9cdc71dd5a4e43bb4b5d20887;ee5e98347d31991dd2b088ae627039ae24f0b10f;362c818a28759e5fca9a7f0d4e92ed695951a5fb;a01715bd830311a59ceda880f94f11a860224873;2b233f3fe9887477b9333af2751d2476954bc901;558e59a9972530acb403739cc7ef0239143f5a8f;ed002af5819415ec0a8b31a69ce941f6e7d53f64",
         "Laboratory-acquired Brucellosis;Acute hepatitis due to brucellosis in a laboratory technician.;Prevention of Laboratory-Acquired Brucellosis;Analysis of risk factors for laboratory-acquired brucella infections.;Use of the Isolator 1.5 Microbial Tube for Detection of Brucella melitensis in Synovial Fluid;Bacterial Pathogens as Biological Weapons and Agents of Bioterrorism;Unusual Complication of Breast Implants: Brucella Infection;Brucellosis in laboratory workers at a Saudi Arabian hospital.;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Brucella abortus Infection Acquired in Microbiology Laboratories;Posttreatment Follow-Up of Brucellosis by PCR Assay;Detection of Brucellae in Blood Cultures;Rapid diagnosis of human brucellosis by peripheral-blood PCR assay;Failure of short antimicrobial treatments for human brucellosis;Comparison of BACTEC 9240 Peds Plus medium and isolator 1.5 microbial tube for detection of Brucella melitensis from blood cultures;Exposure to Brucella in the laboratory;Brucellosis from sniffing bacteriological cultures;Grand Rounds—Hammersmith Hospital: A case of laboratory acquired brucellosis;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Live Brucella spp. fail to induce tumor necrosis factor alpha excretion upon infection of U937-derived phagocytes;Outbreak of Brucella melitensis among microbiology laboratory workers;Unsuspected brucellosis diagnosed in a child as a result of an outbreak of laboratory-acquired brucellosis.;Brucellosis: imported and laboratory-acquired cases, and an overview of treatment trials.;Biochemical mis-identification of Brucella melitensis and subsequent laboratory-acquired infections.;Laboratory diagnosis of brucella infection: some pitfalls;Outbreak of Brucella melitensis among microbiology laboratory workers in a community hospital.;An outbreak of Brucella melitensis infection by airborne transmission among laboratory workers.;Laboratory-acquired meningitis caused by Brucella abortus strain 19.;Past and present hazards of working with infectious agents.;From the Center for Disease Control;Brucella canis: A cause of undulant fever;An unusual outbreak of brucellosis.;Brucellosis: I. Laboratory-Acquired Acute Infection;Brucellosis in the United States.;Brucella species;Brucellosis: An occupational hazard for medical laboratory personnel. Report of five cases;Beer-Sheva 84101, Israel, fax: 972-8640-3541;Prevention of laboratory-acquired brucellosis.;(revised). Geneva: World Health Organization;Basic protocols for level A laboratories for the presumptive identification of Brucella species. Washington: American Society for Microbiology;Exposure of hospital personnel to Brucella melitensis and occurrence of laboratory-acquired disease in an endemic area.;Biosafety in microbiological and biomedical laboratories;Laboratory-associated infections: incidence, fatalities, causes, and prevention.;Principles and practice of infectious diseases. Vols 1 and 2.;Brucellosis in the United States, 1965-1974.;Laboratory Infections Due to Brucella"
        ],
        [
         "14",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "https://hal.inria.fr/inria-00182046/file/2005-petti-fraichard-iros.pdf",
         "Engineering,Computer Science",
         "2005 IEEE/RSJ International Conference on Intelligent Robots and Systems",
         "S. Petti,Thierry Fraichard",
         "40180963,1694234",
         "287",
         "17.0",
         "2005-12-05",
         "2005",
         "3d89fe08f16bf8ecabb7bb26115b8baccbdd8712;64be486150521eddcf83ef356a0719d1ef65ceb8;0cfc462c907a790ecc56fba3ffeade9f1fbae757;3c0ced2de89291e1e94a68217a2d27f4b8ba1b64;9cec1297bfce7ddd63390670467f1af6ccae4165;e4ed63ae63e6a8634e115925284f8c644b943623;9dddc932f553f31c14fdf941abe4f8961eef34f8;1281372e4b79bdb0d58a3d1dd2a813d23b2490c0;b0501adbca9917c19ae7b343e74b6e163753a227;62113b0fcb182853446679430aaefa9c367fe438;dc6e901b8c62d90c2d77877488b51c46f8bdf8a8;60f389d8def6c93aab17faf369f0727edbd51f47;afcd60559f9bc54bc2e0fd063fc0eb2bef87482e;9ba11955f7d8c93163288c33f9c3ed4cffc89a31;3f77f8b0e1a3e3dcedb0953a42018b6948b60617;ccfb06935ea9c8832ec6086000f71cf584a79679;7e81dac6260c4768af3a28ac21c78c5a38a5f7d0;2078d9a14df37d0992283c03cf42f591d71aeb3b;4d95e5c7c3b7d8a2a8a273e69a5b4fb98557b912;90c8e1ff97ffede807ce0e4bd20b984dd96725d9;ba40bb0b5d17afdab9adfcad9648a184b0a6c7f9;e8d60f2e1d51566953256020c71fee569865c948;09ddcaaf43aa3115ffa86aeb106604c9233717d5;6260d36c5fcb7fefb997ede20c2ba983adfa2470",
         "Motion prediction for moving objects: a statistical approach;Inevitable collision states. A step towards safer robots?;Towards real-time global motion planning in a dynamic environment using the NLVO concept;An integrated approach to goal-directed obstacle avoidance under dynamic constraints for dynamic environments;Real-time randomized path planning for robot navigation;Improved fast replanning for robot navigation in unknown terrain;Learning motion patterns of persons for mobile service robots;Reactive navigation for non-holonomic robots using the ego-kinematic space;Randomized Kinodynamic Motion Planning with Moving Obstacles;Real-time motion planning for agile autonomous vehicles;Kinodynamic motion planning amidst moving obstacles;VFH/sup */: local obstacle avoidance with look-ahead verification;Randomized Kinodynamic Planning;High-speed navigation using the global dynamic window approach;Motion Planning in Dynamic Environments Using Velocity Obstacles;The dynamic window approach to collision avoidance;Probabilistic roadmaps for path planning in high-dimensional configuration spaces;The curvature-velocity method for local obstacle avoidance;The Focussed D* Algorithm for Real-Time Replanning;Reflexive collision avoidance: a generalized approach;Motion planning in the presence of moving obstacles;Robot Motion Planning;Sensor-Based Motion Control for Mobile Robots;Kinodynamic planning in a structured and time-varying 2-D workspace"
        ],
        [
         "15",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2004.04487.x",
         "Biology,Medicine",
         "The FEBS Journal",
         "T. Utsumi,H. Ohta,Y. Kayano,N. Sakurai,Y. Ozoe",
         "6319788,122141924,49256899,48638495,6146172",
         "22",
         "0.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "16",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "https://europepmc.org/articles/pmc1134930?pdf=render",
         "Medicine,Biology",
         "Biochemical Journal",
         "S. Mousavi,Mitsuru Sato,Marita Sporstøl,B. Smedsrød,T. Berg,N. Kojima,H. Senoo",
         "145278181,47702160,1402638735,1432387861,145476071,50662160,4710114",
         "37",
         "1.0",
         "2005-04-01",
         "2005",
         "a6d3bbeb84717deccff079bd319413fc54b50e16;100f45a6b0045379892054aa70a738df681c6e97;85f7f66dc1be37706f712d6616387d33bb209a44;f86dc51f49b3f3d30e8e008c327f8b5402cddaf5;00be5eb58ac778cc624178ac010099a0601b7296;7fa779abd761c9306c433a9e87c1d11f8df19c1b;eb97c3f3cf1881a3908a6908f5e5a37e6d9c2145;cd648ef152b1f5a5d2b0f4645192b1c9b0ef17ec;2120667e80c26b7240ccac884467e869f6ce4c5f;3ea0c614b0b3edc4cb61b22054bb4f35e2bbeb5c;1c54f9a2d4aed39b8ec37e71cc63424a47526376;1ac7d0b397db7e0260c7bcf5ae4b4736f44f09a2;252991b9bbe9a0eb815051a797ae3a154600b74a;2dbfc38f53fc5d1c5441892b50d6cb0e5dfd170d;961e7f9ed82dbb973b0db57f7103e3c867d158ef;50c37eea83465e0ac7ebca071c3b75dc559f81d1;ae03a6754c9d89bdeca4474383cb078429c3830f;2303a910b94222ed2388c2206fa36060bdfeaf28;116139463e233dfbe4e8b71aa1a858b0b1145c22;c65bae3717eda5fa4a9cb63a6bf7c09040773c00;e0400ec8e27dd9036ad838755534df4bdfad87d9;2170efb311a61e99b0c7a960aa715a39d237aed1;c76fd3c2bb59cd99649849e51b66fbb8ee75cd36;c7b72147f381caa4b35b44ca665b144474030d9c;0fa5ea6e6e3b8020866ffbd9b3a3235e295a1432;eca866fb91011bad32af30aab5638562bea0275b;c9d968f1085827fd0af82e323ec482e0f1caa8a3;da432fa0f66c6a0fe29a8032ed229112c6679365;075cf9dd18564794e48c3d4736a84a9c4433562d;b0d137da67e4e7ecd9887ee57a7260f18d4710d9;bdad5d7454e41e03771edff7c7c86a83d512094c;8a7b2a7500846faa86f207bf2c90aff7c07691b0;8aa62d7c8426b02ddafb2df88390688a92690c1f;14253d459b1d6329ff175d332ece39d6ef03b9ed;7269b727ca35ac19da971ed902ed32dd7b8785f4;f999493daa293f2d24782a9120dcf3d1d880b0fa;6a4b503f0840c26dab7ce2f0012715b47d054152;9c9ba006ad1b6be24f473a4a1ad36681f7fbc744;fb76c9e322005d0334146cfb3b248d32f9837f93;1c610c79254816af5845d65e4e4ff90213f00267;2d4cea38f64c8dcc06d2d1cee84f90f55821c32f;c3aacfc7fc9e06c6960d46653260edbee87858fc;b8a05a8ab5e142cc0348cf5a31a16d0aed1568f8;c99120417ac2adc583085b340766dbaf52d071c2;bc0ebe8a73c8efbdb7d0016eec8c513dcdd0a748;634668d47753c48db112e9377445fc37f9eeca22;26270ce70c5c8e8d40201fa3dd9fa717c2029329;f0d468bd28cbaa9d08f27278bcf6b372a6d846f9;e2b117fe55064f148332db4e1b36f52a55f2b3b0;38017c1032349f35c0b1efeafaf5861a9aa4c718;3cf858a3295d7b6f4b5fc7b234945ab25b65a51e;c568ad1bc32539c8bb0b8809873e7ff9fe73396e;1db7bf5ff49493dcfc46c66ed29a4b663cd78295",
         "uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV.;Identification and characterization of the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor.;A targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen uptake;Liver extracellular matrix in health and disease;Hepatic stellate cells: unique characteristics in cell biology and phenotype.;uPARAP/Endo180 is essential for cellular uptake of collagen and promotes fibroblast collagen adhesion;Characterization of Sugar Binding by the Mannose Receptor Family Member, Endo180*;The mannose receptor family.;The C-type Lectin Receptor Endo180 Displays Internalization and Recycling Properties Distinct from Other Members of the Mannose Receptor Family*;Collagen-receptor signaling in health and disease.;The Urokinase Plasminogen Activator Receptor–Associated Protein/Endo180 Is Coexpressed with Its Interaction Partners Urokinase Plasminogen Activator Receptor and Matrix Metalloprotease-13 during Osteogenesis;Hepatic Blood Flow Regulation by Stellate Cells in Normal and Injured Liver;Extracellular Matrix Degradation and the Role of Hepatic Stellate Cells;Wortmannin-sensitive trafficking steps in the endocytic pathway in rat liver endothelial cells.;Effects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy.;Endo180, an endocytic recycling glycoprotein related to the macrophage mannose receptor is expressed on fibroblasts, endothelial cells and macrophages and functions as a lectin receptor.;Selective adhesion of macrophages to denatured forms of type I collagen is mediated by scavenger receptors.;A Urokinase Receptor-associated Protein with Specific Collagen Binding Properties*;The C‐type lectin superfamily in the immune system;An orphan receptor tyrosine kinase family whose members serve as nonintegrin collagen receptors.;The discoidin domain receptor tyrosine kinases are activated by collagen.;Vascular Smooth Muscle αvβ3 Integrin Mediates Arteriolar Vasodilation in Response to RGD Peptides;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Cell adhesion receptors for native and denatured type I collagens and fibronectin in rabbit arterial smooth muscle cells in culture.;An endothelial cell receptor for plasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of t-PA-dependent plasminogen activation.;Differential expression of matrix-metalloproteinase-1 and -2 genes in normal and fibrotic human liver.;Transfer of retinol-binding protein from HepG2 human hepatoma cells to cocultured rat stellate cells.;Fat-storing cells as liver-specific pericytes. Spatial dynamics of agonist-stimulated intracellular calcium transients.;Affinity of integrins for damaged extracellular matrix: αvβ3 binds to denatured collagen type I through RGD sites;Identification of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin in collagen.;Internalization of retinol-binding protein in parenchymal and stellate cells of rat liver.;p180, a novel recycling transmembrane glycoprotein with restricted cell type expression;The alpha 2 beta 1 integrin cell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen.;Activation of cultured rat hepatic lipocytes by Kupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation of cell proliferation via induction of platelet-derived growth factor receptors.;Characterization of desmin‐positive rat liver sinusoidal cells;Hepatic lipocytes: the principal collagen-producing cells of normal rat liver.;A radioiodinated, intracellularly trapped ligand for determining the sites of plasma protein degradation in vivo.;The effect of leupeptin on intracellular digestion of asialofetuin in rat hepatocytes.;The structures of the galactose-containing sugar chains of ovalbumin.;HIGH-YIELD PREPARATION OF ISOLATED RAT LIVER PARENCHYMAL CELLS;Distribution of Endo180 receptor and ligand in developing articular cartilage.;Endocytic uptake of advanced glycation end products by mouse liver sinusoidal endothelial cells is mediated by a scavenger receptor distinct from the macrophage scavenger receptor class A.;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-α and TGF-β1;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1.;Vascular smooth muscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD peptides.;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Affinity of integrins for damaged extracellular matrix: alpha v beta 3 binds to denatured collagen type I through RGD sites.;Expression of the gene of the α-smooth muscle-actin isoform in rat liver and in rat fat-storing (ITO) cells;Acidification of the endocytic and exocytic pathways.;Collagen production by rat liver fat-storing cells in primary culture.;Newly administered [3H]retinol is transferred from hepatocytes to stellate cells in liver for storage.;Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes), their related structure in and around the liver sinusoids, and vitamin A-storing cells in extrahepatic organs.;Preparation of isolated rat liver cells.;Hepatic stellate cells: morphology, function, and regulation;Isolation of perisinusoidal stellate cells (vitamin A-storting cells, fat-storting cells) of the rat liver"
        ],
        [
         "17",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "https://europepmc.org/articles/pmc538921?pdf=render",
         "Medicine,Biology",
         "Antimicrobial Agents and Chemotherapy",
         "L. Poirel,M. Lartigue,J. Decousser,P. Nordmann",
         "6593096,40361075,46491845,5228247",
         "233",
         "8.0",
         "2005-01-01",
         "2005",
         "",
         ""
        ],
        [
         "18",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "https://zenodo.org/record/1259441/files/article.pdf",
         "Biology,Medicine",
         "Virus Research",
         "I. Kuzmin,G. Hughes,A. Botvinkin,L. Orciari,C. Rupprecht",
         "31991394,145378585,4167987,4216439,1727205",
         "187",
         "15.0",
         "2005-07-01",
         "2005",
         "f9833b6bdeb64d7b59c9f7d9b344ce25d5c90dba;2b97169a29a168ab41c3ec1a78164b96a6969a41;7a98ad002f81a2f7aab8d8491cadbc88aa8c3155;d1b3e4ff02d07be96c06fbfaf555406f17196741;33be5abf0ecb7dfdc6402e68dc86a1126c13adc8;1a9bc3e0a07a9d95d4f35323ab0f1c8c1413ef76;4001a31540fceaa58f5f393584a71522e1e31e4c;6b4c6ff3535d650d61c57ab38337d12b62314d01;7bcecb53c6e78442750e3e9b72ba3a80e769f70f;511a070c6dfd315dc7c4ae0e59c07957737618d8;dd06144588032875303e5ad4531de249890411a2;51a951e0338efd7eac40cb05a8d8bb1f9c0b2dc9;3e25fda5c88fe7969682f3203a9707ef42f0e551;6bb3b70653cf04a5b2c68240e740df7412f30ea2;725a3cbac5e12ee0dcd1dbd036929dc6d7783b58;ebb01f73c93b2d963c3dd09e607d85ac2921bd08;283f8d4a6b328ecf87aa6896307fa92a4159e282;a4bd32fbe148185f7f96bb6a8f057a8bee258e55;7696a27bd12a890797d339f730585d240ed59c1f;f63ea68c269895152c1aff73e5b1927b6fb32949;ad56575dae7f33f0a62df28b1bdc1c8344e4352a;8d54df6996acbb4c6758544b046278166afa4a9a;b0caf52d27797f3f413444bb15f5d6692b6375c9;cd2628e16a0c5978ccd8ab019308853ec712cc4b;aedc61581cafb2a48d3a0095d8694570f6d1e5d0;93260ef642602445acfac1a3130c86e85873cc38;2fa65c170ab7e679cbc0eecc5b03d3e25dac6556;5d8c958b183dad733d0e1003b74569b79a349478;1dfb94db76de9db1b86b2bd896a1991cdaf33c66;37815c1e6cce8b572a44892b360e619b10bfd28f;321bbf45869d401205db0123dd09aa5ddb78324b;f3b9fe7f78f8dd0673a811fec3b3741d652dcd8e;36f2bc416239289425d64110703dd037196571da;f1f369b6f62a2374dc6cba39c6ca58aa2544b016;1de7a444fac66d2c1f3a318d415714d524940c9d;e63bdf8efd8a01e5b53deee54a5837c8538b7cfe;970725ff0887cbdbf57e820588e4d7bb047383bf;c0f8a73ab829481eb8f97763d38a4d8eedaaf1d5;47c403f4295ab506971e1581475501908ac30e4e;385852642f3ada1e3dbf8ed90c005263a0be280a;dd20fc8f997d821dee710fe01a27049c12b5dfaa;54dfa2d112efd63b772c8c70795cd9cd7d3fc8dd;5685328fbbbf401d8b5b77924c616a825a56ea91;ba000c39195024b6f16e4eda48468db5be1506c0;f3ac4157691f496505db5c7b808b079f0e9d2363;682a5700489eec2f9c1f4f6246b73e5bcdb5fa97;f8d1d71b86d771cebd9061fba6e8459db171bbfc;096a93c9e0841bd3f99bfb486b170ed2c1b5d434;762cb1a4d0b79d9e9287cea102930294c87bf3ab;19b2eaa20f170259f94c5a84e1ff1952237bf92e;53bd1d94388d92e50aee35e9f5eb43a501b2a5e9;029463508fe0009869be5a3736075ab528d63664;adfd3c0cf96e20f5714a2deb0bb69772182b80da;2507c1996314911d9e998281f95f2a597358f74b;3842589a2100702965d4123501ff19858dc62cee;ec6f2977cb6c80aa845eafc14d46417a92ec3a00;17eab3eab209ab9b87425de3b51540b5efd9dd9e;b1d9cae15b280af1d4bc9001f066b757df14f5c4;e9038f1b41d390f5e6eeb9d8da000a94eec21e86;d3f4ba36f4de1d317078f90aa82b7e1526b5fb94;6993234eb3aba662d7d19b2b2c9b2f6f7304f830;a9c8b15829024098a9825321727fcd9e0d12c57d;0ae262d9cf78536754bc064e07113ab5e978f208;73679f1ba00de9e73492f3f5d53bdcef50af3fb4;85e77924e184aa659599cbc1675ac66142a3683c;924c6ec5ea4524ab6aa2e8f6e66bcdbcc4a7ba2b;94f848c5d0621d840bb13305ae945100c3551e77;cb592f5cbc4a9b871a79d9256404617ef6e5892f;2d10e6a2e1ffcb5608fa78ea47bab7927e53994a;0b8eb3ae0d8c670378d55f04ba69fa2f26ae2d31",
         "Novel Lyssaviruses Isolated from Bats in Russia;Bat lyssaviruses (Aravan and Khujand) from Central Asia: phylogenetic relationships according to N, P and G gene sequences.;A comparison of DNA vaccines for the rabies-related virus, Mokola.;New Lyssavirus Genotype from the Lesser Mouse-eared Bat (Myotis blythi), Kyrghyzstan;A molecular epidemiological study of Australian bat lyssavirus.;Competition Commission inquiry into POMs.;Characterisation of an Australian bat lyssavirus variant isolated from an insectivorous bat.;Phylogenetic comparison of the genus Lyssavirus using distal coding sequences of the glycoprotein and nucleoprotein genes;Lyssavirus P gene characterisation provides insights into the phylogeny of the genus and identifies structural similarities and diversity within the encoded phosphoprotein.;Increased positive selection pressure in persistent (SSPE) versus acute measles virus infections.;European Bat Lyssavirus Infection in Spanish Bat Populations;Serologic Evidence of Lyssavirus Infections among Bats, the Philippines;Genetic constraints and the adaptive evolution of rabies virus in nature.;Expression of the B-cell and T-cell epitopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG and induction of an humoral response in mice.;MEGA2: molecular evolutionary genetics analysis software;Extensive Attenuation of Rabies Virus by Simultaneously Modifying the Dynein Light Chain Binding Site in the P Protein and Replacing Arg333 in the G Protein;Molecular basis for the interaction between rabies virus phosphoprotein P and the dynein light chain LC8: dissociation of dynein-binding properties and transcriptional functionality of P.;Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains;Host Switching in Lyssavirus History from the Chiroptera to the Carnivora Orders;Evidence of Two Lyssavirus Phylogroups with Distinct Pathogenicity and Immunogenicity;Antigenic and genetic divergence of rabies viruses from bat species indigenous to Canada.;Codon-substitution models for heterogeneous selection pressure at amino acid sites.;Sequencing and position analysis of the glycoprotein gene of four Chinese rabies viruses;Ecology and evolution of rabies virus in Europe.;Phylogeographic patterns exhibited by Ontario rabies virus variants;Chimeric Lyssavirus Glycoproteins with Increased Immunological Potential;Multiple sequence alignment with Clustal X.;Rabies virus quasispecies: implications for pathogenesis.;DNA-based immunization for exploring the enlargement of immunological cross-reactivity against the lyssaviruses.;PAML: a program package for phylogenetic analysis by maximum likelihood;Evolution of European bat lyssaviruses.;The complete Mokola virus genome sequence: structure of the RNA-dependent RNA polymerase.;A Biographical History of Veterinary Pathology;Ten-year survey of British bats for the existence of rabies;Encephalitis caused by a Lyssavirus in fruit bats in Australia.;Characterization of a unique variant of bat rabies virus responsible for newly emerging human cases in North America.;The design of strain-specific polymerase chain reactions for discrimination of the racoon rabies virus strain from indigenous rabies viruses of Ontario.;Genetic polymorphism in the rabies virus nucleoprotein gene.;Molecular epidemiology of lyssaviruses: focus on the glycoprotein and pseudogenes.;Molecular diversity of the Lyssavirus genus.;Isolation and characterization of 115 street rabies virus isolates from Ethiopia by using monoclonal antibodies: identification of 2 isolates as Mokola and Lagos bat viruses.;[A lyssavirus with an unusual antigenic structure isolated from a bat in southern Kyrgyzstan].;Rapid sequence evolution of street rabies glycoprotein is related to the highly heterogeneous nature of the viral population.;PCR technique as an alternative method for diagnosis and molecular epidemiology of rabies virus.;T helper cell epitope of rabies virus nucleoprotein defined by tri‐ and tetrapeptides;Molecular cloning and complete nucleotide sequence of the attenuated rabies virus SAD B19.;Rabies-related Yuli virus, identification with a panel of monoclonal antibodies.;Efficacy of rabies vaccines against Duvenhage virus isolated from European house bats (Eptesicus serotinus), classic rabies and rabies-related viruses.;Immunity against the European bat rabies (Duvenhage) virus induced by rabies vaccines: an experimental study in mice.;Localization and immunological characterization of antigenic domains of the rabies virus internal N and NS proteins.;Walking along the rabies genome: is the large G-L intergenic region a remnant gene?;HUMAN RABIES OF BAT ORIGIN IN EUROPE;Rabies in the Tropics;Rabies-related viruses.;Rabies virus infection of a flying fox bat, Pteropus policephalus in Chandigarh, Northern India.;Giulio Alfredo Maccacaro.;Experimental infection of dogs and monkeys with two rabies serogroup viruses, Lagos bat and Mokola (IbAn 27377): clinical, serologic, virologic, and fluorescent-antibody studies.;An unusual case of human rabies thought to be of chiropteran origin.;Two African Viruses Serologically and Morphologically Related to Rabies Virus;Isolation of Rabies Virus from Fruit Bats in Thailand;Isolation of a virus from Nigerian fruit bats.;New Cases of Mokola Virus Infection in South Africa: A Genotypic Comparison of Southern African Virus Isolates;BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT;MODELTEST: testing the model of DNA substitution;Immunoprotection by rabies virus nucleoprotein.;Nucleotide and deduced amino acid sequences of the nominal nonstructural phosphoprotein of the ERA, PM and CVS-11 strains of rabies virus.;Rabies and Rabies-related viruses in Zimbabwe historical, virological and ecological aspects.;Rabies-Related Viruses;Characterization of an antigenic determinant of the glycoprotein that correlates with pathogenicity of rabies virus.;Antigenic characterization of the bangui strain (ANCB-672d) of lagos bat virus"
        ],
        [
         "19",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         "https://www.jneurosci.org/content/jneuro/25/48/11071.full.pdf",
         "Psychology,Medicine",
         "Journal of Neuroscience",
         "Y. Sugase-Miyamoto,B. Richmond",
         "1400827629,3007646",
         "110",
         "5.0",
         "2005-11-30",
         "2005",
         "",
         ""
        ],
        [
         "20",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "https://europepmc.org/articles/pmc1323209?pdf=render",
         "Biology,Medicine",
         "Proceedings of the National Academy of Sciences of the United States of America",
         "Huiling He,K. Jażdżewski,Wei Li,S. Liyanarachchi,R. Nagy,S. Volinia,G. Calin,Chang-gong Liu,K. Franssila,S. Suster,R. Kloos,C. Croce,A. de la Chapelle",
         "2328535,3744462,2157336158,2934528,32344884,1906416,2124909,16073087,4761141,3162753,3481071,144749275,3597724",
         "1251",
         "59.0",
         "2005-12-27",
         "2005",
         "5631e50fd39d9c7a54dec2bb5e5a016f8402c8b5;10bae8b93245299d0e4c4079e76f4d8c10458709;8de8119e007331a5b9e11c3ef99e15b2c7c8edce;e181f1874c85c6d42bf794f4e8aef5be2f77306a;c7497d6d32d3dd6f893609a17848c807134e1cc4;d8138beef46811bcb2ec68a4ae029b274ee4e1d1;77ef9aac7e0199184cb3169ac5e4de1d5fd7152b;94e90f0d2dba39b2357283e78f1a2dcb3f3ec358;240aa3fbe87b1fb085815baa87cca8bd04f52f70;631cf40b915e9f9b176d7a9be8a22c66a27a1acc;aab9e4d3502a20eb48d22ae8bb6ce29aabfe6a0a;01ae7d930255dd36a3308bd912d29d79859934db;62278466704924429db9b60bebc4ce8b4b7f2521;f60f9040c85eb47659079adf779a31bdb58014eb;c87c7646aa4273e73cdb1f2a7a8804281b478b4e;c8bb4cfcb04928f5312c66cc0239cca93839ff8b;72136fce3ef7aa30bf7ce640f44715daafe93c03;eab6ab732d7951e804769646a42a261665ed7297;8c1f06689340bbbd5f56d90511a338ebc958c517;2459b8e56f02c9d9c3b19ae8fcd75fedf05391d5;d577f656be9a1aa6d951dfd8cb02867d7194f4a8;3ed944a9b0fcd2a77298049f092a3b97b4633bc7;cde7b10619a54868cfbdacf99d051bc631bb86d7;97c4ead52f67e84d6ab55bb4f73a192533a95e1a;0442c89a09bfd5ba7f09340d0b23143f6c3f32b7;c81a5f7469419748daff6272230ed330001ee8f2;06cc16cdaf46fa657bb001faec28021b8ab5a3a1;5be20043c72560471655eba6f2a23329f5ea5f5e;b8f81ba06d314b8b112298c5b600557108c45872;92dcca877b55b85ceaa139bc5a6d4a563d598971;8259057fd4d7fb899894ff37c3fa9c2d1cca8798;726fa5af9a99f441cdddb2f86ffec6d6c968a6d5;80ce85acd05007231bbb58f41a4ee746ff5a6bd6;a96a1fee948a06d083e9d7bf57480666bd1653f3;4dac942817175d54e581803c90d20209cb0752b2;df292306c38ea75a87b649b3bd1833cdde89b128;f73bf5bf32c20821b00c854e60b21cff6f6a8e56;f5779281d7d71ae5a9d2f6a7ddc57699293d28aa;7a50462b065cdfddb9f96730fa68706174005e33;ab539ed05d3fc88082d19e3c5829da1d167ca066;234fd0f30e731e7d96e713d041c84fdc3bea8299;09b60a85059b98b1f3df8646742cf086edc512ea;b0d11c4e598c4c683cecac6640fae6e5330ce1ed;1bab97c3a3e9dc5c9f6545753bbd6fc7b58f6e68;bbefe294e85cb153d4bbf417b24b4073e0ec6999;cb9627792a808bf3921d00b62173fba1a2d2dfd3",
         "MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation.;MicroRNA gene expression deregulation in human breast cancer.;MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells.;miRNAs, Cancer, and Stem Cell Division;MicroRNA expression profiles classify human cancers;Cancer genomics: Small RNAs with big impacts;A microRNA polycistron as a potential human oncogene;Small regulatory RNAs in mammals.;Combinatorial microRNA target predictions;The RET/PTC-RAS-BRAF linear signaling cascade mediates the motile and mitogenic phenotype of thyroid cancer cells.;MicroRNAs: a developing story.;RAS Is Regulated by the let-7 MicroRNA Family;Accumulation of miR-155 and BIC RNA in human B cell lymphomas.;A new mechanism of BRAF activation in human thyroid papillary carcinomas.;Detection of Overexpressed and Phosphorylated Wild-Type Kit Receptor in Surgical Specimens of Small Cell Lung Cancer;Kit as a human oncogenic tyrosine kinase.;Oncogenic protein tyrosine kinases;Human MicroRNA Targets;Fast and effective prediction of microRNA/target duplexes.;Genetic analysis of genome-wide variation in human gene expression;MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias.;Annual report to the nation on the status of cancer, 1975–2001, with a special feature regarding survival;An oligonucleotide microchip for genome-wide microRNA profiling in human and mouse tissues.;Reduced Expression of the let-7 MicroRNAs in Human Lung Cancers in Association with Shortened Postoperative Survival;MicroRNAs Genomics, Biogenesis, Mechanism, and Function;Prediction of Mammalian MicroRNA Targets;BRAF mutations in thyroid tumors are restricted to papillary carcinomas and anaplastic or poorly differentiated carcinomas arising from papillary carcinomas.;Reduced accumulation of specific microRNAs in colorectal neoplasia.;Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma;BRAF mutation in papillary thyroid carcinoma.;High prevalence of BRAF mutations in thyroid cancer: genetic evidence for constitutive activation of the RET/PTC-RAS-BRAF signaling pathway in papillary thyroid carcinoma.;Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia;Environmental and heritable causes of cancer among 9.6 million individuals in the Swedish family‐cancer database;Gene expression in papillary thyroid carcinoma reveals highly consistent profiles;Localization of a susceptibility gene for familial nonmedullary thyroid carcinoma to chromosome 2q21.;Papillary thyroid carcinoma associated with papillary renal neoplasia: genetic linkage analysis of a distinct heritable tumor syndrome.;The biology of stem cell factor and its receptor C-kit.;A gene predisposing to familial thyroid tumors with cell oxyphilia maps to chromosome 19p13.2.;Paraffin section detection of the c-kit gene product (CD117) in human tissues: value in the diagnosis of mast cell disorders.;c-Kit proto-oncogene is more likely to lose expression in differentiated thyroid carcinoma than three thyroid-specific genes: thyroid peroxidase, thyroglobulin, and thyroid stimulating hormone receptor.;Transformation of thyroid epithelium is associated with loss of c-kit receptor.;Systematic population-based assessment of cancer risk in first-degree relatives of cancer probands.;Expression of c-kit receptor in normal and transformed human nonlymphoid tissues.;RNA silencing.;M  ED  IC  A  L  SC  IE  N  CE;Loss of heterozygosity in papillary and follicular thyroid carcinoma: a mini review.;de la Chapelle, A.;Age at diagnosis Gender Race Diagnosis"
        ],
        [
         "21",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/cncr.20947",
         "Biology,Medicine",
         "Cancer",
         "R. Kim",
         "2592641",
         "315",
         "15.0",
         "2005-04-15",
         "2005",
         "376109aaa3cbc518162072d892982ee7d7c61716;ce3c0b44cd2216676a66ed0c658d2bc298883f47;ef47428ff7b435d697f530965245854043545ab3;aeeddeb465b14955c0c207ae5d8facd8d4e01ce8;f172956a2b5badf538115971e667d34bebd87d22;9e4b1bab7acb0526340bd10134c672d3db8d445d;cdf7719d569d5b5ee447d329a25bbf19fdcc37c3;1c624d6499bd865abc6f8e400b24c8b79f6b620f;529a77664ace0111bcc3e9220770f8897ef023e5;c548c49f6baee6772d6e9fe3cbb9dde108a707b3;bf5f409029a90feff7df4cacdbd467d8d1eb00d8;6d6a4eef1ac55af590484c713f2f576c85c78d98;f357979269b1757c92d0c092be99730778c09d24;b3a2c655c5e01b71f590b2151cff8f00a02e1226;c45854aadb1d8e2e411e87a467f9f065f4866966;92f639e758c61f702a9c00cbea9be0976b20d894;53a50ae0268994aed5cf952d6c99043de7e35ebd;d37d31faa3a00280d21f5e1a648f98b66c8e3432;11b3fec8df3f80b2bec1488a6f0df6c4d8441c71;799ab82ec856050d460526143b7d0718b96e14cb;b3fae1ae6bc52cfe558a594ba83f5dc8a700f43c;51525a9c8be7e63f959ac2bfa3158f9db5b10d7c;b5672e0714e39e7bda99ea73b0bcebfdc6b05571;6da399d8f4d3460dde33aa226c27b631d47734ba;e83e6c40a82928c05944bc56ab2ae4a7d8ccf302;05d42e39ca41624d718474dffb3bcdfd01359b4e;a675c38d04171783d9a60886cc54ece593194dfe;bd437570db7af2cab4d48a6ce8301493ced7d301;045bc283c609699990dff04aa9b4514027dfceca;ad8cba15117d0a7073e3638b8a22daaac3588483;0bb77a1432b8d5752838df435480e44e3c50fff0;2ba0fec7e8cf702393a1382ca856855a7837556f;0291f1c9ff0da520e91419ae2db08999ee399725;e54cab9b65df38aa1f0a9519923c159b51fb68b6;5df427a0bafeb5240b6f577879c0de8e63ee7417;bd0da4411086bad78115eaf6dd0b45004209ff44;a2b0a9954056c4b757c828e6b16144251a158da1;7b6d94906866ab52104a903207582f2436ea697d;4fe6cc56b37705ac433ce4cd16853d5fe9fceeac;3c49e9197ceb2728b27e913342fc321637cf6238;3c8a2cf8e6c03282842828cd36f189d835c21f6e;fd0960e7ef910dc32ca134e477f05dc9818f6351;1b7d30f14e6b49cc7f21c2392bd968bf0cfe625f;382eec2dc4340b1331798554b7a49218ccc478e3;e806d85d49e988ee33f5e36bffd2a984449cda39;1b4b00fb7e6f96fe6e94b2d2e274ee431d4796f3;27930771ff138d36d235401e67bc181c075ab02f;43325ff8d344de359f0f6ccc23ba171a7cc26748;65e6570ac8da2eab3e549c8d0fa921e6b56a442c;803b1672ed76dc13f50b020abd30e76a010be24b;75f6318d79eb3ef37dc5e6ef5069864829e4e1d0;8b3b03b3843bc2489aea05e5bd575719abdc31c3;367f3536f0b382b32d38034fcf9b2bc30304d6c6;6a0effba2107618e090d29a519bc8cc0d95574bc;2368dde26aa4ed09380222cfc8d8081de22ab899;4feb355910a6276d6bb1a790d1b76315bae25072;55617d0d12a80e300ee19353cb9234189d563685;6b5e978b6ee2b44ed9651d1a391eff867f9582b4;4af3826564c80d0e09c2c58b4919a6839dd544bc;c8c7d9d1cdefc34132dd7af4cf41042bc8142c77;9bb678d996f92800823bbb881c51c7ae5d1d6fe7;f50232e9fd7cb6b62e70966fc65676393a2ff40e;6a064731895d6c8b46146a46dd99be1f8fba54dd;607f07d507527a3765bceb8483c69486e1ff86a6;d211b8513a89e4274e69a40062724877c389d783;3a40fdacf4cd8ece6443f92033e8f6539ce19720;83c29f099cb279c1bee03261845205562f74fa29;be92bf9b710be989ba332bef5013ae49be45fb60;c2f7cf5511d33da829564a022e41742fc1255477",
         "Neutralization of Smac/Diablo by Inhibitors of Apoptosis (IAPs);Aging as a catabolic malfunction.;Apoptosis, autophagy, and more.;Dynamic evolution of the adenine nucleotide translocase interactome during chemotherapy-induced apoptosis;Bax is an important determinant for radiation sensitivity in esophageal carcinoma cells.;Apollon ubiquitinates SMAC and caspase-9, and has an essential cytoprotection function;A Phase I Pharmacokinetic and Biological Correlative Study of Oblimersen Sodium (Genasense, G3139), an Antisense Oligonucleotide to the Bcl-2 mRNA, and of Docetaxel in Patients with Hormone-Refractory Prostate Cancer;DAP-kinase as a target for drug design in cancer and diseases associated with accelerated cell death.;Endogenously released Smac is insufficient to mediate cell death of human lung carcinoma in response to etoposide.;Smac induces cytochrome c release and apoptosis independently from Bax/Bcl-xL in a strictly caspase-3-dependent manner in human carcinoma cells;Nuclear and mitochondrial conversations in cell death: PARP-1 and AIF signaling.;Ceramide-mediated Macroautophagy Involves Inhibition of Protein Kinase B and Up-regulation of Beclin 1*;JNK promotes Bax translocation to mitochondria through phosphorylation of 14‐3‐3 proteins;Multiple cell death pathways as regulators of tumour initiation and progression;Apoptosis defects and chemotherapy resistance: molecular interaction maps and networks;Changes in Therapy for Solid Tumors: Potential for Overcoming Drug Resistance In Vivo with Molecular Targeting Agents;Molecular therapy for peritoneal dissemination of xenotransplanted human MKN-45 gastric cancer cells with adenovirus mediated Bax gene transfer;Mitochondrial protease Omi/HtrA2 enhances caspase activation through multiple pathways;Apoptosis pathways in cancer and cancer therapy;In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker.;Endonuclease G is required for early embryogenesis and normal apoptosis in mice;Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor;Phosphoinositide 3-kinase accelerates autophagic cell death during glucose deprivation in the rat cardiomyocyte-derived cell line H9c2;Endostatin induces autophagic cell death in EAhy926 human endothelial cells.;HSpin1, a transmembrane protein interacting with Bcl-2/Bcl-xL, induces a caspase-independent autophagic cell death;Direct Interaction between Survivin and Smac/DIABLO Is Essential for the Anti-apoptotic Activity of Survivin during Taxol-induced Apoptosis*;Omi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP) irreversibly inactivates IAPs and facilitates caspase activity in apoptosis.;Transformation-selective apoptotic program triggered by farnesyltransferase inhibitors requires Bin1;The CD95 type I/type II model.;Amino Acids Interfere with the ERK1/2-dependent Control of Macroautophagy by Controlling the Activation of Raf-1 in Human Colon Cancer HT-29 Cells*;Induction of autophagic cell death in malignant glioma cells by arsenic trioxide.;Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs.;Chemotherapy enhances TNF-related apoptosis-inducing ligand DISC assembly in HT29 human colon cancer cells;Molecular response of human glioblastoma multiforme cells to ionizing radiation: cell cycle arrest, modulation of the expression of cyclin-dependent kinase inhibitors, and autophagy.;Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?;Current status of the molecular mechanisms of anticancer drug-induced apoptosis;Hypoxia and Nitric Oxide Treatment Confer Tolerance to Glucose Starvation in a 5′-AMP-activated Protein Kinase-dependent Manner*;Mediation of Poly(ADP-Ribose) Polymerase-1-Dependent Cell Death by Apoptosis-Inducing Factor;DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death;Bcl-2 and Bcl-xL Inhibit CD95-mediated Apoptosis by Preventing Mitochondrial Release of Smac/DIABLO and Subsequent Inactivation of X-linked Inhibitor-of-Apoptosis Protein*;Increased Ras expression and caspase-independent neuroblastoma cell death: possible mechanism of spontaneous neuroblastoma regression.;Paclitaxel triggers cell death primarily via caspase-independent routes in the non-small cell lung cancer cell line NCI-H460.;The Apaf-1 apoptosome: a large caspase-activating complex.;Apoptosis-inducing factor (AIF): a novel caspase-independent death effector released from mitochondria.;Identification of Omi/HtrA2 as a Mitochondrial Apoptotic Serine Protease That Disrupts Inhibitor of Apoptosis Protein-Caspase Interaction*;The Tumor Suppressor PTEN Positively Regulates Macroautophagy by Inhibiting the Phosphatidylinositol 3-Kinase/Protein Kinase B Pathway*;Autophagy delays sulindac sulfide-induced apoptosis in the human intestinal colon cancer cell line HT-29.;Bax translocation is crucial for the sensitivity of leukaemic cells to etoposide-induced apoptosis;The target of rapamycin (TOR) proteins;Proapoptotic BAX and BAK: A Requisite Gateway to Mitochondrial Dysfunction and Death;Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function.;PTEN controls tumor-induced angiogenesis;Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis;Bcl-2 down-regulation causes autophagy in a caspase-independent manner in human leukemic HL60 cells;Molecular Determinants of the Caspase-promoting Activity of Smac/DIABLO and Its Role in the Death Receptor Pathway*;BNIP3 and Genetic Control of Necrosis-Like Cell Death through the Mitochondrial Permeability Transition Pore;Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments.;Distinct Classes of Phosphatidylinositol 3′-Kinases Are Involved in Signaling Pathways That Control Macroautophagy in HT-29 Cells*;Induction of autophagy and inhibition of tumorigenesis by beclin 1;Autophagy Is Activated by Apoptotic Signalling in Sympathetic Neurons: An Alternative Mechanism of Death Execution;Cloning and genomic organization of beclin 1, a candidate tumor suppressor gene on chromosome 17q21.;Oncogenic Ras triggers cell suicide through the activation of a caspase-independent cell death program in human cancer cells;BIN1 is a novel MYC–interacting protein with features of a tumour suppressor;Ultrastructural and immunocytochemical characterization of autophagic vacuoles in isolated hepatocytes: effects of vinblastine and asparagine on vacuole distributions.;Studies on the mechanisms of autophagy: formation of the autophagic vacuole;Vinblastine-induced autophagic vacuoles in mouse liver and Ehrlich ascites tumor cells as assessed by freeze-fracture electron microscopy.;The morphology of various types of cell death in prenatal tissues.;Inducing cancer cell death by targeting transcription factors.;Mitochondrial dysfunction is an essential step for killing of non-small cell lung carcinomas resistant to conventional treatment"
        ],
        [
         "22",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2005.04623.x",
         "Biology,Medicine",
         "The FEBS Journal",
         "Ronja Tasler,T. Moises,N. Frankenberg‐Dinkel",
         "5801680,14847601,1399196272",
         "104",
         "6.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "23",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "http://www.jlr.org/article/S0022227520340323/pdf",
         "Biology,Medicine",
         "Journal of Lipid Research",
         "C. Villeneuve,M. Gilleron,I. Maridonneau-Parini,M. Daffé,C. Astarie-Dequeker,G. Etienne",
         "48980754,144505377,1397966565,5407662,1401604315,47231979",
         "107",
         "6.0",
         "2005-03-01",
         "2005",
         "",
         ""
        ],
        [
         "24",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D4334A1A0F9096B08FAE923BAB0CA36D/S0029665105000194a.pdf/div-class-title-insights-from-the-developing-world-thrifty-genotypes-and-thrifty-phenotypes-div.pdf",
         "Medicine,Biology",
         "Proceedings of the Nutrition Society",
         "A. Prentice,P. Rayco-Solon,S. Moore",
         "3674816,1401327164,48773910",
         "205",
         "7.0",
         "2005-05-01",
         "2005",
         "bc9ce03c2c4c174bd6c3b1d0be8900145edff1db;06acdfded5f54064fede109f07d9f36d8873834d;78d45a925322ae5987536246dd9a512b3d791ed7;c28997f8a141c0176ae7f5c06db2818ad40e220d;c57f0bf1bf34631a10d5f1d27191b88ed2f9b687;b3211005e2be14d595907435efc89659f76fa5c8;b02f1a164c91647b502be658d3c0e9e4c1f57f2c;27396abd2923770a9663d7ea65aa7a04ca6efc83;b23eef326baea71254d7c63656e97f638b9192ae;c137226bbdf92efc3da973dfd9e87a19dd042d68;eefefe28334b759b8b53e25caefca368a161dc98;2ae29359abae064e93f7d910e0c25aed49a88f4e;89c1180a10cd2d4f6b69b05faabcc6bb87929593;9ebf1a1f44e3658b2dd389b0526829653d05f0fd;de71e00c157f2a6252d6332c5a1e6063e6c4158f;f624515d506367c34202b1ccfb3a62566caf54b4;2f5d63fc5dd3fd93d6c8632642e9799cdb2dc667;8b025f19fc6c3a2565f2753b3d22219f0967e756;3b0189be73140b24df1fd28a3633c5debad6a958;4e4e8e7c72f63610dc20f83f07c02d220c438373;4a7f9ea4dd88c1d1793b79539dc887b93ee88c1c;a587a2b28dd5e8577fbd4d0a98dbf07506df40d6;d475e72c9d720f611b3728771f23082782877ebb;57784681819cecc1028626455fb79c57fe1030fd;95acbce1e13dea72bf4ecceef8070cd632c1905c;a1c3dfb03a261423e7f757718e61af931c1ac36d;0f4caae8c99e0d72292d71addb310524aec78c0d;a00002c79570f71907233f59273571ea44e67a27;7cfb934057f219e58da9c815e49c687cd5f3e305;c61d6adf94da35fa7b1ebd4332607ece26b12cff;0b69c9ba93b0e2d5d10c6b81cb65b179b0ce0e0e;00462a440fcfd0497a8769f2e3e94e40c50929c1;545f324e5db2615ddae40d7c685fd96ccb624f5d;e50748d4ac19be31f2f1ffe6591ef8b61a3bc136;864414f69552e3955b428c875a040785e2351d83;b7da62272a15b2eab222feeb806974255ffc6eb6;5c0dde78840d8c1c914f59179afc4c0993ce60fa;a37ef5d0f1eb4d9f2ee2c5e1ea49b3bceb1ca8eb;e946add8a70509c349ae15054c3c1052a981165d;5363b5010f319dc63abcdbfd1c868daebd725306;1ec845128a00e13b08eea726e8e8d784f844dfbd;795430c208a98348adb6fbbe7a962453d149c787;7454fa4f1eb0acfc56c67a288ac61bdebeb35970;8b6c6100fc489b4e951b9ee4f33f743d14b321e6;8246e8f6dbd713319cb843bf7e4990ad29df914e;f70b5135140b82250b8ccbcacde4973320f57b05;6ce812c3d9eb7a671e1a987517564a2b739870be;7d9c2c4768b5de3a533aa002b359449fabcdc151;b212cd8394f5fec259916490c9cf7668ed72160b;bc2e45abdfc013d33d5f1ba5c0934a8d4cc3574b;51c82b8dfc9ac7e33c7ecfaf023ddfe5c507de26;5522565260f1bd56280a2f5c50c5496b4ccad736;fb2b7baf210fb38b4a0501b2f5cd200abce4bb54;6a03fe8f413a6d83198f817eeaf8e1f27d9912fd;f83f5704a37c5776d21c4e950a45c27aaaecaac3;1036fd9c08f2f55be8974e20253875fd60ac16e0;323b16f39deeceae6fb5b285a9bd48c3e069a783;fc1819f465702c13883870784f9508e4054d00dc;935ec7c48c94bf948f1fbee7eb961255aa9eef24;c530eb353abee06e3b3cc5eea087d28aa0a800bb;6f38ee886f39daa656b1ac9ceb8a3c0785e100eb;d72154ce1a31048035a9ed91860acd6c895f735c;b5b8c35ff6c516a6106cfa930490e0779e8c43a4;d7d39f115e1b9b5482867486fcf2184576e8b72d;2a308549d0abed485e230c108a8bbe9338e2c926;90644eb817116ef816566ac87e3cc4b2804aaf26;94bdea56b64a2a3269c927af53a5f70a0abf389e;c49d0548da17d0e28c7d48fbfe7154cf5d523d7a;85d0d6037f14ad6bc592310d1f3350d87d424004;b79916ed1461af81a0559432924f559a13032c6f;9b47fdd07197bf012ba0b0bd9153b9dd70981bc1;8861cd3593c4334c59f75a859df8652cc4bb688d",
         "Fetal origins of insulin resistance and obesity;Variation at the insulin gene VNTR (variable number tandem repeat) polymorphism and early growth: studies in a large Finnish birth cohort.;Apolipoprotein E polymorphism and fertility: a study in pre-industrial populations.;Opposing influences of prenatal and postnatal weight gain on adrenarche in normal boys and girls.;Maternal-fetal interactions and birth order influence insulin variable number of tandem repeats allele class associations with head size at birth and childhood weight gain.;The human obesity gene map: the 2003 update.;Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA;Intrauterine factors, adiposity, and hyperinsulinaemia;Global haplotype diversity in the human insulin gene region.;Birthweight and perinatal mortality: paradoxes, social class, and sibling dependencies.;The double puzzle of diabetes;The epidemiology of adverse pregnancy outcomes: an overview.;Neonatal anthropometry: the thin–fat Indian baby. The Pune Maternal Nutrition Study;Adiposity and hyperinsulinemia in Indians are present at birth.;The role of uncoupling proteins in pathophysiological states.;Racial differences in the relation between uncoupling protein genes and resting energy expenditure.;An overview on the nutrition transition and its health implications: the Bellagio meeting;An adipose-specific control of thermogenesis in body weight regulation;Insulin gene variable number of tandem repeat genotype and the low birth weight, precocious pubarche, and hyperinsulinism sequence.;On the importance--and the unimportance--of birthweight.;The thrifty phenotype hypothesis.;Obesity and undernutrition and cardiovascular risk factors in rural and urban Gambian communities.;Glucose, insulin and lipid metabolism in rural Gambians exposed to early malnutrition;Uncoupling proteins: functional characteristics and role in the pathogenesis of obesity and Type II diabetes;Uncoupling proteins: their roles in adaptive thermogenesis and substrate metabolism reconsidered;Fires of life: the struggles of an ancient metabolism in a modern world;The nutrition transition and obesity in the developing world.;Thrifty Genotypes and Phenotypes in the Pathogenesis of Type 2 Diabetes Mellitus;Uncoupling to survive? The role of mitochondrial inefficiency in ageing;The Fetal and Childhood Growth of Persons Who Develop Type 2 Diabetes;Energy adaptations in human pregnancy: limits and long-term consequences.;Interactions of perturbations in intrauterine growth and growth during childhood on the risk of adult-onset disease;Insulin resistance syndrome in 8-year-old Indian children: small at birth, big at 8 years, or both?;Transmission ratio distortion at the INS-IGF2 VNTR;Apolipoprotein E (APOE) allele distribution in the world. Is APOE*4 a ‘thrifty’ allele?;The insulin gene VNTR, type 2 diabetes and birth weight;The thrifty genotype in type 2 diabetes;The INS 5′ Variable Number of Tandem Repeats Is Associated with IGF2 Expression in Humans*;Association of the INS VNTR with size at birth;The Great Famine: Northern Europe in the Early Fourteenth Century;Bergmann's rule and the thrifty genotype.;Imprinted and genotype-specific expression of genes at the IDDM2 locus in pancreas and leucocytes.;Susceptibility to human type 1 diabetes at IDDM2 is determined by tandem repeat variation at the insulin gene minisatellite locus;Biochemical control logic and the metabolism of glutamine.;Type 2 (non-insulin-dependent) diabetes mellitus: the thrifty phenotype hypothesis;The possible role of glutamine substrate cycles in skeletal muscle.;Maternal weight gain, infant birth weight, and diet: causal sequences.;Role of triglyceride-fatty acid cycle in controlling fat metabolism in humans during and after exercise.;A Polymorphic Locus Near the Human Insulin Gene Is Associated with Insulin-dependent Diabetes Melliitus;Poverty and famine: An essay on entitlement and deprivation : , Oxford, New York: Oxford University Press, 1981. xi + 257 pp., index. $17.95;Malnutrition and fertility.;Poverty and famines. An essay on entitlement and deprivation [incl. case studies of famines in Bengal, Ethiopia, Sahel, and Bangladesh].;Famine in Tudor and Stuart England.;The high prevalence of diabetes mellitus on a Central Pacific island;Famine and Human Development: The Dutch Hunger Winter 1944-1945;Faminee of history and of today;Diabetes in American Indians and Other Native Populations of the New World;The determinants of growth and form.;Review lecture - The determinants of growth and form;Diabetes mellitus: a \"thrifty\" genotype rendered detrimental by \"progress\"?;Biology of Human Starvation;A critical evaluation of the fetal origins hypothesis and its implications for developing countries.;Lack of support for a role of the insulin gene variable number of tandem repeats minisatellite (INS-VNTR) locus in fetal growth or type 2 diabetes-related intermediate traits in United Kingdom populations.;Differential reproductive efficiency associated with common apolipoprotein e alleles in postreproductive-aged subjects.;Eating, exercise, and \"thrifty\" genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases.;The insulin gene variable number tandem repeat class I/III polymorphism is in linkage disequilibrium with birth weight but not Type 2 diabetes in the Pima population.;Floods, Famines and Emperors. London: Pimlico;PPARgamma, the ultimate thrifty gene.;Fetal origins of adult disease-the hypothesis revisited.;Genomic imprinting of fetal growth;The Demography of Famine: An Indian Historical Perspective;Biochemical control logic and the metabolism of glutamine.;Agriculture’s mixed blessings;The Third Chimpanzee;Determinants of low birth weight: methodological assessment and meta-analysis.;Origin of Species"
        ],
        [
         "25",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "https://ir.vanderbilt.edu/bitstream/1803/13704/1/ms-thesis-manish-kushwaha.pdf",
         "Computer Science",
         "IEEE International Conference on Mobile Adhoc and Sensor Systems Conference, 2005.",
         "Manish Kushwaha,K. Molnár,J. Sallai,P. Völgyesi,M. Maróti,Á. Lédeczi",
         "2181068,2065280567,1717457,1740862,1692190,1721042",
         "70",
         "5.0",
         "2005-12-12",
         "2005",
         "813ffc490393534e9244924dcd0e4337b4bf5fa0;571eab8ea81d7c54bf5454c67c2d28f6aac3f086;35b225cf1cb2ff030eff1ffd9c554a87418b16ee;5e1c7fcad70403f2da76849eaf4551143c008116;a4ac5b5e63942b95c5ebbf51019869950282212d;cd608034cca0eb48676f3fc4f6082e4a504df42f;25a5f84ee5164162ad04e0a1d69c35525ce556de;26d7477c2f05594eb98a191826dcbdb836c9fb45;8b2815e893b0d4a476f27b15b69cec7f2a43d8e8;2cdd6e0dd343a94480adfeba247d3f33c429113a;d7d1ba245d7397c0cead0191c08aeb0da6fb2316;24741a5d686df667874f43d79e6359346444cd3e;6d5239e7d50c83dad3621b65a447ba8191b7694c;be8ac44dd2bb5f5d23cdb902dabd332969222c9a;869b5434530fd46dc84ca30661ebf8b3a697600b;2d4ad6826ed95a6b17709278b9f2b6c729b1163f;fba126954f354e24cc57d1159dfe7d5ae16d11ef;234efad9a8efb5d417743c70fc9f2d0ca6eac97f;6d6e4c92401f60682f00777238d4afeb729340bc;4f264407361a47c81695a03a8d96c078e1f588b0;0e4ade87ce468690926f153bc413d53a5034a9e4;22188b246b50e273b2f8de308c091d0c082aca03;15f705d41cfb39860c33d15435defaa4c6469d29;93172a2fb5c410498079361e806e9c43549c8d2b;fb5232fe382f4c801a0a0ecb4a6e92dab0f72889;fa9d7cdee95b6e780b5006606a8a96cdd62417d8;ba062f1deca36fdc287609771e47a92d26973ed9;0115f7ae387b4116acaafe84176f38bb2e7bab6f;96aa4d55a3870193b23c9c5d3b838e2be6f17c33;b2fe77a389e749359f12ba483664429e11182f62;46a1dbf20cd13dc3204eb0cc35fc34aaa3e2a70c;f5ed53760000b1851c10734f271a97b7017ecfb8",
         "Resilient Localization for Sensor Networks in Outdoor Environments;Mobile-assisted localization in wireless sensor networks;Sensor network-based countersniper system;Robust distributed network localization with noisy range measurements;The flooding time synchronization protocol;Localization of wireless sensor networks with a mobile beacon;Smart sensors to network the world.;Range-free localization schemes for large scale sensor networks;The n-Hop Multilateration Primitive for Node Localization Problems;Ad hoc positioning system (APS) using AOA;Mica: A Wireless Platform for Deeply Embedded Networks;Middleware challenges for wireless sensor networks;The bits and flops of the n-hop multilateration primitive for node localization problems;Wireless sensor networks for habitat monitoring;A survey on position-based routing in mobile ad hoc networks;Robust range estimation using acoustic and multimodal sensing;Location Systems for Ubiquitous Computing;Smart kindergarten: sensor-based wireless networks for smart developmental problem-solving environments;Exposure in wireless Ad-Hoc sensor networks;Dynamic fine-grained localization in Ad-Hoc networks of sensors;The cricket compass for context-aware mobile applications;GPS-free Positioning in Mobile Ad Hoc Networks;GPS-less low-cost outdoor localization for very small devices;Embedding the Internet: introduction;Acoustic Ranging in Resource-Constrained Sensor Networks;IEEE Computer Graphics & Applications: Projects In VR Pervasive Pose-Aware Applications and Infrastructure;VR Pervasive Pose-Aware Applications and Infrastructure. IEEE Distributed Systems Online;Range-free localization schemes in large scale sensor network;This copyright notice must be included in the reproduced paper. USENIX acknowledges all trademarks herein. Robust Positioning Algorithms for Distributed Ad-Hoc Wireless Sensor Networks;The Design of Calamari : an Ad-hoc Localization System for Sensor Networks;Complex Behavior at Scale: An Experimental Study of Low-Power Wireless Sensor Networks;Convex Optimization Methods for Sensor Node Position Estimation;SpotON: An Indoor 3D Location Sensing Technology Based on RF Signal Strength;The Cricket location-support system;Global Positioning System Overview (last modified 5.1.2000);Numerical Optimization, pages 250–275;Theory and application of digital signal processing;If both | µ i | and σ i are large then again location might be affected by echo and we follow the same procedure as in case 2 above;If µ i is close to zero but σ i is large then either the range deviations are spread around zero or few large deviations caused σ i to be large;If | µ i | is large but σ i is small then all elements of ∆ i are large i.e. the node location is deﬁnitely incorrect"
        ],
        [
         "26",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "http://archive.nyu.edu/bitstream/2451/14996/2/Infosec_Book%2bWattal%2bTelang.pdf",
         "Business,Computer Science",
         "Workshop on the Economics of Information Security",
         "Rahul Telang,S. Wattal",
         "145961354,2628487",
         "80",
         "3.0",
         "2005-02-01",
         "2005",
         "",
         ""
        ],
        [
         "27",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-95-1423",
         "Biology,Medicine",
         "Phytopathology",
         "C. Silvar,J. Diaz,F. Merino",
         "6248808,152435507,145620908",
         "63",
         "1.0",
         "2005-12-01",
         "2005",
         "",
         ""
        ],
        [
         "28",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "https://europepmc.org/articles/pmc1069588?pdf=render",
         "Biology,Medicine",
         "Molecular and Cellular Biology",
         "T. Adachi-Yamada,Toshiyuki Harumoto,Kayoko Sakurai,R. Ueda,K. Saigo,M. O’Connor,H. Nakato",
         "1399592175,6637499,3515490,34604795,33751317,7615853,3971771",
         "23",
         "1.0",
         "2005-04-01",
         "2005",
         "",
         ""
        ],
        [
         "29",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "https://europepmc.org/articles/pmc2522321?pdf=render",
         "Medicine,Biology",
         "Journal of Proteome Research",
         "K. Anderson,J. LaBaer",
         "2700838,2112063",
         "311",
         "12.0",
         "2005-06-18",
         "2005",
         "a5fec970c69680e7d7ec9fe90b057291b4e7773d;8ef30a4dcb0ef1af8522d2a4ac3922457ba62a8c;733b342cac9c3cf6fbb70715bc3ecca5b730b349;c3c2808e82d98a4159cabcad3802dd7dc1705f7f;58ee959e3fe9a781bf59ab64549810149f593474;0069e9285f277056500448265d34b03a44372a08;a9d867d185ae257493226d2a53a94b7b682328a2;13bb6ea8a95a04aff56f95a013c3219d3cb57ff3;c4130add7c4c78affcff33f330dec1019373ec22;a982595680c8bcce7fcea5f16ac98e00e1113d2a;13d7452b1f93ae19f9db16cbe60f681616d02198;6e2c87806883391644aaec82861a2d4e64d3a503;7cb9f8e54e2786196b7d0f7178af3e32a66a294f;405a39a2fbd43b5bb6a73f12e6d880a38c0ec469;00ce84e1bc0fb55d3e4ac1eafef11137b842b674;94cbbf1805238625054ddaee315edbe98db15b99;e37bd6ce0c355fbe970a72b5d5ab2c61e76f1e31;7933e84c6162015607118f650818a04029a123a9;1a9168875ee9115b0130a978779867d6c86c164d;d14f3e86ee66258fe9e194939b2eff10a26deac8;251328149f10c2264a5b81e1e2163518d83e7af2;8458139de08cf9af5a8a3db11f9f56b641e0f9ae;3dea83a9f695382d933eb3cd5e89df303b16383c;0c70af8c234cad6a3716ec219fde55a52bca5d8c;96cb8f7f09a8fde293b5675da621836cd4a25442;17c551aca1c09efe7308f3a370e14b6f43eed293;2ce4c74262c38b472473cd089b3f6e1546ba989b;617018364b7ad94d15b34c751700371258488a24;363c94c7ab5aa4baebc089c8a6697c2d6188f054;b9cd2802cfb0e4b3d9c01132f470a2daced6a6b0;9aecd5b116595b1355f5dad17c919dcf448a4d6c;49de046c674e6b1be12427b81126f921c66e06e3;09e506bad5617ee5c1c9fcb2fc1baff71d33c898;1de2e10678cd99ffc6c12a7c80b45300c1f74b9e;9411bbe9e8ba032141dbc5c1904c65b8dce131bd;2b7da19d167b41357fa323a83fec7f69db36a6f6;6285b7e9f8e7cdf299920247ca0e0e14b37b1e8b;640a97b4aeb462b135971ed4acc2a243b55f6b4c;13cadee9e67644d3729d471bc3757c70e9fe173f;ad7be2b4e10846bcd85bdbf69c3e6652d50af488;c0adf2095391d71f8e92eb33df92c66b45ff51f1;48d6fc3223842c715e80bf6096c43cb3a8730e76;02074b7a8b2742ce30eecff1fead09b735be7b58;576afd62b7e1da025747950bfc0031ac070e14d9;901bde6f666a8933f2a1631a28b3c2e85f552cd4;d18a6692c38fa5c91bbe586aaca0d79f7d73ab1f;7d56a74dd196c2e81415b5d95a55b232a60d52f6;c5e573e73615df34b8e755dcd37c71f51a2aef8b;d36b4e618097ff1f97fb54e4fe8338e6539b9419;906df3ac3c0efc4ae49abb03eccb8a1a56eb6d8d;cacf432fbf7361cfc1b5a21e7653f4abeeb608bc;89a91f216ed67b56dbd2e897888fb03d3b7689b4;b8d5884927255b8a1ee8666d42dce3c7d8edc8bc;adc0cef49de7640091c30c64b5b0971fa15f8cec;77c7be9fba69091cdd99b89ae78bc1224eb6427f;296a0473a4537088e01704a8af41c749916998de;dc2aba9d4f2ca0d1a87e3fc0c3d5dbd89347657f;45e8b0e8aba7bce41416d9dc50f1175b8e546537;faf3728220c32d50d6f6311096ec3d70b0efa2af;417636fc2fa5a79d0a9e151ca2fadf4d896c3a6a;06f479de8c40c51ccbe591c004ee358beea7e3cc;d41eb0fce92491576a05850c935f894bbf09bde8;75521ab6ea29e022f51c4ed91382365836db205e;1690493dc16a64b7103da134383d970e407576af;24b56291be5bfeec9b69ecda07e5bc8167b4b326;a566936accc6ba41396d83ab68232ad0083689ee;d028d2e7b78ceb3103cb6e51ac8866e32ca28952;a8df84e42991b3fe1eac759a9491d542673b7c12;ca6ac75d2408d9fbc3ff49e8d62341821574337b;f26551154538a2f1d5799b9540b24cb74fb89763;61dc4870984aa539428bd0fea54fda55149f0eeb;e92fe6ff345e23d95f5d4c263ef2b66f1cbaa947;cf9c0737484df260b4407aceca30c6dbd7e0c08e;24aacae008a93fc5f912537d90b3808da25245bd;1ca341b8f40b748249717f6b26db00517a65645b;9649c703b781c322fad5f5d2e44af14460049978;e29c64ec56f634049c8931616bbc34807e53ccbe;b76586c209648e3f66a48b075a5a3b85b780c8e2;6783882e887d1fd25d84d4ef22b7c4fb6d5982e6;2bc35ccc858ecff282da44ce7c971b3efe3f6edb;1557de0758d8e90f89806d5aac1e02a788941a0e;eb61c808a6b210f8134d3c3ecbb7eb4cec0df4a6;49790e49ee605e518c6c164d6dcd3709b3b3d366;4bd9e587ccb7e702f9b83ce79e9a14d5889a365b;0a7e447ffa5921c3cefbd0a4110d0b73fa24fae4;cc2f46e6bdf0681fd05b7b5d41a74f52aa131813;e4a27eac6551682770c0af50db31723ad34edd55;6ea42f29588a3699a8daaf03e613cdc385a07a98;6984707cd20cab11a337aae74d31aa28f28506f1;bfd9b207d5df59cb51ee9ed7cd70749a9abcb154;16f54f42f2b97dc6548eac350b67227bf4f24715;f09ce709ee7be9223c98fe2729ac39e3654eb432;6046d06aa295bfb20be8e952b5124375ac25d5bd;eec606cec01c5c86a6195abd33dcf9178e4c9dd6;ce54d815b5ea33a108a04557a2a5855c259bd5de;9d4028799d9b9b282a190b3e96498d824aa0fe31;2a0b10e367b0786d4600b336aefc5a15072e5d3e;31cc5c4ab5f488e2a50b692de8264c4e5886e3bc;477e6297459532d8a651d8947d471b69a10c0bee;a5f1d509eb95129272afe5db38d1fa08afc1cf82;0b3b00a3c3dcd8577c6594781da13e90d11a25da;1e672c6d143b318e918a8424e1026a9433fc5952;54e66e26f22f9573b1ab00ef154e4b9ec97662fb;df9f47703c6299a257cd25ef20c82a66d39fd355;dcb0df34754e316e80826416031bc497c2eec162;341d10aacb81f6688bd0eb8566f6b76b1a8ad7b3;fc612a59bce380077f4cb8080bf4f7dcfb3ee494;f7c6c19558fbfec06849749ffa2f18fc4de63455;8c140318c3811d98a91afc7e3bbec42f62dee579;432d239dac42a5263ed2010313d582cfc6c32018;478504ff6ae608efcfcc5d125196cd8e6df85d3e;28a9f8b464aeda844f2233a2da2a00ac86597c56;7e520b5bdc68f2520e402593aa7c5b56ef1a3ad4;33c9970d75ac2c8b0561cfb2c3a9db7f319ada71;c0e41bffc9acee371c0365d1a79d37475c27aff4;238954a402bc9a038f88f0cbde3d801628f64e59;df36002142e55b86221f4ec5ac2722edfb86f73c;c28b6762b25ff1f624cea7a775d7c6a2a4214832;61e58171fd3059916c2e021dee9683e5bcb901b3",
         "T Cell–Dependent Antibody Responses against Aberrantly Expressed Cyclin B1 Protein in Patients with Cancer and Premalignant Disease;Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles;Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast Cancer;A phase I trial of DNA vaccination with a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate cancer;Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant induces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans.;Self-Assembling Protein Microarrays;Protein microarray technology;Minor Histocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem Cell Transplantation;An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in association with HLA-DR.;Development of natural protein microarrays for diagnosing cancer based on an antibody response to tumor antigens.;NY-ESO-1 Expression and Immunogenicity in Malignant and Benign Breast Tumors;Clinical proteomics: revolutionizing disease detection and patient tailoring therapy.;The clinical value of intracellular autoantigens B-cell epitopes in systemic rheumatic diseases.;Serological cloning of cancer/testis antigens expressed in prostate cancer using cDNA phage surface display;Complete response to donor lymphocyte infusion in multiple myeloma is associated with antibody responses to highly expressed antigens.;Humoral immune response against melanoma antigens induced by vaccination with cytokine gene‐modified autologous tumor cells;Identification of a shared epitope recognized by melanoma-specific, HLA-A3-restricted cytotoxic T lymphocytes.;Genome-wide Characterization of a Viral Cytotoxic T Lymphocyte Epitope Repertoire*;Microarrays of tumor cell derived proteins uncover a distinct pattern of prostate cancer serum immunoreactivity;Protein arrays for autoantibody profiling and fine‐specificity mapping;Protein microarrays: Promising tools for proteomic research;Recursive partitioning as an approach to selection of immune markers for tumor diagnosis.;Antigen-specific immunity in neuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen.;Expression of cancer/testis (CT) antigens in lung cancer.;Unique conformation of cancer autoantigen B23 in hepatoma: A mechanism for specificity in the autoimmune response;NY-ESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial ovarian cancer.;Identification of tumor‐associated antigens by screening phage‐displayed human cDNA libraries with sera from tumor patients;Tumor-specific immunological recognition of frameshift-mutated peptides in colon cancer with microsatellite instability.;CD1d-restricted T cells regulate dendritic cell function and antitumor immunity in a granulocyte–macrophage colony-stimulating factor-dependent fashion;Melanoma inhibitor of apoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction;The use of HPLC-MS in T-cell epitope identification.;Immunomic analysis of human sarcoma;Enhancement of antibody detection in cancer using panel of recombinant tumor-associated antigens.;Immunoscreening of phage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease.;Integrated functional genomics approach for the design of patient-individual antitumor vaccines.;Proteomics Approaches to Uncover the Repertoire of Circulating Biomarkers for Breast Cancer;CML28 is a broadly immunogenic antigen, which is overexpressed in tumor cells.;Towards patient‐specific tumor antigen selection for vaccination;Cancer/testis antigens: an expanding family of targets for cancer immunotherapy;Serological identification of antigens associated with renal cell carcinoma.;Characterisation of tumour-associated antigens in colon cancer;Solid-Phase Epitope Recovery: A High Throughput Method for Antigen Identification and Epitope Optimization;Enhanced humoral immune response correlates with improved disease-free and overall survival in American Joint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine.;Identification of Breast Cancer-Restricted Antigens by Antibody Screening of SKBR3 cDNA Library Using a Preselected Patient's Serum;Posttranslational protein modifications: new flavors in the menu of autoantigens;ATP6S1 elicits potent humoral responses associated with immune-mediated tumor destruction;Identification of cancer/testis genes by database mining and mRNA expression analysis;Autoantigen microarrays for multiplex characterization of autoantibody responses;Miniaturised multiplexed immunoassays.;Identification of Cyclin B1 as a Shared Human Epithelial Tumor-Associated Antigen Recognized by T Cells;Trends in biomarker research for cancer detection.;Autoimmune diseases: are markers ready for prediction?;Post-translational protein modifications in antigen recognition and autoimmunity.;Identification of a tissue-specific putative transcription factor in breast tissue by serological screening of a breast cancer library.;Profiling the immune response in patients with breast cancer by phage‐displayed cDNA libraries;Tumor-associated antigen TA90 immune complex assay predicts recurrence and survival after surgical treatment of stage I-III melanoma.;Evidence that specific T lymphocytes may participate in the elimination of chronic myelogenous leukemia;Pre-existent immunity to the HER-2/neu oncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer;Profiling the immune responses in patient sera with peptide and cDNA display libraries.;Reactivity of natural and induced human antibodies to MUC1 mucin with MUC1 peptides and n‐acetylgalactosamine (GalNAc) peptides;Rapid degradation of a large fraction of newly synthesized proteins by proteasomes;Serological identification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung cancer.;p53 Antibodies in the sera of patients with various types of cancer: a review.;Expression of cancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a new CT gene, CT9.;CT10: A new cancer‐testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational‐difference analysis;p53 gene mutation, tumor p53 protein overexpression, and serum p53 autoantibody generation in patients with breast cancer.;Cancer-testis antigens and ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-specific ING1 transcripts and a homologue gene.;Antigens recognized by autologous antibody in patients with renal‐cell carcinoma;SYFPEITHI: database for MHC ligands and peptide motifs;Cleavage by Granzyme B Is Strongly Predictive of Autoantigen Status;Humoral immune responses of cancer patients against \"Cancer-Testis\" antigen NY-ESO-1: correlation with clinical events.;The telomerase catalytic subunit is a widely expressed tumor-associated antigen recognized by cytotoxic T lymphocytes.;Isoforms of the human PDZ-73 protein exhibit differential tissue expression.;The 72-kDa Component of Signal Recognition Particle Is Cleaved during Apoptosis*;The Central Role of CD4+ T Cells in the Antitumor Immune Response;Characterization of human colon cancer antigens recognized by autologous antibodies;A Survey of the Humoral Immune Response of Cancer Patients to a Panel of Human Tumor Antigens;Human lung cancer antigens recognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor gene locus on chromosome 3p21.3.;High-titer HER-2/neu protein-specific antibody can be detected in patients with early-stage breast cancer.;SSX: A multigene family with several members transcribed in normal testis and human cancer;Autoantibodies to p53 in ovarian cancer patients and healthy women: a comparison between whole p53 protein and 18-mer peptides for screening purposes.;A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening.;Humoral immune response against p53 protein in patients with colorectal carcinoma;DNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to detect all known p53 alterations in human malignancies.;Anti-p53 antibodies in sera from patients with chronic obstructive pulmonary disease can predate a diagnosis of cancer.;Anti-p53 antibodies in sera of workers occupationally exposed to vinyl chloride.;Absence of p53 autoantibodies in sera from glioma patients.;A motif in human histidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-coil that is essential for enzymatic activity and contains the major autoantigenic epitope.;Clinical utility of the immunocytochemical detection of p53 protein in cytological specimens.;Role of bone marrow-derived cells in presenting MHC class I-restricted tumor antigens.;Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains.;Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-macrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor immunity.;Development of antibodies against p53 in lung cancer patients appears to be dependent on the type of p53 mutation.;HLA-A2 molecules in an antigen-processing mutant cell contain signal sequence-derived peptides;Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry.;Very long charge runs in systemic lupus erythematosus-associated autoantigens.;Characterization of naturally occurring minor histocompatibility peptides including H-4 and H-Y.;The fine specificity of anti-La antibodies induced in mice by immunization with recombinant human La autoantigen.;Post-translational modifications of proteins: some problems left to solve.;CA125 Antigen Levels in Obstetric and Gynecologic Patients;Purification of a human prostate specific antigen.;Detection of a transformation-related antigen in chemically induced sarcomas and other transformed cells of the mouse.;SPECIFIC CARCINOEMBRYONIC ANTIGENS OF THE HUMAN DIGESTIVE SYSTEM;Immunotherapy (APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3 randomized placebo-controlled trial;Efficient purification of unique antibodies using peptide affinity-matrix columns.;Endogenous immune response to early- and intermediate-stage melanoma is correlated with outcomes and is independent of locoregional relapse and standard prognostic factors.;Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem cell transplantation and in healthy female donors.;Recombinant antigen expression on yeast surface (RAYS) for the detection of serological immune responses in cancer patients.;Immunotherapy of hematologic malignancy.;p53 autoantibodies can be indicative of the development of breast cancer relapse.;The autoantibody repertoire: searching for order;MUC1-like tandem repeat proteins are broadly immunogenic in cancer patients.;Epitope identification and vaccine design for cancer immunotherapy.;Immunotherapeutic Targets Cancer : Identification of Potential Diagnostic and Cancer-related Serological Recognition of Human Colon Updated;Identification of tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by serological screening of a testicular library with breast cancer serum.;Humoral immunity to human breast cancer: antigen definition and quantitative analysis of mRNA expression.;Monitoring of p53 autoantibodies after resection of colorectal cancer: relationship to operative curability.;Gure, A. O. et al. CT10: a new cancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-difference analysis. Int. J. Cancer 85, 726-732;MHCPEP, a database of MHC-binding peptides: update 1997;Cancer,Facts and Figures about."
        ],
        [
         "30",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "http://www.jbc.org/article/S0021925820792379/pdf",
         "Chemistry,Medicine",
         "Journal of Biological Chemistry",
         "E. Gagnon,M. Bergeron,Nikolas D. Daigle,Marie-Hélène Lefoll,P. Isenring",
         "117842903,5297299,40563519,4943153,144981400",
         "30",
         "1.0",
         "2005-09-16",
         "2005",
         "",
         ""
        ],
        [
         "31",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         "https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-10/issue-5/051604/Prospective-grading-of-neoplastic-change-in-rat-esophagus-epithelium-using/10.1117/1.2102767.pdf",
         "Medicine",
         "Journal of Biomedical Optics",
         "A. Wax,J. Pyhtila,R. Graf,R. Nines,C. Boone,R. Dasari,M. Feld,V. Steele,G. Stoner",
         "6601645,2596802,39855573,5362380,48807863,6398265,145506659,5086274,3306303",
         "65",
         "2.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "32",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "https://access.archive-ouverte.unige.ch/access/metadata/414fbfe1-4481-4240-92c1-e62a3593f0f0/download",
         "Psychology,Medicine",
         "Emotion",
         "T. Brosch,D. Sharma",
         "2256291,34546725",
         "153",
         "10.0",
         "2005-09-01",
         "2005",
         "804eeefaf41247d1927a7e9d4ab27f06c3917fab;24cea7f3a3d8e4d490725d55933a724421b8c086;e35f63ac711e7253049a4811a354b93478d4959e;240247d67cc26cd7474e02ed75de6682e181839d;494771f2b2bcff4d795dc4e974524c5a7ec5572f;2da6cce61dd4defa4c44e82d1e0bb9920ee193bb;3542ff0be48d6196677c85fb6b65dbf7180d0e46;bdfeee608d25196733e80800f43a57b768a039ab;fbe379683d148cdf82a03010f1e79f22d16657e6;4f3ef78375b4c147ab794ca9a8e9bca1fe4daf4e;76361a44e145732a39dbc68d9418871038c83be2;6871c3babd4f47895d616d15b17b7ba471afa89a;bc44f92d97e409449ce3af72f7d2cb4ac544245a;c26f4aef22b2682db192d60b932455df09702e92;d506a8fdc86aeb83c28d4b5a87cf35e3c0e14910;46c99f0a5935572dd8b19936afef8faa8de25fbf;05c2b29d567263cb034f28eefe078545ad357d73;bdeee63dc6398e2d9d1fe9571d23dcf141903ee1;2445bf8b2442a04e9a4db6819aaa267f8e45f4a7;9508efad2d667fa38e4c9a50efd91c117a7a7943;74257ff0d6645ab73650fdc9faaf8fe1a6a430b5;7f57c5462b5728661608c308e76be0e2f4a0408a",
         "Attentional control of the processing of neural and emotional stimuli.;Searching for threat;Emotion drives attention: detecting the snake in the grass.;The face in the crowd revisited: a threat advantage with schematic stimuli.;Fear and anxiety: animal models and human cognitive psychophysiology.;Facial Expressions of Emotion: Are Angry Faces Detected More Efficiently?;A subcortical pathway to the right amygdala mediating \"unseen\" fear.;It Takes a Confounded Face to Pop Out of a Crowd;On the automatic nature of phobic fear: conditioned electrodermal responses to masked fear-relevant stimuli.;Orienting and schizophrenia: stimulus significance, attention, and distraction in a signaled reaction time task.;A feature-integration theory of attention;The premise of equipotentiality in human classical conditioning: conditioned electrodermal responses to potentially phobic stimuli.;One-trial learning and superior resistance to extinction of autonomic responses conditioned to potentially phobic stimuli.;Phobias and preparedness;The cognitive neuroscience of emotion;Fears, phobias, and preparedness: toward an evolved module of fear and fear learning.;Unconscious emotion : Evolutionary perspectives, psychophysiological data, and neuropsychological mechanisms;A threat imminence approach to human fear responding : direction of threath, aversive contexts, and electrodermal responses;Preparedness and electrodermal fear-conditioning: ontogenetic vs phylogenetic explanations.;Finding the face in the crowd: an anger superiority effect.;Feature analysis in early vision: evidence from search asymmetries.;The emotional brain.;E-Prime;32306-1270), as editor of the Clinician's Research Digest newsletter for"
        ],
        [
         "33",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "https://www.jstage.jst.go.jp/article/expanim/54/5/54_5_461/_pdf",
         "Medicine",
         "Experimental animals",
         "M. Asakawa,T. Yoshioka,I. Hikita,T. Matsutani,T. Hirasawa,A. Arimura,T. Sakata,T. Horikawa",
         "50156427,2001345,5223435,5352485,34764605,47810324,48565673,143893766",
         "23",
         "1.0",
         "2005-10-01",
         "2005",
         "abd6726370ffc4c6e7ed4b64164bf5d2b42e1fcb;d1076a4457fff22f01c397ef080a731c23e5e807;a884242808a1ace5f72ba4d59c97a6360e3644e7;2571e34834098d5d1ec06e89c81458ec84a41b8e;6c644bd483d2639bcfddeaf8bcb696cd2df1ef77;e056d6b051a9582e3f3f1d81d84a99b329946b64;b3327aae68ba3bd0c99e3dc1504d0a118d9fa133",
         "Role of the Nh (Non-hair) mutation in the development of dermatitis and hyperproduction of IgE in DS-Nh mice.;DS–Nh as an experimental model of atopic dermatitis induced by Staphylococcus aureus producing staphylococcal enterotoxin C;Characterization of dermatitis arising spontaneously in DS-Nh mice maintained under conventional conditions: another possible model for atopic dermatitis.;Atopic dermatitis: new insights and opportunities for therapeutic intervention.;Locus of dominant hairless gene (Ht) causing abnormal hair and keratinization maps to rat chromosome 10.;Characteristics of spontaneous erythema appeared in hairless rats.;IgE production by normal human lymphocytes is induced by interleukin 4 and suppressed by interferons gamma and alpha and prostaglandin E2."
        ],
        [
         "34",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "http://www.cell.com/article/S009286740500098X/pdf",
         "Biology,Medicine",
         "Cell",
         "I. Krylova,E. Sablin,Jamie M. R. Moore,R. Xu,G. Waitt,J. A. MacKay,D. Juzumiene,J. Bynum,K. Madauss,V. Montana,L. Lebedeva,M. Suzawa,Jon D. Williams,Shawn P. Williams,R. Guy,J. W. Thornton,R. Fletterick,T. Willson,H. Ingraham",
         "34437421,5695081,2108562213,4605417,7978640,144066376,47866582,48218038,3502869,4051165,2106094788,40606710,51017082,23919632,144399402,3722546,4009116,5064590,3514504",
         "405",
         "18.0",
         "2005-02-11",
         "2005",
         "1d6bd7b0c8db990d8d67f496ede67ae2827cae6f;9603421bed7605ac327a47ae9982d0ab8c6eeb58;c84e4fce3b6dadc50ce3c174b3bc4759442e38e9;d91a2295136ab9944fcd756bddab143c84804b59;ce6902fec41ee6fd52c9e3eefa059bc6bb415eef;44ef03895ff4e14bf5d226ef3d05f0e097d80486;87f895a0ecdc49df01a9cbef037f898e90d2c91d;89544b7b1dfd705c9d5199e342e8cd0fa7221a51;926e9a38d4e6ee4ff925995e6625106a2513183a;cb4f09dcf20c7700c3a5c1c5e2ca4c981182ce68;f256f33b7a0cf35735fba4d5defa2c3da988e490;fe6adae44ba4ded648fe28fe3837a9d22eab5d08;63523500744c4a42596cef3a31049061b7999a0c;4bd5ee4101ea924889a1ea398fb3cdd39310ba74;7599925bf30ff65b9c4e63464a1d7d7d014c09a7;bf6d61b49f81eeea09540d0a5924f026230c8d14;69faa8f5d27ed9f6cb09a4ba5a251a52aa698fea;83011a2d9fafdebe71b685684d22e7d5e37af1f7;badcfc9340eee368b92961ba52fb94012a7f031f;3902088e094524712a09886b265e77cb3d5d4bb3;701fab8b80fffcd0765465a8ed7a523fa3429208;c740aba3971f2e536e4e70ae166ba46debd24086;52ca6d637dd35ebbe5659f31342260a24b354988;2a9dce8ca5350d4137a5debd70e144e145e0468e;bada3a161bb601bd7b0ef07a901379d8ff22298c;2f1bdce0bca9ff5556e0c237c232b92fc64fe1e5;ee5445492daa6341cab66a5ecf24aa041bb6129f;b3ac08c338b82741e97e0671aea2a3381cd0a57b;bac05ffabbbe20bbfd3f9ac7bf2bb0d627b364e0;6160c4d09e8745db0d669351d138547cfab8909c;885007d7cdff2b2f9330a93eb9d568d0bc76403a;2bb11f7aae32348cc314171336d2e27bebf32513;d615c6e4c2161a309b9c63972867db1adf646a7e;2a8fa95224b6a8a54368262f621e3bb36e6a62c6;eb4c2bdb02b2fbca400d8a645440fcf16d25b0e5;2749817f25ac423a826977acad0284405c163d0e;67c9984328066bf9019152a7c4c325941694bfa7;193b40d2589ce17655a376a82f32747b6edaafc0;91b1bd09d655da66f30a0b2b416be982383e4f93;363c04c59e1ccf958552fe68eed66e274977c70d;f9845ded6f23c4d2e620a88f7a765ec38dc22bef;8df18f32bb193c4c9cfae684ea4a80d616601331;20cc588436334fd28ca613a68cffe81c9ec29844;ecbf24f75bd3d9dd4b3b303994f77850f57e2ca7;847eab3db971b86aa434f2a60ffeef7317be5d42;0aef55484c9bcde366848592dd43b45aa85d55a5;4ab6abd506eb7b4b10836e83a03ff274278bb0c5;cbb0f3690661c288f1a383293a67d42218081e2c;1117b106700a5d022a3af1103c9c370f95ea0fcc;e08c7acadbbeada1f429d43231aa02d92c3b3ba1;6332dfdced172ca5ffa1971c1b285c5ab3119281",
         "The DEAD-Box Protein DP103 (Ddx20 or Gemin-3) Represses Orphan Nuclear Receptor Activity via SUMO Modification;Amplification of the steroidogenic factor 1 gene in childhood adrenocortical tumors.;Identification of liver receptor homolog-1 as a novel regulator of apolipoprotein AI gene transcription.;Synergy between LRH-1 and beta-catenin induces G1 cyclin-mediated cell proliferation.;Quantitative Proteomics of the Thyroid Hormone Receptor-Coregulator Interactions*[boxs];LRH-1: an orphan nuclear receptor involved in development, metabolism and steroidogenesis.;Sex determination: a ‘window’ of DAX1 activity;Generation and Characterization of Human Steroidogenic Factor 1 LBD Crystals with and without Bound Cofactor Peptide;SHIP-2 and PTEN Are Expressed and Active in Vascular Smooth Muscle Cell Nuclei, but Only SHIP-2 Is Associated with Nuclear Speckles*;Resurrecting the Ancestral Steroid Receptor: Ancient Origin of Estrogen Signaling;MrBayes 3: Bayesian phylogenetic inference under mixed models;Activation of nuclear receptors: a perspective from structural genomics.;StAR-related Lipid Transfer (START) Proteins: Mediators of Intracellular Lipid Metabolism*;The Drosophila Orphan Nuclear Receptor DHR38 Mediates an Atypical Ecdysteroid Signaling Pathway;Structural basis for ligand-independent activation of the orphan nuclear receptor LRH-1.;Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors;Nuclear lipid signalling;Nuclear lipids: New functions for old molecules?;X-Ray Structure of the hRORα LBD at 1.63 Å;Elevation of plasma phospholipid transfer protein in transgenic mice increases VLDL secretion Published, JLR Papers in Press, August 16, 2002. DOI 10.1194/jlr.M200166-JLR200;Crystal Structure of the HNF4α Ligand Binding Domain in Complex with Endogenous Fatty Acid Ligand*;Phosphorylation and Intramolecular Stabilization of the Ligand Binding Domain in the Nuclear Receptor Steroidogenic Factor 1;Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids.;Inhibition of androgen receptor (AR) function by the reproductive orphan nuclear receptor DAX-1.;Control of Cholesterol Turnover in the Mouse*;Phosphatidic Acid-Mediated Mitogenic Activation of mTOR Signaling;The PITP family of phosphatidylinositol transfer proteins;Characterization of a G Protein-activated Phosphoinositide 3-Kinase in Vascular Smooth Muscle Cell Nuclei*;Binding of ligands and activation of transcription by nuclear receptors.;Structure of a Multifunctional Protein;Crystal Structure of the Ligand-binding Domain of the Ultraspiracle Protein USP, the Ortholog of Retinoid X Receptors in Insects*;Haploinsufficiency of steroidogenic factor-1 in mice disrupts adrenal development leading to an impaired stress response.;Nuclear Lipid Signaling;Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors.;Ligand binding and nuclear receptor evolution;A new method to localize and test the significance of incongruence: detecting domain shuffling in the nuclear receptor superfamily.;Inferring the historical patterns of biological evolution;Oxysterol Regulation of Steroidogenic Acute Regulatory Protein Gene Expression;Structure and specificity of nuclear receptor-coactivator interactions.;25-Hydroxycholesterol is not a ligand for the orphan nuclear receptor steroidogenic factor-1 (SF-1).;Wilms' Tumor 1 and Dax-1 Modulate the Orphan Nuclear Receptor SF-1 in Sex-Specific Gene Expression;Activation of the orphan nuclear receptor steroidogenic factor 1 by oxysterols.;DAX-1 inhibits SF-1-mediated transactivation via a carboxy-terminal domain that is deleted in adrenal hypoplasia congenita;A new method of inference of ancestral nucleotide and amino acid sequences.;CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;The nuclear receptor steroidogenic factor 1 acts at multiple levels of the reproductive axis.;A cell-specific nuclear receptor is essential for adrenal and gonadal development and sexual differentiation;Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology;Repression of orphan nuclear recep - or activity via SUMO - modification and the Dead - Box Protein P 103 ( Ddx 20 , Gemin - 3 );Nuclear lipids: key signaling effectors in the nervous system and other tissues;X-ray structure of the hRORalpha LBD at 1.63 A: structural and functional data that cholesterol or a cholesterol derivative is the natural ligand of RORalpha.;Nuclear-receptor ligands and ligand-binding domains.;Standard Fmoc protocols.;Methods Enzymol"
        ],
        [
         "35",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.HYP.0000184251.01159.72",
         "Chemistry,Medicine",
         "HYPERTENSION",
         "C. Zeng,Z. Wang,U. Hopfer,L. Asico,G. Eisner,R. Felder,P. Jose",
         "49386778,48708059,4337525,6316449,38877762,1955072,3561892",
         "39",
         "1.0",
         "2005-10-01",
         "2005",
         "",
         ""
        ],
        [
         "36",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         "https://jcp.bmj.com/content/jclinpath/58/9/991.full.pdf",
         "Biology,Medicine",
         "Journal of Clinical Pathology",
         "Giulio Rossi,G. Sartori,Riccardo Valli,F. Bertolini,Nazzarena Bigiani,L. Schirosi,A. Cavazza,G. Luppi",
         "2248858530,143679786,2251605562,2251555357,12435222,48791235,3835552,2251360364",
         "23",
         "3.0",
         "2005-08-26",
         "2005",
         "",
         ""
        ],
        [
         "37",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/129/2/1290171.pdf",
         "Medicine,Biology",
         "Reproduction",
         "J. Ito,M. Hirabayashi,Megumi Kato,A. Takeuchi,Mayumi Ito,M. Shimada,S. Hochi",
         "38681521,34621985,32885513,152918649,50230871,10276380,5629712",
         "41",
         "3.0",
         "2005-02-01",
         "2005",
         "",
         ""
        ],
        [
         "38",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "http://arxiv.org/pdf/1111.4445.pdf",
         "Mathematics,Computer Science",
         "SIAM Journal on Numerical Analysis",
         "Y. Kuznetsov,W. Govaerts,E. Doedel,Annick Dhooge",
         "1775504,1797969,2966430,2713848",
         "35",
         "2.0",
         "2005-04-01",
         "2005",
         "56ff3caaf8a0dd6ed415f8b84f6c8f9041d2a0fb;6ed02cffaaf644f63912f41fbfa00f6f4406c464;7838818b776dd024c885b857c56815ddf215cf60;435ab7306b96edfc3fad6fe165e22d6d6abe531b;526c44f9a4fdac6824d7b1813869649797defd08;3239959373f015e193b6493d7fd64b603547c6a2;8b6f7d0b305d6cba163ef568cdc361ada56e6b7a;cd060346f7d865fff39eb01922207776811cdfa4;b32aba832ca2be4a84c687b040bc39fc8a8413bb;09f52abb09d783865c4ef2cf5af6454802bd2b93;42763fcda07ec7c1a6a93aba8c8f83352c6299c3;cf2c8d80c7bcba7a507801c6e3da8cac70b85bd8;0cbe5c98f1b9c0273629c41745fbec5750b6cf9c;23a8fe3d34a78f2a2fe3f8b0ff32919586dbe96b;340d7ac3b6e13a63e0c47b46543e4377a55f6963;2b6adbad7d8f4c5f1ade291264df221e1b7e22eb;fc93cfbfe36648f105fd4034bbc2471c3367dff8;a594a834dd17b28e2b5ddd42cbd1d9e62efc1b7a;90c0d56498401779d928cd4d8d24c66705fad561;93b1a0baf18ec6b3278805931585392000877ff8;6df4bbd7d59ccce8b034ad9eca8ee1c5a4355592;54cce58d16bd817c766c9495c57eac4f77993b71;1cdcc6db6b8139eb1c1110ba44933727760a17dd;1c46bd1bfbfe4248aa642f5210841379d5923763;d84c4a0dc989082a28956c95ba8ff3a220182091;d7cbbcb3b90deef39fc3b1f4e2eecb80e78a3c08;17be3ad189a2e01f96648ab4b890ddc1c2cf641f;c2e294e7b709dcdae66b600a3c8d3fa459745d0f;2ce5b9b35b5941749cb614e72a69e7ee0de9184c;d35064f6159576e2d9a49a443f8abcdb681cc49a;70e3da6c426ca384f78f77474cbbf00a436038a2;52c0a963506ad2df44e4d5e4358b1e12f500b983;747fa5fcf394aeedc790488ce725a35e3351f310;0b41448a2d92cccfcf4a9ce1e0d68371ce200cfb;39f578060187f86bc80f645b39d735479e664d86;03894e8313ad4c1b36af338151a01baa88cc479e;eea110aa0ea13cc090cebe08f30c3178cc0ad359;7921f44af253340514c9c98af417b0cb07c75afd;4ae84da06e7dbb22fcf248ce4bfa31569dfbad6b;2f59d80b6cccd2b0a28a437d81f360c9fc701287;36ba81e158b864ed1576224f83da232b6031465c;6c1638a43f9bab7409de246138866814cffde0f2;b862a20ee79a56c79e03701fefd4c5b14d53a542",
         "Numerical Methods for Two-Parameter Local Bifurcation Analysis of Maps;Newton's descent method for the determination of invariant tori.;A Newton Method for Locating Invariant Tori of Maps;Sensitivity Analysis of ODES/DAES Using the Taylor Series Method;Numerical Normal Forms for Codim 2 Bifurcations of Fixed Points with at Most Two Critical Eigenvalues;Numerical Methods for the Continuation of Invariant Tori;MATCONT: A MATLAB package for numerical bifurcation analysis of ODEs;Cl_matcont: a continuation toolbox in Matlab;Computation of Periodic Solution Bifurcations in ODEs Using Bordered Systems;Chapter 4 – Numerical Continuation, and Computation of Normal Forms;Computing Periodic Orbits and their Bifurcations with Automatic Differentiation;Numerical Normalization Techniques for All Codim 2 Bifurcations of Equilibria in ODE's;Topics in bifurcation theory and applications;Elements of applied bifurcation theory (2nd ed.);Harmonic balance analysis of period-doubling bifurcations with implications for control of nonlinear dynamics;Algorithm 755: ADOL-C: a package for the automatic differentiation of algorithms written in C/C++;Bifurcation and predictability analysis of a low-order atmospheric circulation model;Models of complex dynamics in nonlinear systems;Bifurcation sequences at 1:4 resonance: an inventory;Harmonic balance methods for the analysis of chaotic dynamics in nonlinear systems;BIFURCATIONS AND CHAOS IN A PERIODIC PREDATOR-PREY MODEL;NUMERICAL ANALYSIS AND CONTROL OF BIFURCATION PROBLEMS (II): BIFURCATION IN INFINITE DIMENSIONS;Numerical Calculation of Invariant Tori;THE QUASIPERIODIC ROUTE TO CHAOS IN A MODEL OF THE PEROXIDASE-OXIDASE REACTION;Global Characterization of the Normal Form for a Vector Field Near a Closed Orbit;A simple global characterization for normal forms of singular vector fields;Normal form reduction for time-periodically driven differential equations;Numerical computation of invariant circles of maps;NONLINEAR OSCILLATIONS, DYNAMICAL SYSTEMS, AND BIFURCATIONS OF VECTOR FIELDS (Applied Mathematical Sciences, 42);Irregularity: a fundamental property of the atmosphere*;Nonlinear Oscillations, Dynamical Systems, and Bifurcations of Vector Fields;Collocation at Gaussian Points;Continuation of Quasi-periodic Invariant Tori;The fold-flip bifurcation;On the computation of characteristic multipliers for predicting limit cycle bifurcations;Elements of applied bifurcation theory;AUTO 2000 : CONTINUATION AND BIFURCATION SOFTWARE FOR ORDINARY DIFFERENTIAL EQUATIONS (with HomCont);CONTENT: A multiplatform environment for analyzing dynamical systems;Normal forms of bifurcating periodic orbits;Numerical computation of periodic solution branches and oscillatory dynamics of the stirred tank reactor with A → B → C reactions;Geometrical Methods in the Theory of Ordinary Differential Equations;Elementary stability and bifurcation theory;Bifurcation of maps and applications;Topics in bifurcation theory;Numerical solution of boundary value problems in ordinary differential equations;Numerical Continuation, And Computation Of Normal Forms;CAPD: Computer assisted proofs in dynamics"
        ],
        [
         "39",
         "002dea70291555c99aa2d88b87db7e8d3ccad66a",
         "https://www.nature.com/articles/6602902.pdf",
         "Biology,Medicine",
         "British Journal of Cancer",
         "E. Petroulakis,Y. Mamane,O. Bacquer,D. Shahbazian,Nahum Sonenberg",
         "4783187,4405032,2096936102,6155126,2242737161",
         "216",
         "11.0",
         "2005-12-13",
         "2005",
         "",
         ""
        ],
        [
         "40",
         "002fbca2bbbe668326f59226c20ed85af17d375a",
         "https://link.springer.com/content/pdf/10.1007%2F11554554_19.pdf",
         "Computer Science",
         "International Conference on Theorem Proving with Analytic Tableaux and Related Methods",
         "J. Otten",
         "34835300",
         "42",
         "2.0",
         "2005-09-14",
         "2005",
         "",
         ""
        ],
        [
         "41",
         "002fe46a688bf95a086a415278e9bff1496107b5",
         "http://www.cell.com/article/S153561080500231X/pdf",
         "Biology,Medicine",
         "Cancer Cell",
         "S. Moody,D. Perez,Tien-Chi Pan,C. J. Sarkisian,C. P. Portocarrero,Christopher J. Sterner,Kathleen L Notorfrancesco,R. Cardiff,L. Chodosh",
         "5209530,80808143,6093778,39129050,4947009,32683945,14281764,144192072,1904147",
         "750",
         "27.0",
         "2005-09-01",
         "2005",
         "d1685e7f7ca8e4fd2d66009b9ae7f9a946cef27c;b188eecc76ca241738297f518d7402f823902225;73d6948a5fb62bd8c3fbbd612b1064003bcb2222;df36888b3309ec7750074df622b47f9c3295c6ea;554b2695193aaa9b56afbc246977d45d4d6bf069;e4a02e7e42ad6e6c04a03705e5f5ae21a9f75921;ca8e6394bb4addf7fdd0f8d9d6acc31526b82c5e;1f43bbef5c6a613e66ac020a36573a1a1695abcc;26887a17ef00fc2a6b58d931b821c48d5160f4a1;5a58345878984b345468544e58fe6413a8e43a8e;368bc1eb2691ef36df9762069ae631045f290c08;5434dcc794f95181631d023d84fe07144101fe0f;c8ab2dddf63bea242ddf0c441812eaf76297fff4;2bc3b91c9e03942285ae75d47b9f8db048c27299;beedb4835b03f6a4e111ba29534b3ae9b24880f0;4ffdfd9fe14e81e22bdbc72d39637d22e700e4e4;ba66c1e646953a8045c3749417eb944c98d33573;aa3a96180cdb983f4a4f4af2496bfa7d00a0a158;e9d95a8b880a55e2146d7b7ec3232d0b9763d1be;cd39231c21e5a75888fb3690738ef4c0d81f16b4;dcdca38000e63cf50c77659f350f813304f853e9;ffe43ab57cb77c267775fdce32136fafa56127a5;1b37279dce5ca9606cbf62a1daf2cbc4dfecc339;e7dd79965df81538976ccbc60fb4dfad89ca5a22;5e71b372e6fc7b2913723f17f0c6886a1d31c5f1;15bbd26a5a6ba6542015cce8a5a5d7bfe0456871;2a0bf30dc789b2f9aa9f1078cff496afe54134e2;3e3205e0cab80625c4c8998840ee4b1bf9939c18;5c41be65df72756eeb403cfd34e22f066c5aaf45;bb486cda6ec9d6c724ff4b7deda7d21987347975;cafcdee34c4bc3318ca15695af06ec6df4762ddb;095ff22fc2cbfb79a6a92bccd7e139cf75431f18;4596fbec21a29a5e6473a76c04b3717da1b9a87b;2c298a2ec7dab478f7d68861a80cd349e3d268ce;5c2b1f0ee20ea6870be56b2ed586336506e4cfd5;47271366e6aecb37cc3cfbac261c5ca78e5fc056;717abb278d928cca9e81b1fdfa973b881dfa63bb;880a4052294e09991d691f265c72d0f91950a3bc;370c1be9e80695d41d3ca267c83221f0550c7c3f;ffd292bd329bb8d290e60f8c780746ccd9701512;7af1562e5f5d0b6f367d09da3ed626719ba381b7;378557e08f29840a0a90f6193e11d99b8ab090f2;97affe65ca9e938972c073b892b49ac6ab4dddd5;9855f7793bbec7ee5a2079afc7f0dba1480e992f;4baac9512e789b12fb9ae1e2b44148b7c31f52f0;b808df891a28cfb776d57b0f0286b1df2cc1b7e5",
         "Pak1 phosphorylation of snail, a master regulator of epithelial-to-mesenchyme transition, modulates snail's subcellular localization and functions.;Global Cancer Statistics, 2002;Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer;Glycogen synthase kinase-3 is an endogenous inhibitor of Snail transcription;Menopausal status dependence of the timing of breast cancer recurrence after surgical removal of the primary tumour;Dual regulation of Snail by GSK-3β-mediated phosphorylation in control of epithelial–mesenchymal transition;Treatment of lymph-node-negative, oestrogen-receptor-positive breast cancer: long-term findings from National Surgical Adjuvant Breast and Bowel Project randomised clinical trials;Twist, a Master Regulator of Morphogenesis, Plays an Essential Role in Tumor Metastasis;A two-gene expression ratio predicts clinical outcome in breast cancer patients treated with tamoxifen.;Prognostic molecular markers in early breast cancer;Detection and clinical implications of early systemic tumor cell dissemination in breast cancer.;Vimentin expression in breast carcinomas.;Repeated observation of breast tumor subtypes in independent gene expression data sets;Natural history of more than 20 years of node-positive primary breast carcinoma treated with cyclophosphamide, methotrexate, and fluorouracil-based adjuvant chemotherapy: a study by the Cancer and Leukemia Group B.;C-myc, not HER-2/neu, can predict recurrence and mortality of patients with node-negative breast cancer;Conditional activation of Neu in the mammary epithelium of transgenic mice results in reversible pulmonary metastasis.;The prognostic significance of epithelial-mesenchymal transition in breast cancer.;Twist function is required for the morphogenesis of the cephalic neural tube and the differentiation of the cranial neural crest cells in the mouse embryo.;Correlation of Snail expression with histological grade and lymph node status in breast carcinomas;The SLUG zinc-finger protein represses E-cadherin in breast cancer.;A novel doxycycline‐inducible system for the transgenic analysis of mammary gland biology;Efficacy and safety of trastuzumab as a single agent in first-line treatment of HER2-overexpressing metastatic breast cancer.;Gene expression profiling predicts clinical outcome of breast cancer;Overview of treatment results with trastuzumab (Herceptin) in metastatic breast cancer.;The Mouse Snail Gene Encodes a Key Regulator of the Epithelial-Mesenchymal Transition;Targeting HER2: recent developments and future directions for breast cancer patients.;Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications;Long-term results of local recurrence after breast conservation treatment for invasive breast cancer.;Mechanisms of inactivation of E-cadherin in breast carcinoma: modification of the two-hit hypothesis of tumor suppressor gene;Role of isolated locoregional recurrence of breast cancer: results of four prospective studies.;The Akt Proto-oncogene Links Ras to Pak and Cell Survival Signals*;The transcription factor Snail is a repressor of E-cadherin gene expression in epithelial tumour cells;The transcription factor Snail controls epithelial–mesenchymal transitions by repressing E-cadherin expression;Separating favorable from unfavorable prognostic markers in breast cancer: the role of E-cadherin.;Annual hazard rates of recurrence for breast cancer after primary therapy.;Phase II study of weekly intravenous recombinant humanized anti-p185HER2 monoclonal antibody in patients with HER2/neu-overexpressing metastatic breast cancer.;Correlation of E-cadherin expression with differentiation grade and histological type in breast carcinoma.;Transgenic oncogene mice. Tumor phenotype predicts genotype.;Significance of ipsilateral breast tumour recurrence after lumpectomy;Relation of tumor size, lymph node status, and survival in 24,740 breast cancer cases;Correlation of c-erbB-2 gene amplification and protein expression in human breast carcinoma with nodal status and nuclear grading.;Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene.;Mutations and Chromosomal Rearrangements Affecting the Expression of Snail, a Gene Involved in Embryonic Patterning in DROSOPHILA MELANOGASTER.;Patterns of relapse and survival following radical mastectomy. Analysis of 716 consecutive patients;Local failure is responsible for the decrease in survival for patients with breast cancer treated with conservative surgery and postoperative radiotherapy.;E-cadherin mRNA expression in breast carcinomas correlates with overall and disease-free survival."
        ],
        [
         "42",
         "00309d5539a3db7b5bf065351c3b085e87a3b1f3",
         "https://jb.asm.org/content/jb/187/9/2948.full.pdf",
         "Biology,Medicine",
         "Journal of Bacteriology",
         "M. Cao,Charles M. Moore,J. Helmann",
         "144352095,36936788,2126233",
         "48",
         "0.0",
         "2005-05-01",
         "2005",
         "21a0391b16bfba4dd1ec6b1451db187c299c0c55;9f327bca0e5096f66fa906f802da94952f8e45d5;ae02c3273ba2f9f0e7aebdd27de80a4e91a564aa;0b70fa144ca906b2679b03dfabd523e54eec998e;7ad5d5c5c7c0075e0ab077e220eff99133675a64;231281e6c32f18da451360ef1c45653ef3eba76c;40597683823edf006b0c60130641e43b243e44bd;91d771131618194090b73a343c034191b681bf67;24035a56c47f48c3b50082e288816cfe863cce10;d921c135e1f9aca08057d7658c3c6f6c45db3789;6f9c032f3922339d2118adaac0cf625c1012972a;22e1e16de8054c04426c9b2d8760b8667bfc86f8;a3ef354966193cbb6b4b0bb3da048d3da2ac1cf0;60dfdb80d740da1fce7c5bb5c1a6f442dfc00528;d65c6830c897f878825478d833eb2b51031091f5;ce48cb03c907ffe1c636348c59850c9c18764d0e;724a0cb0239610b8c7e8f689ff435dc1f9d8a94d;ce7d6f9e4bdca60c39221e2577ab392988ae1a14;d1be3315921318c7ab4a14c973f098f856bb8f01;1f8a47d2392339ec4f7f8ed6109a4d6b59ef78ce;ee006c457feb833a03f05bb973bd424b5db7f1f1;5d4763271093a35e4dd88e900af8d076355fb2fd;53a57620b1daa9b4efbdd855e2892877f4f9d231;057e0ef90a3a312e5d97da50e4bb3c6ca078ecc7;76d935a97f4f34198281879a544b48b433828e8d;c89d2d78a3a0827b52e0e3c0bd99a43c61bbaef1;1dea7e87fc024322898533dddba45d7072f8503e;d3b8861b628bc574806cc24cfd4071fb40c909d8;eb2c22f1ff4b6af040a4569eb5c3d1ed0b7ef4c7;c984c08b7f030d02e1b451f19e3c9d48eab2d4bc;e191e013c2ca518e727212a1d7d91b4c3d746cec;6ae75779c47231af9276685c0bd339a5ef9609f2;44982b39a6f0006f4f0e7e49d92801fae908cf2e;232f0ddc40b5c2f47b4f0e92f1c6a7dbde637165;67d8736d2f328ca5e7d978edffa73a5a1f8a9247;b225032ce942696e38f481f92dc0529a9b812ef8;5f9aeab139cf9d9c1b3c5a80e80b71044ffe3dd3;f9d01a59cc8bf44af50055c8d88d25fb4929883a;40f56f83134d8f13ea753b249ff0cc2dde10e94c;d92afca0d27594b4fbb3554712f2531406055806;6d4ed4694c3512153428bb542cbbeaeebbd18a12;11762fd32ba5ba871d7b311e61643b3647b7164b;3c6fb08f4a5ce937d2fe3405b9a809ab35bde094;6b1003fb7556179dd87fbe3a84dfd2633b1109b3;004106a0ae81098983c2403bd3190a4ab10c478e;0d3d604ac94b5fd0f761e212401e10dbeabd5bc0;7f91cdb628be19c4def56b7812ab2af1787da1ea;fb412abafe778810cd3f72f5519cff6ee62c1c8b;7a4909f510fb4709d3ed176f795b05ae22560034;3746dae3f1b1bb6703440befd579cebebe642903;237341a0fdeb2f41392c59a4a68268ac66c16b11;099ed3e2061cbed0cfedd45f0dd4ebfa39a5cc96;2a6058a27e5f8002a97fc27257e096425993c77d;d0a38395eafcb2cb7209a74de398d19ed4f8f266;7394dae1544c770625d8ead56df63433ca2be85b",
         "Response of Bacillus subtilis to Nitric Oxide and the Nitrosating Agent Sodium Nitroprusside;The Bacillus subtilisσW anti‐sigma factor RsiW is degraded by intramembrane proteolysis through YluC;Spectroscopic description of the two nitrosyl-iron complexes responsible for fur inhibition by nitric oxide.;Spx-RNA Polymerase Interaction and Global Transcriptional Control during Oxidative Stress;Interaction of Bacillus subtilis extracytoplasmic function (ECF) sigma factors with the N-terminal regions of their potential anti-sigma factors.;σ Factors and Global Gene Regulation in Mycobacterium tuberculosis;Transcriptome and proteome analysis of Bacillus subtilis gene expression in response to superoxide and peroxide stress.;The pqrAB Operon Is Responsible for Paraquat Resistance in Streptomyces coelicolor;Spx-dependent global transcriptional control is induced by thiol-specific oxidative stress in Bacillus subtilis;RshA, an anti‐sigma factor that regulates the activity of the mycobacterial stress response sigma factor SigH;In Bacillus subtilis W23, the duet σXσM, two sigma factors of the extracytoplasmic function subfamily, are required for septum and wall synthesis under batch culture conditions;Regulation of the Bacillus subtilis Extracytoplasmic Function Protein σY and Its Target Promoters;SigM, an Extracytoplasmic Function Sigma Factor of Bacillus subtilis, Is Activated in Response to Cell Wall Antibiotics, Ethanol, Heat, Acid, and Superoxide Stress;Characterization of YqjM, an Old Yellow Enzyme Homolog from Bacillus subtilis Involved in the Oxidative Stress Response*;Global Characterization of Disulfide Stress in Bacillus subtilis;DNA microarray analysis of Bacillus subtilis sigma factors of extracytoplasmic function family.;A Bacitracin-Resistant Bacillus subtilis Gene Encodes a Homologue of the Membrane-Spanning Subunit of the Bacillus licheniformis ABC Transporter;The Global Transcriptional Response of Bacillus subtilis to Peroxide Stress Is Coordinated by Three Transcription Factors;Regulation of the Bacillus subtilis bcrC Bacitracin Resistance Gene by Two Extracytoplasmic Function σ Factors;The Salmonella enterica sv. Typhimurium smvA, yddG and ompD (porin) genes are required for the efficient efflux of methyl viologen;Antibiotics that inhibit cell wall biosynthesis induce expression of the Bacillus subtilisσW and σM regulons;'New uses for an Old Enzyme'--the Old Yellow Enzyme family of flavoenzymes.;Defining the Bacillus subtilis sigma(W) regulon: a comparative analysis of promoter consensus search, run-off transcription/macroarray analysis (ROMA), and transcriptional profiling approaches.;FosB, a Cysteine-Dependent Fosfomycin Resistance Protein under the Control of ςW, an Extracytoplasmic-Function ς Factor in Bacillus subtilis;Mutational analysis of RsrA, a zinc‐binding anti‐sigma factor with a thiol–disulphide redox switch;Alpha/Beta-hydrolase fold enzymes: structures, functions and mechanisms.;The role of the bacitracin ABC transporter in bacitracin resistance and collateral detergent sensitivity.;Resistance to Bacitracin as Modulated by an Escherichia coli Homologue of the Bacitracin ABC Transporter BcrC Subunit from Bacillus licheniformis;σM, an ECF RNA polymerase sigma factor of Bacillus subtilis 168, is essential for growth and survival in high concentrations of salt;σR, an RNA polymerase sigma factor that modulates expression of the thioredoxin system in response to oxidative stress in Streptomyces coelicolor A3(2);Purification and Preliminary Characterization of a Serine Hydrolase Involved in the Microbial Degradation of Polychlorinated Biphenyls*;Identification of target promoters for the Bacillus subtilis sigma X factor using a consensus-directed search.;Alkyl hydroperoxide reductase, catalase, MrgA, and superoxide dismutase are not involved in resistance of Bacillus subtilis spores to heat or oxidizing agents;The Bacillus subtilis sigma(X) protein is an extracytoplasmic function sigma factor contributing to survival at high temperature;PCR‐synthesis of marker cassettes with long flanking homology regions for gene disruptions in S. cerevisiae;Bacillus licheniformis bacitracin‐resistance ABC transporter: relationship to mammalian multidrug resistance;The methyl viologen-resistance-encoding gene smvA of Salmonella typhimurium;Analysis of the Streptomyces coelicolor sigE gene reveals the existence of a subfamily of eubacterial RNA polymerase sigma factors involved in the regulation of extracytoplasmic functions.;Mutations that relieve nutritional repression of the Bacillus subtilis dipeptide permease operon;Isolation and characterization of methyl viologen-sensitive mutants of Escherichia coli K-12;Role of AbrB in Spo0A- and Spo0B-dependent utilization of a sporulation promoter in Bacillus subtilis;Effects of salts on the lethality of paraquat;Biodegradation of paraquat dichloride.;Superoxide dismutase: improved assays and an assay applicable to acrylamide gels.;The extracytoplasmic function (ECF) sigma factors.;Antibiotics that inhibit cell wall biosynthesis induce expression of the Bacillus subtilis sigma(W) and sigma(M) regulons.;FosB, a cysteine-dependent fosfomycin resistance protein under the control of sigma(W), an extracytoplasmic-function sigma factor in Bacillus subtilis.;sigmaR, an RNA polymerase sigma factor that modulates expression of the thioredoxin system in response to oxidative stress in Streptomyces coelicolor A3(2).;Molecular cloning and nucleotide sequence of the superoxide dismutase gene and characterization of its product from Bacillus subtilis;Plasmids for ectopic integration in Bacillus subtilis.;The methyl viologen-resistance-encoding gene smvA of Salmonella typhimurium.;Integrational Vectors for Genetic Manipulation in Bacillus subtilis;Bacillus Subtilis and Other Gram-Positive Bacteria: Biochemistry, Physiology, and Molecular Genetics;Molecular biological methods for Bacillus;Use of transposons and integrational vectors for mutagenesis and construction of gene fusions in Bacillus species;Biodegradation of paraquat dichloride.;Experiments in molecular genetics"
        ],
        [
         "43",
         "003437ce9cbb8b1b43b45bed926c2e1a54a3d389",
         "https://www.nature.com/articles/pr2005710.pdf",
         "Biology,Medicine",
         "Pediatric Research",
         "B. Kramer,U. Kaemmerer,M. Kapp,Daniele Herbst,A. Marx,D. Berg,P. Groneck,C. Speer",
         "144440679,115294665,37420194,38854866,145186037,152469561,5245808,3501891",
         "50",
         "0.0",
         "2005-09-01",
         "2005",
         "",
         ""
        ],
        [
         "44",
         "00357f6991440ffb57022cdc2b7ab210c1ed63ec",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/ijc.21130",
         "Biology,Medicine",
         "International Journal of Cancer",
         "M. Kuokkanen,R. Butzow,H. Rasinperä,K. Mȩdrek,M. Nilbert,S. Malander,J. Lubiński,I. Järvelä",
         "6979516,152417084,12184971,3700472,4054056,5379508,5632238,3974297",
         "38",
         "1.0",
         "2005-10-20",
         "2005",
         "",
         ""
        ],
        [
         "45",
         "00376fbb6ade96f6daf3b760897fc030dfb1a7aa",
         "https://europepmc.org/articles/pmc1087915?pdf=render",
         "Biology,Medicine",
         "Proceedings of the National Academy of Sciences of the United States of America",
         "D. Langenau,H. Feng,S. Berghmans,J. Kanki,J. Kutok,A. Look",
         "6269963,144021289,3549403,6122783,4661510,145348521",
         "241",
         "10.0",
         "2005-04-12",
         "2005",
         "a65920b90e3a51e5d8fec2905c28756ffd4de591;18d8660a3df9a0bcc0a17a5d6738de4791c0fe48;05642b715dc8a2e5b0419796e2a73cf399feabdf;2dd6f6304825aee5d0cdcb1b30ac1bf6d6b3ce09;3be8c6bc670b7db556e08b52bc7de92e647fa52f;2b5523ac3224550307c21c248c2b3ab0306cd729;fe2a8c5e151feedd307ce3de106e871a67eddab6;0a075766ada3cea70469852df58abe27644d9c0d;31f80148922aa4373537ed50b4f9b605d93ab41d;c4637a20c4eba4fc71084907edd661dc62e59379;e6ae3be94e375b255725e70dbad9dd8aaadd4495;d4171b7d92f5303d83649f1c7fdb240ca2f74d63;6e2b7ac82811e941cd3a32a0143fdd72d853d622;22999e5d2f3ca346a5210affc487f9aa3262275f;acd36b17c486957cebacc3ad68bd83ed417bf9cc;5b894af3465b65e67bc61fb686253b0889d289f9;af9c6b21564579fea18f72859fd93b10120061eb;c7bb60ebdc1bf1c63103cf3d20f4b2184e34c0a8;b90a5a71958248f0a793b49bd9c5f1def6d03e38;6e5fed71cc1ab3ecdf7ecd6ef67644af6f3fe161;1f0e357c124f6dd024aea1e0e0bd0f43fa549c49;a703fd2cdb6ff7336de0bfaeb04fb7111d571505;f2e7f4edb273288e40c51734eac22afb2d613825;5787fead3b2e49da44474c9029ed6c950cd8b189;7ee166a0c266ffdf11435fc5893c50fac44c6f5e;ec15caaf364dfebee374d6939205fbe2cee56bae;66a75156226cc6e4dfe2672c1fc24004ef854e4b;c445bce364fb50abafc2284666a31750b08a1a6f;3dbc3bf962fb4bce2aaeb342b36b892d40ec9a74;e4ca573284a45ecccb9380b6d6c010d980f6735f;d18e6e79601ebd339da2d20a5a27e31a14bdae1c;0442b2d4da6b9584431bd4a6831c66ecd07f23f9;32c95d6e1e2379c2cda75a7e534b013d79f56b27;8732d72c6656c5b25c527bfe2b90473e33475e30;46a8aa95ce6abd5057e835ce90090314376b06f1;9054f8c8b088df5c5c4914d6e6acc4ff9e9c8e30;b85376ba4333d820ce5e5fbda47a6d28e1276277;4687fef6e9508778ae704aa3821e2c8ccdea85d5;177bf3f156c53edd18a394c74fa314a8fc3e4b14;a717606c3a35d3f56e741995d2f2a372300877f5;e57d3dd011c04f63c338000466e5b4f9d3eca6e8",
         "Suppression of apoptosis by bcl-2 overexpression in lymphoid cells of transgenic zebrafish.;tp53 mutant zebrafish develop malignant peripheral nerve sheath tumors.;Activating Mutations of NOTCH1 in Human T Cell Acute Lymphoblastic Leukemia;In vivo tracking of T cell development, ablation, and engraftment in transgenic zebrafish.;Chemical suppression of a genetic mutation in a zebrafish model of aortic coarctation;Biallelic transcriptional activation of oncogenic transcription factors in T-cell acute lymphoblastic leukemia.;Prognostic importance of TLX1 (HOX11) oncogene expression in adults with T-cell acute lymphoblastic leukaemia;Gene expression profiling in T-cell acute lymphoblastic leukemia.;Expression of HOX11 in childhood T-lineage acute lymphoblastic leukaemia can occur in the absence of cytogenetic aberration at 10q24: a study from the Children's Cancer Group (CCG);Tagging muscle cell lineages in development and tail regeneration using Cre recombinase in transgenic Xenopus.;The UMD‐p53 database: New mutations and analysis tools;Myc-Induced T Cell Leukemia in Transgenic Zebrafish;Molecular cloning and developmental expression of Tlx (Hox11) genes in zebrafish (Danio rerio);HOX11L2 expression defines a clinical subtype of pediatric T-ALL associated with poor prognosis.;The IARC TP53 database: New online mutation analysis and recommendations to users;Insertional mutagenesis in zebrafish rapidly identifies genes essential for early vertebrate development;Classification, subtype discovery, and prediction of outcome in pediatric acute lymphoblastic leukemia by gene expression profiling.;Gene expression signatures define novel oncogenic pathways in T cell acute lymphoblastic leukemia.;One INK4 gene and no ARF at the Fugu equivalent of the human INK4A/ARF/INK4B tumour suppressor locus;Convergence of distinct pathways to heart patterning revealed by the small molecule concentramide and the mutation heart-and-soul;A new recurrent and specific cryptic translocation, t(5,14)(q35,q32), is associated with expression of the Hox11L2 gene in T acute lymphoblastic leukemia;FLP and Cre recombinase function in Xenopus embryos.;Concurrent expression of recombination activating genes 1 and 2 in zebrafish olfactory sensory neurons;Promoter demethylation accompanies reactivation of the HOX11 proto‐oncogene in leukemia;Small molecule developmental screens reveal the logic and timing of vertebrate development.;Clinical implications of recurring chromosomal and associated molecular abnormalities in acute lymphoblastic leukemia.;Characterization of three isotypes of immunoglobulin light chains and T-cell antigen receptor α in zebrafish;The t(14,21)(q11.2,q22) chromosomal translocation associated with T-cell acute lymphoblastic leukemia activates the BHLHB1 gene.;Disruption of the ARF-Mdm2-p53 tumor suppressor pathway in Myc-induced lymphomagenesis.;Myc signaling via the ARF tumor suppressor regulates p53-dependent apoptosis and immortalization.;Oncogenic transcription factors in the human acute leukemias.;The identification of genes with unique and essential functions in the development of the zebrafish, Danio rerio.;A genetic screen for mutations affecting embryogenesis in zebrafish.;Inducible gene targeting in mice;Frequent deletion of p16INK4a/MTS1 and p15INK4b/MTS2 in pediatric acute lymphoblastic leukemia.;Clinical significance of p53 mutations in relapsed T-cell acute lymphoblastic leukemia.;Deletion of a DNA polymerase beta gene segment in T cells using cell type-specific gene targeting.;Infrequent mutations in the P53 gene in primary human T-cell acute lymphoblastic leukemia.;RAG-1 and RAG-2, adjacent genes that synergistically activate V(D)J recombination.;The V(D)J recombination activating gene, RAG-1;Multiple recurrent genomic defects in follicular lymphoma. A possible model for cancer."
        ],
        [
         "46",
         "003800d8d89dfba1f08640cd3aa49c844956a22f",
         "https://www.jneurosci.org/content/jneuro/25/17/4330.full.pdf",
         "Biology",
         "Journal of Neuroscience",
         "Weiqi Zhang,A. Rohlmann,V. Sargsyan,G. Aramuni,R. Hammer,T. Südhof,M. Missler",
         "2028237355,34658898,143907507,6138592,90422041,4704732,5496005",
         "137",
         "1.0",
         "2005-04-27",
         "2005",
         "b790b21cacc3c115d6c046bfabd060cbad25a673;cb0278101e241da456f22b0b0829f013ebfe3f86;46da5ca41d406fe12ab4623dfcfff123f9adfd01;ff7382c0da0dee8b29e4efae3eecbae8bf32106d;6221285a5663cb7e84b645375f6c1a7dbf0cfeca;4b93eea1b5bb1fd30d9202758463386e4406706d;bff200f6aec36eee2990e88f3c1ea8260a434ed4;1a22dc11984db9bc2bb3c0714b0a8cbd85115c3d;fa4a3f10db2ffec2ccb32484e410802d2d02e908;03cd7d140256489b2730aab31325229e0759fce6;7966395d4c7766c579d0f29bfab0bc4ec8cb8465;ccd2842b2a4f8cb1e8041495ef549d69ae73e760;7a6b4411eba71509155e6b193a67e8187fdc9b3c;e4e3f9a6ee1384e72f623b5e3cb1ac69b0e3c50e;7a570435b461204057c29fb91a57b60f95e8a80a;952ca7eb5e7e789c023e880bf86f3ef0c7ac65e6;de97c42235518ccf7082e6f01d15a09cfd11cf81;efc04ff16ad89a15ad9767c3e6508cf3a96c47a5;9ff63153988379b6206c59a4fd66acab990f7acc;7872dd9199cf265c66771399b1af1ad24bb1f4f4;b0bb0cdee8166a43b9d4acab39592f967ec0322c;5861abf1b1166143bd276d1fbdd52095347137a8;0fee7f15bb7d5813de71b2951c5d118fbcdd4832;c54c23a15c307715e899aa1d7ec398e8cb98caf9;3b3f373d04f5c6dda23294b5e8559d9124ae1edb;0b594a97d18e0acf1a17e5e13b5dab91401cb565;faef3ed6624730160517e48fb2eb810699147bfa;ba7d82ba518c836b2a80fabc39a59cc55e923802;e499f3a53cffbcac2239419dee1586391423412d;56ed4d8823300a900e083c9039ccfca6ca3b89aa;39d856bb689ed3ffbe38d82221c59d64ea3f5651;fbda846eb3fafbf780a408fffba917952d41ba90;df20509f0ab3a0eec5f1e7730a65def36097b28c;6427d81b67f5958136fefe8757a82eb39665c500;f931a6d6a3c0d730c468203eb44bd5565e09bf1e;e914817d16c133427baac039bfd4d7ff8c6b3c9b;1b4cb06ada513363c41f37d5ecb395c5570332c8;894ded51d4931ebc47bc2d845ec00604d5e26352;8fb409b3c108b0ce4e38cf0d847b6483c9b7ade2;26256ae4cf6134048181bc1758343a10cc6cc0dd;22b476dc1bc562e670dbd4f015bd9e2a58af4d40;d77b13539deabb22bcaae1215ce032353b4bf09e;b79a4b0cc84243fe6bfde71480b713b26280aa96;479848ea3d3a699d25c5084d1c91587ae7aab865;0826bdd2d88ee13279adea6c69d665917356839d;48b2694d75a300dccc825cd32b56a81f5ce570a3;d6c7f4c65a02781488594e7a56ffe08a33e9ed38;9886bf6643514b971c88929bf9789de30ef6e23b;9f2bce56145c5f458ba8ac95dc4375c9d51066d7;b5ad20df307643b864d175a9ced00277dc28b053;29c1657abdffbdea8805e546ea8788c24c9049c1;feee9ad07b5b8dd1df28209a8ef39de6f059b378;128df89abcc575ecf797e8c01a46a04d46bf3d70;c6d1f83211e357faa68113724073d95d771a632e;4cf4b35ef9e0f0d9feb98bf3bb23d89cb59d9c77;c6f71fae30449c448155355e2316ea45402c8a5c",
         "Selective Capability of SynCAM and Neuroligin for Functional Synapse Assembly;Neurexins Induce Differentiation of GABA and Glutamate Postsynaptic Specializations via Neuroligins;Cell-cell signaling during synapse formation in the CNS.;CaV1.3 Channels Are Essential for Development and Presynaptic Activity of Cochlear Inner Hair Cells;Neurexin mediates the assembly of presynaptic terminals;α-Neurexins couple Ca2+ channels to synaptic vesicle exocytosis;Synaptic cell adhesion goes functional;Probing the Intracellular Calcium Sensitivity of Transmitter Release during Synaptic Facilitation;Gamma Protocadherins Are Required for Survival of Spinal Interneurons;Dual regulation of voltage-gated calcium channels by PtdIns(4,5)P2;Sense and sensibility in the regulation of voltage-gated Ca2+ channels;Impairment of L-type Ca2+ Channel-Dependent Forms of Hippocampal Synaptic Plasticity in Mice Deficient in the Extracellular Matrix Glycoprotein Tenascin-C;Cadherin Regulates Dendritic Spine Morphogenesis;Structure and evolution of neurexin genes: insight into the mechanism of alternative splicing.;Analysis of the human neurexin genes: alternative splicing and the generation of protein diversity.;A stoichiometric complex of neurexins and dystroglycan in brain;Regulated Endocytic Routing Modulates Wingless Signaling in Drosophila Embryos;Functional and physical coupling of voltage‐sensitive calcium channels with exocytotic proteins: ramifications for the secretion mechanism;Calcium dynamics associated with action potentials in single nerve terminals of pyramidal cells in layer 2/3 of the young rat neocortex;Intracellular calcium dependence of transmitter release rates at a fast central synapse;Calcium sensitivity of glutamate release in a calyx-type terminal.;Shared and Unique Roles of Cap23 and Gap43 in Actin Regulation, Neurite Outgrowth, and Anatomical Plasticity;Neuroligin Expressed in Nonneuronal Cells Triggers Presynaptic Development in Contacting Axons;Postnatal development of GABAB receptor‐mediated modulation of voltage‐activated Ca2+ currents in mouse brain‐stem neurons;Genetic dissection of endocytic trafficking in Drosophila using a horseradish peroxidase-bride of sevenless chimera: hook is required for normal maturation of multivesicular endosomes.;Interactions of Presynaptic Ca2+ Channels and Snare Proteins in Neurotransmitter Release;Neurexins Are Functional α-Latrotoxin Receptors;Neuroligin 1 is a postsynaptic cell-adhesion molecule of excitatory synapses.;Calcium Channel Types with Distinct Presynaptic Localization Couple Differentially to Transmitter Release in Single Calyx-Type Synapses;Neurexophilin binding to alpha-neurexins. A single LNS domain functions as an independently folding ligand-binding unit.;Target cell-specific modulation of transmitter release at terminals from a single axon.;A Tripartite Protein Complex with the Potential to Couple Synaptic Vesicle Exocytosis to Cell Adhesion in Brain;Target-cell-specific facilitation and depression in neocortical circuits;Differential signaling via the same axon of neocortical pyramidal neurons.;N- and P/Q-Type Ca2+ Channels Mediate Transmitter Release with a Similar Cooperativity at Rat Hippocampal Autapses;R-type Ca2+ currents evoke transmitter release at a rat central synapse.;Target-specific expression of presynaptic mossy fiber plasticity.;Neurexin Iα Is a Major α-Latrotoxin Receptor That Cooperates in α-Latrotoxin Action*;CASK: a novel dlg/PSD95 homolog with an N-terminal calmodulin-dependent protein kinase domain identified by interaction with neurexins;Expression of APP in transgenic mice: a Comparison of neuron-specific promoters;Neuroligin 1: A splice site-specific ligand for β-neurexins;Exocytotic Ca2+ channels in mammalian central neurons;Conserved domain structure of beta-neurexins. Unusual cleaved signal sequences in receptor-like neuronal cell-surface proteins.;Inactivation of the N-CAM gene in mice results in size reduction of the olfactory bulb and deficits in spatial learning;The probability of transmitter release at a mammalian central synapse;Genetic deletion of a neural cell adhesion molecule variant (N-CAM-180) produces distinct defects in the central nervous system;Nonuniform probability of glutamate release at a hippocampal synapse.;Neurexin III alpha: extensive alternative splicing generates membrane-bound and soluble forms.;Neurexins: synaptic cell surface proteins related to the alpha-latrotoxin receptor and laminin.;Affinity purification of antibodies from diazotized paper blots of heterogeneous protein samples.;Determinants of G protein inhibition of presynaptic calcium channels;alpha-Latrotoxin and its receptors: neurexins and CIRL/latrophilins.;Structure and regulation of voltage-gated Ca2+ channels.;Neurexins: three genes and 1001 products.;Neurexin I alpha is a major alpha-latrotoxin receptor that cooperates in alpha-latrotoxin action.;Neuroligin 1: a splice site-specific ligand for beta-neurexins.;Mints as adaptors. Direct binding to neurexins and recruitment of munc18"
        ],
        [
         "47",
         "003bffead36371a8f49d13dbbc128130d1d9bd90",
         "http://www.jbc.org/article/S0021925820692419/pdf",
         "Biology,Medicine",
         "Journal of Biological Chemistry",
         "K. Toth,J. Mazurkiewicz,K. Rippe",
         "49457210,2056367601,2408430",
         "48",
         "8.0",
         "2005-04-22",
         "2005",
         "4329c6f6babfb8925a4e0a7d8dfba6fd02cb991c;fed01679f6eaa4a4ce3e4be98987a1a0961c7096;26550f2e86b7a1d0ef9e281381756ab19532c2f4;c01b571f85838c1f8653ddf3d9e7a22f244363a7;9bb73d97dced4284368fe5cf7207ca1d6c317391;0e045753580a44eb6108d2161d2f6e7821b8aeee;a677da02b0d10c31efec3f20343e36bfcf77d735;225844802e76bb7cfa814b2895ea551789150afe;5cbc859c0b71ec873c8e3524b8f455a3ab206694;c9cdcf2e525247e53ab4e549af67d5244c75ad94;108f745dd8a6751880ceeccb37f3b9d436d692d9;22bb3871f69ede06d924dea6ebf6fff1aeaa295e;0a47ce160a1e79c6be42a798bd8d4badb85e14ea;7a6704079bc4e862a18c4de4940e057bdc032f1d;a056600c5598f8e09aebc8da8544c1d20704c267;a5ff4937b6f8b8f723aed91d977ffb2294510de3;e8b853b9ce4e26ae75f751b62882d96f0cfda19b;8a403d60d51dec65a47b55d590f9e0c3beb39bab;6c2e21d747612c55069d4542d101af5974976a90;07975bba7a9f710c3ce23d90953d180faa323d60;408455ae2c3176d193bde479c67aeab403b68fe8;31e8b4cec5f1e509a3983690a31e6d8bcfd9b466;3a04d4393d6ec0babe3d38a21937e0c8f049f836;b12f56a5c596d7597e2371db787d3d388e0cfcf7;7a68077de33d92b91d3bb4821cca0b1e4302b66a;6ef26557a173a2504cf4baa9871c2f812fc48ada;e50c229c549a1daaf36e6ca4001dec3e2e10ee80;21a72a6834c0c19316ee171363fe6ba6cae35d86;b64a452db3b22a37461bb1cbed4870b5e48c2e7a;a2ab6fc4a8f020508605aca53d6133ad44a05ba7;22fda6fe1df6680e3ea6f855c1fc508e0d7eaa8c;c84c48e95aa4aefaaa4432f042653d35fe1eb7c6;7d110aab354a185aea7fbf2a4d5803783b98ea82;5c2bc44a1874cdbe181d69a235bb038f35e5f5bd;fa7e70b5ee68ebf657e92618310483aab46f870b;555d86f630cd40769b943ba1e9d08f80228e8f47;98506d195600e23a060aa39ba88bf193dd9b6874;06c4dd54335bc2f6235d8fb8748428a77e7b416c;1a4d35412117b68655dfcabe150674cace542d87;691e5f4f3a297334d4dd4d11dd18df8dc5289ba3;ebc2d9fdf7db04104a77a2a7a5d23c2952d67ebc;05bd59f66abdfdb194af8f01418246ba2c1c59ab;399517eb7617169039c09cdc652351bcdf9e6861;a8595112c58293554ae09ab21d109b442e2d9bbd;7a2596d7acc844021326c49b54527dad2b964f3f;0ff08ca8bf795fdc786821e7c42d8a34c60143b3;790747b08c7e10760cb82d59ce3ec8a5a1466cde;4e0d882e10b4e01c24d2df80b0206e461ac391a5;5a8d112396e3b0f5b060a659f6a98a462125d1fd;8b88df2b71bf4b8bbf25bd79fdfaaadb9864bf95;4e546daf7ff921e3cb64c1ec07bd15bdcc9d35c5;55ab82851153b7db17fbc85b263c7d53c0810d23;dc04e5d72408462e98b00cfc41d524d879d6d4f3;7e995c11afd7539be6880e141d8ad2013fd8c61f;67c4b0ff4b48d6d66ddafbfc72d4c4d91bbb15d1;9034a3a93e60d4e449b3a5d4ec2b39688c8a28ac;048c42c2a9c03d22084c2acf33f222b8e3a61e60;814860fddff2b6d2363af2032d02afe8ef3af17f",
         "Self-association of the yeast nucleosome assembly protein 1.;Nucleoplasmin: a nuclear chaperone.;Histone chaperones, a supporting role in the limelight.;Histone release during transcription: NAP1 forms a complex with H2A and H2B and facilitates a topologically dependent release of H3 and H4 from the nucleosome.;Chromatin remodeling by RNA polymerases.;ATP-Driven Exchange of Histone H2AZ Variant Catalyzed by SWR1 Chromatin Remodeling Complex;Histone H3.1 and H3.3 Complexes Mediate Nucleosome Assembly Pathways Dependent or Independent of DNA Synthesis;Conformation of reconstituted mononucleosomes and effect of linker histone H1 binding studied by scanning force microscopy.;Histone H2A/H2B dimer exchange by ATP-dependent chromatin remodeling activities.;Counting nucleosomes in living cells with a combination of fluorescence correlation spectroscopy and confocal imaging.;Preferential Binding of the Histone (H3-H4)2 Tetramer by NAP1 Is Mediated by the Amino-terminal Histone Tails*;Global analysis of protein expression in yeast;On the analysis of protein self-association by sedimentation velocity analytical ultracentrifugation.;Involvement of Nucleocytoplasmic Shuttling of Yeast Nap1 in Mitotic Progression;FACT Facilitates Transcription-Dependent Nucleosome Alteration;Chromatin assembly by DNA-translocating motors;Genome-wide expression analysis of NAP1 in Saccharomyces cerevisiae.;Histone chaperones and nucleosome assembly.;Regulation of biosynthesis and intracellular localization of rice and tobacco homologues of nucleosome assembly protein 1;Modern analytical ultracentrifugation in protein science: A tutorial review;Solvent mediated interactions in the structure of the nucleosome core particle at 1.9 a resolution.;Cell cycle-dependent assembly of a Gin4-septin complex.;A polymer model for large-scale chromatin organization in lower eukaryotes.;Dual Roles of p300 in Chromatin Assembly and Transcriptional Activation in Cooperation with Nucleosome Assembly Protein 1 In Vitro;Chromatin assembly. Cooperation between histone chaperones and ATP-dependent nucleosome remodeling machines.;The crystal structure of nucleoplasmin-core: implications for histone binding and nucleosome assembly.;Multistep Chromatin Assembly on Supercoiled Plasmid DNA by Nucleosome Assembly Protein-1 and ATP-utilizing Chromatin Assembly and Remodeling Factor*;Kinetics of Core Histones in Living Human Cells;Asymmetries in the nucleosome core particle at 2.5 A resolution.;p300-mediated acetylation facilitates the transfer of histone H2A-H2B dimers from nucleosomes to a histone chaperone.;Polyglutamylation of Nucleosome Assembly Proteins*;Anomalous diffusion of fluorescent probes inside living cell nuclei investigated by spatially-resolved fluorescence correlation spectroscopy.;NAP-2: histone chaperone function and phosphorylation state through the cell cycle.;ACF consists of two subunits, Acf1 and ISWI, that function cooperatively in the ATP-dependent catalysis of chromatin assembly.;Chromatin assembly: biochemical identities and genetic redundancy.;Association states of the transcription activator protein NtrC from E. coli determined by analytical ultracentrifugation.;Assembly, Remodeling, and Histone Binding Capabilities of Yeast Nucleosome Assembly Protein 1*;Characterization of nucleosome core particles containing histone proteins made in bacteria.;Crystal structure of the nucleosome core particle at 2.8 Å resolution;Functional characterization of human nucleosome assembly protein-2 (NAP1L4) suggests a role as a histone chaperone.;The N-terminal tail of histone H2A binds to two distinct sites within the nucleosome core.;Histones in transit: cytosolic histone complexes and diacetylation of H4 during nucleosome assembly in human cells.;Drosophila NAP-1 is a core histone chaperone that functions in ATP-facilitated assembly of regularly spaced nucleosomal arrays;Stimulation of transcription factor binding and histone displacement by nucleosome assembly protein 1 and nucleoplasmin requires disruption of the histone octamer;A histone‐binding protein, nucleoplasmin, stimulates transcription factor binding to nucleosomes and factor‐induced nucleosome disassembly.;Functional analysis of nucleosome assembly protein, NAP-1. The negatively charged COOH-terminal region is not necessary for the intrinsic assembly activity.;The nucleosomal core histone octamer at 3.1 A resolution: a tripartite protein assembly and a left-handed superhelix.;Associative behavior of the histone (H3-H4)2 tetramer: dependence on ionic environment.;Purification and characterization of CAF-I, a human cell factor required for chromatin assembly during DNA replication in vitro;Rapid purification of nucleosome assembly protein (AP-I) and production of monoclonal antibodies against it.;Purification and initial characterization of a protein which facilitates assembly of nucleosome-like structure from mammalian cells.;Regulation of cell size in the yeast Saccharomyces cerevisiae;The Sda1 protein is required for passage through start.;Preparation of nucleosome core particle from recombinant histones.;Modern applications of analytical ultracentrifugation.;Handbook of fluorescent probes and research chemicals;Analytical ultracentrifugation in biochemistry and polymer science;Macromolecular crowding : obvious but underappreciated;hp://w w w .jb.org/ D"
        ],
        [
         "48",
         "003e9dc9c9aadce5d8504e376cd2d60a37563b17",
         "https://europepmc.org/articles/pmc1135011?pdf=render",
         "Biology,Medicine",
         "Biochemical Journal",
         "J. D. Leonard,F. Lin,M. Milla",
         "13991395,2113696401,4009920",
         "66",
         "2.0",
         "2005-05-01",
         "2005",
         "",
         ""
        ],
        [
         "49",
         "004051ae2ab180bb719c695daae098eabb8a2afd",
         "http://www.jbc.org/article/S0021925820615307/pdf",
         "Medicine,Biology",
         "Journal of Biological Chemistry",
         "E. Park,Joseph M. Lee,Jaime D. Blais,J. Bell,J. Pelletier",
         "47543639,2108300261,39423183,2750710,48465443",
         "32",
         "3.0",
         "2005-06-03",
         "2005",
         "",
         ""
        ]
       ],
       "shape": {
        "columns": 12,
        "rows": 2117
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>fieldsOfStudy</th>\n",
       "      <th>venue</th>\n",
       "      <th>authorName</th>\n",
       "      <th>authorId</th>\n",
       "      <th>citationCount</th>\n",
       "      <th>influentialCitationCount</th>\n",
       "      <th>publicationDate</th>\n",
       "      <th>publicationYear</th>\n",
       "      <th>referenceIds</th>\n",
       "      <th>referenceTitles</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>https://aacrjournals.org/clincancerres/article...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Clinical Cancer Research</td>\n",
       "      <td>T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...</td>\n",
       "      <td>14839966,10239302,2000448727,3201672,7147569,2...</td>\n",
       "      <td>88</td>\n",
       "      <td>6.0</td>\n",
       "      <td>2005-04-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>http://www.merl.com/publications/docs/TR2005-0...</td>\n",
       "      <td>Computer Science,Mathematics</td>\n",
       "      <td>IEEE Transactions on Communications</td>\n",
       "      <td>Juntan Zhang,M. Fossorier</td>\n",
       "      <td>3193350,48891339</td>\n",
       "      <td>319</td>\n",
       "      <td>26.0</td>\n",
       "      <td>2005-03-07</td>\n",
       "      <td>2005</td>\n",
       "      <td>17d0afe4f814e5abc6252e572e3b039082bf6f07;02464...</td>\n",
       "      <td>Low-Density Parity-Check Codes;Turbo decoder a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>https://www.jstage.jst.go.jp/article/bpb/28/8/...</td>\n",
       "      <td>Medicine</td>\n",
       "      <td>Biological and Pharmaceutical Bulletin</td>\n",
       "      <td>Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...</td>\n",
       "      <td>11809650,6617669,123635715,11305673,2152146096...</td>\n",
       "      <td>24</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2005-08-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/pdfdirect/...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Developmental Dynamics</td>\n",
       "      <td>Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....</td>\n",
       "      <td>35690923,2114125286,2113252574,2134773,3658233</td>\n",
       "      <td>53</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2005-01-01</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820589394/pdf</td>\n",
       "      <td>Chemistry,Medicine</td>\n",
       "      <td>Journal of Biological Chemistry</td>\n",
       "      <td>M. Westfall,A. M. Lee,D. Robinson</td>\n",
       "      <td>5025983,49249911,48706510</td>\n",
       "      <td>70</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2005-12-16</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2112</th>\n",
       "      <td>0a32f6f23c05827e466580647467a322b7db9f7d</td>\n",
       "      <td>https://hal.inria.fr/inria-00548510/file/lana_...</td>\n",
       "      <td>Mathematics,Computer Science</td>\n",
       "      <td>Tenth IEEE International Conference on Compute...</td>\n",
       "      <td>Svetlana Lazebnik,C. Schmid,J. Ponce</td>\n",
       "      <td>1749609,2462253,144189388</td>\n",
       "      <td>214</td>\n",
       "      <td>15.0</td>\n",
       "      <td>2005-10-17</td>\n",
       "      <td>2005</td>\n",
       "      <td>72bf4b2ce534b95bc24118491dbc4f8d550734a2;8c04f...</td>\n",
       "      <td>A sparse texture representation using local af...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2113</th>\n",
       "      <td>0a3370922b04f1789a8ccfa57ae8f11188b0d378</td>\n",
       "      <td>https://www.pnas.org/doi/pdf/10.1073/pnas.0506...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Proceedings of the National Academy of Science...</td>\n",
       "      <td>S. Lau,P. Woo,K. S. Li,Yi Huang,H. Tsoi,Beatri...</td>\n",
       "      <td>145931871,144306257,145781175,46844089,6780849...</td>\n",
       "      <td>1496</td>\n",
       "      <td>108.0</td>\n",
       "      <td>2005-09-27</td>\n",
       "      <td>2005</td>\n",
       "      <td>4e91f5fa9098067a1de83a61af810b263e40ff9a;82bfa...</td>\n",
       "      <td>Antigenic Cross-Reactivity between Severe Acut...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2114</th>\n",
       "      <td>0a37fc79e27f8f4a15cf83d126bd1c1f01b5d573</td>\n",
       "      <td>https://oem.bmj.com/content/oemed/62/11/793.fu...</td>\n",
       "      <td>Biology,Medicine</td>\n",
       "      <td>Occupational and Environmental Medicine</td>\n",
       "      <td>Roel C. H. Vermeulen,H. Kromhout</td>\n",
       "      <td>2239539881,2581089</td>\n",
       "      <td>20</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2005-10-18</td>\n",
       "      <td>2005</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2115</th>\n",
       "      <td>0a38af2050b6e1ce5f20cc9df4d6e64b4af05c95</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820565265/pdf</td>\n",
       "      <td>Chemistry,Medicine</td>\n",
       "      <td>Journal of Biological Chemistry</td>\n",
       "      <td>R. Jacobs,L. M. Stead,C. Devlin,I. Tabas,M. Br...</td>\n",
       "      <td>40605588,32493479,39055507,2073142235,1690700,...</td>\n",
       "      <td>94</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2005-08-05</td>\n",
       "      <td>2005</td>\n",
       "      <td>71257e319d4522d5125024bea18dc79398c1d456;36e6a...</td>\n",
       "      <td>Targeted Deletion of Hepatic CTP:phosphocholin...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2116</th>\n",
       "      <td>0a3a0a1df691f2d0d3646bedd9dc48f208a3843a</td>\n",
       "      <td>http://www.cell.com/article/S0960982205008250/pdf</td>\n",
       "      <td>Medicine,Biology</td>\n",
       "      <td>Current Biology</td>\n",
       "      <td>H. Berg</td>\n",
       "      <td>46826168</td>\n",
       "      <td>39</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2005-08-09</td>\n",
       "      <td>2005</td>\n",
       "      <td>0fca691640f44c44151ad05c5dd9fcaeed4711d9;7ad17...</td>\n",
       "      <td>Species Diversity and Genetic Diversity: Paral...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2117 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                       paperId  \\\n",
       "0     0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1     000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2     000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3     0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4     00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "...                                        ...   \n",
       "2112  0a32f6f23c05827e466580647467a322b7db9f7d   \n",
       "2113  0a3370922b04f1789a8ccfa57ae8f11188b0d378   \n",
       "2114  0a37fc79e27f8f4a15cf83d126bd1c1f01b5d573   \n",
       "2115  0a38af2050b6e1ce5f20cc9df4d6e64b4af05c95   \n",
       "2116  0a3a0a1df691f2d0d3646bedd9dc48f208a3843a   \n",
       "\n",
       "                                          openAccessPdf  \\\n",
       "0     https://aacrjournals.org/clincancerres/article...   \n",
       "1     http://www.merl.com/publications/docs/TR2005-0...   \n",
       "2     https://www.jstage.jst.go.jp/article/bpb/28/8/...   \n",
       "3     https://onlinelibrary.wiley.com/doi/pdfdirect/...   \n",
       "4      http://www.jbc.org/article/S0021925820589394/pdf   \n",
       "...                                                 ...   \n",
       "2112  https://hal.inria.fr/inria-00548510/file/lana_...   \n",
       "2113  https://www.pnas.org/doi/pdf/10.1073/pnas.0506...   \n",
       "2114  https://oem.bmj.com/content/oemed/62/11/793.fu...   \n",
       "2115   http://www.jbc.org/article/S0021925820565265/pdf   \n",
       "2116  http://www.cell.com/article/S0960982205008250/pdf   \n",
       "\n",
       "                     fieldsOfStudy  \\\n",
       "0                 Biology,Medicine   \n",
       "1     Computer Science,Mathematics   \n",
       "2                         Medicine   \n",
       "3                 Biology,Medicine   \n",
       "4               Chemistry,Medicine   \n",
       "...                            ...   \n",
       "2112  Mathematics,Computer Science   \n",
       "2113              Biology,Medicine   \n",
       "2114              Biology,Medicine   \n",
       "2115            Chemistry,Medicine   \n",
       "2116              Medicine,Biology   \n",
       "\n",
       "                                                  venue  \\\n",
       "0                              Clinical Cancer Research   \n",
       "1                   IEEE Transactions on Communications   \n",
       "2                Biological and Pharmaceutical Bulletin   \n",
       "3                                Developmental Dynamics   \n",
       "4                       Journal of Biological Chemistry   \n",
       "...                                                 ...   \n",
       "2112  Tenth IEEE International Conference on Compute...   \n",
       "2113  Proceedings of the National Academy of Science...   \n",
       "2114            Occupational and Environmental Medicine   \n",
       "2115                    Journal of Biological Chemistry   \n",
       "2116                                    Current Biology   \n",
       "\n",
       "                                             authorName  \\\n",
       "0     T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...   \n",
       "1                             Juntan Zhang,M. Fossorier   \n",
       "2     Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...   \n",
       "3     Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....   \n",
       "4                     M. Westfall,A. M. Lee,D. Robinson   \n",
       "...                                                 ...   \n",
       "2112               Svetlana Lazebnik,C. Schmid,J. Ponce   \n",
       "2113  S. Lau,P. Woo,K. S. Li,Yi Huang,H. Tsoi,Beatri...   \n",
       "2114                   Roel C. H. Vermeulen,H. Kromhout   \n",
       "2115  R. Jacobs,L. M. Stead,C. Devlin,I. Tabas,M. Br...   \n",
       "2116                                            H. Berg   \n",
       "\n",
       "                                               authorId  citationCount  \\\n",
       "0     14839966,10239302,2000448727,3201672,7147569,2...             88   \n",
       "1                                      3193350,48891339            319   \n",
       "2     11809650,6617669,123635715,11305673,2152146096...             24   \n",
       "3        35690923,2114125286,2113252574,2134773,3658233             53   \n",
       "4                             5025983,49249911,48706510             70   \n",
       "...                                                 ...            ...   \n",
       "2112                          1749609,2462253,144189388            214   \n",
       "2113  145931871,144306257,145781175,46844089,6780849...           1496   \n",
       "2114                                 2239539881,2581089             20   \n",
       "2115  40605588,32493479,39055507,2073142235,1690700,...             94   \n",
       "2116                                           46826168             39   \n",
       "\n",
       "      influentialCitationCount publicationDate  publicationYear  \\\n",
       "0                          6.0      2005-04-01             2005   \n",
       "1                         26.0      2005-03-07             2005   \n",
       "2                          2.0      2005-08-01             2005   \n",
       "3                          2.0      2005-01-01             2005   \n",
       "4                          3.0      2005-12-16             2005   \n",
       "...                        ...             ...              ...   \n",
       "2112                      15.0      2005-10-17             2005   \n",
       "2113                     108.0      2005-09-27             2005   \n",
       "2114                       1.0      2005-10-18             2005   \n",
       "2115                       3.0      2005-08-05             2005   \n",
       "2116                       2.0      2005-08-09             2005   \n",
       "\n",
       "                                           referenceIds  \\\n",
       "0                                                         \n",
       "1     17d0afe4f814e5abc6252e572e3b039082bf6f07;02464...   \n",
       "2                                                         \n",
       "3                                                         \n",
       "4                                                         \n",
       "...                                                 ...   \n",
       "2112  72bf4b2ce534b95bc24118491dbc4f8d550734a2;8c04f...   \n",
       "2113  4e91f5fa9098067a1de83a61af810b263e40ff9a;82bfa...   \n",
       "2114                                                      \n",
       "2115  71257e319d4522d5125024bea18dc79398c1d456;36e6a...   \n",
       "2116  0fca691640f44c44151ad05c5dd9fcaeed4711d9;7ad17...   \n",
       "\n",
       "                                        referenceTitles  \n",
       "0                                                        \n",
       "1     Low-Density Parity-Check Codes;Turbo decoder a...  \n",
       "2                                                        \n",
       "3                                                        \n",
       "4                                                        \n",
       "...                                                 ...  \n",
       "2112  A sparse texture representation using local af...  \n",
       "2113  Antigenic Cross-Reactivity between Severe Acut...  \n",
       "2114                                                     \n",
       "2115  Targeted Deletion of Hepatic CTP:phosphocholin...  \n",
       "2116  Species Diversity and Genetic Diversity: Paral...  \n",
       "\n",
       "[2117 rows x 12 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "paperId",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "authorName",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "openAccessPdf",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_blocks",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pdf_status",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "referenceTitles",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "valid_pages",
         "rawType": "object",
         "type": "string"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "5782d0d4-c0a6-4b8f-b40f-5bbda8dce6ec",
       "rows": [
        [
         "0",
         "0001d5fbff6f7c763a78dd5d141292416dcfae59",
         "Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which are unknown. We investigated the involvement of fibroblast growth factor (FGF) signals in synovial sarcoma and evaluated the therapeutic effect of inhibiting the FGF signal. Experimental Design: The expression of 22 FGF and 4 FGF receptor (FGFR) genes in 18 primary tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR. Effects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein kinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory effects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo. Results: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed in neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma cell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal–regulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the FGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle arrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the FGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not p38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sarcoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2. Conclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibitory molecules will be of potential use for molecular target therapy in synovial sarcoma.",
         "T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. Nishijo,K. Shibata,Y. Shima,S. Nagayama,T. Katagiri,Yusuke Nakamura,Takashi Nakamura,J. Toguchida",
         "https://aacrjournals.org/clincancerres/article-pdf/11/7/2702/1962059/2702-2712.pdf",
         "\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\nDisruption of Fibroblast Growth Factor Signal Pathway Inhibits the\nGrowth of Synovial Sarcomas: Potential Application of Signal\nInhibitors to MolecularTarget Therapy\nTatsuya Ishibe,1,2 Tomitaka Nakayama,2 Takeshi Okamoto,1,2 Tomoki Aoyama,1Koichi Nishijo,1,2\nKotaro Roberts Shibata,1,2 Yasuko Shima,1,2 Satoshi Nagayama,3 Toyomasa Katagiri,4\nYusuke Nakamura,4 Takashi Nakamura,2 and Junya Toguchida1\n**BLOCK**fs== 9.0**b== 0.4**t== 0.3**l== 0.1**r== 0.2**\nAbstract Purpose: Synovial sarcoma is a soft tissue sarcoma, the growth regulatory mechanisms of which\nare unknown.We investigated the involvement of fibroblast growth factor (FGF) signals in synovial\nsarcoma and evaluated the therapeutic effect of inhibiting the FGF signal.\nExperimental Design:The expression of 22 FGF and 4 FGF receptor (FGFR) genes in18 prima-\nry tumors and five cell lines of synovial sarcoma were analyzed by reverse transcription-PCR.\nEffects of recombinant FGF2, FGF8, and FGF18 for the activation of mitogen-activated protein\nkinase (MAPK) and the growth of synovial sarcoma cell lines were analyzed. Growth inhibitory\neffects of FGFR inhibitors on synovial sarcoma cell lines were investigated in vitro and in vivo.\nResults: Synovial sarcoma cell lines expressed multiple FGF genes especially those expressed\nin neural tissues, among which FGF8 showed growth stimulatory effects in all synovial sarcoma\ncell lines. FGF signals in synovial sarcoma induced the phosphorylation of extracellular signal ^\nregulated kinase (ERK1/2) and p38MAPK but not c-Jun NH2-terminal kinase. Disruption of the\nFGF signaling pathway in synovial sarcoma by specific inhibitors of FGFR caused cell cycle ar-\nrest leading to significant growth inhibition both in vitro and in vivo. Growth inhibition by the\nFGFR inhibitor was associated with a down-regulation of phosphorylated ERK1/2 but not\np38MAPK, and an ERK kinase inhibitor also showed growth inhibitory effects for synovial sar-\ncoma, indicating that the growth stimulatory effect of FGF was transmitted through the ERK1/2.\nConclusions: FGF signals have an important role in the growth of synovial sarcoma, and inhibi-\ntory molecules will be of potential use for molecular target therapy in synovial sarcoma.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nSynovial sarcoma is the most frequent soft-tissue sarcoma\n(STS) among patients in the third to fourth decade of life (1)\nand accounts for about 7% to 10% of all human STSs (2). It\npredominantly affects the lower extremities but can occur in\nany part of the body. Surgical resection with an adequate\nfor\nsurgical margin is the definitive choice of\nprimary tumors and has been shown to control\nlocal\nrecurrence (3, 4). Disseminated distant metastasis is the major\ncause of poor outcome, and several reports describing the\nresults of current therapy showed a 5-year survival rate of\naround 50% to 60% (3, 4). The efficacy of adjuvant\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\ntreatment\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAuthors’ Affiliations: 1Institute for Frontier Medical Sciences, Departments of\n2Orthopaedic Surgery, 3Surgery Surgical Basic Science, Graduate School of Medi-\ncine, Kyoto University, Kyoto, Japan; and 4Institute of Medical Science, University\nof Tokyo, Tokyo, Japan\nReceived 10/7/04; revised 1/4/05; accepted 1/13/05.\nGrant support: Ministry of Education, Culture, Sports, Science, and Technology\nscientific research grant 11177101 (J. Toguchida).\nThe costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked advertisement in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\nRequests for reprints: Junya Toguchida, Department of Tissue Regeneration,\nInstitute for Frontier Medical Sciences, Kyoto University, 53 Kawahara-cho,\nShogoin, Sakyo-ku, Kyoto, 606-8507, Japan. Phone: 81-75-751-4134; Fax: 81-75-\n751-4646; E-mail: togjun@frontier.kyoto-u.ac.jp.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nF 2005 American Association for Cancer Research.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nchemotherapy is still a matter of debate, and the development\nof a new therapeutic approach is required to improve the\nprognosis.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nthe reciprocal\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nDespite little progress in clinical treatment during the last\n20 years, cytogenetic and molecular genetic analyses have greatly\nimproved the understanding of this type of tumor, especially\nwith the discovery of\ntranslocation\nt(18;X)(q11;p11) creating an SYT-SSX fusion gene as a synovial\nsarcoma – specific genetic alteration (5, 6). Thus far, three SSX\ngenes (SSX1, SSX2, and SSX4) have been identified as a partner\nof the SYT gene, and >95% of synovial sarcoma tumors carried\none of these fusion genes (7). Although the precise function and\nthe mechanism of oncogenesis are not yet clearly shown, the\nhigh sensitivity and specificity of the SYT-SSX fusion gene in\nsynovial sarcoma have proven to be useful for molecular\ndiagnosis (8). In particular types of tumors with a specific\nreciprocal translocation such as PML-RARa in acute promyelo-\ncytic leukemia (9) and BCR-ABL in chronic myelogenous\nleukemia (10), fusion gene products themselves serve as targets\nfor the therapy. Although immunotherapy using a peptide\nderived from SYT-SSX protein as a specific vaccine has been\ninvestigated (11), no therapeutic approach has been discovered\nfor directly targeting the fusion protein or its function.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nGene expression profiling of tumors has been shown a\npowerful tool with which to isolate a molecular target for\n**BLOCK**fs== 9.0**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\ntherapy (12). We have done a gene expression analysis of\nsynovial sarcoma using a genome-wide cDNA microarray and\nfound that synovial sarcoma shared its molecular signature with\nmalignant peripheral nerve sheath tumor, of which the\nprecursors were neural crest – derived cells, and also identified\na set of genes commonly up-regulated in synovial sarcoma\nincluding the fibroblast growth factor 18 (FGF18) gene (13). The\nFGF signaling pathway seems to play significant roles in tumor\ndevelopment and progression (14 – 16), and recently FGF18 was\nidentified as an autocrine growth factor involved in colon\ncancers (17). Thus far, 22 genes have been identified as members\nof the FGF family, and our cDNA microarray contained 10 FGF\ngenes (FGF2, FGF3, FGF4, FGF7, FGF9, FGF11, FGF12, FGF13,\nFGF18, and FGF19). We found that some FGF genes other\nthan FGF18 were also expressed in synovial sarcoma (data\nshown). Based on these results, we have done an\nnot\nintensive analysis of FGF and its receptor (FGFR) genes in\nsynovial sarcoma and also investigated whether inhibition of\nthe FGF signal is a new therapeutic modality for synovial\nsarcoma.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nMaterials and Methods\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nTissue samples and cell lines.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nTumor tissues were obtained at either\nbiopsy or resection surgery and kept at 80jC. Informed consent was\nobtained from each patient, and tumor samples were approved for\nanalysis by the Ethics Committee of the Faculty of Medicine, Kyoto\nUniversity. Five human synovial sarcoma cell lines (YaFuSS, HS-SY-II,\nSYO-1, Fuji, and 1273/99) were used in this study. YaFuSS and HS-SY-II\ncells have the SYT-SSX1 fusion gene and the others have the SYT-SSX2\nfusion gene (data not shown). YaFuSS was established in our laboratory\nfrom a monophasic synovial sarcoma in a 28-year-old male. HS-SY-II\nwas a gift from H. Sonobe (Kochi University, Japan; ref. 18), SYO-1\nfrom A. Kawai (Okayama University, Japan; ref. 19), Fuji from S.\nTanaka (Hokkaido University, Japan; ref. 20), and 1273/99 from\nO. Larsson (Karolinska Institute, Sweden). Among control cell lines,\nNMS-2 (malignant peripheral nerve sheath tumor; ref. 21) was\nprovided by A. Ogose (Niigata University,\nJapan), and Saos2\n(osteosarcoma), HT1080 (fibrosarcoma), COLO205 (colon cancer),\nand SW480 (colon cancer) cells were purchased from American Type\nCulture Collection (Manassas, VA). Cells were maintained in RPMI\n1640 (Sigma-Aldrich, St. Louis, MI) for Fuji, SW480, and COLO205\nwith 10% fetal bovine serum (FBS, HyClone, Road Logan, UT) or\nDMEM (Sigma, St. Louis, MO) for other cells with 10% FBS. OPTI-\nMEM I, which contains insulin and transferrin as protein supplements\n(Invitrogen, Carlsberg, CA), was used in serum-free cultures.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nReagents and antibodies.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nThe FGFR-specific tyrosine kinase\ninhibitor SU5402 was purchased from Calbiochem (La Jolla, CA;\nref. 22). PD166866 was provided by Pfizer Global Research and\nDevelopment (Groton, CT; ref. 23). The MAPK/extracellular signal –\nregulated kinase (ERK) kinase (MEK) inhibitor U0126 was purchased\nfrom Promega (Madison, WI; ref. 24). Recombinant human FGF18\n(rhFGF18) was purchased from Wako Pure Chemical Industry (Osaka,\nJapan),\nrhFGF8 from PeproTech (London, United Kingdom),\nand rhFGF2 from Oncogene Research Products (San Diego, CA).\nAnti-ERK1 (M12320), anti-phosphorylated-p38 (P39520), anti-pan-\np38 (P19820), anti-phosphorylated-JNK1/SAPK1 (S37220), and anti-\npan-JNK1/SAPK1 (M54920) antibodies were purchased from\nBD Biosciences PharMingen (San Diego, CA); anti-phosphorylated-\nERK1/2 (sc-7383), anti-FGFR3 (sc-123), and peroxidase-conjugated\nanti-mouse IgG (sc-2005) from Santa Cruz Biotechnology (Santa\nCruz, CA); anti-phosphotyrosine (PY20) from Zymed Laboratories\n(South San Francisco, CA); anti-FGF18 (MAB667) from R&D Systems\n(Minneapolis, MN); peroxidase-conjugated anti-rabbitimmunoglobu-\nlin (P0399) from DakoCytomation (Glostrup, Denmark).\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nTotal RNA was extracted using TRIzol\nreagent (Invitrogen) following the manufacturer’s instructions. After\ntreatment with DNase I (Nippon Gene, Osaka, Japan), 1 Ag of total\nRNA was reverse transcribed for single-stranded cDNAs using oligo(dT)\nprimer and Superscript II reverse transcriptase (Invitrogen). PCR was\ndone using 1 AL of RT product in a final volume of 25 AL containing\n20 pmol each of the sense and antisense primers, 2.5 mmol/L MgCl2,\n0.2 mmol/L of each deoxynucleotide triphosphate, and 1 unit of rTaq\npolymerase (TOYOBO, Osaka, Japan). All PCRs were done using\nGeneAmp 9700 (PE Applied Biosystems, Foster City, CA). Information\nof the primers are available upon request.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nQuantitative reverse transcription-PCR (RT-PCR) analyses were done\nin selected FGF and FGFR genes with ABI PRISM 7700 Sequence\nDetection System (PE Applied Biosystems). One microliter of RT\nproduct in a final volume of 25 AL containing 12.5 AL of 2 SYBR\nGreen Mastermix (PE Applied Biosystems). Information for primers are\navailable upon requests. The mean of triplicated data was used to\ncalculate the ratio of target gene/b-actin expression. Statistical analysis\nwas done by t test after logarithmic transformation.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nWestern blot analyses.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nTo detect FGF18 protein in the supernatant,\ncells were cultured up to 80% confluency in 100-mm dish. After\nwashing the dish, cells were further incubated in 5 mL OPTI-MEM I for\n4 days. The supernatant was harvested, mixed with 8 mL ice-cold\nacetone, and kept at 80jC for 1 hour. The mixture was centrifuged at\n10,000  g for 15 minutes, and the precipitate was suspended by lysis\nbuffer (100 AL) containing aprotinin (1 Ag/mL), leupeptin (1 Ag/mL),\npepstatin A (1 Ag/mL), and phenylmethylsulfonyl fluoride (1 mmol/L)\nfollowed by sonication and centrifugation. Twenty microliters of the\nsupernatant were electrophoresed on a 10% SDS-polyacrylamide gel\nand transferred onto a polyvinylidene difluoride membrane (Millipore,\nBedford, MA). After blocking with 3% skim milk, membranes were\nprobed with anti-FGF18 antibody at 1:1,000 dilutions for 1 hour and\nwith peroxidase-conjugated anti-mouse IgG 1:2,000 for 1 hour.\nImmunoreactive bands were detected with Enhanced Chemilumines-\ncence Plus (Amersham Biosciences, Little Chalfont Buckinghamshire,\nUnited Kingdom).\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nTo detect FGFR3 protein in cell lysate, cells were harvested by the\nsame lysis buffer mentioned above. The lysate (40 Ag) was electro-\nphoresed using 8% SDS-polyacrylamide gel, blotted, and blocked by\n5% skim milk. The membrane was probed with anti-FGFR3 antibody at\n1:500 dilutions and with peroxidase-conjugated anti-rabbit immuno-\nglobulin at 1:2,000.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nPhosphorylation analyses.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nTo evaluate the phosphorylation status\nof FGFR3, serum-starved cells were treated by 20 Amol/L SU5402 for\n30 minutes followed by the treatment with rhFGF18 or vehicle. The\ncells were harvested by 1 mL modified radioimmunoprecipitation assay\nbuffer containing protease inhibitors and sodium orthovanadate,\nsonicated, and centrifuged at 15,000  g for 10 minutes. After\npreclearing by 100 AL protein A agarose (Upstate, Lake Placid, NY),\n1 mg of the protein was added with 4 Ag anti-FGFR3 antibody and\nrotated for 2 hours at 4jC, and the immunocomplex was mixed with\n100 AL protein A agarose and rotated for another 2 hours. The mixture\nwas washed twice by PBS, and the precipitate was boiled for 5 minutes\nwith 60 AL of 2 sample buffer, centrifuged, and the supernatant was\nused for Western blotting using anti-FGFR3 antibody (1:500) or anti-\nphosphotyrosine (1:500).\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nTo evaluate the phosphorylation of MAPKs, cells (4  105) were\nseeded on 60-mm dishes and incubated overnight with medium\ncontaining 10% FBS followed by serum starvation for 24 hours. After\ntreatment with each reagent for 15 minutes in the serum-free medium,\ncells were lysed in a buffer (100 AL) containing sodium orthovanadate\n(5 mmol/L) and protease inhibitors, sonicated, and centrifuged.\nProteins (15 Ag) were used for Western blotting.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nBromodeoxyuridine incorporation assay. Cells (4  103) were\nseeded on 96-well plates and incubated overnight with the medium\ncontaining 10% FBS. Then the medium was replaced with the serum-\nfree medium, in which cells were further incubated with each reagent\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nfor 48 hours. Bromodeoxyuridine (BrdUrd, 10 Amol/L) was added to\nthe culture medium during the last 4 hours, and the incorporated\nBrdUrd was detected with BrdUrd Detection Kit (Roche Molecular\nBiochemicals, Manheim, Germany), according to the manufacturer’s\ninstructions. Experiments were done in triplicate at least.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nin DMEM with 10% FBS,\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nCell cycle analysis. Cells (1  105) were incubated overnight in\n60-mm dishes\nfollowed by serum\nstarvation for 24 hours. After treatment with SU5402 (20 Amol/L)\nfor 30 minutes, FBS (1%) was added to the medium and cells were\nfurther incubated for 48 hours. Cells were then harvested, washed\nonce with PBS, and fixed in 70% ethanol overnight. After an\nincubation in PBS containing propidium iodide (10 Ag/mL) and\nRNaseA (10 Ag/mL) for 30 minutes at 37jC, the DNA content of\nthese cells was analyzed using FACScan and CELL Quest Software\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n(Becton Dickinson, San Jose, CA). The data from 10,000 cells were\ncollected and analyzed.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nIn vivo growth assay. All animal studies were approved by the\nAnimal Research Committee, Graduate School of Medicine, Kyoto\nUniversity and done according to the Guideline for Animal Experi-\nments of Kyoto University. First,\nthe effect of PD166866 on\nphosphorylated ERK1/2 in vivo was investigated. SYO-1 (5  106)\ncells suspended in 100 AL of PBS were s.c. injected into the hind\nflank region of male BALB/c nu/nu athymic mice at 5 weeks of age\n(Japan SLC, Hamamatsu,\nJapan). PD166866 (0.1 or 0.5 mg)\nsuspended in 50 AL of DMSO was given i.p. when the tumor\nvolume was f1,500 mm3. Tumors were dissected 30 minutes after\nthe injection, and tissue blocks (5  5  5 mm) were prepared from\nthe central and peripheral portion of tumors. Whole cell lysates were\n**BLOCK**fs== 7.0**b== 0.3**t== 0.5**l== 0.7**r== 0.1**\nFig. 1. Expression of FGF and FGFR in cell\nlines. mRNA expression of 22 FGF genes\n(A) and four FGFR genes including splicing\nvariants (B) was analyzed by RT-PCR in\nfive synovial sarcoma cell lines as well as cell\nlines of malignant peripheral nerve sheath\ntumor (NMS-2), fibrosarcoma (HT1080),\nosteosarcoma (Saos2), and colon\ncarcinoma (COLO205 and SW480).\nC, Western blotting for FGF18 using\nconcentrated supernatant of synovial\nsarcoma cells. D, Western blotting for\nFGFR3 using cell lysates of synovial\nsarcoma cells and NMS-2. Two isoforms\n(125 and 135 kDa) corresponding to\nFGFR3b and FGFR3c were detected\nin synovial sarcoma cells but not in NMS-2.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.3**l== 0.1**r== 0.7**\nFig. 2. Expression of FGF and FGFR genes\nin tumor tissues. A, RT-PCR analysis of the\nexpression of five FGF genes (FGF2, FGF8,\nFGF9, FGF11, and FGF18) in tumor tissues.\nAmong 18 synovial sarcoma (SS) tumors,\nseven were SYT-SSX1-positive\nbiphasic, six were SYT-SSX1-positive\nmonophasic, and five were\nSYT-SSX2-positive monophasic tumors.\nAbbreviations: PLS, pleomorphic\nliposarcoma; LMS, leiomyosarcoma; MFH,\nmalignant fibrous histiocytoma; MPNST,\nmalignant peripheral nerve sheath tumor.\nB, quantitative RT-PCR analyses of the\nexpression of five FGF genes. ., tumors\nother than synovial sarcoma; o, synovial\nsarcoma tumors. C, RT-PCR analysis of the\nexpression of FGFR genes.\nD, quantitative RT-PCR analyses of the\nexpression of FGFR2b, FGFR2c, and\nFGFR3 genes in tumor tissues. ., tumors\nother than synovial sarcoma; o,\nmonophasic synovial sarcoma tumors;\n, biphasic synovial sarcoma tumors.\n**, P < 0.01.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nprepared from each tissue block and used for Western blot. Growth\ninhibition experiments were done using SYO-1 and HT1080. Cells\n(5  106) were inoculated as described above. When the tumor\nvolume reached about 50 mm3 (usually 11-12 days after inocula-\ntion), PD166866 was given i.p., a treatment which was repeated\nthereafter once a day, 6 days a week, for 2 weeks. Tumor size was\nmeasured with vernier calipers, and the volume was calculated as k/\n6  length  width  height.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nProliferating cell nuclear antigen staining.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nImmunohistochemical\nstaining of proliferating cell nuclear antigen (PCNA) was done using\nEPOS anti-PCNA/horseradish peroxidase system (DakoCytomation).\nOn day 13 (the day of the last administration), tumor tissues were\ndissected, fixed in 10% formalin, and embedded in paraffin. Sections\n(5-Am-thick) were probed with anti-PCNA antibody for 1 hour and\ncounterstained with hematoxylin. To quantify the PCNA-positive area,\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nmicroscopic images were digitally acquired, the color blue (hematox-\nylin) was subtracted, and the brown area (PCNA-positive nucleus) was\ncalculated as a digital value using Scion Image software. Five different\nsites were randomly acquired and the average value for the PCNA-\npositive area was calculated.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nStatistical analysis.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nSignificant differences between experimental\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nvalues were determined using t test.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nResults\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nSynovial sarcoma cell lines expressed multiple fibroblast growth\nfactor and fibroblast growth factor receptor genes. The mRNA\nexpression of 22 subtypes of FGF genes was analyzed by RT-\nPCR in five synovial sarcoma and five other cell lines (Fig. 1A).\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\nfive synovial sarcoma cell\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nNone of the five synovial sarcoma cell lines expressed the\nFGF5, FGF6, FGF12, FGF14, FGF16, FGF17, FGF22, or FGF23\ngene, whereas the FGF2, FGF8, FGF9, FGF11, and FGF18 genes\nwere expressed in all\nlines.\nExpression of the remaining nine FGF genes varied among\ncell\nlines. The expression profile in NMS-2 (malignant\nperipheral nerve sheath tumor) was similar to that in synovial\nsarcoma, which may relate to the overall similarity of the gene\nexpression profile between synovial sarcoma and malignant\nperipheral nerve sheath tumor (13). HT1080 (fibrosarcoma)\nexpressed a similar set of FGF genes to malignant peripheral\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nnerve sheath tumor but at much lower levels, and the\nexpression profile of Saos2 (osteosarcoma) was quite different\nbeing positive only for FGF1 and FGF13. Interestingly, synovial\nsarcoma shared with colon carcinoma cell lines the expression\nof some FGF genes such as FGF3 and FGF18 genes, which were\nnot expressed in mesenchymal cell lines. Secretion of these FGF\nproteins by synovial sarcoma cell lines was confirmed by the\nWestern blotting of culture supernatant. FGF18 proteins were\nclearly detected in culture supernatant of synovial sarcoma cell\nlines, although the level of protein was not completely agreed\nwith the level of RNA expression (Fig. 1C).\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.7**r== 0.1**\nFig. 3. Growth stimulatory effect of FGFs\nin synovial sarcoma cell lines. Synovial\nsarcoma cells were treated with FGF8 (A),\nFGF18 (B), or FGF2 (C) at the indicated\nconcentration, and the amount of\nincorporated BrdUrd in each sample was\npresented as a relative BrdUrd uptake value\nusing the uptake of a control sample treated\nwith DMSO as a standard value (100%).\n*, P < 0.05 and **, P < 0.01compared with\neach control sample.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nIn contrast with the heterogeneous expression pattern of\nthe expression profile of receptors (FGFR) was\nligands,\nrelatively homogeneous among synovial sarcoma cell lines,\nbeing positive for all subtypes analyzed except FGFR2b, which\nis an epithelia-specific FGFR expressed in colon carcinomas\n(Fig. 1B). In addition to FGFR2b, NMS-2 was negative for\nFGFR3, and HT1080 was negative for FGFR2c. Expression of\nFGFR3 in synovial sarcoma cell lines was confirmed at the\nprotein level by the Western blotting using cell lysates of each\ncell line (Fig. 1D).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nExpression of fibroblast growth factor and fibroblast growth\nfactor receptor genes in tumor tissues.\nThe FGF2, FGF8,\nFGF9, FGF11, and FGF18 genes, which were commonly ex-\npressed in the five synovial sarcoma cell lines, were inves-\ntigated in tumor tissues of 18 synovial sarcomas and 11 STSs\nof other types (Fig. 2A). In accordance with the results in the\ncell lines, almost all 18 synovial sarcoma tumors expressed\nall of these FGF genes (Fig. 2A). Among five FGF genes, the\nFGF2, FGF9, and FGF11 genes were also expressed in some\ntumors other than synovial sarcoma. In contrast, expression of\nFGF8 and FGF18 genes was only weakly detected in other\ntumors. Expression profiles of these five FGF genes were further\nanalyzed by the quantitative RT-PCR (Fig. 2B). The level of\nexpression of FGF8 and FGF18 genes in synovial sarcoma\ntumors were significantly higher than in tumor tissues of other\ntype of tumors, whereas those of FGF2, FGF9, and FGF11 genes\nshowed no difference.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nAs for receptor genes, synovial sarcoma tumors expressed all\ntypes of FGFR genes analyzed including the FGFR2b gene\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\n(Fig. 2C), which was not detected in cell lines (Fig. 1B). In\ncontrast with the uniform expression pattern observed in\nsynovial sarcoma, expression of the FGFR genes in other types\nof tumors seemed to vary among tumors. For example, FGFR3\nwas expressed in one of three leiomyosarcomas and one of two\nmalignant\nfibrous histiocytoma. The quantitative RT-PCR\nanalyses showed that the expression level of FGFR2b and\nFGFR3 but not FGFR2c genes were significantly higher in\nsynovial sarcoma than those in other types of STS (Fig. 2D). As\nexpected, the expression level of FGFR2b gene was higher in\nbiphasic than in monophasic tumors, although some mono-\nphasic tumors expressed this gene at the same level observed in\nbiphasic tumors.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nGrowth stimulatory effects of\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nfibroblast growth factors in\nsynovial sarcoma cell lines. Expression of FGF and FGFR genes\nand proteins suggested the involvement of FGFs as autocrine/\nparacrine growth factors in synovial sarcoma. Among FGF genes\nexpressed in synovial sarcoma both in vitro and in vivo, we\nfocused on the FGF2, FGF8, and FGF18 genes in subsequent\nexperiments. The FGF2 gene was selected as a gene commonly\nup-regulated in STS (Fig. 2A). FGF8 and FGF18 genes were\nselected as synovial sarcoma – specific up-regulated FGF genes\n(Fig. 2A).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nGrowth stimulatory effects of each rhFGF protein were\nserum-starved conditions. Recombinant\nanalyzed under\nhFGF8 showed a dose-dependent growth-promoting effect\nin all synovial sarcoma cell lines (Fig. 3A). On the treatment\nwith rhFGF18, only HS-SY-II\nshowed a dose-dependent\nincrease (Fig. 3B). A heterogeneous response to FGF was\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.7**\nFig. 4. Activation of ERK1/2 and p38 by\nFGFs in synovial sarcoma cell lines.\nA, Western blot analysis of total ERK1and\nphosphorylated ERK1/2 (pERK1/2) in\nsynovial sarcoma cell lines after treatment\nwith FGF8, FGF18, or FGF2. B, Western blot\nanalysis of total p38 and phosphorylated\np38 (pp38) in synovial sarcoma cell lines\nafter treatment with FGF8, FGF18, or FGF2.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.3**l== 0.7**r== 0.1**\nFig. 5. Antiproliferative effect of FGFR\ninhibitor on synovial sarcoma cell lines.\nA, inhibition of phosphorylation of FGFR3\nby SU5402. Immunoprecipitated\nFGFR3 was hybridized with anti-\nphosphotyrosine antibody (top) and anti-\nFGFR3 antibody (bottom) in three synovial\nsarcoma cell lines. Amount of incorporated\nBrdUrd was analyzed after the treatment\nwith FGFR inhibitor SU5402 in synovial\nsarcoma cell lines as well as other cell lines,\nand the relative uptake value of\neach sample was calculated using the value\nof a control as a standard (100%).\nExperiments were done in the culture\nmedium with 10% FBS (B) or 1% FBS (C).\n*, P < 0.05 and **, P < 0.01.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nmore clearly observed in the case of rhFGF2, in which the\ngrowth of HS-SY-II\nincrease,\nwhereas a growth inhibitory effect was observed in the case\nof YaFuSS (Fig. 3C).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nshowed a dose-dependent\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nActivation of mitogen-activated protein kinases by fibroblast\ngrowth factor signals. One of major signal pathways located\ndownstream of FGFR is the MAPKs, for which three subtypes are\nknown: ERK, p38 MAPKs, and c-Jun NH2-terminal kinases\n(JNK; ref. 25). To investigate the events downstream of activated\nFGFRs, the phosphorylation of these MAPKs was investigated. In\nall synovial sarcoma cell lines analyzed, ERK1/2 was phosphor-\nylated under serum-free conditions, suggesting the presence of\nendogenous stimulatory signals (Fig. 4A). The amount of\nphosphorylated ERK1/2 (pERK1/2) increased on treatment\nwith FGF8 in a dose-dependent manner in all four cell lines\nanalyzed, with the response of HS-SY-II being the highest\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\n(Fig. 4A). FGF18 also induced phosphorylation in all cell lines\n(Fig. 4A). An induction by FGF2 was also observed in HS-SY-II\nand 1273/99, but not remarkably in YaFuSS and SYO-1 (Fig. 4A).\nAs in the case of ERK1/2, phosphorylated p38MAPK (pp38)\nwas observed in all synovial sarcoma cell lines under serum-\nfree conditions (Fig. 4B). The level of pp38 increased on\ntreatment with FGF8 in all synovial sarcoma cell lines, among\nwhich HS-SY-II showed the clearest induction (Fig. 4B). FGF18\nalso induced pp38 in HS-SY-II and other cell lines but at much\nlower\nincrease in pp38 upon\ntreatment with FGF2 was observed in SYO-1 (Fig. 4B).\nPhosphorylation of JNK was not observed under serum-free\nconditions or after the treatment with FGFs in any synovial\nsarcoma cell lines (data not shown). Therefore, exogenous FGF\nsignals were transmitted through both ERK1/2 and p38MAPK\nin synovial sarcoma cell lines.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nlevels. A dose-dependent\n**BLOCK**fs== 9.0**b== 0.4**t== 0.1**l== 0.1**r== 0.5**\nFibroblast growth factor receptor – specific inhibitors reduce the\nlines in accordance with\ngrowth of synovial sarcoma cell\ninactivation of fibroblast growth factor receptor. MAPKs in\nsynovial sarcoma are activated by intrinsic pathways, and\nexogenous FGFs can stimulate the growth of synovial sarcoma,\nsuggesting that inhibition of the intrinsic FGF signal pathway\nmay inhibit the growth of synovial sarcoma. SU5402 is known\nto inhibit FGFR autophosphorylation at an IC50 of 10 to\n20 Amol/L (22). To confirm the effect of SU5402 on synovial\nsarcoma cell lines, the phosphorylation status of FGFR3 was\nanalyzed by the immunoprecipitation and Western blotting in\nthree synovial sarcoma cell lines (YaFuSS, SYO-1, and 1273/\n99; Fig. 5A). Phosphorylated FGFR3 was detected in these cell\nlines with serum-starved condition, and the treatment with\nrhFGF18 protein increased the amount of phosphorylated\nFGFR3 (Fig. 5A), which was compatible with the autocrine/\nparacrine model of FGFs in synovial sarcoma cells. Treatment\nwith SU5402 effectively decreased the phosphorylation of\nFGFR3 into almost undetectable level (Fig. 5A), indicating that\nSU5402 can be used as an inhibitor of FGFR in synovial\nsarcoma cells. By the treatment with SU5402, the growth of all\nfive synovial sarcoma cells was inhibited in a dose-dependent\nmanner (Fig. 5B), and the IC50 value ranged from 8.5 to 17.2\nAmol/L (Table 1). The growth inhibitory effect of SU5402 for\ncell lines other than synovial sarcoma was not remarkable. The\nsame experiments were done under low-serum conditions (1%\nFBS, Fig. 5C), resulting in similar growth inhibition in all\nsynovial sarcoma cell lines. The IC50 was almost identical in\nthe three cell lines (YaFuSS, HS-SY-II, and SYO-1), whereas the\nlines (Fuji and 1273/99) required a higher\nother two cell\nconcentration of SU5402 (Table 1). Growth inhibitory effects\nof another FGFR-specific inhibitor, PD166866 (23), was also\ninvestigated under the low-serum conditions. All five synovial\nsarcoma cell lines had an IC50 value (0.08-4.7 Amol/L) much\nlower than those of other cell lines (>10 Amol/L, Table 1).\nSimilar to the data obtained using SU5402, IC50 values in\nYaFuSS, HS-SY-II, and SYO-1 were much lower than those in\nFuji and 1273/99.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nTable 1. IC50 (Amol/L) of FGFR inhibitors in STS cell\nlines\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nInhibitor\n(concentration\nof serum*)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.9**\nc\nc\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nc\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.9**\nYaFuSS\nHS-SY-II\nc\nSYO-1\nc\nFuji\n1273/99\nNMS-2\nHT1080\nSaos2\nCOLO205\nSW480\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nSU5402\n(10%)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.7**l== 0.2**r== 0.7**\n11.5 F 1.8\n15.1 F3.4\n8.5 F 2.5\n17.2 F 1.8\n12.8 F 6.1\n46.4 F 6.9\n65.4 F 2.6\n>80\n>80\n54.4 F 21.5\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nSU5402\n(1%)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.7**l== 0.3**r== 0.6**\n11.4 F 3.6\n4.9 F 0.5\n8.6 F 1.6\n25.3 F 3.6\n29.9 F 16.6\n60.2 F 20.6\n63.7 F 27.8\n55.8 F 19.1\n>80\n78.1 F20.3\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nPD166866\n(1%)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.7**l== 0.4**r== 0.5**\n0.08 F 0.04\n0.77 F 0.52\n0.18 F 0.11\n4.0 F 1.6\n2.7 F 0.4\n>10\n>10\n>10\n>10\n>10\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\n*Concentration of fetal bovine serum in medium used for measuring IC50s.\ncSynovial sarcoma cell lines.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.1**l== 0.5**r== 0.1**\nGrowth inhibition of fibroblast growth factor receptor inhibitor\nassociated with the reduction of phosphorylation of extracellular\nsignal – regulated kinase 1/2 but not of p38.\nTo investigate\nwhich signals may contribute to the growth of synovial\nsarcoma, the phosphorylation of ERK1/2 and p38 before and\nafter the treatment with SU5402 was investigated. In three\nsynovial sarcoma cell lines (YaFuSS, HS-SY-II, and SYO-1),\nwhich were relatively sensitive to the growth inhibitory effect of\nSU5402, phosphorylation of ERK1/2 under low-serum con-\nditions was completely inhibited by SU5402, whereas the\namount of phosphorylated p38 showed no change (Fig. 6A). A\nsmall amount of pERK1/2 was still observed after SU5402\ntreatment in 1273/99, whereas no significant decrease was\nobserved in Fuji, both of which showed no significant change in\nthe pp38 (Fig. 6A). In cell lines other than synovial sarcoma,\nSU5402 showed no significant changes in the amount of\npERK1/2 or pp38 (data not shown). These results suggested that\nthe signal to induce the phosphorylation of ERK1/2 in synovial\nsarcoma was sent mainly through the FGF and FGFR pathway\nand that although the FGF signal induced the phosphorylation\nof both ERK1/2 and p38, the growth stimulatory effects were\ntransmitted through ERK1/2. To further confirm that the growth\ninhibition of SU5402 was due to the reduction in pERK1/2, the\nfunction of MAPK/ERK kinase 1/2 in synovial sarcoma, which is\na kinase of ERK1/2, was inhibited by U0216. Phosphorylation\nof ERK1/2 was effectively inhibited in all five cell lines, in which\n1273/99 still had a small amount of pERK1/2 at the lower\nconcentration (Fig. 6B). BrdUrd uptake was inhibited in a dose-\ndependent manner in all five cell lines, in which 1273/99 was\nrelatively resistant at the lower concentration, consistent with\nthe status of pERK1/2 (Fig. 6C). These results further indicated\nthat the growth inhibition of SU5402 in synovial sarcoma was\nachieved through the inhibition of signals through ERK1/2.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nGrowth inhibition by fibroblast growth factor receptor inhibitor\nin synovial sarcoma is associated with cell cycle arrest. The\nfraction of each cell cycle phase was analyzed before and after\nthe treatment with SU5402 in YaFuSS cells. SU5402 signifi-\ncantly increased the G1 and subG1 fractions, and decreased the\nS fraction (Fig. 6D). These data suggested that the growth\ninhibition of synovial sarcoma by SU5402 was due to its\ninduction of G1 arrest as previously reported for other tyrosine\nkinase inhibitors (26).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nFibroblast growth factor receptor inhibitors reduced the\nFor in vivo study,\ngrowth of synovial sarcoma cells in vivo.\nwe used SYO-1 because of\nits consistency in developing\ntumor, and PD166866 was used because of its lower IC50\nvalue. To confirm its efficacy in vivo, tumor tissue was taken\n30 minutes after the injection of PD166866. We found that\n0.1 mg of PD166866 was enough to reduce the phosphory-\nlated ERK1/2 in the tumor (Fig. 7A). Based on this result,\nthe growth inhibitory effect of PD166866 (0.1 and 0.5 mg/\nbody) was analyzed in vivo. Although PD166866 was unable\nto stop the growth of the tumor completely, a significant\ninhibition of growth was observed during the period of\nadministration (P < 0.01), and the effect was retained even at\nday 21 at the higher dose (P = 0.02, Fig. 7B). No definite\nreduction of body weight or pathologic changes in vital\norgans such as liver and kidney was observed (data not\nshown). No growth inhibitory effect was observed when\nHT1080 was used instead of SYO-1 even at the higher dose\n(data not shown).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nHistologic examination of tumor tissues showed no signifi-\ncant morphologic changes or focal necrosis. The number of\nPCNA-positive cells, however, was clearly smaller in PD166866-\ntreated tumors (19.3 F 6.6%, Fig. 7D) than control tumors\n(53.0 F 2.0%, Fig. 7C), indicating that PD166866 reduced the\nnumber of cells in S phase. No such difference was observed\nwhen HT1080 was used instead of SYO-1 (data not shown).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nDiscussion\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFGFs may activate genetic programs which promote cell\ngrowth by at least one of three general mechanisms: first, as\nmitogens\nsecond, by\npromoting angiogenesis to supply a growing tumor; and\nthird, by inhibiting apoptosis and allowing tumor cells to\ncontinue to grow beyond normal constraints. We found that\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\nthe tumor cells\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nthemselves;\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nfor\n**BLOCK**fs== 7.0**b== 0.5**t== 0.3**l== 0.7**r== 0.1**\nFig. 6. Effect of FGFR inhibitor on down\nstream molecules and cell cycle profiles\nin synovial sarcoma cells. A, amount of\nppERK1/2 and pp38MAPK after the\ntreatment with SU5402 was analyzed by\nWestern blotting in synovial sarcoma cell\nlines. Total ERK1and p38 were also\nanalyzed. B, Western blot analysis of pERK1/\n2 and ERK1after treatment with the MAPK/\nERK kinase inhibitor U0126 in synovial\nsarcoma cell lines. C, BrdUrd uptake of\nsynovial sarcoma cell lines after treatment\nwith U0126. The relative uptake value of\neach sample was calculated using the\nvalue of a control as a standard (100%).\n**, P < 0.01. D, effect of FGFR inhibitor on\ncell cycle profiles of synovial sarcoma cell\nline. DNA contents of YaFuSS after\ntreatment with vehicle (a) or 20 Amol/L\nSU5402 (b) were analyzed by\nfluorescence-activated cell sorting.\n*, P < 0.05 and **, P < 0.01.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nsynovial sarcoma cells expressed a number of FGF genes, of\nwhich FGF2 , FGF8 , FGF9 , FGF11 , and FGF18 were\ncommonly expressed. The involvement of FGF2 in malignant\ntumors as an autocrine growth factor and/or a paracrine\nangiogenic factor was first shown in brain tumors (27), and\nsubsequently in non – small cell carcinoma (14). FGF8 was\noriginally identified as an androgen-induced growth factor\nfound in conditioned medium of androgen-dependent mouse\nmammary carcinoma SC-3 cells and reported to have a\ngrowth stimulatory effect (28). The human FGF8 gene was\nexpressed in breast and prostate cancers (15, 16, 29) and\nassociated with anchorage-independent proliferation and\ninvasion (30, 31). FGF9 was originally purified from the\nconditioned medium of\nline NMC-G1 and\ndesignated as glia-activating factor due to its mitogenic\nactivity toward glial cells (32). FGF11 was one of four FGF\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nthe glial cell\n**BLOCK**fs== 9.0**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\nhomology factors, which shared structural homology with\nfailed to activate FGFRs (33). Recently,\nclassic FGFs but\nFGF11 was identified as one of the gene products enriched in\nneuronal precursors during the development of the nervous\nsystem (34). FGF18 is cloned by its homology to FGF8 and\nFGF9 (35, 36). FGF18, in association with FGF8, is expressed\nat various times and places during embryogenesis, especially\nneurogenesis (33, 37). Mitotic activity was shown in colon\ncarcinomas,\nin which the expression of FGF18 was up-\nregulated by WNT signals (17). Overall, we found that\nsynovial sarcoma expressed several FGF genes, particularly\nthose expressed in neural tissues, which may further suggest\nthe neural origin of precursor cells of synovial sarcoma as we\npreviously proposed (13). Mitotic activity of exogeneous FGFs\nwas not remarkable in some cell lines (Fig. 3). The serum-\nfree medium used in this study contained insulin, which is\nknown to potentiate the effect of FGF, and the further\naddition of insulin showed no increase of the BrdUrd uptake\n(data not shown). It is possible that the extent of autocrine\nstimulation may already be saturating so that further FGF\nstimulation does not lead to additional growth.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nThe expression of FGFR1, FGFR2c, FGFR3, and FGFR4, to\nwhich FGF2 (1c, 3c and 4), FGF8 (2c, 3c, 4), and FGF18 (2c,\n3c, 4) preferentially bind (38, 39), was confirmed in both tumor\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFig. 7. Effect of FGFR inhibitor on the growth of synovial sarcoma cells in vivo.\nA, Western blot of phosphorylated ERK1/2 obtained using tissue blocks dissected\nfrom the peripheral (P) and central (C) portion of SYO-1tumors in vivo. B, growth\ncurve of SYO-1tumors in vivo. PD166866 (0.1or 0.5 mg) was injected\nintraperitoneally on the days indicated by an arrow. C and D, immunohistochemistry\nfor PCNA in SYO-1tumors. Tumor tissues treated with vehicle (C) or 0.5 mg\nPD166866 (D) were dissected at day 13, and processed for the staining of PCNA.\nOriginal magnification, 400.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.1**l== 0.5**r== 0.1**\ntissues and cell lines. The expression of FGFR2b gene, which was\ndetected in tumor tissues but not in cell lines, is intriguing.\nFGFR2b is a receptor for keratinocyte growth factor (FGF7), and\nexpressed in epithelial cells (40). As expected, all biphasic\nsynovial sarcomas expressed the FGFR2b gene, whereas the\nexpression was low in most of the monophasic synovial\nsarcomas and other STSs. It is, however, intriguing that some\nmonophasic synovial sarcomas with no apparent epithelial\nstructures expressed the FGFR2b gene at a level comparable with\nthat in biphasic synovial sarcoma (Fig. 2D). All five synovial\nsarcoma cell lines used in this study including SYO-1, which was\nestablished from a biphasic synovial sarcoma, were negative for\nthe FGFR2b gene. These results suggested that synovial sarcoma\ntumor cells may have an intrinsic mechanism to express FGFR2b\nin vivo, which was somehow inhibited or disappeared in vitro.\nAlternatively, synovial sarcoma cells positive for FGFR2b may\nhave growth disadvantages, and therefore only negative cells\nwere established as cell\nlines. Recently, single nucleotide\npolymorphism in FGFR4 is reported to associate with poor\nprognosis in high-grade soft tissue sarcomas including synovial\nsarcomas (41) and both FGF8 and FGF18 bind to FGFR4.\nAlthough the status of polymorphism was not investigated in our\nsamples, FGF signals through FGFR4 may contribute to the\naggressiveness of synovial sarcoma.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nThe binding of FGF to FGFR leads to receptor dimerization\nand tyrosine autophosphorylation, followed by downstream\nactivation of PLCg, Crk, and SNT-1/FRS2 signaling pathways\namong which the SNT-1/FRS2 is the major pathway for\nmitogenic signals (25). Phosphorylated SNT-1/FRS2 directly\nbinds to the GRB2-SOS complex, and then membrane-\nassociated RAS recruits RAF-1, which in turn activates MAPKs\n(42). MAPKs are composed of three well-characterized sub-\nfamilies, ERK, p38 MAPK, and JNK (25), and we found that the\ngrowth-promoting signal of FGF was transmitted mainly\nthrough ERK1/2. Although ERK1/2 are activated by signals\nthrough other receptor tyrosine kinases (43), phosphorylation\nof ERK1/2 was completely inhibited by FGFR inhibitors, which\nindicated that the major signal to activate ERK1/2 in synovial\nsarcoma is FGF signaling, suggesting the important role of FGF\nin synovial sarcoma.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nWe have no clear explanation for the heterogeneous response\nin synovial sarcoma cell lines to FGF2, which stimulated the\ngrowth of HS-SY-II, but inhibited the growth of YaFuSS. In\ncontrast with many studies describing the role of FGF2 as a\nmitogen, several reports showed that FGF2 acted as a growth\ninhibitor of tumors of Ewing’s sarcoma family, which are\nsupposed to be of neuroectodermal origin (44, 45). Multiple\nFGF signals are involved in the biological process in synovial\nsarcoma, and the response may depend on the cell specificity.\nNevertheless, the disruption of FGF signals was found to cause\ngrowth inhibition in all synovial sarcoma cell lines, suggesting\nthat an inhibitory molecule for FGFR will be a promising tool\nfor molecular target therapy in synovial sarcoma.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nMolecular therapy targeted to FGFR has been investigated\nwith hematologic malignancies carrying a translocation involv-\ning the FGFR gene such as chronic myeloid leukemia with the\nBCR-FGFR1 gene (46) and multiple myeloma with the MMSET-\nFGFR3 fusion gene (47). In both instances, SU5402 was found\nto effectively inhibit the growth of cell lines with an IC50\nequivalent to that in synovial sarcoma. For experiments in vivo,\nwe used PD166866 instead of SU5402 because of its lower IC50\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nvalue and found that i.p. given PD166866 efficiently inhibited\ntumor causing an\nthe phosphorylation of ERK1/2 in s.c.\ninhibition of growth without significant side effects. As in the\ncase of other tyrosine kinase inhibitors, PD166866 did not\nstrongly induce apoptosis or necrosis, and therefore no\ncomplete tumor remission was observed. However, cell cycle\narrest caused by the treatment with PD166866 may enhance\nthe cytotoxic effects of anticancer drugs as previously shown in\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nsome instances (48, 49), and therefore further in vivo studies\nusing the treatment combined with suitable anticancer drugs\nmay provide a promising approach for clinical application.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nAcknowledgments\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nWe thank Drs. H. Sonobe, A. Kawai, S.Tanaka, O. Larsson, and A. Ogose for pro-\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nviding cell lines and Y. Furukawa for useful comments.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nReferences\n1.Weiss SW, Goldblum JR. Enzinger andWeiss’s soft tis-\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nsue tumors. 4th ed. St. Louis: Mosby; 2001.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\n2. Fisher C. Synovial sarcoma. Ann Diagn Pathol 1998;\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n2:401 ^ 21.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\n3. Spillane AJ, A’Hern R, Judson IR, Fisher C,\nThomas JM. Synovial sarcoma: a clinicopathologic,\nstaging, and prognostic assessment. J Clin Oncol\n2000;18:3794 ^ 803.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\n4. Lewis JJ, Antonescu CR, Leung DH, et al. Synovial\nsarcoma: a multivariate analysis of prognostic factors\nin 112 patients with primary localized tumors of the ex-\ntremity. J Clin Oncol 2000;18:2087 ^ 94.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\n5. Turc-Carel C, Dal Cin P, Limon J, et al. Involvement of\nchromosome X in primary cytogenetic change in hu-\nman neoplasia: nonrandom translocation in synovial\nsarcoma. Proc Natl Acad Sci U S A 1987;84:1981 ^ 5.\n6. Clark J, Rocques PJ, Crew AJ, et al. Identifi-\ninvolved in\ncation of novel genes, SYT and SSX,\nthe t(X;18)(p11.2;q11.2) translocation found in human\nsynovial sarcoma. Nat Genet 1994;7:502 ^ 8.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\n7. Dos Santos NR, De Bruijn DR, Van Kessel AG.\nMolecular mechanisms underlying human synovial\nsarcoma development. Genes Chromosomes Can-\ncer 2001;30:1 ^ 14.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\n8. Ladanyi M. Fusions of the SYTand SSX genes in sy-\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nnovial sarcoma. Oncogene 2001;20:5755 ^ 62.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.7**\n9. Huang ME,YeYC, Chen SR, et al. Use of all-trans ret-\ninoic acid in the treatment of acute promyelocytic leu-\nkemia. Blood 1988;72:567 ^ 72.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\n10. Druker BJ,Tamura S, Buchdunger E, et al. Effects of\na selective inhibitor of the Abl tyrosine kinase on the\ngrowth of Bcr-Abl positive cells. Nat Med 1996;2:\n561 ^ 6.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\n11. Sato Y, Nabeta Y, Tsukahara T, et al. Detection and\ninduction of CTLs specific for SYT-SSX-derived pep-\ntides in HLA-A24(+) patients with synovial sarcoma.\nJ Immunol 2002;169:1611 ^ 8.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\n12. Zanders ED. Gene expression analysis as an aid to\nthe identification of drug targets. Pharmacogenomics\n2000;1:375 ^ 84.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\n13. Nagayama S, Katagiri T, Tsunoda T, et al. Genome-\nwide analysis of gene expression in synovial sarco-\nmas using a cDNA microarray. Cancer Res 2002;62:\n5859 ^ 66.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.7**\n14. Berger W, Setinek U, Mohr T, et al. Evidence for a\nrole of FGF-2 and FGF receptors in the proliferation\nof non-small cell lung cancer cells. Int J Cancer\n1999;83:415 ^ 23.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\n15. Tanaka A, Miyamoto K, Matsuo H, Matsumoto K,\nYoshida H. Human androgen-induced growth factor in\nprostate and breast cancer cells: its molecular cloning\nand growth properties. FEBS Lett 1995;363:226 ^ 30.\n16. Dorkin TJ, Robinson MC, March C, Bjartell A,\nNeal DE, Leung HY. FGF8 over-expression in pros-\ntate cancer is associated with decreased patient\nsurvival and persists in androgen independent dis-\nease. Oncogene 1999;18:2755 ^ 61.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\n17. Shimokawa T, Furukawa Y, Sakai M, et al. Involve-\nment of the FGF18 Gene in colorectal carcinogenesis\nas a novel downstream target of the h-catenin/T-cell\nfactor complex. Cancer Res 2003;63:6116 ^ 20.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\n18. Sonobe H, Manabe Y, Furihata M, et al. Establish-\nment and characterization of a new human synovial\nsarcoma cell line, HS-SY-II. Lab Invest 1992;67:\n498 ^ 505.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n19. Kawai A, Naito N, Yoshida A, et al. Establishment\nand characterization of a biphasic synovial sarcoma\ncell line, SYO-1. Cancer Lett 2004;204:105 ^ 13.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\n20. Nojima T, Wang YS, Abe S, Matsuno T, Yamawaki\nS, Nagashima K. Morphological and cytogenetic\nstudies of a human synovial sarcoma xenotrans-\nplanted into nude mice. Acta Pathol Jpn 1990;\n40:486 ^ 93.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\n21. Imaizumi S, Motoyama T, Ogose A, Hotta T,\nTakahashi HE. Characterization and chemosensitivity\nof two human malignant peripheral nerve sheath tu-\nmour cell lines derived from a patient with neurofibro-\nmatosis type 1.Virchows Arch 1998;433:435 ^ 41.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\n22. Mohammadi M, McMahon G, Sun L, et al. Struc-\ntures of the tyrosine kinase domain of fibroblast\ngrowth factor receptor in complex with inhibitors. Sci-\nence 1997;276:955 ^ 60.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\n23. Panek RL, Lu GH, DahringTK, et al. In vitro biologi-\ncal characterization and antiangiogenic effects of\nPD166866, a selective inhibitor of the FGF-1receptor\ntyrosine kinase. J Pharmacol Exp Ther 1998;286:\n569 ^ 77.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n24. Favata MF, Horiuchi KY, Manos EJ, et al. Identifica-\ntion of a novel inhibitor of mitogen-activated protein\nkinase kinase. J Biol Chem 1998;273:18623 ^ 32.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\n25. Powers CJ, McLesky SW, Wellstein A. Fibroblast\ngrowth factors, their receptors, and signaling. Endocr\nRelat Cancer 2000;7:165 ^ 97.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n26. Shintani S, Li C, Mihara M, et al. Gefitinib (‘‘Iressa’’,\nZD1839), an epidermal growth factor receptor tyrosine\nkinase inhibitor, up-regulates p27KIP1and induces G1\narrest in oral squamous cell carcinoma cell lines. Oral\nOncol 2004;40:43 ^ 51.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\n27. TakahashiJ, Mori H, Fukumoto M, et al. Gene expres-\nsion of fibroblast growth factors in human gliomas and\nmeningiomas: demonstration of cellular source of basic\nfibroblast growth factor mRNA and peptide in tumor\ntissues. Proc Natl Acad Sci US A1990;87:5710 ^ 4.\n28. Tanaka A, Miyamoto K, Minamino N, et al. Cloning\nand characterization of an androgen-induced growth\nfactor essential for the androgen-dependent growth\nof mouse mammary carcinoma cells. Proc Natl Acad\nSci U S A 1992;89:8928 ^ 32.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.4**r== 0.4**\n29. Tanaka A, Furuya A,Yamasaki M, et al. High frequen-\ncy of fibroblast growth factor (FGF) 8 expression in\nclinical prostate cancers and breast tissues, immuno-\nhistochemically demonstrated by a newly established\nneutralizing monoclonal antibody against FGF 8. Can-\ncer Res 1998;58:2053 ^ 6.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\n30. Ruohola JK, Viitanen TP, Valve EM, et al. Enhanced\ninvasion and tumor growth of fibroblast growth factor\n8b-overexpressing MCF-7 human breast cancer cells.\nCancer Res 2001;61:4229 ^ 37.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\n31. Song Z, Powell WC, Kasahara N, van Bokhoven A,\nMiller GJ, Roy-Burman P. The effect of fibroblast\ngrowth factor 8, isoform b, on the biology of prostate\ncarcinoma cells and their interaction with stromal cells.\nCancer Res 2000;60:6730 ^ 6.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\n32. Miyamoto M, Naruo K, Seko C, Matsumoto S,\nKondo T, Kurokawa T. Molecular cloning of a novel\ncytokine cDNA encoding the ninth member of the\nfibroblast growth factor family, which has a unique\nsecretion property. Mol Cell Biol 1993;13:4251 ^ 9.\n33. Smallwood PM, Munoz-Sanjuan I,Tong P, et al. Fi-\nbroblast growth factor (FGF) homologous factors:\nnew members of the FGF family implicated in nervous\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nsystem development. Proc Natl Acad Sci U S A 1996;\n93:9850 ^ 7.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\n34. Luo Y, Cai J, Liu Y, et al. Microarray analysis of se-\nlected genes in neural stem and progenitor cells.\nJ Neurochem 2002;83:1481 ^ 97.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.1**\n35. Ohbayashi N, Hoshikawa M, Kimura S,Yamasaki M,\nFukui S, Itoh N. Structure and expression of the mRNA\nencoding a novel fibroblast growth factor, FGF-18.\nJ Biol Chem 1998;273:18161 ^ 4.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\n36. Hu MC, Qiu WR, Wang YP, et al. FGF-18, a novel\nmember of the fibroblast growth factor family, stimu-\nlates hepatic and intestinal proliferation. Mol Cell Biol\n1998;18:6063 ^ 74.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.1**\n37. Ohuchi H, Kimura S, Watamoto M, Itoh N. Involve-\nment of fibroblast growth factor (FGF)18-FGF8\nsignaling in specification of left-right asymmetry and\nbrain and limb development of the chick embryo.\nMech Dev 2000;95:55 ^ 66.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\n38. Ornitz DM, Xu J, Colvin JS, et al. Receptor specific-\nity of the fibroblast growth factor family. J Biol Chem\n1996;271:15292 ^ 7.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\n39. Ellsworth JL, Berry J, Bukowski T, et al. Fibroblast\ngrowth factor-18 is a trophic factor for mature chon-\ndrocytes and their progenitors. Osteroarthritis Carti-\nlage 2002;10:308 ^ 20.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.6**r== 0.1**\n40.Werner S. Keratinocyte growth factor: a unique play-\ner in epithelial repair processes. Cytokine Growth Fac-\ntor Rev 1998;9:153 ^ 65.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\n41. MorimotoY, Ozaki T, Ouchida M, et al. Single nucle-\notide polymorphism in fibroblast growth factor recep-\ntor 4 at codon 388 is associated with prognosis in\nhigh-grade soft tissue sarcoma. Cancer 2003;98:\n2245 ^ 50.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\n42. Kouhara H, Hadari YR, Spivak-Kroizman T, et al.\nA lipid-anchored Grb2-binding protein that links FGF-\nreceptor activation to the Ras/MAPK signaling path-\nway. Cell 1997;89:693 ^ 702.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.6**r== 0.1**\n43. Denhardt DT. Signal-transducing protein phosphor-\nylation cascades mediated by Ras/Rho proteins in the\nmammalian cell: the potential for multiplex signaling.\nBiochem J 1996;318:729 ^ 47.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\n44. Sturla LM, Westwood G, Selby PJ, Lewis IJ,\nBurchill SA. Induction of cell death by basic fibro-\nblast growth factor in Ewing’s sarcoma. Cancer Res\n2000;60:6160 ^ 70.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.6**r== 0.1**\n45. Kim MS, Kim CJ, Jung HS, et al. Fibroblast growth\nfactor 2 induces differentiation and apoptosis of Askin\ntumour cells. J Pathol 2004;202:103 ^ 12.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\n46. Demiroglu A, Steer EJ, Heath C, et al.The t(8;22) in\nchronic myeloid leukemia fuses BCR to FGFR1: trans-\nforming activity and specific inhibition of FGFR1fusion\nproteins. Blood 2001;98:3778 ^ 83.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\n47. Paterson JL, Li Z,Wen XY, et al. Preclinical studies of\nfibroblast growth factor receptor 3 as a therapeutic\ntarget in multiple myeloma. Br J Haematol 2004;124:\n595 ^ 603.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.6**r== 0.1**\n48. Garofalo A, Naumova E, Manenti L, et al. The com-\nbination of the tyrosine kinase receptor inhibitor\nSU6668 with paclitaxel affects ascites formation\nand tumor spread in ovarian carcinoma xenografts\ngrowing orthotopically. Clin Cancer Res 2003;9:\n3476 ^ 85.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.1**\n49. Abdollahi A, Lipson KE, Sckell A, et al. Combined\ntherapy with direct and indirect angiogenesis inhibi-\ntion results in enhanced antiangiogenic and antitumor\neffects. Cancer Res 2003;63:8890 ^ 8.",
         "extracted",
         "",
         "Disruption of Fibroblast Growth Factor Signal Pathway Inhibits the Growth of Synovial Sarcomas: Potential Application of Signal Inhibitors to Molecular Target Therapy",
         "None"
        ],
        [
         "1",
         "000372290caae5482dcdd7954feb895ccbab921d",
         "Shuffled versions of iterative decoding of low-density parity-check codes and turbo codes are presented. The proposed schemes have about the same computational complexity as the standard versions, and converge faster. Simulations show that the new schedules offer better performance/complexity tradeoffs, especially when the maximum number of iterations has to remain small.",
         "Juntan Zhang,M. Fossorier",
         "http://www.merl.com/publications/docs/TR2005-063.pdf",
         "\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\nMITSUBISHI ELECTRIC RESEARCH LABORATORIES\nhttp://www.merl.com\n**BLOCK**fs== 14.3**b== 0.7**t== 0.3**l== 0.3**r== 0.3**\nReplica Shufﬂed Iterative Decoding\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.3**\nZhang, J.; Wang, Y.; Fossorier, M.; Yedidia, J.S.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nTR2005-063 September 2005\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nAbstract\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.2**\nReplica shufﬂed versions of interactive decoders of turbo codes, low-density parity-check codes\nand turbo product codes are presented. The proposed schemes converge faster than standard and\npreviously proposed shufﬂed approaches. Simulations show that the new schedules offer good\nperformance versus complexity/latency trade-offs.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.4**\nIEEE International Symposium on Information Theory (ISIT)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in whole or in part\nwithout payment of fee is granted for nonproﬁt educational and research purposes provided that all such whole or partial copies include\nthe following: a notice that such copying is by permission of Mitsubishi Electric Research Laboratories, Inc.; an acknowledgment of\nthe authors and individual contributions to the work; and all applicable portions of the copyright notice. Copying, reproduction, or\nrepublishing for any other purpose shall require a license with payment of fee to Mitsubishi Electric Research Laboratories, Inc. All\nrights reserved.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nCopyright c Mitsubishi Electric Research Laboratories, Inc., 2005\n201 Broadway, Cambridge, Massachusetts 02139\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nMERLCoverPageSide2\n**BLOCK**fs== 14.3**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\nReplica Shufﬂed Iterative Decoding\n**BLOCK**fs== 10.9**b== 0.7**t== 0.3**l== 0.3**r== 0.2**\nJuntan Zhang, Yige Wang, Marc Fossorier, and Jonathan S. Yedidia\n**BLOCK**fs== 10.9**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nTR-2005-063 February 2005\n**BLOCK**fs== 10.9**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nAbstract\n**BLOCK**fs== 10.9**b== 0.4**t== 0.5**l== 0.2**r== 0.2**\nReplica shufﬂed versions of iterative decoders of turbo codes, low-density parity-check\ncodes and turbo product codes are presented. The proposed schemes converge faster\nthan standard and previously proposed “shufﬂed” approaches. Simulations show that\nthe new schedules offer good performance versus complexity/latency trade-offs.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nThis work may not be copied or reproduced in whole or in part for any commercial purpose. Permission to copy in\nwhole or in part without payment of fee is granted for nonproﬁt educational and research purposes provided that all such\nwhole or partial copies include the following: a notice that such copying is by permission of Mitsubishi Electric Research\nLaboratories, Inc.; an acknowledgment of the authors and individual contributions to the work; and all applicable portions\nof the copyright notice. Copying, reproduction, or republishing for any other purpose shall require a license with payment\nof fee to Mitsubishi Electric Research Laboratories, Inc. All rights reserved.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nCopyright c Mitsubishi Electric Research Laboratories, Inc., 2005\n201 Broadway, Cambridge, Massachusetts 02139\n**BLOCK**fs== 10.9**b== 0.9**t== 0.1**l== 0.2**r== 0.2**\nTo appear in the Proceedings of the 2005 International Symposium on Information Theory.\n**BLOCK**fs== 23.9**b== 0.8**t== 0.1**l== 0.2**r== 0.2**\nReplica Shufﬂed Iterative Decoding\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.5**\nJuntan Zhang, Yige Wang and Marc Fossorier\nDepartment of Electrical Engineering\nUniversity of Hawaii at Manoa\nHonolulu, HI 96822\nEmail: juntan, yige, marc@spectra.eng.hawaii.edu\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nJonathan S. Yedidia\nMitsubishi Electric Research Laboratories\n201 Broadway, Cambridge, MA 02139\nEmail: yedidia@merl.com\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nAbstract— Replica shufﬂed versions of iterative decoders of\nturbo codes, low-density parity-check codes and turbo product\ncodes are presented. The proposed schemes converge faster\nthan standard and previously proposed “shufﬂed” approaches.\nSimulations show that the new schedules offer good performance\nversus complexity/latency trade-offs.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nI. INTRODUCTION\n**BLOCK**fs== 10.0**b== 0.2**t== 0.4**l== 0.1**r== 0.5**\nIterative decoding based on belief propagation (BP) [1] has\nreceived signiﬁcant attention recently, mostly due to its near-\nShannon-limit error performance for the decoding of low-\ndensity parity-check (LDPC) codes [2] and turbo codes [3].\nLike maximum a posterior probability (MAP) decoding [4],\nit is a symbol-by-symbol soft-in/soft-out decoding algorithm.\nIt processes the received symbols recursively to improve the\nreliability of each symbol based on constraints that specify\nthe code. In the ﬁrst iteration, the decoder only uses the\nchannel output, and generates soft output for each symbol.\nSubsequently, the output reliability measures of the decoded\nsymbols at the end of each decoding iteration are used as\ninputs for the next iteration. The decoding iteration process\ncontinues until a certain stopping condition is satisﬁed. Then\nhard decisions are made based on the output reliability mea-\nsures of the decoded symbols at the last decoding iteration.\nThe standard BP decoder of LDPC codes often needs several\ntens or hundreds of iterations for the decoding process to\nconverge, which is not desirable because of high decoding\ndelay. Furthermore, LDPC codes of interest can have large\nthe\ncodeword length and it can be difﬁcult\ndecoding in hardware in a fully parallel way. Because of\nthe serial property of the BCJR algorithm, decoders of turbo\ncodes cause high delay as well. In [5], [6] and [7], “shufﬂed”\nmethods were presented to reduce the required number of\niterations for decoding LDPC and turbo codes, respectively.\nA speedup factor of 2 for LDPC codes and the saving of\none iteration for turbo codes were reported. The aim of this\npaper is to introduce a “replica shufﬂed” scheme which further\naccelerate the decoding process for turbo codes, LDPC codes,\nturbo product codes and other iterative decodable codes.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nto implement\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nII. ITERATIVE DECODING OF TURBO CODE\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nA turbo code [3] encoder is constructed using a concatena-\ntion of two (or more) convolutional encoders, and its decoder\nconsists of two (or more) soft-in/soft-out convolutional de-\ncoders which feed reliability information to each other. For\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\nis the output code block at\n**BLOCK**fs== 10.0**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nsimplicity, we consider a turbo code that consists of two rate-\n1/n systematic convolutional codes with encoders in feedback\nform. Let u = (u1, u2, . . . , uK) be an information block of\nlength K and c = (c1, c2, . . . , cK) be the corresponding\ncoded sequence, where ck = (ck,1, ck,2, . . . , ck,n), for k =\n1, 2, . . . , K,\ntime k. Suppose\nBPSK transmission over an AWGN channel, with uk and\nck,j all taking values in {+1, -1} for k = 1, 2, . . . , K and\nj = 1, 2, . . . , n. Let y = (y1, y2, . . . , yK) be the received\nsequence, where yk = (yk,1, yk,2, . . . , yk,n) is the received\nblock at time k. Let ˆu = {ˆu1, ˆu2, . . . , ˆuK} denote the estimate\nof u. Let sk denote the encoder state at time k. Following\n1 ), γk(s, s) = p(sk =\n[4], deﬁne: αk(s) = p(sk = s, yk\ns, yk|sk−1 = s), βk(s) = p(yK\na =\n(ya, ya+1, . . . , yb), and let α(m)\n(s) be\nk\nthe corresponding values computed in component decoder m,\nwith m = 1, 2. Let L(i)\nem(ˆuk) denote the extrinsic value of the\nestimated information bit ˆuk delivered by component decoder\nm at the ith iteration [8].\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\nk+1|sk = s), where yb\n(s), γ(m)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n(s, s), β(m)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\nk\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nk\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nA. Standard serial and parallel turbo decoding\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nThe decoding approach proposed in [3] operates in serial\nmode, i.e., the component decoders take turns generating the\nextrinsic values of the estimated information symbols, and\neach component decoder uses the extrinsic messages delivered\nby the last component decoder as the a priori values of the\ninformation symbols. The disadvantage of this scheme is high\ndecoding delay. In the parallel turbo decoding algorithm [9],\nall component decoders operate in parallel at any given time.\nAfter each iteration, each component decoder delivers extrinsic\nmessages to other decoder(s) which use these messages as a\npriori values at the next iteration.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nB. Plain shufﬂed turbo decoding\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nAlthough the parallel turbo decoding reduces the decoding\ndelay of serial decoding by half, the extrinsic messages are not\ntaken advantage of as soon as they become available, because\nthe extrinsic messages are delivered to component decoders\nonly after each iteration is completed. The aim of the shufﬂed\nturbo decoding is to use the more reliable extrinsic messages\nat each time. Let ˜u = (˜u1, ˜u2, . . . , ˜uK) be the sequence\npermuted by the interleaver corresponding to the original\ninformation sequence u = (u1, u2, . . . , uK), according to the\nmapping ˜uk = uπ(k), for k = 1, 2, . . . , K. We assume that k =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nπ(k), ∀k. There is a unique corresponding reverse mapping\nuk = ˜uπ−(k), for k = 1, 2, . . . , K and k = π−(k), ∀k. In\nshufﬂed turbo decoding, the two component decoders operate\nsimultaneously as in the parallel turbo decoding scheme, but\nthe messages are updated during each iteration based on π(k)\nand π−(k) [7]. Correspondingly it provides a faster decoding\nconvergence.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nC. Replica shufﬂed turbo decoding\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\n−→\nL\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n←−\nD 1 and\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n−→\nD 1 and\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\n−→\nD 1, which are denoted by\n**BLOCK**fs== 10.0**b== 0.3**t== 0.3**l== 0.1**r== 0.5**\nIn the plain shufﬂed turbo decoding summarized in Section\nthe component decoders process the\nII-B, we assume all\nbackward recursion followed by the forward recursion. Let\n−→\nus refer to the two component decoders as\nD 2.\nNaturally another possible scheme is to operate in the reverse\norder, i.e, all the component decoders process the forward\nrecursion followed by the backward recursion and we refer\n←−\nto them as\nD 2. In terms of error performance, there\nis no difference between these two approaches. However, the\nreliabilities of the extrinsic messages concerning a certain\ninformation bit delivered by these two shufﬂed turbo decoders\nare not the same. In general, the more independent information\nis used, the more reliable the delivered messages become.\nTherefore for the extrinsic messages delivered by component\n(i)\ndecoder\ne1 (ˆuk), the larger k\nis, the more reliable this message is. Following a similar\n←−\nanalysis, for the extrinsic message\nD 1,\nthe smaller k is, the more reliable this message is. It is natural\nto expect a faster decoding convergence if these two shufﬂed\nturbo decoders operate cooperatively instead of independently.\nBecause in this approach two sets of shufﬂed component\ndecoders are used to decode the same sequence of information\nbits, we refer to it as replica shufﬂed turbo decoding. In replica\nshufﬂed turbo decoding, two plain shufﬂed turbo decoders\n−→\n(processing recursions in opposite directions)\nD 2 and\n←−\n←−\nD 1 ,\nD 2 operate simultaneously and exchange more reliable\nextrinsic messages. We assume that the component decoders\n−→\n\n\ndeliver extrinsic messages synchronously, i.e.,\nk =\nk =\nT\n←−\n←−\n−→\n←−\n←−\n\n\n\n\n\nk)and\nk (\nT\nk, where the\nk) denote the times\nT\nT\nT\nk =\nT\n←−\n−→\n←−\n−→\nat which\nD 2) deliver the extrinsic values\nD 2 (\nD 1) and\nD 1 (\nof the k-th ((K + 1 − k)-th) estimated symbol of the original\ninformation sequence u and of the interleaved sequence ˜u,\nrespectively.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.5**\n(i)\ne1 (ˆuk) delivered by\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−→\nD 1 ,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nk (\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n←−\nL\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−→\nT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n−→\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.6**\n−→\nD 1. After time\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nLet us ﬁrst consider the forward recursion stage at the i-\n−→\n\nk−1,\nT\nth iteration of component decoder\nthe values of −→α (1)\nk (s) should be updated and the values\nof −→γ (1)\nk (s) are needed. There are two possible cases. The\nﬁrst case is k > π−(k), which means the extrinsic value\n−→\n(i)\ne2 (ˆuk) of the information bit ˆuk has already been delivered\nL\n−→\nD 2. As in plain shufﬂed turbo decoding, this newly\nby decoder\n−→\n(i)\ne2 (ˆuk) is used to compute the values −→γ (1)\nL\navailable\nk (s),\n−→\n(i)\n−→α (1)\ne1 (ˆuk). The second case is k < π−(k), which\nL\nk (s), and\n(i)\ne2 (ˆuk) of the information bit ˆuk\nmeans the extrinsic value\n−→\nD 2. Then in plain shufﬂed\nhas not been delivered yet by\nturbo decoding, the values α(1)\ne1 (ˆuk) are updated\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n−→\nL\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n←−\nL\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n←−\nL\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n−→\nL\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n←−\nL\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n−→\nL\nk (s), and\n**BLOCK**fs== 10.0**b== 0.5**t== 0.1**l== 0.5**r== 0.1**\nbased on the extrinsic messages delivered at last iteration.\nIn replica shufﬂed turbo decoding, however, there are two\nfurther subcases. The ﬁrst subcase is K + 1 − k < π−(k),\n(i)\ne2 (ˆuk) of the information\nwhich means the extrinsic value\n←−\nbit ˆuk has already been delivered by decoder\nD 2. Then this\n(i)\n(i−1)\ne2 (ˆuk), instead of\n(ˆuk) is used to\nnewly available\ne2\n−→\n(i)\ncompute the values −→γ (1)\nk (s), −→α (1)\ne1 (ˆuk). The\nL\nsecond subcase is K + 1 − k < π−(k), which means both\n(i)\nextrinsic messages of the information bit ˆuk, i.e,\ne2 (ˆuk)\n−→\n(i)\ne2 (ˆuk) are not available yet. In this subcase, the values\nL\nand\n(i)\nof −→α (1)\ne1 (ˆuk) are updated based on the extrinsic\nk (s) and\nmessages delivered at (i − 1)-th iteration. The recursions of\n←−\n←−\ncomponent decoders\nD 2 are realized based on\nD 1 and\nthe same principle. After Imax iterations, the shufﬂed turbo\ndecoding algorithm outputs ˆu = (ˆu1, ˆu2, . . . , ˆuK) as the de-\n←−\n(i)\ncoded codeword, where ˆuk =sgn[(\ne1 (ˆuk))/2 +\nL\n−→\n(i)\ne2 (ˆuk))/2 + 4\nyk,1], which is different from\nL\n(\nN0\nthat in the standard turbo decoding [3] and plain shufﬂed\nturbo decoding. It is straightforward to generalize the replica\nshufﬂed turbo decoding to multiple turbo codes which consist\nof more than two component codes. Also group of bits can\nbe updated periodically only to reduce information exchanges\nbetween replicas. Based on the above descriptions with two\nreplicas, the total computational complexity of the replica\nshufﬂed turbo decoding for multiple turbo codes at each\ndecoding iteration is about twice that of the parallel turbo\ndecoding.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n(i)\ne2 (ˆuk) +\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n(i)\ne1 (ˆuk) +\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−→\nD 2,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n←−\nL\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n−→\nL\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.3**\nD. Simulation results\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nFig. 1 depicts the bit error performance of a turbo code\nwith two component codes (rate-1/3) and interleaver size\n16384, with standard parallel decoding, plain shufﬂed and\nreplica shufﬂed decoding. We observe that, to obtain the same\nerror performance, the replica turbo decoding requires about\nhalf the number of iterations of that of the standard parallel\nturbo decoding. Hence latency is greatly reduced (by half if\ninformation exchanges are ignored).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nIII. ITERATIVE DECODING OF LDPC CODES\nLet H = [Hmn] be the parity check matrix which deﬁnes\nan LDPC code. We denote the set of bits that participate in\ncheck m by N (m) = {n : Hmn = 1} and the set of checks\nin which bit n participates as M(n) = {m : Hmn = 1}.\nAssume a codeword w = (w1, w2, . . . , wN ) is transmitted\nover an AWGN channel with zero mean and variance N0/2\nusing BPSK signaling and let y = (y1, y2, . . . , yN ) be the\ncorresponding received sequence.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nA. Standard BP for iterative decoding of LDPC codes\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nBased on [1], let Fn be the log-likelihood ratio (LLR) of\nbit n and initially set Fn = 4\nmn be the\nN0\nLLR of bit n which is sent from check node m to bit node n\nand sent from the bit node n to check node m, respectively.\nLet z(i)\nn denote the a posteriori LLR of bit n. The standard\nBP algorithm [1] is carried out as follows:\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.8**r== 0.1**\nmn and z(i)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\nyn. Let ε(i)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.7**\nIM ax. For each m,n, set z(0)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.3**r== 0.6**\nmn = Fn.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nStep 1\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n(i) Horizontal Step, for 1 ≤ n ≤ N and each m ∈\nM(n), process:\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nτ (i)\nmn =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\ntanh(z(i−1)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nmn /2)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n(1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.5**\nn∈N (m)\\n\n1 + τ (i)\nmn\n1 − τ (i)\nmn\n(ii) Vertical Step, for 1 ≤ n ≤ N and each m ∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nε(i)\nmn = log\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n(2)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nM(n), process:\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nz(i)\nmn = Fn +\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nε(i)\nmn\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nz(i)\nn = Fn +\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\nm∈M(n)\\m\nε(i)\nmn\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nm∈M(n)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n(3)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n(4)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n(i)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nStep 2 Hard decision and stopping criterion test:\nCreate ˆw(i) = [ ˆw(i)\nn ] such that ˆw(i)\nz(i)\nn > 0, and ˆw(i)\nn = 1 if z(i)\nn < 0.\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 3. Otherwise set\ni := i + 1 and go to Step 1.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nn = 0 if\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n(ii)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nStep 3 Output ˆw(i) as the decoded codeword.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nB. Plain shufﬂed BP for iterative decoding of LDPC codes\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nAt the i-th iteration of the standard BP algorithm, ﬁrst all\nvalues of the check-to-bit messages are updated by using the\nvalues of the bit-to-check messages obtained at the (i − 1)-\nth iteration, i.e., each ε(i)\n:\nn ∈ N (m)\\n}. Then, all values of the bit-to-check messages\nare updated by using the values of the check-to-bit messages\nnewly obtained at the i-th iteration, i.e., each z(i)\nmn is updated\nfrom {ε(i)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.5**\nmn is updated by using {z(i−1)\nmn\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nmn : m ∈ M(n)\\m}.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nWe observe that certain values z(i)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nin (1) to compute the remaining values ε(i)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nmn could already be\ncomputed in (3) based on a partial computation of the values\nε(i)\nmn obtained from (2), and then could be used instead of\nz(i−1)\nmn. Hence Step\nmn\n1 of the shufﬂed BP algorithm is performed as: for 1 ≤ n ≤ N\nand each m ∈ M(n), process the horizontal step and vertical\nstep jointly, with (1) modiﬁed as:\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.7**\ntanh(z(i)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nmn /2)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\ntanh(z(i−1)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nmn /2)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nτ (i)\nmn =\n**BLOCK**fs== 5.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nn∈N (m)\\n\nn<n\n**BLOCK**fs== 5.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nn∈N (m)\\n\nn>n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nSince this algorithm becomes totally serial, bits can be grouped\nto maintain a sufﬁcient level of parallelism with the same error\nperformance [7].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nC. Replica shufﬂed BP decoding for LDPC codes\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nPlain shufﬂed BP decoding is a bit-based sequential ap-\nproach and the method described in Section III-B is based\non a natural increasing order, i.e, belief messages concerning\nbit nodes are updated according to order i = 1, 2, . . . , N .\n**BLOCK**fs== 10.0**b== 0.6**t== 0.1**l== 0.5**r== 0.1**\nThe larger the value of i, the more independent informations\nare used to update the beliefs of bit note i and the more\nreliable these belief messages are. Therefore the reliability of\nbit nodes increases and the error rate decreases as i increases.\nFollowing a similar analysis, in plain shufﬂed BP decoding\nbased on a natural decreasing order, after each iteration, the\nreliability of bit nodes decreases as i increases. As in replica\nshufﬂed turbo decoding, in replica shufﬂed BP decoding, two\nreplica shufﬂed subdecoders based on different updating orders\noperate simultaneously and cooperatively. After each iteration,\neach subdecoder receives more reliable messages from and\nsends more reliable messages to another subdecoder. Based on\nthese more reliable messages, both replica subdecoders begin\n←−\nthe next iteration of decoding. Let\nD denote the replica\nsubdecoder with natural increasing and decreasing updating\norder, respectively. Let −→ε (i)\nmn be the variables\n←−\nassociated with\nD\nare deﬁned in a similar way. The replica shufﬂed BP decoding\nwith two replica subdecoders is carried out as followings:\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.1**\n−→\nD at iteration i. Variables associated with\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\nmn and −→z i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n−→\nD and\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nInitialization Set i = 1, maximum number of iteration to\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\nIM ax. For each m,n, set −→z (0)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nmn = ←−z (0)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nmn = Fn.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nStep 1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.1**\nEach replica subdecoder process as the following\ntwo steps simultaneously. For 0 ≤ n ≤ N − 1 and\neach m ∈ M(n), process\nHorizontal Step\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n(i)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n−→τ (i)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nmn =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\ntanh(−→z (i)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\nmn/2)\n**BLOCK**fs== 5.0**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\nn∈N (m)\\n\nn<n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n·\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.8**r== 0.1**\ntanh(−→z (i−1)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\nmn /2)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n−→ε (i)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nmn = log\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nn∈N (m)\\n\nn>n\n1 + −→τ (i)\nmn\n1 − −→τ (i)\nmn\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\n←−τ (i)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\nmn =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.8**r== 0.1**\ntanh(←−z (i)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.9**r== 0.1**\nmn/2)\n**BLOCK**fs== 5.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nn∈N (m)\\n\nn>n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n·\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\ntanh(←−z (i−1)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nmn /2)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n←−ε (i)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\nmn = log\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\nn∈N (m)\\n\nn<n\n1 + ←−τ (i)\nmn\n1 − ←−τ (i)\nmn\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n(ii)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nVertical Step\n−→z (i)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nmn = Fn +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n←−z (i)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\nmn = Fn +\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nm∈M(n)\\m\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nm∈M(n)\\m\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n−→ε (i)\nmn\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.9**r== 0.1**\n←−ε (i)\nmn\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nStep 2 Exchange of\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nthe more reliable messages. Set\nmn = −→z (i)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nmn for 0 ≤ n < N/2 and ←−z (i)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\nmn\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nmn = ←−z (i)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n−→z (i)\nfor N/2 ≤ n ≤ N − 1.\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.5**r== 0.2**\nA. Conventional decoding methods of TPC\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.2**r== 0.8**\n(i)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n(ii)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n←−ε (i)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nm∈M(n)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.3**r== 0.5**\nn ] such that for 0 ≤ n <\nn = 1 if Fn +\nmn <\nn = 0 otherwise; for N/2 < n ≤\nn = 1 if Fn +\nmn <\nn = 0 otherwise.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.2**r== 0.5**\nCreate ˆw(i) = [ ˆw(i)\nN/2, ˆw(i)\n0, and ˆw(i)\nN −1, ˆw(i)\n0, and ˆw(i)\nIf H ˆw(i) = 0 or IM ax is reached, stop the\ndecoding and go to Step 4. Otherwise set\ni := i + 1 and go to Step 1.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nm∈M(n)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n−→ε (i)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nStep 4 Output ˆw(i) as the decoded codeword.\nAn alternative approach that can be used is to make the\ntwo subdecoders exchange the more reliable messages after\nupdating the reliability message of each bit (or a group of\nbits). We observe that this “simultaneous replica” approach\nprovides a faster convergence, especially for Gallager-type\nLDPC codes. Thus all simulation results of LDPC codes are\nbased on simultaneous updating of each bit (or a group of\nbits). It is also straightforward to extend the replica shufﬂed\nBP decoding to the cases in which more than two replica\nsubdecoders are used. In order to decrease decoding delay\nof plain shufﬂed BP decoding, a parallel version of shufﬂed\nBP named group shufﬂed BP was developed in [7]. In a\nsimilar way, group replica shufﬂed BP can also preserve the\nparallelism advantage of the standard BP algorithm. It is also\nclear this idea can be extended to other other grouping scheme\n(e.g, [11]) and other iterative decoding algorithms, such as bit\nﬂipping and weighted bit ﬂipping decoding.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nD. Simulation results\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nFig. 2 depicts the word error rate (WER) of iterative de-\ncoding of a (8000, 4000)(3, 6) LDPC code, with the standard\nBP, plain shufﬂed and group replica shufﬂed BP algorithm, for\nG = 2, 4, 8, 16 and 8000 and with four replica subdecoders.\nThe maximum number of iterations for plain and group replica\nshufﬂed BP was set to be 10. We observe that the WER\nperformance of replica shufﬂed BP decoding with four subde-\ncoders, Imax=10, and a group number larger or equal to four,\nare approximately the same as that of the standard BP with\nImax=60. Fig. 3 depicts the WER of the standard and replica\nshufﬂed BP decoding of the (16200, 7200) irregular LDPC\ncode which is constructed in a semi-random matter [12].\nThe variable node degree distribution is λ(x) = 0.00006x +\n0.57772x2 + 0.3111x3 + 0.11111x8. The number of replica\nsubdecoders was four. We observe that the replica shufﬂed BP\nwith Imax=10 and G = 32 provides a similar performance as\nthat of the standard BP with Imax=70.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nIV. ITERATIVE DECODING OF TURBO PRODUCT CODES\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nA two-dimension turbo product code (TPC) can be denoted\nas C1\nC2, where C1 and C2 are two linear block codes.\nPlace k1 × k2 information symbols in an array of k1 rows\nand k2 columns, and then encode the k1 rows using code C2.\nAfterwards, the resulting n2 columns are encoded using code\nC1. Usually, we choose C1 the same as C2.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nThe conventional TPC decoder performs row and column\ndecoding in a serial fashion. A soft input soft output (SISO)\ndecoder, such as MAP, is used to decode each row or column.\nA low complexity decoding approach is provided in [13]. It\napplies the Chase algorithm iteratively on the row and column\ndecoding, but still in a serial fashion. In order to halve the\ndecoding latency, a parallel TPC decoder has been proposed\nin [10]. As opposed to the conventional serial TPC decoder,\nthe row and column decoders in this method operate in\nparallel and send each other the updated extrinsic information\nimmediately after a row or column has been decoded. The\nsimulation results reveal that this parallel decoder can reduce\ndecoding latency by half of that of the original decoder.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.3**\nB. Replica decoding of TPC\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nBased on the parallel decoder, we propose a replica parallel\ndecoder as shown in Fig. 4, which can further reduce decoding\nlatency. Both row and column decoders are duplicated, but\nwork from opposite extremes, which means the two row\ndecoders process rows from the top and bottom, respectively\nand the two column decoders process columns from the\nleft and right, respectively. Row (Column) decoders send to\nboth column (row) decoders immediately the latest extrinsic\nmatrices, [Wcol\nB ] ([Wrow\nR ]), after a row\n(column) has been decoded.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nL ] and [Wrow\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nT ] and [Wcol\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe procedure of the replica parallel decoding is shown\nin Fig. 5. The circles denote bit positions that were already\nupdated by other decoders. Their number is much larger than\nthat in the parallel decoder, which greatly beneﬁts the decoding\nbecause most bits have more accurate priori information. The\narrows with letters aT , bT , etc., represent the processing order\nof different decoders. After both row (column) decoders ﬁnish\ndecoding all the rows (columns), the most reliable parts of\nthem are combined and the resulting extrinsic matrix for the\nnext iteration is transmitted to both column (row) decoders.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nC. Simulation results\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nFig. 6 depicts\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nthe performance of TPC (64, 57, 4)2.\nare\nThe weighting factors with the\n[0.1, 0.2, 0.3, 0.5, 0.7, 0.9, 1.0, 1.0]. The MAP algorithm is\nused to decode the two linear block codes. As shown in Fig.\n6, the performance at the third iteration in the replica decoder\nis better than that at the fourth iteration in the parallel decoder.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\niteration number\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n[1] D. J. C. MacKay, “Good error-correcting codes based on very sparse\nmatrices,” IEEE Trans. Inform. Theory, vol. 45, pp. 399-431, Mar. 1999.\n[2] R. G. Gallager, Low-Density Parity-Check Codes. Cambridge, MA:\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nM.I.T. Press, 1963.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n[3] C. Berrou and A. Glavieux, “Near-optimum error-correcting coding and\ndecoding: Turbo-codes,” IEEE Trans. Commun, vol. 44, pp. 1261-1271,\nOct. 1996.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[4] L. Bahl, J. Cocke, F. Jelinek, and J. Raviv, “Optimal decoding of linear\ncodes for minimizing symbol error rate,” IEEE Trans. Inf. Theory, pp.\n284-287, Mar. 1974.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[5] H. Kﬁr and I. Kanter, “Parallel versus sequential updating for belief\npropagation decoding,” submitted to Physical Review E, July 2002.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nFig. 1. Bit error performance of 2-component turbo code with interleaver\nsize 16384, for standard parallel, plain shufﬂed and replica shufﬂed decodings.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nFig. 3.\nirregular LDPC code.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\nError performance for iterative decoding of the (16200, 7200)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nFig. 2. Error performance for iterative decoding of the (8000, 4000)(3, 6)\nLDPC code with group shufﬂed BP algorithm, for G = 1, 2, 8, 100, 8000\nand at most 10 iterations.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n[6] J. Zhang and M. Fossorier, “Shufﬂed Belief Propagation Decoding,”\nProceedings of 36th Annual Asilomar Conference on Signals, Systems\nand Computers, pp. 8-15, Nov. 2002.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n[7] J. Zhang and M. Fossorier, “Shufﬂed Belief Propagation Decoding,”\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nIEEE Trans. Commun., Feb. 2005.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n[8] J. Hagenauer, E. Offer and L. Papke, “Iterative decoding of block and\nconvolutional codes,” IEEE Trans. on Inform., vol. 42, pp. 429-445,\nMarch 1996.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n[9] D. Divsalar and F. Pollara, “Multiple turbo codes for deep-space\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\ncommunications,” JPL TDA Progress Report, 71-78, May 1995.\n[10] C. Argon and S. McLaughlin, “A parallel decoder for low latency\ndecoding of turbo product codes,” IEEE Commun. Lett., vol. 6, pp. 70-\n72, Feb. 2002.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n[11] E. Yeo, P. Pakzad, B. Nikolic and V. Anantharam, “High throughput\nlow-density parity-check decoder architectures,” Proceedings of Global\nTelecommunications Conference, vol. 5, pp. 3019-3024, Nov. 2001.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n[12] “Draft DVB-S2 Standard,” availible at http://www.dvb.org.\n[13] R. M. Pyndiah, ”Near-optimum decoding of product codes: Block turbo\ncodes”, IEEE Trans. Commun., vol. 46, pp. 1003-1010, Aug. 1998.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\nFig. 4. Replica parallel decoder (one iteration)\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\nFig. 5. Procedure of replica parallel decoder\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nFig. 6.\ndecoders.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.1**\nPerformance of TPC (64, 57, 4)2 with parallel and replica parallel",
         "extracted",
         "Low-Density Parity-Check Codes;Turbo decoder architectures for low-density parity-check codes;Shuffled belief propagation decoding;A parallel decoder for low latency decoding of turbo product codes;Parallel versus sequential updating for belief propagation decoding;High throughput low-density parity-check decoder architectures;Good error-correcting codes based on very sparse matrices;Near optimum error correcting coding and decoding: turbo-codes;Multiple turbo codes for deep-space communications;Optimal decoding of linear codes for minimizing symbol error rate (Corresp.);Generic Architecture for LDPC Codes Decoding;Optimal decoding of linear codes for minimizing symbol error rate",
         "Shuffled iterative decoding",
         "None"
        ],
        [
         "2",
         "000376ec6f6ade6261fc12df9eddd25a88c1e9ca",
         "This study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus BUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath. In endothelium-intact aortic rings, A. membranaceus extract induced a significant dose-dependent relaxation of the rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(omega)-nitro-arginine methyl ester or methylthioninium chloride. In endothelium-denuded ones, the extract could dose-dependently relax the rings contracted by phenylephrine, not by KCl; and it could also attenuate contractile response to phenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca(2+)-free medium; but it failed to affect the CaCl(2)-induced enhancement of contractile response to phenylephrine in Ca(2+)-free medium. These results indicate that nitric oxide signaling and Ca(2+)-handling pathway are involved in the A. membranaceus extract-induced vasodilatation.",
         "Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,Jian Sun,Q. Xia,K. Bian",
         "https://www.jstage.jst.go.jp/article/bpb/28/8/28_8_1450/_pdf",
         "\n**BLOCK**fs== 14.0**b== 0.8**t== 0.1**l== 0.1**r== 0.2**\nDiphasic Effects of Astragalus membranaceus BUNGE (Leguminosae) on\nVascular Tone in Rat Thoracic Aorta\n**BLOCK**fs== 7.2**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nc,d\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\nBi-Qi ZHANG,a Shen-Jiang HU,*,a,d Li-Hong QIU,a Qi-Xian SHAN,b Jian SUN,a Qiang XIA,b and\nKa BIAN\na Department of Cardiology, The First Afﬁliated Hospital, College of Medicine, Zhejiang University; No. 79, Qingchun St,\nHangzhou 310003, Zhejiang, P.R. China: b Department of Physiology, College of Medicine, Zhejiang University; No. 353,\nYan’an  St,  Hangzhou  310031,  Zhejiang,  P.R.  China:  c Department  of  Integrative  Biology  and  Pharmacology,  The\nUniversity of Texas-Houston Medical School; 6431 Fannin, Houston, Texas, 77030, U.S.A.: and  d E-Institute of Shanghai\nUniversities, Division of Nitric Oxide and Inﬂammatory Medicine; No. 1200, Chalun St, Shanghai 201203, P.R. China.\nReceived March 22, 2005; accepted April 28, 2005\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nThis study was designed to investigate the effects of the aqueous ethanol extract of Astragalus membranaceus\nBUNGE (Leguminosae) on rat thoracic aorta. Isometric tension was recorded in response to drugs in organ bath.\nIn endothelium-intact aortic rings, A. membranaceus extract induced a signiﬁcant dose-dependent relaxation of\nthe rings precontracted by phenylephrine, which could be inhibited by preincubation with L-N(ww)-nitro-arginine\nmethyl  ester  or  methylthioninium  chloride.  In  endothelium-denuded  ones,  the  extract  could  dose-dependently\nrelax  the  rings  contracted  by  phenylephrine,  not  by  KCl;  and  it  could  also  attenuate  contractile  response  to\nphenylephrine, not to caffeine or phorbol-12,13-diacetate in Ca2-free medium; but it failed to affect the CaCl2-\ninduced enhancement of contractile response to phenylephrine in Ca2-free medium. These results indicate that\nnitric oxide signaling and Ca2-handling pathway are involved in the A. membranaceus extract-induced vasodi-\nlatation.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.2**r== 0.3**\nKey words Astragalus membranaceus; nitric oxide; calcium ion; thoracic aorta; vasomotion\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe  roots  of  Astragalus  membranaceus BUNGE (Legumi-\nnosae) are amongst the most popular and important “Qi (pro-\nnounce  chee)  tonifying”  adaptogenic  herbs  in  China,  their\nuse  dates  back  more  than  2000  years,  and  are  recorded  in\nShen  Nong’s Materia  Medica.  According  to  ethnobotancical\ndata  collected  in  China,  A.  membranaceus has  been  pre-\nscribed  for  centuries  for  diarrhea,  frequent  colds,  sponta-\nneous  sweats,  fatigue,  and  edema.  Furthermore,  A.  mem-\nbranaceus is  alleged  to  possess  the  effects  of  antioxidative\ndamage,1) immune-stimulation,2) and antiviral infection.3)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nRecently,  it  has  received  more  attention  that  the  fact\nrecorded in ancient Chinese texts that A. membranaceus pos-\nsessed an antihypertensive effect by raising the yang qi of the\nspleen and kidney, since deﬁciency qi formed by the disorder\nof spleen and kidney is considered to be the basic pathogene-\nses  of  hypertension.4) A  great  deal  of  studies  demonstrated\nthat  A.  membranaceus was  beneﬁcial  on  the  process  of  hy-\npertension  in  animals  and  humans,5—7) and  a  cooperative\nmechanism  of\nrenin  angiotensin  aldosterone  system,\nkallikrein bradykinin system, and central neuropeptide8) was\nused  to  explain  this  effect  of  A.  membranaceus.  We  also\nfound  in  the  past  experiments  that  the  administration  of  A.\nmembranaceus could attenuate the progression of blood pres-\nsure  in  spontaneously  hypertensive  rats  concomitant  with\nrestoration  of  baroreﬂex  sensitivity  and  reduction  of  Ca2\nconcentration  in  lymphocyte  and  vascular  smooth  muscle\ncells  (VSMC).9—11) Additionally,  in  vivo  experiment  had\ndemonstrated that A. membranaceus could induce the vasodi-\nlatation.12) However,  there  is  no  experiment  evidence  avail-\nable  to  show  the  exact  mechanism  of  vasorelaxation  pro-\nduced by A. membranaceus. The present study is conducted\nto  elucidate  various  possible  mechanisms  in  vasomotor  re-\nsponse of A. membranaceus.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nMATERIALS AND METHODS\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nPlant  Material  and  Preparation  of  the  Extracts A.\nmembranaceus roots were collected in the northern region of\nGansu Province, China, in the autumn (2001) and identiﬁed\nby the Quality Assessment of Di’ao Group, Sichuang, China.\nAnd a voucher specimen was deposited in the Academic De-\npartment of Di’ao Group (STQ-C-00203).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nA. membranaceus roots were air dried. A crude extract was\nprepared  by  decoction  of  2.0102 g  in  500 ml  water  for  3\ntimes (90 min per time). The obtained extract was combined,\nﬁltered  and  concentrated.  Then  all  the  aqueous  extract  was\nextracted  with  400 ml  ethanol  (75%  and  85%)  twice.  After\nthe ethanol was retrieved, the active fractions were recovered,\nﬁltered  and  lyophilized  (yield  0.28%).  Consequently,  the\nﬁnal  extract  was  used  in  the  study  after  dissolution  in  dis-\ntilled water at 37 °C and stirred for 60 min.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nChemicals Phenylephrine  (PE),  acetylcholine  (ACh),  L-\nN(w)-nitro-arginine  methyl  ester  (L-NAME),  methylthion-\ninium  chloride  (MC),  phosphoramidon,  ethylene  glycol-bis\n[b-aminoethyl  ether]-N,N,N,N-tetraacetic  acid  (EGTA),\ncaffeine  and  phorbol-12,13-diacetate  (PD)  were  purchased\nfrom  Sigma  Chemical  Co.  (St.  Louis,  MO,  U.S.A.).  They\nwere  dissolved\nin  distilled  water  and  diluted  with\nKrebs–Henseleit  (K–H)  solution  before  use.  All  chemicals\nwere of the highest purity available.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nExperiment  Animals The  investigation  conformed  to\nthe  Chinese  Regulation  for  Administration  of  Laboratory\nAnimal  (SSTC  1988/2).  Health  male  Sprague–Dawley  rats\nweighing  260—280 g  (Laboratory  Animal  Center  of  Zhe-\njiang  College  of  Traditional  Chinese  Medicine,  Hangzhou,\nChina) were housed in conventional cages with free access to\nwater and rodent chow at the controlled temperature and hu-\nmidity with a 12-h light/dark cycle.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nDawley  Rats Rats  were  euthanized  by  decapitation  using\nsmall animal guillotine. The thoracic aorta was immediately\nisolated  and  immersed  in  oxygenated  K–H  solution  4 °C  of\nthe  following  composition  (in  mM):  NaCl  118.3;  KCl  4.7;\nMgSO4 1.2; KH2PO4 1.2; NaHCO3 25.0; CaCl2 2.5; Glucose\n11.0.  Then  the  adherent  connective  tissue  was  cleaned  and\nblood  vessel  was  cut  into  3—4 mm  rings,  with  the  special\ncare  of  avoiding  damage  of  endothelium.  In  some  prepara-\ntions, endothelium was mechanically removed by gently rub-\nbing  the  lumen.  The  rings  were  mounted  horizontally  be-\ntween two stirrups in organ bath ﬁlled with 10 ml K–H solu-\ntion  at  37 °C,  ventilated  continuously  with  95%  O2 and  5%\nCO2. The isometric tension was recorded with a force trans-\nducer  (JZ101,  metrical  range:  0—5.0 g)  and  MedLab  5.0v\nrecording system (Nanjing Medease Science and Technology\nCo. Ltd).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nExperimental  Procedure  of  Isometric  Tension Rings\nwere equilibrated for 60 min at 2.0 g resting tension, and then\nchallenged with KCl (6.0102\nM) at least 3 times until a re-\nproducible maximal contractile response was obtained. After\na  further  equilibration  period  of  30 min,  the  integrity  of  the\nendothelium was assessed in all preparations by determining\nthe ability of ACh (1.0105\nM) to induce more than 80% re-\nlaxation  of  rings  pre-contracted  with  PE  (1.0106\nM).  The\nendothelium  was  considered  to  be  removed  when  there  was\nless than 10% relaxation response to ACh.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nProtocol 1: Effects of A. membranaceus on Vascular Tone:\nThe ﬁrst series of the experiments were conducted to assess\nendothelium-dependent  or  independent  effects  of  A.  mem-\nbranaceus on isolated aortic rings. When the tension was at\nresting  state  or  reached  a  plateau  induced  by  PE  (3.0\n107\nM),  A.  membranaceus (2.0105—2.0101 g/l)  was\ncumulatively added into the organ bath with 10 min interval.\nThe  rings  with  intact  or  denuded  endothelium  were  always\ntested in parallel.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nProtocol  2:  Endothelium-Independent  Vasomotor  Re-\nsponse by A. membranaceus: The second series of the exper-\niments  were  designed  to  determine  the  underlying  mecha-\nnisms  of  A.  membranaceus-induced  vasomotor  response  in\naortic rings without endothelium. In contrast to the response\nof  A.  membranaceus (2.0105—2.0101 g/l)  to  PE,  KCl\n(3.0102\nM)  was  also  employed  to  constrict  the  denuded\naortic rings. A plateau contractile response to KCl or PE was\nalso obtained as a time control.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.5**\nM), PD (1.0106\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nAdditionally,  after  the  preparations  were  serially  washed\nand equilibrated for 60 min by Ca2-free medium (mM: NaCl\n118.3, KCl 4.7, KH2PO4 1.2, MgSO4 1.2, NaHCO3 25, Glu-\ncose 11.0, EGTA 0.05), PE (3.0107\nM)\n(an  activator  of  protein  kinase  C,  PKC),13) or  caffeine\n(2.5102\nM) (an activator of ryanodine receptor)14) was used\nto contract the denuded rings. In order to explore the proba-\nble  effect  of  extracellular  Ca2 inﬂow,  2.5 mM CaCl2 was\nadded  after  PE  stimulated  the  rings.  Then  the  effect  of  A.\nmembranaceus (6.0103 g/l)  was  tested.  A  time  control\n(same protocol as above except for the omission of A. mem-\nbranaceus to  the  second  application  of  PE,  PD  or  caffeine)\nwas  always  run  in  parallel.  Following  the  initial  control  re-\nsponse in Ca2-free medium and before the second test con-\ntraction (to PE or caffeine), a 6.0102\nM KCl-induced con-\ntraction  in  K–H  solution  was  obtained  to  reﬁll  the  Ca2\nstores in the vascular tissues.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nProtocol 3: Endothelium-Dependent Vasomotor Response\nby A. membranaceus: In the third series of experiments, the\npotential inﬂuence of A. membranaceus on PE-induced vaso-\nmotor function in the intact rings was examined. For pretreat-\nment,  endothelium-intact  rings  were  incubated  with  L-\nNAME (1.0104\nM) (a speciﬁc inhibitor of nitric oxide syn-\nthase), MC (1.0105\nM) (an inhibitor of guanylate cyclase)\nfor 15 min15) and phosphoramidon (5.0106\nM) (an inhibitor\nof  endothelin  converting  enzyme)  for  20 min16) before  ad-\nministration  of  PE.  Then  the  response  curves  of  A.  mem-\nbranaceus were recorded. The rings with or without the treat-\nment of inhibitors were always tested in parallel.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nStatistical  Analysis Relaxant  responses  were  expressed\nas the percentage decreases of the magnitude of the contrac-\ntion  induced  by  PE  or  KCl  (100%)  before  the  application\nof  vasodilators,  and  the  contractile  responses  as  the  ratio  of\nthe  responses  following  and  before  the  application  of  A.\nmembranaceus (second  response/ﬁrst  response).  All  results\nare  expressed  as  meanS.E.M.  Statistical  analysis  was  per-\nformed  with  t-test  or  ANOVA  followed  by  Newman–Keuls\ntest. Differences were accepted as statistically signiﬁcant at p\nvalues 0.05 (GraphPad Prism).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nRESULTS\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nPE  (1.0106\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nM)  induced  a  similar  sustained  contraction\nof  aortic  rings  in  each  group  with  a  peak  tension  of  about\n3.110.27 g  in  intact  aortic  rings  and  3.020.19 g  in  de-\nnuded ones.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nEffects  of  A.  membranaceus on  the  Vasotension All\ndoses (2.0105—2.0101 g/l) of A. membranaceus exhib-\nited  negligible  vasomotor  actions  on  aortic  rings  with  or\nwithout  endothelium  at  resting  tension  (Fig.  1).  A.  mem-\nbranaceus (2.0105—2.0101 g/l)  produced  dose-depen-\ndent  relaxation  in  endothelium-denuded  rings  (Fig.  2),  after\nthe  plateau  tension  induced  by  PE.  However,  in  PE-precon-\ntracted  endothelium-intact  rings,  low  doses  (2.0104—\n6.0103 g/l) of A. membranaceus showed a more signiﬁcant\nrelaxation than that in denuded ones (Fig. 2), while high dose\n(2.0102—2.0101 g/l) of the extract induced a transient\ncontraction (Figs. 2, 3).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nEndothelium-Independent  Vasodilatation  by  A.  mem-\nIn  contrast  to  the  effect  on  PE-contracted  en-\nbranaceus\ndothelium-denuded rings in K–H solution, A. membranaceus\n(2.0104—2.0101 g/l)  had  no  dilatation-effect  on  rings\nstimulated by KCl (3.0102\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nM) (Fig. 4).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nAnd in the experiment with Ca2-free medium, aortic con-\ntraction  elicited  by  PE,  caffeine  or  PD  was  decreased  sub-\nstantially, as these vascular contractions in Ca2-free medium\nare mediated only via the release of Ca2 from the intracellu-\nlar stores or phosphorylation of myosin light chain. A. mem-\nbranaceus (6.0103 g/l)  could  inhibit  the  contraction  in-\nduced  by  PE  (3.0107\nM),  but  not  block  the  extracellular\nCa2 inﬂow,  because  2.5 mM CaCl2 induced  a  similar  en-\nhancement of contraction by PE, compared with the control\n(95.842.63%  vs. 96.961.87%,  p0.05).  Furthermore,  A.\nmembranaceus (6.0103 g/l) failed to induce an attenuation\nof  contractile  response  by  PD  (1.0106\nM)  and  caffeine\n(2.5102\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nM) (Fig. 5).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nEndothelium-Dependent  Vasomotor  Effects  by  A.\nmembranaceus Pre-incubation  of  the  intact  rings  with  L-\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nFig. 1. Effects of A. membranaceus on the Vasomotion of Aorta Rings in the Resting Tension\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nA. membranaceus (2.0105—2.0101 g/l) was cumulatively added into the bathing solution in 10 min interval during the resting phase (2.0 g) in (A) endothelium intact or (B)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nremoved rings. Values of each group are expressed as meanS.E.M. of 8 experiments.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nFig. 2. Vasorelaxant  Effects  of  A.  membranaceus (2.0105—2.0101 g/l)  on  Phenylephrine  (3.0107\nwith or (B) without Endothelium\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.1**\nM)-Induced  Contractions  in  Aortic  Rings  (A)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nValues of both groups are expressed as meanS.E.M. of 8 experiments. Contractile responses are expressed as the percentage from maximal contraction elicited by phenyl-\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nephrine (3.0107\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.2**r== 0.6**\nM). ∗ p0.05, ∗∗ p0.01, compared with the control group.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nFig. 3. Transient Vasocontractile Response of A. membranaceus (Closed Circle) against Phenylephrine-Induced Endothelium-Intact Aortic Rings\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nWith (A) or (B) without the incubation of phosphoramidon (5.0106\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nM) was added, the effect of A. membranaceus (2.0102,\n6.0102 and 2.0101 g/l, showed by a, b, c point) was obtained. For it was similar with Fig. 2, the effect of A. membranaceus (2.0105—6.0103 g/l) was cut. Values of both\ngroups are expressed as meanS.E.M. of 8 experiments. Contractile responses are expressed as the percentage from maximal contraction elicited by phenylephrine (3.0107\nM).\n∗ p0.05, ∗∗ p0.01, compared with corresponding values before administration.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nM) for 20 min before phenylephrine (3.0107\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nNAME  (1.0104\nM)  markedly,  incompletely  inhibited  A.\nmembranaceus (6.0103 g/l)-induced  relaxation  from  36.0\nto 7.3% (Fig. 6). Similarly, MC (1.0105\nM) also showed an\ninhibition  of  endothelium-dependent  vasodilatation  from\n36.0  to  8.0%  (Fig.  6).  Thus,  NO  signaling  might  play  a\nmajor, but not full role in the vasodilatation effect of A. mem-\nbranaceus.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nWhen the dose of A. membranaceus in the organ bath was\nstepwise raised to 2.0102—2.0101 g/l, a transient con-\ntraction  could  be  observed  in  endothelium-intact  rings.  Pre-\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\ntreatment  of  endothelium-intact  rings  with  phosphoramidon\n(5.0106\nM)  signiﬁcantly  attenuated  the  transient  contrac-\ntion response (Fig. 3).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nDISCUSSION\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe present study ﬁrst and detailedly explored the in vitro\nvascular effects of A. membranaceus in aortic rings isolated\nfrom rats. In the course of the study, a number of novel ob-\nservations  had  been  made,  which  contribute  to  better  scien-\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\ntiﬁc  understanding  of  their  therapeutic  use  for  antihyperten-\nsion associated with Qi deﬁciency caused by the disorder of\nspleen and kidney in China.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nFirst, although neurohumoral factors may contribute to an-\ntihypertension for the crude extract of A. membranaceus ob-\nserved previously,8) it is apparent that direct endothelium-de-\npendent  and  independent  vasorelaxation  should  be  promi-\nnently taken into account according to this study.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nSecond,  A.  membranaceus dose-dependently  dilated  en-\ndothelium-denuded  rings  contracted  with  PE.  However,  the\nagent had no effect on the vessel contracted by KCl. The cel-\nlular  mechanism  of  contraction  involved  in  the  response  to\nKCl and PE is different. KCl induce Ca2 inﬂux via voltage-\ndependent  Ca2 channel,  which  further  activate  Ca2-in-\nduced Ca2 release through ryanodine-receptor. PE increase\nintracellular  Ca2 concentration  with  two  mechanisms,  1)\nthrough  receptor-gated  Ca2 channels;  2)  mobilizes  Ca2\nfrom  intracellular  stores  via the  inositol-1,4,5-trisphosphate\n(IP3)  receptor  or  induces  secondly  myosin  light  chain  phos-\nphorylation via activating PKC.17,18) That is, the vasorelaxant\naction of A. membranaceus seems to occur in a receptor-de-\npendent manner in SMVC of rat aortas. However, the result\nthat A. membranaceus failed to inhibit the contraction caused\nby addition of CaCl2 in Ca2 free solution was probably only\ndue to its effects on the intracellular pathways. This hypothe-\nsis was conﬁrmed in the further experiments that the contrac-\ntions induced by PD and caffeine in Ca2-free medium were\nnot  affected  by  A.  membranaceus equally.  Thus,  it  appears\nthat the IP3-induced Ca2 release channels might be the site\nof action for A. membranaceus on endothelium-removed aor-\ntas.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nThird,  NO  is  a  potent  vasodilator  synthesized  in  the  en-\ndothelium19) by  NO  synthase,  and  causes  VSMC  relaxation\nthrough  the  activation  of  soluble  guanylate  cyclase.20) The\npresent  study  demonstrated  that  the  extract  of  A.  mem-\nbranaceus dose-dependently  inhibited  the  contraction  in-\nduced  by  PE  in  intact  aorta  isolated  from  rats.  This  vasore-\nlaxant  action  was  mostly  inhibited  by  treatment  with  L-\nNAME  or  MC.  Endothelium-dependent  relaxation  of  A.\nmembranaceus seemed  to  be  associated  with  NO  signaling\nvia guanylate cyclase activation since both L-NAME and MC\ncould  attenuate  this  response.  This  result  was  supported  by\nother studies which had shown that A. membranaceus could\nincrease the plasma level of NO21) and induce the release of\nNO  from  VSMC  in  vitro  by  NO-soluble  guanylate  cyclase-\ncyclic guanosine monophosphate pathway.22)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nFinally,  it  was  also  observed  that  high  dose  (2.0102—\n2.0101 g/l) A. membranaceus induced a transient contrac-\ntion  in  arterial  rings  pre-constricted  by  PE.  This  result  was\nidentical with that obtained in another report where treatment\nwith  high  dose  of  A.  membranaceus reversibly  increased\nblood pressure of patients without hypertension previously.23)\nThe  mechanism  underlying  is  under  investigation.  Blockage\nof  the  contraction  by  endothelin-converting  enzyme  phos-\nphoramidon  may  suggest  the  involvement  of  an  endothelin\nreceptor mediated signaling.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nIn summary, the ﬁndings indicated that A. membranaceus\nhave  diphasic  effects  of  relaxation  and  contraction  on  the\naorta rings. The mechanisms of relaxation may include inhi-\nbition  of  intracellular  Ca2 release  in  vascular  smooth  mus-\ncle cells, and endothelium dependent relaxation mediated by\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nFig. 4. Vasorelaxant  Effects  of  A.  membranaceus (2.0105—2.0101\ng/l) on KCl (3.0102\nM)-Induced Contractions in Aortic Rings without En-\ndothelium\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nValues of both groups are expressed as meanS.E.M. of 8 experiments. Contractile\nresponses  are  expressed  as  the  percentage  from  maximal  contraction  elicited  by  KCl\n(3.0102\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nM).\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nInhibitory  Effect  of  A.  membranaceus (6.0103 g/l)  on  Phenyl-\nM)  and  Caffeine\nM)-Induced  Contractions  in  Ca2-Free  Medium  in  Endothelium-\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nM)-,  Phorbol-12,13-diacetate  (1.0106\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nEach  column  with  a  bar  represents  meanS.E.M.  of  8  experiments.  Data  are  ex-\npressed as the percentage of the test response ratio (after/before A. membranaceus) with\nrespect to the time control response ratios. ∗∗ p0.01, compared with value in the cor-\nresponding vehicle group.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nFig. 6. Vasorelaxant  Effects  of  A.  membranaceus (6.0103 g/l)  on\nPhenylephrine  (3.0107\nM)-Induced  Contraction  in  Presence  of  L-N(w)-\nNitro-arginine Methyl Ester (L-NAME) (1.0104\nM) and Methylthioninium\nChloride (MC) (1.0105\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.2**r== 0.5**\nM) in K–H Solution in Endothelium-Intact Rings\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nIn the untreatment group, A. membranaceus or vehicle group was not pretreated with\nL-NAME or MC. Each column with a bar represents meanS.E.M. of 8 experiments.\nContractile  responses  are  expressed  as  the  percentage  from  maximal  contraction\nelicited by phenylephrine (3.0107\nM). ∗ p0.05, ∗∗ p0.01, compared with value of\nthe  corresponding  vehicle  group.   p0.01,  compared  with  value  of  the  experiment\nsubgroup in the untreatment group.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nthe NO-GC pathway, whilst the effect of transient contraction\nof A. membranaceus at a high dose was related with endothe-\nlin release.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nAcknowledgements This work was supported by grants\nfrom  Province  Administration  of  Traditional  Chinese  Medi-\ncine,  Zhejiang,  P.R.  China  (Project  Number:  2000C54  and\n2003C078);  Foundation  for  Returnee  from  Personal  Depart-\nment  of  Zhejiang  Province,  P.R.  China  (Project  Number:\nZRZ-2001275);  and  by  E-Institutes  of  Shanghai  Municipal\nEducation  Commission,  P.R.  China  (Project  Number:  E-\n04010). The authors are grateful to Prof. Wenquan Liang of\nthe  Department  of  Pharmacology,  Zhejiang  University,  for\nhis technical help.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n1) Toda S., Shirataki Y., J. Ethnopharmacol., 68, 331—333 (1999).\n2)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nJiao Y., Wen J., Yu X., Chin. J. Integrated Tradit. Chin. West. Med., 19,\n356—358 (1999).\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n3) Huang Z. Q., Qin N. P., Ye W., Chin. J. Integrated Tradit. Chin. West.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nMed., 15, 328—330 (1995).\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n4) Gu  Y.,  Deng  D.  M.,  Chin.  Arch.  Tradit.  Chin.  Med.,  19,  324—326\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\n(2001).\nJia S. Q., Tianjin Pharm., 6, 24—26 (1994).\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\n5)\n6) Zhao C. L., Tradit. Chin. Med. Res., 14, 25—26 (1998).\n7) Zha Y. Z., J. Tradit. Chin. Med., 41, 329 (2000).\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\n8) Song D. J., Gu D. G., Mao S. Y., Chin. Tradit. Herb. Drugs, 20, 25—\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.4**\n28 (1989).\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\n9) Chen  Z.  K.,  Hu  S.  J.,  Sun  J.,  Xia  Q.,  Shen  Y.  L.,  Zheng  X.,  Chin.  J.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.3**\nLab. Diagn., 7, 403—405 (2003).\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n10) Chen  Z.  K.,  Hu  S.  J.,  Sun  J.,  Xia  Q.,  Shen  Y.  L.,  Tradit.  Chin.  Drug\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nRes. & Clin. Pharmacol., 14, 372—374 (2003).\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n11) Chen Z. K., Hu S. J., Zheng X., Wang G. B., Sun J., Xia Q., Shen Y.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nL., Chin. J. Chin. Mater. Med., 28, 155—158 (2003).\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n12) Guo Z. G., Xue S. W., J. Tradit. Chin. Med., 21, 73—75 (1980).\n13) Ko W. H., Yao X. Q., Lau C. W., Law W. I., Chen Z. Y., Kwok W., Ho\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nK., Huang Y., Eur. J. Pharmacol., 399, 187—196 (2000).\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n14) Chew D. K., Orshal J. M., Khalil R. A., Hypertension, 42, 818—824\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n(2003).\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n15) Huang Y., Life Sci., 60, 1749—1756 (1997).\n16) Zimmermann  M.,  Jung  C.  S.,  Vatter  H.,  Raabe  A.,  Seifert  V.,  Acta\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nNeurochir. (Wien), 144, 1213—1219 (2002).\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n17) Karaki H., Ozaki H., Hori M., Mitsui-Saito M., Amano K., Harada K.,\nMiyamoto S., Nakazawa H., Won C. J., Sato K., Pharmacol. Rev., 49,\n157—230 (1997).\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n18) Kobayashi S., Gong M. C., Somlyo A. V., Somlyo A. P., Am. J. Phys-\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\niol., 260, C364—370 (1991).\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n19) Moncada S., Palmer R. M., Higgs E. A., Pharmacol. Rev., 43, 109—\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n142 (1991).\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n20) Martin E., Davis K., Bian K., Lee Y. C., Murad F., Semin. Perinatol.,\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n24, 2—6 (2000).\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n21) He L., Wang R. G., You Z. L., Yang Z. W., Chen X. D., Li J., Zhan X.\nL., Wang J., Acad. J. Hunan Tradit. Chin. Med., 23, 4—6 (2003).\n22) Wang K. H., Foreign Med. Sci.: TCM Section, 18, 38—39 (1996).\n23) Wang  C.  H.,  Zhang  J.  J.,  Shandong  J.  Tradit.  Chin.  Med.,  15,  351\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n(1996).",
         "extracted",
         "",
         "Diphasic effects of Astragalus membranaceus BUNGE (Leguminosae) on vascular tone in rat thoracic aorta.",
         "None"
        ],
        [
         "3",
         "0003a5747ee3676be1b6d2c72c13a04e41811274",
         "In this report, we use zebrafish as a model system to understand the importance of CD146 in vascular development. Endothelial‐specific expression of CD146 was verified by whole‐mount in situ hybridization. Suppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly developed intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood flow through the ISV region, despite that fluorescence microangiography showed that the ISVs were present. This finding suggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally, remodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of CD146 protein expression resulted in a circulation shunt that caused the circulation to by‐pass part of the caudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino oligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N terminus deleted in an in‐frame manner. This region, therefore, is likely to contain elements critical for CD146 function. This study provides the first in vivo functional assessment of CD146 in embryonic development by showing that knockdown of CD146 protein expression severely hinders vascular development in zebrafish. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley‐Liss, Inc.",
         "Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V. Sukhatme",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/dvdy.20220",
         "\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nCritical Roles of CD146 in Zebraﬁsh Vascular\nDevelopment\nBarden Chan,1 Sonia Sinha,1 Dan Cho,1,2 Ramani Ramchandran,1† and Vikas P. Sukhatme1,2*\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.1**\nIn this report, we use zebraﬁsh as a model system to understand the importance of CD146 in vascular\ndevelopment. Endothelial-speciﬁc expression of CD146 was veriﬁed by whole-mount in situ hybridization.\nSuppression of CD146 protein expression by antisense morpholino oligonucleotides (MO) resulted in poorly\ndeveloped intersomitic vessels (ISVs). In CD146 morphants, we observed a lack of blood ﬂow through the\nISV region, despite that ﬂuorescence microangiography showed that the ISVs were present. This ﬁnding\nsuggests that the lumens of the developing ISVs may be too narrow for proper circulation. Additionally,\nremodeling of the caudal vein plexus into functional vascular tubes appeared to be affected. Suppression of\nCD146 protein expression resulted in a circulation shunt that caused the circulation to by-pass part of the\ncaudal artery/vein system. The same vascular defects were recapitulated by using a second morpholino\noligonucleotide. This morphant expressed a truncated CD146 protein with amino acids V32 to T57 at the N\nterminus deleted in an in-frame manner. This region, therefore, is likely to contain elements critical for\nCD146 function. This study provides the ﬁrst in vivo functional assessment of CD146 in embryonic\ndevelopment by showing that knockdown of CD146 protein expression severely hinders vascular\ndevelopment in zebraﬁsh. Developmental Dynamics 232:232–244, 2005. © 2004 Wiley-Liss, Inc.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nKey words: CD146; P1H12; Mel-CAM; zebraﬁsh vascular development; morpholino; ﬂuorescence microangiography\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.4**\nReceived 8 March 2004; Revised 13 August 2004; Accepted 26 August 2004\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nINTRODUCTION\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.6**\nCD146, also known as MUC18, A32,\nMel-CAM, and S-Endo1 (Lehmann et\nal., 1989; Shih, 1999), is a transmem-\nbrane glycoprotein that belongs to the\nimmunoglobulin superfamily with a\ncharacteristic V-V-C2-C2-C2 domain\n(Williams and Barclay,\nstructure\n1988; Sers et al., 1993; Shih, 1999).\nAlthough it was ﬁrst identiﬁed as a\nhuman melanoma-associated antigen\n(Holzmann et al., 1987; Lehmann et\nal., 1987), this protein is also highly\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\nexpressed in the endothelium (Bardin\net al., 1996a,b). In fact, it was deter-\nmined that CD146 was the antigen\nrecognized by a highly endothelial-\nspeciﬁc monoclonal antibody called\nP1H12 (Solovey et al., 2001).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.4**r== 0.4**\nAlthough its cellular function is rel-\natively poorly understood, CD146 is\nbelieved to play a role in cell– cell ad-\nhesion. It has been shown by immuno-\nhistochemical studies that CD146 is\nlocalized at the intercellular junctions\nof conﬂuent endothelial cells in vitro\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.7**r== 0.1**\n(Bardin et al., 1996a, 2001), although\nCD146 does not appear to form part of\nthe tight junctions (Lampugnani and\nDejana, 1997) nor the adherens junc-\ntions (Bardin et al., 2001). In addition,\nthe cell surface expression of CD146\nwas shown to increase by twofold as\nhuman umbilical vein endothelial\ncells (HUVECs) grew from subconﬂu-\nency to complete conﬂuency (Bardin et\nal., 2001). Furthermore, overexpres-\nsion of CD146 in ﬁbroblasts decreased\nparacellular permeability (Bardin et\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nThe Supplementary Materials referred to in this article can be found at http://www.interscience.wiley.com/jpages/1058-8388/suppmat\n1Division of Nephrology, Center for Study of the Tumor Microenvironment, Beth Israel Deaconess Medical Center, and Harvard Medical\nSchool, Boston, Massachusetts\n2Division Hematology-Oncology, Center for Study of the Tumor Microenvironment, Beth Israel Deaconess Medical Center and Harvard\nMedical School, Boston, Massachusetts\n†Dr. Ramchandran’s present address is National Cancer Institute, Key West Center, Bldg 14, 9610, Medical Center Drive, Rockville, MD\n20850.\n*Correspondence to: Vikas P. Sukhatme, 330 Brookline Avenue, RW 563, Beth Israel Deaconess Medical Center, Boston, MA\n02215. E-mail: vsukhatm@bidmc.harvard.edu\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.4**\nDOI 10.1002/dvdy.20220\nPublished online 3 December 2004 in Wiley InterScience (www.interscience.wiley.com).\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFig. 1. Full-length mRNA and protein sequences of zebraﬁsh CD146. The 2.3-kb transcript harbors an open reading frame that encodes a\n617-amino-acid protein. The amino acids are numbered, starting from the translational start codon. The putative polyadenylation site is underscored\nwith a wavy line. The nucleotides deleted due to aberrant splicing in the presence of splice blocker 1 and splice blocker 2 are in bold and underlined,\nrespectively. The premature stop codon present in the aberrantly spliced mRNA induced by splice blocker 1 is in bold and italic.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nal., 2001), whereas treatment of con-\nﬂuent human microvascular endothe-\nlial cells (HMVECs) with monoclonal\nantibody P1H12 increased permeabil-\nity to albumin in vitro (Solovey et al.,\n2001).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.6**\nThe nature of CD146 adhesive prop-\nerty has been addressed in vitro\nmainly through cell-aggregation and\nsolid-phase binding assays. Results\nfrom these experiments suggest that\nCD146 confers cell– cell adhesion by\nmeans of interactions with a hetero-\nphilic ligand that has yet to be identi-\nﬁed (Johnson et al., 1997; Shih et al.,\n1997). This interaction is divalent-cat-\nion independent and can be inhibited\nby a recombinant full-length CD146\nprotein and a polyclonal antiserum\nraised against CD146 (Johnson et al.,\n1997; Shih et al., 1997).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nIn addition to its potential role in\ncell– cell adhesion, CD146 has also\nbeen shown to be a signaling molecule\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.4**r== 0.4**\nthat induces several intracellular mo-\nlecular events upon cross-linking by a\nmonoclonal antibody. For example,\ntreatment of HMVEC with the mono-\nclonal antibody P1H12 caused caveo-\nlin to redistribute from the cell periph-\nery to the cytoplasm and nuclear entry\nof the transcription factor NF-B (So-\nlovey et al., 2001). In addition, anti-\nbody-mediated cross-linking of CD146\nin vitro recruited Fyn to the cytoplas-\nmic domain of CD146, triggered a\nphospholipase C – dependent store-\noperated calcium mobilization and in-\nduced phosphorylation of cellular pro-\nteins such as Pyk2, FAK, CAS, and\npaxillin (Anfosso et al., 1998, 2001;\nSolovey et al., 2001). However, CD146\nhas not been shown to interact with\nother cell-surface adhesive molecules\nsuch as VE-cadherin, PECAM-1, or 1\nintegrin (Bardin et al., 2001).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nRecently, a fully humanized CD146\nmonoclonal antibody ABX-MA1 (Mills\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.7**r== 0.1**\net al., 2002) and a mouse monoclonal\nantibody AA98 (Yan et al., 2003) were\ndeveloped. When tested in mouse tu-\nmor models, both antibodies inhibited\ntumor growth and metastasis and re-\nduced microvascular density in tu-\nmors (Mills et al., 2002; Yan et al.,\n2003). Of interest, the antibody AA98\nwas shown to bind only to stimulated\nbut not serum-starved HUVECs in\nvitro (Yan et al., 2003). Most impor-\ntantly, this antibody was shown to\nrecognize an epitope in CD146 that\nwas only present in tumor endothe-\nin endothelial cells\nlium but not\npresent in normal tissues (Yan et al.,\n2003). AA98 monoclonal antibody was\nshown to inhibit proliferation and mi-\ngration of HUVECs in vitro and sig-\nniﬁcantly\nin\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003). Collectively,\nthese studies suggest\nthat CD146\nplays a signiﬁcant role in blood vessel\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nreduce angiogenesis\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nFig. 2. A: Protein sequence of zebraﬁsh CD146 aligned to the putative orthologs from chicken (accession no. Y08855; Alais et al., 2001), mouse\n(accession no. AAH26985), rat (accession no. BAB16048), and human (accession no. NP_006491; Lehmann et al., 1989). The alignment was performed\nat http://prodes.toulouse.inra.fr/multalin/multalin.html (Corpet, 1988). Highly conserved amino acid residues are highlighted. The leader sequence and\nthe transmembrane domain of human CD146 are underlined by a solid line and a wavy line, respectively. Amino acids of the zebraﬁsh protein are also\nnumbered. B: Phylogenetic analysis based on the CD146 protein sequences. The analysis was conducted with the software MEGA3 (Kumar et al.,\n2001) using the Neighbor-Joining method. The long isoform of DIM-1 from Caenorhabditis elegans (accession no. AAM23316) is a member of the\nimmunoglobulin superfamily (Rogalski et al., 2003) and was used as the outgroup in the analysis. Bootstrap support values based on 100 resamplings\nare presented at branch points.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.6**\ndevelopment as well as in solid tumor\ngrowth.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.6**\nThe ease of manipulation makes the\nzebraﬁsh system ideal to identify en-\ndothelial speciﬁc genes. Speciﬁcally,\nthe antisense morpholino-oligonucleo-\ntide technology (Heasman et al., 2000;\nNasevicius and Ekker, 2000) provides\na rapid way to suppress protein ex-\npression to address the functionality\nof a speciﬁc gene in a developmental\ncontext. One of our goals is to use this\nmodel to rapidly characterize endo-\nthelial-speciﬁc genes with unknown in\nvivo functions. We chose CD146 as our\nﬁrst target gene, because a CD146 ge-\nnetic knockout mouse is not available\nto date. Here, we report the use of the\nzebraﬁsh system to understand the\npotential role of CD146 in vascular\ndevelopment.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nRESULTS\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nCloning of the Zebraﬁsh\nCD146 Gene\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.6**\nA cDNA clone (clone number, 3512;\nGenBank accession no. BG985668) re-\nported in the zebraﬁsh cDNA data-\nbase\n(http://zf.nichd.nih.gov/pubzf/)\nshows a high level of sequence iden-\ntity to mouse and chicken CD146 at\nthe RNA and protein levels, respec-\ntively (Kudoh et al., 2001). Therefore,\nwe performed 5 and 3 rapid ampliﬁ-\ncation of cDNA ends (RACEs) based\non this sequence information to clone\nthe full-length zebraﬁsh CD146 (Fig.\n1).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.6**\nThe transcript is approximately 2.3\nkb long, harboring an open reading\nframe that encodes a 617-amino-acid\nprotein. The 5 untranslated region is\nrelatively short (approximately 73 nu-\ncleotides), similar to that of the tran-\nscript of human origin (26 nucleotides;\nSers et al., 1993). The 3 untranslated\nregion is approximately 350 nucleo-\ntides long and contains a canonical\npolyadenylation\nsequence\nAAUAAA just 5 to the poly-A tail\n(Fig. 1). When this mRNA sequence\nwas compared with the zebraﬁsh\ngenomic sequence, a 200-kb genomic\nfragment (ctg24502.1) was identiﬁed\nto contain the ﬁrst 13 predicted exons\nof CD146 (not shown). This genomic\nfragment covers nucleotides 1 to 1685\nof the mRNA. A similar search was\nperformed at the ENSEMBL Web site\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nsignal\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.4**r== 0.4**\n(http://www.ensembl.org/Danio_rerio/\nblastview). The search result indi-\ncates that CD146 is located on chro-\nmosome 19 of the zebraﬁsh genome.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.4**r== 0.4**\nThe zebraﬁsh CD146 protein shows\napproximately 30% overall sequence\nidentity to that of the human, avian,\nmouse, and rat orthologs (Fig. 2A). Of\ninterest, the transmembrane and the\ncytoplasmic domains are the most con-\nserved regions. When the protein se-\nquence was scanned for protein motifs\n(at http://scansite.mit.edu/; Yaffe et\nal., 2001), ﬁve immunoglobulin do-\nmains (Ig1, K35 to V123; Ig2, C168 to\nV232; Ig3, G273 to S324; Ig4, G359 to\nV408; Ig5, N445 to A507) were identi-\nﬁed in the extracellular region of the\nprotein, consistent with the character-\nistic Ig domain structure present in\nthe subgroup of immunoglobulin su-\nperfamily that CD146 belongs to\n(Johnson et al., 1996). The two puta-\ntive PKC phosphorylation sequences\n(S593GK and S614DK) in human (Alais\net al., 2001) are also conserved in the\nzebraﬁsh\n(S575GK and\nS596GK). The result of a phylogenetic\nanalysis is shown in Figure 2B.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nprotein\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.4**r== 0.4**\nTransient expression of zebraﬁsh\nCD146 tagged with a ﬂuorescent pro-\ntein at the C terminus in 293 cells\nshowed cell surface expression pat-\ntern similar to that of transiently ex-\npressed human CD146/YFP (data not\nshown). In addition, the ﬁrst 11 amino\nacids of zebraﬁsh CD146 directed the\nsecretion of YFP into the culture me-\ndium when fused to the N terminus of\nthe protein and transiently expressed\nin 293 cells, suggesting that this re-\ngion contains a leader-peptide se-\nquence and that Met1 is the authentic\ninitiation methionine\nnot\nshown). We also tested two commer-\ncially available antibodies against hu-\nman CD146 (a polyclonal antibody\nfrom ZYMED and a monoclonal anti-\nbody from CHEMICON) in Western\nblot analysis and determined that the\nantibodies did not cross-react with the\nzebraﬁsh protein overexpressed in\n293 cells (data not shown).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n(data\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nExpression of CD146 in\nZebraﬁsh Embryos Is Highly\nEndothelial Speciﬁc\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nNext, CD146 expression pattern in\nzebraﬁsh embryo was examined by\nwhole-mount in situ hybridization\n**BLOCK**fs== 9.0**b== 0.3**t== 0.1**l== 0.7**r== 0.1**\nby using a probe that was derived\nfrom the ﬁrst 0.6 kb of the mRNA\n(Fig. 3A). In a 24-hr-old embryo,\nmRNA of CD146 was detected in the\ndorsal aorta (Fig. 3B, red arrow),\ncaudal artery (Fig. 3B, red arrow-\nhead), caudal vein (Fig. 3B, blue ar-\nrowhead), and the sprouting inter-\nsomitic vessels\n(Fig. 3B, green\narrow). Expression in neuronal cells\nwas also detected (Fig. 3B, black ar-\nrow). This expression pattern is es-\nsentially identical to that reported in\nthe zebraﬁsh cDNA database (http://\nzf.nichd.nih.gov/pubzf/), where the\nwhole-mount in situ experiment was\nperformed using a 0.5-kb probe de-\nrived from the 3 end of the mRNA.\nOf interest, only a very low level of\nCD146 mRNA expression was de-\ntected in the posterior cardinal vein\nat this stage (Fig. 3B, blue arrow).\nThe expression speciﬁcity was best\nillustrated by a cross section of the\nembryo (Fig. 3D). Intense staining of\nthe message was detected in the\naorta (Fig. 3D, red arrow); only a\nvery low level of staining was de-\ntected in the posterior cardinal vein\n(Fig. 3D, blue arrow). In contrast,\nCD146 expression in the posterior car-\ndinal vein was detected at day 2 (Fig.\n3E, blue arrow) and day 3 (data not\nshown). This ﬁnding highlights the\ndynamics of the expression pattern of\nthis endothelial marker during devel-\nopment. mRNA staining of CD146 in\nthe head showed clear endothelial ex-\npression pattern (Fig. 3C). Certain\ncranial vessels (e.g., the middle cere-\nbral vein) were stained, while others\n(e.g., the dorsal ciliary and median\npalatocerebral veins) did not appear\nto express CD146. Some other vascu-\nlar structures stained included the\naortic arches and the heart.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.7**r== 0.1**\nProtein Expression\nKnockdown of CD146\nInhibits Proper Development\nof Intersomitic Vessels\nTo address the functional\nimpor-\ntance of CD146 in embryonic vascu-\nlar development, we performed pro-\ntein expression knockdown exper-\niments by using antisense MOs. The\nﬁrst MO (splice blocker 1) was de-\nsigned to target the seven nucleo-\ntides at the 3 end of exon 10 and the\nﬁrst 18 nucleotides of intron 10 (Fig.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nFig. 3. Whole-mount in situ hybridization of CD146 in zebraﬁsh. The digoxigenin-labeled antisense probe was derived from the ﬁrst 0.6 kb of the\nmRNA. A: Overall staining pattern of CD146 in a 24-hour postfertilization (hpf) embryo. B: A magniﬁcation at the trunk-tail region of a 24-hpf embryo.\nNotice the staining of the intersomitic vessels (green arrow), dorsal aorta (red arrow), caudal artery (red arrowhead), and the caudal vein (blue\narrowhead). Staining of the posterior cardinal vein was noticeably absent at this stage (blue arrow). Staining of neuronal cells was also seen at this\nstage (black arrow). C: Staining pattern in the head region of an embryo at 3 days postfertilization. D: Cross-section of an embryo at 24 hpf at the trunk\nregion showing the expression of CD146 in the artery and a very low level of expression in the vein. E: Expression pattern of CD146 mRNA in a\n2-day-old embryo showing endothelial speciﬁcity. Notice the expression of CD146 in the posterior cardinal vein. F: Cross-section of a 3-day-old\nembryo at the trunk region showing expression of CD146 in both the dorsal aorta and posterior cardinal vein.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\n4A). Microinjection of splice blocker\n1 into the embryos resulted in aber-\nrant splicing. When total RNA har-\nvested from control MO-injected em-\nbryos was used as the template for\nreverse\ntranscriptase-polymerase\nchain reaction (RT-PCR), a speciﬁc\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nband of 300 bp was ampliﬁed (Fig.\n5A, lane 1). The identity of this PCR\nproduct was conﬁrmed to be wild-\ntype CD146 by DNA sequencing\n(Fig. 6A). When the same analysis\nwas performed using total RNA har-\nvested from splice blocker 1–injected\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.7**r== 0.1**\nembryos, a PCR product with the\nsize 300 bp was obtained (Fig. 5A,\nlanes 3 and 5). DNA sequencing re-\nvealed that this product was derived\nfrom an aberrantly spliced CD146\nmRNA with nucleotides 1260 to 1282\ndeleted (Fig. 6B). Apparently, with\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.7**r== 0.1**\nA subpopulation of embryos injected\nwith the MO showed different de-\ngrees of curvature at the trunk/tail\nregion. An example is shown in\nFigure 7D. However, there did not\nappear to be a correlation between\nvascular defects and the body curva-\nture.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nReduced Protein Expression\nof CD146 Causes Disruption\nof Caudal Plexus\nReorganization\n**BLOCK**fs== 9.0**b== 0.3**t== 0.3**l== 0.7**r== 0.1**\nAdditional vascular defects were also\ndetected in CD146 morphants. For ex-\nample, the angiogenic remodeling pro-\ncess of the caudal plexus into func-\ntional vascular tubes was complete at\nday 3 in the control MO-injected ﬁsh\n(Movie 2A). Blood could be seen trav-\neling through the entire length of the\nembryo by means of the dorsal aorta,\nthe caudal artery, the caudal vein, and\nthe posterior cardinal vein. However,\nthis vascular reorganization of the\nplexus appeared to be defective in the\nsplice blocker 1–injected embryos\n(Supplementary Movie 2B). The cau-\ndal plexus failed to completely re-\nmodel and organize into functional\ntubes. There was a circulation shunt\nthat caused the circulation to by-pass\nmuch of the caudal artery/vein sys-\ntem. A stagnant pool of blood cells ac-\ncumulated at the tail. Although no cir-\nculation was detected, these trapped\ncells appeared to exhibit slight pulsa-\ntile motion, suggesting that the cau-\ndal plexus might not be developed and\norganized properly for circulation but\nconnected enough to allow hemody-\nnamic forces to reach the tail region.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.1**\nCD146 Appears To Be\nImportant in Formation of\nVessels With Proper Lumen\nSize\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.7**r== 0.1**\nTo investigate the integrity of the vas-\nculature, we performed microangiog-\nraphy to detect lumenized vessels.\nEmbryos at 3 dpf were injected with\nred ﬂuorescent microbeads with an\naverage diameter of 0.02 m. The re-\nsults are shown in Figure 8. In control\nMO-injected ﬁsh, vascular develop-\nment as seen by microangiography ap-\npeared to be normal. In contrast, the\noverall vessel integrity of the splice\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nFig. 4. Design of splice blocking antisense morpholino oligonucleotides. Blast analysis revealed\na 200-kb genomic fragment (ctg24502.1) in the zebraﬁsh genomic database containing the ﬁrst 13\npresumed exons of CD146. A,B: Exons 10 (A) and 2 (B) were chosen as the targets. The regions\ncomplementary to the morpholino oligonucleotides used are underlined. A: Splice blocker 1 targets\nthe 3 end of exon 10 (bold) and the predicted GT splice donor site of intron 10. B: Splice blocker\n2 targets the 3 end of exon 2 (bold) and the predicted GT splice donor site of intron 2.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\ngenerating\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\nthe predicted splice donor site of in-\ntron 10 blocked by the morpholino\noligonucleotide (see Fig. 4A), G1260\nof the mRNA was used as an alter-\nnate, cryptic splice donor site (Fig.\n6B),\naberrantly\nspliced mRNA harboring a prema-\nture stop codon 186 amino acids N\nterminal to the predicted transmem-\nbrane domain (Fig. 1, deleted amino\nacids underlined and the premature\nstop codon in bold and italic).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nan\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.6**\nTo conﬁrm the effect of splice\nblocker 1 on CD146 mRNA splicing,\nanother set of PCRs was performed in\ntriplicate by using a pair of isoform-\nspeciﬁc primers. This primer set was\ndesigned to detect\nthe full-length,\nwild-type CD146 cDNA speciﬁcally.\nAs shown in Figure 5B, the full-length\nCD146 was only detected in the con-\ntrol MO injected but not the splice\nblocker 1–injected embryos. As con-\ntrol, detection of another endothelial\nspeciﬁc gene ﬂi-1 was also shown.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nTo ascertain the role of CD146 in\nvessel development, blood circulation\nof embryos at 3 days postfertilization\n(dpf) was monitored (Fig. 7, boxed re-\ngions). Control injected ﬁsh showed\nnormal circulation. Blood cells were\nseen traveling through the length of\nthe embryo by means of the artery/\nvein system (Supplementary Movie\n1A, which is available online at www.\ninterscience.wiley.com/jpages/1058-\n8388/suppmat). Importantly, circula-\n**BLOCK**fs== 9.0**b== 0.1**t== 0.5**l== 0.4**r== 0.4**\ntion was also detected throughout the\nintersomitic vessels, indicating that\nthese vessels were functional at this\nstage of development. In contrast, cir-\nculation was abnormal in embryos in-\njected with splice blocker 1. Although\nblood cells were seen traveling in the\nartery and vein, little or no circulation\nwas detected in the intersomitic ves-\nsels (Supplementary Movie 1B). This\nobservation suggests that suppression\nof CD146 protein expression blocked\nthe development of the intersomitic\nvessels. This effect was tabulated, and\nthe results are shown in Table 1. We\ndecided to score the vessel develop-\nment of the embryos by the percent-\nage of functional intersomitic vessels\npresent at 3 dpf. This titration exper-\niment was done multiple times in\nsmall-scale pilot experiments, and a\nsimilar inhibition pattern was consis-\ntently observed. Next, we chose to in-\nject the embryos with either 1.2 pmol\ncontrol MO or 0.8 pmol splice blocker\n1 in three additional independent ex-\nperiments. The results are summa-\nrized in Figure 7C. Splice blocker 1\nclearly had a deleterious effect on\nthe development of intersomitic ves-\nsels. The deleted amino acids, there-\nfore, are critical to CD146 functions.\nBecause the transmembrane and cy-\ntoplasmic domains of CD146 were\ndeleted in the morphants, intracellu-\nlar signaling events triggered by\nCD146 were presumably abolished.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nFig. 5. Agarose gel analysis showing splice blocker 1 inhibited proper splicing of CD146 pre-mRNA. Fish embryos were injected with either control\nmorpholino oligonucleotide (MO) or the splice blocker 1. Total RNA was extracted at day 2 of development. Reverse transcriptase-polymerase chain\nreaction (RT-PCR) was performed to determine whether splicing was blocked. A: PCR using a set of primers that ampliﬁed both the full-length and\naberrantly spliced CD146 transcripts. The detection of a 300-bp DNA fragment indicated that proper splicing occurred when a control MO was injected\n(lane 1). A slightly shorter DNA fragment was ampliﬁed when the embryos were injected with splice blocker 1 (lanes 3 and 5). Mock treatments without\nRT (lanes 2, 4, and 6) were also included as controls. B: PCR using a set of primers that speciﬁcally ampliﬁed the full-length but not the truncated\nCD146 mRNA. Experiments were performed in triplicate. RT-PCR speciﬁcally ampliﬁed a 1.2-kb band using total RNA derived from control\nMO-injected embryos. No PCR product was detected when the embryos were injected with splice blocker 1, conﬁrming that aberrant splicing took\nplace. As controls, RT-PCR was performed to detect the expression of another endothelial speciﬁc gene called Fli 1. No-template PCR controls are\nindicated by a minus sign.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.6**\nblocker 1–injected ﬁsh appeared to be\ncompromised. For example, distinct\ncranial vessels were seen in the con-\ntrol-injected ﬁsh (Fig. 8A). In contrast,\nthe cranial vessels of\nthe splice\nblocker 1-injected embryo appeared to\nform less-distinct tubular structures\n(Fig. 8B). In particular, a group of ves-\nsels called the cranial division of the\ninternal carotid was less organized\ncompared with the control\n(arrow-\nheads in Fig. 8A,B). Second, the inter-\nsomitic vessels were narrower com-\npared with the control, even though\nthese vessels were detected by mi-\ncroangiography. A movie clip of the\ncirculation again revealed that the de-\nvelopment of intersomitic vessels was\n(Supplementary\nseverely\nMovie 3). However, these ISVs were\nlumenized as seen by ﬂuorescence mi-\ncroangiography (Fig. 8D). A similar\ndefect was observed in the caudal\nplexus region. When comparing the\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\naffected\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.4**r== 0.4**\nmicroangiogram and the blood circu-\nlation of the splice blocker–injected\nembryos, we observed that blood did\nnot ﬂow through part of the lumenized\ncaudal artery and vein vessels (corre-\nsponded to the region marked be-\ntween the arrowhead and the arrow in\nFig. 8D), resulting in a circulation\nshunt. This ﬁnding suggests that the\nlumen at the site where the circula-\ntion shunt occurred was too narrow\nfor blood cells to pass through. Our\nobservations suggest that CD146 may\nbe important in formation of vessels\nwith proper lumen size.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nElements Critical to CD146\nFunction Are Present at the\nN-Terminus of the Protein\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nTo validate our data with splice\nblocker 1, we designed a second an-\ntisense morpholino oligonucleotide\nto target the 3 end of exon 2 and the\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.7**r== 0.1**\npredicted GT splice donor site of in-\ntron 2 (Fig. 4). When this MO was\ndelivered into the embryos by micro-\ninjection,\nit also caused aberrant\nmRNA splicing. RT-PCR analyses\nwere performed with total RNA har-\nvested from control MO- and splice\nblocker 2–injected embryos. The con-\ntrol MO did not affect splicing at the\nexon2/intron2 junction of CD146\npre-mRNA (Fig. 6C). However, se-\nquencing data revealed that splice\nblocker 2 inhibited normal splicing\nbetween exons 2 and 3 (Fig. 6D).\nWith the predicted splice donor site\nof\nthe\nintron 2 blocked, G94 of\nmRNA was used as an alternate,\ncryptic splice donor site. This aber-\nrant splicing event yielded a mutant\nCD146 with an in-frame deletion of\n26 amino acids (V32 to T57) at the N\nterminus (Fig. 1, deleted region in\nbold).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nDeletion of amino acids V32 to T57\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nFig. 6. Sequence of CD146 cDNA generated from control morpholino oligonucleotide (MO)-injected and splice blocker–injected embryos. Nucleo-\ntides are numbered starting from the A in the initiation codon ATG as 1. A: Sequence of CD146 coding region from nucleotide 1246 to 1323 in control\nMO-injected embryos reveals that the control MO had no effects in RNA splicing in this region. Exon 10 (black line) was joined to exon 11 (red line)\nas predicted. B: Sequence analysis reveals that splice blocker 1 inhibited normal splicing between exons 10 and 11. With the predicted splice donor\nsite of intron 10 blocked by the morpholino oligonucleotide (see Fig. 4A), G1260 of the mRNA was used as an alternate, cryptic splice donor site. This\naberrantly spliced mRNA results in a premature stop codon within the open reading frame of CD146. C: Sequence of CD146 coding region from\nnucleotide 88 to 183 in control MO-injected embryos reveals that the control MO had no effects in RNA splicing in this region. Exon 2 (black line) was\njoined to exon 3 (red line) as predicted. D: Sequence analysis reveals that splice blocker 2 inhibited normal splicing between exons 2 and 3. With the\npredicted splice donor site of intron 2 blocked by the morpholino oligonucleotide (see Fig. 4B), G94 of the mRNA was used as an alternate, cryptic\nsplice donor site. This aberrant splicing event yielded a mutant CD146 with an in-frame deletion of 26 amino acids (V32 to T57) at the N terminus, which\ncontains a conserved cysteine residue (C42) presumably important for the folding of the ﬁrst immunoglobulin domain.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.7**r== 0.1**\nﬁsh, no circulation was seen in the\nintersomitic vessels (Supplementary\nMovie 4B and Table 2), a phenotype\nvery similar to that generated by\nsplice blocker 1–induced CD146 pro-\ntein expression knockdown. There-\nfore, these two MOs appeared to have\nphenocopied each other. Furthermore,\nthis result strongly suggests that the\nregion between V32 and T57 is likely\nto contain elements critical for CD146\nfunction.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nDISCUSSION\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nbetween\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nsimilarity\n**BLOCK**fs== 9.0**b== 0.2**t== 0.3**l== 0.7**r== 0.1**\nIn this report, we cloned and charac-\nterized the CD146 ortholog in ze-\nbraﬁsh. The overall amino acid se-\nquence\nthe\nzebraﬁsh and human proteins was ap-\nproximately 30%. Whole-mount\nin\nsitu hybridization results indicate\nthat the CD146 has a dynamic expres-\nsion pattern in the vascular tree dur-\ning development. At 1 dpf, CD146\ntranscript is detected in the dorsal\naorta, caudal artery, and the inter-\nsomitic vessels. In the venous system,\nhowever, CD146 is detected only in\nthe caudal vein but not in the poste-\nrior cardinal vein at this stage (Fig.\n3B). However, CD146 began to ex-\npress in the posterior cardinal vein at\n2 dpf (Fig. 3E). The functional impor-\ntance of the dynamic expression of\nCD146 in the venous system is yet to\nbe understood. CD146 is also ex-\npressed in the heart of developing ze-\nbraﬁsh embryos and may play a role\nin cardiac development. We occasion-\nally observed pericardial edema in the\nmorphants, but there did not seem to\nbe a correlation between this pheno-\ntype and the vascular defects de-\nscribed here. Future studies will in-\nclude the analysis of the relationship\nand\nbetween heart\nCD146 functions. In this report, we\nfocused on the vascular functions of\nCD146 primarily because of the ap-\nparent roles CD146 plays in tumor an-\ngiogenesis (Mills et al., 2002; Yan et\nal., 2003).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.8**r== 0.1**\ndevelopment\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.7**r== 0.1**\nWhen the protein expression of\nCD146 was knocked down, vascular\ndevelopment in the zebraﬁsh embryo\nwas severely affected. One most prom-\ninent defect was the lack of blood cir-\nculating in the intersomitic vessels by\n3 dpf (Supplementary Movies 1A and\nB). This lack of circulation was not\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.4**\nFig. 7. Effects of morpholino oligonucleotide (MO) in vascular development in zebraﬁsh. A,B:\nEmbryos were injected with 0.8 pmol of either the control MO (A) or splice blocker 1 (B) and\nexamined at 3 dpf. Movie clips of blood circulation (boxed regions) observed under light micros-\ncopy were also recorded (see Supplementary Movies 1A and B). C: Histogram showing the\ndistribution of embryos according to percent of functional intersomitic vessels present at 3 days\npostfertilization (dpf). Amounts of the control MO or splice blocker 1 injected were 1.2 or 0.8 pmol,\nrespectively. Results were the averages of three different experiments. The percentages do not add\nup to 100% due to rounding errors. D: A 3-day-old embryo injected with splice blocker 1 showing\na curvature in the trunk/tail region. Scale bars  0.1 mm in A,B,D.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nof CD146 also affected the develop-\nment of intersomitic vessels in ze-\nbraﬁsh embryos. At day three, circu-\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nlation was normal in embryos injected\nwith the control MO (Supplementary\nMovie 4A). In splice blocker 2–injected\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.4**\nTABLE 1. Effects of Splice Blocker 1 Injection on Intersomitic Vessel\nDevelopmenta\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nPercent of\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nfunctional\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nISV\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n80–100\n50–80\n20–50\n0–20\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nControl MO\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n1.2 pmol\n(n  30)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nPercent of embryos\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nSplice blocker 1\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n1.2 pmol\n(n  25)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n0.8 pmol\n(n  34)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n0.4 pmol\n(n  33)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\naMO, morpholine oligonucleotide; ISV, intersomitic vessels.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.4**l== 0.1**r== 0.6**\ndue to the absence of intersomitic ves-\nsels per se, because these vessel struc-\ntures were clearly detected by ﬂuores-\ncence microangiography,\nalthough\nmany of them appeared to have un-\neven thickness and stenosis was evi-\ndent even when observed under a ﬂu-\n(Fig. 8B,D).\norescence microscope\nAlthough the lumen size of the inter-\nsomitic vessels in zebraﬁsh has not\nbeen reported in literature, the diam-\neters of artery and vein in zebraﬁsh\nembryos have been measured to be\n9.0 m,\n12.8\nrespectively\n(Fritsche et al., 2000). The use of ﬂu-\norescent microbeads with an average\ndiameter of 0.02 m allowed us to de-\ntect vessels with a diameter 400\ntimes smaller than that of the vein.\nBased on our results, we speculate\nthat, in the absence of CD146, the\nintersomitic vessels cannot develop\ninto functional tubes with a lumen\nopening large enough for blood cells to\npass through.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nand\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.6**\nThe development of the intersomitic\nvessels is a highly organized and cho-\nreographed process. It was reported\nrecently that formation of the inter-\nsomitic vessels involves three differ-\nent endothelial cell types sprouting\nfrom the dorsal aorta (Childs et al.,\n2002). The ﬁrst two form a T-like\nstructure at the dorsal side of the dor-\nsal aorta and the ventral side of the\ndorsal longitudinal anastomotic ves-\nsel. The third one connects these two\nT structures. Because the intersomitic\nvessels were detected in the mor-\nphants with CD146 protein expres-\nsion suppressed, this ﬁnding suggests\nthat CD146 may play a role in the\nstep(s) after the basic architecture of\nthe intersomitic vessels has been laid\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nout by these three types of endothelial\ncells.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.4**l== 0.4**r== 0.4**\nData from our morpholino experi-\nments indicate that CD146 may play\nan important role in the remodeling of\nthe caudal vein plexus into functional\nvascular tubes as well. In the CD146\nmorphants, circulation ﬂowed from\nthe dorsal aorta into the caudal artery\nand returned prematurely into cardi-\nnal vein, creating a circulation shunt\nin the caudal artery/vein system (Sup-\nplementary Movie 2B). This circula-\ntion shunt appears to be distinct from\nthat observed in the zebraﬁsh mutant\nmindbomb (mib) defective in Notch\nsignaling (Lawson et al., 2001). In the\nmib mutant, the circulation shunt oc-\ncurred in the trunk region where cir-\nculation ﬂowed directly from the dor-\nsal aorta into the posterior cardinal\nvein without entering the caudal ar-\ntery/vein system. This circulation by-\npass was speculated to arise from dis-\norganization of the boundary between\nthe dorsal aorta and the posterior car-\ndinal vein based on histological data\n(Lawson et al., 2001). In the CD146\nmorphants described here, blood cir-\nculation entered the caudal artery but\nreturned into the venous system pre-\nmaturely into the caudal vein. How-\never, ﬂuorescence microangiography\nrevealed that the caudal plexus was\nlumenized (Fig. 8D, see region be-\ntween the arrow and the arrowhead).\nTherefore, the caudal shunt present in\nthe CD146 morphants may be a result\nof restricted lumen diameter, a struc-\ntural defect similar to that observed in\nthe intersomitic vessels.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nOur observation that CD146 is im-\nportant in vascular development is in\nagreement with several studies re-\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.7**r== 0.1**\nported in the literature. For example,\nit was shown the CD146 expression\nwas up-regulated in human microvas-\ncular endothelial cells that had under-\ngone tubular morphological changes\non matrigel (Glienke et al., 2000). In\nadditional, the ability of endothelial\ncells to form tubes on matrigel can be\nblocked by a CD146 neutralizing anti-\nbody (Mills et al., 2002). Functional\nsuppression of CD146 by an antibody\ncan also inhibit endothelial cell migra-\ntion in vitro and neovascularization in\nchicken chorioallantoic membrane as-\nsays (Yan et al., 2003).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.3**l== 0.7**r== 0.1**\nWe have also identiﬁed V32 to T57\nof CD146 to be an important sequence\nelement in its protein function. We\nused an antisense morpholino oligo-\nnucleotide targeting the predicted\nexon 2/intron 2 junction of zebraﬁsh\nCD146 pre-mRNA to create a mutant\nprotein with those 26 amino acids de-\nleted in an in-frame manner. This\ntruncated protein appeared to be non-\nfunctional. This region may contain a\nrecognition epitope for CD146 func-\ntions. Alternatively, the deletion may\nhave affected proper protein folding,\nas the region deleted contains a con-\nserved cysteine residue in the ﬁrst im-\nmunoglobulin domain.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.7**r== 0.1**\nVascular development requires ex-\ntensive dynamic changes of endothe-\nlial morphology. These dramatic mor-\nphological\nchanges were recorded\nrecently by time-lapse microscopy in a\ndeveloping zebraﬁsh embryo in real-\ntime (Lawson and Weinstein, 2002).\nEndothelial cells were seen to extend\nand retract long ﬁlopodial processes\nduring path ﬁnding. Communication\nappears to occur between these acti-\nvated endothelial cells through ﬁlopo-\ndial contacts, resulting in proper vas-\ncular remodeling and regression to\nensure the development of functional\nlumens\n(Lawson and Weinstein,\n2002). The exact molecular basis of\nthese cellular behaviors is certainly\ncomplex, but highly orchestrated and\ndynamic changes of the cytoskeleton\norganization are probably involved\n(Davis et al., 2002). For example,\nCdc42, Rac1, and integrins are some\nof the signaling molecules intimately\nlinked to cytoskeleton functions that\nhave been shown to be critical in en-\ndothelial tube and lumen formation in\nvitro (Davis and Camarillo, 1996; Bay-\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.7**r== 0.1**\neral integrins that are known to be\nexpressed in endothelial cells (Alais et\nal., 2001; Hodivala-Dilke et al., 2003).\nAll these cellular events appear to be\nrelated to the cytoskeleton. Perhaps,\nCD146 mediates the formation of\nproper vessels through regulating the\ndynamic changes in the organization\nof the cytoskeleton during develop-\nment. Further studies that are de-\nsigned to dissect the intracellular sig-\nnaling events important in CD146\nfunctions are currently under way.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\nEXPERIMENTAL\nPROCEDURES\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nZebraﬁsh Care\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\nZebraﬁsh (Danio rerio) was raised and\nmaintained as described (Westerﬁeld,\n2000).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n5 and 3 RACES\n**BLOCK**fs== 9.0**b== 0.1**t== 0.5**l== 0.7**r== 0.1**\nTotal RNA was collected from ze-\nbraﬁsh embryos at 48 hours postfertil-\nization (hpf) by using either the\nRNAaqueous-4PCR kit from Ambion\n(Austin, TX) or the RNeasy Mini kit\nfrom Qiagen (Valencia, CA). The 5\nand 3 RACEs were carried out by us-\ning the GeneRacer Kit from Invitro-\ngen (Carlsbad, CA) according to the\nmanufacturer’s protocol. The kit was\nchosen because the transcriptional\nstart site of the mRNA can be ob-\ntained directly. Brieﬂy, 5 RACE was\nperformed by using 5-GGCACAGG-\nGAGTGTCTTCTTGCCTTATA-3\nas\nthe gene-speciﬁc primer for ﬁrst-\nstrand cDNA synthesis. The cDNA\nwas ampliﬁed by nested PCR using\nthe GC-Advantage DNA polymerase\nfrom Clontech (Palo Alto, CA). The\nprimers for the ﬁrst round of PCR\nwere the GeneRacer 5 Primer (pro-\nvided in the GeneRacer Kit) and the\ngene-speciﬁc primer. For the second\nround of ampliﬁcation, the GeneRacer\n5 Nested Primer (provided in the\nGeneRacer Kit) and a primer with the\nsequence 5-GCAGCTTTCCCTTCTT-\nGTAGAGGTAAT-3 were used. The\nPCR products were cloned into the\nTOPO cloning kit from Invitrogen.\nProtein sequence alignment was per-\nhttp://prodes.toulouse.\nformed\ninra.fr/multalin/multalin.html\n(Cor-\npet, 1988). The mRNA sequence has\nbeen deposited in GenBank (accession\nno. AY497061).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\nat\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.4**\nFig. 8. Visualization of vasculature by microangiography. A–D: Embryos were injected with 0.8\npmol of either the control morpholino oligonucleotide (MO; A,C) or splice blocker 1 (B,D) and\nexamined at 3 days postfertilization. The vessels in the head region of the splice blocker 1–injected\nembryo were less developed when compared with the control MO-injected embryo (see text). The\narrowheads in A and B indicate a group of vessels called the cranial division of the internal carotid.\nImportantly, intersomitic vessels were detected in the angiogram of the morphants, while blood did\nnot circulate through most of these vessels (see Supplementary Movie 3). The white arrow in D\nindicates the point at which blood exits the artery and enters the vein. No circulation was observed\nbetween the arrowhead and the arrow, even though this segment was lumenized. The boxed\nregions in A and B were magniﬁed and are shown in C and D, respectively. Orientation of the\nembryo in C and D: left/caudal and right/rostral. Scale bars  0.1 mm in A–D.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nless et al., 2000; Bayless and Davis,\n2002; Davis and Bayless, 2003).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nAlthough the molecular mechanism\nby which CD146 affects proper lumen\nformation observed in this report is at\nthis point unclear, it is tempting to\nspeculate that CD146 may achieve\nsuch function by regulating the cy-\ntoskeleton. It is known that antibody-\nmediated cross-linking of CD146 in\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nvitro initiates a cascade of intracellu-\nlar signaling events that\nincludes\nphosphorylation of focal adhesion ki-\nnase, paxillin, and p130CAS (Anfosso\net al., 1998, 2001). Additionally,\nCD146 has been shown to interact\nwith the actin cytoskeleton in HUVEC\n(Bardin et al., 2001). Moreover, the\nCD146 ortholog in chicken has been\nshown to modulate expression of sev-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.4**\nTABLE 2. Effects of Splice Blocker 2 Injection on Intersomitic Vessel\nDevelopmenta\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nPercent of\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nfunctional\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nISV\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n80–100\n50–80\n20–50\n0–20\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nControl MO\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n1.2 pmol\n(n  53)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nPercent of embryos\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nSplice blocker 2\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n1.2 pmol\n(n  50)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n0.8 pmol\n(n  31)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n0.4 pmol\n(n  33)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\naMO, morpholine oligonucleotide; ISV, intersomitic vessels.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.6**\nIn Situ Hybridization\nNucleotides spanning the 5 untrans-\nlated region and the ﬁrst 592 bases of\nthe open reading frame of zebraﬁsh\nCD146 were ampliﬁed and cloned into\na vector derived from pCR-BluntII-\nTOPO (Invitrogen). To\ngenerate\ndigoxigenin (DIG)-labeled antisense\nprobes, the DNA plasmid was linear-\nized with EcoRI, followed by tran-\nscription using SP6 polymerase and\nthe 10 DIG RNA labeling mix from\nRoche (Indianapolis, IN). In situ hy-\nbridization was then performed ac-\ncording to a published procedure\n(Broadbent and Read, 1999; Read,\n2000).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.6**\nMicroinjection and\nFluorescence\nMicroangiography\nTwo antisense (splice blocker 1. 5-\nCAGTTGTAGTGACCCACCTCGCA-\nCG-3; splice blocker 2. 5-GTCTTTT-\nTCTCACCGTAAACCAGAT-3) and a\ncontrol (5-GACTCTATATTGCCTAC-\nCTTTACAG-3) morpholino oligo-\nnucleotides were purchased from\nGene-Tools (Philomath, OR). The\nmorpholino oligonucleotides were in-\njected into the yolk sac of zebraﬁsh\nembryos between one- and four-cell\nstages. Embryos were raised in egg\nwater in the presence of 0.003% (w/v)\n1-phenyl-2-thiourea (Sigma, St. Louis,\nMO) to prevent pigmentation (Wester-\nﬁeld, 2000). To verify successful block-\nage of CD146 pre-mRNA splicing by\nthe morpholino oligonucleotides, total\nRNA was collected from the zebraﬁsh\nembryos at 2 dpf by using the RNeasy\nMini kit from Qiagen. Reverse tran-\nscription was carried out by using ei-\n**BLOCK**fs== 9.0**b== 0.3**t== 0.4**l== 0.4**r== 0.4**\nther Thermoscript or Superscript III\nfrom Invitrogen with the gene-speciﬁc\nprimer (5-GGCACAGGGAGTGTCT-\nTCTTGCCTTATA-3). PCR was then\nperformed by using GC-Advantage\nDNA polymerase (Clontech) and the\nfollowing ampliﬁcation primers: 5-\nGATATCACCTTGACTTGCAACG-\nCATTGTCT-3 and 5-GATTGGG-\nTAGCCTTGTGCATAGCAGGTCAG-3\nfor\nsplice blocker 1 injection;\n5-ATGACCTACACCGCACTGCTGC-\nTCGCGGGA-3 and 5-AGGATC-\nGAGAGGGAAGCTGCCCTTGTAC-\nCA-3 for splice blocker 2 injection. A\npair of isoform speciﬁc primers that am-\npliﬁed only the wild-type CD146 cDNA\n(nucleotides 126 to 1287) has the follow-\ning sequences: 5-TACTTACACTCT-\nGACGGAAGAAGCCAAGAT-3 and\n5-CTTTCCTCGCACGTTTACTTGCAC-\nTGA-3. Primers used to ampliﬁed Fli-1\nwere 5-GAAAATGGACGGAACTAT-\nTAAGGAGGCGCT-3 and 5-TCTTC-\nGAGTGCAGTTCAAGTTTTGGCATT-\nTAAGGGT-3.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.4**r== 0.4**\nBlood circulation in anesthetized\nﬁsh embryos was recorded as time-\nlapse images using an inverted phase\ncontrast microscope (Nikon Corpora-\ntion, Tokyo, Japan), a Spot RT cam-\nera, and the Advanced SPOT software\n(Diagnostic Instruments, Inc., Ster-\nling Heights, MI). The images were\nconverted into QuickTime movies us-\ning a program called ImageJ.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nFluorescence\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nmicroangiography\nwas performed by using the red ﬂuo-\ncarboxylate-modiﬁed Fluo-\nrescent\nSpheres (0.02 m)\nfrom Molecular\nProbes (Eugene, OR) as described\n(Weinstein et al., 1995). Fluorescence\nmicroangiograms were recorded with\nthe Nikon ﬂuorescence microscope us-\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.7**r== 0.1**\ning a ﬁlter set (41035:ex. 546; em. 605)\nfrom Chroma (Rockingham, VT).\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.7**r== 0.1**\nACKNOWLEGMENTS\nThis study was supported by seed\nfunds from Beth Israel Deaconess\nMedical Center (Boston, MA).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nAlais S, Allioli N, Pujades C, Duband JL,\nVainio O, Imhof BA, Dunon D. 2001.\nHEMCAM/CD146 downregulates\ncell\nsurface expression of beta1 integrins.\nJ Cell Sci 114:1847–1859.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.7**r== 0.1**\nAnfosso F, Bardin N, Frances V, Vivier E,\nCamoin-Jau L, Sampol J, Dignat-George\nF. 1998. Activation of human endothelial\ncells via S-endo-1 antigen (CD146) stim-\nulates the tyrosine phosphorylation of fo-\ncal adhesion kinase p125(FAK). J Biol\nChem 273:26852–26856.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.4**l== 0.7**r== 0.1**\nAnfosso F, Bardin N, Vivier E, Sabatier F,\nSampol J, Dignat-George F. 2001. Out-\nside-in signaling pathway linked to\nCD146 engagement in human endothe-\nlial cells. J Biol Chem 276:1564 –1569.\nBardin N, Frances V, Lesaule G,\nHorschowski N, George F, Sampol J.\n1996a. Identiﬁcation of the S-Endo 1 en-\ndothelial-associated antigen. Biochem\nBiophys Res Commun 218:210 –216.\nBardin N, George F, Mutin M, Brisson C,\nHorschowski N, Frances V, Lesaule G,\nSampol J. 1996b. S-Endo 1, a pan-endo-\nthelial monoclonal antibody recognizing\na novel human endothelial antigen. Tis-\nsue Antigens 48:531–539.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.7**r== 0.1**\nBardin N, Anfosso F, Masse JM, Cramer E,\nSabatier F, Le Bivic A, Sampol J, Dignat-\nGeorge F. 2001. Identiﬁcation of CD146\nas a component of the endothelial junc-\ntion involved in the control of cell-cell\ncohesion. Blood 98:3677–3684.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.7**r== 0.1**\nBayless KJ, Davis GE. 2002. The Cdc42\nand Rac1 GTPases are required for cap-\nillary lumen formation in three-dimen-\nsional extracellular matrices. J Cell Sci\n115:1123–1136.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.7**r== 0.1**\nBayless KJ, Salazar R, Davis GE. 2000.\nRGD-dependent vacuolation and lumen\nformation observed during endothelial\ncell morphogenesis in three-dimensional\nﬁbrin matrices\nthe alpha-\n(v)beta(3) and alpha(5)beta(1) integrins.\nAm J Pathol 156:1673–1683.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\ninvolves\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nBroadbent J, Read EM. 1999. Wholemount\nin situ hybridization of Xenopus and ze-\nbraﬁsh embryos. Methods Mol Biol 127:\n57– 67.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.1**\nChilds S, Chen JN, Garrity DM, Fishman\nMC. 2002. Patterning of angiogenesis in\nthe zebraﬁsh embryo. Development 129:\n973–982.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nCorpet F. 1988. Multiple sequence align-\nment with hierarchical clustering. Nu-\ncleic Acids Res 16:10881–10890.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nDavis GE, Bayless KJ. 2003. An integrin\nand Rho GTPase-dependent pinocytic\nvacuole mechanism controls capillary lu-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.6**\nmen formation in collagen and ﬁbrin ma-\ntrices. Microcirculation 10:27–44.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nDavis GE, Camarillo CW. 1996. An alpha 2\nbeta 1 integrin-dependent pinocytic\nmechanism involving intracellular vacu-\nole formation and coalescence regulates\ncapillary lumen and tube formation in\nthree-dimensional collagen matrix. Exp\nCell Res 224:39 –51.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\nDavis GE, Bayless KJ, Mavila A. 2002. Mo-\nlecular basis of endothelial cell morpho-\ngenesis in three-dimensional extracellu-\nlar matrices. Anat Rec 268:252–275.\nFritsche R, Schwerte T, Pelster B. 2000.\nNitric oxide and vascular reactivity in\ndeveloping zebraﬁsh, Danio rerio. Am J\nPhysiol Regul Integr Comp Physiol 279:\nR2200 –R2207.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.6**\nGlienke J, Schmitt AO, Pilarsky C, Hin-\nzmann B, Weiss B, Rosenthal A, Thier-\nauch KH. 2000. Differential gene expres-\nsion by endothelial cells in distinct\nangiogenic states. Eur J Biochem 267:\n2820 –2830.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nHeasman J, Kofron M, Wylie C. 2000.\nBeta-catenin signaling activity dissected\nin the early Xenopus embryo: a novel\nantisense approach. Dev Biol 222:124 –\n134.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nHodivala-Dilke KM, Reynolds AR, Reyn-\nolds LE. 2003. Integrins in angiogenesis:\nmultitalented molecules in a balancing\nact. Cell Tissue Res 314:131–144.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\nHolzmann B, Brocker EB, Lehmann JM,\nRuiter DJ, Sorg C, Riethmuller G, John-\nson JP. 1987. Tumor progression in hu-\nman malignant melanoma: ﬁve stages\ndeﬁned by their antigenic phenotypes.\nInt J Cancer 39:466 –471.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nJohnson JP, Rummel MM, Rothbacher U,\nSers C. 1996. MUC18: a cell adhesion\nmolecule with a potential role in tumor\ngrowth and tumor cell dissemination.\nCurr Top Microbiol Immunol 213(Pt 1):\n95–105.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nJohnson JP, Bar-Eli M, Jansen B, Markhof\nE. 1997. Melanoma progression-associ-\nated glycoprotein MUC18/MCAM medi-\nates homotypic cell adhesion through in-\nteraction with a heterophilic ligand. Int\nJ Cancer 73:769 –774.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.4**r== 0.4**\nKudoh T, Tsang M, Hukriede NA, Chen X,\nDedekian M, Clarke CJ, Kiang A,\nSchultz S, Epstein JA, Toyama R, Dawid\nIB. 2001. A gene expression screen in\nzebraﬁsh embryogenesis. Genome Res 11:\n1979 –1987.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nKumar S, Tamura K, Jakobsen IB, Nei M.\n2001. MEGA2: molecular evolutionary\ngenetics analysis software. Bioinformat-\nics 17:1244 –1245.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\nLampugnani MG, Dejana E. 1997. Interen-\ndothelial junctions: structure, signalling\nand functional roles. Curr Opin Cell Biol\n9:674 –682.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nLawson ND, Weinstein BM. 2002. In vivo\nimaging of embryonic vascular develop-\nment using transgenic zebraﬁsh. Dev\nBiol 248:307–318.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nLawson ND, Scheer N, Pham VN, Kim CH,\nChitnis AB, Campos-Ortega JA, Wein-\nstein BM. 2001. Notch signaling is re-\nquired for arterial-venous differentiation\nduring embryonic vascular development.\nDevelopment 128:3675–3683.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\nLehmann JM, Holzmann B, Breitbart EW,\nSchmiegelow P, Riethmuller G, Johnson\nJP. 1987. Discrimination between be-\nnign and malignant cells of melanocytic\nlineage by two novel antigens, a glyco-\nprotein with a molecular weight of\n113,000 and a protein with a molecular\nweight of 76,000. Cancer Res 47:841–\n845.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\nLehmann JM, Riethmuller G, Johnson JP.\n1989. MUC18, a marker of tumor pro-\ngression in human melanoma, shows se-\nquence similarity to the neural cell ad-\nhesion molecules of the immunoglobulin\nsuperfamily. Proc Natl Acad Sci U S A\n86:9891–9895.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nMills L, Tellez C, Huang S, Baker C, Mc-\nCarty M, Green L, Gudas JM, Feng X,\nBar-Eli M. 2002. Fully human antibodies\nto MCAM/MUC18 inhibit tumor growth\nand metastasis of human melanoma.\nCancer Res 62:5106 –5114.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\nNasevicius A, Ekker SC. 2000. Effective\ntargeted gene ‘knockdown’ in zebraﬁsh.\nNat Genet 26:216 –220.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nRead EM. 2000. Wholemount in situ hy-\nbridization of xenopus and zebraﬁsh em-\nbryos. In: Darby IA, editor. In situ hy-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.7**r== 0.2**\nbridization\nHumana Press. p 343.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.1**\nprotocols. Totowa, NJ:\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.1**\nRogalski TM, Gilbert MM, Devenport D,\nNorman KR, Moerman DG. 2003.\nDIM-1, a novel immunoglobulin super-\nfamily protein in Caenorhabditis el-\negans, is necessary for maintaining body\nwall muscle integrity. Genetics 163:905–\n915.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.7**r== 0.1**\nSers C, Kirsch K, Rothbacher U, Riethmul-\nler G, Johnson JP. 1993. Genomic orga-\nnization of the melanoma-associated gly-\ncoprotein MUC18: implications for the\nevolution of\nthe immunoglobulin do-\nmains. Proc Natl Acad Sci U S A 90:8514 –\n8518.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nShih IM. 1999. The role of CD146 (Mel-\nCAM) in biology and pathology. J Pathol\n189:4 –11.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.7**r== 0.1**\nShih LM, Hsu MY, Palazzo JP, Herlyn M.\n1997. The cell-cell adhesion receptor\nMel-CAM acts as a tumor suppressor in\nbreast carcinoma. Am J Pathol 151:745–\n751.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\nSolovey AN, Gui L, Chang L, Enenstein J,\nBrowne PV, Hebbel RP. 2001. Identiﬁca-\ntion and functional assessment of endo-\nthelial P1H12. J Lab Clin Med 138:322–\n331.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.1**\nWeinstein BM, Stemple DL, Driever W,\nFishman MC. 1995. Gridlock, a localized\nheritable vascular patterning defect in\nthe zebraﬁsh. Nat Med 1:1143–1147.\nWesterﬁeld M. 2000. The zebraﬁsh book: a\nguide for the laboratory use of zebraﬁsh\n(Danio rerio). Eugene, OR: University of\nOregon.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.7**r== 0.1**\nWilliams AF, Barclay AN. 1988. The im-\nmunoglobulin superfamily— domains for\ncell surface recognition. Annu Rev Im-\nmunol 6:381–405.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.1**\nYaffe MB, Leparc GG, Lai J, Obata T, Vo-\nlinia S, Cantley LC. 2001. A motif-based\nproﬁle scanning approach for genome-\nwide prediction of signaling pathways.\nNat Biotechnol 19:348 –353.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.7**r== 0.1**\nYan X, Lin Y, Yang D, Shen Y, Yuan M,\nZhang Z, Li P, Xia H, Li L, Luo D, Liu Q,\nMann K, Bader BL. 2003. A novel anti-\nCD146 monoclonal antibody, AA98, in-\nhibits angiogenesis and tumor growth.\nBlood 102:184 –191.",
         "extracted",
         "",
         "Critical roles of CD146 in zebrafish vascular development",
         "None"
        ],
        [
         "4",
         "00043e28cffe744909bf69cde70e6dac0bce0a10",
         "Cardiac troponin I is a phosphorylation target for endothelin-activated protein kinase C. Earlier work in cardiac myocytes expressing nonphosphorylatable slow skeletal troponin I provided evidence that protein kinase C-mediated cardiac troponin I phosphorylation accelerates relaxation. However, replacement with the slow skeletal isoform also alters the myofilament pH response and the Ca2+ transient, which could influence endothelin-mediated relaxation. Here, differences in the Ca2+ transient could not explain the divergent relaxation response to endothelin in myocytes expressing cardiac versus slow skeletal troponin I nor could activation of Na+/H+ exchange. Three separate clusters within cardiac troponin I are phosphorylated by protein kinase C, and we set out to determine the contribution of the Thr144 and Ser23/Ser24 clusters to the endothelin-mediated contractile response. Myocyte replacement with a cardiac troponin I containing a Thr144 substituted with the Pro residue found in slow skeletal troponin I resulted in prolonged relaxation in response to acute endothelin compared with control myocytes. Ser23/Ser24 also is a target for protein kinase C phosphorylation of purified cardiac troponin I, and although this cluster was not acutely phosphorylated in intact myocytes, significant phosphorylation developed within 1 h after adding endothelin. Replacement of Ser23/Ser24 with Ala indicated that this cluster contributes significantly to relaxation during more prolonged endothelin stimulation. Overall, results with these mutants provide evidence that Thr144 plays an important role in the acute acceleration of relaxation, whereas Ser23/Ser24 contributes to relaxation during more prolonged activation of protein kinase C by endothelin.",
         "M. Westfall,A. M. Lee,D. Robinson",
         "http://www.jbc.org/article/S0021925820589394/pdf",
         "\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nTHE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 280, NO. 50, pp. 41324 –41331, December 16, 2005\n© 2005 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.\n**BLOCK**fs== 19.5**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nDifferential Contribution of Troponin I Phosphorylation Sites\nto the Endothelin-modulated Contractile Response*\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nReceived for publication, June 2, 2005, and in revised form, October 3, 2005 Published, JBC Papers in Press, October 18, 2005, DOI 10.1074/jbc.M506043200\nMargaret V. Westfall1, Adonia M. Lee, and Dustin A. Robinson\nFrom the Department of Surgery, University of Michigan, Ann Arbor, Michigan 48109\n**BLOCK**fs== 8.8**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\nCardiac troponin I is a phosphorylation target for endothelin-\nactivated protein kinase C. Earlier work in cardiac myocytes\nexpressing nonphosphorylatable slow skeletal troponin I provided\nevidence that protein kinase C-mediated cardiac troponin I phos-\nphorylation accelerates relaxation. However, replacement with the\nslow skeletal isoform also alters the myofilament pH response and\nthe Ca2 transient, which could influence endothelin-mediated\nrelaxation. Here, differences in the Ca2 transient could not explain\nthe divergent relaxation response to endothelin in myocytes\nexpressing cardiac versus slow skeletal troponin I nor could activa-\ntion of Na/H exchange. Three separate clusters within cardiac\ntroponin I are phosphorylated by protein kinase C, and we set out to\ndetermine the contribution of the Thr144 and Ser23/Ser24 clusters to\nthe endothelin-mediated contractile response. Myocyte replace-\nment with a cardiac troponin I containing a Thr144 substituted with\nthe Pro residue found in slow skeletal troponin I resulted in pro-\nlonged relaxation in response to acute endothelin compared with\ncontrol myocytes. Ser23/Ser24 also is a target for protein kinase C\nphosphorylation of purified cardiac troponin I, and although this\ncluster was not acutely phosphorylated in intact myocytes, signifi-\ncant phosphorylation developed within 1 h after adding endothelin.\nReplacement of Ser23/Ser24 with Ala indicated that this cluster con-\ntributes significantly to relaxation during more prolonged endothe-\nlin stimulation. Overall, results with these mutants provide evi-\ndence that Thr144 plays an important role in the acute acceleration\nof relaxation, whereas Ser23/Ser24 contributes to relaxation during\nmore prolonged activation of protein kinase C by endothelin.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nProtein kinase C (PKC)2 activation is an important pathway involved\nin modulating cardiac contractile function (1–5). A number of end tar-\nget proteins involved in excitation-contraction coupling are phospho-\nrylated in response to PKC activation, including the L-type Ca2 chan-\nnel (6, 7) and myofilament proteins such as cardiac troponin I (cTnI)\n(8 –11), cardiac troponin T (10, 12), and myosin light chain 2 (MLC2) (8,\n13, 14). The contribution of each end target to the contractile response\nto PKC is not well understood. Our laboratory is interested in under-\nstanding the role of cTnI in the PKC-mediated contractile function\nresponse. Increased PKC expression is observed in failing human hearts\n(15, 16), and the relationship between PKC and cTnI phosphorylation\nmay exert an important influence on cardiac function under physiolog-\nical and pathophysiological conditions (17).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n* The costs of publication of this article were defrayed in part by the payment of page\ncharges. This article must therefore be hereby marked “advertisement” in accordance\nwith 18 U.S.C. Section 1734 solely to indicate this fact.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n1 To whom correspondence should be addressed: Dept. of Surgery, 1150 W. Medical\nCenter Dr., B562 MSRB II 0686, Ann Arbor, MI 48109. Tel.: 734-615-8911; Fax: 734-763-\n0323; E-mail: wfall@umich.edu.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n2 The abbreviations used are: PKC, protein kinase C; PKA, protein kinase A; TnI, troponin\nI; cTnI, cardiac TnI; ssTnI, slow skeletal TnI; SL, sarcomere length; TTP, time to peak\nshortening; ANOVA, analysis of variance; DOB, dobutamine; ET, endothelin; EIPA,\n5-N(ethyl-N-iospropyl)amiloride; bis-1, bis-indolylmaleimide-1.\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.1**r== 0.6**\nThis is an Open Access article under the CC BY license.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nEarlier work demonstrated that cTnI phosphorylation is correlated\nwith the positive inotropic response to ET (4, 8, 18). Accelerated relax-\nation also is observed in response to PKC activation by ET in myocytes\nexpressing cTnI (4). PKC phosphorylation of purified cTnI decreases\nmyofilament Ca2 sensitivity in reconstituted myofilaments (11), and\nthis desensitization is expected to contribute to the accelerated relax-\nation. In contrast, peak shortening is diminished and relaxation is\ndelayed rather than accelerated in response to ET when endogenous\ncTnI is replaced with nonphosphorylatable slow skeletal troponin I\n(ssTnI) in adult myocytes. These results are consistent with the idea that\ncTnI phosphorylation enhances peak myocyte shortening and acceler-\nates relaxation rate in response to PKC activation by ET. However,\nssTnI expression in adult myocytes also increases myofilament Ca2\nsensitivity and slows the Ca2 transient (19, 20). Thus, the differential\nresponse to ET observed in myocytes expressing cTnI versus ssTnI may\nresult from the direct actions of cTnI phosphorylation on shortening\nand relaxation or from alterations in the Ca2 transient induced by\nssTnI expression.\n**BLOCK**fs== 8.8**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nOne goal of the present study is to establish whether PKC-mediated\ncTnI phosphorylation acts directly to accelerate relaxation or is\nexplained by parallel alterations in cellular Ca2. Calcium and shorten-\ning responses are measured simultaneously during the ET response for\nthese experiments. Stimulation frequency also alters the Ca2 transient,\nwith accelerated sarcoplasmic reticulum Ca2 uptake observed at\nhigher pacing frequencies (21). The relative contribution of phospho-\nrylated cTnI to the ET-induced relaxation response may decrease as\npacing frequency increases in the event that alterations in cellular Ca2\nare primarily responsible for this relaxation response. Thus, the contri-\nbution of cTnI phosphorylation to the ET-induced relaxation response\nis examined over a range of frequencies.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nA second goal of our study was to differentiate between cTnI phos-\nphorylation-induced and alkalosis-induced changes in the ET-mediated\ncontractile performance response. Previous investigators showed that\nET increases Na/H exchange via PKC (22, 23). The ensuing alkalosis\nincreases myofilament Ca2 sensitivity (24), and ET-induced alkalosis\nhas been shown to contribute to the enhanced peak shortening and\nincreased rate of contraction (22, 23). The diminished peak-shortening\nresponse to ET observed in ssTnI-expressing myocytes (4) also could\nresult from the enhanced basal myofilament Ca2 sensitivity and dimin-\nished pH sensitivity relative to cTnI-expressing myocytes (19). In the\npresent study, the influence of ET-induced alkalosis on peak contrac-\ntion was tested by inhibiting Na/H exchange in myocytes expressing\ncTnI and ssTnI.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe final aim of the present work was to determine the contribution\nof individual phosphorylation sites within cTnI to the PKC-mediated\nchanges in intact adult myocyte relaxation rate in response to ET.\nDetailed work on purified cTnI indicates that PKC phosphorylates three\nseparate clusters of residues, including Ser23/Ser24, Ser43/Ser45, and\nThr144 within cTnI (11, 25). However, the myocardial response to phos-\nphorylation of individual sites within the intact myocyte has yet to be\n**BLOCK**fs== 8.8**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nthoroughly investigated. Studies here are focused on investigating the\ninfluence of the rat cTnI Ser23/Ser24 and Thr144 sites on contractile\nfunction in response to ET. For studies of cTnIThr144, the goal is to\ndetermine whether Thr144 influences the peak contraction and relax-\nation response to PKC activated by ET, by substitution with the non-\nphosphorylatable Pro residue. The Ser23/Ser24 cluster also is a target for\nPKC in purified cTnI and is a major myofilament target for protein\nkinase A (PKA) (26). Phosphorylation of this cluster by PKA signifi-\ncantly decreases myofilament Ca2 sensitivity (27) and accelerates\nrelaxation during -adrenergic activation (28). A similar increase in\nrelaxation is expected if this Ser cluster is phosphorylated by PKC in\nintact adult myocytes. For these studies, the Ser23/Ser24 residues were\nsubstituted with Ala to investigate whether this cluster influences the\nability of cTnI to maintain relaxation in response to ET activation of\nPKC.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nEXPERIMENTAL PROCEDURES\n**BLOCK**fs== 8.8**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nMutagenesis Strategy—Wild-type rat cTnI cDNA (kind gift of A.\nMurphy) was used to construct the cTnIT144P and cTnIS23A/S24A\nmutants. Primers used for mutagenesis of cTnIT144P were 5-gcaagtt-\ntaagcggccgcctctccgaagagtg-3 (sense) and 5-cactcttcggagaggcggccgct-\ntaaacttgc-3 (antisense), and for cTnIS23A/S24A the primers were\n5-ccgacgtcgcgccgcggccaactaccg-3 (sense) and 5-cggtagttggccgcg-\ngcgcgacgtcgg-3 (antisense) using a pGEM-3Z vector and the Strat-\nagene QuikChange site-directed mutagenesis kit (29). The presence of\neach mutant was verified by DNA sequencing.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nGeneration of Adenoviral Vectors—Recombinant adenovirus vectors\nwere constructed by cotransfection of pCA4cTnI cDNA with pJM17 by\ncalcium phosphate into a HEK 293 cell\nline. Replication-deficient,\nrecombinant adenovirus containing wild-type rat cTnI, wild-type\nssTnI, or mutant ssTnI cDNAs, a cytomegalovirus promoter, and an\nSV40 polyadenylation signal were obtained following homologous\nrecombination (19).\n**BLOCK**fs== 8.8**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nMyocyte Isolation—Adult rat ventricular myocytes were isolated as\ndescribed earlier (4, 30). Myocytes were plated on laminin-coated cov-\nerslips in Dulbecco’s modified Eagle’s medium supplemented with 50\nunits/ml penicillin, 50 g/ml streptomycin (pen/strep) and 5% serum\nfor 2 h prior to replacing the medium with serum-free Dulbecco’s mod-\nified Eagle’s medium containing recombinant adenovirus vector (mul-\ntiplicity of infection  500). Gene transfer of exogenous TnI at a mul-\ntiplicity of infection of 500 results in virtually all cells expressing and\nincorporating exogenous TnI into the myofilament (29). After 1 h, Dul-\nbecco’s modified Eagle’s medium with pen/strep (2 ml) was added, with\nregular replacement of media over 4 days in culture.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nAnalysis of Protein Expression—Control and virus-treated adult rat\ncardiac myocytes were scraped from coverslips into sample buffer 4\ndays after gene transfer (31). In some experiments, myocytes were per-\nmeabilized in 0.1% Triton X-100 for 1 min to evaluate incorporation of\nTnI protein into the myofilament after gene transfer. In earlier work,\nexpression in intact and permeabilized myocytes was comparable when\nmyofilament proteins were incorporated into the myofilament (29, 32).\nProteins were then separated by gel electrophoresis and blotted onto\npolyvinylidene difluoride membrane for 2000 V-h with immunodetec-\ntion carried out as described earlier (30) using a 1:1000 dilution of MAB\n1691 (Chemicon Inc., Temecula, CA), a monoclonal antibody that rec-\nognizes all TnI isoforms. A phospho-Ser23/Ser24-specific polyclonal\nantibody (1:1000; Cell Signaling Technology, Beverly, MA) and a paired\nhorseradish peroxidase-conjugated secondary antibody were used to\ndetect Ser23/Ser24 phosphorylation. Myocytes collected for detection of\nphosphorylation were incubated at 37 °C with calyculin A (10 nM) alone\n**BLOCK**fs== 8.8**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nor with the selected PKA and PKC agonist and/or inhibitor for time\nintervals ranging from 2.5 min up to 1 h.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nMeasurement of Sarcomere Shortening in Single Cardiac Myocytes—\nMyocytes to be used for shortening assays were transferred to a Plexiglas\nstimulation chamber consisting of 8 wells containing platinum elec-\ntrodes. Myocytes were stimulated electrically (2.5-ms pulse, 0.2 Hz) in\nmedium 199 supplemented with pen/strep (M199), 10 mM HEPES, 0.2\nmg/ml bovine serum albumin, and 10 mM glutathione (M199), and\nthe media was replaced every 12 h for days 2– 4 post-gene transfer (33).\nThe voltage was set so that 20% of the myocytes were stimulated on\neach coverslip. Four days after gene transfer, individual coverslips were\ntransferred to a temperature-controlled (37 °C) chamber mounted on a\nNikon microscope stage. A video-based detection system (Ionoptix,\nMilton, MA) was used to detect sarcomere shortening in intact myo-\ncytes. For frequency-response studies, myocytes were stimulated at\neach frequency (0.2, 0.5, 0.75, 1, and 2 Hz) until a steady state contrac-\ntion was reached. Sarcomere shortening also was recorded at 0.2 Hz\nbefore and after the increases in stimulation frequency to ensure there\nwere no changes in the rate of contraction, peak contraction, and rate of\nrelaxation after testing at all frequencies. Experiments were recorded\nusing SarcLen software, and an average of 10 twitches/myocyte was\ncollected under basal conditions. Signal-averaged data were analyzed to\ndetermine resting sarcomere length (SLR), percent change in peak sar-\ncomere shortening above base line (Sb), time to peak shortening (TTP),\ntime to 25, 50, and 75% relaxation (TTR25, TTR50, and TTR75, respec-\ntively), and maximum normalized shortening and relaxation velocities\n(dl/dtmax, dl/dtmax, respectively). Media containing ET (10 nM),\n5-N(ethyl-N-iospropyl)amiloride (EIPA, 500 nM) or both reagents were\nperfused via separate individual perfusion tubing to eliminate introduc-\ntion of residual agonists/inhibitors.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nExperiments for Ca2 transient and sarcomere shortening measure-\nments were carried out on myocytes loaded with Fura-2 AM, as\ndescribed previously (34). Briefly, myocytes were loaded for 5 min with\n5 M Fura-2 AM (final Me2SO  0.6%) at 37 °C followed by a 5-min\nwash-out for de-esterification. Myocytes were then stimulated at 0.2 Hz\nat 37 °C, and the base-line sarcomere length and Ca2 transient were\nrecorded from a single cell. Recordings were then made 5, 10, and 15\nmin after the addition of ET (10 nM), with the cell maintained in the dark\nbetween sampling times. Optimal cTnI phosphorylation was observed\nin response to 10 –100 nM ET in an earlier study (4). Ten twitches/\nmyocyte were collected for each sample. In addition to sarcomere short-\nening measurements, the resting and peak ratios, maximum Ca2 tran-\nsient rate (dCa/dtmax), maximum Ca2 decay rate (dCa/dtmax), and\ntime to 50 and 75% decay (TTD50, TTD75, respectively) in the Ca2\ntransient were measured. The Ca2 transient and shortening measure-\nments were maintained over this same time interval in myocytes main-\ntained under basal conditions.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nStatistical Analysis—Values are expressed as mean  S.E. Grouped\ncomparisons are analyzed using a Student’s t test or analysis of variance\ntesting (ANOVA). Statistical differences detected by ANOVA are com-\npared with a Newman-Keuls multiple comparison test with p  0.05\nconsidered significantly different.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nRESULTS\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nRelationship between Ca2 Transient and Contractile Response to\nEndothelin—Sarcomere shortening and Ca2 transients were measured\nbefore and after ET in Fura-2-loaded myocytes expressing cTnI or ssTnI\n4 days after gene transfer (Fig. 1, TABLE ONE). The increased peak\nshortening and enhanced relaxation rate observed in response to 10 nM\nET in earlier work on myocytes expressing cTnI (4), was similarly\n**BLOCK**fs== 7.8**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nTABLE ONE\nComparison of the percent change in sarcomere shortening and Ca2\ntransient measurements in response to 10-min exposure to ET (10\nnM) for myocytes expressing cTnI, ssTnI, or cTnIT144P\nPercent change in resting sarcomere length (SLR), maximum normalized\ndeparture velocity (dl/dtmax), peak shortening normalized for resting length\n(Sb), maximum normalized relaxation velocity (dl/dtmax), and time to 50\nand 75% relaxation from peak (TTR50, TTR75) are shown for the shortening\nmeasurements. Ca2 measurements shown include percent changes in rest-\ning Ca2 fluorescence, maximum normalized Ca2 rise (dCa/dtmax), peak\nCa2 normalized for resting Ca2 (peak Ca2), maximum normalized Ca2\ndecay (dCa/dtmax), and time to 50 and 75% decay from peak (TTD50,\nTTD75). Results are expressed as mean  S.E. Statistical comparisons were\nmade between the control and ssTnI responses and were performed using\nANOVA (p  0.05) and post-hoc Newman-Keuls tests (see Footnotes a\nand b).\n**BLOCK**fs== 7.8**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nMeasurements\n**BLOCK**fs== 7.8**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nShortening\n**BLOCK**fs== 7.8**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nSLR\ndl/dtmax\nSb\ndl/dtmax\nTTR50\nTTR75\n**BLOCK**fs== 7.8**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nCa2 transient\n**BLOCK**fs== 7.8**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nControl (n  35)\n**BLOCK**fs== 7.8**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n% Change in response to ET\nssTnI (n  25)\n**BLOCK**fs== 7.8**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\nT144P (n  25)\n**BLOCK**fs== 7.8**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\n0.76  0.29\n9.22  6.43\n1.37  5.31b\n**BLOCK**fs== 7.8**b== 0.6**t== 0.3**l== 0.6**r== 0.1**\n1.24  0.54\n0.46  0.27\n0.08  4.86\n13.94  12.10\n37.72  6.96a\n44.05  7.24a\n16.33  4.53a 15.67  5.37a,b 21.95  6.45a,b\n37.44  8.35a,b\n0.09  5.21\n28.41  8.26a,b\n6.62  4.60\n**BLOCK**fs== 7.8**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n11.43  4.87\n31.63  13.17a,b\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nFIGURE 1. Composite Ca2 transients (left panels) and shortening traces (right pan-\nels) recorded before and after ET (10 nM) in myocytes expressing cTnI (A) and ssTnI\n(B). A, composite trace of Ca2 transient and shortening made from 35 myocytes\nexpressing cTnI before and after ET. B, composite tracing of Ca2 transient and sar-\ncomere shortening made before and after ET in 25 myocytes expressing ssTnI. Record-\nings were made 4 days after gene transfer in all myocytes, and ET recordings were cap-\ntured between 10 and 15 min after the addition of ET.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nobserved in myocytes loaded with Fura-2 (Fig. 1A). In myocytes\nexpressing nonphosphorylatable ssTnI, the delayed relaxation and loss\nof the enhanced peak shortening observed in response to ET (4) simi-\nlarly developed in Fura-2-loaded cells (Fig. 1B, TABLE ONE).\n**BLOCK**fs== 8.8**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nCa2 transient measurements also were examined in these myocytes.\nThe time to 50% decay for basal Ca2 transients reportedly slows in\ntransgenic mouse myocytes expressing ssTnI (20), and similar results\ndeveloped in rat myocytes expressing ssTnI (TABLE TWO). Peak Ca2\nand the TTP remained unchanged in the ssTnI-expressing myocytes\ncompared with values observed in controls at 0.2 Hz. The Ca2 tran-\nsient change observed in response to ET was then analyzed in myocytes\nexpressing cTnI and ssTnI (TABLE ONE). The percent change in peak\nCa2 amplitude and maximum rate of Ca2 decay (dCa/dtmax) in\nresponse to ET was not significantly changed in myocytes expressing\neither cTnI or ssTnI (Fig. 1, A and B; TABLE ONE). However, signifi-\ncant delays were observed in the time to 75% Ca2 decay in both groups\nof myocytes. The similar Ca2 transient responses to ET coupled with\nthe divergent shortening response of cTnI- versus ssTnI-expressing\nmyocytes indicates that cTnI contributes significantly to the enhanced\npeak shortening and accelerated relaxation rate observed in response to\nPKC activation by ET compared with myocytes expressing ssTnI.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAdditional studies were performed to investigate whether the contri-\nbution of phosphorylated TnI to the ET-mediated change in contractile\nfunction is diminished with higher frequencies. The shortening  fre-\nquency relationship described previously in rodents (33, 35) was similar\nunder basal conditions and in response to ET over a range of frequencies\n(0.2–2 Hz), (TABLE THREE) in control myocytes. The percent decrease\nin peak shortening, dl/dtmax and dl/dtmax in response to increased\nfrequency was similar before and after ET. The consistency of this\nresponse provides evidence that the contribution of cTnI phosphoryla-\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.5**r== 0.2**\nResting Ca2 2.49  0.83\n3.92  2.42\n1.21  28.09\n29.03  25.88\ndCa/dtmax\nPeak Ca2\n2.03  9.21\n29.08  10.79\n16.27  15.47 18.69  12.96\ndCa/dtmax\n39.93  21.69\nTTD50\n64.66  14.09a\nTTD75\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n10.33  5.29\n81.69  17.61a\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.8**r== 0.1**\n1.74  0.71\n3.82  11.22\n18.95  11.39\n9.04  14.73\n23.57  11.07\n90.39  27.97a\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\na Peak response compared with base line.\nb Control versus ssTnI.\n**BLOCK**fs== 7.8**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nTABLE TWO\nComparison of base-line sarcomere shortening and Ca2 transient in\nadult rat cardiac myocytes expressing cTnI (control), ssTnI, or\ncTnIT144P\nResults are expressed as mean  S.E., with p  0.05 (see Footnote a) consid-\nered significantly different from control myocytes when compared by\nANOVA and post-hoc Newman-Keuls tests.\n**BLOCK**fs== 7.8**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nMeasurements\n**BLOCK**fs== 7.8**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\ncTnI (n  35)\n**BLOCK**fs== 7.8**b== 0.4**t== 0.6**l== 0.7**r== 0.1**\nssTnI (n  25) T144P (n  25)\n**BLOCK**fs== 7.8**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nBasal shortening\nSLR (m)\nTTP (s)\nPeak height (m)\nTTR50 (s)\n**BLOCK**fs== 7.8**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nBasal Ca2 transient\n**BLOCK**fs== 7.8**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nTTP (s)\nPeak Ca2 (AU)\nTTD50 (s)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\na p  0.05.\n**BLOCK**fs== 7.8**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n1.821  0.006\n0.070  0.003\n0.182  0.010\n0.035  0.002\n**BLOCK**fs== 7.8**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n1.777  0.019\n0.075  0.006\n0.186  0.017\n0.125  0.012a\n**BLOCK**fs== 7.8**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n1.818  0.010\n0.070  0.004\n0.179  0.015\n0.029  0.002\n**BLOCK**fs== 7.8**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n0.044  0.003\n9.60  1.03\n0.148  0.007\n**BLOCK**fs== 7.8**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\n0.047  0.004\n6.70  1.3\n0.189  0.009a\n**BLOCK**fs== 7.8**b== 0.2**t== 0.7**l== 0.8**r== 0.1**\n0.045  0.004\n7.46  1.89\n0.149  0.006\n**BLOCK**fs== 8.8**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\ntion to contractile function is not altered by frequency-induced changes\nin the Ca2 cycle.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nContribution of Cellular Alkalosis to the Myofilament Response to\nEndothelin—An important component of the contractile response to\nET depends on activation of Na/H exchange and the generation of a\ncellular alkalosis (22, 23). Experiments were carried out in the presence\nof the amiloride analog EIPA, an inhibitor of Na/H exchange (23, 36),\nto identify responses resulting from alkalosis. The myocyte response to\nEIPA is comparable in nontreated myocytes and myocytes expressing\ncTnI after gene transfer (results not shown); the pooled results are\nshown in Fig. 2. Myocyte shortening amplitude in myocytes expressing\n**BLOCK**fs== 7.8**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTABLE THREE\nFrequency response (0.2 and 2 Hz) of electrically stimulated adult rat\ncardiac myocytes observed before and after ET (10 nM) in control\nmyocytes 4 days after myocyte isolation\nResults are expressed as mean  S.E. of the percent change in sarcomere\nshortening measurements made between 0.2 and 2.0 Hz under basal condi-\ntions and in response to 10 –15-min exposure to ET. There were no significant\ndifferences between the frequency response under basal conditions and in\nresponse to ET using a Student’s t test with p  0.05 considered significant.\n**BLOCK**fs== 7.8**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nMeasurements\n**BLOCK**fs== 7.8**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nResting SL\nTTP\ndl/dtmax\nSb\ndl/dtmax\nTTR25\nTTR50\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n% Change from 0.2 and 2 Hz\n**BLOCK**fs== 7.8**b== 0.6**t== 0.2**l== 0.2**r== 0.7**\nBasal (n  13)\n1.00  0.24\n17.38  3.36\n5.85  3.82\n23.56  11.89\n19.80  21.53\n3.50  3.35\n3.69  3.60\n**BLOCK**fs== 7.8**b== 0.6**t== 0.2**l== 0.4**r== 0.5**\nET (n  13)\n1.23  0.29\n10.40  6.25\n6.09  10.83\n33.50  8.96\n15.21  12.65\n6.73  7.53\n8.90  5.46\n**BLOCK**fs== 8.8**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\ncTnI decreased slightly after the addition of EIPA but did not signifi-\ncantly influence peak amplitude in myocytes expressing ssTnI (Fig. 2, A\nand B). Resting sarcomere length (SLR), dl/dtmax, and dl/dtmax\n(dl/dt and dl/dt in Fig. 2, respectively) were not significantly altered\nby EIPA in myocytes expressing cTnI or ssTnI (Fig. 2B). However, EIPA\nblocked a major portion (70%) of the ET-induced increase in peak\nshortening amplitude observed in cTnI-expressing myocytes. In con-\ntrast, EIPA did not significantly modify the accelerated relaxation\nresponse to ET in myocytes expressing cTnI, and the delayed relaxation\nobserved in response to ET was comparable with and without EIPA in\nmyocytes expressing ssTnI (Figs. 1 and 2). These results support the idea\nthat Na/H exchange and the ensuing alkalosis is important for the\nenhanced peak shortening response, whereas cTnI plays a more critical\nrole in the accelerated relaxation rate response to ET.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nRole of cTnI Phosphorylation Residues in the Endothelin Response—\nTo determine the role of the cTnIThr144 phosphorylation site in the\nrelaxation response, the ssTnI Pro residue was substituted for Thr144 to\nform cTnIT144P, and replacement was measured 4 days after gene\ntransfer. The stoichiometric replacement of cTnI with cTnIT144P-\nFLAG was 46  2% (n  13) and comparable with the 53  2% (n  14)\nreplacement with cTnI-FLAG 4 days after gene transfer (Fig. 3).\nReplacement of nontagged cTnI and cTnIT144P is expected to be com-\nparable in myocytes over the same time interval (29, 37). Total TnI\nexpression remained constant in myocytes expressing cTnI and\ncTnIT144P based on the ratio of TnI intensity to the intensity of a\nsilver-stained band on the gel (cTnI set at 1.0; cTnIT144P  1.13  0.10,\nn  11), and isoform expression of tropomyosin and troponin T was\ncomparable in myocytes expressing cTnI and cTnIT144P (results not\nshown). Expression of total TnI also was comparable in intact (I) and\npermeabilized (P) myocytes (Fig. 3, I/P ratio for cTnI  1.10  0.08, n \n5; cTnIT144P  1.05  0.06, n  5), which is consistent with myofila-\nment incorporation of the delivered TnI.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nMeasures of basal contractile function in myocytes expressing\ncTnIT144P are not significantly different from values obtained from\nmyocytes expressing cTnI (TABLE TWO, Fig. 4A). These results are in\nagreement with the earlier reverse substitution of Pro110 with Thr in fast\nskeletal TnI (38). Base-line Ca2 transients in myocytes expressing\ncTnIT144P (Fig. 4B) also showed no significant differences from control\nvalues. Peak shortening increased significantly in response to ET in\ncTnIT144P-expressing myocytes (Fig. 4) and was comparable with the\nET-induced enhanced peak shortening in control myocytes (Fig. 1B).\nHowever, several indices of relaxation, including dl/dtmax and TTR75,\nwere significantly delayed in response to ET in myocytes expressing\n**BLOCK**fs== 7.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nFIGURE 2. Influence of the Na/H exchange inhibitor, EIPA, on sarcomere shorten-\ning in myocytes expressing cTnI and ssTnI before and after ET (10 nM). A, composite\ntraces of sarcomere shortening from 38 myocytes under basal conditions, after the addi-\ntion of EIPA (500 nM; and EIPA plus ET (10 nM). Results from myocytes expressing cTnI with\nor without adenovirus-mediated cTnI gene transfer were not significantly different from\neach other (results not shown) and were therefore pooled for these tracings. B, percent\nchange in shortening measurements from basal to EIPA and from EIPA to EIPA  ET.\nResults are shown for resting sarcomere length (SLR), peak shortening amplitude (Peak),\nmaximum shortening rate (dl/dt), and maximum relaxation rate (dl/dt) in myocytes\nexpressing cTnI (n  38) or ssTnI (n  16) after EIPA (upper panel) and EIPA  ET (lower\npanel). Myocyte shortening was analyzed 4 days after gene transfer, and measurements\nwere carried out 15 min after the addition of EIPA and 10 min after the addition of\nEIPA  ET. Although SLR, dl/dt, and dl/dt were not significantly changed by EIPA, the\npercent change in peak shortening decreased significantly in myocytes expressing cTnI\n(p  0.05 versus base-line level by ANOVA and post-hoc Newman-Keuls tests). There\nwere no significant differences in the response to EIPA when comparing myocytes\nexpressing cTnI versus ssTnI. SLR, peak, and dl/dt were not significantly changed by\nEIPA  ET compared with EIPA alone. However, dl/dt increased significantly in myo-\ncytes expressing cTnI (*, p  0.05), and this increased rate of relaxation was significantly\nhigher than the change in relaxation observed in myocytes expressing ssTnI (, p  0.05\nversus control).\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\ncTnIT144P compared with controls (TABLE ONE). The amplitude of\nthe Ca2 transient response to ET was not significantly different com-\npared with control values for myocytes expressing cTnIT144P (TABLE\nONE, Fig. 4). As with control myocytes, a frequency change from 0.2 to\n2 Hz produced similar changes before and after ET in myocytes express-\ning cTnIT144P (results not shown). The major difference between myo-\ncytes expressing cTnIT144P versus cTnI was the decrease in relaxation\nrate in response to ET; this finding indicates a specific role for Thr144 in\naccelerating relaxation in response to ET.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe contribution of Ser23/Ser24 phosphorylation to the ET-induced\nchange in contractile function also was determined. A phospho-specific\nSer23/Ser24 antibody was used for these studies. Myocytes were incu-\nbated with the PKA agonist dobutamine (DOB) for 2 min to verify\ndetection of the well documented PKA-mediated cTnISer23/Ser24 phos-\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nFIGURE 3. Representative expression of cTnIT144P in adult myocytes 4 days after\ngene transfer. A, gene transfer and expression of cTnI and cTnIT144P with and without\nthe FLAG epitope. All myocytes were collected on day 4 after gene transfer. B, expression\nof cTnI, ssTnI, cTnIT144P, and the FLAG-tagged version of cTnI and cTnIT144P in myo-\ncytes along with cTnI in noninfected myocytes. Expression is measured in intact (I) and\nTriton X-100-permeabilized (P) myocytes. A silver (Ag)-stained portion of each SDS-poly-\nacrylamide gel is shown in the lower panel to indicate protein loading for each lane.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nFIGURE 4. Composite traces of sarcomere shortening (A) and Ca2 transients (B) in\n25 myocytes expressing cTnIT144P before and after ET. Measurements were made 4\ndays after gene transfer, and composite base-line values are shown in TABLE TWO.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nphorylation site (39, 40) (Fig. 5A). Detection of cTnI phosphorylation by\nthis antibody was specific, as it was blocked when, in addition to DOB,\nthe -adrenergic inhibitor propranolol was added to myocytes (results\nnot shown). There was no significant phosphorylation detected in myo-\ncytes incubated with ET for 10 min with (Fig. 5A) or without the PKC\ninhibitor bis-indolylmaleimide-1 (bis-1, results not shown). However,\nsignificant phosphorylation of Ser23/Ser24 developed in control myo-\n**BLOCK**fs== 7.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nFIGURE 5. Representative expression and phosphorylation of cTnISer23/Ser24 in\nresponse to PKC and PKA agonists. A, phosphorylation of cTnI Ser23/Ser24 in response\nto the -adrenergic agonist DOB for 2.5 min (200 nM), under basal conditions (10 and 60\nmin), in response to propranolol (PRO, 10 M, 10 min), ET (10 nM, 10 and 60 min),\nET  PRO (60 min), ET  bis-1 (500 nM, 60 min), phenylephrine (Phe, 1 M) with and\nwithout propranolol, and phorbol 12-myristate 13-acetate (PMA, 200 nM) with and with-\nout propranolol. Significant phosphorylation of cTnI on Ser23/Ser24 is detected in\nresponse to DOB and after 60 min of ET, phenylephrine, and phorbol 12-myristate 13-ac-\netate. Phosphorylation in response to ET is blocked by the PKC inhibitor bis-1, but phos-\nphorylation of Ser23/Ser24 in response to ET, phenylephrine, or phorbol 12-myristate\n13-acetate is not blocked by PRO. In additional experiments (results not shown), phos-\nphorylation of Ser23/Ser24 in response to DOB (2.5 min) was blocked by propranolol (10\nM), and the addition of the PKC agonist fibroblast growth factor-2 (10 ng/ml, 60 min) or\nbis-I alone (60 min) had no significant influence on cTnISer23/Ser24 phosphorylation. B,\nWestern blot detection of cTnI, cTnI-FLAG, cTnIS23A/S24A, (cTnI S23/24A) and cTnIS23A/\nS24A-FLAG (cTnI S23/24A FLAG) expression in intact (I) and permeabilized (P) myocytes 4\ndays after gene transfer. C, phosphorylation of cTnI Ser23/Ser24 by ET (10 nM, 60 min) in\nmyocytes expressing cTnI, cTnIS23A/S24A, or cTnIS23A/S24A-FLAG 4 days after gene\ntransfer. Western analysis of cTnISer23/Ser24 phosphorylation is shown in the top panel.\nExpression of cTnI, cTnIS23A/S24A, and cTnIS23A/S24A-FLAG are shown in the middle\npanel. A silver-stained (Ag stain) portion of the gel is shown in each blot to indicate\nprotein loading.\n**BLOCK**fs== 8.8**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\ncytes over a 1-h time span, and this ET-mediated phosphorylation was\nsignificantly reduced in the presence of the PKC inhibitor bis-1 but not\nby the -adrenergic antagonist propranolol (Fig. 5A). Treatment of\nmyocytes with other PKC agonists, including phenylephrine and phor-\nbol 12-myristate 13-acetate for 1 h also resulted in significant cTnI\nphosphorylation of Ser23/Ser24. Incubation with fibroblast growth fac-\ntor-2 was previously shown to activate PKC (41), yet this agonist did not\nsignificantly influence cTnI phosphorylation over a 1-h time span\n(results not shown). These results provide evidence for temporal phos-\nphorylation of cTnI Ser23/Ser24 in response to PKC activation by several\nPKC agonists.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nTo directly determine the contribution of Ser23/Ser24 to the contract-\nile function response to ET, cTnIS23A/S24A was expressed in adult\nmyocytes. By 4 days after gene transfer, 44  5% of cTnI was replaced\nwith the cTnIS23A/S24A (n  15, Fig. 5B). DOB phosphorylation of\ncTnI was significantly reduced in myocytes expressing cTnIS23A/S24A\nor cTnIS23A/S24A-FLAG (results not shown). In addition, Ser23/Ser24\nphosphorylation was significantly attenuated after 1 h of ET in myocytes\nexpressing tagged and nontagged cTnIS23A/S24A (Fig. 5C). This S23A/\nS24A substitution has previously been shown to have little influence on\nbasal myofilament function (42), and no significant differences were\nobserved in SLR, peak amplitude, dl/dtmax, ordl/dt max in myocytes\nexpressing cTnIS23A/S24A compared with controls (results not\nshown). Control myocytes expressing cTnI or\nthe substituted\ncTnIS23A/S24A responded similarly to ET over the first 15 min (Fig. 6).\nHowever, a significant difference in the response was observed 1 h after\nthe addition of ET in myocytes expressing cTnIS23A/S24A compared\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.1**r== 0.7**\nFIGURE 6. Influence of cTnISer23/Ser24 on the\ncontractile function response to ET (10 nM) in\nmyocytes over time. Results are expressed as per-\ncent change in sarcomere shortening measure-\nments. Measurements are shown for percent\nchange in base-line sarcomere length (baseline SL),\npeak shortening, dl/dtmax, and time to relaxation\n75% (TTR75) and are compared in myocytes\nexpressing cTnI (control; n  24), ssTnI (n  15),\nand S23A/S24A (n  16) 4 days after gene transfer.\nNo significant differences between myocytes\nexpressing cTnI\nand cTnIS23A/S24A were\nobserved in response to 15 min ET, but mainte-\nnance of relaxation at 60 min was significantly\ndecreased in myocytes expressing ssTnI and\ncTnIS23A/S24A compared with controls. *, p \n0.05 versus control by ANOVA and post-hoc New-\nman-Keuls tests.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nwith control values (Fig. 6). Specifically, peak shortening amplitude was\nnot increased and both dl/dtmax and TTR75 indicated relaxation was\nsignificantly prolonged in myocytes expressing cTnIS23A/S24A com-\npared with control myocytes and were comparable with the results\nobtained in myocytes expressing ssTnI (Fig. 6). These results demon-\nstrate Ser23/Ser24 phosphorylation contributes to the accelerated relax-\nation during more prolonged myocyte PKC activation by ET.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nDISCUSSION\n**BLOCK**fs== 8.8**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nResults from the present study demonstrate the important direct role\nfor cTnI phosphorylation in accelerating relaxation in response to PKC\nactivated by ET in adult myocytes (Figs. 1–3). Our results now demon-\nstrate a clear role for cTnI Thr144 and Ser23/Ser24 in contributing to the\naccelerated relaxation in response to ET (Figs. 4 and 6). Substitution\nwith a Pro residue at Thr144 results in the delayed relaxation previously\nobserved in myocytes expressing nonphosphorylatable ssTnI and pro-\nvides evidence that the Thr144 site contributes to accelerated relaxation\nduring the ET response. The Ser23/Ser24 cluster does not appear to\ncontribute to the initial response to ET but, instead, is important for the\naccelerated relaxation during more prolonged stimulation by ET. This\nfinding indicates that the relaxation rate response to ET-induced PKC\npathway activation depends on temporal phosphorylation of individual\nclusters within cTnI.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nTnI Phosphorylation Plays a Direct Role in the Relaxation Response to\nEndothelin—Results from the present study demonstrate the important\nrole of cTnI phosphorylation in the relaxation response during the pos-\nitive inotropic response to ET, independent of cellular Ca2 handling.\nThe only aspect of Ca2 transient that changed significantly in response\nto 10 nM ET was the percent change in TTD75, which decreased in both\ncontrol and ssTnI-expressing myocytes (Fig. 1, TABLE ONE). All other\nmeasurements of the Ca2 transient remained unchanged during the\nET response, and work by other laboratories supports this finding (43,\n44). There are reports that ET activation of PKC enhances the Ca2\ntransient at higher doses of ET (45), which may be due to increased Ca2\nentry via the L-type Ca2 channel (46). However, results from Ebihara et\nal. (47) indicate the increased Ca2 transient is not necessary for the\nenhanced contractile response to ET.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nExperiments with the NHE1 inhibitor EIPA also demonstrate that\ninhibition of NHE1 prevents the ET-mediated enhancement in peak\nshortening amplitude (Fig. 2). ET in the concentration range used here\n**BLOCK**fs== 8.8**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nactivates PKC-induced NHE1 (48) and causes a cellular alkalosis (22,\n23). The alkalosis-mediated increase in myofilament Ca2 sensitivity\n(24) is responsible for the increased peak contractile shortening (22, 23).\nThis enhancement in peak shortening is absent from myocytes express-\ning ssTnI (Fig. 1), which is likely due to the combination of increased\nbasal myofilament Ca2 sensitivity (19, 20) and a decreased pH response\nto alkalosis in these myocytes compared with cTnI-expressing myocytes\n(19).\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nIn contrast to peak shortening, the divergent influence of cTnI versus\nssTnI on the relaxation response to ET was not influenced significantly\nby EIPA (Fig. 2). Although the slowing of relaxation in myocytes\nexpressing ssTnI was attenuated by the presence of EIPA during the ET\nresponse, relaxation was significantly accelerated in myocytes express-\ning cTnI versus ssTnI during this response. Our results now provide\nevidence that phosphorylated cTnI maintains relaxation regardless of\nthe alkalosis. The delayed relaxation response to ET observed in myo-\ncytes expressing cTnIT144P also supports this finding (Fig. 4).\n**BLOCK**fs== 8.8**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nWhat Role Does cTnI Phosphorylation Play in the Contractile\nResponse to ET?—The role of cTnI phosphorylation in the contractile\nfunction response to PKC agonists remains controversial. Earlier in\nvitro work showed that PKC phosphorylation of cTnI decreases myofil-\nament Ca2 sensitivity as well as maximum actomyosin ATPase activity\n(11, 25). PKA phosphorylation of cTnI also decreases myofilament Ca2\nsensitivity and results in accelerated myocyte relaxation (27, 28), and\nsubstitution of Ser23/Ser24 with Asp to mimic phosphorylation acceler-\nates relaxation in transgenic mice (39). However, PKC phosphorylates\nSer43/Ser45 and Thr144 in addition to the Ser23/Ser24 cluster (11). More\nrecent studies in which the Ser23/Ser24, Ser43/Ser45, and Thr144 clusters\nare all replaced with Ala in knock-in mice (49, 50) or substituted with\nAsp to mimic phosphorylation in transgenic mice (37) suggest that cTnI\nworks primarily to delay cardiac relaxation during PKC activation.\nThese results are in contrast to results presented here demonstrating an\naccelerated relaxation response attributable to cTnI phosphorylation\n(TABLE ONE), and the in vitro work demonstrating decreased myofil-\nament Ca2 sensitivity (11). Differences in experimental conditions,\nsuch as load and/or temperature in these various models, do not appear\nto account for the divergent functional response. We propose that these\nseemingly divergent responses may be due to a hierarchy of cTnI phos-\nphorylation in response to a given agonist, with different cTnI clusters\nphosphorylated in response to PKC activation. Ser43/Ser45 phosphoryl-\n**BLOCK**fs== 8.8**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\nation may have a dominant influence on contractile relaxation via its\ninfluence on maximum ATPase activity when all five phosphorylation\nsites are replaced by Ala (49) or replaced with a negatively charged Asp\n(37). However, PKC activation by a specific agonist may differentially\nand/or temporally phosphorylate one or more of the cTnI residues\nrather than phosphorylate all three clusters. In support of this idea, in\nvitro motility assays carried out with cTnISer43/Ser45 containing\ncharged Glu substitutions demonstrated a decrease in myofilament\nCa2 sensitivity and decreased sliding speed, whereas cTnIT144G only\ndecreased myofilament Ca2 sensitivity (51). NMR analysis of Ser43/\nSer45 substituted with Asp provides evidence that phosphorylation of\nthis site may be dominant in that it influences the Ca2 binding affinity\nto TnC as well as protein-protein interactions between TnI and TnC\n(52). Thus, the divergent results are predicted to stem from differential\nphosphorylation of the three cTnI phosphorylation clusters and/or the\nrelative dominance of one cluster in response to activation and translo-\ncation of specific PKC isoforms by a given agonist. This idea will be\nimportant for rigorous testing, with a variety of agonists under physio-\nlogical and pathophysiological conditions, to better understand the\nPKC signaling pathway and its ability to modulate both cTnI phospho-\nrylation and contractile performance.\n**BLOCK**fs== 8.8**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nRole of Individual cTnI-phosphorylated Residues in the Contractile\nResponse to ET—Detailed studies on purified cTnI support the idea that\nthe residue phosphorylated by PKC is important for the resulting func-\ntional relaxation response. Phosphorylation of Thr144 in cTnI produces\na rightward shift in myofilament Ca2 sensitivity for both actomyosin\nATPase activity (11) and force generation in permeabilized trabeculae\n(51, 53). The anticipated cellular response is an increase in myocyte\nrelaxation rate. Our results now show that substitution of Thr144 with\nPro significantly delays relaxation during the ET response (Fig. 4,\nTABLE THREE). The Thr144 site is particularly important because the\ninhibitory peptide region encompassing this phosphorylation target\n(amino acids 129 –149) is important for determining TnI binding to\nactin in the absence of Ca2 and diminished binding to actin in the\npresence of Ca2 (54). Thus, the inhibitory peptide region is important\nfor the molecular switch activity of TnI (55, 56). The delayed relaxation\nresponse to ET in myocytes expressing cTnIT144P compared with con-\ntrol myocytes is consistent with the idea that phosphorylation of this\nresidue in response to ET plays a significant role in the accompanying\nrelaxation rate response.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nExperiments with the phospho-cTnISer23/Ser24 antibody and with\ncTnIS23A/S24A also demonstrate a key role for this cluster during\nmore extended activation of PKC by ET. Phosphorylation of Ser23/Ser24\ndecreases myofilament Ca2 sensitivity and accelerates relaxation in\nresponse to PKA activation (27, 28), and similar shifts in myofilament\nCa2 sensitivity are reported in response to PKC phosphorylation of\nSer23/Ser24 (11). Our results now demonstrate a temporal pattern of\nSer23/Ser24 phosphorylation that is important for enhanced relaxation\nin response to longer intervals of ET in intact myocytes (Fig. 6). In\nbiochemical experiments with purified cTnI, longer treatment with\nPKC also was required for Ser23/Ser24 phosphorylation (11), yet it\nremained unclear whether this result indicated that there was temporal\nphosphorylation in the intact cell or instead indicated that Ser23/Ser24\nwas not a typical in vivo target for activated PKC. Results with myocytes\nexpressing cTnIS23A/S24A (Fig. 6) clearly demonstrate that there is\ntemporal phosphorylation of this site by PKC activated in response to\nET. Several PKC agonists, including ET, are released in a paracrine/\nautocrine pattern (57, 58), and phosphorylation of Ser23/Ser24 during\nmore prolonged intervals of PKC stimulation would then be important\nfor preventing delays in cellular relaxation. Taken together, the results\n**BLOCK**fs== 8.8**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nobtained in experiments to investigate Thr144 and Ser23/Ser24 provide\nevidence that both clusters are important in maintaining relaxation dur-\ning PKC activation. The relative importance, as well as the interactive\neffects, of these sites and the Ser43/Ser45 site in response to PKC ago-\nnists, which appears primarily to decrease peak contraction and slow\nrelaxation (51), will be important to investigate in the future for a com-\nplete understanding of the role played by cTnI phosphorylation in mod-\nulating cardiac function.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nAcknowledgments—We thank Andrea Borton (Cell Signaling Inc.) for the gift\nof the phospho-Ser23/Ser24 antibody. We also thank Ayako Ohata for technical\nassistance, and Sharlene Day, Joseph Metzger, and Daniel Michele for helpful\ncomments on earlier versions of this manuscript.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n1. Gwathmey, J., and Hajjar, R. (1990) Circ. Res. 67, 744 –752\n2. Buenaventura, P., Cao-Danh, H., Takeuchi, K., Takahashi, S., Simpleaceanu, E.,\nMcGowan, F. J., and del Nido, P. (1995) Ann. Thorac. Surg. 60, Suppl. 6, S505–S508\n3. Puceat, M., Clement, O., Lechene, P., Pelosin, J., Ventura-Clapier, R., and Vassort, G.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\n(1990) Circ. Res. 67, 517–524\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n4. Westfall, M. V., and Borton, A. R. (2003) J. Biol. Chem. 278, 33694 –33700\n5. Metzger, J. M., and Westfall, M. V. (2004) Circ. Res. 94, 146 –158\n6. He, J. Q., Pi, Y., Walker, J. W., and Kamp, T. J. (2000) J. Physiol. 524, 807– 820\n7. Alden, K. J., Goldspink, P. H., Ruch, S. W., Buttrick, P. M., and Garcia, J. (2002) Am. J.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\nPhysiol. 282, C768 –C774\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n8. Damron, D. S., Darvish, A., Murphy, L., Sweet, W., Moravec, C. S., and Bond, M.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\n(1995) Circ. Res. 76, 1011–1019\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n9. Noland, T. A., Jr., and Kuo, J. F. (1991) J. Biol. Chem. 266, 4974 – 4978\n10. Mazzei, G. J., and Kuo, J. F. (1984) Biochem. J. 218, 361–369\n11. Noland, T. A., Jr., Raynor, R. L., and Kuo, J. F. (1989) J. Biol. Chem. 264, 20778 –20785\n12. Noland, T. A., Jr., and Kuo, J. F. (1992) Biochem. J. 288, 123–129\n13. Noland, T. A., Jr., and Kuo, J. F. (1993) Biochem. Biophys. Res. Commun. 193, 254 –260\n14. Clement, O., Puceat, M., Walsh, M., and Vassort, G. (1992) Biochem. J. 285, 311–317\n15. Bodor, G. S., Oakeley, A. E., Allen, P. D., Crimmins, D. L., Ladenson, J. H., and\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nAnderson, P. A. W. (1997) Circulation 96, 1495–1500\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n16. Bowling, N., Walsh, R., Song, G., Estridge, T., Sandusky, G., Fouts, R., Mintze, K.,\nPickard, T., Roden, R., Bristow, M., Sabbah, H., Mazrahi, J., Gromo, G., King, G., and\nVlahos, C. (1999) Circulation 99, 384 –391\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n17. Noguchi, T., Hunlich, M., Camp, P. C., Begin, K. J., El Zaru, M., Patten, R., Leavitt, B. J.,\nIttleman, F. P., Alpert, N. R., LeWinter, M. M., and VanBuren, P. (2004) Circulation\n110, 982–987\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n18. Pi, Y., Sreekumar, R., Huang, X., and Walker, J. (1997) Circ. Res. 81, 92–100\n19. Westfall, M. V., Rust, E. M., and Metzger, J. M. (1997) Proc. Natl. Acad. Sci. U. S. A. 94,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\n5444 –5449\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n20. Fentzke, R. C., Buck, S. H., Patel, J. R., Lin, H., Wolska, B. M., Stojanovic, M. O.,\nMartin, A. F., Solaro, R. J., Moss, R. L., and Leiden, J. M. (1999) J. Physiol. (Lond.) 517,\n143–157\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n21. DeSantiago, J., Maier, L. S., and Bers, D. M. (2002) J. Mol. Cell Cardiol. 34, 975–984\n22. Kramer, B. K., Smith, T. W., and Kelly, R. A. (1991) Circ. Res. 68, 269 –279\n23. Goldberg, A. T., Bond, B. R., Mukherjee, R., New, R. B., Zellner, J. L., Crawford, F. A.,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nJr., and Spinale, F. G. (2000) Ann. Thorac. Surg. 69, 711–715\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n24. Mayoux, E., Coutry, N., Lechene, P., Marotte, F., Hoffmann, C., and Ventura-Clapier,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.3**\nR. (1994) Am. J. Physiol. 266, H2051–H2060\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n25. Noland, T. A., Jr., Guo, X., Raynor, R. L., Jideama, N. M., Averyhart-Fullard, V., Solaro,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nR. J., and Kuo, J. F. (1995) J. Biol. Chem. 270, 25445–25454\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n26. Robertson, S. P., Johnson, J. D., Holroyde, M. J., Kranias, E. G., Potter, J. D., and Solaro,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nR. J. (1982) J. Biol. Chem. 257, 260 –263\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n27. Westfall, M. V., Turner, I. I., Albayya, F., and Metzger, J. M. (2001) Am. J. Physiol. 280,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nC324 –C332\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n28. Kentish, J. C., McCloskey, D. T., Layland, J., Palmer, S., Leiden, J. M., Martin, A. F., and\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nSolaro, R. J. (2001) Circ. Res. 88, 1059 –1065\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n29. Westfall, M. V., Borton, A. R., Albayya, F. P., and Metzger, J. M. (2002) Circ. Res. 91,\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n525–531\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n30. Westfall, M. V., Rust, E. M., Albayya, F., and Metzger, J. M. (1997) Methods Cell Biol.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n52, 307–322\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n31. Westfall, M. (2004) Methods Mol. Biol. 219, 159 –166\n32. Westfall, M. V., Albayya, F. P., and Metzger, J. M. (1999) J. Biol. Chem. 274,\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n22508 –22516\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n33. Coutu, P., and Metzger, J. M. (2002) Biophys. J. 82, 2565–2579\n34. Coutu, P., and Metzger, J. M. (2005) Am. J. Physiol. 288, H601–H612\n35. Bolck, B., Munch, G., Mackenstein, P., Hellmich, M., Hirsch, I., Reuter, H., Hattebuhr,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nN., Weig, H. J., Ungerer, M., Brixius, K., and Schwinger, R. H. (2004) Am. J. Physiol.\n287, H1435–H1445\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nP. M., Bluhm, W. F., Meyer, M., Sayen, M. R., Swanson, E., and Dillmann, W. H. (1997)\nJ. Clin. Investig. 100, 380 –389\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n36. Kanaya, N., Murray, P. A., and Damron, D. S. (2001) Anesthesiology 94, 1096 –1104\n37. Sakthivel, S., Finley, N. L., Rosevear, P. R., Lorenz, J. N., Gulick, J., Kim, S., VanBuren,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nP., Martin, L. A., and Robbins, J. (2005) J. Biol. Chem. 280, 703–714\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n38. Strauss, J. D., Van Eyk, J. E., Barth, Z., Kluwe, L., Wiesner, R. J., Maeda, K., and Ruegg,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nJ. C. (1996) Pfluegers Arch. Eur. J. Physiol. 431, 853– 862\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n39. Takimoto, E., Soergel, D. G., Janssen, P. M., Stull, L. B., Kass, D. A., and Murphy, A. M.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(2004) Circ. Res. 94, 496 –504\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n40. Ward, D. G., Cornes, M. P., and Trayer, I. P. (2002) J. Biol. Chem. 277, 41795– 41801\n41. Padua, R., Merle, P.-L., Doble, B., Yu, C.-H., Zahradka, P., Pierce, G., Panagia, V., and\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nKardami, E. (1998) J. Mol. Cell Cardiol. 30, 2695–2709\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n42. Reiffert, S., Maytum, R., Geeves, M., Lohmann, K., Greis, T., Bluggel, M., Meyer, H. E.,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nHeilmeyer, L. M., and Jaquet, K. (1999) Eur. J. Biochem. 261, 40 – 47\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n43. Baudet, S., Weisser, J., Janssen, A., Beulich, K., Bieligk, U., Pieske, B., Noireaud, J.,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nJanssen, P., Hasenfuss, G., and Prestle, J. (2001) Cardiovasc. Res. 50, 486 – 494\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n44. Qiu, Z., Wang, J., Perreault, C. L., Meuse, A. J., Grossman, W., and Morgan, J. P. (1992)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nEur. J. Pharmacol. 214, 293–296\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n45. Damron, D. S., Van Wagoner, D. R., Moravec, C. S., and Bond, M. (1993) J. Biol. Chem.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\n268, 27335–27344\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n46. He, H., Giordano, F. J., Hilal-Dandan, R., Choi, D. J., Rockman, H. A., McDonough,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n47. Ebihara, Y., Haist, J. V., and Karmazyn, M. (1996) J. Mol. Cell Cardiol. 28, 265–277\n48. Slepkov, E., and Fliegel, L. (2002) Biochem. Cell Biol. 80, 499 –508\n49. Pi, Y., Kemnitz, K. R., Zhang, D., Kranias, E. G., and Walker, J. W. (2002) Circ. Res. 90,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n649 – 656\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n50. Pi, Y., Zhang, D., Kemnitz, K. R., Wang, H., and Walker, J. W. (2003) J. Physiol. 552,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n845– 857\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n51. Burkart, E. M., Sumandea, M. P., Kobayashi, T., Nili, M., Martin, A. F., Homsher, E.,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nand Solaro, R. J. (2003) J. Biol. Chem. 278, 11265–11272\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n52. Finley, N. L., and Rosevear, P. R. (2004) J. Biol. Chem. 279, 54833–54840\n53. Sumandea, M. P., Burkart, E. M., Kobayashi, T., de Tombe, P. P., and Solaro, R. J.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\n(2004) Ann. N. Y. Acad. Sci. 1015, 39 –52\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n54. Takeda, S., Yamashita, A., Maeda, K., and Maeda, Y. (2003) Nature 424, 35– 41\n55. Van Eyk, J. E., Strauss, J. D., Hodges, R. S., and Ruegg, J. C. (1993) FEBS Lett. 323,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n223–228\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n56. Li, Z., Gergely, J., and Tao, T. (2001) Biophys. J. 81, 321–333\n57. Zolk, O., Quattek, J., Sitzler, G., Schrader, T., Nickenig, G., Schnabel, P., Shimada, K.,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nTakahashi, M., and Bohm, M. (1999) Circulation 99, 2118 –2123\n58. Suzuki, T., Hoshi, H., and Mitsui, Y. (1990) FEBS Lett. 268, 149 –151",
         "extracted",
         "",
         "Differential Contribution of Troponin I Phosphorylation Sites to the Endothelin-modulated Contractile Response*",
         "None"
        ],
        [
         "5",
         "0004b93367db8768d29c7e21bf410d55a30f5205",
         "This paper documents a study, carried out in the academic year 2003-2004, on fifteen factors that may influence performance on a first year object-oriented programming module. The factors included prior academic experience, prior computer experience, self-perception of programming performance and comfort level on the module and specific cognitive skills. The study found that a student's perception of their understanding of the module had the strongest correlation with programming performance, r=0.76, p‹0.01. In addition, Leaving Certificate (LC) mathematics and science scores were shown to have a strong correlation with performance. A regression module, based upon a student's perception of their understanding of the module, gender, LC mathematics score and comfort level was able to account for 79% of the variance in programming performance results.",
         "Susan Bergin,R. Reilly",
         "http://dl.acm.org/ft_gateway.cfm?id=1047480&type=pdf",
         "\n**BLOCK**fs== 17.9**b== 0.8**t== 0.1**l== 0.2**r== 0.2**\nProgramming: Factors that Inﬂuence Success\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.5**\nSusan Bergin\nDepartment of Computer Science,\nNational University of Ireland Maynooth,\nMaynooth, Co. Kildare.\nsbergin@cs.may.ie\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.1**\nRonan Reilly\nDepartment of Computer Science,\nNational University of Ireland Maynooth,\nMaynooth, Co. Kildare.\nronan@cs.may.ie\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nABSTRACT\nThis paper documents a study, carried out in the academic\nyear 2003-2004, on ﬁfteen factors that may inﬂuence per-\nformance on a ﬁrst year object-oriented programming mod-\nule. The factors included prior academic experience, prior\ncomputer experience, self-perception of programming per-\nformance and comfort level on the module and speciﬁc cog-\nnitive skills. The study found that a student’s perception of\ntheir understanding of the module had the strongest corre-\nlation with programming performance, r = 0.76, p < 0.01.\nIn addition, Leaving Certiﬁcate (LC) mathematics and sci-\nence scores were shown to have a strong correlation with\nperformance. A regression module, based upon a student’s\nperception of their understanding of the module, gender, LC\nmathematics score and comfort level was able to account for\n79% of the variance in programming performance results.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nCategories and Subject Descriptors\nK.3.2 [Computer and Information Science Education]:\nComputer science education\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nGeneral Terms\nHuman Factors, Measurement, Performance\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.7**\nKeywords\nCS1, Programming, Predictors\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\n1.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nINTRODUCTION\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nStudent retention on third-level (post high school or equiv-\nalent) Computer Science (CS) and Information Technology\n(IT) courses is a signiﬁcant problem. Students ﬁnd com-\nputer programming diﬃcult and struggle to master the core\nconcepts. A multi-national, multi-institutional study on the\nprogramming skills of ﬁrst year CS students found that stu-\ndents struggled to achieve an average above 30% on assess-\nments administered as part of their study [14]. Further-\nmore, introductory programming modules tend to have a\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nPermission to make digital or hard copies of all or part of this work for\npersonal or classroom use is granted without fee provided that copies are\nnot made or distributed for proﬁt or commercial advantage and that copies\nbear this notice and the full citation on the ﬁrst page. To copy otherwise, to\nrepublish, to post on servers or to redistribute to lists, requires prior speciﬁc\npermission and/or a fee.\nSIGCSE’05 February 23–27, 2005 St. Louis, Missouri, USA.\nCopyright2005ACM1-58113-997-7/05/0002...$5.00.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nvery high student to lecturer ratio (100:1 or greater) and\noften lecturers do not know how well students are doing un-\ntil after the ﬁrst assessment. In general the ﬁrst assessment\ndoes not take place until six or eight weeks after the mod-\nule has commenced and given the typically high number of\nstudents, marking the assessments can take a considerable\nlength of time. At this stage, it may be too late to intervene\nto prevent struggling students from failing. Even if inter-\nvention is possible, a lecturer is unlikely to know a student\nwell enough or be able to identify individual student prob-\nlems and therefore recognize the most suitable interventions\nto make.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe research documented in this paper is part of a lon-\ngitudinal study on early identiﬁable factors that inﬂuence\nperformance on an introductory programming module.\nIf\nsuch factors can be identiﬁed then it may be possible to de-\nvelop a tool to provide an early diagnosis of a student’s likely\nperformance on a programming module. Interested parties\ncould use the tool to make more informed decisions on ap-\npropriate courses of action and to decide upon personalized\ninterventions that foster a student’s intellectual strengths.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\n2. RELATED RESEARCH\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nWhile a considerable amount of research has been carried\nout on factors that aﬀect programming performance our in-\nterest is on factors that aﬀect programming performance\non an introductory third-level object-oriented programming\nmodule where such factors can be determined early in the\nacademic year. These factors can be categorized as: (1)\nprevious academic and computer experience, (2) cognitive\nskills, (3) personal information and (4) experience on the\nmodule. A brief review of some research studies in each of\nthese categories is presented next.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nPrevious academic experience and programming experi-\nence have often been cited as predictors of programming suc-\ncess. Several studies have found that mathematical ability\nand exposure to maths courses are important predictors of\nperformance on introductory computer science modules [2,\n3, 4, 6, 13, 19]. Similarly, although less studied, performance\nin and experience of science subjects has also been shown\nto be important [2, 19]. Studies have also found prior pro-\ngramming experience and non-programming computer ex-\nperience (for example, experience of computer applications,\nemailing, game playing and surﬁng the web) to be related\nto programming performance [9, 10, 6, 4, 18].\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe role of cognitive factors in programming has also re-\nceived research attention. Certain cognitive factors, includ-\ning problem-solving, abstract reasoning, problem translation\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nskills, logical ability and cognitive style have been identiﬁed\nas possible predictors [7, 12, 11, 15].\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nNumerous studies have been carried out on demographic\nand self-reported personal information. Some studies have\nexamined speciﬁc attributes related to study, for example\npreference for working alone or in a group to solve a pro-\ngramming problem and encouragement or support from oth-\ners to study computers [4, 2, 8].\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nIn recent times researchers have examined the relation-\nship between students’ expectations of and experiences on\nan introductory computing module. A positive relationship\nbetween a student’s mental models of programming and self-\neﬃcacy for programming and performance has recently been\nidentiﬁed [18]. The grade a student expected to achieve in\nan introductory module was found to be the most impor-\ntant indicator of performance in another recent study[17].\nA recent longitudinal study found that the most important\npredictor of students’ performance on an introductory com-\nputer science course was comfort level, determined by the\ndegree of anxiety a student felt about the course [4].\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nAlthough a considerable number of research studies have\nexamined factors that inﬂuence programming performance,\ncomparisons between the various studies and application of\nthe results are diﬃcult because the studies are carried out\nusing diﬀerent parameters. These parameters include:\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n• the type of students (novice to experienced program-\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nmers, academic students to employees),\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n• the content of the course (some courses are solely pro-\ngramming courses while others are introductory com-\nputing courses),\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n• the programming language being taught (older studies\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\ntend to be based on procedural languages),\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n• the educational setting (many of the research studies\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nare based on the US educational system) and\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n• the reference criterion (for example continuous assess-\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nment, end-of-year exam, job performance).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nIn the rest of this paper we describe our study, which\nbuilds upon existing research, in particular the work of [2,\n4], to identify early factors that inﬂuence performance of\nﬁrst year students, on an introductory object-oriented pro-\ngramming module, using the Java programming language.\nThe course is based on the Irish educational system and\nthe reference criterion is the overall result achieved on the\nmodule.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\n3. RESEARCH DESIGN\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nThe introductory programming module at our university\nis composed of a one and a half hour Problem-Based Learn-\ning (PBL) workshop, a one and a half hour laboratory ses-\nsion and three one-hour lectures per week over two semesters.\nStudents in Ireland do not study programming in secondary\nschool and the majority of students taking this module have\nrecently completed second level education.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSelection of factors for this study was restricted for a num-\nber of reasons, including availability of participants, length\nof completion time needed for each instrument and stage in\nyear. With this in mind we attempted to examine the re-\nlationship between and the predictability of ﬁfteen factors\nand performance on our introductory module. The factors\nfall into four broad categories:\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.6**r== 0.1**\n1. previous academic and computer experience: as mea-\nsured by performance in the Irish Leaving Certiﬁcate\n(LC) examinations in mathematics and science sub-\njects and self-reported computer experience,\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\n2. speciﬁc cognitive skills: as measured by an in-house\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\ncognitive test,\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.6**r== 0.1**\n3. personal information: gender, age, work-style prefer-\nence, encouragement from others and the number of\nhours per week working at a part-time job and\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\n4. experience on the module: students own perception of\nhow well they are doing and how comfortable they feel\nwith the module material.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nPerformance on this module is based on continuous assess-\nment (30% of the overall mark) and a ﬁnal examination (70%\nof the overall mark). The measure of performance reported\nupon in this paper is the overall module mark. Continu-\nous assessment and ﬁnal examination marks render similar\nresults and are reported in [1].\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\n3.1 Participants\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nThe study was carried out in the academic year 2003-2004.\nStudents enrolled in the ﬁrst year ‘Introduction to Program-\nming’ module in our department voluntarily participated in\nthis study. Ninety-six students completed the module in the\nacademic year 2003-2004.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n3.2\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nInstruments\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nTwo instruments were used to collect data: a question-\nnaire and a custom-made cognitive test. The questionnaire\ncollected data on the following items: (1) LC mathematics\ngrade, (2) LC physics grade, (3) LC biology grade, (4) LC\nchemistry grade, (5) highest LC science grade, (6) comfort\nlevel on the module, (7) perceived understanding of the mod-\nule material, (8) prior programming experience, (9) prior\nnon-programming computer experience, (10) work-style pref-\nerence (preference to work-alone or as part of a group), (11)\nencouragement from others to study computer science, (12)\nnumber of hours per week working at a (part-time) job. The\ncognitive test was developed in-house1 and comprised items\ninvolving numerical and letter sequencing, arithmetic rea-\nsoning, problem translation skills and logical ability. In ad-\ndition, information on gender, age and overall module results\nwas available for all students taking the module.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nIn total 80 students (49 male, 31 female) completed the\ncognitive test and 30 (19 male, 11 female) students com-\npleted the survey. Both instruments were completed in the\nsecond semester of the module and data collection for both\nwas paper-based.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\n4. RESULTS\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nAn a priori analysis was carried out to verify no signif-\nicant diﬀerence existed between the mean overall module\nscores of the class and the sample. Test assumptions on\nnormality (Kolmogorov-Smirnov test) and the equality of\nvariance (Levine test) were performed and a t-test on the\noverall results, (t(124) = 0.795, p = 0.428), found no signif-\nicant diﬀerences between the mean scores of the class and\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n1Developed by Jacqueline McQuillan, Department of Com-\nputer Science, NUI Maynooth.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTable 1: Pearson correlations for previous academic\nresults and performance\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nTable 2: Dichotomous values for personal factors\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nLC\nMaths\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nLC\nPhys\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nLC\nChem\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nLC\nBio\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nHigh\nSci\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nr\nn\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n0.46**\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n0.59*\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n0.4\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n0.75*\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n0.48**\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nValues\nMale, Female\nGender\nUnder 23, 23+\nAge\nIndividual, Group\nWork style preference\nEncouragement\nYes, No\nPart-time employment No, Yes\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n0.72*\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n0.89** 0.88\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nFemale only\nr\nn\nMale only\n0.27\nr\nn\n\n** Correlation is signiﬁcant at the 0.01 level (2-tailed).\n* Correlation is signiﬁcant at the 0.05 level (2-tailed).\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n0.93** 0.84**\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n-0.04\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n0.16\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n0.16\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0.9\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nthe sample. In the remainder of this section the ﬁndings on\nthe relationship between each of the factors studied and pro-\ngramming performance is presented, followed by an analysis\nof the combination of factors that best predicts performance.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n4.1 Previous academic and computer experi-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nence\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nTo establish the relationship between previous academic\nexperience in mathematics and science, the achievable grades\nfor each subject were ranked, with the highest rank given\nto the highest possible grade and the lowest rank given to\nthe lowest possible grade. Table 1. provides the Pearson\ncorrelations for each of these measures and notable relation-\nships are identiﬁed. LC mathematics was found to have a\nstatistically signiﬁcant relationship with performance, r =\n0.46, p < 0.01. LC physics was found to be moderately\nstrong and signiﬁcant, r = 0.59, p < 0.05 as was LC biol-\nogy, r = 0.75, p < 0.05 for the ﬁnal examination. Highest\nscience result, which includes other less commonly studied\nscience subjects, was also found to be statistically signiﬁ-\ncant, r = 0.48, p < 0.01. No relationship was found be-\ntween LC chemistry and performance. Secondary analysis,\nbased on gender revealed that none of the measures were\nsigniﬁcant for male students and resulted in notably higher\ncorrelations for the female students, as shown in Table 1. A\nrecent study on gender diﬀerences in LC examinations found\nthat (1) more female students are taking higher level LC ex-\naminations than male students and (2) female students are\noutperforming male students on LC maths and physics ex-\naminations (no other science subjects were reviewed in the\nstudy) [5]. This may relate to our ﬁndings and further re-\nsearch is necessary.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe ﬁndings on the relationship between experience in\nmathematics and science subjects, and programming per-\nformance is in line with previous research ﬁndings. The\nstrength of the correlations between LC physics scores and\nparticularly LC biology scores and programming performance\nis interesting and would suggest that science in general has\na signiﬁcant inﬂuence on performance. However, the lack\nof correlation with LC chemistry appears contradictory and\nfurther research is required.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nTable 3: Pearson correlations for comfort level and\nperceived understanding with performance\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\nr\nn\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\nComfort level\n0.55**\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\nUnderstanding\n0.76**\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nFemale only\nr\nn\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nMale only\nr\nn\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n0.62*\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n0.79**\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n0.82**\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.8**r== 0.1**\n0.84**\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n** Correlation is signiﬁcant at the 0.01 level (2-tailed).\n* Correlation is signiﬁcant at the 0.05 level (2-tailed).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nPrevious computer experience was measured by prior pro-\ngramming experience and previous non-programming com-\nputer experience. In both cases student responses were sepa-\nrated into those with previous experience and those without\nprevious experience. Descriptive statistics for each group\nare given in Table 4. T-tests for independent samples were\nused to examine the diﬀerences between the overall mod-\nule results of each group. Before each t-test was carried\nout assumptions of normality and equality of variance were\nconﬁrmed. No signiﬁcant diﬀerences were found between\nstudents with or without previous programming experience\nor between students with or without non-programming com-\nputer experience and performance module. Although pre-\nvious research has found previous programming experience\nand non-programming computing experience to be indica-\ntors of success our results may be partially accounted for by\nthe fact that students cannot study programming or appli-\ncation software at examination level in secondary schools in\nIreland.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\n4.2 Speciﬁc cognitive skills\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nA correlation of r = 0.31, p < 0.01 was found between\nperformance on the cognitive test and performance on the\nmodule. Although this result is weak, subsequent analysis\nfound that a number of items in the test were highly corre-\nlated with programming performance. We anticipate that a\nredesign of the test could result in more signiﬁcant ﬁndings\nin the future.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\n4.3 Personal information\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nGender, age, work-style preference, encouragement by oth-\ners and part-time employment were treated as dichotomous\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTable 4: Comparison of the mean and standard de-\nviation for overall results (as a percentage) grouped\nby: gender, age, work-style preference, encourage-\nment by others, prior programming (Prog. exp.)\nand non-programming computer experience (Non-\nprog. comp. exp.)\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nGender\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nAge\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nWork-style preference\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nEncouragement\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nPart-time job\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nProg. exp.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nNon-prog. comp. exp.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nn Mean\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\nFemale\nMale\nUnder 23\n23+\nIndividual 12\n\nGroup\n\nNo\n\nYes\n\nNo\n\nYes\n\nNone\n\nSome\n\nNone\n\nSome\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.4**r== 0.6**\n(%)\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.4**r== 0.5**\nS.D.\n(%)\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nvariables for analysis purposes. The possible values of each\nfactor are given in Table 2. Students were grouped accord-\ning to the responses they provided for each of the factors.\nDescriptive statistics for each group are given in Table 4.\nT-tests for independent samples were used to examine the\ndiﬀerences between the overall module results for each of\nthe factor values, for example the mean overall module re-\nsult for male students was compared to the mean overall\nmodule result for female students. Before each t-test was\ncarried out assumptions of normality and equality of vari-\nance were conﬁrmed. In each instance, the t-tests revealed\nno signiﬁcant diﬀerences between any of the factors and the\noverall results on the module. We intend to further examine\nthe relationship between work-style preference and perfor-\nmance, as since the introduction of PBL workshops into the\nmodule mean scores have increased, at the top, middle and\nbottom levels of the class. We feel this is a result of the\nPBL workshops and students repeating the module appear\nto concur with us [16].\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n4.4 Experience on the module\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nComfort level was measured as the cumulative response\nto questions on a student’s understanding of programming\nconcepts, diﬃculty designing programs without help and dif-\nﬁculty for completing lab assignments. Each question had\na number of ranked answers and the cumulative rank was\nused to analyze comfort level. The Pearson correlations are\ngiven in table 3. Comfort level was found to be a statistically\nsigniﬁcant indicator of performance with r = 0.516, p < .01.\nUnderstanding was measured by ranked responses to a\nsingle question ‘How do you rate your level of understand-\ning of the programming module?’ A strong signiﬁcant rela-\ntionship between understanding and performance was found,\nr = 0.76, p < 0.01.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nGiven the earlier ﬁndings on gender diﬀerences between\nprevious academic experience and programming performance,\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\ngender based analysis was carried out on comfort level and\nunderstanding. Comfort level was found to have a higher\ncorrelation with performance for male students.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nLike the Cantwell Wilson and Shrock [4] study comfort\nlevel was found to be highly correlated with programming\nperformance. The most signiﬁcant ﬁnding, however, is the\nvery strong correlation between a students’ perception of\ntheir understanding of the programming module. As this\nstudy was carried out in the second semester we intend to\nconduct a further study to identify the point in time per-\nception of module understanding becomes such a reliable\nindicator. If a similarly high correlation can be found early\non in the module then it would be very powerful in diagnos-\ning and subsequently mediating struggling students.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\n4.5 Regression Analysis\n**BLOCK**fs== 9.0**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nTo investigate whether the various factors studied were\npredictive of performance on the module a number of re-\ngression analyzes were conducted. Each analysis was moti-\nvated by the literature review, the authors’ experience work-\ning with ﬁrst year students and the strength of the corre-\nlation coeﬃcients generated in this study. Although, both\nLC biology and LC physics rendered high correlation coeﬃ-\ncients for programming performance, neither variables were\ndirectly included in the regression models as the sample size\nfor each was deemed too small (n=10, n=18 respectively).\nInstead the highest LC science result was included (n=28).\nThe ﬁrst model was designed to determine the earliest\nindicators of programming performance. Consideration was\ngiven to gender, previous academic experience, cognitive test\nscore, previous programming and non-programming com-\nputer performance, encouragement from others, work-style\npreference and hours working at a part-time job. Using a\nstepwise regression method a signiﬁcant model emerged with\nF (2, 27) = 7.113, p < 0.01 with an adjusted R square =30%.\nSigniﬁcant values were found for: LC maths (β = 0.390, p =\n0.021) and gender (β = −0.368, p = 0.028).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThe second model considered all of the predictors used in\nthe ﬁrst model but also included the results of the ﬁrst class\ntest. Class tests are typically the ﬁrst test given to ﬁrst year\nstudents and although they do not test a student’s ability\nto design and code up a solution to a programming problem\nthey do test a students’ understanding of basic programming\nconcepts. A stepwise regression method found a signiﬁcant\nmodel of F (2, 27) = 14.882, p < 0.001 and adjusted R square\n=49%. Signiﬁcant values were found for the class test (β =\n0.563, p = 0.000) and LC maths (β = 0.375, p = 0.01).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe third model included the predictors from the sec-\nond model but also considers the results of the ﬁrst lab\ntest. The lab test is similar to the ﬁnal examination in\nthat students are required to design and code up a solu-\ntion to a programming problem. Although the lab test may\nbe a better predictor of the overall result, a trade oﬀ takes\nplace in that this information is not available until near the\nend of the ﬁrst Semester and at this stage struggling stu-\ndents may have dropped out or given-up hope of succeed-\ning. A stepwise regression method resulted in a signiﬁcant\nmodel of F (2, 25) = 26.38, p < 0.001 and an adjusted R\nsquare = 65%. Signiﬁcant factors were found for the ﬁrst\nlab test (β = 0.700, p = 0.000) and highest science result\n(β = 0.700, p = 0.000).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe fourth model includes the predictors from the sec-\nond model but takes into account a students’ comfort level\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nwith the module and perceived understanding of how they\nare doing. Using a stepwise regression method a signiﬁcant\nmodel emerged with F (4, 23) = 26.03, p < 0.001, adjusted\nR square = 79%. Signiﬁcant values were found for: under-\nstanding (β = 0.505, p = 0.000), gender (β = −0.494, p =\n0.000), comfort level (β = 0.301, p = 0.022), and LC maths\n(β = 0.197, p = 0.047). If the results of the ﬁrst lab test is\nalso considered 84% of the variance in performance can be\naccounted for with F (4, 23) = 36.92, p < 0.001.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nThe factors known at the start of the academic year re-\nsult in a poor prediction of programming performance. The\nresults of the ﬁrst class test (model 2) and subsequently\nthe ﬁrst lab exam (model 3) results in an improved predic-\ntion ability. However, when a students’ perception of their\nunderstanding of the module is considered a very strong\nprediction model occurs. As with the strength of the Pear-\nson correlation coeﬃcient for this variable a further study\nto determine the stage at which a student’s self-perception\nbecomes so accurate would be valuable.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\n5. CONCLUSIONS\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nThis study examined the relationship and predictive abil-\nity between ﬁfteen factors and performance on a program-\nming module. Comfort level on the module, LC maths and\nLC science scores were shown to have a strong correlation\nwith performance, with notable gender diﬀerences identi-\nﬁed. A predictive combination of factors was found to be a\nstudent’s perception of their understanding of the module,\ncomfort level on the module, LC maths score and gender,\naccounting for 79% of the variance in programming perfor-\nmance.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe study found that the strongest relationship existed\nbetween a student’s perception of their understanding of\nthe module and programming performance. The need to\nunderstand the role of self-perception in the process and to\ninvestigate how early it becomes a reliable predictor war-\nrants further research.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n6. REFERENCES\n[1] S. Bergin and R. Reilly. Discovering the likely\nperformance of novice programmers on an\nintroductory programming module. In Technical\nReport NUIM-CS-TR-2004, Department of Computer\nScience, National University of Ireland, Maynooth,\n2004.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n[2] P. Byrne and G. Lyons. The eﬀect of student\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nattributes on success in programming. In Proceedings\nof the 6th annual conference on Innovation and\ntechnology in Computer Science Education, pages\n49–52, 2001.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n[3] P. F. Campbell and G. P. McCabe. Predicting the\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nsuccess of freshmen in a computer science major.\nCommun. ACM, 27(11):1108–1113, 1984.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n[4] B. Cantwell-Wilson and S. Shrock. Contributing to\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nsuccess in an introductory computer science course: a\nstudy of twelve factors. In Proceedings of the\nthirty-second SIGCSE technical symposium on\nComputer Science Education, pages 184–188, 2001.\n[5] J. Elwood and K. Carlisle. Examining Gender. Gender\nand achievement in the Junior and Leaving Certifcate\nExaminations 2000/2001. National Council for\nCurriculum and Assessment, 2003.\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\n[6] G. E. Evans and M. G. Simkin. What best predicts\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.6**r== 0.2**\ncomputer proﬁciency? Commun. ACM,\n32(11):1322–1327, 1989.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n[7] D. C. Gibbs. The eﬀect of a constructivist learning\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nenvironment for ﬁeld-dependent/independent students\non achievement in introductory computer\nprogramming. In Proceedings of the thirty-ﬁrst\nSIGCSE technical symposium on Computer science\neducation, pages 207–211, 2000.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n[8] A. Goold and R. Rimmer. Factors aﬀecting\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\nperformance in ﬁrst-year computing. SIGCSE Bull.,\n32(2):39–43, 2000.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n[9] D. Hagan and S. Markham. Does it help to have some\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.6**r== 0.1**\nprogramming experience before beginning a\ncomputing degree program? In Proceedings of the 5th\nannual SIGCSE/SIGCUE ITiCSEconference on\nInnovation and technology in computer science\neducation, pages 25–28, 2000.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n[10] E. Holden and E. Weeden. The impact of prior\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\nexperience in an information technology programming\ncourse sequence. In Proceeding of the 4th conference on\nInformation technology curriculum, pages 41–46, 2003.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n[11] T. R. Hostetler. Predicting student success in an\nintroductory programming course. SIGCSE Bull.,\n15(3):40–43, 1983.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\n[12] B. L. Kurtz. Investigating the relationship between the\ndevelopment of abstract reasoning and performance in\nan introductory programming class. In Proceedings of\nthe eleventh SIGCSE technical symposium on\nComputer science education, pages 110–117, 1980.\n[13] R. R. Leeper and J. L. Silver. Predicting success in a\nﬁrst programming course. In Proceedings of the\nthirteenth SIGCSE technical symposium on Computer\nscience education, pages 147–150, 1982.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[14] M. McCracken, V. Almstrum, D. Diaz, M. Guzdial,\nD. Hagan, Y. B.-D. Kolikant, C. Laxer, L. Thomas,\nI. Utting, and T. Wilusz. A multi-national,\nmulti-institutional study of assessment of\nprogramming skills of ﬁrst-year cs students. SIGCSE\nBull., 33(4):125–180, 2001.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[15] R. Nowaczyk. Cognitive skills needed in computer\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.6**r== 0.1**\nprogramming. In Annual Meeting of the Southeastern\nPsychological Association, 1983.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[16] J. O’Kelly, A. Mooney, J. Ghent, P. Gaughran,\nS. Dunne, and S. Bergin. An overview of the\nintegration of problem based learning into an existing\ncomputer science programming module. In Pleasure\nBy Learning, (PBL 2004), 2004.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n[17] N. Rountree, J. Rountree, and A. Robins. Predictors\nof success and failure in a cs1 course. SIGCSE Bull.,\n34(4):121–124, 2002.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n[18] D. L. S. Wiedenbeck and V. Kain. Factors aﬀecting\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\ncourse outcomes in introductory programming. In\nProceedings of the 16th Workshop on Psychology of\nProgramming, PPIG’04, pages 97–109, 2004.\n[19] L. H. Werth. Predicting student performance in a\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.6**r== 0.1**\nbeginning computer science class. In Proceedings of\nthe seventeenth SIGCSE technical symposium on\nComputer science education, pages 138–143, 1986.",
         "extracted",
         "The impact of prior experience in an information technology programming course sequence;Predictors of success and failure in a CS1 course;A multi-national, multi-institutional study of assessment of programming skills of first-year CS students;The effect of student attributes on success in programming;Contributing to success in an introductory computer science course: a study of twelve factors;Does it help to have some programming experience before beginning a computing degree program?;Factors affecting performance in first-year computing;The effect of a constructivist learning environment for field-dependent/independent students on achievement in introductory computer programming;What best predicts computer proficiency?;Predicting student performance in a beginning computer science class;Predicting the success of freshmen in a computer science major;Predicting student success in an introductory programming course;Predicting success in a first programming course;Investigating the relationship between the development of abstract reasoning and performance in an introductory programming class;Factors affecting course outcomes in introductory programming;An Overview of the Integration of Problem Based Learning into an existing Computer Science Programming Module;Discovering the likely performance of novice programmers on an introductory programming module;Differential Performance by gender and achievement in the Certificate Examination Results 2000/2001: Final Report, Dublin: National Council for Curriculum and Assessment;Examining Gender;Cognitive skills needed in computer programming;speciﬁc cognitive skills: as measured by an in-house cognitive test;previous academic and computer experience: as measured by performance in the Irish Leaving Certiﬁcate (LC) examinations in",
         "Programming: factors that influence success",
         "None"
        ],
        [
         "6",
         "0004e56a361394b92b9ac1d15a524ef7ecfe2767",
         "Objective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced monocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and shedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs). Methods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of TF-positive monocytes in HRs (32.0±3.5%) versus LRs (11.2±1.2%; P≤0.05), along with a stronger mean fluorescence intensity of TF signal in HRs (7.1±0.5 arbitrary units [AU]) compared with LRs (5.4±0.4 AU; P≤0.05). The LPS-treated blood of the HR group contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular accumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs versus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of surface TF, whereas MNCs from LRs only had a minor increase in surface TF levels. Conclusions—The higher availability of surface TF antigen on MNCs from HRs and TF-containing microparticles might make these individuals more susceptible to hypercoagulation.",
         "E. Egorina,M. Sovershaev,G. Bjørkøy,F. Gruber,J. Olsen,B. Parhami-Seren,K. Mann,B. Østerud",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.ATV.0000168413.29874.d7",
         "\n**BLOCK**fs== 11.5**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nIntracellular and Surface Distribution of Monocyte\nTissue Factor\nApplication to Intersubject Variability\nElena M. Egorina, Mikhail A. Sovershaev, Geir Bjørkøy, Franz X.E. Gruber, Jan O. Olsen,\nBehnaz Parhami-Seren, Kenneth G. Mann, Bjarne Østerud\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nObjective—The high and low responder phenomenon describes individual differences in lipopolysaccharide (LPS)-induced\nmonocyte tissue factor (TF) activity. We characterized patterns of intracellular accumulation, externalization, and\nshedding of TF in response to LPS in mononuclear cells (MNCs) from high responders (HRs) and low responders (LRs).\nMethods and Results—After 2 hours of LPS stimulation of whole blood, flow cytometry analyses revealed a larger population of\nTF-positive monocytes in HRs (32.03.5%) versus LRs (11.21.2%; P0.05), along with a stronger mean fluorescence intensity\nof TF signal in HRs (7.10.5 arbitrary units [AU]) compared with LRs (5.40.4 AU; P0.05). The LPS-treated blood of the HR\ngroup contained 2-fold more TF-positive microparticles than LRs. In-cell Western assay demonstrated higher intracellular\naccumulation of TF in mononuclear cells (MNCs) from LRs because LPS induced a 3.7-fold increase of total TF levels in LRs\nversus a 1.5-fold increase in HRs. In contrast, in response to LPS stimulation, MNCs from HRs exhibited a 4-fold induction of\nsurface TF, whereas MNCs from LRs only had a minor increase in surface TF levels.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nConclusions—The higher availability of surface TF antigen on MNCs from HRs and TF-containing microparticles might\nmake these individuals more susceptible to hypercoagulation. (Arterioscler Thromb Vasc Biol. 2005;25:1493-1498.)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nKey Words: high and low responder phenomenon  monocyte  tissue factor\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nT ightly controlled exposure of tissue factor (TF) to com-\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nponents of the plasma coagulation cascade is important\nfor maintenance of normal rheological properties of blood.\nFailure to manipulate TF levels available for the initiation of\nblood clotting leads to thrombotic or bleeding disorders in\nhumans. Circulating monocytes are presumably the major cell\ntype that respond to variable stimuli by developing coagulant\nactivity1 through the expression of TF.2\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nOriginally, intersubject variability in developing of mono-\ncyte TF activity was described by Østerud et al.3 By compar-\ning lipopolysaccharide (LPS)-induced monocyte TF activity\nand tumor necrosis factor- (TNF-) production in a whole\nblood system, an up to a 50-fold difference between individ-\nuals was observed.4 This finding was defined as the “high–\nlow responder phenomenon.”4 Also noteworthy, the individ-\nual usually remains a high responder (HR) or low responder\n(LR) for several years.5,6 Later, high intersubject variability in\ncytokine production by LPS-stimulated monocytes was dem-\nonstrated.7 It was also shown that patients with high levels of\nTNF- production were more susceptible to heart transplant\nrejection.8 Monocytes isolated from septic shock patient\nsurvivors revealed higher TNF- production than monocytes\nfrom nonsurvivors.9\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nMany studies have been undertaken to describe the signifi-\ncance of this phenomenon, but so far, no general explanation has\nbeen found. Diverse plasma factors and direct cell interactions\nplay an important role in the development of monocyte TF\nactivity.10 –15 High expression of monocyte TF activity is asso-\nciated with higher risk of acute coronary syndrome.16\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nPlatelets have been suggested to be responsible for\ninducing monocyte TF activity.17 Platelet-rich plasma\ninduced significantly higher TF activity in LPS-stimulated\nmonocytes than platelet-poor plasma.18 Moreover, when\nblood cells without platelets from HRs were mixed with\nplatelet-rich plasma of an LR, LPS-induced TF activity\nwas reduced up to 76% compared with an autologous\nsystem.18 It was shown that granulocytes enhance LPS-\ninduced monocyte TF activity in a platelet-dependent\nreaction involving P-selectin, platelet factor 4, platelet-\nactivating factor, hydroxyl-eicosatetraenoic acid, and\nplatelet-derived growth factor.18 –21\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nHere we report several observations concerning the rela-\ntionships between intracellular- and membrane-located TF\nantigen in resting and LPS-stimulated monocytes in groups of\nHRs and LRs, using fluorescence-activated cell sorter\n(FACS) analysis, fluorescence confocal microscopy, in-cell\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nOriginal received February 24, 2005; final version accepted April 14, 2005.\nFrom the Department of Biochemistry (E.M.E., M.A.S., G.B., J.O.O., B.O.), Institute of Medical Biology, University of Tromsø, Norway; the\nDepartment of Immunology and Transfusion Medicine (F.X.E.G.), University Hospital of Northern Norway, Tromsø; and the Department of Biochemistry\n(B.P.-S., K.G.M.), University of Vermont, Burlington.\nE.M.E. and M.A.S. contributed equally to this work.\nCorrespondence to Elena M. Egorina, MD, Department of Biochemistry, Institute of Medical Biology, University of Tromsø, N-9037, Tromsø,\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nNorway. E-mail egorina@fagmed.uit.no\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\n© 2005 American Heart Association, Inc.\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nArterioscler Thromb Vasc Biol. is available at http://www.atvbaha.org\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.7**r== 0.1**\nDOI: 10.1161/01.ATV.0000168413.29874.d7\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nWestern assay, and immunoblotting. In response to LPS\nstimulation, we clearly found different patterns of intracellu-\nlar accumulation and externalization of TF onto the surface of\nMNCs isolated from whole blood of HRs and LRs.\n**BLOCK**fs== 11.5**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\nMaterials and Methods\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nBlood Sampling and Experimental Design\nBlood samples from 16 healthy volunteers (HRs n8; LRs n8)\nwere obtained under protocols approved by the regional committee\nfor medical research ethics. In time-course ex vivo experiments,\nLPS-stimulated and -nonstimulated blood aliquots were incubated\nfor different times with agitation at 37°C. For TF activity assay and\nWestern blotting MNCs were isolated using Lymphoprep (Axis-\nShield) according to manufacturer instructions. For TNF- quanti-\nfication, platelet-poor plasma was obtained from blood aliquots by\ncentrifugation for 10 minutes. For a detailed description, please see\nthe online supplement, available at http://atvb.ahajournals.org.\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nQuantification of Monocyte TF Activity\nTF was measured in the in frozen/thawed preparations of MNCs\nusing a 2-stage clotting assay based on the ability of TF to accelerate\nthe activation of factor X by factor VIIa as described previously.19\nQuantification of TNF-\nThe concentration of TNF- in platelet-poor plasma was determined by\nusing PeliKine compact ELISA kit according to manufacturer instructions.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nAntibody Quality Control and Small\nInterfering RNA\nTo ascertain the suitability of the monoclonal anti-human TF antibody used\nin the present study, we used TF gene silencing by RNA interference. For\na detailed description, please see the online supplement.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFlow Cytometry\nPopulation of monocytes expressing TF (CD14TF cells) was deter-\nmined by staining of whole blood aliquots treated as described in\nexperimental design, using phycoerythrin (PE)-conjugated anti-human\nCD14 monoclonal antibodies (BD Biosciences PharMingen) and\nAlexa488-conjugated mouse anti-human TF monoclonal antibodies.\nMouse monoclonal antibodies recognizing extracellular domain of\nhuman TF were prepared by the monoclonal antibody facility, Depart-\nment of Biochemistry, University of Vermont, Burlington. For a\ndetailed description of the procedure, please see the online supplement.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nFluorescence Confocal Microscopy\nAfter Lymphoprep isolation, MNCs were plated for 30 minutes onto\nflat-bottomed 8-well chambers and fixed. Part of the samples was\npermeabilized with 70% methanol. TF and monocytes were visual-\nized by immunostaining with Alexa488-conjugated mouse anti-\nhuman TF monoclonal antibodies and PE-conjugated mouse anti-\nhuman CD14 monoclonal antibodies. Nuclei of the cells were stained\nusing DRAQ5 dye (Biostatus, Ltd). A fluorescent laser scanning\nmicroscope (LSM510 META; Carl Zeiss AG) with C-Apochromat\n40/1.2 W lens was used for image acquisition.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nQuantitative Immunofluorescence Staining (In-Cell\nWestern Assay)\nWe used in-cell Western assay to characterize total and surface MNC\nTF (MNC-TF) levels. This assay allows for the comparison of\nantigen of interest levels in the cellular context in plated fixed cells.\nMonocytes were prepared as for confocal microscopy as described\nabove. After 4% paraformaldehyde fixation, methanol permeabiliza-\ntion (if applicable), and blocking with 3% goat serum-PBS, the\nsignal from mouse anti-human TF monoclonal antibodies was\ndetected with IRDye800CW-conjugated goat anti-mouse polyclonal\nantibodies (1.25 g/mL; Rockland, Inc.). In RNA interference,\nsignal\nfrom rabbit anti-human GAPDH polyclonal antibodies\n(Sigma-Aldrich) was detected by Alexa680-conjugated goat anti-\nrabbit polyclonal antibodies (1 g/mL; Molecular Probes, Inc). After\nwashing, images of Alexa680 and IRDye800 CW fluorescence were\nobtained on 700 and 800 nm channels of an Odyssey infrared imager\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n(LI-COR Biosciences GmbH). For statistical analyses, integrated\nintensities of fluorescence in wells were analyzed using software\nprovided with the imager station.\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nWestern Blotting\nTF protein levels in lysates of resting and LPS-stimulated 106 MNCs\nwere detected using 10% SDS-PAGE, and immunoblotting of nitrocel-\nlulose membranes with mouse anti-human TF monoclonal antibodies\nand horseradish peroxidase– conjugated secondary antibodies. We used\ndensitometrical reading of 43-kDa immunopositive band for statistical\ncomparisons. For details, please the online supplement.\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nRT-PCR and Real-Time Polymerase\nChain Reaction\nTotal RNA was extracted from MNC pellets with an RNeasy kit\n(Qiagen). Thereafter, single-stranded cDNA was synthesized using\nTaqMan Gold RT-PCR Kit, and real-time PCR analyses of TF\nmRNA expression was performed in duplicates with assay-on-\ndemand real-time PCR kit using ABI PRISM 7700 Sequence\nDetection System (Applied Biosystems). For data normalization, we\nanalyzed the expression of 18S rRNA by using an assay-on-demand\nkit. For details, please see the online supplement.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nCell Count and Viability\nCell counts were performed on a Sysmex K1000 (TOA Medical\nElectronics Co Ltd) in a whole blood system and after isolation of\nMNCs. Cell viability was assessed by using a trypan blue dye exclusion\nassay according to manufacturer instructions (Invitrogen Corp).\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nStatistics\nStatistical comparisons were performed using SigmaPlot 8.0 (SPPS,\nLtd.) and MS Excel. The presented data shown are meanSEM.\nComparisons between mean values were performed using the Stu-\ndent paired t test. P value 0.05 was considered significant.\n**BLOCK**fs== 11.5**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\nResults\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nReactivity of MNCs in HRs and LRs\nThe LPS-induced responses of MNCs from individuals enrolled\nin our study have been repeatedly assessed to determine their\nindividual TF activity and concentration levels of TNF-. These\nhealthy individuals were allocated to HR or LR groups accord-\ning to criteria developed in our laboratory.16 Briefly, healthy\nvolunteers were defined as HRs if their MNC-TF activity\nexceeded 20 mU/106 cells and their plasma TNF- concentra-\ntion levels were 3 ng/mL taken from whole blood stimulated\nwith LPS for 2 hours. We assigned individuals with these\nparameters lower than indicated above to the LR group.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nMeasurements of TF procoagulant activity in lysates of\nMNCs isolated from ex vivo LPS-stimulated whole blood\nrevealed 2 distinct responses. In HRs, LPS stimulation\ninduced significantly higher values of MNC-TF activity than\nin LRs after 2 hours (50.114.3 versus 20.94.2 mU/106\ncells; P0.05) and 4 hours (56.016.3 mU/106 versus\n21.24.9 mU/106 cells; P0.05; Figure 1). We found no\nincrease in TF activity in MNC lysates from time-matched\nnonstimulated samples (Figure 1).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nBecause the HR and LR phenomenon encompasses not only\ndifferences in monocyte TF activity but also in cytokine produc-\ntion,4 we studied the reactivity of whole blood to LPS stimula-\ntion with respect to TNF- production. Analyses of TNF-\nconcentration in plasma obtained from LPS-stimulated whole\nblood showed that in HRs, the levels of TNF-were constantly\nhigher during the experimental time course (3255.6108.7\nversus 2308.8264.6 pg/mL, P 0.05 at 2 hours; and\n4551.0275.4 versus 2977.0496.5 pg/mL at 4 hours, P0.05;\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nFigure 1. Higher LPS-induced TF activity in MNCs from HRs.\nMNCs obtained from LPS-stimulated blood of HRs demon-\nstrated signiﬁcantly higher TF activity when compared with the\nLR group. Data are presented as meanSEM; *P0.05 com-\npared with time-matched LPS-stimulated samples from the LR\ngroup. h indicates hours.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nFigure 2). Levels of TNF- were negligible on baseline and in\nnonstimulated time-matched samples (Figure 2).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nThe differences of MNC-TF activity and the production of\nTNF- in our experimental model could not be caused by\nvariations in degree of cell death because the number of dead\ncells measured by trypan blue exclusion assay did not differ\nbetween groups and was within 3% of the total cell count.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe variable degrees of responsiveness to LPS treatment,\ncharacterized by significantly elevated TF activity in MNC\nlysates in HRs,\nthe comparisons of\nintracellular and surface TF antigen levels under resting and\nstimulated conditions in these individuals.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\nled us to carry out\n**BLOCK**fs== 9.5**b== 0.0**t== 0.6**l== 0.1**r== 0.5**\nIntracellular and Surface Distribution of\nTF Antigen\nAfter we characterized the suitability of available antibodies\nfor detection of denatured TF antigen, immunoblots of MNC\nlysates using mouse anti-human TF monoclonal antibodies\nrevealed a strong immunopositive 43-kDa band and several\nweaker bands that migrated between 50 and 60 kDa, which\nare most likely secondary antibodies reactive with mouse\nimmunoglobulin heavy chain. Mature TF is a protein con-\ntaining 263 amino acids with predicted molecular weight 33\nkDa. However, it is known to be post-translationally modified\nby glycosylation and phosphorylation, which retards its\nelectrophoretic mobility. To identify the TF band on the\nimmunoblots, we depleted TF expression by RNA interfer-\nence. Immunoblots of extracts from human cervical adeno-\ncarcinoma cell line (HeLa; ATCC No. CCL-2) cells trans-\nfected with anti-TF small interfering RNAs (siRNA) revealed\nthat the strong 43-kDa band was significantly reduced com-\npared with HeLa cells transfected with control siRNA. At the\nsame time, we found no reduction in the intensities of the\nweak 50 to 60 kDa bands. The intensity of the signal from\nimmunopositive bands of an irrelevant protein epidermal\ngrowth factor receptor displayed no differences between\nthese samples (Figure IA, available online at http://atvb.\nahajournals.org). This ascertains that monoclonal antibodies\nrevealed TF as 43-kDa band.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nFigure 2. Higher LPS-stimulated TNF- production in whole\nblood of HRs. The HR group revealed signiﬁcantly higher TNF-\nlevels in plasma of LPS-stimulated whole blood aliquots than\nobserved in the LR group. Data are presented as meanSEM;\n*P0.05 compared with time-matched LPS-stimulated samples\nfrom the LR group. h indicates hours.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nBecause we planned to use immunostaining-based methods for\nnative TF antigen detection (such as fluorescence-activated cell\nsorting, fluorescence confocal microscopy and in-cell Western\nassay), we applied RNA interference against TF expression in\nplated HeLa cells. Similarly, the same antibodies detected a 3-fold\nreduction of the in-cell Western assay signal intensity in the wells\nthat contained HeLa cells transfected with siRNA against TF when\ncompared with cells transfected with control siRNA. The signal\nfrom the in-cell Western assay against GAPDH showed no differ-\nences in the same wells (Figure IB).\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nTo characterize levels of TF antigen on the surface and\ninterior of resting and LPS-stimulated monocytes from HRs\nand LRs, we chose FACS analyses of whole blood, fluores-\ncence confocal microscopy, and in-cell Western assay of\npermeabilized and nonpermeabilized MNCs.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.6**l== 0.5**r== 0.1**\nData of FACS analyses revealed striking differences in\nsurface TF antigen levels in LPS-stimulated monocytes of\nHRs and LRs. The population of CD14 cells, expressing TF\non their surface, was significantly lower in LRs (11.21.2%)\nthan in HRs (32.03.5%) after 2 hours of LPS stimulation\n(P0.05; Figure 3A and 3B). The mean fluorescence inten-\nsity (MFI) was also significantly lower in the LRs (5.40.4\narbitrary units [AU]) than in the HRs (7.10.5 AU; P0.05)\nat 2 hours of LPS stimulation (Figure 3C). More rapid\nexternalization of TF antigen on the surface of monocytes\nwas already seen in the HR group after 1 hour of LPS\nstimulation, but these differences did not reach statistical signif-\nicance between groups. Time-matched nonstimulated samples\nhad no significant differences in surface TF expression between\n2 groups and the baseline values. Observed differences in the\ndynamics of TF surface expression on monocytes forced us to\nre-evaluate our flow cytometry data and challenge the frequency\nof TF-containing microparticles. TF-containing microparticles\nwere determined by gating all events, which were smaller in size\nthan platelets, using forward scatter against TF-Alexa488 chan-\nnel plot. Indeed, we found nearly a 2-fold LPS-induced increase\nin the frequency of TF events in samples from HRs and only a\nminor increase in LRs (Figure 3D).\n**BLOCK**fs== 8.0**b== 0.5**t== 0.2**l== 0.7**r== 0.1**\nFigure 3. More monocytes bear surface\nTF in HRs. Nonpermeabilized monocytes\nwere gated by using side scatter channel\n(SSC) vs CD14 –PE channel. The events\nin the monocyte gate were sent to histo-\ngram, where CD14 –PE channel was\nplotted against the TF–Alexa488 channel.\nThe histogram of double-positive cells,\ndepicted in the top right quadrants in A,\ndemonstrates differences between repre-\nsentative HRs and LRs in resting and\nLPS-stimulated conditions. The dynam-\nics of changes in TF surface expression\nas a percentage of double-positive\nmonocytes (B) and the MFI of TF–Al-\nexa488 signal in CD14-positive cells (C)\nin groups of HRs and LRs are presented\nas meanSEM. D shows plotted mean\nfrequencies of TF-containing micropar-\nticles in LPS-stimulated (LPS; 2 hours\n[h]) and time-matched nonstimulated\n(CTR; 2 hours) samples, expressed as\npercentage of baseline (NS); *P0.05\ncompared with time-matched LPS-\nstimulated samples from the LR group.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nWe found no baseline difference in the surface expression\nof TF on nonstimulated CD14 cells from LRs (1.40.1%)\nand HRs (1.50.2%; Figure 3A and 3B). MFI did not differ\nbetween groups and was 4.50.3 AU and 4.20.1 AU in\nLRs and HRs, respectively (Figure 3C).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nBy use of fluorescence confocal microscopy of isolated\nMNCs in resting and LPS-stimulated conditions, we visual-\nized the distribution of TF antigen between cytoplasmic and\nmembrane compartments. Only a few CD14 cells expressed\nTF in permeabilized and nonpermeabilized conditions (Fig-\nure 4A and 4B). Furthermore, we noticed 2 phenotypes when\nimaging nonpermeabilized resting CD14 cells: 1 did not\nexpress TF on the membrane surface, whereas another did\n(Figure 4C and 4D). The pattern of TF surface distribution could\nbe characterized by the presence of 5 to 7 TF-enriched dots or\n“patches.” When permeabilized, resting CD14 cells have TF-\ncontaining speckles in the proximity of the plasma membrane,\nready to be exocytosed to the cell exterior (Figure 4E).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nAfter 2 hours of LPS stimulation, permeabilized mono-\ncytes had their membranes enriched with TF antigen. The\ncytoplasm was depleted from TF-containing granules except\nfor a strong TF-positive staining unilaterally from the nucleus\n(Figure 4F). This could be newly synthesized TF protein.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nTo verify TF antigen distribution data from flow cytometry\nand fluorescence confocal microscopy, we used a quantitative\nimmunofluorescence staining in-cell Western assay.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nThe relationship between intracellular and membrane lo-\ncated TF antigen in resting and LPS-stimulated MNCs is\ndemonstrated in Figure 5A. Surprisingly, when comparing\nintensities of total TF signal in LPS-stimulated MNCs with\ncells from time-matched nonstimulated samples, we found\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nsignificantly higher LPS-induced accumulation of intracellu-\nlar TF antigen in LRs compared with HRs (Figure 5B). At the\nsame time, LPS treatment caused a 4-fold increase of surface\nTF signal intensity in HRs. However, no such increase was\nfound in the surface TF signal of LRs (Figure 5B).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nData from the immunoblotting of MNC lysates against TF\nantigen confirmed that after 2 hours of LPS stimulation,\nMNCs of LRs indeed accumulated TF antigen, whereas in\nHRs,\nthis accumulation was minor (Figure 5C and 5D).\nMNCs from time-matched samples of LRs experienced vir-\ntually no accumulation of the TF antigen in contrast to the\nsignificant accumulation of TF in MNCs of HRs (Figure 5D).\nComparison of real-time PCR data on de novo TF synthesis in\nLPS-stimulated conditions revealed similar mean cycle threshold\nvalues in MNCs of HRs and LRs (Table I, available online at\nhttp://atvb.ahajournals.org). Therefore, LPS stimulation is not only\nnecessary for the induction of TF synthesis, but also for its transport\nvia exocytotic pathway and shedding from the cell membrane.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nDiscussion\nIn the present ex vivo study, using a whole blood system, we\nreport several novel observations characterizing the distribu-\ntion of TF antigen between the interior and surface of MNCs\nfrom HRs and LRs. MNCs from representatives of these\ngroups exhibit distinct patterns of intracellular TF accumula-\ntion, externalization onto cell membrane, and shedding of\nTF-containing microparticles in LPS-treated conditions.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nFACS analyses using double staining against CD14 and TF\nrevealed higher LPS-induced surface expression of TF on\nmonocytes of individuals from the HR group compared with\nLRs. Our data imply that in HRs, a higher frequency of\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFigure 4. Visualization of TF antigen on resting and LPS-\nstimulated monocytes. Nonpermeabilized and permeabilized\nisolated resting monocytes (CD14 –PE; red) visualized only few\ncells positive for TF antigen (TF–Alexa488; green) in representa-\ntive ﬁelds (A and B, white arrowheads). Image of nonpermeabil-\nized resting monocyte demonstrates no surface expression of\nTF (C). Image depicts patches-like pattern of TF surface distri-\nbution on nonpermeabilized resting monocyte (D). Characteristic\nintracellular TF pattern in permeabilized resting monocyte is\ndemonstrated in E. Image demonstrates the surface redistribu-\ntion of TF in permeabilized LPS-stimulated monocyte; nucleus is\nstained with Draq5 dye (purple; F).\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nCD14 cells expose TF on the cell surface after LPS-\nstimulation when compared with LRs.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nData of FACS analyses were in accordance with our\nfindings from fluorescence confocal microscopy, in which we\ndemonstrated redistribution of TF antigen on stimulation with\nLPS onto the cell surface. In stimulated conditions, TF was\ntransported and localized mostly on the membrane because\nthere was a clear depletion of TF staining from the cytoplasm.\nIntense dots present unilaterally from the nucleus could\nrepresent de novo synthesized TF.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nImmunoblotting showed LPS-induced TF antigen accumula-\ntion in the lysates of MNCs from LRs, whereas in MNCs from\nHR individuals, this accumulation was seen in a minor extent.\nAn in-cell Western assay also confirms this finding. Signifi-\ncantly higher total TF antigen accumulation was found in\nLPS-treated MNCs of LRs without exposure of TF antigen to the\ncell membrane. We demonstrate that LPS-stimulated monocytes\nof HRs synthesize, quickly externalize, and release TF antigen,\nwhereas monocytes of LRs retain TF inside and are slower to\nexternalize. HRs and LRs had similar LPS-induced TF synthe-\nsis, as demonstrated by data from relative quantification of TF\nmRNA by real-time PCR. High levels of TF in time-matched\nnonstimulated samples of HRs could be explained by synthesis\nof TF without externalization and “loss” via shedding. Altered\nreactivity of LR monocytes triggered weaker TF synthesis in\n**BLOCK**fs== 8.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nFigure 5. Induction of surface expression in HRs and intracellu-\nlar accumulation in LRs of TF antigen in LPS-stimulated MNCs.\nRepresentative images of the in-cell Western assay (A) and plots\nof LPS-induced fold induction of TF signal (B) depict a signiﬁ-\ncantly higher degree of total TF antigen accumulation in MNCs\nfrom LRs. Contrary, the LPS-induced increase of surface TF\nantigen was more prominent in MNCs from HRs. Note that the\ngreen color was artiﬁcially generated by software to illustrate TF\nimmunostaining image acquired via the infrared dye. Fold induction\nwas calculated by normalizing the TF signal intensities to non-\nstimulated time-matched samples (CTR; 2 hours [h]). The represen-\ntative Western blot demonstrates the total TF antigen in lysates of\nMNCs from HRs and LRs in resting conditions (NS), after 2 hours\nof LPS stimulation (LPS; 2 hours), and nonstimulated time-\nmatched samples (CTR; 2 hours; C). Densitometrical readings of\nTF immunopositive band intensities (n8 in each group), normal-\nized to resting conditions, are presented as meanSEM (D).\n*P0.05 compared with LPS-stimulated sample from the LR\ngroup. MWM indicates molecular weight marker.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\ntime-matched nonstimulated cells. Together, it is possible that\nmonocytes from HRs could synthesize equal or even higher\namounts of TF antigen than those from LRs.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.5**r== 0.1**\nIn both groups, TF was detected in only 1.4% to 1.5% of\ncells of the total CD14 population under resting conditions.\nThe amount of surface TF, localized per individual monocyte,\ndid not differ between groups, as shown by the MFI data.\nAlthough it has been reported that the population of mono-\ncytes expressing TF at the baseline conditions is small,11,22\nour study for the first\ntime characterizes a fraction of\nTF-expressing resting CD14 cells. These data are in accor-\ndance with the recently published results of Butenas et al,\ndemonstrating 0.2% resting MNCs as being TF positive.23\nMinor differences in the fraction of TF-positive cells between\ndata of Butenas et al and from our study reflect regular\nvariations related to FACS methodology, and also that we\nhave analyzed cells from the population of CD14 cells,\nwhereas Butenas et al presented values of TF cells, gated\nfrom the total MNC population.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nImmunostaining against TF and fluorescent confocal mi-\ncroscopy of permeabilized and nonpermeabilized resting\nmonocytes support our data from flow cytometry because\nonly few monocytes expressed TF antigen. The localization\npattern of TF on the monocyte surface in the present study is\nconfined to dotted structures or “patches.” Intracellular local-\nized TF in CD14 cells was restricted to round vesicle-like\nstructures in the proximity of the plasma membrane. Mulder\net al demonstrated the presence of an intracellular TF pool\nassociated with caveolae and multivesicular bodies.24\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nOur data obtained from FACS and immunoblotting are\nconsistent with the previous accepted opinion that resting human\nmonocytes contain nearly undetectable levels of TF antigen.25–30\nHowever, we provide new information about the relationship\nbetween the intracellular and surface pool of TF in MNCs by\nusing in-cell Western assay. This enzyme-independent method\nof antigen detection via antibodies conjugated with infrared dyes\ngives highly reproducible data characterizing antigen levels.\nComparisons of the TF signal from resting MNCs between\npermeabilized and nonpermeabilized conditions revealed that in\nresting conditions, MNCs contain equally distributed surface\nand intracellular TF antigen in HRs and LRs.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nIn conclusion, MNCs of HRs and LRs contain similar, nearly\nundetectable intracellular levels of TF antigen at baseline condi-\ntions. On LPS stimulation, TF is apparently transported faster to the\ncell surface and in a greater amount in the group of HRs. This group\nalso shed more of TF-containing microparticles. The cells of LRs\nhad a lower ability to be stimulated by LPS and retained more TF\nantigen, probably because of a slower exocytosis rate. Furthermore,\nLR monocytes express less TF antigen on their surface and shed\nfewer TF-containing microparticles. The findings of our present\nstudy indicate that elevated levels of the blood coagulation cascade\ninitiator on the surface of circulating monocytes and microparticles\nin HRs might make them more susceptible to hypercoagulation.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nAcknowledgments\nThis work was supported by program project grant HL46703 from\nthe National Institutes of Health (K.G.M.). We thank Dr Terje\nJohansen for sharing his knowledge and necessary reagents, Aud\nØvervatn for invaluable technical expertise, Drs Peter McCourt and\nPaul Johe for their help in preparation of this manuscript, and Serhiy\nPankiv for fruitful scientific discussions.\nReferences\n1. Conkling PR, Greenberg CS, Weinberg JB. Tumor necrosis factor\ninduces tissue factor-like activity in human leukemia cell line U937 and\nperipheral blood monocytes. Blood. 1988;72:128 –133.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n2. Edwards RL, Rickles FR, Bobrove AM. Mononuclear cell tissue factor:\ncell of origin and requirements for activation. Blood. 1979;54:359 –370.\n3. Østerud B, Bjorklid E. The production and availability of tissue throm-\nboplastin in cellular populations of whole blood exposed to various\nconcentrations of endotoxin. An assay for detection of endotoxin. Scand\nJ Haematol. 1982;29:175–184.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n4. Østerud B. The high responder phenomenon: enhancement of LPS\ninduced tissue factor activity in monocytes by platelets and granulocytes.\nPlatelets. 1995;6:119 –125.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n5. Østerud B, Olsen JO, Wilsgard L. The role of arachidonic acid release and\nlipoxygenase pathway in lipopolysaccharide-induced thromboplastin\nactivity in monocytes. Blood Coagul Fibrinolysis. 1990;1:41– 46.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n6. Ernofsson M, Siegbahn A. Platelet-derived growth factor-BB and\nmonocyte chemotactic protein-1 induce human peripheral blood\nmonocytes to express tissue factor. Thromb Res. 1996;83:307–320.\n7. Endres S, Cannon JG, Ghorbani R, Dempsey RA, Sisson SD, Lonnemann\nG, Van der Meer JW, Wolff SM, Dinarello CA. In vitro production of IL\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of\ncyclooxygenase inhibition and evidence of independent gene regulation.\nEur J Immunol. 1989;19:2327–2333.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n8. Turner D, Grant SC, Yonan N, Sheldon S, Dyer PA, Sinnott PJ,\ntransplant\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nHutchinson IV. Cytokine gene polymorphism and heart\nrejection. Transplantation. 1997;64:776 –779.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n9. Munoz C, Carlet J, Fitting C, Misset B, Bleriot JP, Cavaillon JM.\nDysregulation of in vitro cytokine production by monocytes during\nsepsis. J Clin Invest. 1991;88:1747–1754.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n10. Nijziel M, van Oerle R, van ’t Veer C, van Pampus E, Lindhout T,\nHamulyak K. Tissue factor activity in human monocytes is regulated by\nplasma: implications for the high and low responder phenomenon. Br J\nHaematol. 2001;112:98 –104.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n11. Lindmark E, Tenno T, Siegbahn A. Role of platelet P-selectin and CD40\nligand in the induction of monocytic tissue factor expression. Arterioscler\nThromb Vasc Biol. 2000;20:2322–2328.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n12. Evangelista V, Piccardoni P, White JG, de Gaetano G, Cerletti C.\nCathepsin G-dependent platelet stimulation by activated poly-\nmorphonuclear leukocytes and its inhibition by antiproteinases: role of\nP-selectin-mediated cell-cell adhesion. Blood. 1993;81:2947–2957.\n13. Del Prete G, De Carli M, Lammel RM, D’Elios MM, Daniel KC, Giusti\nB, Abbate R, Romagnani S. Th1 and Th2 T-helper cells exert opposite\nregulatory effects on procoagulant activity and tissue factor production by\nhuman monocytes. Blood. 1995;86:250 –257.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n14. Carlsen E, Flatmark A, Prydz H. Cytokine-induced procoagulant activity\nin monocytes and endothelial cells. Further enhancement by cyclosporine.\nTransplantation. 1988;46:575–580.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n15. Maclouf J, Folco G, Patrono C. Eicosanoids and iso-eicosanoids: consti-\ninducible and transcellular biosynthesis in vascular disease.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\ntutive,\nThromb Haemost. 1998;79:691–705.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n16. Østerud B, Elvevoll EO, Brox J, Olsen JO. Cellular activation responses in blood\nin relation to lipid pattern: healthy men and women in families with myocardial\ninfarction or cancer. Blood Coagul Fibrinolysis. 2002;13:399–405.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n17. Niemetz J, Marcus AJ. The stimulatory effect of platelets and platelet\nmembranes on the procoagulant activity of leukocytes. J Clin Invest.\n1974;54:1437–1443.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n18. Halvorsen H, Olsen JO, Østerud B. Granulocytes enhance LPS-induced\ntissue factor activity in monocytes via an interaction with platelets.\nJ Leukoc Biol. 1993;54:275–282.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n19. Østerud B. Platelet activating factor enhancement of lipopolysaccharide-\ninduced tissue factor activity in monocytes: requirement of platelets and\ngranulocytes. J Leukoc Biol. 1992;51:462– 465.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n20. Engstad CS, Lia K, Rekdal O, Olsen JO, Østerud B. A novel biological\neffect of platelet factor 4 (PF4): enhancement of LPS-induced tissue\nfactor activity in monocytes. J Leukoc Biol. 1995;58:575–581.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n21. Lorenzet R, Niemetz J, Marcus AJ, Broekman MJ. Enhancement of\nmononuclear procoagulant activity by platelet 12-hydroxyeicosatetraeno-\nic acid. J Clin Invest. 1986;78:418 – 423.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n22. Ungerstedt JS, Heimersson K, Soderstrom T, Hansson M. Nicotinamide\ninhibits endotoxin-induced monocyte tissue factor expression. J Thromb\nHaemost. 2003;1:2554 –2560.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n23. Butenas S, Bouchard BA, Brummel-Ziedins KE, Parhami-Seren B, Mann\nKG. Tissue factor activity in whole blood. Blood. 2005;105:2764 –2770.\n24. Mulder AB, Smit JW, Bom VJ, Blom NR, Ruiters MH, Halie MR, van\nder Meer J. Association of smooth muscle cell tissue factor with caveolae.\nBlood. 1996;88:1306 –1313.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n25. Niemetz J, Herbert V. The role of protein synthesis on the generation of tissue\nfactor activity by leukocytes. Proc Soc Exp Biol Med. 1972;139:1276–1279.\n26. Drake TA, Ruf W, Morrissey JH, Edgington TS. Functional tissue factor\nis entirely cell surface expressed on lipopolysaccharide-stimulated human\nblood monocytes and a constitutively tissue factor-producing neoplastic\ncell line. J Cell Biol. 1989;109:389 –395.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n27. Drake TA, Morrissey JH, Edgington TS. Selective cellular expression of\ntissue factor in human tissues. Implications for disorders of hemostasis\nand thrombosis. Am J Pathol. 1989;134:1087–1097.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n28. Carson SD. Manifestation of cryptic fibroblast tissue factor occurs at\ndetergent concentrations which dissolve the plasma membrane. Blood\nCoagul Fibrinolysis. 1996;7:303–313.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n29. Wilcox JN, Smith KM, Schwartz SM, Gordon D. Localization of tissue\nfactor in the normal vessel wall and in the atherosclerotic plaque. Proc\nNatl Acad Sci U S A. 1989;86:2839 –2843.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n30. Le DT, Rapaport SI, Rao LV. Relations between factor VIIa binding and\nexpression of factor VIIa/tissue factor catalytic activity on cell surfaces.\nJ Biol Chem. 1992;267:15447–15454.",
         "extracted",
         "The production and availability of tissue thromboplastin in cellular populations of whole blood exposed to various concentrations of endotoxin. An assay for detection of endotoxin.;Tissue factor activity in whole blood.;Nicotinamide inhibits endotoxin‐induced monocyte tissue factor expression;Cellular activation responses in blood in relation to lipid pattern: healthy men and women in families with myocardial infarction or cancer;Tissue factor activity in human monocytes is regulated by plasma: implications for the high and low responder phenomenon;Role of Platelet P-Selectin and CD40 Ligand in the Induction of Monocytic Tissue Factor Expression;Eicosanoids and Iso-Eicosanoids: Constitutive, Inducible and Transcellular Biosynthesis in Vascular Disease;Cytokine gene polymorphism and heart transplant rejection.;Platelet-derived growth factor-BB and monocyte chemotactic protein-1 induce human peripheral blood monocytes to express tissue factor.;Association of smooth muscle cell tissue factor with caveolae.;Manifestation of cryptic fibroblast tissue factor occurs at detergent concentrations which dissolve the plasma membrane;A novel biological effect of platelet factor 4 (PF4): enhancement of LPS‐induced tissue factor activity in monocytes;Th1 and Th2 T-helper cells exert opposite regulatory effects on procoagulant activity and tissue factor production by human monocytes.;Granulocytes enhance LPS‐induced tissue factor activity in monocytes via an interaction with platelets;Cathepsin G-dependent platelet stimulation by activated polymorphonuclear leukocytes and its inhibition by antiproteinases: role of P-selectin-mediated cell-cell adhesion.;Relations between factor VIIa binding and expression of factor VIIa/tissue factor catalytic activity on cell surfaces.;Platelet activating factor enhancement of lipopolysaccharide‐induced tissue factor activity in monocytes: requirement of platelets and granulocytes;Dysregulation of in vitro cytokine production by monocytes during sepsis.;The role of arachidonic acid release and lipoxygenase pathway in lipopolysaccharide-induced thromboplastin activity in monocytes.;In vitro production of IL 1β, IL 1α, TNF and IL 2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation;Functional tissue factor is entirely cell surface expressed on lipopolysaccharide-stimulated human blood monocytes and a constitutively tissue factor-producing neoplastic cell line;Localization of tissue factor in the normal vessel wall and in the atherosclerotic plaque.;Cytokine-induced procoagulant activity in monocytes and endothelial cells. Further enhancement by cyclosporine.;Tumor necrosis factor induces tissue factor-like activity in human leukemia cell line U937 and peripheral blood monocytes.;Enhancement of mononuclear procoagulant activity by platelet 12-hydroxyeicosatetraenoic acid.;The stimulatory effect of platelets and platelet membranes on the procoagulant activity of leukocytes.;The Role of Protein Synthesis on the Generation of Tissue Factor Activity by Leukocytes;Monocytes.;The high responder phenomenon: enhancement of LPS induced tissue factor activity in monocytes by platelets and granulocytes.;Selective cellular expression of tissue factor in human tissues. Implications for disorders of hemostasis and thrombosis.;In vitro production of IL 1 beta, IL 1 alpha, TNF and IL2 in healthy subjects: distribution, effect of cyclooxygenase inhibition and evidence of independent gene regulation.;Mononuclear cell tissue factor: cell of origin and requirements for activation.;After blocking with 5% non-fat skimmed milk, membranes were incubated with mouse anti-human TF monoclonal antibodies;After Lymphoprep isolation, total RNA was extracted from MNC pellets with an;OR) anti-human TF monoclonal antibodies (10 µg/ml) for 30 min on ice. Mouse monoclonal antibodies recognizing extracellular domain of human TF were prepared by the Monoclonal Antibody Facility",
         "Intracellular and Surface Distribution of Monocyte Tissue Factor: Application to Intersubject Variability",
         "None"
        ],
        [
         "7",
         "000646db4a4a269f9261f5b285ba1d07ef006809",
         "ABSTRACT The erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin clinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-susceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of methicillin-resistant S. aureus strains were D-test positive.",
         "T. Levin,B. Suh,P. Axelrod,A. Truant,T. Fekete",
         "https://europepmc.org/articles/pmc549252?pdf=render",
         "\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Mar. 2005, p. 1222–1224\n0066-4804/05/$08.000 doi:10.1128/AAC.49.3.1222–1224.2005\nCopyright © 2005, American Society for Microbiology. All Rights Reserved.\n**BLOCK**fs== 16.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nPotential Clindamycin Resistance in Clindamycin-Susceptible,\nErythromycin-Resistant Staphylococcus aureus: Report of\na Clinical Failure\nTodd P. Levin,* Byungse Suh, Peter Axelrod, Allan L. Truant, and Thomas Fekete\nTemple University School of Medicine, Philadelphia, Pennsylvania\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nReceived 25 June 2004/Returned for modiﬁcation 3 October 2004/Accepted 4 November 2004\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.2**\nThe erm gene product confers clindamycin resistance on Staphylococcus aureus. We report a clindamycin\nclinical failure where resistance developed on therapy in a D-test-positive strain. D tests of 91 clindamycin-\nsusceptible, erythromycin-resistant S. aureus isolates showed that 68% of methicillin-susceptible and 12.3% of\nmethicillin-resistant S. aureus strains were D-test positive.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nClindamycin-susceptible, erythromycin-resistant Staphylo-\ncoccus aureus (clindamycin-erythromycin discordant) may de-\nvelop clindamycin resistance (7, 8). The erm gene product is a\nribosome methylase whose expression is normally minimal.\nErythromycin induces the production of this methylase, which\nis why these strains are erythromycin resistant, but mutations\nin the promoter region of erm allow production of methylase\nwithout an inducer (18, 19). These mutants are stably erythro-\nmycin and clindamycin resistant. Since erythromycin resistance\ncan occur with other mechanisms (e.g., efﬂux pumps and en-\nzymatic modiﬁcation) (15), the D-test identiﬁes inducible re-\nsistance that might presage mutational clindamycin constitu-\ntive resistance. The D-test is performed by placing clindamycin\nand erythromycin disks at an edge-to-edge distance of 15 to 20\nmm and looking for ﬂattening of the clindamycin zone nearest\nthe erythromycin disk (2). A positive D-test suggests the pres-\nence of an erm gene that could result in constitutive clindamy-\ncin resistance and clinical failure.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nThere are few published clinical failures of clindamycin with\nemergence of resistance (1, 3, 9, 12, 16). However, there are\nalso reports of successful use of clindamycin in treating pa-\ntients with D-test-positive isolates (3, 9). We report a clinical\nfailure with documented emergence of resistance. In order to\navoid poor clinical outcomes but retain the usefulness of clin-\ndamycin, it would be helpful to know the prevalence of induc-\nible resistance in clindamycin-erythromycin discordant bacte-\nria. This prevalence varies by geographic location, patient age,\nbacterial species, and bacterial susceptibility proﬁle (4, 5, 10,\n11, 13, 14, 16). For example, a pediatric population in Houston,\nTex., with methicillin-resistant S. aureus (MRSA) had a D-test\npositivity rate of 2.2% (9) compared to a similar population of\nchildren with MRSA in Chicago, Ill., whose D-test positivity\nrate was 94% (3). We determined the prevalence of D-test\npositivity in clindamycin-erythromycin discordant S. aureus iso-\nlates in our institution in early 2004. We tested isolates from\ninfected body sites where clindamycin might be considered for\ntherapy.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n* Corresponding author. Mailing address: Temple University Hos-\npital, Section of Infectious Diseases, 3401 North Broad St., Philadel-\nphia, PA 19140. Phone: (215) 707-3807. Fax: (215) 707-4414. E-mail:\nTLevin9696@yahoo.com.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.4**l== 0.5**r== 0.1**\nThe patient was a 44-year-old man with fever, chills, and\nincreased low back pain 10 days after a lumbar discectomy. At\nadmission he had a temperature of 102.4°F and complained of\n3 days of erythema, bloody drainage, and pain along his inci-\nsion. He had a wound exploration, and the tissue Gram stain\nshowed white blood cells and gram-positive cocci. Fluid and\nblood cultures grew clindamycin-erythromycin discordant me-\nthicillin-susceptible S. aureus (MSSA), which was treated with\n2 weeks of oxacillin followed by 2 weeks of oral clindamycin.\nTen days after completing clindamycin therapy, he had severe\nback pain, chills, a temperature of 101.6°F, and purulent drain-\nage from his wound. A magnetic resonance image showed new\nevidence of lumbar osteomyelitis, discitis, and enhancement of\nparavertebral musculature with a small ﬂuid collection. On\nvancomycin, he had surgical debridement. Bone, blood, and\nwound cultures all grew MSSA with the same antibiogram as\nthat of his prior isolate except for new clindamycin resistance.\nHe successfully completed therapy with intravenous (oxacillin)\nand oral (combinations of minocycline, trimethoprim-sulfame-\nthoxazole, and rifampin) antibiotics for a total of 5 months.\nThe D-test on the original ﬂuid isolate was positive (Fig. 1);\nhowever, this result was not available at the time of his original\ntherapy. The original S. aureus wound isolate and the blood\nand bone isolates collected after completion of primary ther-\napy were indistinguishable by ribotyping with EcoRI restriction\nenzyme (Christiana Care Health Services, Wilmington, Del.).\nWe collected all 168 S. aureus isolates from wounds, ab-\nscesses, bone, cerebrospinal ﬂuid, pleural ﬂuid, ascitic ﬂuid,\nand joint ﬂuid from 1 January to 28 February 2004 at Temple\nUniversity Hospital (Philadelphia, Pa.). The strains (adult, pe-\ndiatric, inpatient, and outpatient) were stored in glycerol at\n74°C until batch testing in April 2004. Antibiotic susceptibil-\nities were determined by the Vitek-1 test (bioMerieux, Hazel-\nwood, Mo.); erythromycin-intermediate strains were considered\nresistant. The D-test was performed on clindamycin-erythromycin\ndiscordant strains by preparing an 0.5 McFarland suspension of\nbacteria in tryptic soy broth (2) inoculated on Mueller-Hinton\nplates incubated at 35°C for 18 to 24 h with erythromycin (15 g)\nand clindamycin (2 g) disks placed 17 mm apart (edge to edge).\nControls with known D-test positivity and negativity were also\ntested. Results were photographed and visually inspected by two\ninvestigators.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.2**\nFIG. 1. D-test positivity on the original wound culture isolate from the case patient. E, erythromycin; CL, clindamycin.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nNinety-one (54.2%) of 168 isolates were clindamycin-eryth-\nromycin discordant. There was 100% concordance of the D-\ntest results between the investigators. The majority (72.2%) of\nthe discordant strains were MRSA; however, only 8 (12.3%) of\n65 strains were D-test positive (Fig. 2). Twenty-ﬁve (27.5%) of\nthe 91 discordant strains were MSSA, but 17 (68%) of the 25\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nstrains were D-test positive. Overall, 25 (27.5%) of the 91\ndiscordant strains were D-test positive.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nThe mechanisms of clindamycin resistance have been stud-\nied for years, and the D-test was developed to identify poten-\ntial clindamycin resistance so that possible ineffective therapy\nis not started when conventional tests show clindamycin MICs\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nFIG. 2. Summary of all clindamycin-erythromycin discordant isolates.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nwithin the susceptible range (0.5 g/ml). Molecular markers\nfor the erm gene are available, but they are costly and incon-\nvenient for everyday use (8, 15, 17). The D-test is easy to\nperform and interpret, reproducible, and inexpensive but still\nnot universally used. There is a good correlation of standard\nD-tests with a modiﬁcation applied to purity plates done at the\ntime of specimen preparation for the Vitek-2 test (6).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nClindamycin is frequently used to treat skin and bone infec-\ntions because of its tolerability, cost, oral form, and good tissue\npenetration. With this high prevalence of D-test positivity, why\nare clindamycin clinical failures seldom reported? It may take\ntime for a mutant strain to develop, and the immune system\nmay have already controlled the infection. New agents active\nagainst gram-positive bacteria have been developed, so clinda-\nmycin may now be used less often. Finally, although a D-test-\npositive isolate may mutate on therapy, the mutation rate in\nclinical infections is unknown and may be rare.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nWe studied the prevalence of D-test positivity in isolates that\nmight be treated with clindamycin. We documented a clinda-\nmycin failure when resistance developed in a D-test-positive\nMSSA isolate. S. aureus clindamycin susceptibility reporting\nshould be suppressed when erythromycin resistance is found\n(unless a D-test is negative). If D-testing is delayed until re-\nsistance testing is complete, the results may not be available for\nmaximal clinical utility. Alternatively, the clinician may have to\nrequest it speciﬁcally.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nIt may be risky to use clindamycin when erythromycin testing\nshows a resistant or intermediate phenotype. The risk might\ncurrently be lower with MRSA, but this disparity may not\ncontinue or not be true elsewhere. Routine D-testing might\nallow clinicians to retain conﬁdence in clindamycin when eryth-\nromycin resistance is present.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n1. Drinkovic, D., E. R. Fuller, K. P. Shore, D. J. Holland, and R. Ellis-Pegler.\n2001. Clindamycin treatment of Staphylococcus aureus expressing inducible\nclindamycin resistance. J. Antimicrob. Chemother. 48:315–316.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n2. Fiebelkorn, K. R., S. A. Crawford, M. L. McElmeel, and J. H. Jorgensen.\n2003. Practical disk diffusion method for detection of inducible clindamycin\nresistance in Staphylococcus aureus and coagulase-negative staphylococci.\nJ. Clin. Microbiol. 41:4740–4744.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n3. Frank, A. I., J. F. Marcinak, P. D. Mangat, J. T. Tjhio, S. Kelkar, P. C.\nSchreckenberger, and J. P. Quinn. 2002. Clindamycin treatment of methi-\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\ncillin-resistant Staphylococcus aureus infections in children. Pediatr. Infect.\nDis. J. 21:530–534.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n4. Hamilton-Miller, J. M. T., and S. Shah. 2000. Patterns of phenotypic resis-\ntance to the macrolide-lincosamide-ketolide-streptogramin group of antibi-\notics in staphylococci. J. Antimicrob. Chemother. 46:941–949.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n5. Jennssen, W. D., S. Thakker-Varia, D. T. Dubin, and M. P. Weinstein. 1987.\nPrevalence of macrolides-lincosamides-streptogramin B resistance and erm\ngene classes among clinical strains of staphylococci and streptococci. Anti-\nmicrob. Agents Chemother. 31:883–888.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n6. Jorgensen, J. H., S. A. Crawford, M. L. McElmeel, and K. R. Fiebelkorn.\n2004. Detection of inducible clindamycin resistance of staphylococci in con-\njunction with performance of automated broth susceptibility testing. J. Clin.\nMicrobiol. 42:1800–1802.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n7. Leclercq, R., and P. Courvalin. 1991. Bacterial resistance to macrolide,\nlincosamide, and streptogramin antibiotics by target modiﬁcation. Antimi-\ncrob. Agents Chemother. 35:1267–1272.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n8. Leclercq, R. 2002. Mechanisms of resistance to macrolides and lincosamides:\nnature of the resistance elements and their clinical implications. Clin. Infect.\nDis. 34:482–492.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n9. Martines-Aquilar, G., W. A. Hammerman, E. O. Mason, Jr., and S. L.\nKaplan. 2003. Clindamycin treatment of invasive infections caused by com-\nmunity-acquired, methicillin-resistant and methicillin-susceptible Staphylo-\ncoccus aureus in children. Pediatr. Infect. Dis. J. 22:593–598.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n10. Nicola, F. G., L. K. McDougal, J. W. Biddle, and F. C. Tenover. 1998.\nCharacterization of erythromycin-resistant isolates of Staphylococcus aureus\nrecovered in the United States from 1958 through 1969. Antimicrob. Agents\nChemother. 42:3024–3027.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n11. Panagea, S., J. D. Perry, and F. K. Gould. 1999. Should clindamycin be used\nas treatment of patients with infections caused by erythromycin-resistant\nstaphylococci? J. Antimicrob. Chemother. 44:577–582.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n12. Rao, G. G. 2000. Should clindamycin be used in treatment of patients with\ninfections caused by erythromycin-resistant staphylococci? J. Antimicrob.\nChemother. 45:715–716.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n13. Sanchez, M. L., K. K. Flint, and R. N. Jones. 1993. Occurrence of macrolide-\nlincosamide-streptogramin resistance among staphylococcal clinical isolates\nat a university medical center. Diagn. Microbiol. Infect. Dis. 16:205–213.\n14. Sattler, C. A., E. O. Mason, Jr., and S. L. Kaplan. 2002. Prospective com-\nparison of risk factors and demographic and clinical characteristics of com-\nmunity-acquired, methicillin-resistant versus methicillin-susceptible Staphy-\nlococcus aureus infection in children. Pediatr. Infect. Dis. J. 21:910–916.\n15. Schmitz, F.-J., J. Petridou, A. C. Fluit, U. Hadding, G. Peters, and C. von\nEiff. 2000. Distribution of macrolide-resistant genes in Staphylococcus aureus\nblood-culture isolates from ﬁfteen German university hospitals. Eur. J. Clin.\nMicrobiol. Infect. Dis. 19:385–387.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n16. Siberry, G. K., T. Tekle, K. Carroll, and J. Dick. 2003. Failure of clindamycin\ntreatment of methicillin-resistant Staphylococcus aureus expressing inducible\nclindamycin resistance in vitro. Clin. Infect. Dis. 37:1257–1260.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n17. Sutcliffe, J., T. Grebe, A. Tait-Kamradt, and L. Wondrack. 1996. Detection\nof erythromycin-resistant determinants by PCR. Antimicrob. Agents Che-\nmother. 40:2562–2566.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n18. Weisblum, B. 1995. Insights into erythromycin action from studies of its\nactivity as inducer of resistance. Antimicrob. Agents Chemother. 39:797–805.\n19. Weisblum, B., and V. Demohn. 1969. Erythromycin-inducible resistance in\nStaphylococcus aureus: survey of antibiotic classes involved. J. Bacteriol.\n98:447–452.",
         "extracted",
         "",
         "Potential Clindamycin Resistance in Clindamycin-Susceptible, Erythromycin-Resistant Staphylococcus aureus: Report of a Clinical Failure",
         "None"
        ],
        [
         "8",
         "0008826f79633b9d5738551ac67593d6456acd43",
         "We propose a constructive approach for distributed source coding of correlated binary sources using linear channel codes that can achieve any point of the Slepian-Wolf achievable rate region. Our approach is very intuitive and can be used with systematic and nonsystematic linear codes. Moreover, the proposed coding strategy can easily be extended to the case of more than two sources.",
         "N. Gehrig,P. Dragotti",
         "http://www.commsp.ee.ic.ac.uk/~ng1/pdf/gehrigD_CL_Jan05.pdf",
         "\n**BLOCK**fs== 23.9**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nSymmetric and A-Symmetric Slepian-Wolf Codes\nwith Systematic and Non-Systematic Linear Codes\n**BLOCK**fs== 11.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nNicolas Gehrig, Student Member, IEEE, and Pier Luigi Dragotti, Member, IEEE\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.0**r== 0.5**\nAbstract— We propose a constructive approach for distributed\nsource coding of correlated binary sources using linear channel\ncodes that can achieve any point of the Slepian-Wolf achievable\nrate region. Our approach is very intuitive and can be used\nwith systematic and non-systematic linear codes. Moreover, the\nproposed coding strategy can easily be extended to the case of\nmore than two sources.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nIndex Terms— Distributed source coding, Slepian-Wolf theo-\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.0**r== 0.6**\nrem, symmetric encoding, linear channel codes.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nI. INTRODUCTION\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.0**r== 0.5**\nT HE Slepian-Wolf theorem [1] states that separate encod-\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.0**r== 0.5**\ning of the outputs of two correlated sources can be as\nefﬁcient as joint encoding, assuming that the two compressed\nsignals can be jointly decoded. The achievable rate region for\nsuch a system is given by: RX ≥ H(X|Y ), RY ≥ H(Y |X)\nand RX + RY ≥ H(X, Y ).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.0**r== 0.5**\nAlthough this theoretical result has been known for about\nthree decades, it is only recently that practical coding ap-\nproaches have been proposed. In [2], a ﬁrst coding technique\nusing channel coding principles was introduced. Practical\ndesigns mainly based on Turbo and LDPC codes have since\nbeen presented in several other papers (see [3], [4], [5] for\nexample). Most of these approaches focus on the asymmetric\nscenario, where one of the two sources is transmitted perfectly\nto the receiver.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.0**r== 0.5**\nFor practical applications, it might be necessary to have\nmore ﬂexibility in the repartition of the bit-rates between\nthe encoders. In [6], Pradhan and Ramchandran proposed\na technique based on their original work (DISCUS [2]) in\norder to achieve any point of the Slepian-Wolf achievable rate\nregion. Their method creates two sub-codes of a single channel\ncode by splitting the original generator matrix in two. Each\nencoder uses then one of these sub-codes to encode its data.\nIn this letter, we propose a constructive approach that allows\nfor a ﬂexible repartition of the transmission rates between\nthe encoders. Our technique uses a single linear channel code\nthat can be non-systematic. The performance of our approach\ndepends only on the quality of the channel code used. Actually,\nthe two correlated sources can be seen respectively as the\ninput and the output of a certain channel used to model their\ncorrelation. We refer to this virtual channel as the correlation\nchannel of the two sources. If we can ﬁnd a code that achieve\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.0**r== 0.5**\nManuscript received April 8, 2004. The associate editor coordinating the\nreview of this letter and approving it for publication was Prof. Chi-chao Chao.\nThe authors are with the Communications and Signal Processing Group,\nElectrical and Electronic Engineering Department, Imperial College London,\nUK (e-mail: {nicolas.gehrig, p.dragotti}@imperial.ac.uk).\nDigital Object Identiﬁer 10.1109/LCOMM.2005.01035.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nthe capacity of this correlation channel, then our distributed\nsource coding approach can reach the Slepian-Wolf bound and\nis therefore optimal.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nNotice that similar approaches have recently been proposed\nin [7], [8] and [9]. Although our own approach is relatively\nsimilar in spirit to the one in [7], our scheme can also be\nused with non-systematic codes, whereas their technique can\nonly be used with systematic codes. Note that good capacity\nachieving LDPC codes are usually non-systematic. In [8]\nand [9], iterative decoding procedures are proposed in order\nto decode the two correlated blocks simultaneously. Their\nmain strategy is to apply the standard sum-product algorithm\n(message-passing decoding [10]) on an extended factor graph,\ncorresponding to two standard LDPC decoders connected\nthrough correlation nodes modeling the joint distribution be-\ntween the sources. In Section III, we show that our approach\ndoes not require the use of such extended graphs since our\nmethods only needs to decode one single block (the difference\npattern). A standard iterative decoding scheme similar to the\none proposed in [5] can therefore be used in our case. Our\napproach can thus be seen as an intuitive extension of the\nasymmetric approach proposed in [5] in order to achieve the\nentire Slepian-Wolf achievable rate region.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nII. A SIMPLE EXAMPLE WITH THE HAMMING (7, 4) CODE\nWe consider the Hamming (7, 4) code C whose parity check\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nmatrix is given by:\n⎛\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nH =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n⎝\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n⎞\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n⎠ .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n(1)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nWe know that a codeword x belongs to the Hamming code C\nif and only if its syndrome is equal to zero:\nsx = HxT = 0 ⇐⇒ x ∈ C.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.9**r== 0.1**\n(2)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThe minimum distance between any two of the 16 codewords\nof the Hamming code is three. This code is therefore able to\ncorrect up to one bit error per codeword. Assume ei is the\nerror pattern corresponding to an error at bit position i (ei has\nsix 0s and one 1 at position i). We deﬁne y = x ⊕ ei where\n⊕ corresponds to the binary addition. This codeword y does\nclearly not belong to C since its distance from x is equal to\none. Its syndrome is given by:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\nsy = HyT = H(x ⊕ ei)T = HxT ⊕ HeT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\ni = HeT\ni .\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\n(3)\n**BLOCK**fs== 10.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nthe syndrome of an erroneous\nWe can therefore see that\ncodeword does not depend on the original codeword but only\non the error pattern. This means that if we change the ith bit\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.4**r== 0.4**\n1089-7798/05$20.00 c 2005 IEEE\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.0**r== 0.5**\nof all the codewords of C, this produces 16 new codewords, all\nhaving syndrome HeT\ni . This new set of codewords is called\ncoset number i and has the same properties as C (coset 0),\nthat is, the minimum distance between any two codewords is\nstill three. All the 27 possible 7-bit blocks are thus distributed\nin 8 distinct cosets. Notice that this Hamming (7, 4) code has\na particular structure such that the syndrome of an erroneous\ncodeword gives the binary representation of the error position,\nor similarly, the coset number.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.0**r== 0.5**\nConsider now two discrete memoryless uniformly dis-\ntributed 7-bit binary random variables x and y, correlated such\nthat their Hamming distance is at most one (dH (x, y) ≤ 1).\nAssume that x and y belong to cosets i and j respectively.\nThe difference between x and y is given by the error pattern\nek = x ⊕ y (x and y differs at position k). We know that the\nsyndromes of x and y are given by sx = HxT = HeT\ni and\nsy = HyT = HeT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nj respectively. We can see that:\nk = H(x ⊕ y)T = HxT ⊕ HyT = sx ⊕ sy.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nsk = HeT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n(4)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.0**r== 0.5**\nThis result shows that knowing only the syndromes of x and\ny, we can retrieve the syndrome of their difference pattern and\ntherefore, the bit position where they differ.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.0**r== 0.5**\nOur coding technique can now be presented as follows:\nAssume the following block representations for x, y and H:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nx = [xa xb]\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\ny = [ya\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nyb]\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nH = [Ha Hb]\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n(5)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.0**r== 0.5**\nwhere the ﬁrst and the second blocks are of length 4 and 3\nrespectively. The syndromes of x and y are computed at their\nrespective encoders as: sx = HxT = HaxT\nb and\na ⊕ HbyT\nsy = HyT = HayT\nb . Encoder 1 transmits sx together\nwith a subset of xa. Encoder 2 transmits sy together with the\nsubset of ya which is complementary to the one chosen by the\nﬁrst encoder. For example, as presented in Fig. 1, the encoder\n1 could send: [x1 x2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nsT\nx ] and encoder 2: [y3\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\na ⊕ HbxT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nsT\ny ].\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\ny4\n**BLOCK**fs== 12.6**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\nxa\n**BLOCK**fs== 10.8**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nEncoder 1\nEncoder 2\n**BLOCK**fs== 12.6**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nx1\ny1\n**BLOCK**fs== 12.6**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nx2\ny2\n**BLOCK**fs== 12.6**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nx3\ny3\n**BLOCK**fs== 12.6**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nx4\ny4\n**BLOCK**fs== 12.6**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nya\n**BLOCK**fs== 12.6**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nsx\n**BLOCK**fs== 12.6**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nsy\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.0**r== 0.5**\nFig. 1. Example of distributed source coding of two correlated 7-bit blocks.\nOnly the gray squares are transmitted.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nAt\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\na ).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nthe decoder,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nb = H−1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nb (sx ⊕ HaxT\n**BLOCK**fs== 10.0**b== 0.0**t== 0.8**l== 0.0**r== 0.5**\nthe syndrome of the difference pattern\nbetween x and y is obtained by computing the sum of the two\nsyndromes sx ⊕ sy. Using the corresponding error pattern, the\nmissing bits of xa and ya can easily be retrieved. Finally,\nxb and yb are obtained as: xT\na ) and\nb = H−1\nb (sy ⊕ HayT\nyT\nSince x and y are uniformly distributed, we have: H(x) =\nH(y) = 7 bits. We know that y can take 8 different equiprob-\nable values for a speciﬁc x. Hence, H(y|x) = H(x|y) = 3\nbits. The joint entropy of x and y is therefore equal\nto\nH(x, y) = H(x) + H(y|x) = 10 bits. Our coding scheme\nuses 6 bits to send the two syndromes and a total of 4 bits\nto send the two complementary subsets of xa and ya and is\ntherefore optimal.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.6**r== 0.2**\nIII. CONSTRUCTIVE APPROACH\nUSING ANY LINEAR CHANNEL CODE\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nAssume we have an (n, k) binary linear code C with parity\ncheck matrix H in its reduced form such that: H = [H1 H2],\nwhere H1 and H2 are of size (n − k × k) and (n − k × n − k)\nrespectively. Assume without loss of generality that H2 is non\nsingular. Notice that if the code is systematic, we know that\nthe generator matrix G is of the form: G = [Ik P], where P\nis of size (k×n−k). The parity check matrix H has to satisfy:\nGHT = 0 and can therefore be given by: H = [−PT\nIn−k].\nH2 is thus simply the identity matrix in that case.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nAssume C is able to correct up to M errors per n-bit code\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nblock. We know that the following relation must hold:\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\n2n−k ≥\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\nM\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nj=0\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\nn\nj\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.1**\n(sphere packing bound).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n(6)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nThis code C generates 2n−k cosets each containing 2k code-\nwords of length n. We know that x belongs to the coset i,\nsuch that sx = HxT = HeT\n(ei is the coset leader of coset\ni\nnumber i, i.e., the codeword with minimum weight).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nbi\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nLet xi be a binary block of length n represented as: xi =\nqi], where ai, bi and qi are of length k1, k2 and n−k\n[ai\nrespectively (k1 and k2 are chosen such that their sum is equal\nbi]T ⊕\nto k). The syndrome of xi is deﬁned as: si = H1[ai\nH2qT\ni .\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nConsider now two n-bit blocks x1 and x2, correlated such\nthat their Hamming distance dH (x1, x2) is at most m (Assume\nthat M ≥ m). Our distributed coding strategy consists in\nsT\n2 ] from the encoders 1 and\nsending only [a1\n2 respectively. The transmission bit-rates are therefore given\nby: R1 = n − k2 bits and R2 = n − k1 bits, corresponding to\na total of 2n − k bits.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nsT\n1 ] and [b2\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.8**r== 0.1**\n1 ⊕ xT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.8**r== 0.2**\nd = H(xT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nAt the receiver, we let ed correspond to the “difference\npattern” between x1 and x2 as: ed = x1 ⊕ x2. We know that\nthe syndrome of ed is given by sd = HeT\n2 ) =\ns1⊕s2. We can now retrieve the error pattern ed corresponding\nto this syndrome sd using one of the following techniques:\nIf the code is not too large, a simple lookup table storing\nthe corresponding pattern error for each possible syndrome\ncan be used. For larger code, an iterative method has to be\nused. Using an iterative decoding scheme such as the one\nproposed in [5], we can recover ed as the closest codeword\nto the all zero sequence satisfying the syndrome sd. Notice\nthat this iterative decoding approach is particularly suited for\nLDPC codes which are amongst the best block codes known\nfor memoryless channels [10].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nb2] ⊕ ek\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\nd, where ek\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nb1]T ⊕ H2qT\nb1]T . We can now retrieve q1 by computing: qT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nk ﬁrst bits of x1 and x2 are easily obtained as: [a2\n[a1\n**BLOCK**fs== 10.0**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nKnowing the difference pattern ed, the missing bits of the\nb1] =\nd corresponds to the k ﬁrst bits of ed.\nWe know that the syndrome of x1 corresponds to: s1 =\n1 . Let z1 be deﬁned as: z1 = s1 ⊕\nH1[a1\n1 =\nH1[a1\nH−1\ncan be obtained using Gaussian\nelimination and that, if C is systematic, we can choose H\nsuch that H2 = I and q1 = zT\n1 . The inversion of H2 is\nactually done only once (off-line) and does not introduce extra\ncomplexity to the decoding phase. This inversion is the one\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\n2 z1. Notice that H−1\n**BLOCK**fs== 6.5**b== 0.8**t== 0.1**l== 0.1**r== 0.9**\nX1\nX2\nX3\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nXL\n**BLOCK**fs== 9.8**b== 0.7**t== 0.1**l== 0.3**r== 0.7**\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n**BLOCK**fs== 9.8**b== 0.7**t== 0.1**l== 0.4**r== 0.6**\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n. . .\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nsyndromes (n-k bits)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.0**r== 0.5**\nFig. 2. Our encoding strategy for L correlated binary sources. Each encoder\nsends the syndrome and a subset of the ﬁrst k bits of their input block.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.0**r== 0.5**\nthat is usually done in order to compute the generator matrix\nfrom a parity check matrix.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.0**r== 0.5**\nKnowing q1, we have now completely recovered x1 and we\ncan easily obtain x2 as x2 = x1 ⊕ ed. We can summarize our\ncoding approach as follows:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.0**r== 0.5**\nProposition 1: Assume X and Y are two binary sequences\nof length n, correlated such that their Hamming distance is\nat most m. Consider an (n, k) linear channel code C that can\ncorrect up to M ≥ m errors per n-bit block. The following\ndistributed coding strategy uses a total of 2n−k bits to encode\nthe two sequences and is sufﬁcient to allow for a perfect\nreconstruction of them at the decoder:\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nProposition 2: Assume x1, . . . , xL are L binary sequences\nlength n correlated such that\nthe Hamming distance\nof\nbetween two consecutive sequences is at most m (i.e.,\ndH (xi, xi+1) ≤ m for i = 1, . . . , L − 1). Consider an (n, k)\nlinear channel code C that can correct up to M ≥ m errors per\nn-bit block. The following distributed coding strategy uses a\ntotal of n + (L − 1)(n − k) bits to encode the L sequences\nand is sufﬁcient to allow for a perfect reconstruction of all of\nthem at the decoder:\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n From each encoder, send the syndrome si of the corre-\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nsponding block xi.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n Send only complementary subsets of their ﬁrst k bits such\nthat each bit position is sent from only one encoder.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nAt\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nthe decoder,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nthe L − 1 difference patterns can be\nrecovered from the L syndromes, allowing then to complete\nthe ﬁrst k bits of each sequence. The decoding method used\nhere is similar to the one presented in the previous section.\nThe difference pattern of each pair of consecutive blocks is\nretrieved by running the standard iterative decoding method\nwith the sum of the two syndromes. Knowing all the difference\npatterns and having received complementary subsets of the\nﬁrst k bits, the ﬁrst k bits of each block can then be recovered.\nFinally, each original block xi := [ai\nqi] (i = 1, . . . , L) is\ncompleted by recovering its last n − k bits as:\ni = H−1\nqT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n2 (si ⊕ H1aT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\ni ).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n(7)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n Send the syndromes of X and Y from their respective\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nIt is possible to show that this coding strategy is optimal\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nencoders.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.0**r== 0.5**\n Send only complementary subsets of their ﬁrst k bits.\nIn terms of performance, we can say that the ability of\nour distributed source coding technique to work close to the\nSlepian-Wolf bound only depends on the quality of the channel\nif X and Y are uniformly\ncode used. More speciﬁcally,\ndistributed and p(Y |X) is the transition probability, then the\ncloser the channel code C gets to the capacity of the binary\nchannel p(Y |X), the closer our system gets to the Slepian-\nWolf bound. The design of capacity achieving channel codes,\nhowever, is beyond the scope of this paper.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.0**r== 0.5**\nSince our decoding strategy can use an iterative decoding\napproach similar to the one proposed in [5], similar perfor-\nmances in terms of decoding error probability are expected\nif the same linear code is used by both systems. Our work\ncan thus be seen as an intuitive extension of the work in [5]\nin order to cover the full Slepian-Wolf achievable rate region\nwithout any performance loss.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.0**r== 0.5**\nWe have run some simulations and we have obtained numer-\nical evidence that our approach presents similar performances\nthan the one proposed in [5].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.0**r== 0.5**\nIV. GENERALIZATION TO MORE THAN TWO SOURCES\nThe approach of the previous section can be extended to\nany number of correlated sources (see Fig. 2), as indicated in\nthe following proposition:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nfor some particular cases.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[1] D. Slepian and J. K. Wolf, “Noiseless coding of correlated information\nsources,” IEEE Trans. Inform. Theory, vol. 19, pp. 471-480, July 1973.\n[2] S. S. Pradhan and K. Ramchandran, “Distributed source coding using\nsyndromes (DISCUS): Design and construction,” in IEEE International\nConference on Data Compression (DCC), 1999, pp. 158-167.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.2**\ncodes,” IEEE Commun. Lett., vol. 5, pp. 417-419, Oct. 2001.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[4] A. Aaron and B. Girod, “Compression with side information using turbo\ncodes,” in IEEE International Conference on Data Compression (DCC),\nApr. 2002, pp. 252-261.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[5] A. D. Liveris, Z. Xiong, and C. N. Georghiades, “Compression of binary\nsources with side information at the decoder using LDPC codes,” IEEE\nCommun. Lett., vol. 6, pp. 440-442, Oct. 2002.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n[6] S. S. Pradhan and K. Ramchandran, “Distributed source coding: Sym-\nmetric rates and applications to sensor networks,” in IEEE International\nConference on Data Compression (DCC), 2000, pp. 363-372.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n[7] V. Stankovic et al., “Design of Slepian-Wolf codes by channel code\npartitioning,” in IEEE International Conference on Data Compression\n(DCC), Mar. 2004, pp. 302-311.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n[8] D. Schonberg, S. S. Pradhan, and K. Ramchandran, “Distributed code\nconstructions for the entire Slepian-Wolf rate region for arbitrarily corre-\nlated sources,” in IEEE International Conference on Data Compression\n(DCC), Mar. 2004, pp. 292-301.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n[9] T. P. Coleman et al., “On some new approaches to practical Slepian-\nWolf compression inspired by channel coding,” in IEEE International\nConference on Data Compression (DCC), Mar. 2004, pp. 282-291.\n[10] T. J. Richardson and R. L. Urbanke, “The capacity of low-density parity-\ncheck codes under message-passing decoding,” IEEE Trans. Inform.\nTheory, vol. 47, pp. 599-618, Feb. 2001.",
         "extracted",
         "Design of Slepian-Wolf codes by channel code partitioning;On some new approaches to practical Slepian-Wolf compression inspired by channel coding;Distributed code constructions for the entire Slepian-Wolf rate region for arbitrarily correlated sources;Compression of binary sources with side information at the decoder using LDPC codes;Compression with side information using turbo codes;Compression of correlated binary sources using turbo codes;The capacity of low-density parity-check codes under message-passing decoding;Distributed source coding: symmetric rates and applications to sensor networks;Distributed source coding using syndromes (DISCUS): design and construction;Noiseless coding of correlated information sources",
         "Symmetric and asymmetric Slepian-Wolf codes with systematic and nonsystematic linear codes",
         "None"
        ],
        [
         "9",
         "000bc721cefe3f2ba2425e8b95b042dc1e9778f9",
         "In this work we use Machine Learning (ML) techniques to learn the CPU time-slice utilization behavior of known programs in a Linux system. Learning is done by an analysis of certain static and dynamic attributes of the processes while they are being run. Our objective was to discover the most important static and dynamic attributes of the processes that can help best in prediction of CPU burst times which minimize the process TaT (Turn-around-Time). In our experimentation we modify the Linux Kernel scheduler (version 2.4.20-8) to allow scheduling with customized time slices. The \"Waikato Environment for Knowledge Analysis\" (Weka), an open source machine-learning tool is used to find the most suitable ML method to characterize our programs. We experimentally find that the C'4.5 Decision Tree algorithm most effectively solved the problem. We find that predictive scheduling could reduce TaT in the range of 1.4% to 5.8%. This was due to a reduction in the number of context switches needed to complete the process execution. We find our result interesting in the context that generally operating systems presently never make use of a program's previous execution history in their scheduling behavior.",
         "A. Negi,K. P. Kishore",
         "http://www.cs.ucr.edu/%7Ekishore/papers/tencon.pdf",
         "\n**BLOCK**fs== 23.9**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nApplying Machine Learning Techniques to improve\nLinux Process Scheduling\n**BLOCK**fs== 11.0**b== 0.7**t== 0.2**l== 0.3**r== 0.3**\nAtul Negi, Senior Member, IEEE, Kishore Kumar P.\nDepartment of Computer and Information Sciences\nUniversity of Hyderabad\nHyderabad, INDIA 500046\natulcs@uohyd.ernet.in, kishoregupta os@yahoo.com\n**BLOCK**fs== 9.0**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\nAbstract In this work we use Machine Learning (ML) tech-\nniques to learn the CPU time-slice utilization behavior of known\nprograms in a Linux system. Learning is done by an analysis of\ncertain static and dynamic attributes of the processes while they\nare being run. Our objective was to discover the most important\nstatic and dynamic attributes of the processes that can help best\nin prediction of CPU burst times which minimize the process\nTaT (Turn-around-Time). In our experimentation we modify the\nLinux Kernel scheduler (version 2.4.20-8) to allow scheduling\nwith customized time slices. The Waikato Environment for\nKnowledge Analysis (Weka), an open source machine-learning\ntool is used to nd the most suitable ML method to characterize\nour programs. We experimentally nd that the C4.5 Decision\nTree algorithm most effectively solved the problem. We nd that\npredictive scheduling could reduce TaT in the range of 1:4%\nto 5:8%. This was due to a reduction in the number of context\nswitches needed to complete the process execution. We nd our\nresult interesting in the context that generally operating systems\npresently never make use of a program’s previous execution\nhistory in their scheduling behavior.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nIndex Terms Machine learning, Process Scheduling, turn-\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\naround-time.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nI. INTRODUCTION\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nG ENERALLY process schedulers allocate CPU time\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nslices to a process according to a scheduling algorithm\nthat does not use any previous execution history of the\nprocess. It might be possible to better utilize resources if\nwe could recognize a program and predict\nits resource\nrequirements. For example, consider a process which was\nallocated a xed CPU time is pre-empted while having a\nvery small CPU time needed for completion. Such preemption,\nincreases number of context switches (also called as a process\nswitch or task switch), it causes invalidation of caches and\npipelines, swapping of buffers and so on [6]. Thus ultimately\nthis increases TaT of the program. (TaT is the delay between\nprocess submission and process completion). Thus we observe\nthat by characterizing or recognizing programs it may be\npossible to understand their previous execution history and\npredict their resource requirements. In this paper we address\nthe problem of, how to minimize the TaT of programs by\nusing ML techniques.We discover certain static and dynamic\ncharacteristics of a program are taken as features which the\nmachine learning techniques use to predict CPU burst times.\nWe call Special Time Slice or STS as the CPU burst time that\nminimizes turnaround time.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nThe rest of this paper rst discusses related work in Section II,\nan overview of machine learning techniques in Section III, we\nrerview Linux process scheduling in Section IV, and then de-\nscribes our implementation in Section V. Section VI describes\nour experimentation. Section VII presents the conclusions and\ndescribes possible future work.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nII. RELATED WORK\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nThe work [10] by Suranauwarat and Taniguchi, presents\nan approach to remember the previous execution behavior of\ncertain well-known programs. They study the process times\nof these programs in variousimilaritys states. The knowledge\nof the program ow sequence (PFS) is used to extend the\nCP U time slice of a process. P F S of a process is computed\nfrom its past execution history. P F S characterizes the process\nexecution behavior and is used to decide whether the program\nexecuting currently needs additional time. They set a threshold\nTm called as maximum dispatch delay time, which determines\nthe time limit for context switching; and is a multiple of the\ndelay for minimum process switching time. They control the\nCP U time of a process Tp, by either reducing or increasing\nsome scaling feature. They conclude from experimental results\nthat overall processing time is reduced for known programs.\nTo schedule a process they search for its name in the PFS\nknowledge base and thus improve its behavior.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nIn the paper by Smith et al. [7], the authors predicted\nthe application run times using historical information. They\npresent a technique for deriving predictions for the run\ntimes of parallel applications from the run times of similar\napplications that have executed in the past. They use the\nfollowing characteristics to dene similarity: user, queue, load\nleveler script, arguments, network adapter, number of nodes,\nmaximum run time, submission time, start time, run time.\nThese characteristics are used to make a template which can\nnd the similarity by matching. They use genetic algorithms\nfor identifying good templates for a particular workload.\nA template denes the similarity between two applications.\nThey use Genetic Algorithm (GA) techniques to determine\nthose application characteristics that yield the best denition\nof similarity for making predictions. GAs are probabilistic\ntechniques and are well known for exploring large search\nspaces. However from a machine learning perspective, GAs\nare expensive in terms of computation [15] and also their\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\nresults are considered fragile.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nIn the paper by Gibbons [5], the authors used Statistical\nRegression methods for prediction. Wh ile regression methods\nwork well on numeric data but cannot readily be used for\nnominal data. In the paper by Fredrik et al. [4], the authors\ndescribed an application signature model used for predicting\napplication performance on a given set of grid resources.\nIn this model,\nthey introduced the notion of application\nintrinsic behaviour to separate the performance effects of the\nruntime system from the behaviour inherent in the application\nitself.The signature model is used as the basis for performance\npredictions. So their approach combines the knowledge of\napplication intrinsic behaviour with run-time predictions of\nresources. They also dene application intrinsic metrics as\nmetrics that are solely dependent on the application code and\nproblem parameters.\nFrom this brief review of related literature, we draw the\nfollowing conclusions:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n It is possible to protably predict scheduling behavior.\n However the success of the approach depends upon\nthe ML technique used to train on previous programs\nexecution behavior.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n More importantly for the ML techniques to succeed, it\nneeds a suitable characterization of the program attributes\n(features) that are most signicant for prediction.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nIII. AN INTRODUCTION TO MACHINE LEARNING\nTECHNIQUES USED\n**BLOCK**fs== 10.0**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nIn this section we bring in some background information\nabout the machine learning techniques which were used in\nthe experimentation. The eld of machine learning (ML) [15]\nis concerned with the question of how to construct computer\nprograms that automatically improve with experience. ML\nmethods are also similar to Pattern Recognition methods\nis that while training\nas [16]. The most\nor learning stage several attributes are used to describe the\nobjects being studied. In our case the objects being studied\nare programs and we have come up with the best attributes\nthat can desribe programs for prediction. [8]. However the\nsuccess of a ML method also depends upon the classiers\nused. Certain classiers are more successful for different\ndata or problem domains. Here we just give a cursosry\nintroduction to the two most effective which were used [8]\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nimportant aspect\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe Learning Algorithms (Classiers): For our experi-\nments, we selected a representative set of standard machine\nlearning algorithms with different model classes. All of these\nare available in the Waikato Environment for Knowledge\nAnalysis (Weka) [9] , an open source machine learning tool.\nThe selected learner types (classier types) were: Trees and\nLazy which are listed below:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n C4.5 (or J48 in Weka [9] ) is a Reduced-Pruned Decision\nTree Learner. Decision Tree learning is one of the most\nwidely used and practical methods for inductive infer-\nence. It is a method for approximating discrete-valued\nfunctions, and is robust to noisy data and capable of\nlearning disjunctive expressions.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n IB k Learner (k-NN): This is an instance based learner\nantd assumes all instances are points in an n-dimensional\nspace. The nearest neighbors (nn) of an instance are\ndened in terms of the standard Euclidean Distance. The\nclass label of a new instance is found from the k instances\nnearest to it by assigning to it the majority label of the\nk-NN [15] [9].\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.6**r== 0.1**\nIV. A REVIEW OF LINUX PROCESS SCHEDULING\nCONCEPTS\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nIn this section, we review the Linux process scheduling\nconcepts which we use to develop the modied scheduler as\ndescribed in Section V.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nScheduler: The idea behind a scheduler is simple. Its\nobjective is to best utilize processor time. Assuming there are\nrunnable processes, a process should always be running. If\nthere are more processes than processors in a system, some\nprocesses will not always be running. These processes are\nwaiting to run. Deciding which process runs next, given a\nset of runnable processes, is the fundamnetal decision the\nscheduler must take.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nTime slice: Linux uses a tick to allocate processor time\nto individual processes. There are several different hardware\ntimers that the kernel can access. One is loaded at boot time\nwith values to interrupt the system every tick. The kernel\nuses the CPU clock speed given as HZ in the kernel to set\nthe length of a single tick to approximately 2.5 nanoseconds\n[2]. Consequently, every 2.5 nanoseconds, the system takes\nover, decrements the amount of ticks that the current process\nhas, and returns the CPU to the process for another tick.\nIf the process running is out of CPU time, the scheduler\nis invoked. If there are more processes in the run-queue\nthat have been allocated CPU time, the scheduler will nd\nthe best one and switch to it. The best process is selected\nusing the task timeslice() macro dened in sched:c. This\ntask timeslice() macro calculates a weight for the process\nusing bonus which is calculated from nice values etc.. The\nhighest weight is the best process to schedule [14] [2].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nProcess Descriptor: The process descriptor [13] is dened\nin sched:h and holds all information about a process. The\namount of ticks of CPU time that a process receives before\nanother process is scheduled is stored in the process descriptor.\nWhen all processes in the run-queue have exhausted their\nCPU time,\nthe scheduler recalculates the amount of CPU\ntime for each process using the macro task timeslice(). The\ntask timeslice() uses the process’ nice value in the process\ndescriptor to determine the number of ticks the process gets.\nHere a process is referred to as p. Variables in the process\ndescriptor are referred to as p!variable. For example p!nice\nrefers to the nice value of process p.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nContext switching: The context of a process [3] consists\nof the contents of its (user) address space and the contents of\nhardware registers and kernel data structures that relate to the\nprocess. In multi-processing environments, a context switch\n(or process switch or task switch) is when one process is\nsuspended from execution , its context is recorded and another\nprocess starts its execution in the CPU.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.6**\nV. IMPLEMENTATION\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nIn this section, we briey desribe about kernel modications\nand process charcterization. The kernel modications subsec-\ntion shows the kernel data structures modied for nding STS.\nThe process characterization section shows, how to nd the\nstatic and dynamic characteristics of a process.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nA. Kernel modications\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nOur idea of kernel modications is similar to that of [12],\nbut the difference is that we describe the modications to\nO(1) scheduler of Linux Kernel 2.4.20-8.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\n1) Process descriptor: The Linux process descriptor is\ndened in sched:h as a structure called task struct. More than\none process can request a larger amount of processor time and\nthe amount of extra time to be given is specic to the process.\nTherefore the extra time a process gets is stored in the\nprocess descriptor. A single integer eld, special time slice is\nadded to the process descriptor and is sufcient to store the\nnumber of ticks that a CPU-bound process should receive.\nThis single variable can also be used to distinguish between\nCPU-bound process from the other processes in the system.\nIf p!special time slice is less than 0 then this process should\nbe allocated the standard amount of ticks dened by the\nmacro task timeslice() in sched:c. If p!special time slice\nis greater than 0 then the process is given special time slice\nticks of processing time instead of using the standard macro,\ntask timeslice(). This requires that when processes are created,\nthe special time slice variable must be initialized to 1 in\nfork.c. The next sections describe how the modied scheduler\nuses the special time slice variable in the process descriptor.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\n2) System calls: Two system calls have been added to\nthe kernel for controlling CPU-bound processes. One system\ncall is necessary to allocate more processor time for a single\nprocess. This system call performs two simple tasks. First,\nthe special time slice variable is set to the number of ticks\nrequested by the process via an argument to the system call.\nSecond, the time slice variable of the process descriptor is set\nto the value passed to the system call. The process then returns\nto run on the processor with an increased amount of processor\ntime and special handling from the scheduler. Therefore the\nprocess gets special time slice ticks of processor time, each\ntime the scheduler recalculates the process time slices.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nThe second system call\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nis used to return a process to\nthe normal state in which p!time slice is calculated using\ntask timeslice() and p!special time slice is less than 0.\nto\nThe next\ntask timeslice(p), which is the standard method of assigning\np!time slice.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\ntime the scheduler runs the time slice is set\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n3) Modications to O(1) scheduler data structures: The\nscheduler tick() function of sched:c has been modied to\nhandle any process with special time slice > 0 differently\nfrom other processes.If this value is exhausted (decremented to\n0) then the scheduler gives the processor to the next process.\nThe run-queue is a circular doubly linked list of process\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\ndescriptors in the TASK RUNNING state.The rst thing that the\nnew scheduler does is check whether process currently running\nhas been granted more processor time (p! special time slice\n> 0). If this is true, and the process still has some CPU time\nleft (p!time slice > 0), then the scheduler immediately gives\nthe processor back to the current process. This guarantees that\neach processor-bound process uses all of its processor time\nimmediately, even if the scheduler is called in the middle\nof its running. If the CPU-bound process has used all of its\nCPU time (p!time slice > 0), then the scheduler continues\nits normal operation and switch to the next process.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nCalculating the goodness [2] for each process remains\nthe same. Since CPU-bound processes have large time slice\nvalues, they are not favored over other processes. This is\nacceptable since the CPU-bound processes are typically the\nonly things using the processor on a system. The CPU-\nbound process may be last to run, but there should not be\nmuch waiting to run in front of them, and from the previous\nmodication these processes is guaranteed to stay on the\nprocessor until they have used all of there CPU time.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nA slight modication to the recalculation of the time slice\nvariables checks each process to see if it can be allot-\nted more than task timeslice(p) ticks of processing time. If\nthe p!special time slice > 0, then p!time slice is set to\np!special time slice. There is a small amount of overhead\nadded to the scheduling algorithm, because each process has\nto be checked to see if it is a CPU-bound process. All other\nprocesses receive the same time slice value, as they would\nunder normal operation of the scheduler. The rest of the\nscheduling algorithm remains the same.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nB. Process characterization\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThis section describes , how we characterized the programs.\nWe studied the execution behaviour of several programs to nd\nout the characteristics that can be used to predict the STS. We\nhave taken representative programs : matrix multiplication,\nsorting programs, recursive bnocci number generating pro-\ngrams and random number generator programs. Any Machine\nlearning technique [15] requires fully labeled (categorized)\ndata for training. We categorized our data into classes and\ntrained the classiers (machine-learning techniques) [15].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe experimental procedure is divided into two phases. In\nthe rst phase, we create the data set from the program’s run\ntraces and make the data base with the static and dynamic\ncharactersitics of the programs and train it with machine\nlearning techniques. In the second phase, we classify this data\naccording to the tness function, STS by using machine-\nlearning techniques. The trained classiers are then used on\na different data set called as a test data set. Often due to\nlimitations in size of the data, we train classiers using a leave-\none-out technique [16] which is a standard technique.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n1) Creating the dataset: To characterize the program ex-\necution behaviour , we need to nd the static and dynamic\ncharacteristics. We used readelf [1]and size commands to\nget the attributes as shown in Table I. To nd STS, a script\nran the matrix multiplication program of size 700 x 700\nmultiple times with different values of STS on P4 Linux\n**BLOCK**fs== 6.4**b== 0.9**t== 0.1**l== 0.2**r== 0.6**\nATTRIBUTES AND THEIR MEANING\n**BLOCK**fs== 6.4**b== 0.9**t== 0.1**l== 0.7**r== 0.2**\nTHE INSTANCE\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nAttribute\nHash\nDynamic\nDynstr\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nmsh\nDynamic\nDynstr\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nDynsym\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nGot\nPlt\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nRoData\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nRyl.Dyn\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nText\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nData\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nBss\nTotal size\nProgram Name\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nInput Size\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nInput Type\nSpecialTimeSlice(STS)\nSTSclass\n**BLOCK**fs== 8.0**b== 0.5**t== 0.2**l== 0.2**r== 0.5**\nDenition\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for\ndynamic linking.\nThis is symbol hash table size in bytes.\nThis dynamic linking information size in bytes.\nThis is size ( bytes )of strings needed for dy-\nnamic linking, most commonly the strings that\nrepresent\nthe names associated with symbol\ntable entries .\nThis is size ( bytes) of the dynamic linking\nsymbol table .\nThis is the global offset table size in bytes.\nThis is the procedure linkage table size in\nbytes.\nThis is read-only data size (bytes) that typically\ncontribute to a non-writable segment\nin the\nprocess image.\nThis is the size (bytes) of the rellocation infor-\nmation size.\nThis is the text or executable instructions,\nsize (bytes )of a program.\nThis is the size(bytes) of the initialized data\nthat contribute to the size of the program’s\nmemory image size.\nThis section holds uninitialized data size\nThis is total size (bytes) of the program.\nThis is name of the program and a nominal\nattribute.\nValue of InputSize which depends on the pro-\ngrams type.\nType of the Input which is a nominal value.\nValue of the best time slice.\nClass of the STSs\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nSystem and selected the STS, which gives minimum TaT. The\nrst 18 (characteristics) attributes are used to predict the target\nattribute STSclass.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nWe build the data set of 84 execution instances of ve\nprograms: matrix multiplication, quick sort, merge sort, heap\nsort and a recursive Fibonacci number generator and the\nTable II shows an example of a training example or an\ninstance. We collected the data like the above for 5 programs\nwith different input sizes and different best STSs. Data of 84\ninstances of the ve programs was collected and made into\n11 categories based on the attribute STSclass with each class\nhaving an interval of 50 ticks.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n2) Training and Testing methodology: We performed two\ntypes of tests on the training examples with all the learners\ndescribed in the section, Overview of machine learning\nalgorithms, on the data sets collected in the rst phase. The\ntests are:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n Use Training Set [9]: The classier is evaluated on how\nwell it predicts the class of the instance it was trained on.\n Cross-Validation [9]: The classier is evaluated by cross-\nvalidation, using the number of folds that are entered\nin the Folds text eld (Weka). Recognition accuracy\nwas tested via cross-validation. In this test, the training\nexamples are divided into 10 parts and the classier\nclassies by taking one part as a test set and other 9\nparts as training set. Likewise, we continue for all parts.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.6**r== 0.3**\nAttribute\nHash\nDynamic\nDynstr\nDynsym\nGot\nPlt\nRoData\nRyl.Dyn\nText\nData\nBss\nTotal size\nProgram Name\nInput Size\nInput Type\nSpecialTimeSlice (STS)\nSTSclass\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.7**r== 0.1**\nValue\n\n\n\n\n\n\n\n\n\n\n\n\nmm\n\ni1 input type1\n\nt2 (101 ticks-149 ticks)\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nProgram X\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nKnowledge Base\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nDecision Tree Program\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nExecution History of Programs\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nUser Space\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nSTS\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nKernel Space\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nOur System call\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nModified Scheduler\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\nNow CPU allocates\nSTS ticks to  Program X\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nTimeSlice = STS ticks\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.8**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nHardware\n**BLOCK**fs== 4.8**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 4.8**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nCPU\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nFig. 1. The design of modied scheduling process.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nThe results of these tests explained in the next section Eval-\nuation and Results.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\n3) Extracting the best attributes: Attributes selection1 [11]\ninvolves searching through all possible combinations of at-\ntributes in the data to nd which subset of attributes works best\nfor predicting the program execution behavior. Here our goal\nis to predict the target attribute. To do this, two objects must\nbe set up ; an attribute evaluator and a search method. The\nevaluator determines what method is used to assign a value\n(weight or worth) to each subset of attributes. Here, we used\nExhaustive and Genetic search methods, and Co-rrelation\nbased feature selection (CfsSubsetval) [9] evaluation method.\nA detailed description of feature selection is presented in [11].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nC. The design of modied scheduling process\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThe Figure 1 shows the design of modied scheduling\nprocess and the steps to minimize TaT of a program. The steps\nto minimize TaT of a program are as shown in the Figure 1\nwith numbers from 1 to 5.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n1) The program ’X’ is given to C4.5 decision tree as an\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\ninput.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n1Extracting the best attributes is nothing but feature selection\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n2) The decision tree will classify ’X’ and output the STS.\n3) We send this STS information to modied scheduler\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nthrough a system call.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n4) The scheduler instructs the CPU such that CPU allocates\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nper second.le:///usr/share/doc/HTML/index.html Therefore\non 1.6GHz Intel P4 machine , we can save 1600MHz = 16 x\n108 clocks/second = [16 x 108 x no.of pipelines x (60320)]\nlow-level operations , with special time slice (STS) = 200.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nSTS ticks to ’X’.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n5) The CPU allocates STS ticks to ’X’ and it will run with\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nB. Selecting the best machine Learning technique\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nminimum TaT.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nD. Overall method\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nOur method is explained as follows :\n1) Run the programs with different special time slices\nwith modied O(1) scheduler and nd STS (best spe-\ncial time slice) which gives minimum turn-around-time\n(TaT).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n2) Build the knowledge base of static and dynamic char-\nacteristics of the programs from the run traces obtained\nin step 1 and train them with the C4.5 decision tree\nalgorithm.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n3) If a new program comes, classify it and run the program\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nwith this predicted STS.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n4) If the new program instance is not in the knowledge-\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nbase, go to step 1.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nE. Experimental environment\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFor this work, we used GNU/Linux 2.4.20-8 operating sys-\ntem, 1.6GHz Intel P4 processor, 128MB RAM main memory,\n512KB Cache memory, Vi editor and GNU gcc compiler and\nso on. WEKA(Waikato Environment for Knowledge Analysis),\nan open source machine-learning tool was used to nd the\nmost suitable ML method to characterize our programs.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nVI. EXPERIMENTS AND RESULTS\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nWe explain the experimentation process by taking the matrix\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nmultiplication program as an example.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nA. Effectiveness of STS\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nA script ran the matrix multiplication program of size 700\nx 700, multiple times with different values of STS on P4\nLinux System. The table III shows how the turn-around-\ntime changed as the CPU burst time (STS) of the process\nchanged. From the table III, STS (or Best special time slice)\n**BLOCK**fs== 6.4**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nEFFECT OF SPECIAL TIME SLICE ON TURN-AROUND-TIME(TAT).\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nTABLE III\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nAverage TaT(secs)\n16.372123\n16.900567\n15.863590\n.....\n15.880436\n...\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nspecial time slice (no.of ticks)\n\n\n\n....\n\n...\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nis 200 and corresponding TaT = 15.863590sec. Standard\nTaT (with unmodied scheduler) is 16.466sec. The microsec-\nonds saved per second = ( 16.466880 - 15.863590 ) / 16\n= 60320 micro-seconds. About 60320 micro-seconds saved\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nWe used WEKA for training and testing the data set of 84\ninstances of those ve programs to nd a good ML technique\n[8]. The table IV shows the best machine learning technique\nparticularly for our data set. From the table IV, C4.5 is the\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nTABLE IV\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nMACHINE LEARNING TECHNIQUES AND PREDICTION ACCURACIES ON 84\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nINSTANCES.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nML Technique % Classcation with\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nC4.5(J48)\nIBK\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.2**\nCrossValidation Test\n91.1667%\n89.5432%\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\n% Classication with\nUse Training Set Test\n94.0476%\n93.6480%\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nbest technique2 to predict the STS [8].\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nC. Finding the best attributes which can characterize STS\nclass\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nWe applied, exaustive search with CfsSubset [11] evaluation\nmethod on the training set and nd the best attributes, which\ncan characterize the STS Class. The Table V shows the best\nattributes.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nTABLE V\n**BLOCK**fs== 6.4**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nTHE BEST ATTRIBUTES.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nAttribute\nInputSize\nProgramSize\nBSS\nRoData\nText\nInputType\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nIts Rank\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nD. Effectiveness of our method in reducing the Turn-around-\ntime.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nThe table VI shows that how our Modied Scheduler 3\nreduces turn-around-time of Matrix Multiplication program\ninput sizes. In table VI, MS is Modied\nwith different\nScheduler and UMS is Unmodied Scheduler.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nInuence of STS on a heavy load system : We ran the matrix\nmultiplication programs as a batch on a heavy load system\nto test the effectiveness of our modied scheduler. There are\n10 programs in a batch. The table VII shows the results. We\nran four programs which have operations like opening several\nles, writing large number of random numbers in them and\nclosing them to make the system as a heavy load system. We\nmonitored the load-average using top command.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n2IBK is also a good technique but it is costly in terms of computation as\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\ncompared to C4.5\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n3UnModied Scheduler is O(1) Scheduler of Linux Kernel 2.4.20-8 and\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nModied scheduler is modied O(1) Scheduler of Linux Kernel 2.4.20-8.\n**BLOCK**fs== 6.4**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTHE COMPARISION OF UNMODIFIED SCHEDULER AND MODIFIED\nSCHEDULER OF MATRIX MULTIPLICATION PROGRAMS.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nMatrix Sizes\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n700 x 700\n750 x 750\n800 x 800\n850 x 850\n900 x 900\n950 x 950\n1000 x 1000\n1050 x 1050\n1100 x 1100\n1150 x 1150\n1200 x 1200\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\nTaT with UnModi-\ned Scheduler\n16466 millisecs\n19568 millisecs\n23820 millisecs\n28789 millisecs\n33540 millisecs\n39997 millisecs\n47370 millisec\n54912 millisecs\n63782 millisecs\n72704 millisecs\n85776 millisecs\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\nTaT with Mod-\nied Scheduler\n15864 millisecs\n19228 millisecs\n23190 millisecs\n28112 millisecs\n32029 millisecs\n39388 millisecs\n46110 millisecs\n54204 millisecs\n61580 millisecs\n70264 millisecs\n80654 millisecs\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nin\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nReduction\nTaT\n602 millisecs\n340 millisecs\n630 millisecs\n677 millisecs\n1511 millisecs\n609 millisecs\n1260 millisecs\n708 millisecs\n2200 millisecs\n2440 millisecs\n5122 millises\n**BLOCK**fs== 6.4**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nTABLE VII\nCOMPARISON OF MODIFIED SCHEDULER AND UNMODIFIED SCHEDULER\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nFuture work: Our future work will include extending our\ntechnique to parallel programs, more comprehensive study\nof high performance application characteristics is being con-\nducted on an 8-node Linux cluster, and will also include to\nadd our idea of predictability to Portable Batch Scheduler of\nLinux Cluster.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nACKNOWLEDGMENT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nWe would like to thank Sasi Kanth Ala for giving valuable\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nadvice through out of our work.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n[1] Hongjiu Lu, ELF: From The Programmer’s Perspective, Technical Report,\nNYNEX Science and Technology, 500 Westchester Avenue, White Plains,\nNY 10604, USA, May,1995.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n[2] Aivazian, Tigran, Linux Kernel 2.4 Internals, The Linux Documentation\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nProject, August, 2002.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n[3] Maurice Bach, The design of the Unix operating system,\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\nPearson\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nON A HEAVY LOAD SYSTEM.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nEducation Asia, pp:159-170, 2002.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nMatrix Sizes\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nBatch\nSize\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nload-\naverage\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n800 x 800\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n900 x 900\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n7.38\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n10.34\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\nTaT\nwith\nUMS\n165.57\nsecs\n469.23\nsecs\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\nTaT\nwith\nMS\n163.23\nsecs\n463.86\nsecs\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nReduction\nin TaT\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\n2.34\nsecs\n5.37\nsecs\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nDecision tree overhead: We implemented C4.5 decision\ntree for training and testing and its running time is around\n220 milliseconds. The overhead of this is around 4% when\ncompared to the reduction in turn-around time in case of\nmatrix multiplication program of size 1200 x 1200. So we\ncan neglect the overhead of decision tree.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nLimitations of our method.: The characteristics of the\nprograms and the gain in the turn-around-time depends on\nthe architecture and operating system. Our modied scheduler\nis not designed with any security features that would prevent a\nuser from writing their own process and requesting INT MAX\nprocessor time via system call which is described in the\nSection V-A.2. Setting p >special time slice to INT MAX\ncould signicantly slow down a system.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nVII. CONCLUSIONS\nBy considering the static and dynamic characteristics of\na program, we can schedule it using modif ied scheduler\nsuch that the turn-around-time of it is minimized. We show\nthat machine learning algorithms are very efcient\nin the\nprocess characterization process. The C4.5 decision tree\nalgorithm achieved good prediction (91%  94%), which\nindicates that when suitable attributes are used, a certain\namount of predictability does exist for known programs. Our\nexperiments show that 1:4% to 5:8% reduction in turn-around-\ntime is possible and this reduction rate slowly increases with\nthe input size of the program. From our experiments, we\nnd the best features : input size, program size, bss, text,\nrodata and input type of a program that can characterize\nits execution behavior. We conclude that our technique can\nimprove the scheduling performance in a single system.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n[4] Fredrik Vraalsen, Performance Contracts: Predicting and monitoring grid\napplication behavior, In Proceedings of the 2nd Internationsl Workshop\non Grid computing, November, 2001.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n[5] Richard Gibbons, A Historical Application Proler for Use by Parallel\nSchedulers , Lecture Notes on Computer Science, Volume : 1297, pp:\n58-75, 1997.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n[6] Hyok-Sung Choi and Hee-Chul Yun, Context Switching and IPC\nPerformance Comparison between uClinux and Linux on the ARM9\nbased Processor , Linux in embedded applications,\nJan., 2005.\nhttp://www.linuxdevices.com/articles/AT2598317046.html.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n[7] Warren Smith, Valerie Taylor, Ian Foster, Predicting Application Run-\nTimes Using HistoricalInformation, Job Scheduling Strategies for Par-\nallel Processing, IPPS/SPDP’98 Workshop, March, 1998.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n[8] Kishore Kumar. P and Atul Negi, Characterizing Process Execution\nBehaviour Using Machine Learning Techniques, In DpROM WorkShop\nProceedings, HiPC 2004 International Conference, December, 2004.\n[9] Garner, S .R., WEKA: The Waikato Environment for Knowledge Anal-\nysis, In Proc. of the NewZealand Computer Science Research Students\nConference,\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\npp : 57-64, 1995.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[10] Surkanya Suranauwarat, Hide Taniguchi, The Design, Implementation\nand Initial Evaluation of An Advanced Knowledge -based Process Sched-\nuler, ACM SIGOPS Operating Systems Review,\nvolume: 35, pp: 61-81,\nOctober, 2001.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n[11] Mark A. Hall, Co-rrelation based feature selection for Machine Learn-\nDepartment of Computer Science, University of\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.2**\ning, Master Thesis,\nWaikato, pp: 7-9, 12-14, April, 1999.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[12] Andrew Marks, A Dynamically Adaptive CPU Scheduler,\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nMaster\nThesis, Department of Computer Science, Santa Clara University, pp :5-\n9, June, 2003.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\n[13] Robert Love, Linux Kernel Development, 1st ed,\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nThe Pearson\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nEducation,2004.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[14] Danie P. Bovet, Marc, Understanding the Linux Kernel, 2nd ed,\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nO’\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nReilly and Associates, Dec., 2002.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[15] Tom Mitchell, Machine Learning, 1st ed, pp: 52-75, 154-183, 230-244,\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nThe Mc-Graw Hill Company.Inc.International Edition,1997.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n[16] O. Duda, P. E.Hart , Pattern Classication, Wiley, NewYork,1973.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nAtul Negi works as a reader in the Department of Computer and Informations\nSciences, University of Hyderabad, Hyderabad. He is presently the IEEE\nComputer Society Chapter Chair for IEEE Hyderabad Section. His interests\nare in applications of Machine Learning, Pattern Recognition and Document\nImage Analysis.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nKishore Kumar P. is a Masters student in the Department of Computer and\nInformations Sciences, University of Hyderabad, Hyderabad. His interests are\nin Operating Systems and Cluster computing.",
         "extracted",
         "Performance Contracts: Predicting and Monitoring Grid Application Behavior;The design, implementation and initial evaluation of an advanced knowledge-based process scheduler;Understanding the Linux Kernel;Predicting Application Run Times Using Historical Information;A Historical Application Profiler for Use by Parallel Schedulers;Context Switching and IPC Performance Comparison between uClinux and Linux on the ARM9 based Processor;Characterizing Process Execution Behaviour Using Machine Learning Techniques;The Pearson Education;Linux Kernel Development , 1st ed;Correlation-based Feature Selection for Machine Learning;A Dynamically Adaptive CPU Scheduler;Linux Kernel 2.4 Internals;Co-rrelation based feature selection for Machine Learning, Master Thesis, Department of Computer Science, University of Waikato;Machine Learning , 1st ed, pp: 52-75, 154-183, 230-244, The;WEKA: The Waikato Environment for Knowledge Analysis;ELF: From The Programmer's Perspective;The Design of the UNIX Operating System;Build the knowledge base of static and dynamic characteristics of the programs from the run traces obtained in step 1 and train them with the C4.5 decision tree algorithm;is a Masters student in the Department of Computer and Informations Sciences;We send this STS information to modi(cid:2)ed scheduler through a system call;The scheduler instructs the CPU such that CPU allocates STS ticks to ’X’;decision tree will classify ’X’ and output the STS. per second.(cid:2)le;Aivazian , Tigran , Linux Kernel 2 . 4 Internals , The Linux Documentation Project , August , 2002",
         "Applying Machine Learning Techniques to Improve Linux Process Scheduling",
         "None"
        ],
        [
         "10",
         "000d7634ea82cea4ea3d1aac5e632f88e6736f05",
         "ABSTRACT Neuronal leucine-rich repeat proteins (NLRRs) are type I transmembrane proteins and expressed in neuronal tissues, but their function remains unknown. Here, we describe the identification and characterization of a new member of the NLRR family, NLRR4, and its potential role in long-lasting memory. We generated NLRR4-deficient (NLRR4−/−) mice and found that they showed impaired memory retention. In hippocampus-dependent learning tasks, NLRR4−/− mice were able to learn and maintain the memories for one day but unable to retain the memories for four days after learning. In contrast, in a hippocampus-independent task, NLRR4−/− mice were able to retain the memory normally for at least seven days. These results suggest that NLRR4 plays a key role in hippocampus-dependent long-lasting memory.",
         "Takayoshi Bando,K. Sekine,Shizuka Kobayashi,A. M. Watabe,Armin Rump,Minoru Tanaka,Yoshikuni Suda,S. Kato,Y. Morikawa,T. Manabe,A. Miyajima",
         "https://europepmc.org/articles/pmc1087730?pdf=render",
         "\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nMOLECULAR AND CELLULAR BIOLOGY, May 2005, p. 4166–4175\n0270-7306/05/$08.000 doi:10.1128/MCB.25.10.4166–4175.2005\nCopyright © 2005, American Society for Microbiology. All Rights Reserved.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nNeuronal Leucine-Rich Repeat Protein 4 Functions in\nHippocampus-Dependent Long-Lasting Memory\nTakayoshi Bando,1 Keisuke Sekine,1 Shizuka Kobayashi,3 Ayako M. Watabe,3 Armin Rump,1\nMinoru Tanaka,1 Yoshikuni Suda,1 Shigeaki Kato,2 Yoshihiro Morikawa,4\nToshiya Manabe,3 and Atsushi Miyajima1*\nLaboratory of Cell Growth and Differentiation1 and Laboratory of Neuclear Signaling,2 Institute of Molecular and\nCellular Biosciences, University of Tokyo, Yayoi, Bunkyo-ku, Tokyo 113-0032, Japan; Division of Neuronal\nNetwork, Department of Basic Medical Sciences, Institute of Medical Science, University of Tokyo, 4-6-1\nShirokanedai, Minato-ku, Tokyo 108-8639, Japan3; and Department of Anatomy and Neurobiology,\nWakayama Medical University, 811-1 Kimidera, Wakayama 641-8509, Japan4\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.3**r== 0.3**\nReceived 11 November 2004/Accepted 13 February 2005\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.2**\nNeuronal leucine-rich repeat proteins (NLRRs) are type I transmembrane proteins and expressed in\nneuronal tissues, but their function remains unknown. Here, we describe the identiﬁcation and characteriza-\ntion of a new member of the NLRR family, NLRR4, and its potential role in long-lasting memory. We generated\nNLRR4-deﬁcient (NLRR4/) mice and found that they showed impaired memory retention. In hippocampus-\ndependent learning tasks, NLRR4/ mice were able to learn and maintain the memories for one day but\nunable to retain the memories for four days after learning. In contrast, in a hippocampus-independent task,\nNLRR4/ mice were able to retain the memory normally for at least seven days. These results suggest that\nNLRR4 plays a key role in hippocampus-dependent long-lasting memory.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe hippocampus plays a role in declarative memory such as\nepisodic and spatial memories, but unimodal memories such as\ncued fear memories occur independently of the hippocampus.\nHippocampal lesions result in severe amnesia of contextual\nand spatial learning and memory but do not affect normal\nlearning, as seen by the cued fear-conditioning test (3, 19, 24).\nMemory formation can be divided into three phases by differ-\nences in the length of the time from acquisition; short-term\nmemory, long-term memory, and long-lasting memory (22).\nNew protein synthesis is not required for short-term memory\nbut is required for long-term memory (18, 21).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nThe hippocampus plays a role in the acquisition of short-\nterm memory and also for the conversion of short-term mem-\nory to long-term memory within several hours after acquisition,\na process known as cellular memory consolidation (22). Long-\nterm memory is converted to long-lasting memory that is per-\nmanently stored in the cortex (7, 8). This process is known as\nsystem consolidation, and the memory acquired in the past is\nkept intact even if the hippocampus is damaged. Some proteins\nimplicated in cellular memory consolidation have been re-\nported, such as N-methyl-D-aspartate receptor, calcium-calm-\nodulin-dependent protein kinase IV (CaMKIV), and cyclic\nAMP (cAMP)-responsive element-binding protein (CREB) in\nmice (5, 16, 18, 26). Cellular consolidation requires new pro-\ntein synthesis directed by CREB-mediated transcription.\n-Calcium-calmodulin-dependent protein kinase II (CaMKII)\nis the only molecule known to be involved in system consoli-\ndation, and CaMKII heterozygous mice are defective in long-\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n* Corresponding author. Mailing address: Laboratory of Cell\nGrowth & Differentiation, Institute of Molecular and Cellular Bio-\nsciences, University of Tokyo, Yayoi, Bunkyo-ku, Tokyo 113-0032,\nJapan. Phone: 81-3-5841-7884. Fax: 81-3-5841-8475. E-mail: miyajima\n@ims.u-tokyo.ac.jp.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nlasting memory and cortical long-term potentiation (7). How-\never, how CaMKII is involved in this process is not fully\nunderstood.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nIn this study, we describe a novel type I transmembrane\nprotein termed NLRR4 which exhibits similarity to neuronal\nleucine-rich repeat proteins NLRR1 to NLRR3. NLRR1 and\nNLRR2 are expressed in the developing nervous system (4, 14,\n28, 29). NLRR3 is induced by brain injury and regulated by\nRas-mitogen-activated protein kinase signaling (9, 10, 15).\nHowever, their functions remain to be studied. To uncover the\nfunction of NLRR4 in vivo, we generated NLRR4-deﬁcient\n(NLRR4/) mice by replacing its exons with the -galactosi-\ndase gene by homologous\nrecombination. Homozygous\nNLRR4/ mice were viable and fertile. We found that their\nlong-term memory in hippocampus-dependent tasks was se-\nlong-term potentiation\nverely impaired, while hippocampal\nwas intact. Interestingly, hippocampus-independent memory\nassessed by the cued fear conditioning in NLRR4/ mice was\nnormal. These data indicate that NLRR4 is important for\nmaintenance of hippocampus-dependent memories.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nMATERIALS AND METHODS\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nMice. C57/BL6J and ICR mice were purchased from Nippon CLEA (Tokyo,\nJapan) and mutant mice were maintained and mated in our animal facility. All\nmice were housed in a speciﬁc-pathogen-free barrier animal facility. NLRR4/\nmice were backcrossed to the C57BL/6J strain at least 6 times before being used\nfor behavioral and electrophysiological experiments. All experiments were per-\nformed according to the guidelines of the Animal Care and Experimentation\nCommittee of University of Tokyo and Wakayama Medical University.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nExpression cloning of NLRR4 cDNA. Expression cloning of a cDNA encoding\nthe B61 antigen was carried out using COS7 cells and a cDNA library prepared\nfrom LO cells as previously described (13). We used anti-rat immunoglobulin\nG-conjugated magnetic beads and a magnetic cell sorter to enrich COS7 cells\nexpressing the B61 antigen. Plasmids recovered from enriched COS7 cells were\ntransfected in COS7 cells. By repeating this selection 3 times, we obtained a\ncDNA encoding the B61 antigen NLRR4.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nFIG. 1. Structural properties of NLRR4. (a) The deduced amino acid sequences of mouse (upper) and human (lower) NLRR4. The putative\nsignal peptide is underlined, and the transmembrane region is boxed. Identical amino acid residues are indicated by dots, and the leucine-rich\nrepeats (LRRs) and ﬁbronectin III (FN III) domains are indicated by the arrows underneath the sequences. (b) Structures of the NLRR family\nmembers. All members are type I transmembrane proteins with leucine-rich repeats and ﬁbronectin III motifs in their extracellular region. SP,\nsignal peptide; AF, amino-ﬂanking region; CF, carboxy-ﬂanking region; TM, transmembrane.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nNorthern blotting. Total RNA was extracted using Trizol reagent (Invitrogen)\nand polyadenylated RNA was prepared from various tissues using an Oligo-\nMAG mRNA puriﬁcation kit (Takara) according to the manufacturer’s protocol.\nNorthern blot analysis was carried out as previously described by using digoxi-\ngenin-labeled single-stranded DNA probe.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nGeneration of NLRR4-deﬁcient mice. To construct the NLRR4 targeting vec-\ntor, overlapping NLRR4 genomic clones were isolated from a phage library\nmade from 129SVJ strain mice (Stratagene). The targeting construct contained\nthe 5.5-kb 5 and 4.5-kb 3 homology regions and the 2.5-kb fragment containing\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nthe exon with initiation codon was replaced with the -galactosidase gene. The\nneomycin resistance gene was inserted between the homologous regions, and the\nherpes simplex virus thymidine kinase cassette was ligated to the 3 end of the\nvector. TT2 embryonic stem (ES) cells were electroporated and selected with\nG418 and 2-deoxy-2-ﬂuoro--D-arabinofuranosyl-5-iodouracil. Drug-resistant\nclones were analyzed by Southern blotting. BamHI- or EcoRI-digested genomic\nDNA was hybridized with the 400-bp 3 genomic fragment that corresponded to\ngenomic sequences outside of the targeting vector or 500-bp 5 genomic frag-\nment. Chimeric mice were generated by the aggregation method as previously\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n60 s were guided to the platform and allowed to stay there for 30 s. Each mouse\nwas trained four trials per day for 5 consecutive days. A different starting point\nwas used for each of the trials. The time to reach the platform was recorded\n(escape latency). Probe tests were performed at three time points (retention day)\nafter the last training. In the probe test, the platform was removed and the mice\nwere allowed to swim for 60 s. The swimming speed was calculated based on\nmovement data. For all experiments, the movement of each mouse was moni-\ntored by a charge-coupled device camera and the data were processed with NIH\nImage WM 2.12r (O’Hara & Co.).\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nElectrophysiology. All experiments were performed to compare NLRR4/\nmice with wild-type mice in a blind fashion using male littermates. Nine- to\n11-week-old NLRR4/ and wild-type mice were decapitated under deep halo-\nthane anesthesia, and both hippocampi were removed. Hippocampal slices (400\nm thick) were cut with a Vibratome tissue slicer and placed in a humidiﬁed\nholding chamber for at least 2 h. A single slice was then transferred to a\nrecording chamber maintained at 25°C, and submerged beneath a continuously\nperfusing medium that had been saturated with 95% O2/5% CO2. The medium\ncomprised 119 mM NaCl, 2.5 mM KCl, 1.3 mM MgSO4, 2.5 mM CaCl2, 1.0 mM\nNaH2PO4, 26.2 mM NaHCO3 and 11 mM glucose. All perfusing solutions con-\ntained picrotoxin (100 M) to block gamma-aminobutyric acid A receptor-\nmediated inhibitory synaptic responses. Field potential recordings were made\nusing a glass electrode ﬁlled with 3 M NaCl and placed in the stratum radiatum\nin the hippocampal CA1 region. An Axopatch-1D ampliﬁer was used, and the\nsignal was ﬁltered at 1 kHz and digitized at 10 kHz.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nTo evoke synaptic responses, a bipolar tungsten electrode was placed in the\nstratum radiatum, and Schaffer collateral and commissural ﬁbers were stimulated\nat 0.1 Hz (test pulses). A single high-frequency stimulus train (100 Hz, 1 s) was\napplied with the test pulse intensity to induce long-term potentiation. The input-\noutput relationship of basal synaptic responses was examined in the presence of\nD-2-amino-5-phosphonovaleric acid (25 M) to block N-methyl-D-aspartate re-\nceptor-mediated synaptic responses. A low concentration of 6-cyano-7-nitroqui-\nnoxaline-2,3-dione (1 M) was also present to partially block alpha-amino-3-\nhydroxy-5-methyl-4-isoxazole propionate (AMPA) receptors. This enables more\naccurate measurements of the input-output relationship, since the presence of\nlow concentrations of 6-cyano-7-nitroquinoxaline-2,3-dione reduces the nonlin-\near summation of ﬁeld excitatory postsynaptic potentials when strong stimulus\nstrengths are used. For measurements of paired-pulse facilitation, afferent ﬁbers\nwere stimulated twice at intervals of 50, 100, and 200 ms in the presence of 25\nM D-2-amino-5-phosphonovaleric acid. All values were expressed as the mean\n standard errors of the mean. Student’s t test (two-tailed, unpaired) was used\nto determine whether there was a signiﬁcant difference in the means between two\nsets of data.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nRESULTS\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nMolecular cloning of the NLRR4 gene. In our studies on the\ndevelopment of hematopoiesis, we generated a number of\nmonoclonal antibodies against LO cells (12), a hemangioblast-\nlike cell line. B61 is one such antibody and recognizes a cell\nsurface molecule on LO cells. We employed an expression\ncloning strategy to identify the B61 antigen by using COS7 cells\nand an expression library of LO cells, and we obtained a 2.7-kb\ncDNA encoding the B61 antigen. While the B61 antigen was\nhighly expressed in the hemangioblast-like cell line LO, B61\nmRNA was not found in hematopoietic tissues (data not\nshown). Instead, we found its expression in neuronal tissues as\ndescribed below. Sequence analysis revealed that the B61\ncDNA has an open reading frame encoding 735 amino acid\nresidues with two hydrophobic segments, one at the amino\nterminus (amino acids 1 to 21) and the second (amino acids\n679 to 704) in the internal region. There are two distinct motifs\nin the extracellular region, leucine-rich repeats and ﬁbronectin\ntype III-like repeats and a small intracellular region consisting\nof only 30 amino acid residues without any known motifs. Due\nto the similarity to the previously identiﬁed neuronal leucine\nrich-repeat proteins NLRR1 to NLRR3 (4, 14, 28, 29), we\nnamed B61 NLRR4. These members are type I transmem-\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nFIG. 2. Expression of NLRR4. (a) Northern blot analysis of\nNLRR4 mRNA in the adult brain. (b) Northern blot analysis of\nNLRR4 in various adult mouse tissues. Polyadenylated RNA was\nprepared from various parts of the adult brain or various adult tissues.\nOne g of mRNA for each sample was separated on an agarose gel\nand then transferred to a nylon membrane. After UV cross-linking, the\nmembrane was hybridized with a digoxigenin-labeled full-length\nNLRR4 probe. The lower panel shows equal loading of the sample\ndehydrogenase\nhybridized with\n(GAPDH) probe.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nglyceraldehyde-3-phosphate\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\na\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\ndescribed (25). Genotyping of progenies was performed by Southern blot anal-\nysis using the 3 probe or by PCR. The PCR-speciﬁc primers used were NLRR4S\n(5-TACGGACTCTCTCTGTGTAGGACTCCC-3), NLRR4AS (5-AGGTTG\nTGACTGAGGTCGAGCTCAC-3), NeoS (5-ACGACGGGCGTTCCTTGC\nGCAGCTGTG-3), and NeoAS (5-TCAGAAGAACTCGTCAAGAAGGCG\nATA-3)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nFlow cytometry. Cells harvested from cultures were resuspended in phosphate-\nbuffered saline and ﬁltered through 70-m nylon mesh (cell strainer, Falcon) to\nremove cell debris. The cells were then incubated with monoclonal antibodies for\n30 min on ice and analyzed by ﬂuorescence-activated cell sorting (Becton Dick-\ninson)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n-Galactosidase staining of whole brains. For -galactosidase staining, brains\nwere ﬁxed in phosphate-buffered saline containing 2% paraformaldehyde, 0.2%\nglutaraldehyde, and 0.02% NP-40 and sectioned at 300-m thickness by vi-\nbratome. After washing, the sections were placed in staining buffer (44 mM\nHEPES [pH 7.4], 3 mM potassium ferricyanide, 15 mM NaCl, 1.3 mM MgCl2, 0.5\nmg/ml 5-bromo-4-chloro-3-indolyl-- D-galactopyranoside [X-Gal] in phosphate-\nbuffered saline) at room temperature.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nContextual fear-conditioning tests. For each experiment, a mouse was placed\nin the conditioning chamber and given a series of foot shocks alone (2-s duration,\n0.3 mA, 1 min apart). To test its memory, the mouse was placed in the same\nconditioning chamber and freezing responses were analyzed using Image FZC\n(O’Hara & Co.), with modiﬁed software available in the public-domain National\nInstitutes of Health Image program. In the experiment, a mouse was trained by\nthree foot shocks and then its behavior was tested 1, 4, or 7 days after training.\nCued fear-conditioning tests. For training, a mouse was placed in the condi-\ntioning chamber for 2 min before the onset of conditioned stimuli, a tone which\nlasted for 30 s at 10 kHz and 70 dB. The last 2 s of the conditioned stimulus was\npaired with am unconditioned stimulus, 0.3 mA of a continuous electric foot\nshock. After an additional 30 s in the chamber, the mouse was returned to its\nhome cage. Four and seven days after training, the mouse was placed in a\ndifferent chamber in which the same tone as the conditioned stimuli was applied\nfor 120 s after a 1-min habituation period (preconditioned stimulus).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nMorris water maze test. The water pool used in these experiments was 1 m in\ndiameter and made up of white polyvinyl chloride and the temperature of the\nwater was 27  1°C. For the hidden-platform task, an acrylic transparent plat-\nform (10 cm in diameter) was submerged 5 mm below the surface of water that\nwas made opaque by adding nontoxic odorless white paint. The location of the\nplatform was ﬁxed during the trials for each mouse. If it located the platform\nwithin 60 s, it was kept on it for 30 s. Mice that failed to ﬁnd the platform within\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFIG. 3. Targeted disruption of the NLRR4 gene. (a) Structures of the NLRR4 protein, cDNA, genomic locus, targeting vector, and mutated\nallele. Three boxes in the NLRR4 genome represent exons. The locations of the probes used for Southern blotting are shown (probes 1 and 2).\nE, EcoRI; B, BamHI; lacZ, Escherichia coli -galactosidase cDNA; neo, neomycin resistance gene; HSV-tk, herpes simple virus thymidine kinase.\n(b) Identiﬁcation of a homologous recombinant clone (A29) by southern blot analysis of BamHI- or EcoRI-digested genomic DNA from ES clones.\n(c) Southern blot analysis of BamHI-digested genomic DNA from mouse tails using probe 1. (d) Flow cytometric analysis using an NLRR4\nantibody was performed on cells derived from wild-type and NLRR4/ embryos. NLRR4 protein was not detected in NLRR4/ cells.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nbrane proteins with multiple leucine-rich repeats and a ﬁ-\nbronectin type III-like domain.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nAn NLRR4-homologous gene was found in the human ge-\nnome but not in Caenorhabditis elegans or Drosophila melano-\ngaster. A human cDNA homologous to NLRR4 encoded a type\nI transmembrane protein with 640 amino acids (Fig. 1a) with\nan overall homology of 64% at the amino acid level. NLRR4\nshowed 21% and 20% homology at the amino acid level to\nNLRR1 and NLRR3 (28, 29), respectively. Although there are\ntwo immunoglobulin C2 loops in NLRR1 and NLRR3, no such\nstructure was found in NLRR4 (Fig. 1b). Northern blot anal-\nysis revealed that NLRR4 mRNA was expressed in lung, heart,\nand ovary (Fig. 2b). In the adult brain, NLRR4 was strongly\nexpressed in the hippocampus and weakly in the cerebellum\n(Fig. 2a).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nTargeted disruption of the NLRR4 gene. To uncover the\nfunction of NLRR4 in vivo, we generated an NLRR4-deﬁcient\nmouse line. As shown in Fig. 3a, we replaced the exons of the\nNLRR4 gene with the -galactosidase gene and the neomycin\nresistance gene (neo). The deleted exon encoded the initiation\ncodon, the signal peptide, and the leucine-rich repeat domains,\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nand the targeting vector was designed to express -galactosi-\ndase from NLRR4 regulatory elements. We isolated six inde-\npendent ES clones with homologous recombination (Fig. 3b),\nand chimeric male mice were generated from three of the\nclones. Heterozygous embryos from three independent chi-\nmeric mice exhibited the same -galactosidase expression pat-\ntern. Genotyping by Southern blotting showed successful tar-\ngeting of the NLRR4 gene (Fig. 3c).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nNLRR4 protein expression was not detected in NLRR4/\nembryos by ﬂow cytometric analysis of primary culture cells\nderived from wild-type and NLRR4/ embryos with an anti-\nNLRR4 monoclonal antibody (Fig. 3d). We genotyped 372\npups born from heterozygous matings to assess whether the\nexpected number of knockout and heterozygote pups was af-\nfected. The number of NLRR4/ to NLRR4/ to\nNLRR4/ was 83:196:93, as expected for unbiased Mende-\nlian frequencies (NLRR4/), and the mice grew with normal\nbody weights. Both males and females are fertile, and no in-\ncreased mortality was observed up to 1 year of age.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nExpression of NLRR4 in the adult brain and anatomy of the\nNLRR4/ hippocampus. Since NLRR4 mRNA was found in\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nFIG. 4. Expression of NLRR4 and hippocampal anatomy in the NLRR4/ adult brain. NLRR4 expression was assessed by -galactosidase\nstaining of a sagittal section (a) and a coronal section (b) of the adult brain of 2-month-old heterozygous mice. -Galactosidase activity was\ndetected in the hippocampus, layers V and VI in the cortex, the piriform cortex, the inner granule layer, and Purkinje cells in the cerebellum.\n-Galactosidase staining was performed in coronal sections of the NLRR4/ and NLRR4/ cerebrum. Similar staining patterns were observed\nin both NLRR4/ and NLRR4/ mice (c, d, e, and f). Gross hippocampal anatomy in NLRR4/ mice was evaluated by histochemical and\nimmunohistochemical staining. Hematoxylin-eosin staining (g and h) and immunostaining of a neuron-speciﬁc maker, NeuN (i and j), showed that\nthere were no signiﬁcant differences between the two genotypes. Abbreviations: h, hippocampus; c, cortex; p, piriform cortex.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nthe brain by Northern blotting analysis (Fig. 2), we precisely\nanalyzed the expression of NLRR4 in the brain. As the -ga-\nlactosidase gene was inserted in the exon of NLRR4, we per-\nformed -galactosidase staining of the adult brain of two-\nmonth-old heterozygous mice (NLRR4/) (Fig. 4a, b, c, and\ne). NLRR4 was expressed in the CA1 and CA3 regions and\ndentate gyrus of the hippocampus. In addition, NLRR4 was\nexpressed in layers V and VI in the neocortex and piriform\ncortex. It was also expressed in the inner granular layer and\nPurkinje cells of the cerebellum. The NLRR4/ brain was\nalso stained for -galactosidase activity, and there was no dif-\nference in the staining pattern between NLRR4/ and\nNLRR4/ mice (Fig. 4c, d, e, and f).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nWe performed histochemical staining to examine hippocam-\npal anatomy in wild-type and NLRR4/ mice. Hematoxylin-\neosin staining and immunohistochemical analysis of NeuN, a\nneuron-speciﬁc DNA binding protein, showed no difference\nbetween wild-type and NLRR4/ mice (Fig. 4g, h, i, and j).\nThese results indicated that the gross anatomy of hippocampus\nwas not affected by the NLRR4 mutation.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nDefective contextual fear memory in NLRR4/ mice. Since\nNLRR4 is expressed in the hippocampus, which plays a role in\nlearning and memory (3, 24), we considered the possibility that\nNLRR4 plays a role in memory formation. To address this, we\nexamined the associative emotional memory of the mutant\nmice using the contextual fear-conditioning test. If mice learn\nfear such as that resulting from an electric foot shock in a\nspatial context, they show some defensive responses, including\nfreezing behavior in the conditioning chamber (3).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.7**\nfear-conditioning test,\n**BLOCK**fs== 9.0**b== 0.1**t== 0.5**l== 0.1**r== 0.5**\nWithout experiencing the electric shock, wild-type and\nNLRR4/ mice showed little of this behavior. Mice with\neither genotype spent the same time freezing in the context at\n24 h after the training, indicating that NLRR4/ mice are\nable to learn fear by contextual fear-conditioning. However,\nNLRR4/ mice froze signiﬁcantly less than wild-type mice 4\nand 7 days after the training (Fig. 5a). There were no signiﬁ-\ncant differences between the two genotypes in terms of noci-\nceptive sensitivity to an electrical foot shock (Fig. 5b). For the\ncontextual\nfear-eliciting experiences\nsuch as an electric foot shock are input into the amygdala and\nspatial information such as the conditioning chamber is sent to\nthe hippocampus. Both sets of information are consolidated\nand stored in a manner dependent on the hippocampus and\namygdala. As NLRR4/ mice showed normal freezing behav-\nior during the 1-day retention period, cellular consolidation in\nNLRR4/ mice was normal. However, as the freezing time\nwas signiﬁcantly reduced in NLRR4/ mice 4 days after the\ntraining, their memory was impaired after long retention delay.\nNormal tone-dependent cued fear memory in NLRR4/\nmice. As contextual fear-conditioning tasks are dependent on\nthe hippocampus and amygdala, we examined hippocampus-\nindependent fear memory to assess the function of amygdala in\nNLRR4/ mice. The auditory cued fear-conditioning test is\nan amygdala-dependent but hippocampus-independent task\n(19). Wild-type and NLRR4/ mice were exposed simulta-\nneously to a tone (conditioned stimulus) and an electric foot\nshock (unconditioned stimulus). After this learning process, we\nmeasured the freezing time when the tone was replayed in a\ndifferent chamber. Mice with either genotype did not show any\nfreezing behavior before presenting conditioned stimuli (pre-\n**BLOCK**fs== 8.0**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nFIG. 5. Contextual and cued fear memory in NLRR4/ mice. (a)\nFreezing ratio for contextual fear at different retention delays for\nNLRR4/ and wild-type mice (n  13 for each genotype and each\ntime point). No difference was found in the time spent on freezing\nbehavior after 1 day of retention (wild-type, 58.7  15.1%; NLRR4/,\n59.5  12.6%). Freezing behavior was signiﬁcantly reduced in the\nNLRR4/ mice after 4 days (wild-type, 59.5  12.6%, NLRR4/,\n20.1  11.3%, Student’s t test P  0.001) and 7 days retention (wild-\ntype, 57.2  17.5%; NLRR4/, 17.8  9.8%, Student’s t test, P \n0.001). All data points represent the mean ( standard error of the\nmean). *, P  0.001; **, P  0.001. (b) Normal pain sensitivity of\nNLRR4/ and wild-type mice. Pain sensitivities were calculated by\nthe total pixels that mice moved before and after electric foot shock.\nAll data points represent the mean ( standard error of the mean). (c)\nPercent of freezing time at preconditioned or conditioned stimuli after\ncued fear conditioning for NLRR4/ and wild-type mice. Tests were\nperformed 4 and 7 days after training. No signiﬁcant difference be-\ntween the genotypes was found for either time points (day 4 after\ntraining: wild-type, 45.1  12.5%; NLRR4/, 49.2  13.4%; day 7\nafter training: wild-type, 66.4  20.7%; NLRR4/, 57.4  27.7% [n \n10 for each genotype and each time point]). All data points represent\nthe mean ( standard error of the mean).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nconditioned stimuli). In the presence of a conditioned stimu-\nlus, mice of either genotype showed the same level of freezing\ntime at both time points after the conditioning (Fig. 5c). No\nsigniﬁcant differences were observed in freezing time between\nthe two genotypes 4 and 7 days after the training, indicating\nthat NLRR4 is dispensable for tone information processing\nand fear memory is fully integrated into the amygdala without\nNLRR4 after long retention.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nDefective spatial memory in NLRR4/ mice. For the fear\nmemory tests, NLRR4/ mice showed defects in the hippocam-\npus-dependent memory (contextual fear-conditioning test), but\nnot the hippocampus-independent memory (tone fear-condition-\ning test). These results indicated that NLRR4/ mice are defec-\ntive in hippocampus-dependent fear memories. We then exam-\nined another type of hippocampus-dependent memory without\nthe experience of fear in NLRR4/ mice. We employed the\nhidden-platform Morris water maze test, which is a hippocampus-\ndependent learning task. Wild-type and NLRR4/ mice im-\nproved their performance during 5 days of training at 4 trials/day\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nFIG. 6. Spatial learning and memory in the hidden-platform Morris water maze task. (a) Average of the escape latency for each day for\nhidden-platform Morris water maze training sessions. Four trials were performed each day for 5 consecutive days (n  20 for each genotype). No\ndifference was observed between the two genotypes, indicating that the mutant mice learned equally well. All data points represent the mean (\nstandard error of the mean). (b) The probe tests were performed at three different retention time points (1, 4, and 7 days after the last day of\nacquisition for the hidden-platform task). Different groups of mice were used at each time point. One day after training, no difference was observed\nbetween the genotypes (n  10 for each genotype), indicating that the learning and acquisition of initial memory are normal for NLRR4/ mice.\nFour and seven days after the last training, the time that NLRR4/ mice spent in the target quadrant was signiﬁcantly less than that of wild-type\nmice. Statistical signiﬁcance was calculated by the Scheffe’s test following two-way analysis of variance. *, P  0.05. (n  10 for each genotype and\neach time point). (c) Swimming speeds were calculated from these movement data. No differences were observed between the genotypes during\nthe training sessions. (d) Time spent in each quadrant of the pool 4 days after training. The time spent by NLRR4/ mice in the target quadrant\nwas shorter than that by the wild-type control mice.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\n(Fig. 6a). There was no signiﬁcant difference in escape latencies\nbetween the two genotypes on any day during training. For the\nprobe test on 1 day after training, no signiﬁcant difference be-\ntween the two genotypes was observed for the time spent in the\ntarget quadrant, indicating that cellular consolidation of long-\nterm memory in NLRR4/ mice was normal. However, the\nNLRR4/ mice spent less time in the target quadrant than that\nof wild-type mice 4 days after the training (Fig. 6b). NLRR4/\nmice spent almost the same time in all quadrants 4 days after the\ntraining (Fig. 6d), showing that NLRR4/ mice randomly swam\nin all quadrants. Swimming speed in the training session was not\ndifferent between the two genotypes (Fig. 6c). These results indi-\ncate that NLRR4 is required for hippocampus-dependent long-\nlasting memory.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nNormal hippocampal long-term potentiation in NLRR4/\nmutants. The ﬁnding that NLRR4/ mice showed normal\nshort-term memory but\nimpaired long-lasting memory\nprompted us to examine whether the physiological properties\nof the hippocampus were altered by the lack of NLRR4. We\nﬁrst examined the input-output relationship of synaptic trans-\nmission to assess possible changes in basal synaptic properties,\nusing hippocampal slices. AMPA receptor-mediated excitatory\npostsynaptic potentials in the CA1 region in response to vari-\nintensities were indistinguishable between\nous\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nstimulus\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nNLRR4/ and wild-type mice (Fig. 7a), indicating that the\nbasal synaptic efﬁcacy was not altered in NLRR4/ mice.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\ntransmitter\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.7**r== 0.1**\nrelease probability were intact\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nWe then examined paired-pulse facilitation, which is a pre-\nsynaptic form of short-term plasticity and is sensitive to pre-\nsynaptic release probability (20). The paired-pulse facilitation\nobserved in slices from NLRR4/ mice did not signiﬁcantly\ndiffer from that of wild-type mice at any interstimulus interval\nexamined (Fig. 7b), indicating that presynaptic properties in-\ncluding\nin\nNLRR4/ mice. Furthermore, we analyzed long-term synap-\ntic plasticity and found that a single high-frequency stimulus\ntrain (100 Hz/s) produced long-term potentiation in mutant\nslices that was indistinguishable from that of wild-type slices\n(Fig. 7c and d), which is consistent with the results that\nNLRR4/ mice performed normally hippocampus-depen-\ndent memory tasks 24 h after the training. Since there was a\ntendency of decreased long-term potentiation 60 min after the\ninduction in NLRR4/ mice, we also followed the time\ncourse of long-term potentiation for another hour, but found\nthat long-term potentiation 120 min after the induction was\nalso normal (Fig. 7e and f). Taken together, these results\nindicate that NLRR4 is not required for basal synaptic trans-\nmission or short-term and the early phase of long-term plas-\nticity at the Schaffer collateral-CA1 synapse in the hippocampus.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.7**l== 0.1**r== 0.1**\nFIG. 7. Normal basal synaptic transmission, paired-pulse facilitation and long-term potentiation in NLRR4/ mice. (a) The input-output\nrelationships of AMPA receptor-mediated excitatory postsynaptic potentials of wild-type (open circles, n  12) and mutant (closed circles, n \n9) mice. Shown are the mean synaptic responses to stimuli of various strengths for the NLRR4/ and wild-type (/) slices. There was no\nsigniﬁcant difference between the two genotypes. Insets display representative traces evoked with four different stimulus intensities of 1.8 to 3.42\nV. (b) Paired-pulse facilitation induced by stimulating afferent ﬁbers twice at 50, 100, and 200 ms in wild-type (open circles) and NLRR4/ (solid\ncircles) mice. The ratio of the slope of the second excitatory postsynaptic potentials to that of the ﬁrst excitatory postsynaptic potentials was\ncalculated (at 50-ms interstimulus interval (ISI): NLRR4/, 1.41  0.04%, n  7; wild-type, 1.45  0.05%, n  10, P  0.6; at 100-ms ISI:\nNLRR4/, 1.38  0.04%, n  7; wild-type, 1.39  0.03%, n  10, P  0.8; at 200-ms ISI: NLRR4/, 1.26  0.03%, n  7; wild-type, 1.29 \n0.04%, n  10, P  0.6). (c) The averaged time course of long-term potentiation for 60 min after tetanic stimulation for wild-type (open circle,\nn  20) and NLRR4/ (solid circle, n  18) mice. At time zero, tetanic stimulation (100 Hz, 1 s) was delivered to the Schaffer collateral/\ncommissural pathway. Initial excitatory postsynaptic potentials slopes were normalized for each experiment to the averaged slope value during the\nbaseline period (30 to 0 min). Representative traces (average of 10 consecutive responses) in the inset were excitatory postsynaptic potentials\nobtained at the times indicated by the numbers on the graph. (d) Summary of long-term potentiation calculated as the percent increase in the mean\nexcitatory postsynaptic potential slope from 50 to 60 min after tetanic stimulation compared with the mean excitatory postsynaptic potential slope\nduring the baseline period (NLRR4/, 137.8  6.5% of baseline, n  18; wild type, 146.7  3.9% of baseline, n  20, P  0.2). (e) The averaged\ntime course of long-term potentiation for 120 min after tetanic stimulation for wild-type (open circle, n  5) and NLRR4/ (solid circle, n  6)\nmice. This graph consists of a part of the data shown in Fig. 8C, which was obtained from the slices with 1 h of additional recording. (f) Summary\nof long-term potentiation calculated as the percent increase in the mean excitatory postsynaptic potential slope from 110 to 120 min after tetanic\nstimulation compared with the mean excitatory postsynaptic potential slope during the baseline period (NLRR4/, 143.2  9.8% of baseline, n\n 6; wild-type, 146.3  7.2% of baseline, n  5, P  0.8).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nWe isolated a cDNA encoding a new member of the NLRR\nfamily, a type I transmembrane protein with leucine-rich re-\npeats and a ﬁbronectin type III repeat. Leucine-rich repeats\nconsist of multiple copies of the conserved sequence. Proteins\nwith leucine-rich repeats mediate homophilic or heterophilic\nprotein-protein interactions. Slits function as a ligand for the\nRobo receptors through its leucine-rich repeat (2, 11), and\nNgR and Trks are receptors for Nogo and neurotrophic factors\n(6, 17), respectively. CAPS and connectin function as cell rec-\nognition molecules in neuromuscular development via their\nhomophilic interaction (23, 27). We therefore tested a ho-\nmophilic interaction of NLRR4 protein. However, cell aggre-\ngation assays using NLRR4-expressing cells failed to show any\ninteraction (data not shown), suggesting that NLRR4 may be\ninvolved in cell-cell interactions as a component or it may be a\nreceptor or a ligand for an unknown molecule.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nNLRR4-deﬁcient mice generated by inserting the -galac-\ntosidase cDNA into the NLRR4 gene locus developed nor-\nmally and exhibited no anatomical abnormalities. The NLRR4\nexpression revealed by -galactosidase activity indicates that\nNLRR4 is expressed in the hippocampus, layers V and VI in\nthe cortex and piriform cortex, the internal granule layer, and\nPurkinje cell in the cerebellum. The hippocampus plays a role\nin the initial storage and the cellular consolidation of spatial\ninformation (22, 26). Since NLRR4 is highly expressed in the\nhippocampus, it is possible that NLRR4 is involved in learning\nand/or memory. In fact, we found that\nthe memory of\nNLRR4/ mice was severely impaired for two learning tests\nthat are dependent on the hippocampus; the contextual fear-\nconditioning and hidden-platform Morris water maze tests.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nIn the contextual fear-conditioning test, NLRR4/ mice\nwere able to learn normally as shown by the normal freezing\nbehavior 1 day after training. However, NLRR4/ mice failed\nto retain the memory longer as their freezing behavior was\nsigniﬁcantly reduced 4 and 7 days after training. We obtained\nsimilar results in the hidden-platform Morris water maze\ntest. Thus,\nin the hippocampus-dependent behavioral tests\nNLRR4/ mice are able to learn and retain memories nor-\nmally memory 1 day after training. Consistently, NLRR4/\nmice showed normal long-term potentiation in the hippocam-\npus, indicating that initial memory and cellular consolidation\nare normal in NLRR4/ mice. Contextual and spatial mem-\nories are known to be retained in the hippocampus for several\ndays and long-lasting memory is permanently stored in the\ncortex (7, 8). As memory retention 4 days after training was\nseverely impaired in NLRR4/ mutant mice, it is likely that\nNLRR4 is required for long-lasting memory.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nIn contrast to hippocampus-dependent memories, NLRR4/\nmice showed normal cued fear memory in which a tone was\npaired with an electric foot shock. This task depends on the\namygdala but is independent of the hippocampus. It is known\nthat hippocampus lesion results in impaired spatial memory\nbut normal memory in the cued fear-conditioning tests (19).\nLikewise, the hippocampal CA1 region-speciﬁc knockout of\nthe NR1 gene encoding an essential subunit of the N-methyl-\nD-aspartate receptor results in amnesia of contextual fear\nmemory without affecting cued fear memory (26). Axon of\nCA1 pyramidal cells mainly extend to the layer V in entorhinal\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\ncortex and NLRR4 is expressed in CA1, CA3, and dentate\ngyrus in the hippocampus and in layers V and VI in the cortex.\nThese results collectively suggest that the impaired retention of\nmemories in NLRR4/ mice depends on the hippocampal\nfunction but is independent of the amygdala.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nAmnesia in NLRR4/ mice is similar to that in CaMKII\nheterozygous mice, which exhibit severely impaired hippocam-\npus-dependent\nlong-lasting memory, while the molecular\nmechanism how CaMKII is involved in long-lasting memory\nstill remains to be elucidated, there may be a link between\nNLRR and CaMKII. CREB is required for the cellular con-\nsolidation of long-term memory, indicating that a protein(s)\nregulated by CREB plays a role in long-term memory (1, 5,\n18). Therefore, the impaired memory in NLRR4/ mice\nmight be due to an alteration in this pathway. As NLRR4 is a\ntransmembrane protein with protein-protein interaction mo-\ntifs, it is tempting to speculate that NLRR4 may generate an\nintracellular signal that affects the CREB pathway in the hip-\npocampus. It will be of interest to ﬁnd a molecule that interacts\nwith NLRR4.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\nACKNOWLEDGMENTS\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nWe thank S. Kida for helpful discussion and critical reading of the\nmanuscript and T. Fukuda, T. Watanabe, H. Kawano, and T. Sato for\ntechnical advice. We are also grateful to T. Akiyama and his staff for\ntheir help for behavioral tests and to Y. Iwakura for providing us with\na mouse genomic DNA library and the LacZ plasmid.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThis work was supported in part by Grants-in-Aid for Scientiﬁc\nResearch, the 21st Century COE program, the Center for Biosignaling\nand Integrated Brain Medical Science, and Special Coordination\nFunds for Promoting Science and Technology from the Ministry of\nEducation, Culture, Sports and Technology, by RISTEX and CREST\nof the Japan Science and Technology Agency, and by the Inoue Foun-\ndation for Science.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n1. Abel, T., and E. Kandel. 1998. Positive and negative regulatory mechanisms\nthat mediate long-term memory storage. Brain Res. Brain Res. Rev. 26:360–\n378.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n2. Battye, R., A. Stevens, R. L. Perry, and J. R. Jacobs. 2001. Repellent signal-\ning by Slit requires the leucine-rich repeats. J. Neurosci. 21:4290–4298.\n3. Best, P. J., A. M. White, and A. Minai. 2001. Spatial processing in the brain:\nthe activity of hippocampal place cells. Annu. Rev. Neurosci. 24:459–486.\n4. Bormann, P., L. W. Roth, D. Andel, M. Ackermann, and E. Reinhard. 1999.\nzfNLRR, a novel leucine-rich repeat protein is preferentially expressed dur-\ning regeneration in zebraﬁsh. Mol. Cell. Neurosci. 13:167–179.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n5. Bourtchuladze, R., B. Frenguelli, J. Blendy, D. Ciofﬁ, G. Schutz, and A. J.\nSilva. 1994. Deﬁcient long-term memory in mice with a targeted mutation of\nthe cAMP-responsive element-binding protein. Cell 79:59–68.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n6. Fournier, A. E., T. GrandPre, S. M. Strittmatter. 2001. Identiﬁcation of a\nreceptor mediating Nogo-66 inhibition of axonal regeneration. Nature 409:\n341–346.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n7. Frankland, P. W., C. O’Brien, M. Ohno, A. Kirkwood, and A. J. Silva. 2001.\n-CaMKII-dependent plasticity in the cortex is required for permanent\nmemory. Nature. 411:309–313.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n8. Frankland, P. W., B. Bontempi, L. E. Talton, L. Kaczmarek, and A. J. Silva.\n2004. The involvement of the anterior cingulate cortex in remote contextual\nfear memory. Science 304:881–883.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n9. Fukamachi, K., Y. Matsuoka, C. Kitanaka, Y. Kuchino, and H. Tsuda. 2001.\nRat neuronal leucine-rich repeat protein-3: cloning and regulation of the\ngene expression. Biochem. Biophys. Res. Commun. 287:257–263.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n10. Fukamachi, K., Y. Matsuoka, H. Ohno, T. Hamaguch, and H. Tsuda. 2002.\nNeuronal leucine-rich repeat protein-3 ampliﬁes MAPK activation by epi-\ndermal growth factor through a carboxyl-terminal region containing endo-\ncytosis motifs. J. Biol. Chem. 277:43549–43552.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n11. Guthrie, S. 1999. Axon guidance: starting and stopping with slit. Curr. Biol.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n9:R432–435.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n12. Hara, T., Y. Nakano, M. Tanaka, K. Tamura, T. Sekiguchi, K. Minehata,\nN. G. Copeland, N. A. Jenkins, M. Okabe, H. Kogo, Y. Mukouyama, and A.\nMiyajima. 1999. Identiﬁcation of podocalyxin-like protein 1 as a novel cell\nsurface marker for hemangioblasts in the murine aorta-gonad-mesonephros\nregion. Immunity 11:567–578.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n13. Harada, N., B. E. Castle, D. M. Gorman, N. Itoh, J. Schreurs, R. L. Barrett,\nM. Howard, and A. Miyajima. 1990. Expression cloning of a cDNA encoding\nthe murine interleukin 4 receptor based on ligand binding. Proc. Natl. Acad.\nSci. USA. 87:857–861.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n14. Hayata, T., T. Uochi, and M. Asashima. 1998. Molecular cloning of\nXNLRR-1, a Xenopus homolog of mouse neuronal leucine-rich repeat pro-\ntein expressed in the developing Xenopus nervous system. Gene 221:159–\n166.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n15. Ishii, N., A. Wanaka, and M. Tohyama. 1996. Increased expression of\nNLRR-3 mRNA after cortical brain injury in mouse. Brain Res. Mol. Brain\nRes. 40:148–152.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n16. Kang, H., L. D. Sun, C. M. Atkins, T. R. Soderling, M. A. Wilson, and S.\nTonegawa. 2001. An important role of neural activity-dependent CaMKIV\nsignaling in the consolidation of long-term memory. Cell 106:771–783.\n17. Kaplan, D. R., and R. M. Stephens. 1994. Neurotrophin signal transduction\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nby the Trk receptor. J. Neurobiol. 25:1404–1417.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n18. Kida, S., S. A. Josselyn, S. P. de Ortiz, J. H. Kogan, I. Chevere, S. Masushige,\nand A. J. Silva. 2002. CREB required for the stability of new and reactivated\nfear memories. Nat. Neurosci. 5:348–355.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n19. Kim, J. J., and M. S. Fanselow. 1992. Modality-speciﬁc retrograde amnesia\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nof fear. Science 256:675–677.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n20. Manabe, T., D. J. Wyllie, D. J. Perkel, and R. A. Nicoll. 1993. Modulation of\nsynaptic transmission and long-term potentiation: effects on paired pulse\nfacilitation and EPSC variance in the CA1 region of the hippocampus.\nJ. Neurophysiol. 70:1451–1549.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n21. Matthies, H. 1989. In search of cellular mechanisms of memory. Prog.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nNeurobiol. 32:277–349.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n22. McGaugh, J. L. 2000. Memory–a century of consolidation. Science 287:248–\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n251.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n23. Nose, A., V. B. Mahajan, and C. S. Goodman. 1992. Connectin: a homophilic\ncell adhesion molecule expressed on a subset of muscles and the motoneu-\nrons that innervate them in Drosophila. Cell. 70:553–567.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n24. Scoville, W. B., and B. Milner. 1957. Loss of recent memory after bilateral\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nhippocampal lesions. J. Neurochem. 20:11–21.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n25. Sekine, K., H. Ohuchi, M. Fujiwara, M. Yamasaki, T. Yoshizawa, T. Sato, N.\nYagishita, D. Matsui, Y. Koga, N. Itoh, and S. Kato. 1999. Fgf10 is essential\nfor limb and lung formation. Nat. Genet. 21:138–141.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n26. Shimizu, E., Y. P. Tang, C. Rampon, and J. Z. Tsien. 2000. NMDA receptor-\ndependent synaptic reinforcement as a crucial process for memory consoli-\ndation. Science 290:1170–1174.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n27. Shishido, E., M. Takeichi, and A. Nose. 1998. Drosophila synapse formation:\nregulation by transmembrane protein with Leu-rich repeats, CAPRICIOUS.\nScience 280:2118–2121.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n28. Taguchi, A., A. Wanaka, T. Mori, K. Matsumoto, Y. Imai, T. Tagaki, and M.\nTohyama. 1996. Molecular cloning of novel leucine-rich repeat proteins and\ntheir expression in the developing mouse nervous system. Brain Res. Mol.\nBrain Res. 35:31–40.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n29. Taniguchi, H., M. Tohyama, and T. Takagi. 1996. Cloning and expression of\na novel gene for a protein with leucine-rich repeats in the developing mouse\nnervous system. Brain Res. Mol. Brain Res. 36:45–52.",
         "extracted",
         "",
         "Neuronal Leucine-Rich Repeat Protein 4 Functions in Hippocampus-Dependent Long-Lasting Memory",
         "None"
        ],
        [
         "11",
         "000fd65de92d766ada2dab424f7f66e526e2f063",
         "We present an encoding framework which exploits semantics for video content delivery. The video content is organized based on the idea of main content message. In the work reported in this paper, the main content message is extracted from the video data through semantic video analysis, an application-dependent process that separates relevant information from non relevant information. We use here semantic analysis and the corresponding content annotation under a new perspective: the results of the analysis are exploited for object-based encoders, such as MPEG-4, as well as for frame-based encoders, such as MPEG-1. Moreover, the use of MPEG-7 content descriptors in conjunction with the video is used for improving content visualization for narrow channels and devices with limited capabilities. Finally, we analyze and evaluate the impact of semantic video analysis in video encoding and show that the use of semantic video analysis prior to encoding sensibly reduces the bandwidth requirements compared to traditional encoders not only for an object-based encoder but also for a frame-based encoder.",
         "A. Cavallaro,Olivier Steiger,T. Ebrahimi",
         "https://infoscience.epfl.ch/record/87259/files/Cavallaro2005_1275.pdf",
         "\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 23.9**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nSemantic video analysis for adaptive content\ndelivery and automatic description\n**BLOCK**fs== 11.0**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\nAndrea Cavallaro, Olivier Steiger, Touradj Ebrahimi\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nAbstract— We present an encoding framework which exploits\nsemantics for video content delivery. The video content is orga-\nnized based on the idea of main content message. In the work\nreported in this paper, the main content message is extracted from\nthe video data through semantic video analysis, an application-\ndependent process that separates relevant information from non\nrelevant information. We use here semantic analysis and the\ncorresponding content annotation under a new perspective: the\nresults of the analysis are exploited for object-based encoders,\nsuch as MPEG-4, as well as for frame-based encoders, such as\nMPEG-1. Moreover, the use of MPEG-7 content descriptors in\nconjunction with the video is used for improving content visual-\nization for narrow channels and devices with limited capabilities.\nFinally, we analyze and evaluate the impact of semantic video\nanalysis in video encoding and show that the use of semantic\nvideo analysis prior to encoding sensibly reduces the bandwidth\nrequirements compared to traditional encoders not only for an\nobject-based encoder but also for a frame-based encoder.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nIndex Terms— Video analysis, video encoding, object segmen-\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\ntation, metadata, MPEG.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.2**r== 0.6**\nI. INTRODUCTION\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nThe diffusion of network appliances such as cellular phones,\npersonal digital assistants, and hand-held computers creates a\nnew challenge for content delivery: how to adapt the media\ntransmission to various device capabilities, network charac-\nteristics, and user preferences [1], [2], [3]. Each device is\ncharacterized by certain display capabilities and processing\npower. Moreover, such appliances are connected through dif-\nferent kinds of networks with diverse bandwidths. Finally,\nusers with different preferences access the same multimedia\ncontent. Therefore there exists a need to personalize the way\nmedia content is delivered to the end user. In addition to the\nabove, recent devices, such as digital radio receivers, and new\napplications, such as intelligent visual surveillance, require\nnovel forms of video analysis for content adaptation and sum-\nmarization. Digital radios allow for the display of additional\ninformation alongside the traditional audio stream to enrich the\naudio content. For instance, digital audio broadcasting (DAB)\nallocates 128 Kb/s to streaming audio, whereas 8Kb/s can\nbe used to send additional information, such as visual data\n[4]. Moreover, the growth of video surveillance systems poses\nchallenging problems for the automatic analysis, interpretation\nand indexing of video data as well as for selective content\nﬁltering for privacy preservation. Finally, the instantaneous\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nA. Cavallaro is with the Multimedia and Vision Laboratory, Queen Mary,\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nUniversity of London (QMUL), E1 4NS London, United Kingdom.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nO. Steiger and T. Ebrahimi are with the Signal Processing Institute, Swiss\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFederal Institute of Technology (EPFL), CH-1015 Lausanne, Switzerland.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nindexing of video content is also an desirable feature for sports\nbroadcasting [5].\n**BLOCK**fs== 10.0**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nTo cope with these challenges, video content needs to be\nautomatically analyzed and adapted to the needs of the speciﬁc\napplication, to the capabilities of the connected terminal and\nnetwork, and to the preferences of the user. Three main\nstrategies for adaptive content delivery have been proposed\nthroughout the literature, namely Info Pyramid, scalable cod-\ning and transcoding. The work presented in this paper aims\nto go beyond traditional adaptation techniques. We focus on\nsemantic encoding by looking to exploit video analysis prior\nto encoding (Figure 1). Speciﬁcally, we use semantic video\nanalysis to extract relevant areas of a video. These areas are\nencoded at a higher level of quality or summarized in textual\nform. The idea behind this approach is to organize the content\nso that a particular network or device does not inhibit the main\ncontent message. The main content message is dependent on\nthe speciﬁc application. In particular, for applications such as\nvideo surveillance and sport video the main content message\nis deﬁned based on motion information.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThe contribution of this paper is twofold. On the one\nhand, a framework for adaptive video delivery is deﬁned and\nimplemented based on video objects and on their associated\nmetadata. On the other hand, two new modalities of video\ndelivery are proposed in such a framework. The ﬁrst modality\ncombines semantic analysis with a traditional frame-based\nvideo encoder. The second modality uses metadata to efﬁ-\nciently encode the main content message. In particular, the\nuse of metadata enables not only to make the content more\nsearchable, but also to improve visualization and to preserve\nprivacy in video-based applications.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nThe paper is organized as follows. Section II is an overview\nof existing adaptation techniques. Section III presents the\nalgorithm for extracting the main content message, the frame-\nwork for adaptive video delivery and automatic description\nusing semantic video analysis. Section IV discusses quality\nassessment issues, whereas experimental results are presented\nin Section V. Finally, Section VI concludes the paper.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nII. BACKGROUND\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThree main approaches have been presented in the literature\nto provide adaptive content delivery, namely Info Pyramid,\nscalable coding and transcoding. Info Pyramid provides a\ngeneral framework for managing and manipulating media\nobjects [6], [7]. Info Pyramid manages different versions, or\nvariations, of media objects with different modalities (e.g.,\nvideo, image, text, and audio) and ﬁdelities (summarized, com-\npressed, and scaled variations). Moreover, it deﬁnes methods\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nVideo\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nSemantic\nvideo analysis\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nForeground\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nBackground\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nBackground\nsimplification\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nM\nU\nX\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nM\nU\nX\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nFrame-based\nencoder\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nTemporal\ndownsampling\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nSpatial\ndownsampling\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nCompositing\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nObject-based\nencoder\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nMetadata\nencoder\n**BLOCK**fs== 6.4**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\nStill image\nencoder\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n(1-5)\n**BLOCK**fs== 6.4**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n(8)\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n(6, 7)\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n(9)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nFig. 1. Flow diagram of the proposed encoding framework based on semantic video analysis and description\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nfor manipulating, translating, transcoding, and generating the\ncontent. When a client device requests a multimedia document,\nthe server selects and delivers the most appropriate variation.\nThe selection is made based on network characteristics and\nterminal capabilities, such as display size, frame rate, color\ndepth and storage capacity.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.5**l== 0.1**r== 0.5**\nAs opposed to Info Pyramid, scalable coding processes\nmultimedia content only once. Lower qualities, lower spatial\nand temporal resolutions of the same content are then obtained\nby truncating certain layers or bits from the original stream [8].\nBasic modes of video scalability include quality scalability,\nspatial scalability, temporal scalability, and frequency scala-\nbility. Combinations of these basic modes are also possible.\nQuality scalability is deﬁned as the representation of a video\nsequence with varying accuracies in the color patterns. This\nis typically obtained by quantizing the color values with\nincreasingly ﬁner quantization step sizes. Spatial scalability\nis the representation of the same video in varying spatial\nresolutions. Temporal scalability is the representation of the\nsame video at varying temporal resolutions or frame rates.\nFrequency scalability includes different frequency components\nin each layer, with the base layer containing low-frequency\ncomponents and the other layers containing increasingly high-\nfrequency components. Such decomposition can be achieved\nvia frequency transforms like the DCT or wavelet transforms.\nFinally, the basic scalability schemes can be combined to reach\nﬁne-granularity scalability, such as in MPEG–4 FGS [9].\nThe various scalable coding methods introduced previously\nperform the same operation over the entire video frame. In\nobject-based temporal scalability (OTS), the frame rate of\nforeground objects is enhanced so that it has a smoother\nmotion than the background.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nVideo transcoding is the process of converting a compressed\nvideo signal into another compressed signal with different\nproperties. Early solutions to video transcoding determine the\noutput format based on network and appliance constraints,\nindependently of the semantics in the content (content-blind\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\ntranscoding). Content-blind transcoding strategies include spa-\ntial resolution reduction, temporal resolution reduction, and\nbit-rate reduction [10]. Recent transcoding techniques make\nuse of semantics to minimize the degradation of important\nimage regions [11], [12]. In [13], optimal quantization pa-\nrameters and frame skip are determined for each video object\nindividually. The bit-rate budget for each object is allocated by\na difﬁculty hint, a weight indicating the relative encoding com-\nplexity of each object. Frame skip is controlled by a shape hint,\nwhich measures the difference between two consecutive shapes\nto determine whether an object can be temporally downsam-\npled without visible composition problems. Key objects are\nselected based on motion activity and on bit complexity.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThe transcoding strategies described thus far are referred\nto as intramedia transcoding strategies and do not change the\nmedia nature of the input signal. On the other hand, intermedia\ntranscoding, or transmoding,\nis the process of converting\nthe media input\ninto another media format. Examples of\nintermedia transcoding include speech-to-text [14] and video-\nto-text [15] translation. Both the intramedia and the intermedia\nadaptation concepts are used in this paper for video encoding,\nas described in the following section.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nIII. ADAPTIVE CONTENT DELIVERY AND DESCRIPTION\nUSING SEMANTICS\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThe proposed framework for adaptive video delivery and\nautomatic description uses video content analysis and semantic\npre-ﬁltering prior to encoding (Figure 1) in order to improve\nthe perceived content quality and to provide additional func-\ntionalities, such as privacy preservation and automatic video\nindexing. Semantic video analysis and semantic encoding are\ndescribed next.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nA. Semantic video analysis\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nSemantic video analysis is used to extract the main content\nmessage from the video. The semantics to be included in\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nthe analysis is dependent on the speciﬁc application. In the\nfollowing, we discuss possible semantics and, in particular,\nwe describe the use of motion as semantics.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nSemantic video analysis refers to a human abstraction and\nuses a priori information to translate the semantics into rules.\nThe rules are then applied through an algorithm. Examples\nof semantic video analysis based on a priori information are\ntemplate matching, extraction of captions and text, face de-\ntection, and moving object segmentation. Template matching\nis used to implement the semantics when the shape objects\nwe want to segment is known a priori. In this case, which\nincludes in particular the detection of captions and text, the\nextraction method searches for speciﬁc object features in terms\nof geometry. For segmenting faces of people, color-based\nsegmentation can be used [16]. The face detection task consists\nin ﬁnding the pixels whose spectral characteristics lie in a\nspeciﬁc region in the chromaticity diagram. For extracting\nmoving objects, motion information can be used as semantics.\nSeveral applications, such as sport broadcasting and video\nsurveillance, deal with segmenting moving objects.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.4**l== 0.1**r== 0.5**\nA typical tool used to tackle the problem of object segmen-\ntation based on motion is change detection. Different change\ndetection techniques can be employed for moving camera\nand static camera conditions. If the camera moves, change\ndetection aims at recognizing coherent and incoherent moving\nareas. The former correspond to background areas, the latter\nto video objects. If the camera is static, the goal of change\ndetection is to recognize moving objects (foreground) and the\nstatic background. The semantic analysis we use addresses\nthe static camera problem and is applicable in the case of a\nmoving camera after global motion compensation. The change\ndetector decides whether in each pixel position the foreground\nsignal corresponding to an object is present. This decision\nis taken by thresholding the frame difference between the\ncurrent frame and a frame representing the background. The\nframe representing the background is dynamically generated\nbased on temporal information [17]. The thresholding aims\nat discarding the effect of the camera noise after frame\ndifferencing. A locally adaptive threshold, τ (i, j), is used that\nmodels the noise statistics and applies a signiﬁcance test. To\nthis end, we want to determine the probability that frame\ndifference at a given position (i, j) is due to noise, and not to\nother causes. Let us suppose that there is no moving object in\nthe frame difference. We refer to this hypothesis as the null\nhypothesis, H0. Let g(i, j) be the sum of the absolute values\nof the frame difference in an observation window of q pixels\naround (i, j). Moreover, let us assume that the camera noise is\nadditive and follows a Gaussian distribution with variance σ.\nGiven H0, the conditional pdf of the frame difference follows\na χ2\nq distribution with q degrees of freedom deﬁned by\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\nf\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nH0\ng(i, j)\n|\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n2q/2σqΓ(q/2)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.5**\ng(i, j)(q−2)/2e−g(i,j)\n**BLOCK**fs== 5.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n/2σ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n(1)\n\nwhere Γ(\n) is the Gamma function, that can be evaluated as\n·\nΓ(x+ 1) = xΓ(x), and Γ(1/2) = √π. To obtain a good trade-\noff between robustness to noise and accuracy in the detection\n5 window centered in (i, j)). It is now\nwe choose q = 25 (5\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n(a)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n(c)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.8**r== 0.1**\n(b)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n(d)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nFig. 2. Example of semantic video analysis results for the test sequences (a)\nHall Monitor and (c) Highway: (b) separation of foreground and background\nfor Hall Monitor. (d) separation of foreground and background for Highway.\nThe background is color-coded in black\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\npossible to derive the signiﬁcance test as\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\ng(i, j)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nP\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n{\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nH0}\nτ (i, j)\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n≥\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\nΓ(q/2, g(i, j)2/2σ2)\nΓ(q/2)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n(2)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nWhen this probability is smaller than a certain signiﬁcance\nlevel, α, we consider that H0 is not satisﬁed at the pixel\nposition (i, j). Therefore we label that pixel as belonging to a\nmoving object. The signiﬁcance level α is a stable parameter\nthat does not need manual tuning along a sequence or for\ndifferent sequences. Experimental results indicate that valid\nvalues fall in the range from 10−2 to 10−6.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThe change detection process produces the segmentation of\nthe moving objects from the background (Figure 2) and, cou-\npled with video object tracking [18], enables the subsequent\nextraction of object metadata, as described in the following\nsection.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nB. Semantic encoding\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe decomposition computed with semantic video analysis\nis used with an object-based encoder as well as with a\ntraditional frame-based encoder. We will refer to the former\ncase as object-based encoding and to the latter as frame-\nbased encoding. Furthermore, metadata are used to efﬁciently\nencode relevant information and to enhance relevant part of a\nlow-quality coded video. These approaches are referred to as\nmetadata-based encoding and metadata-enhanced encoding,\nrespectively. Relevant examples of the modalities presented\nin this section are illustrated in Figure 3. The analysis and\nevaluation of the different approaches in terms of results and\nbandwidth requirements are presented in Section V.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n1) Object-based encoding: With object-based encoding, the\nencoder needs to support the coding of individual video objects\n(e.g., MPEG–4 object-based). Each video object is assigned to\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n(a)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n(b)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n(c)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n(d)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n(e)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nFig. 3. Examples of encoding modalities. (a) Sample frame from the sequence Soccer; (b) Semantic frame-based encoding: the background is selectively\nlowpass-ﬁltered prior to encoding; (c) Metadata-based encoding: object shapes are superimposed on the background; (d) Spatial resolution reduction; (e)\nMetadata-enhanced encoding: metadata are used to enhance relevant portions of a video\n**BLOCK**fs== 10.0**b== 0.3**t== 0.3**l== 0.1**r== 0.5**\na distinct object class, according to its importance in the scene.\nThe encoding quality can be set depending on the object class:\nthe higher the relevance, the higher the encoding quality. One\nadvantage of this approach is the possibility of controlling\nthe sequencing of objects: video objects may be encoded\nwith different degree of compression, thus allowing better\ngranularity for the areas in the video that are of more interest to\nthe viewer. Moreover, objects may be decoded in their order\nof priority, and the relevant content can be viewed without\nhaving to reconstruct the entire image (network limitations).\nAnother advantage is the possibility of using a simpliﬁed\nbackground (appliance limitation), so as to enhance the rel-\nevant objects. Using a simpliﬁed background aims at taking\nadvantage of the task-oriented behavior of the human visual\nsystem for improving compression ratios. Recent work on\nfoveation [19] demonstrated that using nonlinear integration\nof low-level visual cues mimicking the processing in primate\noccipital and posterior parietal cortex allows one to sensibly\nincrease compression ratios. Moreover, the work reported in\n[20] demonstrated that an overall increase in image quality\ncan be obtained when the increase in quality of the relevant\nareas of an image more than compensates for the decrease in\nquality of the image background.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\n2) Semantic frame-based encoding: The semantic frame-\nbased encoding mode exploits semantics in a traditional frame-\nbased encoding framework (e.g., MPEG–1). The use of the\ndecomposition of the scene into meaningful objects prior\nto encoding, referred here as semantic pre-ﬁltering, helps\nsupport low bandwidth transmission. The areas belonging to\nthe foreground class, or semantic objects, are used as region\nof interest. The areas not included in the region of interest\nmay either be eliminated, that is set to a constant value, or\nlowered in importance by using a low-pass ﬁlter. The latter\nsolution simpliﬁes the information in the background, while\nstill retaining essential contextual information. An example\nof this solution is reported in Fig. 4 (a). On the other hand,\nﬁltering the entire image inhibits the main content message\nFig. 4 (b). Another way to take into account less relevant\nportions of an image before coding is to take advantage of the\nspeciﬁcs of the coding algorithm. In the case of block-based\ncoding, each background macroblock can be replaced by its\nDC value. Semantic frame-based encoding mimics the way\nhumans perceive visual information and allows for a reduction\nof information to be coded.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n(a)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.8**r== 0.1**\n(b)\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nFig. 4.\n(a) Selective lowpass-ﬁltering simpliﬁes the information in the\nbackground, while still retaining essential contextual information; (b) ﬁltering\nthe entire image inhibits the main content message\n**BLOCK**fs== 10.0**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\n3) Metadata-based and metadata-enhanced encoding: A\nfurther processing of the video content is performed to cope\nwith limited device or network capabilities as well as to\nautomatically generate metadata. Such processing transforms\nthe foreground objects extracted through semantic analysis\ninto quantitative descriptors and enables video annotation.\nVideo annotation is desirable for applications such as video\nsurveillance, where terabytes of data are produced and need\nto be searched quickly. Moreover, the descriptors can be\ntransmitted instead of the video content itself and superim-\nposed by the terminal on a still background. For example, an\nobject identiﬁer and a shape descriptor are used in [21]. The\nobject identiﬁer is a unique numerical identiﬁer describing\nthe spatial location of each object in the scene. The shape\ndescriptor is used to represent the shape of an object, ranging\nfrom a bounding box to a polygonal representation with a\ndifferent number of vertices (Figure 3(c)). This approach is\nuseful to preserve privacy in video surveillance applications\nas well as to reduce bandwidth requirements under critical\nnetwork conditions. A progressive representation is used where\nthe number of vertices corresponding to the best resolution\nis computed, and any number of vertices smaller that this\nmaximum can be used according to the requirements of the\napplication. In addition to the above, other features such as\ncolor and texture descriptors may be added in the description\nprocess. The choice of these additional features depends on\nthe application at hand.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nIn addition to the above, the descriptors can be transmitted\nalong with the video itself and used for rendering the video\ncontent. This solution, consisting in a mixture of video-based\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nand text-based modalities, is here referred toe as metadata-\nenhanced encoding. Using metadata-enhanced encoding, con-\ntent descriptors help enhance parts of the video that are hidden\nor difﬁcult to perceive due to heavy compression. In this case,\nthe video itself is the background and the descriptors highlight\nrelevant portions of the data. One example is the ball in a\nfootball match for transmission to a PDA or a mobile phone,\nas shown in Figure 3(e).\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nIV. QUALITY ASSESSMENT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nis a difﬁcult\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nPerceptual video quality assessment\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\ntask\nalready when dealing with traditional coders [22]. When\ndealing with object-based coders,\nthe task becomes even\nmore challenging. For this reason, we use a combination of\nsubjective and objective evaluation techniques to compare the\nperformance of the different encoding modalities. Subjective\nevaluation includes the visual comparison of frames and frame\ndetails. This analysis is performed at different bitrates and\nat different frame resolutions. Objective evaluation includes\ntemporal signal-to-noise ratio analysis and the analysis of rate-\ndistortion curves.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nA. Semantic peak signal-to-noise ratio\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nTraditional peak signal-to-noise ratio (PSNR) analysis uni-\nformly weights the contribution of each pixel in an image\nwhen computing the mean squared error (MSE). This analysis\ngives the same importance to relevant as well as less relevant\nareas of an image. To account for the way humans perceive\nvisual\ninformation, different areas of an image, or object\nclasses, should be considered [11]. We take into account\nobject classes through a distortion measure, the semantic mean\nsquared error, SMSE, deﬁned as:\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nN\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nSMSE =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nwk ·\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nMSEk,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n(3)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nP\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nXk=1\nwhere N is the number of object classes and wk the weight of\nclass k. Class weights are chosen depending on the semantics,\nN\ni=1 wk = 1. The mean\nwith wk ≥\nsquared error of each class, MSEk, can be written as\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nk = 1, . . . , N and\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n∀\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nMSEk =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nd2(i, j),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n(4)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nCk|\n|\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nCk| X(i,j)∈Ck\nwhere Ck is the set of pixels belonging to the object class k\nand\nis its cardinality. The class membership of each pixel\n(i, j) is deﬁned by semantic video analysis. The error d(i, j)\nbetween the original image IO and the distorted image ID in\nEq.(4) is the pixel-wise color distance. The color distance is\ncomputed in the 1976 CIE Lab color space in order to consider\nperceptually uniform color distances with the Euclidean norm\nand is expressed as:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nd(i, j) =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n∆I L(i, j)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n+\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n∆I a(i, j)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n+\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n∆I b(i, j)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nq\n\n\nD(i, j), ∆I a(i, j) = I a\nI L\nwith ∆I L(i, j) = I L\n−\nD(i, j), and ∆I b(i, j) = I b\nI a\nO(i, j)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n\nO(i, j)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n\nO(i, j)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n,\n(5)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.5**r== 0.3**\nevaluation metric,\nSPSNR, is the following:\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nthe semantic peak signal-to-noise ratio,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nSPSNR = 10 log10\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nV 2\nM\nSMSE !\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n(6)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nwhere VM is the maximum peak-to-peak value of the color\nrange. When the object classes are foreground and back-\nground, then N = 2 in Eq (3). If we denote with wf the\nforeground weight, then SPSNR\nPSNR when wf = 0.5.\nThe larger wf , the more important the contribution of the\nforeground. When wf = 1,\nthen the foreground only is\nconsidered in the evaluation of the peak signal-to-noise ratio.\nAn illustration of the impact of wf in the distortion measure\nis shown in in Fig. 5. The ﬁgure presents a comparison of the\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n≡\n**BLOCK**fs== 4.9**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 4.9**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 4.9**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 1.4**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n]\nb\nd\n[\n**BLOCK**fs== 3.3**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nR\nN\nS\nP\nS\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n(1)\n(2)\n(3)\n(4)\n(5)\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n0.5\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n0.55\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n0.6\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n0.65\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n0.8\n0.75\n0.7\nForeground weight\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n0.85\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n0.9\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n0.95\n**BLOCK**fs== 4.9**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nIllustration of the impact of wf in the distortion measure: average\nFig. 5.\nSPSNR vs. foreground weight for Hall monitor sequence. The ﬁve labels\ncorrespond to the following sequence types: (1) coded original; (2) tempo-\nrally down-sampled; (3) spatially down-sampled; (4) static background; (5)\nsimpliﬁed background. Content-blind coding methods (1)-(3) decrease their\nperformance when the foreground is given more importance. Methods based\non semantic, (4) and (5), increase their performance when the foreground is\ngiven more importance\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\naverage SPSNR of the sequence Hall Monitor for the different\nencoding modalities described in Section III-B as function of\nwf . The value of wf to be used is estimated as described in\nthe next section.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nB. Determination of the foreground weight\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nSubjective performance evaluation experiments have been\nperformed to estimate the foreground weight leading to the\nclosest match between SPSNR prediction and human judg-\nment. Twenty non-expert observers of different ages and\nbackgrounds have been presented a series of video sequences\naccording to ITU-T Recommendation P.910, Absolute Cate-\ngory Rating [23]. The evaluation has been carried out using\nthe MPEG–4 test sequences Akiyo, Hall Monitor, Children,\nand Coastguard. Video sequences have been generated using\nthe encoding strategies described in Section III-B, at different\nbitrates, and rated by the observers on a scale ranging from 0\n(bad) to 100 (excellent). This range of values was presented\nto the observers in a training phase.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe foreground weight, wf , is determined for each test\nsequence by maximizing the Pearson correlation [24] between\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 10.0**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\nSPSNR and subjective results. The results are summarized in\nTable I. For the sequence Akiyo, where the foreground covers\na large area of each frame and the background is simple, the\nthe observers focused mostly on foreground, thus leading to a\nvalue of wf = 0.97. For Hall Monitor, whose background is\nmore complex and objects are smaller, the foreground attracted\nslightly more the attention than the background (wf = 0.55).\nThe sequence Children has a very complex and colored back-\nground that attracted the observers’ attention, thus resulting\nin foreground and background being equally weighted (wf =\n0.5). The sequence Coastguard contains camera motion that\nprevented the observer from focusing on background steadily,\neven though the background is quite complex. In this case, the\nresulting foreground weight is wf = 0.7. In general, results\nconﬁrm that large moving objects and complex background\ntend to attract user’s attention. Based on the data collected with\nsubjective experiments, it is possible to predict the foreground\nweight based on the following formula:\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nwf = α\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nr + (δ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nβ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n·\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n·\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nr)σb + γ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nv + δ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n·\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n(7)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n+\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nCb|\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\nCb|\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nCf |\n/(\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nwhere r represents the portion of the image occupied by\nforeground pixels, expressed as r =\n), with\nCf |\n|\nrepresenting the number of foreground and\nCf |\n|\nbackground pixels, respectively. The background complexity\nis taken into account with σb, the standard deviation of the\nluminance of background pixels. The presence of camera\nmotion is considered with the term v: v = 1 for moving\ncamera, and v = 0 otherwise. α, β, γ, and δ are constants\nwhose values are determined based on the results of the\nsubjective experiments and are the following: α = 5.7, β =\n0.108, γ = 0.2 and δ = 0.01. The ﬁnal value of wf is the\naverage of the foreground weights over the sequence.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nIn addition to semantic weight, Table I provides information\nabout accuracy, monotonicity and consistency of the SPSNR\nmetric. Accuracy is given by Pearson linear correlation coefﬁ-\ncient rp, monotonicity by Spearman rank-order correlation co-\nefﬁcient rs, and consistency by outliers ratio ro [24]. Pearson\ncorrelation of PSNR, rp(0.5), is given for comparison. Pearson\ncorrelation rp and Spearman correlation rs are close to 1 for\nall sequences. Thus, accuracy and monotonicity of SPSNR are\nhigh. Outliers ratio ro is around 10%, thus consistency of the\nmetric is good as well. Note that using semantics improves\naccuracy by up to 8% (Akiyo), as compared to PSNR.\n**BLOCK**fs== 6.4**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nTABLE I\nFOREGROUND WEIGHT AND SPSNR ACCURACY\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nwf\nrp(wf )\nrp(0.5)\nrs(wf )\nro(wf )\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\nAkiyo\n0.97\n0.95\n0.87\n0.90\n0.10\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.7**\nHall monitor\n0.55\n0.90\n0.89\n0.84\n0.11\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nChildren\n0.50\n0.95\n0.95\n0.95\n0.07\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\nCoastguard\n0.7\n0.92\n0.90\n0.93\n0.07\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.2**r== 0.6**\nV. EXPERIMENTAL RESULTS\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nIn this section, experimental results of the proposed seman-\ntic video encoding and annotation framework with standard\ntest sequences are presented. The results illustrate the impact\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nof semantic analysis on the encoding performance of frame-\nbased as well as object-based coders and demonstrate the use\nof the proposed approach for advanced applications, such as\nprivacy preservation in video surveillance. Sample results are\nshown from the MPEG–4 test sequence Hall Monitor and\nfrom the MPEG–7 test sequence Highway. Both sequences\nare in CIF format at 25 Hz. The modalities under analysis are:\n(1) coded original sequence; (2) temporal resolution reduction\n(from 25 frames/s. to 12.5 frames/s.); (3) spatial resolution\nreduction (from CIF to QCIF); (4,6) video objects composited\nwith static background; (5,7) video objects composited with\nsimpliﬁed background. The background is simpliﬁed using a\nGaussian 9x9 low-pass ﬁlter with µ = 0 and σ = 2.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nThe following coders have been used in the encoding\nprocess: (i) TMPGEnc 2.521.58.169 using constant bitrate\n(CBR) rate control for frame-based MPEG–1; (ii) MoMuSys\nMPEG-4 VM reference software version 1.0 using VM5+\nglobal rate control for object-based MPEG–4; (iii) Expway\nMPEG-7 BiM Payload encoder/decoder version 02/11/07 for\nMPEG–7 metadata; (iv) Kakadu JPEG2000 codec version\n4.2 for JPEG200 still images. The value of the foreground\nweight used in the objective evaluation is wf = 0.55 for\nHall Monitor, as determined with the subjective experiments,\nand wf = 0.53 for Highway, computed using Eq. (7) with\nr = 0.07, σb = 48, v = 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nFigure 6 shows the rate-distortion diagrams for the test\nsequences. The average SPSNR for ﬁve encoding modalities is\nplotted against the encoding bitrate. Figures 6 (a) and (b) show\nthe rate-distortion diagrams for MPEG–1 at bitrates between\n150 Kbit/s and 1000 Kbit/s. At low bitrates (150-300 Kbit/s),\nsemantic encoding with static background (4) leads to a larger\nSPSNR than any of the content-blind methods (1-3). This is\nbecause inter-coded static background blocks do not produce\nresidue and most of the available bitrate can be allocated to\nforeground objects. In Figures 6 (c) and (d), foreground and\nbackground are encoded in two separate streams using object-\nbased MPEG–4 at bitrates between 100 Kbit/s and 500 Kbit/s.\nHere semantic analysis is used in all ﬁve modalities. It possible\nto notice that quality is improved at low bitrates by low-pass\nﬁltering the background or using a still frame representing the\nbackground.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nFigure 7 shows a sample frame from each test sequence\ncoded with MPEG–1 at 150 Kbit/s with and without semantic\npre-ﬁltering. Figure 8 shows magniﬁed excepts of both test\nsequences coded with MPEG–1 at 150 Kbit/s. Figure 8 (top)\nshows the person that carries a monitor in Hall monitor. The\namount of coding artifacts is notably reduced by semantic\npre-ﬁltering ((d) and (e)). In particular, the person’s mouth\nand the monitor are visible in (e), whereas they are corrupted\nby coding artifacts in the non-semantic modalities. Similar\nobservations can be made for Figure 8 (bottom), which shows\na blue truck entering the scene at the beginning of the Highway\nsequence. Coding artifacts are less disturbing on the object in\n(d) and (e) than in (a)-(c). Moreover, the front-left wheel of\nthe truck is only visible with semantic pre-ﬁltering ((d) and\n(e)).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nNext, we evaluate the cost of sending metadata for metadata-\nbased and metadata-enhanced encoding. Table II shows the\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 5.8**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 1.6**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n]\n**BLOCK**fs== 3.2**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nb\nd\n**BLOCK**fs== 1.6**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n[\n**BLOCK**fs== 3.8**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nR\nN\nS\nP\nS\n**BLOCK**fs== 1.6**b== 0.5**t== 0.4**l== 0.1**r== 0.9**\n]\nb\nd\n[\n**BLOCK**fs== 3.8**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nR\nN\nS\nP\nS\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n(1) Original sequence\n(2) 12.5 frames/s.\n(3) QCif\n(4) Lowpass−filtered bckg\n(5) Static bckg\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nBitrate [Kbit/s]\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n(a)\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n(1) Original sequence\n(2) 12.5 frames/s.\n(3) QCif\n(6) Lowpass−filtered bckg\n(7) Static bckg\n**BLOCK**fs== 5.8**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 1.6**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n]\n**BLOCK**fs== 3.2**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nb\nd\n**BLOCK**fs== 1.6**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n[\n**BLOCK**fs== 3.8**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nR\nN\nS\nP\nS\n**BLOCK**fs== 1.6**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n]\nb\nd\n[\n**BLOCK**fs== 3.8**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nR\nN\nS\nP\nS\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n(1) Original sequence\n(2) 12.5 frames/s.\n(3) QCif\n(4) Lowpass−filtered bckg\n(5) Static bckg\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nBitrate [Kbit/s]\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n(b)\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n(1) Original sequence\n(2) 12.5 frames/s.\n(3) QCif\n(6) Lowpass−filtered bckg\n(7) Static bckg\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nBitrate [Kbit/s]\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n(d)\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nBitrate [Kbit/s]\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(c)\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 5.8**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFig. 6. Rate-distortion diagrams. (a) Hall monitor, MPEG–1; (b) Highway, MPEG–1; (c) Hall monitor, MPEG–4 object-based; (d) Highway, MPEG–4\nobject-based\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nlocators of the foreground objects,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nbitrate required by three types of description for Hall Monitor\nand Highway using MPEG–7 binary format (BiM). MPEG–\n7 binary format is used for sending summary information\nto terminals with limited capabilities and to enhance heavily\ncompressed videos. The descriptions are represented by the\nspatial\ntheir bounding\nboxes, and an approximation of their shape with 20-sided\npolygons, respectively. The metadata size increases with the\ndescription complexity and with the number of objects in the\nscene (Hall Monitor vs. Highway). The cost for metadata-\nenhanced encoding can be further reduced by sending the\ndescription of critical objects only. In addition to the above,\n**BLOCK**fs== 6.4**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAVERAGE BITRATE OF MPEG–7 BIM SEQUENCE DESCRIPTION\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nTABLE II\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nDESCRIPTION\nHall monitor\nHighway\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\nSpatial locator\n21 Kbit/s\n26 Kbit/s\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nBounding box\n59 Kbit/s\n66 Kbit/s\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\nPolygon shape\n89 Kbit/s\n98 Kbit/s\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nlevel of information hiding obtained using object descriptors\nfor the sequence Hall Monitor. A surveillance operator can\nbe shown different video types, ranging from the full appear-\nance of the objects (Figure 9 (a)) to the visualization of a\nposition locator that allows the operator to derive statistics\nabout number of objects, their behavior and position without\ndisclosing their identity (Figure 9 (d)). Intermediate levels of\nvisualization include the approximation of object shapes that\nhides the identity of the subjects captured by the surveillance\ncamera, while allowing to derive information about their size\nand form (Figure 9 (b)), and the bounding box (Figure 9 (c)).\nThe encoding cost associated with this additional functionality\nadded to a surveillance system is 21 Kbit/s for the spatial\nlocator, 59 Kbit/s for the bounding box and 89 Kbit/s for the\npolygonal shape. The choice of the description to be used\ndepends on the trade-off between privacy and the monitoring\ntask at hand.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nVI. CONCLUSIONS\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nmetadata-enhanced encoding is used for privacy preservation\nin video surveillance. Figure 9 shows an example of different\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nWe presented a content-based video encoding framework\nwhich is based on semantic analysis. Semantic analysis enables\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n(a)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n(b)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n(c)\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nFig. 7. Frame 190 of Hall monitor (top) and frame 44 of Highway (bottom) coded with MPEG–1 at 150 Kbit/s using different modalities: (a) coded original\nsequence; (b) static background; (c) simpliﬁed background\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nthe decomposition of a video into meaningful objects. Using\nthis decomposition, the encoder may adapt its behavior to code\nrelevant and non relevant objects differently. Three modalities\nof video delivery have been discussed, analyzed, and com-\npared using standard encoders. The ﬁrst exploits semantics\nin traditional frame-based encoding. Semantically pre-ﬁltering\nthe video prior to coding leads to signiﬁcant improvements in\nvideo compression efﬁciency in terms of bandwidth require-\nments as well as visual quality at low bitrates. The second\nmodality uses metadata to efﬁciently encode relevant infor-\nmation. Object descriptors are generated for content retrieval\nas well as used for coding at very low bit-rates or for devices\nwith limited capabilities. The third modality combines video\nand metadata for visualization. Metadata are used for content\nenhancement at low bitrates and for preserving privacy in\nvideo surveillance applications.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nIn the speciﬁc implementation discussed in Section V, the\nsemantics is deﬁned by motion. Given the modularity of the\nproposed encoding framework other semantics can also be\nused in the analysis step. Examples are face detection and\ntext segmentation.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe quality metric used in this work is a promising ﬁrst step\ntowards measuring the quality taking semantics into account.\nFuture work includes the study and deﬁnition of a perceptual\nmetric that accounts for user satisfaction, depending on the\napplication and the user preferences. To this end, an object of\ninterest metric, such as that used in [3], will be an important\nbuilding block of the overall quality metric. This quality\nmetric will be used to automatically select the best encoding\ntechnique that maximizes user experience.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n[1] P. van Beek, J. Smith, T. Ebrahimi, T. Suzuki, and J. Askelof, “Metadata-\ndriven multimedia access,” IEEE Signal Processing Magazine, pp. 40–\n52, March 2003.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[2] R. Mohan, J. Smith, and C.-S. Li, “Adapting multimedia internet content\nfor universal access,” IEEE Transactions on Multimedia, vol. 1, no. 1,\npp. 104–114, 1999.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[3] A. Vetro, H. Sun, and Y. Wang, “Object-based transcoding for adaptable\nvideo content delivery,” IEEE Transactions on Circuits and Systems for\nVideo Technology, vol. 11, no. 3, pp. 387–401, 2001.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n[4] I. P. Duncumb, P. F. Gadd, G. Wu, and J. L. Alty, “Visual radio: Should\nwe paint pictures with words, or pictures?,” Tech. Rep., Loughborough\nUniversity, UK, 2004.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[5] Gopal S. Pingali, Agata Opalach, Yves D. Jean, and Ingrid B. Carlbom,\n“Instantly indexed multimedia databases of real world events,” IEEE\nTransactions on Multimedia, vol. 4, no. 2, pp. 269–282, June 2002.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[6] Chung-Sheng Li, Rakesh Mohan, and John R. Smith,\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\n“Multimedia\ncontent description in the Info Pyramid,”\nthe IEEE\nInternational Conference on Accoustics, Speech and Signal Processing,\nMay 1998, pp. 171–178.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\nin Proc. of\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n[7] John R. Smith, Rakesh Mohan, and Chung-Sheng Lj,\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n“Scalable\nmultimedia delivery for pervasive computing,” in Proc. of the ACM\nconference on Multimedia, Oct.-Nov. 1999, vol. 1, pp. 131–140.\n[8] Yao Wang, J¨orn Ostermann, and Ya-Qin Zhang, Video Processing and\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nCommunications, Prentice Hall, 1 edition, 2001.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n[9] W. Li,\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\n“Overview of ﬁne granularity scalability in MPEG–4 video\nstandard,” IEEE Trans. on Circuits and Systems for Video Technology,\nvol. 11, no. 3, pp. 301–317, March 2001.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n[10] A. Vetro, C. Christopoulos, and H. Sun, “Video transcoding architectures\nand techniques: an overview,” IEEE Signal Processing Magazine, vol.\n20, no. 2, pp. 18–29, March 2003.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[11] R. Cucchiara, C. Grana, and A. Prati, “Semantic transcoding for live\nin Proceedings of ACM Multimedia, Juan–Les–Pins\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.2**\nvideo server,”\n(France), December 2002, pp. 223–226.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[12] Andrea Cavallaro, Olivier Steiger, and Touradj Ebrahimi, “Semantic\nsegmentation and description for video transcoding,” in Proceedings of\nthe IEEE International Conference on Multimedia and Expo, July 2003,\nvol. 3, pp. 597–600.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[13] A. Vetro, H. Sun, and Y. Wang, “Object-based transcoding for adaptable\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n(a)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n(b)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n(c)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n(d)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\n(e)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFig. 8. Details of frame 280 of Hall monitor (top) and frame 16 of Highway (bottom). The sequences are encoded with MPEG–1 at 150 Kbit/s using\ndifferent encoding modalities: (a) coded original sequence; (b) temporal resolution reduction; (c) spatial resolution reduction; (d) static background; (e)\nsimpliﬁed background\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n(a)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n(b)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n(c)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\n(d)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nFig. 9. Example of use of the proposed encoding framework for privacy preservation in an indoor surveillance application. Four different methods are shown\nrepresenting different privacy levels. a) Video objects; (b) Object shape; (c) Bounding box; (d) Object position. The method can also be used to adapt the\nvideo delivery to the channel capacity and the terminal characteristics.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nvideo content delivery,” IEEE Trans. on Circuits and Systems for Video\nTechnology, vol. 11, no. 3, pp. 387–401, March 2001.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n[14] N. Morgan and H. Bourlard, “Continuous speech recognition,” IEEE\nSignal Processing Magazine, vol. 12, no. 3, pp. 24–42, May 1995.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\nMay 2004.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[16] R.L. Hsu, M.Abdel-Mottaleb, and A. Jain, “Face detection on color\nimages,” IEEE Trans. on Pattern Analysis and Machine Intelligence,\nvol. 24, no. 5, pp. 696–706, 2002.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n[15] K. Jung, K.I. Kim, and A.K. Jain, “Text information extraction in images\nand video: a survey,” Pattern Recognition, vol. 37, no. 5, pp. 977–997,\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[17] A. Cavallaro and T. Ebrahimi, “Video object extraction based on adap-\ntive background and statistical change detection,” in Proceedings of SPIE\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nSUBMITTED TO IEEE TRANSACTIONS ON CIRCUITS AND SYSTEMS FOR VIDEO TECHNOLOGY\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nElectronic Imaging - Visual Communications and Image Processing, San\nJose, California, USA, 2001, pp. 465–475.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n[18] A. Cavallaro, O. Steiger, and T. Ebrahimi, “Multiple object tracking in\ncomplex scenes,” in Proceedings of ACM Multimedia, Juan–Les–Pins\n(France), December 2002, pp. 523–532.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n[19] L. Itti, “Automatic foveation for video compression using a neurobio-\nlogical model of visual attention,” IEEE Trans. on Image Processing,\nvol. 13, no. 10, pp. 1304–1318, 2004.\n[20] A.P. Bradley and F.W.M. Stentiford,\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n“Visual attention for region of\ninterest coding in jpeg 2000,” Journal of Visual Communication and\nImage Representation, vol. 14, pp. 232–250, 2003.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n[21] O. Steiger, A. Cavallaro, and T. Ebrahimi, “MPEG-7 description of\ngeneric video objects for scene reconstruction,” in Proceedings of SPIE\nElectronic Imaging, San Jose, California, USA, January 2002, pp. 223–\n226.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\n[22] S. Olsson, M. Stroppiana, and J. Baina,\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n“Objective methods for\nassessment of video quality : state of the art,” IEEE Transactions on\nBroadcasting, vol. 43, no. 4, pp. 487–495, 1997.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n[23] ITU,\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n“Subjective video quality assessment methods for multimedia\napplications,” Tech. Rep. P.910, ITU-T Recommandation, September\n1999.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n[24] David Freedman, Robert Pisani, and Roger Purves, Statistics, W.W.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nNorton & Company, 3 edition, 1997.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n[25] Yap-Peng Tan, Yongqing Liang, and Haiwei Sun, “On the methods\nand performances of rational downsizing video transcoding,” Signal\nProcessing: Image Communications, vol. 19, pp. 47–65, 2004.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n[26] Anthony Vetro, Toshihiko Hata, Naoki Kuwahara, Hari Kalva, and Shun-\nIchi Sekiguchi, “Complexity-quality analysis of transcoding architec-\ntures for reduced spatial resolution,” in IEEE Transactions on Consumer\nElectronics, August 2002, pp. 515–521.\n[27] P. A. A. Assunc¸ ˜ao and M. Ghanbari,\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n“A frequency-domain video\ntranscoder for dynamic bit-rate reduction of MPEG–2 bit streams,” IEEE\nTrans. on Circuits and Systems for Video Technology, vol. 8, no. 8, pp.\n953–967, Dec. 1998.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n[28] T. Shanableh and M. Ghanbari, “Heterogeneous video transcoding to\nlower spatio-temporal resolutions and different encoding formats,” IEEE\nTrans. on Multimedia, vol. 2, no. 2, pp. 101–110, June 2000.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n[29] Y. Liang and Y.-P. Tan, “A new content-based hybrid video transcoding\nmethod,” in Proc. of IEEE Int. Conf. on Image Processing 2001, Oct.\n2001, vol. 1, pp. 429–432.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n[30] A. Cavallaro, E. Salvador, and T. Ebrahimi, “Shadow detection in image\nsequences,” in Proc. of IEE Conference on Visual Media Production,\nLondon, UK, 2004, pp. 165–174.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n[31] A. Vetro and H. Sun, “Media conversions to support mobile users,” in\nProc. of IEEE Canadian Conf. on Electrical and Computer Engineering,\nCCECE 2001, May 2001, vol. 1, pp. 607–612.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n[32] R. Mohan, J.R. Smith, and C.-S. Li, “Adapting multimedia internet\ncontent for universal access,” IEEE Trans. on Multimedia, vol. 1, no. 1,\npp. 104–114, March 1999.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n[33] H. Sun, A. Vetro, and K. Asai, “Resource adaptation based on MPEG–\n21 usage environment description,” in Proc. IEEE Int. Symposium on\nCircuits and Systems, May 2003, vol. 2, pp. 536–539.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n[34] ISO/IEC, “Information technology – generic coding of moving pictures\nand associated audio information: Video, 2nd ed.,” Tech. Rep. ISO/IEC\nFDIS 13818-2:2000, ISO/IEC JTC 1/SC29/WG11, 2000.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n“Information technology – coding of audio-visual objects\n– part 2 visual–amendment 2: Streaming video proﬁles,” Tech. Rep.\nISO/IEC FDIS 14496-2:2001, ISO/IEC JTC 1/SC29/WG11, 2001.\n[36] Julien Bourgeois, Emmanuel Mory, and Franc¸ois Spies, “Video trans-\nmission adaptation on mobile devices,” Journal of System Architecture,\nvol. 49, pp. 475–484, 2003.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n“Dave: A system for quality\ndriven adaptive video delivery,” Proceedings of the 5th ACM SIGMM\ninternational workshop on Multimedia information retrieval, pp. 223–\n230, 2003.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n“Video transcoding for\nuniversal multimedia access,” in ACM Multimedia Workshop, 2000, pp.\n75–79.",
         "extracted",
         "Automatic foveation for video compression using a neurobiological model of visual attention;Text information extraction in images and video: a survey;DAVE: a system for quality driven adaptive video delivery;Video transmission adaptation on mobile devices;Visual attention for region of interest coding in JPEG 2000;Semantic segmentation and description for video transcoding;Resource adaptation based on MPEG-21 usage environment descriptions;Video transcoding architectures and techniques: an overview;Metadata-driven multimedia access;Semantic transcoding for live video server;Multiple video object tracking in complex scenes;Complexity-quality analysis of transcoding architectures for reduced spatial resolution;Instantly indexed multimedia databases of real world events;Face detection in color images;MPEG-7 description of generic video objects for scene reconstruction;A new content-based hybrid video transcoding method;Video Processing and Communications;Media conversions to support mobile users;Object-based transcoding for adaptable video content delivery;Overview of fine granularity scalability in MPEG-4 video standard;Video object extraction based on adaptive background and statistical change detection;Video transcoding for universal multimedia access;Heterogeneous Video Transcoding to Lower Spatio-Temporal Resolutions and Different Encoding Formats;Scalable multimedia delivery for pervasive computing;Adapting Multimedia Internet Content for Universal Access;A frequency-domain video transcoder for dynamic bit-rate reduction of MPEG-2 bit streams;Multimedia content description in the InfoPyramid;Objective methods for assessment of video quality : state of the art;On the methods and performances of rational downsizing video transcoding;Shadow detection in image sequences;Video Transcoding Architectures and Techniques : An Overview;Visual radio: should we paint pictures with workds, or pictures?;Dave: A system for qual ity driven adaptive video delivery,”Proceedings of the 5th ACM SIGMM international workshop on Multimedia information retriev;Processing and;Information technology – coding of audio-visual objects – part 2 visual–amendment 2: Streaming video profiles;Statistics, 3 ed;Continuous speech recognit  io;Submitted by;Information technology — Generic coding of moving pictures and associated audio information — Part 2: Video;SUBJECTIVE VIDEO QUALITY ASSESSMENT METHODS FOR MULTIMEDIA APPLICATIONS",
         "Semantic video analysis for adaptive content delivery and automatic description",
         "None"
        ],
        [
         "12",
         "0010aa2b264826b06d3839c2f139da8286930131",
         "ABSTRACT Of the nasopharyngeal cultures recovered from 942 day care center (DCC) attendees in Lisbon, Portugal, 591 (62%) yielded Streptococcus pneumoniae during a surveillance performed in February and March of 1999. Forty percent of the isolates were resistant to one or more antimicrobial agents. In particular, 2% were penicillin resistant and 20% had intermediate penicillin resistance. Multidrug resistance to macrolides, lincosamides, and tetracycline was the most frequent antibiotype (17% of all isolates). Serotyping and molecular typing by pulsed-field gel electrophoresis were performed for 202 out of 237 drug-resistant pneumococci (DRPn). The most frequent serotypes were 6B (26%), 14 (22%), 19F (16%), 23F (10%), and nontypeable (12%). The majority (67%) of the DRPn strains were representatives of nine international clones included in the Pneumococcal Molecular Epidemiology Network; eight of them had been detected in previous studies. Fourteen novel clones were identified, corresponding to 26% of the DRPn strains. The remaining 7% of the strains were local clones detected in our previous studies. Comparison with studies conducted since 1996 in Portuguese DCCs identified several trends: (i) the rate of DRPn frequency has fluctuated between 40 and 50%; (ii) the serotypes most frequently recovered have remained the same; (iii) nontypeable strains appear to be increasing in frequency; and (iv) a clone of serotype 33F emerged in 1999. Together, our observations highlight that the nasopharynxes of children in DCCs are a melting pot of successful DRPn clones that are important to study and monitor if we aim to gain a better understanding on the epidemiology of this pathogen.",
         "S. Nunes,R. Sá-Leão,João André Carriço,C. Alves,R. Mato,A. B. Avô,J. Saldanha,J. Almeida,I. Sanches,H. de Lencastre",
         "https://europepmc.org/articles/pmc1081291?pdf=render",
         "\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nJOURNAL OF CLINICAL MICROBIOLOGY, Mar. 2005, p. 1285–1293\n0095-1137/05/$08.000 doi:10.1128/JCM.43.3.1285–1293.2005\nCopyright © 2005, American Society for Microbiology. All Rights Reserved.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.1**\nTrends in Drug Resistance, Serotypes, and Molecular Types of\nStreptococcus pneumoniae Colonizing Preschool-Age Children\nAttending Day Care Centers in Lisbon, Portugal: a Summary\nof 4 Years of Annual Surveillance\nS. Nunes,1 R. Sa´-Lea˜o,1 J. Carric¸o,2 C. R. Alves,1 R. Mato,1 A. Brito Avoˆ,3 J. Saldanha,4\nJ. S. Almeida,2,5 I. Santos Sanches,1,6 and H. de Lencastre1,7*\nLaborato´rio de Gene´tica Molecular1 and Grupo de Biomatema´tica,2 Instituto de Tecnologia Quı´mica e Biolo´gica,\nUniversidade Nova de Lisboa, and Centro de Sau´de de Oeiras,3 Oeiras, Hospital de Santa Maria, Lisbon,4 and\nFaculdade de Cieˆncias e Tecnologia, Universidade Nova de Lisboa, Monte da Caparica,6 Portugal;\nDepartment of Biometry and Epidemiology, Medical University of South Carolina,\nCharleston, South Carolina5; and Laboratory of Microbiology, The Rockefeller\nUniversity, New York, New York7\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\nReceived 14 September 2004/Returned for modiﬁcation 18 October 2004/Accepted 5 November 2004\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.2**\nOf the nasopharyngeal cultures recovered from 942 day care center (DCC) attendees in Lisbon, Portugal, 591\n(62%) yielded Streptococcus pneumoniae during a surveillance performed in February and March of 1999. Forty\npercent of the isolates were resistant to one or more antimicrobial agents. In particular, 2% were penicillin\nresistant and 20% had intermediate penicillin resistance. Multidrug resistance to macrolides, lincosamides,\nand tetracycline was the most frequent antibiotype (17% of all isolates). Serotyping and molecular typing by\npulsed-ﬁeld gel electrophoresis were performed for 202 out of 237 drug-resistant pneumococci (DRPn). The\nmost frequent serotypes were 6B (26%), 14 (22%), 19F (16%), 23F (10%), and nontypeable (12%). The majority\n(67%) of the DRPn strains were representatives of nine international clones included in the Pneumococcal\nMolecular Epidemiology Network; eight of them had been detected in previous studies. Fourteen novel clones\nwere identiﬁed, corresponding to 26% of the DRPn strains. The remaining 7% of the strains were local clones\ndetected in our previous studies. Comparison with studies conducted since 1996 in Portuguese DCCs identiﬁed\nseveral trends: (i) the rate of DRPn frequency has ﬂuctuated between 40 and 50%; (ii) the serotypes most\nfrequently recovered have remained the same; (iii) nontypeable strains appear to be increasing in frequency;\nand (iv) a clone of serotype 33F emerged in 1999. Together, our observations highlight that the nasopharynxes\nof children in DCCs are a melting pot of successful DRPn clones that are important to study and monitor if\nwe aim to gain a better understanding on the epidemiology of this pathogen.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nThe ecological niche of Streptococcus pneumoniae is the na-\nsopharynxes of humans. Although pneumococci are widely\nspread in the community, carriage rates are particularly high in\npreschool children attending day care centers (DCCs) (24, 45).\nS. pneumoniae can cause a spectrum of diseases including\ninvasive disease, such as meningitis, bloodstream infections,\nand pneumonia, as well as infections of the upper respiratory\ntract. S. pneumoniae is a major cause of morbidity and mor-\ntality in young children (6). In the last decade an increasing\nproportion of pneumococci that are not susceptible to penicil-\nlin has been observed, reaching up to 50% in some areas of the\nworld (1, 10, 16, 37, 39, 43).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nIn 1996 our group began yearly surveillance of the nasopha-\nryngeal carriage of pneumococci and other bacterial respira-\ntory tract pathogens in DCCs located in the Lisbon area of\nPortugal. Since 1996, a total of 23 DCCs and over 1,800 chil-\ndren participated in this study. Serotypes, antibiotypes, and\nmolecular types of the majority of drug-resistant S. pneu-\nmoniae isolates were determined during these annual surveil-\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n* Corresponding author. Mailing address: The Rockefeller Univer-\nsity, 1230 York Ave., New York, NY 10021. Phone: (212) 327-8278.\nFax: (212) 327-8688. E-mail: lencash@rockefeller.edu.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nlances (8, 9, 34, 35), allowing one to obtain a view of temporal\nchanges of drug-resistant pneumococci (DRPn) inhabiting the\nnasopharyngeal ﬂora, an important ecological reservoir which\nmay be a major source of drug-resistant strains causing pedi-\natric and adult disease. The importance of such surveillances\nfor both the national and international communities has been\nhighlighted in recent reports (15, 32, 38).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nIn the study described here we provide new data on the\nnasopharyngeal carriage of drug-resistant pneumococci during\nthe 1999 surveillance. We also summarize temporal changes\nnoted in the same surveillance system between 1996 and 1999.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nMATERIALS AND METHODS\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nStudy population. Between February and March of 1999, nasopharyngeal\nsamples were recovered from children attending DCCs in the Lisbon area.\nLisbon is the capital of Portugal and, according to the last census (of 2001),\napproximately 2 million people live in the city and its suburbs. The 14 DCCs were\nselected to be of diverse geographic locations and sizes and to include children\nof different social strata. The target population was 1,454 children, of which a\ntotal of 942 children (480 females and 462 males), with ages ranging from 6\nmonths to 6 years (average, 3.3 years old), participated in the study.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nQuestionnaire. Questionnaires on antibiotic consumption habits of the chil-\ndren were addressed to their parents and/or guardians at the time of sampling.\nThe questions were as follows. (i) Is the child taking any antibiotics now? (ii) Has\nthe child taken antibiotics within the last month? (iii) Has the child received\nrepeated courses (more than three) of antibiotics in the last 6 months? (iv) If the\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nchild took antibiotics recently, which antibiotic did the child take and what was\nthe reason?\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nNasopharyngeal swab and isolation of S. pneumoniae. Nasopharyngeal secre-\ntions were taken by a pediatric nurse using swabs and modiﬁed Stuart’s medium\n(Mini-Tip Culturette; Becton Dickinson Microbiology Systems, Cockeysville,\nMd.). Swabs were transferred to the central research laboratory at the Instituto\nde Tecnologia Quı´mica e Biolo´gica and were inoculated into appropriate culture\nmedia within 4 h after the sample collection, to select and identify S. pneumoniae.\nBlood agar with gentamicin (5 mg/ml) and anaerobic incubation were used for\nselective culture of pneumococci. The pneumococci were identiﬁed by optochin\nsusceptibility and colony morphology. One colony was picked and a pure culture\nwas frozen. All isolates were frozen in Mueller-Hinton broth (Difco, Detroit,\nMich.) and 15% (vol/vol) glycerol (Merck, Darmstadt, Germany) and kept at\n70°C (8).\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nAntimicrobial susceptibility testing. Susceptibility testing was performed using\nthe Kirby-Bauer technique, according to the NCCLS recommendations and\ndeﬁnitions (28). The antimicrobial agents tested were chloramphenicol, eryth-\nromycin, clindamycin, tetracycline, and sulfamethoxazole-trimethoprim (SXT).\nAntibiotic disks were purchased from Oxoid (Hampshire, England). Isolates\nwere also screened for MICs of penicillin and ceftriaxone with the E-test (AB\nBiodisk, Solna, Sweden) according to the manufacturer’s recommendations. In\nthe interpretation of decreased penicillin susceptibility, isolates were divided into\nintermediately resistant (0.1 g/ml  MIC  1.5 g/ml) and resistant (MIC \n1.5 g/ml) strains, according to NCCLS guidelines (28).\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nSerotyping. Strains resistant to at least one of the antimicrobial agents tested\n(except those resistant only to SXT) were serotyped by the Quellung reaction\nusing commercially available antisera (Statens Seruminstitut, Copenhagen, Den-\nmark) (40).\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nPFGE. The strains that were serotyped were also typed by pulsed-ﬁeld gel\nelectrophoresis (PFGE). Preparation of chromosomal DNA, restriction with\nSmaI endonuclease, and PFGE were done as previously described (35). PFGE\npatterns were assigned by visual inspection of the macrorestriction proﬁles, by\nusing currently accepted criteria (42). The particular PFGE patterns, which were\ndetected in previous studies (8, 35), received the same uppercase letter; the new\nPFGE patterns were assigned two distinct uppercase letters.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nMLST. Multilocus sequence typing (MLST) was performed on selected iso-\nlates and interpreted as described previously (14, 33). DNA sequencing was done\neither at the Rockefeller University Protein/DNA Technology Center (New\nYork, N.Y.) or at Macrogene, Inc. (Seoul, Korea).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nRESULTS\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nStudy population. Between February and March of 1999,\n942 children were screened, corresponding to 65% of the chil-\ndren attending the 14 DCCs studied. The number of children\nenrolled at each day care center varied from 44 to 178, with an\naverage of 104 children. Participation rates in this study in each\nDCC ranged from 40 to 75%, and the number of children\nsampled in each center ranged from 20 to 132.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nPneumococcal carriage and resistance to antimicrobial\nagents. A total of 591 children (63%) of the 942 sampled were\npneumococcal carriers. Of the 591 pneumococcal isolates, 60%\nwere susceptible to all antimicrobial agents tested, 18% were\nsusceptible to penicillin but resistant to at least one of the\nother antimicrobials, and 22% had decreased susceptibility to\npenicillin: the penicillin MIC for 2% of the isolates was equal\nto or higher than 1.5 g/ml and 20% had intermediate resis-\ntance. Rates of resistance to antimicrobial agents other than\npenicillin were 26% for tetracycline, 25% for SXT, 8.5% for\nchloramphenicol, and 2% for ceftriaxone. Concerning macro-\nlide-lincosamide resistance, 22% of the isolates were resistant\nto erythromycin and 20.5% were resistant to clindamycin. Mul-\ntidrug resistance (deﬁned as resistance to three or more classes\nof antimicrobial agents) was common (26% of all pneumococ-\ncal isolates) and frequently (17%) included resistance to eryth-\nromycin, clindamycin, and tetracycline (Table 1).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nCarriage of antibiotic-resistant S. pneumoniae varied widely\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nAntibiotype resistant toa:\n**BLOCK**fs== 8.0**b== 0.5**t== 0.2**l== 0.5**r== 0.3**\nPG R, SXT\nPG R, CHL, TET, SXT\nPG I, CHL, ERY, CLI, TET, SXT\nPG I, CHL, ERY, CLI, TET\nPG I, CHL, ERY, CLI, SXT\nPG I, ERY, CLI, TET, SXT\nPG I, CHL, TET, SXT\nPG I, ERY, CLI, TET\nPG I, ERY, CLI, SXT\nPG I, ERY, TET, SXT\nPG I, CHL, SXT\nPG I, CHL, TET\nPG I, ERY, CHL\nPG I, ERY, TET\nPG I, TET, SXT\nPG I, ERY\nPG I, TET\nPG I, SXT\nPG I\nCHL, ERY, CLI, TET, SXT\nCHL, ERY, CLI, TET\nERY, CLI, TET, SXT\nERY, CLI, TET\nERY, CLI, SXT\nERY, TET, SXT\nERY, CLI\nERY, SXT\nTET, SXT\nERY\nSXT\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nNo. of\nisolates\n**BLOCK**fs== 8.0**b== 0.5**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nSusceptible to all antimicrobial agents\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\na PG R, penicillin MIC  1.5g/ml; PG I, penicillin 0.1 g/ml  MIC  1.5\ng/ml; TET, tetracycline; ERY, erythromycin; CLI, clindamycin; CHL, chlor-\namphenicol. Multidrug resistance to erythromycin, clindamycin, and tetracycline\nis highlighted in bold.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nfrom one DCC to another (Fig. 1), reinforcing the idea that\nDCCs are autonomous epidemiological units (35). In DCC17,\nfor example, all strains recovered (n  13) were susceptible to\npenicillin, but half of them were resistant to at least another\nantimicrobial agent. By contrast, in DCC19, all strains (n  17)\nbut one were susceptible to all antimicrobial agents tested; in\nDCC22, similar proportions of isolates susceptible to all drugs\n(n  13), susceptible to penicillin but resistant to some other\ndrugs (n  17), and intermediately resistant to penicillin (n \n13) were found. We used chi-square and Fisher exact tests to\ndetermine whether this diversity was associated with antibiotic\nintake in the month prior to the sampling and found no sig-\nniﬁcant associations at a 95% conﬁdence level (data not\nshown).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nQuestionnaire. Seven percent of the 942 children were tak-\ning antimicrobials at the time of sampling; close to 23% of the\nchildren had taken antimicrobial agents in the previous month,\nand 21% had taken three or more courses of antimicrobial\nagents in the previous 6 months. The antibiotics most fre-\nquently reported by the parents were amoxicillin (39%) and\namoxicillin-clavulanic acid (41%); antibiotic use was mainly\ndue to otitis media (50%) and tonsillitis (24%).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nSerotyping and molecular typing of DRPn isolates. A total\nof 202 DRPn out of 237 were characterized by serotyping and\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nFIG. 1. Distribution of antimicrobial resistance among S. pneumoniae isolates recovered in different day care centers. White bars, isolates\nresistant to penicillin; gray bars, isolates intermediately resistant to penicillin; dashed bars, isolates susceptible to penicillin but resistant to other\nantimicrobial agents; black bars, isolates susceptible to all antimicrobial agents tested.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nPFGE. The 35 out of 237 DRPn that were resistant to SXT\nonly were not further characterized, because previous studies\ndemonstrated a high degree of diversity among SXT-resistant\nstrains in terms of serotypes and molecular types, suggesting\nfrequent independent emergence of resistance to this drug (8).\nAmong the 12 serotypes found the most frequent were 6B\n(26%), 14 (22%), 19F (16%), and 23F (10%). Other capsular\ntypes detected were 3 (1%), 6A (1%), 9V (4%), 15A (4%),\n15B (1%), 19A (1%), 23A (1%), and 33F (2%). Twelve\npercent of the DRPn were nontypeable.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nPFGE analysis grouped the isolates in 29 patterns, some of\nwhich are shown in Fig. 2. Table 2 summarizes relevant prop-\nerties of the 202 DRPn strains, grouping them by level of\nresistance to penicillin and PFGE pattern. Serotypes, antibi-\notic resistance proﬁles, and distribution of isolates by DCC are\nalso shown. PFGE patterns were associated with particular\nserotypes; the association with antibiotypes was less obvious. In\nall DCCs but one (DCC19, in which only one drug-resistant\nstrain was detected), four or more PFGE patterns were iden-\ntiﬁed among the DRPn isolates.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nBy comparing the 29 PFGE patterns (clonal types) with\npatterns from previous studies (8, 34), the 29 clones were\ngrouped as follows: representatives of Pneumococcal Molecu-\nlar Epidemiology Network (PMEN) clones; Lisbon DCC\nclones, which included strains that do not belong to PMEN\nclones and have PFGE patterns identiﬁed in our previous\nstudies (8, 34); and novel clones, which included strains with\nPFGE proﬁles that we could not identify in our previous stud-\nies (1996 to 1998). The majority (67%) of drug-resistant pneu-\nmococci were PMEN clones, 26% were novel clones, and 7%\nwere Lisbon DCC clones. PMEN clones were present in all day\ncare centers and, together, they accounted from 40 to 100% of\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nthe DRPn strains in various DCCs, thus representing the ma-\njority of the isolates. Strains belonging to the Lisbon DCC\nclones were detected in 5 DCCs, and the novel clones were\nidentiﬁed in 11 of the 14 DCCs (Table 3). Some relevant\nproperties of the clones of each group are described below.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nPMEN clones. Nine PMEN clones were detected: clone\nSpain23F-1 present in ﬁve DCCs; Spain6B-2 in one DCC;\nSpain9V-3 in 12 DCCs; England14-9 in three DCCs;\nPoland6B-20 in six DCCs; clone Portugal19F-21 in ﬁve DCCs;\nclone Greece6B-22 in ﬁve DCCs; clone Sweden15A-25 in three\nDCCs, and clone Colombia23F-26 in two DCCs.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nClone B (Spain9V-3) was the most prevalent (20% of the\nDRPn strains) and also the most widely disseminated—de-\ntected in 12 out of 14 DCCs. Strains were either of serotype 9V\nor 14. Although the majority of the strains had intermediate\nresistance to penicillin and were also resistant to erythromycin\nand clindamycin, strains with other biotypes were also de-\ntected. Clone SI (Spain6B-2) was detected in DCC16. In our\nprevious studies (8, 34) no strains of this clone were identiﬁed.\nMLST was performed for two strains and the allelic proﬁle\nST90 characteristic of the Spain6B-2 clone reference strain was\nidentiﬁed (Table 4) (27). The remaining PMEN clones\nPortugal19F-21,\n(Spain23F-1, England6B-9,\nGreece6B-22, Sweden15A-25, and Colombia23F-26) had all been\nrecovered in 1996 to 1998 and had the same properties de-\nscribed previously (8, 34).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nPoland6B-20,\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nLisbon DCC clones. Six clonal types were identiﬁed: NNN,\nD, X, XXX, EEE, and WWW. The six strains of clone NNN\nwere all nontypeable, were resistant to erythromycin, clinda-\nmycin, tetracycline, and SXT, and were either susceptible to\npenicillin or intermediately resistant to penicillin. This clone\nwas detected in four DCCs. In our earlier studies clone NNN\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nFIG. 2. PFGE proﬁles of some drug-resistant S. pneumoniae clones isolated from children attending day care centers in Lisbon, Portugal. The\n ladder and R6 are indicated and were used as molecular weight markers. Clonal types of strains identical to PMEN representatives are designated\nby the PFGE code letter and the PMEN code between the brackets; other clonal types are designated by the serotype and PFGE code letter. NT,\nnontypeable.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nwas recovered in a single DCC only (DCC2) in 1997 and 1998\n(35). By MLST this clone had ST344, i.e., had the same allelic\nproﬁle of two nontypeable strains recovered from patients with\ninvasive disease in Norway in 1996 and Australia in 1998 (Ta-\nble 4). Four strains of clone D and capsular type 19F were\nrecovered in DCC15. The strains had intermediate resistance\nto penicillin and were also resistant to erythromycin, clinda-\nmycin, tetracycline, and SXT. This clone was also detected in\nour earlier studies since 1996 but only in DCC7, and the pre-\nvious isolates were resistant to penicillin and SXT only (8).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nNovel clones. Three of the 14 novel clonal types (NF, NK,\nand BE) were nontypeable. The other clones were of serotypes\n6A, 15A, 19A, 19F, and 33F. Three clones (BE, AI, and AG)\nhad a high number of isolates accounting for 52% of these\nisolates. Representatives of these three clones were selected\nfor MLST.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nClone BE was found in two DCCs and all the strains were\nnontypeable and resistant to erythromycin, clindamycin, tetra-\ncycline, and SXT; six out of eight strains also had intermediate\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nresistance to penicillin. This clone, although distinct by PFGE,\naccording to MLST is a double-locus variant of the clone of\nPFGE type NNN (also nontypeable), described above (Table\n4).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nAll strains of clone AI had capsular type 19F and were\nresistant to chloramphenicol, tetracycline, and SXT. These\nstrains were recovered in three DCCs (DCC16, -20, and -22).\nStrains recovered from children in DCC20 were also resistant\nto erythromycin and clindamycin. MLST of two representative\nisolates yielded two sequence types (ST87 and ST1198), dif-\nfering in the gki and ddl alleles. Comparison with data from the\nMLST database identiﬁed one strain also with ST87, which was\nfound to cause invasive disease in Denmark in 1997, and three\nisolates with double-locus variants of ST87 that were detected\nin Spain in 1989 and 1997 and in 1999 in Italy (Table 4).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nAll strains of clone AG had capsular type 33F and were\nresistant to erythromycin and clindamycin and were found in\nthree DCCs (DCC7, -18, and -23). By MLST, two ST types\nwere identiﬁed, ST1199 and ST717 (differing only in the ddl\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nTABLE 2. Summary of clonal analysis of 202 drug-resistant pneumococcal isolates\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nPFGE type\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nCapsular type\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nAntibiotype resistant toe:\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nNo. of\nisolates\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.1**\nDCC code (no. of strains)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nResistant to penicillina\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nB (Spain9V-3)\nA (Spain23F-1)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nIntermediate resistance to\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\npenicillinb\nA (Spain23F-1)\nB (Spain9V-3)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nSI (Spain6B-2)\nR (England14-9)\nFF (Colombia23F-26)\nE (Poland6B-20)\nDDD (Sweden15A-25)\nNNN\nNF\nBZ\nD\nAI\nNK\nAQ\nBE\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nOthers with only one isolate\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nPenicillin susceptible but\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nresistant to other drugsc\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nB (Spain9V-3)\nH (Portugal19F-21)\nR (England14-9)\nM (Greece6B-22)\nE (Poland6B-20)\nAG\nAQ\nX\nBE\nNNN\nCA\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nOthers with only one isolate\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n9V/14\n23F\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nPG, SXT\nPG, CHL, TET, SXT\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n2 (4), 5 (1), 19 (1)\n1 (2), 15 (2)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n23F\n9V/14\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nPG, CHL, TET (12), SXT\nPG, ERY (11), CHL (9), TET (10), SXT (13)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n6B\n\n23F\n6B\n15A\nNTd\nNT\n15A\n19F\n19F\nNT\n19A\nNT\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n9V/14\n19F\n\n6B\n6B\n33F\n19F\n19F\nNT\nNT\n6A\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nPG, CHL, ERY (4), CLI (4), TET (8), SXT\nPG, ERY, CLI, SXT\nPG\nPG, CHL (2), ERY, CLI, TET, SXT (3)\nPG, ERY, CLI, TET\nPG, ERY, CLI, TET, SXT\nPG, ERY, TET, SXT\nPG, ERY, CLI, TET\nPG, ERY, CLI, TET, SXT\nPG, CHL, ERY (3), CLI (3), TET, SXT\nPG, TET, SXT\nPG, ERY, CLI, TET, SXT (1)\nPG, ERY, CLI, TET, SXT\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\nERY, CLI, TET (8), SXT (6)\nERY, CLI, TET (4)\nERY, CLI (5), SXT (2)\nCHL (2), ERY, CLI, TET, SXT (5)\nERY (12), CLI (12), TET, SXT (1)\nERY, CLI\nERY, CLI, TET\nERY, CLI, TET\nERY, CLI, TET, SXT\nERY, CLI (2), TET, SXT\nERY (1), CLI (1), TET, SXT\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\n1 (8), 2 (1), 5 (1), 7 (2), 15 (1)\n1 (8), 2 (1), 5 (3), 7 (2), 9 (3), 15 (2),\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\n16 (3), 18 (2), 21 (1), 22 (1)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.7**r== 0.1**\n16 (10)\n22 (2)\n2 (2), 16 (1)\n2 (3), 5 (2), 7 (1), 16 (2), 22 (6)\n1 (1), 20 (3), 21 (2)\n1 (1), 2 (1), 9 (1)\n2 (2)\n9 (1), 22 (1)\n15 (4)\n16 (9), 20 (3), 22 (2)\n2 (3)\n21 (1), 22 (1)\n21 (5), 5 (1)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.7**r== 0.1**\n1 (4), 17 (2), 21 (3)\n1 (1), 2 (1), 17 (2), 18 (3), 20 (2)\n9 (6), 21 (2)\n5 (1), 7 (6), 16 (1), 21 (1), 17 (2)\n2 (3), 5 (2), 22 (7), 23 (1)\n7 (3), 18 (1), 23 (1)\n16 (2)\n1 (2)\n21 (2)\n2 (1), 15 (2)\n22 (1), 23 (2)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\na MIC  1.5 g/ml. Total isolates, 10 (5%).\nb 0.1 g/ml  MIC  1.5 g/ml. Total isolates, 120 (59%).\nc Total isolates, 72 (36%).\nd NT, nontypeable.\ne PG, penicillin; TET, tetracycline; ERY, erythromycin; CLI, clindamycin; CHL, chloramphenicol. Numbers in parentheses indicate the number of isolates resistant\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nto the antimicrobial agent (no number, all isolates were resistant).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nallele). ST717 was previously identiﬁed in one invasive strain\nfrom the United Kingdom in 2003, which was also of type 33F\n(Table 4).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nDISCUSSION\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThis study was a follow-up of the Lisbon Day Care Center\nInitiative which started in 1996 and was conducted on a yearly\nbasis (8, 9, 34, 35). The main purpose of the initiative was to\nmonitor the prevalence of drug-resistant pneumococcal strains\namong children attending DCCs and to characterize the strains\nby serotyping and molecular typing techniques. No such stud-\nies had ever been conducted in Portugal and, to our best\nknowledge, this remains the only surveillance program of this\ntype in the country. In addition, although other studies on the\npneumococci carried by DCC attendees have been conducted\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nin other countries (3, 4, 19, 29), we are not aware of efforts that\nhave lasted for several years and are of such large magnitude,\nwhere all DRPn (including strains susceptible to penicillin)\nhave been extensively characterized by serotyping and geno-\ntyping (and antibiotyping), with the possible exception of a\nstudy described by Nilsson and Laurell (29) in Malmo, Sweden,\nwhich included close to 3,000 children attending 63 DCCs\nduring a 3-year period (1995 to 1997). In that study, molecular\ntyping was done by AP-PCR and BOX-A PCR. Other DCC\nstudies carried out in Greece (4), The Netherlands (3), and\nIsrael (19) have been described. These studies had smaller\nnumbers of children and of pneumococcal isolates. In addition,\nthese studies were point-prevalence ones and did not include\nyearly surveillances like ours, and they also had different de-\nsigns. In particular, the Israeli study was a 5-month longitudi-\nnal study of 252 children; the authors used ribotyping to char-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nTABLE 3. Prevalence of PMEN clones, Lisbon DCC clones, and\nnovel clones in each DCCa\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nStrain classiﬁcation\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nDCC code\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nNo. of DRPn\nfully\ncharacterized\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nNo. of\nPFGE\ntypes\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n% Found\nof PMEN\nclones\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n% Found\nof Lisbon\nDCC clones\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n% Found\nof novel\nclones\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nTotal\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n135 (67%)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n15 (7%)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n52 (26%)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\na PMEN clones found in our study: A, clone Spain23F-1; SI, clone Spain6B-2;\nB, clone Spain9V-3; R, clone England14-9; E, clone Poland6B-20; H, clone\nPortugal19F-21; M, clone Greece6B-22; DDD, clone Sweden15A-25; FF, clone\nColombia23F-26. Lisbon DCC clones, clones detected in our previous day care\ncenter studies; Novel clones, clones found in this study.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nacterize the 594 pneumococci (19). Two Greek studies were\nperformed since December of 1995. The ﬁrst one was per-\nformed with 338 children attending seven day care centers. The\nstudy was conducted from December of 1995 to February of\n1996. The second study, was conducted from February of 1997\nto February of 1999, but only 95 of the 2,448 children sampled\nattended day care centers; the authors used RFEL and MLST\nto characterize 128 isolates susceptible to penicillin but resis-\ntant to other antimicrobials (4). A study from The Netherlands\ncompared 259 children attending 16 DCCs with 276 children\nwho did not attend DCCs. Children were sampled twice from\nJanuary to March of 1999, and a total of 305 pneumococci were\nrecovered (3).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nIn fact, in the 4 years of the study, 23 DCCs and 1,800\nchildren were enrolled, generating 3,053 nasopharyngeal sam-\nples of which 1,687 were positive for pneumococci. All pneu-\nmococcal isolates were characterized by antibiotype. Of the\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\n740 DRPn recovered, 615 pneumococci strains were com-\npletely characterized by serotyping and PFGE typing (8, 35).\nThese data have been a rich source of information on the\npneumococcal population and have enabled us to observe\ntrends of the DRPn in its ecological niche. In addition, these\nstudies have provided a good baseline for two intervention\nstudies that began in 2001 aimed to decrease carriage of\nDRPn: one based on education on antibiotic use and hygiene\ncontrol measures (the EURIS project, European Resistance\nIntervention Study) and the other based on the licensed 7-va-\nlent conjugate vaccine Prevenar (in preparation). The EURIS\nproject is now being continued in a sequel named PREVIS\n(Pneumococcal Resistance, Epidemicity, and Virulence—an\nInternational Study) and, by 2006, we will have completed a\ndecade of annual-biannual surveillance of pneumococci car-\nried by DCC attendees.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nThe data from the 1999 surveillance described here indicate\nthat the carriage rate of S. pneumoniae in Portuguese DCCs\nhas increased continuously since 1996, from 47% to 48, 60, and\n63% in subsequent years (35). The reasons for this increase are\nnot clear. We can only speculate that the nurses who collected\nthe samples have been mastering their sampling technique,\nyielding higher pneumococcal recovery rates. Similar high\nrates of pneumococcal carriage in DCC children have been\nreported in other countries such as Iceland (50%) (23), Swe-\nden (52%) (22), Lithuania (51%) (23), France (54.7%) (12),\nIsrael (63%) (45), and Greece (48%) (44).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nComparison of the 1999 results with results of the previous\nyears of surveillance indicated that the proportion of DRPn\nranged between 40 to 50% of all isolates. The carriage rate of\npenicillin-resistant and penicillin-intermediate pneumococci\nalso ﬂuctuated—ranging from 2 to 8% and 12 to 27%, respec-\ntively—over time. Interestingly, 1999 was the year with the\nlowest rate of penicillin resistance (2%) (Fig. 3).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nAnalysis of Portuguese invasive strains, recovered between\n1999 and 2001 from pediatric patients of the same age group,\nidentiﬁed higher values of DRPn (63.8%) and of resistance to\npenicillin (4.7%) (38). However, drug resistance to antimicro-\nbials other than penicillin was 15.9% among invasive pediatric\nisolates, a value comparable to the 18% carriage rate of such\nstrains identiﬁed in our study. In 1999, 64% of the penicillin-\nsusceptible strains were coresistant to macrolides, clindamycin,\nand tetracycline, a trend previously observed in 1997 and 1998\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nTABLE 4. Multilocus sequence typing of selected isolatesa\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nStrain\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nAntibiotype\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nSerotype\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.6**\nPG, CHL, TET, ERY, CLI, SXT\nPG, CHL, TET, SXT\nPG, ERY, CLI, TET, SXT\nERY, CLI, TET, SXT\nPG, ERY, CLI, TET, SXT\nERY, CLI, TET, SXT\nPG, ERY, TET, SXT\nPG, CHL, ERY, CLI, TET, SXT\nERY, CLI\nERY, CLI\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\n6B\n6B\nNT\nNT\nNT\nNT\n19F\n19F\n33F\n33F\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nPFGE\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nSI\nSI\nNNN\nNNN\nBE\nBE\nAI\nAI\nAG\nAG\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nST\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nObservationsb\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.1**\nSpain6B-2\nSpain6B-2\nNorway, Australia\nNorway, Australia\ndlv of ST344\ndlv of ST344\nDenmark\nslv of ST88 (Spain, Italy)\nslv of ST717 (United Kingdom)\nUnited Kingdom\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\na PG, penicillin; TET, tetracycline; ERY, erythromycin; CLI, clindamycin; CHL, chloramphenicol; ST, sequence type; dlv, double locus variant; slv, single locus\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nvariant; NT, nontypeable.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nb Country where ST or related ST was detected. Further details on epidemiological data are provided in the text.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nFIG. 3. Comparison of 4 years of the study. White bars, isolates resistant to penicillin; light gray bars, isolates intermediately resistant to\npenicillin; dark gray bars, isolates susceptible to penicillin but resistant to other antimicrobial agents; black bars, isolates susceptible to all\nantimicrobial agents tested. The total number of pneumococci was 277, 354, 465, and 591 in the four consecutive years, respectively.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nstudies (17, 33–35). DCC studies in Greece (44) found anti-\nbiotypes and levels of erythromycin and tetracycline resistance\ncomparable to those of our study. The Portuguese and Greek\ndata contrast with data from pneumococcal isolates from DCC\nattendees in Sweden which had similar levels of penicillin non-\nsusceptibility (20%) (36) but only rarely were resistant to\nerythromycin, tetracycline, and/or SXT.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe antimicrobial consumption was similar in all years and\nthe antibiotics most frequently used were always amoxicillin-\nclavulanic acid and amoxicillin. In our studies otitis has always\nbeen the major cause for antibiotic consumption reported by\nthe parents. Otitis is highly recognized as the most frequent\nafﬂiction responsible for visits to pediatricians and for antibi-\notic prescription, and thus is a major cause of morbidity and\nhas high costs associated with it (6). In addition, a study on the\nsimilarity of strains recovered from otitis and carriage (20)\nsuggests that intervention aimed to decrease carriage will also\ntarget otitis media.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nThe most frequent serotypes recovered in 1999 (6B, 14, 19F,\n23F, and nontypeable) were all (except for nontypeable iso-\nlates) included in the 7-valent conjugate vaccine. The 7-valent\nconjugate vaccine covered 78% of the DRPn isolates. These\ncapsular types had already been included among the most\nfrequent serotypes recovered in the past years (8, 34). These\nwere also the serotypes most commonly found in healthy chil-\ndren in Sweden (36), Israel (31), the United Kingdom (5), and\nGreece (41), although in a different rank order. Serotypes 14,\n23F, and 6B were also among the ﬁve most common capsular\ntypes in invasive strains recovered in 1999 to 2001 in Portugal\nfrom children younger than 6 years old (38). In our study,\nnonvaccine types (3, 6A, 15A, 15B, 19A, and 33F) together\nwith nontypeable strains accounted for 22% of the DRPn. All\nthese nonvaccine types had been detected in previous years\napart from serotype 33F (8, 35).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nMolecular typing identiﬁed several PMEN clones among the\nisolates. The international clone Spain9V-3, ﬁrst detected in Spain\n(7) and France (18, 25), was the clone most frequently isolated in\nour study. Interestingly, strains belonging to this clonal type also\nincluded isolates that had intermediate levels of resistance or\nwere fully susceptible to penicillin. Strains of Spain9V-3 have most\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\noften been described as fully resistant to penicillin (27) and iso-\nlates of invasive origin have been recovered in several countries\n(11, 13, 26). In our study, the majority of strains belonging to this\nclonal type expressed serotype 14, which represents one of the\nmost frequent serotypes among invasive isolates in Europe, the\nUnited States, and Canada (21).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nInterestingly, the relative proportion of strains of serotype\n23F (10%) decreased when compared to those from 1998 (P \n0.0038), reﬂecting the lower prevalence of international clone\nSpain23F-1 in the present study (Table 5). On the other hand, the\nmultiresistant Spain6B-2 clone was now detected, for the ﬁrst\ntime, in Portuguese DCCs. The clone was detected in a single\nDCC. Another six PMEN clones were identiﬁed (England14-9,\nPoland6B-20, Portugal19F-21, Greece6B-22, Sweden15A-25, and\nColombia23F-26); all of them had also been recovered in earlier\nPortuguese DCC studies (8, 34). Clones D and NNN, found since\n1996 in a single DCC (35), continued to be detected in other\nDCCs and thus are persisting and appear to be spreading through\ntime (Table 5).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nNovel clones were detected and accounted for 26% of the\nDRPn strains isolated. Three of them, BE, AI, and AG, had\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nTABLE 5. Prevalence over time of major clones found in 1999\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nClone\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nSpain23F-1\nSpain9V-3\nPoland6B-20\nPortugal19F-21\nColombia23F-26\nGreece6B-22\nEngland14-9\nSweden15A-25\nSpain6B-2\nD\nNNN\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nNew\nUnique\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nPercent found in:\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n(n  91)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n(n  131)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n(n  191)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n(n  202)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n22.0\n11.0\n5.5\n4.4\n0.0\n19.8\n1.1\n0.0\n0.0\n12.1\n0.0\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n13.2\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n16.0\n1.5\n9.9\n8.4\n10.7\n7.6\n0.0\n6.1\n0.0\n9.9\n0.8\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n12.2\n10.7\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.8**r== 0.2**\n14.7\n13.6\n2.1\n5.2\n4.7\n18.8\n14.1\n3.7\n0.0\n0.0\n1.6\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n15.2\n5.2\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.9**r== 0.1**\n8.4\n20.3\n13.4\n4.5\n1.5\n5.4\n5.0\n3.0\n1.0\n2.0\n3.0\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\n24.8\n7.4\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nMLST sequence types that were identical or differed in only\none or two loci from strains of invasive origin recovered in\nother countries. Interestingly, clone AG was associated with\nserotype 33F, an unusual capsular type in carriage (2); strains\nof clone BE were nontypeable. Both clones have “capsules”\nnot included in 7-valent conjugate vaccine and are multiresis-\ntant to antimicrobial agents. An increased prevalence of these\nstrains among carried isolates could be, as postulated by others\n(36), a preview of what will soon appear in the pneumococcal\ndisease population.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nIn summary, the majority of drug-resistant strains recovered\nfrom DCC children in Portugal are members of multidrug-\nresistant international clones that can cause illness (invasive\nand noninvasive). The emergence of a serotype 33F clone and\nthe increasing number of nontypeable clones, both with capac-\nity to cause acute otitis media (5, 30), highlight that the naso-\npharynxes of DCC children are a melting pot that is important\nto monitor (2) if we aim to gain a better understanding of the\nepidemiology of this pathogen.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nACKNOWLEDGMENTS\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nPartial support for this work was provided by contracts EURIS\n(QLK2-CT-2000-01020) from the European Community, PRAXIS/P/\nSAU/14051/1998 from Fundac¸a˜o para a Cieˆncia e Tecnologia, Portu-\ngal, and a grant from Fundac¸a˜o Calouste Gulbenkian, Portugal,\nawarded to H. de Lencastre. R. Sa´-Lea˜o and J. Carric¸o were supported\nby grants SFRH/BPD/14596/2003 and SFRH/BD/3123/2000, respec-\ntively, both from Fundac¸a˜o para a Cieˆncia e Tecnologia, Portugal. S.\nNunes was supported by grant 011/BIC/01 from contract QLK2-CT-\n2000-01020 and C. R. Alves was supported by grant 001/99/BIC/P from\ncontract PRAXIS/P/SAU/14051/1998.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nWe thank Natacha Sousa for assistance in MLST typing, Idalina\nBonﬁm for helping with isolation and characterization of the strains,\nand the experienced state-registered pediatric nurse Anabela Gon-\nc¸alves. We thank Alexander Tomasz of The Rockefeller University for\nhelpful suggestions and critical reading of the manuscript.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n1. Baquero, F., J. A. Garcia-Rodriguez, J. Garcia de Lomas, and L. Aguilar,\nand The Spanish Surveillance Group for Respiratory Pathogens. 1999. An-\ntimicrobial resistance of 1,113 Streptococcus pneumoniae isolates from pa-\ntients with respiratory tract infections in Spain: results of a 1-year (1996-\n1997) multicenter surveillance study. Antimicrob. Agents Chemother 43:\n357–359.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n2. Bogaert, D., R. De Groot, and P. W. Hermans. 2004. Streptococcus pneu-\nmoniae colonisation: the key to pneumococcal disease. Lancet Infect. Dis.\n4:144–154.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n3. Bogaert, D., M. N. Engelen, A. J. Timmers-Reker, K. P. Elzenaar, P. G.\nPeerbooms, R. A. Coutinho, R. de Groot, and P. W. Hermans. 2001. Pneu-\nmococcal carriage in children in The Netherlands: a molecular epidemio-\nlogical study. J. Clin. Microbiol. 39:3316–3320.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n4. Bogaert, D., P. W. Hermans, I. N. Grivea, G. S. Katopodis, T. J. Mitchell, M.\nSluijter, R. De Groot, N. G. Beratis, and G. A. Syrogiannopoulos. 2003.\nMolecular epidemiology of penicillin-susceptible non-beta-lactam-resistant\nStreptococcus pneumoniae isolates from Greek children. J. Clin. Microbiol.\n41:5633–5639.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n5. Brueggemann, A. B., D. T. Grifﬁths, E. Meats, T. Peto, D. W. Crook, and\nB. G. Spratt. 2003. Clonal relationships between invasive and carriage Strep-\ntococcus pneumoniae and serotype- and clone-speciﬁc differences in invasive\ndisease potential. J. Infect. Dis. 187:1424–1432.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n6. Centers for Disease Control and Prevention. 1997. Prevention of pneumo-\ncoccal disease: recommendations of the Advisory Committee on Immuniza-\ntion Practices (ACIP). Morb. Mortal. Wkly. Rep. 46:1–24.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n7. Coffey, T. J., C. G. Dowson, M. Daniels, J. Zhou, C. Martin, B. G. Spratt, and\nJ. M. Musser. 1991. Horizontal transfer of multiple penicillin-binding pro-\ntein genes, and capsular biosynthetic genes, in natural populations of Strep-\ntococcus pneumoniae. Mol. Microbiol. 5:2255–2260.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n8. de Lencastre, H., K. G. Kristinsson, A. Brito-Avoˆ, I. S. Sanches, R. Sa´-Lea˜o,\nJ. Saldanha, E. Sigvaldadottir, S. Karlsson, D. Oliveira, R. Mato, M. Aires\nde Sousa, and A. Tomasz. 1999. Carriage of respiratory tract pathogens and\nmolecular epidemiology of Streptococcus pneumoniae colonization in healthy\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nchildren attending day care centers in Lisbon, Portugal. Microb. Drug Resist.\n5:19–29.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n9. de Lencastre, H., I. Santos Sanches, A. Brito-Avoˆ, R. Sa´-Lea˜o, J. Saldanha,\nK. G. Kristinsson, and A. Tomasz. 1999. Carriage and antibiotic resistance of\nrespiratory pathogens and molecular epidemiology of antibiotic-resistant\nStreptococcus pneumoniae colonizing children in day-care centers in Lisbon:\nthe Portuguese day-care center initiative. Clin. Microbiol. Infect. 5:S55–S63.\n10. Di Fabio, J. L., E. Castaneda, C. I. Agudelo, F. De La Hoz, M. Hortal, T.\nCamou, G. Echaniz-Aviles, M. Noemi, C. Barajas, I. Heitmann, J. C.\nHormazabal, M. C. Brandileone, V. S. Dias Vieira, M. Regueira, R. Ruvin-\nski, A. Corso, M. Lovgren, J. A. Talbot, C. De Quadros, et al. 2001. Evolution\nof Streptococcus pneumoniae serotypes and penicillin susceptibility in Latin\nAmerica, Sireva-Vigia Group, 1993 to 1999. Pediatr. Infect. Dis. J. 20:959–\n967.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n11. Doit, C., B. Picard, C. Loukil, P. Geslin, and E. Bingen. 2000. Molecular\nepidemiology survey of penicillin-susceptible and -resistant Streptococcus\npneumoniae recovered from patients with meningitis in France. J. Infect. Dis.\n181:1971–1978.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n12. Dunais, B., C. Pradier, H. Carsenti, M. Sabah, G. Mancini, E. Fontas, and\nP. Dellamonica. 2003. Inﬂuence of child care on nasopharyngeal carriage of\nStreptococcus pneumoniae and Haemophilus inﬂuenzae. Pediatr. Infect. Dis.\nJ. 22:589–592.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n13. Enright, M. C., A. Fenoll, D. Grifﬁths, and B. G. Spratt. 1999. The three\nmajor Spanish clones of penicillin-resistant Streptococcus pneumoniae are the\nmost common clones recovered in recent cases of meningitis in Spain.\nJ. Clin. Microbiol. 37:3210–3216.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n14. Enright, M. C., and B. G. Spratt. 1998. A multilocus sequence typing scheme\nfor Streptococcus pneumoniae: identiﬁcation of clones associated with serious\ninvasive disease. Microbiology 144:3049–3060.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n15. Felmingham, D., C. Feldman, W. Hryniewicz, K. Klugman, S. Kohno, D. E.\nLow, C. Mendes, and A. C. Rodloff. 2002. Surveillance of resistance in\nbacteria causing community-acquired respiratory tract infections. Clin. Mi-\ncrobiol. Infect. 8:12–42.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n16. Felmingham, D., R. R. Reinert, Y. Hirakata, and A. Rodloff. 2002. Increasing\nprevalence of antimicrobial resistance among isolates of Streptococcus pneu-\nmoniae from the PROTEKT surveillance study, and comparative in vitro\nactivity of the ketolide, telithromycin. J. Antimicrob. Chemother. 50:25–37.\n17. Garcia-Rey, C., E. Bouza, L. Aguilar, J. Garcia-de-Lomas, and F. Baquero.\n2003. Evolution of penicillin and erythromycin co-resistance in Streptococcus\npneumoniae in Spain. Int. J. Antimicrob. Agents 22:541–544.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n18. Gasc, A. M., P. Geslin, and A. M. Sicard. 1995. Relatedness of penicillin-\nresistant Streptococcus pneumoniae serogroup 9 strains from France and\nSpain. Microbiology 141:623–627.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n19. Givon-Lavi, N., R. Dagan, D. Fraser, P. Yagupsky, and N. Porat. 1999.\nMarked differences in pneumococcal carriage and resistance patterns be-\ntween day care centers located within a small area. Clin. Infect. Dis. 29:\n1274–1280.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n20. Hanage, W. P., K. Auranen, R. Syrjanen, E. Herva, P. H. Makela, T. Kilpi,\nand B. G. Spratt. 2004. Ability of pneumococcal serotypes and clones to\ncause acute otitis media: implications for the prevention of otitis media by\nconjugate vaccines. Infect. Immun. 72:76–81.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n21. Hausdorff, W. P., J. Bryant, C. Kloek, P. R. Paradiso, and G. R. Siber. 2000.\nThe contribution of speciﬁc pneumococcal serogroups to different disease\nmanifestations: implications for conjugate vaccine formulation and use, part\nII. Clin. Infect. Dis. 30:122–140.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n22. Henriques Normark, B., B. Christensson, A. Sandgren, B. Noreen, S. Sylvan,\nL. G. Burman, and B. Olsson-Liljequist. 2003. Clonal analysis of Streptococ-\ncus pneumoniae nonsusceptible to penicillin at day-care centers with index\ncases, in a region with low incidence of resistance: emergence of an invasive\ntype 35B clone among carriers. Microb. Drug Resist. 9:337–344.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n23. Hjaltested, E. K., J. Bernatoniene, H. Erlendsdottir, P. Kaltenis, G. Berna-\ntoniene, T. Gudnason, A. Haraldsson, and K. G. Kristinsson. 2003. Resis-\ntance in respiratory tract pathogens and antimicrobial use in Icelandic and\nLithuanian children. Scand. J. Infect. Dis. 35:21–26.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n24. Kellner, J. D., E. L. Ford-Jones, et al. 1999. Streptococcus pneumoniae\ncarriage in children attending 59 Canadian child care centers. Arch. Pediatr.\nAdolesc. Med. 153:495–502.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n25. Lefevre, J. C., M. A. Bertrand, and G. Faucon. 1995. Molecular analysis by\npulsed-ﬁeld gel electrophoresis of penicillin-resistant Streptococcus pneu-\nmoniae from Toulouse, France. Eur. J. Clin. Microbiol. Infect. Dis. 14:491–\n497.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n26. Marchese, A., M. Ramirez, G. C. Schito, and A. Tomasz. 1998. Molecular\nepidemiology of penicillin-resistant Streptococcus pneumoniae isolates recov-\nered in Italy from 1993 to 1996. J. Clin. Microbiol. 36:2944–2949.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n27. McGee, L., L. McDougal, J. Zhou, B. G. Spratt, F. C. Tenover, R. George, R.\nHakenbeck, W. Hryniewicz, J. C. Lefevre, A. Tomasz, and K. P. Klugman.\n2001. Nomenclature of major antimicrobial-resistant clones of Streptococcus\npneumoniae deﬁned by the pneumococcal molecular epidemiology network.\nJ. Clin. Microbiol. 39:2565–2571.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n28. National Committee for Clinical Laboratory Standards. 1997. Performance\nstandards for antimicrobial disk susceptibility tests, 6th ed., vol. 17, no. 1.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nApproved standard M2-A6. National Committee for Clinical Laboratory\nStandards, Wayne, Pa.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n29. Nilsson, P., and M. H. Laurell. 2001. Carriage of penicillin-resistant Strep-\ntococcus pneumoniae by children in day-care centers during an intervention\nprogram in Malmo, Sweden. Pediatr. Infect. Dis. J. 20:1144–1149.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n30. Porat, N., G. Barkai, M. R. Jacobs, R. Treﬂer, and R. Dagan. 2004. Four\nantibiotic-resistant Streptococcus pneumoniae clones unrelated to the pneu-\nmococcal conjugate vaccine serotypes, including 2 new serotypes, causing\nacute otitis media in southern Israel. J. Infect. Dis. 189:385–392.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n31. Regev-Yochay, G., M. Raz, R. Dagan, N. Porat, B. Shainberg, E. Pinco, N.\nKeller, and E. Rubinstein. 2004. Nasopharyngeal carriage of Streptococcus\npneumoniae by adults and children in community and family settings. Clin.\nInfect. Dis. 38:632–639.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n32. Sahm, D. F., J. A. Karlowsky, L. J. Kelly, I. A. Critchley, M. E. Jones, C.\nThornsberry, Y. Mauriz, and J. Kahn. 2001. Need for annual surveillance of\nantimicrobial resistance in Streptococcus pneumoniae in the United States:\n2-year longitudinal analysis. Antimicrob. Agents Chemother. 45:1037–1042.\n33. Sa´-Lea˜o, R., A. Tomasz, and H. de Lencastre. 2001. Multilocus sequence\ntyping of Streptococcus pneumoniae clones with unusual drug resistance\npatterns: genetic backgrounds and relatedness to other epidemic clones.\nJ Infect. Dis. 184:1206–1210.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n34. Sa´-Lea˜o, R., A. Tomasz, I. S. Sanches, A. Brito-Avoˆ, S. E. Vilhelmsson, K. G.\nKristinsson, and H. de Lencastre. 2000. Carriage of internationally spread\nclones of Streptococcus pneumoniae with unusual drug resistance patterns in\nchildren attending day care centers in Lisbon, Portugal. J. Infect. Dis. 182:\n1153–1160.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n35. Sa´-Lea˜o, R., A. Tomasz, I. S. Sanches, S. Nunes, C. R. Alves, A. B. Avoˆ, J.\nSaldanha, K. G. Kristinsson, and H. de Lencastre. 2000. Genetic diversity\nand clonal patterns among antibiotic-susceptible and -resistant Streptococcus\npneumoniae colonizing children: day care centers as autonomous epidemio-\nlogical units. J. Clin. Microbiol. 38:4137–4144.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n36. Sandgren, A., K. Sjostrom, B. Olsson-Liljequist, B. Christensson, A. Sam-\nuelsson, G. Kronvall, and B. Henriques Normark. 2004. Effect of clonal and\nserotype-speciﬁc properties on the invasive capacity of Streptococcus pneu-\nmoniae. J. Infect. Dis. 189:785–796.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n37. Schito, G. C., E. A. Debbia, and A. Marchese. 2000. The evolving threat of\nantibiotic resistance in Europe: new data from the Alexander Project. J.\nAntimicrob Chemother. 46:3–9.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n38. Serrano, I., M. Ramirez, and J. Melo-Cristino. 2004. Invasive Streptococcus\npneumoniae from Portugal: implications for vaccination and antimicrobial\ntherapy. Clin. Microbiol. Infect. 10:652–656.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n39. Song, J. H., N. Y. Lee, S. Ichiyama, R. Yoshida, Y. Hirakata, W. Fu, A.\nChongthaleong, N. Aswapokee, C. H. Chiu, M. K. Lalitha, K. Thomas, J.\nPerera, T. T. Yee, F. Jamal, U. C. Warsa, B. X. Vinh, M. R. Jacobs, P. C.\nAppelbaum, and C. H. Pai. 1999. Spread of drug-resistant Streptococcus\npneumoniae in Asian countries: Asian Network for Surveillance of Resistant\nPathogens (ANSORP) Study. Clin. Infect. Dis. 28:1206–1211.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n40. Sorensen, U. B. 1993. Typing of pneumococci by using 12 pooled antisera.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nJ. Clin. Microbiol. 31:2097–2100.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n41. Syrogiannopoulos, G. A., G. D. Katopodis, I. N. Grivea, and N. G. Beratis.\n2002. Antimicrobial use and serotype distribution of nasopharyngeal Strep-\ntococcus pneumoniae isolates recovered from Greek children younger than 2\nyears old. Clin. Infect. Dis. 35:1174–1182.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n42. Tenover, F. C., R. D. Arbeit, R. V. Goering, P. A. Mickelsen, B. E. Murray,\nD. H. Persing, and B. Swaminathan. 1995. Interpreting chromosomal DNA\nrestriction patterns produced by pulsed-ﬁeld gel electrophoresis: criteria for\nbacterial strain typing. J. Clin. Microbiol. 33:2233–2239.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n43. Thornsberry, C., D. F. Sahm, L. J. Kelly, I. A. Critchley, M. E. Jones, A. T.\nEvangelista, and J. A. Karlowsky. 2002. Regional trends in antimicrobial\nresistance among clinical isolates of Streptococcus pneumoniae, Haemophilus\ninﬂuenzae, and Moraxella catarrhalis in the United States: results from the\nTRUST Surveillance Program, 1999-2000. Clin. Infect. Dis. 34:S4–S16.\n44. Tsolia, M., G. Kouppari, A. Zaphiropoulou, S. Gavrili, M. Tsirepa, D.\nKafetzis, and T. Karpathios. 1999. Prevalence and patterns of resistance of\nStreptococcus pneumoniae strains isolated from carriers attending day care\ncenters in the area of Athens. Microb. Drug Resist. 5:271–278.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n45. Yagupsky, P., N. Porat, D. Fraser, F. Prajgrod, M. Merires, L. McGee, K. P.\nKlugman, and R. Dagan. 1998. Acquisition, carriage, and transmission of\npneumococci with decreased antibiotic susceptibility in young children at-\ntending a day care facility in southern Israel. J. Infect. Dis. 177:1003–1012.",
         "extracted",
         "",
         "Trends in Drug Resistance, Serotypes, and Molecular Types of Streptococcus pneumoniae Colonizing Preschool-Age Children Attending Day Care Centers in Lisbon, Portugal: a Summary of 4 Years of Annual Surveillance",
         "None"
        ],
        [
         "13",
         "00111aa397aff132da8dcb2e4e9276d68d1cd4a6",
         "Laboratory exposures to brucellae indicate unfamiliarity with organism.",
         "P. Yagupsky,E. Baron",
         "https://wwwnc.cdc.gov/eid/article/11/8/pdfs/04-1197.pdf",
         "\n**BLOCK**fs== 24.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nLaboratory Exposures to Brucellae\nand Implications for Bioterrorism\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.4**\nPablo Yagupsky* and Ellen Jo Baron†\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nBrucellae are class 3 organisms and potential agents\nof  bioterrorism.  Because  of  effective  public  health  meas-\nures, brucellosis has become a rare disease in industrial-\nized  countries,  and  clinical  microbiology  laboratories  are\nfrequently unfamiliar with the genus. A low index of suspi-\ncion by physicians or failure to notify the laboratory, equiv-\nocal  Gram-stain  results,  misidentification  of  the  organism\nby  commercial  systems,  unsafe  laboratory  practices,  and\nlaboratory accidents have been responsible for numerous\ncases of exposure to the organism and laboratory-acquired\ndisease in recent years. Discovery of a laboratory exposure\nto  brucellae  should  prompt  an  exhaustive  investigation  of\nthe event and its circumstances, definition of the population\nat  risk,  enforcement  of  safe  laboratory  practices,  and\nantimicrobial  drug  prophylaxis  for  exposed  persons.\nInadvertent exposures to brucellae in the clinical laboratory\nindicate  a  widespread  lack  of  preparedness  to  cope  with\neventual biologic threats involving use of the organism.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nby contact with infected animals and consumption of\ncontaminated animal products (1,2). Because of compulso-\nry pasteurization of milk products and strict control of the\ndisease  in  dairy  cattle,  the  incidence  of  brucellosis  has\nsteadily  declined  in  most  industrialized  countries  during\nthe last 50 years. However, the disease remains among the\nmost commonly recognized causes of laboratory-transmit-\nted infections; 2% of all brucellosis cases are laboratory-\nacquired (1,3–9).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSeveral  biologic  characteristics  make  brucellae  easily\ntransmissible within the close confinement of the clinical\nmicrobiology laboratory, including the facts that the infect-\ning dose for humans is low, and the organism may enter the\nbody in many ways relevant to laboratory practices (e.g.,\nthrough  the  respiratory  mucosa,  conjunctivae,  gastroin-\ntestinal tract, or abraded skin) (1,2). Rare cases of acquisi-\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n*Ben-Gurion  University  of  the  Negev,  Beer-Sheva,  Israel;  and\n†Stanford University Medical School, Stanford, California, USA\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\ntion of the organism through organ transplantation, sexual\ncontact,  breastfeeding,  or  the  transplacental  route  have\nalso been reported (1). Because person-to-person transmis-\nsion does not occur, infected persons do not pose a threat\nto their surroundings.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nand\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nfusion\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nreproducing\n**BLOCK**fs== 10.0**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nSoon  after  entry  into  the  body  through  the  skin  or\nmucous  membranes,  brucellae  are  ingested  by  polymor-\nphonuclear and mononuclear phagocytes. The organism is\nable to escape phagocytic killing by inhibiting the phago-\nsome-lysosome\ninside\nmacrophages  (1,10).  After  a  variable  incubation  period\nranging  from  <1  week  to  several  months  (usually  2–4\nweeks),  nonspecific  systemic  symptoms  such  as  fever,\nheadache,  malaise,  night  sweats,  and  arthralgia  follow,\nresembling a flulike disease (1,2). During the early stages\nof  the  disease,  patients  are  frequently  bacteremic.  This\nbacteremia  has  a  continuous  pattern,  making  circulating\nbrucellae  easily  detectable  by  blood  culture.  Once  in  the\nbloodstream, the organism is seeded to multiple organ sys-\ntems and especially to those rich in reticuloendothelial tis-\nsue,  such  as  the  liver,  spleen,  and  the  skeletal  and\nhematopoietic systems, where it may cause localized dis-\nease such as hepatitis or arthritis (1,2).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nBecause  of  the  variable  manifestations  of  human  bru-\ncellosis, a wide array of different clinical specimens may\ncontain viable brucellae, including pus, blood, bone mar-\nrow,  synovial  fluid  and  tissues,  and  more  rarely,  cere-\nbrospinal  fluid,  urine,  and  genital  exudates.  The\nconcentration  of Brucella organisms  in  the  blood  (11,12)\nand synovial fluid (13) of patients with brucellosis is usu-\nally low, and therefore, these clinical specimens probably\npose  a  low  risk  for  contagion  for  laboratory  personnel.\nHowever,  the  danger  of  clinically  relevant  exposure\nincreases exponentially after incubation of both solid and\nliquid media. Seeded media harbor considerable amounts\nof  viable Brucella organisms,  and  routine  bacteriologic\nprocedures  such  as  preparing,  centrifuging,  and  vigorous\nagitation (vortexing) of bacterial suspensions, performing\nsubcultures  and  biochemical  testing,  and  particularly  the\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.2**r== 0.3**\nEmerging Infectious Diseases • www.cdc.gov/eid • Vol. 11, No. 8, August 2005\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\ncatalase test, may create dangerous aerosols and the poten-\ntial for accidental spillage (14).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nMost  cases  of  laboratory-acquired  brucellosis  have\nbeen  caused  by  the  more  virulent  Brucella  melitensis\nspecies  (3,4,6,8,14–29).  B.  suis (4,6),  B.  abortus (3,6,8),\nand B. canis (30) have also been implicated, and transmis-\nsion of the attenuated B. abortus 19 and B. melitensis Rev-\n1 vaccine strains has also been reported (31,32).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nClinical Manifestations of\nLaboratory-acquired Infections\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nPatients involved in laboratory outbreaks of brucellosis\nhave shown almost the entire range of clinical manifesta-\ntions of the disease, ranging from the common prolonged\nfebrile  syndrome  (undulant  fever)  (14)  and  a  flulike  dis-\nease (25,27), to focal signs and symptoms, such as hepati-\ntis (17,33), lymphadenopathy (17,25), uveitis (14), breast\nabscess  (28)  epididymitis  (29),  arthritis  (17),  discitis\n(29,34), pneumonitis (17), deep vein thrombosis (29), and\nmeningitis  (31).  Deaths  are  rare  (<5%)  even  among\nuntreated  patients  with  brucellosis,  but  5  fatal  cases\noccurred among the 426 persons with laboratory-acquired\ndisease summarized by Pike in 1978 (5).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nLaboratory Exposure to Brucella Organisms\n**BLOCK**fs== 10.0**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nBecause  of  effective  control  measures  in  animals  and\nanimal  products,  brucellosis  has  been  almost  eradicated\nfrom  most  industrialized  countries,  where  the  disease  is\nusually limited to persons who have traveled to developing\ncountries  or  ingested  imported  contaminated  food  (14).\nHuman brucellosis has become so rare in the United States\nthat <300 cases have been reported annually in the last 4\ndecades (9,15) and ≈100 per year in the last 10 years (2).\nBecause of the rare occurrence of brucellosis, technicians\nworking in industrialized countries have become unfamil-\niar with the staining and other phenotypic characteristics of\nthe  organism.  In  addition,  physicians  frequently  do  not\nconsider  the  diagnosis  of  brucellosis  or  fail  to  communi-\ncate their suspicion to the clinical microbiology laborato-\nry, which results in inadvertent handling of cultures on an\nopen bench.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAlthough  the  identification  of  the  genus  Brucella is\nstraightforward  (small  gram-negative  coccobacilli;  posi-\ntive oxidase, catalase, and urease test results; no sugar fer-\nmentation;  and  requirement  of  aerobic  conditions  and\nadded CO2 for its growth), laboratory-acquired disease has\nfrequently resulted from misidentification of the organism\n(35). In 2 published outbreaks, the Brucella isolate resisted\ndecoloration  and  appeared  as  a  gram-positive  or  gram-\nvariable  coccobacillus  and  thus  was  misidentified  as\nmicrococcus  or  a  coryneform  bacillus  (14,29).  In  other\ncases,  B.  melitensis organisms  tested  with  the  API20NE\n(bioMérieux,  Marcy  l’Etoile,  France)  identification  kit\n**BLOCK**fs== 10.0**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\nproduced  the  1200004  or  1000004  biochemical  profile\nleading  to  misidentification  of  the  isolate  as  Moraxella\nphenylpyruvica at  a  good  identification  level  (90.5%)\n(8,18–21). The  frequent  failure  of  clinical  laboratories  to\ncorrectly identify isolates as brucellae is particularly wor-\nrisome because these organisms are regarded as potential\nagents for bioterrorism. Brucellae are inexpensive to pro-\nduce and disperse, and transmission to humans may result\nin prolonged illness and long-term sequelae. These organ-\nisms  are  considered  category  B  select  agents  (2). The\noccurrence in the last few years of inadvertent exposures to\nbrucellae  in  laboratories  in  industrialized  countries\n(7,8,14,17–26)  indicates  a  lack  of  preparedness  to  deal\nwith a real biologic threat. Lack of recognition of an iso-\nlate as a Brucella sp. by a laboratory may enable a bioter-\nrorism-related  attack  to  go  undetected,  whereas  a\nfalse-positive identification may cause unnecessary alarm.\nIn laboratories serving countries in which human bru-\ncellosis is still endemic, the degree of potential exposure to\nthe  organism  may  be  extremely  high.  In  a  microbiology\nlaboratory in Ankara, Turkey, the average annual number\nof cultures positive for Brucella spp. reaches 400 (34). A\nlaboratory serving a brucellosis-endemic area in southern\nIsrael processes ≈150 per year; at the peak of the Brucella\nseason  (April–June),  10%  of  all  positive  blood  cultures\ngrow B. melitensis (27). The risk for exposure in develop-\ning  countries  is  frequently  aggravated  by  lack  of  safety\nequipment and inadequate laboratory facilities (34). In the\naforementioned Turkish laboratory, the disease affected 10\n(18%) of 55 laboratory workers, and the calculated hazard\nwas 8% per employee-year (34).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nMechanism of Transmission\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nthe\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nThe probable source of the infection is usually apparent\nwhen disease occurs in laboratories where isolation of bru-\ncellae  is  rare  (21).  In  laboratories  serving  brucellosis-\ntime  and  circumstances  of\nendemic  areas,\ncontamination  are  more  difficult  to  trace  because  of  the\nexistence  of  multiple  potential  sources  (22,27). Although\nthe exact form of transmission often remains speculative,\naerosols have been implicated in most cases (24,27,29,32).\nThis  assumption  has  been  strengthened  by  spread  of  the\ndisease  to  distant  areas  through  common  ventilation\n(27,32).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nLaboratory  accidents  such  as  breakage  of  centrifuge\ntubes (7) or blood-culture vials (16) play a minor role in\nlaboratory-acquired  disease  and  are  responsible  for  only\n20% of cases (6). More commonly, exposures are the result\nof  unsafe  laboratory  practices,  such  as  sniffing  plates\n(8,24,26,29,34);  working  on  an  open  bench  with  viable\norganisms  (14,16,17,20,22);  not  using  protective  equip-\nment such as gloves, masks, and goggles (34); or ingesting\nsuspensions of living brucellae during mouth pipetting (3).\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.3**r== 0.3**\nEmerging Infectious Diseases • www.cdc.gov/eid • Vol. 11, No. 8, August 2005\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nHowever, even when no apparent breach in safety proce-\ndures  is  apparent,  transmission  may  occur  (17).  Rare\nevents—such as self-inoculation of brucellae by syringes\nloaded with a suspension of the organism or with synovial\nfluid from an infected patient (3,16), injury to the conjunc-\ntiva with a broken tube that had contained a living culture\n(3), or participation in a laboratory exercise in which still-\nviable  organisms  were  inadvertently  used  (15)—are  also\nresponsible  for  a  small  number  of  laboratory  exposures\nand outbreaks of disease.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nMost  cases  of  laboratory-acquired  brucellosis  have\noccurred  in  clinical  laboratories,  but  transmission  of  the\ndisease  in  research  facilities  (4,25)  and  laboratories  that\nmanufactured Brucella vaccines has also been document-\ned  (21,22).  A laboratory  technician  at  the  Centers  for\nDisease  Control  and  Prevention  (CDC)  became  infected\nwhile working in a safety cabinet with a B. melitensis iso-\nlate that had originally caused an outbreak of laboratory-\nacquired  disease  in  a  community  hospital  (17).  In  a\nseparate episode, the organism isolated from a microbiolo-\ngy technologist with laboratory-acquired disease infected a\nlaboratory worker at the hospital to which the technologist\nhad been admitted (14).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nThe attack rate of laboratory-associated infections has\nranged  from  30%  to  100%  depending,  among  other  fac-\ntors, on the location of workers, whether aerosol-generat-\ning  procedures  have  been  performed,  and\nthe\nconcentration  of  microorganisms  in  the  contaminated\nmedia  (7,8).  However,  the  hazard  of  transmission  is  not\nlimited to persons who worked with the isolate. Among the\n74 reported cases of laboratory-acquired brucellosis from\n1897  to  1939  reviewed  by  Meyer  and  Eddie,  the  disease\nalso affected janitors and occasional visitors (3). In a large\noutbreak of laboratory-acquired brucellosis, 3 of 7 cases of\ndisease  occurred  among  persons  who  briefly  visited  the\nfacility but did not enter the room where Brucella cultures\nwere processed (27).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nRecognizing sporadic cases and even outbreaks of lab-\noratory-acquired brucellosis is not always easy because the\ndisease lacks distinctive clinical features (14,17) and has a\nwidely variable incubation period (7,17,19,32). In a cluster\nof disease that originated in a single exposure, the onset of\nsymptoms of the 7 affected laboratory workers spanned 5\nmonths (17). Although a temporal clustering of cases usu-\nally  suggests  a  large  exposure  from  a  pinpoint  source\n(7,32),  this  is  not  always  the  case.  When  biotyping  was\nperformed in the isolates recovered within a 5-week peri-\nod  from  7  infected  hospital  workers  in  southern  Israel,  3\ndistinct  strains  were  recognized,  demonstrating  that  the\noutbreak was caused by at least 3 separate exposures (27).\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nGuidelines for Investigating Outbreaks\n**BLOCK**fs== 10.0**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\nIn-depth  investigation  of  laboratory-acquired  cases  of\nbrucellosis may lead to the identification of unsafe bacte-\nriologic  practices,  suitable  for  correction  by  educational\nand technical measures. To guide the investigation, the fol-\nlowing  recommendations  are  made.  1)  Send  isolates  to  a\nreference  laboratory  for  confirmation.  Organisms  should\nbe shipped as “dangerous goods,” according to the guide-\nlines established by the World Health Organization and the\nOffice  of  Biosafety  at  CDC  (36).  In  the  United  States,\norganisms presumptively identified as brucellae should be\nsent to CDC or another public health laboratory following\nthe specific guidelines for transferring “select agents.” 2)\nInform infection control services and public health author-\nities,  who  may  choose  to  involve  CDC.  3)  Conduct  a\nmeticulous epidemiologic investigation. 4) Determine the\ndate  and  circumstances  of  the  exposure  (17).  5)  Exclude\nother  potential  sources  of  transmission,  such  as  previous\nlaboratory exposures, travels to brucellosis-endemic areas,\nconsumption  of  unpasteurized  dairy  products,  and  han-\ndling  of  farm  or  laboratory  animals  (14,15,17,27,29).  6)\nKeep all relevant data and records. 7) Define the exposed\npopulation. 8) Determine the level of risk on the basis of\ntype of laboratory procedures performed, proximity to the\nsource,  duration  of  the  exposure,  and  the  like,  to  define\npersons  to  whom  postexposure  prophylaxis  should  be\noffered (8,32). 9) Check biologic safety cabinets (27,29).\n10) Check the ventilation system (27). 11) Collect baseline\nserum  samples  from  all  known  potentially  exposed  per-\nsons  (17,27).  12)  Freeze  isolates  for  future  typing,  espe-\ncially in brucellosis-endemic areas (23,27).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nPrevention of Laboratory-acquired Cases\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nBrucellae  are  considered  class  3  organisms.  CDC  has\nstrongly recommended that live Brucella cultures and sus-\npicious  organisms  be  manipulated  in  a  class  II  biologic\nsafety  cabinet  (37).  This  recommendation,  however,  is\nclearly insufficient for preventing laboratory-acquired dis-\nease because by the time the organism is suspected or con-\nfirmed as Brucella, exposure of laboratory personnel may\nhave occurred (27). On the other hand, converting a large\nand busy clinical microbiology laboratory into a biosafety\nlevel III facility, where all specimens are handled in biolog-\nic safety cabinets (32), is both impractical and unnecessary,\nespecially in laboratories in areas where the disease is not\nendemic. However, in regions where brucellosis is highly\nprevalent, an enhanced safety policy should be adopted. All\nblood  culture  vials  detected  as  positive  by  the  automated\nblood culture system, as well as all bone marrow and syn-\novial  fluid  specimens,  should  be  manipulated  in  biologic\nsafety hoods until the isolated microorganisms are defini-\ntively determined to be other than Brucella sp. (27).\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.2**r== 0.3**\nEmerging Infectious Diseases • www.cdc.gov/eid • Vol. 11, No. 8, August 2005\n**BLOCK**fs== 10.0**b== 0.1**t== 0.2**l== 0.1**r== 0.5**\nFor  laboratories  serving  areas  where  brucellosis  is\nuncommon, the following recommendations are made. 1)\nBecause of the low incidence of brucellosis, physicians in\nareas not endemic for the disease are unfamiliar with the\nclinical and epidemiologic features of human brucellosis,\nand the possibility of brucellosis is rarely considered in the\ndifferential diagnosis. Periodic education of physicians on\nthis  subject  is  indicated.  2)  Communication  between\nattending  physicians  and  the  laboratory  should  be\nimproved. The clinical microbiology laboratory should be\ninformed  in  advance  when  clinical  specimens  had  been\nobtained  from  patients  with  risk  factors  for  brucellosis,\nsuch  as  recent  history  of  travel  to  brucellosis-endemic\nareas,  consumption  of  local  or  imported  unpasteurized\ndairy products, or professional exposures in veterinarians,\nshepherds,  slaughterhouse  employees,  and  laboratory\nworkers. 3) The use of automated, continuous monitoring\nblood culture systems for patients with suspected brucel-\nlosis  should  be  preferred  over  the  lysis-centrifugation\nmethod  because  the  latter  involves  centrifuging  clinical\nspecimens  and  visually  inspecting  plates  to  detect  the\norganism and probably increases the risk for transmission\n(11,27). Although the use of blood lysis-based methods has\nbeen  advocated  in  the  past  for  improving  detection  of\nBrucella bacteremia, modern automated blood culture sys-\ntems are faster and more sensitive (11,12). 4) The familiar-\nity of laboratory technicians with the characteristics of the\norganism,  as  well  as  with  the  safe  handling  of  cultures,\nshould be improved and maintained through periodic edu-\ncation.  The  reader  is  referred  to  the  excellent  review  by\nGilligan  and York  for  handling  and  identifying  presump-\ntive Brucella organisms (35). 5) Standard precautions and\nstrict adherence to good laboratory practices must be com-\npletely  adopted,  reinforced,  and  regularly  monitored.  6)\nAll work with gram-negative or gram-variable small rods\nor coccobacilli isolated from tissues, blood, bone marrow,\nbone, or synovial fluid exudates should be carried out in a\nbiosafety  cabinet  until  Brucella has  been  ruled  out.  7)\nPlates  should  be  sealed  for  safety  when  not  in  use  and\nappropriately disposed and sterilized as soon as they are no\nlonger being actively used (17). 8) Because brucellae are\nrelatively slow-growing bacteria, cultures for the organism\nhave been traditionally kept for several weeks. However,\nmodern  blood  culture  systems  enable  brucellae  to  be\ndetected within the routine 5-day incubation period insti-\ntuted  in  most  clinical  laboratories  (12). Therefore,  safety\nprecautions should not be limited to organisms that tend to\ngrow slowly (8,18,20,26). 9) Antimicrobial drug–suscepti-\nbility  testing  of  Brucella  organisms  is  not  indicated\nbecause the therapeutic regimen for brucellosis is standard,\nand  the  organism  does  not  usually  acquire  antimicrobial\nresistance.  Performance  of  this  and  other  unnecessary\ntests,  and  especially  of  laboratory  procedures  known  to\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nproduce aerosols, should be strongly discouraged (27). 10)\nIf a suspension of living brucellae is spilled and the organ-\nism is recognized, the entire laboratory should be immedi-\nately  evacuated,  doors  should  be  shut,  and  an  effective\ngermicide  such  as  3%  phenol  or  10%  bleach  should  be\napplied  by  a  trained  person  wearing  a  safety  mask,  gog-\ngles, an impermeable laboratory gown, and gloves (7).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nPostexposure Prophylaxis\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nBecause  of  ethical  considerations,  the  heterogeneous\nnature of the events leading to Brucella exposure, difficul-\nties in determining the actual risk for individual workers,\nlate recognition of outbreaks, and the small number of per-\nsons involved in each outbreak, no controlled studies have\nbeen performed to assess the value of administering post-\nexposure  prophylaxis  to  persons  at  risk.  However,  anec-\ndotal  evidence  suggests  that  administering  prophylactic\nantimicrobial drug therapy may reduce the risk of develop-\ning clinical disease (7,8,26).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nIn a recently reported event, an isolate from a chest wall\nexudate  culture  from  an  Indian  patient,  which  was  origi-\nnally identified as M. phenylpyruvica, was correctly recog-\nnized  as  B.  melitensis 22  days  after  the  specimen  was\nobtained (8). By that time, 26 laboratory workers had been\npotentially  exposed,  and  6  had  actually  manipulated  the\norganism. These 6 workers were considered to be at high\nrisk and offered a 3-week prophylactic course of combined\ndoxycycline-rifampin  or  trimethoprim-sulfamethoxazole\ntherapy.  None  of  the  5  laboratory  technologists  who\nreceived  postexposure  antimicrobial  drugs  became  ill  or\ndeveloped an antibody response. However, Brucella bac-\nteremia  developed  in  the  only  laboratory  worker  who\nrefused therapy, and she seroconverted 10 weeks after the\nspecimen  was  received  (8).  Neither  clinical  disease  nor\nseroconversion developed in the remaining 19 laboratory\nworkers who were only present in the laboratory but were\nnot in direct contact with the organism (8).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nIn  another  event,  3  laboratory  technologists  who  had\ninadvertently  worked  with  a  B.  melitensis isolate  on  an\nopen bench and sniffed the plates were given 1 week of pro-\nphylactic  doxycycline  within  24  hours  of  the  exposure;\nnone of them became ill or seroconverted (26). In a large\nexposure caused by breakage of a tube containing B. abor-\ntus, patients who started therapy immediately after serocon-\nversion was detected and before symptoms developed (10\nweeks after the laboratory accident) had a benign clinical\ncourse, suggesting that even if the disease was not prevent-\ned, a certain degree of attenuation probably occurred (7).\nBecause  of  the  high  attack  rate  of  brucellosis  among\nexposed  workers,  the  unpredictable  and  often  chronic\ncourse of the disease, and the difficulties in eradicating the\norganism  once  a  symptomatic  infection  has  been  estab-\nlished, postexposure prophylaxis is probably indicated for\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.3**r== 0.3**\nEmerging Infectious Diseases • www.cdc.gov/eid • Vol. 11, No. 8, August 2005\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nall  persons  after  an  obvious  exposure  to  living  brucellae\n(26). Antimicrobial  drug  prophylaxis  with  a  combination\nof oral doxycycline 100 mg twice a day plus rifampin 600\nmg 4 × per day for 3 weeks should be started as soon as the\nexposure  to  confirmed  Brucella organisms  is  recognized\n(8). For pregnant women, administration of trimethoprim-\nsulfamethoxazole 160/800 mg 2 × per day for 3 weeks has\nbeen advocated (8).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nPostexposure Follow-up\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nWhether  exposed  persons  have  received  prophylactic\ntherapy or not, increased surveillance for clinical signs of\ndisease should be conducted for at least 6 months. In addi-\ntion,  exposed  laboratory  workers  should  be  followed  for\npossible subclinical infections or early signs of disease by\nperiodic serologic testing. Weekly or semiweekly serolog-\nic surveillance is recommended for the first 3 months and\nonce  a  month  thereafter  for  3  to  9  additional  months  to\ndetect late infections resulting from prolonged incubation\n(7,8).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nHuman brucellosis has variable clinical manifestations\nand may mimic other infections, particularly influenza, as\nwell as noninfectious conditions. Therefore, the diagnosis\nof  the  disease  requires  a  high  index  of  suspicion.\nEducation  of  exposed  personnel  on  the  symptoms  of  the\ndisease and the need for periodic and timely serologic fol-\nlow-up are particularly important. Increased vigilance dur-\ning  the  flu  season  employing  a  broad  case  definition  is\nclearly  needed,  and  administration  of  influenza  vaccine\nshould be strongly recommended to all Brucella-exposed\npersons. If symptoms develop, blood cultures and cultures\nof other normally sterile body fluids, as clinically indicat-\ned, should be obtained, and a new antibody titer should be\ndetermined.  Although  use  of  nucleic  acid  amplification\nmethods  has  been  shown  to  enable  early  detection  of\ninfected persons (38), these tests are not yet commercially\navailable.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nTreating Infected Persons and\nPosttreatment Follow-up\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nPersons  in  whom  culture-confirmed  or  serologically\nproven disease develops should receive therapy with oral\ndoxycycline and rifampin for 6 weeks, or a combination of\noral doxycycline for 6 weeks and an intramuscular amino-\nglycoside  (gentamicin  or  streptomycin)  for  the  first  2\nweeks (2,7). Use of 3 drugs is usually reserved for compli-\ncated cases or life-threatening clinical manifestations, such\nas endocarditis or meningitis (1). Pregnant women should\nreceive  trimethoprim-sulfamethoxazole  160/800  mg  2  ×\nper day for 6 weeks (1).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nEven  when  patients  are  appropriately  treated,  the  risk\nfor  relapse  remains  high  (≈20%)  (39,40).  Therefore,\npatients who have completed a full therapeutic antimicro-\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nbial course should be followed clinically and serologically\nfor  1  year.  Failure  to  show  declining  antibody  titers  may\nindicate incomplete cure. If symptoms consistent with bru-\ncellosis develop, blood cultures and serologic tests should\nbe performed to detect relapses of the disease (39). Use of\nnucleic acid amplification methods in the future may fur-\nther improve the detection of patients in whom the organ-\nism was not eliminated (40).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nAlthough human brucellosis has been eradicated from\nmost  industrialized  countries,  isolated  cases  of  disease,\nusually  related  to  travel  or  import  of  contaminated  food\nfrom disease-endemic areas, continue to occur. Because of\nthe  low  incidence  of  disease  in  industrialized  countries,\nclinical  laboratory  technologists  have  become  unfamiliar\nwith  identifying  and  handling  Brucella species.  Unsafe\nlaboratory  practices  while  manipulating  Brucella isolates\nhave resulted in inadvertent exposure to the organism and\nmany cases of laboratory-acquired disease, indicating lack\nof preparedness to cope with bioterrorism threats involv-\ning  brucellae.  Education  of  laboratory  personnel  on  the\nidentification  of  Brucella  species,  adherence  to  and\nenforcement  of  standard  precautions,  thorough  investiga-\ntion of laboratory exposures, administration of prophylac-\ntic  antimicrobial  drug  therapy,  and  close  follow-up  of\ndirectly exposed persons are strongly recommended.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nDr.  Yagupsky  is  the  director  of  the  Clinical  Microbiology\nLaboratory  of  the  Soroka  University  Medical  Center,  Beer-\nSheva, Israel. His main research areas are Kingella kingae infec-\ntions and diagnosis of human brucellosis.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nDr.  Baron  is  director  of  the  Clinical  Microbiology\nLaboratory at Stanford University Medical Center and professor\nof pathology at Stanford University Medical College. Her inter-\nests include the development of rapid infectious disease diagnos-\ntic  tests  and  educating  both  scientists  and  the  public  about\nrelevant aspects of infectious diseases.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nReferences\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n1. Young  EJ.  Brucella  species.  In:  Mandell  GL,  Bennett  JE,  Dolin  R,\neditors.  Mandell,  Douglas  and  Bennett’s  principles  and  practice  of\ninfectious  diseases.  Philadelphia:  Elsevier,  Churchill,  Livingstone;\n2005. p. 2669–74.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n2. Greenfield  RA,  Drevets  DA,  Machado  LJ,  Voskuhl  GW,  Cornea  P,\nBronze MS. Bacterial pathogens as biological weapons and agents of\nbioterrorism. Am J Med Sci. 2002;323:299–315.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n3. Meyer KF, Eddie B. Laboratory infections due to Brucella. J Infect\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nDis. 1941;68:24–32.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n4. Trever  RW,  Cluff  LE,  Peeler  RN,  Bennett  IL.  Brucellosis.  I.\nIntern  Med.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\ninfection.  Arch\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nacute\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nLaboratory-acquired\n1959;103:381–97.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n5. Pike RM. Past and present hazards of working with infectious agents.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nArch Pathol Lab Med. 1978;102:333–6.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n6. Pike  RM.  Laboratory  associated  infections:  incidence,  fatalities,\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.2**r== 0.3**\nEmerging Infectious Diseases • www.cdc.gov/eid • Vol. 11, No. 8, August 2005\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n7. Fiori PL, Mastrandrea S, Rappelli P, Cappuccinelli P. Brucella abor-\ntus infection acquired in microbiology laboratories. J Clin Microbiol.\n2000;38:2005–6.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n24. Grammont-Cupillard M, Berthet-Badetti, Dellamonica P. Brucellosis\nfrom sniffing bacteriological cultures. Lancet. 1996;348:1733–4.\n25. Arlett  PR.  A case  of  laboratory-acquired  brucellosis.  BMJ.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n8. Robichaud S, Libman M, Behr M, Rubin E. Prevention of laborato-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1996;313:1130–2.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nry-acquired brucellosis. Clin Infect Dis. 2004;38:e119–e22.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n26. Zervos MJ, Bostic G. Exposure to Brucella in the laboratory. Lancet.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n9. Fox MD, Kaufman AF. Brucellosis in the United States, 1965–1974.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1997;349:651.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nJ Infect Dis. 1977;136:312–6.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n10. Caron E, Peyrard T, Kohler S, Cabane S, Liautard JP, Dornard J. Live\nBrucella spp.  fail  to  induce  tumor  necrosis  alpha  excretion  upon\ninfection\nImmun.\n1994;62:5267–74.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nof  U937-derived\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nphagocytes.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nInfect\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n11.  Yagupsky  P,  Peled  N,  Press  J,  Abramson  O,  Abu-Rashid  M.\nComparison  of  BACTEC  9240  Peds  Plus  medium  and  isolator  1.5\nmicrobial  tube  for  detection  of  Brucella  melitensis  from  blood  cul-\ntures. J Clin Microbiol. 1997;35:1382–4.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n27. Yagupsky P, Peled N, Riesenberg K, Banai M. Exposure of hospital\npersonnel  to  Brucella  melitensis and  occurrence  of  laboratory-\nacquired  disease  in  an  endemic  area.  Scand  J  Infect  Dis.\n2000;32:31–5.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n28. Memish  ZA,  Alazzawi  M,  Bannatyne  R.  Unusual  complication  of\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nbreast implants: Brucella infection. Infection. 2001;29:291–2.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n29. Memish ZA, Mah MW. Brucellosis in laboratory workers at a Saudi\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nArabian hospital. Am J Infect Control. 2001;29:48–52.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n30. Blankenship  RM,  Sanford  JP.  Brucella  canis.  A cause  of  undulant\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n12. Yagupsky  P.  Detection  of  brucellae  in  blood  cultures.  J  Clin\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.3**\nfever. Am J Med. 1975;59:424–6.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nMicrobiol. 1999;37:3437–42.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n13. Yagupsky P, Peled N. Use of the Isolator 1.5 microbial tube for detec-\ntion  of  Brucella  melitensis in  synovial  fluid.  J  Clin  Microbiol.\n2002;40:3878.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n14. Noviello S, Gallo R, Kelly M, Limberger RJ, DeAngelis K, Cain L,\nInfect  Dis.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\net  al.  Laboratory-acquired  brucellosis.  Emerg\n2004;10:1848–50.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n31. Montes  J,  Rodriguez  MA,  Martin T,  Martin  F.  Laboratory-acquired\nmeningitis  caused  by  Brucella  abortus strain  19.  J  Infect  Dis.\n1986;154:915–6.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n32. Olle-Goig  JE,  Canela-Soler  J.  An  outbreak  of  Brucella  melitensis\ninfection by airborne transmission among laboratory workers. Am J\nPublic Health. 1987;77:335–8.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n33. Ozaras R, Celik AD, Demirel A. Acute hepatitis due to brucellosis in\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n15. Nelson KE, Ruben Fl, Andersen B. An unusual outbreak of brucel-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\na laboratory technician. Eur J Intern Med. 2004;15:264.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nlosis. Arch Intern Med. 1975;135:691–5.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n16. Al-Aska  AK,  Chagla  AH.  Laboratory-acquired  brucellosis.  J  Hosp\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nInfect. 1989;14:69–71.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n34. Ergonul  O,  Celikbas  A,  Tezeren  D,  Guvener  E,  Dokuzoguz  B.\nAnalysis of risk factors for laboratory-acquired Brucella infections. J\nHosp Infect. 2004;56:223–7.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n17. Staszkiewicz J, Lewis CM, Colville J, Zervos M, Band J. Outbreak of\nBrucella  melitensis  among  microbiology  laboratory  workers  in  a\ncommunity hospital. J Clin Microbiol. 1991;29:287–90.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n35. Gilligan  PH, York  MK.  Basic  protocols  for  level A laboratories  for\nthe  presumptive  identification  of  Brucella species.  Washington:\nAmerican Society for Microbiology; 2002.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n18. Peiris V, Fraser S, Fairhurst M, Weston D, Kaczmarski E. Laboratory\nsome  pitfalls.  Lancet.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\ninfection:\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.7**\ndiagnosis  of  Brucella\n1992;339:1415–6.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n36. Transport of infectious substances. In: Laboratory biosafety manual.\n2nd  ed.  (revised).  Geneva:  World  Health  Organization;  2003.  p.\n353–4.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n19. Batchelor  BI,  Brindle  RJ,  Gilks  GF,  Selkon  JB.  Biochemical  mis-\nidentification  of  Brucella  melitensis  and  subsequent  laboratory-\nacquired infections. J Hosp Infect. 1992;22:159–62.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n37. Centers  for  Disease  Control  and  National  Institutes  of  Health.\nBiosafety in microbiological and biomedical laboratories. Pub no. 17-\n40-50-3. Washington: US Government Printing Office; 1988.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n20. Luzzi  GA,  Brindle  R,  Sockett  PN,  Solera  J,  Klenerman  P,  Warrell\nDA.  Brucellosis:  imported  and  laboratory-acquired  cases,  and  an\noverview  of  treatment  trials.  Trans  R  Soc  Trop  Med  Hyg.\n1993;87:138–41.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n21. Chusid MJ, Russler SK, Mohr BA, Margolis DA, Hillery CA, Kehl\nKC.  Unsuspected  brucellosis  diagnosed  in  a  child  as  a  result  of  an\noutbreak  of  laboratory-acquired  brucellosis.  Pediatr  Infect  Dis  J.\n1993;12:1031–3.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n22. Martin-Mazuelos  E,  Nogales  MC,  Florez  C,  Gomez-Mateos  JM,\nLozano F, Sanchez A. Outbreak of Brucella melitensis among micro-\nbiology laboratory workers. J Clin Microbiol. 1994;32:2035–6.\n23. Gruner E, Bernasconi E, Galeazzi RL, Buhl D, Heinzle R, Nadal D.\nBrucellosis; an occupational hazard for medical laboratory personnel.\nReport of five cases. Infection. 1994;22:33–6.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n38. Queipo-Ortuño MI, Morata P, Ocon P, Manchado P, Colmenero JD.\nRapid diagnosis of human brucellosis by peripheral-blood PCR assay.\nJ Clin Microbiol. 1997;35:2927–30.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n39. Abramson O, Abu-Rashid M, Gorodischer R, Yagupsky P. Failure of\nshort  antimicrobial  treatments  for  human  brucellosis.  Antimicrob\nAgents Chemother. 1997;41:1621–2.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n40. Morata  P,  Queipo-Ortuño  MI,  Reguera  JM,  Garcia-Ordoñez  MA,\nPichardo  C,  Colmenero  JD.  Post-treatment  follow-up  of  brucellosis\nby PCR assay. J Clin Microbiol. 1999;37:4163–6.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nAddress  for  correspondence:  Pablo  Yagupsky,  Clinical  Microbiology\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nLaboratory, Soroka University Medical Center, Ben-Gurion University of\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nthe  Negev,  Beer-Sheva  84101,  Israel;  fax:  972-8640-3541;  email:\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.3**\nyagupsky@bgumail.bgu.ac.il\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.3**r== 0.3**\nEmerging Infectious Diseases • www.cdc.gov/eid • Vol. 11, No. 8, August 2005",
         "extracted",
         "Laboratory-acquired Brucellosis;Acute hepatitis due to brucellosis in a laboratory technician.;Prevention of Laboratory-Acquired Brucellosis;Analysis of risk factors for laboratory-acquired brucella infections.;Use of the Isolator 1.5 Microbial Tube for Detection of Brucella melitensis in Synovial Fluid;Bacterial Pathogens as Biological Weapons and Agents of Bioterrorism;Unusual Complication of Breast Implants: Brucella Infection;Brucellosis in laboratory workers at a Saudi Arabian hospital.;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Brucella abortus Infection Acquired in Microbiology Laboratories;Posttreatment Follow-Up of Brucellosis by PCR Assay;Detection of Brucellae in Blood Cultures;Rapid diagnosis of human brucellosis by peripheral-blood PCR assay;Failure of short antimicrobial treatments for human brucellosis;Comparison of BACTEC 9240 Peds Plus medium and isolator 1.5 microbial tube for detection of Brucella melitensis from blood cultures;Exposure to Brucella in the laboratory;Brucellosis from sniffing bacteriological cultures;Grand Rounds—Hammersmith Hospital: A case of laboratory acquired brucellosis;Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases;Live Brucella spp. fail to induce tumor necrosis factor alpha excretion upon infection of U937-derived phagocytes;Outbreak of Brucella melitensis among microbiology laboratory workers;Unsuspected brucellosis diagnosed in a child as a result of an outbreak of laboratory-acquired brucellosis.;Brucellosis: imported and laboratory-acquired cases, and an overview of treatment trials.;Biochemical mis-identification of Brucella melitensis and subsequent laboratory-acquired infections.;Laboratory diagnosis of brucella infection: some pitfalls;Outbreak of Brucella melitensis among microbiology laboratory workers in a community hospital.;An outbreak of Brucella melitensis infection by airborne transmission among laboratory workers.;Laboratory-acquired meningitis caused by Brucella abortus strain 19.;Past and present hazards of working with infectious agents.;From the Center for Disease Control;Brucella canis: A cause of undulant fever;An unusual outbreak of brucellosis.;Brucellosis: I. Laboratory-Acquired Acute Infection;Brucellosis in the United States.;Brucella species;Brucellosis: An occupational hazard for medical laboratory personnel. Report of five cases;Beer-Sheva 84101, Israel, fax: 972-8640-3541;Prevention of laboratory-acquired brucellosis.;(revised). Geneva: World Health Organization;Basic protocols for level A laboratories for the presumptive identification of Brucella species. Washington: American Society for Microbiology;Exposure of hospital personnel to Brucella melitensis and occurrence of laboratory-acquired disease in an endemic area.;Biosafety in microbiological and biomedical laboratories;Laboratory-associated infections: incidence, fatalities, causes, and prevention.;Principles and practice of infectious diseases. Vols 1 and 2.;Brucellosis in the United States, 1965-1974.;Laboratory Infections Due to Brucella",
         "Laboratory Exposures to Brucellae and Implications for Bioterrorism",
         "None"
        ],
        [
         "14",
         "00127bef3dbe9388b9ddf936fc4d242c208dc606",
         "This paper addresses the problem of motion planning (MP) in dynamic environments. It is first argued that dynamic environments impose a real-time constraint upon MP: it has a limited time only to compute a motion, the time available being a function of the dynamicity of the environment. Now, given the intrinsic complexity of MP, computing a complete motion to the goal within the time available is impossible to achieve in most real situations. Partial motion planning (PMP) is the answer to this problem proposed in this paper. PMP is a motion planning scheme with an anytime flavor: when the time available is over, PMP returns the best partial motion to the goal computed so far. Like reactive navigation scheme, PMP faces a safety issue: what guarantee is there that the system will never end up in a critical situation yielding an inevitable collision? The answer proposed in this paper to this safety issue relies upon the concept of inevitable collision states (ICS). ICS takes into account the dynamics of both the system and the moving obstacles. By computing ICS-free partial motion, the system safety can be guaranteed. Application of PMP to the case of a car-like system in a dynamic environment is presented.",
         "S. Petti,Thierry Fraichard",
         "https://hal.inria.fr/inria-00182046/file/2005-petti-fraichard-iros.pdf",
         "\n**BLOCK**fs== 17.2**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nSafe Motion Planning in Dynamic Environments\nStephane Petti, Thierry Fraichard\n**BLOCK**fs== 14.3**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nTo cite this version:\n**BLOCK**fs== 10.9**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nStephane Petti, Thierry Fraichard. Safe Motion Planning in Dynamic Environments. Proc. of the\nIEEE-RSJ Int. Conf. on Intelligent Robots and Systems, Aug 2005, Edmonton, AB (CA), France.\n￿inria-00182046￿\n**BLOCK**fs== 14.3**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nHAL Id: inria-00182046\n**BLOCK**fs== 14.3**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\nhttps://inria.hal.science/inria-00182046v1\n**BLOCK**fs== 10.9**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nSubmitted on 24 Oct 2007\n**BLOCK**fs== 10.9**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nHAL is a multi-disciplinary open access\narchive for the deposit and dissemination of sci-\nentific research documents, whether they are pub-\nlished or not. The documents may come from\nteaching and research institutions in France or\nabroad, or from public or private research centers.\n**BLOCK**fs== 10.9**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nL’archive ouverte pluridisciplinaire HAL, est\ndestinée au dépôt et à la diffusion de documents\nscientifiques de niveau recherche, publiés ou non,\némanant des établissements d’enseignement et de\nrecherche français ou étrangers, des laboratoires\npublics ou privés.\n**BLOCK**fs== 19.9**b== 0.8**t== 0.1**l== 0.2**r== 0.2**\nSafe Motion Planning in Dynamic Environments\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nSt´ephane Petti\nInria Rocquencourt & Ecole des Mines de Paris\nEmail: stephane.petti@inria.fr\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nThierry Fraichard\nInria Rhˆone-Alpes & Gravir Lab., Grenoble\nEmail: thierry.fraichard@inria.fr\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nAbstract— This paper addresses the problem of motion plan-\nning (MP) in dynamic environments. It is ﬁrst argued that\ndynamic environments impose a real-time constraint upon MP:\nit has a limited time only to compute a motion, the time available\nbeing a function of the dynamicity of the environment. Now,\ngiven the intrinsic complexity of MP, computing a complete\nmotion to the goal within the time available is impossible to\nachieve in most real situations. Partial Motion Planning (PMP)\nis the answer to this problem proposed in this paper. PMP is a\nmotion planning scheme with an anytime ﬂavor: when the time\navailable is over, PMP returns the best partial motion to the goal\ncomputed so far. Like reactive navigation scheme, PMP faces a\nsafety issue: what guarantee is there that the system will never\nend up in a critical situation yielding an inevitable collision?\nThe answer proposed in this paper to this safety issue relies\nupon the concept of Inevitable Collision States (ICS). ICS takes\ninto account the dynamics of both the system and the moving\nobstacles. By computing ICS-free partial motion, the system\nsafety can be guaranteed. Application of PMP to the case of a\ncar-like system in a dynamic environment is presented.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nIndex Terms— Safety - Motion Planning - Dynamic Environ-\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nments.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nI. INTRODUCTION\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nA. Overview of the Problem\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nThe problem of autonomous navigation has attracted a lot\nof interest for various robotic systems during last decades.\nTwo main paradigms have arisen to tackle this problem, the\ndeliberative approaches resulting in global motion planning\nschemes, i.e. the determination of a complete plan based on a\npriori known information and the reactive approaches taking\ndecision from real-time data by means of exteroceptive sen-\nsors, while moving. Though ﬁrst problems involved mostly\nsimple systems evolving among a stationary environment,\nit became clear that for real applications, dynamic systems\nas well as a dynamic environment had to be considered.\nthe system has\nWithin a dynamic environment however,\nthe obligation to make a decision, within a bounded time,\notherwise it might be in danger by the sole fact of being\npassive. This limited available time for the system to make\na decision, i.e. plan a motion, depends on the nature and\ndynamicity of the environment and is a hard real\ntime\nconstraint. Unfortunately, global motion planning schemes\nbased on techniques from computational geometry that were\nused for the ﬁrst problems [1], did not give much hope to\nfulﬁl such a timing constraint due to their inherent NPHard\ncomplexity [2]. Hence, reactive methods have been preferred\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nand several schemes have been presented [3], [4], [5]. Nev-\nertheless, reactive approaches exhibit strong limitations. In\norder to take fast decisions, these methods explore locally\nthe velocity space of the system from which one admissible\ncontrol is selected at a time. As a consequence these methods\nexhibit a lack of lookahead, conducting the robot\nto be\ntrapped in local minima during its trip, and a weak goal\ndirectedness keeping the robot from reaching the objective.\nDespite recent modiﬁcations on a few schemes in order to\nimprove these techniques, once systems with kinematic or\ndynamic constraints are considered, speciﬁc schemes are pro-\nposed [3], [6], [7] but cannot handle such complex systems\nin a general form. Fortunately, in the mid 90’s, probabilistic\nplanners appeared [8] and brought a new powerful\ntool\nfor rapid exploration of high dimensional state-time space,\nframework presented in [9] used for complex systems and\ndynamic environment description. Some recent work could\ndemonstrate fast planning within a dynamic environment\nusing probabilistic techniques [10] and [11].\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nHowever, this work does not take into consideration the\nfact that a complete trajectory to the goal might not be\nfound before the available allotted time has elapsed. In fact,\nas [12], we believe that for complex systems or environment,\na complete trajectory to the goal cannot be found over a\nlimited time, in general. In such a case, it becomes of the\nutmost importance to consider the behaviour of the system\nat the end of the trajectory. What if a car ends its trajectory\nin front of a wall at high speed? It becomes clear that strong\nguarantees should be given to this trajectory in order to\nhandle the safety issues raised by such a partial planning.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nA few papers only have addressed the issue of safety\nfor partial or incomplete trajectory by providing, in another\ncontext braking policies [13] or more recently by calculating\nevasive plan [14] or by insuring the system to be collision\nfree during an empirical period of time τ [12]. We believe\nthat these methods propose partial answers to the problem of\nsafety. In Fig. 1 we consider a selected milestone of a point\nmass robot with non zero velocity moving to the right (a state\nof P is therefore characterised by its position (x, y) and its\nspeed v). Depending upon its speed there is a region of states\n(in grey) for which P, even though it is not in collision and\nis τ -safe, will not have the time to brake and avoid collision.\nThese states for which no matter what the future trajectory of\nthe system is, a collision with the obstacle occurs, have been\n**BLOCK**fs== 7.4**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nP\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nv\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nτ\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nInevitable Collision\nObstacle\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nFig. 1.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nτ -safe State vs. Inevitable Collision State\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\ncharacterized by [15] as Inevitable Collision States (ICS).\nThe framework of ICS provides a suitable framework to\nestablish the relation of the collision states and the dynamic\nconstraints in order to have strong safety guarantees.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nB. Contribution of the Paper\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nIn this paper, we ﬁrst detail the mechanism of a Partial Mo-\ntion Planner (PMP) designed to handle the partial plans that\nwould result from planning under hard real time constraint\nwithin a dynamic environment. Then, we address the safety\nissue related to such a scheme by calculating trajectories\nof safe states using the framework of ICS. We provide a\nsufﬁcient condition for hard safety guarantee for the PMP,\nallowing discrete calculation of the ICS, thus reducing the\ncomputational complexity of the scheme while insuring the\nsafety of the whole trajectory.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe paper is organised as follows: §2 brieﬂy reviews\nthe work related to trajectory planning within a dynamic\nenvironment and detail the existing safety guarantees that\nhave been provided in recent approaches. After introducing\nin §3 useful notations and deﬁnitions, we detail in §4 the\nmechanism of the PMP algorithm for which we provide\nand demonstrate in §5 a sufﬁcient hard safety condition. We\npresent ﬁnally in §6 our preliminary results for a car-like\nrobot and draw our conclusion in §7.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nII. PREVIOUS WORKS\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nRecently, a few reactive approaches have been modiﬁed\nin order to increase their lookahead and goal directedness.\nBasically, these modiﬁed methods rely on a map, built at\nexecution-time, like an occupancy grid, or known a priori.\nThese modiﬁed reactive methods use techniques stemming\nin order to ex-\nfrom global motion planning techniques,\nplore the velocity space of the system and select a discrete\nsequence of velocities, introducing in a sense the concept\nof partial planning. The trajectory to be followed, ie this\ndiscrete sequence of velocities, is determined by mean of\na navigation function [16] or an incremental algorithm that\nbuilds a tree within the velocity space [17], [18], [19].\nHowever, complex systems or environments remain difﬁcult\nto handle in a general form.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.1**l== 0.5**r== 0.1**\nthe environment, resulting thus in safer motion. Beside early\nwork based on dynamic programming, like the dynamic A*\nalgorithm (D*) [20], [21],\nthe approach of motion plan-\nning has been mainly reconsidered with the new tools of\nprobabilistic techniques based on graph or tree construction,\nshowing impressive results [10], [11]. Only latest work on\nthis issue, recognise the possibility of complete trajectory\nplanning failure [12], [14] and attempts to provide safety\nguarantee. However the guarantees apply over speciﬁc, em-\npirical time period without obvious physical relation with the\nconsidered dynamic system. In [14] the authors complement\nthe work presented in [10] and deﬁne safe planning as\nthe capacity of the planner to calculate an escape plan in\ncase it has failed ﬁnding a complete trajectory to the goal\nduring the allotted time. But no details with respect to the\nvalid time length of the escape plan is given. In the work\nof [12] the authors do not attempt to plan at each step a\ncomplete trajectory, the computational constraints being to\nhigh, hence considering the eventuality of partial planning.\nThey present the concept of τ -safety as a guarantee of no\ncollision during τ seconds for each milestone of the tree.\nSince the planner builds a tree between primary milestones\nat the equilibrium state, such a criteria might sufﬁce in the\npresented scheme, nevertheless it is too weak for a general\nkinodynamic framework.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nIII. NOTATIONS AND DEFINITIONS\nLet A denote a robotic system placed in a workspace\nW. The motion model of A is described by a differential\nequation of the form ˙s = f (s, u) where s ∈ S is the state\nof the system, ˙s its time derivative and u ∈ U a control. S\nis the state space and U the control space of A. Let φ ∈ Φ:\n[t0, tf ] −→ U denote a control input, ie. a time-sequence\nof controls. Starting from an initial state s0, at time t0, and\nunder the action of a control input φ, the state of the system\nA at time t is denoted by s(t) = φ(s0, t). An initial state and\na control input deﬁne a trajectory for A, ie a time sequence of\nstates. The environment is cluttered with a set of obstacles.\nAn obstacle B is a closed subset of W. The deﬁnition of\na moving obstacle is time-dependent and denoted by B(t).\nLet B(t0, ∞) denotes the obstacle B from time t0 to ∞, it\nmodels the future behaviour of B. A state s is a collision\nstate at time t if and only if ∃B such that A(s) ∩ B(t) = ∅.\nAs per [15], let us recall the formal deﬁnition of an Inevitable\nCollision State.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nDEFINITION 1 (INEVITABLE COLLISION STATE)\nA state s is an Inevitable Collision State (ICS) if and only if\n∀φ, ∃tc (for a given φ) such that φ(s, tc) is a collision state.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.3**\nA safe trajectory is deﬁned as:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nGlobal motion planning schemes have been modiﬁed as\nwell in order to gain some reactivity toward changes within\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nDEFINITION 2 (SAFE TRAJECTORY)\nA trajectory, deﬁned by the initial state s0 at time t0 and the\n**BLOCK**fs== 6.8**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nδ\n0p\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nPLANNING\n**BLOCK**fs== 9.3**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nφ\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nδ\n1p\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nPLANNING\nEXECUTION\n**BLOCK**fs== 9.2**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nφ\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nδ\n0h\n**BLOCK**fs== 7.7**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n...\n**BLOCK**fs== 3.9**b== 0.8**t== 0.1**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 2.6**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n1−np\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nPLANNING\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nPLANNING\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n(goal)\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\ntime\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nEXECUTION\n**BLOCK**fs== 9.3**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nφ\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nEXEC.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nδ\n1h\n**BLOCK**fs== 9.3**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nφ\n**BLOCK**fs== 5.4**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n1−n\n**BLOCK**fs== 6.2**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nEXECUTION\n**BLOCK**fs== 7.7**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n...\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nδ\n2h\n**BLOCK**fs== 5.2**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n0t\n**BLOCK**fs== 6.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nδ\nc\n**BLOCK**fs== 6.2**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nstep\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1t\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n2t\n**BLOCK**fs== 6.9**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nδ\nc\n**BLOCK**fs== 6.2**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nstep 1\n**BLOCK**fs== 6.9**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nδ\nc\n**BLOCK**fs== 6.2**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nstep 2\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n3t\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n1−nt\n**BLOCK**fs== 6.8**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nδ\nc\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nnt\n**BLOCK**fs== 6.2**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nstep n-1\n**BLOCK**fs== 7.2**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nδ\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n1−nh\n**BLOCK**fs== 6.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nδ\nc\n**BLOCK**fs== 6.2**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nstep n\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nft\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n1+nt\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nFig. 2. Partial Motion Planning iterative cycle\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\ncontrol input φ over [t0, tf ], is safe if and only if ∀t ∈ [t0, tf ],\ns(t) = φ(s0, t) is not an ICS.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nUsing these notations and deﬁnitions,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nthe next section\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\ndetails the Partial Motion Planning (PMP) algorithm.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nIV. PARTIAL MOTION PLANNER\n**BLOCK**fs== 10.0**b== 0.2**t== 0.4**l== 0.1**r== 0.5**\nIn general, a robotic system cannot safely stand still in\na dynamic environment (it might be collided by a moving\nobstacle). It has to plan a motion within a bounded time and\nthen execute it in order to remain safe. The time δc available\nto calculate a new motion is function of the nature and\ndynamicity of the environment. Today, the only motion plan-\nning techniques able to deal with reasonably complex robotic\nsystems are randomized. Unfortunately, these techniques do\nnot have any running-time upper-bound: there is no guarantee\nwhatsoever that a complete motion can be computed within\nthe time δc available. Like motion planning, partial motion\nplanning requires a model of the environment, the ﬁrst step\nis aimed at getting this model. This paper does not address\nhowever the problem of model construction. This model can\nbe given a priori or built using sensor observations. In space\napplications for instance, most of the moving obstacles follow\nKepler’s law, which provides a means to compute a full a\npriori model of the future [12]. In many other applications\nhowever, the moving obstacles have their own free will and\ntheir future behaviour is only partially predictable (if at all).\nIn this kind of situations,\nthe model of the future must\nbe predicted using prediction techniques such as the ones\npresented in [22] or [23].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nNote that when predictions are used, it is likely that the\nmodel of the future that is obtained will have a limited\nvalidity duration δv. This is an additional argument for\npartial motion planning: what is the point of a complete\nexploration of the future if the model of the future used is\nlikely to be grossly inaccurate? It is better to iterate a partial\nmotion planning process taking as input a regularly updated\npredicted model of the future. The periodic iterative PMP\nscheme proposed in this paper therefore accounts for both\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nthe planning time constraints and the validity duration of the\nmodel of the environment.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nThus, the PMP algorithm iterates over a cycle of duration\nδc ≤ δv as depicted in Fig. 2. It is assumed that the initial\nstate of A is ICS-free. Let us focus on the planning iteration\nstarting at time ti:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n1) An updated model of the future is acquired, ie B(ti, ∞)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nfor each moving obstacles.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n2) The state-time space of A is searched using an ran-\ndomised tree rooted at the state s(ti+1) with ti+1 =\nti + δc.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n3) At time ti+1, the current iteration is over, the best safe\npartial trajectory φi in the tree is selected according to a\ngiven criterion and is fed to the robot that will execute\n]\nit from now on. φi is deﬁned over [ti+1, ti+1 + δhi\nwith δhi the trajectory duration.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nThe algorithm operates until the last state of the planned\ntrajectory reaches a neighbourhood of the goal state.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.7**r== 0.2**\nV. SAFETY ISSUES\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nLike every method that computes partial motion only, PMP\nhas to face a safety issue: since PMP has no control over\nthe duration of the partial trajectory that is computed, what\nguarantee do we have that A will never end up in a critical\nsituations yielding an inevitable collision? The answer to that\nproblem lies in the very fact that the partial trajectory that is\ncomputed is ICS-free. Meaning that, even in the worst case\nscenario where the duration δhi of the partial trajectory is\nshorter than the cycle time δc, A can always execute one of\nthe existing safe trajectory. The overall safety is guaranteed\nas long as the initial state is ICS-free (which is something\nthat has been assumed).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nNow, determining whether a given state of A is an ICS\nor not is a complex task since it requires to consider all\npossible future trajectories for A. However, it i possible to\ntake advantage of the approximation property demonstrated\nin [15] in order to compute a conservative approximation of\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nthe set of ICS. This is done by considering only a subset of\nthe full set of possible future trajectories (see §VI.B).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nBesides, in order to further reduce the complexity of the\nPMP algorithm, we present a property that simpliﬁes the\nsafety checking of a trajectory. To begin with, we demon-\nstrate a property that states that for a given control input, all\nthe states between an ICS and the corresponding collision\nstate, are ICS.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nPROPERTY 1\nLet s be an ICS at time t0. For a given control input φ, let\ntc denote the time at which a collision occurs. Then ∀t ∈\n[t0, tc], s(t) = φ(s, t) is also an ICS.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nProof: suppose that a state si = φ(s, ti), with ti ∈ [0, tc],\nis not an ICS. By deﬁnition, ∃φj that yields no collision when\napplied to si. Let φi denote the part of φ deﬁned over [t0, ti].\nClearly, the combination of φi and φj also yields no collision\nwhen applied to s  contradiction. \n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nWe can now provide a sufﬁcient safety condition for a\npartial\ntrajectory that states that provided a trajectory is\ncollision free, if the last state of the trajectory is not an\ninevitable collision state then none of the states of the\ntrajectory are inevitable collision states.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nPROPERTY 2 (PMP SUFFICIENT SAFETY CONDITION)\nGiven a trajectory deﬁned over [t0, tf ], if\n(H1) the trajectory is collision-free and\n(H2) s(tf ) is not an ICS\nthen ∀t ∈ [t0, tf ], s(t) is not an ICS.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nProof: Suppose that ∃ti ∈ [t0, tf ] such that si = s(ti) is\nan ICS. Then, by deﬁnition, ∀φ, ∃tc such that φ(si, tc) is a\ncollision state. If t0 ≤ tc ≤ tf then collision occurs before\ntf  contradiction with H1. Now, if tc > tf then by previous\nproperty P1, we must have s(tf ) is an ICS  contradiction\nwith H2. \n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.2**r== 0.6**\nVI. CASE STUDY\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nθ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\ny\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nL\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nx\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nFig. 3. The car-like vehicle A (bicycle model).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nIn this section we present the application of PMP to the\ncase of a car-like vehicle A moving on a planar surface\nW and within a fully observable environment cluttered with\nstationary and dynamic obstacles.\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nMoving\nObstacle\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nInevitable\nCollision States\n**BLOCK**fs== 4.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nStatic Obstacle\n**BLOCK**fs== 4.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nMoving\nObstacle\n**BLOCK**fs== 4.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nInevitable\nCollision States\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nCar-like Robot\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nTrajectory\n**BLOCK**fs== 4.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nCar-like Robot\n**BLOCK**fs== 4.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nTrajectory\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n(a) (θ, v) = (0.9, 1.0)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n(b) (θ, v) = (0.9, 2.0)\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nStatic Obstacle\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nStatic Obstacle\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nMoving\nObstacle\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\nMoving\nObstacle\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nInevitable Collision\nObstacles\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nInevitable Collision\nObstacles\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nCar-like Robot\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nCar-like Robot\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n(c) (θ, v) = (0.9, 1.0)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\n(d) (θ, v) = (0.9, 2.0)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n(θ, v)-slices of\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nthe state space of A. Shaded regions\nFig. 4.\nare ICS respectively deﬁned for\nthe braking trajectory of control\n(αmin, ˙ξmin) (top), and all braking trajectories of control selected from\n[(αmin, ˙ξmax), (αmin, 0), (αmin, ˙ξmin)] (bottom).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nA. Vehicle Model\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nA moves like a car-like vehicle and its dynamics follows\nthe bicycle model. A state of A is deﬁned by the 5-tuple\ns = (x, y, θ, v, ξ) where (x, y) are the coordinates of the\nrear wheel, θ is the main orientation of A, v is the linear\nvelocity of the rear wheel, and ξ is the orientation of the\nfront wheels (Fig. 3). A control of A is deﬁned by the couple\n(α, γ) where α is the rear wheel linear acceleration. and γ\nthe steering velocity. The motion of A is governed by the\nfollowing differential equations:\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n˙x\n˙y\n˙θ\n˙vr\n˙ξ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\nvr cos θ\nvr sin θ\ntan ξvr\nL\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n+\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.8**r== 0.2**\n\n\n\n\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.8**r== 0.2**\n\n\n\n\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nα +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\nγ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n(1)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nwith α ∈ [αmin, αmax]\n(acceleration bounds), γ ∈\n[γmin, γmax] (steering velocity bounds), and |ξ| ≤ ξmax\n(steering angle bounds). L is the wheelbase of A.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nB. ICS Calculation\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nIn general, computing the ICS for a given system is an\nintricate problem since it would require to consider the set\nof all the possible future trajectories. To compute in practice\nthe ICS for a system such as A, it is taken advantage of\nthe approximation property established in [15] showing that\na conservative approximation of the ICS can be obtained\nby considering only a ﬁnite subset I of the whole set of\npossible future trajectories. For practical reasons, the duration\nof the trajectories of I has to be limited to a given time\nhorizon that determines the overall level of safety of A.\nthis horizon can be\nIn partially predictable environment,\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nStatic\nObstacle\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nMoving\nObstacle\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nICS\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nICS-free\ntrajectory\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nCollision States\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nMoving\nObstacle\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\nFig. 8. Parking lot conﬁguration for practical experiments\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\n(a) Cycab platform from INRIA.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n(b) Cameras on a parking lot.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nFig. 5. The state labelled ICS is an ICS since the three braking trajectories\nissued from it yield collisions.\n**BLOCK**fs== 5.4**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\ntree root\nSi\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\ntree nodes\n**BLOCK**fs== 5.4**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\nSc\n**BLOCK**fs== 3.5**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nSn1\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\ntarget Sr\n**BLOCK**fs== 5.4**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nICS-free\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nSn5\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nSn2\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nSn3\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nSn4\n**BLOCK**fs== 5.4**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nICS\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nFig. 6. Tree construction principle.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nset up to δv, thus providing the maximum possible safety\nguarantee. In our case, the subset I considered includes\nthe braking trajectories with a constant control selected\nfrom [(αmin, ˙ξmax), (αmin, 0), (αmin, ˙ξmin)], and applied\nover the time necessary for A to stop.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFigure 4 depicts the ICS obtained when different set of\nbraking trajectories are considered. Each subﬁgure represents\na (θ, v)-slice of the full 5D state space of A. In the top sub-\nﬁgures, only the braking trajectory of control (αmin, ˙ξmin)\nis considered.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nFor instance, the shaded regions in subﬁgure (a) corre-\nspond to states with (θ, v) = (0.9, 1.0) for which the braking\ntrajectory of control (αmin, ˙ξmin) yields a collision with one\nof the obstacles, ﬁxed or moving. In the bottom subﬁgures,\nthe three braking trajectories are considered.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nIn PMP, checking whether a state is an ICS or not is carried\nout by testing if all the braking trajectories yield a collision\nwith one of the moving obstacles. In case all the trajectories\nappear to be in collision in the future, this state is an ICS\nand is not selected. Fig. 5 illustrates how a state of the partial\ntrajectory is checked to be an ICS or not. The collision states\nrepresent the collision that will occur in the future from this\nstate for all trajectories of I. In this case, since all trajectories\ncollide in the future, this state is an ICS.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nC. Tree Construction\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n| α ∈(αmin, 0, αmax) and ˙ξ ∈( ˙ξmax, 0, ˙ξmin) } are then\napplied to sc for a duration , they yield a set of candi-\ndate trajectories ending in given states sni. These candidate\ntrajectories are pruned out: only are kept the trajectories\nthat are collision-free and whose ﬁnal state is ICS-free (as\nper property 2, such trajectories are ICS-free). Finally, the\ntrajectory whose ﬁnal state is closer to sr is selected and\nadded up to the tree. This process is repeated until the end of\nthe time available where the best partial trajectory extracted\nfrom the tree is returned.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\nD. Preliminary Results\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nIn this implementation, the duration of a cycle δc is 1s,\nand the integration step of the differential equation is 0.5s.\nIn Fig. 7 we can see an example of a navigation from a\nstill starting state to a still goal state. The environment is\ncluttered by moving and static obstacles. In 7(a) one can\nobserve in front of the car how the safe partial trajectory is\ncalculated and planned within the time-state space in order\nto avoid the obstacle moving upward. The states behind\nthe car, deﬁne the trajectory, built from partial trajectories\nfrom the previous PMP cycles and (ideally) executed by\nthe robot. In 7(b) we can observe that the car was obliged\nto slow down at the intersection of several obstacles, since\nno other safe trajectories could be found, before to re-\naccelerate. In 7(d) the system has planned a partial trajectory\nthat avoids the last static obstacle. Videos of this simula-\ntion can be found at http://emotion.inrialpes.fr\n/film-gallery.php.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nThis work is currently being integrated on a real platform\n(ﬁg. 8(a)) moving within a parking lot. External cameras\nabove the parking (ﬁg. 8(b)) have been installed in order to\nfully observe the environment and thus localize our system\nand detect static (parked cars) as well as moving obstacles\n(pedestrians and cars) whereas the trajectories of the moving\nobstacles will be predicted using the work of [23].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe exploration method used is the well known Rapidly-\nExploring Random Tree method (RRT) [24]. RRT incre-\nmentally builds a tree in the state space of A. The basic\nprinciple of RRT is depicted in Fig. 6. A state sr is randomly\nselected ﬁrst. Then, the closest node in the tree, say sc,\nis determined. Constant controls selected from U={ (α, ˙ξ)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nVII. DISCUSSION AND CONCLUSION\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nIn this paper we tackled the problem of motion planning\nwithin a dynamic environment and proposed a Partial Motion\nPlanning scheme (PMP) which handles the real-time con-\nstraint inherent to such environment. The PMP algorithm\nconsists in iteratively exploring the state-time space during\n**BLOCK**fs== 4.5**b== 0.8**t== 0.1**l== 0.3**r== 0.7**\nGoal\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nStatic\nObstacles\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nMoving\nObstacles\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nStatic\nObstacles\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nTrajectory\nexecuted\n**BLOCK**fs== 4.5**b== 0.8**t== 0.1**l== 0.5**r== 0.5**\nGoal\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nMoving\nObstacles\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nMoving Obstacles\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nTrajectory\nexecuted\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nStart\n**BLOCK**fs== 4.4**b== 0.8**t== 0.1**l== 0.7**r== 0.3**\nGoal\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nSafe Partial\nTrajectory\nPlanned\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nStatic\nObstacles\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nSafe Partial\nTrajectory Planned\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nStart\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nSafe Partial\nTrajectory Planned\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nTrajectory\nexecuted\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nStart\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nMoving\nObstacles\n**BLOCK**fs== 4.5**b== 0.8**t== 0.1**l== 0.9**r== 0.1**\nGoal\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nSafe Partial\nTrajectory\nPlanned\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nTrajectory\nexecuted\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nStart\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nStatic\nObstacles\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.3**r== 0.5**\n(b) Braking in front of unavoidable\nmoving obstacles\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.3**\n(c) Braking in front of moving obsta-\ncles\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.7**r== 0.1**\n(d) Avoidance of the last static obstacle\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nFig. 7. Results of a 2D safe iterated Partial Motion Planning (δc = 1s, vmax = 2.0m/s, ξmax = π/3rad, ˙ξmax = 0.2rad/s, αmax = 0.1m/s2)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\na ﬁxed limited time, by building a tree using probabilistic\ntechniques. During a cycle, a complete trajectory calculation\nto the goal can not be guaranteed in general, which raises\nthe issue of the safety of our system. We use the formalism\nof the Inevitable collision States (ICS) as the theoretical\nanswer to this safety problem. A trajectory of ICS-free states\nguarantees that our system always have a possibility to escape\ncritical situations. ICS computation remains however a very\ncomplicated task in its general form and we have to rely\non conservative approximation for practical ICS calculation.\nIn this paper, we also demonstrate a property that simpliﬁes\nthe safety checking of a trajectory. Finally, we present an\nimplementation of the PMP algorithm for a car-like robot\nand provide simulation results of the safe trajectory found\nby PMP for our system.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFuture work includes the coupling of the PMP algorithm\nwith a closed loop control and its integration on an exper-\nimental vehicle. Our goal is to perform experimentations\nwithin a real environment, for which a model of the future\nobstacles’ behaviour will be determined thanks to a predic-\ntion technique.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nACKNOWLEDGMENT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nThis work was partially supported by Aisin AW Corp. and\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nthe French CNRS Parknav and Predit Mobivip projects.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n[1] J.-C. Latombe, Robot motion planning. Kluwer Academic Publishers,\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nBoston, MA, 1991.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n[2] J. H. Reif and M. Sharir, “Motion planning in the presence of moving\nobstacles,” in Symp. on the Foundations of Computer Science, Portland,\nOR (US), Oct. 1985, pp. 144–154.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n[3] D. Fox, W. Burgard, and S. Thrun, “The dynamic window approach\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nto collision avoidance, Tech. Rep. IAI-TR-95-13, 1 1995.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n[4] M. Khatib, “Sensor-based motion control for mobile robots,” Ph.D.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\ndissertation, LAAS-CNRS December, 1996, 1996.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n[5] P. Fiorini and Z. Shiller, “Motion planning in dynamic environments\nusing velocity obstacles,” International Journal of Robotics Research,\nvol. 17, no. 7, pp. 760–772, July 1998.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n[6] R. Simmons, “The curvature velocity method for local obstacle\navoidance,” in International Conference on Robotics and Automation,\nMinneapolis (USA), april 1996, pp. 3375–3382.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n[7] J. Minguez, L. Montano, and J. Santos-Victor, “Reactive navigation for\nnon-holonomic robots using the ego kinematic space,” in Int. Conf. on\nRobotics and Automation, Washington (US), May 2002.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n[8] L. Kavraki, P. Svestka, J.-C. Latombe, and M. H. Overmars, “Proba-\nbilistic roadmaps for path planning in high dimensional conﬁguration\nspaces,” IEEE Transactions on Robotics and Automation, vol. 12, pp.\n566–580, 1996.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n[9] T. Fraichard and C. Laugier, “Kinodynamic planning in a structured\nand time-varying 2D workspace.” in Proceedings IEEE International\nConference on Robotics and Automation, Nice, (FR), May 1992.\n[10] D. Hsu, R. Kindel, J.-C. Latombe, and S. Rock, “Kinodynamic motion\nplanning amidst moving obstacles,” in Int. Conf. on Robotics and\nAutomation, San Francisco (US), April 2000.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n[11] J. Bruce and M. Veloso, “Real-time randomized path planning for robot\nnavigation,” in Int. Conf. on Intelligent Robots and Systems, Lausanne,\nSwitzerland, October 2002.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n[12] E. Feron, E. Frazzoli, and M. Dahleh, “Real-time motion planning\nfor agile autonomous vehicles,” in AIAA Conference on Guidance,\nNavigation and Control, Denver (US), August 2000.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n[13] T. S. Wikman, M. S. Branicky, and W. S. Newman, “Reﬂexive collision\navoidance: a generalized approach,” in Int. Conf. on Robotics and\nAutomation, vol. 3, Atlanta (US), May 1993, pp. 31–36.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n[14] D. Hsu, R. Kindel, J.-C. Latombe, and S. Rock, “Randomized kin-\nodynamic motion planning with moving obstacles,” Int. Journal of\nRobotics Research, vol. 21, no. 3, pp. 233–255, March 2002.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[15] T. Fraichard and H. Asama, “Inevitable collision states - a step towards\nsafer robots?” Advanced Robotics, vol. 18, no. 10, pp. 1001–1024,\n2004.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n[16] O. Brock and O. Khatib, “High-speed navigation using the global\ndynamic window approach,” in Int. Conf. on Robotics and Automation,\nDetroit (US), May 1999.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n[17] L. Ulrich and J. Borenstein, “VFH*: Local obstacle avoidance with\nlook-ahead veriﬁcation,” in Int. Conf. on Robotics and Automation,\nSan Fransisco (US), April 2000, pp. 2505–2511.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[18] F. Large, S. Sekhavat, Z. Shiller, and C. Laugier, “Towards real-time\nglobal motion planning in a dynamic environment using the NLVO\nconcept,” in Int. Conf. on Intelligent Robots and Systems, Lausanne,\nSwitzerland, October 2002.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n[19] C. Stachniss and W. Burgard, “An integrated approach to goal-directed\nobstacle avoidance under dynamic constraints for dynamic environ-\nments,” in Int. Conf. on Intelligent Robots and Systems, Lausanne,\nSwitzerland, October 2002.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n[20] A. Stentz, “The focussed D* algorithm for real-time replanning,” in\nInt. Joint Conf. on Artiﬁcial Intelligence, Montreal, Quebec, 1995, pp.\n1652–1659.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n[21] S. Koenig and M. Likhachev, “Improved fast replanning for robot nav-\nigation in unknown terrain,” in Int. Conf. on Robotics and Automation,\n2002.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n[22] M. Bennewitz, W. Burgard, and S. Thrun, “Learning motion patterns\nof persons for mobile service robots,” in Int. Conf. on Robotics and\nAutomation, Washington DC (US), 2002, pp. 3601–3606.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n[23] D. Vasquez and T. Fraichard, “Motion prediction for moving objects:\na statistical approach,” in Int. Conf. on Robotics and Automation, New\nOrleans, LA, April 2004.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n[24] S. LaValle and J. Kuffner, “Randomized kinodynamic planning,” in\nInt. Conf. on Robotics and Automation, Detroit (US), May 1999, pp.\n473–479.",
         "extracted",
         "Motion prediction for moving objects: a statistical approach;Inevitable collision states. A step towards safer robots?;Towards real-time global motion planning in a dynamic environment using the NLVO concept;An integrated approach to goal-directed obstacle avoidance under dynamic constraints for dynamic environments;Real-time randomized path planning for robot navigation;Improved fast replanning for robot navigation in unknown terrain;Learning motion patterns of persons for mobile service robots;Reactive navigation for non-holonomic robots using the ego-kinematic space;Randomized Kinodynamic Motion Planning with Moving Obstacles;Real-time motion planning for agile autonomous vehicles;Kinodynamic motion planning amidst moving obstacles;VFH/sup */: local obstacle avoidance with look-ahead verification;Randomized Kinodynamic Planning;High-speed navigation using the global dynamic window approach;Motion Planning in Dynamic Environments Using Velocity Obstacles;The dynamic window approach to collision avoidance;Probabilistic roadmaps for path planning in high-dimensional configuration spaces;The curvature-velocity method for local obstacle avoidance;The Focussed D* Algorithm for Real-Time Replanning;Reflexive collision avoidance: a generalized approach;Motion planning in the presence of moving obstacles;Robot Motion Planning;Sensor-Based Motion Control for Mobile Robots;Kinodynamic planning in a structured and time-varying 2-D workspace",
         "Safe motion planning in dynamic environments",
         "None"
        ],
        [
         "15",
         "001356ecc6393aeb0997ee1bc5b47b423587fdff",
         "In eukaryotic cellular proteins, protein N‐myristoylation has been recognized as a protein modification that occurs mainly on cytoplasmic or nucleoplasmic proteins. In this study, to search for a eukaryotic N‐myristoylated transmembrane protein, the susceptibility of the N‐terminus of several G‐protein‐coupled receptors (GPCRs) to protein N‐myristoylation was evaluated by in vitro and in vivo metabolic labeling. It was found that the N‐terminal 10 residues of B96Bom, a Bombyx mori GPCR, efficiently directed the protein N‐myristoylation. Analysis of a tumor necrosis factor (TNF) fusion protein with the N‐terminal 90 residues of B96Bom at its N‐terminus revealed that (a) transmembrane domain 1 of B96Bom functioned as a type I signal anchor sequence, (b) the N‐myristoylated N‐terminal domain (58 residues) was translocated across the membrane, and (c) two N‐glycosylation motifs located in this domain were efficiently N‐glycosylated. In addition, when Ala4 in the N‐myristoylation motif of B96Bom90‐TNF, Met‐Gly‐Gln‐Ala‐Ala‐Thr(1–6), was replaced with Asn to generate a new N‐glycosylation motif, Asn‐Ala‐Thr(4–6), efficient N‐glycosylation was observed on this newly introduced N‐glycosylation site in the expressed protein. These results indicate that the N‐myristoylated N‐terminus of B96Bom is translocated across the membrane and exposed to the extracellular surface. To our knowledge, this is the first report showing that a eukaryotic transmembrane protein can be N‐myristoylated and that the N‐myristoylated N‐terminus of the protein can be translocated across the membrane.",
         "T. Utsumi,H. Ohta,Y. Kayano,N. Sakurai,Y. Ozoe",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2004.04487.x",
         "\n**BLOCK**fs== 14.9**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nThe N-terminus of B96Bom, a Bombyx mori G-protein-\ncoupled receptor, is N-myristoylated and translocated\nacross the membrane\nToshihiko Utsumi1, Hiroto Ohta2, Yoshiyuki Kayano1, Nagisa Sakurai1 and Yoshihisa Ozoe2\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\n1 Department of Biological Chemistry, Faculty of Agriculture, Yamaguchi University, Japan\n2 Department of Life Science and Biotechnolgy, Faculty of Life and Environmental Science, Shimane University, Japan\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nKeywords\nB96Bom; biogenic amine receptor;\nG-protein-coupled receptor; membrane\nprotein; protein N-myristoylation\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nCorrespondence\nT. Utsumi, Department of Biological\nChemistry, Faculty of Agriculture,\nYamaguchi University, Yamaguchi 753-8515,\nJapan\nFax: +81 83 933 5820\nTel: +81 83 933 5856\nE-mail: utsumi@yamaguchi-u.ac.jp\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.7**\n(Received 5 August 2004, revised 4\nNovember 2004, accepted 16 November\n2004)\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\ndoi:10.1111/j.1742-4658.2004.04487.x\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.4**r== 0.1**\nIn eukaryotic cellular proteins, protein N-myristoylation has been recog-\nnized as a protein modiﬁcation that occurs mainly on cytoplasmic or\nnucleoplasmic proteins. In this study, to search for a eukaryotic N-myris-\ntoylated transmembrane protein, the susceptibility of the N-terminus of\nseveral G-protein-coupled receptors (GPCRs) to protein N-myristoylation\nwas evaluated by in vitro and in vivo metabolic labeling. It was found that\nthe N-terminal 10 residues of B96Bom, a Bombyx mori GPCR, efﬁciently\ndirected the protein N-myristoylation. Analysis of a tumor necrosis factor\n(TNF) fusion protein with the N-terminal 90 residues of B96Bom at its\nN-terminus revealed that (a) transmembrane domain 1 of B96Bom func-\ntioned as a type I signal anchor sequence, (b) the N-myristoylated N-ter-\nminal domain (58 residues) was translocated across the membrane, and (c)\ntwo N-glycosylation motifs located in this domain were efﬁciently N-gly-\ncosylated. In addition, when Ala4 in the N-myristoylation motif of\nB96Bom90-TNF, Met-Gly-Gln-Ala-Ala-Thr(1–6), was replaced with Asn\nto generate a new N-glycosylation motif, Asn-Ala-Thr(4–6), efﬁcient N-gly-\ncosylation was observed on this newly introduced N-glycosylation site in\nthe expressed protein. These results indicate that\nthe N-myristoylated\nN-terminus of B96Bom is translocated across the membrane and exposed\nto the extracellular surface. To our knowledge, this is the ﬁrst report show-\ning that a eukaryotic transmembrane protein can be N-myristoylated and\nthat the N-myristoylated N-terminus of the protein can be translocated\nacross the membrane.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nCovalent attachment of fatty acids to proteins is a\nwell-recognized form of protein modiﬁcation. Two\nforms of protein fatty acylation, protein N-myristoy-\nlation and protein palmitoylation, occur on a wide\nvariety of\neukaryotic and viral proteins. Protein\nN-myristoylation is the attachment of myristic acid, a\n14-carbon saturated fatty acid, to the N-terminal Gly\nof proteins [1–5]. Many of the N-myristoylated pro-\nteins play key roles in regulating cellular structure and\nfunction. They include proteins involved in a wide\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nvariety of cellular signal-transduction pathways such\nas protein kinases, phosphatases, guanine nucleotide-\nbinding proteins, and Ca2+-binding proteins. In many\ncases, the functions of these N-myristoylated proteins\nare regulated by reversible membrane binding medi-\nated by protein N-myristoylation.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nProtein N-myristoylation has been recognized as a\nprotein modiﬁcation that occurs mainly on cytoplasmic\nand nucleoplasmic proteins, and no eukaryotic trans-\nmembrane protein has been demonstrated to be\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nAbbreviations\nADRb1, human b-adrenoceptor type 1; B96Bom, B. mori tyramine receptor; DRD4, human dopamine D4 receptor; GPCR, G-protein-coupled\nreceptor; HBV, hepatitis B virus; TM1, transmembrane domain 1; TNF, tumor necrosis factor.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nN-myristoylated so far. In viral proteins, however,\nprotein N-myristoylation has been found to occur on\ntransmembrane proteins such as the large L envelope\nprotein of hepatitis B virus (HBV), a polytopic mem-\nbrane protein [6,7]. Therefore, it seemed possible that\nN-myristoylated eukaryotic transmembrane proteins\nexist. In this study, we tried to search for eukaryotic\ntransmembrane proteins\nthat are N-myristoylated.\nAs protein palmitoylation, another type of protein\nacylation,\nis often observed on G-protein-coupled\nreceptors (GPCRs) [8], which are seven transmembrane\nreceptors, we focused our attention on these recep-\ntors to search for N-myristoylated transmembrane\nproteins.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nGPCRs represent a superfamily of proteins\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nthe\nfunction of which is dependent on heterotrimeric\nGTP binding and hydrolyzing proteins (G-proteins).\nThey link the cells to their environment by receiving\nstimuli, relaying the message to the cells, and regula-\nting the cellular response [9,10]. GPCRs represent one\nof the largest protein superfamilies in the eukaryotic\ngenome [11,12]. In fact,  9000 entries are listed in the\nGPCR database which is available as a public WWW-\nserver (http://www.gpcr.org/7tm/) [13]. To search for\nN-myristoylated GPCRs, possible candidates with an\nN-myristoylation consensus motif at their N-termini\nwere ﬁrst extracted from the GPCR database and then\nthe susceptibility of the proteins to protein N-myristoy-\nlation was examined experimentally. It was found that\nthe N-terminus of B96Bom [14,15], a Bombyx mori\nGPCR (tyramine receptor), is efﬁciently N-myristoylat-\ned and that\nthe N-myristoylated N-terminus of\nB96Bom is translocated across the membrane and\nexposed to the extracellular surface. To our knowledge,\nthis is the ﬁrst report showing that a eukaryotic trans-\nmembrane protein can be N-myristoylated and that the\nN-myristoylated N-terminus of\nthe protein can be\ntranslocated across the membrane.\n**BLOCK**fs== 10.7**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nResults\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.1**r== 0.6**\nThe N-terminus of B96Bom, an insect GPCR,\nis N-myristoylated\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nTo search for N-myristoylated transmembrane pro-\nteins, the susceptibility of the N-terminal sequence of\nseveral GPCRs to protein N-myristoylation was evalu-\nated by metabolic labeling using an in vitro translation\nsystem. Three GPCRs\ntyramine receptor\n(B96Bom), human dopamine D4 receptor (DRD4),\nand human b-adrenoceptor type 1 (ADRb1)] with an\nN-myristoylation consensus motif at their N-termini\nwere selected as possible candidates (Fig. 1).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n[B. mori\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nFig. 1. The N-myristoylation consensus motif\nresides in the\nN-terminus of various biogenic amine receptors. Amino acids at\npositions 3 and 6 are boxed.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\nThe N-terminal 10 residues, including the initiating\nMet of the mature domain of tumor necrosis factor\n(TNF), were replaced with those of these three recep-\ntors to generate B96Bom10-TNF, DRD4-TNF, and\nADRb1-TNF, respectively. Then, the susceptibility of\nthese proteins to protein N-myristoylation was evalu-\nated by in vitro metabolic labeling using rabbit reticu-\nlocyte lysate. In this case, Dpro-TNF,\nthe mature\ndomain of TNF in which the initiating Met was intro-\nduced at the N-terminus, was used as a nonmyristoyl-\nated control TNF. As shown in Fig. 2B, translation of\nmRNAs\ncoding for Dpro-TNF, B96Bom10-TNF,\nDRD4-TNF, and ADRb1-TNF in the presence of\n[3H]leucine gave rise to protein bands with the expec-\nted molecular mass (17 kDa). Translation of mRNA\ncoding for Gi1a-TNF, in which the N-terminal 10 resi-\ndues of the naturally N-myristoylated Gi1a protein\nwere linked to the N-terminus of the mature domain\nof TNF, gave rise to a protein band with an expected\nmolecular mass of 19 kDa. The fainter protein bands\nwith a molecular mass  2 kDa larger than expected\nwere also observed in the translation products of Gi1a-\n[3H]Myristic acid labeling\nTNF and Dpro-TNF.\nrevealed that Dpro-TNF and ADRb1-TNF were not\nN-myristoylated,\nand\nDRD4-TNF were N-myristoylated (Fig. 2C). The efﬁ-\nciency of [3H]myristic acid incorporation ([3H]myristic\nacid incorporation ⁄ [3H]leucine\ninto\nB96Bom10-TNF was comparable to that into Gi1a-\nindicating that B96Bom10-TNF is efﬁciently\nTNF,\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nB96Bom10-TNF\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nincorporation)\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nwhereas\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nA\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nB\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nC\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nA\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nB\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nC\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFig. 2. The N-terminal 10 residues of B96Bom efﬁciently direct pro-\ntein N-myristoylation.\n(A) mRNAs encoding Gi1a-TNF, Dpro-TNF,\nB96Bom10-TNF, DRD4-TNF and ADRb1-TNF were translated\nin vitro in the presence of [3H]leucine or [3H]myristic acid using rab-\nbit reticulocyte lysate. After immunoprecipitation with antibody to\nTNF, the labeled translation products were analyzed by SDS ⁄ PAGE\nand ﬂuorography. (B) [3H]Leucine incorporation; (C) [3H]myristic acid\nincorporation.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nN-myristoylated. In contrast, the efﬁciency of [3H]my-\nristic acid incorporation into DRD4-TNF was relat-\nively low.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nTo conﬁrm the N-myristoylation of B96Bom, the\nsusceptibility of B96Bom to protein N-myristoylation\nwas evaluated by in vitro metabolic labeling using\nintact B96Bom cDNA. As the efﬁciency of protein\nexpression directed by full-length B96Bom cDNA was\nvery low in both in vitro and in vivo expression systems\n(data not shown), two fusion proteins (B96Bom58-\nTNF, B96Bom90-TNF) in which the N-terminal 58 or\n90 residues of B96Bom were linked to the mature\ndomain of TNF were generated. Then their susceptibil-\nity to protein N-myristoylation was evaluated. The\nN-terminal 58 residues are the putative N-terminal ex-\noplasmic domain of B96Bom, and N-terminal 90 resi-\ndues include transmembrane domain 1 (TM1) and its\nN-terminal and C-terminal ﬂanking regions. As shown\nin Fig. 3B,\nfor\nB96Bom58-TNF gave rise to two translation products:\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.3**r== 0.5**\nof mRNA coding\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\ntranslation\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nFig. 3. The N-terminus of B96Bom is N-myristoylated. (A) mRNAs\nencoding B96Bom58-TNF and B96Bom90-TNF were translated\nin vitro in the presence of [3H]leucine or [3H]myristic acid using rab-\nbit reticulocyte lysate. After immunoprecipitation with antibody to\nTNF, the labeled translation products were analyzed by SDS ⁄ PAGE\nand ﬂuorography. (B) [3H]Leucine incorporation; (C) [3H]myristic acid\nincorporation.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\none was the major product with an apparent molecular\nmass of 28 kDa, and the other was a fainter band with\nan  4 kDa larger molecular mass. [3H]Myristic acid\nlabeling revealed that the 28 kDa major protein band\nwas efﬁciently N-myristoylated (Fig. 3C). Translation\nof mRNA coding for B96Bom90-TNF gave rise to two\ntranslation products: one was the major product with\nan apparent molecular mass of 27 kDa, and the other\nwas a fainter band with a molecular mass of 20 kDa.\n[3H]Myristic acid labeling revealed that the 27 kDa\nmajor protein band was also efﬁciently N-myristoylat-\ned (Fig. 3C). These results indicate that the N-terminus\nof intact B96Bom is cotranslationally N-myristoylated.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.2**\nThe N-terminal domain of B96Bom is\ntranslocated across the membrane\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nIn many of the GPCRs, the N-terminal domain is\ntranslocated across the endoplasmic reticulum during\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nprotein synthesis and then targeted to the plasma\nmembrane and exposed to the extracellular space. To\ndetermine whether\nthe N-myristoylated N-terminal\ndomain of B96Bom is translocated across the mem-\nbrane, the susceptibility of two N-glycosylation motifs\n[Asn-Tyr-Thr(11–13), Asn-Tyr-Thr(16–18)]\nlocated at\nthe N-terminal domain of B96Bom to protein N-glyco-\nsylation was evaluated by in vivo metabolic labeling in\ntransfected COS-1 cells. When pro-GLC-TNF, a\nmodel N-glycosylated type II transmembrane protein\nwith an N-glycosylation site in the mature domain of\nTNF, was expressed in COS-1 cells, a protein band\nwith a molecular mass of 30 kDa was generated, as\nshown in Fig. 4B (lane 1). Glycopeptidase F treatment\nof this protein gave rise to a protein band with an\n 4 kDa smaller molecular mass (26 kDa), indicating\nthe pro-GLC-TNF is N-glycosylated when\nthat\nexpressed in COS-1 cells (lane 2). When B96Bom58-\nTNF cDNA was\ntransfected into COS-1 cells, a\n28 kDa protein was expressed (lane 3); however, the\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nA\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nB\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nC\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nFig. 4. The N-terminal domain of B96Bom is translocated across the\nmembrane. (A) cDNAs encoding B96Bom58-TNF and B96Bom90-\nTNF were transfected into COS-1 cells, and their expression and N-\nmyristoylation of the products were evaluated by Western blotting\nanalysis and [3H]myristic acid labeling, respectively. N-Glycosylation\nof the expressed protein was determined by the change in its\nmolecular mass by treatment of the immunoprecipitated protein with\nglycopeptidase F (GPF). (B) Western blotting; (C) [3H]myristic acid\nincorporation.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nmolecular size of this protein was not affected by treat-\nment with glycopeptidase F (lane 4). In contrast,\nexpression of B96Bom90-TNF cDNA gave rise to two\nproteins: one was the major product with a molecular\nmass of 33 kDa, and the other was a fainter band of\n27 kDa. Glycopeptidase F treatment of these two pro-\nteins gave rise to three protein bands: a major 27 kDa\nprotein, a minor 30 kDa protein and a trace amout of\n33 kDa protein. These results indicate that both of the\ntwo N-glycosylation motifs located at the N-terminal\ndomain of B96Bom90-TNF directed protein N-glyco-\nsylation, and diglycosylated protein with a molecular\nmass of 33 kDa was generated. [3H]Myristic acid labe-\nling revealed that both the diglycosylated 33 kDa pro-\nprotein were\ntein\n27 kDa\nefﬁciently N-myristoylated (Fig. 4C,\nlane 3). These\nresults clearly indicate that the TM1 of B96Bom func-\ntioned as a type I signal anchor sequence, and the\nN-terminal domain (58 residues) was\ntranslocated\nacross the membrane.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nnonglycosylated\n**BLOCK**fs== 9.5**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nand\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nThe N-myristoylated N-terminus of B96Bom\nis translocated across the membrane\n**BLOCK**fs== 9.5**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\nIn the membrane proteins with extracellular N-termini\nthat lack a cleaved signal sequence, the TM1 functions\nas a type I signal anchor sequence, and the portion\nbefore TM1 translocates the membrane post-transla-\ntionally [16,17]. It is clear from the results obtained in\nsection that Asn11 and Asn16 of\nthe previous\nB96Bom is translocated across the membrane, as these\ntwo Asn residues are N-glycosylated. However, whe-\nther or not\nthe N-myristoylated N-terminus of\nB96Bom is translocated across the membrane is not\nclear from these experimental results. To clarify this,\nwe introduced an additional N-glycosylation site at a\nposition close to the N-terminus and evaluated its\nthis\nsusceptibility to protein N-glycosylation. For\npurpose, Ala4 in B96Bom90-TNF was replaced with\nAsn to generate a new N-glycosylation site, Asn-\nAla-Thr(4–6), and the N-glycosylation of the mutant\nobtained B96Bom90A4N-TNF expressed in COS-1\ncells was evaluated by glycopeptidase F treatment.\nWhen the B96Bom90A4N-TNF was expressed in\nCOS-1 cells, two protein bands were obtained: one\nwas a major protein band with an apparent molecular\nmass of 35 kDa, and the other was a fainter protein\nband with 27 kDa molecular mass, as\nshown in\nFig. 5B (lane 3). The molecular size of the major pro-\ntein band was slightly larger than that of diglycosylat-\nindicating that\ned B96Bom90-TNF (lanes 1 and 3),\nthe expressed B96Bom90A4N-TNF was triglycosylat-\ned. As shown in Fig. 5B lane 2, mild treatment of the\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nA\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nB\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nC\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nD\n**BLOCK**fs== 7.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nFig. 5. The N-myristoylated N-terminus of B96Bom is translocated\nacross the membrane.\n(A) cDNAs encoding B96Bom90-TNF and\nB96Bom90A4N-TNF were transfected into COS-1 cells, and their\nexpression and N-myristoylation of the products were evaluated by\nWestern blotting analysis and [3H]myristic acid labeling, respect-\nively. N-Glycosylation of the expressed protein was determined by\nthe change in its molecular mass by treatment of the immunopre-\ncipitated protein with a low (B) or high (C,D) concentration of gly-\ncopeptidase F (GPF). (B,C) Western blotting; (D) [3H]myristic acid\nincorporation.\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nthat the N-myristoylated N-terminus of B96Bom90-\nTNF is\nthe membrane and\nexposed to the extracellular space.\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\ntranslocated across\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nN-Myristoylation of the N-terminus of B96Bom is\nnot involved in the membrane translocation of\nthe N-terminal domain of B96Bom\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nthe\nTo determine whether the N-myristoylation of\nN-terminus of B96Bom plays a role in the membrane\ntranslocation of the N-terminal domain of B96Bom,\nthe Gly residue at position 2 of B96Bom90-TNF was\nreplaced with Ala to generate a nonmyristoylated\nmutant, B96Bom90G2A-TNF, and its membrane\ntranslocation was evaluated by assessing the suscepti-\nbility of this protein to protein N-glycosylation. As\nin vitro translation of\nshown in Fig. 6B lane 1,\nB96Bom90-TNF cDNA gave rise to two protein\nbands: one was the major product with an apparent\nmolecular mass of 27 kDa, and the other was a fainter\nband with a molecular mass of 20 kDa. In contrast,\nin vitro translation of B96Bom90G2A-TNF cDNA\ngave rise to three protein bands: a major protein band\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nA\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nB\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nC\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nimmunoprecipitated B96Bom90-TNF with glycopepti-\ndase F gave rise to one protein band with an interme-\ndiate size between those of the two original proteins.\nIn contrast, the same treatment of the immunoprecipi-\ntated B96Bom90A4N-TNF gave rise to two protein\nbands between the two original protein bands (lane 4),\nindicating that the major product of B96Bom90A4N-\nTNF expressed in COS-1 cells\ntriglycosylated.\n[3H]Myristic acid labeling revealed that the triglycosyl-\nated B96Bom90-TNF is N-myristoylated, and the\nN-myristoylated triglycosylated protein band was shif-\nted to a nonmyristoylated 27 kDa protein band by\ntreatment with glycopeptidase F (Fig. 5D, lanes 3 and\n4). These results indicate that Asn4, which is close to\nin B96Bom90A4N-TNF\nthe N-myristoylated Gly2,\nis quite probable\nwas N-glycosylated. Therefore,\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nis\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nit\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n(A) mRNAs\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nencoding B96Bom90-TNF\n**BLOCK**fs== 7.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nthe N-terminus of B96Bom is not\nFig. 6. N-Myristoylation of\ninvolved in the membrane translocation of the N-terminal domain\nof B96Bom.\nand\nB96Bom90G2A-TNF were translated in vitro in the presence of\n[3H]leucine or [3H]myristic acid using rabbit reticulocyte lysate. After\nimmunoprecipitation with antibody to TNF, the labeled translation\nproducts were analyzed by SDS ⁄ PAGE and ﬂuorography (B).\ncDNAs encoding B96Bom90-TNF and B96Bom90G2A-TNF were\ntransfected into COS-1 cells, and their expression was evaluated\nby Western blotting analysis. N-Glycosylation of the expressed pro-\ntein was determined by the change in its molecular mass by treat-\nment of\nthe immunoprecipitated protein with glycopeptidase F\n(GPF) (C).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nwith an apparent molecular mass of 29 kDa and two\nfainter bands with molecular masses of 31 kDa and\n[3H]Myristic acid labeling revealed\n20 kDa (lane 2).\nthat none of these three protein bands was N-myristo-\nlane 4). When B96Bom90G2A-TNF\nylated (Fig. 6B,\nwas expressed in COS-1 cells, three protein bands with\nmolecular masses of 35, 32, and 29 kDa were detected\n(Fig. 6C, lane 3). Glycopeptidase F treatment of these\nproteins gave rise to a single protein band with a\nmolecular mass of 29 kDa (lane 4),\nindicating that\nthe two protein bands with higher molecular mass\nwere N-glycosylated. These results clearly indicated\nthe N-terminal domain of nonmyristoylated\nthat\nB96Bom90G2A-TNF was\nthe\nmembrane as efﬁciently as that of N-myristoylated\nB96Bom90-TNF. These results reveal that N-myristoy-\nlation of the N-terminus of B96Bom does not play a\nrole in the membrane translocation of the N-terminal\ndomain of B96Bom.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\ntranslocated across\n**BLOCK**fs== 10.7**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nDiscussion\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\ntranslocation of\n**BLOCK**fs== 9.5**b== 0.1**t== 0.5**l== 0.1**r== 0.5**\nIn eukaryotic cellular proteins, protein N-myristoyla-\ntion occurs mainly on cytoplasmic or nucleoplasmic\nproteins. In viral proteins, however, protein N-myris-\ntoylation has been found to occur on transmembrane\nproteins. It has been found that the large L envelope\nprotein of HBV, a polytopic membrane protein,\nis\nN-myristoylated, and this modiﬁcation is involved in\nthe intracellular retention of this protein and thus is\nessential for viral\ninfectivity [6,7,18,19]. This protein\nhas the peculiar capacity to form two transmembrane\ntopologies via an as yet uncharacterized process of\nits pre-S\npartial post-translational\ndomain across membranes [20–22]. The HBV L protein\nwith its N-myristoylated N-terminal pre-S domain\nplays vital roles in the viral\nlife cycle by mediating\nreceptor binding during host cell attachment, by per-\nforming a matrix-like function in nucleocapsid envel-\nopment and by exerting various regulatory effects. The\nmultifunctional nature of HBV L protein depends on\nits dual transmembrane topology, which is established\nafter the completion of polypeptide synthesis [23,24].\nThus, protein N-myristoylation occurs on this viral\ntransmembrane protein and seems to be involved in\nthe speciﬁc functions of the protein. However, so far,\nprotein N-myristoylation has not been observed on\neukaryotic transmembrane proteins. In this study, to\nsearch for a eukaryotic N-myristoylated transmem-\nbrane protein, the susceptibility of the N-terminus of\nGPCR, a polytopic transmembrane protein with vital\nroles in intracellular signal-transduction cascades, to\nprotein N-myristoylation was evaluated.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n[1,26,27]. These\n**BLOCK**fs== 9.5**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\nTo search for N-myristoylated GPCRs, possible can-\ndidates with an N-myristoylation consensus motif at\ntheir N-terminus were ﬁrst extracted from the GPCR\ndatabase. Protein N-myristoylation is catalyzed by\nN-myristoyltransferase, a member of the GCN5 acetyl-\ntransferase superfamily of proteins [25]. The precise\nsubstrate speciﬁcity of this enzyme has been character-\nized using puriﬁed enzyme and synthetic peptides\nsequences of known\nderived from the N-terminal\nN-myristoylated proteins\nstudies\nrevealed that,\nin addition to Gly at position 2, the\namino acids at positions 3, 6, and 7 play an important\nrole in substrate recognition by N-myristoyltransferase.\nRecently, we studied the amino-acid requirements at\npositions 3, 6 and 7 in the N-myristoylation motif by\nvertical-scanning mutagenesis of amino acids in a\nthe\nmodel N-myristoylation motif. We found that\ncombination of amino acids at positions 3, 6 and 7 is\na major determinant of protein N-myristoylation\n[28,29]. For example, when Ser was located at position\n6, as many as 11 amino acids (Gly, Ala, Ser, Cys, Thr,\nVal, Asn, Leu, Ile, Gln, His) were permitted at posi-\ntion 3 to allow efﬁcient protein N-myristoylation. In\ncontrast, when other amino acids such as Thr, Ala or\nPhe were located at position 6, only three amino acids\n(Ala, Asn, Gln) were permitted at position 3 to allow\nefﬁcient protein N-myristoylation. To search for\nN-myristloylated GPCRs, these rules were applied to\nthe N-terminal sequence of GPCRs listed in the GPCR\ndatabase [13]. As a result, several biogenic amine re-\nceptors such as octopamine receptor, tyramine recep-\ntor, dopamine receptor D4, and serotonin receptor\nwere found to have N-terminal sequences that are con-\nsistent with these rules.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nIn previous studies, we have shown that the suscepti-\nbility of a protein to cotranslational N-myristoylation\ncan easily be detected by metabolic labeling in an in vitro\nor in vivo expression system using a fusion protein in\nwhich the N-terminal sequence ( 10 amino-acid resi-\ndues) of the query protein was fused to a model protein\n[28–32]. In the present study, we used this strategy to\ndetect protein N-myristoylation of GPCRs. We found\nthat the N-terminus of B96Bom, a B. mori GPCR, is\nefﬁciently N-myristoylated. We have recently cloned the\ncDNA of this protein and expressed it in HEK-293 cells.\nWe showed that B96Bom is a B. mori tyramine receptor\nwhich is negatively coupled to adenylate cyclase [14,15].\nUsing this B96Bom cDNA, we further analyzed the\nN-myristoylation of this protein.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe seven transmembrane domains characteristic of\nGPCRs give rise to several structural features. Each\nGPCR has an N-terminal extracellular domain, the\nseven transmembrane helices, three extracellular and\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nthe\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nreticulum.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nendoplasmic\n**BLOCK**fs== 9.5**b== 0.3**t== 0.2**l== 0.1**r== 0.5**\nthree intracellular loops, and an intracellular C-ter-\nminal domain. In the membrane proteins with extracel-\nlular N-termini that lack a cleaved signal sequence,\nincluding GPCRs, TM1 functions as a type I signal\nanchor sequence, and the portion before TM1 translo-\ncates the endoplasmic reticulum post-translationally\n[16,17]. Protein N-myristoylation is a very early\ncotranslational protein modiﬁcation, which may occur\nas early as when the ﬁrst 50 amino acids of the nascent\npolypeptide chain are polymerized [33]. Therefore,\nit\nseems likely that the N-myristoylation of B96Bom pre-\ncedes the translocation of the N-terminus of this pro-\nIn many\ntein across\nN-myristoylated proteins, the N-terminal myristic acid\nmoiety functions as a membrane anchor and immobil-\nizes the protein on the membrane. Therefore, it was\npossible that the membrane binding mediated by the\nN-myristoylated N-terminus of B96Bom inihibits the\nmembrane translocation of the N-terminal domain of\nthis protein. The experimental results obtained with\nB96Bom90A4N-TNF clearly show that this is not the\ncase. The oligosaccharyl transferase is required for the\ntransfer of oligosaccharides from a dolichol carrier to\nAsn-X-Ser ⁄ Thr acceptor sites for N-glycosylation in\nnascent polypeptides. N-linked glycosylation occurs on\nthe lumenal side of the endoplasmic reticulum during\nor soon after translocation of nascent polypetides [34].\nAs the active site of the oligosaccharyl transferase is\nlocated  15 residues away from the endoplasmic reti-\nculum membrane [35], the fact that Asn4 close to the\nN-myristoylated Gly2 in this mutant was N-glycosylat-\ned strongly indicated that the N-myristoylated N-ter-\nminus of B96Bom90-TNF is translocated across the\nendoplasmic reticulum membrane through transloca-\ntion machinery and exposed to the lumenal side of the\nendoplasmic reticulum. It\nis not clear whether the\nN-myristoylated N-terminus of B96Bom anchors to\nthe membrane or not. Therefore, there are two poss-\nible models for the transmembrane topology of the\nN-terminal domain of B96Bom, as shown in Fig. 7.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nThe physiological roles of protein N-myristoylation\nof B96Bom are not clear at present. The fact that the\nN-terminus of nonmyristoylated B96BomG2A-TNF\nwas translocated across the membrane as efﬁciently as\nthat of N-myristoylated B96Bom90-TNF indicated\nthat N-myristoylation of the N-terminus of B96Bom\ndoes not play a role in the membrane translocation of\nthe N-terminal domain of B96Bom. It has been found\nthat post-translational modiﬁcations such as N-glyco-\nsylation, phosphorylation, and palmitoylation play\nimportant roles in the functional regulation of GPCRs\n[8,36,37]. Of these three modiﬁcations, only N-glycosy-\nlation occurs on the N-terminal extracellular domain,\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nFig. 7. Two possible models for the transmembrane topology of\nthe N-terminal domain of B96Bom.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nas does protein N-myristoylation. N-Glycosylation of\nGPCRs on one or more Asn residues in the N-terminal\ndomain seems to be involved in ligand binding and cell\nsurface expression of the receptor [37]. Therefore, it is\npossible that N-myristoylation of B96Bom is also\nIn addition to the\ninvolved in these phenomena.\nB96Bom receptor, several other biogenic amine recep-\ntors\nsuch as octopamine receptors, dopamine D4\nreceptors, and a serotonin receptor were found to have\nthe N-myristoylation consensus motif at their N-ter-\nmini (Fig. 1). However, none of these receptors has\nbeen shown to be N-myristoylated so far. To our\nknowledge, no obvious similarity has been reported in\nthe function of these receptors except that all these are\nstudies will be\nbiogenic amine receptors. Further\nrequired to elucidate the roles of protein N-myristoyla-\ntion in the functions of the B96Bom receptor or other\nbiogenic amine receptors.\n**BLOCK**fs== 10.7**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nExperimental procedures\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nMaterials\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nRestriction\nenzymes,\nendonucleases, DNA-modifying\nRNase inhibitor, and Taq DNA polymerase were purchased\nfrom Takara Shuzo, Kyoto Japan. The mCAP RNA cap-\nping kit and proteinase K were from Stratagene (La Jolla,\nCA, USA). RNase was purchased from Boehringer-Mann-\nheim (Mannheim, Germany). Rabbit reticulocyte lysate was\nfrom Promega (Madison, WI, USA). [3H]Leucine, [3H]my-\nristic acid and Amplify were from Amersham, Piscataway,\nNJ, USA. The Dye Terminator Cycle Sequencing kit was\nfrom Applied Biosystems (Foster City, CA, USA). Polyclonal\nantibody to human TNF was purchased from R & D Sys-\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\ntems (Minneapolis, MN, USA). Protein G–Sepharose was\nfrom Pharmacia Biotech (Piscataway, NJ, USA). Other rea-\ngents purchased from Wako Pure Chemical (Osaka, Japan),\nDaiichi Pure Chemicals (Tokyo, Japan), and Seikagaku\nKogyo (Tokyo, Japan) were of analytical or DNA grade.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nPlasmid construction\n**BLOCK**fs== 8.7**b== 0.3**t== 0.3**l== 0.1**r== 0.5**\nPlasmid pBluescript II SK(+) lacking ApaI and HindIII\nsites was constructed as previously described [38], and des-\nignated pB. Plasmid pBDpro-TNF, which contains a cDNA\ncoding for the mature domain of TNF, was constructed as\ndescribed [32]. Plasmid pBB96Bom10-TNF was constructed\nusing PCR. pBDpro-TNF served as the template and two\noligonucleotides (Bom10, B1) served as primers (Table 1).\nAfter digestion with BamHI and PstI, the ampliﬁed product\nwas subcloned into pB at the BamHI and PstI sites. Plas-\nmids pBDRD4-TNF and ADRb1-TNF were constructed\nby a method similar to that used for pBB96Bom10-TNF\nusing two oligonucleotide pairs\n(DRD4 plus B1 and\nADRB1 plus B1, respectively) as primers (Table 1). Plasmid\npBGi1a-TNF was constructed as previously described [32].\nThe cDNAs coding for B96Bom58-TNF and B96Bom90-\nin which the N-terminal 58 and 90 residues of\nTNF,\nB96Bom were linked to the N-terminus of\nthe mature\ndomain of TNF, were constructed by using PCR. For this\nprocedure, previously cloned B96Bom cDNA [14] sub-\ncloned in pcDNA3 served as the template, and two oligo-\nnucleotide pairs (BomN plus Bom58 and BomN plus\nBom90, respectively) served as primers (Table 1). After\ndigestion with BamHI and XhoI, the ampliﬁed products\nwere subcloned into pBGi1a-TNF at the BamHI and XhoI\nsites. Plasmid pBB96Bom90A4N-TNF was constructed by\na method similar to that used for pBB96Bom90-TNF using\ntwo oligonucleotides\nas primers\n(Table 1). Plasmid pBB96Bom90G2A-TNF was constructed\nby a method similar to that used for pBB96Bom90-TNF\nusing two oligonucleotides (BomG2A, Bom90) as primers\n(Table 1). Plasmid pBpro-GLC-TNF was constructed by\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\n(BomA4N, Bom90)\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nusing PCR. In this mutant, an N-glycosylation site (Asn-X-\nSer) was introduced at amino acid positions 45–47 in the\nmature domain of TNF. For this procedure, pBpro-TNF\nserved as the template and two sets of oligonucleotides (T3\nplus GLC-6 and MUTB1 plus B1) as primers. After diges-\ntion with SacI and KpnI, the ampliﬁed product was sub-\ncloned into pB at the SacI and KpnI sites. The DNA\nsequences of these recombinant cDNAs were conﬁrmed by\nthe dideoxynucleotide chain termination method [39]. The\ncDNAs coding for TNF mutants were subcloned into\npcDNA3, and the pcDNA3 constructs obtained were used\nfor transfection assays.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.5**r== 0.2**\nIn vitro transcription and translation\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nMethods essentially the same as those described previously\nwere used [29]. T3 polymerase was used to obtain transcripts\nof these cDNAs subcloned into pB vector. These transcripts\nwere puriﬁed by phenol ⁄ chloroform extraction and ethanol\nprecipitation before use. Subsequently,\ntranslation was\ncarried out using rabbit reticulocyte lysate (Promega) in the\npresence of [3H]leucine or [3H]myristic acid under conditions\nrecommended by the manufacturer. The mixture [17.5 lL\nrabbit reticulocyte lysate; 0.5 lL 1 mm leucine-free amino-\nacid mixture or 1 mm complete amino acid mixture; 4.0 lL\n[3H]leucine (5 lCi) or [3H]myristic acid (25 lCi); 3.0 lL\nmRNA] was incubated at 30 C for 90 min.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nTransfection of COS-1 cells and determination\nof N-myristoylated and N-glycosylated proteins\n**BLOCK**fs== 8.7**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThe simian virus 40-transformed African Green monkey\nkidney cell\nline, COS-1, was maintained in Dulbecco’s\nmodiﬁed Eagle’s medium (Invitrogen, Carlsbad, CA, USA)\nsupplemented with 10% fetal bovine serum (Invitrogen).\nCells (2 · 105) were plated on to 35-mm diameter dishes\n1 day before transfection. pcDNA3 construct (2 lg; Invitro-\ngen) containing mutant TNF cDNA was used to transfect\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nTable 1. Nucleotide sequences of oligonucleotides used for the construction of mutant TNF cDNAs.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nPrimer\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.9**\nBom10\nDRD4\nADRB1\nBomN\nBom58\nBom90\nBomG2A\nBomA4N\nGLC-6\nMUTB1\nT3\nB1\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nSequence (5¢ to 3¢)\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.4**r== 0.1**\nGCGCGGATCCATGGGTCAAGCAGCTACTCACGATGCAAATGACAAGCCTGTAGCC\nGCGCGGATCCATGGGGAACCGCAGCACCGCGGACGCGGACGACAAGCCTGTAGCC\nGCGCGGATCCATGGGCGCGGGGGTGCTCGTCCTGGGCGCCGACAAGCCTGTAGCC\nGCGCGGATCCATGGGTCAAGCAGCT\nGCGCCTCGAGAGCTGTACAAATAGC\nGCGCCTCGAGGTTCTGAACAATTCT\nGCGCGGATCCATGGCTCAAGCAGCTACT\nGCGCGGATCCATGGGTCAAAACGCTACTCACGATGCA\nTGGCACCACCAGAGAGTTGTTAGACAGCTCCACGCC\nAGCTGGATCTCCACCACGGTGTCGG\nAATTAACCCTCACTAAAGGG\nGCCGGGATCCTAGGGCGAATTGGGTACC\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\nbovine\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n[3H]myristic\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.2**r== 0.6**\nserum containing\n**BLOCK**fs== 8.7**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\neach plate of COS-1 cells along with 3 lL LipofectAmine\n(2 mgÆmL)1; Invitrogen) in 1 mL serum-free medium. After\nincubation for 5 h at 37 C,\nthe cells were refed with\nserum-containing medium and incubated again at 37 C for\n24 h. The cells were then washed twice with 1 mL serum-\nfree Dulbecco’s modiﬁed Eagle’s medium and incubated for\n5 h in 1 mL Dulbecco’s modiﬁed Eagle’s medium with 2%\nacid\nfetal\n(100 lCiÆmL)1). Subsequently, the cells were washed three\ntimes with Dulbecco’s phosphate buffered saline, collected\nwith cell scrapers, and lysed with 200 lL RIPA buffer\n[50 mm Tris ⁄ HCl (pH 7.5), 150 mm NaCl, 1% Nonidet\nP40, 0.5% sodium deoxycholate, 0.1% SDS, proteinase\ninhibitors] on ice for 20 min. The cell lysates were centri-\nfuged at 21 000 g at 4 C for 15 min in a microcentrifuge\n(Hitachi-CF15D2), and the supernatants were collected.\nAfter immunoprecipitation with antibody to TNF, the sam-\nples were analyzed by SDS ⁄ PAGE and ﬂuorography. To\nexamine the N-glycosylation of TNF mutants, 20 lL total\ncell lysate of each group of transfected cells or TNF sample\nimmunoprecipitated from total cell lysates were treated with\n10 or 50 mUÆmL)1 glycopeptidase F at 37 C for 1 h and\nthen analyzed by Western blotting.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nWestern blotting\n**BLOCK**fs== 8.7**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nTNF samples immunoprecipitated from in vitro translation\nproducts or total cell lysates of each group of transfected\ncells were resolved by SDS ⁄ PAGE (12.5% gel) and then\ntransferred to an Immobilon-P transfer membrane (Milli-\npore, Billerica, MA, USA). After being blocked with nonfat\nmilk, the membrane was probed with a speciﬁc goat anti-\nbody to human TNF as described previously [40]. Immuno-\nreactive proteins were speciﬁcally detected by incubation\nwith horseradish peroxidase-conjugated anti-goat\nIgG\n(Santa Cruz, Biotech, Santa Cruz, CA, USA). The mem-\nbrane was developed with ECL Western blotting reagent\n(Amersham) and exposed to an X-ray ﬁlm (Kodak). Quanti-\ntative analysis of immunoreactive proteins on the membrane\nwas carried out using a storage phosphor imaging screen and\nGS-250 Molecular Imager (Bio-Rad, Hercules, CA, USA).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nImmunoprecipitation\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nSamples containing TNF mutants were immunoprecipitated\nwith a speciﬁc goat polyclonal antibody to human TNF (R\n& D Systems) as described [38].\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nSDS ⁄ PAGE and ﬂuorography\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSamples were denatured by boiling for 3 min in SDS sam-\nple buffer followed by analysis by SDS ⁄ PAGE on a 12.5%\ngel. Thereafter, the gel was ﬁxed and soaked in AmplifyTM\n(Amersham) for 30 min. The gel was dried under vacuum\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nand exposed to an X-ray ﬁlm (Kodak) for an appropriate\nperiod. Quantitative analysis of the labeled proteins was\ncarried out by scanning the ﬂuorogram using an imaging\ndensitometer (Bio-Rad GS-700).\n**BLOCK**fs== 10.7**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nAcknowledgements\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nPart of this work was supported by a Grant-in-Aid for\nScientiﬁc Research (No. 12660080, No. 15580080)\nfrom the Ministry of Education, Science, and Culture\nof Japan.\n**BLOCK**fs== 10.7**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nReferences\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n1 Towler DA, Gordon JI, Adams SP & Glaser L (1988)\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\nThe biology and enzymology of eukaryotic protein acy-\nlation. Annu Rev Biochem 57, 69–99.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n2 Spiegel AM, Backlund PS, Butrynski JE, Jones TLZ &\nSimonds WF (1991) The G protein connection: molecu-\nlar basis of membrane association. Trends Biochem\nSci 16, 338–341.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n3 Boutin JA (1997) Myristoylation. Cell Signal 9, 15–35.\n4 Resh MD (1999) Fatty acylation of proteins: new\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\ninsights into membrane targeting of myristoylated and\npalmitoylated proteins. Biochim Biophys Acta 1451,\n1–16.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n5 Farazi TA, Waksman G & Gordon JI (2001) The biol-\nogy and enzymology of protein N-myristoylation. J Biol\nChem 276, 39501–39504.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n6 Persing D, Varmus H & Ganem D (1987) The preS1\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\nprotein of hepatitis B virus is acylated at its amino ter-\nminus with myristic acid. J Virol 61, 1672–1677.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n7 Prange R, Clemen A & Streeck RE (1991) Myristylation\nis involved in intracellular retention of hepatitis B virus\nenvelope proteins. J Virol 65, 3919–3923.\n8 Qanbar R & Bouvier M (2003) Role of\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\npalmitoylation ⁄ depalmitoylation reactions in\nG-protein-coupled receptor function. Pharmacol Ther\n97, 1–33.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n9 Gether U (2000) Uncovering molecular mechanisms\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\ninvolved in activation of G protein-coupled receptors.\nEndocr Rev 21, 90–113.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n10 Pierce L, Premont RT & Lefkowitz RJ (2002) Seven-\ntransmembrane receptors. Nat Rev Mol Cell Biol 3,\n639–650.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n11 Bockaert J & Pin JP (1999) Molecular tinkering of G\nprotein-coupled receptors: an evolutionary success.\nEMBO J 18, 1723–1729.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n12 Bockaert J, Claeysen S, Becamel C, Pinloche S &\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.6**r== 0.1**\nDumuis A (2002) G protein-coupled receptors: domi-\nnant players in cell-cell communication. Int Rev Cytol\n212, 63–132.\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n13 Horn F, Vriend G & Cohen FE (2001) Collecting\nand harvesting biological data: the GPCRDB and\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nNucleaRDB information systems. Nucleic Acids Res 29,\n346–349.\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n14 Ohta H, Utsumi T & Ozoe Y (2003) B96Bom encodes a\nBombyx mori tyramine receptor negatively coupled to\nadenylate cyclase. Insect Mol Biol 12, 217–223.\n15 Ohta H, Utsumi T & Ozoe Y (2004) Amino acid\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n28 Utsumi T, Nakano K, Funakoshi T, Kayano Y, Nakao\nS, Sakurai N, Iwata H & Ishisaka R (2004) Vertical-\nscanning mutagenesis of amino acids in a model N-myr-\nistoylation motif reveals the major amino-terminal\nsequence requirements for protein N-myristoylation. Eur\nJ Biochem 271, 863–874.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\nresidues involved in interaction with tyramine in the\nBombyx mori tyramine receptor. Insect Mol Biol 13,\n531–538.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\n16 Friedlander M & Blobel G (1985) Bovine opsin has\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nmore than one signal sequence. Nature 318, 338–343.\n17 Kanner EM, Klein IK, Friedlander M & Simon SM\n**BLOCK**fs== 8.7**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n(2002) The amino terminus of opsin translocates ‘post-\ntranslationally’ as efﬁciently as cotranslationally.\nBiochemistry 41, 7707–7715.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n18 Gripon P, Le Seyec J, Rumin S & Guguen-Guillouzo C\n(1995) Myristylation of the hepatitis B virus large sur-\nface protein is essential for viral infectivity. Virology\n213, 292–299.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n19 Bruss V, Hagelstein J, Gerhardt E & Galle PR (1996)\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nMyristylation of the large surface protein is required for\nhepatitis B virus in vitro infectivity. Virology 218,\n396–399.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\n20 Bruss V, Lu X, Thomssen R & Gerlich WH (1994)\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nPost-translational alterations in transmembrane topol-\nogy of the hepatitis B virus large envelope protein.\nEMBO J 13, 2273–2279.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n21 Ostapchuk P, Hearing P & Ganem D (1994) A dramatic\nshift in the transmembrane topology of a viral envelope\nglycoprotein accompanies hepatitis B viral morphogen-\nesis. EMBO J 13, 1048–1057.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n22 Lambert C & Prange R (2001) Dual topology of the\nhepatitis B virus large envelope protein: determinants\ninﬂuencing post-translational pre-S translocation. J Biol\nChem 276, 22265–22272.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\n23 Bruss V & Vieluf K (1995) Infection process of the\n**BLOCK**fs== 8.7**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nhepatitis B virus depends on the presence of a deﬁned\nsequence in the pre-S1 domain. J Virol 69, 6652–6657.\n24 Le Seyec J, Chouteau P, Cannie I, Guguen-Guillouzo C\n& Gripon P (1999) Infection process of the hepatitis B\nvirus depends on the presence of a deﬁned sequence in\nthe pre-S1 domain. J Virol 73, 2052–2057.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nrelated N-acetyltransferases: a structural overview. Annu\nRev Biophys Biomol Struct 29, 81–103.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n26 Towler DA, Adams SP, Eubanks SR, Towery DS, Jack-\nson-Machelski E, Glaser L & Gordon JI (1988) Myris-\ntoyl CoA: protein N-myristoyltransferase activities from\nrat liver and yeast possess overlapping yet distinct pep-\ntide substrate speciﬁcities. J Biol Chem 263, 1784–1790.\n27 Rocque WJ, McWherter CA, Wood DC & Gordon JI\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n(1993) A comparative analysis of the kinetic mechanism\nand peptide substrate speciﬁcity of human and Saccharo-\nmyces cerevisiae myristoyl-CoA:protein N-myristoyl-\ntransferase. J Biol Chem 268, 9964–9971.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\n29 Utsumi T, Sato M, Nakano K, Takemura D, Iwata H\n& Ishisaka R (2001) Amino acid residue penultimate to\nthe amino-terminal Gly residue strongly affects two\ncotranslational protein modiﬁcations, N-myristoylation\nand N-acetylation. J Biol Chem 276, 10505–10513.\n30 Utsumi T, Tou E, Takemura D, Ishisaka R, Yabuki M\n& Iwata H (1998) Met-Gly-Cys motif from G-protein\nalpha subunit cannot direct palmitoylation when fused\nto heterologous protein. Arch Biochem Biophys 349,\n216–224.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n31 Utsumi T, Sakurai N, Nakano K & Ishisaka R (2003)\nC-terminal 15 kDa fragment of cytoskeletal actin is\nposttranslationally N-myristoylated upon caspase-\nmediated cleavage and targeted to mitochondria. FEBS\nLett 539, 37–44.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n32 Utsumi T, Kuranami J, Tou E, Ide A, Akimaru K,\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nHung MC & Klostergaard J (1996) In vitro synthesis of\nan N-myristoylated fusion protein that binds to the lipo-\nsomal surface. Arch Biochem Biophys 326, 179–184.\n33 Deichaite I, Casson LP, Ling HP & Resh MD (1988) In\nvitro synthesis of pp60v-src: myristoylation in a cell-free\nsystem. Mol Cell Biol 8, 4295–4301.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n34 Kaplan HA, Welply JK & Lennarz WJ (1987) Oligosac-\ncharyl transferase: the central enzyme in the pathway\nof glycoprotein assembly. Biochim Biophys Acta 906,\n161–173.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n35 Nilsson I & von Heijne G (1993) Determination of the\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\ndistance between the oligosaccharyltransferase active site\nand the endoplasmic reticulum membrane. J Biol Chem\n268, 5798–5801.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n36 Bohm SK, Grady EF & Bunnett NW (1997) Regulatory\nmechanisms that modulate signalling by G-protein-\ncoupled receptors. Biochem J 322, 1–18.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n37 Kaushal S, Ridge KD & Khorana HG (1994) Structure\nand function in rhodopsin: the role of asparagine-linked\nglycosylation. Proc Natl Acad Sci USA 91, 4024–4028.\n38 Utsumi T, Akimaru K, Kawabata Z, Levitan A, Toku-\nnaga T, Tang P, Ide A, Hung MC & Klostergaard J\n(1995) Human pro-tumor necrosis factor: Molecular\ndeterminants of membrane translocation, sorting and\nmaturation. Mol Cell Biol 15, 6389–6405.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\n39 Sanger F, Nicklen S & Coulson AR (1977) DNA\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nsequencing with chain-terminating inhibitors. Proc Natl\nAcad Sci USA 74, 5463–5467.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\n40 Utsumi T, Levitan A, Hung MC & Klostergaad J\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n(1993) Effects of truncation of human pro-tumor necro-\nsis factor transmembrane domain on cellular targeting.\nJ Biol Chem 268, 9511–9516.",
         "extracted",
         "",
         "The N‐terminus of B96Bom, a Bombyx mori G‐protein‐coupled receptor, is N‐myristoylated and translocated across the membrane",
         "None"
        ],
        [
         "16",
         "00147c55961b99b39b3c6ede1d0e231929c4bf10",
         "Tissue remodelling is dependent on the integration of signals that control turnover of ECM (extracellular matrix). Breakdown and endocytosis of collagen, a major component of the ECM, is central to this process. Whereas controlled secretion of matrix-degrading enzymes (such as matrix metalloproteinases) has long been known to mediate ECM breakdown, it is becoming clear that uPARAP/Endo180 (where uPARAP stands for urokinase plasminogen activator receptor-associated protein) serves as a receptor that mediates endocytosis of collagen by several types of cells. In the liver, the stellate cells play a major role in turnover of ECM including collagens. These cells synthesize various collagens and also produce matrix metalloproteinases. In the present study, we investigated the capacity of rat hepatic stellate cells to endocytose and degrade 125I-labelled heat-denatured collagen I. It was found that the collagen is efficiently taken up and degraded by these cells. Degradation was inhibited by inhibitors of lysosomal proteases (leupeptin and E-64d) and the vacuolar proton pump (concanamycin A), indicating that it takes place in lysosomes. Furthermore, endocytosed FITC-labelled collagen was shown to reach late endocytic compartments in which it colocalized with LysoTracker (a marker of late endocytic compartments). Competition experiments showed that uPA and unlabelled collagen are capable of inhibiting binding and uptake of [125I]collagen in a dose-dependent manner. Moreover, Western-blot analysis of cell lysate (using a polyclonal rabbit human-Endo180 antiserum) revealed a single band at 180 kDa. In addition, the antiserum was capable of reducing [125I]collagen binding to the cell surface. Finally, using two primers designed from the human uPARAP/Endo180 mRNA sequence, the expression of uPARAP/Endo180 mRNA was detected by reverse transcriptase-PCR. These results together suggest that uPARAP/Endo180 mediates endocytosis of collagen in rat liver stellate cells.",
         "S. Mousavi,Mitsuru Sato,Marita Sporstøl,B. Smedsrød,T. Berg,N. Kojima,H. Senoo",
         "https://europepmc.org/articles/pmc1134930?pdf=render",
         "\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.1**r== 0.1**\nUptake of denatured collagen into hepatic stellate cells: evidence for\nthe involvement of urokinase plasminogen activator receptor-associated\nprotein/Endo180\nSeyed Ali MOUSAVI*, Mitsuru SATO†, Marita SPORSTØL*, Baard SMEDSRØD‡, Trond BERG*1, Naosuke KOJIMA†\nand Haruki SENOO†\n*Department of Molecular Biosciences, University of Oslo, Blindernveien 31, 0316 Oslo, Norway, †Department of Anatomy, Akita University School of Medicine, 1-1-1 Hondo,\nAkita 010-8543, Japan, and ‡Department of Experimental Pathology, University of Tromsø, Norway\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nTissue remodelling is dependent on the integration of signals\nthat control turnover of ECM (extracellular matrix). Breakdown\nand endocytosis of collagen, a major component of the ECM, is\ncentral to this process. Whereas controlled secretion of matrix-\ndegrading enzymes (such as matrix metalloproteinases) has long\nbeen known to mediate ECM breakdown, it is becoming clear\nthat uPARAP/Endo180 (where uPARAP stands for urokinase\nplasminogen activator receptor-associated protein) serves as a\nreceptor that mediates endocytosis of collagen by several types of\ncells. In the liver, the stellate cells play a major role in turnover\nof ECM including collagens. These cells synthesize various colla-\ngens and also produce matrix metalloproteinases. In the present\nstudy, we investigated the capacity of rat hepatic stellate cells\nto endocytose and degrade 125I-labelled heat-denatured collagen\nI. It was found that the collagen is efﬁciently taken up and\ndegraded by these cells. Degradation was inhibited by inhibitors of\nlysosomal proteases (leupeptin and E-64d) and the vacuolar pro-\nton pump (concanamycin A), indicating that it takes place in lyso-\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nsomes. Furthermore, endocytosed FITC-labelled collagen was\nshown to reach late endocytic compartments in which it co-\nlocalized with LysoTracker (a marker of late endocytic compart-\nments). Competition experiments showed that uPA and unlabelled\ncollagen are capable of inhibiting binding and uptake of [125I]col-\nlagen in a dose-dependent manner. Moreover, Western-blot\nanalysis of cell lysate (using a polyclonal rabbit human-Endo180\nantiserum) revealed a single band at 180 kDa. In addition,\nthe antiserum was capable of reducing [125I]collagen binding\nto the cell surface. Finally, using two primers designed from the\nhuman uPARAP/Endo180 mRNA sequence, the expression of\nuPARAP/Endo180 mRNA was detected by reverse transcriptase–\nPCR. These results together suggest\nthat uPARAP/Endo180\nmediates endocytosis of collagen in rat liver stellate cells.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nKey words: collagen receptor, denatured collagen, lysosomal de-\ngradation, receptor-mediated endocytosis, urokinase plasminogen\nactivator receptor-associated protein (uPARAP).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nINTRODUCTION\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nHepatic stellate cells are located within the perisinusoidal space\nbetween the basolateral surface of hepatocytes and the endothelial\ncells [1]. They are relatively scarce, < 10 % of the total resident\nliver cells are stellate cells [2]. A number of important functions\nhave been attributed to stellate cells. They store vitamin A,\nreceived from the neighbouring hepatocytes [3,4], and, since they\nencircle the hepatic sinusoids and have contractile properties,\nthey may regulate blood ﬂow through the sinusoids [5–7]. They se-\ncrete cytokines and express a number of receptors that play a\ncentral role in cell–matrix and cell–cell interactions, both in\nnormal and in injured liver [2,8]. The stellate cells exhibit two\nmain phenotypes, the ‘quiescent state’ and the ‘activated state’.\nThe quiescent cells have low proliferation rate and low ﬁbrogenic\nactivity, but after any injury to the liver the cells shift towards\na myoﬁbroblastic-like phenotype [2,5,8]. A similar activation\nprocess is observed when primary stellate cells are kept in culture\non plastic support [9,10]: the cells elongate, lose their vitamin A\ndroplets and develop myoﬁlaments in their cytoplasm [5,8]. The\nactivated cells gain functions such as proliferation and produce\nvarious growth factors and most of the ECM (extracellular\nmatrix) components [9–12]. Activated stellate cells also produce\nenzymes that can degrade the normal pericellular ECM before\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nsynthesis of ﬁbrous tissue. Such enzymes include MMPs (matrix\nmetalloproteinases), a family of zinc-dependent enzymes [13,14].\nAlthough activated stellate cells produce several factors needed\nfor ECM degradation, it is conceivable that they in addition may\nremove components of the ECM by means of endocytosis. Very\nfew studies have focused on this process in stellate cells. It was\nshown by using an in vitro system that retinol-binding protein,\nreleased from hepatocytes may be internalized in the stellate cells\nby receptor-mediated endocytosis [4]. A similar mechanism may\noperate when stellate cells take up retinol-binding protein from\nblood [15].\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe aim of the present study was to determine whether stellate\ncells are able to endocytose collagen, a main component of the\nECM. To this end we used cultured rat stellate cells, and heat-\ndenatured collagen I, labelled with 125I, was chosen as a repres-\nentative ligand. Initial experiments demonstrated that the stellate\ncells internalize and degrade [125I]collagen, and that the degrad-\nation is reduced by inhibitors of lysosomal function, indicating\nthat the cells are able to endocytose [125I]collagen. Moreover, the\nbinding of [125I]collagen was shown to be saturable, indicating\nthat it is receptor-mediated. Further investigations indicated that\nthe receptor in question could be the surface receptor uPARAP/\nEndo180 [where uPARAP stands for urokinase plasminogen\nactivator receptor-associated protein]. uPARAP/Endo180 was\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nAbbreviations used: CTLD, C-type lectin-like domain; ECM, extracellular matrix; [125I]TC, [125I]tyramine cellobiose; MMP, matrix metalloproteinase;\nOVA, ovalbumin; RT, reverse transcriptase; TCA, trichloracetic acid; tPA, tissue plasminogen activator; uPA, urokinase plasminogen activator; uPAR, uPA\nreceptor; uPARAP, uPAR-associated protein; V-ATPase, vacuolar H+-ATPase.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n1 To whom correspondence should be addressed (email trond.berg@bio.uio.no).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.1**l== 0.1**r== 0.5**\noriginally identiﬁed as a constitutively recycling surface receptor\n[16] and has recently been shown to function as a collagen recep-\ntor [17–20]. A large pool (70–90 %) of the total uPARAP/Endo180\nis maintained in endosomal compartments and the small plasma\nmembrane pool (10–30 %) is associated with clathrin-coated\npits [16,21]. uPARAP/Endo180 has also been called uPARAP\nbecause it can form a ternary complex with uPAR-bound pro-uPA.\nCollagen can block the formation of this trimolecular complex\n[22]. uPARAP/Endo180 is a member of the mannose receptor\nfamily, which also comprises the mannose receptor, the phospho-\nlipase A2 receptor and the DEC-205/gp200-MR6 receptor [23].\nThese proteins are large in size (175–200 kDa) and contain\nseveral distinct domains including an NH2-terminal cysteine-rich\ndomain, a ﬁbronectin-like type II domain, followed by either eight\nor ten tandemly arranged CTLDs (C-type lectin-like domains,\nnamed CTLD1–CTLD10), a single transmembrane domain and a\nshort cytoplasmic domain containing one or two endocytic motifs\nto direct their internalization through clathrin-coated pits [23,24].\nThe ﬁbronectin-like type II domains of all family members are\nconsidered to have collagen-binding capacity [23], but only the\nmannose receptor (through its CTLD4) and uPARAP/Endo180\n(through its CTLD2) have been shown to bind carbohydrates\n[25,26]. uPARAP/Endo180 has been identiﬁed in many cell\ntypes including monocyte-like U937 cells, vascular smooth\nmuscle cells [22], ﬁbroblasts, vessel endothelial cells, macro-\nphages [16,24,26], osteoblasts [27] and chondrocytes of young\nmice [28]. However, as far as we know, uPARAP/Endo180 has\nnot been reported to be present in stellate cells.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nThe present study presents data indicating that uPARAP/\nEndo180 may be the main receptor responsible for endocytosis of\ndenatured collagen in activated rat hepatic stellate cells.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nEXPERIMENTAL\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nMaterials\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nType I calf skin collagen and type IV collagen were from\nSigma and they were heat denatured by incubation at 60 ◦C for\n30 min. High molecular mass urokinase was from Diagnostica &\nAnalys Service AB (G¨oteborg, Sweden). tPA (tissue plasminogen\nactivator) and E-64d were from Calbiochem (Oslo, Norway).\nAnti-rat CD49b (integrin α2 chain) and anti-rat CD29 (integrin β1\nchain) were from PharMingen International (Laborel AS, Oslo,\nNorway). The tetrapeptide Asp-Gly-Glu-Ala and the tripeptide\nArg-Gly-Asp were from Bachem (Bubendorf, Switzerland). 125I\nwas from Amersham Biosciences. Polyclonal rabbit antibody to\nEndo180 was a gift from Dr C. M. Isacke (The Breakthrough\nBreast Cancer Research Centre, Institute of Cancer Research,\nLondon, U.K.). All other chemicals were purchased from Sigma\nunless otherwise mentioned.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nat 5–10 passages. These stellate cells were in ‘activated state’, as\njudged by the presence of desmin and loss of lipid droplets [9,10].\nCells were suspended by trypsinization when conﬂuent and main-\ntained on ﬂasks as described above. Endocytosis experiments were\nperformed with cells grown on 6- or 24-well plates. After the cells\nhad reached approx. 60–80 % conﬂuence, the wells were washed\nonce with serum-free medium and then incubated in 1 ml of\nserum-free medium containing 2 mM L-glutamine, and 1 % BSA\n(to block non-speciﬁc-binding sites) at 37 ◦C for 30 min before\nexperiments were started. Studies of ligand binding were per-\nformed as described above with the exception that the medium\ncontained 20 mM Hepes and the experiments were started after an\nadditional incubation at 4 ◦C for 30 min. Binding of radiolabelled\nligands to wells without cells in the presence of 1 % BSA was\nnegligible (< 0.05 %).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nRadiolabelling of ligands\nDenatured collagen (100 µg in 100 µl of PBS) was incubated with\n10 µl of 125I (40 MBq) in an Iodogen-coated tube (IODO-GEN;\nBio-Rad Laboratories). After 45–60 min, the labelled protein\nwas separated from free Na125I using a Sephadex G-25 column\n(Amersham Biosciences). The speciﬁc activity of [125I]collagen\nwas 8 × 105 c.p.m./µg. OVA (ovalbumin) or denatured colla-\ngen was labelled with [125I]TC ([125I]tyramine cellobiose) as des-\ncribed previously [32]. The speciﬁc activities of [125I]TC-collagen\nand [125I]TC-OVA were 3 × 105 and 5 × 105 c.p.m./µg res-\npectively.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nEndocytosis experiments\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nFor studies of endocytic uptake of [125I]collagen, the cells were\nincubated with the labelled ligand at 37 ◦C. In experiments using\ninhibitors, cells were incubated for 15 min with inhibitors before\nthe addition of [125I]collagen. At indicated time-points duplicate\n0.1 ml aliquots were taken from the medium of each well for the\ndetermination of the amount of acid-soluble radioactivity released\nto the medium, and the cells were gently washed with ice-cold\nPBS and scraped in lysis buffer (0.1 % SDS, 0.1 M NaOH). The\nsamples were precipitated with TCA (trichloracetic acid) and BSA\nat ﬁnal concentrations of 10 and 0.5 % respectively, and cen-\ntrifuged at 1000 g for 10 min at 4 ◦C. Acid-soluble and acid-\nprecipitable radioactivities were counted in a gamma counter.\nCell-associated ligand was calculated by subtracting radioactivity\nmeasured at time zero from cell-associated radioactivity measured\nat later time points. To determine the actual amount of degraded\nligand released to the medium from the cells, acid-soluble radio-\nactivity in wells without cells was subtracted from values in cor-\nresponding wells with cells. Total cellular uptake is expressed as\nthe sum of cell-associated [125I]collagen and acid soluble [125I]-\ncollagen released to the medium.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nIsolation and culture of stellate cells\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nHepatic parenchymal and non-parenchymal cells were isolated\nfrom male Wistar rats, weighing 250–300 g, by the collagenase\nperfusion method [29,30]. After the separation of parenchymal\ncells by low-speed centrifugation at 50 g for 2 min, the supernatant\ncontaining stellate cells, other non-parenchymal cells and con-\ntaminating parenchymal cells were further centrifuged in a Percoll\ndensity gradient to isolate stellate cells [31]. After primary cul-\nture in plastic ﬂasks using Dulbecco’s modiﬁed Eagle’s medium\n(BioWittaker, MedProbe, Oslo, Norway) containing 10 % (v/v)\nfetal bovine serum and 2 mM L-glutamine and antibiotics\n(BioWhittaker) at 37 ◦C in a humidiﬁed atmosphere of 95 %\nair and 5 % CO2, the stellate cells were used for experiments\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nConfocal ﬂuorescence microscopy\nSerum-free Dulbecco’s modiﬁed Eagle’s medium (120 µl), con-\ntaining 50 000 cells, was seeded on poly(D-lysine)-coated cover\nglasses in 24-well plates for 60 min at 37 ◦C. After washing away\nnon-adherent cells, the cells were incubated with the uptake me-\ndium (serum-free medium containing 5 µg/ml heat-denatured\ntype IV collagen labelled with FITC and 1 % BSA) at 37 ◦C.\nUptake was allowed to proceed for 15 min in the presence and\nabsence of LysoTracker Red (Molecular Probes; 0.5 µM), after\nwhich the cells were washed several times with PBS. The cells\nincubated in the presence of LysoTraker were then ﬁxed in 3 %\n(w/v) paraformaldehyde, whereas cells incubated with FITC-\ncollagen alone were ﬁrst chased for 90 min in ligand-free medium\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nand thereafter for 30 min in the presence of LysoTracker. After\nﬁxation, cover glasses were washed in PBS and then mounted\non to Mowiol. Images were collected using an IX81 confocal\nmicroscope (OLYMPUS, FLUOVIEW, FV1000). FITC was ex-\ncited using an argon laser at 488 nm, whereas LysoTraker was\nexcited using a helium–neon laser at 543 nm.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nLigand-binding experiments\nBinding experiments were performed with cells incubated at 4 ◦C\nwith [125I]collagen in the presence of increasing concentrations of\nunlabelled denatured collagen. For competition experiments, the\ncells were preincubated in the absence or presence of competitive\nmolecules for 60 min at 4 ◦C, then [125I]collagen was added, and\nafter incubation for 2 h at 4 ◦C, the wells were washed gently with\ncold PBS to remove unbound ligand, and the cells were scraped\noff in lysis buffer before the determination of surface-bound [125I]-\ncollagen. Non-speciﬁc binding was calculated by subtracting\nradioactivity measured in the presence of a 50-fold excess of un-\nlabelled denatured collagen.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nPreparation of cell membrane proteins\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nHepatic stellate cell membrane fractions containing the integral\nmembrane proteins were prepared using a membrane extraction\nreagent kit (Mem-PER; Pierce) according to the manufacturer’s\ninstructions.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nLysate preparation, electrophoresis and immunoblot analysis\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nFor SDS/PAGE experiments, cells were allowed to detach from\nthe culture ﬂasks by incubating with PBS at 37 ◦C with occasional\ngentle agitation. The cells were pelleted by centrifugation and then\nlysed in lysis buffer (0.3 M NaCl, 20 mM Hepes, 0.2 mM EDTA,\n1.5 mM MgCl2, 0.1 % Triton X-100 and protease inhibitors) at\n4 ◦C. The whole lysates were centrifuged at 10 000 g for 2 min and\nthe clariﬁed lysates were diluted 1:1 in sample buffer [Tris buffer,\npH 6.8, 2 % SDS and 10 % (w/v) sucrose], heated for 5 min and\nthen separated by SDS/PAGE under non-reducing conditions.\nAfter transfer on to PVDF membranes (Millipore), the membranes\nwere blocked in 5 % (w/v) skimmed milk, incubated with the\nprimary antibody (polyclonal rabbit anti-human-Endo180, diluted\n1:5000) for 1 h, and thereafter washed three times in PBS con-\ntaining 0.1 % Tween 20, and incubated with the secondary anti-\nbody (mouse anti-rabbit IgG conjugated to horseradish peroxi-\ndase) for 1 h. Finally, the membranes were washed three times and\nprotein bands were detected using enhanced chemiluminescence\nECL® reagent (Amersham Biosciences).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nAnalysis of gene expression by RT (reverse transcriptase)–PCR\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nTotal RNA was isolated using TRIzol® (Invitrogen) from rat hepa-\ntic stellate cells, rat Kupffer cells, rat liver endothelial cells, rat\nparenchymal cells and J774 mouse macrophage cell-line accord-\ning to the manufacturer’s instructions. First-strand cDNA was\nsynthesized from 5 µg of total RNA using SuperScript RT\n(Invitrogen). After cDNA synthesis, 20 µl of mixture was diluted\nto a ﬁnal volume of 50 µl and from this cDNA mixture 1 µl was\nused as template in PCR. Primers were designed using the Pri-\nmer 3 Output program (http://frodo.wi.mit.edu/cgi-bin/primer3/\nprimer3 www.cgi) to amplify Endo180 in rat liver stellate cells\nfrom the human Endo180 mRNA sequence. The primers used\nwere: for Endo180: 5-CCTGGGCATGTATGAGTGTG-3 and\n5-TGTCAGTCAGCTGGTCCTTG-3, and for β-actin: 5-AG-\nCCATGTACGTAGCCATCC-3 and 5-TCTCAGCTGTGGTG-\nGTGAAG-3. Routine PCR was performed using DyNAzyme\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n(Finnzymes, Espoo, Finland) according to the manufacturer’s\ninstructions and performed at the following conditions: 94 ◦C for\n5 min, 35 cycles each 30 s at 94 ◦C, 60 ◦C for 30 s and 72 ◦C\nfor 30 s. The PCR products were cloned into pGEM-T Easy Vector\n(Promega) and sequenced (GATC-Biotech, Konstanz, Germany).\nPCR marker was provided by New England Biolabs (Beverly,\nMA, U.S.A.).\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nRESULTS\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nUptake and degradation of [125I]collagen by stellate cells\n**BLOCK**fs== 9.5**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nIn initial experiments, we examined the ability of stellate cells\nto take up and degrade the [125I]collagen. Cells were incubated\nwith [125I]collagen at 37 ◦C for different time periods and uptake\nand degradation of ligand were determined as described in the\nExperimental section. As shown in Figure 1(A), stellate cells take\nup [125I]collagen efﬁciently. After 1 h at 37 ◦C in the presence\nof [125I]collagen, the cells had removed approx. 5 % of the avail-\nable ligand from the medium. To determine whether the ligand\nwas transported to lysosomes for degradation, acid soluble and\nacid precipitable radioactivities were determined in cells and me-\ndium. The results show that degraded ligand accumulated both\ninside the cells and in the medium. Release of acid soluble radio-\nactivity to the medium started after a lag phase lasting approx.\n15 min. Acid precipitable radioactivity in the cells increased for\napprox. 30 min and then levelled off, indicating that an equi-\nlibrium between transport to and degradation in lysosomes had\nbeen reached. Similar results were obtained when denatured 125I-\nlabelled collagen IV was used (results not shown). That de-\ngradation of collagen takes place intracellularly was veriﬁed\nby the ﬁnding that degradation products formed from [125I]TC-\ncollagen were not released to the medium during the incubation\ntime (120 min) (results not shown). Labelled degradation pro-\nducts formed from this probe are trapped in the degradative\ncompartment [33].\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nWe also analysed the transport of ﬂuorescently labelled de-\nnatured collagen to late endocytic compartments in pulse–\nchase experiments. Fluorescence microscopy using FITC-labelled\ncollagen (Figure 2) showed lysosomal/late endosomal staining, as\nindicated by co-localization (yellow) due to overlap of the inter-\nnalized ligand (green) and LysoTracker (red), a marker of acidic\nendocytic compartments.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nBinding and internalization of [125I]collagen to hepatic stellate\ncells is saturable and pH-dependent, but not calcium-dependent\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nTo determine whether endocytosis of [125I]collagen is receptor-\nmediated, cells were incubated with [125I]collagen in the presence\nof increasing concentrations of unlabelled collagen. The results\nobtained (Figure 1B) show that both binding (at 4 ◦C) and uptake\n(at 37 ◦C) of [125I]collagen were reduced by adding unlabelled\ncollagen. At 6 µg/ml of unlabelled collagen, binding and uptake of\n[125I]collagen were decreased by 70 and 90 % respectively. Taken\ntogether, these experiments indicate that the uptake of collagen\nrepresents a receptor-mediated process and not merely ﬂuid-phase\nendocytosis. Binding of [125I]collagen was pH-dependent, and was\nreduced by approx. 50 % when pH was reduced to pH 5.5 (results\nnot shown). Binding was, however, retained in the presence of the\ncalcium chelator EGTA (2.2 mM).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\nDegradation of [125I]collagen is inhibited by protease\nand proton pump inhibitors\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nLeupeptin, an inhibitor of cysteine proteases, inhibited the degra-\ndation of [125I]collagen partially (Figure 3, upper panel). After\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nFigure 1 Uptake and binding of [125I]collagen to hepatic stellate cells\n**BLOCK**fs== 7.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\n(A) Time course of receptor-mediated uptake and degradation of collagen: cells (∼ 106 cells/\nwell) were incubated with [125I]collagen (0.2 µg/ml) at 37 ◦C for the indicated times, washed and\nthen lysed. Cell lysates and medium were treated with TCA to determine acid-soluble and acid-\nprecipitable radioactivities as described in the Experimental section. Values are expressed as\npercentage of total initial acid precipitable radioactivity in the medium and are means +\n− S.D.\nresults for triplicate samples. Equivalent results were obtained in at least six separate experiments.\n(B) Concentration-dependent binding and uptake of collagen in stellate cells: cells were pre-\nincubated at 4 ◦C for 30 min with the indicated concentrations of unlabelled collagen. After\npreincubation, [125I]collagen (20 ng/ml) was added and the cells were incubated for 120 min\nat 4 ◦C or 30 min at 37 ◦C. The cells were then washed three times and the cell-associated\nradioactivity was determined. Values are the means +\n− S.D. results for triplicate samples.\nEquivalent results were obtained in at least three separate experiments.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n90 min of incubation at 37 ◦C, formation of acid soluble radio-\nactivity was reduced by approx. 50 % in the presence of leupeptin\n(2 mM). Release of acid-soluble degradation products from the\ncells was, however, reduced by nearly 80 % by the inhibitor. These\nresults suggest that the processing of internalized [125I]collagen\nﬁrst produces acid-soluble peptides, which are then further\ndegraded to amino acids (released from the cells). Leupeptin\nprimarily inhibits degradation of the smaller peptides. Preventing\ncollagen degradation caused, as expected, the accumulation of\nboth acid-precipitable and acid-soluble radioactivity in the cells.\nWe also assessed the effect of E-64d, a selective and irreversible\ninhibitor of lysosomal cysteine proteases, on the degradation\nof internalized [125I]collagen. As indicated in Figure 3 (upper\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\npanel), treatment of cells with E-64d gave results similar to\nthose seen with leupeptin treatment, although at a much lower\nconcentration (50 µM) of the inhibitor. Lysosomal degradation\nof endocytosed ligands depends on a functional vacuolar H+-\nATPase (V-ATPase) in the endosomal/lysosomal compartments\n[34]. To determine whether the inactivation of V-ATPase affects\ncollagen degradation, the cells were incubated in the presence\nof concanamycin A, a selective inhibitor of V-ATPase [35]. The\nresults obtained (Figure 3, lower panel) show that concanamycin\nA inhibited both uptake and degradation of [125I]collagen. In cells\nincubated for 90 min at 37 ◦C, the degradation of labelled colla-\ngen was reduced approx. 85 % in the presence of 0.1 µM con-\ncanamycin A. Uptake was reduced approx. 60 % at this drug\nconcentration.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nEffects of possible competitive ligands and anti-integrin antibodies\non [125I]collagen binding\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nThe rapid removal of collagen from the culture medium sug-\ngested the existence of a speciﬁc receptor(s) in hepatic stellate\ncells. Several integrins have been reported to act as binding recep-\ntors for denatured collagen, including α2β1 [36,37] and αvβ3\n[38,39]. We therefore used anti-integrin antibodies and ligands of\nthese receptors in competition experiments to see whether they\nwere capable of competing with [125I]collagen for binding. At high\nconcentrations (> 20 µg/ml), the anti-integrin α2 chain inhibited\n[125I]collagen binding by approx. 10 %, whereas anti-integrin β1\nchain did not signiﬁcantly inhibit the binding of [125I]collagen. No\nadditional inhibition was observed using both antibodies together.\nPeptides containing the RGD (Arg-Gly-Asp) sequence have been\nshown to inhibit speciﬁcally the binding of denatured type I\ncollagen to the αvβ3 integrin, possibly because this sequence\nbecomes exposed on collagen denaturation [38], whereas the\nsequence DGEA (Asp-Gly-Glu-Ala) binds to α2β1 integrin [40].\n[125I]Collagen binding was therefore examined in the presence\nof the tripeptide Arg-Gly-Asp and the tetrapeptide Asp-Gly-\nGlu-Ala. Neither of these peptides (> 0.5 mg/ml) signiﬁcantly\ninhibited the binding of [125I]collagen (results not shown).\nScavenger receptors have also been shown to serve as receptors\nfor denatured collagen [41]. No inhibitory effects on the binding\nand uptake of [125I]collagen were, however, observed when cells\nwere incubated with a 50 M excess of formaldehyde-treated BSA,\na ligand of scavenger receptors [42] (results not shown).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nuPA inhibits binding and uptake of [125I]collagen\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nCollagen has been shown to block the association of Endo180 with\nuPAR-bound pro-uPA [22]. Conceivably, uPA might also bind to\nthe collagen-binding receptor in the stellate cells if this receptor\nis related to the uPARAP/Endo180 receptor. It was found that uPA\ninhibits both binding and uptake of [125I]collagen to the stellate\ncells in a concentration-dependent manner (Figure 4). At 3 mg/ml,\nuPA reduced both binding and uptake of labelled collagen by\napprox. 80 %. It may, however, be argued that the reduced binding\nof collagen in the presence of uPA is due to the activation of\ncollagen-degrading enzymes (such as MMP-13) secreted by the\ncells into the medium during the 3.5 h incubation (30 min at 37 ◦C\nand 3 h at 4 ◦C). To rule out this possibility, similar experiments\nwere performed in the presence of tPA, which binds receptors\nother than uPAR [43]. However, incubation with 2.2 mg/ml tPA\ndid not inhibit signiﬁcantly the binding of collagen to the cells\n(binding as percentage of total: 1.04 +− 0.19 and 0.96 +− 0.09;\nn = 3, for control and tPA respectively), indicating that the reduced\nbinding of collagen is due to binding of uPA to its receptor (or\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nFigure 2\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nFluorescence microscopy showing co-staining of the internalized FITC-collagen and a marker of late endocytic compartments\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nCells were pulse-labelled at 37 ◦C for 15 min with FITC-collagen (green) in the presence or absence of LysoTracker (red), then washed and either ﬁxed (images on the left) or chased for 90 min, after\nwhich the incubation continued for an additional 30 min in the presence of LysoTracker (images on the right). Co-localization of FITC-collagen with LysoTracker (yellow) is shown in the merged\nimages (lowest panels). Results represent two or three experiments.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nto uPARAP/Endo180, see below) and not to an indirect effect of\nuPA on the activation status of collagenases in the medium.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nuPARAP/Endo180 is expressed in rat hepatic stellate cells\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nResult from experiments with uPA suggested that uPARAP/\nEndo180 is probably involved in the uptake of collagen. Therefore\nwe performed Western-blot analysis, using a polyclonal antibody\nagainst puriﬁed human uPARAP/Endo180 [16]. U937 cells, the\ncell line originally used to deﬁne uPARAP activity [22], were\nused as positive controls, and HepG2 cells were used as negative\ncontrols since they do not express uPARAP/Endo180 [24]. As\nshown in Figure 5, no band was detected in the preparation from\nHepG2 cells, whereas a speciﬁc band at 180 kDa was revealed\nin preparations from both hepatic stellate cells and U937 cells.\nWe also tested the ability of the Endo180 antiserum to interfere\nwith collagen binding to the cells and found that the binding\nof [125I]collagen to the stellate cells was signiﬁcantly reduced\nby 50 % when 20 µl/ml of the antiserum was used (results not\nshown). Analysis by RT–PCR showed that uPARAP/Endo180 is\nabundantly expressed in stellate cells, but is not detectable in\nKupffer cells, liver endothelial cells, hepatocytes and J774 cells\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n(Figure 6). The PCR product gave the expected length of 422 bp\nwhen compared with the human uPARAP/Endo180 sequence\nfrom which the primers were designed (AF134838). The PCR\nproduct was sequenced and shown to have a homology of 89 %\nwith the human uPARAP/Endo180 (AF107292, NM 006039\nand AF134838) and 94 % homology was seen with the mouse\nmannose receptor uPARAP/Endo180 receptor (NM 008626) as\nwell. The β-actin PCR products (227 bp) showed the same mRNA\ncontent in all cell types tested.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nUptake of OVA\nCTLD2 in Endo180 has been shown to contain a Ca2+-dependent\ncarbohydrate-binding determinant, which speciﬁcally binds to\nN-acetylglucosamine, mannose and fucose [26]. To determine\nwhether hepatic stellate cells express such binding activity, we\nmeasured the uptake of OVA (which contains sugars terminating\nin N-acetylglucosamine and mannose [44]). It was found that the\nstellate cells are also capable of taking up OVA. As shown in\nFigure 7, uptake and degradation of [125I]TC-OVA increases as\na function of time. Moreover, the uptake (measured after 30 min\nincubation at 37 ◦C) was reduced by 90 +− 4 and 87 +− 5 % by the\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nFigure 5\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.2**\nEndo180 is expressed in rat hepatic stellate cells\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nThe whole lysates (40 µg) of hepatic stellate cells (lane 1), U937 cells (lane 3) and HepG2 cells\n(lane 4) were analysed by SDS/PAGE on a 6 % gel and then transferred on to membrane. The\nblot was incubated ﬁrst with a polyclonal antiserum directed against Endo180 and subsequently\nwith mouse anti-rabbit IgG conjugated to horseradish peroxidase followed by detection with\nECL®. Lane 2 corresponds to the cell membrane fraction (20 µg) prepared from stellate cells.\nThe membrane was then stripped, blocked and incubated with anti-β-actin. Relative molecular\nmass standards are shown on the left (masses in kDa). The results shown were reproduced in\nat least three independent experiments.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nFigure 3\ndegradation of [125I]collagen\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nEffect of leupeptin, E-64d and concanamycin on uptake and\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nCells were preincubated with either leupeptin (2 mM), E-64d (2 µM) (upper panel) or increasing\nconcentrations of concanamycin (lower panel) for 15 min at 37 ◦C. After preincubation, radio-\nlabelled collagen (40 ng/ml) was added, followed by 90 min incubation at 37 ◦C and three\nwashes. Cell\nlysates and medium were treated with TCA to determine acid-soluble and\nacid-precipitable radioactivities as described in the Experimental section. Values are expressed\nas percentage of controls and are the means +\n− S.D. results for triplicate samples. Equivalent\nresults were obtained in at least three separate experiments.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nFigure 6\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\nExpression analysis of uPARAP/Endo180 mRNA by RT–PCR\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\ntypes of rat\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nFour different\nliver cells [liver endothelial cells (rLEC), Kupffer cells (rKC),\nparenchymal cells (rPC) and stellate cells (rStel)] and J774 mouse macrophages were tested\nfor mRNA expression of Endo180 (lanes 2–6 respectively) with β-actin mRNA as an internal\ncontrol (lanes 7–11). Three experiments gave equivalent results. Sizes are indicated in bp.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFigure 4 Dose-dependent inhibition of binding and uptake of [125I]collagen\nto stellate cells by uPA\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nFigure 7 Uptake and degradation of [125I]TC-OVA by hepatic stellate cells\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nCells were preincubated with increasing concentrations of uPA for 30 min at 4 ◦C. After\npreincubation, radiolabelled collagen (20 ng/ml) was added, followed by 120 min incubation\nat 4 ◦C and three washes. For uptake, uPA was added just before the addition of [125I]collagen\nand cells were incubated at 37 ◦C for 30 min. Results are means +\n− S.D. of a representative\nexperiment performed in triplicate.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nCells (∼ 106 cells/well) were incubated at 37 ◦C with 2 nM [125I]TC-OVA in the presence and\nabsence of a 50-fold excess of unlabelled OVA or 2.2 mM EGTA. At timed intervals the cells were\nwashed and cell-associated radioactivity (acid-soluble and acid-precipitable) was measured as\ndescribed in the Experimental section. Inset: the effect of unlabelled OVA and EGTA on the\nuptake.\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\naddition of 50-fold excess of unlabelled OVA and 2.2 mM EGTA\nrespectively (Figure 7, inset).\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nDISCUSSION\n**BLOCK**fs== 9.5**b== 0.4**t== 0.2**l== 0.1**r== 0.5**\nThe present results demonstrate that the uptake of heat-denatured\n125I-labelled collagen in rat hepatic stellate cells is mediated by\na saturable binding site. To identify the receptor(s) involved, we\nmeasured the uptake of [125I]collagen in the presence and absence\nof antibodies and ligands that bind to potential receptor candi-\ndates. The results obtained seemed to exclude integrins (α2β1 and\nαvβ3) and scavenger receptors. The discoidin domain receptors\n(DDR1 and DDR2) are, probably, not involved in binding, as these\nreceptors require the native, triple-helical conﬁguration of the\ncollagen [45,46]. On the other hand, several observations pointed\nto the uPARAP/Endo180 as the receptor involved in binding colla-\ngen. Behrendt et al. [22] found that collagen prevents the form-\nation of a complex between pro-uPA-uPAR and uPARAP/\nEndo180. We found that the reciprocal interaction, the inter-\naction of uPA-uPAR with uPARAP/Endo180, prevents uPARAP/\nEndo180 from binding to collagen. Alternatively or in addition,\nuPA can directly bind to Endo180. This result suggested that\n[125I]collagen binding in stellate cells could be mediated by\nthe uPARAP/Endo180. That the receptor in question really is\nuPARAP/Endo180 was veriﬁed using Western-blot analysis and\nRT–PCR. Moreover, collagen binding was signiﬁcantly reduced\nby Endo180 antiserum, strongly supporting the involvement of\nuPARAP/Endo180 in mediating the uptake of collagen. Taken\ntogether, these observations are compatible with the newly deﬁned\nrole of uPARAP/Endo180 as an endocytic collagen receptor\n[17–20]. The stellate cells also bind [125I]TC-OVA, compatible\nwith the observation that uPARAP/Endo180, which belongs to\nthe mannose-receptor family, can also function as a lectin, binding\nligands with terminal mannose, fucose and N-acetylglucosamine\n[26]. Moreover, the binding of the OVA is calcium-dependent,\nwhereas the binding of collagen is not. This is in agreement with\nearlier studies of the uPARAP/Endo180 receptor [18].\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nInhibition of [125I]collagen degradation by leupeptin and E-64d,\ntwo inhibitors of lysosomal thiol proteinases and concanamycin\nA, an inhibitor of the vacuolar proton pump [35,47], shows that the\nligand is effectively transported to late endocytic compartments\nfor degradation. These observations are in agreement with recent\nreports demonstrating that uPARAP/Endo180 mediates endo-\ncytosis and lysosomal degradation of collagens in ﬁbroblasts [20]\nand in a breast cancer cell line (MCF-7) [18]. To the best of our\nknowledge, the present study is the ﬁrst to demonstrate receptor-\nmediated endocytosis of collagen in hepatic stellate cells.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nThe afﬁnity of the uPARAP/Endo180 for denatured collagen\nis reduced at pH  6.5 and the ﬁnding that the proton pump in-\nhibitor also reduces cellular uptake of [125I]collagen is probably\na result of increases in pH in early or late endosomes. This will\nprevent dissociation of [125I]collagen from the collagen receptor\nin endosomes, and the ligand will recycle to the plasma membrane\nbound to the receptor. uPARAP/Endo180 has been regarded as a\nconstitutively recycling surface receptor.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nA collagen receptor in activated stellate cells may play a crucial\nrole in the turnover of ECM in liver. The hepatic stellate cells are\nthe major producers of ECM both in normal liver and in the in-\njured liver [48,49]. They synthesize a large variety of ECM\ncomponents [2,5,8] and also participate very actively in their de-\ngradation. To this end different calcium- and zinc-dependent\nenzymes are produced, each being speciﬁc for a group of matrix\ncomponents. MMPs comprise collagenases, gelatinases, strombe-\nlysins and metalloelastases. The stellate cells are the main source\nof MMP synthesis during liver ﬁbrosis [2,9,50]. The main func-\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\ntion of the uPARAP/Endo180 may be to mediate endocytosis of\npartially degraded collagens (and glycoproteins) produced in the\nECM by MMPs. MMP-mediated cleavage of ﬁbrillar collagen\nresults in altered structure accompanied by rapid denaturation\n[51]. These denatured fragments may be treated by uPARAP/\nEndo180 in vivo as heat-denatured collagen is in vitro. The\nuPARAP/Endo180 in stellate cells would thereby co-operate with\nthe secreted MMPs in the remodelling of the matrix, especially\nduring the recovery phase after liver injury.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nWe thank Dr C. M. Isacke and D. Wienke (The Breakthrough Breast Cancer Research\nCentre, Institute of Cancer Research, London, U.K.) for kindly providing the antibody\nagainst Endo180, and Dr L. Malerod for valuable help in the preparation of this paper.\nThis work was supported by the Norwegian Research Council and the Norwegian Cancer\nSociety.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nREFERENCES\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n1 Wake, K. (1980) Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes),\ntheir related structure in and around the liver sinusoids, and vitamin A-storing cells in\nextrahepatic organs. Int. Rev. Cytol. 66, 303–353\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n2 Sato, M., Suzuki, S. and Senoo, H. (2003) Hepatic stellate cells: unique characteristics in\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\ncell biology and phenotype. Cell Struct. Funct. 28, 105–112\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n3 Blomhoff, R., Holte, K., Naess, L. and Berg, T. (1984) Newly administered [3H]retinol is\ntransferred from hepatocytes to stellate cells in liver for storage. Exp. Cell Res. 150,\n186–193\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n4 Senoo, H., Smeland, S., Malaba, L., Bjerknes, T., Stang, E., Roos, N., Berg, T., Norum,\nK. R. and Blomhoff, R. (1993) Transfer of retinol-binding protein from HepG2 human\nhepatoma cells to cocultured rat stellate cells. Proc. Natl. Acad. Sci. U.S.A. 90, 3616–3620\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n5 Bedossa, P. and Paradis, V. (2003) Liver extracellular matrix in health and disease.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nJ. Pathol. 200, 504–515\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n6 Pinzani, M., Failli, P., Ruocco, C., Casini, A., Milani, S., Baldi, E., Giotti, A. and\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nGentilini, P. (1992) Fat-storing cells as liver-speciﬁc pericytes. Spatial dynamics of\nagonist-stimulated intracellular calcium transients. J. Clin. Invest. 90, 642–646\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n7 Rockey, D. C. (2001) Hepatic blood ﬂow regulation by stellate cells in normal and injured\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.3**\nliver. Semin. Liver Dis. 21, 337–349\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n8 Li, D. and Frieman, S. L. (2001) Hepatic stellate cells: morphology, function, and\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nregulation. In The Liver: Biology and Pathobiology, 4th edn (Arias, I. M., Boyer, J. L.,\nChisari, E. V., Fausto, N., Schachter, D. and Shafritz, D. A., eds.), pp. 455–468, Lippincott\nWilliams & Wilkins, Philadelphia\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n9 Friedman, S. L. and Arthur, M. J. (1989) Activation of cultured rat hepatic lipocytes by\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nKupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation\nof cell proliferation via induction of platelet-derived growth factor receptors.\nJ. Clin. Invest. 84, 1780–1785\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n10 Gressner, A. M. (1996) Transdifferentiation of hepatic stellate cells (Ito cells) to\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nmyoﬁbroblasts: a key event in hepatic ﬁbrogenesis. Kidney Int. Suppl. 54, S39–S45\n11 Tsutsumi, M., Takada, A. and Takase, S. (1987) Characterization of desmin-positive rat\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nliver sinusoidal cells. Hepatology 7, 277–284\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n12 Ramadori, G., Veit, T., Schwogler, S., Dienes, H. P., Knittel, T., Rieder, H. and\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nMeyer zum Buschenfelde, K. H. (1990) Expression of the gene of the alpha-smooth\nmuscle-actin isoform in rat liver and in rat fat-storing (ITO) cells. Virchows Arch. B Cell\nPathol. Incl. Mol. Pathol. 59, 349–357\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n13 Benyon, R. C. and Arthur, M. J. (2001) Extracellular matrix degradation and the role of\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.5**r== 0.2**\nhepatic stellate cells. Semin. Liver Dis. 21, 373–384\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n14 Milani, S., Herbst, H., Schuppan, D., Grappone, C., Pellegrini, G., Pinzani, M., Casini, A.,\nCalabro, A., Ciancio, G., Stefanini, F. et al. (1994) Differential expression of matrix-\nmetalloproteinase-1 and -2 genes in normal and ﬁbrotic human liver. Am. J. Pathol. 144,\n528–537\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n15 Senoo, H., Stang, E., Nilsson, A., Kindberg, G. M., Berg, T., Roos, N., Norum, K. R. and\nBlomhoff, R. (1990) Internalization of retinol-binding protein in parenchymal and stellate\ncells of rat liver. J. Lipid Res. 31, 1229–1239\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n16 Isacke, C. M., van der Geer, P., Hunter, T. and Trowbridge, I. S. (1990) p180, a novel\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nrecycling transmembrane glycoprotein with restricted cell type expression.\nMol. Cell. Biol. 10, 2606–2618\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n17 Engelholm, L. H., List, K., Netzel-Arnett, S., Cukierman, E., Mitola, D. J., Aaronson, H.,\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nKjoller, L., Larsen, J. K., Yamada, K. M., Strickland, D. K. et al. (2003) uPARAP/Endo180\nis essential for cellular uptake of collagen and promotes ﬁbroblast collagen adhesion.\nJ. Cell Biol. 160, 1009–1015\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n18 Wienke, D., MacFadyen, J. R. and Isacke, C. M. (2003) Identiﬁcation and characterization\nof the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor.\nMol. Biol. Cell 14, 3592–3604\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n19 East, L., McCarthy, A., Wienke, D., Sturge, J., Ashworth, A. and Isacke, C. M. (2003)\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n36 Staatz, W. D., Walsh, J. J., Pexton, T. and Santoro, S. A. (1990) The alpha 2 beta 1 integrin\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nA targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen\nuptake. EMBO Rep. 4, 710–716\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\ncell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen.\nJ. Biol. Chem. 265, 4778–4781\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n20 Kjoller, L., Engelholm, L. H., Hoyer-Hansen, M., Dano, K., Bugge, T. H. and Behrendt, N.\n(2004) uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV.\nExp. Cell Res. 293, 106–116\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n21 Howard, M. J. and Isacke, C. M. (2002) The C-type lectin receptor Endo180 displays\ninternalization and recycling properties distinct from other members of the mannose\nreceptor family. J. Biol. Chem. 277, 32320–32331\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n37 Yamamoto, K. and Yamamoto, M. (1994) Cell adhesion receptors for native and denatured\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\ntype I collagens and ﬁbronectin in rabbit arterial smooth muscle cells in culture.\nExp. Cell Res. 214, 258–263\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n38 Davis, G. E. (1992) Afﬁnity of integrins for damaged extracellular matrix: alpha v beta 3\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nbinds to denatured collagen type I through RGD sites. Biochem. Biophys. Res. Commun.\n182, 1025–1031\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n22 Behrendt, N., Jensen, O. N., Engelholm, L. H., Mortz, E., Mann, M. and Dano, K. (2000)\nA urokinase receptor-associated protein with speciﬁc collagen binding properties.\nJ. Biol. Chem. 275, 1993–2002\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n39 Mogford, J. E., Davis, G. E., Platts, S. H. and Meininger, G. A. (1996) Vascular smooth\nmuscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD\npeptides. Circ. Res. 79, 821–826\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n23 East, L. and Isacke, C. M. (2002) The mannose receptor family. Biochim. Biophys. Acta\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n40 Staatz, W. D., Fok, K. F., Zutter, M. M., Adams, S. P., Rodriguez, B. A. and Santoro, S. A.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n1572, 364–386\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n24 Sheikh, H., Yarwood, H., Ashworth, A. and Isacke, C. M. (2000) Endo180, an endocytic\nrecycling glycoprotein related to the macrophage mannose receptor is expressed on\nﬁbroblasts, endothelial cells and macrophages and functions as a lectin receptor.\nJ. Cell Sci. 113, 1021–1032\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n25 Weis, W. I., Taylor, M. E. and Drickamer, K. (1998) The C-type lectin superfamily in the\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nimmune system. Immunol. Rev. 163, 19–34\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n26 East, L., Rushton, S., Taylor, M. E. and Isacke, C. M. (2002) Characterization of sugar\nbinding by the mannose receptor family member, Endo180. J. Biol. Chem. 277,\n50469–50475\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n27 Engelholm, L. H., Nielsen, B. S., Netzel-Arnett, S., Solberg, H., Chen, X. D., Lopez Garcia,\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nJ. M., Lopez-Otin, C., Young, M. F., Birkedal-Hansen, H., Dano, K. et al. (2001)\nThe urokinase plasminogen activator receptor-associated protein/endo180 is\ncoexpressed with its interaction partners urokinase plasminogen activator receptor and\nmatrix metalloprotease-13 during osteogenesis. Lab. Invest. 81, 1403–1414\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n(1991) Identiﬁcation of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin\nin collagen. J. Biol. Chem. 266, 7363–7367\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n41 Gowen, B. B., Borg, T. K., Ghaffar, A. and Mayer, E. P. (2000) Selective adhesion of\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nmacrophages to denatured forms of type I collagen is mediated by scavenger receptors.\nMatrix Biol. 19, 61–71\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n42 Matsumoto, K., Sano, H., Nagai, R., Suzuki, H., Kodama, T., Yoshida, M., Ueda, S.,\nSmedsrod, B. and Horiuchi, S. (2000) Endocytic uptake of advanced glycation end\nproducts by mouse liver sinusoidal endothelial cells is mediated by a scavenger\nreceptor distinct from the macrophage scavenger receptor class A. Biochem. J. 352,\n233–240\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n43 Cesarman, G. M., Guevara, C. A. and Hajjar, K. A. (1994) An endothelial cell receptor for\nplasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of\nt-PA-dependent plasminogen activation. J. Biol. Chem. 269, 21198–21203\n44 Yamashita, K., Tachibana, Y. and Kobata, A. (1978) The structures of the galactose-\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.2**\ncontaining sugar chains of ovalbumin. J. Biol. Chem. 253, 3862–3869\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n28 Howard, M. J., Chambers, M. G., Mason, R. M. and Isacke, C. M. (2004) Distribution of\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n45 Vogel, W., Gish, G. D., Alves, F. and Pawson, T. (1997) The discoidin domain receptor\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nEndo180 receptor and ligand in developing articular cartilage. Osteoarthritis Cartilage 12,\n74–82\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\n29 Berry, M. N. and Friend, D. S. (1969) High-yield preparation of isolated rat liver\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nparenchymal cells: a biochemical and ﬁne structural study. J. Cell Biol. 43, 506–520\n30 Seglen, P. O. (1976) Preparation of isolated rat liver cells. Methods Cell Biol. 13, 29–83\n31 Senoo, H. and Hata, R. (1993) Isolation of perisinusoidal stellate cells (vitamin A-storting\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\ntyrosine kinases are activated by collagen. Mol. Cell 1, 13–23\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n46 Shrivastava, A., Radziejewski, C., Campbell, E., Kovac, L., McGlynn, M., Ryan, T. E.,\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nDavis, S., Goldfarb, M. P., Glass, D. J., Lemke, G. et al. (1997) An orphan receptor\ntyrosine kinase family whose members serve as nonintegrin collagen receptors.\nMol. Cell 1, 25–34\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n47 Tolleshaug, H. and Berg, T. (1981) The effect of leupeptin on intracellular digestion of\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\ncells, fat-storting cells) of the rat liver. Connect. Tissue 25, 129–137\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nasialofetuin in rat hepatocytes. Exp. Cell Res. 134, 207–217\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n32 Kjeken, R., Mousavi, S. A., Brech, A., Grifﬁths, G. and Berg, T. (2001) Wortmannin-\nsensitive trafﬁcking steps in the endocytic pathway in rat liver endothelial cells.\nBiochem. J. 357, 497–503\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n48 Friedman, S. L., Roll, F. J., Boyles, J. and Bissell, D. M. (1985) Hepatic lipocytes:\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nthe principal collagen-producing cells of normal rat liver. Proc. Natl. Acad. Sci. U.S.A. 82,\n8681–8685\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n33 Pittman, R. C., Carew, T. E., Glass, C. K., Green, S. R., Taylor, Jr, C. A. and Attie, A. D.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n49 Kawase, T., Shiratori, Y. and Sugimoto, T. (1986) Collagen production by rat liver\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n(1983) A radioiodinated, intracellularly trapped ligand for determining the sites of plasma\nprotein degradation invivo. Biochem. J. 212, 791–800\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n34 Mellman, I., Fuchs, R. and Helenius, A. (1986) Acidiﬁcation of the endocytic and exocytic\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\npathways. Annu. Rev. Biochem. 55, 663–700\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n35 Mousavi, S. A., Kjeken, R., Berg, T. O., Seglen, P. O., Berg, T. and Brech, A. (2001)\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nEffects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy.\nBiochim. Biophys. Acta 1510, 243–257\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nfat-storing cells in primary culture. Exp. Cell Biol. 54, 183–192\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n50 Knittel, T., Mehde, M., Kobold, D., Saile, B., Dinter, C. and Ramadori, G. (1999)\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nExpression patterns of matrix metalloproteinases and their inhibitors in parenchymal and\nnon-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1. J. Hepatol.\n30, 48–60\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n51 Vogel, W. F. (2001) Collagen-receptor signaling in health and disease. Eur. J. Dermatol.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n11, 506–514\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nReceived 8 June 2004/14 October 2004; accepted 27 October 2004\nPublished as BJ Immediate Publication 27 October 2004, DOI 10.1042/BJ20040966",
         "extracted",
         "uPARAP/endo180 directs lysosomal delivery and degradation of collagen IV.;Identification and characterization of the endocytic transmembrane glycoprotein Endo180 as a novel collagen receptor.;A targeted deletion in the endocytic receptor gene Endo180 results in a defect in collagen uptake;Liver extracellular matrix in health and disease;Hepatic stellate cells: unique characteristics in cell biology and phenotype.;uPARAP/Endo180 is essential for cellular uptake of collagen and promotes fibroblast collagen adhesion;Characterization of Sugar Binding by the Mannose Receptor Family Member, Endo180*;The mannose receptor family.;The C-type Lectin Receptor Endo180 Displays Internalization and Recycling Properties Distinct from Other Members of the Mannose Receptor Family*;Collagen-receptor signaling in health and disease.;The Urokinase Plasminogen Activator Receptor–Associated Protein/Endo180 Is Coexpressed with Its Interaction Partners Urokinase Plasminogen Activator Receptor and Matrix Metalloprotease-13 during Osteogenesis;Hepatic Blood Flow Regulation by Stellate Cells in Normal and Injured Liver;Extracellular Matrix Degradation and the Role of Hepatic Stellate Cells;Wortmannin-sensitive trafficking steps in the endocytic pathway in rat liver endothelial cells.;Effects of inhibitors of the vacuolar proton pump on hepatic heterophagy and autophagy.;Endo180, an endocytic recycling glycoprotein related to the macrophage mannose receptor is expressed on fibroblasts, endothelial cells and macrophages and functions as a lectin receptor.;Selective adhesion of macrophages to denatured forms of type I collagen is mediated by scavenger receptors.;A Urokinase Receptor-associated Protein with Specific Collagen Binding Properties*;The C‐type lectin superfamily in the immune system;An orphan receptor tyrosine kinase family whose members serve as nonintegrin collagen receptors.;The discoidin domain receptor tyrosine kinases are activated by collagen.;Vascular Smooth Muscle αvβ3 Integrin Mediates Arteriolar Vasodilation in Response to RGD Peptides;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Cell adhesion receptors for native and denatured type I collagens and fibronectin in rabbit arterial smooth muscle cells in culture.;An endothelial cell receptor for plasminogen/tissue plasminogen activator (t-PA). II. Annexin II-mediated enhancement of t-PA-dependent plasminogen activation.;Differential expression of matrix-metalloproteinase-1 and -2 genes in normal and fibrotic human liver.;Transfer of retinol-binding protein from HepG2 human hepatoma cells to cocultured rat stellate cells.;Fat-storing cells as liver-specific pericytes. Spatial dynamics of agonist-stimulated intracellular calcium transients.;Affinity of integrins for damaged extracellular matrix: αvβ3 binds to denatured collagen type I through RGD sites;Identification of a tetrapeptide recognition sequence for the alpha 2 beta 1 integrin in collagen.;Internalization of retinol-binding protein in parenchymal and stellate cells of rat liver.;p180, a novel recycling transmembrane glycoprotein with restricted cell type expression;The alpha 2 beta 1 integrin cell surface collagen receptor binds to the alpha 1 (I)-CB3 peptide of collagen.;Activation of cultured rat hepatic lipocytes by Kupffer cell conditioned medium. Direct enhancement of matrix synthesis and stimulation of cell proliferation via induction of platelet-derived growth factor receptors.;Characterization of desmin‐positive rat liver sinusoidal cells;Hepatic lipocytes: the principal collagen-producing cells of normal rat liver.;A radioiodinated, intracellularly trapped ligand for determining the sites of plasma protein degradation in vivo.;The effect of leupeptin on intracellular digestion of asialofetuin in rat hepatocytes.;The structures of the galactose-containing sugar chains of ovalbumin.;HIGH-YIELD PREPARATION OF ISOLATED RAT LIVER PARENCHYMAL CELLS;Distribution of Endo180 receptor and ligand in developing articular cartilage.;Endocytic uptake of advanced glycation end products by mouse liver sinusoidal endothelial cells is mediated by a scavenger receptor distinct from the macrophage scavenger receptor class A.;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-α and TGF-β1;Expression patterns of matrix metalloproteinases and their inhibitors in parenchymal and non-parenchymal cells of rat liver: regulation by TNF-alpha and TGF-beta1.;Vascular smooth muscle alpha v beta 3 integrin mediates arteriolar vasodilation in response to RGD peptides.;Transdifferentiation of hepatic stellate cells (Ito cells) to myofibroblasts: a key event in hepatic fibrogenesis.;Affinity of integrins for damaged extracellular matrix: alpha v beta 3 binds to denatured collagen type I through RGD sites.;Expression of the gene of the α-smooth muscle-actin isoform in rat liver and in rat fat-storing (ITO) cells;Acidification of the endocytic and exocytic pathways.;Collagen production by rat liver fat-storing cells in primary culture.;Newly administered [3H]retinol is transferred from hepatocytes to stellate cells in liver for storage.;Perisinusoidal stellate cells (fat-storing cells, interstitial cells, lipocytes), their related structure in and around the liver sinusoids, and vitamin A-storing cells in extrahepatic organs.;Preparation of isolated rat liver cells.;Hepatic stellate cells: morphology, function, and regulation;Isolation of perisinusoidal stellate cells (vitamin A-storting cells, fat-storting cells) of the rat liver",
         "Uptake of denatured collagen into hepatic stellate cells: evidence for the involvement of urokinase plasminogen activator receptor-associated protein/Endo180.",
         "None"
        ],
        [
         "17",
         "0016746ad49041402c38da533049ebe0fe9539d1",
         "ABSTRACT Several expanded-spectrum β-lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enterobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional mechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.",
         "L. Poirel,M. Lartigue,J. Decousser,P. Nordmann",
         "https://europepmc.org/articles/pmc538921?pdf=render",
         "\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Jan. 2005, p. 447–550\n0066-4804/05/$08.000 doi:10.1128/AAC.49.1.447–550.2005\nCopyright © 2005, American Society for Microbiology. All Rights Reserved.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nISEcp1B-Mediated Transposition of blaCTX-M in Escherichia coli\nLaurent Poirel, Marie-Fre´de´rique Lartigue, Jean-Winoc Decousser,\nand Patrice Nordmann*\nService de Bacte´riologie-Virologie, Hoˆpital de Biceˆtre, Assistance Publique/Hoˆpitaux de Paris, Faculte´ de\nMe´decine Paris-Sud, Le Kremlin Biceˆtre, France\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nReceived 14 November 2003/Returned for modiﬁcation 1 February 2004/Accepted 11 September 2004\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\nSeveral expanded-spectrum -lactamase blaCTX-M genes are associated with ISEcp1-like elements in Enter-\nobacteriaceae. We found that ISEcp1B was able to mobilize the adjacent blaCTX-M-19 gene by a transpositional\nmechanism in Escherichia coli by recognizing a variety of DNA sequences as right inverted repeats.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nAn increasing number of reports of expanded-spectrum\n-lactamases of the CTX-M type and plasmid-mediated cepha-\nlosporinases of the AmpC type in Enterobacteriaceae raise the\nquestion of their acquisition (5, 24, 26). The blaCTX-M genes\nbelonging to three out of the ﬁve known clusters (the CTX-\nM-1, -M-2, and -M-9 clusters) are associated with ISEcp1-like\ninsertion sequences (IS) (2, 5–7, 11, 12, 18, 21, 23). Neverthe-\nless, the distance that separates the -lactamase gene from\nISEcp1 varies within a given cluster of CTX-M-type genes,\nindicating that different insertion events have occurred (13,\n19). Analysis of GenBank databases reveals that some plasmid-\nencoded cephalosporinase genes, such as the blaCMY-type\ngenes, may also be associated with ISEcp1 (3, 10, 17).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nISEcp1 is an IS weakly related to other IS elements and\nbelonging to the IS1380 family (IS Database HomePage\n[http://www-is.biotoul.fr/page-is.html]) (8). Nothing is known\nabout the mobilization properties of ISEcp1. Recently, we\ncharacterized ISEcp1B associated with a blaCTX-M-19 gene in a\nKlebsiella pneumoniae ILT-3 clinical\nisolate from Vietnam\n(21). ISEcp1B differs from ISEcp1 (GenBank accession no.\nAJ242809) by three nucleotide substitutions. Their inverted\nrepeat (IR) sequences are identical, and their transposases\ndiffer by a single amino acid change (18). Since ISEcp1-like\nelements are located upstream of several -lactamase genes,\nanalysis of the variable sequences separating these IS elements\nfrom the initiation codons of these genes allowed us to deter-\nmine its boundaries.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nRecently, it has been shown that ISEcp1B brought promoter\nsequences for high-level expression of the blaCTX-M-14/-18,\n-lactamase genes that belong to\nblaCTX-M-17, and blaCTX-M-19\nthe same blaCTX-M-9 subgroup (5, 6, 18) and might be also\nresponsible for their mobilization. The aim of this study was to\nanalyze the mobilization process mediated by ISEcp1B.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nISEcp1B possesses two imperfect IRs likely made of 14 bp,\npreviously estimated to be 18 bp (18), with 12 of these 14 bp\nbeing complementary (Table 1). A detailed analysis of a ca.\n9.5-kb DNA fragment from natural plasmid pILT-3 from K.\npneumoniae ILT-3 revealed that the gene encoding ORF1 in\ntransposon Tn1721 was interrupted by a 4.8-kb potential trans-\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n* Corresponding author. Mailing address: Service de Bacte´riologie-\nVirologie, Hoˆpital de Biceˆtre, 78 rue du Ge´ne´ral Leclerc, 94275 Le\nKremlin-Biceˆtre Cedex, France. Phone: 33-1-45-21-36-32. Fax: 33-1-\n45-21-63-40. E-mail: nordmann.patrice@bct.ap-hop-paris.fr.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nposon bracketed on the left side by the left IR (IRL) of\nISEcp1B and on the right side by an imperfect right IR (IRR)\nnamed IRR1 made of a 14-bp sequence belonging to a gene\nencoding a putative iron transport protein (Fig. 1) (18). This\npotential transposon comprises ISEcp1B, blaCTX-M-19, IS903D,\nand part of the gene encoding an iron transport protein. We\nhave hypothesized that ISEcp1B could mobilize adjacent se-\nquences by transposition.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nSeveral transposition experiments were performed to deter-\nmine (i) the mobility of this potential transposon, (ii) the role\nof ISEcp1B in mobilizing the adjacent -lactamase gene, and\n(iii) the role of IS903D. Different recombinant plasmids were\nconstructed by inserting PCR products that encompassed ei-\nther one or both ISs that bracketed the blaCTX-M-19 gene in\nplasmid pILT-3 into the EcoRV site of plasmid pACYC184\n(Table 2) and then transformed into Escherichia coli reference\nstrain DH10B. Recombinant plasmids were constructed with\nthe pACYC184 backbone to prevent overexpression of the\nISEcp1B transposase. Plasmid pC-1 contained the sequences\nof the potential transposon identiﬁed in plasmid pILT-3,\nwhereas the following constructs lacked the right part of it\n(Fig. 1). Plasmid pC-2 possessed the ISEcp1B sequence but\nlacked part of IS903D, and plasmid pC-3 possessed the entire\nIS903D sequence and a truncated ISEcp1B transposase gene\n(Fig. 1; Table 2). E. coli RR1023 (pOX38) was transformed by\nelectroporation with these recombinant plasmids (Table 2) (9).\nPlasmid pOX38 is a self-conjugative and IS-free plasmid.\nTransposition events were searched between inserts of several\nrecombinant plasmids and recipient plasmid pOX38 after 24 h\nof growth in Trypticase soy broth. Transfer of the recombinant\nplasmids with the pOX38 backbone into streptomycin-resistant\nE. coli reference strain DH10B (Table 2) was then performed by\nconjugation as described previously (20), and transconjugants\nwere selected on agar plates containing 100 g of amoxicillin per\nml (plasmid resistance marker) and 20 g of streptomycin per ml\n(chromosomal marker). Several amoxicillin-resistant transconju-\ngants were extracted and sequenced in part.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nSusceptibility to chloramphenicol was always observed in\ntransconjugant strains, thus ruling out full integration (chlor-\namphenicol resistance) of the recombinant plasmids derived\nfrom pACYC184 into plasmid pOX38. Detection of ISEcp1B\nand IS903D was performed using two sets of primers (primer\nISEcp1A [Table 2] in combination with primer ISEcp1B [5-T\nTTCCGCAGCACCGTTTGC-3] and primer IS903A [5-CA\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nDescription of sequence\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nNucleotide sequence\n(533)a\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nNo. of base pairs identical\nto perfect IRRb\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nSize of transposed\nfragment (bp)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\nIRL of ISEcp1B\nDeduced perfect IRR of ISEcp1B\nIRR of ISEcp1B\nIRR1 at the potential transposon end of pILT-3\nIRR2 in plasmid pACYC184\nIRR3 in plasmid pBK-CMV\nIRR4 in the tnpA gene of IS903D\nIRR5 in the iron gene of pILT-3\nIRR6 in the iron gene of pILT-3\nIRR7 in the iron gene of pILT-3\nIRR8 in the iron gene of pILT-3\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\nGATTCTACGTCAGT\nACGTAGAATCTAGG\nACGTGGAATTTAGG\nGCGTTGATTCCTGG\nGCATGGATTGTAGG\nCCACAGAATCAGGG\nGATGAACTGCGGCG\nGATGTGGAACCTCG\nGCACCGGGATGGCG\nGCGCCGCCGCGGGG\nGGGGTCGAACTGGG\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\na Underlined nucleotides are nucleotides identical to those identiﬁed at the same positions in the deduced IRR of ISEcp1B.\nb A perfect IRR contains 14 bp.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.8**r== 0.1**\n4,798\n4,726\n4,541\n3,273\n3,817\n4,167\n4,273\n4,283\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nCATGAAATCATCTGCGC-3] in combination with primer\nIS903B [Table 2], respectively).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nAmoxicillin-resistant\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\ntransconjugants were not obtained\nwhen E. coli RR1023 (pC-3) lacking ISEcp1B (Fig. 1) was used\nas a donor, suggesting that the IS903D element located down-\nstream of the -lactamase gene did not mobilize this latter\ngene. On the other hand, amoxicillin-resistant transconjugants\nwere obtained when E. coli RR1023 (pC-1) and E. coli RR1023\n(pC-2) were used as donors with a transposition efﬁciency of\nca. 107 per donor. These results indicated that ISEcp1B was\nessential for movement of the blaCTX-M-19 gene.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nUsing E. coli RR1023 (pC-1) as a donor, all amoxicillin-\nresistant transconjugants except one contained ISEcp1B and\nIS903D elements, with the blaCTX-M-19 gene and DNA se-\nquences being located on the right part of IS903D in plasmid\npILT-3 (Fig. 1). Surprisingly, none of the 11 transconjugants\nthat were sequenced had the entire potential transposon iden-\ntiﬁed in natural plasmid pILT-3. Nevertheless, the transposi-\ntion mechanism mediated by ISEcp1B was evident, since 5-bp\nduplications were observed at the insertion sites in plasmid\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\npOX38 (see below). These data suggested that the IRR se-\nquences recognized by ISEcp1B were located inside the poten-\ntial transposon. Analysis of the boundaries of the transposed\nfragments identiﬁed sequences, similar to that of the IRR of\nISEcp1B and deﬁned as IRR-like sequences, that had been\nrecognized as such by the transposase of ISEcp1B during a\nmobilization process.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nFive different IRRs (IRR4 to IRR8) (Table 1) were identi-\nﬁed in ﬁve amoxicillin-resistant transconjugants that were an-\nalyzed, four of them being located in the gene encoding the\niron transport protein and one of them being located in the\ntnpA gene of IS903D (Table 2; Fig. 1). The number of identical\nbase pairs among the sequence deﬁned as IRR varied from\nthree to six, corresponding to less than 50%.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nAnalysis of E. coli RR1023 (pOX38) transconjugants resis-\ntant to amoxicillin, obtained with plasmid pC-2 as a donor, and\nthus lacking the sequence used as IRR in pILT-3, also showed\nthat transposition had occurred using nearby sequences of\npACYC184. The IRR-like sequences bracketing the trans-\nposed fragment in these transconjugants remained unchanged.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nFIG. 1. Schematic map of the potential transposon structure in natural plasmid pILT-3 of K. pneumoniae ILT-3 (13). Open reading frames and\ngenes are shown as boxes with an arrow indicating the orientation of each coding sequence and the gene name shown under the corresponding\nbox. IRL and IRR motifs are indicated by vertical arrows. The IRL and IRR extremities of the ISEcp1B-mobilized fragments are in italics, whereas\nthose of the ISEcp1B element are underlined. The sequences cloned in recombinant plasmids are indicated by double-headed arrows with the\ncorresponding recombinant plasmid names.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nStrain or plasmid\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nStrains\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nE. coli RR1023\nE. coli DH10B\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nPlasmids\npOX38\npACYC184\npILT-3\npC-1\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\npC-2\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\npC-2K\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\npC-3\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.4**\nTABLE 2. Strains and plasmids used in this study\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nRelevant genotype or phenotype\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nSource or reference\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\npOX38 recA56; susceptible strain\naraD139 (ara leu)7697 deoR endA1 galK1 galU nupG recA1 rpsL F mrcA (mrr-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.4**\nhsdRMS-mrcBC)80dlacZM15 lacX74; streptomycin resistant\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nK. pneumoniae ILT-3\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nExtended-spectrum cephalosporin-resistant isolate\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nStratagene\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\nNew England Biolabs\n\nThis study\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nThis study\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\n55 kb, F derivative, conjugative\nChloramphenicol resistant\nNatural ca. 50-kb plasmid of K. pneumoniae ILT-3 producing -lactamase CTX-M-19\nRecombinant plasmid pACYC184 with a 4,877-bp PCR fragment (primers PreA [5-\nGCAGGTCTTTTTCTGCTCC-3] and PreB [5-TTTCCGCAGCACCGTTTGC-\n3]) encompassing the potential transposon identiﬁed in pILT-3\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.2**r== 0.3**\nRecombinant plasmid pACYC184 with a 3,635-bp PCR fragment (primers PreA and\nIS903B [5-CCGTAGCGGGTTGTGTTTTC-3]) containing ISEcp1B, blaCTX-M-19\ngene, and truncated IS903D\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.3**\nRecombinant plasmid pBK-CMV with the same 3,635-bp PCR fragment inserted\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nThis study\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ninto pC-2\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.3**\nRecombinant plasmid pACYC184 with a 4,606-bp PCR fragment (primers ISEcp1A\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\n[5-GCAGGTCTTTTTCTGCTCC-3] and PreB) containing blaCTX-M-19 and\nIS903D but lacking part of ISEcp1B\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nThis study\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nThey shared 9 of 14 bp of the perfect IRR sequence (Table 1)\nand corresponded to a segment belonging to the cloning vec-\ntor. In order to rule out\nthe speciﬁc sequence of\nthat\npACYC184 may play a role in this transposition process, the\nsame inserted PCR product of pC-2 was cloned into the SmaI\nsite of pBK-CMV, giving rise to pC-2K (Table 2; Fig. 1).\nAnalysis of several amoxicillin-resistant transconjugants ob-\ntained with plasmid pC-2K as a donor also showed that trans-\nposition had occurred by involving sequences of pBK-CMV.\nThe IRR-like sequences bracketing the transposed fragment in\nthese transconjugants were always the same and shared 9 of\nthe 14 bp of a perfect IRR (Table 1). These results indicated\nthat sequences of pACYC184 (IRR2) or of pBK-CMV (IRR3)\nmight be used as IRRs by ISEcp1B, but further repeated ex-\nperiments are needed to assess this hypothesis. The reason why\nthe IRR1 of the potential transposon of pILT-3 was not the\nboundary of the transposed fragments in these in vitro exper-\niments remains intriguing. One explanation could be that\nIRR2 and IRR3 share greater nucleotide identity in their 3\nextremities (7 and 8, respectively, of last 10 bp) to a perfect\nIRR than IRR1 (only 6 of last 10 bp).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nNo consensus sequence could be determined by comparing\nthese 14-bp-long IRR sequences of recombinant plasmids and\nthe IRR sequences of ISEcp1B in pILT-3 (Table 1). Neverthe-\nless, a guanosine residue located at the 3 end of these IRRs\nwas always found, likely indicating that this nucleotide was\nnecessary in the transposition process. In addition, the last two\nnucleotides of these IRR sequences were either GG or CG.\nThese data indicated that ISEcp1B was involved in a transpo-\nsition process with weakly related IRRs.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAnalysis of the insertion sites of ISEcp1B-mediated trans-\nposition revealed variable sequences in recombinant plasmids\n(AAGAA, TATAA, TATTA, GCTGA, ATATC, TAATA,\nTTTAC, AATGA, TTATA, ATAAG, and TGATT). No con-\nsensus sequence was identiﬁed among the 5-bp duplicated\nsites, whereas an AT-rich content that may target ISEcp1-\nmediated transposition as observed for IS1 was noticed (16,\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n27). In silico analysis of six ISEcp1-like sequences associated\nwith blaCTX-M-like or blaCMY-like -lactamase genes available\nin the GenBank databases did not identify target site duplica-\ntions just bracketing the insertion sites of ISEcp1-like se-\nlikely indicates that after insertion,\nquences. This result\nISEcp1-like elements may be responsible for mobilization of\nthe adjacent sequences. In addition, this in silico analysis con-\nﬁrmed that the 5-bp sequences adjacent to the IRLs and IRRs\nof ISEcp1 elements were always AT rich.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nISEcp1B may use a wide range of DNA sequences as IRRs\nduring a mobilization process that involves its adjacent se-\nquences, therefore explaining its ability to mobilize structurally\nunrelated -lactamase genes. Similar observations have been\nmade for IS1247 of Xanthobacter autotrophicus, which also\nbelongs to the IS1380 family (25). A single copy of IS1247 may\nmobilize sequences located at its right-end extremity in what is\nconsidered a one-ended transposition process since only a sin-\ngle IS is involved.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThese results may also be compared to those obtained with\nother IS elements, such as IS91 and IS911, that are responsible\nfor mobilization of adjacent sequences by a one-ended trans-\nposition mechanism (4, 22). In addition, transposition of other\nelements belonging to the Tn3 family, such as Tn21, generates\nvariable endpoints as a consequence of one-ended transposi-\ntion (1, 15). This peculiar process is the consequence of rec-\nognition of several different IR sequences by the Tn21 trans-\nposase, generating novel joint molecules (14). However, in our\ncase, some homology between the different IRRs used by\nISEcp1B is noticed, whereas one-ended transposition (for ex-\nample, with Tn21) requires one ﬁxed end and another random\none.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nIn the present study, we have demonstrated the ability for\n-lactamase gene. In addi-\nISEcp1B to mobilize the blaCTX-M\ntion, the IS903D element present in the potential transposon in\npILT-3 had no role in the mobilization process.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nIn conclusion, it remains to be determined why the emerging\nexpanded-spectrum -lactamases of the CTX-M type that are\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nstructurally related to several chromosomally located -lacta-\nmase genes of enterobacterial species are associated speciﬁ-\ncally with ISEcp1-like sequences.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nWe thank Keith Derbyshire for giving us the E. coli RR1023 strain\nharboring plasmid pOX38 and for precious advice. We also thank\nHesna Yigit for contributive discussion.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nThis work was ﬁnanced by a grant from the Ministe`re de l’Education\nNationale et de la Recherche (UPRES grant EA3539), Universite´\nParis XI, Paris, France, and by the European Community (6th PCRD,\nLSHM-CT-2003-503-335). L.P. is a researcher from the INSERM,\nParis, France.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n1. Avila, P., J. Grinsted, and F. De La Cruz. 1988. Analysis of the variable\nendpoints generated by one-ended transposition of Tn21. J. Bacteriol. 170:\n1350–1353.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n2. Baraniak, A., J. Fiett, W. Hryniewicz, P. Nordmann, and M. Gniadkowski.\n2002. Ceftazidime-hydrolysing CTX-M-15 extended-spectrum -lactamase\n(ESBL) in Poland. J. Antimicrob. Chemother. 50:393–396.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n3. Bauernfeind, A., I. Schneider, R. Jungwirth, H. Sahly, and U. Ullmann.\n1999. A novel type of AmpC -lactamase, ACC-1, produced by a Klebsiella\npneumoniae strain causing nosocomial pneumonia. Antimicrob. Agents Che-\nmother. 43:1924–1931.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n4. Bernales, I., M. V. Mendiola, and F. De La Cruz. 1999. Intramolecular\ntransposition of insertion sequence IS91 results in second-site simple inser-\ntions. Mol. Microbiol. 33:223–234.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n5. Bonnet, R. 2004. Growing group of extended-spectrum -lactamases: the\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nCTX-M enzymes. Antimicrob. Agents Chemother. 48:1–14.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n6. Cao, V., T. Lambert, and P. Courvalin. 2002. ColE1-like plasmid pIP843 of\nKlebsiella pneumoniae encoding extended-spectrum -lactamase CTX-M-17.\nAntimicrob. Agents Chemother. 46:1212–1217.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n7. Chanawong, A., F. H. M’Zali, J. Heritage, J. H. Xiong, and P. M. Hawkey.\n2002. Three cefotaximases, CTX-M-9, CTX-M-13, and CTX-M-14, among\nEnterobacteriaceae in the People’s Republic of China. Antimicrob. Agents\nChemother. 46:630–637.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n8. Chandler, M., and J. Mahillon. 2002. Insertion sequences revisited, p. 305–\n366. In N. L. Craig, R. Craigie, M. Gellert, and A. M. Lambowitz (ed.),\nMobile DNA II, ASM Press, Washington, D.C.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n9. Derbyshire, K. M., L. Hwang, and N. D. Grindley. 1987. Genetic analysis of\nthe interaction of the insertion sequence IS903 transposase with its terminal\ninverted repeats. Proc. Natl. Acad. Sci. USA 84:8048–8053.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n10. Girlich, D., T. Naas, S. Bellais, L. Poirel, A. Karim, and P. Nordmann. 2000.\nBiochemical-genetic characterization and regulation of expression of an\nACC-1-like chromosome-borne cephalosporinase from Hafnia alvei. Anti-\nmicrob. Agents Chemother. 44:1470–1478.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n11. Karim, A., L. Poirel, S. Nagarajan, and P. Nordmann. 2001. Plasmid-medi-\nated extended-spectrum -lactamase (CTX-M-3-like) from India and gene\nassociation with insertion sequence ISEcp1. FEMS Microbiol. Lett. 201:237–\n241.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n12. Lartigue, M. F., L. Poirel, C. He´ritier, V. Tolu¨n, and P. Nordmann. 2003.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nFirst description of CTX-M-15-producing Klebsiella pneumoniae in Turkey.\nJ. Antimicrob. Chemother. 52:315–316.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n13. Lartigue, M. F., L. Poirel, and P. Nordmann. 2004. Diversity of genetic\nenvironment of the blaCTX-M genes. FEMS Microbiol. Lett. 234:201–207.\n14. Liebert, C. A., R. M. Hall, and A. O. Summers. 1999. Transposon Tn21,\nﬂagship of the ﬂoating genome. Microbiol. Mol. Biol. Rev. 63:507–522.\n15. Martin, C., J. Grinsted, and F. de la Cruz. 1989. Effects of variation of\ninverted-repeat sequences on reactions mediated by the transposase of Tn21.\nJ. Bacteriol. 171:3996–4001.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n16. Meyer, J., S. Iida, and W. Arber. 1980. Does the insertion element IS1\ntranspose preferentially into AT rich DNA segments? Mol. Gen. Genet.\n178:471–473.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n17. Nadjar, D., M. Rouveau, C. Verdet, L. Donay, J. Herrmann, P. H. Lagrange,\nA. Philippon, and G. Arlet. 2000. Outbreak of Klebsiella pneumoniae pro-\nducing transferable AmpC-type -lactamase (ACC-1) originating from\nHafnia alvei. FEMS Microbiol. Lett. 187:35–40.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n18. Poirel, L., J. W. Decousser, and P. Nordmann. 2003. Insertion sequence\n-lac-\nISEcp1B is involved in the expression and mobilization of a blaCTX-M\ntamase gene. Antimicrob. Agents Chemother. 47:2938–2945.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n19. Poirel, L., M. Gniadkowski, and P. Nordmann. 2002. Biochemical analysis of\nthe ceftazidime-hydrolysing extended-spectrum -lactamase CTX-M-15 and\nof its structurally related -lactamase CTX-M-3. J. Antimicrob. Chemother.\n50:1031–1034.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n20. Poirel, L., M. Guibert, S. Bellais, T. Naas, and P. Nordmann. 1999. Integron-\nand carbenicillinase-mediated reduced susceptibility to amoxicillin-clavu-\nlanic acid in isolates of multidrug-resistant Salmonella enterica serotype ty-\nphimurium DT104 from French patients. Antimicrob. Agents Chemother.\n43:1098–1104.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n21. Poirel, L., T. Naas, I. Le Thomas, A. Karim, E. Bingen, and P. Nordmann.\n2001. CTX-M-type extended-spectrum -lactamase that hydrolyzes ceftazi-\ndime through a single amino acid substitution in the omega loop. Antimi-\ncrob. Agents Chemother. 45:3355–3361.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n22. Pre`re, M. F., M. Chandler, and O. Fayet. 1990. Transposition in Shigella\ndysenteriae: isolation and analysis of IS911, a new member of the IS3 group\nof insertion sequences. J. Bacteriol. 172:4090–4099.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n23. Saladin, M., V. T. Cao, T. Lambert, J. L. Donay, J. L. Herrmann, Z. Ould-\nHocine, C. Verdet, F. Delisle, A. Philippon, and G. Arlet. 2002. Diversity of\nCTX-M -lactamases and their promoter regions from Enterobacteriaceae\nisolated in three Parisian hospitals. FEMS Microbiol. Lett. 209:161–168.\n24. Tzouvelekis, L. S., E. Tzelepi, P. T. Tassios, and N. J. Legakis. 2000. CTX-\nM-type -lactamases: an emerging group of extended-spectrum enzymes.\nInt. J. Antimicrob. Agents 14:137–140.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n25. Van der Ploeg, J., M. Willemsem, G. Van Hall, and D. B. Janssen. 1995.\nAdaptation of Xanthobacter autotrophicus GJ10 to bromoacetate due to\nactivation and mobilization of the haloacetate dehalogenase gene by inser-\ntion element IS1247. J. Bacteriol. 177:1348–1356.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n26. Wu, S. W., K. Dornbusch, G. Kronvall, and M. Norgren. 1999. Character-\nization and nucleotide sequence of a Klebsiella oxytoca cryptic plasmid en-\ncoding a CMY-type -lactamase: conﬁrmation that the plasmid-mediated\ncephamycinase originated from the Citrobacter freundii AmpC -lactamase.\nAntimicrob. Agents Chemother. 43:1350–1357.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n27. Zerbib, D., P. Gamas, M. Chandler, P. Prentki, S. Bass, and D. Galas. 1985.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nSpeciﬁcity of insertion of IS1. J. Mol. Biol. 185:517–524.",
         "extracted",
         "",
         "ISEcp1B-Mediated Transposition of blaCTX-M in Escherichia coli",
         "None"
        ],
        [
         "18",
         "001b635f95e9ed2b69830e84a0be4d7748033106",
         "None",
         "I. Kuzmin,G. Hughes,A. Botvinkin,L. Orciari,C. Rupprecht",
         "https://zenodo.org/record/1259441/files/article.pdf",
         "\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.4**\nVirus Research 111 (2005) 28–43\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.0**\nPhylogenetic relationships of Irkut and West Caucasian bat viruses\nwithin the Lyssavirus genus and suggested quantitative criteria based\non the N gene sequence for lyssavirus genotype deﬁnition\nIvan V. Kuzmin a,b,∗, Gareth J. Hughes a, Alexandr D. Botvinkin c,\nLillian A. Orciari a, Charles E. Rupprecht a\na Centers for Disease Control and Prevention, 1600 Cliffton Road, Atlanta, GA 30333, USA\nb Institute for Natural Foci Infections, 7 Prospekt Mira, Omsk 644080, Russia\nc Plague Control Research Institute of Siberia and Far East, 78 Trilissera Street, Irkutsk 664047, Russia\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.3**\nAvailable online 8 April 2005\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nAbstract\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.2**r== 0.0**\nThe nucleoprotein (N), phosphoprotein (P) and glycoprotein (G) genes of Irkut and West Caucasian bat viruses (WCBV) were sequenced\nand compared with those of other lyssaviruses. N gene nucleotide identities provided unequivocal separation of all lyssavirus genotypes with\nan identity threshold of 82%. On this basis, Irkut virus should be considered as a new genotype with particular relatedness to genotypes 4\nand 5 (78.0–78.6% identity for N gene nucleotides and 90.4–92.6% for amino acids). Furthermore, genotypes 4–6, together with Aravan,\nKhujand and Irkut viruses, present a solid phylogroup of Old World bat lyssaviruses. This relationship is apparent using all three viral genes,\nand causes overlap between intragenotype and intergenotype identities for the P gene (Aravan, Khujand viruses and genotype 6) and for the\nG gene (Aravan, Khujand, genotypes 5 and 6). WCBV is the most divergent of known lyssaviruses with only limited relatedness to genotypes\n2 and 3.\n© 2005 Elsevier B.V. All rights reserved.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nKeywords: Rhabdovirus; Lyssavirus; Rabies; Chiroptera; Bat; Phylogeny; Genotype\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n1. Introduction\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.2**r== 0.4**\nUntil recently, the Lyssavirus genus, family Rhabdoviri-\ndae, was believed to consist of seven distinct genotypes\n(GTs). Rabies virus (RABV; GT 1), is distributed world-\nwide among terrestrial mammals and bats, presents the most\ncomprehensive collection of isolates, and has been exten-\nsively studied, due to its health and economic signiﬁcance.\nLagos bat virus (LBV; GT 2) was ﬁrst isolated in Nigeria\nfrom the frugivorous bat (Eidolon helvum) in 1956 (Boulger\nand Porterﬁeld, 1958). This virus was also isolated from the\nbat Micropteropus pussilus in the Central Africa Republic\n(Sureau et al., 1980), from the bat Epomophorus wahlbergi\nin South Africa (Meredith, 1980), cats in South Africa (King\nand Crick, 1988) and Zimbabwe (Foggin, 1988), and from\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\n∗ Corresponding author. Fax: +1 404 639 1564.\nE-mail address: ibk3@cdc.gov (I.V. Kuzmin).\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.2**r== 0.5**\n0168-1702/$ – see front matter © 2005 Elsevier B.V. All rights reserved.\ndoi:10.1016/j.virusres.2005.03.008\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.6**r== 0.0**\na dog in Ethiopia (Mebatsion et al., 1992). Mokola virus\n(MOKV; GT 3) was ﬁrst isolated from shrews in Nigeria in\n1968 (Shope et al., 1970). Thereafter, MOKV was detected\nin shrews from Nigeria and Cameroon, humans in Nigeria,\ndomestic cats in Zimbabwe, Ethiopia and South Africa, a\ndomestic dog in Zimbabwe and a rodent (Lophuromys sika-\npusi) from the Central Africa Republic (reviewed by Nel\net al., 2000). Duvenhage virus (DUVV, GT 4) was isolated\nfrom a human, who died after a bat bite in 1970 in South\nAfrica (Meredith et al., 1971). It has also been identiﬁed in\nthe insectivorous bat (Miniopterus sp.) in South Africa and\nanother bat species (Nycteris thebaica) in Zimbabwe (Van\nder Merwe, 1982; King and Crick, 1988). Since its discov-\nery in 1968, European bat lyssavirus, type 1 (EBLV-1; GT\n5) has been isolated from a number of European countries,\nwhere it was suspected since the 1950s (Schneider et al.,\n1985). The primary host species for the virus is considered\nthe serotine bat (Eptesicus serotinus) (Amengual et al., 1997;\n**BLOCK**fs== 10.0**b== 0.5**t== 0.1**l== 0.1**r== 0.5**\nSerra-Cobo et al., 2002). One human case of EBLV-1 infec-\ntion occurred following a bat bite in Russia in 1985 (Selimov\net al., 1989). European bat lyssavirus, type 2 (EBLV-2; GT\n6) was isolated in Finland from a biologist, who died of ra-\nbies (Lumio et al., 1986). Later, it was isolated from bats\n(primarily from the genus Myotis) in northwestern Europe\n(Amengual et al., 1997), and more recently caused a sec-\nond recorded human death, this time in the United Kingdom\n(Fooks et al., 2002). Phylogenetic analysis has demonstrated\nthat EBLV-1 and EBLV-2 form relatively homogenous clus-\nters, but each may be additionally subdivided into two lin-\neages, “a” and “b” (Amengual et al., 1997). Australian bat\nlyssavirus (ABLV; GT 7) was discovered in 1996. Initially it\nwas isolated from pteropid bats, and two humans that died\nof rabies following bat exposure. Later ABLV was identiﬁed\nfrom ﬁve different bat species with at least two separate lin-\neages originating from frugivorous and insectivorous species\n(Fraser et al., 1996; Gould et al., 2002; Guyatt et al., 2003).\nClearly, most non-rabies lyssaviruses are associated with\nbats. The ecology of MOKV has not been sufﬁciently stud-\nied, so it is not possible to determine the principal host or\nsuggest inferences regarding its epidemiology. Shortly after\nthe discovery of LBV and MOKV, Africa was considered\nas the probable birth place of the Lyssavirus genus, and the\nChiroptera were suggested as the ﬁrst affected order on the\nroute of adaptation from plant and arthropod rhabdoviruses to\nmammalian hosts (Shope, 1982). Furthermore, since EBLV-1\nhad been demonstrated to be related to DUVV, an introduc-\ntion of the virus from Africa to Europe with migrating bats\nwas hypothesised (Schneider et al., 1985; Amengual et al.,\n1997; Serra-Cobo et al., 2002).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nVery limited information on bat lyssaviruses is available\nfor Asia. Only a few records reported presumable rabies virus\nisolates of Chiropteran origin in India (Pal et al., 1980) and\nThailand (Smith et al., 1968), but these have never been con-\nﬁrmed by further identiﬁcation or other observations. The\npresence of antibodies to ABLV was demonstrated in sera\nfrom bats in the Philippines, however, no isolates were ob-\ntained (Arguin et al., 2002).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nTwo lyssaviruses have been isolated from bats in Central\nAsia. Aravan virus was isolated in southern Kyrgyzstan in\n1991 (Kuzmin et al., 1992). Nucleotide sequencing of the\nentire nucleoprotein (N) gene has provided an indication that\nAravan virus might be regarded as a new lyssavirus genotype\nwith distant relatedness to GT 4 and GT 5 (Arai et al., 2003).\nA second example, Khujand virus, was isolated in Northern\nTajikistan during 2001 (Kuzmin et al., 2002). Further phylo-\ngenetic analysis of the N, phosphoprotein (P) and glycopro-\ntein (G) genes of these viruses suggests that Khujand virus is\nmost related to genotype 6, while Aravan virus is related to\nKhujand virus, and demonstrated moderate similarity to GTs\n4–6 (Kuzmin et al., 2003).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nDuring 2002, two new lyssaviruses have been isolated\nfrom Eurasian bats. According to preliminary identiﬁcation\nwith anti-nucleocapsid monoclonal antibodies and compar-\nison of limited N gene sequences, both may be considered\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nas new genotypes of the Lyssavirus genus (Botvinkin et al.,\n2003). These viruses have been named West Caucasian bat\nvirus (WCBV) and Irkut virus.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nIn this study we describe the genetic properties of Irkut\nvirus and WCBV, determine their phylogenetic relationships\naccording to the entire N, P and G gene sequences, and dis-\ncuss quantitative approaches for lyssavirus genotype deﬁni-\ntion considering the bat lyssaviruses discovered during recent\nyears.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.2**\n2. Materials and methods\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n2.1. Virus isolates\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.6**r== 0.0**\nThe isolation of Irkut virus and WCBV has been described\nearlier (Botvinkin et al., 2003). Brieﬂy, Irkut virus was iso-\nlated in the town of Irkutsk (East Siberia, Baikal lake re-\ngion) from the brain of a greater tube-nosed bat (Murina\nleucogaster) captured in September, 2002. WCBV was iso-\nlated from the brain of a common bent-winged bat (Min-\niopterus schreibersi) in Krasnodar region in July 2002. Both\nviruses were isolated by intracerebral mouse inoculation\n(Koprowski, 1996). In mice, both viruses caused fatal en-\ncephalitis, and typical cytoplasmic inclusions were detected\nin mouse brain impressions using either polyclonal or mon-\noclonal anti-rabies virus antibodies.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\nFor this study both viruses were passaged twice by intrac-\nerebral inoculation of mice, prior to genetic characterization.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\n2.2. RNA extraction, RT-PCR and nucleotide sequencing\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.0**\nTotal RNA was extracted from infected mouse brains, us-\ning TRIzolTM (Life Technologies) according to the manu-\nfacturer’s instructions. RT-PCR and direct sequencing were\nperformed as described earlier (Sacramento et al., 1991) with\nmodiﬁcations (Kuzmin et al., 2003). At least two RNA sam-\nples were processed with each primer pair, and both DNA\nstrands of PCR product were sequenced at least twice.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.2**\n2.3. Phylogenetic analysis\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.6**r== 0.0**\nSequence editing, alignment, consensus sequence genera-\ntion and translation to amino acid sequences were performed\nusing BioEdit software (Hall, 1999). Multiple alignment was\nperformed using the Clustal X package (Jeanmougin et al.,\n1998). Neighbor joining (NJ) analysis was performed us-\ning MEGA version 2.1 (Kumar et al., 2001). Maximum\nparsimony (MP) analysis and maximum likelihood (ML)\nanalysis were performed using PHYLIP version 3.6-alpha\n(Felsenstein, 1993). Nucleotide substitution models used\ntransition/transversion ratios varying from 2 to 4, with em-\npirical base frequencies, gamma distribution of rate variation\namong sites, and a Hidden Markov Model for inferring dif-\nferent rates of evolution at different sites. Bootstrap values\nwere determined using 1000 replicates for the NJ and MP\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.6**\nTable 1\nThe lyssavirus sequences used in the present study\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.3**r== 0.7**\nCountry\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.4**r== 0.5**\nSpecies isolated from\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.4**\nVirus number\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.3**\nReference\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.1**\nGenbank accession no.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.8**\nYear\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.2**r== 0.8**\nGT1\n**BLOCK**fs== 8.0**b== 0.5**t== 0.2**l== 0.2**r== 0.8**\n?\n\n\n\nLaboratory\nstrain\nLaboratory\nstrain\nThe same\nThe same\n\n\n\n\n\n\n?\n?\n?\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n?\n?\n?\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nGT2\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n1990–1991\n\n?\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nGT3\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.2**r== 0.8**\n1990–1991\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nGT4\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nGT5\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\nUSA\nCanada\nRussia\nBurkina Fasso\nFrench Guyana\nSouth Africa\nIran\nYugoslavia\nEstonia\n**BLOCK**fs== 8.0**b== 0.5**t== 0.3**l== 0.3**r== 0.7**\nUSA\nCanada\nSri-Lanka\nCanada\nNepal\nIsrael\nUSA\nCanada\nCanada\nChina\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nUSA\nAlgeria\nThailand\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\nNigeria\nEthiopia\nNigeria\nSouth Africa\nCentral Africa\nRepublic\nNigeria\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.3**r== 0.7**\nZimbabwe\nZimbabwe\nEthiopia\nSouth Africa\nSouth Africa\nNigeria\nZimbabwe\nSouth Africa\nSouth Africa\nSouth Africa\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nSouth Africa\nSouth Africa\nSouth africa\nSouth Africa\nSouth Africa\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nPA R89\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nRaccoon\nLasionycteris noctivagans ML5\nArctic fox\nDog\nDog\nViverridae spp.\nDog\nWolf\nRaccoon dog\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n9141RUS\n88636HAV\n9001GUY\n?\n8681IRA\n8653YOU\n9339EST\nSAD B19\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.2**\nN gene\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.1**\nP gene\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.0**\nG gene\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.6**r== 0.2**\nNadin-Davis et al. (1996)\nNadin-Davis et al. (2001)\nKissi et al. (1995)\nKissi et al. (1995)\nKissi et al. (1995)\nJacobs et al. (unpublished)\nKissi et al. (1995)\nBourhy et al. (1999)\nBourhy et al. (1999)\nConzelmann et al. (1990)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\nU27221\nAF351834\nU22656\nU22486\nU22478\nAF467949\nU22482\nU22482\nU42707\nM31046\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nAF467950\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nM31046\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.0**\nM31046\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nCVS\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nTordo et al. (1986)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nD42112\n**BLOCK**fs== 8.0**b== 0.5**t== 0.3**l== 0.4**r== 0.5**\nRaccoon\nLasionycteris noctivagans\nDog\nArctic fox\nDog\nFox\nLasionycteris noctivagans\nArctic fox\nSkunk\nDog/mouse\nbrain/BHK cell\nculture\nRaccoon\nDog\nDog\n**BLOCK**fs== 8.0**b== 0.5**t== 0.3**l== 0.6**r== 0.2**\nLarson and Wunner (1990)\nMorimoto et al. (1998)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nMorimoto et al. (1996)\nNadin-Davis et al. (1999)\nNadin-Davis et al. (1999)\nTang et al. (2000)\n**BLOCK**fs== 8.0**b== 0.5**t== 0.3**l== 0.5**r== 0.4**\nFL.RAC\n4398.SHB\nV118.DG\nARC5.RFX\nV124.DG\nV660.FX\nSHBRV\n91RABN1035\n92RBGL0867\nCTN\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nNY516\n?\n?\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nNadin-Davis et al. (1999)\nBenmansour et al. (1992)\nMinamoto N. (unpublished)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\nEidolon helvum\nDog\nEidolon helvum\nEpomophorus wahlbergii\nMicropteropus pussilus\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\n8619NGA\nEthlag\nv006\nV267\nLagCAR\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.6**r== 0.2**\nKissi et al. (1995)\nThis paper\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nBadrane et al. (2001)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nEidolon helvum\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nLagNGA\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nBadrane et al. (2001)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.4**r== 0.6**\nCat\nCat\nCat\nCat\nCat\nCrocidura sp.\nCat\nCat\nCat\nCat\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nHuman\nHuman\nMiniopterus sp.\nHuman\nMiniopterus sp.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.5**r== 0.4**\n?\n?\nEthmok\nV635\nV552\nV020\nV628\nV547\nV241\nV550\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n86132AS\nV008\nV268\nDuvSAF1\nDuvSAF2\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.6**r== 0.2**\nKissi et al. (1995)\nLe Mercier et al. (1997)\nThis paper\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nAmengual et al. (1997)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nBadrane et al. (2001)\nBadrane et al. (2001)\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nU22842\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.8**r== 0.2**\nU22843\nY09762\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nU22848\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nX55727\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\nAF369294\nAF369345\nAF369321\nAF369270\nAF369319\nAF369280\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\nAF049114\nAF049119\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.9**r== 0.1**\nAF369378\nAF369376\nAF049116\nAF369377\nAF369374\nAF049118\nAF369375\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\nAF049115\nAF049120\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.0**\nAF042823\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.9**r== 0.0**\nU52946\nU11736\nAF344307\nAY009100\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.0**\nU27214\nM81058\nAB052666\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.9**r== 0.0**\nAF298149\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.9**r== 0.0**\nAF298148\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.9**r== 0.0**\nY09762\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.9**r== 0.0**\nAF298146\nAF298147\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.9**r== 0.0**\nAF298143\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\nFrance\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\nEptesicus serotinus\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\n8918FRA\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.2**\nKissi et al. (1995); Badrane\net al. (2001)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.8**r== 0.2**\nU22845\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\nTable 1 (Continued )\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.8**\nYear\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.7**\nCountry\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.3**r== 0.6**\nSpecies isolated from\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.5**\nVirus number\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.4**\nReference\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.1**\nGenbank accession no.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.8**\nPoland\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.3**r== 0.6**\nEptesicus serotinus\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.5**\n8615POL\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nDenmark\nHolland\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nEptesicus serotinus\nEptesicus serotinus\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nV002\nV023\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nGT6\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nFinland\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nHuman\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n9007FIN\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nHolland\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nMyotis dasycneme\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n9018HOL\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\nSwitzerland\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\nMyotis daubentonii\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\nV286\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\nSaccolaimus ﬂaviventris\nPteropus sp.\nHuman\nPteropus sp.\nPteropus sp.\nPteropus alecto\nPteropus alecto\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nMyotis blythi\nMyotis daubentoni\nMurina leucogaster\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nAF081020\nAF006497\nV481\nV478\nV474\n735-34\n632-37\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nAravan\nKhujand\nIrkut\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nGT7\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\nAustralia\nAustralia\nAustralia\nAustralia\nAustralia\nAustralia\nAustralia\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nUnclassiﬁed\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\nKyrgizstan\nTajikistan\nRussia\n(Irkutsk\nprovince)\nRussia\n(Krasnodar\nregion)\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.2**\nN gene\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.2**\nU22844\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.1**\nP gene\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.0**\nG gene\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.0**\nAF298142\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.6**r== 0.3**\nKissi et al. (1995); Badrane\net al. (2001)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\nKissi et al. (1995); Badrane\net al. (2001)\nKissi et al. (1995); Badrane\net al. (2001)\nNadin-Davis et al. (2002)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nU22846\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nU22847\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nAF081020\nAF006497\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\nGould et al. (2002)\nGould et al. (1998)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nNadin-Davis et al. (2002)\nGuyatt et al. (2003)\nGuyatt et al. (2003)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nAF049113\nAF049117\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.8**r== 0.1**\nAF049121\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\nAF369373\nAF369371\nAF369369\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.0**\nAF298144\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.0**\nAF298145\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.0**\nAF081020\nAF006497\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.9**r== 0.0**\nAF426308\nAF426311\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nKuzmin et al. (2003)\nKuzmin et al. (2003)\nThis paper\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nAY262023\nAY262024\nAY333112\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nAY262023\nAY262024\nAY333112\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.9**r== 0.0**\nAY262023\nAY262024\nAY333112\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nMiniopterus schreibersi\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nWest Caucasian\nbat virus\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nThis paper\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nAY333113\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nAY333113\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.9**r== 0.0**\nAY333113\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nmethods and by using 100 replicates for the ML method.\nThe NJPLOT program from Clustal X and the TreeExplorer\nmodule of MEGA were used to obtain the graphic output.\nVesicular stomatitis virus (Indiana 1 serotype; GenBank ac-\ncession no. AF473864) was used as the outgroup. The other\nsequences used for analysis are presented in Table 1.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\n2.4. Positive selection analysis\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nVarious models of codon substitution (Yang et al., 2000)\nimplemented in the CODEML program of the PAML pack-\nage (Yang, 1997), were applied with varying constraints on\nthe values of synonymous (dS) and nonsynonymous (dN) sub-\nstitution rates, and their ratio (ω). Models allowing for pos-\nitive selection (that is, ω > 1) are nested with models that\ndo not allow for positive selection. This allows the signif-\nicance of the ﬁt for positive selection models to be tested\nusing the likelihood ratio test. Positive selection is inferred if\nthe positive selection model has a signiﬁcantly higher likeli-\nhood than the null model and a value of ω > 1 is estimated. If\nevidence of positive selection is suggested, Bayesian meth-\nods are used to identify which individual codons fall into the\nω > 1 class. Sequence alignments were trimmed to include\nonly non-stop codons. Identical sequences were removed.\nFor each alignment, a ML tree was generated using PAUP*\n4.0, beta 10 (Swofford, 2000) by a heuristic search incorpo-\nrating tree bisection-recombination (TBR) branch swapping.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.0**\nThis tree was then used for positive selection analysis. In\neach case, the model of nucleotide substitution was selected\nusing Modeltest (Posada and Crandall, 1998). Model testing\nusing CODEML was performed as the methods of Woelk et\nal. (2002).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n3. Results\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n3.1. N gene\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.6**r== 0.0**\nThe coding region of the entire N gene consisted of 1353\nnucleotides (nt) (451 deduced amino acids) for Irkut virus\nand 1350 nt (450 deduced amino acids) for the WCBV. Pu-\ntative phosphorylation site mapping to serine at position 389\n(Dietzschold et al., 1987) was conserved. Irkut virus shared\nthe sequence of the NIII epitope (aa 313–337) with GTs\n4 and 5 (-QSC- at positions 331–333), while the WCBV\nshared this epitope with GT 2 and 3 (-QTC- in the same\npositions). With other described T- and B-cell epitopes (Ertl\net al., 1991; Fu et al., 1994; da Cruz et al., 2002), Irkut virus\ndemonstrated particular similarity to GTs 4 and 5 and Aravan\nvirus, while the WCBV demonstrated signiﬁcant diversity\ncompared to all other lyssavirus representatives, except site\n3D (aa 22–35), which was conserved amongst all sequences.\nFor both viruses the N gene terminated with the sequence 5-\nGA7-3 followed by -CT- for Irkut and by -CTAC- for WCBV\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.3**\nTable 2\nIdentity of the N gene and deduced nucleoprotein sequences of lyssaviruses presented in Fig. 2\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.7**\nGenotypes GT 1\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.3**r== 0.7**\nGT 2\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.4**r== 0.6**\nGT 3\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.4**r== 0.5**\nGT 4\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.5**\nGT 5\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.4**\nGT 6\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.3**\nGT 7\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.2**\nAravan\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.1**\nKhujand\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.1**\nIrkut\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.0**\nWCBV\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.2**r== 0.8**\nGT 1\nGT 2\nGT 3\nGT 4\nGT 5\nGT 6\nGT 7\nAravan\nKhujand\nIrkut\nWCBV\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.3**r== 0.7**\na\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.2**r== 0.7**\n93.7−99.9\n82.3−99.4\n72.5–73.9\n69.4–73.8\n70.1–73.4\n70.4–74.9\n70.9–74.1\n72.4–78.0\n73.2–76.2\n75.3–76.5\n73.1–75.2\n72.3–73.0\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.3**r== 0.6**\n81.1–85.1\n94.4\n82.8\n76.1–78.5\n73.4–75.2\n74.0–74.5\n71.9–73.5\n72.5–75.1\n74.2–75.1\n71.9–73.6\n74.7–75.1\n74.1–74.3\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.4**r== 0.6**\n76.4–82.4\n87.1–90.00\n94.0−98.4\n86.1−98.7\n71.5–73.6\n69.8–73.0\n69.0–72.2\n69.7–73.1\n72.3–74.7\n68.2–70.9\n71.1–74.2\n70.8–72.9\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.4**r== 0.5**\n87.3–88.4\n85.7–87.3\n80.6–84.6\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n78.3–79.8\n72.4–73.6\n74.2–75.7\n78.1\n76.0\n78.0\n73.9\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.5**r== 0.4**\n87.1–88.4\n83.1–86.0\n78.8–83.3\n92.7–93.3\n98.6\n96.5\n74.5–76.9\n72.5–75.6\n77.8–78.0\n77.0–77.3\n78.2–78.6\n72.1–72.3\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.6**r== 0.4**\n86.2–87.8\n79.1–81.5\n76.0–79.7\n85.8–86.2\n86.4–88.0\n97.8\n95.9\n72.1–76.0\n76.6–77.1\n79.0\n76.6–76.9\n72.1\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.7**r== 0.3**\n89.1–92.5\n81.3–84.8\n78.8–84.0\n88.9–90.0\n88.2–89.8\n86.4–87.8\n95.6\n84.1\n76.5\n75.8–78.5\n75.2–75.8\n73.0\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.7**r== 0.2**\n88.9–90.1\n84.6–87.7\n80.8–85.3\n91.6\n91.6–92.0\n88.2–88.9\n91.1–92.0\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.7**r== 0.2**\n78.8\n76.2\n73.1\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.8**r== 0.1**\n89.6–90.8\n81.3–84.6\n78.4–82.2\n89.3\n89.1–90.2\n90.2–90.4\n90.4–92.2\n92.7\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n76.1\n70.5\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.9**r== 0.1**\n85.7–87.5\n84.2–85.7\n80.4–84.6\n90.4\n91.7–92.6\n86.2\n87.7\n90.6\n88.0\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n72.1\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.9**r== 0.0**\n80.5–81.2\n82.4–83.1\n78.4–82.4\n84.6\n81.7–82.4\n80.2\n81.3–82.6\n83.5\n81.1\n83.1\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.0**\na Upper triangle and above the dash – amino acid identity (shaded); lower triangle and below the dash – nucleotide identity. When only one sequence of a\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.5**\ncertain genotype was available for comparison, there is no value.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nas was earlier described for other lyssaviruses (Tordo et al.,\n1986).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.2**r== 0.4**\nNucleotide and amino acid identities for the N gene are\nshown in Table 2. The maximum intrinsic diversity for the\ncomplete N gene was found in GT 1: 82.3% and 93.7% iden-\ntity for the nucleotide and amino acid sequences, respectively.\nThese data match earlier estimations based on N gene se-\nquences (Bourhy et al., 1993). The maximum amount of ho-\nmology between different GTs was shared for GTs 4 and\n5 (up to 79.8 and 93.3% for nucleotide and amino acid se-\nquences, respectively). Irkut virus was most related to GT\n5 (78.6% of nucleotide and 92.6% of amino acid identity).\nWCBV remained distant from all other lyssaviruses how-\never, most divergent representative of the genus according to\nN gene, was MOKV.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.4**\nThe distribution of frequencies of pairwise identities be-\ntween the entire N gene nucleotide sequences (Fig. 1a) clearly\ndistinguish intragenotype identity (82.3% and more) from\nintergenotype identity (79.8% and less). The amino acid se-\nquences of the N gene provide limited resolution for geno-\ntyping (Fig. 1b), but there remains no overlap between virus\ngroups that were deﬁned according to nucleotide sequence\nidentities.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.4**\nA comparison of WCBV sequence identity with that of\nthe two lyssavirus phylogroups described earlier (Badrane et\nal., 2001) (Table 3) show that this virus cannot be considered\nas a member of either of these phylogroups.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nRegardless of method, results of the phylogenetic analy-\nsis demonstrated the same tree topology for all main clusters,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.6**r== 0.0**\nexcept Aravan virus (ML and MP trees not shown). The posi-\ntion of Aravan virus was unstable due to its equally moderate\nhomology with Khujand virus and with the clade joining GTs\n4 and 5 (Kuzmin et al., 2003). The same topology was ob-\ntained for the 450 deduced amino acids (Fig. 2). Irkut virus\nwas positioned as a member of the cluster joining GTs 4 and 5\n(bootstrap support of 83% for nucleotides and 90% for amino\nacids). WCBV was connected to the cluster of GTs 2 and 3.\nHowever, although the nucleotide tree joined these genotypes\nwith bootstrap values of 87% (ML) to 92% (NJ), the amino\nacid tree did not conﬁdently support this relationship: 42%\n(NJ) to 60% (MP) bootstrap values.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n3.2. P gene\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.6**r== 0.0**\nThe P gene transcriptional start signal for Irkut virus and\nWCBV was similar to other lyssaviruses: -AACACCYCT-.\nThe coding region of the P gene of Irkut virus consisted of\n894 nt and coded for 298 amino acids; the same length found\nfor the P gene of GTs 4 and 5. The coding region of the P\ngene of WCBV consisted of 891 nt and coded 297 amino\nacids, like most representatives of GT 1, 6 and 7 (Nadin-\nDavis et al., 2002). Methionine at positions 53 and 83 of the\nphosphoprotein was conserved in Irkut virus, but in WCBV\nit was substituted at these positions by valine and tyrosine,\nrespectively. The WCBV had a double proline at positions\n134–135, 137–138 and an additional proline at position 140.\nThe amino acid insertion at 140 is novel to all comparable\nlyssavirus sequences and produced the ﬁrst gap in the align-\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.2**r== 0.0**\nTable 3\nThe N and G ectodomain nucleotide and deduced amino acid sequence identities (%) between lyssaviruses of earlier determined phylogroups (Badrane et al.,\n2001) and WCBV\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nCompared virus groups\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.7**\nWithin phylogroup 1a\nWithin phylogroup 2\nBetween phylogrops 1 and 2\nBetween WCBV and phylogroup 1\nBetween WCBV and phylogroup 2\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\nN nucleotides\n79.2 ± 0.8 b\n83.5 ± 4.4\n72.2 ± 0.3\n72.5 ± 0.3\n72.8 ± 1.0\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nN amino acids\n90.9 ± 0.5\n91.8 ± 2.4\n81.5 ± 0.4\n81.4 ± 0.5\n81.5 ± 1.4\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\nG ectodomain nucleotides\n75.5 ± 0.8\n73.9 ± 4.1\n62.8 ± 0.2\n58.3 ± 0.4\n60.4 ± 0.6\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.9**r== 0.0**\nG ectodomain amino acids\n83.3 ± 0.8\n81.8 ± 4.8\n63.1 ± 0.3\n54.8 ± 0.3\n57.6 ± 0.7\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\na Phylogroup 1 includes genotypes 1, 4-7, Aravan, Khujand and Irkut viruses; phylogroup 2 includes genotypes 2 and 3.\nb Mean and 95% conﬁdence limit.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.0**\nFig. 1. Distribution of frequences of pairwise identities between the entire N gene sequences of lyssaviruses used in this study (a) nucleotides and (b) amino\nacids.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.0**\nFig. 2. Phylogenetic trees based on the entire N gene of the Lyssavirus genus (a) alignment of 1353 nucleotides, (b) alignment of 451 amino acids; pairwise\ndeletion applied for missed values) obtained by neighbor-joining method. GenBank accession numbers are given when virus names were unavailable. Bootstrap\nvalues are presented for key nodes and branch lengths are drawn to scale.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.4**\nments. Further gaps occurred in the variable region 471–573\n(amino acid positions 159–191) of the P. The critical posi-\ntions of the binding site for the cytoplasmic light chain of\ndynein, LC8 (Poisson et al., 2001), were conserved in Irkut\nvirus, but WCBV retained only D144 and Q148 (Fig. 3). A\nparticularly notable substitution in this site was A149 instead\nof T149, which is present in all other compared sequences.\nAnother stabile motif thought to be an important component\nof the C-terminal nucleoprotein binding domain of the phos-\nphoprotein, -FSKKYKF- at positions 209–215 (Jacob et al.,\n2001; Nadin-Davis et al., 2002), was conserved in Irkut virus,\nand had only one initial substitution (F to Y) in the WCBV\nphosphoprotein.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.4**\nNucleotide and amino acid identities for the P gene are\nshown in Table 4. Minimum identity of P gene sequences\nof GT 1 isolates was 75.0 and 80.0% for the nucleotide and\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\namino acid sequences, respectively. The diversity was even\nmore signiﬁcant between two compared sequences of GT 2:\n74.0 and 79.3% for the nucleotide and amino acid sequences,\nrespectively. Irkut virus demonstrated maximum relatedness\nto GT 5 (71.4 and 71.1% for nucleotide and amino acid se-\nquences, respectively). WCBV was equally different from all\nother GTs.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.0**\nThe distribution of frequencies of pairwise identities be-\ntween the P gene sequences revealed three frequency groups\n(not shown). One of them consisted of identities between the\nmost related sequences (>72% for nucleotide identity, and\n>78% for amino acid identity). The second group consisted\nof identities between less related sequences, 60–71% of nu-\ncleotide and 56–75% of amino acid identity. The third group\nconsisted of identities between most distant sequence pairs,\n55% and less of nucleotide identity, and 49% and less of\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.0**\nFig. 3. Partial alignment of lyssavirus phosphoproteins. Amino acid positions are shown above the alignment. LC8 site (aa 144–148) and C-terminal nucleo-\nprotein binding domain (aa 209–215) are framed.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\namino acid identity. Aravan and Khujand viruses belonged\nto the ﬁrst group since their identity to each other (74.6%\nfor nucleotides and 80.0% for amino acids) and identity of\nKhujand virus to GT 6 (74.1% for nucleotides and 78.7% for\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\namino acids) were more signiﬁcant than the intrinsic identity\nof GT 2. Hence, this separation was different from the sepa-\nration provided by N gene identities where all these viruses\nwere recognized as representatives of different genotypes.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nTable 4\nIdentity of the P gene and deduced phosphoprotein sequences of lyssaviruses presented in Fig. 4\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nGenotypes GT 1\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nGT 2\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nGT 3\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nGT 4\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nGT 5\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nGT 6\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nGT 7\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nAravan\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nKhujand\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nIrkut\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.9**r== 0.0**\nWCBV\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\nGT 1\nGT 2\nGT 3\nGT 4\nGT 5\nGT 6\nGT 7\nAravan\nKhujand\nIrkut\nWCBV\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\na\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.2**r== 0.7**\n80.0−97.0\n75.0−98.4\n51.5–54.0\n49.7–53.7\n58.7–63.1\n60.4–63.0\n64.5–65.6\n66.5–70.2\n63.3–65.9\n64.8–67.6\n61.1–65.1\n50.3–52.0\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n44.0–46.3\n79.3\n74.0\n61.4–63.6\n53.0\n54.6–55.2\n52.5\n51.7–54.3\n50.7–52.0\n51.6–52.6\n53.1–54.5\n51.2\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n45.3–47.0\n56.3–59.6\n81.1−99.3\n79.4−99.3\n49.6–51.0\n50.7–52.0\n51.7–52.9\n50.6–52.9\n50.0–52.3\n52.3–54.1\n52.8–54.4\n50.3–52.1\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\n54.0–58.8\n43.9–44.5\n42.2–43.8\n97.3\n97.9\n68.3–69.1\n62.9–63.8\n59.8–62.2\n63.3–64.2\n62.8–63.2\n67.2–67.3\n51.7–52.0\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\n58.1–64.4\n44.8–46.1\n43.8–45.8\n68.3–70.9\n98.3\n98.4\n66.0\n63.9–64.5\n68.4–69.0\n66.2\n71.4\n49.1–49.7\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n64.9–68.5\n45.3–46.3\n45.8–47.1\n57.1–57.8\n66.0\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n65.9–66.7\n72.5\n74.1\n66.4\n52.6\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n64.5–74.3\n43.4–44.6\n44.2–47.5\n55.5–56.5\n59.8–61.1\n67.5–69.1\n82.2−98.6\n79.7−99.1\n67.2–67.7\n67.3\n64.1–65.2\n49.5–50.2\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\n63.0–66.6\n43.5–43.9\n44.5–45.5\n60.2–61.2\n69.3–70.3\n78.4\n66.0–67.3\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n74.6\n65.5\n51.2\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.8**r== 0.2**\n65.1–69.3\n44.7–45.6\n45.8–47.5\n60.3–61.0\n66.2–66.8\n78.7\n67.1–69.0\n80.0\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n66.5\n52.4\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.9**r== 0.1**\n60.9–64.7\n44.9–45.9\n46.2–48.5\n66.1–66.7\n70.8–71.1\n66.7\n63.4–64.4\n67.1\n68.4\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n51.9\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.9**r== 0.0**\n41.3–44.3\n44.4–44.7\n42.7–45.7\n42.3–43.0\n44.0\n43.7\n42.0–44.7\n43.3\n42.0\n43.7\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.2**r== 0.0**\na Upper triangle and above the dash – amino acid identity (shaded); lower triangle and below the dash – nucleotide identity. When only one sequence of a\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\ncertain genotype was available for comparison, there is no value.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.0**\nFig. 4. Phylogenetic trees based on the entire P gene of the Lyssavirus genus (a) alignment of 915 nucleotides, and (b) alignment of 305 amino acids; pairwise\ndeletion applied for missed values) obtained by neighbor-joining method. GenBank accession numbers are given when virus names were unavailable. Bootstrap\nvalues are presented for key nodes and branch lengths are drawn to scale.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nIn all phylogenetic constructions based on the P gene, Irkut\nvirus was joined to the cluster of GTs 4 and 5 (Fig. 4). WCBV\ncould not be joined to any particular group and was placed\nexternally to all other lyssaviruses.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n3.3. G gene\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.2**r== 0.4**\nThe G gene of both Irkut and WCBV began with the signal\npeptide of 19 amino acids common to all lyssaviruses. The en-\ntire G gene of Irkut virus coded for 505 amino acids, similar to\nthe G gene of GTs 1, 5 and 6. The G of WCBV coded for 506\namino acids. This is different from all other lyssavirus rep-\nresentatives (Wunner, 2002). The ectodomain of both Irkut\nvirus and WCBV consisted of 439 amino acids, and did not\nproduce any gaps in an alignment with other lyssaviruses,\nas was also true for the cytoplasmic domain: all alignment\ngaps were produced along the endodomain only. Along the\nectodomain, the recognized antigenic sites were not strictly\nconserved amongst the different GTs. Only C35 and K198\n(antigenic site II), W335 (antigenic site III) and -KG-342-343\n(antigenic site a) were conserved in all compared sequences.\nK330 was conserved in Irkut ectodomain but replaced by\nI330 for WCBV. However, WCBV retained K331 similarly\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\nto some GT 2 representatives (Badrane et al., 2001). R333,\nwhich was reported as an important factor of pathogenic-\nity in a peripheral challenge (Dietzschold et al., 1983;\nBadrane et al., 2001), was conserved in Irkut ectodomain,\nbut not in WCBV, where it was substituted by E333. A poten-\ntial glycosylation site was found in the WCBV ectodomain\nat position 334 as it had been published earlier for GT 2\n(Badrane et al., 2001). A palmitoylation site (C461) was rep-\nresented in both viruses as well as a conserved endodomain\nregion -SWESYK- at positions 493–498 (Badrane et al.,\n2001).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.6**r== 0.0**\nNucleotide and amino acid identities for the G gene are\nshown in Table 5. The greatest amount of intragenotype di-\nversity was detected in GT 2: 76.2 and 83.9% identity for\nnucleotide amino acid sequences respectively. Maximum in-\ntergenotype homology was registered between GTs 5 and 6\n(72.9 and 80.7% for nucleotides and amino acids, respec-\ntively). Irkut virus demonstrated maximum identity with GT\n5: 73.5 and 80.0% of nucleotide and amino acid identity, re-\nspectively. WCBV was the most diverse member of the iden-\ntity table, demonstrating overall identity of only 53.3–57.6%\nat the nucleotide level and 48.1–53.1% identity at the amino\nacid level, to other GTs.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.4**\nTable 5\nIdentity of the G gene and deduced glycoprotein sequences of lyssaviruses presented in Fig. 6\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.8**\nGenotypes GT 1\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.3**r== 0.7**\nGT 2\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.4**r== 0.6**\nGT 3\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.4**r== 0.5**\nGT 4\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.5**\nGT 5\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.4**\nGT 6\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.3**\nGT 7\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.2**\nAravan\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.8**r== 0.2**\nKhujand\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.1**\nIrkut\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.9**r== 0.0**\nWCBV\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.1**r== 0.8**\nGT 1\nGT 2\nGT 3\nGT 4\nGT 5\nGT 6\nGT 7\nAravan\nKhujand\nIrkut\nWCBV\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.3**r== 0.7**\na\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.2**r== 0.7**\n84.3−98.4\n78.9−97.7\n57.6–59.2\n57.3–59.3\n64.3–65.1\n65.6–67.2\n68.4–70.6\n69.1–71.6\n67.9–71.1\n69.0–69.8\n64.7–66.7\n53.3–55.6\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.3**r== 0.7**\n56.2–57.8\n83.9\n76.2\n68.3–70.1\n57.3–58.4\n58.5–60.9\n60.1–60.9\n58.1–59.9\n59.5–60.1\n59.1–59.8\n60.1–61.2\n56.5–56.9\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.4**r== 0.6**\n57.4–58.9\n72.7–73.7\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n58.3\n59.8\n58.5–59.6\n58.5–59.6\n58.0\n58.4\n59.2\n57.6\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.4**r== 0.5**\n67.9–69.6\n54.9–56.2\n55.8\n97.7\n98.1\n71.8–72.4\n70.5–72.1\n66.8–68.4\n69.2–69.5\n70.7\n68.1–68.9\n55.5\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.5**r== 0.4**\n70.0–71.7\n56.8–58.0\n57.3\n78.4–78.8\n97.7\n95.4\n72.3–72.9\n68.3–69.8\n72.6\n71.5\n73.5\n55.2\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.6**r== 0.4**\n71.5–74.4\n55.1–56.4\n55.1–56.4\n77.0–79.0\n79.0–80.7\n96.5\n94.0\n70.9–72.6\n74.6\n77.6–78.2\n70.2–70.6\n55.3–56.9\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.6**r== 0.3**\n73.2–78.2\n57.4–58.0\n57.0–58.2\n71.1–73.6\n73.2–76.3\n75.5–79.9\n90.0−99.8\n82.8−99.8\n70.3–71.4\n73.1\n65.5–68.5\n54.9–55.2\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.7**r== 0.2**\n72.7–74.6\n57.5\n56.6\n76.7–77.4\n82.6–83.0\n83.8\n76.9–79.5\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n74.5\n69.7\n55.6\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.8**r== 0.2**\n73.7–77.4\n56.4–57.6\n56.6\n76.3–77.0\n79.0\n85.9–86.8\n78.2–82.4\n83.5\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n69.7\n55.3\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.9**r== 0.1**\n66.4–69.0\n58.7–59.1\n58.0\n72.4–73.7\n\n75.3–76.5\n71.4–73.3\n77.3\n76.2\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n55.6\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.9**r== 0.0**\n48.7–50.4\n52.3–53.1\n52.7\n48.1–48.6\n50.1\n49.1–50.0\n50.8–51.2\n50.6\n50.4\n50.2\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.0**\na Upper triangle and above the dash – amino acid identity (shaded); lower triangle and below the dash – nucleotide identity. When only one sequence of a\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\ncertain genotype was available for comparison, there is no value.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nThe distribution of frequencies of the pairwise identities\nbetween the G gene sequences, and particularly ectodomain\namino acid sequences (Fig. 5), demonstrated multiple over-\nlaps between Aravan and Khujand viruses and GTs 5–7,\nso that it was impossible to deﬁne a threshold between in-\ntragenotype and intergenotype identities using this part of\nthe genome.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nWhen phylogenetic analysis was performed (Fig. 6), Irkut\nvirus was joined to GT 5 by all methods used for either\nnucleotide or amino acid sequences. However, this place-\nment was not conﬁdently supported, with bootstrap values\nof only 43–69%. Nonetheless, the position of Irkut virus\ninside the cluster of GTs 4 and 5 was conﬁrmed. WCBV\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\ncould not be joined to any cluster and was positioned ex-\nternally to all lyssaviruses. An identical topology was ob-\ntained by different phylogenetic methods applied for the\nectodomain nucleotide and amino acid sequences (results not\nshown).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\n3.4. Positive selection analysis\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.0**\nAnalysis of N and G gene phylogenies revealed no signiﬁ-\ncant evidence of positive selection. For the N gene sequences,\nmodels allowing for positive selection were signiﬁcantly ac-\ncepted in two cases (p < 0.0001), however, no ω value greater\nthan 1 was estimated. Similarly, for the G gene sequences,\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.2**r== 0.0**\nFig. 5. Distribution of frequencies of pairwise identities between the ectodomain amino acid sequences of lyssaviruses used in the current study.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.2**r== 0.0**\nFig. 6. Phylogenetic trees based on the entire G gene of the Lyssavirus genus (a) alignment of 1599 nucleotides, and (b) alignment of 533 amino acids; pairwise\ndeletion applied for missed values) obtained by neighbor-joining method. Genbank accession numbers are given when virus names were unavailable. Bootstrap\nvalues are presented for key nodes and branch lengths are drawn to scale.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\none null model was rejected (p < 0.0001), but no codons were\nestimated to have a ω > 1.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.2**r== 0.4**\nFor P gene sequences, there was very limited evidence\nof positive selection. One null model (M1 which allows for\nneutral evolution where ( is either 1 or 0) was rejected signif-\nicantly (p < 0.0001) in favor of M2, which allows for a ω > 1.\nThe M2 model was the only one of the three positive selection\nmodels (together with M3 and M8; Woelk et al., 2002) which\nestimated a codon class with a ω < 1 (3.18). Consequently, the\npresence of ﬁve positively selected codons along the P gene\n(amino acid positions 61, 67, 151, 159 and 161 [where posi-\ntion 1 is the conserved ATG start codon]) determined by this\nmodel are highly speculative.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n4. Discussion\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.2**r== 0.4**\nDeﬁnition of quantitative criteria for inclusion of a new\ngenotype into the Lyssavirus genus became problematic af-\nter discovery of Aravan and Khujand viruses (Kuzmin et\nal., 2003). Criteria proposed earlier were quite acceptable\nfor GTs 1–7 to be determined independently on the gene\nused: intragenotype identities were greater than intergeno-\ntype identities, and clusters of certain genotype members\nwere supported by signiﬁcant bootstrap values (Bourhy et\nal., 1993; Tordo et al., 1993; Kissi et al., 1995; Badrane\net al., 2001; Johnson et al., 2002). Introducing the newly\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\ndiscovered lyssaviruses required a reassessement of the old\napproaches.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.0**\nIn this analysis, we obtained different quantitative separa-\ntions within the genus according to the pairwise identities of\nthe N and P gene. The G gene sequences, either complete or\npartial could not separate genotypes due to multiple overlaps.\nWe propose that it would be necessary to select one of the\ngenes for taxonomic purposes. But which one provides the\nbest subdivision?\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.6**r== 0.0**\nThe N gene provides the most clear and unequivocal sub-\ndivision of the lyssavirus groups (Fig. 1). It is commonly\nsequenced as part of molecular epidemiological studies, and\nhistorically is associated with earlier lyssavirus classiﬁcation\nbased on reactivity patterns with anti-nucleocapsid mono-\nclonal antibodies (Bourhy et al., 1993; Kissi et al., 1995).\nWe suggest this gene is the most appropriate for quantita-\ntive genotype deﬁnition, and the nucleotide sequence should\nbe preferred to the amino acid sequence due to a higher de-\ngree of resolution. The results of our study suggest that the\nthreshold of 80% nucleotide identity for sequences of the en-\ntire N gene (Kissi et al., 1995) still appears to be reasonable.\nMoreover, we would raise this threshold to 82%. According\nto this approach, both Irkut virus and WCBV, as well as the\nearlier described Aravan and Khujand viruses (Arai et al.,\n2003; Kuzmin et al., 2003), should be considered as different\nlyssavirus genotypes. Partial N gene sequences may be used\nfor classiﬁcation purposes (Amengual et al., 1997; Bourhy et\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nal., 1999), but their appropriateness must be assessed more\nthoroughly for recently discovered lyssaviruses.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nUse of some qualitative criteria for taxonomy purposes,\nsuch as geographic distribution and host species, is an alter-\nnative approach. Unfortunately, our present knowledge of the\nnatural history of most non-rabies lyssaviruses is too limited\nfor successful implementation of this suggestion.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.2**l== 0.1**r== 0.5**\nAll our phylogenetic constructions demonstrate that GTs\n4–6, together with Aravan, Khujand and Irkut viruses, present\na solid phylogroup of Old World bat lyssaviruses (Fig. 7). We\ncould not provide a straight statistical basis for their joining\ninto one phylogroup as it has been done for phylogroups 1\nand 2 (Badrane et al., 2001), due to divergence of marginal\nmembers (GTs 4 and 6). Nonethless, the relatedness of these\nviruses to each other is clearly apparent. This relatedness is\nseen with each gene, and explains why Aravan and Khujand\nviruses could not be separated from GT 6 according to the\nP gene sequence, and why multiple overlaps between mem-\nbers of this phylogroup occurred according to the G gene\nsequence. However, this is not a rationale to tighten some\nof these lineages into one genotype if we consider others\nas independent genotypes. All of these lineages are clearly\nseparated as independent genotypes according to the N gene\nsequences. Relatedness of these viruses cannot be solely at-\ntributed to convergent evolution as a result of host species\nadaptation: bat lyssaviruses of other GTs do not tend to be\nparticularly related to members of this phylogroup, except\nGT 7. As was shown earlier (Kuzmin et al., 2003) and con-\nﬁrmed in this study, GT 7 is more related to Khujand virus\nand GT 6 according to the G amino acid sequence, while ac-\ncording to the N and P sequences it is most related to the GT\n1. This particular homology of the G may be suggested as\nevidence of convergent or parallel evolution.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nNo signiﬁcant evidence of positive selection was found in\nthe compared sequences. Previously, evidence of purifying\nselection has been suggested as a result of genetic constraints\non the rabies virus genes analyzed (G and N) or a lack of ad-\nequate selection pressure (Badrane and Tordo, 2001; Holmes\net al., 2002). We do not consider the results of this study to\nbe unexpected, considering the vast number of independent\nevolutionary paths that have been pooled within the analy-\nsis, a problem that may inhibit tests for positive selection on a\ngenotype basis due to the compartmentalization that undoubt-\nedly governs lyssavirus epidemiology (e.g., Smith, 2002). To\ncorrectly assess the role of positive selection in lyssavirus\nevolution, analysis of phylogenetically distinct lineages may\nbe necessary.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nWCBV is the most divergent representative of the\nLyssavirus genus described to date. The limited relatedness\nof the N gene sequences of WCBV to GTs 2 and 3 may de-\nrive from parental homology, although there are no additional\ndata to support this assertion. The phylogenetic analysis de-\nscribed here raises interesting questions regarding the origin\nand evolution of the Lyssavirus genus. Do Old World bat\nlyssaviruses originate from Africa, as was proposed earlier\nfor the EBLVs (Shope, 1982; Amengual et al., 1997; Serra-\n**BLOCK**fs== 10.0**b== 0.6**t== 0.1**l== 0.6**r== 0.0**\nCobo et al., 2002)? They might, since all putative reservoir\nspecies appear to be broadly distributed, due in no small mea-\nsure to the power of ﬂight. However, only one member of this\ngroup, DUVV, is present in Africa. Other lineages are found\nin Eurasia. Is this observation evidence of independent evo-\nlution after an initial introduction of a progenitor virus from\nAfrica? DUVV is no more closely related to LBV and MOKV\nthan it is to EBLV-1, EBLV-2, Aravan, Khujand and Irkut\nviruses. On the other hand, MOKV, which has been isolated\nonly in Africa was demonstrated to be antigenically related\nto the African arthropod rhabdoviruses, Obodhiang and Ko-\ntonkan, hypothesized as examples for probable progenitors of\nthe lyssavirus genus (Shope, 1982). Has WCBV derived from\nAfrica as well? The principal host of this virus is unknown.\nIt was isolated from Miniopterus schreibersi, a broadly dis-\ntributed colonial species. This bat species does not perform\nlong-distance seasonal migrations, but it is quite mobile and,\naccording to tracking data, the population occupying caves\non both slopes of the Caucasian ridge is permanently inter-\nmixed (Gazaryan, 1999). Thus, more distant movements ap-\npear probable. Additional investigations should be carried out\nto determine genetic and antigenic relations between WCBV,\nLBV, MOKV, Obodhiang and Kotonkan viruses, together\nwith uncharacterized rhabdoviruses.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.4**l== 0.6**r== 0.0**\nAnother important question addresses pathogenicity of\nWCBV. As had been repeatedly demonstrated, R333 (or\nK333) in the glycoprotein ectodomain, presented in all\nphylogroup 1 lyssaviruses (except artiﬁcially modiﬁed\nlaboratory strains), play crucial role in their peripheral\npathogenicity (Dietzschold et al., 1983; Badrane et al., 2001;\nMebatsion, 2001). Pathogenic importance was also suggested\nfor the LC8 binding site of the phosphoprotein (Poisson et al.,\n2001; Mebatsion, 2001). The phylogroup 2 members (LBV\nand MOKV) have D333 in the glycoprotein, and the LC8\nbinding site of their phosphoprotein is different from that\nof the phylogroup 1 lyssaviruses. LBV and MOKV were\napathogenic for mice and dogs, and of low pathogenicity\nfor monkeys by the peripheral route (Tignor et al., 1973;\nDietzschold et al., 1983; Badrane et al., 2001). WCBV has\nE333 in the glycoprotein ectodomain and the LC8 binding\nsite of the phosphoprotein is different from either 1 or 2\nphylogroup lyssaviruses. In our experiments WCBV was ap-\nathogenic for 3-week-old mice by intramuscular and oral\nroutes even when 106 mouse intracerebral lethal doses were\ngiven. But syrian hamsters challenged intramuscularly with\nthe same virus dose developed typical rabies symptoms and\nall succumbed (100% mortality). Also, we should not forget\nthat WCBV and phylogroup 2 lyssaviruses are not labora-\ntory strains. They were isolated from a number of naturally\ninfected mammalian species (MOKV was also isolated from\nhumans), so there should be some adaptive mechanisms pro-\nviding their natural circulation.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.0**\nThe public health implications of newly discovered\nlyssaviruses require introspection. Although no human in-\nfection caused by viruses other then GTs 1, 5 and 6 have\nbeen reported from this region, such observations may be as\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.0**\nFig. 7. Unrooted phylogenetic trees based on the entire lyssavirus N gene nucleotide sequences (a) and glycoprotein ectodomain amino acid sequences (b).\nThe genotypes (GT) recognized earlier are enclosed in circles. Genbank accession numbers are given when virus names are unavailable. Bootstrap values are\npresented for key nodes and branch lengths are drawn to scale.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\na result of inadequate diagnostic tests or inappropriate epi-\ndemiological surveillance (Kuzmin et al., 2003). Moreover,\nconsidering bat mobility and the potential for spatial spread\nof infection, the application of the term “rabies-free” should\nbe used conservatively. For example, the Irkutsk region has\nbeen considered free of rabies for 35 years before the iso-\nlation of Irkut virus (Botvinkin et al., 2003). Similarly, the\nUnited Kingdom was believed to be “rabies-free” until 1996,\nwhen EBLV-2 was isolated from bats (Whitby et al., 1996),\nand a human case of this infection was registered in 2002\n(Fooks et al., 2002). Public health awareness should be raised,\nand attention paid to the protective efﬁcacy of commercially\navailable rabies vaccines, as well as immune globulin reac-\ntivity, with these non-rabies lyssaviruses. Anti-rabies biolog-\nicals provide incomplete protection of experimental animals\nagainst EBLVs, limited protection against LBV, and no pro-\ntection against MOKV (Fekadu et al., 1988; Lafon et al.,\n1988; Mebatsion et al., 1992; Bahlou et al., 1998; Jallet et\nal., 1999; Nel et al., 2003). We should expect the same lack\nof efﬁcacy against WCBV.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nAcknowledgements\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nWe are grateful to Drs. T. Mebatsion and J.H. Cox, (Fed-\neral Institute for Animal Viral Diseases, Tubingen, Germany)\nfor the N gene sequences of LBV and MOKV viruses orig-\ninating from Ethiopia. We thank J.S. Smith (CDC, Atlanta,\nGA, USA) for valuable discussions during the study. IVK\nwas supported by the Russian Foundation of Basic Research\n(grant 00-04-48004) and Association of Public Health Lab-\noratories, international fellowship program of 2002–2003.\nGJH was funded by an ASM/NCID post-doctoral fellowship.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nReferences\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nAmengual, B., Whitby, J.E., King, A., Serra Cobo, J., Bourhy, H., 1997.\nEvolution of European bat lyssaviruses. J. Gen. Virol. 78, 2319–2328.\nArai, Y.T., Kuzmin, I.V., Kameoka, Y., Botvinkin, A.D., 2003. New\nlyssavirus genotype from the lesser mouse-eared bat (Myotis blythi),\nKyrghyzstan. Emerg. Inf. Dis. 9, 333–337.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nArguin, P.M., Murray-Lillibridge, K., Miranda, M.E., Smith, J.S., Calaor,\nA.B., Rupprecht, C.E., 2002. Serologic evidence of Lyssavirus infec-\ntions among bats, the Philippines. Emerg. Inf. Dis. 8, 258–262.\nBadrane, H., Bahloul, C., Perrin, P., Tordo, N., 2001. Evidence of\ntwo Lyssavirus phylogroups with distinct pathogenecity and immuno-\ngenecity. J. Virol. 75, 3268–3276.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nBadrane, H., Tordo, N., 2001. Host switching in Lyssavirus history from\nthe Chiroptera to the Carnivora orders. J. Virol. 75, 8096–8104.\nBahlou, C., Jacob, Y., Tordo, N., Perrin, P., 1998. DNA-based immuniza-\ntion for exploring the enlargement of immunological cross-reactivity\nagainst the lyssaviruses. Vaccine 16, 417–425.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nBenmansour, A., Brahmi, M., Tuffereau, C., Coulon, P., Lafay, F., Fla-\nmand, A., 1992. Rapid sequence evolution of street rabies glycoprotein\nis related to the highly heterogeneous nature of the viral population.\nVirology 187, 33–45.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nBotvinkin, A.D., Poleschuk, E.M., Kuzmin, I.V., Borisova, T.I., Gazaryan,\nS.V., Yager, P., Rupprecht, C.E., 2003. Novel lyssavirus isolated from\nbats Russia. Emerg. Inf. Dis. 9, 1623–1625.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nBoulger, I.R., Porterﬁeld, J.S., 1958. Isolation of a virus from Nigerian\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nfruit bats. Trans. Roy. Soc. Trop. Med. Hyg. 52, 421–424.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nBourhy, H., Kissi, B., Audry, L., Smreczak, M., Sadkowska-Todys, M.,\nKulonen, K., Tordo, N., Zmudzinski, J.F., Holmes, E.C., 1999. Ecol-\nogy and evolution of rabies virus in Europe. J. Gen. Virol. 80,\n2545–2557.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nBourhy, H., Kissi, B., Tordo, N., 1993. Molecular diversity of the\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nLyssavirus genus. Virology 194, 70–81.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nConzelmann, K.K., Cox, J.H., Schneider, L.G., Thiel, H.J., 1990. Molecu-\nlar cloning and complete nucleotide sequence of the attenuated rabies\nvirus SAD B19. Virology 175, 485–499.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.6**r== 0.0**\nda Cruz, F.W., McBride, A.J.A., Conceic¸ ˜ao, F.R., Dale, J.W., McFadden,\nJ., Dellagostin, O.A., 2002. Expression of the B-cell and T-cell epi-\ntopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG\nand induction of an humoral response in mice. Vaccine 20, 731–\n736.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.0**\nDietzschold, B., Wunner, W.H., Wiktor, T.J., Lopes, A.D., Lafon, M.,\nSmith, C.L., Koprowski, H., 1983. Characterization of an antigenic\ndeterminant of the glycoprotein that correlates with pathogenicity of\nrabies virus. Proc. Natl. Acad. Sci. U.S.A. 80, 70–74.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\nDietzschold, B., Lafon, M., Wang, H., Otvos, L.J., Celis, E., Wunner,\nW.H., Koprowski, H., 1987. Localization and immunological charac-\nterization of antigenic domains of the rabies virus internal N and NS\nproteins. Virus Res. 8, 103–125.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.0**\nErtl, H., Dietzchold, B., Otvos, L., 1991. T helper cell epitope of rabies\nvirus nucleoprotein deﬁned by tri- and tetrapeptides. Eur. J. Immunol.\n21, 1–10.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.6**r== 0.0**\nFekadu, M., Shaddock, J.H., Sanderlin, D.W., Smith, J.S., 1988. Efﬁ-\ncacy of rabies vaccines against Duvenhage virus isolated from Euro-\npean house bats (Eptesicus serotinus), classic rabies and rabies-related\nviruses. Vaccine 6, 533–539.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\nFelsenstein, J., 1993. PHYLIP: Phylogenetic Inference Package (Version\n3.5c). Department of Genetics, University of Washington, Seattle, WA,\nUSA.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\nFoggin, C.M., 1988. Rabies and rabies-related viruses in Zimbabwe: his-\ntorical, virological and ecological aspects. Thesis, University of Zim-\nbabwe, Harare.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.6**r== 0.0**\nFooks, A.R., Finnegan, C., Johnson, N., Mansﬁeld, K., McElhinney, L.\nManser, P., 2002. Human case of EBL type 2 following exposure to\nbats in Angus, Scotland. Vet. Rec. 151, 679.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.0**\nFu, Z.F., Wunner, W.H., Dietzschold, B., 1994. Immunoprotection by ra-\nbies virus nucleoprotein. In: Rupprecht, C.E., Dietzschold, B., Ko-\nprowski, H. (Eds.), Lyssaviruses. Springer, Berlin, Germany, pp.\n207–218.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\nFraser, C.F., Hooper, P.T., Lunt, R.A., Gould, A.R., Glesson, L.J., Hyatt,\nA.D., Russell, G.M., Kattenbelt, J.A., 1996. Encephalitis caused by a\nLyssavirus in fruit bats in Australia. Emerg. Inf. Dis. 2, 327–331.\nGazaryan, S.V., 1999. New data on occurrence of Schreiber’s bats (Min-\niopterus schreibersi) in the Western Caucasus. Plecotus et al. 2, 88–93.\nGould, A.R., Kattenbelt, J.A., Gumley, S.G., Lunt, R.A., 2002. Char-\nacterization of an Australian bat lyssavirus variant isolated from an\ninsectivorous bat. Virus Res. 89, 1–28.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\nGuyatt, K.J., Twin, J., Davis, P., Holmes, E.C., Smith, G.A., Smith,\nI.L., Mackenzie, J.S., Young, P.L., 2003. A molecular epidemiological\nstudy of Australian bat lyssavirus. J. Gen. Virol. 84, 485–496.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.6**r== 0.0**\nHall, T.A., 1999. BioEdit: a user-friendly biological sequence alignment\neditor and analysis program for Windows 95/98/NT. Nucl. Acids.\nSymp. Ser. 41, 95–98.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.0**\nHolmes, E.C., Woelk, C.H., Kassis, R., Bourhy, H., 2002. Genetic con-\nstraints and the adaptive evolution of rabies virus in Nature. Virology\n292, 247–257.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nJacob, Y., Real, E., Tordo, N., 2001. Functional\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.0**\ninteraction map of\nlyssavirus phosphoprotein: identiﬁcation of the minimal transcription\ndomains. J. Virol. 75, 9613–9622.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.0**\nJallet, C., Jacob, Y., Bahloul, C., Drings, A., Desmezi`eres, E., Tordo,\nN., Perrin, P., 1999. Chimeric lyssavirus glycoproteins with increased\nimmunological potential. J. Virol. 73, 225–233.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.4**\nJeanmougin, F., Thompson, J.D., Gouy, M., Higgins, D.G., Gibson, T.J.,\n1998. Multiple sequence alignment with Clustal X. Trends. Biochem.\nSci. 23, 403–405.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.2**r== 0.4**\nJohnson, N., McElhinney, L.M., Smith, J., Lowings, P., Fooks, A.R., 2002.\nPhylogenetic comparison of the genus Lyssavirus using distal coding\nsequences of the glycoprotein and nucleoprotein genes. Arch. Virol.\n147, 2111–2123.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nKing, A., Crick, J., 1988. Rabies-related viruses. In: Campbell, J.B.,\nCharlton, K.M. (Eds.), Rabies. Kluwer Acad. Publisher, Boston, pp.\n178–199.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nKissi, B., Tordo, N., Bourhy, H., 1995. Genetic polymorphism in the\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nrabies virus nucleoprotein gene. Virology 209, 526–537.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.4**\nKoprowski, H., 1996. The mouse inoculation test. In: Meslin F.-X., Ka-\nplan, M.M., Koprowski, H. (Eds.), Laboratory Techniques in Rabies,\n4th ed. WHO, Geneva, pp. 80–86.\nKumar, S., Tamura, K., Jakobsen,\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nI.B., Nei, M., 2001. MEGA2:\nmolecular evolutionary genetics analysis software. Bioinformatics 17,\n1244–1245.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nKuzmin,\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nI.V., Orciari, L.A., Arai, Y.T., Smith, J.S., Hanlon, C.A.,\nKameoka, Y., Rupprecht, C.E., 2003. Bat lyssaviruses (Aravan and\nKhujand) from Central Asia: phylogenetic relationships according to\nN, P and G gene sequences. Virus Res. 97, 65–79.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.2**r== 0.4**\nKuzmin, I.V., Botvinkin, A.D., Khabilov, T.K., 2002. A lyssavirus was\nisolated from a whiskered bat in Northern Tajikistan. Plecotus et al.\n4, 75–81, in Russian.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nKuzmin, I.V., Botvinkin, A.D., Rybin, S.N., Bayaliev, A.B., 1992. A\nlyssavirus with unusual antigenic structure isolated from a bat in\nSouthern Kyrghyzstan. Voprosy Virusologii 5–6, 256–259 (in Rus-\nsian).\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.2**r== 0.4**\nLafon, M., Bourhy, H., Sureau, P., 1988. Immunity against the European\nbat rabies (Duvenhage) virus induced by rabies vaccines: an experi-\nmental study in mice. Vaccine 6, 362–368.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nLarson, J.K., Wunner, W.H., 1990. Nucleotide and deduced amino acid\nsequences of the nominal nonstructural phosphoprotein of the ERA,\nPM and CVS-11 strains of\nrabies virus. Nucl. Acids Res. 18,\n7172.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nLe Mercier, P., Jacob, Y., Tordo, N., 1997. The complete Mokola virus\ngenome sequence: structure of the RNA-dependent RNA polymerase.\nJ. Gen. Virol. 78, 1571–1576.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.2**r== 0.4**\nLumio, J., Hillbom, M., Roine, R., Ketonen, L., Haltia, M., Valle, M.,\nNeuvonen, E., Lahdeviktra, J., 1986. Human rabies of bat origin in\nEurope. Lancet 1, 378.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nMebatsion, T., Cox, J.H., Frost, J.W., 1992. Isolation and characteriza-\ntion of 115 street rabies virus isolates from Ethiopia by using mon-\noclonal antibodies: identiﬁcation of 2 isolates as Mokola and Lagos\nbat viruses. J. Inf. Dis. 166, 972–977.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.2**r== 0.4**\nMebatsion, T., 2001. Extensive attenuation of rabies virus by simul-\ntaneously modifying the dynein light chain binding site in the P\nprotein and replacing Arg333 in the G protein. J. Virol., 11496–\n11502.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nNadin-Davis, S.A., Sampath, M.I., Casey, G.A., Tinline, R.R., Wandeler,\nA.I., 1999. Phylogeographic patterns exhibited by Ontario rabies virus\nvariants. Epidemiol. Inf. 123, 325–336.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nNadin-Davis, S.A., Huang, W., Armstrong, J., Casey, G.A., Bahloul, C.,\nTordo, N., Wandeler, A.I., 2001. Antigenic and genetic divergence of\nrabies viruses from bat species indigenous to Canada. Virus Res. 74,\n139–156.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nNadin-Davis, S.A., Abdel-Malik, M., Armstrong, J., Wandeler, A., 2002.\nLyssavirus P gene characterization provides insight into the phylogeny\nof the genus and identiﬁes structural similarities and diversity within\nthe encoded phosphoprotein. Virology 298, 286–305.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.6**r== 0.0**\nNel, L., Jacobs, J., Jafta, J., Von Teichman, B., Bingham, J., 2000. New\ncases of Mokola virus infection in South Africa: a genotypic com-\nparison of Southern African virus isolates. Virus Genes 20, 103–\n106.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.0**\nNel, L.H., Niezgoda, M., Hanlon, C.A., Morril, P.A., Yager, P.A., Rup-\nprecht, C.E., 2003. A comparison of DNA vaccines for the rabies-\nrelated virus, Mokola. Vaccine 21, 2598–2606.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.0**\nPal, S.R., Arora, B.M., Chuttani, P.N., Broer, S., Choudhury, S., Joshi,\nR.M., 1980. Rabies virus infection of a ﬂying fox bat, Pteropus po-\nliocephalus, in Chandigarh, Northern India. Trop. Geogr. Med. 32,\n265–267.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\nPoisson, N., Real, E., Gaudin, Y., Vaney, M.C., King, S., Jacob, Y., Tordo,\nN., Blondel, D., 2001. Molecular basis for the interaction between\nrabies virus phosphoprotein P and the dynein light chain LC8: disso-\nciation of dynein-binding properties and transcriptional functionality\nof P. J. Gen. Virol. 82, 2691–2696.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.0**\nPosada, D., Crandall, K.A., 1998. MODELTEST: testing the model of\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\nDNA substitution. Bioinformatics 14, 817–818.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.6**r== 0.0**\nSacramento, D., Bourhy, H., Tordo, N., 1991. PCR technique as an al-\nternative method for diagnosis and molecular epidemiology of rabies\nvirus. Molec. Cell. Prob. 5, 229–240.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\nSchneider, L.G., Barnard, B.J.H., Schneider, H.P., 1985. Application of\nmonoclonal antibodies for epidemiological investigations and oral vac-\ncination studies. 1. African viruses. In: Kuwert, E., Merieux, C., Ko-\nprowski, H., Bogel, K. (Eds.), Rabies in the Tropics. Springer-Verlag,\nBerlin, Heidelberg, New York, Tokyo, pp. 47–59.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.6**r== 0.0**\nSelimov, M.A., Tatarov, A.G., Botvinkin, A.D., Klueva, E.V., Kulikova,\nL.G., Khismatullina, N.A., 1989. Rabies-related Yuli-virus; identiﬁ-\ncation with a panel of monoclonal antibodies. Acta Virol. 33, 542–\n545.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.0**\nSerra-Cobo, J., Amengual, B., Abellan, C., Bourhy, H., 2002. European\nbat lyssavirus infection in Spanish bat populations. Emerg. Inf. Dis.\n8, 414–420.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\nShope, R.E., Murphy, F.A., Harrison, A.K., Causey, O.R., Kemp, G.E.,\nSimpson, D.I.H., Moore, D., 1970. Two African viruses serologi-\ncally and morphologically related to rabies virus. J. Virol. 6, 690–\n692.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\nShope, R.E., 1982. Rabies-related viruses. Yale J. Biol. Med. 55, 271–\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n275.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nMeredith, C.D., 1980. Rabies conﬁrmed in Megachiroptera in South\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\nSmith, J.S., 2002. Molecular epidemiology. In: Jackson, A.C., Wunner,\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nAfrica. Rabies Inf. Exchange 3, 26.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nMeredith, C.D., Rossouw, A.R., Van Praag Koch, H., 1971. An unusual\ncase of human rabies thought to be of Chiropteran origin. S. Afr.\nMed. J. 45, 767–769.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.2**r== 0.4**\nMorimoto, K., Patel, M., Corisdeo, S., Hooper, D.C., Fu, Z.F., Rupprecht,\nC.E., Koprowski, H., Dietzschold, B., 1996. Characterization of a\nunique variant of bat rabies virus responsible for newly emerging\nhuman cases in North America. Proc. Natl. Acad. Sci. U.S.A. 93,\n5653–5658.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\nMorimoto, K., Hooper, D.C., Carbaugh, H., Fu, Z.F., Koprowski, H., Di-\netzschold, B., 1998. Rabies virus quasispecies: implications for patho-\ngenesis. Proc. Natl. Acad. Sci. U.S.A. 95, 3152–3156.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.4**\nNadin-Davis, S.A., Huang, W., Wandeler, A.I., 1996. The design of strain-\nspeciﬁc polymerase chain reactions for discrimination of the raccoon\nrabies virus strain from indigenous rabies viruses of Ontario. J. Virol.\nMethods 57, 1–14.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\nW.H. (Eds.), Rabies. Acad. Press, San Diego, pp. 79–113.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\nSmith, P.C., Lawhaswasdi, K., Vick, W.E., Stanton, J.S., 1968. Isolation\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\nof rabies virus from fruit bats in Thailand. Nature 216, 384.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.6**r== 0.0**\nSureau, P., Tignor, G.H., Smith, A.L., 1980. Antigenic characterization\nof the Bangui strain (ANCB-672d) of Lagos bat. Ann. Virol. 131,\n25–32.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.0**\nSwofford, D.L., 2000. Phylogenetic Analysis Using Parsimony (* and\nOther Methods). Version 4.0b6. Sinauer Associates, Sunderland,\nMA.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.0**\nTang, Q., Orciari, L.A., Rupprechti, C.E., Zhao, X., 2000. Sequencing\nand Positional Analysis of the Glycoprotein Gene of Four Chinese\nRabies Viruses. Zhongguo Bingduxue 15, 22–33.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.0**\nTignor, G.H., Shope, R.E., Bhatt, P.N., 1973. Experimental infection of\ndogs and monkeys with two rabies serogroup viruses, Lagos Bat and\nMokola (IbAn 27377): clinical, serologic, virologic, and ﬂuorescent-\nantibody studies. J. Inf. Dis. 128, 471–478.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nTordo, N., Poch, O., Ermine, A., Keith, G., Rougeon, F., 1986. Walking\nalong the rabies genome: is the large G-L intergenic region a remnant\ngene? Proc. Natl. Acad. Sci. U.S.A. 83, 3914–3918.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTordo, N., Badrane, H., Bourhy, H., Sacramento, D., 1993. Molecular\nepidemiology of Lyssaviruses: focus on the glycoprotein and pseudo-\ngenes. Onderstepoort J. Vet. Res. 60, 315–323.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nWoelk, C.H., Pybus, O.G., Jin, L., Brown, D.W.G., Holmes, E.C.,\n(SSPE)\nIncreased positive selection pressure in persistent\n2002.\nversus acute measles virus infections. J. Gen. Virol. 83, 1419–\n1430.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nWunner, W.H., 2002. In: Jackson, A.C., Wunner, W.H. (Eds.), Rabies.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.6**r== 0.2**\nAcad. Press, San Diego, pp. 23–78.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nVan der Merwe, M., 1982. Bats as vectors of rabies. S. Afr. J. Sci. 78,\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nYang, Z., 1997. PAML: a program package for phylogenetic analysis by\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n421–422.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nmaximum likelihood. Comput. Appl. Biosci. 13, 555–556.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nWhitby, J.E., Johnstone, P., Parsons, G., King, A.A., Hutson, A.M., 1996.\nTen-year survey of British bats for the existence of rabies. Vet. Rec.\n139, 491–493.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nYang, Z., Nielsen, R., Goldman, N., Pedersen, A.M., 2000. Codon-\nsubstitution models for heterogeneous selection pressure at amino acid\nsites. Genetics 155, 431–449.",
         "extracted",
         "Novel Lyssaviruses Isolated from Bats in Russia;Bat lyssaviruses (Aravan and Khujand) from Central Asia: phylogenetic relationships according to N, P and G gene sequences.;A comparison of DNA vaccines for the rabies-related virus, Mokola.;New Lyssavirus Genotype from the Lesser Mouse-eared Bat (Myotis blythi), Kyrghyzstan;A molecular epidemiological study of Australian bat lyssavirus.;Competition Commission inquiry into POMs.;Characterisation of an Australian bat lyssavirus variant isolated from an insectivorous bat.;Phylogenetic comparison of the genus Lyssavirus using distal coding sequences of the glycoprotein and nucleoprotein genes;Lyssavirus P gene characterisation provides insights into the phylogeny of the genus and identifies structural similarities and diversity within the encoded phosphoprotein.;Increased positive selection pressure in persistent (SSPE) versus acute measles virus infections.;European Bat Lyssavirus Infection in Spanish Bat Populations;Serologic Evidence of Lyssavirus Infections among Bats, the Philippines;Genetic constraints and the adaptive evolution of rabies virus in nature.;Expression of the B-cell and T-cell epitopes of the rabies virus nucleoprotein in Mycobacterium bovis BCG and induction of an humoral response in mice.;MEGA2: molecular evolutionary genetics analysis software;Extensive Attenuation of Rabies Virus by Simultaneously Modifying the Dynein Light Chain Binding Site in the P Protein and Replacing Arg333 in the G Protein;Molecular basis for the interaction between rabies virus phosphoprotein P and the dynein light chain LC8: dissociation of dynein-binding properties and transcriptional functionality of P.;Functional Interaction Map of Lyssavirus Phosphoprotein: Identification of the Minimal Transcription Domains;Host Switching in Lyssavirus History from the Chiroptera to the Carnivora Orders;Evidence of Two Lyssavirus Phylogroups with Distinct Pathogenicity and Immunogenicity;Antigenic and genetic divergence of rabies viruses from bat species indigenous to Canada.;Codon-substitution models for heterogeneous selection pressure at amino acid sites.;Sequencing and position analysis of the glycoprotein gene of four Chinese rabies viruses;Ecology and evolution of rabies virus in Europe.;Phylogeographic patterns exhibited by Ontario rabies virus variants;Chimeric Lyssavirus Glycoproteins with Increased Immunological Potential;Multiple sequence alignment with Clustal X.;Rabies virus quasispecies: implications for pathogenesis.;DNA-based immunization for exploring the enlargement of immunological cross-reactivity against the lyssaviruses.;PAML: a program package for phylogenetic analysis by maximum likelihood;Evolution of European bat lyssaviruses.;The complete Mokola virus genome sequence: structure of the RNA-dependent RNA polymerase.;A Biographical History of Veterinary Pathology;Ten-year survey of British bats for the existence of rabies;Encephalitis caused by a Lyssavirus in fruit bats in Australia.;Characterization of a unique variant of bat rabies virus responsible for newly emerging human cases in North America.;The design of strain-specific polymerase chain reactions for discrimination of the racoon rabies virus strain from indigenous rabies viruses of Ontario.;Genetic polymorphism in the rabies virus nucleoprotein gene.;Molecular epidemiology of lyssaviruses: focus on the glycoprotein and pseudogenes.;Molecular diversity of the Lyssavirus genus.;Isolation and characterization of 115 street rabies virus isolates from Ethiopia by using monoclonal antibodies: identification of 2 isolates as Mokola and Lagos bat viruses.;[A lyssavirus with an unusual antigenic structure isolated from a bat in southern Kyrgyzstan].;Rapid sequence evolution of street rabies glycoprotein is related to the highly heterogeneous nature of the viral population.;PCR technique as an alternative method for diagnosis and molecular epidemiology of rabies virus.;T helper cell epitope of rabies virus nucleoprotein defined by tri‐ and tetrapeptides;Molecular cloning and complete nucleotide sequence of the attenuated rabies virus SAD B19.;Rabies-related Yuli virus, identification with a panel of monoclonal antibodies.;Efficacy of rabies vaccines against Duvenhage virus isolated from European house bats (Eptesicus serotinus), classic rabies and rabies-related viruses.;Immunity against the European bat rabies (Duvenhage) virus induced by rabies vaccines: an experimental study in mice.;Localization and immunological characterization of antigenic domains of the rabies virus internal N and NS proteins.;Walking along the rabies genome: is the large G-L intergenic region a remnant gene?;HUMAN RABIES OF BAT ORIGIN IN EUROPE;Rabies in the Tropics;Rabies-related viruses.;Rabies virus infection of a flying fox bat, Pteropus policephalus in Chandigarh, Northern India.;Giulio Alfredo Maccacaro.;Experimental infection of dogs and monkeys with two rabies serogroup viruses, Lagos bat and Mokola (IbAn 27377): clinical, serologic, virologic, and fluorescent-antibody studies.;An unusual case of human rabies thought to be of chiropteran origin.;Two African Viruses Serologically and Morphologically Related to Rabies Virus;Isolation of Rabies Virus from Fruit Bats in Thailand;Isolation of a virus from Nigerian fruit bats.;New Cases of Mokola Virus Infection in South Africa: A Genotypic Comparison of Southern African Virus Isolates;BIOEDIT: A USER-FRIENDLY BIOLOGICAL SEQUENCE ALIGNMENT EDITOR AND ANALYSIS PROGRAM FOR WINDOWS 95/98/ NT;MODELTEST: testing the model of DNA substitution;Immunoprotection by rabies virus nucleoprotein.;Nucleotide and deduced amino acid sequences of the nominal nonstructural phosphoprotein of the ERA, PM and CVS-11 strains of rabies virus.;Rabies and Rabies-related viruses in Zimbabwe historical, virological and ecological aspects.;Rabies-Related Viruses;Characterization of an antigenic determinant of the glycoprotein that correlates with pathogenicity of rabies virus.;Antigenic characterization of the bangui strain (ANCB-672d) of lagos bat virus",
         "Phylogenetic relationships of Irkut and West Caucasian bat viruses within the Lyssavirus genus and suggested quantitative criteria based on the N gene sequence for lyssavirus genotype definition.",
         "None"
        ],
        [
         "19",
         "001c244e51484df18db5d96722887ea4a0063e0f",
         "The amygdala is critical for connecting emotional reactions with environmental events. We recorded neurons from the basolateral complex of two monkeys while they performed visually cued schedules of sequential color discrimination trials, with both valid and random cues. When the cues were valid, the visual cue, which was present throughout each trial, indicated how many trials remained to be successfully completed before a reward. Seventy-six percent of recorded neurons showed response selectivity, with the selectivity depending on some aspects of the current schedule. After a reward, when the monkeys knew that the upcoming cue would be valid, 88 of 246 (36%) neurons responded between schedules, seemingly anticipating the receiving information about the upcoming schedule length. When the cue appeared, 102 of 246 (41%) neurons became selective, at this point encoding information about whether the current trial was the only trial required or how many more trials are needed to obtain a reward. These cue-related responses had a median latency of 120 ms (just between the latencies in inferior temporal visual area TE and perirhinal cortex). When the monkey was releasing a touch bar to complete the trial correctly, 71 of 246 (29%) neurons responded, with responses in the rewarded trials being similar no matter which schedule was ending, thus being sensitive to the reward contingency. Finally, 39 of 246 (16%) neurons responded around the reward. We suggest that basolateral amygdala, by anticipating and then delineating the schedule and representing reward contingency, provide contextual information that is important for adjusting motivational level as a function of immediate behavior goals.",
         "Y. Sugase-Miyamoto,B. Richmond",
         "https://www.jneurosci.org/content/jneuro/25/48/11071.full.pdf",
         "\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nThe Journal of Neuroscience, November 30, 2005 • 25(48):11071–11083 • 11071\n**BLOCK**fs== 11.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nBehavioral/Systems/Cognitive\n**BLOCK**fs== 22.0**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\nNeuronal Signals in the Monkey Basolateral Amygdala\nduring Reward Schedules\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nYasuko Sugase-Miyamoto1,2,3 and Barry J. Richmond1\n1Laboratory of Neuropsychology, National Institute of Mental Health, National Institutes of Health, Department of Health and Human Services, Bethesda,\nMaryland 20892-4415, 2Neuroscience Research Institute, National Institute of Advanced Industrial Science and Technology, Tsukuba, 305-8568, Japan, and\n3Japan Society for the Promotion of Science, Overseas Research Fellow, Japan Society for the Promotion of Science, 1-6 Chiyoda-ku, Tokyo, 102-8471, Japan\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.1**r== 0.1**\nThe amygdala is critical for connecting emotional reactions with environmental events. We recorded neurons from the basolateral\ncomplex of two monkeys while they performed visually cued schedules of sequential color discrimination trials, with both valid and\nrandom cues. When the cues were valid, the visual cue, which was present throughout each trial, indicated how many trials remained to\nbe successfully completed before a reward. Seventy-six percent of recorded neurons showed response selectivity, with the selectivity\ndepending on some aspects of the current schedule. After a reward, when the monkeys knew that the upcoming cue would be valid, 88 of\n246 (36%) neurons responded between schedules, seemingly anticipating the receiving information about the upcoming schedule length.\nWhen the cue appeared, 102 of 246 (41%) neurons became selective, at this point encoding information about whether the current trial\nwas the only trial required or how many more trials are needed to obtain a reward. These cue-related responses had a median latency of\n120 ms (just between the latencies in inferior temporal visual area TE and perirhinal cortex). When the monkey was releasing a touch bar\nto complete the trial correctly, 71 of 246 (29%) neurons responded, with responses in the rewarded trials being similar no matter which\nschedule was ending, thus being sensitive to the reward contingency. Finally, 39 of 246 (16%) neurons responded around the reward. We\nsuggest that basolateral amygdala, by anticipating and then delineating the schedule and representing reward contingency, provide\ncontextual information that is important for adjusting motivational level as a function of immediate behavior goals.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nKey words: emotion; arousal; motivation; temporal lobe; neurophysiology; visual cue\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nIntroduction\nThe amygdala is important in generating emotional context for\nsituations and events (LeDoux, 2000; Everitt et al., 2003; Phillips\net al., 2003). For example, functional neuroimaging studies in\nhumans show that the amygdala is activated during recognition\nof fearful and happy facial expressions (Morris et al., 2002; Hare\net al., 2005). The amygdala also appears to be involved in evalu-\nating the reward value of a stimulus (Malkova et al., 1997; Par-\nkinson et al., 2001). All of these findings are consistent with the\nview that the amygdala is important for assessing the affective (or\nemotional) valence of a stimulus and/or production of a specific\naffective state (Everitt et al., 2003; Phillips et al., 2003).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nWe have been studying how the brain uses external stimuli,\nparticularly visual cues, to predict how much work (compare\nwith number of trials) is needed to get a reward (visually cued\nreward schedule task) (Bowman et al., 1996). The visual cue,\nwhich predicts the work load remaining in the current schedule,\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nReceived May 4, 2005; revised Oct. 16, 2005; accepted Oct. 20, 2005.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nThis work was supported by the Intramural Research Program of the National Institute of Mental Health. We\nthank Dr. R. C. Saunders, Dr. H. Kosaki, Dr. M. Shidara, Dr. Z. Liu, Dr. S. Ravel, Dr. M. C. Weiner, Dr. I. Mizuhiki, Dr. K.\nKawano, Dr. R. P. Hasegawa, Dr. Y. T. Hasegawa, Dr. S. Yamane, S. Steenrod, and D. Pritchett for their help. We\ngratefully acknowledge support in Japan from Japan Society for the Promotion of Science (B.J.R.).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nCorrespondence should be addressed to Dr. Barry J. Richmond, National Institute of Mental Health/National\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nInstitutes of Health, Building 49, Room 1B80, Bethesda, MD 20892-4415. E-mail: bjr@ln.nimh.nih.gov.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.1**r== 0.8**\nDOI:10.1523/JNEUROSCI.1796-05.2005\nCopyright © 2005 Society for Neuroscience\n**BLOCK**fs== 7.0**b== 0.0**t== 1.0**l== 0.2**r== 0.6**\n0270-6474/05/2511071-13$15.00/0\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\naffects the monkey’s motivational and probably emotional states;\nbecause the visual cue progressively indicates that more work\nremains, the error rates and reaction times increase. The increases\nin error rates and reaction times indicate the monkeys are less\nmotivated. This decrease in motivation in turn shows that the\ncues indicating that more work remains negatively affect the\nmonkey’s emotional state in that the monkeys become less in-\nclined to complete the current trial correctly. Single neuron re-\ncording studies have identified robust signals related to this type\nof reward expectancy in the ventral striatum (Shidara et al.,\n1998), the perirhinal cortex (Liu and Richmond, 2000), and the\nanterior cingulate cortex (Shidara and Richmond, 2002).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nHere we report on single neuronal recordings from monkeys\nperforming this reward schedule task in the basolateral complex\nof the amygdala, a region that receives input from the anterior\ntemporal lobe, including area TE (Cheng et al., 1997) and rhinal\ncortex (Stefanacci et al., 1996), and is high in dopamine (Sadikot\nand Parent, 1990). The signals carried by amygdala neurons, by\ncoding for schedule and the reward contingency, could be used to\nresolve ambiguity about the upcoming reward. The anatomical\nconnections and timing relations of the signals suggest that the\namygdala may be part of a temporal lobe network that takes\ninformation about stimulus identity from temporal lobe sensory\nareas and generates signals that interpret the external stimuli or\nevents by providing an emotional context, allowing the animal to\norganize its behavior.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nest in the rewarded trial (Fig. 1). The brightness was proportionally re-\nlated to the schedule state [pixel intensities in eight bit; 85 (33%) for the\n1/3 state, 127 (50%) for the 1/2, 170 (67%) for the 2/3, and 255 (100%)\nfor all three rewarded trials, namely the 1/1, 2/2, and 3/3 states] (Fig. 1 B).\nIn the random cue condition, the reward schedule sequence remained,\nbut the brightness of the cue was chosen randomly from the four bright-\nness levels trial by trial so that there was no relationship between the cue\nbrightness and the schedule state.\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nIn another no-work, no-cue control condition, designated as free-\nreward condition, a reward was delivered at a regularly irregular rate of\nbetween 5 and 8 s (chosen pseudorandomly) with no work required, nor\nwas any related stimulus shown.\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nEach task was run as a block of trials.\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nSurgical preparation\nAfter the monkeys were trained to perform the reward schedule task, a\ncylinder for microelectrode recording and a head holder (Crist Inc., Ha-\ngerstown, MD) were affixed to the skull above the dorsal surface of the\nbrain during an aseptic surgical procedure with the animal under isoflu-\nrane anesthesia. A scleral magnetic search coil for measuring eye move-\nment was implanted during the same surgery (Robinson, 1963; Judge et\nal., 1980). The monkeys were given a 2 week postoperative recovery\nperiod. After recovery, the monkeys were given practice in the tasks with\na loose fixation requirement.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nRecording of single neuronal activity\nSingle-neuron data and behavioral data were collected while the mon-\nkeys performed the reward schedule task in both the valid and random\ncue conditions. A hydraulic microdrive (Narishige, Tokyo, Japan) or an\nelectromechanical microdrive (Alpha Omega, Nazareth, Israel) was\nmounted on the recording cylinder, and tungsten microelectrodes [im-\npedance, 1.5–1.7 M; Frederick Haer Company (Bowdoinham, ME) or\nRoboz-Microprobe (Rockville, MD)] were inserted vertically through a\nstainless steel guide tube that was placed in a hole of a grid within a\nrecording chamber (Crist et al., 1988) (Crist Inc.). Experimental control\nand data collection were performed by a personal computer, using the\nREX real-time data-acquisition program adapted for the QNX operating\nsystem (Hays et al., 1982). Single-neuron activity was isolated by discrim-\nination using time–voltage window discriminator (Frederick Haer Com-\npany). All event and spike times were stored at 1 ms time resolution. All\nof the experimental procedures were in accordance with the National\nInstitutes of Health Guide for the Care and Use of Laboratory Animals and\nwere approved by the Animal Care and Use Committee of the National\nInstitute of Mental Health.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nRecording sites localization\nIn each recording session, the depth of the recording area was measured\nfrom the upper tip of a guide tube. Before we changed the location of a\nguide tube in the grid, a magnetic resonance (MR) image was acquired to\ndocument the recording location (Saunders et al., 1990). A tungsten\nmicroelectrode was inserted into the monkey’s brain to use as a landmark\nto indicate the recording location on the MR images (susceptibility\nartifact).\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nFor histological examination of our recording sites, electrolytic mic-\nrolesions (30 A, 40 s) were made in the brain of one monkey (monkey\nA). One week later, the monkey was deeply anesthetized with sodium\npentobarbital and perfused through the heart with saline and 10% For-\nmalin fixative. The brain was sectioned into 50 m slices in the coronal\nplane and stained with cresyl violet. The sites of guide tubes and microle-\nsions were identified on the sections under microscopic examination.\nThe histological reconstructions matched the locations determined using\nthe MR, so the MR images alone were used for localization of recording\nsites in the second monkey.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nData analysis\nBehavioral measure. Behavioral performances of the monkeys were mea-\nsured using reaction times and error rates. Reaction times were defined to\nbe the time interval from the appearance of the green spot (go signal) to\nthe time of bar release.\n**BLOCK**fs== 8.5**b== 0.0**t== 1.0**l== 0.5**r== 0.1**\nNeuronal response measure. Neuronal responses were analyzed with\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nFigure 1.\nVisually cued reward schedule task. A, Visually cued two-trial reward schedule.\nTime sequence of stimuli in an individual color discrimination trial is shown. Visual stimuli are\ncentered. B, Visual cues.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nMaterials and Methods\nAnimals, behavioral task, and visual stimuli\nTwo adult rhesus monkeys (Macaca mulatta, 9 and 5 kg, one male and\none female, respectively) were used. During our experiments, the mon-\nkey squatted in a primate chair facing a 21 inch computer monitor.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe monkeys were first trained to perform a sequential red-to-green\ncolor discrimination task (Fig. 1 A). The monkeys were trained without a\nfixation requirement. A display was 43.5 cm in front of the monkey. A\nblack-and-white random dot background covered the whole screen of\nthe display.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nWhen the monkey made contact with a touch lever to initiate a trial, a\nvisual cue (rectangle 38  2° in visual angle) appeared in the center of the\nscreen. Four hundred milliseconds later, a small red spot (0.3  0.3°)\nappeared on the center of the visual cue. After a random period between\n1 and 2 s, the color changed to green. If the monkeys released the touch\nlever within a 0.2–1.3 s period after the appearance of the green spot, the\nspot became blue, indicating that a correct response had occurred. After\nanother 100 –200 ms, the visual cue and the blue spot disappeared, and a\ndrop of liquid was delivered. The intertrial interval was 1.1 s.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nIf the monkey did not release the bar within the designated period (i.e.,\nwithin a 0.2–1.3 s period after the green spot onset), an error was regis-\ntered, the stimuli disappeared, and the intertrial interval before the next\ntrial started.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nAfter the monkeys learned to perform the color discrimination task\n(with 90% correct trials per session), we introduced the visually cued\nreward schedules. In the visually cued reward schedule task (Fig. 1 A), the\nmonkey had to complete one, two, or three color discrimination trials\ncorrectly to obtain a reward, i.e., schedules of one, two, or three trials. We\nidentify the trials in the schedules as schedule states. For labeling of the\nschedule states (Fig. 1), the numerator indicates a progress in the sched-\nule, and the denominator indicates the schedule length (in number of\ntrials). The schedule states are 1/1 for the one-trial schedule, 1/2 and 2/2\nfor the two-trial schedule, and 1/3, 2/3, and 3/3 for the three-trial sched-\nule. A reward was delivered only after the successful completion of the\nfinal trial of a schedule. A new schedule of one, two, or three trials was\npicked randomly after a reward.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nIf the monkeys released the touch lever too early, i.e., earlier than 200\nms after the green spot appeared, or too late, i.e., later than 1.3 s after the\ngreen spot appeared, the stimuli all disappeared, an error was registered,\nand the normal intertrial interval began. There was no explicit punish-\nment for errors, but the schedule state advanced only after a correct trial.\nAfter an error trial, the same trial of the same schedule state was repeated.\nWe tested valid and random cue conditions. In the valid cue condition,\nthe cue became brighter as the rewarded trial approached and was bright-\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.6**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nrespect to several discrete events (Fig. 1 A): touching a bar by the mon-\nkeys, appearance of the cue (cue), appearance of the red spot (wait),\nchange of the red spot to a green spot (go), release of the bar by the\nmonkey (bar release), change of the green spot to a blue spot (correct),\nthe time when both the cue and the spot disappeared, and the time of\nreward apparatus activation, i.e., reward delivery, in the final trial of the\nschedule (reward). We counted number of spikes within different\nperievent periods as neuronal response measure.\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nPrecue activity was measured by counting the number of spikes during\na 300 ms period just before the cue onset. Control activity for the precue\nactivity was measured during a 300 ms period starting 600 ms before the\ncue onset. Go/prebar-release activity was measured by counting the\nnumber of spikes during a 300 ms period just before the bar release.\nReward-related response was measured by counting the number of\nspikes during a 400 ms period after the reward. Control activity for the\nreward-related response was measured during a 400 ms period just be-\nfore the reward.\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nConventional statistical procedures (mentioned in Results) were used\nto assess significance. All data analyses were done in the R statistical\ncomputing environment (Ihaka and Gentleman, 1996).\n**BLOCK**fs== 8.5**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nResponse latency. The latency of the cue-related responses was deter-\nmined by comparing spike counts before cue presentation (background\nactivity) and spike counts in a test window that moved starting from the\ntime of the cue onset. To determine response latency, we used two time\nwindows with different durations. We first used a wider test window (100\nms) and then used a narrow test window (50 ms). Background activity\nwas set as spike counts in a 100 ms period just before the cue onset for the\n100 ms test window and as spike counts in a 50 ms period just before the\ncue onset for the 50 ms test window. The 100 ms sliding time window was\nmoved in 5 ms steps, and the 50 ms window was moved in 3 ms steps.\nUsing the 100 ms time window, we found first four consecutive windows\nthat showed significantly different activity compared with the back-\nground activity (i.e., activity in a 100 ms period before the cue onset;\npaired t test, p  0.05). The first of the four consecutive windows was\npicked. Starting at the beginning of the first window, we then moved a 50\nms window to find first four consecutive windows that showed signifi-\ncantly different activity compared with the background activity (i.e., ac-\ntivity in a 50 ms period before the cue onset). The first window of the\nconsecutive four windows was picked again, and the middle of the win-\ndow was defined as the latency for the neuronal response. Values that are\nmeasured with this procedure are consistent with values we would have\nchosen by eye.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nResults\nBehavioral and electrophysiological data were collected while two\nadult rhesus monkeys performed the visually cued reward sched-\nule task (Fig. 1).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nBehavioral analysis\nAs seen previously (Bowman et al., 1996; Shidara et al., 1998; Liu\nand Richmond, 2000; Shidara and Richmond, 2002), the mon-\nkeys were sensitive to the cue and the work schedule in the valid\ncue condition (Fig. 2 A). The error rates and reaction times were\nindistinguishable across the three rewarded trials, 1/1, 2/2, and\n3/3 (single-factor ANOVA to the two monkeys together; mean\n 1.79, p  0.05; mean reaction times, F(2,735)\nerror rates, F(2,735)\n 0.0022, p  0.05). The error rates and reaction times were\nsignificantly different across the first trials, 1/1, 1/2, and 1/3 (sin-\n 46.72, p  0.0001;\ngle factor ANOVA; mean error rates, F(2,735)\n 73.7, p  0.0001), showing that\nmean reaction times, F(2,735)\nthe monkeys were sensitive to the information provided by the\ncue about the schedule.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nBoth monkeys reacted progressively more quickly as the re-\nwarded trial approached in the two- and three-trial schedules\n(single-factor ANOVA with repeated measures for effect of\n 314.3, p \nschedule progress, two-trial schedule, F(1,245)\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nFigure 2.\nError rates and reaction times in visually cued reward schedule task in both the\nvalid cue and random cue conditions. A, Relationship between monkeys’ behavioral perfor-\nmance and the schedule states. The abscissa shows the schedule states. The ordinate shows the\nproportion of error trials in all trials (left) and mean reaction times (right) in the correct trials.\nBehavioral performances in the valid cue condition are shown with black symbols, and perfor-\nmances in the random cue condition are shown with gray symbols. The data include all of the\nrecording sessions for each monkey (131 for monkey A and 115 for monkey B). Error bars\nindicate SE. B, Relationship between monkey’s behavioral performance and the cue brightness.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n 324.8, p  0.0001). Error\n0.0001; three-trial schedule, F(2,490)\nrates became smaller as the schedule progressed through the\nthree-trial schedule (single-factor ANOVA with repeated mea-\n 79.4, p  0.0001).\nsures for effect of schedule progress, F(2,490)\nIn the two-trial schedule, error rates changed significantly for\n\nmonkey B only (F(1,130)\n109.7, p  0.0001 for monkey B).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\n 1.3, p  0.05 for monkey A; F(1,114)\n**BLOCK**fs== 9.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nAs expected in the random cue condition, the error rates and\nreaction times were the same in all three first trials (1/1, 1/2, and\n 0.81, p \n1/3; single-factor ANOVA; mean error rates, F(2,390)\n 0.18, p  0.05). In the ran-\n0.05; mean reaction times, F(2,390)\ndom cue condition, some information about the reward contin-\ngency is available because the longest possible sequence of unre-\nwarded trials was fixed (two in the three-trial schedule); the\nprobability of a reward is 1/3 in the first trial after a preceding\nreward, 1/2 if one unrewarded trial had passed, and 1, i.e., the\nreward is certain, if two unrewarded trials have passed. In previ-\nous studies, the monkeys had not shown any sensitivity to the\nnumber of trials that had passed (Bowman et al., 1996; Shidara et\nal., 1998; Liu and Richmond, 2000; Shidara and Richmond,\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\n2002). Here, however, both monkeys were sensitive to number of\ntrials that had passed, with a stronger trend in the three-trial\nschedule. For monkey A, the error rates increased as the schedule\nprogressed (single-factor ANOVA with repeated measures for\n 5.8, p  0.01) whereas the\neffect of schedule progress, F(2,146)\nreaction times were not modulated by the schedule state\n 0.63, p  0.05). For monkey B, the reaction times were\n(F(2,146)\nrelated to the schedule states, becoming shorter as the schedule\n 46.6, p  0.0001), whereas the error\nstate progressed (F(2,112)\n 1.5, p \nrates were not modulated by the schedule state (F(2,112)\n0.05). In the states of the two-trial schedule, neither reaction\ntimes nor error rates were significantly different for monkey A.\n 13.6, p \nReaction times decreased for monkey B (F(1,56)\n0.001).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nBoth monkeys were also sensitive to the cue brightness in the\nrandom cue condition (Fig. 2 B). The brightest cue was treated\ndifferently from other cues in both reaction times and error rates\n( p  0.01, t test). Thus, the monkeys were sensitive to two fea-\ntures of the random cue condition, the brightness of the cue, and\nthe number of unrewarded trials that had passed.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.4**l== 0.1**r== 0.5**\nTo examine behavioral variability, we took data from 84 of\n246 recording sessions (46 for monkey A, 38 for monkey B) for\nthe valid cue condition in which at least 20 trials had been per-\nformed for each schedule state. Data were available from the\nrandom cue condition in 40 of these sessions. In the valid cue\ncondition, the reaction times were significantly different across\nthree first trials (1/1, 1/2, and 1/3) in 43 of 84 (51%) sessions\n(single-factor ANOVA, p  0.01). The reaction times were indis-\ntinguishable across the three rewarded trials, i.e., 1/1, 2/2, and\n3/3, for 79 of 84 (94%) sessions (single-factor ANOVA, p  0.05).\nIn the two-trial schedule, the reaction times were significantly\ndifferent along progress for 43 of 84 (51%), and in the three-trial\nschedule for 41 of 84 (49%; single-factor ANOVA with repeated\nmeasures for effect of schedule progress, p  0.01). All 84 record-\ning sessions showed at least one of the following four reaction\ntime characteristics: significantly different across the three first\ntrials, not significantly different across the three rewarded trials\n(i.e., not influenced by the post-reward number as described\nabove for the random cue condition), significantly different\nalong the progress in the two-trial schedule, or significantly dif-\nferent along the progress in the three-trial schedule. Twenty-\nthree sessions showed one, 18 sessions showed two, another 25\nshowed three, and 18 showed a significant effect of all four reac-\ntion time characteristics. Thus, the valid cue seemed to affect the\nanimals in every recording session, although the effect changed\nits pattern from session to session. For 18 of the 23 sessions show-\ning only one criterion, the criterion met was not significantly\ndifferent across the three rewarded trials. The neurons recorded\nin these 18 sessions did not seem to react differently from neurons\nrecorded in the other sessions, suggesting that there was no sub-\nstantial difference between the data collected when the behavior\nwas affected by the schedule in a large way compared with the\ndata when the behavior was less affected. In the random cue\ncondition, the reaction times of three first trials were not signifi-\ncantly different for any sessions (single-factor ANOVA, p \n0.01), and the reaction times decreased significantly as the sched-\nule state progressed in 2 of 15 sessions of monkey B (single-factor\nANOVA with repeated measures for effect of schedule progress,\np  0.01, a result indistinguishable from a random effect).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nWe could not find clear relationships between neuronal re-\nsponses that are presented in the following and these session-to-\nsession variability regarding the reaction times.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nFigure 3.\nHistological localization of recorded neurons. Location of recorded neurons in one\nmonkey (monkey A) are plotted in circles with coronal sections positioned at A26, A25, A24, and\nA23–A22 of the right hemisphere. A26 represents anterior 26 mm in Horsley-Clarke coordinates\n(namely, distance from the plane having the external auditory meatus). AA, Anterior amygda-\nloid nuclei; AB, accessory basal nucleus; B, basal nucleus; Ce, central nucleus; L, lateral nucleus;\nac, anterior commissure; cd, caudate nucleus; cl, claustrum; lv, lateral ventricle; opt, optic tract;\nrs, rhinal sulcus; D, dorsal; L, lateral; M, medial; V, ventral.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nElectrophysiology\nActivity was recorded from 246 (131 from monkey A, 115 from\nmonkey B) single neurons located in the basolateral complex\n[including lateral (L), basal (B), and accessory basal (AB) nuclei]\nof the right amygdala (Fig. 3) of two monkeys during the visually\ncued reward schedule task (Fig. 1). Responses were collected\nwhile single neurons were well isolated, which gave from 6 to 80\nrepetitions (median 21) of each schedule state (1/3, 2/3, 3/3, 1/2,\n2/2, and 1/1) in valid cue condition, and 5– 43 repetitions (me-\ndian 22) in random cue condition. We were unable to identify\nsystematic response differences related to recording site location,\nso we present the neurons as one set.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nWe examined neuronal activity over different task events,\nnamely before and after presentation of visual cue (precue activ-\nity and cue-related response, respectively), before monkey’s bar\nrelease (go/prebar-release activity), and around reward appara-\ntus activation (reward related response). We found that 188 of\n246 neurons (76%) showed a response in one (59 of 188) or more\n(129 of 188) of these time periods (Table 1). Thus, for more than\none-half of neurons (129 of 246, 52%), task-related activity oc-\ncurred at more than one point in the behavioral task.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nSignals related to reward schedules\nPrecue activity\nIn the valid cue condition, 119 of 246 (48%) of the neurons in the\nbasolateral complex of the amygdala showed precue activity that\nis selective for differentiating interschedule from intraschedule\nperiods (t test between activity in the 300 ms precue period before\nthe first trials and activity in the precue period before the non-\nfirst trials, p  0.01) (as in the example in Fig. 4 A). A total of 88 of\n119 neurons (74%) showed this type of preschedule activity\nbuilding toward the end of the interschedule period, i.e., appear-\nance of the first cue (one-tailed t test between activity in the\nprecue period before the first trials and before the non-first trials,\np  0.005). There was virtually no precue activity preceding non-\nfirst trials (2/2, 2/3, and 3/3).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nThe time onset for the preschedule activity varied from trial to\ntrial (compare with Fig. 4 A), usually starting well after the reward\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.6**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.1**r== 0.7**\nTable 1. Number of neurons responding\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nNeuronal response\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nNumber of responses per cell\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nPrecue activity\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nCue-related response\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nGo/prebar-release activity\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nReward-related response\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nNumber of cells\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nTotal number of cells\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n\n\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\n\n\nTotal number of cells\n\nCircles indicate significant response (p  0.01).\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\n\n\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n\n\n\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n59 (31%)\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n81 (43%)\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\n44 (24%)\n4 (2%)\n188 (100%)\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.6**r== 0.1**\nonset compared with just before the cue\nonset (paired t test between activity just\nbefore and after the cue onset; activity was\nmeasured in either a 100 ms period or, if\nthe latency was shorter than 100 ms, a pe-\nriod before latency of the after cue related\nresponse, p  0.01).\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.6**r== 0.1**\nThe other 31 of 119 neurons (26%)\nshowed a complementary response pat-\ntern, i.e., the precue activity in the non-\nfirst trials (2/2, 2/3, and 3/3) was larger\nthan precue activity in the first trials (one-\ntailed t test, p  0.005). No single neuron\nhad precue activity before every cue (t test\nbetween an activity in the precue period\nand an activity in the control period, p \n0.01).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.6**r== 0.1**\nWhen we divided the 300 ms precue\nperiod into two 150 ms periods, the activ-\nity of 33 of 119 neurons (all 33 neurons for\npreschedule activity) was significantly dif-\nferent\nin the two periods (one-tailed\npaired t test on increase, p  0.005), show-\ning that the precue activity of many neu-\nrons was higher just before the appearance\nof the cue than it had been earlier.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.4**\nFigure 4.\nPrecue activity. A, In the valid cue condition, neuronal activity before the cue presentation increased only in the first\ntrial (i.e., preschedule activity). The abscissa shows times from the cue onset (0 ms, vertical line). The ordinate shows a firing\nfrequency per trial (spikes per second per trial). Each line of the dots represents a trial in the task. The earliest trial is placed at the\nbottom. Each curve shows instantaneous firing frequency averaged over all trials and smoothed with a Gaussian filter ( 20\nms). The one-, two-, and three-trial schedules are placed on the top, middle, and bottom rows, respectively. Progress in each\nschedulegoesfromlefttoright.Therewardedtrialsareonthediagonal.Thegraytrianglespointingdownindicatethetimewhenthevisual\ncue and the blue fixation spot disappeared in the previous trial. The black triangles pointing down show the time of reward apparatus\nactivationattheprevioustrial.Themedianlatencyoftheprescheduleactivityfromtheprecedingrewardapparatusactivationwas742ms\n(interquartile range, 451–1018 ms). B, In the random cue condition, preschedule activity disappeared.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\n(700 ms here) and long before (300 ms here) the cue signaling\nthe beginning of the next schedule. The preschedule activity gen-\nerally continued until shortly after cue onset (the ending defined\nas the time when a cue-triggered response began). Only 2 of 88\nneurons showed a significant increase in activity just after the cue\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nA total of 82 neurons were tested in the\nrandom cue condition. The difference in\nactivity between preschedule and intras-\nchedule periods disappeared (49 neurons)\n(Fig. 4 B) or became significantly smaller\n(11 neurons; t test, p  0.005). Thus, the\nprecue activity was, in general, not simply\nrelated to the reward in the preceding trial.\nThe preschedule activity in the population\nwas small in the random cue condition\n(Fig. 5). Thus, the preschedule activity is\nrelated to receiving the information about\nthe next reward schedule that will be indi-\ncated by upcoming valid visual cue, possibly attributable to in-\ncreased arousal, attentiveness, or some other mechanisms.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nBecause amygdala lesions interfere with reinforcer devalua-\ntions (Malkova et al., 1997), we looked for neuronal firing\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nA, B, Strength of precue activity of a neuronal population in the valid cue (A; n \nFigure 5.\n88) and random cue (B; n  68) conditions. In the box plots, middle line indicates the median;\nnotches indicate the 95% confidence interval. Whiskers extend to the most extreme data point\nthat is not 1.5 times the interquartile range. The strength of preschedule activity (i.e., 1/1,\n1/2, and 1/3 states) was significantly larger in the valid cue condition (A) than in the random cue\ncondition (B; Wilcoxon’s test, p  0.0001).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nchanges related to the progress through a recording session (pre-\nsumably the monkeys become satiated as the session progresses,\nalthough each recording session might be too short to detect\neffects related to satiation). We did not identify any trend in the\nprecue activity.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nCue-related response\nFrom inspection, it appeared that many neurons had responses\nafter the appearance of the cues. To identify the period with most\ncue-selective response, a 350-ms-wide analysis window was\nmoved across the responses in 10 ms steps (Fig. 6). In the range\nstarting with the time window from 285 to 75 ms relative to cue\nappearance and ending with the time window from 115 to 465 ms\nrelative to cue appearance, 102 of 246 neurons showed at least\nthree consecutive time windows with significant selectivity across\nthe three first cues or the three non-first cues (single-factor\nANOVAs, p  0.01). Of these 102 neurons, 81 showed significant\ndifferential activity across the responses elicited by the three first\ncues (1/1, 1/2, and 1/3) (Fig. 7), six across the three non-first cues\n(2/2, 2/3, and 3/3), and 15 to both (Fig. 8). The time window from\n100 to 450 ms after cue onset, which was initially chosen by eye,\nwas used to approximate the time period with greatest effect\n(Fig. 6).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nIn the interschedule or intraschedule precue period, only 3 of\n102 neurons (3%, i.e., 5% level) showed a significant differen-\ntial response related to the upcoming schedule during the antic-\nipatory precue period (starting with the time window from 575\nto 225 ms relative to cue appearance ending with the time\nwindow from 355 to 5 ms relative to cue appearance) (Fig.\n6 A). Thus, the precue activity had no systematic effect on the\npostcue selectivity. The effect of the three first or non-first cues\non neuronal activity increased starting at 100 ms after the cue\nonset (Fig. 6). The activity in a 100 ms period just after the cue\nonset, i.e., the continuation of the precue activity after the cue on-\nset, was indistinguishable across the three first cues or the three\nnon-first cues for all 102 neurons (single-factor ANOVA, p \n0.01).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nTo determine whether the responses to the first cues might be\nrelated to the predicted schedule length, pairwise comparisons\n(1/1 vs 1/2, 1/2 vs 1/3, and 1/1 vs 1/3; Tukey’s test to adjust for\nmultiple comparisons, p  0.01) were performed. In this analysis,\n84 of 102 (82%) neurons showed differential activity across the\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nFigure 6.\nTime course of the effect of either the three first cues or three non-first cues on the\nresponses around the cue presentation measured by single-factor ANOVA. The abscissa shows\ntime from the cue onset relative to the end of 350 ms analysis window. A, Number of neurons\nthat showed significant selectivity across the three first or non-first cues (single-factor ANOVAs,\np  0.01). The solid vertical line shows the cue onset. The largest number of neurons (86) with\na significant differential activity was found in the 85– 435 ms time window after cue onset.\nStarting with the time window from 185 to 165 ms after the cue onset, the number of cells\nwith significant differential response increased to 12 (a number significantly different from\nchance; 2 test, p  0.01). B, Mean percentage response variance explained for 102 neurons\nthat showed at least three consecutive time windows with significant selectivity across the\nthree first cues or the three non-first cues (single-factor ANOVAs, p  0.01). The gray curves show SE\nofthepercentageresponsevarianceexplained.Thelargestaveragevariancewasexplainedinthetime\nwindow from 105 to 455 ms after cue onset (33  2%, mean  SE; n  102).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nfirst trials (compatible with, but not identical to, the ANOVA\nabove) (Table 2). For 43 of 84, the responses elicited by 1/1 cue\nwere different from the responses to the 1/2 or 1/3, whereas the\nresponses to these latter cues were indistinguishable (Fig. 7A); for\n23 of 43 neurons, the lack of difference in the responses between\nthe 1/2 and 1/3 cues occurred because there was no response to\nthe 1/2 and 1/3 cues. Thus, there was a significant emphasis of the\n1/1 schedule in the population. Nonetheless, the selectivity of the\nneurons within the population provides information about\nwhich schedule is starting (Table 2).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nNone of the comparisons across first trials was significant for\nthe random cue condition (Figs. 7B, 8 B), as expected if the mon-\nkey was unable to predict the schedule length. The 1/1 cue often\ninduces a significantly different response from the other two cues\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.6**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nFigure 7.\nCue-related response that delineates the shortest schedule length. A, In the valid\ncue condition, this neuron showed unstable preschedule activity starting before the cue onset (0\nms, vertical line). The largest response followed the 1/1 cue. Two comparisons, namely 1/1\nversus 1/2 and 1/1 versus 1/3, were significant for this neuron (Tukey’s test, p  0.01). B, In the\nrandom cue condition, none of the comparisons between responses to two different first trials\n(1/1 vs 1/2, 1/1 vs 1/3, and 1/2 vs 1/3) was significantly different (Tukey’s test, p  0.01). C, In\nthe random cue condition, the cue-related responses sorted by brightness did not depend on\ncue brightness (single-factor ANOVA, p  0.05). Fiducial marking as in Figure 4.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nin the valid cue condition, perhaps attributable to the monkey\nanticipating an immediate reward. However, it is not quite\nstraightforward to attribute this effect to a simple difference in\narousal level as inferred from the behavioral measure. The behav-\nioral results (compare with Fig. 2) suggest that the monkeys\nmight be more aroused (or at least trying harder, i.e., more mo-\ntivated) in the 1/2 state than in the 1/3 state, yet there does not\nseem to be any systematic difference between the cue-related re-\nsponses in the 1/2 and 1/3 states for many neurons. This makes it\ndifficult to invoke a simple connection between probable state of\narousal, motivation, or effort and the first-cue elicited responses.\nPairwise comparisons (2/3 vs 2/2, 2/3 vs 3/3, and 2/3 vs 3/3;\nTukey’s test to adjust for multiple comparisons, p  0.01)\nshowed that 14 of 102 (14%) neurons exhibited differential ac-\ntivity across the non-first trials (Table 3, Fig. 8), revealing that the\nresponses in non-first trials can on occasion be related to the\nreward contingency. Of the 43 neurons that showed a signifi-\ncantly different response to the 1/1 cue than to the other two first\ncues (see above), only five showed differential activity across the\nnon-first trials, suggesting that the neurons responding to the 1/1\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nFigure 8.\nCue-related response. A, In the valid cue condition, after preschedule activity, this\nneuron showed an excitatory response after the cue onset (0 ms, vertical line) of the one-trial\nschedule but stopped firing after the cue presentation of the two- or three-trial schedule. In the\ntwo- and three-trial schedules, the neuron responded again after the cue onset in the rewarded\ntrials, namely both 2/2 and 3/3. Response latencies are 90 ms for 1/1 cue, 173 for 1/2 cue, 93 for\n2/2 cue, 153 for 1/3 cue, and 93 for 3/3 cue. B, In the random cue condition, the neuron showed\na small response only to the first cue; however, the response pattern across the cues was\nsignificantly different from the responses in the valid cue condition. The strength of the excita-\ntory response in 1/1 was significantly smaller compared with the response in the valid cue\ncondition (t test, p  0.0001). Response latencies are 145 ms for 1/1 cue, 108 for 1/2 cue, and\n113 for 1/3 cue. C, Cue-triggered responses in the random cue condition shown in B were sorted\nby the brightness of the cue. Brightness value of each cue is shown as percentage brightness\n(the value 255 is the maximum in 8-bit grayscale and shown as 100%). The strength of the\n 17.7, p  0.0001),\nresponse was modulated by the brightness (single-factor ANOVA, F(3,88)\nwith the brightest cue eliciting the strongest response. The excitatory response to the brightest\ncue was significantly smaller compared with the response either to the 1/1, 2/2, or 3/3 cues in\nthe valid cue condition (t test, p  0.001). Fiducial markings as in Figure 4.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\ncue were not simply responding because the cue signaled a re-\nwarded trial (i.e., the response to the 2/3 cue was generally not\ndifferent from the response to the 2/2 or 3/3 cues). It may be that\nthe responses to the 1/1 cue are special because the cue indicates\nthe conjunction of a first and rewarded trial, the most favorable\nmessage possible in this task.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nSixty-seven of the 102 neurons were tested in the random cue\ncondition. The cue-related responses of 13 neurons were influ-\nenced according to two-level first vs non-first trial discrimina-\ntion, with 8% of response variance being explained [single-factor\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.7**\nANOVA, p  0.01; f test with 65–190 and 1\ndegree of freedom; n  13; interquantile\nrange (iqr) of 6 –11%].\n**BLOCK**fs== 9.5**b== 0.5**t== 0.2**l== 0.1**r== 0.7**\nThe responses of 28 of 67 (42%) neu-\nrons were related to the cue brightness\n(single-factor ANOVA, p  0.01; f test\nwith 54 –213 and 3 degrees of freedom)\n(Fig. 8C), with 18% (median, iqr of 11–\n23%) of response variance being ex-\nplained. Of the 28 neurons, the response\nwas strongest to the brightest cue for 11\nneurons (Fig. 8C) or to the darkest cue for\n17. For 18 of 28 neurons, the strength of\nthe cue-related responses in the valid cue\nsignificantly different\ncondition were\nfrom the responses to the same brightness\ncue in the random cue condition (t test,\np  0.01) (an example shown in Fig. 8),\nindicating that the cue-related responses\nin the valid cue condition was not simply\nrelated to the cue brightness for majority\nof neurons (the 18 plus 39 neurons with-\nout the brightness effect, i.e., 57 of 67\nneurons).\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.4**r== 0.1**\nTable 2. Significant discrimination among cue-related responses across first trials of different schedules (Tukey’s\ntest, p < 0.01; see Results)\nComparison\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n1/1 versus 1/2\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1/1 versus 1/3\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n1/2 versus 1/3\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nNumber of cells\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nTotal number of cells\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nCircles indicate significant comparisons.\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.4**r== 0.1**\nTable 3. Significant discrimination among cue-related responses across non-first trials of multiple-trial schedules\n(Tukey’s test, p < 0.01; see Results)\nComparison\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n2/3 versus 2/2\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n2/3 versus 3/3\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n2/2 versus 3/3\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nNumber of cells\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\nTotal number of cells\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nCircles indicate significant comparisons.\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nFor another 38 of 246 neurons, the ac-\ntivity changed significantly after the cue\nappearance, but the activity was indistinguishable across 1/1, 1/2,\nand 1/3 states (single-factor ANOVA, NS; t test between rate in a\n100 – 450 ms window after cue appearance and a 150 ms window\nbefore the appearance, p  0.01). These responses might be sim-\nply related to arousal induced by the appearance of the cue.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nThe latency of the cue-triggered response was 119 ms (me-\ndian, iqr of 98 –158 ms; n  312; measured for the 113 neurons\nthat showed a clear phasic increase or decrease in activity after the\ncue onset). The median firing rate was 13 spikes/s (iqr of 8 –23\nspikes/s; n  312). The latency was slightly but significantly\nshorter when precue activity preceded the response (median of\n110 ms; iqr of 93–153 ms; n  103) compared with the response\nlatency when there was no precue activity before the response\n(median of 123 ms; iqr of 103–163 ms; n  209; Wilcoxon’s test,\np  0.05). These latencies are longer than those reported for area\nTE and shorter than those reported for perirhinal cortex (Liu and\nRichmond, 2000).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nSignals related to reward\nGo/prebar-release activity\nIn the valid cue condition, we suspected from inspection that the\nactivity after the go signal presentation and just preceding bar\nrelease was the same in the final trials of all schedules (Fig. 9A). A\ntotal of 71 of 246 neurons showed significant modulation accord-\ning to two-level rewarded versus nonrewarded trial discrimina-\ntion (single-factor ANOVA, p  0.01; f test with 45– 440 and 1\ndegree of freedom). Fourteen percent of the response variance\nwas explained (median, n  71; iqr of 9 –25%). When the re-\nsponses of these 71 neurons were analyzed using four levels in\nwhich the levels were the three nonrewarded states (1/3, 2/3, and\n1/2) plus a level for any rewarded trial (thus grouping 1/1, 2/2,\nand 3/3 together; reward proximity in the schedule), the in-\ncreased degrees of freedom significantly improved the explana-\ntory power for 10 neurons (an f test comparing the amount of\nvariability accounted for by the two models, p  0.01) (Venables\nand Ripley, 2002), and 33% of the response variance was ex-\nplained (median, n  10; iqr of 30 – 40%). When the responses of\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nthese 71 neurons were analyzed using the six schedule states, the\nincreased degrees of freedom significantly improved the explan-\natory power for only two neurons, confirming our suspicion that\nthe responses were related to the reward contingency or the num-\nber of trials remaining before the rewarded trial.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nThe responses were time locked to either the bar release (45 of\n71 neurons) or the go signal (26 of 71 neurons; responses classi-\nfied according to the alignment giving the biggest peak response\nin the time period between the go signal appearance and bar\nrelease). The activity was also modulated by progress in both the\ntwo- and three-trial schedules for 25 of 71 neurons ( p  0.01,\nsingle-factor ANOVA with repeated measures for effect of sched-\nule progress; f test with 11–37 and 1 degree of freedom for the\ntwo-trial schedule; f test with 16 – 66 and 2 degrees of freedom for\nthe three-trial schedule). The activity increased as the monkey\napproached the rewarded trial in the schedules (19 of 25 neurons;\none-tailed t test, p  0.005) (Fig. 9A), suggesting that the go/\nprebar-release activity is related to the expectation of forthcom-\ning reward and/or proximity of reward, possibly via increased\narousal.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nForty-five of these 71 neurons were tested in the random cue\ncondition. All but one neuron (44 of 45) lost the differential\nactivity depending on the reward contingency (single-factor\nANOVA, p  0.01; f test with 67–220 and 1 degree of freedom).\nGo/prebar-release activity in the population was present in every\ntrial (Fig. 10) (example of one neuron Fig. 9B). For 33 neurons,\nthe activity was independent of the cue brightness (single-factor\nANOVA, p  0.01; f test with 65–218 and 3 degrees of freedom)\n(Fig. 9C), suggesting that the go/prebar-release activity in the\nvalid cue condition did not encode the cue brightness.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nTrial-by-trial reaction time and go/prebar-release activity was\nsignificantly related only for 9 of 71 neurons (linear regression, 10\nof 426 cases, i.e., 6 states by 71 neurons; p  0.01), showing that\nthe strength of the go/prebar-release activity was only occasion-\nally related to the reaction time. In addition, monkey B showed\nshorter reaction times as number of trials that had passed in-\ncreased in the random cue condition, but no neuron (0 of 13\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.6**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nStrength of go/prebar-release activity of a neuronal population. A, In the valid\nFigure 10.\ncue condition, the activity of a neuronal population was larger in rewarded trials (n  71; paired\nWilcoxon’s test; *p  0.01; **p  0.001; ***p  0.0001). B, In the random cue condition, the\npopulation activity was indistinguishable across schedule states (n  45). The strength of the\nrewarded trials (1/1, 2/2, and 3/3) was not significantly different between the valid and random\ncue conditions. Fiducial markings as in Figure 5.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nReward-related response\nThirty-nine of 246 neurons responded at the time the reward\napparatus was activated (which only occurred in rewarded trials).\nFor 32 neurons, there was an excitatory response (Fig. 11 A), and,\nfor the other seven, there was an inhibitory response (one-tailed t\ntest against strength of activity in a 400 ms period before the\nreward, p  0.005). For seven neurons, the response started be-\nfore reward apparatus activation. For 12 of 39 neurons (including\none inhibitory neuron) that had responses in the rewarded trial,\nwe also found responses in nonrewarded trials at the time when\nthe reward would have been delivered if this had been a rewarded\ntrial. For these 12 neurons, the strength of responses in the re-\nwarded and nonrewarded trials were not consistently different (t\ntest, p  0.005; larger to the rewarded trials, 4 of 12; smaller to the\nrewarded trials, 4 of 12; not significantly different, 4 of 12).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nOf 21 of 39 neurons tested with the random cue condition, 19,\nincluding four that showed anticipatory activity before the re-\nward, responded to the reward (one-tailed t test against strength\nof activity in the control period, p  0.005) (Fig. 11 B). Thus,\nmost of the reward-related signals were found in both the valid\nand random cue conditions (Fig. 12 A), suggesting that they are\nrelated to reward delivery itself in some way. Seven neurons re-\nsponded to the reward only in the random cue condition.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.5**r== 0.1**\nThe free-reward condition provided evidence that some\nreward-related responses might become associated with other\ntask events during the reward schedule task. Among 113 neurons\ntested with both the valid cue and free-reward conditions, 20\nresponded in the free-reward condition only and 16 responded in\nboth conditions (Fig. 12 B) (an example in Fig. 11). Of the 20\nneurons that showed the reward-related response only in the\nfree-reward condition, 14 responded either to the cue presenta-\ntion (Fig. 13) or around the bar release in the valid cue condition.\nIf the neuron shown in Figure 13 had been tested only in the\nfree-reward condition (Fig. 13A), we would have concluded that\nthe response was a “reward-”related response. However, in the\nvalid cue condition, the response occurred to the first cue (Fig.\n13B), suggesting that the neuron has a type of signal similar to the\nprediction signal described for dopamine neurons (Mirenowicz\nand Schultz, 1994; Schultz et al., 1997). The response transfer to\nthe cue presentation or to the bar release in the valid cue condi-\ntion might be related to dopaminergic inputs to basolateral com-\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nActivity during go/prebar-release period. A, In the valid cue condition, the activity\nFigure 9.\nincreased gradually as the bar release approached. The vertical lines show time of bar release.\nThe black triangles represent the onset of the go signal. Trials are sorted according to reaction\ntimes, and the trial with longest reaction time is placed at the bottom. Responses before bar\nrelease were selective for the four levels (trials in schedule before reward; single-factor ANOVA,\n 21.7, p  0.0001). The responses of the rewarded trials, namely 1/1, 2/2, and 3/3\nF(3,138)\n 0.21, p  0.05). The activity became stronger as the\nstates, were indistinguishable (F(2,68)\ntwo- or three-trial schedules progressed (single-factor ANOVA with repeated measures for\n 25.4, p  0.0001; three-trial schedule,\neffect of schedule progress, two-trial schedule, F(1,22)\n 9.9, p  0.0005). B, In the random cue condition, the activity was indistinguishable\nF(2,46)\n 1.17, p  0.32, p  0.05). C,\nacross the schedule states (single-factor ANOVA, F(5,116)\nGo/prebar-release activity in the random cue condition (B) according to the cue brightness.\n\nThere was no modulation depending on the cue brightness (single-factor ANOVA, F(3,118)\n1.30, p  0.05). Fiducial markings as in Figure 4.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nneurons) of the monkey showed significant modulation of the\ngo/prebar-release activity related to reward proximity in the ran-\ndom cue condition (single-factor ANOVA, p  0.01; f test with\n80 –218 and 3 degrees of freedom). Thus, the go/prebar-release\nactivity generally did not appear related to motor aspects of the\nmonkey’s bar release.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nForce measurements taken from the drinking spout (Ravel et\nal., 2001) for one monkey (monkey A) revealed that the monkey\nstarted to lick 300 ms after onset of the wait signal in the re-\nwarded trials, long before (by at least 700 ms) the onset of the go\nsignal. Thus, it is unlikely that the go/prebar-release activity was\nrelated to the monkey’s mouth movements.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nFigure 12.\nA, Valid cue and/or random cue conditions. B, Valid cue and/or free-reward conditions.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\nDistribution of the neurons that responded to reward in different task conditions.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\ndynamics, going from uncertainty to more and more focus on the\nreward, may be related to resolution of an emotionally laden (or\narousing) reward situation.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nPossible origins of the signals\nThe basolateral complex of amygdala receives inputs from a late\nstage of ventral visual processing, having reciprocal connections\nwith the area TE (Aggleton et al., 1980; Amaral and Price, 1984;\nCheng et al., 1997; Stefanacci and Amaral, 2002). Area TE has\nreciprocal connections with the perirhinal cortex (Suzuki and\nAmaral, 1994; Saleem and Tanaka, 1996), and perirhinal cortex\nhas reciprocal connections with the amygdala (Amaral and Price,\n1984; Saunders and Rosene, 1988; Stefanacci et al., 1996). Pre-\nsumably visual information that is first processed in area TE is\nsent to both perirhinal cortex and amygdala, and these three areas\ninfluence each other.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nDuring the visually cued reward schedule task, only cue-\nrelated activity has been observed in area TE and perirhinal cor-\ntex. In contrast, amygdala neurons show responses before the cue\npresentation, before the bar release, and to the reward delivery as\nwell as to the cue. Thus, visual, reward-related, and other signals\nmerge in the amygdala.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nIn the amygdala, in the valid cue condition, the population is\nbiased toward the 1/1 schedule. In the random condition, there is\nsmall but significant response related to brightness of the cue,\nsomewhat like what has be seen in area TE (Liu and Richmond,\n2000). In perirhinal cortex, individual neurons show responses\nthat are idiosyncratically related to subsets of the cues, with the\ncue-related responses encoding information about the schedule\nstates in the population (Liu and Richmond, 2000).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nThe latency of the cue-triggered responses in the amygdala\nneurons is longer than those in area TE and shorter than those in\nperirhinal cortex. These results imply that information process-\ning in the amygdala lies between TE and perirhinal cortex and\nmight contribute to form the associative signals in perirhinal\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nFigure 11.\nA–C, Response triggered by reward delivery (1/1, 2/2, and 3/3 states) in the valid\ncue (A), random cue (B), and free-reward (C) conditions. The vertical lines show time of reward\ndelivery in rewarded trials (black) and equivalent timing without reward delivery in nonre-\nwarded trials (gray). This neuron showed reward-related response in all three conditions. Fidu-\ncial markings as in Figure 4.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nplex (Sadikot and Parent, 1990). A similar type of response trans-\nfer has been found in other brain areas that receive dopaminergic\ninputs, such as the orbitofrontal cortex (Matsumoto et al., 1995;\nTremblay and Schultz, 2000) and the striatum (Apicella et al.,\n1997; Ravel et al., 2001).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nDiscussion\nMore than 70% of recorded neurons showed response modula-\ntion for one or more aspects of this visually cued reward schedule\ntask (Table 1). The selectivity of this population changed dynam-\nically across the schedules and trials. The population begins firing\nbefore the cue presentation; for individual neurons, this activity\noccurs during either the interschedule or the intraschedule peri-\nods but not both, signaling whether the next trial begins a new\nschedule or not. When the first cue appears, the population en-\ncodes which schedule is starting, with the greatest proportion of\nneurons indicating whether or not the one-trial (1/1) schedule\nhas begun. Starting just before the bar release, i.e., from the go/\nprebar-release period, the responses become selective for the re-\nward contingency, i.e., they indicate whether a reward is forth-\ncoming or not. A few neurons responded to reward delivery. The\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.6**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nthat precede and predict reward delivery and to motor acts such\nas bar release that lead to a forthcoming reward (Schultz and\nRomo, 1987). Some amygdala neurons respond to the cue or\naround bar release in the valid cue condition but to reward in the\nfree-reward condition, a situation related to reward anticipation\nor prediction. In the amygdala, the response elicited by the 1/1\ncue is often different from the response to the other two first cues\n(Table 2), a condition that is different from what is found for the\ndopamine neurons, in which the responses to the first cue are\nindistinguishable (S. Ravel and B. J. Richmond, unpublished ob-\nservation). This suggests that the dopamine neurons might play a\nrole in eliciting these cue-related responses but are probably not\nadequate to explain why the cue in the 1/1 condition elicits a\ndifferent response than the 1/2 or 1/3 cues.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nRecent lesion studies have shown that reinforcer devaluation\ndepends on the amygdala and orbitofrontal cortex (Malkova et\nal., 1997; Baxter et al., 2000; Izquierdo and Murray, 2004), sug-\ngesting that both the amygdala and orbitofrontal cortex play a\nrole in association of the conditioned stimulus with the current\nvalue of a reward. Orbitofrontal neurons respond to a visual\nstimulus that is associated with a reward and the reward itself\n(Tremblay and Schultz, 2000). Orbitofrontal cortex has a heavy\nprojection to presumed inhibitory neurons (intercalated masses\nlocated between lateral and basal nucleus) in the amygdala\n(Ghashghaei and Barbas, 2002), and signals in the orbitofrontal\ncortex are thought to modify signal processing within the amyg-\ndala. Inhibition after the preschedule activity, such as is shown in\nFigure 8 for the 1/2 and 1/3 cues, might have been influenced by\nsuch processing. The basolateral amygdala is important for the\norbitofrontal cortex to represent the significance of a reward-\npredicting cue (Schoenbaum et al., 2003); thus, the signals in the\namygdala could influence signals in orbitofrontal cortex, too.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nOther inputs such as those from the posterior thalamus might\nalso play a role in the cue-related response (Komura et al., 2001;\nShi and Davis, 2001).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nFunctional roles of the neuronal responses in the amygdala\nThe basolateral complex of amygdala receives sensory inputs\nfrom sensory association cortices and reward-related informa-\ntion from substantia nigra pars compacta/ventral tegmental area\nand has reciprocal connections with areas that are important for\nmotor selection depending on reward, e.g., the anterior cingulate\ncortices (Amaral and Price, 1984; Stefanacci and Amaral, 2002),\nand the medial and orbitofrontal cortex (Amaral and Price, 1984;\nGhashghaei and Barbas, 2002). It also has projections to the ven-\ntral striatum (Friedman et al., 2002; Fudge et al., 2002). The re-\nsponses we recorded show integration of information from these\nmodalities in a time-multiplexed manner (Table 1). Perhaps the\namygdala plays a role when association among sensory signals,\nmotor outputs, and reward is required for task performance.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nIt is known that amygdala neurons (including the basolateral\ncomplex) are responsive to sensory events associated with reward\nand/or to reward itself in monkeys (food, juice, or intracranial\nself-stimulation) (Nishijo et al., 1986, 1988a,b; Nakamura et al.,\n1992) and that neurons in basolateral nucleus show anticipatory\nactivity before getting a reward in rats (Pratt and Mizumori, 1998;\nSchoenbaum et al., 1998). Responses of amygdala neurons to an\nunconditioned stimulus change depending on satiety (Yan and\nScott, 1996) or taste (Nishijo et al., 1988b). Based on these previ-\nous studies, we had expected responses related to reward and\npossibly sensory events predicting the reward (e.g., cue). The\nprecue activity and the modulations around the bar release were\nunexpected. Comparing our results in other brain regions with\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nFigure 13.\nResponse transfer. The black vertical lines show time of reward delivery, and the\ngray vertical lines show cue onset. A, In the free-reward condition, this neuron showed a re-\nsponse after reward. B, In the valid cue condition, this neuron showed a response to the first cue\n(1/1, 1/2, and 1/3). The response to the 1/1 cue was larger than the response to the 1/2 cue and\n1/3 cue (t test, p  0.005). Fiducial markings as in Figure 4.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\ncortex in which the D2 receptor is crucial for learning the relation\nof the cue to the schedule states (Liu et al., 2000, 2004). There is,\nhowever, a difficulty. At approximately the time of bar release\nand/or a reward delivery, perirhinal cortex neurons did not re-\nspond, so either the cue-triggered amygdala responses do not\ncontribute to perirhinal neuronal responses or cue-triggered ac-\ntivation of perirhinal neurons requires other input in conjunc-\ntion with the amygdala inputs.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nAnother possible source for some of the signals in the amyg-\ndala are dopamine neurons in substantia nigra pars compacta/\nventral tegmental area and/or neurons in orbitofrontal cortex,\nboth of which project strongly to the basolateral complex of\namygdala (Sadikot and Parent, 1990; Ghashghaei and Barbas,\n2002). Dopamine neurons are known to respond to visual stimuli\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nthose presented here, the amygdala is the only area in which this\nstrong and somewhat prolonged precue activity has been seen\nthus far (Bowman et al., 1996; Shidara et al., 1998; Liu and Rich-\nmond, 2000; Shidara and Richmond, 2002). The amygdala is the\nonly area studied thus far for which the population responses\nemphasizes the shortest schedule.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.2**l== 0.1**r== 0.5**\nA number of previous results suggest that the amygdala plays\nan important role in evaluating the value or valence of objects.\nThe anatomical connections and timing relations of the signals\nsuggest that the amygdala is in a temporal lobe network that takes\ninformation about stimulus identity from temporal lobe sensory\nareas and generates signals predicting the behavioral path to be\ntaken for obtaining a reward. Given the anatomical connections\nwith the amygdala and a suggested role of the amygdala in emo-\ntions, it seems likely that emotions aroused by salient stimuli\nparticipate in determining the behavioral path. We have seen a\npreschedule signal that grows as the time for cue appearance\napproaches, possibly because the monkey becomes progressively\nmore aroused while awaiting the appearance of the first cue.\nWhen the first cue appears, the signal becomes specific, encoding\ninformation about which schedule started with emphasis on the\nshortest schedule, presumably the best (for the monkey) situa-\ntion. This is compatible with the role of the amygdala in assessing\nthe valence associated with a stimulus (Everitt et al., 2003), in\nwhich the most desirable information, that this is the one-trial\nschedule, is associated with the most selectivity in the population.\nLater in each trial, the responses are better explained in terms of\nthe reward contingency in the current schedule. These signals\nseem to provide information that interprets external events and\nstimuli so that the motivational level can be adjusted according to\ncurrent behavioral goals. Such signals might be important for\nforming associative signals related to the reward schedules in\nperirhinal cortex and/or ventral striatum or forming long-term\nreward expectation signals in anterior cingulate cortex.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nReferences\nAggleton JP, Burton MJ, Passingham RE (1980) Cortical and subcortical\nafferents to the amygdala of the rhesus monkey (Macaca mulatta). Brain\nRes 190:347–368.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nAmaral DG, Price JL (1984) Amygdalo-cortical projections in the monkey\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n(Macaca fascicularis). J Comp Neurol 230:465– 496.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nApicella P, Legallet E, Trouche E (1997) Responses of tonically discharging\nneurons in the monkey striatum to primary rewards delivered during\ndifferent behavioral states. Exp Brain Res 116:456 – 466.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nBaxter MG, Parker A, Lindner CC, Izquierdo AD, Murray EA (2000) Con-\ntrol of response selection by reinforcer value requires interaction of amyg-\ndala and orbital prefrontal cortex. J Neurosci 20:4311– 4319.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nBowman EM, Aigner TG, Richmond BJ (1996) Neural signals in the mon-\nkey ventral striatum related to motivation for juice and cocaine rewards.\nJ Neurophysiol 75:1061–1073.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nCheng K, Saleem KS, Tanaka K (1997) Organization of corticostriatal and\ncorticoamygdalar projections arising from the anterior inferotemporal\narea TE of the macaque monkey: a Phaseolus vulgaris leucoagglutinin\nstudy. J Neurosci 17:7902–7925.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nCrist CF, Yamasaki DS, Komatsu H, Wurtz RH (1988) A grid system and a\nmicrosyringe for single cell recording. J Neurosci Methods 26:117–122.\nEveritt BJ, Cardinal RN, Parkinson JA, Robbins TW (2003) Appetitive be-\nhavior: impact of amygdala-dependent mechanisms of emotional learn-\ning. Ann NY Acad Sci 985:233–250.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFriedman DP, Aggleton JP, Saunders RC (2002) Comparison of hippocam-\npal, amygdala, and perirhinal projections to the nucleus accumbens: com-\nbined anterograde and retrograde tracing study in the Macaque brain.\nJ Comp Neurol 450:345–365.\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nFudge JL, Kunishio K, Walsh P, Richard C, Haber SN (2002) Amygdaloid\nprojections to ventromedial striatal subterritories in the primate. Neuro-\nscience 110:257–275.\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.1**r== 0.5**\nGhashghaei HT, Barbas H (2002) Pathways for emotion: interactions of\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nprefrontal and anterior temporal pathways in the amygdala of the rhesus\nmonkey. Neuroscience 115:1261–1279.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nHare TA, Tottenham N, Davidson MC, Glover GH, Casey BJ (2005) Con-\ntributions of amygdala and striatal activity in emotion regulation. Biol\nPsychiatry 57:624 – 632.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nHays AV, Richmond BJ, Optican LMA (1982) Unix-based multiple process\nsystem for real-time data acquisition and control. WESCON Conf Proc\n2:1–10.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nIhaka R, Gentleman RR (1996) R: a language for data analysis and graphics.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.3**\nComput Graphical Stat 5:299 –314.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nIzquierdo A, Murray EA (2004) Combined unilateral lesions of the amyg-\ndala and orbital prefrontal cortex impair affective processing in rhesus\nmonkeys. J Neurophysiol 91:2023–2039.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nJudge SJ, Richmond BJ, Chu FC (1980) Implantation of magnetic search\ncoils for measurement of eye position: an improved method. Vision Res\n20:535–538.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nKomura Y, Tamura R, Uwano T, Nishijo H, Kaga K, Ono T (2001) Retro-\nspective and prospective coding for predicted reward in the sensory thal-\namus. Nature 412:546 –549.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nLeDoux JE (2000) Emotion circuits in the brain. Annu Rev Neurosci\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n23:155–184.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nLiu Z, Richmond BJ (2000) Response differences in monkey TE and perirhi-\nnal cortex: stimulus association related to reward schedules. J Neuro-\nphysiol 83:1677–1692.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nLiu Z, Murray EA, Richmond BJ (2000) Learning motivational significance\nof visual cues for reward schedules requires rhinal cortex. Nat Neurosci\n3:1307–1315.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nLiu Z, Richmond BJ, Murray EA, Saunders RC, Steenrod S, Stubblefield BK,\nMontague DM, Ginns EI (2004) DNA targeting of rhinal cortex D2 re-\nceptor protein reversibly blocks learning of cues that predict reward. Proc\nNatl Acad Sci USA 101:12336 –12341.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nMalkova L, Gaffan D, Murray EA (1997) Excitotoxic lesions of the amygdala\nfail to produce impairment in visual learning for auditory secondary re-\ninforcement but interfere with reinforcer devaluation effects in rhesus\nmonkeys. J Neurosci 17:6011– 6020.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nMatsumoto K, Nakamura K, Mikami A, Kubota K (1995) Responses to un-\npredictable water delivery into the mouth of visually responsive neurons\nin the orbitofrontal cortex of monkeys. Abstr IBRO Satellite Meet In-\nuyama 14.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nMirenowicz J, Schultz W (1994) Importance of unpredictability for reward\nresponses in primate dopamine neurons. J Neurophysiol 72:1024 –1027.\nMorris JS, deBonis M, Dolan RJ (2002) Human amygdala responses to fear-\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nful eyes. NeuroImage 17:214 –222.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nNakamura K, Mikami A, Kubota K (1992) Activity of single neurons in the\nmonkey amygdala during performance of a visual discrimination task. J\nNeurophysiol 67:1447–1463.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nNishijo H, Ono T, Nakamura K, Kawabata M, Yamatani K (1986) Neuron\nactivity in and adjacent to the dorsal amygdala of monkey during operant\nfeeding behavior. Brain Res Bull 17:847– 854.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nNishijo H, Ono T, Nishino H (1988a) Topographic distribution of\nmodality-specific amygdalar neurons in alert monkey. J Neurosci\n8:3556 –3569.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nNishijo H, Ono T, Nishino H (1988b) Single neuron responses in amygdala\nof alert monkey during complex sensory stimulation with affective signif-\nicance. J Neurosci 8:3570 –3583.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nParkinson JA, Crofts HS, McGuigan M, Tomic DL, Everitt BJ, Roberts AC\n(2001) The role of the primate amygdala in conditioned reinforcement.\nJ Neurosci 21:7770 –7780.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nPhillips ML, Drevets WC, Rauch SL, Lane R (2003) Neurobiology of emo-\ntion perception. I. The neural basis of normal emotion perception. Biol\nPsychiatry 54:504 –514.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nPratt WE, Mizumori SJ (1998) Characteristics of basolateral amygdala neu-\nronal firing on a spatial memory task involving differential reward. Behav\nNeurosci 112:554 –570.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nRavel S, Sardo P, Legallet E, Apicella P (2001) Reward unpredictability in-\nside and outside of a task context as a determinant of the responses of\ntonically active neurons\nJ Neurosci\n21:5730 –5739.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nin the monkey striatum.\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nRobinson DA (1963) A method of measuring eye movement using a scleral\nsearch coil in a magnetic field. IEEE Trans Biomed Eng 10:137–145.\nSadikot AF, Parent A (1990) The monoaminergic innervation of the amyg-\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.6**\nSugase-Miyamoto and Richmond • Signals in the Amygdala during Reward Schedules\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\ndala in the squirrel monkey: an immunohistochemical study. Neuro-\nscience 36:431– 447.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nSaleem KS, Tanaka K (1996) Divergent projections from the anterior in-\nferotemporal area TE to the perirhinal and entorhinal cortices in the\nmacaque monkey. J Neurosci 16:4757– 4775.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nSaunders RC, Rosene DL (1988) A comparison of the efferents of the amyg-\ndala and the hippocampal formation in the rhesus monkey. I. Conver-\ngence in the entorhinal, prorhinal, and perirhinal cortices. J Comp Neurol\n271:153–184.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nSaunders RC, Aigner TG, Frank JA (1990) Magnetic resonance imaging of\nthe rhesus monkey brain: use for stereotactic neurosurgery. Exp Brain Res\n81:443– 446.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nSchoenbaum G, Chiba AA, Gallagher M (1998) Orbitofrontal cortex and\nbasolateral amygdala encode expected outcomes during learning. Nat\nNeurosci 1:155–159.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nSchoenbaum G, Setlow B, Saddoris MP, Gallagher M (2003) Encoding pre-\ndicted outcome and acquired value in orbitofrontal cortex during cue\nsampling depends upon input from basolateral amygdala. Neuron\n39:855– 867.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nSchultz W, Romo R (1987) Responses of nigrostriatal dopamine neurons to\nhigh-intensity somatosensory stimulation in the anesthetized monkey.\nJ Neurophysiol 57:201–217.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nSchultz W, Dayan P, Montague PR (1997) A neural substrate of prediction\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nand reward. Science 275:1593–1599.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nShi C, Davis M (2001) Visual pathways involved in fear conditioning mea-\nsured with fear-potentiated startle: behavioral and anatomic studies.\nJ Neurosci 21:9844 –9855.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nShidara M, Richmond BJ (2002) Anterior cingulate: single neuronal signals\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nrelated to degree of reward expectancy. Science 296:1709 –1711.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nShidara M, Aigner TG, Richmond BJ (1998) Neuronal signals in the mon-\nkey ventral striatum related to progress through a predictable series of\ntrials. J Neurosci 18:2613–2625.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nStefanacci L, Amaral DG (2002) Some observations on cortical inputs to the\nmacaque monkey amygdala: an anterograde tracing study. J Comp Neu-\nrol 451:301–323.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nStefanacci L, Suzuki WA, Amaral DG (1996) Organization of connections\nbetween the amygdaloid complex and the perirhinal and parahippocam-\npal cortices in macaque monkeys. J Comp Neurol 375:552–582.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nSuzuki WA, Amaral DG (1994) Perirhinal and parahippocampal cortices of\nthe macaque monkey: cortical afferents. J Comp Neurol 350:497–533.\nTremblay L, Schultz W (2000) Reward-related neuronal activity during go-\nnogo task performance in primate orbitofrontal cortex. J Neurophysiol\n83:1864 –1876.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nVenables WN, Ripley BD (2002) Modern applied statistics with S-plus. New\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nYork: Springer.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nYan J, Scott TR (1996) The effect of satiety on responses of gustatory neu-\nrons in the amygdala of alert cynomolgus macaques. Brain Res 740:193–\n200.",
         "extracted",
         "",
         "Neuronal Signals in the Monkey Basolateral Amygdala during Reward Schedules",
         "None"
        ],
        [
         "20",
         "001c41c640543a02eb1cf2aa978093d80f719620",
         "Apart from alterations in the RET/PTC-RAS-BRAF pathway, comparatively little is known about the genetics of papillary thyroid carcinoma (PTC). We show that numerous microRNAs (miRNAs) are transcriptionally up-regulated in PTC tumors compared with unaffected thyroid tissue. A set of five miRNAs, including the three most up-regulated ones (miR-221, -222, and -146), distinguished unequivocally between PTC and normal thyroid. Additionally, miR-221 was up-regulated in unaffected thyroid tissue in several PTC patients, presumably an early event in carcinogenesis. Tumors in which the up-regulation (11- to 19-fold) of miR-221, -222, and -146 was strongest showed dramatic loss of KIT transcript and Kit protein. In 5 of 10 such cases, this down expression was associated with germline single-nucleotide changes in the two recognition sequences in KIT for these miRNAs. We conclude that up-regulation of several miRs and regulation of KIT are involved in PTC pathogenesis, and that sequence changes in genes targeted by miRNAs can contribute to their regulation.",
         "Huiling He,K. Jażdżewski,Wei Li,S. Liyanarachchi,R. Nagy,S. Volinia,G. Calin,Chang-gong Liu,K. Franssila,S. Suster,R. Kloos,C. Croce,A. de la Chapelle",
         "https://europepmc.org/articles/pmc1323209?pdf=render",
         "\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nHuiling He*, Krystian Jazdzewski*, Wei Li*, Sandya Liyanarachchi*, Rebecca Nagy*, Stefano Volinia*, George A. Calin*,\nChang-gong Liu*, Kaarle Franssila†, Saul Suster‡, Richard T. Kloos§, Carlo M. Croce*, and Albert de la Chapelle*¶\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n*Human Cancer Genetics Program, ‡Department of Pathology, and §Departments of Internal Medicine and Radiology, Comprehensive Cancer Center,\nOhio State University, Columbus, OH 43210; and †Department of Pathology, Helsinki University Central Hospital, FIN-00029, Helsinki, Finland\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nContributed by Albert de la Chapelle, November 9, 2005\n**BLOCK**fs== 8.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nApart from alterations in the RETPTC-RAS-BRAF pathway, com-\nparatively little is known about the genetics of papillary thyroid\ncarcinoma (PTC). We show that numerous microRNAs (miRNAs) are\ntranscriptionally up-regulated in PTC tumors compared with unaf-\nfected thyroid tissue. A set of ﬁve miRNAs, including the three most\nup-regulated ones (miR-221, -222, and -146), distinguished un-\nequivocally between PTC and normal thyroid. Additionally, miR-\n221 was up-regulated in unaffected thyroid tissue in several PTC\npatients, presumably an early event in carcinogenesis. Tumors in\nwhich the up-regulation (11- to 19-fold) of miR-221, -222, and -146\nwas strongest showed dramatic loss of KIT transcript and Kit\nprotein. In 5 of 10 such cases, this down expression was associated\nwith germline single-nucleotide changes in the two recognition\nsequences in KIT for these miRNAs. We conclude that up-regulation\nof several miRs and regulation of KIT are involved in PTC patho-\ngenesis, and that sequence changes in genes targeted by miRNAs\ncan contribute to their regulation.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\ngene expression  KIT  DNA polymorphism\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nP apillary thyroid carcinoma (PTC) is the most common\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nmalignancy in thyroid tissue, accounting for 80% of all\nthyroid cancers. The incidence of PTC in the United States has\nincreased in recent years (1). Genetically, PTC is characterized\nby alterations in the RETPTC-RAS-BRAF signaling pathway\n(2, 3). Activating mutations in BRAF and RETPTC gene\nrearrangements are frequent genetic changes in PTC tumors\n(4–6). A strong inherited genetic predisposition is suggested by\ncase-control studies showing a 3- to 8-fold risk in first-degree\nrelatives, one of the highest of all cancers (7, 8). Despite\nunequivocal evidence of an inherited predisposition, large fam-\nilies displaying Mendelian inheritance of PTC are rare, and no\npredisposing gene mutations have been found, even though\nseveral putative loci have been identified by linkage analysis\n(9–11).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nMicroRNAs (miRNAs) represent a previously uncharacter-\nized class of gene products that are believed to function as\nnegative regulators of gene expression (12–16). Recently,\nmiRNA genes have been implicated in several cancers (14,\n17–25). The expression of miRNAs varies between cancer and\nnormal cells and varies among different types of cancer. In most\ncancers studied so far, the expression of miRNAs seems to be\nlower than in the corresponding normal tissue (17, 18, 21, 22, 24).\nWe reasoned that the previous failure to identify genes\npredisposing or contributing to PTC might be because these\ngenes show low penetrance. The mechanisms may require the\ninteraction of two or more genes; thus, regulatory, rather than\nprotein-encoding, genes might be involved. We undertook this\nstudy to elucidate the role of regulatory genes such as the\nmiRNAs in the predisposition and development of PTC.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nMethods\nPatient and Control Samples and Cell Lines: Nucleic Acid Extraction.\nAfter approval of the Institutional Review Board and patient\nconsent, fresh samples from the tumor tissue PTC (T-PTC) and\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nnormal thyroid tissue adjacent to PTC tumors (N-PTC) were\nobtained from 20 patients with sporadic PTC undergoing sur-\ngical resection. The samples were snap-frozen in liquid nitrogen\nand stored at 80°C. Clinical data and information on the\nspecimens are shown in Table 2, which is published as supporting\ninformation on the PNAS web site. Normal thyroid tissue\n(N-Thy, n  6) was collected from consenting individuals who\nhad surgery because of laryngeal malignancy but no thyroid\ndisease. Other samples included: paraffin blocks of thyroid tissue\nfrom Finnish sporadic PTC patients (n  135); blood DNA\nindividuals (n  100);\nsamples of random Finnish control\nlymphoblastoid cell DNA samples (n  24) from Centre d’Etude\ndu Polymorphisme Humain control individuals who are not\ngenetically related. Thyroid cancer cell lines K1, K2, and NPA87\nwere cultured in DMEM mixed with F12 and MCDB (ratio\n2:1:1) medium supplemented with 10% FBS and glutamine in a\n5% CO2 incubator. Genomic DNA and RNA were extracted\nfrom the samples by standard techniques. Total RNA was\nextracted with TRIzol solution (Invitrogen), and the integrity of\nRNA was assessed by using an Agilent BioAnalyzer 2100 (Agi-\nlent, Palo Alto, CA).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nmiRNA Microarrays. Biotin-labeled miRNA was hybridized on\nmiRNA chips as described in ref. 26. Briefly, 5 g of total RNA\nfrom each sample was reverse transcribed by using biotin end-\nlabeled random octamers. Hybridization was carried out on our\nnew custom miRNA microarray chip (OSUCCC version 2.0),\nwhich contained 460 mature miRNA probes spotted in quadru-\nplicate (235 Homo sapiens, 222 Mus musculus, and 3 Arabidopsis\nthaliana) with annotated active sites. The hybridized chips were\nwashed and processed to detect biotin-containing transcripts by\nstreptavidin-Alexa647 conjugate and scanned by using an Axon\n4000B (Axon Instruments). Scanned images were quantified by\nGENEPIX PRO 6.0 (Axon Instruments).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nGene Expression Arrays. Affymetrix HG-U133 plus two chips\n(Affymetrix, Santa Clara, CA) were used to evaluate genome-\nwide gene expression levels. The experimental procedure was\naccording to the Affymetrix Gene-Chip Expression Analysis\nmanual. Double-stranded cDNA was synthesized by using the\nSuperscript Choice System, followed by an in vitro transcription\nreaction with a T-7 (dT24) primer to produce biotinylated\ncRNA. The full-length cRNAs were fragmented to 20 to 200 bp\nand hybridized to Affymetrix GeneChip Human Genome U133\nPlus 2.0 Array.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nConﬂict of interest statement: No conﬂicts declared.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nAbbreviations: miRNA, microRNA; PTC, papillary thyroid carcinoma; N-PTC, normal thyroid\ntissue adjacent to PTC tumors; T-PTC, tumor tissue PTC.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nData deposition: Gene expression data with Affymetrix chips were deposited in the\nNational Center for Biotechnology Information Gene Expression Omnibus database (NCBI\nGEO accession no. GSE3467). MicroRNA chip data have been deposited at the ArrayExpress\ndatabase (accession no. E-TABM-68).\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n¶To whom correspondence should be addressed. E-mail: albert.delachapelle@osumc.edu.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\n© 2005 by The National Academy of Sciences of the USA\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.6**r== 0.1**\nPNAS  December 27, 2005  vol. 102  no. 52  19075–19080\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nStatistical and Bioinformatics Analysis of Microarray Data. Raw data\nfrom miRNA chips were assembled by using BRB ArrayTools\n(Richard Simon and Amy Peng Lam, National Cancer Institute,\nBethesda). BRB ArrayTools generated an average value of the\nfour spot replicates for each miRNA. Background subtracted\nintensities were thresholded to 10 and log transformed. Flagged\nspots corresponding to absent or low-quality signals were re-\nmoved from the analysis before global median normalization.\nSignificant analysis of microarrays and prediction analysis of\nmicroarrays were performed to identify differentially expressed\nmiRNAs in PTC tumors and miRNAs for tumor classification,\nrespectively. Data from Affymetrix chips were analyzed by using\nthe robust multichip average to calculate normalized expression\nvalues. Significance analysis of microarrays and permutation\ntests were used to find differentially expressed genes.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nNorthern Blot Analysis of miRNA. Northern blotting was performed\nas described in ref. 17. Ten micrograms of total RNA from\nsnap-frozen tissues and cell lines was loaded onto a precast 15%\ndenaturing polyacrylamide gel (Bio-Rad). The RNA was then\nelectrophoretically transferred to Bright-Star blotting mem-\nbranes (Ambian). The oligonucleotides used as probes were the\ncomplementary sequences of the mature miRNA (miRNA\nRegistry): miR-221, 5-GAAACCCAGCAGACAATGTA-\nGCT-3; miR-146b, 5-AGCCTATGGAATTCAGTTCTCA-3.\nProbes were end-labeled with [-32P]ATP by T4 polynucleotide\nkinase (USB, Cleveland). Prehybridization and hybridization\nwere carried out in Ultrahyb Oligo solution (Ambian) containing\n106 cpmml probes overnight at 42°C. The most stringent wash\nwas with 2 SSC and 1% SDS at 42°C. For reuse, blots were\nstripped by boiling and reprobed. As a loading control, 5S rRNA\nwas visualized by ethidium bromide staining of the gel before\ntransfer. The image of Northern hybridization signals was pro-\nduced by using STORMSCANNER and IMAGEQUANT TL software\n(Molecular Dynamics).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nSemiquantitative and Quantitative RT-PCR. Total RNA was first\ntreated with DNase-1 (Ambian) and then reverse transcribed\nto cDNA with the SuperScript First Strand Synthesis system\n(Invitrogen). Target genes and an endogenous control gene\nglyceraldehyde-3-phosphate dehydrogenase (GAPDH) were in-\ncluded in the same PCR reaction for semiquantitative RT-PCR.\nAll PCR reactions were verified to be in the linear range by\ntesting with different cycle numbers. Quantitative real-time PCR\nwas performed by using an ABI PRISM 7700 DNA Sequence\nDetection System (Applied Biosystems) and a SYBR Green\nPCR kit (Applied Biosystems). The comparative threshold cycle\nmethod was used to calculate the relative gene expression.\nPrimers for amplification of GAPDH, miR-221, miR-222, miR-\n146a, miR-146b, and CITED1 are listed in Table 3, which is\npublished as supporting information on the PNAS web site.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nWestern Blot. Total protein extract was prepared by using lysis\nbuffer containing 50 mM TrisHCl (pH 7.4), 150 mM NaCl, 1%\nTriton X-100, 0.1% SDS, 1 mM EDTA, plus proteinase inhib-\nitors. The protein concentration was determined by the Bio-Rad\nprotein assay. Thirty micrograms of each protein lysate was\nloaded onto SDSPAGE under reducing conditions and trans-\nferred to poly(vinylidene difluoride) membrane (Millipore). The\nmembrane was incubated in 5% nonfat dry milk in T-TBS (18\nmM TrisHCl, pH 7.6122 mM NaCl0.1% Tween 20) at room\ntemperature for 2 h followed by incubation with a monoclonal\nanti-Kit antibody (BD Biosciences) and a secondary antibody of\nhorseradish peroxidase-conjugated goat anti-mouse IgG. Im-\nmune complexes were detected with the Supersignal Western\nPico Chemiluminescent Substrate (Pierce). Loading control was\ndone by reincubating the same membrane with an anti-GAPDH\nantibody (Novus Biological).\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nTable 1. Differentially expressed miRNAs in PTC tumors\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nMiRNA\n**BLOCK**fs== 7.5**b== 0.6**t== 0.2**l== 0.6**r== 0.4**\nhsa-mir-146\nhsa-mir-221\nhsa-mir-222*\nhsa-mir-21*\nhsa-mir-220\nhsa-mir-181a\nhsa-mir-181c\nhsa-mir-181*\nhsa-mir-155\nhsa-mir-213\nhsa-mir-34a\nhsa-mir-24-2\nhsa-mir-29a-2\nhsa-mir-29b\nhsa-mir-29c\nhsa-mir-102\nhsa-mir-24-1\nhsa-mir-9-3\nhsa-mir-219-1\nhsa-mir-138-1\nhsa-mir-138-2\nhsa-mir-345\nhsa-mir-26a-1\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nFold\n**BLOCK**fs== 7.5**b== 0.6**t== 0.2**l== 0.7**r== 0.3**\n19.3\n12.3\n10.9\n4.3\n4.0\n2.6\n2.4\n2.2\n2.2\n1.9\n1.8\n1.7\n1.7\n1.7\n1.6\n1.6\n1.5\n0.7\n0.6\n0.6\n0.6\n0.6\n0.6\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.8**r== 0.1**\nLocalfdr,\n%\n**BLOCK**fs== 7.5**b== 0.6**t== 0.2**l== 0.9**r== 0.1**\n4.4\n3.2\n2.7\n6.7\n8.6\n6.9\n7.6\n7.2\n9.1\n7.3\n21.9\n10.1\n21.7\n23.7\n21.6\n11.2\n14.5\n7.3\n5.7\n5.0\n4.9\n4.7\n5.0\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nA summary of SAM analysis of paired T-PTC and unaffected thyroid tissue\nfrom 15 PTC patients. Mean of fold changes; Localfdr, local false discovery\nrates. q value (%) of all miRNAs was 0.\n*The chips had two probes of mir-222, two probes of mir-21, and three probes\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nof mir-181b. The different probes showed no signiﬁcant fold variation.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nSequencing. DNA specimens were amplified by using a standard\nPCR protocol. The PCR products were purified with Ex-\noSAP-IT purification kit (USB) and sequenced in both direc-\ntions with the ABI sequencing system (PerkinElmer Applied\nBiosystems). The sequencing results were analyzed by using\nDNAStar SEQMAN software. The PCR primers used for KIT\nexons and miRNA sequencing are listed in Table 3.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nHybridization Prediction. The minimum free energy hybridization\nof miRNAs and target mRNA with all polymorphic variants were\npredicted by RNAHYBRID software (27).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nResults\nDistinct Overexpression of miRNAs in PTC Tumor Tissues. Initially\nmiRNA expression was analyzed in paired PTC tumor and\nunaffected thyroid tissue from nine patients. Pairwise signifi-\ncance analysis of microarray analysis indicated that six miRNA\ngenes were significantly overexpressed in T-PTC with fold\nchanges 1.5 in at least seven patients. These miRNAs were as\nfollows: miR-146, miR-221, miR-222, miR-21, miR-155, and\nmiR-181a (Table 1). The top three miRNAs (miR-146, miR-221,\nand miR-222) showed dramatic overexpression, with 11- to\n19-fold higher level in PTC tumors compared to the adjacent\nunaffected thyroid tissue. We repeated the miRNA chip exper-\niments with samples from another six PTC patients. The over-\nexpression of the six miRNAs identified previously was fully\nconfirmed in these new samples. Data analysis of the combined\n30 samples from 15 PTC patients indicated that the up-\nregulation of miRNAs in PTC tumors represented a distinct\nmiRNA signature comprising 23 miRNAs. Among the signifi-\ncant differentially expressed miRNA genes, overexpression oc-\ncurred in 17. Underexpression occurred in only six miRNAs, and\nin all of them, the change was 2-fold (Table 1).\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nFig. 1. MiRNA expression microarray data conﬁrmation. (A) Semiquantita-\ntive RT-PCR of miR-221, miR-222, miR-146b, and CITED1 in the paired PTC\nsamples from nine patients. GAPDH was used as an internal control for RT-PCR;\nM, DNA marker; () negative control; N, normal thyroid tissue; T, tumor tissue.\n(B) Northern blot showing the overexpression of mature miR-221 and miR-\n146b in PTC tumors. The same ﬁlter was used for both hybridizations after\nstripping. 5S RNA in an ethidium bromide gel was used as loading control.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\ning between normal and cancer tissue, we performed class\nprediction analysis by using the first set (18 samples) to predict\nthe outcome of the second set (12 samples). Results from the\nprediction analysis of microarrays analysis indicated that only\nfive overexpressed miRNAs (miR-221, miR-222, miR-146, miR-\n21, and miR-181a) were sufficient to successfully predict cancer\nstatus. No classification errors were made in this analysis (Fig. 5,\nwhich is published as supporting information on the PNAS web\nsite).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nConfirmation of Overexpression of miR-221, miR-222, and miR-146b in\nPTC Tumors. We performed RT-PCR and Northern blot analysis\nof the top three overexpressed miRNAs in PTC tissue samples\nand in thyroid tumor cell lines and normal thyroid tissue samples.\nOverexpression of miR-221 and miR-222 in PTC tumors was\nconfirmed by both semiquantitative RT-PCR and Northern blot\n(Fig. 1 A and B). The expression patterns of miR-221 and\nmiR-222 in PTC tumors were concordant in most of the cases.\nmiR-221 and miR-222 are located close to each other on the X\nchromosome. The concordant expression patterns in different\nPTC samples suggest shared regulatory mechanisms for the\nexpression of these two clustered miRNAs.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nThe probe sequence of miR-146 on the chip was designed\ncorresponding to miR-146a, located on chromosome 5. Recently\nmiR-146b on chromosome 10 was identified. The mature forms\nof these two miRNAs have high sequence homology, only two\nnucleotides being different. Using primers specific for miR-146a\nand miR-146b in their premature forms, no visible RT-PCR\nband of miR-146a was detectable in thyroid tissue. In contrast,\nmiR-146b was significantly overexpressed in PTC tumor samples\n(Fig. 1 A). This result indicated that miR-146a either was not\nexpressed in thyroid tissue or the expression level was very low.\nThus the overexpressed miR-146 on the microchips apparently\nreflects miR-146b. Indeed, the mature form of miR-146b was\nstrongly overexpressed in PTC tumors as confirmed by Northern\nblot (Fig. 1B).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nOverexpression of miR-221 in N-PTC. The expression of miR-221 was\ndetectable in normal thyroid tissue adjacent to PTC tumors\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nFig. 2. Overexpression of miR-221 in normal thyroid tissue (N-PTC) adjacent\nto PTC tumor (T-PTC). (A) Real-time RT-PCR quantiﬁcation of miR-221 expres-\nsion in N-PTCs and T-PTCs. The expression level of sample N-Thy4 (thyroid\ntissue from an individual without thyroid disease) was set as a reference\n(N-Thy4 expression  1). (B) Semiquantitative RT-PCR of miR-221 expression in\ntwo N-PTC samples and three samples of normal thyroid tissue from individ-\nuals without clinical thyroid disease (N-Thy). RT () and RT (), absence vs.\npresence of reverse transcriptase in the RT-PCR reactions.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\n(N-PTC) from all patients (Fig. 1 A). Interestingly, the miR-221\nexpression level in at least two N-PTC samples was much higher\nthan in the rest of them. For example, there was a strong band\nof miR-221 in samples N-PTC1 and N-PTC3 shown by semi-\nquantitative RT-PCR (Fig. 1 A). To examine whether this ob-\nservation might be an artifact due to tumor tissue contaminating\nthe ‘‘normal’’ tissue, we used the same samples to demonstrate\nthe expression of CITED1, a molecule that is known to be highly\nexpressed in PTC tumors but not in normal thyroid (28).\nExpression of the CITED1 gene was not detectable in the same\nset of RNA samples of these normal thyroid tissues, but over-\nexpressed in the corresponding paired tumors (Fig. 1 A) exclud-\ning significant contamination. The variation of miR-221 in\nnormal thyroid tissues was further tested with other N-PTC\nsamples and compared to normal thyroid tissue from individuals\nwithout any clinical thyroid disease. Quantitative real-time PCR\nrevealed the same observation in five sample pairs (Fig. 2A). The\noverexpressed miR-221 in two N-PTC samples was not due to\ncontamination of the RNA preparation with genomic DNA (Fig.\n2B). Considered together, our data suggest that increased ex-\npression of miR-221 in normal thyroid tissue might be an early\ngenetic event in PTC carcinogenesis.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nSmall Percentage of Putative miRNA Target Genes Showing Down-\nExpression at the mRNA Level in PTC Tumors. miRNAs interact with\ntheir target genes and, thereby, play regulatory roles in many\nphysiological functions and pathophysiological processes. We\nanalyzed the predicted targets of the three most significantly\noverexpressed miRNAs (221, 222, and 146). The analysis was\nperformed by using three publicly available algorithms to predict\nhuman miRNA gene targets, i.e., MIRANDA (29), TARGETSCAN\n(30), and PICTAR (31). To reduce the number of false positives,\nwe listed only putative target genes predicted by at least two of\nthe programs (Table 4, which is published as supporting infor-\nmation on the PNAS web site). The above approach produced\n130 putative targets of miR-221. Although KIT was only pre-\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.6**r== 0.1**\nPNAS  December 27, 2005  vol. 102  no. 52  19077\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nFig. 3. Down-expression of KIT transcript and Kit protein in PTC tumor\ntissues. Semiquantitative RT-PCR of KIT expression in ﬁve paired PTC samples\nand two PTC cell lines. Western blot of Kit in protein extract from the same\nsample set. Bands of 140 kDa and 120 kDa represent the mature and\nimmature forms of Kit protein, respectively. GAPDH was used as a loading\ncontrol.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.3**l== 0.1**r== 0.5**\ndicted by MIRANDA, we included KIT in this list, because KIT is\na target of miR-221 as demonstrated by biological methods (32).\nIt is believed that miRNAs interact with target genes at\nspecific sites by inducing cleavage of the targeted message or by\ninhibiting translation (12). Therefore, one might predict that an\noverexpressed miRNA would be associated with down-\nexpression of its targets at the transcript or protein level. We\nexamined the gene expression level of the putative targets of\nmiR-221, -222, and -146 in the same set of RNA samples. This\nanalysis was done first by evaluating genomewide gene expres-\nsion levels in PTC tumors and matched normal thyroid tissues by\nusing the Affymetrix chip. Significance analysis of microarray\nanalysis revealed a list of genes differentially expressed (unpub-\nlished data). Most of these genes showed expression behaviors\nconsistent with our previously published data (28). For example,\noverexpression of the CITED1 gene was found in all PTC tumors\ntested and was confirmed by semiquantitative RT-PCR (Fig.\n1 A). Comparing the lists of putative miRNA targets (as defined\nabove) and differentially expressed genes, we found that the\ndifferentially expressed mRNAs accounted only for 15% of the\ntotal target genes (data not shown). Further analysis of miR-221\ntargets was performed by k-means clustering of the relative\nexpression changes between tumornormal pairs of target genes\n(using gene expression ratios) and miRNAs (using inverse\nmiRNA expression ratios) by using the EPCLUST program\n(http:ep.ebi.ac.ukEPEPCLUST). It appeared that a group\nof target genes (84 probe sets) that showed down-expression in\nPTC tumors was clustered together with miR-221 and miR-222.\nWithin this cluster, 19 genes were identified whose expression\nlevel was significantly down in PTC tumors compared to their\npaired unaffected tissue (random variance t test P value 0.005;\npermutation test P value 0.05; fold changes 1.5). The ex-\npression patterns of miR-221, miR-222, and these 19 targets are\nillustrated in Fig. 6, which is published as supporting information\non the PNAS web site. KIT was one of the genes showing\nprofound down-expression of mRNA in PTC tumors.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nDown-Expression of KIT Transcript and Protein in PTC Tumors. West-\nern blotting of Kit protein was performed with protein extracts\nobtained from five patients and two cell lines (Fig. 3). Hybrid-\nization with a monoclonal anti-Kit antibody revealed two bands\nof 140 kDa and 120 kDa, respectively, corresponding to the\nmature fully glycosylated and the partially glycosylated Kit (33)\n(Fig. 3). Both forms of Kit were readily detected in normal\nthyroid tissue but were strongly reduced or absent in the four\nsamples showing strong reduction of KIT transcript. The down-\nexpression of KIT transcript and Kit protein level correlated well\nwith the strong overexpression of miRs-221, -222, and -146b in\nthese four PTC patient samples (Fig. 1). However, in two further\ncases with KIT transcript down-regulation, Kit protein was\nunchanged (data not shown).\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nFig. 4.\nComputational modeling of the interaction of miR-221 and miR-146\nwith the KIT gene. (A) Hybridization of miR-221 (green color) and KIT mRNA\n(red color); arrows highlight a polymorphic site within the binding domain\n(SNP rs17084733). mfe, minimum free energy. (B) Hybridization of miR-146b\n(green color) and KIT mRNA (red color); arrows highlight a polymorphic site\nwithin the binding domain (SNP rs3733542).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nMutations in the KIT Gene. We sequenced the miR-221, -222, and\n-146b genes and their flanking regions in samples from 48 PTC\npatients and three PTC cell lines. No previously unidentified\nsequence changes or DNA polymorphisms were found (data not\nshown).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nmiRNA-binding domains in the KIT sequence were predicted\nwith several software programs (27, 29, 31). We sequenced the\nregions harboring two binding domains, one for miR-221 and\nmiR-222, and another one for miR-146a and miR-146b,\nin\ngenomic DNA samples from the PTC patients. We found a\npolymorphism in each recognition site. The 3169G3A SNP\n(rs17084733) is located within the crucial region of the miR-221\nand miR-222 domain in the KIT 3 UTR region. We found GA\nheterozygosity in five PTC patients. Heterozygosity for\n3169G3A leads to a conformation change with an increase of\nfree energy (Fig. 4A). The synonymous 2607G3 C SNP\n(rs3733542) in exon 18 is located within the crucial region of the\nmiR-146a and miR-146b domain. This genotype leads to changes\nin miRNA:target gene mRNA duplex conformation and results\nin hybridization with a different region (Fig. 4B). We found GC\nheterozygosity in the same five PTC patients, suggesting linkage\ndisequilibrium. Notably, the tumors of all these five doubly\nheterozygous patients (PTC3, PTC6, PTC8, PTC10, and\nPTC11) had up-regulation of the three top miRs and profound\ndown-regulation of the KIT transcript. Sequencing of all of the\nremaining KIT exons in nine PTC cases and three cell lines\ndisclosed one additional, previously unreported change in exon\n17 (ex17  33, CT) in one PTC patient. To exclude that the\nrare alleles at these SNPs themselves predispose to PTC, we\ncompared the allele frequencies in cohorts of sporadic PTC\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\npatients and healthy controls (both cohorts from Finland). No\nassociation was found (Table 5, which is published as supporting\ninformation on the PNAS web site).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nDiscussion\nA relatively unexpected finding of ours was the highly prefer-\nential\nincrease rather than decrease in transcript of many\nmiRNA genes in the tumors compared with paired unaffected\nthyroid tissue. Such up-regulation has previously been noted in\na few cancers, but decreases in the tumor seem to be more\ncommon (21, 22, 24). For example, miR-15 and miR-16 are\nfrequently down-regulated in B cell chronic lymphocytic leuke-\nmia (17); miR-143 and miR-145 showed decreased expression in\ncolorectal tumors (34); miRNA let-7 was down-expressed in\nhuman lung cancer, which is associated with poor prognosis (24).\nDown-expression of miRNAs was also found in cancers of the\nbreast, kidney, prostate, and uterus (21, 22). Over-expression of\nmiRNAs was reported in some tumors: miR-155 was overex-\npressed in Burkitt’s lymphomas and human B cell lymphoma\n(35), and a cluster of miRNAs, the miR-17–92 polycistron, were\noverexpressed in B cell lymphoma patients (23). Although we\nhave no clear explanation of the preferential up- rather than\ndown-regulation of miRNAs in PTC, we note that this cancer has\nunusually little loss of heterozygosity (LOH) (36, 37). If LOH is\ngenerally associated with loss of mRNA transcript, then the\npaucity of LOH might, in part, explain our findings. It follows\nthat genomic amplification might explain abundant miRNA\ntranscript.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nComparatively little is known about genetic changes in PTC,\nand no predisposing mutations have been described. Our find-\nings are compatible with the hypothesis that RNA-based regu-\nlatory mechanisms involving miRNAs, and possibly other RNA\ngenes, characterize this tumor. The existence of regulatory\nnetworks, including inherited changes therein, is now established\nin principle, although almost nothing is known about the genes\nthemselves (38).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nWe observed that miR-221 expression was not only up-\nregulated in PTC tumors, but also showed clear-cut variation in\nthe unaffected thyroid tissue among patients with PTC. In at\nleast 3 of 15 patients, the up-regulation was strong. This obser-\nvation suggests that unaffected normal thyroid tissue adjacent to\ntumors may harbor genetic changes before the appearance of\nmorphological malignancy. Overexpression of miR-221 could be\na premalignant change in PTC. Taken together, miRNA dereg-\nulation in the thyroid could be a critical component of PTC\ninitiation and development. We hypothesize that miR-221 may\nfunction as an oncogene in the thyroid. Future study is warranted\nto explore the significance of the overexpression of miRNAs in\nthyroid tissues from PTC patients.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nKit is an important tyrosine kinase receptor in cell differen-\ntiation and growth; it functions as an oncogene in many cancers\n(39, 40). The transcript level of KIT in PTC tumor cells is known\nto be extremely low (41, 42), which is consistent with our data.\nIn contrast, reports about the protein level in PTC are contra-\ndictory (43, 44). Although in several PTC tumors we found Kit\nprotein to be down-regulated, this observation did not apply to\nall tumors, emphasizing the complexity of these regulatory\npathways, which may well be organ- or cell-specific.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\nComputational predictions and experimental approaches sup-\nport the idea that different miRNAs target the same mRNA\n(29). Multiple miRNAs have been predicted to target KIT,\nincluding those overexpressed in PTC (27, 29, 31). We show here\nthat in most PTCs at least three miRs (221, 222, and 146)\ntargeting KIT were up-regulated. It is entirely possible that the\nmultiple interaction opportunities provided by networks or\n‘‘signatures’’ of miRNA dysregulation create different responses\nin target genes under different circumstances and combinations.\nPerhaps most interestingly, our results highlight the role of the\ntarget genes themselves in their responses to miRNA interaction.\nWe found a SNP in KIT precisely in the crucial region of the 3\nUTR-binding domain for miR-221 and -222, and another SNP\nprecisely in the crucial region of the exonic binding domain for\nmiR-146a and -146b. The crucial region is a stretch of seven\nnucleotides in the miRNA:mRNA duplex, usually located in the\n5 end of the miRNA, starting at the first or second position. It\nis likely that variants within the binding regions can acquire new\nmiRNAs as regulating factors due to sequence changes such as\nSNPs. Remarkably, of 10 tumors in which the key miRNA genes\nwere up-regulated, all 10 showed low or very low KIT transcript,\nand 5 of these patients were the only ones (of 15 tested) showing\nheterozygosity for the germline SNPs. These findings suggest\nthat not only changes in miRNAs but also in their target genes\n(inherited or perhaps acquired) profoundly influence PTC car-\ncinogenesis. Our association study suggests that these events are\ndownstream of the putative genetic initiating or predisposing\nevents. More detailed functional studies must address questions\nsuch as how miR-221, -222, and -146b interact with their binding\ndomains in the KIT gene, the effects of the two SNPs on the\ninteractions, and the combinational effects of miR-221, -222, and\n-146b on KIT and other possible targets.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nWe thank Jan Lockman and Heejei Yoon for help and advice. This work\nwas supported by National Institutes of Health (NIH) Grants CA16058\n(to A.d.l.C.) and P01 CA78890 (to C.M.C.). BRB ArrayTools was\ndeveloped by Dr. Richard Simon and Amy Peng Lam. Thyroid tissue\nsamples were provided by the Cooperative Human Tissue Network,\nwhich is funded by the National Cancer Institute.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n1. Jemal, A., Clegg, L. X., Ward, E., Ries, L. A., Wu, X., Jamison, P. M., Wingo,\nP. A., Howe, H. L., Anderson, R. N. & Edwards, B. K. (2004) Cancer 101, 3–27.\n2. Kimura, E. T., Nikiforova, M. N., Zhu, Z., Knauf, J. A., Nikiforov, Y. E. &\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n10. Malchoff, C. D., Sarfarazi, M., Tendler, B., Forouhar, F., Whalen, G., Joshi,\nV., Arnold, A. & Malchoff, D. M. (2000) J. Clin. Endocrinol. Metab. 85,\n1758–1764.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nFagin, J. A. (2003) Cancer Res. 63, 1454–1457.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n3. Melillo, R. M., Castellone, M. D., Guarino, V., De Falco, V., Cirafici, A. M.,\nSalvatore, G., Caiazzo, F., Basolo, F., Giannini, R., Kruhoffer, M., et al. (2005)\nJ. Clin. Invest. 115, 1068–1081.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n4. Cohen, Y., Xing, M., Mambo, E., Guo, Z., Wu, G., Trink, B., Beller, U., Westra,\nW. H., Ladenson, P. W. & Sidransky, D. (2003) J. Natl. Cancer Inst. 95, 625–627.\n5. Nikiforova, M. N., Kimura, E. T., Gandhi, M., Biddinger, P. W., Knauf, J. A.,\nBasolo, F., Zhu, Z., Giannini, R., Salvatore, G., Fusco, A., et al. (2003) J. Clin.\nEndocrinol. Metab. 88, 5399–5404.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n6. Fusco, A., Viglietto, G. & Santoro, M. (2005) J. Clin. Invest. 115, 20–23.\n7. Goldgar, D. E., Easton, D. F., Cannon-Albright, L. A. & Skolnick, M. H. (1994)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nJ. Natl. Cancer Inst. 86, 1600–1608.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n8. Czene, K., Lichtenstein, P. & Hemminki, K. (2002) Int. J. Cancer 99,\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\n260 –266.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n9. McKay, J. D., Lesueur, F., Jonard, L., Pastore, A., Williamson, J., Hoffman, L.,\nBurgess, J., Duffield, A., Papotti, M., Stark, M., et al. (2001) Am. J. Hum. Genet.\n69, 440–446.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n11. Canzian, F., Amati, P., Harach, H. R., Kraimps, J. L., Lesueur, F., Barbier, J.,\nLevillain, P., Romeo, G. & Bonneau, D. (1998) Am. J. Hum. Genet. 63,\n1743–1748.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n12. Bartel, D. P. (2004) Cell 116, 281–297.\n13. Baulcombe, D. (2005) Trends Biochem. Sci. 30, 290–293.\n14. Croce, C. M. & Calin, G. A. (2005) Cell 122, 6–7.\n15. Mattick, J. S. & Makunin, I. V. (2005) Hum. Mol. Genet. 14, Spec. No. 1,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nR121–R132.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n16. Pasquinelli, A. E., Hunter, S. & Bracht, J. (2005) Curr. Opin. Genet. Dev. 15,\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n200–205.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n17. Calin, G. A., Dumitru, C. D., Shimizu, M., Bichi, R., Zupo, S., Noch, E., Aldler,\nH., Rattan, S., Keating, M., Rai, K., et al. (2002) Proc. Natl. Acad. Sci. USA 99,\n15524–15529.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n18. Calin, G. A., Liu, C.-G., Sevignani, C., Ferracin, M., Felli, N., Dumitru, C. D.,\nShimizu, M., Cimmino, A., Zupo, S., Dono, M., et al. (2004) Proc. Natl. Acad.\nSci. USA 101, 11755–11760.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\n19. Meltzer, P. S. (2005) Nature 435, 745–746.\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.6**r== 0.1**\nPNAS  December 27, 2005  vol. 102  no. 52  19079\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n20. Johnson, S. M., Grosshans, H., Shingara, J., Byrom, M., Jarvis, R., Cheng,\nA., Labourier, E., Reinert, K. L., Brown, D. & Slack, F. J. (2005) Cell 120,\n635– 647.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n31. Krek, A., Grun, D., Poy, M. N., Wolf, R., Rosenberg, L., Epstein, E. J.,\nMacMenamin, P., da Piedade, I., Gunsalus, K. C., Stoffel, M. & Rajewsky, N.\n(2005) Nat. Genet. 37, 495–500.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n21. Iorio, M. V., Ferracin, M., Liu, C. G., Veronese, A., Spizzo, R., Sabbioni, S.,\nMagri, E., Pedriali, M., Fabbri, M., Campiglio, M., et al. (2005) Cancer Res. 65,\n7065–7070.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n32. Felli, N., Fontana, L., Pelosi, E., Botta, R., Bonci, D., Facchiano, F., Liuzzi, F.,\nLulli, V., Morsilli, O., Santoro, S., et al. (2005) Proc. Natl. Acad. Sci. USA 102,\n18081–18086.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n22. Lu, J., Getz, G., Miska, E. A., Alvarez-Saavedra, E., Lamb, J., Peck, D.,\nSweet-Cordero, A., Ebert, B. L., Mak, R. H., Ferrando, A. A., et al. (2005)\nNature 435, 834–838.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n33. Tamborini, E., Bonadiman, L., Negri, T., Greco, A., Staurengo, S., Bidoli, P.,\nPastorino, U., Pierotti, M. A. & Pilotti, S. (2004) Clin. Cancer Res. 10,\n8214–8219.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n23. He, L., Thomson, J. M., Hemann, M. T., Hernando-Monge, E., Mu, D.,\nGoodson, S., Powers, S., Cordon-Cardo, C., Lowe, S. W., Hannon, G. J. &\nHammond, S. M. (2005) Nature 435, 828–833.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n24. Takamizawa, J., Konishi, H., Yanagisawa, K., Tomida, S., Osada, H., Endoh,\nH., Harano, T., Yatabe, Y., Nagino, M., Nimura, Y., et al. (2004) Cancer Res.\n64, 3753–3756.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n25. Chan, J. A., Krichevsky, A. M. & Kosik, K. S. (2005) Cancer Res. 65, 6029–6033.\n26. Liu, C.-G., Calin, G. A., Meloon, B., Gamliel, N., Sevignani, C., Ferracin, M.,\nDumitru, C. D., Shimizu, M., Zupo, S., Dono, M., et al. (2004) Proc. Natl. Acad.\nSci. USA 101, 9740–9744.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n27. Rehmsmeier, M., Steffen, P., Hochsmann, M. & Giegerich, R. (2004) RNA 10,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n1507–1517.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n28. Huang, Y., Prasad, M., Lemon, W. J., Hampel, H., Wright, F. A., Kornacker,\nK., LiVolsi, V., Frankel, W., Kloos, R. T., Eng, C., et al. (2001) Proc. Natl. Acad.\nSci. USA 98, 15044–15049.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n34. Michael, M. Z., O’Connor, S. M., van Holst Pellekaan, N. G., Young, G. P. &\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nJames, R. J. (2003) Mol. Cancer Res. 1, 882–891.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n35. Eis, P. S., Tam, W., Sun, L., Chadburn, A., Li, Z., Gomez, M. F., Lund, E. &\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nDahlberg, J. E. (2005) Proc. Natl. Acad. Sci. USA 102, 3627–3632.\n36. Gillespie, J. W., Nasir, A. & Kaiser, H. E. (2000) In Vivo 14, 139–140.\n37. Huang, Y., de la Chapelle, A. & Pellegata, N. S. (2003) Int. J. Cancer 104,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n735–744.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n38. Morley, M., Molony, C. M., Weber, T. M., Devlin, J. L., Ewens, K. G., Spielman,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nR. S. & Cheung, V. G. (2004) Nature 430, 743–747.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n39. Ashman, L. K. (1999) Int. J. Biochem. Cell Biol. 31, 1037–1051.\n40. Kitamura, Y. & Hirotab, S. (2004) Cell Mol. Life Sci. 61, 2924–2931.\n41. Natali, P. G., Berlingieri, M. T., Nicotra, M. R., Fusco, A., Santoro, E., Bigotti,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.2**\nA. & Vecchio, G. (1995) Cancer Res. 55, 1787–1791.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n42. Tanaka, T., Umeki, K., Yamamoto, I., Kotani, T., Sakamoto, F., Noguchi, S.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\n& Ohtaki, S. (1995) Endocr. J. 42, 723–728.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n29. John, B., Enright, A. J., Aravin, A., Tuschl, T., Sander, C. & Marks, D. S. (2004)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n43. Arber, D. A., Tamayo, R. & Weiss, L. M. (1998) Hum. Pathol. 29,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nPloS Biol. 2, e363.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n498 –504.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n30. Lewis, B. P., Shih, I. H., Jones-Rhoades, M. W., Bartel, D. P. & Burge, C. B.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n44. Natali, P. G., Nicotra, M. R., Sures, I., Santoro, E., Bigotti, A. & Ullrich, A.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(2003) Cell 115, 787–798.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\n(1992) Cancer Res. 52, 6139–6143.",
         "extracted",
         "MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation.;MicroRNA gene expression deregulation in human breast cancer.;MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells.;miRNAs, Cancer, and Stem Cell Division;MicroRNA expression profiles classify human cancers;Cancer genomics: Small RNAs with big impacts;A microRNA polycistron as a potential human oncogene;Small regulatory RNAs in mammals.;Combinatorial microRNA target predictions;The RET/PTC-RAS-BRAF linear signaling cascade mediates the motile and mitogenic phenotype of thyroid cancer cells.;MicroRNAs: a developing story.;RAS Is Regulated by the let-7 MicroRNA Family;Accumulation of miR-155 and BIC RNA in human B cell lymphomas.;A new mechanism of BRAF activation in human thyroid papillary carcinomas.;Detection of Overexpressed and Phosphorylated Wild-Type Kit Receptor in Surgical Specimens of Small Cell Lung Cancer;Kit as a human oncogenic tyrosine kinase.;Oncogenic protein tyrosine kinases;Human MicroRNA Targets;Fast and effective prediction of microRNA/target duplexes.;Genetic analysis of genome-wide variation in human gene expression;MicroRNA profiling reveals distinct signatures in B cell chronic lymphocytic leukemias.;Annual report to the nation on the status of cancer, 1975–2001, with a special feature regarding survival;An oligonucleotide microchip for genome-wide microRNA profiling in human and mouse tissues.;Reduced Expression of the let-7 MicroRNAs in Human Lung Cancers in Association with Shortened Postoperative Survival;MicroRNAs Genomics, Biogenesis, Mechanism, and Function;Prediction of Mammalian MicroRNA Targets;BRAF mutations in thyroid tumors are restricted to papillary carcinomas and anaplastic or poorly differentiated carcinomas arising from papillary carcinomas.;Reduced accumulation of specific microRNAs in colorectal neoplasia.;Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma;BRAF mutation in papillary thyroid carcinoma.;High prevalence of BRAF mutations in thyroid cancer: genetic evidence for constitutive activation of the RET/PTC-RAS-BRAF signaling pathway in papillary thyroid carcinoma.;Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia;Environmental and heritable causes of cancer among 9.6 million individuals in the Swedish family‐cancer database;Gene expression in papillary thyroid carcinoma reveals highly consistent profiles;Localization of a susceptibility gene for familial nonmedullary thyroid carcinoma to chromosome 2q21.;Papillary thyroid carcinoma associated with papillary renal neoplasia: genetic linkage analysis of a distinct heritable tumor syndrome.;The biology of stem cell factor and its receptor C-kit.;A gene predisposing to familial thyroid tumors with cell oxyphilia maps to chromosome 19p13.2.;Paraffin section detection of the c-kit gene product (CD117) in human tissues: value in the diagnosis of mast cell disorders.;c-Kit proto-oncogene is more likely to lose expression in differentiated thyroid carcinoma than three thyroid-specific genes: thyroid peroxidase, thyroglobulin, and thyroid stimulating hormone receptor.;Transformation of thyroid epithelium is associated with loss of c-kit receptor.;Systematic population-based assessment of cancer risk in first-degree relatives of cancer probands.;Expression of c-kit receptor in normal and transformed human nonlymphoid tissues.;RNA silencing.;M  ED  IC  A  L  SC  IE  N  CE;Loss of heterozygosity in papillary and follicular thyroid carcinoma: a mini review.;de la Chapelle, A.;Age at diagnosis Gender Race Diagnosis",
         "The role of microRNA genes in papillary thyroid carcinoma.",
         "None"
        ],
        [
         "21",
         "001d36fbb5d1be8a5cb40d2df36bc05179f15ff4",
         "Over the past two decades, the role of apoptosis in the cytotoxicity of anticancer drugs has become clear. Apoptosis may occur via a death receptor‐dependent (extrinsic) or independent (intrinsic or mitochondrial) pathway. Mitochondria play a central role in cell death in response to DNA damage, and mediate the interaction(s) of various cytoplasmic organelles, including the endoplasmic reticulum, Golgi apparatus, and lysosomes. The mitochondrial pathway of cell death is mediated by Bcl‐2 family proteins, a group of antiapoptotic and proapoptotic proteins that regulate the passage of small molecules, such as cytochrome c, Smac/Diablo, and apoptosis‐inducing factor, which activates caspase cascades, through the mitochondrial transition pore. In addition, apoptosis can induce autophagic cell death via crosstalk between the two pathways upon treatment with anticancer drugs. The current review focused on recent advances surrounding the mechanism(s) of cell death induced by anticancer agents and discussed potential molecular targets for enhancing the chemotherapeutic effect(s) of anticancer agents. Cancer 2005. © 2005 American Cancer Society.",
         "R. Kim",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/cncr.20947",
         "\n**BLOCK**fs== 22.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nRecent Advances in Understanding the Cell Death\nPathways Activated by Anticancer Therapy\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nRyungsa Kim, M.D., Ph.D.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\nInternational Radiation Information Center, Re-\nsearch Institute for Radiation Biology and Medi-\ncine, Hiroshima University, Hiroshima, Japan.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nAddress for reprints: Ryungsa Kim, M.D., Ph.D.,\nInternational Radiation Information Center, Re-\nsearch Institute for Radiation Biology and Medi-\ncine, Hiroshima University, 1-2-3 Kasumi Minami-\nku, Hiroshima 734-8553, Japan; Fax: (011) 81-82-\n257-5877; E-mail: rkim@hiroshima-u.ac.jp\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nReceived October 26, 2004; revision received De-\ncember 3, 2004; accepted December 10, 2004.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.1**\nOver the past two decades, the role of apoptosis in the cytotoxicity of anticancer\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.1**\ndrugs has become clear. Apoptosis may occur via a death receptor-dependent\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.1**\n(extrinsic) or independent (intrinsic or mitochondrial) pathway. Mitochondria play\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\na central role in cell death in response to DNA damage, and mediate the interac-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\ntion(s) of various cytoplasmic organelles, including the endoplasmic reticulum,\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\nGolgi apparatus, and lysosomes. The mitochondrial pathway of cell death is me-\ndiated by Bcl-2 family proteins, a group of antiapoptotic and proapoptotic proteins\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\nthat regulate the passage of small molecules, such as cytochrome c, Smac/Diablo,\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\nand apoptosis-inducing factor, which activates caspase cascades, through the\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\nmitochondrial transition pore. In addition, apoptosis can induce autophagic cell\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.1**\ndeath via crosstalk between the two pathways upon treatment with anticancer\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.1**\ndrugs. The current review focused on recent advances surrounding the mecha-\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.1**\nnism(s) of cell death induced by anticancer agents and discussed potential molec-\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.1**\nular targets for enhancing the chemotherapeutic effect(s) of anticancer agents.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.2**\nCancer 2005;103:1551– 60. © 2005 American Cancer Society.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.4**r== 0.2**\nKEYWORDS: anticancer drug, apoptosis, autophagy, cell death.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.4**r== 0.1**\nP rogrammed cell death occurs during organ development, and\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.4**r== 0.1**\nplays an important role in cellular homeostasis.1 Three types of\ncell death are known: apoptosis (type I), autophagy (type II), and\nnecrosis (type III).2 Apoptosis is induced by a number of stimuli,\nincluding anticancer drugs, growth factor deprivation, and irradia-\ntion. The molecular mechanism by which apoptosis occurs has been\nextensively examined. Apoptosis in response to chemotherapy is\nknown to correlate with a reduction in tumor volume, and resistance\nto apoptosis causes drug resistance.3–5 Introduction of a proapoptotic\ngene such as Bax,6,7 and inhibition of an antiapoptotic gene such as\nBcl-2,8,9 enhances the effects of chemotherapy. Thus, apoptosis\nclearly inﬂuences the antitumor effect(s) of chemotherapeutic agents.\nIn solid tumors, hypoxia and glucose deprivation enable tumor\nprogression,10 whereas depletion of adenosine 5-triphosphate (ATP)\nresults in necrosis. The production of lactic acid under anaerobic\nconditions does not necessarily induce necrosis, rather, autophagic\ncell death may depend on susceptibility to apoptosis, which can be\ninhibited by activation of Akt by environmental stimuli.10,11 Drug\nresistance might involve resistance to autophagic cell death, as well as\napoptosis. In fact, treatment with tamoxifen causes autophagy of\nMCF-7 breast carcinoma cells, as well as apoptotic cell death via a\ncaspase 3-independent pathway.12 Autophagy is also induced by au-\nrintricarboxylic acid via activation of the ERK1/2 signaling pathway in\nRas-mutated HT-29 colon carcinoma cells.13 In addition, a human\ntumor suppressor gene, known as Beclin 1, induces autophagy.14,15\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\n© 2005 American Cancer Society\nDOI 10.1002/cncr.20947\nPublished online 1 March 2005 in Wiley InterScience (www.interscience.wiley.com).\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nReduced expression of this gene occurs in breast and\novarian carcinoma due to a 40 –75% deletion of the\ngene.16 Although resistance to autophagy and apopto-\nsis is likely involved in tumor progression and acqui-\nsition of drug resistance by solid tumors, the exact\nmolecular mechanism(s) by which this occurs, and the\nrelation between autophagy and apoptosis, remain\nunknown. Here, we review evidence regarding the\nmolecular mechanism(s) of anticancer drug-induced\ncell death and discuss potential therapeutic targets by\nwhich signal transduction pathways leading to cell\ndeath might be modulated by cancer therapy.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.3**l== 0.1**r== 0.5**\nAPOPTOTIC PATHWAYS OF CELL DEATH\nCaspase-Dependent Pathway\nGiven that a number of anticancer agents induce ap-\noptosis, a lot of research has focused on the various\npathways that mediate apoptosis. Two are known to\nmediate anticancer drug-induced apoptosis, a death\nreceptor-dependent (extrinsic) and a mitochondria-\ndependent (intrinsic) pathway.17 The death receptor-\ndependent pathway involves activation of death re-\nceptors, such as Fas and TRAIL receptors (DR4, DR5).\nDeath receptor activation is mediated by a death-\ninducing signaling complex, which is formed by re-\ncruitment of a Fas-associated death domain and pro-\ncaspase 8 to the death receptor, thereby activating\ncaspase 8.3,18 Caspase 8 directly activates caspase 3,\nleading to apoptosis (type I cell death).19 The mito-\nchondria-dependent pathway involves cleavage of a\nproapoptotic protein, Bid, resulting in the production\nof truncated Bid (tBid), by caspase 8, within the mito-\nchondria. Heterodimerization of tBid and Bak results\nin release of cytochrome c from mitochondria and\nactivation of caspase 3 and caspase 9 (type II cell\ndeath).19 The predominance of one pathway over an-\nother depends on cell type.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nUnlike the death receptor-dependent pathway,\nthe mitochondria-dependent pathway is mediated by\nBax/Bak and involves the release of cytochrome c.3,18\nIn general, Bax is associated with 14-3-3 protein,\nwhich anchors Bax in the cytoplasm. However, in the\npresence of DNA damage, c-Jun NH(2)-terminal ki-\nnase phosphorylates 14-3-3, resulting in dissociation\nof Bax from this protein.20 Homodimerization of Bax,\nor heterodimerization of Bax ank Bak, results in trans-\nlocation of Bax from the cytoplasm to mitochondria.21\nBax homodimers and heterodimers interact with a\nvoltage-dependent anion channel (VDAC) in the mi-\ntochondrial outer membrane to release cytochrome c\nby increasing mitochondrial membrane permeability\n(MMP) via opening of the mitochondrial permeability\ntransition pore.22 The mitochondrial permeability\ntransition pore complex (PTPC) mediates permeability\n**BLOCK**fs== 9.5**b== 0.1**t== 0.2**l== 0.5**r== 0.1**\nof the mitochondrial membrane during apoptosis, ne-\ncrosis, and autophagy. PTPC is composed of an ade-\nnine nucleotide translocator and VDAC, both of which\nare targeted by a variety of proapoptotic inducers. The\nrelease of cytochrome c results in the formation of an\napoptosome, consisting of apoptosis activating factor\n1 and procaspase 9, after which dATP activates\ncaspase 9, leading to apoptosis.23 The activated initi-\nator caspase, caspase 9, leads to activation of one of\nthe effector caspases, caspase 3 or caspase 7, which\ncleaves an inhibitor of caspase-activated DNAase and\nproduces poly (ADP-ribose) polymerase (PARP), re-\nsulting in DNA fragmentation. The release of cyto-\nchrome c through the VDAC-mediated permeability\ntransition pore is inhibited by Bcl-2 and Bcl-xL. In\naddition to cytochrome c, Smac/Diablo is released\nfrom mitochondria when there is a loss of membrane\npotential. Smac/Diablo binds to X chromosome-en-\ncoded IAP, cellular IAP-1 (cIAP-1), and cellular IAP-2,\nas well as survivin, to inhibit apoptosis,24 and it di-\nrectly activates caspase 3. Inhibitors of apoptosis pro-\nteins (IAPs) block apoptosis by binding to and inhib-\niting caspases, as well as by activating caspase-\nindependent mechanisms.25\ninhibit\nactivation of executioner caspases activated by extrin-\nsic or intrinsic pathways. The Smac/Diablo complex\ninduces apoptosis via the apoptosome-dependent\npathway,26 and release of Smac/Diablo is inhibited by\nBcl-2 and Bcl-xL.27 There is also a report that Smac\ninduces cytochrome c release and apoptosis in the\nabsence of Bax/Bcl-xL via activation of caspase 3 in\nhuman HCT116 and DU145 colon carcinoma cells.28\nSmac is capable of circumventing defects in mito-\nchondrial apoptosis signaling, such as loss of Bax or\noverexpression of Bcl-xL, which frequently occurs in\ntumor cells resistant to anticancer therapy. A recent\nstudy suggests that an IAP family protein, Apollon,\nbinds to, ubiquitinates, and facilitates the proteasomal\ndegradation of Smac and caspase 9, thereby prevent-\ning Smac-induced apoptosis.29 The role of Apollon in\ndrug resistance remains to be clariﬁed. Translocation\nof endogenous Smac into the cytosol, and release of\nSmac/Diablo during anticancer drug-induced apopto-\nsis, does not appear to play a major role in cell death\nafter treatment of human lung carcinoma with etopo-\nside,30 because cytochrome c and mitochondrial pro-\ntease Omi/HtrA2 are still detectable in the cytosol in\nthe absence of Smac. Thus, the signiﬁcance of Smac/\nDiablo in anticancer drug-induced mitochondrial\npathway-mediated apoptosis remains unclear.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nIAPs\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\ncan\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nCaspase-Independent Pathway\nAlthough caspase cysteine proteases execute apopto-\nsis, a mitochondrial serine protease released into the\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nFIGURE 1. The role of mitochondria in\ncell death. Mitochondria play a central\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nrole in regulating the intrinsic and ex-\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\ntrinsic apoptotic pathway responses to\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nDNA damage. Apoptotic cell death can\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nbe induced by a number of\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\ntriggers,\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nresulting in the release of cytochrome c,\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nSmac/Diablo, and apoptosis-inducing\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\nfactor (AIF), from the mitochondrial outer\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nmembrane,\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\ninducing caspase-depen-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\ndent and independent pathways of cell\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\ndeath. The release of these small mol-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\necules\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nfrom mitochondria\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\ncan\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nbe\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nblocked by overexpression of Bcl-2.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nPARP-1: poly\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n(ADP-ribose) polymer-\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nase-1; STS: staurosporine; ICAD: inhibi-\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\ntor of caspase-activated DNAase; PK:\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nprotein kinase.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\ncytosol during apoptosis, Omi/HtrA2, is required to\nantagonize inhibitors of apoptosis (IAPs), thereby en-\nabling apoptosis to occur via an indirect mechanism.31\nCleavage of c-IAP-1 by Smac/Diablo and Omi/HtrA2 is\nirreversible, thus, signiﬁcant inactivation of IAPs oc-\ncurs, enabling increased caspase activity after tumor\nnecrosis factor apoptosis related ligand (TRAIL)-in-\nduced apoptosis.32 IAP cleavage by Omi is indepen-\ndent of caspase activity. In addition, Omi/HtrA2 might\nactivate more than one pathway of caspase activation\nbecause extramitochondrial expression of Omi-HtrA2\nresults in increased mitochondrial permeability and\ncytochrome c-induced caspase activation in HeLa\ncells.33\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nApoptosis-inducing factor (AIF) is a phylogeneti-\ncally conserved ﬂavoprotein within the mitochondrial\nmembrane, which has the ability to induce apoptosis\nvia a caspase-independent pathway.34 AIF induces nu-\nclear chromatin condensation and large-scale DNA\nfragmentation (producing approximately 50-kilobase\n[kb] fragments), and is essential for programmed cell\ndeath. On lethal signaling, AIF translocates through\nthe cytosol to the nucleus where it binds to DNA and\nprovokes caspase-independent chromatin condensa-\ntion. Responses to injection of anti-AIF antibodies, or\nknockout of the AIF gene, suggest that AIF may be\nrequired for cell death in response to certain stimuli.35\nStaurosporine (STS) induces mitochondrial dysfunc-\ntion and translocation of AIF into the nucleus after\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nactivation of nuclear apoptosis in nonsmall cell lung\ncarcinoma cells resistant to radiotherapy or chemo-\ntherapy.36 This suggests that, although caspase-de-\npendent and independent pathways coexist,\nthe\ncaspase-dependent pathway might be less efﬁcient.\nFurthermore, resistant cells might be made more sus-\nceptible to treatment by activation of the AIF-medi-\nated caspase-independent pathway. Moreover, recent\nreports indicate that AIF is necessary for PARP-1– de-\npendent cell death.37,38 PARP-1 generates a number of\nlong, branched PARPS after DNA damage. Activation\nof PARP-1 initiates a nuclear signal that triggers the\nrelease of AIF from mitochondria. AIF then shuttles\nfrom mitochondria to the nucleus and induces periph-\neral chromatin condensation and large-scale DNA\nfragmentation.38 Endonuclease G (Endo G) is a nucle-\nar-encoded mitochondrial protein believed to be im-\nportant for nuclear DNA fragmentation during apo-\nptosis, as well as mitochondrial DNA replication. Cells\nfrom Endo G heterozygous mutant mice are resistant\nto tumor necrosis factor alpha and STS-induced cell\ndeath.39\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThus, increased MMP in response to DNA damage\nresults in the release of a number of small molecules,\nsuch as cytochrome c, Smac/Diablo, and AIF from\nmitochondria. This results in activation of caspase-\ndependent and independent apoptotic pathways, de-\npending on which triggers and cell types are involved.\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nOur current understanding regarding the role of mi-\ntochondria in cell death is summarized in Figure 1.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.2**l== 0.1**r== 0.5**\nAUTOPHAGIC PATHWAY OF CELL DEATH\nAutophagic cell death, otherwise known as type II cell\ndeath, is characterized by the appearance of large auto-\nphagic vacuoles in the cytoplasm. It occurs during em-\nbryogenesis and differs from type I programmed cell\ndeath.40 Autophagy is a bulk protein degradation system\nthat is essential for normal cell activity and survival when\nnutrients are scarce.41 The ﬁrst step that occurs in auto-\nphagic cell death is the formation of a double-mem-\nbrane vacuole, the autophagosome, which is derived\nfrom part of the endoplasmic reticulum42 or from the\ncytoplasmic lipid pool.43 The autophagosome fuses with\na lysosome, after which its contents are degraded by\nlysosomal hydrolytic proteases. Despite recent advances\nregarding the molecular mechanism(s) underlying auto-\nphagy, many details remain poorly understood. Of note,\nclass I phosphoinositide 3-kinase (PI 3-kinase) inhibits,\nwhereas Class III PI 3-kinase enhances, autophagy in\nHT-29 human colon carcinoma cells.44 Class I PI 3-ki-\nnase mediates various cellular functions, including insu-\nlin and glycogen synthesis, and suppresses apoptosis\nthrough the Akt-signaling pathway, whereas Class III PI\n3-kinase inﬂuences membrane trafﬁcking. Despite the\nﬁnding that insulin-like growth factor I (IGF-I) normally\ninhibits apoptosis, IGF-I accelerates cell death when glu-\ncose and amino acids are limited, leading to accumula-\ntion of autophagic vacuoles within the cytoplasm and\nevidence of chromosome condensation.40 Autophagic\ncell death is inhibited by 3-methyladenine (3-MA) and\nLY294002, which inhibit PI 3-kinase, but not by the pan-\ncaspase inhibitor, z-VAD-fmk. Of importance, LY294002\nhas no effect on cell death in PC12 and HepG2 carci-\nnoma cells during glucose deprivation,45 suggesting that\ninterference with the PI 3-kinase/Akt signaling pathway\ncauses autophagy of cancer cells but not normal cells.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nA mammalian gene capable of inducing autoph-\nagy, Beclin 1, is a component of the PI 3-kinase com-\nplex responsible for autophagic vesicle nucleation.16\nThis gene is deleted in 75% of ovarian, 50% of breast,\nand 40% of prostate carcinoma cases.16 Frequent al-\nlelic deletions are observed in breast carcinoma cell\nlines in association with signiﬁcant decreases in Beclin\n1 protein levels. Beclin 1 functions as a tumor sup-\npressor, because there is an attenuation of MCF-7–\nmediated tumorigenicity in nude mice overexpressing\nBeclin 1.46 Although Beclin 1 is known to interact with\nBcl-2 in yeast, the relation between this association\nand tumor suppression and autophagy is not yet clear.\nDownregulation of Bcl-2 by antisense Bcl-2 in HL-60\ncells induces autophagy independent of caspase-de-\npendent and mitochondria-dependent pathways,47\n**BLOCK**fs== 9.5**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\nsuggesting a potential role of Bcl-2 in autophagic cell\ndeath, as well as apoptosis. Other autophagy-inducing\ngenes, including bridging integrator 1 (Bin 1) and\ndeath-associated protein kinase (DAPK), exist. Bin 1\ninteracts with c-Myc and inhibits its transformation\nactivity.48 Bin 1 is regulated by alternate splicing, and\nthe splice forms that interact with c-Myc have tumor\nsuppressor properties. This gene is frequently missing\nor functionally inactivated in malignant melanoma,\nbreast, and prostate carcinoma cells, and loss of Bin 1\nexpression may promote tumor progression, while\nlimiting susceptibility to farnesyltransferase inhibitors\n(FTIs). Bin 1 adapter proteins act downstream or par-\nallel to RhoB to exert their effect on cell signaling.49\nDAPK is a calcium-regulated serine/threonine kinase,\nand an important mediator of autophagy induced by\nnutrient deprivation, antiestrogens, and tumor necro-\nsis factor.50 Cancer and cancer cell lines frequently\ndemonstrate decreased DAPK expression due to DNA\nmethylation.51 Transfection of Beclin 1, Bin 1, or an\nactive form of DAPK, into cancer cells induces auto-\nphagic cell death. A human homolog of the Drosophila\nspin gene product (HSpin 1) exists, which binds to\nBcl-2 and Bcl-xL. HSpin 1-induced cell death results in\nformation of autophagic vacuoles and release of a\nmature form of cathepsin D, suggesting a novel\ncaspase-independent cell death, pathway resulting in\nautophagy.52\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nThe tumor suppressor phosphatase and tensin\nhomolog deleted on chromosome10 (PTEN), which is\nan inhibitor of the PI 3-kinase/Akt signaling pathway,\npromotes autophagy in HT-29 colon carcinoma\ncells.53 PTEN PI phosphatase activity inhibits Akt ac-\ntivity, resulting in induction of autophagy. Loss of\nfunction of PTEN increases the expression of Class I PI\n3-kinase and Akt in human cancer cells. Activation of\nAkt and constitutive expression of active Akt block\nautophagic cell death, leading to inhibition of apopto-\nsis and activation of a mammalian target of rapamycin\n(mTor) protein. It is likely that activation of Akt and\nthe mTor signaling pathway, due to loss of PTEN,\ncontributes to malignant transformation by simulta-\nneous inhibition of autophagic and apoptotic cell\ndeath. Another oncoprotein, Ha-Ras, induces autoph-\nagic cell death in glioma and gastric carcinoma cells54\nvia a p53-independent pathway that is not blocked by\nBcl-2. Another Ras family, including K-Ras and N-Ras\nproteins, also induces autophagic cell death.\nIn-\ncreased Ras expression is observed in spontaneous\nregression of neuroblastoma, which is characterized\nby autophagic cell death and a lack of apoptosis.55 In\naddition, Ha-Ras overexpression induces autophagic\ncell death in neuroblastoma cell\nlines. Ha-Ras–in-\nduced autophagy in HT-29 colon carcinoma cells has\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nFIGURE 2. A proposed mechanism for regulation of autophagic cell death. A mammalian gene capable of inducing autophagy, Beclin 1, is activated via a Class\nIII phosphoinositide 3-kinase (PI 3-kinase)– dependent pathway, which is stimulated by lysosomal activation, ceramide, and Gi3 protein. A number of other genes,\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nincluding Bin 1, DAPK, and HSpin 1 genes, are also involved in autophagic cell death. Autophagy can be induced by antimicrotubule agents, such as vinblastine\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nand paclitaxel, as well as by irradiation. A tumor suppressor gene, PTEN, induces autophagic cell death, and interference with PTEN expression results in activation\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nof the Class I PI 3-kinase/Akt pathway, which inhibits apoptosis. Activation of the PI 3-kinase/Akt signaling pathway leads to activation of the mammalian target\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nof rapamycin (mTor) through PDK1, which is inhibited by rapamycin, and induces autophagic cell death. Autophagic cell death is inhibited by 3-methyladenine\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n(3-MA), and treatment with PI 3-kinase inhibitors, such as wortmannin and LY294002, also inhibits autophagy, even if apoptosis is activated. There is likely some\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\noverlap in the pathways mediating autophagy and apoptosis. However, further research is required to clarify the molecular mechanism(s) underlying these pathways.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nFTI: farnesyltransferase inhibitors; IGF-IR: insulin-like growth factor-I receptor.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nsome overlap with the Raf 1/ERK signaling pathway of\nautophagic cell death.56 The ability of Ras to cause\nautophagy might involve activation of ErbB2 and\nErbB3. It is interesting to note that one of the signaling\npathways regulated by Ras, the Raf1/ERK pathway,\ntriggers autophagy, whereas the PI 3-kinase/Akt path-\nway, which is downstream of Ras signaling, inhibits it.\nRas-induced increases in the expression and trafﬁck-\ning of lysosomal cathepsins might contribute to Ras-\ninduced autophagy. A proposed molecular model of\nautophagy is illustrated in Figure 2.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nA death-promoting “BH3-only” family protein, Bcl-2/\nadenovirus E1B 19-kilodalton interacting protein 3\n(BNIP3), differs from other proteins of this class.\nBNIP3 can directly integrate into the mitochondrial\nouter membrane via its BH3 domain in the absence of\nBax/Bak, thereby disrupting the MMP, generating re-\nactive oxygen species (ROS) and promoting autoph-\nagic degeneration.57 Overexpression of BNIP3, a hy-\npoxia-inducible protein with a BH3 domain, results in\nautophagic cell death in cancer cell lines, after cyto-\nchrome c or AIF nuclear translocation. BNIP3-induced\nautophagy can be attenuated by overexpression of Bcl-2,\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nwhich interacts with BNIP3 protein. Autophagic cell\ndeath can be activated in cancer cells in response to\nvarious anticancer drugs. Treatment of MCF-7 breast\ncarcinoma cells with tamoxifen induces autophagy be-\nfore apoptotic cell death.12 Treatment with estradiol and\n3-MA inhibits tamoxifen-induced cell death. Similarly,\nother anticancer drugs, such as vinblastin, paclitaxel,\ninduce autophagic cell death.58 – 60\nand rapamycin,\nTreatment of malignant glioma cells with arsenic triox-\nide induces autophagic cell death in association with\nG2/M arrest.61 In addition, radiotherapy induces auto-\nphagic cell death in several cancer cell lines, including\nbreast, colon, and prostate carcinoma, as well as glio-\nblastoma multiforme.62 Despite several reports of induc-\ntion of autophagy by treatment with anticancer drugs\nand/or irradiation, the clinical signiﬁcance of autophagy\nas a result of chemotherapy remains unclear.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nINTERACTION BETWEEN APOPTOTIC AND\nAUTOPHAGIC PATHWAYS OF CELL DEATH\nDifferent types of cell death differ in their morphologic\ncharacteristics (Fig. 3 and Table 1). In fact, ceramide,\nwhich is believed to activate apoptosis, also induces\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nFIGURE 3. Schematic illustration of\nthe different types of cell death. Cell\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\ndeath can be classiﬁed as controlled or\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nuncontrolled, referring to programmed\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nor necrotic cell death. Programmed cell\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\ndeath includes apoptosis, and autoph-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.7**r== 0.1**\nagy, among other types of cell death.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\nApoptosis usually occurs through a\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\ncaspase-dependent pathway. However,\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\nit sometimes occurs via an alternative\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\npathway, which overlaps with that of\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\nautophagy. MMP: matrix metallopro-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nteases.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nTABLE 1\nDifferences in Characteristics between Type I and Type II Cell Death\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nFactor\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nMorphology\nNucleus\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nCytoplasm\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.2**r== 0.7**\nMembrane\nPrimary proteases\nATP requirement\nInhibition\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nDetection\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nApoptosis (type I)\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nAutophagy (type II)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.5**l== 0.3**r== 0.5**\nChromatin condensation\nDNA laddering and fragmentation\nPyknosis\nCytoplasmic condensation\nFragmentation to apoptotic bodies\nIncrease in MMP\nActivation of caspase cascade\nPotential release of lysosomal enzyme\nOrganelles do not swell\nBlebbing\nCaspases such as caspase 3\nYes\nz-VAD-fmk\nXIAP\nBcl-2/Bcl-xL\nSometimes with actinomycin D\nSometimes with cyclophosphamide\nDNA laddering test\nCaspase activation and the substrate\nTUNEL and annexin V staining\nFACS analysis\nElectron microscopy\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.6**r== 0.2**\nPartial chromatin condensation\nNo DNA laddering and fragmentation\nSometimes pyknosis\nMany large autophagic vacuoles\nMany autophagosomes\nLysosomal activation\nCaspase independent\nPotential involvement of MMP\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.6**r== 0.2**\nBlebbing\nCathepsins and proteasomal proteins\nYes (AMPK, AMP/ATP ratio)\n3-methyladenine (3-MA)\nPI3K inhibitors\nPI3K-I/Akt\nActinomycin D\nCyclophosphamide\nElectron microscopy\nLysosome activity test\nCytoplasmic sequestration test\nLC3 associated with autophagosome membrane\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\nMMP: mitochondrial membrane permeability; XIAP: X chromosome-encoded inhibitors of apoptosis proteins; PI3K: phosphoinositide 3-kinase; TUNEL: terminal\ndeoxynucleotidyl transferase nick-end labeling; FACS: ﬂuorescence-activated cell sorting; AMPK: AMP-activated protein kinase; AMP: adenosine 5-monophosphate;\nATP: adenosine 5-triphosphate.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nautophagy.63 Similarly, proteins like DAPK are capable\nof inducing both apoptosis and autophagy depending\non the cell type involved, suggesting that they may\nfunction as molecular switches or integrators of these\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\ntwo types of programmed cell death. Commitment to\nautophagy or apoptosis may depend on the cell type\ninvolved, as well as the microenvironment. Depending\non the setting, initiation of autophagy may induce,\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\ndelay, or antagonize apoptosis. Three scenarios must\nbe considered when contemplating the interaction be-\ntween apoptosis and autophagic cell death.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nFirst, autophagy may precede apoptosis. Treat-\nment of sympathetic neurons with cytosine arabino-\nside, thereby depleting neural growth factor, results in\na 30-fold increase in autophagy, followed by DNA\ndegradation and apoptosis.64 Delayed induction of ap-\noptosis occurs with 3-MA, which inhibits cytochrome\nc release and caspase activation. These ﬁndings may\nmodel cell death after treatment with anticancer drugs\nor growth factor deprivation.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\nSecond, autophagy may antagonize apoptosis.\nConversely, a reduction in autophagy might increase\nthe susceptibility of cells to apoptotic stimuli. Accel-\nerated sulindac sulﬁde-induced apoptosis occurs in\nmutant clone HT-29 colon carcinoma cells, which lack\nautophagic capability due to overexpression of a mu-\ntant G (alpha i3) protein.65 In contrast, treatment of\nparent HT-29 cells with 3-MA increases sensitivity to\nsulindac sulﬁde-induced apoptosis. Even though\nthere are no differences in the expression patterns of\nCOX-2, Bcl-2, Bcl-xL, Bax, and Akt/PKB among mutant\nand parent HT-29 cells, greater cytochrome c release\noccurs in mutant G (alpha i3) protein overexpressing\ncells, compared with parent cells. It is possible that\nautophagy retards apoptosis in colon carcinoma cells\nby sequestering mitochondrial death-promoting fac-\ntors, such as cytochrome c.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nThird, apoptosis and autophagy may be mutually\nexclusive. Inhibition of autophagy may lead to apo-\nptotic cell death. A recent report reveals that treat-\nment of malignant glioma cells with arsenic trioxide\ninduces autophagic cell death. However, inhibition of\nautophagy by treatment with baﬁlomycin, a H-AT-\nleads to apoptosis.61 However, al-\nPase inhibitor,\nthough treatment of neurons and HeLa cells with a\npancaspase inhibitor prevents apoptosis, selective mi-\nimpairment results in autophagy.66 In\ntochondrial\nlight of several reports indicating that treatment with\ncaspase inhibitors causes a shift from apoptotic cell\ndeath to autophagy, it is possible that this also occurs\nunder other circumstances. For example, inhibition of\napoptosis in Bax/Bak double-deﬁcient DKO cells in-\nduces autophagic cell death.67 The ability of cells to\nrevert between apoptosis and autophagy might be\nclinically signiﬁcant because drug resistance to apo-\nptosis might be overcome by activation of autophagic\ncell death. Along this line, multidrug-resistant MCF-7\ncells can be made susceptible to doxorubicin by treat-\nment with verapamil, after which autophagic cell\ndeath occurs, suggesting that the autophagic cell\ndeath pathway remains viable after inhibition of the\napoptotic pathway (unpublished data). The molecular\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nmechanism(s) by which reversion from apoptosis to\nautophagy occurs remains unclear. Although overex-\npression of Bcl-2 or Bcl-xL may inhibit autophagic cell\ndeath, the role of these antiapoptotic proteins in au-\ntophagic cell death requires further clariﬁcation.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.2**l== 0.5**r== 0.1**\nThe role of Bcl-2 in apoptosis in response to mi-\ntochondrial damage is well established. However, sig-\nnaling as a result of mitochondrial damage might also\nactivate the autophagic pathway of cell death. It ap-\npears that mitochondria are involved in integrating\nthe apoptotic and autophagic pathways of cell death.\nLysosomal enzymes might activate apoptosis because\nlysosome rupture activates apoptosis. It is interesting\nto note that apoptosis after lysosome rupture is pre-\nceded by autophagy. Sequestration of mitochondria\nwithin autophagic vacuoles might protect cells from\napoptosis by delaying the release of cytochrome c.\nThus, autophagy might regulate apoptosis through ly-\nsosome formation and enzyme activity. With regard to\nthe possibility of signaling between lysosomes and\nmitochondria, a recent study reveals that production\nof H2O2 from ROS in response to DNA damage acti-\nvates lysosomal enzymes, such as cathepsin D and B.68\nCathepsin B directly or indirectly results in transloca-\ntion of proapoptotic proteins, such as Bid and Bax,\nfrom the cytoplasm to mitochondria. This alters MMP\nand results in caspase activation subsequent to the\nrelease of cytochrome c, Smac/Diablo, and AIF from\nthe mitochondria. Conversely, under different circum-\nstances, production of H2O2 from ROS in response to\nDNA damage may inhibit the release of small mole-\ncules, such as cytochrome c, Smac/Diablo, and AIF,\nfrom the mitochondrial membrane and activate lyso-\nsomal activity. Thus, crosstalk might occur between\nthe pathways mediating autophagy and apoptosis by\nway of lysosomes and mitochondria. Further research\nis required to clarify the signaling that occurs between\nthese pathways.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nFUTURE PERSPECTIVES REGARDING CELL DEATH\nAND CANCER THERAPY\nIncreasing information is known regarding the molec-\nular mechanism(s) by which cell death is mediated by\nanticancer drugs. Proapoptotic and antiapoptotic pro-\nteins are now being targeted to enhance the effect(s)\nof chemotherapy. In general, the antitumor effect of\nan anticancer agent is measured in terms of its ability\nto induce apoptotic cell death within tumors, which is\nwell correlated with outcome in most preclinical and\nclinical studies. Nevertheless, despite signiﬁcant re-\nductions in tumor volume, high rates of apoptosis do\nnot always occur within solid tumors after chemother-\napy. This suggests that other types of cell death, such\nas autophagy and necrosis, might also occur as a result\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nof chemotherapy. Research into other types of cell\ndeath, particularly autophagic cell death, is important\nto better understand the effects of anticancer drugs. In\naddition, the relation between autophagic cell death\nand apoptosis needs to be examined, because several\nautophagy-related proteins, including Beclin 1, BNIP3,\nDAPK, and HSpin 1, may inﬂuence apoptosis. Because\napoptosis and autophagy are mutually exclusive in\nsome cell lines, molecular switches between the two\nprocesses must exist.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\nResistance to autophagy, as well as apoptosis,\nmight lend resistance to anticancer drugs. Likewise,\ninduction of autophagic cell death in cancer cells may\nbe of therapeutic beneﬁt. Cells transformed with the\ntumor suppressor gene Beclin 1 are less likely to un-\ndergo autophagy and are more susceptible to onco-\ngenesis. Alterations in autophagic genes reduce the\nincidence of autophagy in cancer cells. However, even\nin cells resistant to apoptosis or cells in which apo-\nptosis is inhibited, nonapoptotic cell death pathways,\nsuch as autophagy, may be activated. Targeting alter-\nnative pathways of cell death might overcome drug\nresistance or explain why the effects of chemotherapy\nand irradiation complement each other in the clinical\nsetting. Alterations or defects in apoptotic signal\ntransduction pathways might result in drug resistance,\nhowever, irradiation induces autophagic cell death.\nAlterations of one or both pathways (autophagic or\napoptotic) might determine susceptibility to cell death\nafter chemotherapy because anticancer drugs might\ntrigger both apoptotic caspase-dependent and auto-\nphagic caspase-independent pathways at the same\ntime.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nAutophagic cell death might also inhibit angio-\ngenesis.69 The angiogenesis inhibitor, endostatin, in-\nduces endothelial cell death via the caspase-indepen-\ndent autophagic pathway. In addition, the tumor\nsuppressor gene, PTEN, which induces autophagy, in-\nhibits tumor angiogenesis in nude mice with ortho-\ntopic brain tumors.70 Reconstitution of wild-type\nPTEN has no effect on in vitro proliferation, but dra-\nmatically decreases tumor growth in vivo and pro-\nlongs survival in mice, suggesting that PTEN regulates\ntumor-induced angiogenesis via regulation of PI-de-\npendent signals. The role of autophagy in tumor an-\ngiogenesis might be an important means of inhibiting\ntumor progression and increasing the effectiveness of\nanticancer therapy.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nIt is now clear that a number of different types of\ncell death, including apoptosis and autophagy, play a\nrole in the antitumor effect(s) of treatment with anti-\ncancer drugs. Targeting therapy at genes related to\napoptosis might increase the therapeutic efﬁcacy of\nanticancer agents by modulating signal transduction\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\npathways, and other cell death pathways might be\ninﬂuenced by changes in the microenvironment and\nby crosstalk between the various cell death pathways.\nFurther research into the molecular mechanism(s) be-\nhind autophagic cell death as a result of chemother-\napy might provide a new strategy for overcoming drug\nresistance by exploring this alternative pathway of cell\ndeath.\n**BLOCK**fs== 11.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nREFERENCES\n1.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nJaattela M. Multiple cell death pathways as regulators of\ntumour initiation and progression. Oncogene. 2004;23:2746 –\n2756.\nLockshin RA, Zakeri Z. Apoptosis, autophagy, and more. Int\nJ Biochem Cell Biol. 2004;36:2405–2419.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n2.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n3. Kim R, Tanabe K, Uchida Y, Emi M, Inoue H, Toge T.\nCurrent status of the molecular mechanisms of anticancer\ndrug-induced apoptosis. The contribution of molecular-\nlevel analysis to cancer chemotherapy. Cancer Chemother\nPharmacol. 2002;50:343–352.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n4. Kim R, Tanabe K, Emi M, Uchida Y, Inoue H, Toge T.\nInducing cancer cell death by targeting transcription fac-\ntors. Anticancer Drugs. 2003;14:3–11.\nPommier Y, Sordet O, Antony S, Hayward RL, Kohn KW.\nApoptosis defects and chemotherapy resistance: molecular\ninteraction maps and networks. Oncogene. 2004;23:2934 –\n2949.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n5.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n6. Kim R, Inoue H, Toge T. Bax is an important determinant for\nradiation sensitivity in esophageal carcinoma cells. Int J Mol\nMed. 2004;14:697–706.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n7. Tsunemitsu Y, Kagawa S, Tokunaga N, et al. Molecular ther-\napy for peritoneal dissemination of xenotransplanted hu-\nman MKN-45 gastric cancer cells with adenovirus mediated\nBax gene transfer. Gut. 2004;53:554 –560.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n8. Tanabe K, Kim R, Inoue H, Emi M, Uchida Y, Toge T.\nAntisense Bcl-2 and HER-2 oligonucleotide treatment of\nbreast cancer cells enhances their sensitivity to anticancer\ndrugs. Int J Oncol. 2003;22:875– 881.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n9. Tolcher AW, Kuhn J, Schwartz G, et al. A phase I pharma-\ncokinetic and biological correlative study of oblimersen so-\ndium (genasense, G3139), an antisense oligonucleotide to\nthe bcl-2 mRNA, and of docetaxel in patients with hormone-\nrefractory prostate cancer. Clin Cancer Res. 2004;10:5048 –\n5057.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n10. Esumi H, Izuishi K, Kato K, et al. Hypoxia and nitric oxide\ntreatment confer tolerance to glucose starvation in a 5-\nAMP-activated protein kinase-dependent manner. J Biol\nChem. 2002;277:32791–32798.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n11. Kim R, Toge T. Changes in therapy for solid tumors: poten-\ntial for overcoming drug resistance in vivo with molecular\ntargeting agents. Surg Today. 2004;34:293–303.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n12. Bursch W, Hochegger K, Torok L, Marian B, Ellinger A,\nHermann RS. Autophagic and apoptotic types of pro-\ngrammed cell death exhibit different fates of cytoskeletal\nﬁlaments. J Cell Sci. 2000;113:1189 –1198.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n13. Pattingre S, Bauvy C, Codogno P. Amino acids interfere with\nthe ERK1/2-dependent control of macroautophagy by con-\ntrolling the activation of Raf-1 in human colon cancer HT-29\ncells. J Biol Chem. 2003;278:16667–16674.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n14. Aita VM, Liang XH, Murty VV, et al. Cloning and genomic\norganization of beclin 1, a candidate tumor suppressor gene\non chromosome 17q21. Genomics. 1999;59:59 – 65.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n15. Yue Z, Jin S, Yang C, Levine AJ, Heintz N. Beclin 1, an\nautophagy gene essential for early embryonic development,\nis a haploinsufﬁcient tumor suppressor. Proc Natl Acad Sci\nUSA. 2003;100:15077–15082.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n16. Liang XH, Jackson S, Seaman M, et al. Induction of autoph-\nagy and inhibition of tumorigenesis by beclin 1. Nature.\n1999;402:672– 676.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n17. Debatin KM. Apoptosis pathways in cancer and cancer ther-\napy. Cancer Immunol Immunother. 2004;53:153–159.\n18. Lacour S, Micheau O, Hammann A, et al. Chemotherapy\nenhances TNF-related apoptosis-inducing ligand DISC as-\nsembly in HT29 human colon cancer cells. Oncogene. 2003;\n22:1807–1816.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n19. Barnhart BC, Alappat EC, Peter ME. The CD95 type I/type II\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nmodel. Semin Immunol. 2003;15:185–193.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n20. Tsuruta F, Sunayama J, Mori Y, et al. JNK promotes Bax\ntranslocation to mitochondria through phosphorylation of\n14-3-3 proteins. EMBO J. 2004;23:1889 –1899.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n21. Jia L, Patwari Y, Srinivasula SM, et al. Bax translocation is\ncrucial for the sensitivity of leukaemic cells to etoposide-\ninduced apoptosis. Oncogene. 2001;20:4817– 4826.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n22. Verrier F, Deniaud A, LeBras M, et al. Dynamic evolution of\nthe adenine nucleotide translocase interactome during che-\nmotherapy-induced apoptosis. Oncogene. 2004;23:8049 –\n8064.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n23. Cain K, Bratton SB, Cohen GM. The Apaf-1 apoptosome: a\nlarge caspase-activating complex. Biochimie. 2002;84:203–\n214.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n24. Song Z, Yao X, Wu M. Direct interaction between survivin\nand Smac/DIABLO is essential for the anti-apoptotic activity\nof survivin during Taxol-induced apoptosis. J Biol Chem.\n2003;278:23130 –23140.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n25. Wilkinson JC, Wilkinson AS, Scott FL, Csomos RA, Salvesen\nGS, Duckett CS. Neutralization of Smac/Diablo by IAPs: a\ncaspase-independent mechanism for apoptotic inhibition.\nJ Biol Chem. 2004;279:51082–51090.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n26. Srinivasula SM, Datta P, Fan XJ, Fernandes-Alnemri T,\nHuang Z, Alnemri ES. Molecular determinants of\nthe\ncaspase-promoting activity of Smac/DIABLO and its role in\nthe death receptor pathway. J Biol Chem. 2000;275:36152–\n36157.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n27. Sun XM, Bratton SB, Butterworth M, MacFarlane M, Cohen\nGM. Bcl-2 and Bcl-xL inhibit CD95-mediated apoptosis by\npreventing mitochondrial release of Smac/DIABLO and\nsubsequent inactivation of X-linked inhibitor-of-apoptosis\nprotein. J Biol Chem. 2002;277:11345–11351.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n28. Hasenjager A, Gillissen B, Muller A, et al. Smac induces\ncytochrome c release and apoptosis independently from\nBax/Bcl-x(L) in a strictly caspase-3-dependent manner in\nhuman carcinoma cells. Oncogene. 2004;23:4523– 4535.\n29. Hao Y, Sekine K, Kawabata A, et al. Apollon ubiquitinates\nSMAC and caspase-9, and has an essential cytoprotection\nfunction. Nat Cell Biol. 2004;6:849 – 860.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n30. Bartling B, Lewensohn R, Zhivotovsky B. Endogenously re-\nleased Smac is insufﬁcient to mediate cell death of human\nlung carcinoma in response to etoposide. Exp Cell Res. 2004;\n298:83–95.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n31. Hegde R, Srinivasula SM, Zhang Z, et al. Identiﬁcation of\nOmi/HtrA2 as a mitochondrial apoptotic serine protease\nthat disrupts inhibitor of apoptosis protein-caspase interac-\ntion. J Biol Chem. 2002;277:432– 438.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n32. Yang QH, Church-Hajduk R, Ren J, Newton ML, Du C.\nOmi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nirreversibly inactivates IAPs and facilitates caspase activity\nin apoptosis. Genes Dev. 2003;17:1487–1496.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n33. Suzuki Y, Takahashi-Niki K, Akagi T, Hashikawa T, Taka-\nhashi R. Mitochondrial protease Omi/HtrA2 enhances\ncaspase activation through multiple pathways. Cell Death\nDiffer. 2004;11:208 –216.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n34. Cande C, Cecconi F, Dessen P, Kroemer G. Apoptosis-in-\nducing factor (AIF): key to the conserved caspase-indepen-\ndent pathways of cell death? J Cell Sci. 2002;115:4727– 4734.\n35. Cande C, Cohen I, Daugas E, et al. Apoptosis-inducing factor\n(AIF): a novel caspase-independent death effector released\nfrom mitochondria. Biochimie. 2002;84:215–222.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n36. Joseph B, Marchetti P, Formstecher P, Kroemer G, Lewen-\nsohn R, Zhivotovsky B. Mitochondrial dysfunction is an\nessential step for killing of non-small cell lung carcinomas\nresistant to conventional treatment. Oncogene. 2002;21:65–\n77.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n37. Yu SW, Wang H, Poitras MF, et al. Mediation of poly(ADP-\nribose) polymerase-1-dependent cell death by apoptosis-\ninducing factor. Science. 2002;297:259 –263.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n38. Hong SJ, Dawson TM, Dawson VL. Nuclear and mitochon-\ndrial conversations in cell death: PARP-1 and AIF signaling.\nTrends Pharmacol Sci. 2004;25:259 –264.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n39. Zhang J, Dong M, Li L, et al. Endonuclease G is required for\nearly embryogenesis and normal apoptosis in mice. Proc\nNatl Acad Sci USA. 2003;100:15782–15787.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n40. Schweichel JU, Merker HJ. The morphology of various types\nof cell death in prenatal tissues. Teratology. 1973;7:253–266.\n41. Mizushima N, Yamamoto A, Matsui M, Yoshimori T, Oh-\nsumi Y. In vivo analysis of autophagy in response to nutrient\nstarvation using transgenic mice expressing a ﬂuorescent\nautophagosome marker. Mol Biol Cell. 2004;15:1101–1111.\n42. Dunn WA Jr. Studies on the mechanisms of autophagy:\nformation of the autophagic vacuole. J Cell Biol. 1990;110:\n1923–1933.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n43. Fengsrud M, Roos N, Berg T, Liou W, Slot JW, Seglen PO.\nUltrastructural and immunocytochemical characterization\nof autophagic vacuoles in isolated hepatocytes: effects of\nvinblastine and asparagine on vacuole distributions. Exp\nCell Res. 1995;221:504 –519.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n44. Petiot A, Ogier-Denis E, Blommaart EF, Meijer AJ, Codogno\nP. Distinct classes of phosphatidylinositol 3-kinases are\ninvolved in signaling pathways that control macroautoph-\nagy in HT-29 cells. J Biol Chem. 2000;275:992–998.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n45. Aki T, Yamaguchi K, Fujimiya T, Mizukami Y. Phosphoino-\nsitide 3-kinase accelerates autophagic cell death during glu-\ncose deprivation in the rat cardiomyocyte-derived cell line\nH9c2. Oncogene. 2003;22:8529 – 8535.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n46. Liang XH, Yu J, Brown K, Levine B. Beclin 1 contains a\nleucine-rich nuclear export signal that is required for its\nautophagy and tumor suppressor function. Cancer Res.\n2001;61:3443–3449.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n47. Saeki K, Yuo A, Okuma E, et al. Bcl-2 down-regulation\ncauses autophagy in a caspase-independent manner in hu-\nman leukemic HL60 cells. Cell Death Differ. 2000;7:1263–\n1269.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n48. Sakamuro D, Elliott KJ, Wechsler-Reya R, Prendergast GC.\nBIN1 is a novel MYC-interacting protein with features of a\ntumour suppressor. Nat Genet. 1996;14:69 –77.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n49. DuHadaway JB, Du W, Donover S, et al. Transformation-\nselective apoptotic program triggered by farnesyltransferase\ninhibitors requires Bin1. Oncogene. 2003;22:3578 –3588.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n50. Inbal B, Bialik S, Sabanay I, Shani G, Kimchi A. DAP kinase\nand DRP-1 mediate membrane blebbing and the formation\nof autophagic vesicles during programmed cell death. J Cell\nBiol. 2002;157:455– 468.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n51. Bialik S, Kimchi A. DAP-kinase as a target for drug design in\ncancer and diseases associated with accelerated cell death.\nSemin Cancer Biol. 2004;14:283–294.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n52. Yanagisawa H, Miyashita T, Nakano Y, Yamamoto D.\nHSpin1, a transmembrane protein interacting with Bcl-2/\nBcl-xL,\ninduces a caspase-independent autophagic cell\ndeath. Cell Death Differ. 2003;10:798 – 807.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n53. Arico S, Petiot A, Bauvy C, et al. The tumor suppressor PTEN\npositively regulates macroautophagy by inhibiting the phos-\nphatidylinositol 3-kinase/protein kinase B pathway. J Biol\nChem. 2001;276:35243–35246.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n54. Chi S, Kitanaka C, Noguchi K, et al. Oncogenic Ras triggers\ncell suicide through the activation of a caspase-independent\ncell death program in human cancer cells. Oncogene. 1999;\n18:2281–2290.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n55. Kitanaka C, Kato K, Ijiri R, et al. Increased Ras expression\nand caspase-independent neuroblastoma cell death: possi-\nble mechanism of spontaneous neuroblastoma regression.\nJ Natl Cancer Inst. 2002;94:358 –368.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n56. Pattingre S, Bauvy C, Codogno P. Amino acids interfere with\nthe ERK1/2-dependent control of macroautophagy by con-\ntrolling the activation of Raf-1 in human colon cancer HT-29\ncells. J Biol Chem. 2003;278:16667–16674.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n57. Vande Velde C, Cizeau J, Dubik D, et al. BNIP3 and genetic\ncontrol of necrosis-like cell death through the mitochondrial\npermeability transition pore. Mol Cell Biol. 2000;20:5454 –\n5468.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n58. Hirsimaki Y, Hirsimaki P, Lounatmaa K. Vinblastine-in-\nduced autophagic vacuoles in mouse liver and Ehrlich as-\ncites tumor cells as assessed by freeze-fracture electron\nmicroscopy. Eur J Cell Biol. 1982;27:298 –301.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n59. Raught B, Gingras AC, Sonenberg N. The target of rapamy-\ncin (TOR) proteins. Proc Natl Acad Sci USA. 2001;98:7037–\n7044.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n60. Huisman C, Ferreira CG, Broker LE, et al. Paclitaxel triggers\ncell death primarily via caspase-independent routes in the\nnon-small cell lung cancer cell line NCI-H460. Clin Cancer\nRes. 2002;8:596 – 606.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n61. Kanzawa T, Kondo Y, Ito H, Kondo S, Germano I. Induction\nof autophagic cell death in malignant glioma cells by arsenic\ntrioxide. Cancer Res. 2003;63:2103–2108.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n62. Yao KC, Komata T, Kondo Y, Kanzawa T, Kondo S, Germano\nIM. Molecular response of human glioblastoma multiforme\ncells to ionizing radiation: cell cycle arrest, modulation of\nthe expression of cyclin-dependent kinase inhibitors, and\nautophagy. J Neurosurg. 2003;98:378 –384.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n63. Scarlatti F, Bauvy C, Ventruti A, et al. Ceramide-mediated\nmacroautophagy involves inhibition of protein kinase B and\nup-regulation of beclin 1. J Biol Chem. 2004;279:18384 –\n18391.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n64. Xue L, Fletcher GC, Tolkovsky AM. Autophagy is activated by\napoptotic signalling in sympathetic neurons: an alternative\nmechanism of death execution. Mol Cell Neurosci. 1999;14:\n180 –198.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n65. Bauvy C, Gane P, Arico S, Codogno P, Ogier-Denis E. Auto-\nphagy delays sulindac sulﬁde-induced apoptosis in the hu-\nman intestinal colon cancer cell line HT-29. Exp Cell Res.\n2001;268:139 –149.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n66. Xue L, Fletcher GC, Tolkovsky AM. Mitochondria are selec-\ntively eliminated from eukaryotic cells after blockade of\ncaspases during apoptosis. Curr Biol. 2001;11:361–365.\n67. Wei MC, Zong WX, Cheng EH, et al. Proapoptotic BAX and\nBAK: a requisite gateway to mitochondrial dysfunction and\ndeath. Science. 2001;292:727–730.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n68. Terman A, Brunk UT. Aging as a catabolic malfunction. Int\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nJ Biochem Cell Biol. 2004;36:2365–2375.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n69. Chau YP, Lin SY, Chen JH, Tai MH. Endostatin induces\nautophagic cell death in EAhy926 human endothelial cells.\nHistol Histopathol. 2003;18:715–726.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n70. Wen S, Stolarov J, Myers MP, et al. PTEN controls tumor-\ninduced angiogenesis. Proc Natl Acad Sci USA. 2001;98:\n4622– 4627.",
         "extracted",
         "Neutralization of Smac/Diablo by Inhibitors of Apoptosis (IAPs);Aging as a catabolic malfunction.;Apoptosis, autophagy, and more.;Dynamic evolution of the adenine nucleotide translocase interactome during chemotherapy-induced apoptosis;Bax is an important determinant for radiation sensitivity in esophageal carcinoma cells.;Apollon ubiquitinates SMAC and caspase-9, and has an essential cytoprotection function;A Phase I Pharmacokinetic and Biological Correlative Study of Oblimersen Sodium (Genasense, G3139), an Antisense Oligonucleotide to the Bcl-2 mRNA, and of Docetaxel in Patients with Hormone-Refractory Prostate Cancer;DAP-kinase as a target for drug design in cancer and diseases associated with accelerated cell death.;Endogenously released Smac is insufficient to mediate cell death of human lung carcinoma in response to etoposide.;Smac induces cytochrome c release and apoptosis independently from Bax/Bcl-xL in a strictly caspase-3-dependent manner in human carcinoma cells;Nuclear and mitochondrial conversations in cell death: PARP-1 and AIF signaling.;Ceramide-mediated Macroautophagy Involves Inhibition of Protein Kinase B and Up-regulation of Beclin 1*;JNK promotes Bax translocation to mitochondria through phosphorylation of 14‐3‐3 proteins;Multiple cell death pathways as regulators of tumour initiation and progression;Apoptosis defects and chemotherapy resistance: molecular interaction maps and networks;Changes in Therapy for Solid Tumors: Potential for Overcoming Drug Resistance In Vivo with Molecular Targeting Agents;Molecular therapy for peritoneal dissemination of xenotransplanted human MKN-45 gastric cancer cells with adenovirus mediated Bax gene transfer;Mitochondrial protease Omi/HtrA2 enhances caspase activation through multiple pathways;Apoptosis pathways in cancer and cancer therapy;In vivo analysis of autophagy in response to nutrient starvation using transgenic mice expressing a fluorescent autophagosome marker.;Endonuclease G is required for early embryogenesis and normal apoptosis in mice;Beclin 1, an autophagy gene essential for early embryonic development, is a haploinsufficient tumor suppressor;Phosphoinositide 3-kinase accelerates autophagic cell death during glucose deprivation in the rat cardiomyocyte-derived cell line H9c2;Endostatin induces autophagic cell death in EAhy926 human endothelial cells.;HSpin1, a transmembrane protein interacting with Bcl-2/Bcl-xL, induces a caspase-independent autophagic cell death;Direct Interaction between Survivin and Smac/DIABLO Is Essential for the Anti-apoptotic Activity of Survivin during Taxol-induced Apoptosis*;Omi/HtrA2 catalytic cleavage of inhibitor of apoptosis (IAP) irreversibly inactivates IAPs and facilitates caspase activity in apoptosis.;Transformation-selective apoptotic program triggered by farnesyltransferase inhibitors requires Bin1;The CD95 type I/type II model.;Amino Acids Interfere with the ERK1/2-dependent Control of Macroautophagy by Controlling the Activation of Raf-1 in Human Colon Cancer HT-29 Cells*;Induction of autophagic cell death in malignant glioma cells by arsenic trioxide.;Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs.;Chemotherapy enhances TNF-related apoptosis-inducing ligand DISC assembly in HT29 human colon cancer cells;Molecular response of human glioblastoma multiforme cells to ionizing radiation: cell cycle arrest, modulation of the expression of cyclin-dependent kinase inhibitors, and autophagy.;Apoptosis-inducing factor (AIF): key to the conserved caspase-independent pathways of cell death?;Current status of the molecular mechanisms of anticancer drug-induced apoptosis;Hypoxia and Nitric Oxide Treatment Confer Tolerance to Glucose Starvation in a 5′-AMP-activated Protein Kinase-dependent Manner*;Mediation of Poly(ADP-Ribose) Polymerase-1-Dependent Cell Death by Apoptosis-Inducing Factor;DAP kinase and DRP-1 mediate membrane blebbing and the formation of autophagic vesicles during programmed cell death;Bcl-2 and Bcl-xL Inhibit CD95-mediated Apoptosis by Preventing Mitochondrial Release of Smac/DIABLO and Subsequent Inactivation of X-linked Inhibitor-of-Apoptosis Protein*;Increased Ras expression and caspase-independent neuroblastoma cell death: possible mechanism of spontaneous neuroblastoma regression.;Paclitaxel triggers cell death primarily via caspase-independent routes in the non-small cell lung cancer cell line NCI-H460.;The Apaf-1 apoptosome: a large caspase-activating complex.;Apoptosis-inducing factor (AIF): a novel caspase-independent death effector released from mitochondria.;Identification of Omi/HtrA2 as a Mitochondrial Apoptotic Serine Protease That Disrupts Inhibitor of Apoptosis Protein-Caspase Interaction*;The Tumor Suppressor PTEN Positively Regulates Macroautophagy by Inhibiting the Phosphatidylinositol 3-Kinase/Protein Kinase B Pathway*;Autophagy delays sulindac sulfide-induced apoptosis in the human intestinal colon cancer cell line HT-29.;Bax translocation is crucial for the sensitivity of leukaemic cells to etoposide-induced apoptosis;The target of rapamycin (TOR) proteins;Proapoptotic BAX and BAK: A Requisite Gateway to Mitochondrial Dysfunction and Death;Beclin 1 contains a leucine-rich nuclear export signal that is required for its autophagy and tumor suppressor function.;PTEN controls tumor-induced angiogenesis;Mitochondria are selectively eliminated from eukaryotic cells after blockade of caspases during apoptosis;Bcl-2 down-regulation causes autophagy in a caspase-independent manner in human leukemic HL60 cells;Molecular Determinants of the Caspase-promoting Activity of Smac/DIABLO and Its Role in the Death Receptor Pathway*;BNIP3 and Genetic Control of Necrosis-Like Cell Death through the Mitochondrial Permeability Transition Pore;Autophagic and apoptotic types of programmed cell death exhibit different fates of cytoskeletal filaments.;Distinct Classes of Phosphatidylinositol 3′-Kinases Are Involved in Signaling Pathways That Control Macroautophagy in HT-29 Cells*;Induction of autophagy and inhibition of tumorigenesis by beclin 1;Autophagy Is Activated by Apoptotic Signalling in Sympathetic Neurons: An Alternative Mechanism of Death Execution;Cloning and genomic organization of beclin 1, a candidate tumor suppressor gene on chromosome 17q21.;Oncogenic Ras triggers cell suicide through the activation of a caspase-independent cell death program in human cancer cells;BIN1 is a novel MYC–interacting protein with features of a tumour suppressor;Ultrastructural and immunocytochemical characterization of autophagic vacuoles in isolated hepatocytes: effects of vinblastine and asparagine on vacuole distributions.;Studies on the mechanisms of autophagy: formation of the autophagic vacuole;Vinblastine-induced autophagic vacuoles in mouse liver and Ehrlich ascites tumor cells as assessed by freeze-fracture electron microscopy.;The morphology of various types of cell death in prenatal tissues.;Inducing cancer cell death by targeting transcription factors.;Mitochondrial dysfunction is an essential step for killing of non-small cell lung carcinomas resistant to conventional treatment",
         "Recent advances in understanding the cell death pathways activated by anticancer therapy",
         "None"
        ],
        [
         "22",
         "001e671482c0a1336ad28cbfdfd76af665227433",
         "Phytochromes are photochromic biliproteins found in plants as well as in some cyanotrophic, photoautotrophic and heterotrophic bacteria. In many bacteria, their function is largely unknown. Here we describe the biochemical and spectroscopic characterization of recombinant bacterial phytochrome from the opportunistic pathogen Pseudomonas aeruginosa (PaBphP). The recombinant protein displays all the characteristic features of a bonafide phytochrome. In contrast with cyanobacteria and plants, the chromophore of this bacterial phytochrome is biliverdin IXα, which is produced by the heme oxygenase BphO in P. aeruginosa. This chromophore was shown to be covalently attached via its A‐ring endo‐vinyl group to a cysteine residue outside the defined bilin lyase domain of plant and cyanobacterial phytochromes. Site‐directed mutagenesis identified Cys12 and His247 as being important for chromophore binding and photoreversibility, respectively. PaBphP is synthesized in the dark in the red‐light‐absorbing Pr form and immediately converted into a far‐red‐light‐absorbing Pfr‐enriched form. It shows the characteristic red/far‐red‐light‐induced photoreversibility of phytochromes. A chromophore analog that lacks the C15/16 double bond was used to show that this photoreversibility is due to a 15Z/15E isomerization of the biliverdin chromophore. Autophosphorylation of PaBphP was demonstrated, confirming its role as a sensor kinase of a bacterial two‐component signaling system.",
         "Ronja Tasler,T. Moises,N. Frankenberg‐Dinkel",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1111/j.1742-4658.2005.04623.x",
         "\n**BLOCK**fs== 14.9**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nBiochemical and spectroscopic characterization of the\nbacterial phytochrome of Pseudomonas aeruginosa\nRonja Tasler, Tina Moises and Nicole Frankenberg-Dinkel\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nInstitute for Microbiology, Technical University Braunschweig, Germany\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\nKeywords\nbiliverdin; histidine kinase; linear\ntetrapyrrole; photoreceptor; two-component\nsystem\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nCorrespondence\nN. Frankenberg-Dinkel, Institute for\nMicrobiology, Technical University\nBraunschweig, Spielmannstr. 7, 38106\nBraunschweig, Germany\nFax: +49 531 391 5854\nTel: +49 531 391 5815\nE-mail: n.frankenberg@tu-bs.de\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\n(Received 2 February 2005, revised 17\nFebruary 2005, accepted 21 February 2005)\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\ndoi:10.1111/j.1742-4658.2005.04623.x\n**BLOCK**fs== 9.5**b== 0.6**t== 0.4**l== 0.4**r== 0.2**\nfrom the opportunistic pathogen Pseudomonas\n**BLOCK**fs== 9.5**b== 0.4**t== 0.3**l== 0.4**r== 0.1**\nPhytochromes are photochromic biliproteins found in plants as well as in\nsome cyanotrophic, photoautotrophic and heterotrophic bacteria. In many\nbacteria, their function is largely unknown. Here we describe the biochemi-\ncal and spectroscopic characterization of recombinant bacterial phyto-\naeruginosa\nchrome\n(PaBphP). The recombinant protein displays all the characteristic features\nof a bonaﬁde phytochrome. In contrast with cyanobacteria and plants, the\nchromophore of this bacterial phytochrome is biliverdin IXa, which is pro-\nduced by the heme oxygenase BphO in P. aeruginosa. This chromophore\nwas shown to be covalently attached via its A-ring endo-vinyl group to a\ncysteine residue outside the deﬁned bilin lyase domain of plant and cyano-\nbacterial phytochromes. Site-directed mutagenesis identiﬁed Cys12 and\nHis247 as being important for chromophore binding and photoreversibility,\nrespectively. PaBphP is synthesized in the dark in the red-light-absorbing\nPr form and immediately converted into a far-red-light-absorbing Pfr-\nenriched form. It shows the characteristic red ⁄ far-red-light-induced photo-\nreversibility of phytochromes. A chromophore analog that\nlacks the\nC15 ⁄ 16 double bond was used to show that this photoreversibility is due to\na 15Z ⁄ 15E isomerization of the biliverdin chromophore. Autophosphoryla-\ntion of PaBphP was demonstrated, conﬁrming its role as a sensor kinase of\na bacterial two-component signaling system.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nPhytochromes are biliprotein photoreceptors in plants\nbut have recently also been discovered in bacteria [1].\nIn plants, the family of phytochromes sense red and\nfar-red light and therefore play a key role in mediating\nresponses to light quality, quantity, direction and dur-\nation throughout plant development [2]. Plant phyto-\nchromes are homodimers composed of  125-kDa\nsubunits each with a thioether-linked phytochromobi-\nlin prosthetic group [3]. Unlike the light-harvesting\ncyanobacterial phycobiliproteins which require a lyase\nfor the covalent attachment of the linear tetrapyrrole\n(bilin) chromophore, bilin attachment to apo-phyto-\nchromes is autocatalytic [4]. The action of phyto-\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nchrome depends on its ability to photointerconvert\nbetween the red-light-absorbing Pr form and the far-\nred-light-absorbing Pfr form, a property conferred by\nthe covalently bound phytochromobilin in the plant\nholophytochrome. The ﬁrst phytochrome from a bac-\nterial source to be discovered was Cph1 (cyanobac-\nterial phytochrome 1) from Synechocystis sp. PCC6803\nwhich was followed by the discovery of bacterial phy-\ntochromes (BphPs) from nonphotosynthetic bacteria\n[1,5,6]. BphPs are typical sensor kinases of a two-com-\nponent signaling system. Most BphPs including that of\nPseudomonas aeruginosa (PaBphP) carry a C-terminal\nhistidine kinase module, and it has been shown that\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\nAbbreviations\nBLD, bilin lyase domain; BVR, biliverdin reductase; PaBphP, Pseudomonas aeruginosa bacterial phytochrome; PAS, PER ⁄ ARNT ⁄ SIM\nrepeats.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.2**l== 0.1**r== 0.5**\nmany of them, such as Synechocystis Cph1 [6], Agro-\nbacterium Agp1 and Agp2 [7,8] and also Pseudomonas\nsyringae BphP [9], are light-regulated histidine kinases.\nUnlike plant and cyanobacterial phytochromes,\nwhich carry a phytochromobilin or phycocyanobilin\nchromophore, BphPs have been shown to utilize a bili-\nverdin chromophore [9]. Apart from Cph1, most mem-\nbers of the BphP family lack the conserved cysteine\nresidue in the conserved bilin lyase domain (BLD).\nThis domain has been deﬁned as the minimal GAF\ndomain, capable of autocatalytic assembly with bilin\nchromophores [10]. GAF domains are small\nligand-\nbinding domains found in vertebrate cGMP-speciﬁc\nphosphodiesterases, cyanobacterial adenylate cyclases\nand the formate hydrogen lyase transcription activator\nFhlA [11]. In most phytochromes, the BLD is preceded\nby the P2 domain, which is often recognized as a PAS\ndomain in the Pfam database (protein families data-\nbase; http://www.sanger.ac.uk/Software/Pfam/)\n[12].\nPAS domains are tandem repeats ﬁrst described in the\ntranscriptional regulatory proteins period clock (PER)\nfrom Drosophila melanogaster,\nthe murine aromatic\nhydrocarbon receptor nuclear translocator (ARNT)\nand single minded (SIM) from D. melanogaster [13].\nInterestingly, a cysteine residue in this P2 domain has\nbeen shown to be the site of chromophore attachment\nin Agp1\n[7,14].\nAnother characteristic domain in phytochromes is the\nPHY domain which corresponds to a GAF-related\ndomain located C-terminally to the BLD (Scheme 1).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nfrom Agrobacterium tumefaciens\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nRecently we have shown that BphP from P. aerugi-\nnosa is able to bind biliverdin IXa and biliverdin IXd,\nwhich are produced by the two heme oxygenases BphO\nand PigA [15]. As bphO is chromosomally located\nupstream of bphP and the afﬁnity for biliverdin IXa\nwas about ﬁvefold higher than for biliverdin IXd, we\nconcluded that biliverdin IXa is the natural chromo-\nphore of PaBphP. Furthermore, we presented data\nindicating an involvement of BphP in biliverdin release\nfrom BphO, as this is the rate-limiting step of the\nBphO reaction.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nHere we describe the further biochemical and spect-\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nroscopic characterization of BphP.\n**BLOCK**fs== 10.3**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nC\n**BLOCK**fs== 10.3**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nP2 BLD PH Y HKD\n**BLOCK**fs== 10.7**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nResults\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nExpression, puriﬁcation and initial characterization\nof recombinant P. aeruginosa phytochrome\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nThe P. aeruginosa bphP was expressed using a tet pro-\nmoter-driven C-terminal Strep tag expression system.\nRecombinant BphP was always puriﬁed in the apo\nform, and the homogeneity after puriﬁcation was\n 98% (Fig. 1,\ninset). A single band migrating at\n 80 kDa was obtained on SDS ⁄ PAGE, which corre-\nlates with the predicted molecular mass calculated\nfrom the amino-acid composition (80.1 kDa). The\nyield of puriﬁed BphP was typically 5 mg per litre of\nbacterial culture. Analytical gel permeation chromato-\ngraphy revealed that the apo form, as well as the\nassembled holo form, of BphP is eluted as a dimer\nfrom a Superdex 200 column (data not shown; [15]).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nAssembly and chromophore binding\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nPaBphP is able to autocatalytically form a photocon-\nvertible holo-phytochrome with the proposed natural\nchromophore biliverdin IXa. Illumination of recombin-\nant holo-BphP with saturating red light (630 nm) resul-\nted in the formation of the Pfr form (Pfr-enriched)\nwhich could be converted back into the Pr form through\nillumination with far-red light (750 nm) (Fig. 1A). The\nresultant calculated difference spectrum shows the char-\nacteristic phytochrome signature (Fig. 1B) with maxima\nof 700 and 754 nm for the Pr and Pfr form, respectively.\nThese far-red absorbance maxima seem to be typical of\nbiliverdin-binding phytochromes and represent the most\nred-shifted phytochrome forms described so far [7,16].\nThe covalent binding of biliverdin IXa was conﬁrmed\nby zinc-induced red ﬂuorescence (Fig. 3C).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nThe form initially synthesized after the addition of\nbiliverdin IXa to apo-BphP in the dark is the Pr form,\nwhich is immediately converted nonphotochemically\ninto a Pfr-enriched form. This nonphotochemical con-\nversion reaches an equilibrium between Pr and Pfr\nforms after 90 min (Fig. 2A). Irradiation with far-red\nlight leads to the formation of the Pr form with one\npeak at 700 nm, which can be converted back into the\nPfr form by irradiation with red light. Both the Pr and\nthe Pfr form are unstable in the dark and convert back\ninto a dark form, a Pfr-enriched mixture of Pr and Pfr\n(Fig. 2B,C).\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nScheme 1. Domain structure of the P. aeruginosa phytochrome.\nP2, PAS domain; BLD, bilin lyase domain (a GAF domain); PHY,\nphytochrome domain (GAF-related domain); HKD, histidine kinase\ndomain.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\nChromophore–protein interaction\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nTo determine which part of the bilin chromophore is\ninvolved in covalent attachment to the protein, various\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nA\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nB\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nA\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nB\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nC\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nFig. 1. (A) Absorbance spectra of recombinant BphP incubated with\nbiliverdin IXa. Pfr, Pfr-enriched form obtained after illumination with\nred light (630 nm) (dashed line); Pr, Pr form obtained after illumin-\nation with far-red light (750 nm) (solid line). The inset shows the\nSDS ⁄ PAGE analysis of BphP after afﬁnity chromatography. (B) Cal-\nculated Pr–Pfr difference spectrum.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nbiliverdin derivatives were used (see Fig. 3B for chem-\nical structures). The resultant chromoproteins were\ncharacterized by red ⁄ far-red-light-induced difference\nspectroscopy (Fig. 3A). The spectral properties are\nsummarized in Table 1. BphP is able to covalently bind\nbiliverdin IXd and biliverdin XIIIa, which was con-\nﬁrmed by zinc-induced red ﬂuorescence (Fig. 3C). Fur-\nthermore, these biliverdin adducts were able to form\na photoconvertible holophytochrome. No characteristic\ndifference spectrum nor covalent binding was observed\nwith biliverdin IXb, biliverdin IXc, mesobiliverdin,\n31,32-dihydrobiliverdin and biliverdin IIIa (Fig. 3B,C\nand [15]). The common feature of all covalently bound\nbiliverdin derivatives is an A-ring endo-vinyl group,\nindicating that this side chain is absolutely required for\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nFig. 2. Spectral properties of holo-BphP. (A) Absorbance spectrum\nchanges during 3 h in the dark after assembly of apo-BphP with bili-\nverdin. (B) Dark reversion of BphP photoconverted in the Pr form.\n(C) Dark reversion of BphP photoconverted in the Pfr form. Inserts\nin (B) and (C) show the time-dependent absorbance changes at\n750 nm.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\ncovalent attachment. Furthermore, these results imply\nthat the ring substituents of the other pyrrole rings do\nnot seem to be critical for photoconversion.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nA\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nB\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nC\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nFig. 3. (A) Difference spectroscopy of BphP incubated with biliverdin isomers. From top to bottom: BphP incubated with biliverdin IXc,\nmesobiliverdin, 31,32-dihydrobiliverdin, biliverdin IIIa and biliverdin XIIIa. For difference spectrum of BphP–biliverdin IXa, see Fig. 1B; BphP–\nbiliverdin IXb ⁄ d [15]. (B) Chemical structures of the biliverdin isomers. (C) Zinc-induced red ﬂuorescence of BphP with different chromo-\nphores. Apo-BphP was incubated with different biliverdin isomers; after SDS ⁄ PAGE analysis (labeled protein) and electroblotting, covalently\nbound bilins were visualized using zinc-induced red ﬂuorescence (labeled zinc).\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nPhotoisomerization of PaBphP\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nThe primary photoreaction of plant phytochromes is\nknown to be the 15Z ⁄ 15E isomerization of the phyto-\nchromobilin chromophore [17]. If the C15 double bond\nis missing (i.e. in phycoerythrobilin), the corresponding\nphytochrome adduct is unable to undergo photoiso-\nmerization but instead is highly ﬂuorescent [18]. This\nﬂuorescent adduct of phytochromes is also known as a\nphytoﬂuor [19]. To elucidate whether the photoisomeri-\nzation in PaBphP is also due to a 15Z ⁄ 15E isomeriza-\ntion of\nthe bilin prosthetic group (in this case\nbiliverdin), we incubated apo-BphP with 15,16-dihydro-\nbiliverdin (see Fig. 4 for structure). 15,16-Dihydrobiliv-\nerdin can be synthesized in vitro from biliverdin by the\nferredoxin-dependent bilin reductase PebA [20]. Apo-\nBphP is able to bind 15,16-dihydrobiliverdin and is\norange ﬂuorescent under UV light\n(312 nm). This\nphenomenon was investigated ﬂuorospectrometrically.\nExcitation at 570 nm resulted in a ﬂuorescent phyto-\nﬂuor with an emission maximum of 630 nm (Fig. 4).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nBphP lacks the conserved cysteine residue involved in\ncovalent bilin attachment in plant and most cyanobac-\nterial phytochromes, and therefore the site and kind of\nattachment of the bilin chromophore in the bacterial\nphytochromes is controversial [1,21,22]. To investigate\nwhether the chromophore is attached via a thioether\nlinkage to a cysteine residue, the protein was treated\nwith iodoacetamide. This reagent speciﬁcally modiﬁes\ncysteine residues. If a chromophore-binding cysteine is\naccessible to iodoacetamide, a subsequent covalent\nchromophore attachment should be inhibited. Addition\nof increasing amounts of iodoacetamide leads to a\nreduction in photoisomerzation, as visualized by differ-\nence spectroscopy and covalent chromophore binding\n(i.e. decreased zinc-induced red ﬂuorecence). Full inhi-\nbition was observed with 1 mm iodoacetamide (data\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nTable 1. Spectral properties of BphP reconstituted with different\nchromophores. ND, not detected.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nk (DAmax)\n(nm)\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nk (DAmax)\n(nm)\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nBiliverdin IXa\nND\nBiliverdin IXb\n\nBiliverdin IXd\nND\nBiliverdin IXc\nMesobiliverdin\n\nDihydrobiliverdin ND\nND\nBiliverdin IIIa\n\nBiliverdin XIIIa\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\nND\n\nND\n**BLOCK**fs== 5.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nDAmax\n**BLOCK**fs== 5.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nDAmin\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nDDA\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\n0.022\nND\n0.003\nND\n0.006\n0.000\n0.000\n0.009\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n)0.026\nND\n)0.004\nND\n)0.012\n)0.010\n)0.009\n)0.012\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\n0.048\nND\n0.008\nND\n0.018\n0.010\n0.009\n0.021\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nFig. 4. Phytoﬂuor ﬂuorescence spectra of BphP incubated with\n15,16-dihydrobiliverdin. Fluorescence excitation (dashed) and emis-\nsion spectra (solid) of the phytoﬂuor obtained after incubation of\napo-BphP with 15,16-dihydrobiliverdin. The excitation spectrum was\nmonitored with an emission wavelength of 630 nm. The emission\nspectrum was obtained at an excitation wavelength of 570 nm.\nStructure of 15, 16-dihydrobiliverdin is also shown.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n[7,21]. C248 is\n**BLOCK**fs== 9.5**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\nnot shown). These results imply that the site of chro-\nmophore attachment in P. aeruginosa BphP is most\nlikely a cysteine residue. BphP contains twelve cysteine\nresidues, two of which, at position 12 and 248, could\npossibly serve as the chromophore-binding site. A cys-\nteine corresponding to position 12 has already been\nreported to be the site of chromophore attachment in\nAgp1 from A. tumefaciens\nlocated\nwithin the BLD and is adjacent to the chromophore-\nbinding site in cyanobacterial and plant phytochromes.\nTo further investigate the potential site of chromo-\nphore attachment, site-directed mutants (C12A, C12S\nand C248A) were generated and analyzed using the\nabove methods. Neither BphP C12A nor C12S showed\ncharacteristic difference spectra. The difference spectra\nof these variants (Fig. 5A) were very similar to the\niodoacetamide-blocked wild-type spectra (data not\nshown). The variant BphP C248A was able to form a\nphotoconvertible holoform with maximum and mini-\nmum identical with those of the wild-type (Table 2).\nOnly the C248A variant showed covalent biliverdin\nbinding, as demonstrated by zinc-induced red ﬂuores-\ncence (Fig. 5B). The covalent attachment of biliverdin\nto BphP was further conﬁrmed using a biliverdin\nreductase (BVR) assay. In this assay, only free biliver-\ndin can be converted by BVR into bilirubin. The addi-\ntion of BVR and NADPH to C12A:biliverdin and\nC12S:biliverdin resulted in the conversion of the bound\n**BLOCK**fs== 8.8**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nA\n**BLOCK**fs== 8.8**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nB\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFig. 5. (A) Absorbance difference spectra of BphP variants with bili-\nverdin IXa. Difference spectra of BphP C248A (solid line), BphP\nH247Q (long dashed line), BphP C12S (short dashed line) and BphP\nC12A (dotted line). (B) Zinc-induced red ﬂuorescence. ApoBphP wild-\ntype and variants were incubated with biliverdin IXa, and, after\nSDS ⁄ PAGE (labeled protein) and electroblotting, covalently bound bi-\nlins were visualized using zinc-induced red ﬂuorescence (labeled zinc).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nbiliverdin into bilirubin, which was accompanied by a\ncolor change from green to yellow, indicating the forma-\ntion of bilirubin. No biliverdin conversion was detected\nafter addition of BVR to wild-type BphP and the other\nvariants investigated (data not shown). Overall, these\nresults are in agreement with the data from Agp1 and\nindicate the importance of Cys12 in covalent chromo-\nphore binding.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nBesides this N-terminally located cysteine residue, a\nhistidine residue in the BLD has been discussed as the\nchromophore-binding site in Deinococcus radiodurans\nBphP and Calothrix sp. PCC7601 CphB [1,23]. This his-\ntidine residue is located adjacent to the conserved cys-\nteine residue in cyanobacterial and plant phytochromes.\nTo investigate the role of this histidine residue, a H247Q\nmutant was generated. H247Q was able to form a pho-\ntoconvertible holoform with blue-shifted extrema (694\nand 746 nm) (Table 2). For this variant, covalent bili-\nverdin binding was demonstrated using zinc-induced\nred ﬂuorescence and the BVR assay (data not shown).\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nTable 2. Spectral properties of BphP variants assembled with bili-\nverdin IXa.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nMaximum (nm)\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nWild-type\nC248A\nH247Q\nC12A\nC12S\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nA\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nprotein\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n32P\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n–\n–\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nB\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n32P\n**BLOCK**fs== 5.6**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nn\ne\nu\nt\nr\na\n**BLOCK**fs== 2.2**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nl\n**BLOCK**fs== 7.2**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nM\nH\nC\n**BLOCK**fs== 2.2**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nl\n**BLOCK**fs== 5.6**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\nM\nN\na\nO\nH\n**BLOCK**fs== 5.6**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\na\np\no\n**BLOCK**fs== 6.7**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nP\n**BLOCK**fs== 2.8**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nf\nr\n**BLOCK**fs== 6.7**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nP\n**BLOCK**fs== 3.3**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nr\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nFig. 6. Autoradiogram of BphP. Autoradiogram after [32P]ATP[cP]\nlabeling, SDS ⁄ PAGE and electroblotting. (A) Autoradiogram of the\napo and holo forms of BphP. (B) Stability of the autophosphoryla-\ntion after incubation for 1 h at room temperature in 50 mM Tris\n(pH 7.0) ⁄ 1 M HCl ⁄ 3 M NaOH.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nAutophosphorylation of BphP\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nLight-regulated His phosphorylation has been demon-\nstrated for several bacterial phytochromes. Amino-acid\nsequence analysis revealed that BphP also contains a\nhistidine kinase module (Scheme 1). Autophosphoryla-\ntion of BphP was determined after incubation of puri-\nﬁed apo-BphP and holo-BphP (Pr and Pfr form) with\n[32P]ATP[cP]. Both forms of BphP displayed auto-\nphosphorylation activity (Fig. 6A). Although the Pfr-\nenriched form shows slightly higher kinase actvity, no\nstrong light-dependence could be detected. BphP was\nconﬁrmed to be a histidine kinase, as the phosphoryla-\ntion was stable in alkaline solution and labile in acid\n(Fig. 6B). This was further conﬁrmed by replacing the\npotential phosphorylation site (H513) by alanine. No\nautophosphorylation was detected in this H513A\nmutant (data not shown).\n**BLOCK**fs== 10.7**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nDiscussion\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nPaBphP is a bacterial phytochrome using a\nbiliverdin chromophore\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nPaBphP was among the ﬁrst bacterial phytochromes\nto be discovered, and it has already been shown that\nthis BphP together with other members of this phyto-\nchrome class utilizes a biliverdin chromophore [1,9,15].\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nthe\nOur laboratory has recently demonstrated that\nPaBphP chromophore is produced by one of the two\nP. aeruginosa heme oxygenases. The BphO heme oxy-\ngenase is encoded in the same operon as bphP, and we\nwere able to show that BphP is involved in the release\nof the biliverdin produced from BphO. The major\nfunction of BphP remains unknown, but these results\nprovide biochemical evidence that recombinant BphP\nhas all the characteristics of a red ⁄ far-red-light-respon-\nsive photoreceptor.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nTo date only a few bacterial phytochromes have\nbeen biochemically characterized in detail. Among\nthem are Agp1 and Agp2 (also known as AtBphP1\nand AtBphP2) from A. tumefaciens. The most interest-\ning spectral observation for several BphPs including\nPaBphP is the Pfr-like ground state [8,16]. Assembly\nof PaBphP with biliverdin ﬁrst generates a transient\nPr-like intermediate, which is then nonphotochemically\ntransformed into a stable Pfr-enriched form (Fig. 2A).\nInterestingly, illumination with red light does not fully\nconvert this form into a solely Pfr form (Fig. 2C). The\nPfr-enriched form found after dark assembly is differ-\nent from that obtained through dark conversion of\neither the Pr or Pfr forms (Fig. 2B,C). Incubation of\npre-illuminated BphP always resulted in the formation\nof a Pr ⁄ Pfr equilibrium in the dark.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nAlthough autophosphorylation activity was demon-\nstrated for BphP, only a weak light-dependence has\nbeen observed with the Pfr-enriched form diplaying\nhighest kinase activity. However, this may also be due\nto the amount of Pr present in the Pfr-enriched form.\nThe observed Pfr ground state is the opposite of that\nused by almost all other known members of the classic\nphytochrome family [12,24]. More recent reports also\nrevealed the presence of the Pfr ground state in Agp2\nfrom A. tumefaciens and the BphPs of Rhodopseudo-\nmonas palustris and Bradyrhizobium ORS278 [8,16].\nFor the latter organisms, the Pfr ground state has been\nimplicated to be necessary for maximal photoregula-\ntion of photosynthesis by not overlapping with chloro-\nphyll absorption [16]. The reason for the Pfr ground\nstate in Agp2 and PaBphP is still not known, but if\nPaBphP indeed functions as a photoreceptor\nin\nP. aeruginosa, it would be expected to serve as a sensor\nof the ratio between far-red and red light.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nAn A-ring endo-vinyl group is required for\ncovalent attachment\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nOur data obtained using biliverdin derivatives and site-\ndirected mutagenesis of PaBphP are in agreement with\ndata obtained for Agp1 from A. tumefaciens [7,21].\nBoth BphPs seem to covalently bind the biliverdin\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nchromophore at a conserved cysteine residue in the P2\ndomain close to the N-terminus of the protein. An\nA-ring endo-vinyl group of the chromophore is abso-\nlutely required for this covalent attachment [15,21].\nOur data support the proposal that the lack of the\nconserved cysteine residue in the BLD correlates with\nthe use of biliverdin as the chromophore and the bind-\ning to a conserved cysteine residue in the P2 domain\n[25]. Nevertheless, the BLD still seems to be quite\nimportant for the photochemical reaction, as a H247Q\nmutation resulted in a spectral shift of the Pr and Pfr\nforms. Therefore, the BLD may play a role in stabil-\nthe chromophore and\nization and co-ordination of\npossibly its covalent attachment to Cys12 (i.e. the bilin\nlyase function).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nInterestingly, PaBphP Cys12 mutants assembled bili-\nverdin, but the afﬁnity of biliverdin was about ﬁvefold\nlower than wild-type BphP. The assembled Cys12 vari-\nants displayed a Pr-like aborption spectrum, which did\nnot alter upon red-light illumination (data not shown).\nThis observation is in contrast with data obtained for\nAgp1. An Agp1 C20A mutant was fully photoreversi-\nble, but had a reduced absorption coefﬁcient, a blue-\nshifted Pfr maximum, and a reduced ratio of Pfr to Pr\nabsorption [7]. In the case of PaBphP, the mutation of\nthis conserved cysteine residue is much more dramatic\nthan in Agp1. It seems that, in PaBphP, mutation of\nthis residue not only abolishes covalent binding, but\nstabilization of the Pfr form is also lost. This may be\ndue to a loosely or wrongly oriented biliverdin in the\nchromophore pocket. At this point it is worth men-\ntioning that, although many of our data point towards\nCys12 as the site of covalent chromophore attachment,\nit still cannot be ruled out that this cysteine residue\nonly plays a structural role (i.e. disulﬁde bond forma-\ntion). Consequently, its mutation would lead to a loss\nof the structural environment necessary for covalent\nbinding.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nPhotoconversion of BphPs involves 15Z ⁄ 15E\nisomerization\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nit has always been\nSince the discovery of BphPs,\nassumed that the photochemical reaction is similar to\nthat found in plant phytochromes, which involves a\n15Z ⁄ 15E isomerization of the phytochromobilin chro-\nmophore [17]. This assumption has not yet been\nexperimentally conﬁrmed. We used a chromophore\nanalog that lacks the C15 ⁄ C16 double bond to investi-\ngate the photoisomerization of BphP. The phytoﬂuor\nthe\nadduct obtained conﬁrmed the involvement of\nC15 ⁄ C16 double bond in photoisomerization, as the\ndihydrobiliverdin adduct is highly ﬂuorescent. Further-\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nmore, these data imply that, although the chromo-\nphore is attached at a different position, the geometry\nin PaBphP is\nof\nprobably similar to that of plant phytochromes.\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nthe chromophore-binding pocket\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nConclusion and outlook\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nWe have shown that recombinant PaBphP has all\nfeatures of a ‘true’ phytochrome photoreceptor. It cova-\nlently binds a biliverdin chromophore, which, upon illu-\nmination with red and far-red light, photoisomerizes at\nthe C15 ⁄ C16 double bond. Furthermore, we conﬁrmed\nthat PaBphP is a histidine kinase. The results of this\nwork support the proposal of a separate bacterial phyto-\nchrome class with a new chromophore-binding site in\nthe P2 domain, although a solely structural role for this\nresidue cannot be completely ruled out. The function of\nBphPs in nonphotosynthetic micro-organisms remains a\nmystery. To elucidate this further, we have constructed\nchromosomal knock-out mutations in the P. aeruginosa\nbphOP operon, which are currently being investigated\nusing proteomic and transcriptomics analysis.\n**BLOCK**fs== 10.7**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nExperimental procedures\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nReagents\n**BLOCK**fs== 8.7**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nAll chemicals were purchased from Sigma (Munich, Ger-\nmany) and were American Chemical Society grade or bet-\nter. Restriction enzymes were from Invitrogen (Cleveland,\nOH, USA). MasterTaqTM was purchased from Eppendorf\nScientiﬁc (Westbury, NY, USA). The expression vector\npASK-IBA3, Strep Tactin Sepharose, and anhydrotetra-\ncycline were obtained from IBA GmbH (Go¨ ttingen, Ger-\nmany). Centricon-10 concentrator devices were purchased\nfrom Amicon (Beverly, MA, USA). Biliverdin IXa was\nobtained from Frontier Scientiﬁc (Logan, UT, USA).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nBilin preparations\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n15,16-Dihydrobiliverdin, biliverdin IXb, biliverdin IXc, bili-\nverdin IXd and phycocyanobilin were prepared as described\npreviously [15,20]. Biliverdin XIIIa, biliverdin IIIa, 31,32-\ndihydrobiliverdin and mesobiliverdin were gifts\nfrom\nJ.C. Lagarias (UC Davis, CA, USA) and K. Inomata\n(Kanazawa University, Japan) [21,26,27].\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nConstruction of expression vectors\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe P. aeruginosa bphP (PA 4117) gene was ampliﬁed by\nPCR from chromosomal DNA using a hot start proto-\ncol with the following primers, which contained the\nindicated and underlined restriction sites: bphPXbaRBSfwd:\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.7**r== 0.1**\nbphPXhonoSTOPrev:\n**BLOCK**fs== 8.7**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\n5¢-CGTCTAGATAACGAGGGCAAAAAATGACGAG\nCATCACCCGGTTACC-3¢;\n5¢-CC\nCTCGAGGGACGAGGAGCCGGTCTCCG-3¢. The PCR\nproduct was digested with the indicated enzymes and\ncloned into XbaI ⁄ XhoI-digested expression vector pASK-\nIBA3 (IBA). The integrity of the plasmid construct was veri-\nﬁed by DNA sequence determination of the insert (SeqLab,\nGo¨ ttingen, Germany). The resulting ORF encodes BphP\nwith a C-terminal Strep Tag with a total addition of 20\namino-acid residues under the control of a tet promoter.\nSite-directed mutagenesis of bphP was performed using the\nQuikChange site-directed mutagenesis kit (Stratagene, La\nJolla, CA, USA), according to the instructions of\nthe\nmanufacturer with the following primers (only one primer\nshown, second primer is complement, the underlined codon\nrepresents the introduced mutation). bphP C12A, 5¢-GGT\nTACCCTGGCGAACGCCGAGGACGAACCCATCC-3¢;\nbphP C12S 5¢-GGTTACCCTGGCGAACTCCGAGGAC\nGAACCCATCC-3¢; bphP H247Q, 5¢-GCAGCGTTTCG\nCCGATCCAGTGCGAATACCTGACC-3¢; bphP C248A,\n5¢-CGTTTCGCCGATCCACGCCGAATACCTGACCA\nAC-3¢ and bphP H513A, 5¢-GCGGTGCTCGGCGCCG\nACCTGCGCAAC-3¢. Mutants were also conﬁrmed by\nDNA sequencing (SeqLab).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nProtein production and puriﬁcation\n**BLOCK**fs== 8.7**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\nRecombinant P. aeruginosa BphP was produced using a\ntet promoter-driven Strep tag system ([28]; IBA) in the\nEscherichia coli strain DH5a and was grown at 37 C\nampicillin (100\nin Luria–Bertani medium containing\nlgÆmL)1) to an A578 of 0.5. Cultures were induced by the\naddition of 0.2 lgÆmL)1 anhydrotetracycline and incuba-\nted at 25 C overnight. The bacterial pellet from 3 L of\nculture was resuspended in lysis buffer (50 mm Tris ⁄ HCl,\npH 8.0, 100 mm NaCl, 0.05% Triton X-100) (3 mL buffer\nper g of cells) and disrupted by sonication. Cell debris\nwas removed by ultracentrifugation (30 min, 100 000 g),\nand the supernatant was subjected to a 40% (NH4)2SO4\ncut. The resultant pellet was dissolved in buffer W\n(20 mm\n1 mm\ndithiothreitol), and after 20 min centrifugation (23 000 g),\nthe supernatant was incubated with 40 lgÆmL)1 avidin\nleast 10 min on ice. The\n(ﬁnal concentration)\nresulting supernatant was loaded on to a Strep-Tactin\nSepharose column (5 mL), which had previously been\nequilibrated with buffer W. The puriﬁcation was per-\nformed according to the instructions\nsupplied by the\nmanufacturer (IBA). Fractions containing BphP were fur-\nther puriﬁed using anion-exchange chromatography on\nQ Sepharose (Amersham Biosciences) using a linear gra-\nin 50 mm Hepes ⁄ KOH, pH 8.0.\ndient of KCl\nBphP was eluted with 500 mm KCl from the Q Sepha-\nrose column.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n20 mm NaCl,\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nTris ⁄ HCl,\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n(0–1 m)\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\npH 8.0,\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\nfor at\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nProtein determination\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nProtein concentration was determined by the Bradford\nmethod with BSA as standard [29] or by measuring A280\nusing the calculated e280nm ¼ 78 457 m)1Æcm)1 for BphP\n[30].\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nAnalytical gel permeation chromatography\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nGel permeation chromatography experiments were carried\nout using a Superdex 200 HR10 ⁄ 30 column as described\npreviously [15].\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nAssembly of PaBphP\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nIn vitro chromophore assembly of PaBphP was tested using\n20 lm recombinant apo-BphP, which was incubated with\n40 lm chromophore for 30 min at room temperature in\nthe dark (ﬁnal volume 50 lL). Absorbance spectra were\nobtained after 3 min of incubation with red light at 630 nm\n(Pfr spectrum) and after 3 min of incubation with far red\nin a volume of 500 lL\nlight at 750 nm (Pr spectrum)\n(50 mm Hepes ⁄ HCl, pH 8.0, 20 mm KCl), and the differ-\nence was calculated.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nTo characterize the different forms of holo-BphP spec-\ntroscopically, absorbance spectra between 500 and 800 nm\nwere obtained. Biliverdin IXa (20 lm) was added to 10 lm\nBphP in a ﬁnal volume of 500 lL, and spectra were meas-\nured during incubation in the dark or during irradiation\nwith red and far red light, respectively.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nTo test covalent chromophore attachment to BphP, cova-\nlently bound bilins were visualized by zinc-induced red\nﬂuorescence as described previously [31]. For iodoacetamide\ntreatment, BphP apoprotein was mixed with different con-\ncentrations of the blocking reagent from a 5 mm stock solu-\ntion and incubated for 20 min at room temperature [32].\nSpectra and zinc-induced red ﬂuorescence were measured as\ndescribed above.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nFluorescence spectroscopy\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nRoom temperature ﬂuorescence emission and excitation\nspectra were recorded using a Perkin–Elmer LS50B spectro-\nﬂuorimeter. Fluorescence spectra were measured at 570 nm\nexcitation (the absorption maximum of dihydrobiliverdin)\nor at 630 nm emission.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nBVR assay\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nA BVR assay was used to characterize the complex of\nBphP–biliverdin IXa. BVR catalyzes the conversion of bili-\nverdin IXa into bilirubin IXa, which absorbs at 450 nm.\nBVR can only convert noncovalenty bound biliverdin IXa\ninto bilirubin. Apo-BphP was incubated with an excess of\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nbiliverdin IXa for 30 min in the dark. The BphP–biliverdin\ncomplex was separated from free biliverdin IXa using\nNAP-5 desalting columns (Amersham Biosciences), which\nwere equilibrated with buffer (100 mm Tris ⁄ HCl, pH 8.7).\nThe concentration of protein-bound biliverdin IXa was\nmeasured spectroscopically and estimated using e680 ¼\n12 400 m)1Æcm)1 for free biliverdin IXa. An aliquot of\n 5 lg crude soluble protein extract of recombinant rat\nBVR was added to 20 lm biliverdin in a complex of BphP–\nbiliverdin in 100 mm Tris ⁄ HCl, pH 8.7. The reaction was\nstarted by the addition of an NADPH-regenerating system\ncontaining 6.5 mm glucose 6-phosphate, 0.82 mm NADP+\nand 1.1 UÆmL)1 glucose-6-phosphate dehydrogenase. Spec-\ntral changes between 300 and 800 nm were monitored.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nProtein kinase assays\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nAutophosphorylation was performed as described for Cph1\n[6]. Holo-BphP was irradiated with saturating red (630 nm)\nor far-red (750 nm) light before the addition of [32P]ATP[cP]\nand subsequently incubated for 30 min at room temperature\nwith the corresponding light. Radioisotope imaging was\nmonitored using a Bio-Rad Molecular Imager FX.\n**BLOCK**fs== 10.7**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nAcknowledgements\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nWe are grateful to Drs Inomata (Kanazawa University,\nKanazawa, Japan) and Lagarias (UC Davis, Davis,\nCA, USA) for the gift of chromophores. We thank\nMaria Sowa and Thorben Dammeyer for technical\nassistance. This work was supported by the Emmy-\nNoether-Program of\nthe Deutsche Forschungsge-\nmeinschaft and funds from the Fonds der Chemischen\nIndustrie to N.F.-D.\n**BLOCK**fs== 10.7**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nReferences\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n1 Davis SJ, Vener AV & Vierstra RD (1999) Bacteriophy-\ntochromes: phytochrome-like photoreceptors from non-\nphotosynthetic eubacteria. Science 286, 2517–2520.\n2 Quail PH, Boylan MT, Parks BM, Short TW, Xu Y &\nWagner D (1995) Phytochromes: photosensory percep-\ntion and signal transduction. Science 268, 675–680.\n3 Lagarias JC & Rapoport H (1980) Chromopeptides\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nfrom phytochrome. The structure and linkage of the Pr\nform of the phytochrome chromophore. J Am Chem\nSoc 102, 4821–4828.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n4 Terry MJ, Wahleithner JA & Lagarias JC (1993) Bio-\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nsynthesis of the plant photoreceptor phytochrome. Arch\nBiochem Biophys 306, 1–15.\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n5 Hughes J, Lamparter T, Mittmann F, Hartmann E,\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nGa¨ rtner W, Wilde A & Bo¨ rner T (1997) A prokaryotic\nphytochrome. Nature 386, 663.\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n6 Yeh KC, Wu SH, Murphy JT & Lagarias JC (1997) A\ncyanobacterial phytochrome two-component light sen-\nsory system. Science 277, 1505–1508.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n7 Lamparter T, Michael N, Mittmann F & Esteban B\n(2002) Phytochrome from Agrobacterium tumefaciens\nhas unusual spectral properties and reveals an N-term-\ninal chromophore attachment site. Proc Natl Acad Sci\nUSA 99, 11628–11633.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n8 Karniol B & Vierstra RD (2003) The pair of bacterio-\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nphytochromes from Agrobacterium tumefaciens are histi-\ndine kinases with opposing photobiological properties.\nProc Natl Acad Sci USA 100, 2807–2812.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n9 Bhoo SH, Davis SJ, Walker J, Karniol B & Vierstra\n**BLOCK**fs== 8.7**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nRD (2001) Bacteriophytochromes are photochromic his-\ntidine kinases using a biliverdin chromophore. Nature\n414, 776–779.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n10 Wu SH & Lagarias JC (2000) Deﬁning the bilin lyase\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\ndomain: lessons from the extended phytochrome super-\nfamily. Biochemistry 39, 13487–13495.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n11 Aravind L & Ponting CP (1997) The GAF domain: an\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nevolutionary link between diverse phototransducing pro-\nteins. Trends Biochem Sci 22, 458–459.\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n20 Frankenberg N, Mukougawa K, Kohchi T & Lagarias\nJC (2001) Functional genomic analysis of the HY2\nfamily of ferredoxin-dependent bilin reductases from\noxygenic photosynthetic organisms. Plant Cell 13,\n965–978.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n21 Lamparter T, Michael N, Caspani O, Miyata T, Shirai\nK & Inomata K (2003) Biliverdin binds covalently to\nagrobacterium phytochrome Agp1 via its ring A vinyl\nside chain. J Biol Chem 278, 33786–33792.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n22 Jorissen HJ, Quest B, Remberg A, Coursin T, Braslav-\nsky SE, Schaffner K, de Marsac NT & Ga¨ rtner W\n(2002) Two independent, light-sensing two-component\nsystems in a ﬁlamentous cyanobacterium. Eur J Biochem\n269, 2662–2671.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n23 Quest B & Ga¨ rtner W (2004) Chromophore selectivity\nin bacterial phytochromes: dissecting the process of\nchromophore attachment. Eur J Biochem 271, 1117–\n1126.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n24 Smith H (2000) Phytochromes and light signal percep-\ntion by plants: an emerging synthesis. Nature 407,\n585–591.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n25 Lamparter T (2004) Evolution of cyanobacterial and\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n12 Montgomery BL & Lagarias JC (2002) Phytochrome\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nplant phytochromes. FEBS Lett 573, 1–5.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nancestry: sensors of bilins and light. Trends Plant Sci 7,\n357–366.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n13 Huang ZJ, Edery I & Rosbash M (1993) PAS is a\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\ndimerization domain common to Drosophila period and\nseveral transcription factors. Nature 364, 259–262.\n14 Lamparter T, Carrascal M, Michael N, Martinez E,\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nRottwinkel G & Abian J (2004) The biliverdin chromo-\nphore binds covalently to a conserved cysteine residue\nin the N-terminus of Agrobacterium phytochrome Agp1.\nBiochemistry 43, 3659–3669.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n15 Wegele R, Tasler R, Zeng Y, Rivera M & Frankenberg-\nDinkel N (2004) The heme oxgygenase(s)-phytochrome\nsystem of Pseudomonas aeruginosa. J Biol Chem 279,\n45791–45802.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n16 Giraud E, Fardoux J, Fourrier N, Hannibal L, Genty\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nB, Bouyer P, Dreyfus B & Vermeglio A (2002) Bacterio-\nphytochrome controls photosystem synthesis in anoxy-\ngenic bacteria. Nature 417, 202–205.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n17 Li L & Lagarias JC (1992) Phytochrome assembly.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nDeﬁning chromophore structural requirements for cova-\nlent attachment and photoreversibility. J Biol Chem 267,\n19204–19210.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n18 Li L, Murphy JT & Lagarias JC (1995) Continuous\nﬂuorescence assay of phytochrome assembly in vitro.\nBiochemistry 34, 7923–7930.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n19 Murphy JT & Lagarias JC (1997) The phytoﬂuors: a new\nclass of ﬂuorescent protein probes. Curr Biol 7, 870–876.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n26 Elich TD, McDonagh AF, Palma LA & Lagarias JC\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\n(1989) Phytochrome chromophore biosynthesis. Treat-\nment of tetrapyrrole-deﬁcient Avena explants with\nnatural and non-natural bilatrienes leads to formation\nof spectrally active holoproteins. J Biol Chem 264,\n183–189.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n27 Cornejo J, Beale SI, Terry MJ & Lagarias JC (1992)\nPhytochrome assembly: the structure and biological\nactivity of 2(R),3(E)-phytochromobilin derived from\nphycobiliproteins. J Biol Chem 267, 14790–14798.\n28 Skerra A (1994) Use of the tetracycline promoter for\nthe tightly regulated production of a murine antibody\nfragment in Escherichia coli. Gene 151, 131–135.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n29 Bradford MM (1976) A rapid and sensitive method for\nthe quantitation of microgram quantities of protein util-\nizing the principle of protein-dye binding. Anal Biochem\n72, 248–254.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n30 Gill SC & von Hippel PH (1989) Calculation of protein\nextinction coefﬁcients from amino acid sequence. Anal\nBiochem 182, 319–326.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n31 Berkelman TR & Lagarias JC (1986) Visualization of\nbilin-linked peptides and proteins in polyacrylamide\ngels. Anal Biochem 156, 194–201.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n32 Lamparter T, Esteban B & Hughes J (2001) Phyto-\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nchrome Cph1 from the cyanobacterium Synechocystis\nPCC6803. Puriﬁcation, assembly, and quaternary struc-\nture. Eur J Biochem 268, 4720–4730.",
         "extracted",
         "",
         "Biochemical and spectroscopic characterization of the bacterial phytochrome of Pseudomonas aeruginosa",
         "None"
        ],
        [
         "23",
         "001fbed2006728779468663f65cccd51285c9f7d",
         "Two subfamilies of the polar glycopeptidolipids (GPLs) located on the surface of Mycobacterium smegmatis, along with unknown phospholipids, were recently shown to participate in the nonopsonic phagocytosis of mycobacteria by human macrophages (Villeneuve, C., G. Etienne, V. Abadie, H. Montrozier, C. Bordier, F. Laval, M. Daffe, I. Maridonneau-Parini, and C. Astarie-Dequeker. 2003. Surface-exposed glycopeptidolipids of Mycobacterium smegmatis specifically inhibit the phagocytosis of mycobacteria by human macrophages. Identification of a novel family of glycopeptidolipids. J. Biol. Chem. 278: 51291–51300). As demonstrated herein, a phospholipid mixture that derived from the methanol-insoluble fraction inhibited the phagocytosis of M. smegmatis. Inhibition was essentially attributable to phosphatidylinositol mannosides (PIMs), namely PIM2 and PIM6, because the purified phosphatidylethanolamine, phosphatidylglycerol, and phosphatidylinositol were inactive. This was further confirmed using purified PIM2 and PIM6 from M. bovis BCG that decreased by half the internalization of M. smegmatis. Both compounds also inhibited the uptake of M. tuberculosis and M. avium but had no effect on the internalization of zymosan used as a control particle of the phagocytic process. When coated on latex beads, PIM2 and polar GPL (GPL III) favored the particle entry through complement receptor 3. GPL III, but not PIM2, also directed particle entry through the mannose receptor. Therefore, surface-exposed mycobacterial PIM and polar GPL participate in the receptor-dependent internalization of mycobacteria in human macrophages.",
         "C. Villeneuve,M. Gilleron,I. Maridonneau-Parini,M. Daffé,C. Astarie-Dequeker,G. Etienne",
         "http://www.jlr.org/article/S0022227520340323/pdf",
         "\n**BLOCK**fs== 19.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nMycobacteria use their surface-exposed glycolipids\nto infect human macrophages through a\nreceptor-dependent process\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nChristelle Villeneuve, Martine Gilleron, Isabelle Maridonneau-Parini, Mamadou Daffé,\n\n\nCatherine Astarie-Dequeker,\nand Gilles Etienne\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nDépartement Mécanismes Moléculaires des Infections Mycobactériennes, Institut de Pharmacologie et\nBiologie Structurale, Unité Mixte de Recherche 5089-Centre National de la Recherche Scientiﬁque et\nUniversité Paul Sabatier, 31077 Toulouse, France\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nM. smegmatis\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nMycobacterium  smegmatis\n**BLOCK**fs== 9.5**b== 0.2**t== 0.4**l== 0.1**r== 0.5**\nAbstract Two  subfamilies  of  the  polar  glycopeptidolipids\n(GPLs) located on the surface of\n,\nMycobacterium smegmatis\nalong with unknown phospholipids, were recently shown to\nparticipate in the nonopsonic phagocytosis of mycobacteria\nby human macrophages (Villeneuve, C., G. Etienne, V. Abadie,\nH. Montrozier, C. Bordier, F. Laval, M. Daffe, I. Maridonneau-\nParini, and C. Astarie-Dequeker. 2003. Surface-exposed gly-\ncopeptidolipids  of\nspeciﬁcally\ninhibit the phagocytosis of mycobacteria by human macro-\nphages. Identiﬁcation of a novel family of glycopeptidolipids.\n278: 51291–51300). As demonstrated herein, a\nJ. Biol. Chem.\nphospholipid mixture that derived from the methanol-insol-\nuble fraction inhibited the phagocytosis of\n. In-\nhibition was essentially attributable to phosphatidylinositol\nmannosides  (PIMs),  namely  PIM\n,  because  the\nand  PIM\n\n\npuriﬁed phosphatidylethanolamine, phosphatidylglycerol, and\nphosphatidylinositol  were  inactive.  This  was  further  con-\nﬁrmed  using  puriﬁed  PIM\nBCG\nM.  bovis\nfrom\nand  PIM\n\n\nthat  decreased  by  half  the  internalization  of\nM.  smegmatis\n.\nBoth compounds also inhibited the uptake of\nM. tuberculosis\nand\nbut had no effect on the internalization of zy-\nmosan used as a control particle of the phagocytic process.\nWhen coated on latex beads, PIM\nand polar GPL (GPL III)\n\nfavored the particle entry through complement receptor 3.\nGPL III, but not PIM\nalso directed particle entry through\n2,\nthe mannose receptor.\nTherefore, surface-exposed myco-\nbacterial  PIM  and  polar  GPL  participate  in  the  receptor-\ndependent  internalization  of  mycobacteria  in  human  mac-\nrophages.\n—Villeneuve, C., M. Gilleron, I. Maridonneau-Parini,\nMycobacte-\nM. Daffé, C. Astarie-Dequeker, and G. Etienne.\nria  use  their  surface-exposed  glycolipids  to  infect  human\nmacrophages through a receptor-dependent process.\nJ. Lipid\nRes.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nM. avium\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n475–483.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n2005.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n46:\n**BLOCK**fs== 9.8**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nThe outcome of mycobacterial infections is thought to\nbe  critically  dependent  on  the  nonopsonic  invasion  and\ncolonization of macrophages during primary infection of\ninhaled bacteria. This is why the initial step of mycobacte-\nria phagocytosis by macrophages received considerable at-\ntention, including efforts to delineate the molecular mech-\nanisms that underlie this process.\n**BLOCK**fs== 9.8**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nNonopsonic  internalization  of  mycobacteria  in  macro-\nphages  can  be  mediated  by  different  kinds  of  receptors\nthat speciﬁcally recognize ligands expressed at the surface\nof bacilli (1, 2). Glycophosphoinositol-anchored receptors\nsuch as CD14 (3, 4), scavenger receptor (5), and transfer-\nrin receptor (6, 7) have been suggested as putative routes\nof entry for mycobacteria into macrophages. Additionally,\ncomplement  receptor  3  (CR3)  (8–10)  and  mannose  re-\nceptor (MR) (8, 11) are both well-known receptors for the\nbinding  and  internalization  of  mycobacteria  in  macro-\nphages.  Both  receptors  exhibited  lectin-like  properties.\nIndeed, MR is a monomeric transmembrane protein with\nan  extracellular  domain  containing  eight  carbohydrate\nrecognition  domains  with  C-type  lectin  properties  that\nrecognize mannose, fucose, and\n-acetylglucosamine resi-\ndues present on the cell surface of pathogens, such as my-\ncobacteria  (12–14).  CR3  is  a  heterodimeric  type  1  trans-\nmembrane  glycoprotein  consisting  of  a  CD18\nsubunit\nnoncovalently associated with the CD11b\nchain. The ex-\ntracellular portion of CD11b contained distinct functional\nbinding domains. One of these, called the I domain, is es-\nsential for the recognition and internalization of C3bi op-\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nN\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nSupplementary  key  words\nphatidylinositol mannosides\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nmycobacterium\nphagocytosis\n•\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nphospholipids\n•\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nphos-\n•\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nManuscript received 13 August 2004 and in revised form 4 November 2004.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nPublished, JLR Papers in Press, December 1, 2004.\nDOI 10.1194/jlr.M400308-JLR200\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nCopyright © 2005 by the American Society for Biochemistry and Molecular Biology, Inc.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nAbbreviations:  CR3,  complement  receptor  3;  GPL,  glycopeptido-\nlipid; MALDI-TOF, matrix-assisted laser-desorption/ionization time of\nﬂight;  Man-BSA,  mannosylated  bovine  serum  albumin;  MDM,  mac-\nrophages  derived  from  monocytes;  MR,  mannose  receptor;  PE,  phos-\nphatidylethanolamine;  PG,  phosphatidylglycerol;  PI,  phosphatidylino-\nsitol; PIM, phosphatidylinositol mannoside.\n**BLOCK**fs== 5.6**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.1**\nTo whom correspondence should be addressed.\ne-mail: catherine.astarie-dequeker@ipbs.fr (C.A-D.);\ngilles.etienne@ipbs.fr (G.E.)\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.1**r== 0.6**\nThis is an Open Access article under the CC BY license.\n**BLOCK**fs== 9.8**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nsonized  particles  (15,  16).  Another  one,  located  on  the\ncarboxyl-terminal  end  of  the  extracellular  portion  of\nCD11b, is a nonopsonic binding site with lectin properties\n(17). This latter domain is mainly involved in phagocytosis\nof  particles  containing\n-glucan,  such  as  zymosan  (18),\nbut also interacts with mannose, glucose, and\n-glu-\ncosamine (17). More recently, a second nonopsonic bind-\ning  domain  distinct  from  the  lectin  one  has  been  pro-\nposed to bind and internalize mycobacteria (19).\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n-acetyl-\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nN\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nd\n**BLOCK**fs== 9.8**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\ns\n**BLOCK**fs== 9.8**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nmegmatis\n**BLOCK**fs== 9.8**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nWe  recently  demonstrated  that  mycobacteria  express\nmolecules at their surface that play a role in their internal-\nization  by  human  macrophages  (20).  Using  a  biological\ntest  of  phagocytosis  inhibition,  we  found  that  lipids,  but\nnot proteins, isolated from the outermost layer of\nMycobac-\nterium\nby  a  mechanical  treatment  decrease  the\nnonopsonic phagocytosis of mycobacteria by human mac-\nrophages without affecting the internalization of particles\nsuch as zymosan or latex beads (20). A detailed analysis of\nthe lipid fractions has indicated that the major inhibitory\ncompounds  belong  to  the  glycopeptidolipid  (GPL)  and\nphospholipid  families  (20).  When  the  former  family  was\n**BLOCK**fs== 9.8**b== 0.9**t== 0.1**l== 0.7**r== 0.3**\nFig. 1\n**BLOCK**fs== 9.8**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nchemically  and  structurally  analyzed,  several  inhibitory\nC-type GPLs were identiﬁed, among which a novel class of\nsuccinylated GPLs (\n) passed unnoticed until now (20).\nWe  have  also  proposed  that  their  capacity  to  inhibit  the\nphagocytosis  of  mycobacteria  is  dependent  on  common\nstructural features. Identiﬁcation of these molecules and\ntheir  putative  role  as  surface-exposed  ligands  of  phago-\ncytic receptors would help us understand the host-patho-\ngen interactions at the molecular level. The present study\nwas  designed  ﬁrst  to  identify  the  mycobacteria  surface-\nexposed  phospholipids  that  play  a  role  in  their  internal-\nization  by  human  macrophages  and  second  to  examine\nthe roles of GPLs and phospholipids as potential ligands\nof phagocytic receptors.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nMETHODS\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nReagents and antibodies\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nMannosylated  BSA  (Man-BSA),  zymosan,  soya  phosphatidyl-\ninositol  (PI),  and  ﬂuorescein  isothiocyanate  were  from  Sigma\n**BLOCK**fs== 8.5**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\nFig. 1. Structures of the major glycopeptidolipid (GPL) and phosphatidylinositol mannoside (PIM) spe-\ncies found in M. smegmatis. R1, H or CH3; R2, CH3 in GPL I, succinyl in GPL IIIa, 2-O-succinyl-rhamnosyl in\nGPL IIIb. Asterisks indicate the positions of acylation in PIMs that are always diacylated on the glycerol moi-\nety; they may also be diacylated at the indicated positions.\n**BLOCK**fs== 8.7**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nChemical Co. (St Louis, MO). Polystyrene microspheres were from\nPolysciences, Inc. RPMI 1640, Ficoll-Hypaque,\n-glutamine, and\nantibiotics were purchased from Eurobio (Les Ulysses, France).\nMouse  antibodies  against  the  CD11b  extracellular  domain  was\nused [2LPM (1:20); IgG1\n; Dako, Glostrup, Denmark]. Goat anti-\n\n(\n);\nhuman MR was a generous gift of P. Stahl (1:50). Anti-mycobac-\nterium rabbit antibody Camelia (1:50) was obtained as previously\ndescribed (19). Secondary Abs were purchased from Sigma.\n**BLOCK**fs== 8.7**b== 0.9**t== 0.1**l== 0.4**r== 0.6**\nl\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nMycobacteria\nM. smegmatis\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nM. avium\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nM. smegmatis\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nM. tuberculosis\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n(ATCC 607) and\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nC without agitation;\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nand\nM. smegmatis\n**BLOCK**fs== 8.7**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nH37Rv (ATCC\n27294)  were  grown  as  surface  pellicles  on  liquid  Sauton’s  me-\n\ndium at 37\n(serovar 4), known to\nform more clumps when grown as surface pellicles than the two\nformer species, was cultured under shaking at 250 rpm in Mid-\ndlebroock  7H9  medium  supplemented  with  10%  albumin-dex-\ntrose-catalase  enrichment  (Difco).  It  is  important  to  point  out\nthat the qualitative composition of the surface-exposed materials\nis independent of growth conditions (i.e., surface pellicles or\nshaken cultures) (21). Single cell suspensions were prepared with\nlate log-phase cultures (6 days for\nM. tuber-\nculosis\n) as previously described (20). Brieﬂy, pelli-\nM. avium\ncles (\n) were harvested by pouring\noff the medium, dispersed by gentle shaking for 30 s with 5 g of\nglass beads (4 mm diameter), and resuspended in PBS, pH 7.4.\nIn the case of\nM. avium\n, the culture was recovered by centrifuga-\ntion at 10,000\nfor 10 min. To remove the remaining clumps, the\ng\nbacterial suspensions were centrifuged for 10 min at 200\n. Up\nto  90%  of  mycobacteria  were  individualized,  and  their  viability\naveraged  85%.  When  required,  mycobacteria  were  labeled  with\nFITC, as previously described (22). We checked that their rate of\ninternalization and their route of entry within macrophages de-\nrived from monocytes (MDM) were identical to those obtained\nby  green  ﬂuorescent  protein-expressing\n(data  not\nshown), ruling out a side effect of FITC labeling.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nM. tuberculosis\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nM.  smegmatis\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n, 3 weeks for\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nand\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\ng\n**BLOCK**fs== 9.8**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\nExtraction and puriﬁcation of glycolipids\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n/CH\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nM. smegmatis\n**BLOCK**fs== 8.7**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nGlycolipids and phospholipids are present in both the outer-\nmost layer of the cell envelope and whole mycobacteria (21, 23).\nTherefore,  the  present  study  was  performed  on  lipid  extract\nfrom\nwhole cells to get enough material for further\nfractionation experiments (20). Mycobacterial wet cells were ex-\ntracted  with  CHCl\nOH  (1:2,  v/v)  and  then  at  last  three\n\ntimes  with  CHCl\nOH  (1:1,  v/v).  The  organic  phases  were\n/CH\n\npooled,  extensively  washed  with  water,  and  evaporated  to  dry-\nness. Lipids were then resuspended in a minimal volume of chlo-\nroform and precipitated with trickling methanol. After standing\n\nfor 2 h at 4\nC, methanol-insoluble lipids were recovered by cen-\n\ntrifugation at 4\n). Lipids were further sepa-\nrated using an anion-exchange QMA-silica gel (Chromabond SB;\nMacherey-Nagel). Phospholipids were eluted using a stepwise gra-\ndient: 0.1 M ammonium acetate in 33% methanol in chloroform\n(fractions 9 and 10), 0.2 M ammonium acetate in 33% methanol\nin chloroform (fractions 11–13), and 0.2 M ammonium acetate\nin  methanol  (fraction  14).  Puriﬁcation  of  phosphatidylethanol-\namine (PE) and phosphatidylglycerol (PG) was achieved by pre-\nparative  TLC.  Plates  were  developed  in  CHCl\nO\n\n(56:38:10, v/v/v); silica bands were scraped and extracted three\ntimes in the same solvent, adding trace amounts of acetic acid.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nC for 20 min (8,000\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nOH/H\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n/CH\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\ng\n**BLOCK**fs== 6.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.1**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe  puriﬁcation  of  GPL  has  been  described  in  detail  (20).\nBrieﬂy, the methanol-soluble lipids were chromatographed on a\nFlorisil column irrigated with chloroform and then with a step-\nwise gradient of increasing concentrations of methanol and wa-\nter in chloroform. GPL I was eluted with CHCl\nOH (90:10,\n\nv/v), whereas GPL III was eluted in more polar fractions with\na  mixture  of  CHCl\nO  (65:25:4,  v/v/v);  the  latter\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nOH/H\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n/CH\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n/CH\n**BLOCK**fs== 6.1**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.7**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nwere further separated from contaminating phospholipids using\nan anion-exchange QMA-silica gel (Chromabond SB; Macherey-\nNagel).\n**BLOCK**fs== 6.1**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.1**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n/CH\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nOH/H\n**BLOCK**fs== 8.7**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nAll of the puriﬁcation steps were monitored by TLC on silica\ngel  60-precoated  plates  (0.25  mm  thickness;  Merck)  developed\nwith  CHCl\nO  (60:35:8)  for  the  phospholipid-con-\n\ntaining  fractions.  Sugar-containing  compounds  were  visualized\nby  spraying  plates  with  1.5%  orcinol  in  30%  aqueous  sulfuric\nacid,  followed  by  heating,  whereas  the  Dittmer-Lester  reagent\nand  ninhydrin  were  used  to  detect  phosphorus-  and  amino\ngroup-containing  substances.  Phosphatidylinositol  mannosides\ntype 2 and 6 (PIM\n) (Fig. 1) were puriﬁed on a QMA-\n\nSpherosil M (BioSepra) column as previously described (24).\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nand PIM\n**BLOCK**fs== 9.8**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nMass spectrometry\n**BLOCK**fs== 8.7**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nMatrix-assisted laser-desorption/ionization time of ﬂight (MALDI-\nTOF) mass spectrometry analysis of lipids was performed as previ-\nously described (25). MALDI-TOF spectra were acquired on a Voy-\nager-DE STR mass spectrometer (PerSeptive Biosystems) equipped\nwith  a  pulsed  nitrogen  laser  emitting  at  337  nm  and  were  ana-\nlyzed in the Reﬂectron mode using an extraction delay time set\nat 100 ns and an accelerating voltage operating in negative ion\nmode of 20 kV.\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nHuman MDM\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nHuman  peripheral  blood  monocytes  were  isolated  as  previ-\nously described (11) and cultured for 7 days on sterile glass cov-\nerslips on 24-well tissue culture plates (5\ncells/well) con-\ntaining RPMI medium with 10% heat-inactivated fetal calf serum\nin 5% CO\n. The culture medium was renewed on the third day.\n\nBefore use, MDM were washed twice with fresh RPMI medium.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.8**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nCoating of latex beads\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nm diameter were washed twice (10,000\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nLatex  beads  were  coated  as  described  by  Schlesinger,  Hull,\n\nand Kaufman (26). Brieﬂy, 2\npolystyrene microspheres of\n\n\n, 10 min) in 0.05 M\ncarbonate-bicarbonate  buffer,  pH  9.6.  Beads  were  then  incu-\n\nbated for 1 h at 37\ng of\nvarious glycolipids, Man-BSA, or BSA in a reaction volume of 1\nml. The microspheres were washed twice, incubated in 5% BSA\n\nin  PBS  for  2  h  at  37\nC,  resuspended  in  5%  BSA  in  PBS,  and\n\n.\nstored at 4\nC until the experiments\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.7**r== 0.1**\nC under gentle agitation with either 50\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\ng\n**BLOCK**fs== 9.8**b== 0.3**t== 0.6**l== 0.5**r== 0.2**\nInfection of adherent macrophages and\nphagocytosis assay\n**BLOCK**fs== 8.7**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\n\nWhen speciﬁed, MDM were ﬁrst pretreated for 15 min at 37\nC\nwith the puriﬁed lipid fractions, which were previously resuspended\nin  sterile  apyrogenic  water  and  sonicated  for  10  min,  and  then\nadded with particles at the appropriate multiplicity of infection.\nAt  the  end  of  infection,  cells  were  washed  twice  with  fresh  me-\ndium to remove unbound particles. Phagocytosis of mycobacteria,\nzymosan,  and  latex  beads  was  assessed  by  immunoﬂuorescence\nas  previously  described  (11).  For  each  set  of  conditions,  dupli-\ncate experiments were performed, and at least 100 cells per slide\nwere  counted  by  ﬂuorescence  microscopy.  The  percentage  of\nphagocytic cells having ingested at least one particle or bacterium\nwas calculated as follows: number of phagocytic cells/number of to-\ntal cells\n100. Data are expressed as the percentage of phagocyto-\nsis inhibition determined as follows: number of phagocytic cells\nin  the  control  minus  number  of  phagocytic  cells  under  treated\nconditions/number of phagocytic cells in the control\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.9**r== 0.1**\n100.\n**BLOCK**fs== 8.7**b== 0.1**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 9.8**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nStatistics\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nData are presented as means\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n\nSEM of the indicated number\nof experiments (n) performed in duplicate. The signiﬁcance of the\ndifferences was determined by paired or unpaired Student’s\n-test.\nt\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.3**r== 0.7**\nRESULTS\n**BLOCK**fs== 9.8**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nInhibitory activity of surface-exposed phospholipids on\nthe phagocytosis of\n**BLOCK**fs== 9.8**b== 0.8**t== 0.1**l== 0.2**r== 0.7**\nM. smegmatis\n**BLOCK**fs== 6.8**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.8**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nFig. 2\n**BLOCK**fs== 9.8**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nand PIM\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nM. smegmatis\n**BLOCK**fs== 9.8**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nIn the present study, we ﬁrst attempted to identify the\nsurface-exposed phospholipids implicated in the internal-\nization  of  mycobacteria.  For  this  purpose,  lipid  extract\nfrom\nwas ﬁrst enriched in phospholipids by\nmethanol precipitation. As depicted in\n, the metha-\nnol-insoluble fraction, analyzed by TLC, contained several\ncompounds: PIMs, in particular PIM\n(Fig. 1) as\nthe  main  subspecies,  PI  (Fig.  2A),  PG  (Fig.  2B),  and  PE\n(Fig. 2B). Examination of the blocking effect of the meth-\nanol-insoluble fraction on phagocytosis was the approach\nchosen  to  determine  the  role  of  phospholipids  in  the\nphagocytic process of mycobacteria. MDM were pretreated\nfor 15 min with the phospholipid-enriched extract at con-\ncentrations  ranging  from  10  pg  to  100\ng  lipid/ml  and\nthen put into contact with\nfor 45 min. At the\nlowest tested concentration (i.e., 10 pg lipid/ml), treated\nand untreated macrophages equally internalized\nM. smeg-\n\nmatis\n2). In con-\ntrast, increasing lipid concentration to 10 ng/ml resulted\n\nin a 30% decrease in the uptake of\nM. smegmatis\n6% to 45\n2). This inhibitory effect remained\nstable up to 100\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n5%, respectively; n\n**BLOCK**fs== 9.8**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\n\ng/ml.\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nM. smegmatis\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n6% and 61\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n(from 62\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n1%; n\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(62\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 9.8**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 9.8**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 9.8**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nPuriﬁcation and identiﬁcation of the active phospholipid\nfractions of\n**BLOCK**fs== 9.8**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nM. smegmatis\n**BLOCK**fs== 9.8**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nTo establish the nature of the active phospholipids, we\n\ng/ml\n. Prein-\ng/ml commercial-\n**BLOCK**fs== 9.8**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\nindividually tested the effect of the addition of 10\nPI, PE, and PG on the phagocytosis of\nM. smegmatis\n\ncubation of MDM for 15 min with 10\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nFig. 2.\nIdentiﬁcation of phospholipids in lipids extracted from M.\nsmegmatis whole cells. Phospholipids from M. smegmatis were enriched\nby methanol precipitation. Composition of the methanol-insoluble\nfraction (MIF) was analyzed by TLC using CHCl3/CH3OH/H2O\n(60:35:8, v/v/v) as the solvent system. Different phospholipids [i.e.,\nPIM2, PIM6, phosphatidylinositol (PI), phosphatidylethanolamine\n(PE), phosphatidylglycerol (PG)] were used as controls. Compounds\nwere revealed by spraying with orcinol, following by heating (A) or\nby the Dittmer reagent (B).\n**BLOCK**fs== 9.8**b== 0.9**t== 0.1**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.8**b== 0.9**t== 0.1**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 9.8**b== 0.9**t== 0.1**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n5%;  n\n**BLOCK**fs== 9.8**b== 0.9**t== 0.1**l== 0.8**r== 0.1**\n7% vs. 50\n**BLOCK**fs== 9.8**b== 0.9**t== 0.1**l== 0.7**r== 0.2**\n3). Similarly, 10\n**BLOCK**fs== 9.8**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nM. smegmatis\n\n4% and 49\n**BLOCK**fs== 9.8**b== 0.9**t== 0.1**l== 0.5**r== 0.2**\ncompared with untreated cells (52\n**BLOCK**fs== 9.8**b== 0.5**t== 0.1**l== 0.5**r== 0.1**\nized PI did not inﬂuence the internalization of\nM. smegma-\n\ntis\n5%,\nrespectively; n\ng/ml PE or PG, puri-\nﬁed  from  mycobacteria  cell  extracts  by  preparative  TLC,\ndid not affect the phagocytosis of\ncompared\n\nwith untreated macrophages (43\n3%, re-\n\nspectively,  vs.  50\n3).  These  data  strongly  sug-\ngested  that  the  observed  inhibitory  activity  of  the  phos-\npholipid-rich  methanol-insoluble  fraction  was  essentially\nattributable  to  PIM.  This  hypothesis  was  further  con-\nﬁrmed using PIM-enriched fractions obtained from a\nM.\nsmegmatis\nlipid  extract  as  follows:  methanol-insoluble  lip-\nids  were  fractionated  by  ion-exchange  chromatography\ninto 14 fractions eluted with increasing concentrations of\nammonium  acetate  in  chloroform-methanol.  TLC  and\nMALDI-TOF analysis showed that the ﬁrst 8 fractions con-\ntained  traces,  if  any,  of  phospholipids  (data  not  shown),\nwhereas  fractions  9–14  contained  essentially  PIM  and  PI\nFig. 3A\n(\n). Therefore, these last six fractions were tested on\nphagocytosis of\nat 10 ng lipid/ml, the low-\nest active concentration for the phospholipid mixture. As\nshown in Fig. 3B, the tested fractions exhibited an inhibi-\ntory  effect  of\n50%.  This  set  of  experiments  indicated\nthat PIMs were the compounds responsible for the activity\nof  the  phospholipid  mixture  and  therefore  were  the  es-\nsential  phospholipids  involved  in  the  phagocytic  process\nof mycobacteria.\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nM. smegmatis\n**BLOCK**fs== 9.8**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 6.8**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 6.8**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 9.8**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\nPIM\n**BLOCK**fs== 9.8**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n(Ac\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nM. bovis\n**BLOCK**fs== 9.8**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nand PIM\n**BLOCK**fs== 9.8**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nsubfamilies\n**BLOCK**fs== 9.8**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nM. smegmatis\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.6**r== 0.2**\n(27).  As  depicted  in\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.6**r== 0.2**\nsubfamilies isolated from\n**BLOCK**fs== 9.8**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nInhibition of mycobacterial internalization by PIM\nand PIM\nPIM\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.6**r== 0.2**\nsubfamilies  reduced  phagocytosis  of\n**BLOCK**fs== 9.8**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nPIM\n\nand  triacylated  PIM\n**BLOCK**fs== 9.8**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\n(Fig. 1) represent the main mycobacte-\nrial glycophospholipids described to date (24, 27, 28). As\nexpected, these two subclasses of PIM constituted the ma-\njority of glycophospholipids of\nand were dif-\nferently distributed in fractions 9–14 (Fig. 3A). PIM\nand\ntriacylated  PIM\n)  were  preferentially  eluted  in\n(Ac\nfractions  9–11.  PIM\n)  ap-\npeared  mainly  in  fractions  12–14.  However,  we  failed  to\nproperly separate PIM\nby the puriﬁcation pro-\nfrom PIM\ntocol  used.  This  prompted  us  to  use  puriﬁed  PIM\nand\nPIM\nBCG by Gilleron\net al. (24), which are structurally identical to those of\nM.\nsmegmatis\nand\nPIM\n.\nM.  smegmatis\nTheir inhibitory activity increased from 100 pg to 10 ng of\nlipid per milliliter, PIM\nbeing slightly more efﬁcient than\nPIM\n. A plateau of inhibition was then reached at 10 ng/\nml up to 10\nand PIM\ng/ml. Importantly, 10 ng/ml PIM\n\nsubfamilies did not affect zymosan internalization, used as\na control of phagocytic process (33\n11%,\nrespectively,  vs.  35\n3  and  4),  indicating  a\nspeciﬁc  inhibition  of  the  MDM-particle  interactions  by\nstructurally  deﬁned  molecules.  We  then  addressed  the\nquestion  of  whether  PIM\ninterfered  with  the\n\nphagocytosis  of  the  pathogenic  mycobacteria  (e.g.,\nM.\navium\n). When tested at 10 ng/ml, both\ncompounds  reduced  the  internalization  of  bacteria  (Fig.\n4B). However, micromolar concentrations of PIM were re-\nquired  to  achieve  an  inhibition  of  phagocytosis  of  the\n**BLOCK**fs== 9.8**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nM. tuberculosis\n**BLOCK**fs== 9.8**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\n9% and 36\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n,  both  PIM\n**BLOCK**fs== 9.7**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\nand  PIM\n**BLOCK**fs== 9.8**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n9%;  n\n**BLOCK**fs== 9.8**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nFig.  4A\n**BLOCK**fs== 9.8**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\nand\n**BLOCK**fs== 9.8**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.8**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 9.8**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 9.8**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.8**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 6.8**b== 0.1**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 6.8**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.8**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.8**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.1**\nFig. 3. The phospholipid-enriched fractions of M. smegmatis inhibit its phagocytosis. A: Methanol-insoluble lipids of M. smegmatis were sep-\narated by ion-exchange chromatography and eluted with a stepwise gradient of 0.1 M ammonium acetate in 33% methanol in chloroform\n[fractions 9 (F9) and F10], 0.2 M ammonium acetate in 33% methanol in chloroform (F11 to F13), and 0.2 M ammonium acetate in meth-\nanol (F14). The various PIM species and their relative abundances were determined by matrix-assisted laser-desorption/ionization time of\nﬂight mass spectrometry in the negative mode. PI was present in each fraction. PIM2 and the triacylated form of PIM2 (Ac1PIM2) were en-\nriched in fractions 9–11 and to a lesser extent in fractions 11–14. PIM6 and the triacylated form of PIM6 (Ac1PIM6) appeared in fractions 12–14.\nB: Macrophages derived from monocytes (MDM) were preincubated with fractions 9–14 used at 10  g lipid/ml for 15 min and then chal-\nlenged with FITC-stained M. smegmatis for 45 min at 37C (25 particles per cell). Phagocytosis was determined as the number of macro-\nphages having engulfed at least one particle. Phagocytosis inhibition is expressed as the percentage effect compared with the control. Results\nare means  SEM of three independent experiments performed in duplicate. Signiﬁcance of the treatment effect was assessed by paired\nStudent’s t-test. * P  0.05, ** P  0.01 compared with untreated conditions.\n**BLOCK**fs== 9.8**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\npathogenic species (Fig. 4B) comparable to what was ob-\nserved  with  nanomolar  concentrations  of\nM.  smegmatis\n(Fig. 4A). Nevertheless, these observations clearly demon-\nstrated that subfamilies of PIM are involved in the nonop-\nsonic phagocytic process of mycobacteria.\n**BLOCK**fs== 9.8**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nRecognition of PIM and GPL by nonopsonic\nphagocytic receptors\n**BLOCK**fs== 9.8**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nRecently,  we  reported  that  subfamilies  of  another  sur-\n,  GPL  (Fig.1),  in-\n**BLOCK**fs== 9.8**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nface-exposed  glycolipid  of\n**BLOCK**fs== 9.8**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nM.  smegmatis\n**BLOCK**fs== 9.8**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nhibit the phagocytosis of mycobacteria (20). We have also\ndemonstrated  that  the  nonopsonic  phagocytosis  of  both\npathogenic  and  nonpathogenic  mycobacteria  by  human\nphagocytes is mediated at least in part by two major recep-\ntors, CR3 and MR (11, 29). Therefore, the ability of PIM\nand GPL to inhibit mycobacteria internalization by block-\ning their entry through these receptors was suspected and\ninvestigated. To this aim, latex beads were coated with pu-\nriﬁed PIM or GPL using the method previously described\nby  Schlesinger,  Hull,  and  Kaufman  (26).  As  PIM\nand\n**BLOCK**fs== 9.8**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\ntween the internalization of PIM2- or GPL- and BSA-coated\nbeads (data not shown). As expected, pretreatment of MDM\nwith 2LPM (5 g/ml) for 30 min at 37C signiﬁcantly de-\ncreased  the  internalization  of  serum-opsonized  and  non-\nopsonized zymosan but did not affect the phagocytosis of\nBSA-coated beads (Fig. 5A). Such pretreatment also reduced\nby approximately one-third the entry of both GPL III- and\nPIM2-coated beads but not that of GPL I (Fig. 5A), indicat-\ning  that  entry  of  PIM2-  and  GPL  III-coated  beads,  but  not\nthat of beads coated with GPL I, involves CR3.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nFig. 4. PIM2 and PIM6 inhibit the phagocytosis of pathogenic and\nnonpathogenic mycobacteria. MDM were preincubated for 15 min\nat 37C with or without puriﬁed PIM2 (black diamonds and open\nbars) and PIM6 (gray squares and closed bars) from M. bovis BCG at\nconcentrations ranging from 100 pg lipid/ml to 10 g lipid/ml\nand then challenged with FITC-M. smegmatis (25 particles/cell) for\n45 min at 37C (A) or from 10 ng lipid/ml to 10 g lipid/ml and\nthen put in contact with M. avium and M. tuberculosis (25 particles\nper cell) for 45 min at 37C (B). Inhibition is expressed as the per-\ncentage effect compared with the control. Results are means  SEM\nof three to four independent experiments in duplicate. Statistical\ndifferences were determined for cells treated with 10 g lipid/ml\ncompared with 10 ng lipid/ml by paired Student’s t-test. * P  0.05.\n**BLOCK**fs== 9.8**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nPIM6 displayed comparable activities on the phagocytosis\nof  mycobacteria,  PIM2,  the  quantitatively  major  constitu-\nent of PIM, was isolated from M. bovis BCG and used for\nthe  subsequent  experiment  as  representative  of  the  PIM\nfamily. Beads were also coated with either GPL III or GPL I\nfrom M. smegmatis, two classes of GPL that have been shown\nto exhibit different inhibitory activities on MDM (20).\n**BLOCK**fs== 9.8**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe capacity of the coated beads to enter macrophages\nthrough MR or CR3 was then investigated using a compe-\ntition approach with blocking antibodies directed against\nreceptors or soluble ligands. Monoclonal antibody 2LPM\nwas selected for its capacity to prevent both opsonic and\nnonopsonic internalization through CR3 (19, 30, 31). Se-\nrum-opsonized  and  nonopsonized  zymosan  were  used  as\npositive CR3 binding particles (19) and BSA-latex beads as\na negative control. We did not observe any difference be-\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nFig. 5.\nInteraction of PIM2 and GPL III with complement recep-\ntor 3 (CR3) and mannose receptor (MR). A: MDM were incubated\nfor 15 min at 37C with or without 5 g/ml 2LPM Ab directed\nagainst CR3 and then allowed to ingest either serum-opsonized zy-\nmosan (OZ; 10 particles per cell) for 15 min or zymosan (Z; 25 par-\nticles per cell) and latex beads (LB; 50 particles per cell) coated\nwith BSA, GPL I, GPL III, PIM2, or mannosylated BSA (Man-BSA)\nfor 45 min. B: MDM were pretreated at 37C with or without anti-\nMR antiserum (1:50) for 30 min or 300 g/ml Man-BSA for 15 min\nand then challenged with BSA-, GPL I-, GPL III-, PIM2-, or Man-\nBSA-coated beads (50 particles per cell) for 45 min. Intracellular zy-\nmosan, opsonized zymosan, and coated latex beads were visualized\nby immunoﬂuorescence after staining of the phagosomal mem-\nbrane. Data are expressed as percentage of phagocytosis inhibition\ncompared with untreated macrophages. Inhibition is expressed\nas the percentages effect compared with the control. Results are\nmeans  SEM of three to four independent experiments in dupli-\ncate. Signiﬁcance of the treatment effect was assessed by paired Stu-\ndent’s t-test. * P  0.05, ** P  0.01 compared with untreated con-\nditions.\n**BLOCK**fs== 9.8**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\nThe entry of beads through the MR was also evaluated\nusing a competitive ligand, Man-BSA (11), or a goat anti-\nhuman  MR  antiserum  selected  for  its  capacity  to  block\nMR-dependent  functions  (32,  33).  Man-BSA-  and  BSA-\ncoated latex beads were used as positive and negative con-\ntrols of MR-dependent phagocytosis, respectively (11). In\nthis  set  of  experiments,  GPL  III-coated  beads  were  inter-\nnalized as efﬁciently as beads coated with Man-BSA (45 \n5% vs. 33  7%; n  3), whereas PIM2-coated beads were\nslightly better internalized (58  1% vs. 33  7%; n  3).\nAs  expected,  pretreatment  of  MDM  with  anti-MR  antise-\nrum (1:50) or 300 g/ml Man-BSA for 30 min at 37C dra-\nmatically  reduced  the  internalization  of  Man-BSA-coated\nbeads  (Fig.  5B).  Such  pretreatments  did  not  affect  the\nphagocytosis of either GPL I- or PIM2-coated beads. In con-\ntrast,  both  treatments  slightly  but  signiﬁcantly  decreased\nthe  phagocytosis of GPL III-coated beads  (Fig.  5B), indi-\ncating that only GPL III-coated beads enter MDM through\nthe MR. Taken together, these data demonstrate that both\nPIM2 and GPL III participate to the internalization of my-\ncobacteria and thereby are potential ligands of phagocytic\nreceptors expressed at the surface of MDM.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\nDISCUSSION\n**BLOCK**fs== 9.8**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nThe  outermost  cell  envelope  layer,  also  called  a  cap-\nsule in the case of pathogenic mycobacteria, represents a\nprivileged interface between mycobacteria and host cells.\nSome  of  its  components,  such  as  glucan  (34)  and  GPL\n(20),  have  been  implicated  in  the  interaction  with  mac-\nrophages. Other constituents of the mycobacterial cell en-\nvelope  have  also  been  shown  to  induce  various  host  cell\nresponses (35), but their localization at the cell surface is\nstill a matter of debate. We show here that the outermost\nlayer  of  M.  smegmatis  contained  PIMs  that  also  partici-\npate in the nonopsonic phagocytosis of mycobacteria. The\nPIMs, described by Ballou and colleagues (36) in the 1960s,\nare  based  on  PI  and  are  formed  of  one  to  six  mannosyl\nresidues  attached  to  the  inositol  moiety  of  the  phospho-\nlipid. The PIMs usually occur as a mixture of compounds\ndiffering  from  one  another  by  the  number  of  mannosyl\nresidues and fatty acyl groups. The nonpathogenic strain\nM.  smegmatis  ATCC  607  has  been  described  to  contain\nmainly  two  PIM  subfamilies,  PIM2  and  PIM6  (24,  27),  in\nagreement  with  the  composition  of  the  crude  PIM  frac-\ntions reported herein.\n**BLOCK**fs== 9.8**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAlthough other forms of PIM have been reported in the\nliterature (24), they were undetectable under our experi-\nmental  conditions,  probably  because  they  occur  in  too\nsmall quantities. Therefore, the crude PIM-dependent in-\nhibition  of  mycobacterial  phagocytosis  was  essentially  at-\ntributable to PIM2 and PIM6. This was further conﬁrmed\nusing enriched PIM2 and PIM6 fractions isolated from M.\nbovis  BCG.  Nanomolar  concentrations  of  PIM  decreased\nthe internalization of M. smegmatis by MDM, whereas much\nhigher concentrations of lipids were required to affect M.\navium  or  M.  tuberculosis  phagocytosis  to  the  same  extent.\nPIMs  are  found  on  the  bacterial  surface  of  both  patho-\n**BLOCK**fs== 9.8**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\ngenic  and  nonpathogenic  mycobacteria  (27,  37).  How-\never, there are strain differences in the surface exposure\nof  PIMs  that  are  inversely  correlated  with  the  abundant\nexpression of capsular polysaccharides composed of -glu-\ncan that can partially mask capsular PIM and directly act\non  phagocytic  receptor,  such  as  CR3  (30).  If  we  assume\nthat PIMs, exposed at the surface of M. smegmatis, have a\nlower afﬁnity for the phagocytic receptors than the outer-\nmost polysaccharides of pathogenic species (38), this may\nexplain why different PIM concentrations are required to\ninhibit  to  a  comparable  extent  the  phagocytosis  of  M.\nsmegmatis and M. avium or M. tuberculosis.\n**BLOCK**fs== 9.8**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nIt has to be noted that the occurrence of a carbohydrate\nportion on phospholipid is critical for the inhibitory activ-\nity, because PI displayed no effect but PIM2 did. However,\nsurprisingly,  the  length  of  the  oligomannosyl  chain  does\nnot appear to be important for the activity, because PIM2\nand PIM6 equally inhibited phagocytosis. Previous studies\nhave  reported  that  soya  PI  and,  to  a  lesser  extent,  myco-\nbacterial PI inhibited the nonopsonic binding of M. tuber-\nculosis to murine macrophages (39, 40). This is in apparent\nconﬂict with the data presented here. Such a discrepancy\ncould  be  attributed  to  interspecies  differences  in  the\nPI  mode  of  action  between  murine  and  human  macro-\nphages. Our data, together with our previous report (20),\ncollectively  indicate  that  surface-exposed  PIM  and  GPL\nparticipate in the internalization of mycobacteria by mac-\nrophages and, as such, are potential ligands of phagocytic\nreceptors.  It  is  possible,  however,  that  other  constituents\nof  the  mycobacterial  envelope,  such  as  glucan  and  un-\nidentiﬁed molecules, could also be involved in the interac-\ntions of bacilli with macrophages, as illustrated by the vari-\nety of routes of entry of mycobacteria into host cells.\n**BLOCK**fs== 9.8**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nThe  molecular  mechanisms  by  which  surface-exposed\nGPLs  and  PIMs  participate  in  the  phagocytosis  of  myco-\nbacteria deserve consideration. PIMs have been proposed\nto  participate  in  the  receptor-mediated  uptake  of  myco-\nbacteria (26, 41). For instance, they have been  shown to\nact  as  an  adhesin,  allowing  the  binding  of  M.  tuberculosis\nand  M.  smegmatis  to  hamster  ovary  ﬁbroblasts,  either  di-\nrectly or after opsonization with mannose binding protein\n(41). Here, we demonstrate that coating latex beads with\nPIM2  favors  their  internalization  through  a  CR3-depen-\ndent process, probably through the lectin binding site of\nCR3, which might recognize the mannosyl moiety of PIM,\nas mentioned above. Like other mannosylated conjugates,\nPIMs have been suspected to be ligands of the MR. We ad-\ndressed  this  question  and  show,  for  the  ﬁrst  time,  that\nPIMs, at least PIM2, do not in fact interact with this recep-\ntor in our cell model. As mentioned above, the length of\nthe  oligomannosyl  chain  was  not  critical  for  PIM  inhibi-\ntion of phagocytosis, suggesting that PIM6 was not a part-\nner of MR. The mode of action of GPL as a putative ligand\nof  macrophages  has  been  poorly  investigated.  We  found\nthat  GPL  III  ensures  the  entry  of  particles  through  both\nCR3 and MR. Although all types of GPLs share the same\npeptidolipid core, GPL I, which is inactive, lacks the succi-\nnyl  substituent  that  occurs  in  GPL  III  and  is  3,4-O-acety-\nlated on the 6-deoxytalosyl moiety (20). At present, these\n**BLOCK**fs== 9.8**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nmoieties have not been proposed among those described\nto be recognized by MR and CR3, and their putative bind-\ning  to  these  receptors  need  to  be  further  investigated.\nNonetheless,  we  propose  that  surface-exposed  PIMs  and\nGPLs coated on latex beads are potential ligands of CR3\nand/or MR, thereby mimicking part of the entry process\nof  mycobacteria  in  macrophages.  When  used  as  soluble\nmolecules,  PIMs  (present  study)  as  well  as  GPLs  (20)  af-\nfect the internalization of mycobacteria but not that of zy-\nmosan, whereas the two processes are well known to involve\nboth MR and CR3. Concerning CR3, the fact that zymosan\nand mycobacteria use distinct nonopsonic binding sites of\nthe receptor (19) suggests that the site involved in zymo-\nsan internalization does not recognize PIMs and GPLs.\n**BLOCK**fs== 9.8**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nIn conclusion, we propose that surface-exposed GPLs and\nPIMs interact with the nonopsonic receptors CR3 and/or\nMR involved in the phagocytosis of mycobacteria. Charac-\nterization of their binding properties is beyond the scope\nof this report, but it would clearly help to further investi-\ngate  their  signaling  pathways.  Understanding  the  myco-\nbacterial  strategy  to  infect  macrophages  without  trigger-\ning  bactericidal  activity  is  still  a  challenge.  Finally,  PIMs\nand GPLs, as inhibitors of mycobacteria phagocytosis, may\nalso  help  to  design  new  pharmacological  drugs  for  the\ncontrol of macrophage infection.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe authors are grateful to Drs. G. Puzo and C. Guilhot [Insti-\ntut de Pharmacologie et Biologie Structurale (IPBS), Toulouse,\nFrance)]  for  critical  reading  of  the  manuscript  and  to  Drs.  F.\nLaval  and  H.  Montrozier  (IPBS)  for  technical  assistance  and\nhelpful discussions. This work was supported in part by Euro-\npean Community Grant QLK2CT 19990193.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n1. Ernst, J. D. 1998. Macrophage receptors for Mycobacterium tuberculo-\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nsis. Infect. Immun. 66: 1277–1281.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n2. Bermudez,  L.  E.,  and  F.  J.  Sangari.  2001.  Cellular  and  molecular\nmechanisms  of  internalization  of  mycobacteria  by  host  cells.  Mi-\ncrobes Infect. 3: 37–42.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n3. Peterson, P. K., G. Gekker, S. Hu, W. S. Sheng, W. R. Anderson,\nR. J. Ulevitch, P. S. Tobias, K. V. Gustafson, T. W. Molitor, and C. C.\nChao. 1995. CD14 receptor-mediated uptake of nonopsonized My-\ncobacterium tuberculosis by human microglia. Infect. Immun. 63: 1598–\n1602.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n4. Khanna,  K.  V.,  C.  S.  Choi,  G.  Gekker,  P.  K.  Peterson,  and  T.  W.\nMolitor.  1996.  Differential  infection  of  porcine  alveolar  macro-\nphage  subpopulations  by  nonopsonized  Mycobacterium  bovis  in-\nvolves CD14 receptors. J. Leukoc. Biol. 60: 214–220.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n5. Zimmerli, S., S. Edwards, and J. D. Ernst. 1996. Selective receptor\nblockade  during  phagocytosis  does  not  alter  the  survival  and\ngrowth of Mycobacterium tuberculosis in human macrophages. Am. J.\nRespir. Cell Mol. Biol. 15: 760–770.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n6. Bermudez,  L.  E.,  A.  Parker,  and  J.  R.  Goodman.  1997.  Growth\nwithin macrophages increases the efﬁciency of Mycobacterium avium\nin  invading  other  macrophages  by  a  complement  receptor-inde-\npendent pathway. Infect. Immun. 65: 1916–1925.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n7. Roecklein, J. A., R. P. Swartz, and H. Yeager, Jr. 1992. Nonopsonic\nuptake of Mycobacterium avium complex by human monocytes and\nalveolar macrophages. J. Lab. Clin. Med. 119: 772–781.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n8. Schlesinger,  L.  S.  1993.  Macrophage  phagocytosis  of  virulent  but\nnot  attenuated  strains  of  Mycobacterium  tuberculosis  is  mediated  by\nmannose receptors in addition to complement receptors. J. Immu-\nnol. 150: 2920–2930.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\n9. Bermudez, L. E., L. S. Young, and H. Enkel. 1991. Interaction of\nMycobacterium  avium  complex  with  human  macrophages:  roles  of\nmembrane receptors and serum proteins. Infect. Immun. 59: 1697–\n1702.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n10. Stokes, R. W., I. D. Haidl, W. A. Jefferies, and D. P. Speert. 1993.\nMycobacteria-macrophage  interactions.  Macrophage  phenotype\ndetermines the nonopsonic binding of Mycobacterium tuberculosis to\nmurine macrophages. J. Immunol. 151: 7067–7076.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n11. Astarie-Dequeker,  C.,  E.  N.  N’Diaye,  V.  Le  Cabec,  M.  G.  Rittig,  J.\nPrandi, and I. Maridonneau-Parini. 1999. The mannose receptor\nmediates uptake of pathogenic and nonpathogenic mycobacteria\nand bypasses bactericidal responses in human macrophages. Infect.\nImmun. 67: 469–477.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n12. Pontow, S. E., V. Kery, and P. D. Stahl. 1992. Mannose receptor. Int.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nRev. Cytol. 137B: 221–244.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n13. Largent,  B.  L.,  K.  M.  Walton,  C.  A.  Hoppe,  Y.  C.  Lee,  and  R.  L.\nSchnaar.  1984.  Carbohydrate-speciﬁc  adhesion  of  alveolar  mac-\nrophages to mannose-derivatized surfaces. J. Biol. Chem. 259: 1764–\n1769.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n14. Stahl, P., P. H. Schlesinger, E. Sigardson, J. S. Rodman, and Y. C.\nLee. 1980. Receptor-mediated pinocytosis of mannose glycoconju-\ngates by macrophages: characterization and evidence for receptor\nrecycling. Cell. 19: 207–215.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n15. Ross, G. D., and V. Vetvicka. 1993. CR3 (CD11b, CD18): a phago-\ncyte and NK cell membrane receptor with multiple ligand speciﬁc-\nities and functions. Clin. Exp. Immunol. 92: 181–184.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n16. Sengelov,  H.  1995.  Complement  receptors  in  neutrophils.  Crit.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nRev. Immunol. 15: 107–131.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n17. Thornton, B. P., V. Vetvicka, M. Pitman, R. C. Goldman, and G. D.\nRoss. 1996. Analysis of the sugar speciﬁcity and molecular location\nof the beta-glucan-binding lectin site of complement receptor type\n3 (CD11b/CD18). J. Immunol. 156: 1235–1246.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n18. Ross, G. D., J. A. Cain, and P. J. Lachmann. 1985. Membrane com-\nplement receptor type three (CR3) has lectin-like properties anal-\nogous to bovine conglutinin as functions as a receptor for zymosan\nand rabbit erythrocytes as well as a receptor for iC3b. J. Immunol.\n134: 3307–3315.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n19. Le  Cabec,  V.,  C.  Cols,  and  I.  Maridonneau-Parini.  2000.  Nonop-\nsonic phagocytosis of zymosan and  Mycobacterium kansasii by CR3\n(CD11b/CD18) involves distinct molecular determinants and is or\nis not coupled with NADPH oxidase activation. Infect. Immun. 68:\n4736–4745.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n20. Villeneuve, C., G. Etienne, V. Abadie, H. Montrozier, C. Bordier, F.\nLaval, M. Daffe, I. Maridonneau-Parini, and C. Astarie-Dequeker.\n2003. Surface-exposed glycopeptidolipids of Mycobacterium smegma-\ntis speciﬁcally inhibit the phagocytosis of mycobacteria by human\nmacrophages.  Identiﬁcation  of  a  novel  family  of  glycopeptidolip-\nids. J. Biol. Chem. 278: 51291–51300.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n21. Etienne, G., C. Villeneuve, H. Billman-Jacobe, C. Astarie-Dequeker,\nM. A. Dupont, and M. Daffe. 2002. The impact of the absence of\nglycopeptidolipids on the ultrastructure, cell surface and cell wall\nproperties, and phagocytosis of Mycobacterium smegmatis. Microbiol-\nogy. 148: 3089–3100.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n22. N’Diaye,  E.  N.,  X.  Darzacq,  C.  Astarie-Dequeker,  M.  Daffe,  J.\nCalafat,  and  I.  Maridonneau-Parini.  1998.  Fusion  of  azurophil\ngranules  with  phagosomes  and  activation  of  the  tyrosine  kinase\nHck are speciﬁcally inhibited during phagocytosis of mycobacteria\nby human neutrophils. J. Immunol. 161: 4983–4991.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n23. Ortalo-Magne, A., A. Lemassu, M. A. Laneelle, F. Bardou, G. Silve,\nP. Gounon, G. Marchal, and M. Daffe. 1996. Identiﬁcation of the\nsurface-exposed lipids on the cell envelopes of Mycobacterium tuber-\nculosis and other mycobacterial species. J. Bacteriol. 178: 456–461.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n24. Gilleron,  M.,  C.  Ronet,  M.  Mempel,  B.  Monsarrat,  G.  Gachelin,\nand  G.  Puzo.  2001.  Acylation  state  of  the  phosphatidylinositol\nmannosides from Mycobacterium bovis bacillus Calmette Guerin and\nability  to  induce  granuloma  and  recruit  natural  killer  T  cells.  J.\nBiol. Chem. 276: 34896–34904.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n25. Laval,  F.,  M.  A.  Laneelle,  C.  Deon,  B.  Monsarrat,  and  M.  Daffe.\n2001. Accurate molecular mass determination of mycolic acids by\nMALDI-TOF mass spectrometry. Anal. Chem. 73: 4537–4544.\n26. Schlesinger, L. S., S. R. Hull, and T. M. Kaufman. 1994. Binding of\nthe  terminal  mannosyl  units  of  lipoarabinomannan  from  a  viru-\nlent strain of Mycobacterium tuberculosis to human macrophages. J.\nImmunol. 152: 4070–4079.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n27. Gilleron, M., V. F. Quesniaux, and G. Puzo. 2003. Acylation state of\nthe phosphatidylinositol hexamannosides from Mycobacterium bovis\nbacillus Calmette Guerin and Mycobacterium tuberculosis H37Rv and\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nits  implication  in  Toll-like  receptor  response.  J.  Biol.  Chem.  278:\n29880–29889.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n28. Khoo, K. H., A. Dell, H. R. Morris, P. J. Brennan, and D. Chatter-\njee.  1995.  Structural  deﬁnition  of  acylated  phosphatidylinositol\nmannosides  from  Mycobacterium  tuberculosis:  deﬁnition  of  a  com-\nmon anchor for lipomannan and lipoarabinomannan. Glycobiology.\n5: 117–127.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n29. Cougoule, C., P. Constant, G. Etienne, M. Daffe, and I. Maridon-\nneau-Parini. 2002. Lack of fusion of azurophil granules with pha-\ngosomes during phagocytosis of Mycobacterium smegmatis by human\nneutrophils is not actively controlled by the bacterium. Infect. Im-\nmun. 70: 1591–1598.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n30. Cywes, C., N. L. Godenir, H. C. Hoppe, R. R. Scholle, L. M. Steyn,\nR. E. Kirsch, and M. R. Ehlers. 1996. Nonopsonic binding of Myco-\nbacterium  tuberculosis  to  human  complement  receptor  type  3  ex-\npressed  in  Chinese  hamster  ovary  cells.  Infect.  Immun.  64:  5373–\n5383.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n31. Peyron, P., C. Bordier, E. N. N’Diaye, and I. Maridonneau-Parini.\n2000.  Nonopsonic  phagocytosis  of  Mycobacterium  kansasii  by  hu-\nman neutrophils depends on cholesterol and is mediated by CR3\nassociated  with  glycosylphosphatidylinositol-anchored  proteins.  J.\nImmunol. 165: 5186–5191.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n32. Prigozy, T. I., P. A. Sieling, D. Clemens, P. L. Stewart, S. M. Behar,\nS.  A.  Porcelli,  M.  B.  Brenner,  R.  L.  Modlin,  and  M.  Kronenberg.\n1997. The mannose receptor delivers lipoglycan antigens to endo-\nsomes for presentation to T cells by CD1b molecules. Immunity. 6:\n187–197.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n33. Milone, M. C., and P. Fitzgerald-Bocarsly. 1998. The mannose re-\nceptor mediates induction of IFN-alpha in peripheral blood den-\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.1**\ndritic  cells  by  enveloped  RNA  and  DNA  viruses.  J.  Immunol.  161:\n2391–2399.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n34. Stokes,  R.  W.,  R.  Norris-Jones,  D.  E.  Brooks,  T.  J.  Beveridge,  D.\nDoxsee, and L. M. Thorson. 2004. The glycan-rich outer layer of\nthe cell wall of Mycobacterium tuberculosis acts as an antiphagocytic\ncapsule  limiting  the  association  of  the  bacterium  with  macro-\nphages. Infect. Immun. 72: 5676–5686.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n35. Vergne, I. I., and M. Daffe. 1998. Interaction of mycobacterial gly-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\ncolipids with host cells. Front. Biosci. 3: 865–876.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n36. Grado, C., Ballou, C. E. 1960. Myo-inositol phosphates from beef\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nbrain phosphoinositide. J. Biol. Chem. 235: PC23–24.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n37. Ortalo-Magne, A., A. B. Andersen, and M. Daffe. 1996. The outer-\nmost capsular arabinomannans and other mannoconjugates of vir-\nulent and avirulent tubercle bacilli. Microbiology. 142: 927–935.\n38. Lemassu, A., A. Ortalo-Magne, F. Bardou, G. Silve, M. A. Laneelle,\nand M. Daffe. 1996. Extracellular and surface-exposed polysaccha-\nrides  of  non-tuberculous  mycobacteria.  Microbiology.  142:  1513–\n1520.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n39. Stokes, R. W., and D. P. Speert. 1995. Lipoarabinomannan inhibits\nnonopsonic  binding  of  Mycobacterium  tuberculosis  to  murine  mac-\nrophages. J. Immunol. 155: 1361–1369.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n40. Thorson,  L.  M.,  D.  Doxsee,  M.  G.  Scott,  P.  Wheeler,  and  R.  W.\nStokes. 2001. Effect of mycobacterial phospholipids on interaction\nof  Mycobacterium  tuberculosis  with  macrophages.  Infect.  Immun.  69:\n2172–2179.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n41. Hoppe, H. C., B. J. de Wet, C. Cywes, M. Daffe, and M. R. Ehlers.\n1997. Identiﬁcation of phosphatidylinositol mannoside as a myco-\nbacterial  adhesin  mediating  both  direct  and  opsonic  binding  to\nnonphagocytic mammalian cells. Infect. Immun. 65: 3896–3905.",
         "extracted",
         "",
         "Mycobacteria use their surface-exposed glycolipids to infect human macrophages through a receptor-dependent process Published, JLR Papers in Press, December 1, 2004. DOI 10.1194/jlr.M400308-JLR200",
         "None"
        ],
        [
         "24",
         "0020220e317cd7c4ef87a8296677701d4e6a7c42",
         "Few researchers would dispute that the pandemic of obesity is caused by a profound mismatch between humanity's present environmental circumstances and those that have moulded evolutionary selection. This concept was first articulated when gestational diabetes was described as being the result of a ‘thrifty genotype rendered detrimental by progress’. More recently, this hypothesis has been extended to the concept of a ‘thrifty phenotype’ to describe the metabolic adaptations adopted as a survival strategy by a malnourished fetus; changes that may also be inappropriate to deal with a later life of affluence. Both the thrifty genotype and the thrifty phenotype hypotheses would predict that populations in some areas of the developing world would be at greater risk of obesity and its co-morbidities; a proposition to be explored in the present paper. To date thrifty genes remain little more than a nebulous concept propagated by the intuitive logic that man has been selected to survive episodic famine and seasonal hungry periods. Under such conditions those individuals who could lay down extra energy stores and use them most efficiently would have a survival advantage. The search for candidate thrifty genes needs to cover every aspect of human energy balance from food-seeking behaviour to the coupling efficiency of oxidative phosphorylation. The present paper will describe examples of attempts to find thrifty genes in three selected candidate areas: maternally-transmitted mitochondrial genes; the uncoupling proteins; apoE4, whose geographical distribution has been linked to a possible thrifty role in lipoprotein and cholesterol metabolism.",
         "A. Prentice,P. Rayco-Solon,S. Moore",
         "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D4334A1A0F9096B08FAE923BAB0CA36D/S0029665105000194a.pdf/div-class-title-insights-from-the-developing-world-thrifty-genotypes-and-thrifty-phenotypes-div.pdf",
         "\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nProceedings of the Nutrition Society (2005), 64, 153–161\ng The Authors 2005\n**BLOCK**fs== 15.9**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nInsights from the developing world: thrifty genotypes and\nthrifty phenotypes\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.0**\nAndrew M. Prentice, Pura Rayco-Solon and Sophie E. Moore\nMRC International Nutrition Group, London School of Hygiene and Tropical Medicine, Keppel Street, London, WC1E 7HT,\nUK and MRC Keneba, Keneba, The Gambia\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.3**r== 0.2**\nFew researchers would dispute that the pandemic of obesity is caused by a profound mismatch\nbetween humanity’s present environmental circumstances and those that have moulded\nevolutionary selection. This concept was ﬁrst articulated when gestational diabetes was\ndescribed as being the result of a ‘thrifty genotype rendered detrimental by progress’. More\nrecently, this hypothesis has been extended to the concept of a ‘thrifty phenotype’ to describe\nthe metabolic adaptations adopted as a survival strategy by a malnourished fetus; changes that\nmay also be inappropriate to deal with a later life of afﬂuence. Both the thrifty genotype and\nthat populations in some areas of the\nthe thrifty phenotype hypotheses would predict\ndeveloping world would be at greater risk of obesity and its co-morbidities; a proposition to be\nexplored in the present paper. To date thrifty genes remain little more than a nebulous concept\npropagated by the intuitive logic that man has been selected to survive episodic famine and\nseasonal hungry periods. Under such conditions those individuals who could lay down extra\nenergy stores and use them most efﬁciently would have a survival advantage. The search for\ncandidate thrifty genes needs to cover every aspect of human energy balance from food-seeking\nbehaviour to the coupling efﬁciency of oxidative phosphorylation. The present paper will\ndescribe examples of attempts to ﬁnd thrifty genes in three selected candidate areas:\nthe uncoupling proteins; apoE4, whose geo-\nmaternally-transmitted mitochondrial genes;\ngraphical distribution has been linked to a possible thrifty role in lipoprotein and cholesterol\nmetabolism.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\nObesity: Thrifty genes: Mitochondrial genes: Uncoupling protein: ApoE\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nThe obesity epidemics of many afﬂuent nations have\nrecently started to coalesce and to engulf many less-\nafﬂuent countries. Quite remarkably, it is even penetrating\nthe world’s poorest countries in their urban areas (van der\nSande et al. 2001), as well as the rural areas of countries a\nlittle more advanced along the epidemiological transition.\nThe very high levels of obesity in some of the least-\nprivileged peoples, in whom circumstance has encouraged\na rapid acculturation of diets and lifestyle (e.g. the Pima\nIndians of Arizona and Nauruan Polynesians), have\ngenerated seminal studies of obesity and diabetes over\nrecent decades, and have illustrated the investigative power\nof studying rapid cultural and behavioural\ntransitions\n(Bindon & Baker, 1997; Zimmet et al. 1997). The purpose\nof the present review is to try to extend these speciﬁc\nexamples to a more general interpretation of lessons that\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\nmight be learnt by considering the early development of\nthe obesity trend within populations assumed to have been\nadapted to frugal environments until the very recent past.\nSpecial attention will be given to the twin concepts of the\n‘thrifty phenotype’ and ‘thrifty genotype’.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nThrifty phenotypes\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.0**\nThe thrifty phenotype concept, of an adaptively growth-\nrestricted organism, has existed for many decades but has\nbeen most clearly articulated by Hales & Barker (1992,\n2001), who also coined the term ‘thrifty phenotype’. The\nconcept has been described in some detail in another paper\nin this symposium (Stocker et al. 2005), so current\ncomments will be restricted to a few speciﬁc observations\nin relation to the developing country setting.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nAbbreviations: INS-VNTR, insulin variable number of tandem repeats; UCP, uncoupling protein.\nCorresponding author: Professor A. M. Prentice, fax +44 20 7958 8111, email Andrew.prentice@lshtm.ac.uk\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.1**r== 0.8**\nIt\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.2**r== 0.6**\nis in such countries that\n**BLOCK**fs== 10.0**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\nthe majority of growth-\nrestricted (or small-for-gestational-age) babies are born.\nThe currently-accepted thesis is that\nthis early growth\nrestriction will represent a successful adaptation (although\nso long as the nutritional\nnot being without costs)\ncircumstances in later life continue to match those that\nhave induced the initial adaptation. Hence, it has been\nshown in The Gambia that small babies, including those\nborn during a nutritionally-debilitating hungry season,\nmaintain excellent cardiovascular health into adulthood,\nwith a complete absence of the metabolic syndrome, so\nlong as they retain their ‘lean, ﬁt and frugal’ lifestyle in the\nrural areas (Moore et al. 2001). In practice, however, this\nlifestyle is frequently not retained and many individuals\nwho have experienced fetal and early-life nutritional\nhardships transfer to the urban areas and to a diet of\nrelative energy excess with high intakes of fats and reﬁned\ncarbohydrates (Popkin, 2001, 2002; Adair & Prentice,\n2004). It is this rapid transition within a single lifespan that\nis implicated by many researchers as a crucial causative\nagent in the explosion of non-communicable diseases in\nmany such urban areas.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nFetal growth restriction in developing countries has\nmultifactorial origins, but attempts to quantify the con-\ntribution attributable to each cause indicate that under-\nnutrition is likely to be the most important single factor\n(Kramer, 1987, 2003). Both acute and chronic exposures\ncan be critical, although it seems likely that acute energy\nor nutrient insufﬁciency has to be quite harsh before it has\nreproducibly demonstrable effects on the baby (Susser,\n1991). Chronic food shortages have their effect by creating\na small mother, which imposes ‘maternal uterine con-\nstraint’ on the fetus probably mediated through a complex\ninterplay between insulin-like growth factors and their\nbinding proteins (Kelsey et al. 1999). These constraints\nthe fetus does not outgrow the\nattempt\nmother’s capacity to carry it safely to term and to deliver it\nnaturally. Such growth-restricted babies may be ‘small but\nperfectly formed’, and it\nto assume that\nthey necessarily carry the same perinatal risks as a low-\nbirth-weight baby of a much larger mother (Wilcox, 2001;\nMelve & Skjaerven, 2003). They may be thought of as\nhaving a harmonious growth well adapted for their uterine\nenvironment. Since the uterine environment\nis itself a\nreﬂection of the historic dietary conditions of the family,\nthe baby can be viewed as well adapted to its recent\nhistory, and hence to its best available prediction of likely\nfuture circumstances.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nto ensure that\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nis incorrect\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nProblems arise when this harmony of growth is\ndisrupted. The best documented evidence of this dishar-\nmony comes from collaborative studies between Ranjan\nYajnik and colleagues in Pune, India and Caroline Fall and\ncolleagues in Southampton, UK (Bavdekar et al. 1999;\nYajnik, 2000; Yajnik et al. 2002a,b). They have demon-\nstrated a phenomenon that Yajnik (Yajnik et al. 2002a,b)\ndescribes as the ‘thin–fat’ baby syndrome (Prentice, 2003).\nThe Indian mothers are small but carry a disproportion-\nately high level of abdominal fat. Comparison between\nthe Indian and Southampton babies reﬂects a similar\npicture, with the Indian babies being much smaller than\ntheir UK counterparts in all dimensions except triceps and\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nespecially subscapular skinfolds, suggesting that they are\nrelatively fat. Raised glucose and insulin levels are already\npresent in the cord blood of the Indian babies (Yajnik et al.\n2002b) and studies later in childhood also reveal early\nevidence of insulin resistance (Bavdekar et al. 1999;\nYajnik, 2000). There are several possible interpretations of\nthese data, including the possibility that the relative fatness\nof\nIndian mothers has induced an altered\n‘disharmonious’ pattern of tissue deposition in the fetus\nthat already carries detectable metabolic penalties.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nthe small\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nin view of\n**BLOCK**fs== 10.0**b== 0.4**t== 0.2**l== 0.6**r== 0.0**\nPostnatal inﬂuences have so far attracted less interest in\nthe developmental origins of health and adult disease ﬁeld\nthan prenatal inﬂuences. However, they are now beginning\nthe\nto receive much greater attention,\nit may be the accelerated catch-up\ndemonstrations that\ngrowth of some small babies that\nis most strongly\ncorrelated with later ill-health outcomes (Lucas et al.\n1999; Forse´n et al. 2000).\nIn the developing world\ndeviations in postnatal growth all\ntend to be in the\ndownwards direction and, by analogy with the speculations\nabout impaired fetal growth, might also create a disadapted\nphenotype if such children grow up into conditions of\nnutrient surfeit. Using data from rural Gambians it has\nagain been demonstrated that there is an absence of any\nimportant risk factors for CVD or diabetes in adults who\nwere malnourished as children but still live their traditional\nlifestyle (Moore et al. 2001). This ﬁnding\nfrugal rural\nholds true even in the lowest quartile of childhood\nnutritional status, for which the mean weight-for-age Z-\nscore is worse than –3.3. Such ﬁndings do not necessarily\nchallenge the developmental origins of health and adult\ndisease thesis, but do emphasise the need for datasets in\nwhich early malnutrition has been followed by later\nafﬂuence as part of a rapid demographic transition. Until\nsuch data are obtained, much of the current prediction that\nthe\nrapidly-developing nations will bear\ndevelopmental origins of health and adult disease thesis\nremains speculative.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\nthe brunt of\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nThrifty genotypes\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.6**r== 0.0**\nNeel (1962) at the University of Minnesota (Minneapolis,\nMN, USA) coined the term ‘thrifty genotype’ in a paper\nentitled: Diabetes mellitus: a ‘thrifty’ genotype rendered\ndetrimental by ‘progress’. This phrase has become one of\nthose that are sufﬁciently catchy and self-explanatory to have\nentered the lexicon and achieved wide currency in both lay\nand scientiﬁc writing. In doing so it has gathered a cloak of\nmisuse that hampers constructive discussion of its possible\nrelevance in understanding the current susceptibility to\nobesity. An attempt will be made to deconstruct some of\nthis casual usage in order to try and deﬁne it better as a\ntheoretical construct. In attempting to do so it is recognised\nthat it may be deprived of one of its major attributes, i.e. a\nﬂexibility that has been part of its usefulness.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.1**\nGeneralisability of the thrifty genotype concept\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.0**\nThe disciples of the original thesis of Neel (1962) have\nit could explain the very high\nfrequently argued that\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.4**r== 0.6**\nIt\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.4**r== 0.5**\nis argued that\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nprevalence of obesity and diabetes within some isolated\npopulations, particularly on Polynesian islands (West,\n1974; Joffe & Zimmet, 1998).\nthe\nhighly-susceptible islanders are the descendants of a few\nfounders who had been selected as the survivors of a long\nsea crossing in which their less-thrifty shipmates died of\nhunger. Such a view has been widely held by Western\nscientists, but refuted as ethnographically inaccurate by\nPolynesian historians who claim that their forebears were\nmasters of the sea and able to travel\nindeﬁnitely by\ncatching rainwater and eating ﬁsh and turtles (Prentice,\n2001).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nThe issue can be taken a step further and it can be\nargued that, irrespective of the validity or otherwise of the\nfounder-effect\ntheory in populations with very high\nsusceptibility to diabetes, the metabolic legacy of man-\nkind’s struggles against starvation has imparted a much\nwider inﬂuence on the human genome.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nParadoxically,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nit was the dawn of agriculture that\nheralded episodes of mass starvation (Diamond, 1993;\nPrentice, 2001). It is believed that before this time the\nhunter–gatherer forbears of man avoided starvation by\nexploiting a highly-varied diet, living in small sustainable\ngroups and being prepared to move to new hunting\ngrounds when the need arose. They undoubtedly would\nhave experienced hunger between successful kills, but\nwould have very rarely been exposed to prolonged\nstarvation. Agriculture, on the other hand, fuelled popula-\ntion growth and diminished dietary diversity (McCance,\n1975; Fagan, 2000). This situation is sustainable as long as\nclimatic conditions remain stable (and in the absence of\nwarfare) but is highly vulnerable to even a single year’s\npoor rain, a single night of unseasonable frost or a crop\npest or infection such as potato blight.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nA selection of the mass of historical evidence that\nrecords the inﬂuence of widespread catastrophic famine on\nthe human race has been documented in greater detail\nelsewhere (Prentice, 2001). These records document\nextreme privation, with frequent references to cannibalism\n(especially of children) as a means of survival. Some of the\nearliest and best-known references are the following:\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n‘I am mourning on my high throne for the vast\nmisfortune, because the Nile ﬂood in my time has not\ncome for seven years! Light is the grain; there is a lack\nof crops and of all kinds of food. Each man has become\na thief\nto his neighbour. They desire to hasten and\ncannot walk. . . The counsel of the great ones in the\ncourt\nis but emptiness. Torn open are the chests of\nprovisions, but instead of contents there is air. Every-\nthing is exhausted’ The Stella of Famine; chiselled into\nEgyptian rock in the time of Tcheser about 2000BC\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n‘All of Upper Egypt was dying of hunger, to such a\ndegree that everyone had come to eating his children\n. . . the entire country had become like a starved grass-\nhopper. . .’ The Sepulchers of Ankhtiﬁ 2180–60BC\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n‘If there be a cutting down of the food offerings of the\ngods,\nthen a million men perish among mortals,\ncovetousness is practiced, the entire land is in a fury,\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nand great and small are on the execution block’ Middle\nKingdom Hymn to the Nile\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\n‘And there was famine in the land: And Abram went\ndown into Egypt to sojourn there: For the famine was\ngrievous in all the land’ Abram’s journey into Egypt;\nGenesis 12\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\n‘. . .in the city we are exchanging our children and\neating them. . .’ China 594BC, The Siege of the Chong\nCapital\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.0**\n‘One could scarcely see the water in the Vistasa,\nentirely covered as the river was with corpses soaked\nand swollen by the water in which they had long been\nlying. The land became densely covered with bones in\nall directions until it was like one great burial ground,\ncausing terror to all beings’ Kashmir in the years\n917–8AD as related in Kalhana’s ‘Rajatarangini’\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\n‘Parents killed their children and children killed\nparents, and the bodies of executed criminals were\neagerly snatched from the gallows’ Contemporary\nreport\nthe Great\nFamine 1319\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\nfrom Flanders at\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nthe beginning of\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.0**\n‘The region has been visited by the exterminating angel.\nEvery scourge has been unloosed. Everywhere I have\nfound men dead of cold and hunger’ Mirabeau’s report\nfrom a visit to Provence immediately before the French\nRevolution\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nindividual\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nsummaries of\n**BLOCK**fs== 10.0**b== 0.2**t== 0.5**l== 0.6**r== 0.0**\nFor greater detail about the effects of starvation the reader\nis referred to Prentice (2001) or to the two-volume work\nThe Biology of Human Starvation (Keys et al. 1950).\nExcellent\nfamines are also\navailable (for example, see Jordan, 1996). Only the salient\npoints will be considered here, namely: (a) there is wide-\nspread evidence that starvation and famines must have\nexerted a strong selection effect on the human genome.\nThis effect on the genome would have occurred through\nsuppression of fertility as well as actual mortality; (b) these\nfamines should not be viewed as something happening\nonly in far-off places or in far-off times. Keys et al. (1950)\nestimate that there have been >190 documented famines in\nBritain alone over the past 2000 years; (c) that whilst in\nmany respects the human genome has remained very stable\nover the evolutionarily-trivial time of 12 000 years, since\nthe dawn of agriculture, it is contended that the approxi-\nmately 600 generations during this time will have been\nsufﬁcient to permit considerable selection of ‘thrifty genes’,\nespecially if mediated through effects on reproduction (as\nwill be argued later). In summary, in concurrence with\nDarwin (1989) famine should be placed among the three\nmajor drivers of natural selection.\nIn the concluding\nremarks to The Origin of Species Darwin (1989; originally\npublished in 1859) notes that ‘The evolution of higher\nanimals directly follows . . . from the war of nature, from\nfamine and death . . .’.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.0**\nHaving established that there is ample evidence that\nnatural selection could have favoured the selection of\nthrifty genes, the next task is to deconstruct the concept in\nan attempt to deﬁne it more precisely. The ﬁrst stage in\nthis process is to address a logical corollary of ‘thrifty\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\ngenes’ that at ﬁrst sight seems surprising,\nto\nbelieve in thrifty genes is to accept that the natural state of\naffairs is unthrifty; why might this situation exist?\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.5**r== 0.5**\nthat\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.5**r== 0.5**\ni.e.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nBelief in ‘thrifty genes’ implies that the normal state\nis unthrifty\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nIt is unlikely that evolution has left man in a physiologi-\ncally-unthrifty state through simple carelessness. Instead, it\nis much more likely that there is a series of reasons for\nenergetic proﬂigacy, the advantages of which outweigh\ntheir energy costs. Some possible candidates are listed in\nTable 1; there are no doubt many more candidates.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nThe fructose-1,6-biphosphatase–phosphofructokinase\nshuttle is the most famous example of a so-called ‘futile\ncycle’ in which substrate is utilised without a net gain in\nATP. The favoured teleological explanation for\nthis\nit enhances metabolic\napparently ‘futile’ cycle is that\ncontrol enabling substrate to be rapidly provided, since one\nside of a rapidly-shuttling cycle can be shut down (hence\ndiverting substrate) much more rapidly than the other side\ncould be started up. Similar lines of argument are used to\nexplain why the extrahepatic extra-adipocyte fatty acid\ncycle circulates much more substrate than is required\n(Wolfe et al. 1990), and to explain the high rates of muscle\nglutamine cycling (Parry-Billings & Newsholme, 1991;\nNewsholme, 1994).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nSubstrate usage for thermoregulatory thermogenesis is\nan obvious reason to maintain an unthrifty state in small\nand cold-adapted animals, but\nlimited\nimportance in man.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nis probably of\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe need to quench oxidant stress, especially in the\nmitochondria, may also require a proﬂigate use of reduced\nsubstrates. Brand and colleagues (Brand, 2000) have shown\nthat all mammalian organisms so far studied appear to\nhave a high rate of proton leakage across the mitochondrial\ninner membrane. In his ‘uncoupling to survive’ hypothesis\nBrand (2000) argues that this ‘inefﬁciency’ is necessary to\ncounteract\nthe high levels of reactive oxygen species\ngenerated during normal mitochondrial metabolism.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nAnother well-known reason that an organism might need\nto intentionally burn off energy is in order to concentrate\ndiets with inadequate levels of protein or micronutrients.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nTable 1. Possible evolutionary beneﬁts of energy or substrate\nproﬂigacy\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.6**\nEnhancement of metabolic control\nWhole-body fatty acid cycling\nGlutamine cycling and immunity\nFructose-1,6-biphosphatase–phosphofructokinase\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nThermoregulatory thermogenesis\nBrown adipose tissue (UCP1)\nUCP2 and UCP3?\nFructose-1,6-biphosphatase–phosphofructokinase\nShivering\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nQuenching oxidant stress\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nNeeds high levels of mitochondrial proton shuttling\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nBalancing nutrient-deﬁcient diets\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nExcess energy must be burnt off to ‘concentrate’ nutrients\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nUCP, uncoupling protein.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nThis possibility was demonstrated many decades ago by\nMcCance and colleagues (Widdowson & McCance, 1974),\nand more recently Dulloo (Dulloo & Jacquet, 2001) has\nre-presented some of Stock’s early data to suggest that the\nprotein level of the diet is also a key mediator of adaptive\nthermogenesis in man.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.0**\nBehavioural unthriftiness can also be added to the\nmetabolic unthriftiness listed in Table 1. The young of\nmost species expend high levels of energy in apparently-\nunnecessary play, but this activity almost certainly has a vital\nrole in the development of their motor and cognitive skills.\nIn summary, it is possible to construct many arguments\nto explain why the normal physiological state would utilise\nan extent of unthriftiness. Such traits will presumably have\nbeen selected for in natural selection’s pursuit of optimum\nperformance. Thus,\nthrifty genes will only have been\nselected if there is a more powerful selective force at play,\ni.e. famine.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\nThe multiple possible forms of thrift\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\ntraits,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nbroader\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nencapsulate\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.6**r== 0.0**\nOne of the problems of using the ‘thrifty gene’ concept is\nthat most individuals associate it with a metabolic trait\nrelated to a frugal utilisation of fuel, which indeed was\nNeel’s (1962) ﬁrst intention. However, the phrase could\nequally well\nincluding\nbehavioural ones that would favour survival during famine\nand hence could become selected in the genome. A\nselection of possibilities is listed in Table 2. It should be\nnoted that these examples could operate on the intake side\nof the energy balance equation, although such genes, for\nwhich there is considerable evidence (Snyder et al. 2004),\nmight best be termed ‘greedy genes’. Other researchers\nbelieve that genes operative on the expenditure side of the\nequation may operate through an effect on individuals’ pro-\npensity for physical activity (Chakravarthy & Booth, 2004).\nHaving emphasised this important point about the likely\npleiotropy of thrifty genes the remainder of the present\npaper will concentrate on the possible metabolic aspects.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.1**\nCharacteristics predicted for a thrifty gene\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.0**\nA good candidate as a thrifty gene should exhibit two\nimportant features. The ﬁrst is plasticity in response to\nenvironmental circumstances. A gene that is highly thrifty\nand always in its switched-on mode would presumably\nlead to gross obesity. While the extra fat stores would aid\nsurvival through famine they would, at extreme levels,\nimpair reproductive performance in both males and females.\nThe second feature that would be expected is evidence of\n‘signatures of selection’ such as striking geographical\ndistribution.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nTable 2. Examples of the forms ‘thriftiness’ can take\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.1**\nMetabolic: ‘energy-sparing’ super-efﬁcient metabolism\nAdipogenic: propensity to rapid fat gain\nPhysiological: ability to switch off non-essential processes\nGluttony: tendency to gorge when food is available\nSlothfulness: tendency to conserve energy through inactivity\nBehavioural: hoarding, meanness, theft, etc.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.2**r== 0.6**\nTable 3. Possible sites for metabolic thrift\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.6**\nSubstrate cycling\nMitochondrial proton leakage\nCoupled v. uncoupled oxidative phosphorylation\nNa+/K+ pumping\nCellular repair (e.g. protein turnover)\nReduction in non-essential processes (e.g. immunity)\nFuel channelling: muscle v. adipose tissue\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nPossible candidates for metabolic thrifty genes\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nTable 3 lists some of the possible sites and functions from\nwhich an organism might withhold energy when times are\nharsh. The tightening up of efﬁciency in these areas might\nbe only a temporary phenomenon (e.g.\nin limiting\nresources allocated to cognate cell-mediated immunity).\nIn making such savings the organism would essentially be\ntaking a gamble that it could weather the current famine\nand survive to continue to reproduce when the environ-\nmental conditions improved.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nWithin the areas listed in Table 3 several authors (for\nexample, see Auwerx, 1999; Corbo & Scacchi, 1999; Ong\n& Dunger, 2000; Kimm et al. 2002) have made claims for\nspeciﬁc ‘thrifty genes’. Some of these claims are mislead-\ning, and hence extremely unhelpful, since they apply the\nconcept in an imprecise manner. For instance, it has been\nstated that ‘. . .PPARg was shown to have a key role in\nadipogenesis and be a master controller of the ‘thrifty gene\nresponse’ leading to efﬁcient energy storage.’ (Auwerx,\n1999), but no evidence is provided to suggest that PPARg\nhas been selected for thrift or exhibits any of the deﬁning\ncharacteristics set out earlier. Hence, emphasis will be\ngiven to several candidates with more speciﬁc, albeit very\nearly, evidence that they may make good candidates for\nthrifty genes.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nGenes regulating fuel channelling\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nNeel’s (1962) original ideas focused on the possibility that\nalterations in the insulin signalling pathway could mod-\nulate fuel channelling between utilization and storage, and\nhence provide a likely mechanism for rapid fuel deposition\nin times of plenty. (This general idea also underlies the\nHales & Barker (1992, 2001) thrifty phenotype concept.)\nThe entirety of\nthe insulin signalling pathways are\ncomplex and contain many possible candidate genes,\nalthough to date none has been reliably pin-pointed as\nsuch.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nAn interesting candidate that is currently being investi-\ngating in our Gambian studies is the insulin variable\nnumber of\ntandem repeats (INS-VNTR) microsatellite\n(Bell et al. 1984). This gene lies on the promoter region\nof the insulin gene and is upstream of the gene encoding\ninsulin-like growth factor 2. In non-African populations\nthere are only three main lineages (classes I, II and III),\nbased on the number of\nrepeat motifs, and class I\npredominates (Stead et al. 2003). Associations have been\nreported with both type 1 and type 2 diabetes (Bell et al.\n1984; Bennett et al. 1995; Ong et al. 1999). In contrast,\nAfrican populations possess at least twenty-one different\nvariants, with class III predominating (Stead et al. 2003).\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nThe INS-VNTR class III variant has been associated\nwith higher birth weight (Dunger et al. 1998; Lindsay et al.\n2003), although recent reports have failed to replicate this\nﬁnding (Bennett et al. 2004; Mitchell et al. 2004), and\nfaster postnatal growth (Ong et al. 2004a). It will be\nargued that genes regulating fetal growth and survival may\nbe particularly central to lineage survival in the face of\nfamine, and that INS-VNTR may be a good candidate.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nMitochondrial genes\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\nOxidative phosphorylation within the mitochondrion does\nnot achieve 100 % conversion of the potential energy of\nfuel\ninto ATP, because of uncoupling of the electron-\ntransport chain and the leakage of electrons. The extent of\nmitochondrial coupling, and any genes that might affect it,\nprobably therefore represent a fertile hunting ground for\nthrifty genes. There is evidence for striking geographical\nvariation in some mitochondrial genes (Ruiz-Pesini et al.\n2004), although the current\nthe\nvariations have been driven by the need for unthriftiness\nin order to drive thermogenesis and allow the carriers to\ncolonise more northerly latitudes.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\ninterpretation is that\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.6**r== 0.0**\nIn a paper entitled: Effects of purifying and adaptive\nselection on regional variation in human mitochondrial\nDNA, Wallace and his team (Ruiz-Pesini et al. 2004) have\nused phylogenetic analysis of global human mitochondrial\nDNA to show that mitochondrial DNA lineages from\nhigher latitudes are most highly conserved. These varia-\ntions correlate with energy deﬁciency diseases and lon-\n‘Speciﬁc mtDNA\ngevity. The authors conclude that\nreplacement mutations permitted our ancestors to adapt to\nmore northern climates, and these same variants are\naffecting our health today’ (Ruiz-Pesini et al. 2004). It\nhas further been speculated that the development of these\nhigh-energy-requiring unthrifty variants may have driven\nthe appetite for, and development of, high-fat diets.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\nThis example is the best one so far of the sort of genetic\ndetective work that will be necessary to ultimately validate\nor refute Neel’s (1962) hypothesis. The fact that it has\napparently identiﬁed unthrifty genes rather than thrifty\ngenes does not detract from the principle that analogous\nthrifty variants may ultimately be identiﬁed, and that\nthe mitochondrion is a likely target, whether through\nmitochondrial or nuclear DNA variation.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\nThe uncoupling proteins 2 and 3\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.6**r== 0.0**\nThe original mitochondrial uncoupling protein from brown\nadipose tissue, uncoupling protein (UCP)1, is generally\nthought to be of limited importance in man beyond the\nvery early neonatal stage (Dalgaard & Pedersen, 2001;\nDulloo & Samec, 2001; Argiles et al. 2002). The more\nrecent discovery of its homologues UCP2 and UCP3,\nwhich are expressed more widely including in adipose\ntissue and muscle,\ninterest as\npossible ‘obesity genes’ (and by inference thrifty genes)\n(Dulloo & Samec, 2001). Subsequent research, however,\nhas yielded a confused picture, which is summarised\nbrieﬂy in Table 4. Certain observations of the phenotypic\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\ninitially created great\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nTable 4. The uncoupling proteins (UCP): a confusing picture at\npresent\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nUCP-knock-out models have unexpected phenotypes\nUCP2 and UCP3 mRNA up regulated by fasting\nPossible role in mitochondrial fatty acid shuttling (UCP3)\nPossibly reduce fatty acid accumulation (UCP3)\nPossibly enhance muscle fatty acid oxidative capacity (UCP3)\nProbably quench reactive oxygen species (UCP2)\nPossible immunological role (UCP2)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nbehaviour, for instance that UCP3 is up regulated not down\nthese UCP are not a\nregulated by fasting,\nimply that\nthe energy regulatory\nstraightforward component of\napparatus as had ﬁrst been anticipated (Dulloo & Samec,\n2001). Instead, they may have more esoteric roles such as\nthe quenching of oxidant stress. These latest insights may\nargue against their candidacy as thrifty genes.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nNotwithstanding this argument, there has been interest\nin the possibility of identifying thrifty traits within the\nUCP, but little has emerged. Suggestive evidence has been\nobtained that a UCP3 exon 5 variant is associated with\nresting energy expenditure in African-American\nlower\nwomen (but not white women) and hence may be a thrifty\nallele. This evidence was, however, based on a small\nsample size and, judging by the history of such genetic\nobservations,\nis unlikely to replicate in future studies\n(Kimm et al. 2002). Currently, there appear to be no other\nsimilar claims.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nApoE\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nThe global distribution of apoE4 (the variant associated\nwith raised cardiovascular\nrisk) shows a signature of\nselection of the type predicted for a good functional thrifty\ngene candidate (Table 5). ApoE2 is most common in the\nMediterranean basin where agriculture is thought to have\nbegun. ApoE4 (the ancestral lineage) is more common\nin pigmies,\nthe Khoi San, Aboriginals, some Native\nAmericans and Lapps, leading Corbo and colleagues (Corbo\n& Scacchi, 1999) to speculate that it could represent a\nmore lipid-thrifty variant\nin\nsequestering cholesterol and has been better maintained\nin populations with a tenuous dietary supply.\n(This\nargument runs counter to the thesis that it is agricultural\nsocieties that are more susceptible to episodic famine; see\nalso the ‘famine’ quotes concerning the Mediterranean\nbasin listed earlier (see p. 155).)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nis more efﬁcient\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nthat\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nCorbo (Corbo et al. 2004a,b) argues that the selection\neffect may be mediated through the fact that the apoE4\nIn\nvariant\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\nfavours steroidogenesis and hence fertility.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nsupport of this argument Corbo et al. (2004a) have shown\nin a random sample of post-menopausal women that car-\nriers of apoE4 had more children than women with apoE2.\nThis conjecture in relation to reproductive efﬁciency\nleads to the ﬁnal observation to be offered in relation to the\nsearch for thrifty genes; that it is highly likely that the best\ncandidates will be associated in some way with reproduc-\ntive efﬁciency (see p. 159). Before addressing this topic it\nis pertinent to comment on a recent hypothesis proposed to\nexplain the differential diabetes susceptibility rates in\nEuropean and non-European peoples.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nThe double puzzle of diabetes\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.6**r== 0.0**\nIn a recent essay Diamond (2003) has addressed the puzzle\nof why Europeans seem to have lower susceptibility to\ntype 2 diabetes than other\nraces, after allowing for\ndifferences in BMI. Some of the generic data cited are\nsummarised in Table 6. Diamond (2003) downplays the\nlikely contribution of thrifty genes or thrifty phenotypes in\nthe non-Europeans and instead suggests that peoples of\nEuropean origin must have passed through previous times\nin which epidemics of diabetes have selected resistant\nindividuals. This explanation would seem to be highly\nunlikely on the grounds that it is known that an epidemic\nof diabetes would have to be driven by an epidemic of\nobesity (and there are no previous records of such), and\nthat in any case diabetes generally kills after the time of\npeak reproductive performance.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.1**\nFamines and reproduction\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.6**r== 0.0**\nSevere and extended energy depletion shuts down the\nreproductive axis in both genders of many (probably most)\nspecies, including man. The adults concentrate all avail-\nin the hope that\nable resources on their own survival\nconditions will improve and allow them to recommence\nactive reproduction in the future. Consequently, conception\nand birth rates fall precipitately during extreme famine,\nand these variations have been closely documented, using\nhospital records, in the Dutch Hunger Winter (Stein et al.\n1975) and some of\nfamines (Sen, 1981;\nthe Bengal\nMaharatna, 1996). The fact that this effect is speciﬁc to\nfamine, rather than a side-effect of the general horror of\nthe times, has been elegantly illustrated by an analysis of\nchurch registers of marriages, births and deaths in Keswick\n(northwest England) during the Tudor and Stuart famine of\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.6**r== 0.0**\nTable 6. Diabetes rates ( % population) for Europeans, who seem\nto have lower susceptibility to type 2 diabetes, and those of other\nraces (see Diamond, 2003)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\nTable 5. ApoE: evidence of ‘signatures of selection’\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nEuropeans: UK\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nApoE2 most common in Mediterranean basin\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nApoE4 (the ancestral lineage) more common in:\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nPigmies Khoi San\nAboriginals\nSome Native Americans\nLapps\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nGermany\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.2**\nAustralia\nUSA\nPaciﬁc Islanders: Nauru: 1952\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nAborigines: Traditional\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nWesternised\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nApoE4 associated with increased steroidogenesis and fertility\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nAsian Indians: Urban\n**BLOCK**fs== 3.7**b== 0.8**t== 0.1**l== 0.6**r== 0.4**\nm\nu\nv\no\ng\nn\ni\nt\ns\ne\nR\n**BLOCK**fs== 8.1**b== 0.9**t== 0.1**l== 0.6**r== 0.4**\n99 %\n**BLOCK**fs== 8.1**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n43 %\n**BLOCK**fs== 3.7**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nm\nu\nv\no\ne\np\nR\n**BLOCK**fs== 1.7**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\ni\n**BLOCK**fs== 3.4**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nm\nu\nv\no\nd\ne\ns\ni\nl\ni\nt\nr\ne\nF\n**BLOCK**fs== 8.1**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n64 %\n**BLOCK**fs== 3.7**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\no\ny\nr\nb\nm\ne\nd\ne\nt\nn\na\n**BLOCK**fs== 1.7**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nl\n**BLOCK**fs== 3.7**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\np\nm\n**BLOCK**fs== 1.7**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nI\n**BLOCK**fs== 8.1**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n12 %\n**BLOCK**fs== 8.1**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n60 %\n**BLOCK**fs== 3.4**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\ny\nb\na\nb\ny\nh\nt\nl\n**BLOCK**fs== 3.4**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\na\ne\nH\n**BLOCK**fs== 3.4**b== 0.7**t== 0.3**l== 1.0**r== 0.0**\ne\nh\nc\nr\na\nn\ne\nM\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nFig. 1. Possible selective effects of seasonal variations in fertility.\n($), Bangladesh; (%), The Gambia. (\n), Data suggest the possibility\nthat only ‘thrifty’ parents are able to reproduce at bad times.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\nFig. 2. Some available approximations for oocyte, embryo and\nfetal losses (attrition rate) in normal human pregnancies, indicating\nthat most genetic selection probably occurs before birth in man.\nOdds of any ovum producing a grandchild 0.0007.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n1623 (Appelby, 1978). The Tudor famine caused a 5-fold\nincrease in mortality and a coincident 3-fold drop in\nconceptions (as judged by the number of christenings). An\noutbreak of plague that occurred just 23 years later caused\na very similar increase in deaths, but no change in birth\nrate until a few years later.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nAny genetic trait that allows certain women to continue\nto reproduce in energy-depleted circumstances when others\naround them have stopped could be viewed as thrifty. The\nexample of apoE4 given earlier could be one such gene.\nFig. 1 illustrates how such genes could have been under\ncontinuous selection in addition to the episodic selection of\ncatastrophic famines. It shows the very strong seasonal\nvariation in birth rates in two populations exposed to\nin Bangladesh and The\nseasonal agricultural patterns,\nGambia. The precise reason for this striking variation is\nnot known with certainty. However, in The Gambia it is\nentirely consistent with the Frisch (1982) hypothesis of\nsuppressed fertility during times of maternal energy\ndepletion, since the lowest birth rate falls exactly 9 months\nafter the time of greatest maternal weight loss during each\nyear’s hungry season. The implication is that there are\nsome women who manage to conceive all year round,\nwhilst others are constrained to the more energy-replete\nseason of the year. Do these women have a thrifty trait?\nDoes this trait result in greater overall fertility? These\nissues are currently under analysis.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nfor\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\nselection by famine. This aspect\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nThere is an additional reason to believe that genes\ninﬂuencing reproduction and fetal well-being are likely\ncandidates\nis\nsome available\nillustrated in Fig. 2, which shows\napproximations for oocyte, embryo and fetal\nlosses in\nnormal human pregnancies. It shows the little-appreciated\nfact that the majority of natural selection of human genes\nprobably occurs before birth. If this concept is valid, then\ngenes favouring fetal survival will be under very strong\nselective pressure. This explanation would be consistent\nwith some of the previous observations that metabolic\nplasticity is probably maximised during pregnancy (Pre-\nntice & Goldberg, 2000).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.4**l== 0.6**r== 0.0**\nIn studies in progress in The Gambia it is speculated that\nthe class III allele of INS-VNTR may be one such variant\nif it favours more rapid fetal growth. It is also speculated\nthat it may be associated with the ability to conceive in the\nhungry season and hence may show different allelic\nfrequencies according to month of birth. Evidence already\nexists that INS-VNTR may have an important role in\nmodulating fetal survival because it is one of the ﬁrst\ngenes shown to display differential transmission. Eaves\net al. (1999) have shown that the offspring of heterozygous\nclass III/class I parents have a skewed ratio with 54% class\nI v. 46 % class III alleles. Such a marked transmission bias\nindicates a remarkably profound effect, which presumably\nmust be offset by a reverse selection in postnatal life in\norder to explain the persistence of the class III variant. The\nclass I allele is associated with higher insulin expression\nin the fetal pancreas (Vaﬁadis et al. 1996) and insulin-like\ngrowth factor 2 in the placenta (Paquette et al. 1998),\nwhich may increase the chances of survival of the fetus\n(Eaves et al. 1999). However, the class I allele has also\nbeen associated with low birth weight and increased\nandrogen levels before puberty, which may increase risk\nfor later adult disease (Ibanez et al. 2001; Ong et al.\n2004a,b).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\nTable 7 lists some of the other candidate genes that are\ncurrently being investigated in relation to possible effects\non fertility.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.0**\nTable 7. Some known genetic regulators of pregnancy outcome?\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nFetal growth: INS VNTR\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nMODY\nGCK-30\nmt16189\nG protein b3-subunit (825T)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\nFecundity: Twinning genes\nApoE\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nFetal survival: INS VNTR\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.0**\nINS VNTR, insulin variable number of tandem repeats; MODY, maturity-onset\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.1**\ndiabetes of the young; GCK, glucokinase; mt, mitochondrial.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nThe twin concepts of ‘thrifty phenotype’ and ‘thrifty\ngenotype’ each have merit insofar as they are based on\nintuitively-attractive ideas that, in the broadest of terms,\nhave plenty of support within biology. Each theory,\nhowever, currently lacks the level of precise evidential\nsupport\nthat will be needed to transform them from\nhypothesis to accepted fact. In the continued search for\nsuch evidence, which may ultimately inform the under-\nstanding of\nindividual variability in susceptibility to\nobesity and hence to reﬁned therapeutic strategies, it is\nlikely that research in the developing world will be most\nfruitful. In the present review an attempt has been made\nto suggest some guidelines to inform a more efﬁcient\nsearch.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nReferences\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nAdair L & Prentice AM (2004) A critical evaluation of the fetal\norigins hypothesis and its implications for developing coun-\ntries. Journal of Nutrition 134, 191–193.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nAppleby A (1978) Famine in Tudor and Stuart England.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nLiverpool: Liverpool University Press.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nArgiles JM, Busquets S & Lopez-Soriano FJ (2002) The role\nof uncoupling proteins in pathophysiological states. Biochem-\nical and Biophysical Research Communications 293, 1145–\n1152.\nAuwerx J\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nthe ultimate thrifty gene.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n(1999) PPARgamma,\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nDiabetologia 42, 1033–1049.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nBavdekar A, Yajnik CS, Fall CHD, Bapat S, Pandit AN,\nDeshpande V, Bhave K, Kellingray SD & Jogelkar CV\nInsulin resistance in 8 year old Indian children.\n(1999)\nDiabetes 48, 2242–2249.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nBell GI, Horita S & Karam JH (1984) A polymorphic locus near\nthe human insulin gene is associated with insulin-dependent\ndiabetes mellitus. Diabetes 33, 176–183.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nBennett AJ, Sovio U, Ruokonen A, Martikainen H, Pouta A,\nTaponen S, Hartikainen AL, King VJ, Elliott P, Jarvelin MR &\nMcCarthy MI (2004) Variation at\nthe insulin gene VNTR\n(variable number tandem repeat) polymorphism and early\ngrowth: studies in a large Finnish birth cohort. Diabetes 53,\n2126–2131.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nBennett ST, Lucassen AM, Gough SC, Powell EE, Undlien DE,\nPritchard LE et al. (1995) Susceptibility to human type 1\ndiabetes at IDDM2 is determined by tandem repeat variation\nthe insulin gene minisatellite locus. Nature Genetics 9,\nat\n284–292.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nBindon JR & Baker PT (1997) Bergmann’s rule and the thrifty\ngenotype. American Journal of Physical Anthropology 104,\n201–210.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nBrand MD (2000) Uncoupling to survive? The role of mito-\nchondrial inefﬁciency in ageing. Experimental Gerontology 35,\n811–820.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nChakravarthy MV & Booth FW (2004) Eating, exercise, and\n‘thrifty’ genotypes: connecting the dots toward an evolutionary\nunderstanding of modern chronic diseases. Journal of Applied\nPhysiology 96, 3–10.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nCorbo RM & Scacchi R (1999) Apolipoprotein E (APOE) allele\ndistribution in the world. Is APOE*4 a ‘thrifty’ allele? Annals\nof Human Genetics 63, 301–310.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nCorbo RM, Scacchi R & Cresta M (2004a) Differential\nreproductive efﬁciency associated with common apolipoprotein\nE alleles in postreproductive-aged subjects. Fertility and\nSterility 81, 104–107.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nCorbo RM, Ulizzi L, Scacchi R, Martinez-Labarga C & De\nStefano GF (2004b) Apolipoprotein E polymorphism and\nfertility: a study in pre-industrial populations. Molecular\nHuman Reproduction 10, 617–620.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nDalgaard LT & Pedersen O (2001) Uncoupling proteins:\nfunctional characteristics and role in the pathogenesis of\nobesity and type II diabetes. Diabetologia 44, 946–965.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nDarwin C (1989) The Origin of Species. London: Wordsworth\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nEditions Ltd.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nDiamond J (1993) Agriculture’s mixed blessings. The Third\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nChimpanzee, pp. 180–191. New York: Harper Perennial.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nDiamond J (2003) The double puzzle of diabetes. Nature 423,\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n599–602.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.6**r== 0.0**\nDulloo A & Samec S (2001) Uncoupling proteins: their roles in\nadapataive thermogenesis and substrate metabolism. British\nJournal of Nutrition 86, 123–139.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.6**r== 0.0**\nDulloo AG & Jacquet J (2001) An adipose-speciﬁc contribution\nof thermogenesis in body weight regulation. International\nJournal of Obesity 25, Suppl. 5, S22–S29.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\nDunger DB, Ong KK, Huxtable SJ, Sherriff A, Woods KA,\nAhmed ML, Golding J, Pembrey ME, Ring S, Bennett ST &\nTodd JA (1998) Association of the INS VNTR with size at\nbirth. ALSPAC Study Team. Avon Longitudinal Study of\nPregnancy and Childhood. Nature Genetics 19, 98–100.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.0**\nEaves IA, Bennet ST, Forster P, Ferber KM, Ehrmann D, Wilson\nAJ, Bhattacharyya S, Ziegler AG, Brinkman B & Todd JA\n(1999) Transmission ratio distortion at the INS-IGF2 VNTR.\nNature Genetics 22, 324–325.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.0**\nFagan B (2000) Floods, Famines and Emperors. London:\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nPimlico.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\nForse´n T, Tuomilehto J, Reunanen A, Osmond C & Barker D\n(2000) The fetal and childhood growth of persons who\ndevelop type II diabetes. Annals of Internal Medicine 33,\n176–182.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\nFrisch RE (1982) Malnutrition and fertility. Science 215,\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n1272–1273.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.6**r== 0.0**\nHales CN & Barker DJP (1992) Type 2 (non-insulin-dependent)\ndiabetes mellitus: the thrifty phenotype hypothesis. Diabeto-\nlogia 35, 595–601.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.0**\nHales CN & Barker DJP (2001) The thrifty phenotype hypothesis.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\nBritish Medical Bulletin 60, 5–20.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\nInsulin gene variable number of\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.0**\nIbanez L, Ong K, Potau N, Marcos MV, de Zegher F & Dunger D\n(2001)\ntandem repeat\ngenotype and the low birth weight, precocious pubarche, and\nhyperinsulinism sequence. Journal of Clinical Endocrinology\nand Metabolism 86, 5788–5793.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\nJoffe B & Zimmet P (1998) The thrifty genotype in type 2\ndiabetes: an unﬁnished symphony moving to its ﬁnale?\nEndocrine 9, 139–141.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.6**r== 0.0**\nJordan WC (1996) The Great Famine: Northern Europe in the Early\nFourteenth Century. Princeton, NJ: Princeton University Press.\nKelsey G, Constancia M, Dean WL, Feil RP & Reik W (1999)\nGenomic imprinting of fetal growth. In Fetal Programming:\nInﬂuences on Development and Disease in Later Life, pp.\n73–84 [PMS O’Brien, T Wheeler and DJP Barker, editors].\nLondon: RCOG Press.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.6**r== 0.0**\nKeys AJ, Brozek J, Henschel O, Michelson O & Taylor HL\n(1950) The Biology of Human Starvation. Minneapolis, MN:\nUniversity of Minnesota Press.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.6**r== 0.0**\nKimm SY, Glynn NW, Aston CE, Damcott CM, Poehlman ET,\nDaniels SR & Ferrell RE (2002) Racial differences in the\nrelation between uncoupling protein genes and resting energy\nexpenditure. American Journal of Clinical Nutrition 75,\n714–719.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.6**r== 0.0**\nKramer MS (1987) Determinants of low birth weight: methodo-\nlogical assessment and meta-analysis. Bulletin of the World\nHealth Organization 65, 663–737.\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nKramer MS (2003) The epidemiology of adverse pregnancy\noutcomes: an overview. Journal of Nutrition 133, 1592S–\n1596S.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nLindsay RS, Hanson RL, Wiedrich C, Knowler WC, Bennett PH\n& Baier LJ (2003) The insulin gene variable number tandem\nrepeat class I/III polymorphism is in linkage disequilibrium\nwith birth weight but not\ntype 2 diabetes in the Pima\npopulation. Diabetes 52, 187–193.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nLucas A, Fewtrell MS & Cole TJ (1999) Fetal origins of adult\ndisease – the hypothesis revisited. British Medical Journal 319,\n245–249.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nMcCance RA (1975) Famines of history and of today. Proceed-\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\nings of the Nutrition Society 34, 161–166.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nMaharatna A (1996) The Demography of Famine: An Indian\nHistorical Perspective. Delhi, India: Oxford University Press.\nMelve KK & Skjaerven R (2003) Birthweight and perinatal\nmortality: paradoxes, social class, and sibling dependencies.\nInternational Journal of Epidemiology 32, 625–632.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nMitchell SM, Hattersley AT, Knight B, Turner T, Metcalf BS,\nVoss LD, Davies D, McCarthy A, Wilkin TJ, Smith GD,\nBen-Shlomo Y & Frayling TM (2004) Lack of support for a\nrole of the insulin gene variable number of tandem repeats\nminisatellite (INS-VNTR) locus in fetal growth or type 2\ndiabetes-related intermediate traits in United Kingdom popula-\ntions. Journal of Clinical Endocrinology and Metabolism 89,\n310–317.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nMoore SE, Halsall I, Howarth D, Poskitt EME & Prentice AM\n(2001) Glucose,\ninsulin and lipid metabolism in rural\nGambians exposed to early malnutrition. Diabetic Medicine\n18, 646–653.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nNeel JV (1962) Diabetes mellitus: a ‘thrifty’ genotype rendered\ndetrimental by ‘progress’. American Journal of Human\nGenetics 14, 353–362.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nNewsholme EA (1994) Biochemical control\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nlogic and the\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nmetabolism of glutamine. Nutrition 10, 178–179.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nOng KK & Dunger DB (2000) Thrifty genotypes and phenotypes\nin the pathogenesis of type 2 diabetes mellitus. Journal of\nPediatric and Endocrinology Metabolism 13, Suppl. 6,\n1419–1424.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nOng KK, Petry CJ, Barratt BJ, Ring S, Cordell HJ, Wingate DL,\nPembrey ME, Todd JA & Dunger DB (2004a) Maternal-\nfetal\ninteractions and birth order inﬂuence insulin variable\nnumber of tandem repeats allele class associations with head size\nat birth and childhood weight gain. Diabetes 53, 1128–1133.\nOng KK, Phillips DI, Fall C, Poulton J, Bennett ST,\nGolding J, Todd JA & Dunger DB (1999) The insulin gene\nVNTR, type 2 diabetes and birth weight. Nature Genetics 21,\n262–263.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nOng KK, Potau N, Petry CJ, Jones R, Ness AR, Honour JW, de\nZegher F, Ibanez L & Dunger DB (2004b) Opposing inﬂuences\nof prenatal and postnatal weight gain on adrenarche in normal\nboys and girls. Journal of Clinical Endocrinology and\nMetabolism 89, 2647–2651.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nPaquette J, Giannoukakis N, Polychronakos C, Vaﬁadis P & Deal\nC (1998) The INS 5k variable number of tandem repeats is\nassociated with IGF2 expression in humans. Journal of\nBiological Chemistry 273, 14158–14164.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nParry-Billings M & Newsholme EA (1991) The possible role of\nglutamine substrate cycles in skeletal muscle. Biochemistry\nJournal 279, 327–328.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\ndeveloping world. Journal of Nutrition 131, 871S–873S.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nPopkin BM (2002) An overview on the nutrition transition and its\nthe Bellagio meeting. Public Health\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nhealth implications:\nNutrition 5, 93–103.\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nPrentice AM (2001) Fires of life: The struggles of an ancient\nmetabolism in a modern world. BNF Nutrition Bulletin 26,\n13–27.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.6**r== 0.0**\nPrentice AM (2003) Intrauterine factors, adiposity, and hyper-\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\ninsulinaemia. British Medical Journal 327, 880–881.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nPrentice AM & Goldberg GR (2000) Energy adaptations in human\nlimits and long-term consequences. American\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\npregnancy:\nJournal of Clinical Nutrition 71, 1226S–1232S.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.0**\nRuiz-Pesini E, Mishmar D, Brandon M, Procaccio V & Wallace\nDC (2004) Effects of purifying and adaptive selection on\nregional variation in human mtDNA. Science 303, 223–226.\nSen AK (1981) Poverty and Famine: An Essay on Entitlement\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.6**r== 0.2**\nand Deprivation. Oxford: Clarendon Press.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.6**r== 0.0**\nSnyder EE, Walts B, Perusse L, Chagnon YC, Weisnagel SJ,\nRankinen T & Bouchard C (2004) The human obesity gene\nmap: The 2003 update. Obesity Research 12, 369–410.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.6**r== 0.0**\nStead JDH, Hurles ME & Jeffreys AJ (2003) Global haplotype\ndiversity in the human insulin gene region. Genome Research\n13, 2101–2111.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.6**r== 0.0**\nStein Z, Susser M, Saenger G & Marolla F (1975) Famine and\nHuman Development: The Dutch Hunger Winter of 1944–1945.\nNew York: Oxford University Press.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.0**\nStocker CJ, Arch JRS & Cawthorne MA (2005) Fetal origins of\ninsulin resistance and obesity. Proceedings of the Nutrition\nSociety 64, 143–151.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.0**\nSusser M (1991) Maternal weight gain, infant birth weight, and\ndiet: causal sequences. American Journal of Clinical Nutrition\n53, 1384–1396.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.6**r== 0.0**\nVaﬁadis P, Bennett ST, Colle E, Grabs R, Goodyer CG &\nImprinted and genotype-speciﬁc\nthe IDDM2 locus in pancreas and\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.6**r== 0.1**\nPolychronakos C (1996)\nexpression of genes at\nleucocytes. Journal of Autoimmunity 9, 397–403.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.6**r== 0.0**\nvan der Sande MA, Ceesay SM, Milligan PJ, Nyan OA, Banya\nWA, Prentice A, McAdam KP & Walraven GE (2001) Obesity\nand undernutrition and cardiovascular risk factors in rural and\nurban Gambian communities. American Journal of Public\nHealth 91, 1641–1644.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.6**r== 0.0**\nWest KM (1974) Diabetes in American Indians and other native\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\npopulations of the New World. Diabetes 23, 841–855.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.0**\nWiddowson EM & McCance RA (1974) The determinants of\ngrowth and form. Proceedings of the Royal Society London\n185, 1–17.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.0**\nWilcox AJ (2001) On the importance – and the unimportance – of\nJournal of Epidemiology 30,\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nInternational\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nbirthweight.\n1233–1241.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.6**r== 0.0**\nWolfe RR, Klein S, Carraro F & Weber JM (1990) Role of\ntriglyceride-fatty acid cycle in controlling fat metabolism in\nhumans during and after exercise. American Journal of\nPhysiology 258, E382–E389.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.6**r== 0.0**\nYajnik C (2000) Interactions of perturbations in intrauterine\ngrowth and growth during childhood on the risk of adult-\nonset disease. Proceedings of\nthe Nutrition Society 59,\n257–265.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.6**r== 0.0**\nYajnik CS, Fall CH, Coyaji KJ, Hirve SS, Rao S, Barker DJ,\nJoglekar C & Kellingray S (2002a) Neonatal anthropometry:\nThe thin-fat Indian baby. The Pune Maternal Nutrition Study.\nInternational Journal of Obesity 27, 173–180.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.6**r== 0.0**\nYajnik CS, Lubree HG, Rege SS, Naik SS, Deshpande JA,\nJS (2002b)\nDeshpande SS,\nAdipocity and hyperinsulinaemia in Indians are present at\nbirth. Journal of Clinical Endocrinology and Metabolism 87,\n5575–5580.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nJogelkar CV & Yudkin\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.6**r== 0.0**\nZimmet P, Taft P, Guinea A, Guthrie W & Thoma K (1997) The\nhigh prevalence of diabetes mellitus on a Central Paciﬁc\nIsland. Diabetologia 31, 111–115.",
         "extracted",
         "Fetal origins of insulin resistance and obesity;Variation at the insulin gene VNTR (variable number tandem repeat) polymorphism and early growth: studies in a large Finnish birth cohort.;Apolipoprotein E polymorphism and fertility: a study in pre-industrial populations.;Opposing influences of prenatal and postnatal weight gain on adrenarche in normal boys and girls.;Maternal-fetal interactions and birth order influence insulin variable number of tandem repeats allele class associations with head size at birth and childhood weight gain.;The human obesity gene map: the 2003 update.;Effects of Purifying and Adaptive Selection on Regional Variation in Human mtDNA;Intrauterine factors, adiposity, and hyperinsulinaemia;Global haplotype diversity in the human insulin gene region.;Birthweight and perinatal mortality: paradoxes, social class, and sibling dependencies.;The double puzzle of diabetes;The epidemiology of adverse pregnancy outcomes: an overview.;Neonatal anthropometry: the thin–fat Indian baby. The Pune Maternal Nutrition Study;Adiposity and hyperinsulinemia in Indians are present at birth.;The role of uncoupling proteins in pathophysiological states.;Racial differences in the relation between uncoupling protein genes and resting energy expenditure.;An overview on the nutrition transition and its health implications: the Bellagio meeting;An adipose-specific control of thermogenesis in body weight regulation;Insulin gene variable number of tandem repeat genotype and the low birth weight, precocious pubarche, and hyperinsulinism sequence.;On the importance--and the unimportance--of birthweight.;The thrifty phenotype hypothesis.;Obesity and undernutrition and cardiovascular risk factors in rural and urban Gambian communities.;Glucose, insulin and lipid metabolism in rural Gambians exposed to early malnutrition;Uncoupling proteins: functional characteristics and role in the pathogenesis of obesity and Type II diabetes;Uncoupling proteins: their roles in adaptive thermogenesis and substrate metabolism reconsidered;Fires of life: the struggles of an ancient metabolism in a modern world;The nutrition transition and obesity in the developing world.;Thrifty Genotypes and Phenotypes in the Pathogenesis of Type 2 Diabetes Mellitus;Uncoupling to survive? The role of mitochondrial inefficiency in ageing;The Fetal and Childhood Growth of Persons Who Develop Type 2 Diabetes;Energy adaptations in human pregnancy: limits and long-term consequences.;Interactions of perturbations in intrauterine growth and growth during childhood on the risk of adult-onset disease;Insulin resistance syndrome in 8-year-old Indian children: small at birth, big at 8 years, or both?;Transmission ratio distortion at the INS-IGF2 VNTR;Apolipoprotein E (APOE) allele distribution in the world. Is APOE*4 a ‘thrifty’ allele?;The insulin gene VNTR, type 2 diabetes and birth weight;The thrifty genotype in type 2 diabetes;The INS 5′ Variable Number of Tandem Repeats Is Associated with IGF2 Expression in Humans*;Association of the INS VNTR with size at birth;The Great Famine: Northern Europe in the Early Fourteenth Century;Bergmann's rule and the thrifty genotype.;Imprinted and genotype-specific expression of genes at the IDDM2 locus in pancreas and leucocytes.;Susceptibility to human type 1 diabetes at IDDM2 is determined by tandem repeat variation at the insulin gene minisatellite locus;Biochemical control logic and the metabolism of glutamine.;Type 2 (non-insulin-dependent) diabetes mellitus: the thrifty phenotype hypothesis;The possible role of glutamine substrate cycles in skeletal muscle.;Maternal weight gain, infant birth weight, and diet: causal sequences.;Role of triglyceride-fatty acid cycle in controlling fat metabolism in humans during and after exercise.;A Polymorphic Locus Near the Human Insulin Gene Is Associated with Insulin-dependent Diabetes Melliitus;Poverty and famine: An essay on entitlement and deprivation : , Oxford, New York: Oxford University Press, 1981. xi + 257 pp., index. $17.95;Malnutrition and fertility.;Poverty and famines. An essay on entitlement and deprivation [incl. case studies of famines in Bengal, Ethiopia, Sahel, and Bangladesh].;Famine in Tudor and Stuart England.;The high prevalence of diabetes mellitus on a Central Pacific island;Famine and Human Development: The Dutch Hunger Winter 1944-1945;Faminee of history and of today;Diabetes in American Indians and Other Native Populations of the New World;The determinants of growth and form.;Review lecture - The determinants of growth and form;Diabetes mellitus: a \"thrifty\" genotype rendered detrimental by \"progress\"?;Biology of Human Starvation;A critical evaluation of the fetal origins hypothesis and its implications for developing countries.;Lack of support for a role of the insulin gene variable number of tandem repeats minisatellite (INS-VNTR) locus in fetal growth or type 2 diabetes-related intermediate traits in United Kingdom populations.;Differential reproductive efficiency associated with common apolipoprotein e alleles in postreproductive-aged subjects.;Eating, exercise, and \"thrifty\" genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases.;The insulin gene variable number tandem repeat class I/III polymorphism is in linkage disequilibrium with birth weight but not Type 2 diabetes in the Pima population.;Floods, Famines and Emperors. London: Pimlico;PPARgamma, the ultimate thrifty gene.;Fetal origins of adult disease-the hypothesis revisited.;Genomic imprinting of fetal growth;The Demography of Famine: An Indian Historical Perspective;Biochemical control logic and the metabolism of glutamine.;Agriculture’s mixed blessings;The Third Chimpanzee;Determinants of low birth weight: methodological assessment and meta-analysis.;Origin of Species",
         "Insights from the developing world: thrifty genotypes and thrifty phenotypes",
         "None"
        ],
        [
         "25",
         "002348cfd4535febd8cc865cddd1f98a5de57c32",
         "We present a mobile acoustic beacon based sensor node localization method. Our technique is passive in that the sensor nodes themselves do not need to generate an acoustic signal for ranging. This saves cost, power and provides stealthy operation. Furthermore, the beacon can generate much more acoustic energy than a severely resource constrained sensor node, thereby significantly increasing the range. The acoustic ranging method uses a linear frequency modulated signal that can be accurately detected by matched filtering. This provides longer range and higher accuracy than the current state-of-the-art. The localization algorithm was especially designed to work in such acoustically reverberant environment, as urban terrain. The algorithm presented handles non-Gaussian ranging errors caused by echoes. Node locations are computed centrally by solving a global non-linear optimization problem in an iterative and incremental fashion",
         "Manish Kushwaha,K. Molnár,J. Sallai,P. Völgyesi,M. Maróti,Á. Lédeczi",
         "https://ir.vanderbilt.edu/bitstream/1803/13704/1/ms-thesis-manish-kushwaha.pdf",
         "\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nSENSOR NODE LOCALIZATION USING MOBILE ACOUSTIC BEACONS\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nBy\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nManish Kushwaha\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nThesis\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nSubmitted to the Faculty of the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nGraduate School of Vanderbilt University\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nin partial fulﬁllment of the requirements\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nfor the degree of\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.3**\nMASTER OF SCIENCE\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nin\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nComputer Science\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nAugust, 2005\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nNashville, Tennessee\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nApproved:\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nProfessor Gabor Karsai\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nProfessor Akos Ledeczi\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nThis research was sponsored by the Defense Advanced Research Projects Agency\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nI would like to thank my graduate advisor, Dr. Gabor Karsai, for helping me focus\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\non my studies and research.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.1**\nI am also very grateful to my research advisor, Dr.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nAkos Ledeczi, who motivated and guided me through my work. I would also like to\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nthank Dr. Miklos Maroti for the insightful ideas he put forward that made this work\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\npossible. I am further grateful for the cooperation of Dr. Gyula Simon, Janos Sallai,\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nBranslav Kusy, Gyorgy Balogh, Karoly Molnar and other members of the NEST\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nproject team at ISIS. Their help, criticism and advice saw me through a number of\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nrough spots.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nI would like to thank my parents, who always believed in me and always encour-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\naged and supported me. Thanks to my sister, Sonal, for the faith she has in me.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nThanks to my little brother, Dhruv, who thinks of me as the smartest person. Fi-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nnally, thanks to the human curiosity which constantly drives us to do the unthinkable,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nto reach the places where no human has gone before.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nACKNOWLEDGEMENT . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nLIST OF TABLES . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nPage\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nii\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nv\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nLIST OF FIGURES . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nvi\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nChapter\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nI.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\nINTRODUCTION . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\nNode Localization . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\nSensor Network Deployment . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nII.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\nAPPLICATIONS . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.3**r== 0.1**\nHabitat Monitoring . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nShooter Localization . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nIII.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nEXISTING LOCALIZATION TECHNOLOGIES . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\nRadio Connectivity and Hop Based Ranging . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\nRadio Signal Strength Based Ranging . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\nAcoustic Time of Flight Based Ranging . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nIV.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\nAPPLIED ACOUSTIC RANGING . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\nHardware . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\nRanging Algorithm . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nResults\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.4**r== 0.1**\n. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\nSELF LOCALIZATION . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\nFormalization . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\nError Model . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\nDistance Optimization . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nPruned Distance Optimization . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nPenalty Functions . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nComposition Of Least-Square Optimization Problems . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nVI.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\nLOCALIZATION ALGORITHM . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\nIterative Incremental Localization Algorithm . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\nSub-System Selection . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\nSub-System Localization . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\nSub-System Evaluation . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nVII.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\nIMPLEMENTATION AND RESULTS . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nSimulated Data Generation . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nResults\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.1**\n. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nVIII.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nCONCLUSIONS . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nBIBLIOGRAPHY . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nTable\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nPage\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nVII.1. Localization results . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nFigure\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\nII.1.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.3**r== 0.4**\nSystem architecture for habitat monitoring\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.7**r== 0.1**\n. . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.5**\nII.2. Muzzle blast and shock wave\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n. . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.4**\nIII.1. Radio connectivity over distance\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.6**r== 0.1**\n. . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nPage\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nIII.2. Radio signal strength . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nIII.3. Flip and Flex Ambiguities . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nIV.1. Ranging hardware . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nIV.2. The emitted acoustic signal\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n. . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nIV.3. Ranging measurement results . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nIV.4.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.5**\nStandard deviation of ranging\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\n. . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nV.1.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.3**r== 0.2**\nImportance of node localization in location-critical applications\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n. .\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nV.2. Node Locus . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.8**\nV.3.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.3**r== 0.5**\nIllustration of eﬀect of echo.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n. . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nV.4.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\nIllustration of pruned least square optimization . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nV.5.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\nPenalty functions . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nVI.1. An example optimization landscape . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nVI.2. Categorization of computed locations . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nVII.1. A typical sensor network topology . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nVII.2. Simulated ranging errors . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nVII.3. Comparison of true and computed topology in XY . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nVII.4. Comparison of true and computed topology in XZ . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nVII.5. Localization results for ranging data w/o echoes . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.3**\nVII.6. Localization results for ranging data w/ echoes\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\n. . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nVII.7. Categorization of sensor nodes\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\n. . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nINTRODUCTION\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe revolutionary new technology of wireless sensor networks has opened a wide\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\narea of exciting and powerful applications that will connect the cyber-world more\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nintimately with the real world. A wireless sensor network consists of a large number of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nsmall-scale, resource-constrained computing nodes, outﬁtted with sensors and linked\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ntogether by radios that form a perceptive network that is able to monitor an ecosystem\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nor detect a speciﬁc phenomenon [6]. Although these smart sensors have limited power,\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\ncommunication and processing capabilities, an assembly of hundreds of them can\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nspontaneously organize into an ad hoc perceptive network that is spread throughout\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nthe physical world and is able to perform tasks no ordinary computer system could.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nWireless sensor networks have a middleware system whose main purpose is to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nsupport the deployment, execution and maintenance of sensing applications. R¨omer\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n[20] notes that the scope and functionality of a middleware system for WSN includes,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nbut is not limited to, “mechanisms for formulating complex high-level sensing tasks,\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\ncommunicating this task to the WSN, coordination of sensor nodes to split the task\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nand distribute it to the individual sensor nodes, data fusion for merging the sensor\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nreadings of the individual sensor nodes into a high-level result, and reporting the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nresult back to the task issuer. Moreover, appropriate abstractions and mechanisms\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nfor dealing with the heterogeneity of sensor nodes should be provided. All mechanisms\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nprovided by a middleware system should accommodate special characteristics of WSN,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\nwhich mostly boils down to energy eﬃciency, robustness, and scalability”.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nThis thesis describes a middleware service called node localization and presents a\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nnew method for node localization. Node localization, also known as self-localization, is\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nthe problem of localizing physical sensor nodes in a given sensor network deployment.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nLocalization is an essential tool for the deployment of low-cost sensor networks for\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nuse in location-aware applications [27, 33, 24] and ubiquitous networking [7, 19]. In\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\na typical sensor network application each sensor node monitors and gathers local\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ninformation. This local information has signiﬁcance if it can be tied to the physical\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nlocation it belongs to. For example in a habitat monitoring application a temperature\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nor humidity measurement by a smart sensor doesn’t provide the complete information\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nbut once combined with the physical location of the sensor, the measurements can be\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nused to build a temperature or humidity map of the local region. In location-critical\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\napplications, such as shooter-localization, sub-meter accuracy of 3D node locations is\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.3**\nan absolute necessity for the correct operation of the system [12].\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nLocalization can be either range-free [34] or range-based. Range-free localization\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\ntechniques provide rough estimates of node positions only. Ranging methods for\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nrange-based localization fall into two main classes: acoustic and radio signal strength-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nbased. The latter requires extensive calibration, yet it still achieves low accuracy and\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nlimited range. Acoustic ranging has relatively high accuracy, but short range. The\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nmain reasons are the limited acoustic energy a sensor node can emit and the relatively\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nhigh environmental noise. Having a speaker or sounder on every node adds size and\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ncost also. When stealthy operation is required, only ultrasound can be used. But\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nultrasonic ranging has even more limited range and directionality constraints.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.4**r== 0.3**\nSensor Network Deployment\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nThere are many sensor network deployment scenarios suggested for practical and\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nplacing the nodes on speciﬁed locations or random deployment of nodes in a rec-\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\ntangular grid. We propose a new practical sensor network deployment scenario with\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nmany favorable characteristics in numerous application areas is the dispersal of sensor\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nnodes from a low-ﬂying unmanned aerial vehicle (UAV) platform. An acoustic beacon\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nmounted on the aircraft can send a radio message followed by an acoustic signal at\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nrandom intervals. All the nearby sensor nodes can estimate their distance from the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nbeacon by measuring the time-of-ﬂight of the sound. As size and power are not as\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nbig constraints on a UAV as on a sensor node, the maximum range can be signiﬁ-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ncantly increased by increasing the size and power of the sound source. Furthermore,\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nthe nodes do not reveal their positions since they are only passive listeners in this\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nscenario.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nIn general the node localization problem can be deﬁned as ﬁnding physical loca-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ntions of sensor nodes in a given network deployment scenario. This work redeﬁnes\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nthe self-localization problem for our network deployment scheme as ﬁnding the sensor\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nnode locations given only the distance measurements between unknown mobile bea-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\ncon transmission locations and the sensor nodes. Neither the mobile beacon positions\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nnor the sensor nodes themselves are located necessarily on a plane. Therefore, the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nlocalization problem needs to be solved in 3D. In complex environments sensor nodes\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nmight not have direct line-of-sight with mobile beacon but they receive the signal via\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nmultipath propagation. These multipaths cause ambiguity in ranging data and pro-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nduce false results. To our knowledge, no solutions exist in the literature that handles\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nmultipath eﬀects in 3D environment. For urban deployments this problem needs to\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nbe addressed as well.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThe main contribution of this work is the localization algorithm based on the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nnovel idea of a mobile beacon and its ability to handle multipath eﬀects. The ranging\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nmethod is based on the time-of-ﬂight measurement of an acoustic signal emitted\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nby a single beacon from multiple locations. The acoustic signal used is a linear\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nfrequency modulated (chirp) signal, that can be identiﬁed with high accuracy by\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nmatched ﬁltering at the sensors even at low SNR. Self localization is modeled as a\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nnon-linear optimization problem where node locations are the optimization variables\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nand distance equations involving node locations are non-linear objective functions.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe localization algorithm is both iterative and incremental. At each iteration a part\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nof the sensor network is selected, localized and evaluated. It is incremental because\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nat each iteration the part of sensor network selected will grow around the previously\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nlocalized nodes. This method is a generalization of iterative localization algorithms\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nwhere node location is improved at each iteration.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nThe document is organized as follows. Chapter I provides motivation, brief in-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ntroduction and challenges in node localization. Chapter II introduces two interesting\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nsensor network applications that require node localization. Related research work in\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nself localization is summarized in chapter III. Chapter IV presents the novel acoustic\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nranging method used in this work. Formal problem deﬁnitions and mathematical tool\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\ndeﬁnitions are presented in chapter V. In Chapter VI we present the main localization\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nalgorithm in detail. The implementation and evaluation of localization technique is\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nprovided in chapter VII. Chapter VIII concludes this document with a discussion on\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nthe signiﬁcance of deployment strategy and localization algorithm presented in this\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nthesis.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nAPPLICATIONS\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nCurrent and potential application areas for wireless sensor networks include habi-\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ntat monitoring, geophysical monitoring, precision agriculture, military systems, sur-\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nveillance systems, traﬃc monitoring, business processes management, smart building\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\nmanagement, and in the future, possibly smart homes for everybody.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nTwo very interesting applications are described in this chapter. The ﬁrst one is\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nthat of habitat monitoring of Great Duck Island oﬀ the coast of Maine designed by\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nUniversity of California at Berkeley [1], and the second is a countersniper system\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\ndesigned and developed at Vanderbilt University [12].\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\nHabitat Monitoring\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nIn fragile habitats, such as a small island, human intervention for observation and\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\ndata gathering can adversely aﬀect the small ecosystem. Sensor networks provide\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nnon-invasive methods of observation of such fragile ecosystems. Sensor networks also\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nenable life science researchers to observe and monitor habitats that are not easily\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\naccessible to humans.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nSensor nodes with weather sensor boards were deployed in the observation area\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nand their location were recorded. The weather sensor boards had temperature, light,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nbarometric pressure, humidity and passive infrared sensors. The data recorded by\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nsensors would be transmitted to the base-station. Base-stations would transfer the\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\ndata over network to data storage or client data processing units. In this way sensor\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nnetworks allow clients to remotely monitor the habitat. Figure II.1 [1] shows the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.3**r== 0.2**\nFigure II.1: System architecture for habitat monitoring\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nShooter Localization\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nCountersniper systems are important tools for armed forces and law enforcement\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nagencies. Many systems have been developed in the past but only a few can meet the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nrequirements in complex environment as urban terrain. Simon [12] notes that “the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nmain problems degrading the performance of these systems are the poor coverage due\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.2**\nto the shading eﬀect of the buildings and the presence of multipath eﬀects”.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nSeveral physical phenomenon associated with shot ﬁring can be used for shooter\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nlocalization. The most obvious acoustic event due to ﬁring of a conventional weapon\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nis the muzzle blast. The countersniper system in [12] utilizes this muzzle blast and\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nthe shockwave generated by the supersonic bullet to localize the shooter. Figure\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nII.2 [12] shows the simpliﬁed geometry of the bullet trajectory and the associated\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nmuzzle blast and shockwave fronts. The muzzle blast produces a spherical wave\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nfront, traveling at the speed of sound (vs) from the muzzle (A) to the sensor (S). The\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nshock wave is generated in every point of the trajectory of the supersonic projectile\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nproducing a cone-shaped wave front, assuming the speed of the projectile is constant\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nFigure II.2: Muzzle blast and shock wave\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nvB. The shockwave reaching sensor S was generated in point X. The angle of the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nshockwave cone is determined by the speed of the projectile. The sensor nodes detect\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nand measure the time of arrival (TOA) of shockwaves and muzzle blasts. The system\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nthen utilizes the measurements to localize the source of muzzle blast.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nIt is crucial to have accurate sensor node locations for the correct and accurate op-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\neration of shooter localization application described above. The shooter-localization\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\napplication localizes the shooter based on the TOA of muzzle blast and shockwaves\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nand the locations of the sensor nodes that detect muzzle blast. The errors in sensor\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nnode locations are propagated to the shooter localization errors. A more detailed il-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nlustration of eﬀect of node location errors on shooter location error is given in chapter\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nV.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.3**r== 0.2**\nEXISTING LOCALIZATION TECHNOLOGIES\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nSelf localization, due to its importance in sensor network applications, has been\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nan active research area for the past few years. An early survey of localization systems\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nin presented by Hightower and Boriello in [14]. Many of these systems adopt a simple\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nconnectivity based ranging approach, while some of them further reﬁne range esti-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nmates between node pairs by measuring the received radio signal strength. However,\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nRSS based ranging requires extensive calibration and still yields inaccurate range es-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\ntimates [22] resulting in coarse localization. A number of recent localization systems\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nuse acoustic time of ﬂight for ranging. The calibration required for acoustic ranging\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nis minimal. The localization system presented in this work uses acoustic ranging.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nOne of the earliest and most popular localization systems is GPS or Global Po-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nsitioning System [13]. GPS was designed by and is controlled by the United States\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nDepartment of Defense. GPS is a satellite navigation system used for global outdoor\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nlocalization and providing highly accurate global time reference, which is useful, for\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nexample, in military operations. The project, started in 1978, uses a multi-million\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ndollar infrastructure of 52 GPS satellites, the latest of which was launched in 2004.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nDespite the popularity and easy accessibility of GPS receivers they are not suitable\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nfor localization in wireless sensor networks. The reasons for this are the cost factor,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\npower consumption and the localization accuracy. Even the best GPS receivers do\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nnot claim more than 2-3 meter resolution and can have up to 10-20 meter localization\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nerror. This accuracy is acceptable for applications like vehicle navigation or landmark\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.2**\nlocalization but it is not suﬃcient for many sensor network applications.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nMany localization algorithms are based on simple circular radio connectivity model\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\n[8]. In this approach the radio transmitter is sitting at the center of the circular region\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nand all nodes within this region are considered connected to the transmitter. This\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nmodel is very simple and doesn’t require any special ranging infrastructure. The\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nnetwork routing infrastructure can be used for radio connectivity based ranging. The\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nlocalization systems that are based on this approach are [28, 10]. There are several\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nproblems with ranging based on radio connectivity. Asymmetries in the environment\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nor in the antenna’s orientation can aﬀect radio connectivity. Furthermore, the radio\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nranges show probabilistic behavior i.e.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\ntwo radio nodes are connected with some\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nprobability. This link quality is illustrated in Figure III.1 [8]. Apart from the problem\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nFigure III.1: Radio connectivity over distance\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nof being probably connected links there are two more issues with this approach. Even\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nif two radio nodes are connected to a transmitter, the model is insuﬃcient to tell\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nwhich node is closer. Finally, many pairs whose connectivity falls near the boundary\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nwill oscillate between being connected and disconnected.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nA simple connectivity-based localization system by Bulushu [28] employs a grid\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nof reference nodes with overlapping regions. Unknown nodes localize themselves to\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nthe centroid of their proximate reference nodes. The accuracy of localization is then\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ndependent on the separation distance between two adjacent reference nodes and the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ntransmission range of these reference nodes. Experiments show localization accuracy\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.2**\nof about one third of separation distance between reference nodes.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nNiculescu and Nath [10] proposed a distributed hop by hop localization algorithm\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ncalled APS (Ad hoc Positioning System). The idea here was to propagate the anchor\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nnode location information in the network. Nodes with unknown location note the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nshortest hop count to each of the anchor nodes and multiply this with an average hop\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ndistance to get an approximate distance to each of the anchor nodes. Nodes can then\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nperform triangulation to get an estimate of their locations. They describe three diﬀer-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nent propagation methods, DV-Hop, DV-distance and Euclidean propagation methods,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\neach providing diﬀerent tradeoﬀ between accuracy, signaling complexity, coverage and\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nisotropy of the network. Typical location errors were of the order of 20 − 150% of\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nradio range.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nDoherty [21] formulated self localization as a geometric constraint feasibility prob-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nlem based on node connectivity. The problem was formulated as, given anchor node\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nlocations ﬁnd a possible position for each unknown node subjected to the proximity\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nconstraints between node pairs imposed by known connections. The problem was\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nsolved using convex optimization. Additionally, rectangular bounds on node posi-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ntions were used for tighter geometric constraints. Doherty noted that provided tight\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nenough geometric constraints, simulations show that the node estimates become close\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nto actual node locations. For sensor network of 200 nodes in a 10×10 R region, where\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nR was the maximum range. The simulations show that the mean location errors were\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\n1 − 9 R for diﬀerent anchor nodes ratios.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.4**r== 0.3**\nRadio Signal Strength Based Ranging\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nSeveral localization systems use received radio signal strength to estimate ranges\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nbetween transmitter and receiver. Recent research projects in ad-hoc localization\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nusing signal strength are SpotON [15] and AHLoS [4] among others. For a symmetric\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ntransmitting antenna in a near-ideal environment signal strength at a receiver with\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\ndistance r from a transmitter can be described by RADAR equation,\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nPr =\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nPtGtAr\n4πR2\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nwhere Pr is the received power, Pt is the transmit power, Gt is antenna gain, Ar is\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nthe eﬀective receiver area and R is the distance between transmitter and receiver.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nSince receiver area Ar is constant, signal strength is eﬀectively inversely proportional\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nto square of the distance. This simple radio signal strength model is unreliable in\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\ncomplex indoor or urban environments due to obstacles and reﬂections. Figure III.2\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\n[11] shows the observed mean and standard deviation of signal strength with distance\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nin a near-ideal environment i.e. outdoor ﬁeld.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nSavarese [5] follows two phase localization algorithm,, containing a start-up and\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\na reﬁnement phase. The start-up phase utilizes the hop-TERRAIN algorithm which\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nis similar to DV-hop [10]. The hop-TERRAIN algorithm ﬁnds the number of hops\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nfrom a node to each anchor nodes and then multiplies this hop count by an average\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nhop distance to estimate the range from anchors. The nodes with distances from\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nanchors and known anchor location triangulate their estimated position. The second\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nstage called the reﬁnement is an iterative algorithmic step that uses the ranges and the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.3**\nFigure III.2: Radio signal strength\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nin a number of steps. Savarese [5] also introduces a crude notion of conﬁdence value,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.4**\na metric for the quality of location estimate.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nSavvides [3] solves for unknown node position estimates by setting up a global non-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nlinear optimization problem and solving it using iterative least-squares. The method\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nrequires the known beacons to surround the unknown nodes, which the author calls\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nbeacon-unknown node convexity. However, this topology constraint is hard to satisfy\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.3**\nwith airborne deployment of sensor nodes in hostile urban areas.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nAcoustic Time of Flight Based Ranging\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThe ranging techniques based on time of ﬂight (TOF) measurements of signals\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nprovide better results that are suﬃcient for ﬁne-grained localization [18]. RF time of\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nﬂight based techniques, like GPS, have limited applicability in sensor networks. Much\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nbetter ranging results can be achieved when acoustic and RF signals are combined\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\n[22, 2]. Recently, several localization methods have been proposed that utilize distance\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nare temperature dependent and require an unobstructed line of sight. In a typical\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nindoor or urban environment non-line of sight range measurements, or multi-paths\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nproduce false ranges that are diﬃcult to separate from good ranges. Also, the ratio of\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nmulti-paths to line-of-sight range measurements is considerable, thus any algorithm\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nlocalizing nodes in such scenarios has to consider multi-path propagation.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nCricket [30, 31] is an indoor location support system for pervasive and sensor-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nbased computing applications. Cricket is intended for use indoors or in urban areas\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nwhere outdoor systems like the Global Positioning System (GPS) don’t work well.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nCricket uses a combination of RF and ultrasound to provide a location-support ser-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nvice to users and applications. Wall and ceiling-mounted beacons are spread through\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nthe building, publishing location information on an RF signal. With each RF adver-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ntisement, the beacon transmits a concurrent ultrasonic pulse. The listeners receive\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nthese RF and ultrasonic signals, correlate them to each other, estimate distances to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nthe diﬀerent beacons using the diﬀerence in RF and ultrasonic signal propagation\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\ntimes, and therefore infer the space they are currently in.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nWe use acoustic time of ﬂight based ranging developed at Vanderbilt University\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.2**\n[18] in our localization system. It is described in detail in section IV.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nThere are few approaches that deal with multi-path propagation. One such ap-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.3**\nproach for two dimensions is presented by Moore [9].\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.7**r== 0.1**\nIt identiﬁes multi-paths as\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\ngeometric impossibilities. Moore et. al.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\nformulated node localization as a two-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\ndimensional graph realization problem. They identiﬁed that due to insuﬃcient or\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nnoisy data there could be ambiguity or uncertainty in node positions. The two types\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nof ambiguities in a graph that prevent unique realization are ﬂip and ﬂex ambiguities\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nas shown in Figure III.3[9]. As deﬁned in [9], “Flip ambiguities (Figure III.3a) occur\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nfor a graph in a d-dimensional space when the positions of all neighbors of some ver-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\ntex span a (d − 1)-dimensional subspace. In this case, the neighbors create a mirror\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nFigure III.3: (a) Flip ambiguity. Vertex A can be reﬂected across the line connecting\nB and C with no change in the distance constraints. (b) Discontinuous ﬂex ambiguity.\nIf edge AD is removed, then reinserted, the graph can ﬂex in the direction of the arrow,\ntaking on a diﬀerent conﬁguration but exactly preserving all distance constraints.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nthrough which the vertex can be reﬂected. Discontinuous ﬂex ambiguities (Figure\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nIII.3b) occur when the removal of one edge will allow part of the graph to be ﬂexed\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nto a diﬀerent conﬁguration and the removed edge reinserted with the same length”.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThey introduce the concept of robust quadrilaterals to overcome these ambiguities.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nIn terms of graph theory a robust quadrilateral is a set of four fully-connected nodes\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nthat are unique up to a global rotation, translation and reﬂection, i.e. it is globally\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nrigid. The idea can be extended to three dimensions but as [9] noted, under low\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nconnectivity or high measurement noise conditions the algorithm may be unable to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nlocalize a useful number of nodes. Another case where the geometric constraint based\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\necho identiﬁcation fails is when there is disproportionate geometry thickness in one\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ndirection. In a typical sensor network the X and Y distribution of nodes is much\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nhigher than that in Z which can aﬀect the performance of the algorithm.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nThe localization algorithm presented in this thesis models the problem as global\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nnon-linear optimization problem as in [3], however it goes one step further to deal\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nwith echoes and non-convexity of anchor-unknown node topology.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.4**r== 0.3**\nAPPLIED ACOUSTIC RANGING\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe localization system described in this work uses acoustic time-of-ﬂight based\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nranging developed at Vanderbilt University [18]. The concept of acoustic ranging is\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nbased on measuring the time of ﬂight of the sound signal between the signal source\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n(beacon) and the acoustic sensor. The range estimate can be trivially calculated from\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nthe time measurement. However, the speed of sound is temperature dependent. As-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nsuming insigniﬁcant spatial temperature distribution in the sensor ﬁeld, this problem\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\ncan be solved by a single temperature measurement at the base station. An appealing\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ncharacteristic of the proposed ranging algorithm is that this is the only calibration\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nthat is needed. That is the sensors do not need individual calibration at all.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nHardware\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nThe acoustic ranging application targets the MICA2 motes developed at UC\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nBerkeley [16]. The mote is equipped with a custom acoustic sensor board, which\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nwas developed at the Vanderbilt University for a shooter localization application [12].\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nThe heart of the sensor board is a low-power ﬁxed point ADSP-2189 digital signal\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nprocessor running at 50 MHz (see Figure IV.1). The availability of the DSP enables\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\nthe implementation of sophisticated digital signal processing algorithms .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThere are two independent analog input channels on the board, furnished with\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nlow-cost electret microphones and 2-stage ampliﬁers with software programmable gain\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\n(0-54dB). The analog channels are sampled by A/D converters at up to 100kSPS with\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\n12-bit resolution. The board also has an analog output channel capable of driving a\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n(a)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n(b)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nFigure IV.1: Ranging hardware (a) MICA2 motes, (b) acoustic sensor board\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n250 mW external loudspeaker. The board is connected to the mote by programmable\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.3**\ninterrupt and acknowledgment lines and a standard I2C bus.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nIn the current implementation the mobile beacon is based on a MICA2 mote\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nand the same sensor board with an active loudspeaker attached to its analog output\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nchannel. The maximum output power is 105 dB measured 10 cm away from the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nloudspeaker.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nRanging Algorithm\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nIn order to calculate the range from the time-of-ﬂight of the acoustic signal, the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ndeparture and arrival times of the signal have to be identiﬁed and measured precisely.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nThe beginning of the transmission can be measured at the beacon, while the time of\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\narrival is measured at the receiving sensors. The range calculation is performed on\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nthe receivers, thus the beacon has to send the starting time to the receivers in a radio\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nmessage.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nEmploying a sophisticated time synchronization mechanism is essential to accu-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nrately measure the time of ﬂight. Our approach employs the message time-stamping\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nprimitives introduced in [26]. The synchronization between the source and sensor\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nnodes is implemented as follows.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nThe source queries its local time t0 and decides that it will emit an acoustic signal\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nat time tsend = t0 + δ. The source sends the value tsend to all the sensors in a radio\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nmessage. Therefore the value of δ is chosen such that it is greater than the time\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nrequired by the sensors to process the radio message and to prepare for reception.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe sensors schedule their acoustic board for sampling when the beacon starts the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.5**\ntransmission of the acoustic signal.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nWe assume that the skew of the local clocks is negligible during the short time\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nof the measurement, but we allow arbitrary clock oﬀsets. Since neither the source,\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nnor the sensors have knowledge of a global time, the sensors need to convert tsend\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nincluded in the message from the local time of the source to their own local times.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThis is achieved by timestamping the radio message at transmission and at reception\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nas well. Since the radio signal is traveling at the speed of light, the diﬀerence between\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nthe transmit time instant and the receive time instant is negligible, hence the trans-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nmit timestamp (given by the local clock of the beacon) and the receive timestamps\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n(in the local time of the receivers) are assumed to represent the same global time\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\ninstance. Thus, a sensor can use the diﬀerence of the transmit timestamp and its\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nreceive timestamp to calculate the oﬀset of its local clock from the local clock of the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nbeacon. This oﬀset is added to the received tsend to convert it to the local time of the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n(a)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n(b)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nFigure IV.3: Ranging measurement results (a) without outlier rejection, (b) with\noutlier rejection\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nThe sensor node also has to measure the time of arrival of the acoustic signal. The\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\naccurate detection of the signal is not trivial in a noisy environment, as it is diﬃcult\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nto emit sharp rising edges or pulses with general purpose loudspeakers. Additionally,\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nthe signal has to be emitted with the highest power available in order to maximize\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nthe range of the measurement. These requirements are analogous to the problems\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nof radar signals, a well researched area [29, 23]. The problem arises as the limited\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nbandwidth of the analog output channel restricts the emission of rising edges with\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\narbitrarily steep slope. The contradiction is resolved by long duration signals with\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nshort duration correlation functions, so when the received signal goes through an\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nappropriate matched ﬁlter, the output will be a sharp pulse. The emitted signal is\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ntherefore a Gaussian-windowed linear frequency modulated (chirp) signal shown in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nFig.IV.2, that is commonly used in radar applications. The windowing is needed due\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\nto the limited bandwidth of the acoustic channel.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nA similar solution is presented in [22], where the emitted signal is a binary phase\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nshift keying (BPSK) spread spectrum signal. Since our method does not require to\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\ndistinguish multiple sources, the use of linear frequency modulated signal is more\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nnatural.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nThe frequency span of this signal is spread in the whole acoustic band of the analog\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nchannels. The matched ﬁlter is realized as an FIR ﬁlter on the DSP. The matched\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nﬁltering essentially means the correlation of the expected signature with the measured\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ndata, therefore the length of the FIR ﬁlter is the same as the length of the expected\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nsignature. To avoid a high order FIR ﬁlter which would be computationally expensive,\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\neither the length of the chirp signal or the sampling rate has to be decreased. However,\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nas the length of the chirp signal can not be arbitrarily short due to the limited\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nbandwidth of the physical hardware, the sample rate has to be decreased. Thus, the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nraw data is decimated to a lower sampling frequency before the matched ﬁltering.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nIn order to increase the signal-to-noise ratio (SNR), one range measurement con-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nsists of a series of time-of-arrival measurements. As the delays between the consec-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nutive chirps are known a-priori, an accurate combined result can be calculated by\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\naveraging these measurements. In the averaged signal the chirp signature component\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nis preserved as it is added up at the same phase, but the noise which is assumed to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nbe independent Gaussian white noise is decreased by\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n√\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.7**r== 0.1**\nN where N is the number of\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nchirps added. Currently we use 8 chirps, thus the SNR of the averaged signal is 9 dB\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\nhigher than the SNR of a single chirp.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nDelays between consecutive chirps are varied to avoid a situation when multiple\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nruns have the same noise pattern at the same oﬀset, which is a common phenom-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nenon caused by acoustic multi-path eﬀects. Hence the independent nature of the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\ndisturbances is preserved.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThe decimation ﬁltering is running online on the DSP, and the decimated signal\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nis stored in RAM buﬀer. The consecutive measurements are added together in the\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nsame buﬀer and after all the chirps are received, the matched ﬁltering and the peak-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\ndetection algorithm is performed oﬄine. The peak-detection algorithm is simply a\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nmaximum ﬁnder above a threshold level, as the output of the matched ﬁlter has\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\ndistinctive peaks at chirps. The time of arrival of the chirp signal can easily be\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nidentiﬁed based on the location of the peak.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nResults\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe above algorithm was tested on a grassy ﬁeld with a single beacon, and multiple\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nreceivers. In Fig. IV.3 the ranging results are presented, and in Fig. IV.4 the standard\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ndeviation of the measurements is shown, after outlier rejection. Outlier rejection is\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ndone by a simple median ﬁlter, where the values greatly diﬀering from the mean of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nthe measurements are rejected. Note that multiple measurements are needed for each\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nbeacon position to perform the rejection algorithm.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.4**r== 0.3**\nFigure IV.4: Standard deviation of ranging\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nThe eﬀective range of the presented implementation is 30 meters, as the number\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nof outliers and the standard deviation of the measurements are getting signiﬁcantly\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nhigh above this value. Below 30 meters the standard deviation grows approximately\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nlinearly, with\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nST D ∼= k1d + k2\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\nwhere k1 = 0.011 and k2 = 0.024 and d denotes the actual distance.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nThe eﬀective range of the measurements is more than two times larger than in\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nprevious acoustic ranging experiments [18, 25], where the reliable range was 10 m on\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nasphalt and 15 m on grass, respectively. The standard deviation is also signiﬁcantly\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nimproved.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\nIn [18], the output power of the sounder was limited (88 dB at 10 cm\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nfrom source) and no custom DSP board was used. In [25] the power of the beacon is\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\napproximately the same as in the presented solution (105 dB at 10 cm from source),\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nhowever our use of the DSP board and the linear frequency modulated signal provides\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.6**\nbetter performance.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nThese experimental results are very promising and justify the presented approach.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nMoreover, the current limits on range and precision are primarily caused by issues\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nwith the current implementation. First, the power of the emitted acoustic signal is\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nstill constrained by the gain on the output channel of the board. Second, the analog\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ninput channels of the DSP board also limit the range, as it was designed for a shooter\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\ndetection application, where even the maximum gain is relatively low.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.4**r== 0.3**\nSELF LOCALIZATION\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nAs described in chapter I, high node localization accuracy is an absolute necessity\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nfor the correct operation of location-critical applications, such as shooter-localization.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe application uses TOA of acoustic signals and locations of the nodes that detect\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nthe muzzle blast of shooter. If the node locations are erroneous, the error will prop-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nagate to the shooter localization error.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nAn illustration to show the eﬀect of node localization accuracy on acoustic-beacon\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nlocalization is presented below. The simulation setup consists of four sensor nodes and\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nan acoustic beacon in 2D plane. The goal of the simulation is to localize the beacon,\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ngiven the TOA of acoustic signal from beacon to each of the sensor nodes. Sensor\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nnode locations are given with zero mean and given standard deviation Gaussian error.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nThe simulation uses least-squares optimization for beacon localization. Figure V.1(a)\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n(a)\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n(b)\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nFigure V.1:\n(a) Simulation setup. Big dots denote actual node/beacon location\nwhile the small dots indicate computed locations. (b) shows the behavior of beacon\nlocalization error w.r.t. the standard deviation in node location error.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nshows the 2D simulation setup while Figure V.1(b) shows the beacon localization\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nerror against standard deviation in sensor node location error. Beacon localization\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nerror increases at least linearly with sensor node location error. Hence, it is very\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nimportant to have accurate sensor node locations for accurate operation of location-\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\ncritical applications.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nFormalization\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nFormally, a generalized self localization problem can be deﬁned as follows. Given\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nnode IDs and their ranges from each other, conjecture the relative physical location\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nof each node in the network. Few anchor nodes can be provided to transform relative\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nnode locations to absolute positions.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThere are many challenges to be addressed in this problem. First we deﬁne some\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nterms that we will use in the rest of the document.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nDistance Matrix\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nDistance Matrix D is a matrix such that dij is the range measurement between\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nnode i and node j. Distance is negative for node pairs for which range measurement\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nis not known. Number of positive entries in row i represents the number of neighbors\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nof node i.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\nD =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n\n\n\n\n\n\n\n\n\n\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nd12\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n. . . d1n\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\nd21\n...\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\n...\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n. . . d2n\n...\n. . .\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\ndn1 dn2\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n. . .\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n\n\n\n\n\n\n\n\n\n\n\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.9**r== 0.1**\n(V.1)\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\nNecessary Condition for Localization\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nNecessary Condition for Localization in 3 dimensions states that a node\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nshould have distance measurements with at least four non-coplanar neighbor nodes.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nFollowing ﬁgure illustrates the necessity of four non-coplanar neighbor nodes. Fig-\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nure V.2 shows the locus of node location with number of neighbors. In case of one\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nneighbor the locus is a sphere; for two neighbors the locus is a closed curve that is\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nintersection of two spheres (a circle). For three neighbors the locus is reduced to two\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\npoints. We need fourth neighbor to get rid of this mirror ambiguity.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nFigure V.2: Node Locus\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nMulti-path\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nIn a typical urban environment many sensor nodes might not have line-of-sight\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nwith the mobile beacon but they can receive the acoustic signal via multipath. These\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nmultipath ranges or echoes when used for localization tend to produce false or infeasi-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nble results. Consider the following 2-dimensional example (Figure V.3) for illustration.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nSound sources S1 and S2 have line-of-sight with sensor node U but the direct path to\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nFigure V.3: Illustration of eﬀect of echo.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nsound source S3 is obstructed by wall W1, thus U receives the acoustic signal from S3\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nvia multipath through wall W2. Due to this echo the range measurement between S3\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nand U is d0 instead of the actual value d. Based on these range measurements, node\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nU 0 is localized away from its actual location U . If the wall W2 were moved little far-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nther the echo distance d0 would increase and at some point no feasible location would\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nexist for node U . Similar illustration can be drawn for a 3-dimensional case. It is\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nclear from the above illustration that range measurements with echoes would not only\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nprevent some nodes to be localized but also—even worse—cause mis-localization.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nNotice also that the amount of echoes present in the range measurements depends\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\non the topology. We found no way of estimating the extent of echoes in the measure-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nment data. In typical urban environments, low network connectivity and non-uniform\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nnode distribution in the vertical (Z-)direction further deteriorate the localization ac-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ncuracy. These eﬀects can cause even larger localization errors at boundary nodes.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nFor most measuring instruments, the actual physical measurement is aﬀected by\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nsome sources of errors that we call noise. The most common form of noise is the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nwhite Gaussian noise, which exhibits a Gaussian distribution with zero mean and an\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\ninstrument or measurement process dependent standard deviation.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nRanging measurements also show white Gaussian noise for a “near ideal” environ-\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nment, like an outdoor ﬁeld where all sensor nodes have direct line of sight without\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nany obstructions. Multipath propagation in urban or complex environments causes\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nnon-Gaussian errors in ranging measurements.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nMost of the localization systems do not consider non-Gaussian errors in ranging\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nerror distribution. Considering the non-Gaussian nature of the ranging error is essen-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ntial to solve the localization problem for satisfactory results. There are many ways\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nranging error distribution can be modeled. It can be considered as non-parametric\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ndistribution or as combined distribution of Gaussian and chi-squared distribution.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nThis work doesn’t directly address the error modeling problem and doesn’t explore\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nthe non-Gaussian nature of ranging errors. This work does acknowledge that ranging\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nare non-Gaussian, and deals with them by using a technique called pruned least-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nsquared optimization. Pruned least-squared optimization is described later in this\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nchapter.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.4**r== 0.3**\nDistance Optimization\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThe self localization problem in its most basic form can be modeled as a distance\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\noptimization problem. In the distance optimization problem the independent opti-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nmization variables are node locations and the non-linear objective functions are the\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\ndiﬀerences between distances computed from node locations and range measurements\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nfor all node pairs for which range measurements exist. It can be observed that the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nare the non-linear functions of node locations. Least squares optimization is known\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nto work best for function-ﬁtting problems [17]. The mathematical formulation of\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\ndistance optimization problem is presented below.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nFind x∗, a global minimizer for objective function\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nF (x) =\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nN\nX\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nN, ˆdij ≥0\nX\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\ni=1\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nj=1\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\ndij − ˆdij\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n2\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n(V.2)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nwhere\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\ndij = {(xi − xj)2 + (yi − yj)2 + (zi − zj)2}1/2\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nis the computed distance between nodes i and j, and ˆdij is the measured distance.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe optimization variable is x = [x1y1z1x2y2z2 . . . xnynzn]T where [xiyizi] is the 3D\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\ncoordinate of node i.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe non-linear objective function F (x) is the square sum of distance errors for\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nall pairs (i, j) for which range measurement exists ( ˆdij ≥ 0). The components of the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.2**\noptimization variable x are subjected to the boundary value constraints.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nxmin ≤ xi ≤ xmax\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nymin ≤ yi ≤ ymax\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nzmin ≤ zi ≤ zmax\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n(V.3)\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.4**r== 0.3**\nPruned Distance Optimization\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nAs mentioned in section V we have non-Gaussian error in the form of echoes\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nin range measurements. In least-square optimization terminology these echo ranges\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nare outliers that tend to shift the least-square model from the actual model. It is\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nlogical not to consider these outliers in optimization. For this reason the optimization\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nproblem described in previous section is modiﬁed to account for probable outliers. The\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\ndeﬁnition of the new optimization problem requires an operator min which is deﬁned\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nbelow.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nOperator min\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nDefinition 1. Let fi be a list of N function evaluations (or numbers), then minpfi\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nis the list of dpN e-many smallest function evaluations (or numbers) where d e\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.5**\nis ceiling operator and 0 ≤ p ≤ 1.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nDefinition 2. Let PN\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\ni fi be a series sum of N function evaluations (or numbers),\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nthen PN\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\ni minpfi is the series sum of dpN e-many smallest function evaluations\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.4**\nwhere d e is ceiling operator and 0 ≤ p ≤ 1.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe mathematical formulation of the new distance optimization problem is presented\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nbelow.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.5**\nFind x∗, a global minimizer for\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nF (x) =\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nN\nX\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nN, ˆdij ≥0\nX\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\ni=1\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nj=1\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nminp\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\ndij − ˆdij\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n2\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n(V.4)\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nwhere ˆdij and dij are the range measurement and distance computed from localized\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nnodes i and j and optimization variable is x = [x1y1z1x2y2z2 . . . xnynzn]T.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nIf the optimizer x is close to global optimizer x∗ then all function evaluations\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.3**\nbut those corresponding to echoes will be close to zero.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.7**r== 0.1**\nIn other words, near the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nglobal minimizer all consistent range measurements will have function evaluations\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nwith values close to zero while echoes, which are non-consistent range measurements,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\nwill produce function evaluations with larger values.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nLeast-square optimization works best if the errors have Gaussian distribution.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nWhen we discard few largest function evaluations using the min operator, we are\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\ndiscarding the most signiﬁcant outliers in the distribution and hence obtaining an\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\napproximate Gaussian distribution. Figure V.4 illustrates pruned optimization ap-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\nFigure V.4: Illustration of pruned least square optimization\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nproach for a line ﬁtting problem. In Figure V.4(a) all the data points are considered\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nin optimization. The resulting curve is highly oﬀset from the actual curve due to the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nnon-Gaussian errors. In Figure V.4(b) data points with large deviations from opti-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nmized solution are removed from computation and curve is re-optimized for remaining\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ndata points. Figure V.4(c) and (d) show the subsequent steps; the ﬁnal solution is\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nvery close to the actual curve.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nThe standard optimization solvers used in this work were designed for uncon-\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nstrained optimization. The bounded-value constraints on the optimization variables\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nare incorporated separately by modeling them as penalty functions in the objective\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nfunction. Penalty functions incorporate a penalty value if variables go out of bound.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe most intuitive form of a penalty function is a rectangular penalty wherein\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\na constant high penalty is incorporated if the variable goes out of bounds. For op-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ntimization purposes rectangular penalty does not provide motivation (descent direc-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ntion) for the variable to fall within bounds. Other forms of penalty functions are\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nlinear or quadratic penalties which grow linearly or quadratically with the oﬀset from\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nthe bounds. Logarithmic penalty functions are most suitable for bounded-value con-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nstraints because of their sudden descent near boundary values. Figure V.5 shows the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.4**r== 0.3**\nFigure V.5: Penalty functions\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ncomparison of diﬀerent penalty functions. The least-square optimization problem for\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\npenalty functions is deﬁned below.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.5**\nFind x∗, a global minimizer for\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nN\nX\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.2**\nwhere κ is penalty constant and ∆xoﬀ,i is the oﬀset from feasible boundary,\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n∆xoﬀ,i =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n|xi − xmin|\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nif xi < xmin\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nif xmin ≤ xi ≤ xmax\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n(V.6)\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n|xi − xmax|\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nif xi > xmax\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.2**\nComposition Of Least-Square Optimization Problems\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nTwo or more least-square optimization problems can be composed as follows.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nConsider two least-square optimization problems P1 and P2 on optimization variable\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nx and objective functions PN\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\ni fi(x) and PM\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\nj gj(x) then the combined least-square\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\nFP (x) =\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\nN\nX\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\ni\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nfi(x) +\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\nM\nX\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nj\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\ngj(x)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.9**r== 0.1**\n(V.7)\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.4**r== 0.3**\nLOCALIZATION ALGORITHM\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nAn obvious and straightforward localization algorithm would be to solve for all\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\nunknown node locations simultaneously (Algorithm 1).\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.5**\nAlgorithm 1 Self localization algorithm\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\n1: Consider 3D coordinates of all unknown nodes in optimization\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nvariable.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n2: Construct and solve non-linear least-square optimization problem\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nwith objective function in equation (V.2).\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThis approach has some serious disadvantages. Convergence of the optimization\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nproblem strongly depends upon the initial guess given to the solver. A close-to-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\noptimum initial guess would converge to global optimum in considerably less time\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nwhile a bad initial guess for the same problem might end up in local optima. Initial\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nestimates for nodes can be computed by using an extension of the bounding box\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\ntechnique described in [3]. But due to the large size of the sensor network and\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nrelatively few randomly distributed anchor nodes, it is possible that we do not have\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ngood initial estimates for the whole network but only for the part close to the anchors.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.3**r== 0.2**\nIterative Incremental Localization Algorithm\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nAn iterative incremental approach wherein a part of the network near anchor\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nnodes is localized ﬁrst and then the node locations are propagated further seems\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nsuitable. The idea here is to iteratively select and localize a part of network (a\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nsub-system) for which a good initial estimate is available. At each iteration the\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\npart of the network selected for localization will grow, consisting of nodes that are\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nalready localized and few unknown neighboring nodes that have better estimates\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\nin the current iteration.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.4**r== 0.1**\nIn each iteration ranges that are believed to be echoes\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nare identiﬁed and removed from computation. The algorithm is presented below\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\n(Algorithm 2). Symbol x represents the 3D location vector of nodes, xest and xsol\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ndenotes estimated and localized node location vectors respectively. N denotes the set\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nof nodes in the network and η (described in section sub-system evaluation later in\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.3**\nAlgorithm 2 Incremental iterative self localization algorithm\n1: xest ← 0, xsol ← 0\n2: for run = 1 to runmax do\n3:\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nConfigure parameters, read distance matrix d, set sub-system\n˜N ← ∅\nrepeat\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.2**\ni ← x ∈ Bi ∀i ∈ N − ˜Nold based on neighbor polling\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nsatisfies goodness ∀i ∈ ˜N\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.3**r== 0.4**\n˜Nold ← ˜N\nEstimate bounding-box Bi ∀i ∈ N\nChoose xest\nSelect ˜N ⊆ N such that xest\nOptimize x for sub-system ˜N\nxest ← x\nfor all i ∈ ˜N do\nCompute ηi\n˜Nsol ← ∅\nif ηi acceptable then\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\ni\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\nxsol\ni ← xi\n˜Nsol ← ˜Nsol ∪ {i}\n**BLOCK**fs== 10.0**b== 0.2**t== 0.5**l== 0.2**r== 0.7**\n4:\n5:\n6:\n7:\n8:\n9:\n10:\n11:\n12:\n13:\n14:\n15:\n16:\n17:\n18:\n19:\n20: end for\n21: Output xsol\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nend if\nend for\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nuntil ˜Nsol − ˜Nold = ∅\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThere are two levels of looping in the algorithm. The outer loop starts with an\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nestimate, xest for the whole network. The ﬁrst run of the outer loop starts with a\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nrandom (or user given) estimate. Each run afterward starts with the ﬁnal estimate\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nof the previous run. The inner loop corresponds to the incremental selection and\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nlocalization of a sub-system, ˜N , that we will call an iteration. At each iteration, the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nselected sub-system will increase in size, more nodes will be localized with higher\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\naccuracy until there are no more nodes to be localized or no more nodes can be\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nlocalized (i.e. the necessary condition for localization does not hold). Later sections\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\ndescribe each step of the algorithm in detail.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\nSub-System Selection\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nEach node is represented by a bounded-box with lower and upper bounds (xlb, xub).\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nThe node coordinates can take any value in the closed interval [xlb xub]. Since anchor\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nnodes are known with high accuracy, their bounding-box is very small. Initially, the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nbounding-boxes for all unknown nodes can be set to the size of the ﬁeld and can be\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nupdated using range measurements ˆdij between node i and its neighbors j.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\nxlb,i = min\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\nj\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.5**r== 0.3**\n{(xlb,j − ˆdij · 1), xlb,i}\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nxub,i = min\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nj\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\n{(xub,j + ˆdij · 1), xub,i}\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nThe order in which bounding-box updates should be done is also important. Con-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nsidering the sensor network as a graph, it turns out that a variant of the topological\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nsorting (Algorithm 3) on the network graph will provide the required node ordering.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nThe main idea in this algorithm is that unknown nodes that are closer to known\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.6**\nAlgorithm 3 Topological sort\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\n1: Set known neighbor index, κ = ∞ for anchors and κ = 0 for all\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nother vertices\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.2**\nFind a vertex u with highest κ[u]\nOutput u\nDelete all edges e = (u, v) of u, increment κ[v] by 1\nDelete u from graph\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nFor node i that already has an estimate xest\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\ni\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nand conﬁdence value ηi, the bounds\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nare set as follows. Conﬁdence values for node location estimates are computed in the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nsub-system evaluation section which is described later.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nxlb,i = max{(xest\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\ni − ηi · 1), xlb,i}\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nxub,i = min{(xest\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\ni + ηi · 1), xub,i}\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nFor all other nodes a location estimate is picked from the bounding-box. The most\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nobvious way would be to pick the center of the box but a more heuristic method\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ninvolving bounding-box partitioning is used instead. The bounding-box of a node, if\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nlarger than a critical size, is partitioned into smaller boxes and neighbors are polled\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nfor the partition in which the node is most likely to be present. The center of that\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\npartition is assumed to be the estimated location for that node. A polling index Cp is\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\ncomputed for each partition p, which is essentially a weighted sum of distance errors\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nfor all neighbors j of node i.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nCp = X\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n\nxp − xest\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nj\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n\n − ˆdij\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n\n · ηj\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nj∈N eigh(i)\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nwhere xp is the center point of partition p. The center point of the partition with\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nminimum polling index is chosen as estimated location for that node.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nA part of the network is selected based the following notion of goodness of esti-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nmated node locations. An estimated location for node i is considered good if the node\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nhas at least three neighbors and it’s bounding-box satisﬁes two properties. First, its\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nvolume Vi is smaller than a critical volume V and second, its aspect ratio αi is greater\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nthan a critical ¯αadaptive. Aspect ratio αi is a measure of cubeness of the bounding-box.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nαi is expressed in terms of bounding-box volume Vi, space diagonal di and surface\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n√\n3 · Vi\n\nAi · di\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nαi =\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nNotice that for nodes with small bounding-boxes an estimate is acceptable even if it\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nhas smaller aspect ratio. For this reason the critical aspect ratio is made adaptive,\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nquadratically depending on the bounding-box volume. In the equation below ¯αmin\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nand ¯αmax are constants.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n¯αadaptive = ¯αmax −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n2\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n− 1\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n Vi\nV\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n· (¯αmax − ¯αmin)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n(VI.7)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nSub-System Localization\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe distance optimization problem for the selected sub-system is solved in multiple\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.3**\nstages. At each stage the solution is moved closer to the optima.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nWe solve the objective function in Equation (V.4) or the combination of objective\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nfunction in Equations (V.4) and (V.5) at each stage. The solution from the previous\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nstage is used as a starting point for the current stage. At the end of each stage\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nsome range measurements that are believed to have non-Gaussian errors (echoes) are\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nidentiﬁed and removed from the distance matrix. The diﬀerent stages of sub-system\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\nlocalization are described below.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\n• Stage I. At this stage echo ranges are identiﬁed and discarded based on the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nevaluation of the objective function in Equation (V.4) at the current optimizer\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.2**r== 0.7**\nxest.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\n• Stage II. At this stage of the optimization problem, Equation (V.4) is opti-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nmized in a ﬁxed number of iterations. The solver is stopped even if the optimizer\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nhas not converged. Let us visualize this stage as a 3D earth terrain optimization\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nproblem where x and y directions are optimization variables and altitude from\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nsea-level i.e. z is the optimization function. The global optimization in this\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nproblem is looking for the deepest trench on terrain. Optimizing for a ﬁxed\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nnumber of iterations can be visualized as going downwards a local trench but\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nnot going all the way down because that would take unlimited amount of time.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\n• Stage III. At the previous stage we did not consider bounded-value constraints\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\non the optimization variable. The variable might go out of the feasible region as\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nguided by the objective function. In this stage the combination of the optimiza-\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ntion problems, Equations (V.4) and (V.5) are optimized in a ﬁxed number of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\niterations. The objective function in Equation (V.5) ensures that the variable\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nwill fall within the feasible region. The reason for having stage II separate from\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nstage III is that there exist a possibility that the path to the global optimizer\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\ngoes through a region that might not be part of the feasible region. Figure VI.1\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nshows a case when such a path exists. The optimization variable from initial\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nFigure VI.1: An example optimization landscape. −2 ≤ x ≤ 1 and −1 ≤ y ≤ 2 is\nthe feasible region.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nguess X will optimize to A while the global optimum is at P . But if we use two\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nstage optimization the optimizer will converge to P .\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\n• Stage IV. This ﬁnal stage is similar to stage III except parameter p in Equation\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\n(V.4) is set to 1.0, i.e. no pruning of the distance matrix is done. It is expected\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nthat by the end of stage III we would have discarded most signiﬁcant echo\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nmeasurements.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nSub-System Evaluation\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\nAn important measure for any algorithm is its performance metric.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nIn case of\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nself localization the metric can be deﬁned as the closeness of the computed locations\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nto ground truth (actual node locations); we call this the localization error. Since in\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\npractical scenarios the ground truth is not known, it would be very helpful if there\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nwas an indirect performance metric that correlates with the localization error.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nThe quality of computed locations produced by the solver is evaluated using a\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nmeasure called the conﬁdence value. The conﬁdence value is an indicator of uncer-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\ntainty in node location around the current location estimate.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nThe algorithm to compute conﬁdence value is the following. Compute the ranges\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nbetween computed node locations for all node pairs for which measured range exist.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nNext compute the deviation of these computed ranges from measured ranges. Now\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nfor each node i we have a deviation vector ∆i whose elements are the deviations\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nof computed ranges from measured ranges for all its neighbors. A large value in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\n∆i indicate that either (1) the node location is incorrect or (2) the corresponding\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nrange measurement is incorrect. If the node location is incorrect then most of the\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nelements of ∆i should be large. If only a few range measurements are incorrect then\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nthe mean and the variance of ∆i should be small except for those incorrect range\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nmeasurements. Practically, all node locations are categorized based on mean µi and\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nstandard deviation σi in ∆i. The categorization of node locations is described below.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nConﬁdence value ηi is computed as σi exp(|µi|).\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nFigure VI.2: Categorization of computed locations based on computed errors. Thin\nlines show the error distribution while dash-dotted line indicates the mean and grey\nshaded region show the standard deviation. The dashed line is the zero mean.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\n1. If both µi and σi are close to zero then the node location is correct.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\n2. If µi is close to zero but σi is large then either the range deviations are spread\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\naround zero or few large deviations caused σi to be large. We say that the node\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nlocation may be aﬀected by echo. In this case we remove few data points with\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nlarge deviations and re-categorize the location based on a recomputed mean\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nand standard deviation.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\n3. If |µi| is large but σi is small then all elements of ∆i are large i.e. the node\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\n4. If both |µi| and σi are large then again location might be aﬀected by echo and\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nwe follow the same procedure as we followed in case 2 above.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\n5. If |µi| and σi are neither large nor small then location correctness is undecided.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.2**\nWe follow the same procedure here as we followed in case 2 and 4.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe threshold values for mean, µ and standard deviation, σ are set depending on the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nmean can be 0.05 m and 0.30 m respectively. The minimum and maximum standard\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ndeviation can be around 0.15 m and 0.8 m. The above values were used to evaluate the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nalgorithm on simulated topologies, described in next chapter. Figure VI.2 shows the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\ncharacteristics of each of the above category. Node locations categorized as incorrect\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nor as echoes are considered not localized.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.4**r== 0.3**\nIMPLEMENTATION AND RESULTS\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nWe implemented the proposed localization algorithm in MATLAB and ran it with\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nsimulated sensor network topologies and ranging data. The Levenberg-Marquardt\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nsolver was used for optimization in MATLAB.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nSimulated Data Generation\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nThe network topologies and ranging data for localization algorithm were generated\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nin a Java based simulator. The Java simulator generates node locations on sensor\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nnetwork ﬁeld. The simulator also generates sound source locations arranged on a\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\npath. The separation between two consecutive sound source locations is bounded.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nThis simulates the movement of a mobile acoustic beacon that has speciﬁed speed\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nand sound signal emitting rate. The simulator then generates ranges between sound\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nsources and nodes in its vicinity. Gaussian noise is added to the ranges according\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nto the experimental observation in Figure IV.4 (Eqn. IV.1). Non-Gaussian errors or\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nechoes and negative-echoes are also added to the ranging data according to the trend\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nobserved in experimental data. Experiments show that the probability of a range\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nbeing an echo grows with the range. The extent of echo, which is the deviation of\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\necho range from actual value, depends entirely on the environment.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThe simulator takes number of sensor nodes, number of anchor nodes, number of\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nacoustic beacon paths and number of sound sources on each path as input parame-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nters. Other adjustable parameters include sensor network ﬁeld size, maximum and\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nminimum separation of consecutive sound sources and maximum range of acoustic\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.2**\nFigure VII.1: XY view of a typical sensor network topology.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nA topology of 50 sensor node locations was generated randomly in a 100 × 100 ×\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n20 m ﬁeld with at least half of the nodes on ground level. 80 sound sources were\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\ngenerated on four random paths such that the separation between successive sound\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nsources is bounded (0 − 8 m). Also, the Z variation of the sources was limited to 2 m\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nto simulate a mobile beacon, which is moving on the ground in the sensor ﬁeld. This\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nconstraint can be relaxed if we consider mobile beacons to be on a UAV that can vary\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nits altitude. Maximum acoustic range was set to 30 m which is in accordance with\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nour experimental observations. Five sensor nodes were assumed to be known anchor\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nlocations. Two diﬀerent ranging data sets, one with echoes and another one without\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nechoes, were generated for same topology. Figure VII.1 shows XY view of a typical\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nranging errors are shown in Figure VII.2. Note that the standard deviation in ranging\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nerror increases with ranges. Number of non-Gaussian errors also increases with range.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nFigure VII.2: Simulated ranging errors\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nThe ranging data generated by the simulator closely emulates the behavior of actual\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nacoustic ranging.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nResults\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nIn the presence of ground truth, the performance of the algorithm can be evaluated\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nusing localization error which is the diﬀerence between computed locations and the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nground truth. Localization error for node i is,\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.4**r== 0.3**\np,i = (xi − ˜xi)2 + (yi − ˜yi)2 + (zi − ˜zi)2\nσ2\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n(VII.1)\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nwhere xi, yi and zi are the computed coordinates of node i, and ˜xi, ˜yi and ˜zi are the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ntrue location coordinates of the same node. Figure VII.3 and Figure VII.4 compare\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nthe computed node locations to their true locations in XY and XZ views for ranging\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ndata with echoes. Diﬀerent solid lines show multiple paths of the sound source. Solid\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\narrows in Figure VII.3 indicate the sensor nodes that have the highest localization\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nerror. Notice that all such nodes are very far from their nearest sound source.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nFigure VII.3: Comparison of computed node locations to their true values in XY\nplane for ranging data w/ echoes.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nFigures VII.5 and VII.6 show the histograms of 3D localization error without and\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nwith simulated echoes respectively. Table VII.1 summarizes the localization results.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nThe steeper distribution in case (a) indicates that number of nodes with lower\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.3**r== 0.5**\nUnlocalized sensors\nMean error (2D) [m]\nMax error (2D) [m]\nMean error (3D) [m]\nMax error (3D) [m]\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nRanges\nw/o echoes\n\n0.3041\n2.5436\n0.8962\n4.3252\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\nRanges w/\nechoes\n\n0.4871\n4.4795\n1.0664\n4.5119\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.4**r== 0.3**\nTable VII.1: Localization results\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nlocalization accuracy is more than that in case (b), where the number of nodes with\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nhigher localization errors in signiﬁcant. In other words, more nodes were localized\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\nwith better accuracy when we did not have echoes in ranging data.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n(b)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nFigure VII.4: Comparison of computed node locations to their true values in XZ\nplane for ranging data w/ echoes.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nFrom Figures VII.5 and VII.6 we can see that the computed locations of sound\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nsources are more accurate than that of sensor nodes. This high accuracy can be\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nattributed to the topological fact that sensor nodes are distributed around the sound\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nsources. For node localization application we are actually not concerned about the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ncomputed sound source locations. However, it is an important observation, that if we\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ndistribute the sound sources uniformly around sensor nodes then we can get higher\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\nlocalization accuracy for the sensors.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nThe localization evaluation scheme described in chapter VI was used to evaluate\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nand categorize localization results. Figure VII.7 shows number of sensor nodes in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\neach category. The ﬁgure shows localization error distribution for each category. As\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nexpected, most of the nodes under correct localization category have small localization\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nerror. The categorization algorithm was able to identify four incorrect localization\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nresults, two of them have location error less than 1.0 m. Few localization results\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\ncase (a)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nFigure VII.5: Histograms of 3D localization error for all sensor nodes and sound\nsources without echoes in ranging data.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\ncase (b)\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nFigure VII.6: Histograms of 3D localization error for all sensor nodes and sound\nsources with echoes in ranging data.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nwith high location errors were categorized as others. Nodes that were classiﬁed under\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.3**\naﬀected-by-echo category have 1.0 m or more location error.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nSub-meter localization accuracy is a requirement for correct operation of shooter-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nlocalization application. An important concern for node localization is the identiﬁca-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ntion of nodes with high location error. The algorithm localized approximately 75% of\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nthe nodes with sub-meter accuracy and less than 5 m maximum error for simulated\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nsensor networks. The categorization scheme identiﬁed approximately 50% of the node\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nlocations with 2 m or more location error. Based on localization on simulated data,\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nFigure VII.7: Categorization of sensor nodes\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nthe presented localization and categorization algorithms show a potential of close to\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.4**\nsub-meter accuracy in a complex urban environment.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nCONCLUSIONS\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThe presented sensor node localization technique has several contributions. The\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\naerial distribution of sensor nodes from a low-ﬂying UAV and acoustic ranging using\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\na mobile acoustic beacon mounted on UAV is a realistic deployment strategy with\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nmany advantages in complex, urban environments.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nThe method is passive since only the mobile beacon needs to emit acoustic signals.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nThis saves energy, size and cost on the sensor nodes and provides stealthy operation.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nFurthermore, the mobile beacon can emit much higher-energy sound than the sensor\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nnodes, thereby increasing the eﬀective range. The applied acoustic ranging method\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nhas the longest range for mote class devices, even when normalized by the emitted\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nsound energy. This is due to the signal processing algorithms implemented on the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nsensor board. The sound source installed on mobile UAV can have signiﬁcant height\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nvariation thereby facilitating 3D localization. The uniform speed of UAV with uniform\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nacoustic signal emitting rate provides consistent separation between sound source\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.7**\nlocations.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nThe iterative and incremental non-linear optimization technique introduced pro-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nvides an eﬀective way to deal with acoustic multipath eﬀects and works well for 3D\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nlocalization. There is little work in the wireless sensor network literature that ad-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ndresses these problems. The algorithm provides suﬃcient localization results even\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nwhen number of sound source locations is kept low for practicality. Pruned least-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nsquare optimization and other mathematical tools are formally deﬁned and used in\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nlocalization algorithm. The idea behind pruned least-square optimization is to elim-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\ninate the outlier data points in case the error distribution is not Gaussian.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nThe simulated ranging data used in the evaluation of the localization algorithms\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nare based on experimental observation and are realistic. The localization results\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nproduced by the algorithm shows mean localization error under 50 cm in XY and close\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nto 1 m in 3D. The results are satisfactory and assuring for centralized localization.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nAs a future direction in this work, the localization algorithm can be implemented\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nin Java and integrated with online ranging to provide online node localization in a\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nreal urban deployment.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\n[1] Mainwaring A., Polastre J., Szewczyk R., Culler D., and Anderson J. Wireless\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nSensor Networks for Habitat Monitoring. In ACM International Workshop on\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nWireless Sensor Networks and Applications (WSNA’02), 2002.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\n[2] Savvides A., Han C., and B. Strivastava M. Dynamic Fine-Grained Localization\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nin Ad-Hoc Networks of Sensors. Mobile Computing and Networking, pages 166–\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n179, 2001.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n[3] Savvides A., Park H., and Srivastava M. B. The Bits and Flops of the N-hop\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nMultilateration Primitive for Node Localization Problems. In WSNA, 2002.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\n[4] Savvides A., Park H., and Srivastava M. B. The N-hop Multilateration Primitive\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nfor Node Localization Problems.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nIn Mobile Networks and Applications, pages\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\n443–451, 2003.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n[5] Savarese C., Rabay J., and Langendoen K. Robust Positioning Algorithms for\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nDistributed Ad-Hoc Wireless Sensor Networks. In USENIX Technical Annual\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.6**\nConference, 2002.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\n[6] Culler D. and Mulder H. Smart Sensors to Network the World. Scientiﬁc Amer-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\nican, Volume 290, June 2004.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\n[7] Estrin D., Govindan R., and Heidemann J. Embedding the Internet: Introduc-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\ntion. Communications of the ACM, 43(5), 2000.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\n[8] Ganesan D., Krishnamachari B., Woo A., Culler D., Estrin D., and Wicker S.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nComplex Behavior at Scale: An Experimental Study of Low-Power Wireless\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nSensor Networks. Technical Report CSDTR 02-0013, UCLA, February 2002.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\n[9] Moore D., Leonard J., Rus D., and Teller S. Robust Distributed Network Local-\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nization with Noisy Range Measurements. In SenSys ’04: Proceedings of the 2nd\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nInternational Conference on Embedded Networked Sensor Systems, pages 50–61,\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n2004.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\n[10] Niculescu D. and Nath B. Ad Hoc Positioning System (APS) Using AOA. In\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.5**\nProceedings of INFOCOM, 2001.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\n[11] Whitehouse C. D. The Design of Calamari: An Ad-Hoc Localization System for\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nSensor Networks. Master’s thesis, University of California at Berkeley, 2002.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n[12] Simon G., Mar´oti M., L´edeczi A., Balogh G., Kusy B., N´adas A., Pap G., Sallai\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nJ., and Frampton K. Sensor Network-Based Countersniper System. In SenSys\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\n04, November 2004.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\n[13] Dana P. H. Global Positioning System Overview (last modiﬁed 5.1.2000). Avail-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nable from: www.colorado.edu/geography/gcraft/notes/gps/gps.html.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n[14] Hightower J. and Borriella G. Location Systems for Ubiquitous Computing.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.5**\nIEEE Computer, pages 57–66, 2001.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\n[15] Hightower J., Want R., and Borriello G. SpotON: An Indoor 3D Location Sensing\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nTechnology Based on RF Signal Strength. Technical report, 2000.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\n[16] Hill J. and Culler D. Mica: A Wireless Platform for Deeply Embedded Networks.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.5**\nIEEE Micro, 22(6):12–24, 2002.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\n[17] Nodedal J. and Wright S. J. Numerical Optimization, pages 250–275. Springer-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\nVerlag New York, Inc., 1999.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\n[18] Sallai J., Balogh G., Maroti M., Ledeczi A., and Kusy B. Acoustic Ranging in\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nResource-Constrained Sensor Networks. In International Conference on Wireless\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\n[19] Teller S. J., Chen J., and Balakrishnan H. IEEE Computer Graphics & Appli-\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\ncations: Projects In VR Pervasive Pose-Aware Applications and Infrastructure.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nIEEE Distributed Systems Online, 4(8), 2003.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\n[20] R¨omer K., Kasten O., and Mattern F. Middleware Challenges for Wireless\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nSensor Networks. SIGMOBILE Mobile Computing and Communications Review,\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n6, 2002.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\n[21] Doherty L., Pister K. S. J., and Ghaoui L. E. Convex Optimization Methods for\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\nSensor Node Position Estimation. In Proceedings of INFOCOM, 2001.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n[22] Girod L. and Estrin D. Robust Range Estimation Using Acoustic and Multi-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.6**\nmodal Sensing.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.4**r== 0.1**\nIn IEEE/RSJ International Conference on Intelligent Robots\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nand Systems (IROS), October 2001.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\n[23] R. Rabiner L. and Gold B. Theory and Application of Digital Signal Processing,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.3**\npages 715–716. Prentice-Hall, Englewood Cliﬀs, NJ, 1975.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n[24] B. Srivastava M., Muntz R. R., and Potkonjak M. Smart Kindergarten: Sensor-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nBased Wireless Networks for Smart Developmental Problem-Solving Enviro-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.5**\nments. In MOBICOM, 2001.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\n[25] Kwon Y. M., Mechitov K., Sundresh S., Kim W., and Agha G. Resilient Localiza-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ntion for Sensor Networks in Outdoor Environments. Technical Report UIUCDCS-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nR-2004-2449, Department of Computer Science, University of Illinois at Urbana\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nChampaign, 2004.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\n[26] Mar´oti M., Kus´y B., Simon G., and L´edeczi ´A. The Flooding Time Synchro-\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\nnization Protocol. In SenSys 04, November 2004.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\n[27] Mauve M., Widmer J., and Hartenstein H. A Survey on Position-Based Routing\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\n[28] Bulusu N., Heidemann J., and Estrin D. GPS-less Low Cost Outdoor Localiza-\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\ntion for Very Small Devices. Technical report, Computer Science Department,\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nUniversity of Southern California, April 2000.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\n[29] Levanon N. and Mozeson E. Radar Signals. Hoboken, NJ John Wiley & Sons,\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nInc., 2004.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\n[30] Priyantha N., Chakraborty A., and Balakrishnan H. The Cricket Location-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nSupport System. In MobiCom’00: Proceedings of the 6th Annual ACM Interna-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\ntional Conference on Mobile Computing and Networking, 2000.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n[31] Priyantha N., Miu A., Balakrishnan H., and Teller S. The Cricket Compass\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nfor Context-Aware Mobile Applications. In MobiCom’01: Proceedings of the 7th\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nAnnual International Conference on Mobile Computing and Networking, 2001.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\n[32] Capkun S., Hamdi M., and Hubaux J. P. GPS-Free Positioning in Mobile Ad-Hoc\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.4**\nNetworks. In Proceedings of HICSS, 2001.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n[33] Meguerdichian S., Koushanfar F., Qu G., and Potkonjak M. Exposure in Wireless\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nAd-Hoc Sensor Networks. In MOBICOM, 2001.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\n[34] He T., Huang C., Blum B., Stankovic J., and Abdelzaher T. Range-Free Local-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.3**\nization Schemes in Large Scale Sensor Networks, 2003.",
         "extracted",
         "Resilient Localization for Sensor Networks in Outdoor Environments;Mobile-assisted localization in wireless sensor networks;Sensor network-based countersniper system;Robust distributed network localization with noisy range measurements;The flooding time synchronization protocol;Localization of wireless sensor networks with a mobile beacon;Smart sensors to network the world.;Range-free localization schemes for large scale sensor networks;The n-Hop Multilateration Primitive for Node Localization Problems;Ad hoc positioning system (APS) using AOA;Mica: A Wireless Platform for Deeply Embedded Networks;Middleware challenges for wireless sensor networks;The bits and flops of the n-hop multilateration primitive for node localization problems;Wireless sensor networks for habitat monitoring;A survey on position-based routing in mobile ad hoc networks;Robust range estimation using acoustic and multimodal sensing;Location Systems for Ubiquitous Computing;Smart kindergarten: sensor-based wireless networks for smart developmental problem-solving environments;Exposure in wireless Ad-Hoc sensor networks;Dynamic fine-grained localization in Ad-Hoc networks of sensors;The cricket compass for context-aware mobile applications;GPS-free Positioning in Mobile Ad Hoc Networks;GPS-less low-cost outdoor localization for very small devices;Embedding the Internet: introduction;Acoustic Ranging in Resource-Constrained Sensor Networks;IEEE Computer Graphics & Applications: Projects In VR Pervasive Pose-Aware Applications and Infrastructure;VR Pervasive Pose-Aware Applications and Infrastructure. IEEE Distributed Systems Online;Range-free localization schemes in large scale sensor network;This copyright notice must be included in the reproduced paper. USENIX acknowledges all trademarks herein. Robust Positioning Algorithms for Distributed Ad-Hoc Wireless Sensor Networks;The Design of Calamari : an Ad-hoc Localization System for Sensor Networks;Complex Behavior at Scale: An Experimental Study of Low-Power Wireless Sensor Networks;Convex Optimization Methods for Sensor Node Position Estimation;SpotON: An Indoor 3D Location Sensing Technology Based on RF Signal Strength;The Cricket location-support system;Global Positioning System Overview (last modified 5.1.2000);Numerical Optimization, pages 250–275;Theory and application of digital signal processing;If both | µ i | and σ i are large then again location might be affected by echo and we follow the same procedure as in case 2 above;If µ i is close to zero but σ i is large then either the range deviations are spread around zero or few large deviations caused σ i to be large;If | µ i | is large but σ i is small then all elements of ∆ i are large i.e. the node location is deﬁnitely incorrect",
         "Sensor node localization using mobile acoustic beacons",
         "None"
        ],
        [
         "26",
         "00236516f5be65f61e6fffa20a9ea4ff630b9a7e",
         "Researchers in the area of information security have mainly been concerned with tools, techniques and policies that firms can use to protect themselves against security breaches. However, information security is as much about security software as it is about secure software. Software is not secure when it has defects or flaws which can be exploited by hackers to cause attacks such as unauthorized intrusion or denial of service attacks. Any public announcement makes about a software defect is termed as 'vulnerability disclosure'. Although research in software economics have studied firms' incentive to improve overall quality, there have been no studies to show that software vendors have an incentive to invest in building more secure software. In this paper, we use the event study methodology to examine software vendors' incentives to build more secure software. We collect data from leading national newspapers and industry sources like CERT by searching for reports on published software vulnerabilities. We show that vulnerability disclosures lead to a negative and significant change in market value for a software vendor. On average, a vendor loses around 0.6% value in stock price when a vulnerability is reported. This is equivalent to a loss in market capitalization values of $0.86 billion per vulnerability announcement. To provide further insight, we use the information content of the disclosure announcement to classify vulnerabilities into various types. We find that the change in stock price is more negative if the vendor fails to provide a patch at the time of disclosure. Moreover, vulnerabilities which cause a confidentiality related breach cause a greater decline in the market value for a vendor than the vulnerabilities which cause non-confidentiality related breaches. Also, more severe flaws have a significantly greater impact than flaws with low or moderate severity. Finally, we find that the markets do not punish a software vendor more severely if a third party discovers a flaw in its product than if the vendor itself discovers the flaw. Our analysis provides many interesting implications for software vendors as well as policy makers.",
         "Rahul Telang,S. Wattal",
         "http://archive.nyu.edu/bitstream/2451/14996/2/Infosec_Book%2bWattal%2bTelang.pdf",
         "\n**BLOCK**fs== 16.0**b== 0.7**t== 0.2**l== 0.2**r== 0.2**\nImpact of Software Vulnerability Announcements on the\nMarket Value of Software Vendors – an Empirical\nInvestigation1\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nRahul Telang, Sunil Wattal\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\n{rtelang, swattal}@andrew.cmu.edu\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nCarnegie Mellon University\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nFebruary 2005\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n1  The  authors  would  like  to  thank  Ashish  Arora,  Ramayya  Krishnan,  Sandra  Slaughter,  participants  at  the\nSecurity Working Group Meeting at Heinz School of Public Policy, and Workshop on Information Systems\nand Economics (WISE 2004) for helpful comments and suggestions.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.5**r== 0.4**\nABSTRACT\n**BLOCK**fs== 11.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nResearchers  in  the  area  of  information  security  have  mainly  been  concerned  with  tools,\n**BLOCK**fs== 11.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ntechniques  and  policies  that  firms  can  use  to  protect  themselves  against  security  breaches.\n**BLOCK**fs== 11.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nHowever,  information  security  is  as  much  about  security  software  as  it  is  about  secure  software.\n**BLOCK**fs== 11.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nSoftware  is  not  secure  when  it  has  defects  or  flaws  which  can  be  exploited  by  hackers  to  cause\n**BLOCK**fs== 11.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nattacks such as unauthorized intrusion or denial of service attacks. Any public announcement about\n**BLOCK**fs== 11.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\na software defect is termed as ‘vulnerability disclosure’.\n**BLOCK**fs== 11.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nAlthough  research  in  software  economics  have  studied  firms’  incentive  to  improve  overall\n**BLOCK**fs== 11.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nquality,  there  have  been  no  studies  to  show  that  software  vendors  have  an  incentive  to  invest  in\n**BLOCK**fs== 11.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nbuilding more secure software. In this paper, we use the event study methodology to examine the\n**BLOCK**fs== 11.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nrole that financial markets play in determining software vendors’ incentives to build more secure\n**BLOCK**fs== 11.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nsoftware.  We  collect  data  from  leading  national  newspapers  and  industry  sources  like  CERT  by\n**BLOCK**fs== 11.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nsearching for reports on published software vulnerabilities. We show that vulnerability disclosures\n**BLOCK**fs== 11.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nlead  to  a  negative  and  significant  change  in  market  value  for  a  software  vendor.  On  average,  a\n**BLOCK**fs== 11.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nvendor loses around 0.6% value in stock price when a vulnerability is reported. This is equivalent\n**BLOCK**fs== 11.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nto  a  loss  in  market  capitalization  values  of  $0.86  billion  per  vulnerability  announcement.  To\n**BLOCK**fs== 11.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nprovide further insight, we use the information content of the disclosure announcement to classify\n**BLOCK**fs== 11.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nvulnerabilities  into  various  types.  We  find  that  the  change  in  stock  price  is  more  negative  if  the\n**BLOCK**fs== 11.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nvendor fails to provide a patch at the time of disclosure. Moreover, vulnerabilities which cause a\n**BLOCK**fs== 11.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nconfidentiality  related  breach  cause  a  greater  decline  in  the  market  value  for  a  vendor  than  the\n**BLOCK**fs== 11.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nvulnerabilities which cause non-confidentiality related breaches.  Also, more  severe flaws have  a\n**BLOCK**fs== 11.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nsignificantly  greater  impact  than  flaws  with  low  or  moderate  severity.  Finally,  we  find  that  the\n**BLOCK**fs== 11.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nmarkets  do  not  punish  a  software  vendor  more  severely  if  a  third  party  discovers  a  flaw  in  its\n**BLOCK**fs== 11.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nproduct  than  if  the  vendor  itself  discovers  the  flaw.  Our  analysis  provides  many  interesting\n**BLOCK**fs== 11.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nimplications for software vendors as well as policy makers.\n**BLOCK**fs== 11.0**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\nKeyword: information security, software vulnerability, quality, event study, disclosure policy\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.7**\n1. Introduction\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nMany believe that software vendors typically follow the policy of ‘sell today and fix it\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ntomorrow’;  or  ‘I’d  rather  have  it  wrong  than  have  it  late’  (Paulk  et  al  1994;  Arora,\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nCaulkins,  Telang  2004)  for  launching  software  products  in  the  market.  This  policy,\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\ndictated by the need to launch products quickly before competitors, seemed to work in the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\npast because software errors which escape detection during pre-launch testing appear very\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ninfrequently  in  normal  operations  (once  every  ‘5000’  years,  as  per  Adams,  1980).\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nHowever,  Adams’  ‘5000-year  error’  theory  might  not  hold  in  the  internet  age  because\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nhundreds  (if  not  thousands)  of  people  are  looking  for  flaws  in  other  vendors  software\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nproducts,  drastically  increasing  the  chances  that  a  flaw  will  be  exposed.  Various  people\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nsuch  as  hackers,  independent  security  firms  and  academic  researchers  are  interested  in\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nfinding  flaws  in  other  vendors’  software  for  different  reasons.  Not  only  are  security\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nsoftware products such as firewalls at risk, but software like operating systems, enterprise\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nsoftware and database software also contain flaws which can be exploited to create security\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nrelated attacks.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nThe  Organization  of  Internet  Safety  (OIS)  (www.oisafety.org)  defines  security\nvulnerability as2: “security vulnerability is a flaw within a software system that can cause it\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nto work contrary to its documented design and could be exploited to cause the system to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nviolate  its  documented  security  policy”.  Software  vulnerabilities  have  widespread  impact\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nand can potentially cause billions of dollars in downtime and disruptions to firms (In this\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\npaper, the word ‘firms’ refers to companies which use software products; ‘vendors’ refers\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nto  companies  which  develop  the  software  products).  A  study  by  NIST  in  2002  estimates\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nthe cost of faulty software at $60 bn per year. Incidents like the Code Red virus (in 2001)\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nand  the  Melissa  virus  (in  1999)  occurred  when  hackers  exploited  flaws  in  software.  The\ndamage  due  to  Code  Red  was  estimated  at  $2.1  bn  and  due  to  Melissa  at  $1.1bn3.  The\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nGartner  Group  estimates  that  the  system  downtime  caused  by  security  vulnerabilities\nwould triple from 5% of the total downtime in 2004 to 15% of the total downtime in 2008.4\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nIn  2003,  Computer  Emergency  Response  Team  (CERT)  reported  around  250,000  self\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.2**\n2 In this paper, we use the terms ‘software vulnerability’, ‘security vulnerability’, ‘bug’, ‘flaw’\ninterchangeably. Any other type of vulnerability such as a non-security related vulnerability explicit\nmentioned by name.\n3 Source : www.cisco.com/warp/public/cc/so/neso/sqso/roi1_wp.pdf\n4 http://www.tekrati.com/T2/Analyst_Research/ResearchAnnouncementsDetails.asp?Newsid=3608\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nreported incidences of system breaches, most of which exploit vulnerabilities in software\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ncode  (Applewhite  2004).  Microsoft’s  $200m  campaign  for  .NET  was  marred  by  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ndiscovery of a security flaw in Visual C++ .NET barely a month after Microsoft Chairman\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nBill  Gates  directed  employees  to  focus  on  building  more  secure  software  (dubbed  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\n‘Trustworthy Computing’ initiative). Moreover, vulnerability disclosure is finding its way\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ninto  firms’  strategy  toolkits  as  it  evident  from  a  WSJ  report  in  Feb  2004  that  software\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nvendors are spending time and effort in discovering flaws in their rivals’ software products\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nin  order  to  influence  the  rivals’  stock  prices.  For  example,  security  software  vendor  IDS\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nreleased a vulnerability alert on rival Checkpoint’s firewall software on the day Checkpoint\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nusually  holds  its  annual  US  investor  conference.  Some  examples  of  vulnerability\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nannouncements reported in popular press are:\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\n•  News.com(04/25/2000)  “A  computer  security\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\nfirm  has  discovered  a  serious\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nvulnerability  in  Red  Hat’s  newest  version  of  Linux  that  could  let  attackers  destroy  or\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\ndeface a Web site  - or possibly even take over the machine itself……..”\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n•  WSJ(02/11/2004) “Microsoft Corp. warned customers about serious security problems\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nwith  its  Windows  software  that  let  hackers  quietly  break  into  their  computers  to  steal\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.3**\nfiles, delete data or eavesdrop on sensitive information……..”\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nInspite  of  all  these  concerns  about  software  vulnerabilities,  not  much  has  been\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nmentioned  in  literature  about  the  incentive  of  software  vendors  to  invest  in  defect-free\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nsoftware. Literature on software risks fails to include any measure for security related risks\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\n(Wallace, Keil and Rai, 2004; Barki, Rivard and Talbot, 1993). The closest literature to the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\ntopic of security vulnerabilities is that on software quality (Basili and Musa, 1991; Harter,\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nKrishnan  and  Slaughter,  2000).  However,  in  traditional  literature,  quality  is  measured  in\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nterms  of  reliability  and  integrity  of  the  source  code  –  which  essentially  tests  software\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nagainst specified streams of input from users. However, in today’s internet age, software\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ndesigners  must  not  only  think  of  users,  but  also  malicious  adversaries  (Devanbu  and\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nStubblebine,  2000).  Some  quality  models  such  as  ISO9126  fail  to  include  computer\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsecurity  (Pfleegar  1997).  Therefore  software  which  has  been  certified  as  high  quality,\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nbased  on  existing  definitions  of  software  quality,  can  have  many  security  flaws.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nResearchers in computer science are working on better integration of the two disciplines of\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nsoftware  quality  and  software  security  while  designing  software  (Carr,  Tynan  and  Davis\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n1991, Wang and Wang, 2003; McGraw 2004).\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nSo far, there has only been anecdotal evidence that software vulnerabilities are causing\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nvendors to lose market value. For example, the Wall Street Journal (11/09/2004) reported\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nthat Microsoft’s Internet Explorer (IE) is losing market share in the web browser market to\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ncompetitors like Mozilla’s Firefox, due to numerous flaws discovered in IE. For example,\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nnearly  8  million  people  have  downloaded  the  Firefox  browser  between  September-\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nNovember 2004. While the damage that firms suffer as a result of security breaches can be\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nmeasured  in  terms  of  downtime  and  maintenance  activity,  the  cost  implication  of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nvulnerability disclosures on the software vendors is neither clear nor well studied.  This is\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nthe major goal of this paper. Using an event study methodology, we estimate how the stock\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nprice of the software vendors is affected when vulnerability information in their products is\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\npublished in popular press.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nPrior  literature  on  product  defects  (Jarrell  and  Peltzman  1985)  predicts  that  product\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nrecall announcements in drug and auto industries are associated with loss in market value\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nof a firm. Davidson and Worrell (1992) confirm the negative impact of product defects on\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nstock prices in non-auto industries and also analyze the impact of different types of recall\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n(e.g.  total  product  recall  versus  a  recall  for  repair;  government  ordered  recall  versus\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nvoluntary recall). However the results of these prior studies on product defects cannot be\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ndirectly  applied  to  the  software  industry  because  of  the  following  characteristics  of\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nsoftware products: One, software products generally come with a click-wrap agreement (or\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nEULA  –  End  User  License  Agreement)  which  limits  the  vendors’  liability.  Two,  the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\ngeneral philosophy held by software vendors, software customers and the US courts is that\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nsoftware is a uniquely complex product that will probably have some defects (Cusumano\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\n2004).  Therefore  it  is  not  clear  whether  markets  will  react  adversely  to  the  news  of  a\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsoftware  vulnerability;  because  over  the  long  run,  markets  will  anticipate  the  effect  of  a\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nvulnerability announcements on cash flows of software vendors, so the impact of a specific\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nannouncement  might  not  be  significant.  Three, software vulnerability announcements are\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ngenerally  accompanied  with  a  remedial  patch  which  potentially  protects  customers  from\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nmalicious  exploits.  Hence,  the  market  may  not  behave  adversely  towards  a  vendor  if  it\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nreleases  a  patch  along-with  the  vulnerability  announcement.    Finally,  vulnerability\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nannouncements  are  directly  related  to  the  installed  base  of  a  software  product.  Popular\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nsoftware like those from Microsoft are constantly subject to malicious and non-malicious\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nattacks  and  as  such  have  a  greater  proportion  of  flaws  reported  in  them  as  compared  to\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nsoftware by Apple, where the user base is comparatively smaller. Therefore, the presence\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nof  vulnerabilities  may  not  always  signal  a  lower  quality  product;  it  may  in-fact  signal\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nsuperiority  over  competition.  E.g.  John  Thomson,  CEO  of  Symantec,  predicts  that  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nflaws  in  Linux  will  likely  increase  as  the  installed  base  increases.  In  view  of  these\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\narguments, it would be interesting to understand whether and how the market responds to\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nvulnerability disclosures in software products.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nMotivated  by  these  observations,  in  this  research  we  try  to  quantify  the  losses  that\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nsoftware  vendors  bear  when  a  vulnerability  is  disclosed  in  their  product.  The  main\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nquestions that we seek to answer are:\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\n1.  What  is  the  impact  of  vulnerability  disclosures  on  the  market  value  of  a  software\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nvendor?\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n2.  How do the characteristics of the vulnerability impact this change in market value?\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nOur  research  has  important  policy  implications  in  terms  of  understanding  vendors’\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nincentive to improve pre- and post-launch quality of their software products.  While there\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nis lot of hype surrounding the poor quality dished out by software vendors, their incentives\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nto provide more secure software is still unclear. If we indeed find that the market is willing\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nto  punish  vulnerability  announcements  because  it  perceives  these  announcements  as  a\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nsignal of poor quality software which either increase vendor costs to fix them or erode their\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nmarket share in the long run, then our research provides a direct evidence of incentives to\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nprovide  more  secure  software.  Vulnerabilities  are  disclosed  by  vendors  or  by  third\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nparty/competitors  with  or  without  a  patch.  Since  we  measure  if  there  are  significant\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\ndifferences in market reaction to such disclosures, our paper provides policy guidelines to\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nvendors about whether and how they may disclose the information themselves. Moreover,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nthis  also  provides  evidence  on  what  is  the  value  of  patches  when  vulnerabilities  are\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ndisclosed.  Finally,  we  also  examine  how  vulnerability  characteristics,  effect  of  9/11  etc\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\naffect the market reaction.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nUsing  an  event-study  approach,  we  collect  data  on  about  146  vulnerability  disclosure\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nannouncements over the period of over 5 years for 18 publicly traded vendors. Our results\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nconfirm  that  vulnerability  disclosure  adversely  and  significantly  affects  the  stock\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nperformance  of  a  software  vendor.  We  show  that,  on  average,  a  software  vendor  loses\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\naround  0.63%  of  market  value  on  the  day  of  the  vulnerability  announcement.  This\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ntranslates  to  a  dollar  amount  of  $0.86  bn  loss  in  market  value.  We  also  find  that\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nvulnerabilities  where  the  vendor  fails  to  provide  a  patch  at  the  time  of  disclosure  yield\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nmore negative returns than when the vendor discloses a patch. On average, a vendor loses\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n0.8% more in market value when a patch is not available. This provides evidence as to why\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nvendors are trying to push for legalizing the limited disclosure norms. We also show that\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nvulnerabilities  which  create  confidentiality  related  breaches  result  in  a  greater  loss  than\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nvulnerabilities  which  create  non-confidentiality  related  breaches.  For  example,  vendors\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nlose  0.75%  less  in  market  value  when  the  vulnerability  can  lead  to  a  non-confidentiality\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nrelated breach than when it can lead to a confidentiality related breach. We also find that\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nmore severe software vulnerabilities result in a greater loss in market value than less severe\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe rest of the paper is organized as follows. In the next section, section 2, we provide\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\na literature review. We develop our hypotheses in Section 3. In Section 4, we discuss the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nmethodology of data collection and also describe the event study methodology. In Section\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n5, we present our results using multivariate regression analysis and test various hypothesis\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nrelated  to  how  the  vulnerability  characteristics  effect  the  change  in  stock  prices.  Finally,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nwe present the concluding remarks in Section 6.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.7**\n2. Literature Review\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nMost  prior  research  on  information  security  discusses  the  economics  of  such\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\ninvestments from a customer perspective, rather than from a software vendor perspective\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n(Anderson  2001).  Gordon  and  Loeb  (2002)  show  that  firms  should  make  investments  in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ninformation  security  far  less  than  the  expected  loss  from  a  security  breach.  Gordon  et  al\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n(2002),  Gal-Or  and  Ghose  (2003)  discuss  the  economics  of  information  sharing  among\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nfirms  on  security  related  issues.  Prior  event  study  analyses  on  information  security  have\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nfocused on the change in market value of firms whose systems are breached (Cavusoglu et\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nal (2004) and Kannan, Rees and Sridhar (2004)). These studies show that announcements\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nof  a  security  breach  negatively  impact  the CAR  (Cumulative  Abnormal  Return)  of  firms\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nwhose information systems have been breached. Campbell et al (2003) conduct a similar\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nevent  study  and  find  that  only  the  impact  of  confidentiality  related  security  breaches  is\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nnegative and significant; the impact of non-confidentiality related security breaches is not\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nsignificantly  different  from  zero.    Hovav  and  D’Arcy  (2003)  show  similar  results  by\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nfinding  that  Denial  of  Service  (DoS)  type  attacks  are  not  associated  with  any  significant\nloss in value for firms5.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nDisclosure of vulnerabilities has been one contentious area. Typically, a major portion\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nof the vulnerabilities is reported by benign independent security analysts (ISA). Since no\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nlegal  guidelines  exists  which  dictate  how  vulnerabilities  should  be  handled  by  the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\ndiscoverer,  some ISAs report the vulnerability to the vendor and give it sufficient time to\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\ncome  up  with  a  patch  (The  OIS  recommends  a  time  period  of  30  days  to  be  given  to\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nvendors  to  come  up  with  a  patch).  However,  some  other  ISAs  follow  the  policy  of  full\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\ndisclosure, i.e. they immediately post the vulnerability to a public listing like Bugtraq. One\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nmajor goal of full disclosure is to eventually force vendors to come up with more secure\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nsoftware.  The  debate  on  disclosure  policies  can  be  summarized  by  the  following  quotes\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n(Applewhite  2000):  Jennifer  Granick  of  Stanford  University  defends  full  disclosure  as\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n“You  want  a  free  flow  of  information,  just  like  in  other  scientific  fields,  ideally  without\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nbetraying important interests.” Scott Blake of OIS opposes full disclosure as, “As soon as\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nyou  tell  one  person,  you  have  tipped  the  balance  and  people  will  begin  producing\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nexploits.”  Some  recent  academic  work  is  examining  this  issue  more  formally.  Arora,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nTelang and Xu (2004) study the optimal timing of vulnerability disclosure and show how\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\ndisclosure can force vendors to release patches quickly. Kannan and Telang (2004) explore\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nthe welfare implications of a market mechanism for software vulnerabilities and report that\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\na market based mechanism for software vulnerabilities always underperforms a CERT-type\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nmechanism.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nHowever  none  of  these  studies  measures  the  impact  of  disclosure  on  vendor’s  market\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nvalue or profitability. While one major goal of full disclosure is to eventually force vendors\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nto develop secure software, empirically, there is no evidence that suggests that disclosure\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nindeed  creates  such  incentives.  Our  paper  provides  an  understanding  of  whether  such\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ndisclosures create incentives for the vendors to produce secure software in the first place.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\n5 DoS attacks are classified as non-confidential in the Campbell et al (2001) study.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nOur  methodology  follows  closely  from  prior  event  study  analysis.  Campbell  et  al\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n(1997)  present  a  useful  summary  of  the  event  study  analysis  highlighting  the  history  as\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nwell  as  the  commonly  followed  methodologies.  Event  study  methodologies  are  well\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\naccepted for studying the implications of public announcements on stock prices. Hendricks\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nand  Singhal  (1996)  study  the  impact  of  quality  award  winning  announcements  on  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nmarket  value  of  firms  and  observe  positive  abnormal  returns  generated  by  winning  a\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nquality  award.  They  further  note  that  awards  given  by  independent  organizations  and\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nannouncements by small firms are more likely to have a significant impact on the firms’\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nmarket  value.  In  the  field  of  Information  Systems,  Subramani  and  Walden  (2001)  used\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nevent study analyses to show that e-commerce announcements lead to significant increases\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nin the stock price of firms. Chatterjee, Richardson and Zmud (2001) examine the influence\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nof  newly  announced  CIO  positions  on  shareholder  wealth.  Im,  Dow  and  Grover  (2001)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nexamine  the  changes  in  market  value  of  a  firm  in  response  to  IT  investment\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nannouncements. Bharadwaj and Keil (2003) investigate the impact of IT failures on firms’\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nmarket value.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe  main  contribution  of  this  research  is  that  this  is  one  of  the  first  comprehensive\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nstudies,  to  our  knowledge,  that  has  tried  to  measure  the  impact  of  vulnerabilities  on\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nsoftware  vendors.  Thus,  we  extend  prior  literature  on  product  defects  and  confirm  that\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nsoftware  vendors  too  suffer  a  loss  in  market  value  when  a  flaw  is  discovered  in  their\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nproduct.  This  is  inspite  of  the  fact  that  software  vulnerabilities  are  prevalent  among\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nsoftware of almost all major vendors and that vendors face no legal liability if clients suffer\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nlosses  due  to  the  software  flaw.  As  we  noted  in  the  introduction  that  this  has  important\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nimplications in terms of vendors making investment in software quality as well as policy\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nand legal issues which govern vulnerability disclosures.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.7**\n3. Hypotheses\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nMuch of the prior literature on product defects (Jarrell and Peltzman (1985), Davidson\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nand Worrell (1992)) shows that generally defective product and recall announcements are\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nassociated  with  loss  in  market  value  of  a  firm.  Banker  and  Slaughter  (1998)  find  that\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nunplanned and critical maintenance activities increase software maintenance costs.  Thus,\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nfixing  bugs  entails  cost  to  firms.  For  example,  the  security  fixes  may  cost  about  $2000-\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n$9000 when done during testing phase. However, they may cost more than 4-8 times when\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nfixed after the application has been shipped.6 Slaughter et al (1998) and Westland (2003)\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nsuggest  that  software  defects  are  harder  and  costlier  to  fix  if  discovered  later  in  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nsoftware  development  cycle  (e.g.  when  the  product  has  been  shipped  to  the  customer).\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nMoreover, security breach announcements by the user organizations have been known to\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nhave a negative impact on the share value for firms. Cavusoglu et al (2004) show that the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nmarket  capitalization  values  of  firms  decreases,  on  average,  by  $2.1  billion  within  two\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ndays  of  a  security  breach.  Cyber  insurance  firm  J.S.  Wurzler  charges  an  additional\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\npremium  to  firms  for  using  Windows  NT  due  to  the  number  of  security  breaches  in  the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nsoftware  (Gordon,  Loeb  and  Sohail,  2003).  Clearly,  poor  security  costs  the  users  of  the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nsoftware.  Thus, the cost to the vendor can be written as\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nCost of vulnerability disclosure to vendor = Cost of patching the vulnerability + λ * (Cost\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nto the users of the software due to exploitation, and/or cost to patch the system)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nHere,  λ  is  the  internalization  factor.  That  is  the  user  loss  that  is  internalized  by  the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nvendor  due  to  lost  sale  or  reputation  loss  (or  liability  if  imposed  in  future).  Clearly,  λ\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\ndepends on how willing users are to “punish” the vendor, how competitive the market is\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nand the characteristics of vulnerability (Arora, Telang and Xu 2004). For example, this is a\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\ntypical customer reaction.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n\"We  are  extremely  concerned  by  the  high  amount  of  vulnerabilities  and  patches  from\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nMicrosoft. This goes against the credibility of what they have been saying,\"\nMichael Kamens, global security director at Thermo Electron Corp.7\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nGiven the fact the competitors seem to be using vulnerability disclosure as a strategic way\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nto  undermine  the  rivals’  reputations,  it  seems  that  vulnerability  disclosure  signals  a\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\npotential loss in future cash flows for a software vendor owing to customer dissatisfaction\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nbecause customers suffer a loss if their systems get breached. It also signals an increase in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nproduct  related  costs  due  to  the  time  and  effort  that  the  vendor  spends  in  developing  a\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\npatch or a fix for the flaw. Therefore, we hypothesize that\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nH1: A software vendor suffers a loss in market value when a security related vulnerability\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nis announced in its products.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nOur second hypothesis pertains to whether the software vendor releases a patch for the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nproduct  at  the  time  of  the  vulnerability  announcement.    As  per  the  popular  convention\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nfollowed in the vulnerabilities market, vendors are given some time to work on a patch for\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nthe vulnerability before it is made public. Vendors may also provide a workaround (such as\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\ndisabling features of the software) when a vulnerability is disclosed and choose to address\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nthe  vulnerability  in  a  future  upgrade.  Presence  of  the  patch  is  also  likely  to  reduce\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ncustomers’  loss  if  they  apply  the  patch.  Since  presence  of  patch  also  reflects  vendor’s\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ncommitment  to  its  customers  we  expect  that  vulnerabilities  disclosed  with  a  patch  will\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\ncompensate, to an extent, the negative signal due to vulnerability disclosure. Vendors are\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nalso  pushing  for  limited  disclosures  so  that  they  can  release  the  remedial  patch  in  time.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nThis also suggests that patches play a critical role. Therefore our second hypothesis is:\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nH2: CAR (Cumulative Abnormal Return) [negative] of a stock is greater for vulnerabilities\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nwhere  the  software  vendor  does  not  release  a  patch  at  the  time  of  the  vulnerability\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\ndisclosure.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe  damages  suffered  by  a  software  vendor’s  customers  due  to  a  vulnerability  in  the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nvendor’s software  depend on the  type of security breach that the vulnerability  facilitates.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nCampbell  et  at  (2003)  classify  the  security  breaches  as  confidentiality  related  and  non-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nconfidentiality  related.  Confidentiality  related  breaches  involve  attacks  where  an  intruder\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ncan  gain  access  into  a  system  and  can  steal  sensitive  information.  Non-confidentiality\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nrelated breaches include attacks like denial of service (DoS) attacks where the most likely\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nscenario  is  a  disruption  and/or  a  downtime.  Campbell  et  al  (2003)  further  show  that  the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nloss  in  market  value  is  more  for  confidentiality  related  breaches  than  for  non-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nconfidentiality related breaches. Hovav and D’Arcy (2003) show that DoS attacks are not\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nassociated with any significant loss in market value for a firm. Therefore we would expect\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nthat  the  vendor  loses  more  market  value  if  the  vulnerability  in  its  software  causes  a\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nconfidentiality  related  breach.  The  intuition  is  that  the  negative  reaction  due  to  the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nvulnerability  is  more  if  the  customers  can  potentially  suffer  greater  losses  due  to  the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nvulnerability. Therefore, our next hypothesis is:\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nH3:  CAR  is  greater  for  a  vulnerability  which  can  potentially  cause  a  breach  in\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nconfidentiality as compared to non-confidentiality related breaches.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nThe  impact  of  a  software  flaw  on  a  vendor  also  depends  on  how  severe  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nvulnerability is. Davidson and Worrell (1992) conduct an event study with product defect\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nannouncements  in  the  tire  industry  and  showed  that  the  impact  of  severe  flaws  (which\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ninvolve  a  recall)  is  more  than  that  of  less  severe  flaws  (which  involve  repairs  but  nor\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nrecall). Sometimes an ‘exploit’ exists for the vulnerability at the time of announcement. An\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n‘exploit’  is  a  piece  of  code  which  anyone  can  use  to  compromise  the  security  of  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nsoftware product and subsequently of other information assets. Therefore, we propose our\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nnext set of hypotheses as:\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nH4a:  The  loss  in  market  value  of  a  software  vendor  is  greater  if  the  announced\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nvulnerability has a higher severity.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nH4b: The loss in market value of a software vendor is greater if an exploit exists publicly\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nat the time of the vulnerability announcement.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nA  recent  article  in  the  Wall  Street  Journal  hinted  that  firms  are  using  vulnerability\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\ndisclosure as a strategic weapon against competitors. E.g. ISS disclosed a vulnerability in\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nrival  Checkpoint’s  flagship  firewall  product  just  ahead  of  Checkpoint’s  investor  summit.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nVendors themselves disclose vulnerability information in their products routinely. In fact,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nmany believe that vendors would prefer not to disclose information at all but they fear that\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nsomeone  else  would  do  it.  Generally  vendors  are  likely  to  be  more  careful  about  the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ndisclosure  as  opposed  to  third  party.  Moreover,  disclosure  by  vendors  would  signal  their\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\ncommitment to providing secure software, we hypothesize that\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nH5:  The  loss  in  market  value  for  a  software  vendor  is  lower  in  case  the  security\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nvulnerability is discovered by the vendor itself rather than by rivals or third party security\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nfirms.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nOur  final  hypothesis  relates  to  the  publication  where  vulnerability  is  reported.  Some\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nvulnerabilities are reported in popular press. Others find mention only in industry sources\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nlike  CERT  or  technology  news  portals  like  news.com.  The  question  which  arises  is\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nwhether  both  sources  are  equally  influential  in  affecting  a  stock’s  CAR?  One  would\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsuspect the popular press of being more influential than industry sources because it reaches\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\na broader audience. So our next hypothesis is:\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nH6:  The  magnitude  of  CAR  is  more  when  the  vulnerability  is  reported  in  popular  press\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nthan in industry sources.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n4. Data Description & Methodology\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n4.1 Vulnerability Disclosure Process\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nThe typical process of vulnerability disclosure takes place as shown in Figure 1.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nVulnerability\nDiscovery\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nBy Virus/Hack Attack\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nBy Third Party\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nBy Firm\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nFull /Limited\nDisclosure?\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nFull Disclosure\n(report to public)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nPatch / No\nPatch\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\nDisclosure\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nLimited Disclosure\n(report to firm)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nAnnounce\nment\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.1**\n•  Vulnerability\nCharacteristics\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\nRevised cash flow\nExpectation by\nInvestors\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nAbnormal\nReturns\n**BLOCK**fs== 11.0**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\nFigure 1: Vulnerability Discovery and Disclosure\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nThe  process  starts  with  the  vulnerability  discovery.  There  are  various  sources  for\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nvulnerability discovery. Some vulnerabilities are discovered by amateur researchers or by\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nindependent  security  analysts  (ISA)  like  eEye  Security.    If  the  ISA  chooses  to  follow\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n‘limited  disclosure’,  it  reports  the  vulnerability  to  the  concerned  vendor  or  to  an\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nindependent body like CERT who disclose the vulnerability to public later (generally after\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nthe  fix  has  been  made  available)  in  a  limited  way8.    Sometimes,  the  vendor  itself  might\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ndiscover  a  vulnerability  in  its  products.  On  the  other  hand,  if  the  ISA  chooses  to  follow\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n‘full  disclosure’  or  if  a  hacker  discovers  the  vulnerability,  the  vulnerability  may  get\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nannounced  directly  to  public.  Forums  such  as  Bugtraq  allow  for  full  disclosure  of\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nvulnerability  information.  In  any  case,  after  the  vulnerability  has  been  announced,\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ninvestors  might  re-evaluate  their  projections  on  the  software  vendors’  profitability  based\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\non  the  vulnerability  characteristics.  Consequentially,  the  stock  price  of  the  vendor  might\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nshow an abnormal return due to the vulnerability announcement.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\n4.2 Data\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nOur data comes from the vulnerability disclosures in popular press as well as from the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nadvisory  reports  in  CERT.  We  include  articles  published  by  news  networks  like\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nBusinesswire,  Newswire  and  daily  articles  in  popular  press  like  WSJ,  NY  Times,\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nWashington Post and LA Times. We search for these news articles in Proquest and Lexis-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nNexis  Academic  databases  which,  between  them,  maintain  news  articles  from  major\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nnewspapers  and  news  networks  all  over  the  country.  We  also  include  articles  from\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nnews.com  which  is  a  CNET  owned  site  and  is  a  premier  source  for  round-the-clock,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nbreaking  technology  news  coverage.  We  used  the  following  terms  in  our  search:\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n‘vulnerability  AND  disclosure’,  ‘software  AND  vulnerability’,  ‘software  AND  flaw’,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\n‘virus AND vulnerability’& ‘vulnerability AND patch’.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nWe  also  searched  the  vulnerability  announcements  for  information  on  the  type  of\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.2**\nvulnerabilities. Based on this, we classify vulnerabilities into various categories.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n•  If  the  announcements  contained  words  such  as  ‘serious’  or  ‘severe’  or  ‘dangerous’  to\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ndescribe  the  vulnerability,  we  characterized  the  vulnerability  as  ‘Severe’.  If  the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nannouncement  characterized  the  vulnerability  as  ‘moderately  severe’  or  ‘with  low\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.4**\nseverity’, we characterized it as ‘Non-severe’.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n•  The  vulnerability  announcements  also  had  references  to  what  kind  to  security  breach\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\ncould be facilitated if attackers exploited the vulnerability. If the vulnerability contained\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nterms  such  as  ‘cause  denial  of  service’,    or  ‘disrupt  operations’,  we  classified  the\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.2**r== 0.1**\nvulnerability as type ‘DoS’; otherwise if the vulnerability contained terms such as ‘gain\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n8 CERT then reports the vulnerability to the respective vendor and gives it time (typically 45 days) to come\nout with a patch.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\naccess’  or  ‘steal  information’  or  ‘take  control’,  we  classified  the  vulnerability  as\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\n‘Confidentiality Related’.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n•  Further,  the  announcements  also  described  whether  the  vendor  released  a  patch  at  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\ntime of the vulnerability announcement. If the vendor announced a patch at the time of\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.2**\nvulnerability disclosure, we classified the vulnerability as ‘Fix Available’.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n•  Finally,  we  also  classify  vulnerabilities  on  whether  an  ‘exploit’  exists  for  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nvulnerability  in  the  public  domain.  If  the  vulnerability  announcement  contained  terms\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nsuch as ‘an exploit for the vulnerability is circulating’, we classify the vulnerability as\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\ntype ‘Exploit’.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nAs per convention in prior event study literature (Hendricks & Singhal 1996), we excluded\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nthe following type of announcements from our sample:\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n•  Vulnerability  announcements  in  non-daily  periodicals  like  magazines  because  of  the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.3**\ndifficulty in determining the exact date of the announcement.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n•  Repeat  announcements  of  the  same  event  in  a  different  publication  at  a  later  date.  In\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\ncase of such repeat announcements, the earliest announcement date was chosen as the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nevent day.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\n•  Announcements which were associated with other confounding events like stock splits\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nand mergers on the event date.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\n•  Announcements related to firms not traded on any public exchange in the US.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Announcements  that  point  to  a  fundamental  protocol  flaw  rather  than  a  particular\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nsoftware. E.g. a flaw in the FTP protocol affects multiple vendors. The reason behind\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\ndropping this category is that the flaw exists in the software only because it follows a\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\nflawed protocol, and not due to the vendor.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n•  Software flaws which are not security related.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nOur dataset contains 146 vulnerability announcements pertaining to around 18 firms in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nthe time period January 1999 – May 2004. We capture information on the following details\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nabout the vulnerability announcement: date, firm name, product, who discovered the flaw,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nnews source, whether patch is available and severity. The descriptive statistics of the data\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nare given in Table 1.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.5**r== 0.4**\nTable 1\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nNumber of firms\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\nNumber of announcements\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\n%age of vulnerabilities announcements in popular press\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.3**\n%age of vulnerabilities for which vendor has patch available at the\ntime of the announcement.\n%age of vulnerabilities discovered by the vendor itself\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.3**\n% of vulnerabilities that could potentially result in a security breach\nrelated to confidentiality\n%age of vulnerabilities for which the announcement contained\ninformation of a publicly available ‘exploit’\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nThe year-wise distribution of announcements is given in Table 2 below.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\nYear\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.5**\n2004 (till May 30)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nTable 2\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\nNumber of Announcements\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.7**\n4.3 Methodology\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nWe use the standard event study methodology for this analysis. An event study assumes\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nthat returns on a stock are significantly impacted by an event of interest (in our case, the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nevent of interest is the vulnerability disclosure announcement).  The period of interest for\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nwhich we observe the event is known as the event window. The smallest event window is\none  day  (day  of  the  announcement  or  day  ‘0’9).  In  practice,  the  event  window  is  often\nexpanded  to  include  two  days  (day  0  and  day  110)  to  capture  the  effect  of  price\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nannouncements  made  after  the  close  of  the  markets  on  a  particular  day.  Sometimes\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n9  If  an  announcement  is  made  on  a  day  when  the  markets  are  closed,  we  consider  the  next  day  when  the\nmarkets open as day 0.\n10 Day 1 is the day after the announcement.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nresearchers include a day before the announcements to incorporate any information leaks\nabout the event. In our study we define a one day event window (day 0)11. Hendricks and\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nSinghal (1996) cite two reasons to use a one day event period. One, a shorter event period\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\npermits a better estimation of the effects of information of stock prices since it reduces the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\npossibility  of  other  confounding  factors  not  related  to  the  announcement.  Two,  it  also\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nincreases the power of the statistical tests.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nAbnormal  returns  are  defined  as  the  difference  between  the  actual  return  of  the  stock\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nover the event window minus the expected return of the stock over the event window. The\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nexpected return on the stock is calculated in several ways, but in our analysis, we use the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nmarket  model  which  assumes  a  stable  linear  relation  between  the  market  return  and  the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nreturn  on  the  stock.  We  also  verify  our  results  using  other  methods  such  as  the  market-\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nadjusted  method  and  the  mean-adjusted  method.  The  coefficients  of  the  linear  model  are\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\ncalculated  by  choosing  a  portion  of  the  data  as  the  estimation  window.  The  estimation\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nwindow,  generally  between  120  days  and  200  days  used  in  most  studies,  is  the  period\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nimmediately before the event window. In our case, we use an estimation window of size\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\n160 days, from day -175 to day -16.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nThere  are  three  main  methods  followed  in  the  event  study  methodology  (Campbell,\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nAndrew  and  MacKinlay,  1997;  Hendricks  and  Singhal,  1996)  to  estimate  the  abnormal\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nreturns.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\n1. The Market Model\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.3**\nIn the market model, the abnormal returns are estimated as follows:\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nAR\nit\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nR\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nit\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nβα\n−\nit\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n⋅\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nR\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nmt\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n(1)\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nwhere i denotes the event (i=1,2……N), m denotes the market and t denotes the  day of\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nthe event (e.g t = 0 denotes the day of the vulnerability announcement.). ARit denotes the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nabnormal return of event i at time t, Rit denotes the actual return and Rmt denotes the market\nreturn  at  time  period  t13.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\ndenotes  the  normal  return  of  the  firm  due  to  the\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n+ βα\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nit R⋅\nmt\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nmarket-wide  movement.  The  abnormal  return  is  defined  as  the  difference  between  the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nactual  return  and  the  normal  return.  This  is  the  part  of  the  actual  return  that  cannot  be\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\n11 We also highlight our results using different values of the event window.\n12 Rit for a stock is the percent change in the stock price at time t, (=Pit - Pit-1 )/ Pit-1\n13 We obtain the data on the stock and market returns from Yahoo Finance(http://finance.yahoo.com)\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nexplained  by  market  movements  and  captures  the  effect  of  the  event.  Since  most  of  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ntech stocks are listed on NASDAQ, we use this as our indicator for market returns. We use\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nordinary  least  squares  regression  to  estimate  the  coefficients  α  and  β  for  the  above\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nregression.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\n2. The Market Adjusted Model\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.5**\nIn this case, the abnormal returns are given as\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nAR\nit\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n=\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nR\nit\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nR\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nmt\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\nwhere the terms have the usual meaning as in the Market Model.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\n3. The Mean Adjusted Model\n**BLOCK**fs== 11.9**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nAR\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nit\n**BLOCK**fs== 11.9**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n=\n**BLOCK**fs== 11.9**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nR\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nit\n**BLOCK**fs== 11.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 11.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nR\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ni\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n(2)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n(3)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nwhere\n**BLOCK**fs== 11.9**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nR\n**BLOCK**fs== 11.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ni\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 17.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n∑\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ns\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 6.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nR\nis\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\nis  the  mean  return  on  the  stock  which  made  a  vulnerability\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nannouncement during event i,  over the duration of the estimation period and T is number\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nof days in the estimation period (in our study, T=160).\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nThe  mean  abnormal  return  across  all  observations  on  day  t  of  the  event  is  given  as\n**BLOCK**fs== 11.9**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nA\nt\n**BLOCK**fs== 11.9**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n=\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nN\n**BLOCK**fs== 17.9**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n∑\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\ni\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n=\n**BLOCK**fs== 11.9**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\nAR\nit\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.5**\n. The cumulative abnormal return\n**BLOCK**fs== 11.9**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nCAR\n**BLOCK**fs== 17.9**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n∑=\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nevent\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\ntA\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.6**r== 0.1**\nfor the event is defined as the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nsum  of  the  abnormal  returns  over  the  event  window.  Prior  research  on  event  studies\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n(Brown  &  Warner  1980,  Brown  &  Warner  1985)  presents  a  comprehensive  analysis  of\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nsuitable test statistics for the abnormal mean return.  Since vulnerabilities were disclosed\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nby  more  than  one  vendor  on  a  given  day,  our  statistic  should  allow  for  event  day\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nclustering.  The  following  t-statistic  proposed  by  Brown  &  Warner  (1985)  takes  in  to\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\naccount event day clustering as well as cross-sectional dependence in the security specific\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nexcess returns.\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nt =\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nA\nt\nS\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nA\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nWhere\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nS\n**BLOCK**fs== 7.1**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nA\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n⎛\n⎜\n⎝\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n(4)\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n(\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nA\ns\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nA\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n)\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n⎞\n⎟\n⎠\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.4**r== 0.1**\nand T is the number of days in the estimation period\n**BLOCK**fs== 7.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 18.1**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n∑\n**BLOCK**fs== 7.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\ns\n**BLOCK**fs== 7.1**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n=\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nand\n**BLOCK**fs== 12.1**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\nA\n**BLOCK**fs== 7.1**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 12.1**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n⎛\n= ∑\n⎜\nT\n⎝\n**BLOCK**fs== 7.1**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 7.1**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\ns\n**BLOCK**fs== 7.1**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nsA\n**BLOCK**fs== 12.1**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n⎞\n⎟\n⎠\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nThe  null  hypothesis  is  that  the  abnormal  returns  are  not  significantly  different  from\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nzero.  Under  the  null  hypothesis,  the  abnormal  returns  are  independent  and  identically\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ndistributed  and  normal  with  a  mean  of  zero  and  the  variance  given  by  the  variance  of\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nabnormal returns over the estimation period.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nWe  also  do  multivariate  regression  and  test  our  hypothesis  of  how  CAR  varies  with\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nvarious vendors, or vulnerability characteristics. But before we present the details on our\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nregression analysis, we present the results of the event study.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\n4.2 Event Study Results\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nTable  3  summarizes  the  results  of  our  event  study  and  quantifies  the  effect  of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nvulnerability disclosures on the stock prices of software vendors for our entire sample of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n146 announcements (p-values are in parenthesis)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nTable 3: Cumulative Abnormal Return\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nDay 0     CAR\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.4**r== 0.3**\nMarket Model  Market\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.8**r== 0.1**\nMean Model\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nMean Abnormal Return\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n-0.63   (0.01)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n-0.67   (0.01)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n-0.5    (0.09)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nMedian Abnormal Return\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n-0.44  (0.00)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n-0.5    (0.00)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n-0.55  (0.01)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.7**\nPercent Less than Zero\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n64%   (0.00)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n63.5%  (0.001)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.8**r== 0.1**\n58.7%  (0.03)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nAdjusted Model\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nWe  calculate  CARs  under  there  different  models  (Market  Model,  Market  Adjusted\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nModel and Mean Adjusted Model).  For each of the three models, we use three different\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\ntest  statistics  (Mean  Abnormal  Return,  Median  Abnormal  Return  and  Percent  Less  than\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nZero). The Mean Abnormal Return Test (equations (1) to (4)) is parametric in nature and\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nmakes  assumptions  about  the  distribution  of  abnormal  returns.  We  also  use  two  non-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nparametric tests to strengthen our results. One, we use the Wilcoxon Signed Rank Test to\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\ncalculate the p-value for the median abnormal return; two we use the Sign Test to calculate\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nthe p-value for the percent negative returns. The Wilcoxon Signed Rank Test tests whether\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nthe  median  of  the  abnormal  returns  is  different  than  zero.  The  Sign  Test  is  based  on  the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsign  rather  than  the  magnitude  of  the  abnormal  returns  and  requires  that  under  the  null\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.2**\nhypothesis, the proportion of abnormal returns greater than (or less than) zero is 50%.\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nFrom Table 3, we note that the CAR for day 0 is negative across all the three different\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nmodels. E.g. the Mean Abnormal Return varies between 0.5% - 0.67% depending on the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nmodel  used.  Further  the  Market  Model  and  the  Market  Adjusted  Model  are  statistically\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nsignificant at p < 0.01, while the Mean Adjusted Model is statistically significant at p < 0.1\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nlevel.  The Median Abnormal Returns range between 0.44% - 0.55% and are significant at\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nthe 0.01 level. Finally, the percent observations less than zero range between 57.8% - 64%\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nand  are  significant  at  the  0.05  level.  It  is  clear  that  CAR  is  negative  and  statistically\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nsignificant for all three models and all three tests.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nThus,  our  results  suggest  that  software  vendors  do  tend  to  lose  market  value  when  a\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nvulnerability is announced in their product. To check for the robustness of our results, we\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nnot  only  test  the  mean  abnormal  returns,  but  also  the  median  abnormal  returns  and  the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\npercent  less  than  zero.  This  provides  support  for  hypothesis  H1  that  vulnerability\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nannouncements are associated with a loss in market value of software vendors.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nWe also calculate the abnormal returns using different event windows (beyond 0 days)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nusing  the  market  model  (the  results  do  not  change  substantially  for  other  models).  The\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nresults are given in Table 4.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.3**\nTable 4: CAR for various time periods\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nDay\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n-1\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n0 to 1\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n0 to 2\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n0 to 5\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n0 to 10\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\nCAR\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n(p-value)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n0.25\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(0.4)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n-0.63\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n(0.01)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n-0.65\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n(0.07)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n-0.47\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n(0.35)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n-0.25\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n(0.7)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.8**r== 0.2**\n-0.8\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n(0.36)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nFrom the table, it is clear that the CAR on day 0 is negative and significant at the 0.01\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nevel. However, CAR for day 0 and day 1 combined is significant only at the 0.1 level. This\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nsuggests  that  the  stock  market  is  efficient  in  the  sense  that  the  effect  of  a  software\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nvulnerability  announcement  is  quickly  incorporated  into  a  vendors’  stock  price.  The  p-\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nvalues for day -1 is not statistically significant. A possible explanation for this is that the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\neffect of news leakage through forums like SecurityFocus is not significant. The CARs in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ncolumns  3,  4,  5  and  6  are  negative  but  not  statistically  significant.  However,  it  is\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ninteresting to note that the CARs are negative for even a 10 day window.  Figure 2 shows\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nthe CAR over a 16 day event window (from day -5 to day 10).\n**BLOCK**fs== 10.2**b== 0.8**t== 0.2**l== 0.4**r== 0.3**\nFigure 2: CAR from day -5 to day 10\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n-5\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n-4\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n-3\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n-2\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n-1\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n-0.002\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n-0.004\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nR\nA\nC\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n-0.006\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n-0.008\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n-0.01\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n-0.012\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nDay\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\nFigure 2: CAR over a 16 day event window\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nOur result corroborates prior work on defective products (Jarrell et al 1985, Davidson et\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nal 1992) by showing that product defects lead to a loss in market value of a firm. Our study\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nanalyzes returns on stocks of software vendors and we find that a defective software which\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ncompromises the security of customers’ information systems leads to a negative impact on\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nthe  market  value  of  the  software  vendor.    The  extent  of  losses  suffered  by  a  vendor,  on\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\naverage,  is  moderate  at  about  0.63%  of  its  market  capitalization  value  on  the  day  the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nvulnerability gets announced.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nMarket Capitalization\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nWe  also  calculated  the  abnormal  change  in  market  capitalization  values  of  the  software\nvendor due to the vulnerability announcement14. For each firm, the day 0 change in market\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\ncapitalization  value  was  calculated  by  multiplying  the  day  -1  market  capitalization  value\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nwith the abnormal returns on day 0. On average, we calculate that the software vendors in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nour sample lost $0.86 billion in market capitalization value on the day of the vulnerability\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ndisclosure.  Since Microsoft accounts for more than 40% of our sample, we sub-divide our\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsample into Microsoft and non-Microsoft samples. For the Microsoft sample, the average\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n14 We obtain the market capitalization values from the CSRP database by multiplying the share price with the\nnumber of shares outstanding.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nchange in market capitalization is around $0.92 billion and for the non-Microsoft sample,\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nthe average change in market value is $0.81billion.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nRobustness Checks: We also perform the following robustness checks on our results, as\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.3**\nspecified in the event study by Cooper, Dimitrov and Rau (2001).\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\n1.  Robustness to Outliers:\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nTo  check  the  robustness  of  our  results  to  exclude  the  effect  of  outliers,  we  compute  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nCAR for our sample after excluding the top 10 percentile and the bottom 10 percentile of\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nobservations  (ranked  according  to  the  day  0  mean  abnormal  returns).  We  find  that  our\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nresults remain qualitatively the same. For example, mean abnormal returns for this sample\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nare 0.53% (against 0.63% for the entire sample) and these are significant at the 5% level.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nThis suggests that our results are robust to outliers in the data.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\n2.  Momentum Effect:\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nIt can be argued that the day 0 abnormal returns are caused simply by market momentum\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nthan by the underlying event. As a simple check, we compute the correlation between the\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nabnormal returns before the event and those after the event. In specific, we check the pair-\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nwise correlation (along with level of significance) for three pairs of values : one, day -10 to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nday -1 CAR and day 0 to day 10 CAR; two, day -10 to day -1 CAR and day 0 CAR; and\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nthree, day -1 CAR and day 0 CAR. The pair-wise correlations are as follows:\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\n•  day -10 to day -1 CAR and day 0 to day 10 CAR (correlation: 0.13, p-value 0.12)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.2**\n•  day -10 to day -1 CAR and day 0 CAR (correlation: -0.05, p-value 0.5)\n**BLOCK**fs== 12.1**b== 0.3**t== 0.6**l== 0.2**r== 0.3**\n•  day -1 CAR and day 0 CAR (correlation: 0.03, p-value 0.67)\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nThus  we  find  that  none  of  the  correlations  is  strong  or  significant  at  the  10%  level  and\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nhence the momentum in the stock prices does not seem to be driving our results.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\n5. Regression Analysis\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nTo test the other hypothesis, we develop both a regression model and simple means test\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nto explain the effect of various vulnerability characteristics on abnormal returns. However,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nordinary  least  squares  model  might  not  be  appropriate  because  it  does  not  account  for\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nheterogeneity  among  firms.  The  issue  of  heterogeneity  is  an  important  consideration  in\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nanalyzing panel data. For example, the level of abnormal returns could differ across firms\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nif investors use different valuation models  across  firms  (Malatesta  and  Thompson  1985).\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nEstimating  aggregate  parameters  while  ignoring  heterogeneity  could  lead  to  biased  and\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\ninconsistent estimates (Hsiao 1986).\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nTo incorporate the impact of firm specific heterogeneity in our data, we propose a fixed\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\neffects  model.  The  fixed  effects  model  controls  for  unobservable  firm  specific  variables\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nthat are constant over time. This is equivalent to generating dummy variables for each firm\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nand including them in an ordinary linear regression to control for firm specific effects. The\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nfixed effects model can be specified as:\n**BLOCK**fs== 7.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\ny\nit\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n=\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nX\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n+⋅\nεµβ\nit\ni\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n+\n**BLOCK**fs== 7.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nit\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n(5)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nwhere i = 1…..N (N is the total number of firms) and t = 1……..T (T is the total number\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nof events). yit is the Abnormal Return (ARit) for firm i at event t as calculated according to\nthe  market  model  in  equation  (1)15.  Xit  are  the  independent  variables  which  capture  the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nvarious  vulnerability  characteristics,  ui  is  the  firm  specific  dummy  variable.  The\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.4**\ndescription of the independent variables is as follows:\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nSEVR:  whether  the  vulnerability  has  been  classified  as  severe;  SEVR  =  1  for  a  severe\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nvulnerability and 0 otherwise.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nPATCH: Whether a patch is available at the time of the vulnerability disclosure. PATCH\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n=1 if a patch is available and 0 otherwise.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nDISC:  Whether  the  vulnerability  was  disclosed  by  the  third  party.  DISC  =  1  if  the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nvulnerability was disclosed by the third party and 0 if disclosed by the vendor.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nEXPLOIT:  If  an  exploit  is  publicly  available  at  the  time  of  the  vulnerability\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.4**\nannouncement, then EXPLOIT = 1; otherwise it is zero.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nCERT: If the vulnerability was first reported in CERT, then CERT = 1, it is 0 otherwise.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nPRESS: If the vulnerability was first reported in popular press, the variable PRESS = 1,\notherwise it is zero.16\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nDOS:  If  the  vulnerability  can  potentially  lead  to  a  denial  of  service  type  attack  or  a\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\ndisruption in services, then the variable DOS =1, otherwise it is zero.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nEffect of Investor Sentiments:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n15 Since the market model is the most common model used in event studies, we proceed with the remaining\nanalysis with this model.\n16 For PRESS and CERT, we compare with the baseline case that the vulnerability was reported at the\nCNET/ZDNET owned news.com website.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nTo control for abnormal returns due to overall market sentiments, we introduce a set of\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ndummy  variables  based  on  the  time  when  the  vulnerability  was  announced.  We  use  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nevents surrounding 9/11 as the basis for segmenting our sample into various time periods.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThe stock market crash in late 2000 could also play a role in the negative abnormal returns.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.4**\nWe introduce the following dummy variables in our model:\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nPost_911: If the vulnerability was announced within a year of 9/11, i.e. if the date of\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nthe  announcement  was  between  Sept  11,  2001  and  Sept,  11  2002.  This  constitutes  19\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nobservations from our sample.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nPre_911: If the vulnerability was announced within a year before 9/11, i.e. if the date of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nthe announcement was between Sept 11, 2000 and Sept, 11 2001. This was also the time\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nafter the stock market crashed in mid 2000 and lasted till the first three quarters of 2001\n(Wall  Street  Journal,  200017,  Kannan  et  al  2003,  Wall  Street  Journal,  200318).  This\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nconstitutes 28 observations from our sample.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nYear_99-00: if the announcement was made prior to September 11, 2000, i.e. if the date\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nof the announcement is between Jan 1, 1999 and September 11, 2000.  This constitutes 23\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nobservations from our sample.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nYear_02-03:  If  the  date  of  the  announcement  is  between  September  11,  2002  and\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nSeptember 11, 2003. This constitutes 44 observations from our sample.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nYear_03-04: If the date of the announcement is between September 11, 2003 and June\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n1, 2004. This constitutes 32 observations from our sample. This is the baseline category for\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.7**\nour regression.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n5.1 Results\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nWe  first  compare  the  mean  CARs  of  different  sub  samples  based  on  vulnerability\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\ncharacteristics  to  understand  how  various  factors  affect  market  value  of  the  firms.  The\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\ncharacteristics that we consider are explained above. This is similar to the method followed\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nby Subramani and Walden (2001), Chatterjee, Richardson and Zmud (2001) Im, Dow and\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nGrover (2001). The results are summarized in Table 5.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\n17 Article titled ‘The Internet Bubble Broke Records, Rules and Bank Accounts’ dated 07/14/2000\n18 Article titled ‘Thinking Things Over: On Repairing Economic Damage’ dated 03/10/2003\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nVulnerability\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nCharacteristic\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.4**r== 0.4**\nTable 5: Means Test\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nVariable\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nCAR\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\np-value\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nHigh Severity (81%)\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\nSeverity\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nLow Severity (19%)\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nSource of Discovery\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nDiscovered by Vendor\n(36%)\nDiscovered by Third\nParty (64%)\nPatch Available (24%)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nPatch Availability\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nPatch Not Available (76%)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nCERT (34%)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nSource of Disclosure\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nPress (35%)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nNews.com (30%)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nDenial of Service (24%)\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nType of Attack Possible  Confidentiality (76%)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\nMicrosoft vs Non\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nMicrosoft\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nMicrosoft (46%)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\nNon Microsoft (54%)\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n-0.76\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n-0.04\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n-0.95\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n-0.47\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n-0.37\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n-1.49\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n-0.47\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n-0.98\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n-0.47\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n-0.23\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n-0.78\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n-0.28\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n-0.91\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n0.015\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n0.04\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n0.1\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n0.2\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\n0.28\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n0.05\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n0.4\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n0.04\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n0.3\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n-0.66\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n0.015\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n0.4\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.8**r== 0.2**\n0.13\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nFrom the means analysis it is clear that more severe vulnerabilities affect the stock price\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nmore.  Surprisingly,  when  vendors  disclose  the  information  then  the  investors  seem  to\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nperceives  it  more  negatively  than  when  some  third  party  releases  it.  As  expected,\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\navailability  of  patch  mitigates  the  negative  impact  of  disclosure  somewhat.  Since  the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nvendors  typically  disclose  the  information  with  a  patch,  we  may  find  during  the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nmultivariate regression that vendor release effect may disappear. The source of disclosure\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nis  also  relevant.  Investors  seem  to  pay  more  attention  to  vulnerabilities  published  in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nmainstream newspapers than to CERT. Confidentially breaches are considered more severe\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nand hence have higher significant impact than denial of service vulnerabilities. Finally, we\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ndo not find an evidence of Microsoft effect though, non-Microsoft firms seem to lose more\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nvalue.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nIn  the  second  method,  we  test  our  hypothesis  related  to  the  impact  of  various\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nvulnerability  characteristics  on  the  market  value  of  a  software  vendor  using  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nmultivariate regression model outlined in equation (5). This is similar to the methodology\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nfollowed  by  prior  event  studies  such  as  Hendricks  and  Singhal  (1997),  Chatterjee,\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nRichardson and Zmud (2001), Lane and Robertson (1995). The regression method has the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nadvantage over the sub-sample method used in Table 5 that the regression method captures\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nthe  effect  of  all  the  independent  variables simultaneously.  The  sub-sample  method  could\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ngive spurious results because the effects of a sub-sample, such as patch vs no-patch, can be\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nexplained by a relationship between patch and other independent variables. The parameter\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nestimates as well as the p-values for the various parameters of our fixed effects regression\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nare given in the Table 6 below.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nTable 6 : Regression Estimates\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\nVariable\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nSEVR\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nEXPLOIT\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nPATCH\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\nDISC\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nCERT\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nPRESS\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nDOS\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.7**\nY_99-00\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nPre_911\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nPost_911\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nY_02-03\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.7**\nConstant\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nCoefficient\n-0.006*\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n-0.005\n0.0083*\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\n-0.005\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n0.006\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n-0.0053\n0.0076*\n**BLOCK**fs== 12.0**b== 0.2**t== 0.6**l== 0.5**r== 0.4**\n-0.007\n-0.011*\n-0.02*\n-0.01*\n0.01*\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.7**r== 0.2**\np-value\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n0.1\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n0.24\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n0.04\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\n0.16\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n0.3\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n0.27\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n0.06\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.7**r== 0.2**\n0.26\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n0.05\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n0.001\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n0.05\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\n0.05\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nThe  R2  for  this  regression  is  16.6%.  The  F-test  (p-value  0.001)  for  the  overall  model\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsuggests that our model is highly significant. The constant in the regression is simply the\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\naverage  of  all  the  fixed  effects  in  the  model.  Our  regression  provides  several  interesting\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nobservations regarding the effect of the vulnerability characteristics on the stock price of\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nthe vendor.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\n1.  Severity: The coefficient on the SEVR variable in Table 6 is negative and significant\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\n(though  at  p  =  0.1).  This  is  consistent  with  results  in  Table  5.  More  severe\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nvulnerabilities have a higher potential to cause damage and hence have a larger adverse\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nimpact  on  CAR.  On  average,  a  severe  vulnerability  can  cost  a  software  vendor  0.6%\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.2**r== 0.2**\nmore than a non-severe vulnerability, providing support to hypothesis H4.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n2.  Patch:  We  also  find  that  the  non-availability  of  a  patch  is  positive  and  significantly\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ncorrelated with the market value. This provides support to hypothesis H2 because the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\ncoefficient  on  the  PATCH  variable  suggests  that  on  average,  firms  which  do  not\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nprovide a patch at the time of the vulnerability disclosure suffer a loss of 0.83% more\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nthan firms which provide a patch.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n3.  Confidentiality:  The  coefficient  of  the  DOS  variable  is  0.0076.  This  implies  that\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\nvulnerabilities which result in a non-confidential breach (denial of service, disruptions\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nin  service)  have  an  abnormal  return  of  0.76%  higher  than  the  vulnerabilities  which\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nresult in a confidentiality-related breach. This confirms our hypothesis H3.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nNone of the other coefficients are significant (except time coefficients). Thus our results\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nfail to provide support for hypothesis H5 and H6. From table 5 also, we can observe that\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nnone  of  the  sub-samples  (other  than  the  Press  sub-sample)  has  a  significant  coefficient.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe rejection of hypothesis H5 is especially interesting because it suggests that the markets\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ndo not penalize a vendor any more if the vulnerability is discovered by a third party than\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nby the vendor itself.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nWe also control for the effect of the time period during which the vulnerabilities were\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nannounced  to  control  for  time-specific  investor  sentiments  which  can  affect  abnormal\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nreturns.  We  find  that  software  vendors  suffered  the  greatest  abnormal  returns  in  the  one\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nyear period post 9/11; for example, on average, vendors lost 2% more in market value for\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\neach vulnerability announcement in the year following 9/11 than they did in the baseline\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nperiod  (2003-04).  This  suggests  that  security  concerns  among  investors  were  highest\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nduring this period, as an aftermath of September 11. The next highest period of abnormal\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nreturns was the one-year time frame after the dot-com crash, i.e. from Sept 2000 to Sept\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n2001.  In  this  period,  software  vendors  lost,  on  average,  1.0%  more  on  vulnerability\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nannouncements than in the baseline period. The results also suggest that year 2003-04 is\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nwhen the vendors suffered the least. It is possible  that  vulnerability  announcements  have\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nbecome  commonplace  and  are  not  affecting  the  vendors  as  much.  But,  more  research  is\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nneeded to provide a definitive answer.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\n6. Conclusions and Discussion\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nTo  the  best  of  our  knowledge,  this  is  the  first  study  to  analyze  the  impact  of  product\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\ndefects on software vendors. We also analyze the information content of the vulnerability\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ndisclosure  announcement  and  classify  vulnerabilities  into  various  sub-types  based  on  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nfollowing  characteristics:\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.2**\nthe  source  of  vulnerability  disclosure,  severity  of\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nthe\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nvulnerability, availability of a fix, whether an exploit was publicly available at the time of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\ndiscovery,  the  type  of  security  breach  caused  by  the  vulnerability  and  the  source  of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nvulnerability discovery. Our results show that vulnerability disclosure leads to a significant\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nloss  of  market  value  for  software  vendors.  This  indicates  that  the  stock  markets  react\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nnegatively  to  the  news  of  a  vulnerability  disclosure,  because  the  discovery  of  a\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nvulnerability  could  suggest  a  loss  in  future  cash  flow  of  the  software  vendors.  Software\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nvulnerabilities affect the cash flows of a vendor in two main ways: One, the vendor has to\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nspend time and effort in providing a patch for the vulnerability, which increases the overall\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\ncost  of  the  software  product  and  hence  reduces  profits.  Vulnerabilities  may  also  lead  to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\ncustomer  dissatisfaction  because  the  customer  has  to  spend  time  and  effort  in  installing\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\npatches  and  is  exposed  to  a  risk  of  security  related  attacks.  This  could  further  lead  to\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ncustomers  shifting  to  competitor’s  products  and  hence  reduces  the  cash  flow  for  the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nvendors’  products.  This  has  implications  for  software  vendors  to  invest  in  improving  the\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nquality of their software. While vendors would like to launch software products as soon as\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\npossible, our study shows that vendors need to focus testing in  areas that can potentially\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\ncontain  greater  number  of  security  vulnerabilities.  We  also  show  that  the  effect  of  a\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nvulnerability  announcement  is  quickly  incorporated  into  the  stock  price  and  after  the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nsecond day, there is no significant impact on the stock prices. We check for the robustness\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nof our results in terms of checking for outliers and checking for momentum effects in the\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nstock returns. We find that our results are also robust to the effect of investor sentiments\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nsince we control for the effect of various time periods in our regression.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nOur study also provides preliminary evidence that firms should integrate security into\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nsoftware  quality  practices.  In  a  firm  with  limited  resources,  this  would  mean  focusing\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\ntesting  efforts  in  areas  that  have  a  greater  number  of  security  vulnerabilities.  Although\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nresearchers  in  computer  science  have  stressed  on  this  fact  (McGraw  2004),  there  hardly\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nexists  any  literature  in  software  engineering  economics  which  measures  the  return  on\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ninvestment  of  incorporating  security  based  metrics  in  software  quality  or  software  risk\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nassessment.  While  software  quality  traditionally  deals  with  functional  testing,  complete\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nsecurity  testing  would  incorporate  non-functional  testing  we  well,  i.e.  subjecting  the\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nsoftware to misspecified input streams (Potter and McGraw 2004).\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nOur  results  also  show  that  vulnerabilities  which  are  severe  or  which  can  cause  a\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nconfidentiality  related  attack  are  likely  to  result  in  a  greater  loss  in  market  value  for  a\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nvendor. This has implications for the software vendor in terms of allocating resources for\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nsoftware  testing  and  quality  improvements.  For  example,  vendors  should  focus  on\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nspending more resources in testing those software modules where a flaw is more likely to\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\ncause  a  confidentiality  related  breach.  Campbell  et  al  (2003)  show  that  confidentiality\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nrelated breaches cause a greater loss to firms; we complement that result by showing that\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nsoftware vulnerabilities which cause a confidentiality related breach are likely to result in a\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\ngreater loss in market value for the vendor.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nComparison with prior event studies:\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nIt is interesting to compare how the abnormal returns in our event study compare with\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nresults in prior event studies. Specifically, we compare our results with event studies in the\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nfollowing  categories:  security  breach  related  announcements,  IT  investment  related\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nannouncements and product defect related announcements.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nTable-7  shows  that  our  results  are  comparable  to  prior  studies  on  product  defects  and\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nproduct  recall  announcements.  It  is  especially  interesting  to  note  that  the  loss  in  market\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nvalue that vendors suffer due to a security vulnerability is much less than that suffered by\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nfirms  during  a  security  breach.  A  possible  reason  could  be  that  software  vendors  are\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nprotected  by  click-wrap  agreements  and  have  only  limited  liability  for  any  flaw  in  their\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nproducts.  Another  reason  is  that  firms  usually  supply  a  patch  with  the  vulnerability\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ndisclosure (almost 76% of observations in our sample have a patch available at the time of\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ndisclosure).  Therefore  security  breaches  are  not  so  much  caused  due  to  unprotected\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nvulnerabilities as due to lack of adequate patching done by firms. E.g. the SQL Slammer\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nvirus, which affected millions of servers worldwide, was created when hackers exploited a\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nsix month old vulnerability in SQL. Microsoft had already released a patch for the same,\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nbut,  as  demonstrated  by  the  Slammer,  many  firms  had  not  adequately  protected  their\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nservers by applying the patch.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\nTable 7: Summary of previous event studies\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nClassification of Event\nStudy\nImpact of Vulnerability\nDisclosures on\nSoftware Vendors\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\nImpact of Security\nBreaches on Firms\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nAuthors\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nTelang R and S Wattal (2004)\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.7**r== 0.2**\nTime\nPeriod\n1999-2004\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nCAR\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n-0.63%\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.4**r== 0.3**\nCampbell K, Gordon LA, Loeb MP\nand L Zhou (2003)\nCavusoglu H, Mishra B and S\nRaghunathan (2004)\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n1995-2000\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n-2.0%*\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\n1998-2000\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\n-2.1%\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nImpact of Product\nRecall Announcements\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.7**\nImpact of IT\nInvestment\nAnnouncements\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nHovav A and J D’Arcy (2003)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n1998-2002\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\nKannan K, Rees J and S Sridhar\n(2004)\nJarrell G and S Peltzman (1985)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n1997-2003\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n1967-1981\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n1968-1987\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n1987-1998\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.4**r== 0.2**\nDavidson WL III and DL Worrell\n(1992)\nChatterjee D, Richardson VJ and\nRW Zmud (2001)\nIm KS, Dow KE and V Grover\n(2001)\nSubramani M and E Walden (2001)  Oct 1998-\nDec 1998\n1981-1988\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nDos Santos BL, Peffers K and DC\nMauer (1993)\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n1981-1996\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nNot\nSignificant\n-0.73%\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n-0.81%\n(for auto)\n-0.36%\n(day -1)\n1.16%\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\nNot\nSignificant\n7.5%\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n1%\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nImpact of Winning a\nQuality Award\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nHendricks KB and Singhal VR\n(1996)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\n* Not Significant at the 10% level\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.4**\nImplications for Software Quality and Disclosure Policy\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n1985-1991\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n0.59%\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nAs we noted in introduction, one major argument given by the full disclosure group\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nis that it will eventually force the vendors to improve the quality of their product. From our\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nanalysis, there seems to be some support for this argument. Disclosure, in general (with or\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nwithout a patch) adversely affects the market valuation of the vendors. It is more severe in\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\ncase  of  without  patch  (which  is  what  generally  happens  during  full  disclosure).  Thus,\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\ndisclosure clearly creates some incentives for vendors to produce better quality software.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nHowever,  market  value  is  only  one  metric  to  capture  the  impact  of  disclosure.  A  more\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ninteresting  and  comprehensive  work  would  be  to  measure  the  impact  of  disclosure  on\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nprofit or market share of these firms. But our paper does provide a starting point for why\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nwe  should  analyze  this  issue  in  more  detail.    Another  potential  area  of  future  research\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nwould  be  to  capture  and  test  the  link  between  security  based  risks  and  the  quality  of\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nsoftware systems.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nFrom  our  discussion,  it  is  also  clear  why  vendors  are  pushing  for  a  limited\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ndisclosure policy. Recently, Organization for Internet Safety (OIS), which is consortia of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\n11  large  software  vendors,  announced  a  limited-disclosure  policy  which  requires  the\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\ndiscoverer to notify the vendors and give them some time before making the information\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\npublic. From our analysis, such a policy benefits vendors because limited disclosure gives\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nthem time to release a patch for the vulnerability and availability of patch mitigates some\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nadverse  effects  of  disclosure.    We  also  find  that  whether  the  vulnerability  is  reported  by\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nvendors themselves, or other parties, it has essentially similar impact on market value. To\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nan  extent,  our  study  points  that  vendors  are  not  necessarily  better  off  disclosing\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\ninformation themselves. Generally, an argument could be made that vendors should release\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nthe  information  themselves,  for  if  not,  someone  else  will  and  it  will  lead  to  worse\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nconsequences. However, we do not find any evidence of this. Vendors may be better off\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nkeeping  quiet  and  integrate  their  fixes  as  either  service  packs  (which  do  not  give  micro-\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ndetails on what it fixes) or newer versions and announce the patch only if someone else has\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\ndisclosed it. However, more research is required; especially more data on announcements\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nwhen hackers exploit the vulnerabilities.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nAnother issue raised in discussing software flaws is whether software vendors should be\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nheld responsible by law for vulnerabilities discovered in their products. Currently, the use\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nof  End  User  License  Agreements  (EULA  or  Click-Wrap  Agreements)  that  come  with\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsoftware  products  limit  the  liability  that  vendors  face  if  customers  suffer  a  loss  due  to  a\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nflaw in software. Our results show that liability laws (which anyway are not likely to pass\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsometimes soon) are not the only way to ‘punish’ software vendors for flaws discovered in\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ntheir products. Market seems to act on this information and punishes a vendor, who on an\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\naverage, loses around 0.63% of its market value on the day a vulnerability is reported in its\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nproducts.  Software  liability  could  certainly  cause  the  market  value  of  the  vendors  to\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\ndecline further if a vulnerability is reported in their product.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nSome  vulnerabilities  are  posted  on  a  public  listing  such  as  Bugtraq  before  these  are\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nannounced in popular press or CERT. In that case, the actual vulnerability announcement\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nmay  have  little  surprise  value.  Therefore  our  results  are  a  lower  bound  for  the  actual\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ndecrease  in  stock  prices  experienced  by  the  software  vendor  if  a  flaw  is  reported  in  its\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nproduct. We do not include the vulnerabilities reported on Bugtraq since most of these are\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nnot confirmed vulnerabilities at the time they are posted online.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.1**\nA  limitation  of  our  study  is  that  most  of  the  data  points  in  our  sample  are\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nannouncements  regarding  off-the-shelf  software  products.  Our  analysis  does  not  cover\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nsoftware development projects where a security flaw can cause millions of dollars worth of\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\ndamage.  The  main  reason  for  excluding  them  was  the  lack  of  availability  of  data  on\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nsoftware failures in such cases.  We also reiterate that further analysis in terms of software\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nquality,  market  share  or  profitability  is  needed  to  fully  understand  how  vulnerability\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\ndisclosure  signals  poorer  quality  and  how  it  affects  the  vendors’  incentives  to  provide\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nbetter quality software. Our paper takes the first step in this direction.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nReferences\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Adams  EN  III  (1980)  ‘Minimizing  Cost  Impacts  of  Software  Defects’  IBM  Research\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nReport, RC 8228 April\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Applewhite A (2004) ‘Whose Bug Is It Anyway? The Battle over Handling Software\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nFlaws’ IEEE Software, March/April 2004, 94-97\n**BLOCK**fs== 12.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\n•  Anderson  R  (2001)  ‘Why  Information  Security  is  Hard  –  an  Economic  Perspective’\nProceeding of 17th Annual Computer Security Applications Conference, New Orleans,\nLouisiana\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n•  Arora,  A.,  Caulkins,  J.P.  and  R  Telang,  (2004).  ‘Sell  First,  Fix  Later:  Impact  of\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nPatching on Software Quality’, Carnegie Mellon University, working paper.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n•  Arora, A., Telang, R. and  H Xu, (2004). ‘Optimal  Policy  for  Software  Vulnerability\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nDisclosure’, Carnegie Mellon University, working paper.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n•  Banker RD and SA Slaughter (1997) ‘A Field Study of Scale Economies in Software\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\nMaintenance’, Management Science, 43(12), 1709-1725\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n•  Barki  H,  Rivard  S  and  J  Talbot  ‘Toward  an  Assessment  of  Software  Development\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nRisk’, Journal of Management Information Systems, 10(2),  203-225\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n•  Basili, VR and JD Musa (1991) ‘The Future Engineering of Software: A Management\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.5**\nPerspective’ IEEE Computing, 20(4), 90-96\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n•  Bharadwaj, A and M Keil (2003) ‘The Effects of Information Technology Failures on\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.2**r== 0.5**\nthe Market Value of Firms’, Working paper.\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\n•  Brown SJ and JB Warner (1980) ‘Measuring Security Price Performance’, Journal of\n**BLOCK**fs== 12.1**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nFinancial Economics, 14(1985), 3-31\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n•  Brown  SJ  and  JB  Warner  (1985)  ‘Using  Daily  Stock  Returns:  The  Case  of  Event\n**BLOCK**fs== 12.1**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nStudies’, Journal of Financial Economics, 14(1985), 3-31\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n•  Campbell JY, Andrew WL and AC MacKinlay (1997) ‘The Econometrics of Financial\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nMarkets’ Princeton University Press\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\n•  Campbell  K,  LA  Gordon  LA,  Loeb  MP  and  L  Zhou  (2003)  ‘The  Economic  Cost  of\nPublicly  Announced  Information  Security  Breaches:  Empirical  Evidence  from  the\nStock Market’, Journal of Computer Security, 11(3), 431-448\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n•  Carr  R,  Tynan  M  and  R  Davis  (1991)  ‘Quality  and  Security:  They  Work  Together’\n**BLOCK**fs== 12.1**b== 0.7**t== 0.3**l== 0.2**r== 0.5**\nIEEE AES Systems Magazine, 6(9), 15-19\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\n•  Cavusoglu  H,  Mishra  B  and  S  Raghunathan  (2004)  ‘The  Effect  of  Internet  Security\nBreach  Announcements  on  Market  Value:  Capital  Market  Reactions  for  Breached\nFirms  and  Internet  Security  Developers’  International  Journal  of  Electronic\nCommerce, 9(1), 69\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\n•  Chaney  PK,  Devinney  TM  and  RS  Winer  (1991)  ‘The  Impact  of  New  Product\nIntroductions on the Market Value of Firms’, The Journal of Business, 64(4), 573-610\n•  Chatterjee  D,  Richardson  VJ  and  RW  Zmud  (2001)  ‘Examining  the  Shareholder\nWealth  Effects  of  Announcements  of  Newly  Created  CIO  Positions’,  MIS  Quarterly,\n25(1), 43-70\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n•  Cooper MJ, O Dimitrov and PR Rau (2001) ‘A Rose.com by Any Other Name’, The\n**BLOCK**fs== 12.1**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nJournal of Finance, (6), 2371-2387\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n•  Cusumano,  MA  (2004)  ‘Who  is  Liable  for  Bugs  and  Security  Flaws  in  Software?’\n**BLOCK**fs== 12.1**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nCommunications of the ACM, 47(3), 25-27\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Davidson WL III, DL Worrell(1992) ‘The Effect of Product Recall Announcements on\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\nShareholder Wealth’ Strategic Management Journal, 13(6), 467-473\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Devanbu P and S Stubblebine (2000) ‘Software Engineering for Security : a Roadmap’\n**BLOCK**fs== 12.1**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nFuture of Software Engineering, ACM 225-239\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Dolley J (1933) ‘Characteristics and Procedure  of Common Stock Split-Ups,’ Harvard\n**BLOCK**fs== 12.1**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nBusiness Review, 316-326\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n•  Dos  Santos  BL,  Peffers  K  and  D  Mauer  (1993)  ‘The  Impact  of  Information\nTechnology  on  the  Market  Value  of  the  Firm’,  Information  Systems  Research,  4\n(March), 1-23\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\n•  Gal-Or,  E.  &  A  Ghose  (2003).  ‘The  Economic  Consequences  of  Sharing  Security\nInformation’, In 2nd Workshop on Economics and Information Security, May 29-30.\n•  Gordon LA & MP Loeb(2002) ‘The Economics of Information Security Investments’\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nACM Transactions on Information and Systems Security,5(4),438-457\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n•  Gordon,  L.A.,  Loeb,  M.P.  &  Lucyshyn,  W. (2002) ‘An Economic Perspective on the\nSharing  of  Information  Related  to  Security  Breaches:  Concepts  and  Empirical\nEvidence’, In The 1stWorkshop on Economics and Information Security, May16-17.\n•  Gordon LA and MP Loeb and T Sohail (2003) ‘A Framework for Using Insurance for\n**BLOCK**fs== 12.1**b== 0.1**t== 0.8**l== 0.2**r== 0.3**\nCyber Risk Management’ Communications of the ACM, 46(3), 81-85\n**BLOCK**fs== 12.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n•  Harter  DE,  Krishnan  MS  and  Slaughter  SA  (2000)  ‘Effects  of  Process  Maturity  on\nQuality,  Cycle  Time,  and  Effort  in  Software  Product  Development’  Management\nScience, 46(4), 451-466\n**BLOCK**fs== 12.0**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\n•  Hendricks  KB  and  Singhal  VR  (1996)  ‘Quality  Awards  and  the  Market  Value  of  the\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nFirm: An Empirical Investigation’ Management Science, 42(2), 415-436\n**BLOCK**fs== 12.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n•  Hendricks KB and Singhal VR (1997) ‘Delays in New Product Introductions and the\nMarket  Value  of  the  Firm:  The  Consequences  of  Being  Late  to  the  Market’,\nManagement Science, 43(4), 422-436\n**BLOCK**fs== 12.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\n•  Hovav  A  and  J  D’Arcy  (2003)  ‘The  Impact  of  Denial-of-Service  Attack\nAnnouncements  of  the  Market  Value  of  Firms’,  Risk  Management  and  Insurance\nReview, 6(2), 97-121\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\n•  Hsiao, C (2002) ‘Analysis of Panel Data’ Cambridge University Press\n•\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.1**\nIm  KS,  Dow  KE  and  V  Grover  (2001)  ‘Research  Report:  A  Reexamination  of  IT\nInvestment  and  the  Market  Value  of  the  Firm  –  An  Event  Study  Methodology’,\nInformation Systems Research, 12(1), 103-117\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\n•  Jarrell  G  and  S  Peltzman  (1985)  ‘The  Impact  of  Product  Recalls  on  the  Wealth  of\n**BLOCK**fs== 12.1**b== 0.6**t== 0.4**l== 0.2**r== 0.3**\nSellers’ The Journal of Political Economy, 93(1), 512-536\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n•  Kannan  K  and  R  Telang  (2004)  ‘Market  for  Software  Vulnerabilities?  Think  Again’\n**BLOCK**fs== 12.1**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nManagement Science (Forthcoming).\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n•  Kannan  K,  Rees  J  and  S  Sridhar  (2004)  ‘Reexamining  the  Impact  of  Information\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\nSecurity Breach Announcements on Firm Performance’ Working Paper\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n•  Lane  V  and  R  Jacobson  (1995)  ‘Stock  Market  Reactions  to  Brand  Extension\nAnnouncements: The Effects of Brand Attitude and Familiarity’, Journal of Marketing,\n59 (January), 63-77\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\n•  McGraw G (2004) ‘Software Security’ IEEE Security and Privacy, 2(2), 80-83\n•  Malatesta, P and R Thompson (1985) ‘Partially Anticipated Events: A Model of Stock\nPrice Reactions with an Application to Corporate Acquisitions’, Journal of Financial\nEconomics, 14(2), 237-250\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Paulk M Weber C, Curtis W and Chrissis M (1994) ‘The Capability Maturity Model:\nGuidelines for Improving the Software Process’ Carnegie Mellon University : Software\nEngineering Institute\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n•  Pfleegar CP (1997) ‘The Fundamentals of Information Security’ IEEE Software, 14(1),\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n15-17\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n•  Potter  B  and  G  McGraw  (2004)  ‘Software  Security  Testing’  IEEE  Security  and\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nPrivacy, 2(5), 81-85\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n•  Slaughter SA, DE Harter and MS Krishnan (1998) ‘Evaluating the Cost of Software\n**BLOCK**fs== 12.1**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nQuality’ Communications of the ACM, 41(8), 67-73\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\n•  Wallace L, Keil M and A Rai (2004) ‘How Software Project Risk Affects Project\n**BLOCK**fs== 12.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nPerformance: An Investigation of the Dimensions of Risk and An Exploratory Model’,\nDecision Scineces, 35(2), 289-321\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.2**\n•  Wang H and C Wang (2003) ‘Taxonomy of Security Considerations and Software\n**BLOCK**fs== 12.1**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\nQuality’ Communications of the ACM, 46(6), 75-78\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n•  Westland, JC (2003) ‘The Cost Behavior of Software Defects’, Decision Sciences, 37,\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n229-238",
         "extracted",
         "",
         "Impact of Software Vulnerability Announcements on the Market Value of Software Vendors - an Empirical Investigation",
         "None"
        ],
        [
         "27",
         "0023cadbf311b9bffb7b283fd2e2edded70b640d",
         "ABSTRACT Reliable and sensitive quantification of Phytophthora capsici in pepper plants is of crucial importance in managing the multiple syndromes caused by this pathogen. A real-time polymerase chain reaction (PCR) assay was developed for the determination of P. capsici in pepper tissues. DNA levels of a highly virulent and a less virulent isolate were measured in different pepper genotypes with varying degrees of resistance. Using SYBR Green and specific primers for P. capsici, the minimal amount of pathogen DNA quantified was 10 pg. Pathogen DNA was recorded as early as 8 h postinoculation. Thereafter, the increase was rapid in susceptible cultivars and slower in resistant ones. The amount of pathogen DNA quantified in each pepper genotype correlated with susceptibility to Phytophthora root rot. Likewise, there was a relationship between the virulence of the pathogen and the degree of colonization. Differences also were found in oomycete amount among pepper tissues, with maximal pathogen biomass occurring in stems. The real-time PCR technique developed in this study was sensitive and robust enough to assess both pathogen development and resistance to Phytophthora root rot in different pepper genotypes.",
         "C. Silvar,J. Diaz,F. Merino",
         "https://apsjournals.apsnet.org/doi/pdf/10.1094/PHYTO-95-1423",
         "\n**BLOCK**fs== 9.5**b== 0.9**t== 0.1**l== 0.5**r== 0.4**\nTechniques\n**BLOCK**fs== 16.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nReal-Time Polymerase Chain Reaction Quantification\nof Phytophthora capsici in Different Pepper Genotypes\n**BLOCK**fs== 10.5**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nC. Silvar, J. Díaz, and F. Merino\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nDepartamento de Bioloxía Animal, Bioloxía Vexetal e Ecoloxía, Universidade da Coruña, A Coruña, Spain.\nAccepted for publication 8 August 2005.\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nABSTRACT\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nSilvar,  C.,  Díaz,  J.,  and  Merino,  F.  2005.  Real-time  polymerase  chain\nreaction quantification of Phytophthora capsici in different pepper geno-\ntypes. Phytopathology 95:1423-1429.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nReliable and sensitive quantification of Phytophthora capsici in pepper\nplants is of crucial importance in managing the multiple syndromes caused\nby this pathogen. A real-time polymerase chain reaction (PCR) assay was\ndeveloped for the determination of P. capsici in pepper tissues. DNA levels\nof a highly virulent and a less virulent isolate were measured in different\npepper genotypes with varying degrees of resistance. Using SYBR Green\nand specific primers for P. capsici, the minimal amount of pathogen DNA\nquantified  was  10  pg.  Pathogen  DNA  was  recorded  as  early  as  8  h\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.5**r== 0.0**\npostinoculation. Thereafter, the increase was rapid in susceptible cultivars\nand slower in resistant ones. The amount of pathogen DNA quantified in\neach pepper genotype correlated with susceptibility to Phytophthora root\nrot.  Likewise,  there  was  a  relationship  between  the  virulence  of  the\npathogen and the degree of colonization. Differences also were found in\noomycete amount among pepper tissues, with maximal pathogen biomass\noccurring in stems. The real-time PCR technique developed in this study\nwas sensitive and robust enough to assess both pathogen development and\nresistance to Phytophthora root rot in different pepper genotypes.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.5**r== 0.0**\nAdditional  keywords:  Capsicum  annuum,  PI201234,  Serrano  Criollo  de\nMorelos 331,Yolo Wonder.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nIn a plant–pathogen interaction where the result is disease, the\ndetection and quantification of pathogen biomass in the plants is\ncrucial. Traditionally, the common methods used to evaluate dis-\nease  severity  consisted  of  scoring  visual  symptoms  displayed  by\nthe  plant.  For  pathogens  able  to  kill  plants  completely,  survival\nrates  are  scored  to  evaluate  disease  resistance. Alternatively,  for\npathogens causing necrosis without killing the host, the degree of\nnecrosis over time is used as a measure to monitor resistance (9).\nHowever, these techniques have several limitations. First, they can\nbe used only when the pathogen–host interaction results in visible\ndisease symptoms, which is a limiting factor in the earliest stages\nof  infection.  Second,  there  is  not  always  a  straightforward  rela-\ntionship between disease symptoms and pathogen development in\nplants (24,53). Indeed, a tolerant host can show no disease symp-\ntoms,  without  reduction  in  pathogen  growth  within  the  plant\n(12,42).  Therefore,  a  method  for  direct  assessment  of  pathogen\ndevelopment in infected plants was necessary.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nSeveral techniques have been developed with the aim of quanti-\nfying pathogen biomass. Some of these are based on the measure-\nment of compounds such as ergosterol or chitin (18,21,32,40,43)\nwhich are present in the pathogen but not in the host. The prob-\nlem with these methods is the lack of specificity and the fact that\nthey  can  be  used  only  to  assess  colonization  by  pathogens  that\npossess  these  constituents  (3).  Another  group  of  techniques  in-\ncludes  the  use  of  immunological  methods  for  pathogen  biomass\ndetermination  (13,22,29,38).  However,  among  the  drawbacks  of\nthese  methods  are  the  difficulty  in  obtaining  antibodies  with  the\nrequired  specificity  for  particular  fungi  (56),  false-positive  reac-\ntions (51,52), and interference of plant factors with the measured\nparameter (35,53).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nIn  recent  years,  the  polymerase  chain  reaction  (PCR)  method\nhas been used successfully to detect and identify fungal pathogens\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nCorresponding author: F. Merino; E-mail address: fuenme@udc.es\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nDOI: 10.1094 / PHYTO-95-1423\n© 2005 The American Phytopathological Society\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.5**r== 0.0**\n(48). Among the PCR-based methods, real-time PCR has proven\nto  be  a  simple  yet  reliable  technique  to  quantify  viral,  bacterial,\nfungal,  and  oomycete  plant  pathogens,  and  increasingly  is  being\nused  in  plant  pathology  (5,33,37,47,55).  In  this  technique,  the\nDNA of a specific target organism can be quantified by measure-\nment of the intensity  of fluorescence with time during the  expo-\nnential  phase  of  DNA  amplification.  Quantification  of  DNA  in\nunknown  samples  will  be  possible  by  direct  comparison  with\nstandards amplified in parallel reactions (36). The main advantage\nof this technique compared with the conventional one, end-point\nquantitative PCR, is that the amplification products can be moni-\ntored as they are accumulated  in the log-linear phase of amplifi-\ncation.  Therefore,  it  is  more  accurate,  less  time-consuming,  and\nbetter  suited,  especially  when  the  objective  is  to  discriminate\nbetween slightly different levels of infection (58).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.0**\nPhytophthora root rot, caused by Phytophthora capsici Leonian,\nis  the  most  destructive  disease  of  pepper  crops  in  Spain  (39,44).\nIn  susceptible  plants,  the  pathogen  penetrates  through  the  collar,\ncausing necrosis in that area. At the same time, the plant becomes\nwithered  while  the  pathogen  advances  systemically  toward  both\nthe  stem  and  the  root,  causing  an  obstruction  in  the  vascular\nsystem. The infected tissues become dry, sunken, and parchment-\nlike, and turn dark in color. Finally, the plant dies at the last stage\nof  infection. Although  the  most  effective  method  to  combat  the\npathogen  is  based  on  the  use  of  fungicides,  some  cultivars  with\ndifferent  levels  of  resistance  to  P.  capsici  have  been  reported.\nIndeed, several different screening methods have been developed\nto distinguish Phytophthora-resistant individuals from susceptible\nones (1,7,8,19). Different sources of resistance to P. capsici have\nbeen reported in Capsicum annuum L.; among the genotypes that\nexhibit  resistance  to  Phytophthora  root  rot  are  United  States\nDepartment  of  Agriculture  (USDA)  PI201234  and  USDA\nPI201232, as well as Serrano Criollo de Morelos 331 (SCM331)\nand SCM334, the latter being more resistant to P. capsici than the\nformer (4,20,41).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.0**\nIn a previous study (50), we developed a PCR procedure for the\nearly detection of P. capsici in pepper plants. Specific primers for\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nP. capsici, based on the internal transcribed spacer (ITS) region of\nribosomal DNA, were designed and tested for their sensitivity and\nspecificity  as  well  as  for  their  ability  to  detect  the  pathogen  in\nartificially and naturally infected pepper plants (50). The develop-\nment of a real-time approach enabled the quantification of patho-\ngen  biomass  in  plants  over  the  course  of  infection. This  method\nwill  facilitate  the  characterization  of  host–pathogen  interactions.\nThe aims of the present work were to (i) develop a real-time PCR\nassay  to  quantify  P.  capsici  biomass  in  pepper  plants  and  (ii)\nmonitor pathogen growth in pepper plants in relation to different\nlevels of pathogen virulence and host susceptibility.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nMATERIALS AND METHODS\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nPathogen material. In recent works carried out in our labora-\ntory  (C.  Silvar,  F.  Merino,  and  J.  Díaz,  unpublished  data),  a\npopulation  of  Spanish  P.  capsici  isolates  was  characterized  ac-\ncording  to  their  virulence.  Two  different  isolates,  one  highly\nvirulent  (UDC196Pc)  and  one  less  virulent  (UDC248Pc),  were\nselected  for  this  work.  The  isolates  were  maintained  on  potato\ndextrose  agar  (PDA)  for  further  use.  To  obtain  mycelium,  the\nisolates were grown in still culture in pea broth (11). After 1 week\nof  incubation  at  24°C,  mycelium  was  collected  and  lyophilized\nfor extended storage.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nPlant material. Three different cultivars of pepper plants were\nused:  susceptible  cv. Yolo  Wonder,  and  SCM331  and  PI201234,\neach  with  different  levels  of  resistance  to  P.  capsici,  SCM331\nbeing the most resistant (4). Moreover, a susceptible local cultivar\n(Padrón)  was  included  because  of  its  economic  value  in  north-\nwestern Spain. Plants were transferred into pots and grown sepa-\nrately in a chamber at 25°C and a photoperiod of 16 h of light and\n8 h of darkness until they were 3 weeks old.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nPathogen  inoculation,  quantification  bioassay,  and  disease\nassessment.  A  suspension  of  zoospores  from  UDC196Pc  and\nUDC248Pc were obtained according to Larkin et al. (30). Pepper\nplants were inoculated with a concentration of 104 zoospores ml–1\nby  adding  5  ml  of  the  inoculum  into  the  soil.  For  the  quantifi-\ncation of pathogen DNA, samples of leaves, stems, and roots were\ncollected  at  8,  24,  48,  and  72  h  postinoculation  in  the  different\ncultivars. Three independent experiments were carried out.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nIn parallel, assays were carried out with the aim of studying the\ndisease  progression  in  different  cultivars  during  a  time  period.\nWith this purpose, 10 plants of each cultivar were inoculated with\nthe  two  different  isolates  (UDC196Pc  and  UDC248Pc).  Disease\nseverity was rated for 20 days after inoculation on a scale of 0 to\n5, where 0 = no visible symptoms; 1 = ≤50% of stem with water-\nsoaked lesions, ≤50% of leaves wilted, or both; 2 = >50% of stem\nwith  water-soaked  lesions  or  >50%  of  leaves  wilted;  3  =  both\n≤50%  of  stem  with  water-soaked  lesions  and  >50%  of  leaves\nwilted or both >50% of stem with water-soaked lesions and ≤50%\nof leaves wilted; 4 = >50% of stem with water-soaked lesions and\n>50% of leaves wilted; and 5 = dead plant. Percentage of diseased\nplants (incidence) also was recorded during the time course of in-\nfection. In this case, two independent experiments were performed.\nIsolation of P. capsici from artificially infected plants. Colo-\nnization of pepper plants by P. capsici was determined in each of\nthe  pepper  cultivars  studied  by  isolating  the  pathogen  on  PDA.\nDuring  the  host–plant  infection  assay,  samples  of  plants  were\ncollected at 8, 24, 48, and 72 h postinoculation. Entire plants were\nsurface  disinfected  with  NaClO  (10%)  for  5  min,  washed  with\ndistilled water,  dried,  and plated onto PDA  medium. Plates were\nincubated at 24°C in the dark for 7 days.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nDNA extraction. For DNA extraction, both pathogen and plant\nmaterial (0.2 g fresh weight [FW]) were ground with liquid nitro-\ngen  to  a  fine  powder  using  a  mortar  and  pestle. All  the  powder\nwas transferred to an Eppendorf tube containing extraction buffer\n(0.1 M Tris-HCl, pH 7, 1.4 M NaCl, 0.2% β-mercaptoethanol, 1%\npolyvinyl  pyrrolidone,  and  2%  cetyltrimethylammonium  bro-\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.0**\nmide).  Subsequently,  extraction  with  phenol/chloroform/isoamyl\nalcohol (25:24:1, vol/vol) was carried out. DNA was precipitated\nwith  isopropanol,  washed  with  cold  ethanol  (70%),  and  resus-\npended in sterile distilled water.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.2**l== 0.5**r== 0.0**\nReal-time PCR assay. Real-time PCR assays for quantification\nof both isolates in different cultivars were conducted using primer\npair CAPFW/CAPRV1 (50), which defines an amplicon of 452 bp\nspecific  for  P.  capsici.  Real-time  PCR  was  performed  in  an\niCycler  (BioRad,  Hercules,  CA)  apparatus  and  results  were  ana-\nlyzed with the manufacturer’s software (Optical System Software,\nv. 3.0; BioRad). Each reaction mixture (20 µl) contained 2.5 µl of\nundiluted DNA extract, 1× iQ SYBR Green Supermix (BioRad),\nand 0.3 µM of each primer. In negative controls, the DNA extracts\nwere  obtained  from  noninoculated  plants  of  the  same  age  and\nsame  culture  conditions  as  inoculated  ones.  Additional  negative\ncontrol  reactions  contained  the  same  mixtures  with  2.5  µl  of\nsterile  water  replacing  the  DNA. The  thermal  cycling  conditions\nconsisted of an initial denaturation at 95°C for 2 min followed by\n40  cycles  at  95°C  for  30  s,  56°C  for  30  s,  72°C  for  1  min,  and\n84°C for 15 s. A final extension step at 72°C for 5 min was added.\nFluorescence of the target amplicon (melting temperature (Tm) =\n86.5°C) was detected at 84°C. After the final amplification cycle,\na  melting  curve  temperature  profile  was  obtained  by  heating  to\n95°C, cooling to 65°C, and slowly heating to 95°C at 0.5°C every\n10 s with continuous measurement of fluorescence at 520 nm. A\nstandard  curve  was  constructed  by  plotting  the  logarithm  of\n10-fold  serial  dilutions  from  100  to  10–3  ng  of  P.  capsici  DNA\nagainst the cycle threshold (Ct) values. The Ct value is the cycle\nnumber at which the fluorescence emission of the PCR amplicon\ncould be distinguished from the background.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.0**\nIn  another  assay,  a  10-fold  serial  dilution  of  genomic  DNA\nstarting from 100 ng was added to samples containing 100 ng of\npepper genomic DNA. This was done in order to test whether the\npresence  of  samples  of  plant  DNA  would  interfere  with  the  ac-\ncurate  quantification.  When  constructing  a  regression  line  with\nthese data, a linear relationship was observed, as happened in the\nstandard  curve  (data  not  shown).  From  these  results,  it  was\nconcluded  that  addition  of  host  DNA  has  no  effect  on  pathogen\nDNA quantification.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.0**\nThe amount of pathogen DNA in unknown samples was calcu-\nlated  by  monitoring  the  Ct  value  at  which  the  fluorescence\nemission  could  be  differentiated  from  the  background  and  com-\nparing this Ct value with the point values of the linear regression\nline of the standard curve. Furthermore, to confirm that only one\nPCR product was amplified in the inoculated plants and no ampli-\nfication  product  was  obtained  in  the  negative  controls,  all  reac-\ntions  were  analyzed  by  electrophoresis  in  a  1%  (wt/vol)  agarose\ngel in 1× Tris-borate-EDTA buffer (31) and stained with ethidium\nbromide (5 µg ml–1) for visualization.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.0**\nStatistical analyses. Statistical analyses were performed using\nStatgraphics Plus for Windows, 5.1 Professional Version (StatPoint,\nInc., Herndon, VA). Data of pathogen DNA quantification among\ncultivars  were  analyzed  using  the  Kruskal-Wallis  test  (P  ≤  0.05)\nfollowed by the multiple comparison procedure of Conover (10).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nRESULTS\n**BLOCK**fs== 9.5**b== 0.0**t== 0.8**l== 0.5**r== 0.0**\nDisease assessment in different pepper cultivars. The capac-\nity of the two P. capsici isolates to infect pepper plants was tested\nin  a  group  of  four  pepper  genotypes,  differing  in  their  degree  of\nsusceptibility  to  the  pathogen.  Data  on  disease  severity  were\nplotted  over  time  to  obtain  disease  progress  curves  (Fig.  1).  In\ngeneral,  differences  could  be  observed  among  cultivars,  isolates,\nand  cultivar–isolate  interactions.  Susceptible  cv.  Yolo  Wonder\nshowed high disease severity with both P. capsici isolates. Seed-\nlings were attacked primarily in the collar region and the infection\nrapidly  expanded  up  the  hypocotyl,  reaching  the  cotyledons\nwithin a few days. Host plants revealed the first visible symptoms\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\non the fourth day after inoculation, although the disease progress\nover time was higher in plants inoculated with UDC196Pc than in\nthose infected with UDC248Pc. In the first case, a maximal infec-\ntion (severity index = 4.8) was reached at 17 days after infection\n(Fig. 1A) and the percentage  of infected plants at the end of the\nbioassay  was  95%  (data  not  shown).  In  the  Yolo  Wonder–\nUDC248Pc interaction, however, disease progression was slower\nand the severity rating achieved at the end of the ongoing period\nwas only 1.5 (Fig. 1B), with  only 30%  diseased  plants (data not\nshown). After  inoculation  with  UDC196Pc,  the  infection  pattern\ndisplayed by local cv. Padrón was similar to that in Yolo Wonder,\nbut showed the highest degree of susceptibility (severity rate = 5\nand  percentage  of  infected  plants  =  100%)  (Fig.  1A).  However,\nafter  inoculation  with  UDC248Pc,  the  severity  achieved  by Yolo\nWonder was higher than in Padrón (Fig. 1B).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nIn  contrast,  resistant  genotypes  showed  the  lowest  disease\nseverity.  No  disease  symptoms  were  observed  when  the  plants\nwere infected with UDC248Pc, and slower progress curves were\nobtained in the PI201234– or SCM331–UDC196Pc system com-\npared  with  those  in  susceptible  cultivars.  In  the  last  case,  more\nsymptoms developed in PI201234 than in SCM331, with 40 and\n20%  disease  percentage  (data  not  shown)  and  2  and  1  severity\nindex, respectively, at the  end  of the trials.  Noninoculated plants\ndid not show any symptoms (data not shown).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nP. capsici was isolated successfully from leaves, stems, and roots\nin  all  pepper  genotypes  inoculated  with  both  isolates  at  24,  48,\nand  72  h  after  inoculation.  However,  no  pathogen  isolation  was\npossible at 8 h in any pepper–P. capsici system (data not shown).\nReal-time PCR adjustment and DNA standard curves. Pre-\nliminary runs of real-time PCR without the step at 84°C for 15 s\nwere  carried  out  with  the  aim  of  detecting  some  possible  non-\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.0**\nspecific  amplification  products  or  primer-dimers.  Melting  curve\nprofiles  from  those  runs  revealed  the  presence  of  such  products,\nwith a Tm ≤ 82°C. Therefore, to avoid any possible disturbance in\nthe  quantification  of  the  target  amplicon  (Tm  =  86.5°C),  an  ad-\nditional  step  (84°C  for  15  s)  was  included  in  the  real-time  PCR\namplification  program  and  the  reading  of  the  fluorescence  emis-\nsion was set at this point. Additionally, to confirm the presence of\nthe  single  expected  amplicon  of  452  bp,  final  products  of  real-\ntime  PCR  from  both  standard  curve  and  unknown  samples  were\nvisualized in agarose gels (data not shown).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.0**\nA  standard  curve  was  constructed  by  plotting  known  concen-\ntrations  of  P.  capsici  DNA  against  the  Ct  values  obtained  from\nreal-time PCR (Fig. 2). In order to validate the reproducibility of\nthe  real-time  PCR  method,  PCR  amplifications  were  conducted\nwith duplicate samples of pathogen DNA obtained from four dif-\nferent  preparations.  Highly  reproducible  Ct  values  with  very\nsmall standard deviations were observed, the linear regression co-\nefficient of the  standard curve  being r  = 0.968. Within this  stan-\ndard  curve,  each  10-fold  difference  in  initial  DNA  amounts  was\nrepresented by approximately four-cycle differences in Ct. Under\nthe  PCR  conditions  tested,  the  minimum  starting  quantity  of\npathogen  DNA  that  could  be  accurately  quantified  in  our  assays\nwas 10–2 ng, which corresponded to a Ct value of 36.97 ± 1.06. A\nCt  of  37  (6  ×  10–3  ng  of  DNA  according  to  the  standard  curve)\nwas  considered  to  be  the  threshold  value  suitable  for  quanti-\nfication, because PCR efficiency at a higher number of cycles was\nfar from the value 1 (100% efficiency).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.5**r== 0.0**\nQuantification of P. capsici DNA in pepper plants. Pathogen\nDNA  was  quantified  in  plants  using  real-time  PCR  with  SYBR\nGreen  before  first  symptoms  of  the  disease  were  observed  in\nplants  (Figs.  3,  4,  and  5).  Indeed,  symptoms  did  not  appear  in\ninfected  plants  until  4  days  after  inoculation;  however,  with  the\nreal-time PCR assay, pathogen DNA could be quantified as early\nas  8  h  postinoculation  in  all  the  organs  of  the  plant.  From  that\ntime  point,  the  pathogen  DNA  in  pepper  organs  increased  pro-\ngressively  until  the  end  of  the  bioassay.  Real-time  PCR  quanti-\nfication showed important differences among the different organs.\nIn general, stems were more heavily colonized than roots, and the\nlatter  more  than  leaves.  In  all  cases,  the  maximal  amount  of\npathogen  DNA  occurred  at  72  h  after  inoculation. Although  the\npathogen  was  able  to  colonize  all  the  susceptible  and  resistant\npepper  cultivars  evaluated,  the  extent  of  colonization  varied\nhighly among cultivars with different levels of resistance. In most\nof the cases, higher amounts of pathogen DNA were present at 72\nh  after  inoculation  in  susceptible  cultivars  (Padrón  and  Yolo\nWonder) than in the more resistant ones (PI201234 and SCM331).\nIn  the  susceptible  cultivars,  pathogen  DNA  was  detected  earlier\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nFig.  1.  Disease  progress  curves  of  pepper  cultivars  inoculated  with  A,\nUDC196Pc  or  B,  UDC248Pc.  Each  plotted  point  is  the  mean  value  of  20\nplants  from  two  independent  experiments.  Padrón  (!),  Yolo  Wonder  (\"),\nPI201234 (), and SCM331 ().\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.0**\nFig.  2.  Standard  curve  for  real-time  polymerase  chain  reaction  analysis  of  a\n10-fold serial dilution of Phytophthora capsici DNA.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nand  increased  sharply  over  time  whereas,  in  the  resistant  plants,\nthe variations in pathogen development were slighter. Significant\n(P ≤ 0.05) differences were found among resistant and susceptible\ncultivars  as  early  as  24  h  postinoculation  in  the  stem  for  the\nUDC196Pc  isolate  (Fig.  4A)  although,  at  48  h,  there  was  no\nsignificant  differences.  The  differences  were  significant  at  72  h\npostinoculation in all the organs and isolates, except for the com-\nbination of the roots and the isolate UDC196Pc (Fig. 5A). Com-\nparing  the  cultivars  separately,  susceptible  cvs.  Padrón  and Yolo\nWonder  displayed  a  similar  pattern  of  pathogen  invasion,  but\nsignificant differences between them were found in the roots at 48\nand 72 h postinoculation (Fig. 5A and B). For the resistant culti-\nvars,  significant  differences  were  found  in  SCM331  compared\nwith the partially resistant cv. PI201234 only in the case of roots\nat 48 h postinoculation (Fig. 5A and B). In any case, the accumu-\nlation  of  pathogen  DNA  in  these  resistant  genotypes  was  slower\nthan  that  in  susceptible  cultivars  and  the  amount  of  pathogen\nDNA generally remained lower during the study.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nWith regard to the two P. capsici isolates, a different pattern of\ninfection  was  observed.  Up  to  5.9  ng  g–1  FW  of  pathogen  DNA\nwas  observed  as  soon  as  8  h  postinoculation  in  the  Padrón–  or\nYolo  Wonder–UDC196Pc  interactions  (Fig.  4A)  whereas,  in\nsusceptible cultivar–UDC248Pc interactions, the maximal amount\nof DNA detectable was 0.2 ng g–1 FW (Fig. 4B). In the same way,\nthe  average  pathogen  DNA  in  the  resistant  cultivar–UDC196Pc\nsystem  was  1.4  ng  g–1  FW,  whereas  it  could  not  be  detected  in\nPI201234 or in SCM331 until 24 h after infection, when the plants\nwere  inoculated  with  the  least  virulent  pathogen  (UDC248Pc).\nDuring the remaining period, the amount of UDC196Pc (the iso-\nlate previously characterized as more virulent) DNA was up to 10\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.0**\ntimes higher than UDC248Pc (less virulent) in all cultivars evalu-\nated. These results correspond with those obtained in the severity\nassessment  assay,  where  clear  differences  between  isolates  also\nwere observed.\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nDISCUSSION\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.0**\nOne of the objectives of this study was the development of an\naccurate  quantification  method  of  pathogen  colonization  in  host\ntissue using real-time PCR technology and the use of this method\nfor  a  better  understanding  of  the  interaction  between  different\npepper genotypes and P. capsici isolates. With this purpose, four\npepper cultivars were artificially infected with two P. capsici iso-\nlates, differing in their degree of virulence toward pepper plants.\nPathogen  DNA  was  monitored  and  quantified  in  a  time  course\nafter inoculation using a quantitative SYBR Green PCR assay. In\nrecent years, real-time PCR assays have been widely used for the\ndetection  and  quantification  of  fungal  plant  pathogens  (2,15,48).\nSimilar  studies  have  been  carried  out  in  Phytophthora  spp.  (5,6,\n23,26,55).  However,  to  the  best  of  our  knowledge,  no  attempts\nhad been made to apply the real-time PCR technology for quanti-\nfication of P. capsici in pepper plants.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.5**r== 0.0**\nIn  our  study,  the  results  showed  that  real-time  PCR  can  be\nsuccessfully implemented for the quantification of pepper coloni-\nzation  by  P.  capsici. The  technique  was  highly  reproducible,  the\ncorrelation  coefficient  of  the  standard  curve  constructed  with\ndifferent  replications  being  0.968. At  the  same  time,  the  method\nwas  quite  sensitive,  being  able  to  readily  detect,  using  the  un-\nspecific SYBR Green dye and specific primers to P. capsici, up to\n10–2 ng of pathogen DNA. Similar quantification limits have been\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nFig. 3. Phytophthora capsici DNA quantification in leaves of different pepper\ncultivars inoculated with A,  UDC196Pc  or B,  UDC248Pc. Vertical  arrow ()\nindicates DNA samples (PI201234 and SCM 331) that could not be quantified.\nData  are  the  mean  and  standard  error  of  three  independent  experiments.\nWithin  the  same  time  postinoculation,  different  letters  indicate  significant\ndifferences between cultivars in a Kruskal-Wallis test (P ≤ 0.05) followed by\nthe multiple comparison procedure of Conover (10).\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.0**\nFig. 4. Phytophthora capsici DNA quantification in stems of different pepper\ncultivars inoculated with A,  UDC196Pc  or B,  UDC248Pc. Vertical  arrow ()\nindicates DNA samples (PI201234 and SCM 331) that could not be quantified.\nData  are  the  mean  and  standard  error  of  three  independent  experiments.\nWithin  the  same  time  postinoculation,  different  letters  indicate  significant\ndifferences between cultivars in a Kruskal-Wallis test (P ≤ 0.05) followed by\nthe multiple comparison procedure of Conover (10).\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nfound  in  other  pathogen–plant  interactions  (15–17)  as  well  as  in\nother  Phytophthora  spp.,  although  using  different  real-time\nchemistries  (26).  The  Ct  values  depend  on  both  the  amount  of\nstarting  DNA  and  the  genomic  organization  of  the  target  se-\nquences  recognized  by  the  sequence-specific  primer  pair;  there-\nfore,  the  efficiency  of  the  technique  will  improve  if  the  target\nDNA is present in many copies in the genome, as is the case with\nrDNA.  This  sensitivity  was  similar  to  that  achieved  in  previous\nworks using conventional PCR but less than that obtained with a\nnested PCR assay (50). These results agree with those published\nrecently  by  Bonants  et  al.  (6)  working  on  P.  fragariae.  They\nobtained  equivalent  detection  limits  by  using  either  conventional\nPCR or a TaqMan assay but higher sensitivity with a nested PCR.\nIn any case, with this sensitivity, pathogen abundance can be re-\nliably  quantified  from  the  very  beginning  of  the  infection  time\ncourse and even in tissues exhibiting a low degree of infection.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nIn  a  parallel  assay,  the  influence  of  plant  samples  on  quanti-\nfication was tested (data not shown). Serial dilutions (10-fold) of\npathogen DNA were mixed with an equal amount of plant DNA.\nThis  mixture  possibly  could  reflect  an  infection  from  a  range  of\n1:1 (a very heavy infection) to a range of 1:106 (an early stage of\ninfection). The presence of a host sample of DNA did not produce\nany interference in the quantification. Furthermore, with this tech-\nnique, the sensitivity of the method was not affected by the pres-\nence  of  a  plant  DNA  sample,  as  occurs  with  conventional  and\nnested PCR (50).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nIn a real-time PCR assay, the accumulation of the amplicon can\nbe  detected  using  either  specific  detection  methods  such  as\nTaqMan  probes  (25,54,57)  or  nonspecific  methods  like  SYBR\nGreen (36). The advantage of SYBR Green is that, in contrast to\nfluorogenic probes, it does not require the design of specific com-\nplementary fragments to the target DNA and, therefore, the reac-\ntion cost is limited compared with real-time PCRs that depend on\nhybridization of labeled  sequence-specific probes. The  main dis-\nadvantage  of  this  method  is  that  it  binds  double-stranded  DNA\nand, therefore, it also can associate with primer-dimers and non-\nspecific products, which can seriously disturb the interpretation of\nthe results. However, this major drawback can be overcome with\nthe  use  of  melting  curve  temperature  analysis  at  the  end  of  the\nPCR assay (46). At the end of the cycling reactions, the products\nare melted by slowly increasing the temperature while continually\nmonitoring  the  fluorescence. After  melting  curve  analysis  in  our\nassay, the use of unspecific SYBR Green dye for quantifying the\ntarget amplicon (≈452 bp, Tm = 86.5°C) required a 84°C fluores-\ncence emission reading step in the amplification program to elimi-\nnate  any  interference  caused  by  the  accumulation  of  unspecific\nPCR products. At that temperature, the nonspecific products will\nbe  denatured  and  only  the  fluorogenic  emission  from  the  target\namplicon would result.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nOne of the results in this work is that the real-time PCR method\ndeveloped  here  is  able  to  quantify  pathogen  invasion  in  the\nabsence of visible host plant symptoms. Although the first visible\nsymptoms appeared in plants by day four after infection, quanti-\nfication  was  possible  at  earlier  times  (as  soon  as  8  h  postinocu-\nlation)  in  totally  asymptomatic  plants. Thereafter,  the  amount  of\npathogen  DNA  either  increased  sharply  or  varied  more  slightly\nduring  the  study  period,  but  in  no  case  were  the  symptoms  ob-\nserved in inoculated plants. It must be pointed out that P. capsici\nis detected in leaves at 8 h postinoculation. It is surprising that the\npathogen can invade and colonize the host so fast. This fact prob-\nably is due to the pathogen’s ability to move through the vascular\nsystem and the small size of the plants at the time of the experi-\nments. It was also an unexpected result that the pathogen was not\nisolated at 8 h after inoculation, taking into account that pathogen\nDNA  was detected at that time.  Maybe nonviable pathogen cells\nwere present at that time. Another possibility is that the pathogen\ncolonized  only  the  outer  layers  of  cells  at  that  time,  after  which\nsurface sterilization could have killed the pathogen.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.1**l== 0.5**r== 0.0**\nThe amount of pathogen DNA quantified 72 h after inoculation\nin each pepper genotype correlated with the level of susceptibility\nor  resistance  to  disease  caused  by  P.  capsici  (as  indicated  by\nseverity  of  symptoms).  Maximal  absolute  amounts  of  pathogen\nDNA  were  quantified  in  susceptible  genotypes Yolo Wonder  and\nPadrón  (the  most  severely  diseased  genotypes),  where  measure-\nments in stems ranged from 3.3 × 105 ng g–1 FW in UDC196Pc–\nYolo  Wonder  to  1.6  ×  105  ng  g–1  FW  in  UDC196Pc–Padron.\nActual amounts of P. capsici DNA varied considerably for a given\npepper genotype–P. capsici isolate interaction and, in some cases,\nbetween  P.  capsici  isolates  for  a  given  pepper  genotype.  In  the\nsame way, smaller amounts of pathogen DNA were quantified in\nPI201234 and  SCM331 plants, in that order. These two cultivars\nalso  produced  fewer  diseased  plants  or  even  none  at  all.  There-\nfore,  reduced  disease  severity  in  those  cultivars  considered\nresistant corresponded to the results of the quantitative analysis of\npathogen  DNA.  In  recent  years,  several  different  studies  have\ndemonstrated  the  utility  of  real-time  PCR  to  discriminate  be-\ntween  plant  genotypes  displaying  different  levels  of  resistance\n(9,17,34,45,59). Vandemark and Barker (55), studying the P. medi-\ncaginis–alfalfa  interaction,  were  able  to  discriminate  among\nthree alfalfa populations with  different levels  of resistance. They\nalso  detected  less  pathogen  DNA  in  highly  resistant  alfalfa\npopulations  than  in  more  susceptible  ones.  Moreover,  the  re-\nsults  of  our  work  corroborate  others  published  previously  which\nestablished  clear  differences  in  resistance  among  SCM331  and\nPI201234  (resistant  cultivars)  and  Yolo  Wonder  (susceptible\ngenotype)  (4,19,20).  The  difference  with  the  present  work\nis  that  previous  classifications  of  peppers  susceptible  or  re-\nsistant  to  Phytophthora  root  rot  were  based  on  the  evaluation  of\nexternal  symptoms  and  measurement  of  incidence  and  severity\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.0**\nFig. 5.  Phytophthora capsici DNA  quantification in roots of different  pepper\ncultivars inoculated with A,  UDC196Pc  or B,  UDC248Pc. Vertical  arrow ()\nindicates DNA samples (PI201234 and SCM 331) that could not be quantified.\nData are the mean and standard error of three independent experiments. With-\nin  the  same  time  postinoculation,  different  letters  indicate  significant  differ-\nences  between  cultivars  in  a  Kruskal-Wallis  test  (P  ≤ 0.05)  followed  by  the\nmultiple comparison procedure of Conover (10).\n**BLOCK**fs== 9.5**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nof  plant  symptoms,  which  is  time  consuming  and  sometimes  in-\nconclusive.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nIn terms of pathogen ability to infect the plant, it seemed clear\nthat  the  more  virulent  isolate  had  a  better  capacity  not  only  to\ninfect the plant sooner but also to spread quicker during the time\ncourse  of  infection.  In  general,  the  average  amount  of  pathogen\nDNA  was  higher  for  UDC196Pc-infected  plants  than  for  the\nUDC248Pc-infected  ones.  Moreover,  the  accumulation  of  patho-\ngen DNA was rapid in susceptible genotype–most virulent isolate\ninteractions whereas, in less compatible interactions, it was more\nprogressive  and  notable  increases  in  pathogen  amount  did  not\ntake  place,  the  quantity  of  pathogen  DNA  being  more  constant\nover  time.  Considering  all  these  results,  the  difference  in  viru-\nlence  of  the  two  P.  capsici  isolates  could  be  attributed  to  differ-\nences in pathogen amount in infected plants, among other factors.\nHowever, a clear correlation between the virulence of a pathogen\nand the amount of pathogen present in the plant does not always\nexist.  Sometimes,  more  virulent  isolates  do  not  involve  an  in-\ncrease in pathogen colonization (34,53).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nAnother interesting aspect was the different amounts of patho-\ngen  DNA  in  the  different  tissues  collected.  Comparatively,  the\namount  of  pathogen  DNA  in  leaves  was  much  lower  than  that\nquantified in roots which, in turn, was less than that occurring in\nstems. This tissue difference suggests either a temporal sequence\nof  systemic  colonization  by  the  pathogen  or  a  differential  re-\nsponse of the plant defense mechanisms. Differences in the level\nof  colonization  of  different  tissues  by  the  pathogen  have  been\nreported in other plant–pathogen interactions (34,59). It has been\nreported  that  some  plant  defense-related  genes,  which  encode\npathogenesis-related proteins, display a differential expression in\nvarious  pepper  organs  in  response  to  pathogen  infection  (27,28,\n49).  Such  responses  could  be  responsible  for  that  differential\ndistribution, preventing pathogen advance, especially in the culti-\nvars that are more resistant.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nA final conclusion of the present work was that the local pepper\ncv.  Padrón  displayed  an  infection  and  quantification  pattern\nsimilar  to  that  in  Yolo  Wonder,  suggesting  that  it  may  have  a\nsimilar  degree  of  susceptibility.  Padrón  is  a  pepper  cultivar  of\nhigh  economic  importance  in  Galicia  (northwestern  Spain)  (14).\nThe results in this  study  have  demonstrated its level of suscepti-\nbility  to  Phytophthora  root  rot,  the  most  important  disease  of\npepper  crops  in  this  region  (44).  The  real-time  PCR  developed\nhere, together with disease severity studies, will enable the selec-\ntion  of  the  most  resistant  plants  within  the  Padrón  cultivar  for\ndisease resistance breeding in the future.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nACKNOWLEDGMENTS\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nThis work was supported by grants from MEC (project AGF99-0301)\nand  XUGA  (project  PGDIT99AGR10301).  C.  Silvar  held  a  fellowship\nfrom  MEC.  We  thank  F.  Rodríguez  (SXAIN,  Universidade  da  Coruña,\nSpain)  for  her  technical  assistance  and  T.  Berke  (former  employee  of\nAVRDC, Taiwan) for kindly supplying the PI201234 and SCM331 seed.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nLITERATURE CITED\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n1.  Alcantara, T. P., and Bosland, P. W. 1994. An inexpensive disease screen-\ning  technique  for  foliar  blight  of  chili  pepper  seedlings.  HortScience\n29:1182-1183.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n2.  Bates, J. A., Taylor, E. J. A., Kenyon, D. M., and Thomas, J. E. 2001. The\napplication of real-time PCR to the identification, detection and quantifi-\ncation of Pyrenophora species in barley seed. Mol. Plant Pathol. 2:49-57.\n3.  Bermingham, S., Maltby, L., and Cooke, R. C. 1995. A critical assessment\nof  the  validity  of  ergosterol  as  an  indicator  of  fungal  biomass.  Mycol.\nRes. 99:479-484.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n4.  Black, L. L., and Berke, T. 1998. Breeding for Phytophthora resistance in\npepper. Xth Eucarpia Meeting on Genetics and Breeding of Capsicum and\nEggplant. Avignon, France.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n5.  Böhm, J., Hahn, A., Schubert, R., Bahnweg, G., Adler, N., Nechwatal, J.,\nOehlmann, R., and Obwald, W.  1999. Real-time  quantitative PCR:  DNA\ndetermination  in  isolated  spores  of  the  mycorrhizal  fungus  Glomus\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.6**r== 0.0**\nmosseae  and  monitoring  of  Phytophthora  infestans  and  Phytophthora\ncitricola in their respective host plants. J. Phytopathol. 147:409-416.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.0**\n6.  Bonants, P. J. M., van Gent-Pelzer, M. P. E., Hoofman, R., Cooke, D. E.\nL.,  Guy,  D.  C.,  and  Duncan,  J.  M.  2004. A  combination  of  baiting  and\ndifferent  PCR  formats,  including  measurement  of  real-time  quantitative\nfluorescence,  for  the  detection  of  Phytophthora  fragariae  in  strawberry\nplants. Eur. J. Plant Pathol. 110:689-702.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.0**\n7.  Bosland, P. W., and Lindsey, D. L. 1991. A seedling screen for Phytoph-\nthora root rot of pepper, Capsicum annuum. Plant Dis. 75:1048-1050.\n8.  Bowers, J. H., and Mitchell, D. J. 1991. Relationships between inoculum\nlevel  of  Phytophthora  capsici  and  mortality  of  pepper.  Phytopathology\n81:178-184.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.0**\n9.  Brouwer,  M.,  Lievens,  B.,  van  Hemelrijck, W.,  van  den Ackerveken,  G.,\nCammue,  B.  P.  A.,  and  Thomma,  B.  P.  H.  J.  2003.  Quantification  of\ndisease  progression  of  several  microbial  pathogens  on  Arabidopsis\nthaliana  using  real-time  fluorescence  PCR.  FEMS  Microbiol.  Lett.\n228:241-248.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.0**\n10.  Conover,  W.  J.  1980.  Practical  Nonparametric  Statistics.  John  Wiley  &\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nSons, Inc., New York.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.0**\n11.  Cooke,  D.  E.  L.,  Drenth, A.,  Duncan,  J.  M., Wagels,  G.,  and  Brasier,  C.\nM. 2000. A molecular phylogeny of Phytophthora and related Oomycetes.\nFungal Genet. Biol. 30:17-32.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.0**\n12.  Dan,  H.,  Ali-Khan,  A.  T.,  and  Robb,  J.  2001.  Use  of  quantitative  PCR\ndiagnostics to identify tolerance and resistance to  Verticillium  dahliae in\npotato. Plant Dis. 85:700-705.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.0**\n13.  Dewey,  F.  M., Thornton,  C.  R.,  and  Gilligan,  C. A.  1997.  Use  of  mono-\nclonal antibodies to detect, quantify, and visualize fungi in soil. Adv. Bot.\nRes. 24:276-308.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.0**\n14.  Estrada, B., Bernal, M. A., Díaz, J., Pomar, F., and Merino, F. 2000. Fruit\ndevelopment  in  Capsicum  annuum:  Changes  in  capsaicin,  lignin,  free\nphenolics and peroxidase patterns. J. Agric. Food Chem. 48:6234-6239.\n15.  Filion, M., St.-Arnaud, M., and Jabaji-Hare, S. H. 2003. Quantification of\nFusarium solani f. sp. phaseoli in mycorrhizal bean plants and surround-\ning mycorrhizosphere soil using real-time polymerase chain reaction and\ndirect isolations on selective media. Phytopathology 93:229-235.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.0**\n16.  Filion, M., St-Arnaud, M., and Jabaji-Hare, S. H. 2003. Direct quantifica-\ntion of fungal DNA from soil substrate using real-time PCR. J. Microbiol.\nMethods 53:67-76.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.0**\n17.  Gachon,  C.,  and  Saindrenan,  P.  2004.  Real-time  PCR  monitoring  of\nfungal  development  in  Arabidopsis  thaliana  infected  by  Alternaria\nbrassicicola and Botrytis cinerea. Plant Physiol. Biochem. 42:367-371.\n18.  Gardner,  R.  M.,  Tindall,  G.  W.,  Cline,  S.  M.,  and  Brown,  K.  L.  1993.\nErgosterol  determination  activated  sludge  and  its  application  as  a  bio-\nchemical  marker  for  monitoring  fungal  biomass.  J.  Microbiol.  Methods\n17:49-60.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.0**\n19.  Gil Ortega, R.,  Español, C. P., and  Zueco, J. C.  1991. Genetics  of resis-\ntance to Phytophthora capsici in the pepper line “SCM-334”. Plant Breed.\n107:50-55.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.0**\n20.  Gil Ortega, R., Español, C. P., and Zueco, J. C. 1995. Interactions in the\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\npepper-Phytophthora capsici system. Plant Breed. 114:74-77.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.0**\n21.  Gretenkort,  M.  A.,  and  Ingram,  D.  S.  1993.  The  use  of  ergosterol  as  a\nquantitative measure of the resistance of cultured tissues of Brassica napus\nspp. oleifera to Leptosphaeria maculans. J. Phytopathol. 138:217-224.\n22.  Harrison, J. G., Barker, H., Lowe, R., and Rees, E. A. 1990. Estimation of\namounts  of  Phytophthora  infestans  mycelium  in  leaf  tissue  by  enzyme-\nlinked immunosorbent assay. Plant Pathol. 34:557-564.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.0**\n23.  Hayden, K. J., Rizzo, D., Tse, J., and Garbelotto, M. 2004. Detection and\nquantification  of  Phytophthora  ramorum  from  California  forests  using  a\nreal-time polymerase chain reaction assay. Phytopathology 94:1075-1083.\n24.  Hoffman, T., Schmidt, J. S., Zheng, X., and Bent, A. F. 1999. Isolation of\nethylene-insensitive soybean mutants that are altered in pathogen suscep-\ntibility and gene-for-gene disease resistance. Plant Physiol. 119:935-950.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.0**\n25.  Holland,  P.  M., Abramson,  R.  D., Watson,  R.,  and  Gelfand,  D.  H.  1991.\nDetection  of  specific  polymerase  chain  reaction  product  by  utilizing  the\n5′-3′exonuclease  activity  of  Thermus  aquaticus  DNA  polymerase.  Proc.\nNatl. Acad. Sci. USA 88:7276-7280.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.0**\n26.  Ippolito, A., Schena, L., and Nigro, F., Ligorio, V. S., and Yaseen, T. 2004.\nReal-time  detection  of  Phytophthora  nicotianae  and  P.  citrophthora  in\ncitrus roots and soil. Eur. J. Plant Pathol. 110:833-843.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.0**\n27.  Jung, H. W., and Hwang, B. K. 2000. Pepper gene encoding a basic β-1,3-\nglucanase  is  differentially  expressed  in  pepper  tissues  upon  pathogen\ninfection  and  ethephon  or  methyl  jasmonate  treatment.  Plant  Sci.  156:\n23-34.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.0**\n28.  Jung, H. W., Kim, W., and Hwang, B. K. 2003. Three pathogen-inducible\ngenes  encoding  lipid  transfer  protein  from  pepper  are  differentially  acti-\nvated  by  pathogens,  abiotic,  and  environmental  stresses.  Plant  Cell\nEnviron. 26:915-928.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.0**\n29.  Karpovich-Tate, N., Spanu, P., and Dewe, F. M. 1998. Use of monoclonal\nantibodies  to  determine  biomass  of  Cladosporium  fulvum  in  infected\ntomato leaves. Mol. Plant-Microbe Interact. 11:710-716.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\n30.  Larkin, R. P., Ristaino, J. B., and Campbell, C. L. 1995. Detection and quan-\ntification of Phytophthora capsici in soil. Phytopathology 85:1057-1063.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n31.  Maniatis, T., Fritsch, E. F., and Sambrook, J. 1982. Molecular Cloning: A\nLaboratory Manual. Cold Spring Harbor Laboratory, Cold Spring Harbor,\nNY.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n32.  Martin,  F.,  Delaruelle,  A.,  and  Hilberty,  J.  L.  1990.  An  improved  ergo-\nsterol  assay  to  estimate  fungal  biomass  in  ectomycorrhizas.  Mycol.  Res.\n94:1059-1064.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n33.  McCartney, H. A., Foster, S. J., Fraaije, B. A., and Ward, E. 2003. Molecular\ndiagnostics for fungal plant pathogens. Pest. Manage. Sci. 59:129-142.\n34.  Mercado-Blanco, J., Collado-Romero, M., Parrilla-Araujo, S., Rodríguez-\nJurado,  D.,  and  Jiménez-Díaz,  R.  M.  2003.  Quantitative  monitoring  of\ncolonization  of  olive  genotypes  by  Verticillium  dahliae  pathotypes  with\nreal-time  polymerase  chain  reaction.  Physiol.  Mol.  Plant  Pathol.  63:91-\n105.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n35.  Miller, S. A. 1996. Detecting propagules of plant pathogenic fungi. Adv.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nBot. Res. 23:73-102.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n36.  Morrison,  T.  B.,  Weis,  J.  J.,  and  Wittwer,  C.  T.  1998.  Quantification  of\nlow-copy  transcripts  by  continuous  SYBR  Green  I  monitoring  during\namplification. BioTechniques 24:954-962.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n37.  Mumford, R. A., Walsh, K., Barker, I., and Boonham, N. 2000. Detection\nof  potato  mop  top  virus  and  tobacco  rattle  virus  using  a  multiplex  real-\ntime  fluorescent  reverse-transcription  polymerase  chain  reaction  assay.\nPhytopathology 90:448-453.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n38.  Newton, A. C., and Reglinski, T. 1993. An enzyme-linked immunosorbent\nassay for quantifying mildew biomass. J. Plant. Dis. Prot. 100:176-179.\n39.  Nuez,  F.,  Gil,  R.,  and  Costa  J.  1996.  El  cultivo  de  pimientos,  chiles  y\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\najíes. Ediciones Mundi-Prensa, Madrid.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n40.  Pacovsky,  R.  S.,  and  Bethlenfalvay,  G.  J.  1982.  Measurement  of  the\nextraradical  mycelium  of  a  vesicular-arbuscular  mycorrhizal  fungus  in\nsoil by chitin determination. Plant Soil 68:143-147.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n41.  Palloix, A., Daubeze, A. M., and Pochard, E. 1988. Phytophthora root rot\nof  pepper.  Influence  of  host  genotype  and  pathogen  strain  on  the  inocu-\nlum density-disease severity relationships. J. Phytopathol. 123:25-33.\n42.  Pegg, G. F., and Brady, B. L. 2002. Verticillium Wilts. CAB International,\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nWallingford, UK.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n43.  Plassard,  C.  S.,  Mousain,  D.  G.,  and  Salsac,  L.  E.  1982.  Estimation  of\nmycelia  growth  of  basidiomycetes  by  means  of  chin  determination.\nPhytochemistry 21:345-348.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n44.  Pomar,  F.,  Bernal,  M. A.,  Collar,  J.,  Díaz,  J.,  Caramelo,  C.,  Gayoso,  C.,\nNovo,  M.,  Prego,  C.,  Saavedra,  A.,  Silvar,  C.,  and  Merino,  F.  2001.  A\nsurvey  of  “Tristeza”  of  pepper  in  Galicia  and  the  fungal  pathogens\ncausing the disease. Capsicum Eggplant Newsl. 20:90-93.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n45.  Qi, M., and Yang, Y. 2002. Quantification of Magnaporthe grisea during\ninfection  of  rice  plants  using  real-time  polymerase  chain  reaction  and\nnorthern blot/phosphimaging analyses. Phytopathology 92:870-876.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.0**\n46.  Ririe,  K.  M.,  Rasmussen,  R.  P.,  and  Wittwer,  C.  T.  1997.  Product\ndifferentiation by analysis of DNA melting curves during the polymerase\nchain reaction. Anal. Biochem. 245:154-160.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.0**\n47.  Schaad,  N. W.,  Frederick,  R.  D.,  Shalev, A.,  Shaw,  J.,  Schneider, W.  L.,\nHickson,  R.,  Petrillo,  M.  D.,  and  Luster,  D.  G.  2003.  Advances  in\nmolecular-based  diagnostics  in  meeting  crop  biosecurity  and  phyto-\nsanitary issues. Annu. Rev. Phytopathol. 41:305-324.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.0**\n48.  Schena,  L.,  Nigro,  F.,  Ippolito,  A.,  and  Gallitelli,  D.  2004.  Real-time\nquantitative PCR: A new technology to detect and study phytopathogenic\nand antagonistic fungi. Eur. J. Plant Pathol. 110:893-908.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nthe  hypersensitive  response\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.0**\n49.  Shin,  R.,  Lee,  G.  J.,  Park,  C.  J.,  Kim, T. Y., You,  J.  S.,  Nam, Y. W.,  and\nPaek,  K.  H.  2001.  Isolation  of  pepper  mRNAs  differentially  expressed\nduring\ntobacco  mosaic  virus  and\ncharacterization of a proteinase inhibitor gene. Plant Sci. 161:727-737.\n50.  Silvar, C., Duncan, J. M., Cooke, D. E. L., Williams, N. A., Díaz, J., and\nMerino, F. 2005. Development of specific PCR primers for identification\nand detection of Phytophthora capsici Leon. Eur. J. Plant Pathol. 112:43-\n52.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nto\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.0**\n51.  Srivastava,  A.  K.,  and  Arora,  D.  K.  1997.  Evaluation  of  a  polyclonal\nantibody immunoassay for detection and quantification of Macrophomina\nphaseolina. Plant Pathol. 46:785-794.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.0**\n52.  Takenaka,  S.  1995.  Dynamics  of  fungal  pathogens  in  host  plant  tissues.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nCan. J. Bot. 73:S1275-S1283.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.0**\n53.  Thomma, B. P. H. J., Tadesse, Y. S. H., Jacobs, M., and Browkaert, W. F.\n1999. Disturbed correlation between fungal biomass and β-glucuronidase\nactivity  in  infections  of  Arabidopsis  thaliana  with  transgenic  Alternaria\nbrassicicola. Plant Sci. 148:31-36.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.0**\n54.  Tyagi,  S.,  and  Kramer,  F.  R.  1996.  Molecular  beacons:  Probes  that\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\nfluoresce upon hybridization. Nat. Biotechnol. 14:303-308.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.0**\n55.  Vandemark,  G.  J.,  and  Barker,  B.  M.  2003.  Quantifying  Phytophthora\nmedicaginis  in  susceptible  and  resistant  alfalfa  with  a  real-time  fluores-\ncent assay. J. Phytopathol. 151:577-583.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.0**\n56.  Ward,  E.,  Tahiri-Alaouri,  A.,  and  Antoniw,  J.  F.  1998.  Applications  of\nPCR in fungal–plant interactions. Pages 289-307 in: Application of PCR\nin Mycology. P. D. Bridge, D. K. Arora, C. A. Reddy, and R. P. Elander,\neds. CAB International, London.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.0**\n57.  Whitcombe,  D.,  Theaker,  J.,  Guy,  S.  P.,  Brown,  T.,  and  Little,  S.  1999.\nDetection  of  PCR  products  using  self-probing  amplicons  and  fluores-\ncence. Nat. Biotechnol. 17:804-807.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.0**\n58.  Winton,  L.  M.,  Manter,  D.  K.,  Stone,  J.  K.,  and  Hansen,  E.  A.  2003.\nComparison  of  biochemical,  molecular,  and  visual  methods  to  quantify\nPhaeocryptopus  gaeumannii  in  Douglas-Fir  foliage.  Phytopathology\n93:121-126.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.0**\n59.  Zhou,  X.  G.,  and  Everts,  K.  L.  2004.  Quantification  of  root  and  stem\ncolonization of watermelon by Fusarium oxysporum f. sp. niveum and its\nuse in evaluating resistance. Phytopathology 94:832-841.",
         "extracted",
         "",
         "Real-Time Polymerase Chain Reaction Quantification of Phytophthora capsici in Different Pepper Genotypes.",
         "None"
        ],
        [
         "28",
         "0025bd43353f35f1564e81101155689b6bc62646",
         "ABSTRACT Growth, patterning, and apoptosis are mutually interactive during development. For example, cells that select an abnormal fate in a developing field are frequently removed by apoptosis. An important issue in this process that needs to be resolved is the mechanism used by cells to discern their correct fate from an abnormal fate. In order to examine this issue, we developed an animal model that expresses the dioxin receptor homolog Spineless (Ss) ectopically in the Drosophila wing. The presence of mosaic clones ectopically expressing ss results in a local transformation of organ identity, homeosis, from wing into a leg or antenna. The cells with misspecified fates subsequently activate c-Jun N-terminal kinase to undergo apoptosis in an autonomous or nonautonomous manner depending on their position within the wing, suggesting that a cell-cell interaction is, at least in some cases, involved in the detection of misspecified cells. Similar position dependence is commonly observed when various homeotic genes controlling the body segments are ectopically expressed. The autonomous and nonautonomous apoptosis caused by ss is regulated by a novel leucine-rich repeat family transmembrane protein, Fish-lips (Fili) that interacts with surrounding normal cells. These data support a mechanism in which the lack of some membrane proteins helps to recognize the presence of different cell types and direct these cells to an apoptotic fate in order to exclude them from the normal developing field.",
         "T. Adachi-Yamada,Toshiyuki Harumoto,Kayoko Sakurai,R. Ueda,K. Saigo,M. O’Connor,H. Nakato",
         "https://europepmc.org/articles/pmc1069588?pdf=render",
         "\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nMOLECULAR AND CELLULAR BIOLOGY, Apr. 2005, p. 3140–3150\n0270-7306/05/$08.000 doi:10.1128/MCB.25.8.3140–3150.2005\nCopyright © 2005, American Society for Microbiology. All Rights Reserved.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.1**\nWing-to-Leg Homeosis by Spineless Causes Apoptosis Regulated by\nFish-lips, a Novel Leucine-Rich Repeat Transmembrane Protein†\nTakashi Adachi-Yamada,1,2,3*‡ Toshiyuki Harumoto,2‡ Kayoko Sakurai,4‡ Ryu Ueda,5\nKaoru Saigo,6 Michael B. O’Connor,7,8 and Hiroshi Nakato4,7*\nDepartment of Earth and Planetary System Sciences, Graduate School of Science and Technology,1 and Department of\nSciences for Natural Environment, Faculty of Human Development,2 SORST, and Japan Science and Technology\nAgency,3 Kobe University, Kobe, Department of Biology, Faculty of Science, Tokyo Metropolitan University,\nHachioji,4 Genetic Strains Research Center, National Institute of Genetics, Mishima,5 and Department\nof Biophysics and Biochemistry, Graduate School of Science, University of Tokyo, Tokyo,6 Japan,\nand Department of Genetics, Cell Biology, and Development7 and Howard\nHughes Medical Institute,8 University of Minnesota,\nMinneapolis, Minnesota\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.2**\nReceived 24 May 2004/Returned for modiﬁcation 6 July 2004/Accepted 21 December 2004\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.2**\nGrowth, patterning, and apoptosis are mutually interactive during development. For example, cells that\nselect an abnormal fate in a developing ﬁeld are frequently removed by apoptosis. An important issue in this\nprocess that needs to be resolved is the mechanism used by cells to discern their correct fate from an abnormal\nfate. In order to examine this issue, we developed an animal model that expresses the dioxin receptor homolog\nSpineless (Ss) ectopically in the Drosophila wing. The presence of mosaic clones ectopically expressing ss results\nin a local transformation of organ identity, homeosis, from wing into a leg or antenna. The cells with\nmisspeciﬁed fates subsequently activate c-Jun N-terminal kinase to undergo apoptosis in an autonomous or\nnonautonomous manner depending on their position within the wing, suggesting that a cell-cell interaction is,\nat least in some cases, involved in the detection of misspeciﬁed cells. Similar position dependence is commonly\nobserved when various homeotic genes controlling the body segments are ectopically expressed. The autono-\nmous and nonautonomous apoptosis caused by ss is regulated by a novel leucine-rich repeat family trans-\nmembrane protein, Fish-lips (Fili) that interacts with surrounding normal cells. These data support a mech-\nanism in which the lack of some membrane proteins helps to recognize the presence of different cell types and\ndirect these cells to an apoptotic fate in order to exclude them from the normal developing ﬁeld.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nSignals that regulate cell growth, patterning, and apoptosis\nare interdependent during development (1, 19, 29, 37). In\nparticular, there are likely to be multiple ways to elicit an\napoptotic cell fate since apoptosis serves a variety of functions\nin multicellular organisms (29). In regard to the cell autonomy\nof apoptosis, a cell-autonomous apoptosis occurs during cell\ncompetition, a phenomenon whereby cells that grow slowly due\nto the mutation of essential genes such as Minute or ras are\nremoved later in development (39, 40, 49, 56). In contrast, it\nhas been reported that nonautonomous cell death is often\nassociated with cell fate changes by altering the morphogen\nactivities (2, 3) of Decapentaplegic (Dpp) (32, 41) or Wingless\n(Wg) (42, 60). In these cases, apoptotic cell death occurs both\nwithin and outside of the abnormal cell population and is\nreferred to as morphogenetic apoptosis. Similarly, when spalt\n(sal), a target gene of the Dpp signal, is ectopically expressed\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n* Corresponding authors. Mailing address for T. Adachi-Yamada:\nDepartment of Earth and Planetary System Sciences, Graduate School\nof Science and Technology, Kobe University, Kobe 657-8501, Japan.\nPhone: 81-78-803-7743. Fax: 81-78-803-7743. E-mail: yamadach\n@kobe-u.ac.jp. Mailing address for H. Nakato: Department of Genet-\nics, Cell Biology, and Development, University of Minnesota, Minne-\napolis, MN 55455. Phone: (612) 625-1727. Fax: (612) 626-5652. E-mail:\nnakat003@umn.edu.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n† Supplemental material for this article may be found at http://mcb\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\n.asm.org/.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\n‡ T.A.-Y., T.H., and K.S. contributed equally to this study.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\napoptosis is induced through interaction with the surrounding\nnormal cells (38). Thus, the removal of physiologically abnor-\nmal cells by apoptosis can occur by more than one mechanism,\nalthough the details of the various processes remain to be\nelucidated. In order to search for and analyze the ways to\nremove other types of abnormal cell, we have ectopically ex-\npressed various master genes, including homeotic genes, to\ncreate cell populations with developmentally abnormal fates.\nIn most cases, cells with abnormal fate induced severe apopto-\nsis in a cell autonomous or nonautonomous manner. Among\nthese, we focus on the master gene ss, of which overexpression\nin the wing changes the organ identity into a ventral appendage\nsuch as a leg or antenna.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nBased on sequence identity and splice site conservation, Ss is\nthe closest Drosophila homolog of the mammalian dioxin re-\nceptor (arylhydrocarbon receptor [Ahr]) (17, 27). Both mam-\nmalian and Drosophila proteins can also bind to the xenotoxin\nresponsive element (XRE) and stimulate transcription from\ngenes containing this cis-acting element (18). Although Ss has\nnot been shown to bind to arylhydrocarbons, it regulates nor-\nmal morphogenesis of the leg or antenna and bristles, all of\nwhich are major Drosophila sensor organs or tissues that re-\nspond to environmental chemicals.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nOur results reveal that ectopic ss provokes a wing-to-leg\nand/or antenna homeosis that subsequently elicits apoptosis in\nan autonomous or nonautonomous manner. This apoptotic\nresponse is regulated by a novel transmembrane leucine-rich\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nrepeat (LRR) protein, Fili, and may be a common process\ninduced by ectopic expression of various homeotic genes.\nThese results indicate that homeosis elicits a complex set of\nsignals that inﬂuence the survival of the transformed cells and\ntheir surrounding cells. In addition, these results suggest a\nhypothesis that different LRR family transmembrane proteins\nfunction in different subdomains of a developing ﬁeld to rec-\nognize developmentally misspeciﬁed cells and to regulate cell\nsurvival.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nMATERIALS AND METHODS\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nFly strains. We used two different GAL4 enhancer trap lines of scalloped (sd)\nin the present study. The (s) and (w) lines induce a strong and weak expression\nof UAS-driven genes, respectively. The plasmid construct for tgo-RNAi was made\nby inserting an inverted tgo cDNA into the 5 region of the standard UAS-tgo\nconstruct. When tgo-RNAi was induced singly in the wing, the ﬂy wing showed no\nphenotype (data not shown).\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nThe name ﬁsh-lips was derived from the observation that the expression pat-\ntern of ﬁli in the wing disc made a ﬁsh lips-like shape. ﬁli is identical to the\ntemporarily assigned hypothetical gene CG4054 in the Berkeley Drosophila Ge-\nnome Project. The ﬁli-lacZ ﬂy was identiﬁed during a screen of lacZ enhancer\ntrap strains with P-lacW, a derivative of the transposon P-element (6). The ﬁli\nmutant allele 102 was isolated by an imprecise excision of P-lacW in ﬁli-lacZ\naccording to the standard procedure. The UAS-ﬁli ﬂy was made by inserting the\nﬁli cDNA (2,297 bp, including full-length open reading frame, 18 bp of the 5\nuntranslated region and 62 bp of the 3 untranslated region) into the pUAST\nvector (7) with a white marker and transforming the white ﬂy according to\nstandard procedures.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe other lacZ enhancer trap lines we used are sdETX4, Dll01092, hth05745,\nhid05014, ombP1, fz3SW076-A7.3XW, trnS064117, and pucE69. hid05014 was also used as\na mutant (see Fig. 4E and G). At the late-third-instar larval stage, this hid-lacZ\nis expressed irrespective of apoptosis around the dorsal-ventral boundary and at\nthe two spots in the dorsal and ventral hinge region (data not shown). However,\nat the mid-third-instar larval stage, these expressions do not begin, and the\nexpression is associated with apoptosis. Therefore, we observed the hepCA- or\nss-induced hid-lacZ expression by using the mid-third-instar larvae.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nDetection of apoptosis. Caspase-3 plays a central role in many types of apo-\nptosis, whereas c-Jun N-terminal kinase (JNK) activation elicits a limited group\nof apoptotic events (14). In the Drosophila larval wing disc, activation of JNK\nalways leads to the activation of caspase-3 (2). Puckered (Puc) is a protein\nphosphatase speciﬁcally inactivating JNK, and its transcription is induced by JNK\nsignal, thereby making a negative-regulatory circuit (36). Most of the endoge-\nnous and ectopic puc-lacZ expression observed in the present study was elimi-\nnated in a mutant background of the hep gene (20), which encodes the homolog\nof mitogen-activated protein kinase kinase-7 that activates JNK. Immunoﬂuo-\nrescence was carried out according to the standard procedure. The puc-lacZ\nexpression is detected by measuring mouse anti--galactosidase antibody (1:200\ndilution; Promega Z378) and Cy3-labeled anti-mouse immunoglobulin (1:200\ndilution; Jackson Immunoresearch). The rabbit anti-active caspase-3 antibody\n(CM1) was a gift from Idun Pharmaceuticals. It speciﬁcally recognizes the\ncleaved and activated form of mammalian caspase-3 and has been shown to also\nrecognize the cleaved form of the Drosophila caspase-3 homolog, Drice (59).\nThus, active caspase-3/Drice was detected by CM1 (1:4,000 dilution) and Cy5-\nlabeled anti-rabbit immunoglobulin (1:200 dilution; Jackson Immunoresearch).\nAntibodies. The other antibodies we used were rat anti-Al (from G. Campbell\n[1:1,000 dilution]), rat anti-Sal (from R. Barrio [1:400 dilution]), mouse anti-Dac\n(MABDAC2-3 from the Developmental Studies Hybridoma Bank [1:10 dilution\nof supernatant]), rabbit anti-Vg (from S. Carroll [1:200 dilution]), rabbit anti-p-\nMad (PS1 from P. ten Dijke [1:200 dilution]), mouse anti-Arm (N2 7A1 from the\nDevelopmental Studies Hybridoma Bank [1:10 dilution of supernatant]), mouse\nanti-Dll (MAb DMDll.1 from D. Duncan [1:500 dilution]), and rabbit anti-Caps\n(from A. Nose [1:200 dilution]).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nMosaic overexpression analysis. Discs were prepared from larvae carrying the\nAyGAL4 (actin promoter-FRT-yellow,\nterminator-FRT-GAL4 [28]), hs-FLP\n(yeast FLP recombinase gene driven by heat shock promoter), UAS-ss, and\nUAS-GFP transgenes. The GAL4-expressing clones were induced by heat treat-\nment at 37°C for 20 min at 48 to 72 h after egg-laying and observed 48 to 72 h\nafter heat treatment. In each experiment, we observed more than 10 wing discs\nto conﬁrm the results.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nFIG. 1. Activation of ss leads to loss of wing. (A and B) sd-strong(s)\nand sd-weak(w) expression patterns (green) in the wing imaginal discs.\nAnterior is to the left, and dorsal is at the top. (C) Wild-type ﬂy (left)\nand a ﬂy without wing by overexpression of ss under the control of the\nsd(s)-GAL4 driver (right). (D) Wild-type wing. (E) A wing overex-\npressing ss under the control of the sd(w)-GAL4 driver shows an\nincision of the wing margin. (F) Front view of the normal ﬂy head with\nwhite compound eyes for easier recognition of antenna. (G) Front\nview of the ﬂy head in which tgo is repressed by UAS-tgo-RNAi with\nDll-GAL4. The antenna transforms to a leg-like appendage, which is\nsimilar to the antenna mutant for ss or tgo (17, 18). (H) A wing\noverexpressing ss but repressing tgo under the control of the sd(w)-\nGAL4 driver shows a reversion of the wing margin incision found in\nthe ss overexpression alone.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nRESULTS\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nOverexpression of ss leads to a disappearance of the wing,\nwhich is dependent on tgo. Using the GAL4/UAS system (7),\nwe overexpressed ss in the wing blade region by scalloped (sd)\n(s)-GAL4 (Fig. 1A). As a result, no adults developed wings\n(Fig. 1C). When we expressed ss in a more restricted region\nusing the weaker allele, sd(w)-GAL4 (Fig. 1B), the wing mar-\ngin structure disappeared (Fig. 1D and E). These results sug-\ngest that ectopic expression of ss in the wing leads to massive\napoptosis.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nMammalian Ahr is known to function together with a struc-\nturally similar protein, Arylhydrocarbon receptor nuclear\ntranslocator (Arnt), by forming a heterodimeric transcription\nfactor (27). To test whether the wing apoptosis phenotype by ss\noverexpression is mediated by Tango (Tgo) (43), the Drosoph-\nila homolog of Arnt, we examined ﬂies in which we simulta-\nneously induced both ss expression and tgo-RNAi. Prior to this\nexperiment, we examined whether tgo-RNAi can remove en-\ndogenous tgo function. When tgo-RNAi was induced in the\nantenna by using Distal-less (Dll)-GAL4, the antenna trans-\nformed to a leg-like organ as seen in the mutant of tgo (18)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nFIG. 2. Homeotic transformation of organ identity from that of a wing to that of a leg or antenna by overexpression of ss. (A and A) Expression\nof sd and Dll in the wild-type imaginal discs. Haltere (halt.), leg, wing, antenna (ant.), and eye discs are shown. (B) Expression of various genes\nas markers for the leg or antenna segments in the ss-overexpressing clones. A schematic representation at the bottom of the panel shows normal\nexpression in the antenna (left) and leg (right). The open bar in the sal expression in the antenna indicates the broad expression (in the a2 and\na3 segments) found only in the early stage. This expression is restricted to a narrower region (only a2) later in development. A table indicating\nincreases () or decreases () in gene expression in the ss-overexpressing clones in the wing blade and hinge regions is shown at the top. NE, no\nexpression was seen. (C to H and C to H) Wing discs showing an increase (white arrows) or decrease (gray arrows) of gene expression (magenta)\nin the ss-overexpressing clones (green). ss overexpression was induced, together with DIAP1 (IAP), to delay apoptosis except for panel H. Asterisks\nin panel G indicate normal expression of Dac in the peripodial membrane. (I) An ectopic tarsus-like structure generated in the wing by\ncoexpression of ss and DIAP1 (IAP) with sd(s)-GAL4 driver. (I) t1 segment of the distal mid-leg in wild type. (J and J) Normal expression of ptc\n(green) and Wg (magenta) in the leg and wing. An asterisk indicates a ventral stripe of Wg expression in the leg. (K and K) Expression of ptc\n(green) and Wg (magenta) in the wing in which ss is overexpressed by ptc-GAL4 driver. Asterisk indicates an ectopic stripe of Wg expression in\nthe ventral wing. The gray arrow indicates the reduced Wg expression that is not found in the normal wing.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n(Fig. 1F and G). Thus, the tgo-RNAi was conﬁrmed to repress\nthe endogenous tgo. When ss was induced together with this\ntgo-RNAi, the wing phenotype was completely suppressed (Fig.\n1H), suggesting that the loss of the wing due to ectopic expres-\nsion of ss requires tgo activity and is not simply due to a stress\ncaused by ectopic expression of ss.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nOverexpression of ss leads to a homeotic transformation\nfrom wing to ventral appendage before disappearance of the\ncells. ss is known to specify the identities of tarsus (distal leg)\nand arista (distal antenna) during normal development (17,\n18). Thus, we thought that the ss overexpression in the wing\nmay have removed the organ identity as a wing and underwent\na homeotic change to a leg, antenna, or other ventral append-\nage that does not actually exist in the normal ﬂy prior to\ndisappearance of the wing. To investigate this possibility, we\nnext examined the expression of marker genes speciﬁc to the\nwing or haltere or to leg or antenna. The sd gene, which is\nnormally expressed exclusively in the wing and haltere (10)\n(Fig. 2A), was found to be less strongly expressed in ss-over-\nexpressing clones (Fig. 2C and C). This result suggests that the\nss-overexpressing clones had lost, at least partially, their wing\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nidentity. In contrast, Dll, a master gene that directs the identity\nof the distal leg or whole antenna (13, 45) (Fig. 2A), was\nectopically and cell autonomously induced in the ss-overex-\npressing clones (Fig. 2D and D). During normal wing devel-\nopment, Dll expression is induced in response to Wg signaling.\nHowever, Dll induction in the ss-overexpressing clones ap-\npeared to be independent of the Wg signal since Dfrizzled-3\n(fz3) induction and Armadillo (Arm) accumulation did not\ncoincide with the ss-overexpressing region, as demonstrated\nbelow. Therefore, we hypothesized that this Dll\ninduction\nmight provoke a different Dll function to redirect the identity\nof the wing to that of the distal leg or whole antenna. In fact,\nhigh-level induction of Dll in the wing has been reported to\nlead to its transformation into a distal leg (21). Although nor-\nmal expression of ss in the leg is known to be downstream of\nDll (17), Dll expression is regulated by ectopic ss in the wing,\nindicating the presence of a positive feedback loop between Dll\nand ectopic ss.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nNext, we examined the expression of various genes that are\nmarkers for different subdomains of the leg or antenna (Fig.\n2E to H) (8, 9, 15, 16, 35, 45; Flybase [http://ﬂybase.bio.indiana\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nFIG. 3. Overexpression of ss leads to autonomous and nonautonomous apoptosis in a Dll-dependent manner. (A) A wing imaginal disc with\nmosaic clones overexpressing ss (green). JNK activation visualized by puc-lacZ expression (magenta) and caspase-3 activation (cyan) are shown.\nRed line denotes the D/V boundary. (A-A) Higher magniﬁcation of each clone in panel A. The total number of clones examined is 229. Among\nthese, the number of clones exhibiting autonomous puc-lacZ induction is 30, whereas the number of those exhibiting nonautonomous puc-lacZ\ninduction is 99. (B to B) Wing imaginal disc with ss overexpression driven by the ptc-GAL4 (green). Either cell autonomous or nonautonomous\napoptosis was observed symmetrically as a function of the distance of the cells from the dorsoventral boundary. Arrowheads and arrows denote\na nonautonomous activation of JNK (magenta) and autonomous activation of caspase-3 (cyan), respectively. (C) Wing subdomains based on the\nresponse to ss. The Dll region is white, the vg region is red, and the hth region is blue. (D) Wing imaginal disc with mosaic clones overexpressing\nss (green) in the Dll5/Dll9 mutant background. JNK activation visualized by puc-lacZ expression (magenta) and caspase-3 activation (cyan) are\ngreatly reduced. (E) A wing with ss overexpression by sd(w)-GAL4 in the Dll5/ mutant background shows a strongly suppressed phenotype. (F\nand F) Wing imaginal disc with mosaic clones overexpressing Dll (green). DIAP1 (IAP) is coexpressed to delay apoptosis. puc-lacZ expression\n(magenta). Caspase-3 activation is indicated in cyan. Arrows and arrowheads denote an autonomous and nonautonomous activation of JNK,\nrespectively. (F) High magniﬁcation of the boxed area in panel C. (G) A wing with Dll overexpression by sd(w)-GAL4 shows an incision of the\nwing margin.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\n.edu/]). In summary, ss-overexpressing clones in the blade re-\ngion of the wing have most likely transformed their identity to\nthat of a distal leg (t1 area) rather than that of an antenna (Fig.\n2B). In the hinge region of the wing, Dll expression was not\ninduced by ss (Fig. 2D). Instead, the expression of hth, a master\ngene that directs the identity of the proximal leg or whole\nantenna (12, 44) was enhanced in the ss-overexpressing clones\n(Fig. 2H and H). However, in the normal leg or antenna there\nwas no segment that expresses both ss and hth but not Dll (Fig.\n2B). Thus, the fate of the ss-overexpressing clones in the hinge\nregion of the wing should have transformed to that of a ventral\nappendage, although we have not identiﬁed which normal ap-\npendage segment that the ss-overexpressing clones resemble.\nIn comparison, a leg-antenna intermediate and a hypothetical\nancestor appendage have been observed in Dll and ss mutants,\nrespectively (17, 18).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nWe also obtained other evidence demonstrating that the\ncells with ss overexpression display characters of the ventral\nappendages. First, an ectopic appendage-like structure was\noften generated (Fig. 2I) when ss-induced apoptosis was par-\ntially inhibited by coexpression of Drosophila inhibitor of ap-\noptosis protein 1 (DIAP1 [26]). The structure most similar to\nthis appendage that is seen in external morphology of normal\nﬂy is the tarsus of leg (Fig. 2I). Second, during normal devel-\nopment expression of patched (ptc) was dependent on Hedge-\nhog signaling, which induces a band-like expression of dpp in\nthe wing and the dorsal leg region and that of wg in the ventral\nleg region (5) (Fig. 2J and J). When ss expression was induced\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nby ptc-GAL4, Wg was ectopically induced only in the ventral\nhalf of the ptc-expressing region (Fig. 2K and K), as found in\nthe normal leg or antenna. However, the level of wg induction\nin the ptc-expressing region was weaker than that found in the\nnormal leg. Therefore, the ss-expressing wing appears to have\npartially transformed its identity to that of a leg or antenna.\nThis mixed or conﬂicted identity may also be responsible for\ninduction of apoptosis. Although it has been shown previously\nthat the ectopic expression of ss by ptc-GAL4 leads to a dele-\ntion of the central wing (17), the molecular mechanism was not\nidentiﬁed. Our results suggest that the deletion is triggered\nprimarily by homeotic transformation.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nOverexpression of ss leads to apoptosis in which cell auton-\nomy is position dependent. To study the detailed mechanism\nleading to apoptosis by ectopic expression of ss, we next exam-\nined the cell autonomy of apoptosis in ss-overexpressing\nclones. We monitored caspase-3 and JNK activation by using\nan antibody that recognizes active caspase-3 and the expression\nof a reporter gene, puckered (puc)-lacZ, respectively. After\n48 h of mosaic induction, we observed two types of apoptosis\n(Fig. 3A). The ss-overexpressing clones inside the wing blade\nprimordium showed strong activation of caspase-3 and a weak\nactivation of JNK (Fig. 3A and A). Activation of caspase-3 in\nthe vicinity of the dorsoventral compartment boundary (red\nline in Fig. 3A) was relatively weak and delayed. We can\nrecognize the delay of the cell removal around the dorsoven-\ntral boundary more readily at later stage after the clone induc-\ntion (data not shown). All of these responses were always cell\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nTABLE 1. Position-dependent variance of apoptotic responses to ss\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nSubdomaina\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nJNK\nvariance\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nCaspase-3\nvariance\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nCell autonomy\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nResponse\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nAround DV\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nboundary (Dll)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nRest of blade\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n(vg)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nHinge (hth)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nWeak\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nWeak\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nWeak\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nAutonomous\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nDelayed\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nStrong\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nAutonomous\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nAcute\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nStrong\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nWeak\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nNonautonomous\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nVariable\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\na Representative genes expressed in each subdomain are indicated in paren-\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\ntheses.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nautonomous. In contrast, the ss-overexpressing clones outside\nof the wing blade region (that is the hinge region) showed weak\nactivation of caspase-3 and strong activation of JNK in cells on\neither side of the clone boundary (Fig. 3A). In this region,\nthe outline of the clones which were probably induced at an\nearlier stage, are severely disrupted with strong activation of\nboth caspase-3 and JNK (Fig. 3A).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nThe presence of the two types of apoptosis can clearly be\nrecognized in ss overexpression by ptc-GAL4, which is ex-\npressed in a band intersecting the dorsoventral boundary (Fig.\n3B to B). Strong, autonomous caspase-3 activation was ob-\nserved in the blade region, whereas strong, nonautonomous\nJNK activation was observed around the hinge region. The\nregion around the dorsoventral boundary did not show both\nJNK and caspase-3 activation. Thus, the wing disc can be\ndivided into three subdomains based on the nature of the\napoptotic response to ss overexpression in these regions (Fig.\n3C). Interestingly, these subdomains correlate with the expres-\nsion domains of several key transcriptional regulators. The\nDll-expressing subdomain showed an autonomous and delayed\nresponse. The vestigial (vg)-expressing subdomain showed an\nautonomous and acute response. The homothorax (hth)-ex-\npressing subdomain showed a nonautonomous response.\nThese subdomains can be recognized by the foldings of the disc\ncell layer or by distance between the nuclei, through counter\nstaining with DAPI (4,6-diamidino-2-phenylindole). The\nvariance of apoptotic responses are summarized in Table 1.\nSimilar traits in position-dependent cell autonomy of apoptosis\nhave also been found in the clones mutant for the Dpp recep-\ntor Thick veins (Tkv) (3).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nss-induced apoptosis is dependent on Dll that is responsible\nfor homeotic transformation from wing to leg or antenna. To\nelucidate the role of Dll in ss-induced apoptosis, we placed the\nss-overexpressing clones in a strong Dll mutant Dll5/Dll9 back-\nground. As shown in Fig. 3D, the activation of both JNK and\ncaspase-3 was strongly suppressed by this Dll mutation in the\nwing blade region. Activation was also suppressed in the hinge\nregion, although ss did not induce Dll expression in this region\n(Fig. 2D). A possible explanation is that broader expression of\nDll in the hinge region at an earlier stage of development (data\nnot shown) is required for transformation by ss in later stages.\nThe Dll mutation also suppressed the ss-induced adult wing\nphenotype (Fig. 3E). To test whether Dll overexpression is also\nable to lead to apoptosis, we next generated Dll-overexpressing\nclones in the wing. As shown in Fig. 3F, JNK and caspase-3\nactivation are observed cell autonomously in the Dll-overex-\npressing clones in the wing blade region. Other areas show\nnonautonomy. The wing notching phenotype induced by Dll\noverexpression is found to be similar to that induced by ss\n**BLOCK**fs== 9.0**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\noverexpression (Fig. 3G). Thus, ss-induced apoptosis is likely\nmediated by a pathway that is dependent upon Dll. A discrep-\nancy existed in that the normal expression of Dll in the wing did\nnot induce leg identity or apoptosis. However, the elevated\nexpression of Dll in the wing can induce some leg structure\n(21). Thus, the level of Dll expression, rather than its timing or\nplace, may be crucial for determining the fate as wing or leg.\nCoincidence of round shape and apoptotic response. Apo-\nptosis in each of these regions was dependent on Hemipterous\n(Hep, an activator of JNK [20]), DIAP1, Head involution de-\nfective (Hid; a proapoptotic protein [23]) and Tgo (Fig. 4B to\nF). DIAP1 is known to degrade caspase through its ubiquitin-\nprotein ligase activity. Hid can bind to DIAP1 to stimulate\nautoubiquitination of DIAP1, which prevents the degradation\nof caspase (58). Interestingly, the ss-overexpressing clones in\nhep or hid mutant or in DIAP1 overproducer backgrounds\nretained a round shape, as seen with ss-overexpressing clones\nin the wild-type background (Fig. 4A to E). In contrast, the\nss-overexpressing clones in the tgo-RNAi background lost their\nround shape and invaded the surrounding normal cells (Fig.\n4F). These results indicate that hep and hid only affect the\napoptosis pathway, whereas tgo mediated events more up-\nstream, such as the cell fate change induced by ss. Ectopic\nexpression of ss may make a difference in cell afﬁnity between\nthe clones and surrounding normal cells, which leads to a\nround shape of the clones and subsequent apoptotic response.\nConsistently, activation of JNK pathway by constitutively\nactive Hep leads to hid expression (Fig. 4G). Besides, the\nss-overexpressing clone induces hid expression autonomously\nin the wing blade region but nonautonomously in the wing\nhinge region (Fig. 4H).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nEffect of ss overexpression on the intensity of Dpp and Wg\nsignaling. The JNK-dependent cell death on either side of the\nclone boundary is quite similar to what has been referred to as\n“morphogenetic apoptosis,” through which discontinuities in\nthe Dpp and Wg morphogen activity gradients are corrected\n(2). We next examined the activities of certain morphogen\nsignals in the vicinity of the ss-overexpressing clones. With\nregard to Dpp signaling, we can ﬁnd a slight reduction in the\nexpression of the lacZ enhancer trap of optomotor-blind (omb),\na target gene of the Dpp signal (24) (see Fig. S1A and B in\nthe supplemental material). However, the reduction is always\nfound around the center of the clones and does not coincide\nwith the outline of the clones. We also see a similarly moderate\nalteration of the level of phospho-Mad, an active form of the\nDpp signal transducer “Mothers against dpp” (Mad) (48, 52,\n57) (see Fig. S1A and B in the supplemental material). In the\ncase of Wg signaling, we examined expression of the fz3-lacZ\nenhancer trap (51, 55), a target gene of the Wg signal, and the\nlevel of Arm (46, 54), a Wg signal transducer that is known to\naccumulate in response to the Wg signal (25, 47). We observed\na reduction of fz3-lacZ expression interior to the clone bound-\nary of the ss-overexpressing clones (see Fig. S1C, C, and D to\nD in the supplemental material). However, in the most central\ncells, cytoplasmic accumulation of Arm was conversely ob-\nserved (see Fig. S1C, C, D, and D in the supplemental ma-\nterial). These results suggested that the level of Wg signal is\nthe ss-overexpressing\nnot uniformly received throughout\nclones and is not directly regulated by Ss. Similar responses\nwere observed in clones both in the wing blade and in the hinge\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nregions, as seen in Fig. S1C in the supplemental material.\nTaken together, these results indicate that discontinuities in\nthe strength of Dpp and Wg signal reception does not precisely\ncoincide with the boundary of the ss-overexpressing clones,\nsuggesting that other factors contribute to the nonautonomous\nJNK activation in this case.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\nPosition-dependent variance in the autonomy of apoptosis is\nfound also in clones ectopically expressing homeotic genes. We\nnext examined whether the position-dependent variance in cell\nautonomy of apoptosis is speciﬁc for ss-induced homeosis or a\ngeneral trait exhibited by ectopic expression of diverse ho-\nmeotic genes. The identity of each body segment is deﬁned by\na series of homeotic genes belonging to ANTP-C and BX-C.\nHowever, the Drosophila wing requires no input from ANTP-C\nand BX-C during its development (11). Instead, these ho-\nmeotic genes repress wing development in all segments, except\nfor the mid-thorax, where the wing normally develops. In order\nto produce a misspeciﬁed fate in wing cells, we expressed\nvarious homeotic genes in the wing discs. When labial (lab),\nproboscipedia (pb), Antennapedia (Antp), Ultrabithorax (Ubx),\nabdominal-A (abd-A), or Abdominal-B (Abd-B) were expressed\nby using sd-GAL4 driver, the adult wings showed abnormal\nvein patterns and incision of the wing margin (Fig. 5A to F). In\nthe cases of Ubx or abd-A expression, wing incision was less\nfrequently observed. In these cases, the wing has been shown\nto transform to an organ similar to a haltere (22). When mo-\nsaic clones expressing these homeotic genes are generated,\nJNK activation was found autonomously in the blade region\nbut nonautonomously in the hinge region. All of these re-\nsponses are quite similar to the results of ss overexpression, as\nhas been described above. Thus, the position-dependent vari-\nance in cell autonomy of apoptosis may be a general phenom-\nenon accompanying homeotic transformation.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nCharacterization of the novel gene ﬁli that encodes a LRR\nfamily transmembrane protein. Recently, clones with sal-in-\nduced misspeciﬁed fate were reported to be recognized by the\nsurrounding normal cells through the LRR family transmem-\nbrane proteins, Capricious (Caps) and Tartan (Trn), to prevent\napoptosis (38). The expression pattern of both caps and trn are\ncomplementary to the pattern of sal expression in the late third\ninstar wing (38) (Fig. 6C). Thus, we investigated the role of a\nnewly identiﬁed LRR family protein, Fili, which was expressed\nin the wing in a nearly complementary pattern as that of Dll\n(Fig. 6A and B), in regulating apoptosis in ss-overexpressing\nclones.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe ﬂy strain ﬁli-lacZ was isolated by a screen of enhancer\ntrap lines that displayed a previously unknown expression pat-\ntern. The transposon P-element carrying the lacZ gene was\ninserted at 56 relative to the presumptive transcription start\nsite, TTAGTT (Fig. 7A). The deduced protein, Fili, was com-\nposed of 738 amino acid residues and had a single transmem-\nbrane domain and 14 LRR domains (Fig. 7B). These features\nare common to Caps and Trn, which share 55 and 56% ho-\nmologies with Fili in the LRR domains, respectively (Fig. 7C\nand D). The homologies also extend to the amino- and car-\nboxyl-terminus domains (Fig. 7E and F), suggesting that Fili\nplays a role that is especially related to Caps and Trn among\nnumerous members of LRR family with a variety of functions.\nA mutant, 102, was isolated by imprecise excision of the\nP-element in ﬁli-lacZ. It lacks 534 bp, including a part of the\n**BLOCK**fs== 8.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nFIG. 4. Involvement of clone shape and hid expression. (A) Clones\nexpressing green ﬂuorescent protein alone. All of the clones show an\nirregular shape. (B) ss and DIAP1 (IAP)-overexpressing clones. Most\nof the clones are rounded. (C) Higher magniﬁcation of a clone boxed\nin panel B. JNK activation is found nonautonomously but highly ac-\ncumulated inside the clone, whereas caspase-3 activation is found only\noutside of the clone, which may be due to the ability of DIAP1 to\nbreakdown caspase-3. (D) ss-overexpressing clones in a hep mutant\nbackground. Most of the clones are rounded. (E) ss-overexpressing\nclones in a hid mutant background. Most of the clones are rounded.\n(F) ss-overexpressing clones under the control of tgoRNAi. Most of the\nclones show an irregular shape and do not induce activation of JNK\nand caspase-3 signiﬁcantly. (G and G) Expression of constitutively\nactive Hep (green) induces hid-lacZ expression (magenta). To create\nan intact shape of the clone by repressing apoptosis, the enhancer trap\nmutation hid-lacZ (hid05014) is homozygous in this experiment. (H and\nH) ss overexpression (green) induces hid-lacZ expression (magenta)\nautonomously in the wing blade region (arrow) and nonautonomously\nin the wing hinge region (arrowheads).\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nFIG. 6. Expression patterns of LRR protein genes. (A to A) Ex-\npression of ﬁli-lacZ (magenta) and Dll (green) in the wing, leg, and\nantenna discs; (B to B) in situ hybridization showing ﬁli mRNA; (C)\nexpression of Caps (blue), trn-lacZ (red), and Sal (green) in the wing\ndisc.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nFIG. 5. Ectopic expression of various homeotic genes commonly\ninduces autonomous death in the wing blade region and nonautono-\nmous death in the wing hinge region. (A to F) Wings were generated\nwith ectopic expression of the indicated homeotic genes by using sd-\nGAL4(s) or sd-GAL4(w) drivers. All of the wings show abnormal vein\npatterns and incision of the wing margin at various frequencies. (A to\nF and A to F) Wing imaginal discs with mosaic clones ectopically\nexpressing the indicated homeotic genes (green). puc-lacZ expression\nis indicated in magenta, and caspase-3 activation is indicated in cyan.\nArrows and arrowheads denote an autonomous and nonautonomous\nactivation of JNK, respectively. The total numbers of clones examined\nwere 30 (lab), 22 (pb) 44 (Antp), 16 (Ubx), 48 (abd-A), and 33 (Abd-B).\nAmong these, the numbers of clones exhibiting autonomous puc-lacZ\ninduction are 15 (lab), 6 (pb) 6 (Antp), 4 (Ubx), 2 (abd-A), and 9\n(Abd-B), whereas the numbers of those exhibiting nonautonomous\npuc-lacZ induction are 10 (lab), 4 (pb) 14 (Antp), 4 (Ubx), 4 (abd-A),\nand 16 (Abd-B).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nﬁrst exon, and is lethal at late embryonic stages. The main\ncause of this lethality is unknown. In the wing disc, ﬁli exhibited\na pattern of gene expression that was nearly complementary to\nthat of Dll (Fig. 6A and B). In addition, ﬁli expression was\ncomplementary to Dll expression in antenna discs (Fig. 6A\nand B). In contrast, in the leg disc, ﬁli seemed to partially\noverlap with Dll (Fig. 6A and B). However, we found that\nmost of the cells expressing the respective genes were in dif-\nferent layers of the leg disc folding.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nss-induced apoptosis can be controlled by Fili. If ﬁli is in-\nvolved in the control of apoptosis in ss-overexpressing clone, its\nexpression should be downregulated by ss. As expected, ﬁli\nexpression is repressed in the ss-overexpressing clones (Fig.\n8A). Interestingly, in contrast, expression of caps (Fig. 8A)\nand trn (data not shown) is enhanced by ss. Similar responses\nin ﬁli and caps can also be observed in Dll-overexpressing\nclones (Fig. 8B to B).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nWhen ss was induced together with ﬁli, JNK and caspase-3\nactivation was greatly reduced (Fig. 3A and 9A to A). Resid-\nual JNK and caspase-3 activation was observed only in the\nmost peripheral hinge region, where ﬁli is not expressed in the\nnormal wing. Similarly, in the central wing region, where ﬁli is\nnot expressed in the normal wing, caspase-3 activation was not\nsuppressed by ﬁli coexpression. As a control experiment, we\nobserved that coexpression of both caps and ss did not suppress\nthe ss-induced apoptotic response (Fig. 9B and B). Consis-\ntently, ss overexpression does not repress but rather elevates\nthe caps expression (Fig. 8A). Thus, we conclude that ﬁli\nexpression is sufﬁcient for survival of ss-overexpressing cells in\nthe wing region where endogenous ﬁli is expressed, probably\nthrough recognition by surrounding normal cells transmitting a\nsurvival signal, as shown for Caps and Trn (38).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nWe further carried out a control experiment to test the effect\nof ﬁli on sal-induced apoptosis because sal-induced apoptosis\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nFIG. 7. Structure of ﬁli gene and Fili protein. (A) The horizontal line represents a segment of the genomic DNA around ﬁli gene. The ﬁve exons\nin ﬁli gene are indicated below the DNA. The P-element inserted in the ﬁli-lacZ ﬂy genome is indicated by the downward stippled arrowhead at\nthe position 56 relative to the potential transcription start site. The deleted region in ﬁli102 mutant is indicated around 56. “cen” and “telo”\ndenote the direction toward centromere and telomere, respectively. (B) Domain organization of Fili protein. The deduced Fili protein possesses\ntwo hydrophobic stretches (orange), signal peptide (SP), and transmembrane domain (TM). In the extracellular domain, 14 LRRs are found\n(green). NF and CF refer to the amino-ﬂanking and carboxy-ﬂanking motifs, respectively. (C) Comparison of 14 LRRs in Fili. The identical amino\nacid residues are masked with orange. (D) Comparison of LRR consensus among ﬁve LRR family members. Variable amino acid residues are\nshown with hyphens. (E and F) Comparison of NF and CF among Fili, Trn, and Caps. Identical and similar amino acid residues are shown in\norange and cyan, respectively.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nwas reported to be suppressed by coexpression of caps or trn.\nAt ﬁrst, we observed that the sal-overexpressing clones dis-\nplayed a nonautonomous activation of JNK in the wing region\nwhere sal is not normally expressed (Fig. 9C to C). This\nnonautonomy was quite similar to the morphogenetic apopto-\nsis and ss-induced apoptosis, suggesting that the regulation of\nall of these apoptotic responses may share similar molecular\nmechanism(s). When sal was induced together with ﬁli, JNK,\nand caspase-3 activation was not affected signiﬁcantly (Fig. 9D\nto D). Thus, these results indicate that ﬁli does not suppress\nall cases of nonautonomously induced apoptosis.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nShape and apoptosis of clones mutant for or overexpressing\nﬁli. Next, we examined the relationship of the clone shape and\napoptosis in a simple ﬁli mutant or in its overexpression. The\nﬁli mutant clones showed an irregular shape irrespective of\ntheir position in the wing disc (Fig. 10A) and did not display\nany apoptotic signs. These results are not always discrepant\nfrom the other results and are similar to observations in mu-\ntants for caps or trn alone (38). The other LRR family proteins\nmight function with Fili in a redundant manner in the gener-\nation of afﬁnity with surrounding normal cells. In contrast to\nthe ﬁli mutant, the ﬁli-overexpressing clones showed a smooth\noutline (Fig. 10B). The smooth outlines were found even\nwhere endogenous ﬁli was expressed, which may be due to the\nimmense amount of its expression by using UAS/GAL4 system.\nHowever, the clone in the central region where endogenous ﬁli\nwas not expressed exhibited the most rounded shape (Fig.\n10B). Interestingly, unlike the case of caps and trn, the over-\nexpression of ﬁli alone induced apoptosis in a nonautonomous\nmanner. When UAS-ﬁli was induced by ptc-GAL4 (Fig. 10D),\nnonautonomous activation of JNK was apparent in the cells\naround the posterior edge of ptc expression but not in the cells\naround the anterior edge (Fig. 10D to D). The posterior edge\nis known to create a sharp discontinuity of ptc expression\nlevels, whereas the anterior edge does not (5). More interest-\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\ningly, the area where JNK was activated did not overlap the\narea where endogenous ﬁli was expressed. As a result, the\nareas with JNK activation were divided into three regions\n(arrowheads in Fig. 10D). Also, forced expression of ﬁli in the\ncentral wing blade in the adult wing generated a severely\nnotched phenotype by apoptosis (Fig. 10C). These data\nstrongly suggest that the occurrence of a large discontinuity in\nthe ﬁli expression level activates JNK nonautonomously.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nDISCUSSION\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nEctopic ss induces a homeosis-dependent apoptosis that\nmay cause a rare frequency of wing-to-leg transdetermination.\nHomeosis is a naturally occurring rare phenomenon with for-\nmation of a local body part having characteristics that are\nnormally found in a related part at another location in the\nbody. This phenomenon had been studied exclusively in Dro-\nsophila melanogaster but can also be found in a variety of\ninsects and other organisms (53). Although homeosis is rarely\nfound in wild-type ﬂies, transdetermination, a similar phenom-\nenon, can be easily induced by in vivo culture and subsequent\ntransplantation of imaginal discs between different individuals\nor by directly manipulating functions of various master and/or\nhomeotic genes. Although such developmentally abnormal\ncells occurring in homeosis might a priori be expected to be\nremoved during development to guarantee normal morpho-\ngenesis, apoptosis has not been previously observed to accom-\npany homeosis. Our results suggest that the cells undergoing\nhomeosis do in fact induce apoptotic responses to help prevent\nthem from developing abnormal structures. The rare fre-\nquency in wing-to-leg transdetermination, which was observed\nabout 30 years ago (30), may be derived partly from the high\nfrequency of apoptosis associated with this particular transfor-\nmation.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFIG. 8. Ss and Dll regulate the expression of ﬁli and Caps. (A to\nA) ﬁli-lacZ expression (red) is repressed, whereas Caps expression\n(blue) is induced in the ss-overexpressing clones (green). (B to B)\nﬁli-lacZ expression (red) is repressed, whereas Caps expression (blue)\nis induced in the Dll-overexpressing clones (green). Areas of repres-\nsion (gray arrows) and induction (white arrows) are indicated.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nss-induced autonomous apoptosis is related to cell compe-\ntition. Cell competition is a phenomenon whereby cell clones\nwith a slow growth rate are eliminated during development.\nThe autonomous cell death in cell competition has been dem-\nonstrated to be caused by an impaired reception of extracel-\nlular survival factors. As representative examples, clones with\ncells heterozygous for the M(2)60E mutation show an auton-\nomous death as a result of reduced Dpp reception (40). Thus,\nthe autonomous death in ss-overexpressing cells in the wing\nblade might be a type of cell competition response. In this case,\nhowever, the extracellular factor that the ss-overexpressing\ncells fail to receive is not likely to be Dpp, since we observed\nthat the Dpp signaling level is not strongly affected in the\nss-overexpressing cells (see Fig. S1A and B in the supplemental\nmaterial). Also, ss-overexpressing cells can be removed from\nthe lateral region of the wing blade where the level of Dpp\nsignaling is normally low. In contrast, cells heterozygous for\nM(2)60E can survive in this area due to the loss of competition\nfor Dpp (40). Therefore, Wg or other extracellular survival\nfactors might be affecting the survival of ss-overexpressing\ncells, as discussed below.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nPosition-dependent variance in cell autonomy is found in\nthe apoptotic response to ss overexpression. We have not yet\naddressed the question of why the variance in cell autonomy of\nss-induced apoptosis is related to dorsoventral patterning or\nblade/hinge subdomains. However, the variance suggests an\ninteresting model in which signals that normally regulate dor-\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nFIG. 9. Fili regulates ss-induced apoptosis. Areas of puc-lacZ ex-\npression (red), caspase-3 activation (cyan), and Caps expression (blue)\nare indicated. (A to A) Wing imaginal disc with mosaic clones over-\nexpressing ss and ﬁli (green). JNK activation and caspase-3 activation\nare suppressed. (B to B) Wing imaginal disc with mosaic clones\noverexpressing ss and caps (green). JNK activation is not suppressed by\nCaps. (C to C) Wing imaginal disc with mosaic clones overexpressing\nsal (green). Nonautonomous activation of JNK is found at the region\nwhere endogenous sal is not expressed normally. (D to D) Wing\nimaginal disc with mosaic clones overexpressing sal and ﬁli (green).\nJNK activation is not suppressed.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nsoventral appendage patterning or blade/hinge subdomains\nalso directly affect the apoptotic response or speciﬁcation of\norgan identity as cryptic mechanisms. Interestingly, ectopic\nexpression of most of the segment-specifying homeotic genes\ncommonly shows a similar position-dependent variance in cell\nautonomy (Fig. 5). In this view, Wg, the morphogen controlling\nthe dorsoventral patterning (42, 60), or Vg, the selector of wing\nsubdomain (31, 34), may be involved in the regulation of po-\nsition-dependent variance in the cell autonomy of apoptosis.\nHowever, deﬁnitively demonstrating the involvement of Wg or\nVg is difﬁcult because changing Wg signal or Vg activity alone\nalso induces a severe unrelated apoptosis (2). Thus, the simul-\ntaneous manipulation of ss and Wg/Vg activities may lead to a\ncomplicated result that is difﬁcult to interpret.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nImplication of various LRR family transmembrane proteins\nin the regulation of apoptosis to remove misspeciﬁed cells. A\nrecent study concerning the relation of apoptosis in sal-over-\nexpressing cells and LRR family proteins Caps/Trn has pro-\nvided a new insights into how cell survival of misspeciﬁed cells\nis controlled (38). Cells that overexpress sal lack caps/trn ex-\npression and are removed from the area where sal is not\nnormally expressed. However, cells expressing both sal and\ncaps can survive. In order to control cell survival in the wing,\nCaps and Trn presumably provide position-dependent recog-\nnition cues along the anteroposterior axis, whereas Fili appears\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nvation of mammalian Ahr is known to affect apoptosis, cell\nproliferation, differentiation, and morphogenesis in various tis-\nsues (50). Polycyclic aromatic hydrocarbon, a ligand for Ahr,\ninduced apoptosis accompanied by the activation of JNK (33).\nIt should also be noted that dioxin-induced responses such as\nP450 expression in the liver are not observed in all cells ex-\nposed to dioxin but are observed in a subset of centrilobular\ncells with the highest sensitivity to dioxin (4). Thus, the mosaic\nactivation of Ss and subsequent JNK activation in surrounding\ncells in Drosophila induce circumstances similar to that of liver\ncells exposed to dioxin. With regard to morphogenesis, the\ntype of morphogenetic signals that are modiﬁed or distorted\nduring dioxin action is unknown, despite many examples of\ndioxin-induced mismorphogenesis that have been reported.\nTherefore, our studies on Ss in Drosophila may provide at least\nin part, a common molecular and cellular basis for understand-\ning Ahr-induced apoptosis and mismorphogenesis. It could\ninvolve homeotic transformation of organ identity, recognition\nby surrounding cells through LRR protein, and apoptosis-me-\ndiated large deletion of tissues that can cause a malformation\nduring organogenesis. Interestingly, all of the genes involved in\nthis sequential process are evolutionarily conserved in mam-\nmals. Similar and detailed analysis in mammalian system will\nresolve these issues.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nACKNOWLEDGMENTS\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nWe thank R. Barrio, S. Campbell, S. T. Crews, A. Garcı´a-Bellido,\nB. A. Hay, Y. Hiromi, Y. H. Inoue, K. Irvine, W. Janning, E. Martı´n-\nBlanco, A. Martinez-Arias, A. Nose, S. Noselli, C. Rauskolb, M.\nShinza-Kameda, G. Struhl, K. Takahashi, and D. Yamamoto, Bloom-\nington Stock Center, and GETDB (GAL4-Enhancer Trap Data Base)\nin the National Institute of Genetics in Japan, for the ﬂy strains; R.\nBarrio, G. Campbell, S. B. Carroll, D. Duncan, A. Nose, M. Shinza-\nKameda, and P. ten Dijke, Developmental Studies Hybridoma Bank,\nIowa University, and Idun Pharmaceutical, Inc., for antibodies; J. Kim\nand Y. Lee for technical advice; and Y. Aoki and O. Habara for\ntechnical assistance.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThis study was supported by grants from the Japan Science and\nTechnology Agency; the Ministry of Education, Science, Sports, and\nCulture in Japan (to T.A.-Y.); and NIH and the Human Frontier\nScience Program (to H.N.). M.B.O. is an Investigator of the Howard\nHughes Medical Institute.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n1. Abrams, J. M. 2002. Competition and compensation: coupled to death in\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\ndevelopment and cancer. Cell 110:403–406.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n2. Adachi-Yamada, T., and M. B. O’Connor. 2002. Morphogenetic apoptosis: a\nmechanism for correcting discontinuities in morphogen gradients. Dev. Biol.\n251:74–90.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n3. Adachi-Yamada, T., and M. B. O’Connor. 2004. Mechanisms for removal of\ndevelopmentally abnormal cells: cell competition and morphogenetic apo-\nptosis. J. Biochem. 136:13–17.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n4. Andersen, M. E. 2002. The use of quantitative histological and molecular\ndata for risk assessment and biologically based model development. Toxicol.\nPathol. 30:101–106.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n5. Aza-Blanc, P., and T. B. Kornberg. 1999. Ci: a complex transducer of the\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nhedgehog signal. Trends Genet. 15:458–462.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n6. Bier, E., H. Vaessin, S. Shepherd, K. Lee, K. McCall, S. Barbel, L. Acker-\nman, R. Carretto, T. Uemura, E. Grell, L. Y. Jan, and Y. N. Jan. 1989.\nSearching for pattern and mutation in the Drosophila genome with a P-lacZ\nvector. Genes Dev. 3:1273–1287.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n7. Brand, H., and N. Perrimon. 1993. Targeted gene expression as a means of\naltering cell fates and generating dominant phenotypes. Development 118:\n401–415.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n8. Campbell, G., T. Weaver, and A. Tomlinson. 1993. Axis speciﬁcation in the\ndeveloping Drosophila appendage: the role of wingless, decapentaplegic, and\nthe homeobox gene aristaless. Cell 74:1113–1123.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n9. Campbell, G., and A. Tomlinson. 1998. The roles of the homeobox genes\naristaless and Distal-less in patterning the legs and wings of Drosophila.\nDevelopment 125:4483–4493.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nFIG. 10. Phenotypes of ﬁli mutant and ectopic expression.\n(A) Wing disc with twin spots generated by somatic recombination of\nﬁli102. Clones without green ﬂuorescent protein are the homozygous\nﬁli102 mutant. (A) High magniﬁcation of the boxed area in panel A.\n(B) Wing disc with clones with ﬁli overexpression (green). (B) High\nmagniﬁcation of the boxed area in panel B. (C) Adult wing where\nUAS-ﬁli is driven by ptc-GAL4. (D to D) Wing disc where UAS-ﬁli is\ndriven by ptc-GAL4 (green). Areas of puc-lacZ expression (magenta)\nand caspase-3 activation (cyan) are also shown. Arrowheads indicate\nnonautonomous induction of puc-lacZ expression in places where en-\ndogenous ﬁli is not expressed.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nto provide such cues to neighboring cells along the dorsoven-\ntral axis. These results raise the possibility that cells can rec-\nognize the type of neighboring cells using distinct LRR family\nmembers, depending on the position of the cells in the primor-\ndial tissues. The Drosophila genome contains at least 10 mem-\nbers of LRR family, most of which have not been examined for\nfunction (Flybase).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFunctional relationship between Drosophila Ss and mamma-\nlian Ahr. In the present study, several cellular responses to ss\noverexpression have been described. Clonal overexpression of\nss induces a sequential response composed of an induction of\nleg- or antenna-speciﬁc genes, autonomous homeosis, round-\ning, and autonomous or nonautonomous apoptosis. The acti-\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n10. Campbell, S., M. Inamdar, V. Rodrigues, V. Raghavan, M. Palazzolo, and A.\nChovnick. 1992. The scalloped gene encodes a novel, evolutionarily con-\nserved transcription factor required for sensory organ differentiation in Dro-\nsophila. Genes Dev. 6:367–379.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n11. Carroll, S. B., S. D. Weatherbee, and J. A. Langeland. 1995. Homeotic genes\nand the regulation and evolution of insect wing number. Nature 375:58–61.\n12. Casares F., and R. S. Mann. 1998. Control of antennal versus leg develop-\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nment in Drosophila. Nature 392:723–726.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n13. Cohen, S. M., G. Bronner, F. Kuttner, G. Jurgens, and H. Jackle. 1989.\nDistal-less encodes a homeodomain protein required for limb development\nin Drosophila. Nature 338:432–434.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n14. Davis, R. J. 2000. Signal transduction by the JNK group of MAP kinases.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nCell 103:239–252.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n15. Dong, P. D. S., J. Chu, and G. Panganiban. 2001. Proximodistal domain\nspeciﬁcation and interactions in developing Drosophila appendages. Devel-\nopment 128:2365–2372.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n16. Dong, P. D. S., S. J. Dicks, and G. Panganiban. 2002. Distal-less and homo-\nthorax regulate multiple targets to pattern the Drosophila antenna. Devel-\nopment 129:1967–1974.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n17. Duncan, D. M., E. A. Burgess, and I. Duncan. 1998. Control of distal\nantennal identity and tarsal development in Drosophila by spineless-aristape-\ndia, a homolog of the mammalian dioxin receptor. Genes Dev. 12:1290–1303.\n18. Emmons, R. B., D. Duncan, P. A. Estes, P. Kiefel, J. T. Mosher, M. Son-\nnenfeld, M. P. Ward, I. Duncan, and S. T. Crews. 1999. The Spineless-\nAristapedia and Tango bHLH-PAS proteins interact to control antennal and\ntarsal development in Drosophila. Development 126:3937–3945.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n19. Evan G., and T. Littlewood. 1998. A matter of life and cell death. Science\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n281:1317–1322.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n20. Glise, B., H. Bourbon, and S. Noselli. 1995. hemipterous encodes a novel\nDrosophila MAP kinase kinase, required for epithelial cell sheet movement.\nCell 83:451–461.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n21. Gorﬁnkiel, N., G. Morata, and I. Guerrero. 1997. The homeobox gene\nDistal-less induces ventral appendage development in Drosophila. Genes\nDev. 11:2259–2271.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n22. Grenier, J. K., and S. B. Carroll. 2000. Functional evolution of the Ultra-\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nbithorax protein. Proc. Natl. Acad. Sci. USA 97:704–709.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n23. Grether, M. E., J. M. Abrams, J. Agapite, K. White, and H. Steller. 1995. The\nhead involution defective gene of Drosophila melanogaster functions in pro-\ngrammed cell death. Genes Dev. 15:1694–1708.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n24. Grimm. S, and G. O. Pﬂugfelder. 1996. Control of the gene optomotor-blind\nin Drosophila wing development by decapentaplegic and wingless. Science\n271:1601–1604.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n25. Hamada, F., Y. Tomoyasu, Y. Takatsu, M. Nakamura, S. Nagai, A. Suzuki,\nF. Fujita, H. Shibuya, K. Toyoshima, N. Ueno, and T. Akiyama. 1999.\nNegative regulation of Wingless signaling by D-axin, a Drosophila homolog\nof axin. Science 283:1739–1742.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n26. Hay, B. A., D. A. Wassarman, and G. M. Rubin. 1995. Drosophila homologs\nof baculovirus inhibitor of apoptosis proteins function to block cell death.\nCell 83:1253–1262.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n27. Hoffman, E. C., H. Reyes, F. F. Chu, F. Sander, L. H. Conley, B. A. Brooks,\nand O. Hankinson. 1991. Cloning of a factor required for activity of the Ah\n(dioxin) receptor. Science 252:954–958.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n28. Ito, K., W. Awano, K. Suzuki, Y. Hiromi, and D. Yamamoto. 1997. The\nDrosophila mushroom body is a quadruple structure of clonal units each of\nwhich contains a virtually identical set of neurons and glial cells. Develop-\nment 124:761–771.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n29. Jacobson, M. D., M. Weil, and M. C. Raff. 1997. Programmed cell death in\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nanimal development. Cell 88:347–354.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n30. Kauffman, S. A. 1973. Control circuits for determination and transdetermi-\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nnation. Science 181:310–318.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n31. Kim, J., A. Sebring, J. J. Esch, M. E. Kraus, K. Vorwerk, J. Magee, and S. B.\nCarroll. 1996. Integration of positional signals and regulation of wing for-\nmation and identity by Drosophila vestigial gene. Nature 382:133–138.\n32. Lecuit, T., W. J. Brook, M. Ng, M. Calleja, H. Sun, and S. M. Cohen. 1996.\nTwo distinct mechanisms for long-range patterning by Decapentaplegic in\nthe Drosophila wing. Nature 381:387–393.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n33. Lei, W., R. Yu, S. Mandlekar, and A. N. Kong. 1998. Induction of apoptosis\ninterleukin 1-converting enzyme/Ced-3 protease\nand activation of\n(caspase-3) and c-Jun NH2-terminal kinase 1 by benzo(a)pyrene. Cancer\nRes. 58:2102–2106.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n34. Liu, X., M. Grammont, and K. D. Irvine. 2000. Roles for scalloped and\nvestigial in regulating cell afﬁnity and interactions between the wing blade\nand the wing hinge. Dev. Biol. 228:287–303.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n35. Mardon, G., N. M. Solomon, and G. M. Rubin. 1994. dachshund encodes a\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nnuclear protein required for normal eye and leg development in Drosophila.\nDevelopment 120:3473–3486.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n36. Martı´n-Blanco, E., A, Gampel, J. Ring, K. Virdee, N. Kirov, A. M. Tolkovsky,\nand A. Martinez-Arias. 1998. puckered encodes a phosphatase that mediates\na feedback loop regulating JNK activity during dorsal closure in Drosophila.\nGenes Dev. 12:557–570.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n37. Mila´n, M., S. Campuzano, and A. Garcı´a-Bellido. 1997. Developmental\nparameters of cell death in the wing disc of Drosophila. Proc. Natl. Acad. Sci.\nUSA 94:5691–5696.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n38. Mila´n, M., L. Pe´rez, and S. M. Cohen. 2002. Short-range cell interactions\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.2**\nand cell survival in the Drosophila wing. Dev. Cell 2:797–805.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n39. Morata, G., and P. Ripoll. 1975. Minutes: mutants of Drosophila autono-\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nmously affecting cell division rate. Dev. Biol. 42:211–221.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n40. Moreno, E., K. Basler, and G. Morata. 2002. Cells compete for Decapen-\ntaplegic survival factor to prevent apoptosis in Drosophila wing development.\nNature 416:755–759.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n41. Nellen, D., R. Burke, G. Struhl, and K. Basler. 1996. Direct and long-range\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\naction of a DPP morphogen gradient. Cell 85:357–368.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n42. Neumann, C. J., and S. M. Cohen. 1997. Long-range action of Wingless\norganizes the dorsal-ventral axis of the Drosophila wing. Development 124:\n871–880.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n43. Ohshiro, T., and K. Saigo. 1997. Transcriptional regulation of breathless FGF\nreceptor gene by binding of TRACHEALESS/dARNT heterodimers to\nthree central midline elements in Drosophila developing trachea. Develop-\nment 124:3975–3986.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n44. Pai, C.-Y., T.-S. Kuo, J. J. Thomas, E. Kurant, C.-T. Chen, D. A. Bessarab,\nA. Salzberg, and Y. H. Sun. 1998. The homothorax homeoprotein activates\nthe nuclear localization of another homeoprotein, Extradenticle, and sup-\npresses eye development in Drosophila. Genes Dev. 12:435–446.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n45. Panganiban, G. 2000. Distal-less function during Drosophila appendage and\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nsense organ development. Dev. Dyn. 218:554–562.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n46. Peifer, M., and E. Wieschaus. 1990. The segment polarity gene armadillo\nencodes a functionally modular protein that is the Drosophila homolog of\nhuman plakoglobin. Cell 63:1167–1176.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n47. Peifer, M., D. Sweeton, M. Casey, and E. Wieschaus. 1994. wingless signal\nand zeste-white 3 kinase trigger opposing changes in the intracellular distri-\nbution of Armadillo. Development 120:369–380.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n48. Persson, U., H. Izumi, S. Souchelnytskyi, S. Itoh, S. Grimsby, U. Engstrom,\nC. H. Heldin, K. Funa, and P. ten Dijke. 1998. The L45 loop in type I\nreceptors for TGF- family members is a critical determinant in specifying\nSmad isoform activation. FEBS Lett. 434:83–87.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n49. Prober, D. A., and B. A. Edgar. 2000. Ras1 promotes cellular growth in the\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nDrosophila wing. Cell 100:435–446.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n50. Robertson, J. D., and S. Orrenius. 2000. Molecular mechanisms of apoptosis\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\ninduced by cytotoxic chemicals. Crit. Rev. Toxicol. 30:609–627.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n51. Sato, A., T. Kojima, K. Ui-Tei, Y. Miyata, and K. Saigo. 1999. Dfrizzled-3, a\nnew Drosophila Wnt receptor, acting as an attenuator of Wingless signaling\nin wingless hypomorphic mutants. Development 126:4421–4430.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n52. Sekelsky, J. J., S. J. Newfeld, L. A. Raftery, E. H. Chartoff, and W. M.\nGelbart. 1996. Genetic characterization and cloning of mothers against dpp,\na gene required for decapentaplegic function in Drosophila melanogaster.\nGenetics 139:1347–1358.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n53. Sibatani, A. 1980. Wing homeosis in Lepidoptera: a survey. Dev. Biol. 79:\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1–18.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n54. Siegfried, E., E. L. Wilder, and N. Perrimon. 1994. Components of wingless\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nsignalling in Drosophila. Nature 367:76–80.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n55. Sivasankaran, R., M. Calleja, G. Morata, and K. Basler. 2000. The Wingless\ntarget gene Dfz3 encodes a new member of the Drosophila Frizzled family.\nMech. Dev. 91:427–431.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n56. Simpson, P., and G. Morata. 1981. Differential mitotic rates and patterns of\ngrowth in compartments in the Drosophila wing. Dev. Biol. 85:299–308.\n57. Tanimoto, H., S. Itoh, P. ten Dijke, and T. Tabata. 2000. Hedgehog creates\na gradient of DPP activity in Drosophila wing imaginal discs. Mol. Cell\n5:59–71.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n58. Yoo, S. J., J. R. Huh, I. Muro, H. Yu, L. Wang, S. L. Wang, R. M. Feldman,\nR. J. Clem, H. A. Muller, and B. A. Hay. 2002. Hid, Rpr, and Grim negatively\nregulate DIAP1 levels through distinct mechanisms. Nat. Cell Biol. 4:416–\n424.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n59. Yu, S. Y., S. J. Yoo, L. Yang, C. Zapata, A. Srinivasan, B. A. Hay, and N. E.\nBaker. 2002. A pathway of signals regulating effector and initiator caspases\nin the developing Drosophila eye. Development 129:3269–3278.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n60. Zecca, M., K. Basler, and G. Struhl. 1996. Direct and long-range action of a\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.2**\nwingless morphogen gradient. Cell 87:833–844.",
         "extracted",
         "",
         "Wing-to-Leg Homeosis by Spineless Causes Apoptosis Regulated by Fish-lips, a Novel Leucine-Rich Repeat Transmembrane Protein",
         "None"
        ],
        [
         "29",
         "0025eebeec63c1e7a74d3d1f355fe0cd214a504f",
         "The release of proteins from tumors triggers an immune response in cancer patients. These tumor antigens arise from several mechanisms including tumor-specific alterations in protein expression, mutation, folding, degradation, or intracellular localization. Responses to most tumor antigens are rarely observed in healthy individuals, making the response itself a biomarker that betrays the presence of underlying cancer. Antibody immune responses show promise as clinical biomarkers because antibodies have long half-lives in serum, are easy to measure, and are stable in blood samples. However, our understanding of the specificity and the impact of the immune response in early stages of cancer is limited. The immune response to cancer, whether endogenous or driven by vaccines, involves highly specific T lymphocytes (which target tumor-derived peptides bound to self-MHC proteins) and B lymphocytes (which generate antibodies to tumor-derived proteins). T cell target antigens have been identified either by expression cloning from tumor cDNA libraries, or by prediction based on patterns of antigen expression (\"reverse immunology\"). B cell targets have been similarly identified using the antibodies in patient sera to screen cDNA libraries derived from tumor cell lines. This review focuses on the application of recent advances in proteomics for the identification of tumor antigens. These advances are opening the door for targeted vaccine development, and for using immune response signatures as biomarkers for cancer diagnosis and monitoring.",
         "K. Anderson,J. LaBaer",
         "https://europepmc.org/articles/pmc2522321?pdf=render",
         "\n**BLOCK**fs== 11.0**b== 0.9**t== 0.1**l== 0.2**r== 0.3**\nNIH Public Access\nAuthor Manuscript\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\nPublished in final edited form as:\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nJ Proteome Res. 2005 ; 4(4): 1123–1133. doi:10.1021/pr0500814.\n**BLOCK**fs== 14.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nThe sentinel within: exploiting the immune system for cancer\n**BLOCK**fs== 14.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nbiomarkers\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nKaren S. Anderson1 and Joshua LaBaer2\n1Dana-Farber Cancer Institute, Harvard Medical School, Boston MA\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\n2Harvard Institute of Proteomics, Harvard Medical School, Boston MA 02115\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nAbstract\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.2**r== 0.2**\nThe release of proteins from tumors triggers an immune response in cancer patients. These tumor\nantigens arise from several mechanisms including tumor-specific alterations in protein expression,\nmutation, folding, degradation, or intracellular localization. Responses to most tumor antigens are\nrarely observed in healthy individuals, making the response itself a biomarker that betrays the\npresence of underlying cancer. Antibody immune responses show promise as clinical biomarkers\nbecause antibodies have long half lives in serum, are easy to measure, and are stable in blood samples.\nHowever, our understanding of the specificity and the impact of the immune response in early stages\nof cancer is limited. The immune response to cancer, whether endogenous or driven by vaccines,\ninvolves highly specific T lymphocytes (which target tumor-derived peptides bound to self-MHC\nproteins) and B lymphocytes (which generate antibodies to tumor-derived proteins). T cell target\nantigens have been identified either by expression cloning from tumor cDNA libraries, or by\nprediction based on patterns of antigen expression (“reverse immunology”). B cell targets have been\nsimilarly been identified using the antibodies in patient sera to screen cDNA libraries derived from\ntumor cell lines. This review focuses on the application of recent advances in proteomics for the\nidentification of tumor antigens. These advances are opening the door for targeted vaccine\ndevelopment, and for using immune response signatures as biomarkers for cancer diagnosis and\nmonitoring.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nKeywords\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.2**\nTumor antigen; antibody; protein array; proteomics; tumor immunology; biomarkers\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nIntroduction\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.3**r== 0.1**\nThe challenge faced by our immune system resembles that of an intelligent security system,\nwhich must continually monitor for the presence of foreign invaders, while recognizing and\ndisregarding normal self. Like a vigilant sentry, immunologic memory persists long after\nexposure to the threat has abated. Recognizing the value of this persistent response, clinicians\nhave exploited it for years to test individuals for current or past exposure to a wide variety of\ninfections. Compared to other serum-derived proteins, antibodies are stable, highly specific,\nand readily detected with well-validated secondary reagents, making them ideal for such tests.\nIndeed, the traditional “blood test” required of couples before obtaining a marriage license is\nnothing more than a test for antibodies to the spirochete T. pallidum that causes syphilis. Thus,\nassessing immune responses is one of the oldest and most successful forms of biomarkers in\nmedicine.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.1**\nThe immune system employs complex mechanisms to distinguish between self and non-self.\nIt deletes or renders tolerant any cells which react to the constant stream of benign\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\nmacromolecules in routine circulation. The system is not foolproof, however, and in certain\ndiseases, the immune system responds to self-derived antigens, perhaps because their location,\nabundance, modified form or other features appear unfamiliar. Cancer patients often produce\nresponses to self-proteins that are expressed by their tumors, called tumor antigens, most of\nwhich are altered in some form that renders them immunogenic. These proteins may be unique\nto cancer and germ cells (the “cancer-testis” antigens), found only in specific tumors (prostate-\nspecific antigen)1 or in most tumors (telomerase)2.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.3**r== 0.1**\nThey may be mutated (p53)3, misfolded4, overexpressed (NY-ESO-1) 5, aberrantly\ndegraded6 or aberrantly glycosylated (MUC-1)7. The magnitude of the immune response to\ncancer, in general, is lower than the immune response to infectious agents and the potential\nnumber of tumor antigens encompasses the entire tumor proteome in all its variations. At\npresent, we have a limited understanding of the breadth, extent, impact, and dynamic variation\nof the immune response to cancer (the “cancer immunome”). Identifying the specific targets\nof B- and T- lymphocyte immunity to cancer may 1) identify potential biomarkers for cancer\ndiagnosis, classification, and monitoring of response, 2) determine the impact of immune\nregulation on cancer progression, and 3) identify potential antigens and mechanisms for\nimmunotherapy development.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.3**r== 0.1**\nThe natural immune response is achieved through a tightly regulated, yet flexible network\nincluding antibodies, antigen presenting cells, T lymphocytes, cytokines, chemokines,\nregulatory systems, as well as microenvironmental signals (Figure 1). Of these responses, the\ntargeted responses to protein (and carbohydrate) antigens relies on the development of\nantibodies and/or T lymphocytes to target epitopes. T lymphocytes can respond to antigens\nderived from within cells and without. They primarily recognize short peptides (8-22mer)\nderived from intracellular proteins (i.e., viral antigens) bound to self-MHC molecules for\npresentation to CD8+ T lymphocytes. Exogenous antigens are endocytosed, degraded, and\npresented to CD4+ lymphocytes (Figure 2). Antibody responses increase antigen presentation\nby enhancing uptake through the Fcγ receptors on antigen presenting cells. As a result, antibody\ntargets may contain epitopes that are also recognized by T lymphocytes. This has formed the\nbasis for using antibody responses to identify T cell antigens for immunotherapy.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nLessons from Autoimmune Diseases\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\nSpontaneous autoantibodies were first identified in a series of clinical disorders in which the\npatients' immune systems mount a vigorous response to self antigens, in some cases leading\nto debilitating symptoms. Systemic lupus, myasthenia gravis, rheumatoid arthritis and others\nall involve this process, called autoimmunity. In these illnesses, the titers of autoantibodies\noften track with the severity of the illness and thus have long been used as serum biomarkers\n(reviewed in8, 9).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.3**r== 0.1**\nAlthough post hoc arguments can usually be constructed explaining why proven autoantigens\nmay have appeared unfamiliar to the immune system; in general, we do not understand enough\nabout the characteristics that determine whether a protein will act as an autoantigen to predict\nthem a priori. To date, only one or two percent of proteins have been identified as autoantibody\ntargets10. In autoimmune diseases, the autoantigens identified, such as Ro/SSA, La/SSB, Sm,\nRNP, Scl-70 and Jo-1 are primarily intracellular antigens that function in large complexes with\nnucleic acids involved in protein synthesis8, and are thought to arise as a result of\noverexpression, apoptosis, reduced degradation, or similarity to cross-reacting foreign antigens\n(mimicry). It is often not clear whether these autoantibodies have any direct role in disease\npathophysiology. In contrast, antibodies directed at membrane proteins, such as the target\nantigens of hemolytic anemia or the acetocholine receptor in myasthenia gravis may contribute\nto the disease process through complement activation, antibody-dependent cytotoxicity, or\ninterference with receptor/ligand interactions. Certain structural motifs enhance antigenicity,\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\nsuch as carbohydrates, multivalency, epitope repetition, and coiled-coils11. Some of these\nantigens are linked to degradation during apoptosis12, or are targets of granzyme B cleavage\nduring T lymphocyte-induced cytolysis13, 14. Further elucidation of the mechanisms\nunderlying autoantibody generation will assist in predicting and identifying target antigens.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\np53 as a Model Tumor Autoantigen\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.3**r== 0.1**\nPerhaps the most well-studied autoantigen in cancer is the p53 protein (reviewed in 3). In 1979,\nDeLeo et al15 demonstrated that autoantibodies to some tumor cells in mice were directed\nagainst the p53 protein. Subsequent studies have demonstrated that the half-life of mutated p53\n(several hours) was markedly increased compared to wild-type (several minutes) resulting in\naccumulation in the cell nucleus. p53 autoantibodies are dependent on the type of p53\nmutation16-18. Notably, the immunogenic epitopes have been mapped primarily to both the\nN- and C-terminal portions of the molecule, which are heavily glycosylated, but not to the\ncentral portion of the molecule, which harbors the mutations, suggesting that the accumulation\nof protein, rather than the mutations per se, results in autoantibody generation. Multiple assays\nusing recombinant antigens and p53-derived peptides have shown that some antigenic\ndeterminants are dependent on full-length antigen, conformation, or phosphorylation3, 19.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.3**r== 0.1**\nThe detection of p53 autoantibodies in serum correlates strongly with cancer (p<10-4). p53\nautoantibodies have been studied in over 9489 patients with a wide variety of tumors3. Despite\nthe strong specificity of the response, only 20-40% of patients with cancers harboring p53\nmissense mutations will have p53 antibodies in their sera3. Therefore, there are additional\nunknown factors beyond antigen accumulation that impact the development of autoantibodies.\nThis illustrates one of the major challenges facing the use of single autoantibody tests for cancer\ndetection. Although the appearance of such antibodies is highly suggestive of cancer, many\npatients will be missed because they do not respond to p53. p53 autoantibodies have been\ndetected in body fluids, such as ascites, pleural effusions, and saliva, and these correlate with\nserum autoantibodies19-21. Interestingly, gliomas, which are associated with p53 mutations,\nare not associated with the development of p53 antibodies22, 23, suggesting either that\ndevelopment of tumors in the CNS leads to immunologic privilege, or the use of steroids in\nthis patient population could dampen the serologic immune response.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.5**\nAnti-tumor Effects of Autoantibodies in Cancer\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.3**r== 0.1**\nIt is not known if autoantibodies reflect underlying immunosurveillance of cancer or have an\nimpact on the clinical outcome of the disease. For example, multiple studies have attempted\nto correlate p53 autoantibodies with prognosis showing mixed results3. As the development\nof B cell immunity often depends on concordant T cell immunity, autoantibody identification\ncan lead to the identification of relevant T cell antigens24-26. In patients who have undergone\nautologous tumor vaccination, certain autoantibodies correlate with tumor response to\ntreatment25, 27. In bone marrow transplantation, the development of autoantibodies may\nidentify minor histocompatibility antigens associated with graft versus host disease28 or tumor\nantigens associated with the graft-vs-leukemia (GVL) effects of donor lymphocyte\ninfusion29, 30. In murine tumor models, it is the coordinated activities of CD4+ and CD8+ T\ncells, CD1d-restricted invariant NKT cells, and antibodies that accomplish protective\nimmunity 31, 32 33 34.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\nThe serologic identification of tumor antigens has directly led to the development of a number\nof cancer vaccines currently in clinical trials. The proteinase-3 antigen, first identified as the\ntarget antigen of the ANCA antibody assay for Wegener's granulomatosis, is being tested as a\ntarget for vaccination in CML35. NY-ESO-1 has been used as an immunogen in multiple\nclinical trials36. Antibody response to tumor vaccines have been shown to correlate with\nimproved disease-free and overall survival in stage II melanoma patients vaccinated with a\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\npolyvalent vaccine37. Melanoma patients vaccinated with autologous tumor cells may develop\nhighly-individualized antibody responses38. The identification of these autoantibodies has\ndirectly led to mapping of CD4+ and CD8+ T lymphocyte epitopes. The clinical benefit of\nantigen-specific tumor vaccination strategies is just beginning to emerge, with the recent\ndemonstration of a survival benefit of a dendritic cell-based vaccine in hormone-refractory\nprostate cancer39.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nAutoantibodies as Biomarkers of Cancer\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nIn 2003, it is estimated that 1,334,100 people in the US were newly diagnosed with cancer,\nwith an estimated 556,500 patients dying of the disease40. In this setting, there is intense effort\nin the search for biomarkers that can predict disease, identify biologic subtypes of disease,\ntrack response to therapeutic interventions, predict side effect profiles, and monitor for disease\nprogression and recurrence41.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.3**r== 0.1**\nThere are a limited number of serum protein biomarkers that are widely used in clinical\noncology for prognosis and treatment monitoring, and their use in early cancer diagnosis has\nbeen hampered by false-positive rates in the normal population. The identification and\ndevelopment of these biomarkers took decades of research and required large prospective trials.\nCarcinoembryonic antigen (CEA), widely used for the monitoring of adenocarcinomas, was\noriginally identified from tumor lysates because of its immunogenicity in rabbits42. The\nsubsequent identification of the prostate specific antigen (PSA)43 and the ovarian cancer\nbiomarker CA-12544, among others, have demonstrated both the validity and limitations of\nserum biomarkers in the diagnosis of cancer. Although PSA is now routinely used for screening\nhealthy populations for prostate cancer diagnosis, most serum biomarkers are used for\nmonitoring treatment or for screening highly selected high-risk populations. More recent\ndevelopments in serum proteomics have been reviewed in this issue and elsewhere45.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.3**r== 0.1**\nCompared with other polypeptides, autoantibodies have many appealing features as\nbiomarkers. First, although tumor antigens may circulate only briefly or in low concentration,\nperhaps due to transient shedding by tumors, rapid degradation in the serum, or rapid clearance,\nthe corresponding antibody response is likely to be persistent. Second, antibodies are highly\nstable in serum samples and are not subject to the types of proteolysis that are commonly\nobserved for other polypeptides (discussed elsewhere in this issue), making sample handling\nmuch easier. Third, the t1/2 of antibodies in circulation is >7 days, so hourly or daily fluctuations\nare expected to be minimal, simplifying sample collection. Finally, the biochemical properties\nof antibodies are well understood and there are many available reagents for their detection,\nsimplifying assay development. Given these advantages, the challenge that remains is\ndemonstrating that antibodies can be sufficiently informative to reliably detect cancer.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.1**\nEarly Disease Diagnosis—Very limited data are available on the use of autoantibodies to\nreveal early disease. Antibodies have been detected as early as several years before the clinical\nappearance of cancer46, 47 and in patients with preneoplastic disease48. Anecdotal studies\nhave detected p53 autoantibodies in heavy smokers prior to the diagnosis of lung cancer47.\nAnother study showed p53 autoantibodies in individuals exposed to vinyl chloride, a risk factor\nfor the development of angiosarcoma of the liver46. The p53 autoantibodies predated the\ndiagnosis of angiosarcoma by several years. Although encouraging, these few studies are not\nyet sufficient to endorse the value of antibodies in the early detection setting.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\nMonitoring treatment response and predicting recurrence—Antibodies to tumor\nantigens have been detected in early stages of disease and fluctuate with tumor response. In\npatients with both stage III and stage IV neuroblastoma, 10% have evidence of antibodies to\nthe NY-ESO-1 antigen, but the antibody is not seen in the sera of patients in clinical remission\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\nor in earlier stages of disease49. In melanoma, antibodies to TA90 antigen have been detected\nin only 12% of patients with 1-2mm primary, node negative melanoma who subsequently\nrelapsed, but 62% of case-controlled patients who did not50.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\nAntibodies to the her2/neu tumor antigen have been detected in the sera of 20% of patients\nwith her2+ early-stage breast cancer51 but only 7% of late-stage breast and ovarian cancer\npatients52. Some of the early-stage patients had titers>1:5000, suggesting a strong, possibly\nprotective immune response. In patients with both stage III and stage IV neuroblastoma, 10%\nhave evidence of antibodies to the NY-ESO-1 antigen, but the antibody is not seen in the sera\nof patients in clinical remission or in earlier stages of disease49.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.1**\nThe titer of antibodies to NY-ESO-1 have been shown to correlate with disease progression,\nas well as with disease response/resection53. The titer of anti-p53 antibodies have been shown\nto increase in a subset of patients with early-stage breast cancer prior to disease relapse54, and\ndisappearance of anti-p53 antibodies has been observed in 27 patients after resection of\ncolorectal cancer55. Tumor antigen/antibody immune complexes have been detected up to 19\nmonths prior to the development of clinical relapse and correlate with survival in patients with\nearly-stage melanoma56.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nMethods for assessing autoantibodies\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.3**r== 0.1**\nThe development process for autoantibodies as biomarkers is outlined in Figure 3. Historically,\nthis cycle begins with identifying antigens that are detected by the sera of cancer patients.\nAntigens must then be selected and tested by comparing sera from both patients and healthy\ndonors to determine if the antigens are “informative”, i.e., responses are limited to patients. As\npart of this process, the sensitivity and specificity of the antigens should be determined.\nPromising antigens are then tested on a set of samples separate from the training set to validate\ntheir usefulness. Ultimately prospective trials will establish their value as biomarkers. The\ncycle is completed by using the sera from patients not detected by the test to screen for new\nantigens. In some of the newer proteomic approaches, two of these steps may combine into a\nsingle step.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.4**\nIdentification of novel tumor antigens\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\nSEREX: A variety of techniques have been developed that use patient sera as probes against\ncandidate antigens derived from tumor cells and tumor cell lysates to find novel autoantigens.\nProminent among these, serologic expression cloning (SEREX) was developed ten years ago\n(Figure 4). This method uses patient sera to probe blotted phage expression libraries derived\nfrom tumor cells and has resulted in the identification of over 2000 autoantigens recognized\nby patient sera5, 57-79. Through a database established by the SEREX collaborative group,\nsequences from over 1390 genes have been deposited 80 and\nhttp://www2.licr.org/CancerImmunomeDB/. As SEREX relies on immunoblotting, these\nantigens are limited to linear epitopes and those gene products that can be expressed in bacteria.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.1**\nAs with the autoimmune targets, most of the antigens identified with SEREX are nuclear or\nintracellular antigens, many of which are upregulated, mutated, or specifically expressed in\ntumors. However, altered degradative pathways, such as sensitivity to granzyme B cleavage,\nhas been shown to have a marked impact on the immunogenicity of a hepatocellular tumor\nantigen6. In addition, frameshifts and alternative reading frames, can result in autoantibodies.\nUsing colon cancer sera, the antigen ADO34 was identified with a frameshift insertion81, and\nCDX2 had a frameshift mutation82. Antibodies to the OGFr protein can recognize an\nalternative-reading frame of the molecule79.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\nThe SEREX method identifies autoantigens whether or not they are informative. At present\nonly a handful of antigens have undergone comparison testing, in part because of the technical\nchallenges associated with historical assays for validating autoantigens.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.3**r== 0.1**\nPhage display: Combinatorial phage display is a new and attractive approach for the\nidentification of tumor antigens. By expressing antigens as fusions with phage proteins,\nantigens can be detected more rapidly and with less serum. Rather than immunoblotting as\nwith SEREX, phage display relies on successive rounds of immunoprecipitation of phage\nlibraries using patient serum. This powerful approach has been used in the field of\nrheumatology to identify autoantigens in multiple sclerosis, lupus, rheumatoid arthritis and\nothers83-85. For tumor antigens, one study showed that only 4 of 13 antigens identified with\nprostate cancer sera had already been identified by SEREX technologies, demonstrating that\nnovel tumor antigens may be identified with different screening approaches 86. Phage display\nhas been successfully used to identify tumor autoantigens in cancer86-89. However, antigens\nexpressed by phage display may not be in native conformation, do not have mammalian post-\ntranslational modifications, and each positive phage clone must be individually sequenced.\nYeast display systems are also being investigated90.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.3**r== 0.1**\nValidating autoantibodies—To be useful as biomarkers, the tumor antigens must\ndistinguish between individuals with and without cancer. Thus once a candidate autoantigen\nis identified, comparisons must be made regarding the responses of individuals from both\ngroups. The measurement of an autoantibody response is most often accomplished using the\nenzyme linked immunosorbent assay (ELISA), in which purified protein (the autoantigen) is\nimmobilized in the wells of a microtiter dish and exposed to serum samples (Fig. 4). After\nadequate washing, any bound antibody is revealed using standard anti-human antibody\nreagents linked to enzymatic markers. If the response is very strong, it may be useful to perform\nserial dilutions of the serum to determine the titer of the antibody. By comparing multiple\npatient and normal donor samples it is possible to determine the frequency of response. For\nexample, antibodies to dsDNA are present in only 40-60% of SLE patients, but antibodies to\nhistones are present in >95% of patients with drug-induced SLE. To compute the sensitivity\nand specificity of the test, a threshold value must be set, such that the test is considered positive\nif the antibody response achieves that value or above. Typically, this threshold is either 2 or 3\nstandard deviations above the average response in the normal population.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.3**r== 0.1**\nThe development of clinical ELISAs require the use of recombinant proteins or peptides, which\nmay be complicated by batch-to-batch variations, loss of conformational epitopes or lack of\nmammalian post-translational processing91, 92. Antigenic epitopes from proteins such as Ro,\nLa, SmB, SmD, Scl-70, can be linear epitopes of 10-22 amino acids conformational, or cryptic.\nFurthermore, epitopes may be naturally post-translationally modified by multiple means,\nincluding phosphorylation, glycosylation, acetylation, and methylation (SmD1, SmD3,\nfibrillarin, nucleolin)93, 94, all of which must be considered prior to the development of a\nclinical diagnostic assay. Of the many candidate antigens identified so far, only a handful have\ngone through the validation process to determine their sensitivity and specificity as cancer\npredictors, in part because of the challenges associated with setting up ELISAs.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nProteomic Approaches for the Identification of Tumor Antigens\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\nThe Power of Multiplexing—With all of their potential advantages, the Achilles heel of\nautoantibodies as biomarkers is their sensitivity, i.e., the fraction of true positives that have a\npositive test. In large part, this may reflect the nature of cancer. Unlike infections in which the\nvast majority of patients respond to the same immunodominant antigens, even cancers of the\nsame type represent a mix of different biological subtypes. Thus, patients are likely to mount\nimmune responses to different tumor antigens, and no single antigen is likely to detect all\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.1**l== 0.3**r== 0.1**\ncancers. Typically, only 15-20% of patients demonstrate a response to any given antigen.\nHowever, proteomics may hold the key to success because it provides the means to multiplex.\nBy linking the responses to several antigens together, the sensitivity and specificity of the test\nincreases considerably, presumably because the chance that a patient will respond to at least\none of the antigens is increased76, 95-102. Koziol et al demonstrated that a panel of just seven\ntumor-associated antigens (myc, cyclin B1, p62, IMP-1, Koc, p53, and survivin) could be used\nto segregate sera from patients with cancers from healthy donor sera95. In that study, sera from\nbreast, colorectal, gastric, hepatocellular, lung, and prostate cancers were distinguished with\nsensitivities from 0.77-0.92 and specificities from 0.85-0.91 using no more than 3 tumor\nantigens for any cancer cohort. Scanlan et. al. studied the serologic responses to 13 defined\ntumor antigens in sera from colon cancer patients. Of these, 46% of patients, but not healthy\ndonors, had antibodies to at least 1 of these antigens103.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.3**r== 0.1**\nThese results demonstrate the potential power of simultaneously analyzing multiple\nautoantigens. It provides a greater likelihood of detecting and diagnosing the appropriate cancer\npresumably because the sensitivity will improve (increased chance that the patient will have a\nresponse to at least one of the tested antigens) and the specificity improves (positive responses\non multiple antigens increases certainty). By coupling this with the appropriate statistical\nmodeling, it is likely that patterns or weighted schemes of antibody responses, rather than\nindividual responses, will have the greatest utility in clinical assays. The early experiments\nabove were performed by using recursive partitioning. Given the technical challenges inherent\nin that method for large numbers of antigens, newer technologies are needed for high\nthroughput analysis. To this end, recent development of several proteomic technologies have\nbeen adapted for tumor antigen identification and biomarker development. These include\nprobing fractionated tumor cell lysates, phage display, and protein microarrays (See Table 1).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nProbing fractionated tumor cell lysate blots with serum\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.3**r== 0.1**\nIn this approach, tumor cell lysates are fractionated to separate the various protein species and\nblotted onto a membrane or microarray and then probed with patient or control sera. Response\npatterns are then analyzed to differentiate between the two (Fig. 4). This has been used to\ndemonstrate antibodies in the sera of lung cancer patients96, 104. A similar approach was used\nto distinguish sera from prostate cancer patients from healthy donor sera with 98%\naccuracy105. The development of automated separation systems using liquid-based\nchromatography with subsequent microarray spotting of lysate fractions enhances the\nreproducibility and speed of immunogenic fraction identification. Advantages of this approach\ninclude the ability to query a large fraction of the tumor cell proteome and the preservation of\nthe post translational modifications of proteins. Moreover, when this is executed using a\nmicroarray format, only small amounts of serum are required. However, the reproducibility\ndemanded by a clinical assay requires an identified and validated antigen. Thus, as with other\nchromatographic separation systems, the identity of the proteins in the fraction must eventually\nbe determined, presumably using sensitive mass spectrometric analysis. As each lysate fraction\nmay contain many proteins at very different concentrations, of which the minor component\nmay be the immunogen, antigen identification can be difficult.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nProbing Candidate Antigen Arrays\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\nKnown or predicted tumor antigens may be directly spotted on microarrays and probed with\nhuman sera, with the advantage of reproducibility and more rapid screening of small amounts\nof sera (Figure 4). In this case, the identity of a single protein at each feature on the array is\nknown a priori. This approach has been successfully used to screen autoimmune patient\nsera100, 101. A powerful advantage of this approach is that it offers the ability to screen for\ninformative autoantigens by comparing responses of patients to controls. Because each feature\nof the array represents a single protein, whose identity is known, it is possible to calculate\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\nsensitivity and specificity values for the response to each candidate antigen during the screening\nphase. This allows the rapid determination of which antigens are informative for cancer\ndetection. Ultimately, focused arrays of tumor antigens previously identified by other means\nmay have the greatest utility for immunodiagnostics.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.3**r== 0.1**\nTheir theoretical advantages notwithstanding, protein microarrays have still not found\nwidespread use, in part because producing them is challenging. Historically it has required the\nhigh-throughput production and purification of protein, which then must be spotted on the\narrays. Once printed, concerns remain about the shelf life of proteins on the arrays. Recently,\nprogrammable protein microarrays that consist of anchored cDNA's and in situ transcription\nand translation of tagged proteins have been shown to result in highly reproducible protein\narrays without the requirement of heterologous protein expression and purification106. As the\nproteins are synthesized at the time of the assay, shelf life is not an issue and the proteins are\ntranslated with a mammalian reticulocyte lysate, which reproduces the folding and some of the\npost-translational processing of antigenic epitopes. Unlike tumor lysate fractionation, antigen\nidentification is automatic, although epitopes that depend on abnormal processing by tumor\ncells would not be identified, and transmembrane proteins have not yet been tested in this\nsystem. Like most antigen presentation methods (SEREX, ELISA, phage display, etc.), these\nprogrammable arrays require access to cDNAs to express the antigens. Fortunately, nearly all\nknown tumor antigen genes are available because they were identified with methods that\ninvolved cDNA identification. Moreover, libraries of cDNA clones representing most of the\nhuman proteome are increasingly available.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nProteomics and the Identification of T cell Antigens\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.1**\nIn contrast to the rapid identification of B cell antigens, the identification of T cell antigens\nremains much more difficult. Since T lymphocytes specifically recognize peptides derived\nfrom protein antigens, the isolation and confirmation of these antigens have traditionally relied\non laborious T cell isolation and cloning. Alternatively, direct isolation and sequencing of\nMHC-associated peptides from cells has been limited by the overall low concentration of\nspecific peptide bound to MHC molecules, and the highly polymorphic nature of the MHC\nmolecules themselves. These challenges are being overcome by advances in high-throughput\nMHC-peptide binding studies, the bioinformatics of epitope prediction, and in particular, mass\nspectrometry.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\nWith high-throughput peptide synthesis, direct MHC-peptide binding assays may now be\nperformed on peptides that span target antigens, or using combinatorial peptide libraries107.\nDetailed analysis of peptide binding to common HLA alleles has resulted in the development\nof bioinformatics tools for epitope prediction108, 109. Further limitations imposed by\nproteasomal cleavage patterns can enhance prediction of peptide epitopes. Multiple prediction\nmodels are available, primarily for MHC Class I-binding peptides, due to the more restrictive\nnature of the peptide binding groove. These systems include BIMAS110, SYFPEITHI111,\nMHCPEP112, and RANKPEP113, among others.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.3**r== 0.1**\nAdvances in mass spectrometry have also greatly aided antigen identification. Original studies\nof direct peptide sequencing from purified MHC molecules from cell lines114-116\ndemonstrated that the majority of peptides within the MHC grooves are derived from self-\nproteins, but required billions of cells for the source of antigen. More recent approaches to\nidentify tumor-specific T cell epitopes have used capillary-scale chromatography and tandem\nmass spectrometry (Figure 5) (nanoLC-MS/MS)117-120. To enhance sensitivity, nanospray\nquadrupole-TOF combined with Poisson algorithms121 to specifically evaluate the likelihood\nof predicted peptides in unfractionated mixtures has markedly increased the sensitivity of the\nassay to the range of 1 copy number of peptide/cell in a murine influenza viral model, but is\nlimited to predicted MHC-binding peptide epitopes.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.9**t== 0.1**l== 0.2**r== 0.8**\nSummary\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\nThe development of proteomic-based methods of cell lysate fractionation, phage display,\nprotein microarrays, bioinformatics, and mass spectrometry is resulting in the rapid\nidentification of both B and T cell tumor antigens. These have the potential for clinical\ndiagnosis, identifying targets for immunotherapies, monitoring disease response, and to\nunderstand the breadth, scope, and impact of the immune response to cancer. As each new\ntechnology leads to the discovery of novel antigenic targets, systematic approaches of target\nvalidation and assessment of clinical applications, especially in the area of biomarkers and\ndiagnostic testing, will need to be developed.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nAcknowledgements\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nWe would like to thank Drs. Glenn Dranoff, Ellis Reinherz, and Niroshan Ramachandran for critical review of this\nmanuscript.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.3**r== 0.1**\nSupport: This work has been supported by NCI K08 CA88444-03 (K.S.A.) and NCI P50 CA89393-05, NCI R33\nCA099191-02 and the Breast Cancer Research Foundation (J.L.)\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nReferences\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n1. Pavlenko M, Roos AK, Lundqvist A, Palmborg A, Miller AM, Ozenci V, Bergman B, Egevad L,\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\nHellstrom M, Kiessling R, Masucci G, Wersall P, Nilsson S, Pisa P. A phase I trial of DNA vaccination\nwith a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate\ncancer. Br J Cancer 2004;91:688–694. [PubMed: 15280930]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n2. Vonderheide RH, Hahn WC, Schultze JL, Nadler LM. The telomerase catalytic subunit is a widely\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\nexpressed tumor-associated antigen recognized by cytotoxic T lymphocytes. Immunity 1999;10:673–\n679. [PubMed: 10403642]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n3. Soussi T. p53 Antibodies in the sera of patients with various types of cancer: a review. Cancer Res\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.5**\n2000;60:1777–1788. [PubMed: 10766157]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n4. Schubert U, Anton LC, Gibbs J, Norbury CC, Yewdell JW, Bennink JR. Rapid degradation of a large\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.3**r== 0.2**\nfraction of newly synthesized proteins by proteasomes. Nature 2000;404:770–774. [PubMed:\n10783891]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n5. Chen YT, Scanlan MJ, Sahin U, Tureci O, Gure AO, Tsang S, Williamson B, Stockert E, Pfreundschuh\nM, Old LJ. A testicular antigen aberrantly expressed in human cancers detected by autologous antibody\nscreening. Proc Natl Acad Sci U S A 1997;94:1914–1918. [PubMed: 9050879]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\n6. Ulanet DB, Torbenson M, Dang CV, Casciola-Rosen L, Rosen A. Unique conformation of cancer\nautoantigen B23 in hepatoma: a mechanism for specificity in the autoimmune response. Proc Natl\nAcad Sci U S A 2003;100:12361–12366. [PubMed: 14519847]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n7. von Mensdorff-Pouilly S, Petrakou E, Kenemans P, van Uffelen K, Verstraeten AA, Snijdewint FG,\nvan Kamp GJ, Schol DJ, Reis CA, Price MR, Livingston PO, Hilgers J. Reactivity of natural and\ninduced human antibodies to MUC1 mucin with MUC1 peptides and n-acetylgalactosamine (GalNAc)\npeptides. Int J Cancer 2000;86:702–712. [PubMed: 10797294]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n8. Routsias JG, Tzioufas AG, Moutsopoulos HM. The clinical value of intracellular autoantigens B-cell\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\nepitopes in systemic rheumatic diseases. Clin Chim Acta 2004;340:1–25. [PubMed: 14734193]\n9. Lernmark A. Autoimmune diseases: are markers ready for prediction? J Clin Invest 2001;108:1091–\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n1096. [PubMed: 11602614]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.2**\n10. Plotz PH. The autoantibody repertoire: searching for order. Nat Rev Immunol 2003;3:73–78.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n[PubMed: 12511877]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n11. Brendel V, Dohlman J, Blaisdell BE, Karlin S. Very long charge runs in systemic lupus erythematosus-\nassociated autoantigens. Proc Natl Acad Sci U S A 1991;88:1536–1540. [PubMed: 1996354]\n12. Utz PJ, Hottelet M, Le TM, Kim SJ, Geiger ME, van Venrooij WJ, Anderson P. The 72-kDa\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\ncomponent of signal recognition particle is cleaved during apoptosis. J Biol Chem 1998;273:35362–\n35370. [PubMed: 9857079]\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.3**r== 0.1**\n13. Casciola-Rosen L, Andrade F, Ulanet D, Wong WB, Rosen A. Cleavage by granzyme B is strongly\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.3**r== 0.2**\npredictive of autoantigen status: implications for initiation of autoimmunity. J Exp Med\n1999;190:815–826. [PubMed: 10499920]\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\n14. Raben N, Nichols R, Dohlman J, McPhie P, Sridhar V, Hyde C, Leff R, Plotz P. A motif in human\nhistidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-\ncoil that is essential for enzymatic activity and contains the major autoantigenic epitope. J Biol Chem\n1994;269:24277–24283. [PubMed: 7523371]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\n15. DeLeo AB, Jay G, Appella E, Dubois GC, Law LW, Old LJ. Detection of a transformation-related\nantigen in chemically induced sarcomas and other transformed cells of the mouse. Proc Natl Acad\nSci U S A 1979;76:2420–2424. [PubMed: 221923]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n16. Casey G, Lopez ME, Ramos JC, Plummer SJ, Arboleda MJ, Shaughnessy M, Karlan B, Slamon DJ.\nDNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to\ndetect all known p53 alterations in human malignancies. Oncogene 1996;13:1971–1981. [PubMed:\n8934544]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.3**r== 0.1**\n17. Winter SF, Minna JD, Johnson BE, Takahashi T, Gazdar AF, Carbone DP. Development of antibodies\nagainst p53 in lung cancer patients appears to be dependent on the type of p53 mutation. Cancer Res\n1992;52:4168–4174. [PubMed: 1322237]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n18. Dowell SP, Wilson PO, Derias NW, Lane DP, Hall PA. Clinical utility of the immunocytochemical\ndetection of p53 protein in cytological specimens. Cancer Res 1994;54:2914–2918. [PubMed:\n8187077]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n19. Vennegoor CJ, Nijman HW, Drijfhout JW, Vernie L, Verstraeten RA, von Mensdorff-Pouilly S,\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\nHilgers J, Verheijen RH, Kast WM, Melief CJ, Kenemans P. Autoantibodies to p53 in ovarian cancer\npatients and healthy women: a comparison between whole p53 protein and 18-mer peptides for\nscreening purposes. Cancer Lett 1997;116:93–101. [PubMed: 9177463]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n20. Angelopoulou K, Yu H, Bharaj B, Giai M, Diamandis EP. p53 gene mutation, tumor p53 protein\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\noverexpression, and serum p53 autoantibody generation in patients with breast cancer. Clin Biochem\n2000;33:53–62. [PubMed: 10693987]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n21. Angelopoulou K, Stratis M, Diamandis EP. Humoral immune response against p53 protein in patients\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.3**\nwith colorectal carcinoma. Int J Cancer 1997;70:46–51. [PubMed: 8985089]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.3**r== 0.1**\n22. Rainov NG, Dobberstein KU, Fittkau M, Bahn H, Holzhausen HJ, Gantchev L, Burkert W. Absence\nof p53 autoantibodies in sera from glioma patients. Clin Cancer Res 1995;1:775–781. [PubMed:\n9816045]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n23. Weller M, Bornemann A, Stander M, Schabet M, Dichgans J, Meyermann R. Humoral immune\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\nresponse to p53 in malignant glioma. J Neurol 1998;245:169–172. [PubMed: 9553848]\n24. Zorn E, Miklos DB, Floyd BH, Mattes-Ritz A, Guo L, Soiffer RJ, Antin JH, Ritz J. Minor\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nHistocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem\nCell Transplantation. J Exp Med 2004;199:1133–1142. [PubMed: 15096539]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\n25. Schmollinger JC, Vonderheide RH, Hoar KM, Maecker B, Schultze JL, Hodi FS, Soiffer RJ, Jung\nK, Kuroda MJ, Letvin NL, Greenfield EA, Mihm M, Kutok JL, Dranoff G. Melanoma inhibitor of\napoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction. Proc Natl Acad Sci\nU S A 2003;100:3398–3403. [PubMed: 12626761]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n26. Molldrem JJ, Lee PP, Wang C, Felio K, Kantarjian HM, Champlin RE, Davis MM. Evidence that\nspecific T lymphocytes may participate in the elimination of chronic myelogenous leukemia. Nat\nMed 2000;6:1018–1023. [PubMed: 10973322]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.3**r== 0.1**\n27. Hodi FS, Schmollinger JC, Soiffer RJ, Salgia R, Lynch T, Ritz J, Alyea EP, Yang J, Neuberg D,\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\nMihm M, Dranoff G. ATP6S1 elicits potent humoral responses associated with immune-mediated\ntumor destruction. Proc Natl Acad Sci U S A 2002;99:6919–6924. [PubMed: 11983866]\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\n28. Miklos DB, Kim HT, Zorn E, Hochberg EP, Guo L, Mattes-Ritz A, Viatte S, Soiffer RJ, Antin JH,\nRitz J. Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem\ncell transplantation and in healthy female donors. Blood 2004;103:353–359. [PubMed: 14512314]\n29. Bellucci R, Wu CJ, Chiaretti S, Weller E, Davies FE, Alyea EP, Dranoff G, Anderson KC, Munshi\nNC, Ritz J. Complete response to donor lymphocyte infusion in multiple myeloma is associated with\nantibody responses to highly expressed antigens. Blood 2004;103:656–663. [PubMed: 14563636]\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\n30. Yang XF, Wu CJ, Chen L, Alyea EP, Canning C, Kantoff P, Soiffer RJ, Dranoff G, Ritz J. CML28\nis a broadly immunogenic antigen, which is overexpressed in tumor cells. Cancer Res 2002;62:5517–\n5522. [PubMed: 12359762]\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\n31. Dranoff G, Jaffee E, Lazenby A, Golumbek P, Levitsky H, Brose K, Jackson V, Hamada H, Pardoll\nD, Mulligan RC. Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-\nmacrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor\nimmunity. Proc Natl Acad Sci U S A 1993;90:3539–3543. [PubMed: 8097319]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\n32. Huang AY, Golumbek P, Ahmadzadeh M, Jaffee E, Pardoll D, Levitsky H. Role of bone marrow-\nderived cells in presenting MHC class I-restricted tumor antigens. Science 1994;264:961–965.\n[PubMed: 7513904]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n33. Hung K, Hayashi R, Lafond-Walker A, Lowenstein C, Pardoll D, Levitsky H. The central role of\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nCD4(+) T cells in the antitumor immune response. J Exp Med 1998;188:2357–2368. [PubMed:\n9858522]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.3**r== 0.1**\n34. Gillessen S, Naumov YN, Nieuwenhuis EE, Exley MA, Lee FS, Mach N, Luster AD, Blumberg RS,\nTaniguchi M, Balk SP, Strominger JL, Dranoff G, Wilson SB. CD1d-restricted T cells regulate\ndendritic cell function and antitumor immunity in a granulocyte-macrophage colony-stimulating\nfactor-dependent fashion. Proc Natl Acad Sci U S A 2003;100:8874–8879. [PubMed: 12847287]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n35. Heslop HE, Stevenson FK, Molldrem JJ. Immunotherapy of hematologic malignancy. Hematology\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\n(Am Soc Hematol Educ Program) 2003:331–349. [PubMed: 14633789]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\n36. Davis ID, Chen W, Jackson H, Parente P, Shackleton M, Hopkins W, Chen Q, Dimopoulos N, Luke\nT, Murphy R, Scott AM, Maraskovsky E, McArthur G, MacGregor D, Sturrock S, Tai TY, Green S,\nCuthbertson A, Maher D, Miloradovic L, Mitchell SV, Ritter G, Jungbluth AA, Chen YT, Gnjatic\nS, Hoffman EW, Old LJ, Cebon JS. Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant\ninduces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans. Proc Natl\nAcad Sci U S A 2004;101:10697–10702. [PubMed: 15252201]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n37. DiFronzo LA, Gupta RK, Essner R, Foshag LJ, O'Day SJ, Wanek LA, Stern SL, Morton DL. Enhanced\nhumoral immune response correlates with improved disease-free and overall survival in American\nJoint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine. J Clin\nOncol 2002;20:3242–3248. [PubMed: 12149297]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.3**r== 0.1**\n38. Ehlken H, Schadendorf D, Eichmuller S. Humoral immune response against melanoma antigens\ninduced by vaccination with cytokine gene-modified autologous tumor cells. Int J Cancer\n2004;108:307–313. [PubMed: 14639620]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\n39. Small, EJ.; S, PF.; Higano, C.; Neumanaitis, J.; Valone, F.; Herschberg, RM. Immunotherapy\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n(APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3\nrandomized placebo-controlled trial; 2005 ASCO Prostate Cancer Symposium; 2005.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n40. Cancer Facts and Figures. 2004.\n41. Srinivas PR, Kramer BS, Srivastava S. Trends in biomarker research for cancer detection. Lancet\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.3**r== 0.4**\nOncol 2001;2:698–704. [PubMed: 11902541]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n42. Gold P, Freedman SO. Specific carcinoembryonic antigens of the human digestive system. J Exp\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.4**\nMed 1965;122:467–481. [PubMed: 4953873]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n43. Wang MC, Valenzuela LA, Murphy GP, Chu TM. Purification of a human prostate specific antigen.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.4**\nInvest Urol 1979;17:159–163. [PubMed: 89106]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n44. Niloff JM, Knapp RC, Schaetzl E, Reynolds C, Bast RC Jr. CA125 antigen levels in obstetric and\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\ngynecologic patients. Obstet Gynecol 1984;64:703–707. [PubMed: 6208522]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.3**r== 0.1**\n45. Petricoin E, Wulfkuhle J, Espina V, Liotta LA. Clinical proteomics: revolutionizing disease detection\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.2**\nand patient tailoring therapy. J Proteome Res 2004;3:209–217. [PubMed: 15113096]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n46. Trivers GE, Cawley HL, DeBenedetti VM, Hollstein M, Marion MJ, Bennett WP, Hoover ML, Prives\nCC, Tamburro CC, Harris CC. Anti-p53 antibodies in sera of workers occupationally exposed to\nvinyl chloride. J Natl Cancer Inst 1995;87:1400–1407. [PubMed: 7658501]\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\n47. Trivers GE, De Benedetti VM, Cawley HL, Caron G, Harrington AM, Bennett WP, Jett JR, Colby\nTV, Tazelaar H, Pairolero P, Miller RD, Harris CC. Anti-p53 antibodies in sera from patients with\nchronic obstructive pulmonary disease can predate a diagnosis of cancer. Clin Cancer Res\n1996;2:1767–1775. [PubMed: 9816128]\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.3**r== 0.2**\n48. Suzuki H, Graziano DF, McKolanis J, Finn OJ. T cell-dependent antibody responses against\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\naberrantly expressed cyclin B1 protein in patients with cancer and premalignant disease. Clin Cancer\nRes 2005;11:1521–1526. [PubMed: 15746055]\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\n49. Rodolfo M, Luksch R, Stockert E, Chen YT, Collini P, Ranzani T, Lombardo C, Dalerba P, Rivoltini\nL, Arienti F, Fossati-Bellani F, Old LJ, Parmiani G, Castelli C. Antigen-specific immunity in\nneuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen. Cancer Res\n2003;63:6948–6955. [PubMed: 14583496]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\n50. Litvak DA, Gupta RK, Yee R, Wanek LA, Ye W, Morton DL. Endogenous immune response to early-\nand intermediate-stage melanoma is correlated with outcomes and is independent of locoregional\nrelapse and standard prognostic factors. J Am Coll Surg 2004;198:27–35. [PubMed: 14698308]\n51. Disis ML, Pupa SM, Gralow JR, Dittadi R, Menard S, Cheever MA. High-titer HER-2/neu protein-\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.2**\nspecific antibody can be detected in patients with early-stage breast cancer. J Clin Oncol\n1997;15:3363–3367. [PubMed: 9363867]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n52. Disis ML, Knutson KL, Schiffman K, Rinn K, McNeel DG. Pre-existent immunity to the HER-2/neu\noncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer. Breast\nCancer Res Treat 2000;62:245–252. [PubMed: 11072789]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n53. Jager E, Stockert E, Zidianakis Z, Chen YT, Karbach J, Jager D, Arand M, Ritter G, Old LJ, Knuth\nA. Humoral immune responses of cancer patients against “Cancer-Testis” antigen NY-ESO-1:\ncorrelation with clinical events. Int J Cancer 1999;84:506–510. [PubMed: 10502728]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\n54. Regele S, Vogl FD, Kohler T, Kreienberg R, Runnebaum IB. p53 autoantibodies can be indicative\nof the development of breast cancer relapse. Anticancer Res 2003;23:761–764. [PubMed: 12680180]\n55. Takeda A, Shimada H, Nakajima K, Imaseki H, Suzuki T, Asano T, Ochiai T, Isono K. Monitoring\nof p53 autoantibodies after resection of colorectal cancer: relationship to operative curability. Eur J\nSurg 2001;167:50–53. [PubMed: 11213822]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n56. Kelley MC, Gupta RK, Hsueh EC, Yee R, Stern S, Morton DL. Tumor-associated antigen TA90\nimmune complex assay predicts recurrence and survival after surgical treatment of stage I-III\nmelanoma. J Clin Oncol 2001;19:1176–1182. [PubMed: 11181684]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n57. Tajima K, Obata Y, Tamaki H, Yoshida M, Chen YT, Scanlan MJ, Old LJ, Kuwano H, Takahashi T,\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\nMitsudomi T. Expression of cancer/testis (CT) antigens in lung cancer. Lung Cancer 2003;42:23–\n33. [PubMed: 14512184]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.3**r== 0.1**\n58. Lee SY, Obata Y, Yoshida M, Stockert E, Williamson B, Jungbluth AA, Chen YT, Old LJ, Scanlan\nMJ. Immunomic analysis of human sarcoma. Proc Natl Acad Sci U S A 2003;100:2651–2656.\n[PubMed: 12601173]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n59. Ayyoub M, Hesdorffer CS, Montes M, Merlo A, Speiser D, Rimoldi D, Cerottini JC, Ritter G, Scanlan\nM, Old LJ, Valmori D. An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in\nassociation with HLA-DR. J Clin Invest 2004;113:1225–1233. [PubMed: 15085202]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\n60. Sugita Y, Wada H, Fujita S, Nakata T, Sato S, Noguchi Y, Jungbluth AA, Yamaguchi M, Chen YT,\nStockert E, Gnjatic S, Williamson B, Scanlan MJ, Ono T, Sakita I, Yasui M, Miyoshi Y, Tamaki Y,\nMatsuura N, Noguchi S, Old LJ, Nakayama E, Monden M. NY-ESO-1 expression and\nimmunogenicity in malignant and benign breast tumors. Cancer Res 2004;64:2199–2204. [PubMed:\n15026363]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n61. Odunsi K, Jungbluth AA, Stockert E, Qian F, Gnjatic S, Tammela J, Intengan M, Beck A, Keitz B,\nSantiago D, Williamson B, Scanlan MJ, Ritter G, Chen YT, Driscoll D, Sood A, Lele S, Old LJ. NY-\nESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial\novarian cancer. Cancer Res 2003;63:6076–6083. [PubMed: 14522938]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n62. Scanlan MJ, Gout I, Gordon CM, Williamson B, Stockert E, Gure AO, Jager D, Chen YT, Mackay\nA, O'Hare MJ, Old LJ. Humoral immunity to human breast cancer: antigen definition and quantitative\nanalysis of mRNA expression. Cancer Immun 2001;1:4. [PubMed: 12747765]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n63. Jager D, Unkelbach M, Frei C, Bert F, Scanlan MJ, Jager E, Old LJ, Chen YT, Knuth A. Identification\nof tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by\nserological screening of a testicular library with breast cancer serum. Cancer Immun 2002;2:5.\n[PubMed: 12747750]\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.3**r== 0.1**\n64. Koroleva EP, Lagarkova MA, Mesheryakov AA, Scanlan MJ, Old LJ, Nedospasov SA, Kuprash DV.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.3**r== 0.2**\nSerological identification of antigens associated with renal cell carcinoma. Russ J Immunol\n2002;7:229–238. [PubMed: 12674932]\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\n65. Scanlan MJ, Gure AO, Jungbluth AA, Old LJ, Chen YT. Cancer/testis antigens: an expanding family\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.2**\nof targets for cancer immunotherapy. Immunol Rev 2002;188:22–32. [PubMed: 12445278]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\n66. Forti S, Scanlan MJ, Invernizzi A, Castiglioni F, Pupa S, Agresti R, Fontanelli R, Morelli D, Old LJ,\nPupa SM, Menard S. Identification of breast cancer-restricted antigens by antibody screening of\nSKBR3 cDNA library using a preselected patient's serum. Breast Cancer Res Treat 2002;73:245–\n256. [PubMed: 12160330]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n67. Scanlan MJ, Gordon CM, Williamson B, Lee SY, Chen YT, Stockert E, Jungbluth A, Ritter G, Jager\nD, Jager E, Knuth A, Old LJ. Identification of cancer/testis genes by database mining and mRNA\nexpression analysis. Int J Cancer 2002;98:485–492. [PubMed: 11920606]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.2**\n68. Jager D, Stockert E, Gure AO, Scanlan MJ, Karbach J, Jager E, Knuth A, Old LJ, Chen YT.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nIdentification of a tissue-specific putative transcription factor in breast tissue by serological screening\nof a breast cancer library. Cancer Res 2001;61:2055–2061. [PubMed: 11280766]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.3**r== 0.1**\n69. Gure AO, Stockert E, Scanlan MJ, Keresztes RS, Jager D, Altorki NK, Old LJ, Chen YT. Serological\nidentification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung\ncancer. Proc Natl Acad Sci U S A 2000;97:4198–4203. [PubMed: 10760287]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n70. Scanlan MJ, Altorki NK, Gure AO, Williamson B, Jungbluth A, Chen YT, Old LJ. Expression of\ncancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a\nnew CT gene, CT9. Cancer Lett 2000;150:155–164. [PubMed: 10704737]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\n71. Gure AO, Stockert E, Arden KC, Boyer AD, Viars CS, Scanlan MJ, Old LJ, Chen YT. CT10: a new\ncancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-\ndifference analysis. Int J Cancer 2000;85:726–732. [PubMed: 10699956]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n72. Jager D, Stockert E, Scanlan MJ, Gure AO, Jager E, Knuth A, Old LJ, Chen YT. Cancer-testis antigens\nand ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-\nspecific ING1 transcripts and a homologue gene. Cancer Res 1999;59:6197–6204. [PubMed:\n10626813]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.3**r== 0.1**\n73. Scanlan MJ, Gordan JD, Williamson B, Stockert E, Bander NH, Jongeneel V, Gure AO, Jager D,\nJager E, Knuth A, Chen YT, Old LJ. Antigens recognized by autologous antibody in patients with\nrenal-cell carcinoma. Int J Cancer 1999;83:456–464. [PubMed: 10508479]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n74. Scanlan MJ, Williamson B, Jungbluth A, Stockert E, Arden KC, Viars CS, Gure AO, Gordan JD,\nChen YT, Old LJ. Isoforms of the human PDZ-73 protein exhibit differential tissue expression.\nBiochim Biophys Acta 1999;1445:39–52. [PubMed: 10209257]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\n75. Scanlan MJ, Chen YT, Williamson B, Gure AO, Stockert E, Gordan JD, Tureci O, Sahin U,\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nPfreundschuh M, Old LJ. Characterization of human colon cancer antigens recognized by autologous\nantibodies. Int J Cancer 1998;76:652–658. [PubMed: 9610721]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\n76. Stockert E, Jager E, Chen YT, Scanlan MJ, Gout I, Karbach J, Arand M, Knuth A, Old LJ. A survey\nof the humoral immune response of cancer patients to a panel of human tumor antigens. J Exp Med\n1998;187:1349–1354. [PubMed: 9547346]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.2**\n77. Gure AO, Altorki NK, Stockert E, Scanlan MJ, Old LJ, Chen YT. Human lung cancer antigens\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\nrecognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor\ngene locus on chromosome 3p21.3. Cancer Res 1998;58:1034–1041. [PubMed: 9500467]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.3**r== 0.1**\n78. Gure AO, Tureci O, Sahin U, Tsang S, Scanlan MJ, Jager E, Knuth A, Pfreundschuh M, Old LJ, Chen\nYT. SSX: a multigene family with several members transcribed in normal testis and human cancer.\nInt J Cancer 1997;72:965–971. [PubMed: 9378559]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n79. Mollick JA, Hodi FS, Soiffer RJ, Nadler LM, Dranoff G. MUC1-like tandem repeat proteins are\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.2**\nbroadly immunogenic in cancer patients. Cancer Immun 2003;3:3. [PubMed: 12747745]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n80. Chen, YT.; S, M.; Obata, Y.; Old, LJ. Identification of human tumor antigens by serological expression\ncloning. In: SA, R., editor. Principles and practice of the biologic therapy of cancer. 3rd. Lippincott\nWilliams & Wilkins; Philadelphia: 2000. p. 557-570.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\n81. Line A, Slucka Z, Stengrevics A, Silina K, Li G, Rees RC. Characterisation of tumour-associated\nantigens in colon cancer. Cancer Immunol Immunother 2002;51:574–582. [PubMed: 12384809]\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.3**r== 0.2**\n82. Ishikawa T, Fujita T, Suzuki Y, Okabe S, Yuasa Y, Iwai T, Kawakami Y. Tumor-specific\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.3**r== 0.2**\nimmunological recognition of frameshift-mutated peptides in colon cancer with microsatellite\ninstability. Cancer Res 2003;63:5564–5572. [PubMed: 14500396]\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\n83. Kemp EH, Herd LM, Waterman EA, Wilson AG, Weetman AP, Watson PP. Immunoscreening of\nphage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease.\nBiochem Biophys Res Commun 2002;298:169–177. [PubMed: 12379236]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\n84. Jensen LB, Riise E, Nielsen LK, Dziegiel M, Fugger L, Engberg J. Efficient purification of unique\nantibodies using peptide affinity-matrix columns. J Immunol Methods 2004;284:45–54. [PubMed:\n14736416]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n85. Sioud M, Hansen M, Dybwad A. Profiling the immune responses in patient sera with peptide and\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.2**\ncDNA display libraries. Int J Mol Med 2000;6:123–128. [PubMed: 10891554]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n86. Fossa A, Alsoe L, Crameri R, Funderud S, Gaudernack G, Smeland EB. Serological cloning of cancer/\ntestis antigens expressed in prostate cancer using cDNA phage surface display. Cancer Immunol\nImmunother 2004;53:431–438. [PubMed: 14747957]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.2**\n87. Sioud M, Hansen MH. Profiling the immune response in patients with breast cancer by phage-\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.3**r== 0.2**\ndisplayed cDNA libraries. Eur J Immunol 2001;31:716–725. [PubMed: 11241275]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n88. Minenkova O, Pucci A, Pavoni E, De Tomassi A, Fortugno P, Gargano N, Cianfriglia M, Barca S,\nDe Placido S, Martignetti A, Felici F, Cortese R, Monaci P. Identification of tumor-associated\nantigens by screening phage-displayed human cDNA libraries with sera from tumor patients. Int J\nCancer 2003;106:534–544. [PubMed: 12845649]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n89. Fernandez-Madrid F, Tang N, Alansari H, Granda JL, Tait L, Amirikia KC, Moroianu M, Wang X,\nKarvonen RL. Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast\nCancer. Cancer Res 2004;64:5089–5096. [PubMed: 15289310]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n90. Mischo A, Wadle A, Watzig K, Jager D, Stockert E, Santiago D, Ritter G, Regitz E, Jager E, Knuth\nA, Old L, Pfreundschuh M, Renner C. Recombinant antigen expression on yeast surface (RAYS) for\nthe detection of serological immune responses in cancer patients. Cancer Immun 2003;3:5. [PubMed:\n12828452]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n91. Yan SC, Grinnell BW, Wold F. Post-translational modifications of proteins: some problems left to\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.3**\nsolve. Trends Biochem Sci 1989;14:264–268. [PubMed: 2672445]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n92. St Clair EW, Kenan D, Burch JA Jr, Keene JD, Pisetsky DS. The fine specificity of anti-La antibodies\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.3**r== 0.2**\ninduced in mice by immunization with recombinant human La autoantigen. J Immunol\n1990;144:3868–3876. [PubMed: 1692063]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\n93. Doyle HA, Mamula MJ. Post-translational protein modifications in antigen recognition and\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.3**\nautoimmunity. Trends Immunol 2001;22:443–449. [PubMed: 11473834]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\n94. Doyle HA, Mamula MJ. Posttranslational protein modifications: new flavors in the menu of\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.3**\nautoantigens. Curr Opin Rheumatol 2002;14:244–249. [PubMed: 11981321]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n95. Koziol JA, Zhang JY, Casiano CA, Peng XX, Shi FD, Feng AC, Chan EK, Tan EM. Recursive\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\npartitioning as an approach to selection of immune markers for tumor diagnosis. Clin Cancer Res\n2003;9:5120–5126. [PubMed: 14613989]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n96. Zhang JY, Casiano CA, Peng XX, Koziol JA, Chan EK, Tan EM. Enhancement of antibody detection\nin cancer using panel of recombinant tumor-associated antigens. Cancer Epidemiol Biomarkers Prev\n2003;12:136–143. [PubMed: 12582023]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n97. Joos TO, Stoll D, Templin MF. Miniaturised multiplexed immunoassays. Curr Opin Chem Biol\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.5**\n2002;6:76–80. [PubMed: 11827827]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.3**r== 0.1**\n98. Stoll D, Templin MF, Schrenk M, Traub PC, Vohringer CF, Joos TO. Protein microarray technology.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.4**\nFront Biosci 2002;7:c13–32. [PubMed: 11779717]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n99. Templin MF, Stoll D, Schwenk JM, Potz O, Kramer S, Joos TO. Protein microarrays: promising tools\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nfor proteomic research. Proteomics 2003;3:2155–2166. [PubMed: 14595815]\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\n100. Robinson WH, DiGennaro C, Hueber W, Haab BB, Kamachi M, Dean EJ, Fournel S, Fong D,\nGenovese MC, de Vegvar HE, Skriner K, Hirschberg DL, Morris RI, Muller S, Pruijn GJ, van\nVenrooij WJ, Smolen JS, Brown PO, Steinman L, Utz PJ. Autoantigen microarrays for multiplex\ncharacterization of autoantibody responses. Nat Med 2002;8:295–301. [PubMed: 11875502]\n101. Robinson WH, Steinman L, Utz PJ. Protein arrays for autoantibody profiling and fine-specificity\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nmapping. Proteomics 2003;3:2077–2084. [PubMed: 14595805]\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.3**r== 0.2**\n102. Shin BK, Wang H, Hanash S. Proteomics approaches to uncover the repertoire of circulating\nbiomarkers for breast cancer. J Mammary Gland Biol Neoplasia 2002;7:407–413. [PubMed:\n12882525]\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\n103. Scanlan MJ, Welt S, Gordon CM, Chen YT, Gure AO, Stockert E, Jungbluth AA, Ritter G, Jager\nD, Jager E, Knuth A, Old LJ. Cancer-related serological recognition of human colon cancer:\nidentification of potential diagnostic and immunotherapeutic targets. Cancer Res 2002;62:4041–\n4047. [PubMed: 12124339]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\n104. Qiu J, Madoz-Gurpide J, Misek DE, Kuick R, Brenner DE, Michailidis G, Haab BB, Omenn GS,\nHanash S. Development of natural protein microarrays for diagnosing cancer based on an antibody\nresponse to tumor antigens. J Proteome Res 2004;3:261–267. [PubMed: 15113102]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n105. Bouwman K, Qiu J, Zhou H, Schotanus M, Mangold LA, Vogt R, Erlandson E, Trenkle J, Partin\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\nAW, Misek D, Omenn GS, Haab BB, Hanash S. Microarrays of tumor cell derived proteins uncover\na distinct pattern of prostate cancer serum immunoreactivity. Proteomics 2003;3:2200–2207.\n[PubMed: 14595819]\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.1**\n106. Ramachandran N, Hainsworth E, Bhullar B, Eisenstein S, Rosen B, Lau AY, Walter JC, LaBaer J.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.2**\nSelf-assembling protein microarrays. Science 2004;305:86–90. [PubMed: 15232106]\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\n107. Lawendowski CA, Giurleo GM, Huang YY, Franklin GJ, Kaplan JM, Roberts BL, Nicolette CA.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.1**\nSolid-phase epitope recovery: a high throughput method for antigen identification and epitope\noptimization. J Immunol 2002;169:2414–2421. [PubMed: 12193709]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\n108. Rammensee HG, Weinschenk T, Gouttefangeas C, Stevanovic S. Towards patient-specific tumor\nantigen selection for vaccination. Immunol Rev 2002;188:164–176. [PubMed: 12445290]\n109. Sette A, Keogh E, Ishioka G, Sidney J, Tangri S, Livingston B, McKinney D, Newman M, Chesnut\nR, Fikes J. Epitope identification and vaccine design for cancer immunotherapy. Curr Opin Investig\nDrugs 2002;3:132–139.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n110. Parker KC, Bednarek MA, Coligan JE. Scheme for ranking potential HLA-A2 binding peptides\nbased on independent binding of individual peptide side-chains. J Immunol 1994;152:163–175.\n[PubMed: 8254189]\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.1**\n111. Rammensee H, Bachmann J, Emmerich NP, Bachor OA, Stevanovic S. SYFPEITHI: database for\nMHC ligands and peptide motifs. Immunogenetics 1999;50:213–219. [PubMed: 10602881]\n112. Brusic V, Rudy G, Harrison LC. MHCPEP, a database of MHC-binding peptides: update 1997.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.3**r== 0.4**\nNucleic Acids Res 1998;26:368–371. [PubMed: 9399876]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n113. Reche PA, Glutting JP, Zhang H, Reinherz EL. Enhancement to the RANKPEP resource for the\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nprediction of peptide binding to MHC molecules using profiles. Immunogenetics 2004;56:405–419.\n[PubMed: 15349703]\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\n114. Hunt DF, Henderson RA, Shabanowitz J, Sakaguchi K, Michel H, Sevilir N, Cox AL, Appella E,\nEngelhard VH. Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass\nspectrometry. Science 1992;255:1261–1263. [PubMed: 1546328]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.3**r== 0.1**\n115. Wei ML, Cresswell P. HLA-A2 molecules in an antigen-processing mutant cell contain signal\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\nsequence-derived peptides. Nature 1992;356:443–446. [PubMed: 1557127]\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n116. Rotzschke O, Falk K, Wallny HJ, Faath S, Rammensee HG. Characterization of naturally occurring\nminor histocompatibility peptides including H-4 and H-Y. Science 1990;249:283–287. [PubMed:\n1695760]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.3**r== 0.1**\n117. Kao H, Marto JA, Hoffmann TK, Shabanowitz J, Finkelstein SD, Whiteside TL, Hunt DF, Finn OJ.\nIdentification of cyclin B1 as a shared human epithelial tumor-associated antigen recognized by T\ncells. J Exp Med 2001;194:1313–1323. [PubMed: 11696596]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n118. Hogan KT, Coppola MA, Gatlin CL, Thompson LW, Shabanowitz J, Hunt DF, Engelhard VH,\nSlingluff CL Jr, Ross MM. Identification of a shared epitope recognized by melanoma-specific,\nHLA-A3-restricted cytotoxic T lymphocytes. Immunol Lett 2003;90:131–135. [PubMed:\n14687714]\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.2**\n119. Lemmel C, Stevanovic S. The use of HPLC-MS in T-cell epitope identification. Methods\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.3**r== 0.5**\n2003;29:248–259. [PubMed: 12725790]\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.3**r== 0.1**\n120. Weinschenk T, Gouttefangeas C, Schirle M, Obermayr F, Walter S, Schoor O, Kurek R, Loeser W,\nBichler KH, Wernet D, Stevanovic S, Rammensee HG. Integrated functional genomics approach\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.3**r== 0.1**\nfor the design of patient-individual antitumor vaccines. Cancer Res 2002;62:5818–5827. [PubMed:\n12384544]\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.3**r== 0.1**\n121. Zhong W, Reche PA, Lai CC, Reinhold B, Reinherz EL. Genome-wide characterization of a viral\ncytotoxic T lymphocyte epitope repertoire. J Biol Chem 2003;278:45135–45144. [PubMed:\n12960169]\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.3**r== 0.1**\nFigure 1. The application of cancer proteomics to tumor immunology\nThe immune response to cancer, whether endogenous or driven by immunotherapy, involves\na complex array of interactions between the tumor, host microenvironment, lymphocytes,\nantigen presenting cells, antibodies, cytokines and chemokines. Major arms of the immune\nsystem are shown in blue, and selected techniques for evaluation are shown in green. The\nidentification of target antigens of B- and T-lymphocyte recognition may be identified on a\nproteome-wide basis, through cell fractionation, protein microarrays, nanospray mass\nspectrometry, and protein expression profiling of tumors in vivo. Cellular assays may use\nprotein microarrays, microfluidics, and nanotechnology to identify target antigens as well as\nhost responses. Microenvironmental interactions can be analyzed, as well as signal transduction\npathways and costimulatory/regulatory molecules. This review will focus on B-cell and T-cell\ntarget antigen identification.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.1**\nFigure 2. Classical model of antigen presentation\nIntracellular antigens (i.e., viral) are cleaved into peptides by the proteasome and bound to\nMHC Class I molecules at the surface of cells for presentation to CD8+ T lymphocytes.\nExogenous antigens are endocytosed by antigen presenting cells (APC), which is markedly\nenhanced by binding of serum antibodies both to antigen and to the Fcγ receptor on the APC,\nwith subsequent degradation and binding of peptides to MHC Class II molecules for stimulation\nof CD4+ T lymphocytes. Because of this coordinated immune response, tumor antigens that\ncause antibody responses in patient sera may contain immunogenic T cell peptide epitopes as\nwell (modified from D. Miklos).\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.3**r== 0.1**\nFigure 3. The development of autoantibody biomarkers in cancer\nThe identification of target antigens using antibodies derived from patient sera is the first step\nin identification of potential biomarkers for cancer diagnosis or immune monitoring. In the\nfield of autoimmunity, detailed evaluation of the autoantibody quantity (titration) and\nspecificity has required development of a confirmatory ELISA using heterologous recombinant\nprotein. Antigens that segregate patient sera from control sera (“informative antigens”) would\nbe tested further. Analogous to DNA microarrays, well-annotated and blinded test sets and\nvalidation sets of sera (such as sets of sera from different stages or types of cancers) are required\nto identify areas of potential clinical applications. Finally, prospective clinical trial analysis is\nrequired to further validate the sensitivity, specificity, and predictive value of the assay. For\ntumor antigens, gap analysis that identifies false-negative sera (“missed cancers”) would then\nbe further screened for additional autoantibody biomarkers. The potential clinical applications\nof these biomarkers include diagnosis, immune monitoring, vaccine development, and\nendpoint analysis of cancer therapeutics.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.3**r== 0.1**\nFigure 4. Methods of antigen identification\nIn traditional library screening (top row), cDNA libraries from tumor cells are expressed by\nphage, blotted on a membrane, and probed with patient sera. Confirmation of sensitivity and\nspecificity requires recombinant expression of protein for ELISA analysis. Similar approaches\nof library screening may use phage display, yeast display or peptide libraries. Cell fractionation\napproaches (middle row) are a modification of the traditional western blot. Tumor cell lysates\nmay be fractionated using microfluidic 2-dimensional or 3-dimensional separations, followed\nby printing of fractions on microarrays for probing with patient sera. Fractions identified in\npatient sera but not normal sera (“informative fractions”) are then further evaluated by mass\nspectrometry to identify target antigens, which then must be confirmed for sensitivity and\nspecificity using ELISA or similar methods. In the bottom row, protein microarrays can be\nprinted using recombinant proteins or proteins translated in situ, then probed with patient sera.\nBecause the protein microarrays are addressable, no further antigen identification is required,\nand the sensitivity and specificity of multiple antigens may be evaluated in parallel with\nscreening.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.1**\nFigure 5. Identification of HLA-binding peptides by mass spectrometry\nPeptide-MHC complexes are immunopurified from tumor cell (or antigen presenting cell,\nAPC) lysates using antibodies directed against HLA molecules. Bound peptide mixtures are\neluted in acid, and separated by nano-LC followed by MS/MS identification (top arrow).\nAlternatively, peptides may be directed applied to nanospray quadrupole TOF (bottom arrow),\nand predicted epitopes identified by fragmentation patterns.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nCurrent Methodologies for the Detection of Autoantigens\n**BLOCK**fs== 9.0**b== 0.9**t== 0.1**l== 0.4**r== 0.5**\nTable 1\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nAdvantages\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nDisadvantages\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nApplication\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nMethod\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nSEREX\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nMany clones, clone for gene\nautomatically available\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nWestern Blotting\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nReproducible\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nPhage display\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nCell Fractionation\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nELISA\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nCandidate antigen\nprotein microarrays\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.3**r== 0.5**\nLarge number of clones, little serum\nrequired, ability to refine selection with\nmultiple rounds, clone for gene\nautomatically available\nRetains post translational modification,\nmultiple arrays allow for screening for\nfractions that are informative for cancer,\ndo not need to have cloned gene or\nlibrary for screening\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.3**r== 0.5**\nReproducible, useful for comparing\nmany serum samples\nScreen many candidate antigens,\nvalidation and antigen discovery can be\ndone simultaneously, very little serum\nrequired, clone for gene automatically\navailable\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.3**\nDenatured epitopes, laborious, large\namounts of serum, requires separate\nvalidation step\nLarge amounts of serum, requires separate\nsteps to identify and validate antigens\nPartial gene products, gene\noverrepresentation, requires separate\nvalidation step\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nTedious, must identify the relevant antigen\nin potentially complex fraction, may need\nto clone gene for identified protein, requires\nseparate validation step, post translational\nmodification may not be preserved in the\nvalidation step\nTedious to produce protein for the assay.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nAntigen discovery\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nAntigen Discovery\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.8**r== 0.1**\nAntigen discovery\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\nAntigen discovery\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nValidation\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nArrays can be challenging to produce, novel\ntechonology\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nAntigen discovery and\nvalidation\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\nJ Proteome Res. Author manuscript; available in PMC 2008 August 26.",
         "extracted",
         "T Cell–Dependent Antibody Responses against Aberrantly Expressed Cyclin B1 Protein in Patients with Cancer and Premalignant Disease;Enhancement to the RANKPEP resource for the prediction of peptide binding to MHC molecules using profiles;Autoantibodies to Annexin XI-A and Other Autoantigens in the Diagnosis of Breast Cancer;A phase I trial of DNA vaccination with a plasmid expressing prostate-specific antigen in patients with hormone-refractory prostate cancer;Recombinant NY-ESO-1 protein with ISCOMATRIX adjuvant induces broad integrated antibody and CD4(+) and CD8(+) T cell responses in humans.;Self-Assembling Protein Microarrays;Protein microarray technology;Minor Histocompatibility Antigen DBY Elicits a Coordinated B and T Cell Response after Allogeneic Stem Cell Transplantation;An immunodominant SSX-2-derived epitope recognized by CD4+ T cells in association with HLA-DR.;Development of natural protein microarrays for diagnosing cancer based on an antibody response to tumor antigens.;NY-ESO-1 Expression and Immunogenicity in Malignant and Benign Breast Tumors;Clinical proteomics: revolutionizing disease detection and patient tailoring therapy.;The clinical value of intracellular autoantigens B-cell epitopes in systemic rheumatic diseases.;Serological cloning of cancer/testis antigens expressed in prostate cancer using cDNA phage surface display;Complete response to donor lymphocyte infusion in multiple myeloma is associated with antibody responses to highly expressed antigens.;Humoral immune response against melanoma antigens induced by vaccination with cytokine gene‐modified autologous tumor cells;Identification of a shared epitope recognized by melanoma-specific, HLA-A3-restricted cytotoxic T lymphocytes.;Genome-wide Characterization of a Viral Cytotoxic T Lymphocyte Epitope Repertoire*;Microarrays of tumor cell derived proteins uncover a distinct pattern of prostate cancer serum immunoreactivity;Protein arrays for autoantibody profiling and fine‐specificity mapping;Protein microarrays: Promising tools for proteomic research;Recursive partitioning as an approach to selection of immune markers for tumor diagnosis.;Antigen-specific immunity in neuroblastoma patients: antibody and T-cell recognition of NY-ESO-1 tumor antigen.;Expression of cancer/testis (CT) antigens in lung cancer.;Unique conformation of cancer autoantigen B23 in hepatoma: A mechanism for specificity in the autoimmune response;NY-ESO-1 and LAGE-1 cancer-testis antigens are potential targets for immunotherapy in epithelial ovarian cancer.;Identification of tumor‐associated antigens by screening phage‐displayed human cDNA libraries with sera from tumor patients;Tumor-specific immunological recognition of frameshift-mutated peptides in colon cancer with microsatellite instability.;CD1d-restricted T cells regulate dendritic cell function and antitumor immunity in a granulocyte–macrophage colony-stimulating factor-dependent fashion;Melanoma inhibitor of apoptosis protein (ML-IAP) is a target for immune-mediated tumor destruction;The use of HPLC-MS in T-cell epitope identification.;Immunomic analysis of human sarcoma;Enhancement of antibody detection in cancer using panel of recombinant tumor-associated antigens.;Immunoscreening of phage-displayed cDNA-encoded polypeptides identifies B cell targets in autoimmune disease.;Integrated functional genomics approach for the design of patient-individual antitumor vaccines.;Proteomics Approaches to Uncover the Repertoire of Circulating Biomarkers for Breast Cancer;CML28 is a broadly immunogenic antigen, which is overexpressed in tumor cells.;Towards patient‐specific tumor antigen selection for vaccination;Cancer/testis antigens: an expanding family of targets for cancer immunotherapy;Serological identification of antigens associated with renal cell carcinoma.;Characterisation of tumour-associated antigens in colon cancer;Solid-Phase Epitope Recovery: A High Throughput Method for Antigen Identification and Epitope Optimization;Enhanced humoral immune response correlates with improved disease-free and overall survival in American Joint Committee on Cancer stage II melanoma patients receiving adjuvant polyvalent vaccine.;Identification of Breast Cancer-Restricted Antigens by Antibody Screening of SKBR3 cDNA Library Using a Preselected Patient's Serum;Posttranslational protein modifications: new flavors in the menu of autoantigens;ATP6S1 elicits potent humoral responses associated with immune-mediated tumor destruction;Identification of cancer/testis genes by database mining and mRNA expression analysis;Autoantigen microarrays for multiplex characterization of autoantibody responses;Miniaturised multiplexed immunoassays.;Identification of Cyclin B1 as a Shared Human Epithelial Tumor-Associated Antigen Recognized by T Cells;Trends in biomarker research for cancer detection.;Autoimmune diseases: are markers ready for prediction?;Post-translational protein modifications in antigen recognition and autoimmunity.;Identification of a tissue-specific putative transcription factor in breast tissue by serological screening of a breast cancer library.;Profiling the immune response in patients with breast cancer by phage‐displayed cDNA libraries;Tumor-associated antigen TA90 immune complex assay predicts recurrence and survival after surgical treatment of stage I-III melanoma.;Evidence that specific T lymphocytes may participate in the elimination of chronic myelogenous leukemia;Pre-existent immunity to the HER-2/neu oncogenic protein in patients with HER-2/neu overexpressing breast and ovarian cancer;Profiling the immune responses in patient sera with peptide and cDNA display libraries.;Reactivity of natural and induced human antibodies to MUC1 mucin with MUC1 peptides and n‐acetylgalactosamine (GalNAc) peptides;Rapid degradation of a large fraction of newly synthesized proteins by proteasomes;Serological identification of embryonic neural proteins as highly immunogenic tumor antigens in small cell lung cancer.;p53 Antibodies in the sera of patients with various types of cancer: a review.;Expression of cancer-testis antigens in lung cancer: definition of bromodomain testis-specific gene (BRDT) as a new CT gene, CT9.;CT10: A new cancer‐testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational‐difference analysis;p53 gene mutation, tumor p53 protein overexpression, and serum p53 autoantibody generation in patients with breast cancer.;Cancer-testis antigens and ING1 tumor suppressor gene product are breast cancer antigens: characterization of tissue-specific ING1 transcripts and a homologue gene.;Antigens recognized by autologous antibody in patients with renal‐cell carcinoma;SYFPEITHI: database for MHC ligands and peptide motifs;Cleavage by Granzyme B Is Strongly Predictive of Autoantigen Status;Humoral immune responses of cancer patients against \"Cancer-Testis\" antigen NY-ESO-1: correlation with clinical events.;The telomerase catalytic subunit is a widely expressed tumor-associated antigen recognized by cytotoxic T lymphocytes.;Isoforms of the human PDZ-73 protein exhibit differential tissue expression.;The 72-kDa Component of Signal Recognition Particle Is Cleaved during Apoptosis*;The Central Role of CD4+ T Cells in the Antitumor Immune Response;Characterization of human colon cancer antigens recognized by autologous antibodies;A Survey of the Humoral Immune Response of Cancer Patients to a Panel of Human Tumor Antigens;Human lung cancer antigens recognized by autologous antibodies: definition of a novel cDNA derived from the tumor suppressor gene locus on chromosome 3p21.3.;High-titer HER-2/neu protein-specific antibody can be detected in patients with early-stage breast cancer.;SSX: A multigene family with several members transcribed in normal testis and human cancer;Autoantibodies to p53 in ovarian cancer patients and healthy women: a comparison between whole p53 protein and 18-mer peptides for screening purposes.;A testicular antigen aberrantly expressed in human cancers detected by autologous antibody screening.;Humoral immune response against p53 protein in patients with colorectal carcinoma;DNA sequence analysis of exons 2 through 11 and immunohistochemical staining are required to detect all known p53 alterations in human malignancies.;Anti-p53 antibodies in sera from patients with chronic obstructive pulmonary disease can predate a diagnosis of cancer.;Anti-p53 antibodies in sera of workers occupationally exposed to vinyl chloride.;Absence of p53 autoantibodies in sera from glioma patients.;A motif in human histidyl-tRNA synthetase which is shared among several aminoacyl-tRNA synthetases is a coiled-coil that is essential for enzymatic activity and contains the major autoantigenic epitope.;Clinical utility of the immunocytochemical detection of p53 protein in cytological specimens.;Role of bone marrow-derived cells in presenting MHC class I-restricted tumor antigens.;Scheme for ranking potential HLA-A2 binding peptides based on independent binding of individual peptide side-chains.;Vaccination with irradiated tumor cells engineered to secrete murine granulocyte-macrophage colony-stimulating factor stimulates potent, specific, and long-lasting anti-tumor immunity.;Development of antibodies against p53 in lung cancer patients appears to be dependent on the type of p53 mutation.;HLA-A2 molecules in an antigen-processing mutant cell contain signal sequence-derived peptides;Characterization of peptides bound to the class I MHC molecule HLA-A2.1 by mass spectrometry.;Very long charge runs in systemic lupus erythematosus-associated autoantigens.;Characterization of naturally occurring minor histocompatibility peptides including H-4 and H-Y.;The fine specificity of anti-La antibodies induced in mice by immunization with recombinant human La autoantigen.;Post-translational modifications of proteins: some problems left to solve.;CA125 Antigen Levels in Obstetric and Gynecologic Patients;Purification of a human prostate specific antigen.;Detection of a transformation-related antigen in chemically induced sarcomas and other transformed cells of the mouse.;SPECIFIC CARCINOEMBRYONIC ANTIGENS OF THE HUMAN DIGESTIVE SYSTEM;Immunotherapy (APC8015) for androgen independent prostate cancer (AIPC): Final survival data from a phase 3 randomized placebo-controlled trial;Efficient purification of unique antibodies using peptide affinity-matrix columns.;Endogenous immune response to early- and intermediate-stage melanoma is correlated with outcomes and is independent of locoregional relapse and standard prognostic factors.;Antibody response to DBY minor histocompatibility antigen is induced after allogeneic stem cell transplantation and in healthy female donors.;Recombinant antigen expression on yeast surface (RAYS) for the detection of serological immune responses in cancer patients.;Immunotherapy of hematologic malignancy.;p53 autoantibodies can be indicative of the development of breast cancer relapse.;The autoantibody repertoire: searching for order;MUC1-like tandem repeat proteins are broadly immunogenic in cancer patients.;Epitope identification and vaccine design for cancer immunotherapy.;Immunotherapeutic Targets Cancer : Identification of Potential Diagnostic and Cancer-related Serological Recognition of Human Colon Updated;Identification of tumor-restricted antigens NY-BR-1, SCP-1, and a new cancer/testis-like antigen NW-BR-3 by serological screening of a testicular library with breast cancer serum.;Humoral immunity to human breast cancer: antigen definition and quantitative analysis of mRNA expression.;Monitoring of p53 autoantibodies after resection of colorectal cancer: relationship to operative curability.;Gure, A. O. et al. CT10: a new cancer-testis (CT) antigen homologous to CT7 and the MAGE family, identified by representational-difference analysis. Int. J. Cancer 85, 726-732;MHCPEP, a database of MHC-binding peptides: update 1997;Cancer,Facts and Figures about.",
         "The sentinel within: exploiting the immune system for cancer biomarkers.",
         "None"
        ],
        [
         "30",
         "00274747f36c8ebe700bd4a2c0a8e4d851ddf807",
         "Two variants of the renal Na+-K+-Cl- cotransporter (NKCC2), called NKCC2A and NKCC2F, display marked differences in Na+, Rb+, and Cl- affinities, yet are identical to one another except for a 23-residue membrane-associated domain that is derived from alternatively spliced exons. The proximal portion of these exons is predicted to encode the second transmembrane domain (tm2) in the form of an α-helix, and the distal portion, part of the following connecting segment (cs1a). In recent studies, we have taken advantage of the A–F differences in kinetic behavior to determine which regions in tm2-cs1a are involved in ion transport. Functional characterizations of chimeras in which tm2 or cs1a were interchanged between the variants showed that both regions are important in specifying ion affinities, but did not allow delineating the contribution of individual residues. Here, we have extended these structure-function analyses by studying additional mutants in which variant residues between A and F were interchanged individually in the tm2-cs1a region (amino acid number 216, 220, 223, 229, or 233 in NKCC2). None of the substitutions were found to affect Km (C1-), suggesting that the affinity difference for anion transport is conveyed by a combination of variant residues in this domain. However, 2 substitutions in the tm2 of F were found to affect cation constants specifically; interestingly, one of these mutations (residue 216) only affected Km (Rb+) while the other (residue 220) only affected Km (Na+). We have thus identified two novel residues in NKCC2 that play a key role in cation transport. Because such residues should be adjacent to one another on the vertical axis of the tm2 α-helix, our results imply, furthermore, that the ion transport sites in NKCC2 could be physically linked.",
         "E. Gagnon,M. Bergeron,Nikolas D. Daigle,Marie-Hélène Lefoll,P. Isenring",
         "http://www.jbc.org/article/S0021925820792379/pdf",
         "\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nTHE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 280, NO. 37, pp. 32555–32563, September 16, 2005\n© 2005 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nMolecular Mechanisms of Cation Transport by the Renal\nNa-K-Cl Cotransporter\nSTRUCTURAL INSIGHT INTO THE OPERATING CHARACTERISTICS OF THE\nION TRANSPORT SITES*\nReceived for publication, May 19, 2005, and in revised form, June 29, 2005 Published, JBC Papers in Press, July 18, 2005, DOI 10.1074/jbc.M505511200\nE´ dith Gagnon1, Marc J. Bergeron, Nikolas D. Daigle, Marie-He´ le` ne Lefoll, and Paul Isenring2\nFrom the Nephrology Research Group, Department of Medicine, Faculty of Medicine, Laval University,\nQue´bec G1R 2J6, Canada\n**BLOCK**fs== 8.8**b== 0.3**t== 0.3**l== 0.1**r== 0.5**\nTwo variants of the renal Na-K-Cl cotransporter (NKCC2),\ncalled NKCC2A and NKCC2F, display marked differences in Na,\nRb, and Cl affinities, yet are identical to one another except for a\n23-residue membrane-associated domain that is derived from alter-\nnatively spliced exons. The proximal portion of these exons is pre-\ndicted to encode the second transmembrane domain (tm2) in the\nform of an -helix, and the distal portion, part of the following\nconnecting segment (cs1a). In recent studies, we have taken advan-\ntage of the A–F differences in kinetic behavior to determine which\nregions in tm2-cs1a are involved in ion transport. Functional char-\nacterizations of chimeras in which tm2 or cs1a were interchanged\nbetween the variants showed that both regions are important in\nspecifying ion affinities, but did not allow delineating the contribu-\ntion of individual residues. Here, we have extended these structure-\nfunction analyses by studying additional mutants in which variant\nresidues between A and F were interchanged individually in the\ntm2-cs1a region (amino acid number 216, 220, 223, 229, or 233 in\nNKCC2). None of the substitutions were found to affect Km (C1),\nsuggesting that the affinity difference for anion transport is con-\nveyed by a combination of variant residues in this domain. However,\n2 substitutions in the tm2 of F were found to affect cation constants\nspecifically; interestingly, one of these mutations (residue 216) only\naffected Km (Rb) while the other (residue 220) only affected\nKm (Na). We have thus identified two novel residues in NKCC2 that\nplay a key role in cation transport. Because such residues should be\nadjacent to one another on the vertical axis of the tm2 -helix, our\nresults imply, furthermore, that the ion transport sites in NKCC2\ncould be physically linked.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nThe second isoform of the Na-K-Cl cotransporter (NKCC2)3 is\nconfined to the apical membrane of the thick ascending limb of Henle,\nwhere it promotes substantial NaCl reabsorption (1– 4). Alternative\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n* This work was supported by Grants MOP-68949 and MOP-15405 from the Canadian\nInstitute of Health and Research and the Kidney Foundation of Canada. The costs of\npublication of this article were defrayed in part by the payment of page charges. This\narticle must therefore be hereby marked “advertisement” in accordance with 18 U.S.C.\nSection 1734 solely to indicate this fact.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n1 A Fonds de Recherche en Sante´ du Que´ bec (FRSQ) scholar.\n2 FRSQ Junior II and Associate Professor of Medicine at Laval University. To whom corre-\nspondence should be addressed: L’Hoˆ tel-Dieu de Que´ bec Research Center, 10 Rue\nMcMahon (Rm. 3852), Que´bec (QC) G1R 2J6, Canada. Tel.: 418-691-5151 (15477); Fax:\n418-691-5787; E-mail: paul.isenring@crhdq.ulaval.ca.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n3 The abbreviations used are: NKCC, Na-K-Cl cotransporter; cs, connecting segment;\nFR, flux rate; NMG, N-methyl glucamine; o, extracellular; R, regular; R, hypertonic\nregular; R, modified regular; sa, Squalus acanthias; saA, S. acanthias NKCC2A; saA/F,\nchimera in which tm2 is from saNKCC2A and cs1a from saNKCC2F; saF, S. acanthias\nNKCC2F; saF/A, chimera in which tm2 is from saNKCC2F and cs1a from saNKCC2A; tm,\ntransmembrane domain.\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.1**r== 0.6**\nThis is an Open Access article under the CC BY license.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nsplicing of the primary transcript leads to the formation of three variants\n(called NKCC2B, -A, and -F) that are identical to one another except for\nthe exon 4-encoding region, which corresponds to the second trans-\nmembrane domain (tm2) and part of the following connecting segment\n(cs1a). Interestingly, these variants are distributed differentially along\nthe thick ascending limb as follows: B is in macula densa cells, A in the\nouter medullary stripe, and F in the inner medullary stripe (6, 7). In\ncontrast to NKCC2, the first isoform of NKCC (NKCC1) is widely dis-\ntributed, arising in non-polarized cells as well as in the basolateral pole\nof many secretory epithelia (8 –12).\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe main function of both NKCC1 and NKCC2 is to couple the\nmovement of Na-K-Cl across the cell surface. Based on various\nstudies, the stoichiometry of ion transport has been shown to be 1Na:\n1K:2Cl, implying that there is no net movement of charge during the\ntranslocation cycle (13–15). In some studies, however, alternate trans-\nport modes have been proposed, e.g. 2Na:1K:3Cl for squid NKCC1\n(16), 1Na:1K:1Cl for NKCC2A (17, 18), and K/K exchange for\nboth NKCC1 and NKCC2A (17–19). We and others have hypothesized\nthat large differences in Cl affinities between the 2 Cl binding sites\ncould account for the apparent 1Na:1K:1Cl stoichiometry (17, 18),\nand incomplete reactions during the transport cycle, for the Na-inde-\npendent exchange mode (18 –19).\n**BLOCK**fs== 8.8**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nThe kinetic characteristics of NKCC1 vary widely among species and\nthose of NKCC2, among the splice variants; for example, ion constants\nare lower for human NKCC1 compared with shark (sa) NKCC1 (9,\n22–26), and lower in NKCC2A compared with F (17, 18, 27, 28). In\nprevious work, we have exploited such differences to identify residues\nthat are involved in ion movement by the transporters. Indeed, struc-\nture-function studies of wild-type and chimeric NKCCs have shown\nthat the kinetic characteristics of these proteins are specified, at least in\npart, by variant residues that are localized in the tm2 and/or cs1a regions\n(18, 22–25). Interestingly, such studies have also led to the identification\nof a high affinity-Cl binding site in cs1a.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nChimera approaches can be used quite effectively to identify variant\nresidues that convey differences in the operations of homologous carri-\ners. The main disadvantage of such studies is that key residues will be\nmissed if they are fully conserved between the carriers. This limitation is\nillustrated by recent analyses in which the behavior of various chimeras\nled to slightly different interpretations of where affinity-modifying\ndomains are localized depending on which NKCC was used to gen-\nerate the mutants. Based on these analyses, for instance, it was not\npossible to determine whether the cs1a region of NKCC1 is involved\nin ion transport (18, 22–25).\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe NKCC ion binding sites are known to be functionally interde-\npendent; that is, a change in ion affinity at one site induces a change in\nion affinity at another site (18 –21). One implication of this character-\n**BLOCK**fs== 8.8**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nistic is that it could be difficult to determine which ion constants are\nmodified directly as a result of a residue substitution. Fortunately, we\nhave recently determined the manner in which these constants are\ninterrelated based on kinetic studies of wild-type NKCC1 expressed in\nHEK-293 cells (24, 25). These data, which were analyzed in the context\nof previous kinetic models (19 –21), indicate that at high extracellular\n(o) [Na] or [K], changes in either Na or K affinity will have small\neffects on Km (Cl), whereas at high [Cl]o, a change in Cl affinity will\nhave a larger effect on Km (Na) and Km (Rb).\n**BLOCK**fs== 8.8**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nOne of the structure-function studies that was discussed above led to\nthe identification of five candidate residues in tm2-cs1a that play a def-\ninite role in Cl transport by NKCC2, but the same degree of resolution\nconcerning the structural determinants of cation transport was not\nobtained (18). The approach that was used here was valuable in this\nregard, allowing us to uncover two residues that are important in deter-\nmining Na and Rb affinities. Quite interestingly, our kinetic studies\nalso suggest that the residues identified belong to distinct ion binding\nsites in the tm2-cs1 region.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nMATERIALS AND METHODS\n**BLOCK**fs== 8.8**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nOverview—Structure-function studies were carried out to identify\naffinity-modifying residues in the tm2-cs1a region of Squalus acanthias\n(sa) NKCC2A and saNKCC2F (called saA and saF in this work). Mutant\nsaNKCC2A and saNKCCF were expressed in Xenopus laevis oocytes\nand characterized through flux assays. Unless mentioned otherwise,\nchemicals, reagents, and kits were from different suppliers or collabo-\nrators.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\ncDNAs—A total of 10 different cDNA constructs were used in these\nassays. They were subcloned in Pol1, a X. laevis oocyte expression vec-\ntor that is derived from pGEM and that contains the T7 promoter, a\nmultiple cloning site flanked by the X. laevis -globin untranslated\nregions and a poly(A) tract (17, 18, 29, 30). All of the constructs were\npropagated in XL1-blue cells (Stratagene) and purified on silica-based\ncolumns (Qiagen).\n**BLOCK**fs== 8.8**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nsaA, saF, saA/F, and saF/A—These constructs were generated in pre-\nvious studies but were not recharacterized for this analysis (18). They\ncorrespond to wild-type saA and saF, and to a pair of complementary\nchimeras in which tm2 or cs1a was interchanged between saA and saF.\nThese chimeras are called saA/F (tm2 from A, cs1a from F) and saF/A\n(tm2 from F, cs1a from A). The wild-type saA and the tm2-cs1a regions\nof saA, saF, saA/F, and saF/A are represented in Fig. 1 through hydrop-\nathy plot models to illustrate how they differ between each other and to\nshow where the junction point of the chimeras occur, thereby providing\na rationale for the choice of substitutions that were created in the cur-\nrent work.\n**BLOCK**fs== 5.5**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nsaF(L220I),\n**BLOCK**fs== 5.5**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nsaF(V216I),\n**BLOCK**fs== 5.5**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nsaA(I220L),\n**BLOCK**fs== 5.5**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nsaA(L223I),\n**BLOCK**fs== 5.5**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nsaA(I216V),\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nsaF(I223L),\nsaA(S229C), saF(C229S), saA(C233V), and saF(V233C)—These constructs con-\nsist of saA or saF inserts in which one of the tm2-cs1a codons was\ninterchanged between the variants. All of the mutations were generated\nthrough pairs of mutagenic oligonucleotides (Sigma) using the\nQuikChange Mutagenesis kit (Stratagene). The resulting constructs\nwere called saA(x) or saF(x), where A and F indicate the NKCC2 variant\nin which the residue was substituted, and x, the substitution that was\ngenerated based on the amino acid number in saNKCC2. The muta-\ngenic oligonucleotides that were used to create these mutants are shown\nin TABLE ONE, and the position of the substituted residues are also\nillustrated in Fig. 1.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nIt should be mentioned here that the choice of substitutions was not\nbased only on differences in residues between saA and saF but also on\nthe results of recent structure-function analyses that were concerned\nwith the identification of ion transport sites in both NKCC1 and -2 (18,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nFIGURE 1. Hydropathy plot models of saNKCC2 of the tm2-cs1a region. Each symbol\nrepresents an amino acid residue. A, hydropathy plot model of the entire coding\nsequence of saNKCC2 (dark and light blue, saNKCC2A; light blue, exon 4 of saNKCC2A). B,\nenlarged models of the tm2-cs1a region (yellow, saNKCC2F; red, residues that are iden-\ntical between the variants). The two models on the right show the chimeric junction\npoint of saA/F and saF/A, two saNKCC2 mutants that were analyzed in a previous study\n(18). C and D, enlarged models of the tm2-cs1a region showing the localization of various\npoint substitutions that were generated in saA or in saF. Here, the number in parentheses\nrefers to the amino acid number substituted in NKCC2. All of the models shown here\nwere drawn using the program Plot by Biff Forbush.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\n22–25). In saA/F or saF/A, for example, 9 residue substitutions were\ncreated in tm2 and 5 residue substitutions in cs1a, suggesting that a\nnumber of these residues were responsible for differences in kinetic\nbehaviors between saA and saF. However, based on the behavior of\nsaNKCC1 and human (h) NKCC1 chimeras, both these carriers also\nexhibit dramatic differences in ion affinities, we had predicted that non-\nconserved residues between the two NKCC1s as well as between high\nand low affinity wild-type NKCCs (from shark, rat, rabbit, human)\nwould be important in determining cation affinities, whereas noncon-\nserved residues between high affinity and low affinity NKCC2s (from\nshark, rat, rabbit) would be important in determining anion affinity (17,\n18, 22–28). This interpretation led to the suggestion that the main affin-\nity-modifying residues in saNKCC2 would be found at position 216,\n220, 223, 229, or 233.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nExpression in X. laevis Oocytes—To obtain synthesis of saNKCC2s in\nthis system, defolliculated stage V-VI oocytes were injected with\nsaNKCC2/Pol1-derived cRNA (5–25 ng/oocyte diluted in H2O) and\n**BLOCK**fs== 7.8**b== 0.8**t== 0.1**l== 0.1**r== 0.2**\nTABLE ONE\nOligonucleotides used to introduce single point substitutions\nOligonucleotides are all written 5 to 3. Capital letters in the sequence designate codons that were used for the substitutions.\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nConstructs\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nOligonucleotides\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nResidue change(s)\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nAmino acid number\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nsaA/F\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nsaF/A\n**BLOCK**fs== 5.0**b== 0.6**t== 0.2**l== 0.1**r== 0.8**\nsaA(I216V)\nsaF(V216I)\nsaA(I220L)\nsaF(L220I)\nsaA(L223I)\nsaF(I223L)\nsaA(S229C)\nsaF(C229S)\nsaA(C233V)\nsaF(V233C)\n**BLOCK**fs== 7.8**b== 0.6**t== 0.2**l== 0.3**r== 0.5**\nca att act TGC ATC tct ATG tcc gct ata\nTGT acc aat gga GTT gtt cgt gga ggt gg\ngtc tta aca GGG TTG tct ACC tca gcc att\nTCG acc aac gga TGT gtt cga g\nctg gct act GTG gta aca tc\nctg agt acg ATA gta act gtc\nct ata gta aca tca CTT acg ggg ttg tc\ncg gtg gta act gtc ATA aca tgt atc tc\nca att act ggg ATC tct acc tct gc\nc tta aca tgt TTG tct atg tca gcc\ncc tct gct ata TGC aca aat gg\ng tca gcc att TCA acc aat g\nca aat gga GTT gtt cgt gga gg\ncc aat gga TGT gtt cga gga gg\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nG,L,T,S,C3C,I,M,C,V\n**BLOCK**fs== 7.8**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nC,I,M,C,V3G,L,T,S,C\n**BLOCK**fs== 7.8**b== 0.6**t== 0.2**l== 0.6**r== 0.3**\nI3V\nV3I\nI3L\nL3I\nL3I\nI3L\nS3C\nC3S\nC3V\nV3C\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\na Indicates that the oligonucleotides were used in a previous study to engineer saA/F and saF/A.\n**BLOCK**fs== 7.8**b== 0.6**t== 0.2**l== 0.8**r== 0.1**\na\n222,223\n225, 229, 233\n222, 223a\n225, 229, 233\n**BLOCK**fs== 7.8**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nTABLE TWO\nComposition of flux solutions\nMedium R also contains 84 mM sucrose and Barth’s medium (B) 0.7 mM NO3\nwhereas Rb was used for the others.\n**BLOCK**fs== 7.8**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n2 and 2.4 mM HCO3\n**BLOCK**fs== 7.8**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\n. All solutions are at pH 7.4. K was used for the wash solution,\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nion\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\nNa\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nRb or K\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\nCl\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nCa2\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\nMg2\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nR: Regular\nR: Hyperosmolar\nB: Barth’s medium\n\nR: Rb\nR: Na\nR: Cl\nR: Wash\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\ndependence\ndependence\ndependence\n**BLOCK**fs== 4.5**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 4.5**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\n90.4\n\n0–87\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n0.1–20\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n0–86\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n0.7\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n0.8\n**BLOCK**fs== 5.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n2\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nPO4\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nmM\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n2\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nSO4\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n0.8\n\n\n1–10\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\nHEPES\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.7**r== 0.2**\nNMG\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.8**r== 0.1**\nGluconate\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.9**r== 0.1**\nOsM\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\n0–20\n0–87\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.8**r== 0.2**\n0–78\n**BLOCK**fs== 7.8**b== 0.4**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.8**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nmaintained for 3 days at 18 °C in Barth’s (B) medium (see TABLE TWO)\n 125 M furosemide. Groups of oocytes were also injected with H2O\nalone as controls.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nImmunolocalization Studies\n**BLOCK**fs== 8.8**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nsaNKCC2-expressing X. laevis\nof\nOocytes—They were performed as previously described (29, 30). Here,\nthe anti-P-NKCC immunoglobulin (Ig), which is specific to the phos-\nphorylated form of various NKCCs (29, 31), and which was a gift from\nDr. B. Forbush (Yale University), was used as primary antibody, whereas\nthe goat Alexa Fluor® 488-conjugated goat anti-rabbit anti-IgG (Molec-\nular Probes, A11008) was used as secondary antibody.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nKinetic Studies—Ion transport rates were determined by 86Rb influx\nmeasurements at 22 °C using different types of media (shown in\nTABLE TWO). Each of these media is derived from a regular (R)\nmedium that is iso-osmolar relative to plasma for the X. laevis frog. One\nof the modified media (called R) was obtained by adding 84 mOsM\nsucrose, and the other types of media (called R) were obtained by\nreplacing the ions of medium R with K or N-methyl gluconate (as\n2 (as anion substituents).\ncation substituents) or with gluconate  SO4\nBefore, the flux assay per se, furosemide was first removed through\nseveral rinses in medium R. Eggs were then subjected to two consecutive\nincubation steps: one of 60 min in medium R to activate the cotrans-\nporter and another of 45 min in various medium R  1–2 Ci/ml\n86Rb  10 M ouabain  250 M bumetanide to measure NKCC-\nspecific accumulation of the isotope. During this second 45-min time\ninterval, transport activity versus time is in a linear range, regardless of\nthe ion concentration used. Here, the various medium R were pro-\nduced by varying [Na] from 0 to 87 mM, [Rb] from 0.1 to 20 mM or\n**BLOCK**fs== 8.8**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n[Cl] from 0 to 86 mM (see TABLE TWO and figure legends). Fluxes\nwere terminated with washes in a high K medium R  10 M oua-\nbain  250 M bumetanide, and oocytes were transferred to 96-well\nplates (1 oocyte/well) prefilled with 2% SDS and scintillation fluid. After\na short stabilization period, accumulated 86Rb was detected with the\nTopCountNXT microplate counter (Packard).\n**BLOCK**fs== 8.8**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nTo minimize cell damage during the flux assays, all of the incubation\nsteps were carried out in 12-well plates by transferring oocytes with a\nPasteur pipette from one prefilled well to another, rather than by chang-\ning the media through aspiration and refilling of the same well. To avoid\nsolution contamination, however, each of these steps was performed\nusing three different wells prefilled with the same solution, the first two\nwells to incubate oocytes for a very short period of time, and the last well\nto complete the incubation step per se (except for the final wash during\nwhich all of the incubation times were brief). With this method, we have\nfound that ion concentrations in the final incubating solutions change\nby less than 0.5%.4\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nIn each experiment, counts among several oocytes (generally 4 – 6)\nwere averaged and transposed into flux rates (FRs). The equation used\nfor this calculation was FRs  (counts/oocyte  [85Rb] in the flux\nsolution)  ([counts] in the flux solution  time of incubation in the\nflux solution), assuming that mean membrane surface areas between\ngroups of stage V-VI oocytes are very similar; for kinetic determinations\n(see below), mean FRs were also normalized to those obtained at the\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\n4 E. Gagnon, L. Caron, and P. Isenring, unpublished results.\n**BLOCK**fs== 8.8**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nhighest ion concentration. All of these mean normalized or non-nor-\nmalized values were subsequently reaveraged from several replicate\nexperiments (n  2–5), and the data obtained were expressed as\nmean  S.E.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nIon constants (expressed as Km) and closeness of fits were calculated\nwith the programs PLOT (Biff Forbush) or SigmaPlot 4.00 for Windows\nusing the Michaelis-Menten equation (for a one binding site model), the\nHill equation with a coefficient (n) of 2 (for a two binding site model), or\nthe Hill equation with an undetermined value of n. In each case, least\nsquare analyses were carried out with a 3 or 4parameter equation (vari-\n Hill) to allow a 0 offset, assuming that 86Rb\nables: Vmax, Km, and Vmin\ninflux is never completely absent even if one of the cotransported ions is\nremoved from the flux solution (as occurs when the transporter is in the\nK/K exchange mode for example). In the current study, these kinetic\nparameters are presented as 1) averages of mean values from replicate\nexperiments (in which case differences between variables were analyzed\nthrough Student’s two tailed t-tests, assuming a null hypothesis for p\nvalues  0.05), and as 2) the results of single fits using activity versus\nconcentration data points that were also obtained by averaging mean\nvalues from replicate experiments. The latter method was employed to\nadjust for intra-experimental variability, which could introduce system-\natic errors, and because it is not possible to obtain time-matched data\nfor all of the conditions tested. The types of calculations carried out, and\nvariables that were used are specified further in the legends of tables or\nfigures.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nLast of all, it should be noted that kinetic parameters were calculated\nwithout subtracting bumetanide-sensitive 86Rb influxes measured in\nH20-injected eggs. In fact, these eggs exhibit very low levels of 86Rb\ninfluxes under the experimental conditions used for the flux assays (17,\n18, 29, 30). Hence, the dependence of saNKCC2 activity on [Na],\n[Rb], or [Cl] should remain unaltered whether this background activ-\nity is subtracted or not from the total counts. As will be discussed later,\nprevious studies have confirmed this prediction on multiple occasions\neven for heterologously expressed carriers with transport activities that\nwere as low as 3-fold above background.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nRESULTS\n**BLOCK**fs== 8.8**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nPreamble—Throughout the following sections, the kinetic behaviors\nof our 10 mutants will be compared with those of four other carriers\n(saA, saF, saA/F, and saF/A) that were tested in a previous study using\nthe same expression system (18). We elected not to repeat previous\ncharacterizations for the following reasons. 1) We have observed that\nthe kinetic features of saNKCC2s in X. laevis oocytes were very repro-\nducible over time. 2) The functional assays were carried out by the same\ninvestigators. 3) The main conclusions in this article are based on the\nkinetic characteristics of the 10 novel mutants. 4) As will be reported\nlater, mutant carriers in which a substitution was predicted to produce\nno effect on kinetic parameters behaved as such, exhibiting ion con-\nstants that were identical to previously published values (18); for saA,\nthese values, expressed as Km (Na), Km (Rb), and Km (Cl), were 15.3, 2.2,\nand 6.9 mM, and for saF, 93.0, 4.6, and 69.1 mM. 5) Recent experiments\n(n  1) revealed that the kinetic behavior of saA/F and saF/A was con-\nsistent with expectation based on our previous publication (18), which\nshowed Km (Na), Km (Rb), and Km (Cl) for saA/F of 18.0, 2.2, and 70.0\nmM, and Km for saF/A of 22.6, 1.7, and 8.7 mM.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nExpression of saNKCC2s in Oocytes—Immunofluorescence studies of\noocytes expressing wild-type or mutant saNKCC2s revealed that the\nanti-P-NKCC Ig, which was used as primary antibody, labels the heter-\nologous proteins specifically, and that most of the signal occurs at the\ncell surface (results not shown). These studies also revealed that protein\nexpression levels were relatively similar among all of the saNKCC-ex-\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nFIGURE 2. Basal characterization of saNKCC2-expressing X. laevis oocytes. Oocytes\nwere assayed for 86Rb influx in medium R  1–2 Ci/ml 86Rb  10 M ouabain  250\nbumetanide after a 1-h incubation in medium R. The data are shown as averages  S.E.\nof bumetanide-sensitive influx rates among 2–3 experiments (6 –12 oocytes/experi-\nments). Composition of media used for these studies are shown in TABLE TWO.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\npressing oocytes tested. Functional analyses, during which cotrans-\nporter activity was determined under subsaturating concentrations of\nNa, Rb, and Cl, confirmed the localization data in showing that all\nof the expressed carriers generate bumetanide-sensitive 86Rb influx\nrates that are clearly above those of the X. laevis NKCC1. As seen in Fig.\n2, e.g. these rates are between 0.8 and 7.3 nmol/oocyte/h for the mutant\nNKCC2s, that is, at least 4-fold above background for all of the carriers\nor 8-fold if saA(S229C) is excluded.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nKinetics of Na Transport: Role of tm2—As mentioned earlier, we and\nothers (17, 18, 27, 28) have observed that saA and saF exhibit substantial\ndifferences in regard to apparent Na affinities, that is, Km (Na) for saA\nis 5-fold lower compared with saF. We have also observed that\nKm (Na) remains low even when the tm2 of saA is replaced by that of\nsaF, as in the chimera saF/A, suggesting that variant residues in this\ndomain convey no differences in Na affinity as long as cs1a is from saA\n(18). Hence, replacing residue 216 or 220 in saA by those of saF should\nnot lead to changes in Km (Na) values.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nTo confirm this prediction, we measured the dependence of 86Rb\ninflux on [Na] for saA(I216V) and saA(I220L), and analyzed results in the\nlight of previously observed behaviors for saA, saF, and saF/A (18). Data\nare shown in Fig. 3A (upper row) fit by a model of Na binding at a single\nsite, and Km (Na) derived from the activity versus [ion] relationships are\nshown on Fig. 3B (first 2 bars). In TABLE THREE, similar data are\npresented except that the Km values were calculated with best values of\nHill coefficients. As expected, these mutants exhibit high Na affinities\nthat are in the range of those reported for saA and saF/A. It can be noted,\nadditionally, that NKCC activity at 0 mM Na\no is quite substantial, as\nhad also been reported for saA and saF/A, suggesting that residues 216\nor 220 do not account for the ability of certain NKCCs to induce Rb/\nRb(K) exchange at the cell surface.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nIn our recent structure-function study of saNKCC2 mutants, the kinetic\nbehavior of saF/A in regard to Na transport was also seen to be very similar\nto that of saA/F (18). These findings, which were somewhat unexpected to\nus, imply that Km (Na) for saF decreases following a Phe 3 Ala tm2 substi-\ntution, and that variant residues in this domain do convey differences in\nNa affinity but only when cs1a is from saF. They also imply that differ-\nences between saF and saA/F can be exploited to determine which residues\nin the tm2 of saA are important in determining Km (Na).\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nFig. 3A (lower row) presents the dependence of 86Rb influx on [Na]\nfor saF(V216I) and saF(L220I), and the Km values obtained are shown in Fig.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nFIGURE 3. Dependence of 86Rb influx on [Na] for saA(I216V), saF(V216I), saA(I220L), and saF(L220I). [Na] was varied from 0 to 87 mM whereas [Rb] and [Cl] were maintained\nconstant at 5 and 86 mM, respectively. A, transport activity in relative units versus [Na]. All of the data points in these panels were obtained by averaging normalized FRs from 4 – 6\noocytes/concentration/experiment among 3–5 experiments (see “Materials and Methods”). In each panel, the data points shown were fit by the Michaelis-Menten equation,\nassuming a model of Na binding at a single site. B, Km (Na) values for saA(I216V), saF(V216I), saA(I220L), and saF(L220I). These values were obtained by a single fit of the averaged data,\nusing error bars to express the closeness of fits, and the signs , *, or §, to indicate that ion constants obtained by averaging mean values from replicate experiments are not different\nstatistically (p  0.05) between the carriers () or that they are different (p  0.05), either between the carriers (*) or when compared with those calculated for the other carriers (§).\n**BLOCK**fs== 7.8**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nTABLE THREE\nKinetic constants for 10 different mutants\nIn each experiment, Km values and Hill coefficients were determined by least\nsquare analyses using a 4-parameter Hill equation (variables: Km, Vmin, Vmax,\nand Hill). They are shown here as means  S.E. that were obtained by aver-\naging mean values from 4 to 12 oocytes among 2–5 replicate experiments.\nHere, all of the Hill values  1.21 were statistically different from those  1.71\n(p  0.04).\nIons\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nHill values  S.E.\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nMutants\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nKms  S.E.\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nNa\n**BLOCK**fs== 7.8**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nRb\n**BLOCK**fs== 7.8**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nCl\n**BLOCK**fs== 5.0**b== 0.1**t== 0.5**l== 0.2**r== 0.8**\nsaA(I216V)\nsaA(I220L)\nsaF(V216I)\nsaF(L220I)\nsaA(L223I)\nsaA(S229C)\nsaA(C233V)\nsaF(I223L)\nsaF(C229S)\nsaF(V233C)\nsaA(I216V)\nsaA(I220L)\nsaF(V216I)\nsaF(L220I)\nsaA(L223I)\nsaA(S229C)\nsaA(C233V)\nsaF(I223L)\nsaF(C229S)\nsaF(V233C)\nsaA(I216V)\nsaA(I220L)\nsaF(V216I)\nsaF(L220I)\nsaA(L223I)\nsaA(S229C)\nsaA(C233V)\nsaF(I223L)\nsaF(C229S)\nsaF(V233C)\n**BLOCK**fs== 7.8**b== 0.1**t== 0.5**l== 0.3**r== 0.7**\nmM\n16.8  2.4\n22.7  5.2\n93.3  15.1\n38.2  5.9\n11.7  4.1\n15.5  5.8\n15.7  2.9\n85.5  14.1\n84.6  8.5\n82.0  10.4\n1.6  0.5\n1.7  0.4\n2.5  0.3\n4.5  0.9\n2.5  0.2\n2.2  0.4\n1.0  0.6\n4.7  1.1\n6.6  1.3\n6.7  1.5\n12.8  3.4\n8.3  5.6\n102.2  25.1\n78.0  12.4\n22.0  3.7\n20.3  3.4\n13.5  4.6\n85.9  10.2\n65.8  13.3\n66.5  11.9\n**BLOCK**fs== 7.8**b== 0.1**t== 0.5**l== 0.4**r== 0.6**\n0.82  0.21\n0.71  0.24\n1.04  0.12\n1.13  0.17\n1.03  0.03\n0.81  0.21\n0.71  0.15\n1.03  0.06\n1.04  0.16\n1.11  0.09\n1.18  0.07\n1.06  0.14\n1.12  0.03\n0.97  0.06\n0.96  0.19\n1.10  0.15\n1.02  0.08\n1.04  0.17\n0.82  0.22\n0.91  0.23\n1.13  0.17\n0.92  0.21\n1.74  0.12\n1.73  0.13\n0.88  0.19\n0.87  0.11\n1.21  0.07\n1.71  0.09\n1.76  0.05\n1.80  0.21\n**BLOCK**fs== 8.8**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\n3B (last 2 bars) and TABLE THREE; data are illustrated as above using\nthe same models of ion binding. It is seen here, that Km (Na) for\nsaF(V216I) is much higher than those for saA(I216V) or saA(I220L), but that\nKm (Na) for saF(L220I) is intermediate between those measured for the\nsaA mutants and saF(V216I). These new observations suggest that residue\n220 in the tm2 of saF accounts, at least in part, for the functional char-\nacteristic that was displayed by saA/F in regard to Na transport. It can\nalso be noted here that carrier activity at 0 mM is, as would have been\nexpected, lower for saF(V216I) and saF(L220I) compared with saA(I216V)\nand saA(I220L), but similar to that reported for saF and saA/F previously.\nKinetics of Rb Transport: Role of tm2—For either of the recently\ncharacterized saA, saF/A, saA/F, and saF, the kinetics of Rb transport\nwere shown to be very similar to those of Na transport with the excep-\ntion that NKCC activity at 0 mM Rb and 0 mM K is equally low for all\nthe carriers tested (18). Otherwise, saA, saF/A, and saA/F display a\nsimilar affinity for Rb that is 2-fold higher compared with saF and\nthe activity versus [ion] relationships is best fit with a model of substrate\nbinding at one site. Accordingly, Km (Rb) values for saA(I216V) and\nsaA(I220L) should be similar to those for saA and saF/A, whereas Km (Rb)\nvalues for saF(V216I) or saF(L220I) could potentially differ when compared\nwith those for saF.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe kinetics of Rb transport by saA(I216V), saA(I220L), saF(V216I), and\nsaF(L220I) are summarized in Fig. 4 and TABLE THREE with the repre-\nsentation schemes and binding models that were used to describe Na\ntransport. Here, the behavior of saA(I216V) and saA(I220L) confirms our\npredictions; Km (Rb) values for these mutants are in the low range, sim-\nilar to those reported for saA and saA/F. The behaviors of saF(V216I) and\nsaF(L220I), on the other hand, differ from one another; here, Km (Rb) for\nsaF(L220I) is similar to that reported for saF, but for saF(V216I), it is inter-\nmediate between that for saA(I216V) or saA(I220L) and saF(L220I). These\nfindings suggest that residue 216 accounts to some extent for the\ndecrease in Rb affinity that occurs following a Phe 3 Ala tm2 substi-\ntution. They also indicate that residues 216 and 220 play different roles\nin cation transport.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nKinetics of Cl Transport: Role of tm2—Kinetic parameters for the\nrecently characterized saF/A were found to be similar to those for saA in\nregard to Cl transport, suggesting that substitution of residue 216 or\n220 in saA by the corresponding residue in saF will produce no effects\non Km (Cl) (18). As seen in Fig. 5A (upper row), Fig. 5B (first 2 bars) and\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nFIGURE 4. Dependence of 86Rb influx on [Rb] for saA(I216V), saF(V216I), saA(I220L), and saF(L220I). [Rb] was varied from 0.1 to 20 mM whereas [Na] and [Cl] were maintained\nconstant at 87 and 86 mM, respectively. A, transport activity in relative units versus [Rb]. All of the data points shown in these panels were obtained as outlined in figure legend 3\n(4 – 6 oocytes/concentration/experiment among 3– 4 experiments) and were fit by the Michaelis-Menten equation, assuming a model of ion binding at a single site. B, Km (Rb)\nvalues and closeness of fits for saA(I216V), saF(V216I), saA(I220L), and saF(L220I). These data were also obtained and analyzed as in figure legend 3.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.4**l== 0.1**r== 0.7**\nFIGURE 5. Dependence of 86Rb influx on [Cl]\nfor saA(I216V), saF(V216I), saA(I220L), and saF(L220I).\n[Cl] was varied from 0 to 86 mM while [Rb] and\n[Cl] were maintained constant at 5 and 86 mM,\nrespectively. A, transport activity in relative units ver-\nsus [Cl]. All of the data points in these panels were\nobtained as outlined in figure legends 3 and 4 (4 – 6\noocytes/concentration/experiment among three\nexperiments) but were fit by one of two equations,\nthat is, the Michaelis-Menten equation for saA(I216V)\nand saA(I220L) (one binding site model) and the Hill\nequation for saF(V216I), saF(L220I) (two binding site\nmodel). Here, the equation that yielded the best\ncloseness of fit was selected to fit the data. B, Km (Cl)\nvalues and closeness of fits for saA(I216V), saF(V216I),\nsaA(I220L), and saF(L220I). The manner in which these\ndata were obtained and the meaning of  or § are as\nin legends for Figs. 3 and 4.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nTABLE THREE, using once more the presentation schemes of Figs. 3\nand 4 as well as a model of ion binding at one site (see below), the\ndependence profiles and Km (Cl) values for saA(I216V) and saA(I220L)\nconfirm the above predictions; both values are in the low range. For\nthese mutants, in addition, it can seen that the data points of Fig. 5A are\nvery will fit with the chosen Hill coefficient, as would have been\nexpected based on values that were obtained through least square anal-\nyses (TABLE THREE). In this regard, the behavior of saA(I216V) and\nsaA(I220L) was also similar to that of saA and saF/A.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nIn our structure-function studies of chimeric saNKCC2s, we\nreported that Cl transport kinetics for saF/A are identical to those for\nsaA, but that the behavior of saA/F is unlike that of either saA or saF\n(18). Indeed, Km (Cl) for saA/F is similar to that for saF but calculated\nHill(Cl\n) similar to that for saA. Hence, substituting residue 216 or 220\nin the tm2 of saF by the corresponding residue in saA should lead to no\nchanges in Km (Cl) values but could still alter the calculated Hill(Cl\n)s.\nBased on the kinetic characteristics of saF(V216I) and saF(L220I) (lower row\nof Fig. 5A, last 2 bars of Fig. 5B, and TABLE THREE), it is in fact seen\nthat both Km (Cl) and Hill(Cl\n) values are similar to those reported for\nsaF. These results suggest that other residues in the tm2 of saF or,\nconceivably, the presence of both residues in this domain account for\ndifferences in apparent number of transport sites for Cl.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.8**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nKinetics of Cation Transport: Role of cs1a—The kinetic behavior of\nsaA/F and saF/A showed that cs1a could also play an important, perhaps\nindirect role in cation transport (18). Accordingly, we measured the\ndependence of saA(L223I), saA(S229C), saA(C233V), saF(I223L), saF(C229S),\nsaF(V233C) on [Na] or [Rb], expecting that residue substitutions in the\ncs1a of saA would generate no kinetic changes but that residues substi-\ntutions in the cs1a of saF could alter cation affinities substantially.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nResults of these studies are illustrated in Figs. 6 and 7 using a single\nsite ion binding model and in TABLE THREE. As anticipated, Km values\nfor saA(L223I), saA(S229C), and saA(C233V) (higher rows) are all in the low\nrange, but surprisingly, those for saF(I223L), saF(C229S), or saF(V233C)\n(lower rows) are all in the high range, similar to those reported for saF.\nThese observations suggest that other residues in the cs1a of saF or\nperhaps a combination of specific residues among those that were sub-\nstituted account for the previously observed differences in Km (Na) and\nKm (Rb) between the saF and saF/A.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nKinetics Cl Transport: Role of cs1a—Although tm2 has been shown\nto play an important role in Cl transport, we have been unable to\nidentify the individual residues involved (18). Interestingly, differences\nbetween saF and saA/F had revealed the existence of a low affinity Cl\nbinding site in tm2 (Km (Cl) values for both carriers are in the high range\nbut calculated Hill(Cl\n) values are lower for saF/A) implying that the\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nFIGURE 6. Dependence of 86Rb influx on [Na] for saA(L223I), saF(I223L), saA(S229C), saF(C229S), saA(C233V), and saF(V233C). Experimental conditions and calculations were as\noutlined in figure legend 3. A, transport activity in relative units versus [Na]. The data points were from 4 to 6 oocytes/concentration/experiment among 2– 4 experiments. B,\nKm (Na) values and closeness of fits for saA(L223I), saF(I223L), saA(S229C), saF(C229S), saA(C233V), and saF(V233C) based on the data shown in A. The meaning of  or § is as in figure legend 3.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFIGURE 7. Dependence of 86Rb influx on [Rb] for saA(L223I), saF(I223L), saA(S229C), saF(C229S), saA(C233V), and saF(V233C). Experimental conditions and calculations were as\noutlined in figure legend 4. A, transport activity in relative units versus [Rb]. The data points were from 4 to 6 oocytes/concentration/experiment among 2–5 experiments. B,\nKm (Rb) values and closeness of fits for saA(L223I), saF(I223L), saA(S229C), saF(C229S), saA(C233V), and saF(V233C) based on data shown in A. The meaning of  or § is as in figure legend 4.\n**BLOCK**fs== 8.8**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nhigh affinity Cl binding site is in cs1a. In fact, the behavior of saA and\nsaF/A in our recent study suggested that this is precisely the case and\ncould have explained why cation affinities for saF/A are surprisingly\nhigh. However, because neither of residues 223, 229, or 233 have been\nfound to modulate cation transport in the current study, it is unlikely\nthat Km (Cl) values for saA(L223I), saA(S229C), saA(C233V), saF(I223L),\nsaF(C229S), and saF(V233C) will differ from those of the parent carriers.\n**BLOCK**fs== 8.8**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nThe data in Fig. 8 and TABLE THREE, which show the dependence of\nthese six mutants on [Cl] and kinetic parameters derived from the\nactivity versus [ion] relationships, are once again consistent with expec-\ntations. Indeed, all of the saAx mutants are seen to have higher affinities\nthan the saFx mutants, and the range of Km values observed are similar\nto those reported for their parent carriers. Hence, the residues or com-\nbination of residues that account for differences in Cl transport kinet-\nics between saF and saF/A or between saA and saA/F have not been\nrevealed through these mutagenic analyses.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nDISCUSSION\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nIn this study, differences in kinetic properties between saA and saF\nwere exploited to identify novel ion transport sites in the tm2-cs1a\nregion of NKCC. Our main approach was to interchange variant resi-\ndues between saNKCC2s, creating a single point substitution for each\nmutant, and determine which of these residues account for the distinc-\ntive behaviors. The role of variant residues was analyzed further by\ncomparing the effects of single point substitutions to those of combined\n**BLOCK**fs== 8.8**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nsubstitutions, exploiting our recent observation that both saA/F and\nsaF/A also differ in kinetic behavior when compared with either saA or\nsaF (18). Based on such analyses, we were able to identify two residues\nthat account for differences in cation affinity between saA and saF.\nQuite remarkably, we found that one residue (position 220) is important\nin determining Na affinity, whereas the other residue (position 216 in\nthe same domain) is important in determining Rb affinity.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nHydropathy plot models of NKCC2 have led to the prediction that the\n5-end of exon 4 encodes tm2 and the 3-end of this exon, cs1a. They\nhave also led to the prediction that the central domain of NKCC2 is\nconstituted of 12 tms that are organized as -helicoidal protein seg-\nments (8 –11, 32, 33). For NKCC1, a related carrier that share high levels\nof homology with NKCC2, such models have been confirmed indirectly\nby the identification of regions that behave as tm when expressed in the\nform of fusion proteins by rabbit reticulocyte lysates (34). Based on\nthese structural considerations, the identification of two residues that\nmodulate Na and Rb (K) affinity differentially, but that are very\nclose to one another in the tm2 region, bears a number of important\nimplications.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nOne such implication is that the ion transport sites of NKCC,\nalthough distinct kinetically, may be capable of interacting with one and\nanother through certain key residues. Indeed, the structural models of\nthe tm2-cs1a region also predicts that residues 216 and 220 (Ile and Ile in\nsaA and Val and Leu in saF) are adjacent to one another on the same\nvertical axis of the -helix, suggesting that their peptide backbones\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nFIGURE 8. Dependence of 86Rb influx on [Cl] for saA(L223I), saF(I223L), saA(S229C), saF(C229S), saA(C233V) and saF(V233C). Experimental conditions and calculations were as\noutlined in figure legend 5. A, transport activity in relative units versus [Cl]. The data points were from 6 to 12 oocytes/concentration/experiment among 2–3 experiments. Here,\ndata points for saA, saA(L223I), saA(S229C), and saA(C233V) were fit by the Michaelis-Menten equation, assuming a model of ion binding at one site, whereas data points for saF(I223L),\nsaF(C229S), and saF(V233C) were fit by the Hill equation, assuming a model of ion binding at two sites. B, Km (Cl) values and closeness of fits for saA(L223I), saF(I223L), saA(S229C), saF(C229S),\nsaA(C233V), and saF(V233C) based on data shown in A.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nand/or aliphatic R groups are connected through H atoms (35, 36). In\nthe case of NKCC, for which ions are proposed to move single file down\na translocation pocket (19 –21, 24, 25), such physical interactions could\nrepresent a key aspect of the carrier’s modus operandi, accounting, in\nparticular, for the kinetic interdependence of the ion binding sites.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nAnother implication of the findings that are discussed here is that we\nhave identified a coherent structural correlate to the widely accepted\nmodel of ordered ion binding proposed by Lytle et al. (19). Indeed, this\nmodel predicts that extracellular ions bind to the translocation pocket\nof NKCCs in the order Na-Cl-K-Cl and are released from this\npocket on the inside of the cell in the exact same order. One would thus\nassume based on such a binding scheme that the Na transport site is\nseated more deeply than the K transport site relative to the extracel-\nlular space. Our findings are in fact consistent with this interpretation in\nthat residue 216, which is localized in the middle of tm2, confers differ-\nences in Km (Rb) between saA and saF, and residue 220, which is local-\nized further into the bilayer, confers differences in Km (Na).\n**BLOCK**fs== 8.8**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nA third implication related to the identification of affinity-modifying\nresidues that are “ion-specific” is that the observed changes in Km values\nwere probably not due to substitution-induced conformational arti-\nfacts. Because Michaelis constants for ion movement through a sym-\nporter are related to several kinetic parameters that are not limited to\nthe “true binding affinities” for transported substrates (14, 24, 25), such\nartifacts could have accounted for a change in kinetic behavior by affect-\ning, e.g. a rate constant during the translocation cycle. Under such cir-\ncumstances, however, one would have expected parallel changes in Km\nvalues for all of the cotransported ions. Our results are thus reconcilable\nwith a simple model in which Km values were altered individually\nthrough substitutions of affinity-modifying residues.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nResidues 216 and 220 are nonpolar and, accordingly, predicted to lie\non the hydrophobic pole of the -helix. This localization does not nec-\nessarily imply that these two residues play no role in ion transport.\nIndeed, their peptide backbone or their R groups could still be close to\nthe hydrophilic pole of the -helix and interact directly with ions. Alter-\nnatively, residues 216 and 220 could play an important role in ion coor-\ndination from the hydrophobic pole of the -helix by influencing ion\nbinding sites through local steric interactions. To this effect, it should be\nmentioned that the current mutagenic analysis was design to identify\nresidues that confer differences in kinetic parameters and that such\nresidues, even if they are included in the Na-K-Cl transport sites, may\nnot be the same as those that are considered essential for substrate\nbinding.\n**BLOCK**fs== 8.8**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nWe have stated earlier that ion constants for NKCC1 are interrelated\nin a predictable manner but that the effect of a change in one constant\nwas not necessarily the same on all of the other constants (19 –21, 24,\n25). For example, it has been shown that the apparent affinity of an ion\nis most influenced by changes in the true affinities of the ion that binds\nnext in the translocation pocket, implying that the effect of change in\nKm (Cl) on Km (Na) and Km (Rb) will be more important than the effect\nof a change in Km (Na) and Km (Rb) on Km (Cl). Based on such inter-\nrelationships, it is reasonable to propose that the isolated decrease in\nKm (Na) resulting from a residue substitution at amino acid number 220\nin saF was related to an increase in true Na affinity, and that the\nisolated decrease in Km (Rb) resulting from a residue substitution at\namino acid number 216 was related to an increase in true Rb affinity.\nThe validity of these interpretations is substantiated further by the find-\ning that none of the substitutions in cs1a led to changes Km (Na) or\nKm (Rb).\n**BLOCK**fs== 8.8**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nCertain residues in the transport sites of NKCC1 could influence ion\naffinity only when they operate in concert with specific sets of contigu-\nous residues. Hence, the importance of variant residues in conferring\nkinetic differences between saA and saF could have gone undetected in\nthis work because our main approach was to determine the effect of\nsingle point mutations on Michaelis parameters. An example that illus-\ntrates this possibility comes from the observation that Km (Cl) values for\nsaF/A, which corresponds to a saNKCC2 harboring five substitutions in\ncs1a, were significantly different from those for saA (18), whereas\nKm (Cl) values for saA(L223I), saA(S229C), or saA(C233V), which correspond\nto saNKCC2s harboring single point substitutions in cs1a, were identi-\ncal to those for saA. It may be thus useful in future studies to examine\nthe importance of cs1a in conferring kinetic singularities by exchanging\nvariant residues 2 by 2 or 3 by 3.\n**BLOCK**fs== 8.8**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThe concept of affinity-modifying residue groups might also explain\nwhy a number of the complementary mutants did not exhibit reciprocal\nkinetic behaviors. As described under “Results,” for example, substitu-\ntions in the tm2 of saA (I216V or I220L) led to no changes in kinetic\nconstants, whereas substitutions in the tm2 of saF (V216I or L220I) did\nproduce very noticeable changes. These findings imply that certain\ncombinations of residues (e.g. Gly222-Leu223-Thr225-Ser229-Cys233) may\nplay such a preponderant role in determining ion constants, either\ndirectly or through conformational interactions between the transport\nsites, that they could mask the importance of contiguous affinity-mod-\nifying residues (Ile216-Ile220 or Val216-Leu220), whereas other combina-\n**BLOCK**fs== 8.8**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\ntions of residues at the same positions (e.g. Cys222-Ile223-Met225-Cys229-\nVal233) may play a less influential role.\n**BLOCK**fs== 8.8**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nIn this work, functional expression levels tended to differ among\nmutants (Fig. 2), whereas protein expression levels were relatively sim-\nilar based on the IF studies. Although, in principle, these observations\ncould suggest that certain substitutions altered Vmax, it is not possible at\nthis time to draw any firm conclusions in this regard without the avail-\nability of more precise quantitative estimates of protein expression at\nthe cell surface. On the other hand, it appears unlikely that changes in\ncell surface delivery or in Vmax, if they occurred, accounted for the\nobserved changes in Km values. Indeed, previous studies in oocytes have\nshown that the dependence of saA on [Na], [Rb], or [Cl] remains\nstable whether the carrier is expressed at low or high levels by varying\nthe quantities of cRNA injected.4 Here, in addition, we have observed\nthat all of the cs1a mutants that were expected to behave as saA or as saF\ndid so despite the noted differences in Vmax.\n**BLOCK**fs== 8.8**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nIn conclusion, we have identified two residues in NKCC that appear to\nplay an important role in cation transport. These residues, which are in the\ntm2-cs1a region of saNKCC2 (position 216 and 220) and which differ\nbetween saA and saF, convey variant behaviors to each of the wild-type\ncarrier and they also modulate Km (Na) and Km (Rb) differentially. Interest-\ningly, the proximity in which they occur implies that the interdependence\nof kinetic constants for this type of carrier could be mediated through com-\nplex physical interactions between the ion binding sites. Based on the effect\nof various substitutions on Km (Cl), in addition, we can postulate that\nkinetic constants for the NKCC transport system are probably each speci-\nfied by different sets of residue combinations.\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\n7. Igarashi, P., Vanden Heuvel, G. B., Payne, J. A., and Forbush, B. (1995) Am. J. Physiol.\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.5**r== 0.4**\n269, F405–F418\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n8. Xu, J. C., Lytle, C., Zhu, T. T., Payne, J. A., Benz, E., and Forbush, B. (1994) Proc. Natl.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nAcad. Sci. U. S. A. 91, 2201–2205\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n9. Payne, J. A., Xu, J. C., Haas, M., Lytle, C. Y., Ward, D., and Forbush B. (1995) J. Biol.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nChem. 270, 17977–17985\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n10. Gamba, G., Miyanoshita, A., Lombardi, M., Lytton, J., Lee, W. S., Hediger, M. A., and\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nHebert, S. C. (1994) J. Biol. Chem. 269, 17713–17722\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n11. Delpire, E., Rauchman, M. I., Beier, D. R., Hebert, S. C., and Gullans, S. R. (1994) J. Biol.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nChem. 269, 25677–25683\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n12. Lytle, C., Xu, J. C., Biemesderfer, D., and Forbush, B. (1995) Am. J. Physiol. 269,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nC1496 –C1505\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n13. Haas, M., and Forbush, B. (2000) Annu. Rev. Physiol. 62, 515–534\n14. Russell, J. M. (2000) Physiol. Rev. 80, 211–276\n15. Geck, P., Pietrzyk, C., Burckhardt, B. C., Pfeiffer, B., and Heinz, E. (1980) Biochim.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nBiophys. Acta 600, 432– 447\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n16. Russell, J. M. (1983) J. Gen. Physiol. 81, 909 –925\n17. Gagnon, E., Forbush, B., Caron, L., and Isenring, P. (2003) Am. J. Physiol. Cell Physiol.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n284, C365–C370\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n18. Gagnon, E., Bergeron, M. J., Brunet, G. M., Daigle, N. D., Simard, C. F., and Isenring,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.3**\nP. (2004) J. Biol. Chem. 279, 5648 –5654\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n19. Lytle, C., McManus, T. J., and Haas, M. (1998) Am. J. Physiol. 274, C299 –C309\n20. Benjamin, B. A., and Johnson, E. A. (1997) Am. J. Physiol. 273, F473–F482\n21. Miyamoto, H., Ikehara, T., Yamaguchi, H., Hosokawa, K., Yonezu, T., and Masuya, T.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\n(1986) J. Membr. Biol. 92, 135–150\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n22. Isenring, P., and Forbush, B. (1997) J. Biol. Chem. 272, 24556 –24562\n23. Isenring, P., Jacoby, S. C., and Forbush, B. (1998) Proc. Natl. Acad. Sci. U. S. A. 95,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n7179 –7184\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n24. Isenring, P., Jacoby, S. C., Chang, J., and Forbush, B. (1998) J. Gen. Physiol. 112,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n549 –558\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n25. Isenring, P., and Forbush, B. (2001) Comp. Biochem. Physiol. A Mol. Integr. Physiol.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n130, 487– 497\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n26. Isenring, P., Jacoby, S. C., Payne, J. A., and Forbush, B. (1998) J. Biol. Chem. 273,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n11295–11301\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nAcknowledgments—We thank Luc Caron and Micheline Noe¨l for superb tech-\nnical assistance.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n27. Gimenez, I., Isenring, P., and Forbush, B. (2002) J. Biol. Chem. 277, 8767– 8770\n28. Plata, C., Meade, P., Vazquez, N., Hebert, S. C., and Gamba, G. (2002) J. Biol. Chem.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n1. Greger, R. (1985) Physiol. Rev. 65, 760 –797\n2. Gimenez, I., and Forbush, B. (2003) J. Biol. Chem. 278, 26946 –26951\n3. Takahashi, N., Chernavvsky, D. R., Gomez, R. A., Igarashi, P., Gitelman, H. J., and\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nSmithies, O. (2000) Proc. Natl. Acad. Sci. U. S. A. 97, 5434 –5439\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n4. Yang, T., Huang, Y. G., Singh, I., Schnermann, J., and Briggs, J. P. (1996) Am. J. Physiol.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n271, F931–F939\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n5. Kaplan, M. R., Plotkin, M. D., Lee, W. S., Xu, Z. C., Lytton, J., and Hebert, S. C. (1996)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nKidney Int. 49, 40 – 47\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n6. Payne, J. A., and Forbush, B. (1994) Proc. Natl. Acad. Sci. U. S. A. 91, 4544 – 4548\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n277, 11004 –11012\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n29. Gagnon, E., Forbush, B., Flemmer, A. W., Gimenez, I., Caron, L., and Isenring, P.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.2**\n(2002) Am. J. Physiol. Renal Physiol. 283, F1046 –F1055\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n30. Caron, L., Rousseau, F., Gagnon, E., and Isenring, P. (2000) J. Biol. Chem. 275,\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n32027–32036\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n31. Flemmer, A. W., Gimenez, I., Dowd, B. F., Darman, R. B., and Forbush, B. (2002)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nJ. Biol. Chem. 277, 37551–37558\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n32. Hebert, S. C., Mount, D. B., and Gamba, G. (2004) Pflugers Arch. 447, 580 –593\n33. Park, J. H., and Saier, M. H. Jr. (1996) J. Membr. Biol. 149, 161–168\n34. Gerelsaikhan, T., and Turner, R. J. (2000) J. Biol. Chem. 275, 40471– 40477\n35. Chou, P. Y., and Fasman, G. D. (1978) Adv. Enzymol. 47, 45–148\n36. Doyle, D. A. (2004) Eur. Biophys. J. 33, 175–179",
         "extracted",
         "",
         "Molecular Mechanisms of Cation Transport by the Renal Na+-K+-Cl- Cotransporter",
         "None"
        ],
        [
         "31",
         "002a7a3e268e18203c78c3e2902ac1cd39f5dff9",
         "Angle-resolved low-coherence interferometry (a/LCI) is used to obtain quantitative, depth-resolved nuclear morphology measurements. We compare the average diameter and texture of cell nuclei in rat esophagus epithelial tissue to grading criteria established in a previous a/LCI study to prospectively grade neoplastic progression. We exploit the depth resolution of a/LCI to exclusively examine the basal layer of the epithelium, approximately 50 to 100 microm beneath the tissue surface, without the need for exogenous contrast agents, tissue sectioning, or fixation. The results of two studies are presented that compare the performance of two a/LCI modalities. Overall, the combined studies show 91% sensitivity and 97% specificity for detecting dysplasia, using histopathology as the standard. In addition, the studies enable the effects of dietary chemopreventive agents, difluoromethylornithine (DFMO) and curcumin, to be assessed by observing modulation in the incidence of neoplastic change. We demonstrate that a/LCI is highly effective for monitoring neoplastic change and can be applied to assessing the efficacy of chemopreventive agents in the rat esophagus.",
         "A. Wax,J. Pyhtila,R. Graf,R. Nines,C. Boone,R. Dasari,M. Feld,V. Steele,G. Stoner",
         "https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-10/issue-5/051604/Prospective-grading-of-neoplastic-change-in-rat-esophagus-epithelium-using/10.1117/1.2102767.pdf",
         "\n**BLOCK**fs== 20.0**b== 0.8**t== 0.1**l== 0.1**r== 0.2**\nProspective grading of neoplastic change in rat\nesophagus epithelium using angle-resolved\nlow-coherence interferometry\n**BLOCK**fs== 9.6**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nAbstract. Angle-resolved low-coherence interferometry a/LCI\nis\nused to obtain quantitative, depth-resolved nuclear morphology mea-\nsurements. We compare the average diameter and texture of cell nu-\nclei in rat esophagus epithelial tissue to grading criteria established in\na previous a/LCI study to prospectively grade neoplastic progression.\nWe exploit the depth resolution of a/LCI to exclusively examine the\nbasal layer of the epithelium, approximately 50 to 100 m beneath\nthe tissue surface, without the need for exogenous contrast agents,\ntissue sectioning, or ﬁxation. The results of two studies are presented\nthat compare the performance of two a/LCI modalities. Overall, the\ncombined studies show 91% sensitivity and 97% speciﬁcity for de-\ntecting dysplasia, using histopathology as the standard. In addition,\nthe studies enable the effects of dietary chemopreventive agents, dif-\nluoromethylornithine DFMO and curcumin, to be assessed by ob-\nserving modulation in the incidence of neoplastic change. We dem-\nonstrate that a/LCI is highly effective for monitoring neoplastic change\nand can be applied to assessing the efﬁcacy of chemopreventive\nagents in the rat esophagus. © 2005 Society of Photo-Optical Instrumentation Engi-\nneers.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nDOI: 10.1117/1.2102767\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nKeywords: light scattering; low-coherence interferometry; cell diagnostics.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nPaper SS04251R received Dec. 16, 2004; revised manuscript received Apr. 8, 2005;\naccepted for publication Apr. 13, 2005; published online Oct. 14, 2005.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nAdam Wax\nJohn W. Pyhtila\nRobert N. Graf\nDuke University\nDepartment of Biomedical Engineering\nDurham, North Carolina 27708\nE-mail: a.wax@duke.edu\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\nRonald Nines\nThe Ohio State University\nComprehensive Cancer Center\nColumbus, Ohio 43210\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nCharles W. Boone\nNational Cancer Institute\nDivision of Cancer Prevention\nBethesda, MD 20892-7322\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nRamachandra R. Dasari\nMichael S. Feld\nMassachusetts Institute of Technology\nG. R. Harrison Spectroscopy Laboratory\nCambridge, Massachusetts 02139-4307\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nVernon E. Steele\nNational Cancer Institute\nDivision of Cancer Prevention\nBethesda, MD 20892-7322\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.7**\nGary D. Stoner\nThe Ohio State University\nComprehensive Cancer Center\nColumbus, Ohio 43210\n**BLOCK**fs== 11.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n1 Introduction\n**BLOCK**fs== 9.6**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nNew avenues for monitoring cancer progression based on op-\ntical imaging techniques are under development using small\nanimal models.1–3 Such techniques employ ﬂuorescent tags\nthat exhibit preferential uptake by tumors. However, interac-\ntions between ﬂuorescent tags and experimental therapeutics,\nsuch as potential chemopreventive agents, are unknown and\nmay limit the applicability of these imaging techniques. The\nuse of intrinsic contrast mechanisms, such as those used in\nlight-scattering techniques, can circumvent these difﬁculties\nand provide a useful alternative for monitoring cancer pro-\ngression.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nAddress all correspondence to Adam Wax, Biomedical Engineering, Duke Uni-\nversity, Box 90281, Durham, NC 27708. Tel.: 919 660-5143; Fax: 919 684-\n4488; E-mail: a.wax@duke.edu\n**BLOCK**fs== 9.6**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nRecently, light-scattering techniques were shown to be ca-\npable of detecting neoplastic transformation in animal\nmodels.4,5 These techniques utilize the interaction between\nlight and cellular structures to obtain quantitative information\non nuclear morphology4 or tissue micro/nanoarchitecture5 in\nepithelial layers without the need for biopsy and histopatho-\nlogic evaluation. These noninvasive, nonperturbative tech-\nniques provide a unique opportunity to study the biology of\nneoplastic progression in living tissue, free from artifacts as-\nsociated with preparation of histopathological slides for diag-\nnostic evaluation, and at times before the tumors become vis-\nible.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nPresented here is a prospective study using angle-resolved\nlow-coherence interferometry a/LCI, a light-scattering tech-\nnique, to monitor neoplastic progression through measure-\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\n1083-3668/2005/105/051604/10/$22.00 © 2005 SPIE\n**BLOCK**fs== 9.6**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\nments of nuclear morphology.4,6 The a/LCI technique obtains\nnuclear structural information by examining the angular dis-\ntribution of backscattered light. With a/LCI, depth resolution\nis achieved by the use of a short-coherence-length light source\nin an interferometry scheme. This method of coherence gating\nis also employed to obtain depth resolution in optical coher-\nence tomography OCT, a technique used to construct to-\nmographic images of living tissues.7,8 However, a/LCI has the\nadded feature of resolving the angular distribution of scattered\nlight, enabling structural measurements that exceed the reso-\nlution obtained with OCT. In a/LCI measurements, just as in\nOCT, the depth resolution serves as a powerful means of re-\njecting multiply scattered light, enabling selective measure-\nments of light that has interacted only with a tissue region of\ninterest. This capability has been exploited to examine exclu-\nsively the morphology of basal cell nuclei, rejecting the light\nscattered by the suprabasal layers.\n**BLOCK**fs== 9.6**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nIn the presented studies, quantitative measurements of\nnuclear morphology were used as biomarkers for detecting\nand characterizing the degree of neoplastic change in the rat\nesophagus. Two a/LCI systems are used to measure in situ,\nthe mean diameter of cell nuclei and to parameterize the cy-\ntonuclear texture of cells in living tissue of rat esophageal\nepithelium without the need for ﬁxation, sectioning, and stain-\ning. The mean diameter results, a measure of the degree of\nneoplastic transformation, are used to classify the tissue sites\nin a prospective fashion based on grading criteria established\nin a previous a/LCI study.4 The performance of a/LCI to pro-\nspectively grade the tissue sites is compared with the ob-\nserved modulation of the incidence of dysplasia used to assess\nthe efﬁcacy of chemopreventive agents.\n**BLOCK**fs== 9.6**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n2 Materials and Methods\n2.1 Chemicals\nNMBA, with a purity of 99% was purchased from Ash\nStevens, Inc. Detroit, Michigan and solubilized in 20%\nDMSO. -Diﬂuoromethylornithine\ndimethyl\nDFMO was obtained from the NCI DCPC Repository c/o\nMcKesson BioServices, Rockville, Maryland. DMSO and\ncurcumin were purchased from Sigma Aldrich Chemical\nCompany Milwaukee, Wisconsin and LKT Laboratories St.\nPaul, Minnesota, respectively.\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nsulfoxide\n**BLOCK**fs== 9.6**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\n2.2 Animals and Housing\nAll of the experimental protocols were in accordance with\nNational Institutes of Health NIH guidelines and approved\nby the Institutional Animal Care and Use Committees of Duke\nUniversity, the Massachusetts Institute of Technology MIT,\nand the Ohio State University OSU. Male F344 rats, 4 to 5\nweeks of age, were obtained from Harlan Sprague Dawley\nIndianapolis, Indiana and housed at OSU. The animals were\nkept\ntemperature,\n50± 10% relative humidity, and in a 12-h light/dark cycle.\nTwice weekly cage changes ensured maintenance of hygienic\nconditions. The animals were allowed to acclimate to the fa-\ncility for 2 weeks before randomization into experimental\ntreatment groups. The diet consisted of modiﬁed AIN-76A\ncontaining 20% casein, 0.3% D,L-methionine, 52% corn-\nstarch, 13% dextrose, 5% cellulose, 5% corn oil, 3.5% AIN\nsalt mixture, 1.0% AIN vitamin mixture, and 0.2% choline\n**BLOCK**fs== 9.6**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nthree per cage at 20± 2 ° C ambient\n**BLOCK**fs== 9.6**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nbitartrate Dyets, Bethlehem, Pennsylvania. Yellow lights\nwere used in rooms where the animals were housed, since the\nchemopreventive agent, DFMO, is light sensitive. Food and\nwater were available ad libitum.\n**BLOCK**fs== 9.6**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\n2.3 Experimental Protocol\nCarcinogen-treated animals Table 1; groups 2, 3, 6, and 7\nwere given subcutaneous injections of NMBA 0.25 mg/ kg\nbody weight in the intrascapular region three times a week\nfor 5 weeks with concentrations adjusted weekly based on\naverage body weight. The solvent for NMBA was 20%\nDMSO: H2O and the injection volume was 0.2 ml. Control\ngroups groups 1 and 5 received a similar injection regimen\nof 20% DMSO: H2O. At 8, 12, and 20 weeks after the initial\ninjection of NMBA, NMBA-treated and control rats were\nshipped to MIT DFMO studies or Duke University cur-\ncumin studies to be harvested for optical spectroscopic analy-\nsis. The rats were euthanized by CO2 asphyxiation and sub-\njected to gross necropsy. The entire esophagus was excised\nand opened longitudinally for immediate optical spectroscopic\nanalysis. Following spectroscopic analysis, the esophagus was\nﬁxed in 10% neutral buffered formalin.\n**BLOCK**fs== 9.6**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\n2.4 Chemopreventive Agents\nSelected groups of rats were fed the chemopreventive agents,\nDFMO 1000 ppm, groups 3 and 4 or curcumin 6000 ppm,\ngroups 7 and 8 in modiﬁed AIN-76A diet beginning 1 week\nafter cessation of NMBA treatment until the time of analysis.\nThe diets containing DFMO or curcumin were prepared\nweekly and stored at 4 ° C before administration to the ani-\nmals. The concentrations of DFMO and curcumin in the diets\nwere conﬁrmed by high-performance liquid chromatography\nHPLC.\n**BLOCK**fs== 9.6**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 9.6**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\n2.5 Angle-Resolved Low-Coherence Interferometry\nIntact ex vivo tissue samples were examined using the a/LCI\ntechnique, implemented in one of two modalities. Samples\nfrom the DFMO study groups 1 to 4 were examined using\nthe original a/LCI prototype Fig. 1 based on a modiﬁed\nMichelson interferometer geometry, which used a superlumi-\nnescent diode SLD\n0 = 845 nm, FWHM bandwidth\n22 nm as a light source.9,10 Light from the SLD is divided by\na beamsplitter BS into a reference beam and an input beam\nto the sample. The reference beam is reﬂected by a mirror M\nand recombined at BS with light reﬂected by the sample. The\nmixed ﬁelds generate an interference pattern provided that the\ntwo optical path lengths are matched to within the coherence\nlength of the source lc = 2 ln 2 /  2 /  = 14.3 m. A\nheterodyne signal is generated by translating the reference\nmirror M at a constant velocity 4 mm/ s, resulting in an\nthe corresponding\noscillation in the heterodyne signal at\nDoppler-shifted frequency 9.5 kHz. The beat intensity is de-\ntermined by calculating the power spectrum of the digitized\nphotocurrent sampled at 333 kHz and bandpassing the sig-\nnal at the heterodyne frequency.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe original prototype a/LCI system uses four achromatic\nimaging lenses L1 to L4, focal lengths f 1 to f 4\n to enable the\nmeasurement of angular scattering distributions. These lenses\nare arranged to form multiple 4f imaging systems, which im-\nage both the phase and amplitude of the light ﬁeld. The ref-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.2**r== 0.2**\nTable 1 Protocol for administration of NMBA and chemopreventives in the a/LCI study of chemopre-\nventive efﬁcacy in the rat esophagus epithelium.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nGroup\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nTreatment\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.5**\nDose\nmg/kg body weight\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nNumber of Rats Sacriﬁced at\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nDiet\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n8 weeks\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n12 weeks\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n20 weeks\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nDMSO+\nH2O\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nNMBA\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nNMBA\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nNone\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nDMSO+\nH2O\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nNMBA\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nNMBA\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0.25\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0.25\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n0.25\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n0.25\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nNone\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nAIN-76A\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nAIN-76A\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nAIN-76A+\nDFMO 1000 ppm\nweeks 6 to 25\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nAIN-76A+\nDFMO 1000 ppm\nweeks 6 to 25\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nAIN-76A\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nAIN-76A\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nAIN-76A+\ncurcumin 6000\nppm\nweeks 6 to 25\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nAIN-76A+\ncurcumin 6000\nppm\nweeks 6 to 25\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n—\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n—\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n—\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n—\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nerence ﬁeld is made to cross the detector plane at a variable\nangle by scanning lens L2 f 2 = 4.5 cm a distance y perpen-\ndicular to the beam path. It can be shown using Fourier optics\nthat this translation causes the reference ﬁeld to be reproduced\nin the plane of the detector D with its angle of propagation\nchanged by T = 2y / f 2 but its position unchanged.10\n**BLOCK**fs== 9.6**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nSamples from the curcumin study groups 5 to 8 were\nprobed using a new, second-generation a/LCI system Fig. 2\n**BLOCK**fs== 9.6**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nbased on a Mach-Zehnder interferometer, which used a mode-\nlocked Ti:sapphire laser \n0 = 830 nm, FWHM bandwidth\n10 nm as a light source. The light is split into signal and\nreference ﬁelds by a beamsplitter and offset in frequency us-\ning acousto-optic modulators AOMs, resulting in a hetero-\ndyne beat signal of 10 MHz when the two ﬁelds are mixed at\nbeamsplitter BS2. By exploiting the short coherence length\n30 m of the source, the detected signal ﬁeld can be re-\nsolved by its optical path length. To obtain information about\nthe signal ﬁeld at various depths within a sample, a retrore-\nﬂector RR has been included in the reference beam path to\nvary its path length by 2 z. The system uses a balanced\ndetector D1 and D2, New Focus Model 1807, with the pho-\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFig. 1 Schematic of original prototype a/LCI system. The displacement\nof lens L2 causes the angle that the reference beam crosses the detec-\ntor plane to vary.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nFig. 2 Schematic of second generation a/LCI system. The displace-\nment of lens L2 causes the scattering angle, which overlaps the refer-\nence beam to vary.\n**BLOCK**fs== 9.6**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\ntocurrent demodulated using a spectrum analyzer HP 8594E\nresulting in better SNR and faster acquisition times while of-\nfering the same precision and accuracy for obtaining struc-\ntural information.11\n**BLOCK**fs== 9.6**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nThe second-generation system measures the angular distri-\nbution of the scattered ﬁeld by scanning lens L2 a distance y\nperpendicular to the beam path. It has been shown using Fou-\nrier optics that this translation will enable selective detection\nof the light that arrives at the plane of L2 traveling at an angle\n= y / f 2 relative to the optical axis, where f 2 = 10 cm is the\nfocal length12 of L2. Lens L1 f 1 = 10 cm is included to alter\nthe wavefront of the reference ﬁeld to compensate for the\neffects of L2. Lenses L3 and L4 form a 4f system, imaging\nboth the phase and amplitude of the scattered ﬁeld onto the\nplane of lens L2. Lens L5 f 5 = 10 cm is included so that L4\ndoes not focus the input beam on the sample, but instead\nrecollimates it into a pencil beam.\n**BLOCK**fs== 9.6**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nMeasured a/LCI data from both systems consist of the an-\ngular distribution of scattered light, which contains informa-\ntion about the tissue structure, as a function of depth within\nthe tissue. The light scattered from a particular depth is selec-\ntively detected using coherence gating. The effects of scatter-\ning by layers superﬁcial to those of interest have been dis-\ncussed previously10 and are negligible for a typical depth of\n50 to 80 m selected for analysis in these studies, which cor-\nresponds to the basal cell layer of the esophageal epithelium.\nThe basal layer is identiﬁed by the topmost sharp peak in the\ndensity proﬁle of the tissue, obtained by integrating the a/LCI\ndata over angle. Note that a thickened keratin layer is ob-\nserved for dysplastic samples, which can cause the basal layer\nto be as deep as 200 m Figs. 3 and 4. Although multiple\nscattering is more signiﬁcant at these depths, previous studies\nhave shown that the oscillations in the angular distribution\nused to determine size are still evident,10 in agreement with\nthe presence of angular oscillations in the current data.\n**BLOCK**fs== 9.6**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nThe diameter of the probe beam, approximately 400 m\nwith both systems, enables a few hundred cells to be sampled\nat once, providing an average nuclear morphology measure-\nment within the probe volume. The use of an average mini-\nmizes the effect of cell-to-cell variations in nuclear morphol-\nogy such as irregular nuclear shape and variations in density\narising from differing phases of the reproductive cycle. The\ndepth of the basal layer may be modulated by tens of mi-\ncrometers over the relatively broad diameter of the probe\nbeam. In such cases, the topmost peak corresponding to the\nbasal layer is broadened, spanning up to 50 m in cases of\nHGD Figs. 3 and 4. To accommodate such variations, the\nimaging depth selected for analysis is chosen to span the full\nwidth at half maximum of the topmost peak.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe a/LCI data from the basal cell layer of each sample\nwas processed and analyzed to determine the mean diameter\nof the cell nuclei, as previously described.6 In brief, a/LCI\ndata consist of an oscillatory component superposed on a\nslowly varying background. The slowly varying feature of the\na/LCI data was modeled using a second-order polynomial and\nremoved. The resulting oscillatory component was then com-\npared to a database of theoretical predictions obtained using\nMie theory from which the slowly varying features were simi-\nlarly removed for analysis. The theoretical predictions were\ncalculated for a Gaussian distribution of sizes characterized\n**BLOCK**fs== 9.6**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nby a mean diameter d and standard deviation D. In ad-\ndition, the refractive index of the nuclei relative to the cyto-\nplasm was varied from 1.015 to 1.072 in 36 discrete steps.\nThe a/LCI technique is most sensitive to changes in mean\nnuclear diameter, the descriptor of nuclear morphology used\nin this study; however, useful information may also be ob-\ntained from the standard deviation D and refractive index\nmeasurements.\n**BLOCK**fs== 9.6**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nOn obtaining the nuclear size distribution, the complete\nMie theory prediction for the determined size distribution was\nsubtracted from the measured a/LCI data. The resulting back-\nground component was then Fourier transformed to yield the\ntwo point correlation function, r, with r as the correlation\ndistance. The general form of r for the tissue samples was\na power law of the form 1 / r, as indicated by its appearance\nas a straight line on a log-log plot. The parameter  was\ndetermined by a least squares ﬁt to the data and related to the\nfractal dimension FD of these structures6 by = 3 FD. The\nuse of the FD provides a physical interpretation of this mea-\nsurement as the texture of the subcellular components.13 A\nlarger FD corresponds to a denser, grainier distribution of\nstructures, whereas a smaller FD indicates a smoother, ﬁner\ndistribution. For each sample, both the mean diameter of the\ncell nuclei and the FD are reported.\n**BLOCK**fs== 9.6**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe a/LCI measurements of mean nuclear diameter are\ncompared to grading criteria established using logistic regres-\nsion in the ﬁrst retrospective a/LCI study of the rat esophagus\ncarcinogenesis REC model4 and classiﬁed as normal 7.5\n d  10.2 m, LGD 10.2 d  13.5 m, HGD d\n 13.5 m, or apoptotic d  7.5 m. Representative\na/LCI data obtained using the original prototype and second\ngeneration systems are shown for each classiﬁcation in Figs. 3\nand 4, respectively.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n2.6 Tumor Metrology, Histological Processing, and\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\nGrading of Neoplastic Tissues\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nopened\n**BLOCK**fs== 9.6**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nPrior to a/LCI analysis, the entire esophagus of each rat was\ntumors\nexcised,\n0.05 mm in a single dimension were counted, mapped and\nmeasured. Tumor volume was calculated using the formula:\nlength width height.\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nlongitudinally,\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\nlesions\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nand\n**BLOCK**fs== 9.6**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nFollowing the a/LCI scan, the ﬂat strip of the each esopha-\ngus was cut laterally into thirds. Each strip was parafﬁn em-\nbedded with the epithelium uppermost. Longitudinal sections\nof each esophagus were cut thickness, 4 m, mounted on a\nmicroscope slide and stained with hematoxylin and eosin\nH&E. The tissue site corresponding to the area scanned us-\ning a/LCI was identiﬁed by measuring its location relative to\nthe muscle gap at the tracheal junction of the esophagus. For\nthe DFMO studies, a single tissue site, located 5 mm distal to\nthe muscle gap and laterally centered, was probed on each\nsample with a/LCI. For the curcumin studies, the use of the\nnew a/LCI system enabled multiple tissue sites on each\nsample to be probed. The additional probed sites were located\ndistal to the ﬁrst site, in increments of 1 mm along the lateral\ncenter. Each probed tissue site was classiﬁed histologically as\nfollows: normal epithelium, LGD, HGD, or epithelium with\napoptotic cells Fig. 1. This classiﬁcation scheme was based\non microscopic descriptions of dysplasia and apoptosis given\nin Cotran et al.14\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFig. 3 Typical a/LCI data solid line obtained with the original proto-\ntype system shown for each of the histological classiﬁcations. The\nmean size of the basal cell nuclei, as determined from the best ﬁt to\nthe data dashed line, is given below. Photomicrographs of ﬁxed and\nstained esophagus sections illustrate the classiﬁcation scheme bar\n= 25 mm for a,d; bar= 100 mm for b,c. Typical example of a\nnormal epithelium, b low-grade dysplasia LGD, c high-grade dys-\nplasia HGD, and d epithelium with apoptotic bodies.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nFig. 4 Typical a/LCI data solid line obtained with the second gen-\neration system are shown for each of the histological classiﬁcations.\nThe mean size of the basal cell nuclei, as determined from the best ﬁt\nto the data dashed line, is given below. Photomicrographs of ﬁxed\nand stained esophagus sections illustrate the classiﬁcation scheme\nbar= 100 mm for a to d. Typical example of a normal epithe-\nlium, b LGD, c HGD, and d epithelium with apoptotic bodies.\n**BLOCK**fs== 9.6**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n3 Results\n3.1 DFMO Study\nThe DFMO study used a/LCI to probe the nuclear morphol-\nogy of the rat esophagus epithelium samples at 8, 12, and\n20 weeks after the ﬁrst NMBA injection. For each probed\ntissue site, the mean nuclear diameter was compared to the\ngrading criteria to make a classiﬁcation, as described above.\nThe classiﬁcation results are presented in Table 2. At each\ntime point, the number of samples for each classiﬁcation for\neach animal group as deﬁned in Table 1 are shown. For each\nclassiﬁcation, the average of the mean nuclear diameter mea-\nsurements, as well as the average fractal dimension are given\nin Table 2.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nAt 8 weeks after the ﬁrst NMBA injection, the average of\nthe mean nuclear diameter measurements for normal tissue\nsites is 8.95± 0.76 m and increases to 11.2 m for the two\nLGD sites. No HGD or apoptotic sites were detected. Note\nthat for the limited number of LGD sites found at this time\npoint, the standard deviation is not meaningful. Standard de-\nviations will be given only for average measurements that\n**BLOCK**fs== 9.6**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\ninclude three or more sites. An increase in FD with neoplastic\nprogression is also observed. The mean FD of the normal\ntissue sites at 8 weeks was found to be 1.91± 0.32 and in-\ncreased to 2.05 for the LGD tissues, a difference that did not\nreach statistical signiﬁcance p = 0.1 with Student t test, used\nfor all statistical analysis here.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nAt 12 weeks after the ﬁrst NMBA injection, the average of\nthe mean nuclear diameter measurements for normal tissue\nsites is 8.95± 1.4 m and increases to 11.7± 0.53 m for the\nLGD sites. Three apoptotic sites, with an average mean\nnuclear diameter measurement of 6.60± 0.60 m, and no\nHGD sites were found. The FD was also found to increase\nwith neoplastic progression, from 1.95± 0.24 for normal tis-\nsue sites to 2.17± 0.06 for the LGD tissue sites, a statistically\nsigniﬁcant difference p  0.02. The average FD of the three\napoptotic samples was 2.12± 0.10, which is greater than that\nfor normal tissues.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nAt 20 weeks after the ﬁrst NMBA injection, the average of\nthe mean nuclear diameter measurements for normal tissue\nsites is 9.70± 1.0 m and increases to 11.0 m for LGD sites\nand 13.9 m for HGD sites. In addition, the tissue sites clas-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTable 2 Classiﬁcation of rat esophagus tissue sites examined in the\nDFMO study at 8, 12 and 20 weeks.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nWeek 8\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nGroup\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nNormal\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nLGD\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nHGD\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nApoptotic\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n1 Normal\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n2 NMBA only\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n3 NMBA+ DFMO\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n4 DFMO only\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nTotal:\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nMean diam. m\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n8.95± 0.76\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n11.2\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nFD\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n1.91± 0.32\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n2.05\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nWeek 12\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nx\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nx\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nx\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nx\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nGroup\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nNormal\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nLGD\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nHGD\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nApoptotic\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n1 Normal\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n2 NMBA only\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n3 NMBA+ DFMO\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n4 DFMO only\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nTotal:\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nMean diam. m\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n8.95± 1.4\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n11.7± 0.53\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\n1.95± 0.24 2.17± 0.06\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nWeek 20\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nx\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nx\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n6.60± 0.60\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n2.12± 0.10\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nGroup\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nNormal\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nLGD\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nHGD\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nApoptotic\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n3 NMBA+ DFMO\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n4 DFMO only\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nTotal:\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nMean Diam. m:\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n9.70± 1.0\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n11.0\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n13.9 5.80± 0.36\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nFD:\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n1.69± 0.25\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n2.09\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n2.08 1.94± 0.10\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nThe number of tissue sites exhibiting the classiﬁcation in each group, as deﬁned\nin Table 1, is given. The summary shows the total number of each classiﬁcation,\nthe average FD and the average mean nuclear diameter for that class, with\nstandard deviations given for classiﬁcations containing three or more samples.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nsiﬁed as apoptotic had an average mean nuclear diameter\nmeasurement of 5.80± 0.36 m. The FD of the tissue sites\nwas found to be 1.69± 0.25 for the normal sites, 2.09 for the\nLGD, 2.08 for the HGD and 1.94± 0.10 for the apoptotic\ntissue sites.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nThe tissue classiﬁcations in the prospective study pre-\nsented here were determined by a/LCI measurements of mean\n**BLOCK**fs== 9.6**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nnuclear diameter using the original prototype system and con-\nﬁrmed by histopathological evaluation of the probed tissue\nsites to determine the accuracy of the a/LCI grading method.\nAt each time point, it was typical for a small number of\nsamples to present signiﬁcant artifacts, which prevented\ncoregistration of the two classiﬁcation methods. Such samples\nhave been omitted from Table 2, resulting in a discrepancy\nbetween the total numbers of samples compared to the groups\npresented in Table 1.\n**BLOCK**fs== 9.6**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nFor samples where coregistration was achieved, the two\nclassiﬁcation methods were found to agree entirely for the\n8-week samples. At 12 and 20 weeks, the a/LCI measure-\nments classiﬁed one tissue site from each time point as LGD\nwhen histopathology indicated that they were normal, result-\ning in two false positives. When these results are taken into\naccount, we ﬁnd the original a/LCI system exhibited 100%\nsensitivity 11/ 11 dysplastic tissue sites and 93% speciﬁcity\n26/ 28 normal tissue sites for detecting dysplastic tissues\nupon comparison to histopathology in the DFMO study.\n**BLOCK**fs== 9.6**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nTo assess the efﬁcacy of DFMO as a chemopreventive\nagent, we compare the incidence of dysplasia LGD+ HGD\nthe group 2 NMBA only and group 3 NMBA\nfor\n+ DFMO samples. At 8 weeks after the ﬁrst NMBA injec-\ntion, we ﬁnd that the incidence of dysplasia is the same\n33%, with one LGD site found in each group. However, at\n12 weeks, a signiﬁcant difference between the two was seen,\nwith the group 2 samples showing a high incidence 71% of\ndysplasia and no apoptotic samples and group 3 showing no\nincidence 0% of dysplasia but with a signiﬁcant number of\napoptotic tissue sites 50%. This difference is also seen at\n20 weeks, with a high incidence 57% of dysplasia and no\napoptotic sites found in group 2 and no incidence 0% of\ndysplasia and a signiﬁcant number of apoptotic sites 50% in\ngroup 3.\n**BLOCK**fs== 9.6**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nThe chemopreventive efﬁcacy of DFMO can also be as-\nsessed by examining the tumor incidence, multiplicity and\nsize data for the samples used in the study Table 3. The data\nshow no signiﬁcant difference between the tumor metrology\nfor group 2 NMBA only and group 3 NMBA+ DFMO at\n8 and 12 weeks after the ﬁrst NMBA injection. Note that the\ntumor incidence, multiplicity and size actually are greater for\ngroup 3 compared to the measurements for group 2 but that\nreach statistical signiﬁcance P\nthe differences do not\n 0.05. At 20 weeks, the tumor incidence and tumor size are\ndecreased for group 3 compared to group 2 but the tumor\nmultiplicity is actually slightly increased. Of these differ-\nences, only the change in tumor size is statistically signiﬁcant\nP  0.005.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThus, for a/LCI analysis, we see no change in incidence at\n8 weeks but a sharp decrease in incidence of dysplasia ac-\ncompanied by an increase in apoptosis for DFMO treated\nsamples at 12 and 20 weeks. In comparison, the tumor me-\ntrology data show no signiﬁcant differences at 8 and\n12 weeks but a statistically signiﬁcant decrease in tumor size\nat 20 weeks.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n3.2 Curcumin Study\nThe curcumin study used the second generation a/LCI system\nto probe nuclear morphology in the rat esophagus at 12 and\n20 weeks after the ﬁrst NMBA injection. For the a/LCI data\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n6.40\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n2.03\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTable 3 Tumor incidence, multiplicity and size data for the samples\nused in the DFMO and curcumin studies.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nTable 4 Classiﬁcation of rat esophagus tissue sites examined in the\ncurcumin study at 12 and 20 weeks.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\nDFMO Study: Group 2 NMBA control\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nWeek 12\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nGroup\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nNormal\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nLGD\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nHGD Apoptotic\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nTumor\nIncidence %\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nTumor Multiplicity\nMean\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nTumor Size\nmm3\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nWeek 8\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nWeek 12\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n0.4\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n0.3\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n1.8± 0.2\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n4.8± 2.4\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nWeek 20\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n3.4± 1.2\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n31.3± 12.6\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nGroup 3 NMBA+ DFMO\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n5 Normal\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n6 NMBA only\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\n7 NMBA+ Curcumin\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n8 Curcumin only\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nTotal:\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nTumor\nIncidence %\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nTumor Multiplicity\nMean\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nTumor Size\nmm3\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nMean diam. mm\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n8.89± 0.55 12.5± 1.5\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nWeek 8\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nWeek 12\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nWeek 20\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n0.2\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n1.2\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nFD\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\n1.79± 0.26 2.16± 0.32 \n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n2.2± 0.5\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n9.0± 3.1\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nWeek 20\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n3.8± 0.4\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n9.8± 4.6\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nGroup\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nNormal\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nLGD\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nHGD Apoptotic\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nCurcumin Study: Group 6 NMBA control\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\nTumor\nIncidence %\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nTumor Multiplicity\nMean\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nTumor Size\nmm3\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nWeek 12\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nWeek 20\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n3.0± 0.7\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n13.1± 9.1\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n2.7± 0.5\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n7.7± 4.3\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nGroup 7 NMBA+ curcumin\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\nTumor\nIncidence %\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\nTumor Multiplicity\nMean\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\nTumor Size\nmm3\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nWeek 12\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nWeek 20\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n66.7\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n0.7± 0.4\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n4.5± 0.0\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n3.5± 0.3\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n20.8± 10.7a\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nSamples treated with DFMO show a statistically signiﬁcant decrease P\n 0.005 in tumor size in comparison to that of untreated samples at 20 weeks.\nThe only other statistically signiﬁcant change is seen in the decrease in tumor\nmultiplicity between curcumin treated and untreated samples at 12 weeks P\n 0.02.\naIncludes one unusually large tumor; average drops to 9.4± 5.9 mm3 if omitted.\n**BLOCK**fs== 9.6**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nobtained from each tissue site, the classiﬁcation was deter-\nmined by comparing the mean nuclear diameter to the grading\ncriteria described above. Table 4 lists the number of samples\nfor each classiﬁcation in each group at 12 and 20 weeks. The\naverage morphological parameters for each classiﬁcation are\nalso given.\n**BLOCK**fs== 9.6**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nis\n**BLOCK**fs== 9.6**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\naverage\n**BLOCK**fs== 9.6**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nincreases\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n8.89± 0.55 m. The\n**BLOCK**fs== 9.6**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAt 12 weeks after the ﬁrst NMBA injection, the average of\nthe mean nuclear diameter measurements for normal tissue\nsites\nto\n12.5± 1.5 m for the LGD tissue sites. No HGD sites were\ndetected at 12 weeks. For the one site classiﬁed as apoptotic,\nthe mean nuclear diameter was found to be 6.40 m with a\nFD of 2.03. The morphological parameters also reveal an in-\ncrease of FD with neoplastic progression. The mean FD of the\nnormal tissue sites at 12 weeks was found to be 1.79± 0.26\nand increased to 2.16± 0.32 for the LGD tissues, a statisti-\ncally signiﬁcant difference p = 0.01.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n5 Normal\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n6 NMBA only\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\n7 NMBA+ Curcumin\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n8 Curcumin only\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nTotal:\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nMean diam. m\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n8.71± 0.43 11.1± 1.4\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n14.7\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n5.80\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nFD\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.1**\n1.69± 0.10 1.72± 0.12 1.59\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n1.86\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nThe number of tissue sites exhibiting the classiﬁcation in each group, as deﬁned\nin Table 1, is given. The summary shows the total number of each classiﬁcation,\nthe average FD and the average mean nuclear diameter for that class, with\nstandard deviations given for classiﬁcations containing three or more samples.\n**BLOCK**fs== 9.6**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\nis 8.71± 0.43 m. The average increases\n**BLOCK**fs== 9.6**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nAt 20 weeks after the ﬁrst NMBA injection, the average of\nthe mean nuclear diameter measurements for the normal tis-\nsue sites\nto\n11.1± 1.4 m for the LGD sites and the mean nuclear diam-\neter determined for the one HGD site was 14.7 m. A single\napoptotic site with a mean nuclear diameter of 5.80 m was\nobserved. The FD was not seen to vary signiﬁcantly with\nneoplastic progression with an FD of 1.69± 0.10 for normal\ntissue sites, 1.72± 0.12 for the LGD sites, and 1.59 for the\nsingle HGD tissue site. The FD of the apoptotic sample was\nfound to be 1.86.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThe tissue classiﬁcations presented in this section were\ndetermined by a/LCI measurements executed with the second\ngeneration system and subsequently conﬁrmed by histopatho-\nlogical evaluation of the probed tissue sites. However, as\nshown in Table 1, fewer animals were used for the curcumin\nstudy than for the DFMO study. This decrease was enabled by\nthe faster data acquisition time of the second generation a/LCI\nsystem, permitting up to six data points to be acquired from\none tissue sample. As with the DFMO study, when the histo-\nlogical slides of the samples presented signiﬁcant artifacts,\nwhich prevented coregistration with the a/LCI probe, the data\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.2**r== 0.2**\nTable 5 Sensitivity and speciﬁcity obtained by comparing a/LCI diagnosis with the results of traditional\nhistopatholgy.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\na/LCI system\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nSensitivity\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nSpeciﬁcity\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nOriginal prototype\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\n100% 11/ 11 dysplastic sites\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\n93% 26/ 28 normal sites\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nSecond generation\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\n82% 9 / 11 dysplastic sites\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\n100% 32/ 32 normal sites\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\nCombined results\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n91%\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n97%\n**BLOCK**fs== 9.6**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nwere omitted from the study. The consequences of discarding\nthese data are described in the discussion section.\n**BLOCK**fs== 9.6**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nFor samples where coregistration was achieved, the a/LCI\nand histopathological grading methods were found to com-\npletely agree for the 12-week samples. For the 20-week\nsamples, a/LCI identiﬁed two LGD tissue sites as normal,\nresulting in two false negatives. To clarify this difference in\ngrading, note that these two false negatives were found to\nhave mean nuclear diameters of 10.2 m, only 0.1 m below\ndecision line. Including these two false negatives, the use of\nthe second-generation a/LCI system to prospectively grade\nthe neoplastic progression of the tissue sites resulted in 82%\nsensitivity 9 / 11 dysplastic tissue sites and 100% speciﬁcity\n32/ 32 normal tissue sites.\n**BLOCK**fs== 9.6**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nTo assess the efﬁcacy of curcumin as a chemopreventive\nagent, we compare the incidence of dysplasia LGD+ HGD\nthe group 6 NMBA only and group 7 NMBA\nfor\n+ curcumin samples. For the 12-week data, there is a slight\ndifference between group 6, showing a high 60% incidence\nof dysplasia, and group 7, which shows a signiﬁcantly lower\nincidence 29% of dysplasia and some apoptotic sites 14%.\nThe difference becomes more pronounced at 20 weeks, with a\nhigh incidence 100% of dysplasia in group 6 and no inci-\ndence 0% of dysplasia in group 7, although no apoptotic\nsites were detected for this group.\n**BLOCK**fs== 9.6**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nThe chemopreventive efﬁcacy of curcumin can also be as-\nsessed by examining the tumor incidence, multiplicity and\nsize data for the samples used in the study Table 3. The data\nshow decreases in tumor incidence, multiplicity and size be-\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nTable 6 Trends in the mean nuclear diameter, given in micrometers,\nfor tissue sites in the DFMO study are given.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\nClassiﬁcation\nby Group\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nNormals from\ngroups 1 and 2\nno DFMO\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nNormal from\ngroups 3 and 4\nDFMO\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nNondysplastic\nfrom groups\n3 and 4 DFMO\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nWeek 8\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nWeek 12\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nWeek 20\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n9.2\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n9.6\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n10.3a\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n8.7\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n8.5\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n8.4\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n8.7\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n7.8\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n6.8\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nSamples treated with DFMO show a decrease in mean nuclear diameter in\ncomparison to mean nuclear diameter of untreated samples.\naIncludes one false positive; average drops to 10.1 m if omitted.\n**BLOCK**fs== 9.6**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\ntween group 2 NMBA only and group 3 NMBA\n+ curcumin at 12 weeks after the ﬁrst NMBA injection. We\nnote that only the decrease in tumor multiplicity has a statis-\ntical signiﬁcance P  0.02. At 20 weeks, none of the tumor\ndata show decreases for group 3 compared to those for group\n2 but the changes are not statistically signiﬁcant P  0.05.\nThus, with a/LCI analysis, we see a signiﬁcant change in\nincidence of dysplasia at 12 weeks with a sharper decrease in\nincidence seen for curcumin treated samples at 20 weeks. In\ncomparison, the tumor metrology data show signiﬁcant de-\ncrease in tumor multiplicity at 12 weeks but no statistically\nsigniﬁcant decreases in any of the tumor metrology data at\n20 weeks.\n**BLOCK**fs== 9.6**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\n4 Discussion\nIn the two studies described here, a/LCI was used to prospec-\ntively grade the neoplastic progression of rat esophagus epi-\nthelium samples through nuclear morphology measurements\ncarried out in freshly excised, intact tissue samples without\nthe need for exogenous staining agents, ﬁxation or tissue sec-\ntioning. The mean diameter of cell nuclei in the basal epithe-\nlial layer was used as the grading criteria, as it was shown to\nbe a powerful indicator of neoplastic progression in a previ-\nous study.4 The utility of a/LCI for monitoring neoplastic pro-\ngression is evident by the high rate of success in classifying\ntissues by their degree of neoplastic change Table 5. The\noriginal prototype a/LCI system exhibited a high sensitivity\nand speciﬁcity 100% and 93% respectively while the second\ngeneration system also showed a lower sensitivity 82% but a\nhigher speciﬁcity 100%. We note that the decrease in sensi-\ntivity of the second-generation system is due to two false\nnegatives seen in the sites examined at 20 weeks for which\nthe measured diameter was only slightly below the decision\nline. Overall, on combining the results of the two studies, we\nﬁnd that a/LCI successfully identiﬁed 58 of 60 normal tissue\nsites 97% speciﬁcity and 20 of 22 dysplastic tissue sites\n91% sensitivity.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nAnalysis of the mean nuclear diameter data reveals an\nanomalous statistical trend in the standard deviation of the\naverage mean nuclear diameter measurements for tissue sites\nclassiﬁed as normal in the DFMO study. The average mean\nnuclear diameter is seen to change from 8.95± 0.76 m at\n8 weeks to 8.95± 1.4 m at 12 weeks to 9.70± 1.0 m at\n20 weeks. On further examination, we can see that this trend\nis due to the emergence of a bimodal distribution of the mean\nnuclear diameters, as shown in Table 6. By separating the\nnormal samples into those treated with DFMO and those\nwithout treatment, we see that the mean nuclear diameter in-\n**BLOCK**fs== 9.6**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nthat this change corresponds to a decrease in smaller struc-\ntures and an increase in larger structures. This physical inter-\npretation agrees nicely with changes in the texture of cell\nnuclei seen in histological images, where clumped, grainy cell\nnuclei are characteristic of dysplasia. Continued study of FDs\nin animal models may lead to an improved understanding of\nmechanisms of carcinogenesis, while improvements in FD\nmeasurements, which yield improved statistical signiﬁcance,5\nwill serve to increase its value as a diagnostic biomarker.\n**BLOCK**fs== 9.6**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nIn addition to monitoring neoplastic transformation, one\ngoal of the studies was to determine the efﬁcacy of the\nchemopreventive agents by measuring changes in nuclear\nmorphometry using a/LCI. In a previous study, we identiﬁed\nthat chemopreventive action can be inferred by using a/LCI to\ndetect the presence of apoptotic cells within the tissue.4 This\nﬁnding is corroborated by the identiﬁcation of stimulation of\napoptosis as the mechanism behind the efﬁcacy of DFMO as\na chemopreventive agent in rat esophagus epithelium.15 For\nthe tissue sites classiﬁed as apoptotic, the a/LCI measure-\nments showed a decrease in mean nuclear diameter and an\nincrease in FD, both features agreeing with the contracted\nnucleus seen in apoptosis. Both the DFMO and curcumin\ntreated samples showed evidence of apoptotic cells, while si-\nmultaneously showing decreased incidence of dysplasia.\n**BLOCK**fs== 9.6**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nThe capability of a/LCI to detect subtle changes in nuclear\nmorphology enables greater sensitivity in assessing chemo-\npreventive efﬁcacy than traditional studies using tumor me-\ntrology data. In the DFMO study, a/LCI detects modulation in\nthe incidence of dysplasia, accompanied by the presence of\napoptotic cells as early as 12 weeks after the initial NMBA\ninjection. The tumor metrology data, on the other hand, does\nnot conﬁrm the chemopreventive efﬁcacy until 20 weeks,\nwith the ﬁrst appearance of statistically signiﬁcant changes.\nFurther, although the tumor data for the curcumin study\nshows a decrease in incidence and tumor multiplicity at\n12 weeks, conﬁrming the a/LCI ﬁndings, the limited number\nof samples in this study yields no statistically signiﬁcant\nchanges at 20 weeks. In these studies, the limited number of\nanimals used per group three to seven animals/group af-\nfected the statistical signiﬁcance of the tumor metrology data.\nNote that the number of animals in these studies was far\nsmaller than the 30 animals per group typically used for a\nclassic bioassay. Thus, we see that a/LCI can assess chemo-\npreventive efﬁcacy with fewer samples than conventional\nmethods.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe sensitivity afforded by a/LCI enables a comparison of\nthe relative chemopreventive efﬁcacy of DFMO and curcumin\nby examining the frequency of the observed classiﬁcations.\nWhile DFMO produced a sharp decrease in the incidence of\ndysplasia at 12 and 20 weeks, accompanied by a signiﬁcant\nnumber of sites with apoptotic cells, it was found that cur-\ncumin produced a milder decrease in both the incidence of\ndysplasia and apoptosis at 12 weeks. This suggests that at the\ndose levels tested, curcumin is not as effective as DFMO as a\nchemopreventive agent in the rat esophagus.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nAt 20 weeks, the decrease of incidence of dysplasia in the\ncurcumin treated rats was similar in degree to that found for\nthe DFMO treated rats. However, only 1 apoptotic tissue site\nout of 13 1 / 13 was seen in the curcumin samples at\n20 weeks, which was smaller than the number seen in the\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nFig. 5 Graphical comparison of the average fractal dimension of nor-\nmal and dysplastic tissue sites for the DFMO and curcumin studies. In\nboth studies, a statistically signiﬁcant difference in the average fractal\ndimension was observed P  0.0002 and P  0.025 for normal and\ndysplastic tissue sites, respectively.\n**BLOCK**fs== 9.6**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nfor\n**BLOCK**fs== 9.6**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nthe\n**BLOCK**fs== 9.6**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\ntime\n**BLOCK**fs== 9.6**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nwith\n**BLOCK**fs== 9.6**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nuntreated\n**BLOCK**fs== 9.6**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nsamples*\ncreases\n9.2 to 9.6 to 10.3 m and slightly decreases with time for\nsamples treated with DFMO 8.7 to 8.5 to 8.4 m. The de-\ncreasing trend of the treated samples becomes even clearer\nwhen all nondysplastic samples normal+ apoptotic are\ncombined 8.7 to 7.8 to 6.8 m. The bimodal trend shows a\nmanifestation of the chemopreventive action of DFMO, in\nthat it causes changes in nuclear morphology that are consis-\ntent with the structural changes seen in apoptosis. Although\nthese trends in nuclear size are not evident in the normal\nsamples from the curcumin study, i.e., no distinct changes in\nthe standard deviation are observed, the results of the DFMO\nstudy suggest that it may be appropriate to include the age of\nthe animals and/or chemopreventive regimen in determining\nthe decision lines used to classify tissues based on a/LCI mea-\nsurements.\n**BLOCK**fs== 9.6**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nIn addition to the mean nuclear diameter, we report on the\nfractal dimension of subcellular structures in the probed tissue\nsites Fig. 5. Although this parameter was not used in grading\nthe tissue sites, the FD has become an area of recent interest\nin animal models of carcinogenesis.4,5 In the DFMO study, the\nnormal tissue sites show an FD of 1.88± 0.29, whereas those\nclassiﬁed as dysplastic show an FD of 2.12± 0.08, a statisti-\ncally signiﬁcant difference p  0.0002. For the curcumin\nstudy, an FD of 1.78± 0.17 is obtained for normal tissue sites\nwith an FD of 2.01± 0.34 seen for dysplastic sites, which is\nalso a statistically signiﬁcant difference p  0.025, although\nnot as great as in the DFMO study. These results are presented\ngraphically in Fig. 5 and are consistent with our previous\nﬁndings using this model4 and those found for a study of the\nFD of precancerous rat colon epithelium.5 In the a/LCI data,\nthe change in the FD arises from corresponding changes in the\nstructural composition of\nin the\n2- to 10-m size range as they undergo neoplastic transfor-\nmation. The correlation function r of the tissues reveals\n**BLOCK**fs== 9.6**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nthe cells and tissues\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n*Note that the 10.3-m measurement is above the decision line between\nnormal and dysplastic tissues. This ﬁgure includes one false positive a/LCI mea-\nsurement. On excluding the false positive, the average measurement falls to\n10.1 m.\n**BLOCK**fs== 9.6**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nDFMO samples 3 / 5 at 20 weeks. The decrease in incidence\nof apoptosis also supports the assertion that curcumin is not as\neffective a chemopreventive agent as DFMO at the tested dos-\nage levels.\n**BLOCK**fs== 9.6**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\n5 Conclusion\nIn summary, the utility of a/LCI measurements of nuclear\nmorphology as biomarkers for monitoring neoplastic transfor-\nmation has been demonstrated via a prospective study, obtain-\ning high sensitivity and speciﬁcity with both instrument de-\nsigns. The in situ nuclear morphology measurements are\naccomplished without the use of exogenous staining agents,\nﬁxatives, or tissue sectioning. The a/LCI technique was ap-\nplied to determine the efﬁcacy of DFMO and curcumin as\nchemopreventive agents in rat esophagus epithelium. Future\napplications of a/LCI will\ninclude surveillance of human\nesophageal tissues in patients with Barrett’s esophagus. The\ntransition to clinical usage will be facilitated by a newly de-\nveloped version of a/LCI with a dramatically improved data\nacquisition time.16\n**BLOCK**fs== 9.6**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nAcknowledgments\nThis work was supported by grants from Hamamatsu Corpo-\nration and the National Institutes of Health through the Na-\ntional Center for Research Resources P41-RR02594 and the\nNational Cancer Institute NCI-CN15011-72. JWP is sup-\nported by a training grant from the National Institutes of\nHealth T32 EB001040.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nReferences\n1. L. M. Kalikin, A. Schneider, M. A. Thakur, Y. Fridman, L. B. Grifﬁn,\nR. L. Dunn, T. J. Rosol, R. B. Shah, A. Rehemtulla, L. K. McCauley,\nand K. J. Pienta, “In vivo visualization of metastatic prostate cancer\nand quantitation of disease progression in immunocompromised\nmice,” Cancer Biol. Ther. 26, 656–660 2003.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n2. G. Choy, S. O’Connor, F. E. Diehn, N. Costouros, H. R. Alexander,\nP. Choyke, and S. K. Libutti, “Comparison of noninvasive ﬂuorescent\nand bioluminescent small animal optical imaging,” BioTechniques\n355, 1022–1026 2003.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n3. Y. Chen, G. Zheng, Z. H. Zhang, D. Blessington, M. Zhang, H. Li, Q.\nLiu, L. Zhou, X. Intes, S. Achilefu, and B. Chance, “Metabolism-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.6**r== 0.1**\nenhanced tumor localization by ﬂuorescence imaging: in vivo animal\nstudies,” Opt. Lett. 2821, 2070–2072 2003.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n4. A. Wax, C. H. Yang, M. G. Muller, R. Nines, C. W. Boone, V. E.\nSteele, G. D. Stoner, R. R. Dasari, and M. S. Feld, “In situ detection\nof neoplastic transformation and chemopreventive effects in rat\nesophagus epithelium using angle-resolved low-coherence interfer-\nometry,” Cancer Res. 6313, 3556–3559 2003.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n5. H. K. Roy, Y. Liu, R. K. Wali, Y. L. Kim, A. K. Kromine, M. J.\nGoldberg, and V. Backman, “Four-dimensional elastic light-scattering\nﬁngerprints as preneoplastic markers in the rat model of colon car-\ncinogenesis,” Gastroenterology 1264, 1071–1081 2004.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n6. A. Wax, C. Yang, V. Backman, K. Badizadegan, C. W. Boone, R. R.\nDasari, and M. S. Feld, “Cell organization and sub-structure mea-\nsured using angle-resolved low coherence interferometry,” Biophys.\nJ. 82, 2256–2264 2002.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n7. J. A. Izatt, M. D. Kulkarni, H. W. Wang, K. Kobayashi, and M. V.\nSivak, “Optical coherence tomography and microscopy in gas-\ntissues,” IEEE J. Sel. Top. Quantum Electron. 24,\ntrointestinal\n1017–1028 1996.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n8. D. Huang, E. A. Swanson, C. P. Lin, J. S. Schuman, W. G. Stinson,\nW. Chang, M. R. Hee, T. Flotte, K. Gregory, C. A. Puliaﬁto, and J. G.\nFujimoto, “Optical coherence tomography,” Science 2545035,\n1178–1181 1991.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n9. A. Wax, C. H. Yang, R. R. Dasari, and M. S. Feld, “Measurement of\nangular distributions by use of low-coherence interferometry for\nlight-scattering spectroscopy,” Opt. Lett. 266, 322–324 2001.\n10. A. Wax, C. Yang, V. Backman, M. Kalashnikov, R. R. Dasari, and M.\nS. Feld, “Determination of particle size using the angular distribution\nof backscattered light as measured with low-coherence interferom-\netry,” J. Opt. Soc. Am. A 19, 737–744 2002.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n11. J. W. Pyhtila, R. N. Graf, and A. Wax, “Determining nuclear mor-\nphology using an improved angle-resolved low coherence interferom-\netry system,” Opt. Express 1125, 3473–3484 2003.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n12. A. Wax and J. E. Thomas, “Optical heterodyne imaging and Wigner\nphase space distributions,” Opt. Lett. 2118, 1427–1429 1996.\n13. A. J. Einstein, H. S. Wu, and J. Gil, “Self-afﬁnity and lacunarity of\nchromatin texture in benign and malignant breast epithelial cell nu-\nclei,” Phys. Rev. Lett. 80, 397–400 1998.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n14. R. S. Cotran, S. L. Robbins, and V. Kumar, Robbins Pathological\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\nBasis of Disease, Saunders, Philadelphia 1994.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n15. L. Y. Y. Fong, V. T. Nguyen, A. E. Pegg, and P. N. Magee, “alpha-\nDiﬂuoromethylornithine induction of apoptosis: a mechanism which\nreverses pre-established cell proliferation and cancer initiation in\nesophageal carcinogenesis in zinc-deﬁcient rats,” Cancer Epidemiol.\nBiomarkers Prev. 103, 191–199 2001.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n16. J. W. Pyhtila and A. Wax, “Rapid, depth-resolved light scattering\nmeasurements using Fourier domain, angle-resolved low coherence\ninterferometry,” Opt. Express 12, 6178–6183 2004, http://\nwww.opticsexpress.org/abstract.cfm?URIOPEX-12-25-6178.",
         "extracted",
         "",
         "Prospective grading of neoplastic change in rat esophagus epithelium using angle-resolved low-coherence interferometry.",
         "None"
        ],
        [
         "32",
         "002b13e918af5ff02210a66ae36361810a1f9d22",
         "It has been argued that phylogenetic fear-relevant stimuli elicit preattentive capture of attention. To distinguish between fear relevance and time of appearance in evolutionary history, the authors compare phylogenetic and ontogenetic fear-relevant and fear-irrelevant stimuli in a visual search task. The authors found no evidence for a special role of phylogenetic fear-relevant stimuli; it seems that fear relevance in general is more important than is the evolutionary age. The pattern of results indicates that attention toward threatening stimuli is mainly affected by a late component that prolongs the disengagement of attention.",
         "T. Brosch,D. Sharma",
         "https://access.archive-ouverte.unige.ch/access/metadata/414fbfe1-4481-4240-92c1-e62a3593f0f0/download",
         "\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nArticle scientifique Article\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.1**\n2005                                     Published version Open Access\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nThis is the published version of the publication, made available in accordance with the publisher’s policy.\n**BLOCK**fs== 14.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nThe Role of Fear-Relevant Stimuli in Visual Search: A Comparison of\n**BLOCK**fs== 14.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\nPhylogenetic and Ontogenetic Stimuli\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nBrosch, Tobias; Sharma, Dinkar\n**BLOCK**fs== 11.7**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nHow to cite\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nBROSCH, Tobias, SHARMA, Dinkar. The Role of Fear-Relevant Stimuli in Visual Search: A Comparison\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nof Phylogenetic and Ontogenetic Stimuli. In: Emotion, 2005, vol. 5, n° 3, p. 360–364. doi: 10.1037/1528-\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n3542.5.3.360\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nThis publication URL:\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.4**\nhttps://archive-ouverte.unige.ch/unige:88832\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nPublication DOI:\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.5**\n10.1037/1528-3542.5.3.360\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.2**\n© This document is protected by copyright. Please refer to copyright holder(s) for terms of use.",
         "extracted",
         "Attentional control of the processing of neural and emotional stimuli.;Searching for threat;Emotion drives attention: detecting the snake in the grass.;The face in the crowd revisited: a threat advantage with schematic stimuli.;Fear and anxiety: animal models and human cognitive psychophysiology.;Facial Expressions of Emotion: Are Angry Faces Detected More Efficiently?;A subcortical pathway to the right amygdala mediating \"unseen\" fear.;It Takes a Confounded Face to Pop Out of a Crowd;On the automatic nature of phobic fear: conditioned electrodermal responses to masked fear-relevant stimuli.;Orienting and schizophrenia: stimulus significance, attention, and distraction in a signaled reaction time task.;A feature-integration theory of attention;The premise of equipotentiality in human classical conditioning: conditioned electrodermal responses to potentially phobic stimuli.;One-trial learning and superior resistance to extinction of autonomic responses conditioned to potentially phobic stimuli.;Phobias and preparedness;The cognitive neuroscience of emotion;Fears, phobias, and preparedness: toward an evolved module of fear and fear learning.;Unconscious emotion : Evolutionary perspectives, psychophysiological data, and neuropsychological mechanisms;A threat imminence approach to human fear responding : direction of threath, aversive contexts, and electrodermal responses;Preparedness and electrodermal fear-conditioning: ontogenetic vs phylogenetic explanations.;Finding the face in the crowd: an anger superiority effect.;Feature analysis in early vision: evidence from search asymmetries.;The emotional brain.;E-Prime;32306-1270), as editor of the Clinician's Research Digest newsletter for",
         "The role of fear-relevant stimuli in visual search: a comparison of phylogenetic and ontogenetic stimuli.",
         "None"
        ],
        [
         "33",
         "002b2d9da5439eb1d847ab261f03e536e0880362",
         "WBN/Kob-Ht rats (Ht-rats) raised under conventional conditions spontaneously developed dermatitis. In this study, we carried out histopathological analysis to elucidate the pathological features of the dermatitis in Ht-rats. We then tried to detect Staphylococcus species recovered from the skin lesions of Ht-rats. We also measured the serum levels of total IgE, IL-4 and IFN-gamma in these rats. The histopathological data indicated that inflammatory cells had infiltrated the skin lesions. Staphylococcus aureus was recovered from the skin lesions, and the serum levels of total IgE and IL-4 were elevated in Ht-rats with dermatitis. These results suggest that dermatitis in Ht-rats is similar to that in the DS-Nh mice, which has recently been proposed as an animal model for human atopic dermatitis.",
         "M. Asakawa,T. Yoshioka,I. Hikita,T. Matsutani,T. Hirasawa,A. Arimura,T. Sakata,T. Horikawa",
         "https://www.jstage.jst.go.jp/article/expanim/54/5/54_5_461/_pdf",
         "\n**BLOCK**fs== 18.0**b== 0.7**t== 0.2**l== 0.2**r== 0.2**\nWBN/Kob-Ht Rats Spontaneously Develop\nDermatitis under Conventional Conditions:\nAnother Possible Model for Atopic Dermatitis\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.2**r== 0.2**\nMakoto ASAKAWA1), Takeshi YOSHIOKA1), Ichiro HIKITA1),\nTakaji MATSUTANI2), Tsutomu HIRASAWA1), Akinori ARIMURA1),\nTsuneaki SAKATA1), and Tatsuya HORIKAWA3)\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.2**r== 0.2**\n1)Shionogi Discovery Research Laboratories, Shionogi Co., Ltd., 3–1–1 Futaba-cho, Toyonaka 561-0825,\n2)Division of Immunology and Embryology, Department of Cell Biology, Tohoku University School of\nMedicine, Sendai 980-8575, 3)Division of Dermatology, Department of Clinical Molecular\nMedicine, Kobe University Graduate School of Medicine, Kobe 650-0017, Japan\n**BLOCK**fs== 9.2**b== 0.3**t== 0.5**l== 0.2**r== 0.2**\nAbstract:  WBN/Kob-Ht rats (Ht-rats) raised under conventional conditions spontaneously\ndeveloped dermatitis.  In this study, we carried out histopathological analysis to elucidate\nthe  pathological  features  of  the  dermatitis  in  Ht-rats.    We  then  tried  to  detect\nStaphylococcus species recovered from the skin lesions of Ht-rats.  We also measured the\nserum levels of total IgE, IL-4 and IFN-γ in these rats.  The histopathological data indicated\nthat  inflammatory  cells  had  infiltrated  the  skin  lesions.    Staphylococcus  aureus  was\nrecovered from the skin lesions, and the serum levels of total IgE and IL-4 were elevated in\nHt-rats with dermatitis.  These results suggest that dermatitis in Ht-rats is similar to that in\nthe DS-Nh mice, which has recently been proposed as an animal model for human atopic\ndermatitis.\nKey words:  atopic dermatitis, DS-Nh mice, WBN/Kob-Ht rats\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nThe WBN/Kob-Ht rat (Ht-rat) is a spontaneous hair-\nless mutant of the Wistar strain that was established at\nIshikawa  Laboratory  Animal  Company,  Saitama,  Ja-\npan.    The  inheritance  mode  of  the  Ht  mutation  is\nautosomal  dominant,  and  the  Ht  locus  is  mapped  on\nchromosome 10 [1].  Since the inheritance mode of the\nHt gene is dominant and Ht homozygote rats have high\nmorbidity, the Ht-rat used for this study was of the Ht\nheterozygote type.  In a previous study, these rats spon-\ntaneously  developed  dermatitis  under  conventional\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nconditions, with an incidence of approximately 4% at\n20 weeks of age [17].  However, the pathogenic mecha-\nnism  and  characteristics  of  the  dermatitis  were  not\nexamined from the viewpoint of studying allergy.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nIn human atopic dermatitis (AD), an increase in the\nnumber of infiltrating CD4- bearing T cells, mast cells,\nand eosinophils has been detected in acute skin lesions\nas compared with the normal skin [8, 9, 14].  The le-\nsions show a significant increase in the number of cells\nexpressing IL-4, IL-5 and IL-13 mRNA, this suggest-\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\n(Received 28 October 2004 / Accepted 14 September 2005)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nAddress corresponding: T. Sakata, Shionogi Discovery Research Laboratories, Shionogi Co., Ltd., 3–1–1 Futaba-cho, Toyonaka, Osaka 561-\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n0825, Japan\n**BLOCK**fs== 8.5**b== 0.4**t== 0.6**l== 0.2**r== 0.2**\nFig. 1. (A) Gross lesions of dermatitis of Ht-rats maintained under conventional or SPF conditions at 25\nweeks of age.  (B) Incidence of dermatitis in male Ht-rats after conventionalization.  Rats were\n: Rats maintained under conventional conditions (n=5),\nconventionalized at 5 weeks of age.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.3**r== 0.5**\n: Rats maintained under SPF conditions (n=5).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\ning preferential accumulation of Th2-type cells.  In ad-\ndition, an increase in the expression of IL-4 and of IL-5\nhas been detected in CD4- and CD8-bearing T cells in\npatients  with  AD  [12,  18].    The  Th2-type  cytokines\nhave a critical role in the initiation of the allergic re-\nsponse in Staphylococcus aureus-associated dermatitis\n[10].  IL-4 plays an important role in the induction of\nIgE  production  [13]  and  as  an  inhibitor  of  bacterial\nclearance  [5,  16].    Kasamatsu  et  al.  demonstrated  an\nincreased amount of serum IL-4 in acute AD compared\nwith  the  normal  control  [7].    Furthermore,  up  to  107\ncolony-forming units of S. aureus can be isolated from\nskin  lesions  of  more  than  90%  of  AD  patients  [11],\nwhile only 5% of normal subjects carry S. aureus on\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\ntheir skin.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nIn this study, we analyzed dermatitis in the Ht-rat to\npursue its potential as an animal model for investigat-\ning the pathogenesis of atopic dermatitis (AD) and to\ndevelop new therapeutic approaches or drugs to treat\nAD.  We discuss the characteristics of Ht-rats in com-\nparison  with  Nh-mice,  which  have  recently  been\nproposed as an animal model for human AD [3, 4, 19].\nHt-rats kept under a conventional condition until 25\nweeks of age displayed prominent symptoms of derma-\ntitis  including  edema  and  erythema  at  the  neck  (Fig.\n1A).  Dermatitis was observed in all Ht-rats that were\nmaintained under conventional conditions for 20 weeks,\nbut not in those maintained under SPF conditions (Fig.\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nTable 1. Comparison of the % frequency of detectable S. aureus and the number of mast cells, eosino-\nphils, and CD4- or CD8-bearing cells in the skin lesions from hairless rats and mice maintained\nunder SPF or conventional conditions for 24 weeks\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nNh-mouse (Nh/+)b\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nHt-rat (Ht/+)\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nSPF\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nConventional\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nSPF\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nConventional\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n% Frequency of S. aureus\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0.0 ± 0.0\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n100.0 ± 0.0\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n0.0 ± 0.0\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n100.0 ± 0.0c\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nNumber of cellsa\nMast cells\nEosinophils\nCD4-bearing cells\nCD8-bearing cells\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n40.0 ± 11.6\n0.0 ± 0.0\n16.0 ± 10.1\n5.4 ± 8.2\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n101.7 ± 28.2c\n7.2 ± 4.2\n81.0 ± 68.6c\n23.5 ± 15.3\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n5.9 ± 1.8\n3.0 ± 2.6\n0.0 ± 0.0\n0.0 ± 0.0\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n16.1 ± 3.8c\n35.6 ± 8.1c\n54.1 ± 37.1c\n27.8 ± 26.1\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.2**r== 0.2**\na: Number of cells per mm2 of skin area.  b: With reference to previous reports [4] and [19].  c: Significant\nincrease (P<0.05) in comparison with SPF.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\n1B).  The rats could develop dermatitis only under con-\nventional conditions.  To evaluate the involvement of\nbacteria in the formation of the lesions, bacterial cul-\ntures were obtained from the facial skin surface of each\nHt-rat  kept  under  conventional  or  SPF  conditions.\nSamples  were  inoculated  onto  a  salt,  egg  yolk,  agar\nplate  (Nissui,  Tokyo,  Japan),  and  incubated  at  37°C.\nTen colonies per rat were randomly picked up, and the\nsub- species of Staphylococci were identified with an\nAN-ID Test-SP18 kit (Nissui, Tokyo, Japan).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nThe animals were humanely euthanized in accordance\nwith  the  guidelines  for  animal  experimentation  at\nShionogi  Research  Laboratories.    Skin  samples  were\nfixed in formalin or frozen in a freezer.  Four-microme-\nter thick paraffin sections of skin lesions of the Ht-rat\nwith or without dermatitis were stained with hematoxy-\nlin-eosin (HE) for histopathological observation, acidic\ntoluidine  blue  to  detect  mast  cells,  and  by  the  Luna\nmethod for eosinophils.  Frozen sections of the Ht-rat\nskin  with  or  without  dermatitis  were  immunostained\nusing  mouse  anti-rat  CD4  (OX-35,  PharMingen,  San\nDiego,  CA,  USA)  and  mouse  anti-rat  CD8a  (OX-8,\nPharMingen) antibodies.  Positive reactions on the sec-\ntions were visualized with peroxidase-diaminobenzidine,\nand the number of positive cells was counted.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nStaphylococcus  aureus  was  not  isolated  from  25-\nweek-old Ht-rats maintained under the SPF condition\n(Table 1), and approximately 70% of the isolated bac-\nteria  from  the  animals  housed  conventionally  were\ncommon  Staphylococcus  species  (S.  cohniiB  and  S.\nxylosus); the rest could not be identified.  These bacte-\nrial  strains  were  completely  replaced  by  S.  aureus  in\n**BLOCK**fs== 10.0**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nconventionally  housed  Ht-rats  with  dermatitis  (Table\n1).  In view of these results, S. aureus appears to play\nan important role in the development of dermatitis in\nHt-rats.  Significant histopathological changes observed\nin the skin lesion of all Ht-rats included hyperkeratosis,\nacanthosis, slight intracellular edema of the epidermis,\nslight swelling of the epidermal cells, and the presence\nof inflammatory cells (Fig. 2A).  Higher levels of eosi-\nnophils  and  whole  and  degranulated  mast  cells  were\nobserved in the skin lesions of Ht-rats with dermatitis\nas compared with skin samples from the rats without\ndermatitis (Table 1, Figs. 2Ac and 2Ad).  In rats main-\ntained under the conventional condition, the number of\nCD4-bearing  T  cells  increased,  while  CD8-bearing  T\ncells  appeared  to  show  a  slight  increase  in  number.\nThe  ratio  of  CD4  to  CD8  cells  was  constantly  high.\nBoth CD4- and CD8-bearing T cells were rarely found\nin Ht-rats maintained under the SPF condition (Table\n1, Figs. 2Ae and 2Af).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nWe measured the serum IgE and IL-4 levels, since\nIgE and IL-4 play important roles in the pathogenesis\nof human AD [2, 12].  Serum samples from rats main-\ntained under the SPF or conventional conditions were\ncollected.  Total IgE and IL-4 levels in the serum were\nmeasured using sandwich ELISA kits (Bethyl, Mont-\ngomery,  TX,  USA;  and  Biosource,  Camarillo,  CA,\nUSA).  The level of serum IgE increased only in the\nHt-rats  that  were  maintained  under  the  conventional\ncondition, but not in those maintained under the SPF\nconditions.  Serum IL-4 was detected only in Ht-rats\nwith dermatitis (Fig. 2B).  Serum IFN-γ was not detect-\nable in any of the rats (data not shown).  Considering\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.2**r== 0.2**\nFig. 2. (A) Histological and immunohistochemical features of the skin lesion of a conventionally housed\nHt-rat at 25 weeks of age.  Paraffin sections stained with HE, of SPF (a) and conventionally (b)\nhoused rats at 24 weeks of age.  Paraffin sections of the skin of an Ht-rat at 24 weeks of age with\ndermatitis stained with acidic TB (c) or by the Luna method (d).  A frozen section immunostained\nfor rat CD4 (e) and rat CD8a (f).  Arrows indicate mast cells, eosinophils, and T cells, respectively.\n(B) Serum IgE and IL-4 levels in each group of Ht-rats.  ND: not detected.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nthe results, we propose that the Ht-rat can serve as a\nnew animal model for human AD.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nThe present rat model was compared with the DS-Nh\nmouse, which was recently introduced by Hikita et al.\n[4] and Yoshioka et al. [19].  The rat and mouse mod-\nels  may  be  similar  to  human  AD  with  respect  to  the\nfollowing features [4, 8, 11, 14, 15]: (1) S. aureus can\nbe isolated from skin lesions, (2) serum levels of IgE\nand IL-4 increase significantly, and (3) the number of\nwhole mast cells and CD4-bearing T cells significantly\nincrease.  Although both rodent models have much in\ncommon, there are some differences.  We found that all\nthe  S.  aureus  from  the  DS-Nh  mice  with  dermatitis\nsecrete  staphylococcal  enterotoxin  (SE)  C,  while  S.\naureus from the Ht-rats did not secrete superantigens\n(SEA, SEB, SEC, SED, SEE, and toxic shock syndrome\ntoxin)  (data  not  shown).    In  addition,  the  number  of\neosinophils significantly increased only in rats that were\nmaintained  under  the  conventional  condition,  but  no\nincrease was observed in the mice.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nAs described above, there are some genetic similari-\nties  between  Ht  and  Nh  mutations.    The  inheritance\nmode of both mutations is autosomal dominant [1, 3].\nThe Nh locus is mapped on mouse chromosome 11 and\nthe Ht locus on rat chromosome 10 [1, 3], and both the\ngenes  are  located  in  contiguity  with  the  linkage  site\n(unpublished data).  Furthermore, there are some phe-\nnotypical  similarities  between  both  mutated  rodents.\nHigher levels of mast cells were observed in the skin\nlesions  of  Ht-  or  Nh-mutated  rodents  compared  with\nskin samples of age-matched rodents without these mu-\ntations (data not shown).  In Ht-rats and Nh-mice, we\nconsidered  that  the  increasing  number  of  mast  cells\ncaused by these mutations and S. aureus colonization\nmay play an important role in developing AD-like der-\nmatitis.    We  are  currently  confirming  whether  these\ngenes are the same by using the positional cloning tech-\nnique.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nIn conclusion, Ht-rats develop dermatitis similar to\nthat in DS-Nh mice and should be useful as an addi-\ntional model for human atopic dermatitis.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nReferences\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n1. Akimoto, T., Suzuki, H., Nakama, K., and Suzuki, K. 2000.\n**BLOCK**fs== 8.7**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nExp. Anim. 49: 137–140.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n2. Bunikowski,  R.,  Mielke,  M.,  Skarabis,  H.,  Herz,  U.,\nBergmann, R.L., Wahn, U., and Renz, H. 1999. J. Allergy.\nClin. Immunol. 103: 119–124.\n**BLOCK**fs== 8.7**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n3. Watanabe, A., Takeuchi, M., Nagata, M., Nakamura, K.,\nNakao,  H.,  Yamashita,  H.,  Makino,  S.,  Harada,  M.,  and\nHirasawa, T. 2003. Exp. Anim. 52: 419–423.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n4. Hikita,  I.,  Yoshioka,  T.,  Mizoguchi,  T.,  Tsukahara,  K.,\nTsuru,  K.,  Nagai,  H.,  Hirasawa,  T.,  Tsuruta,  Y.,  Suzuki,\nR., Ichihashi, M., and Horikawa, T. 2002. J. Dermatol. Sci.\n30: 142–153.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n5. Hultgren,  O.,  Kopf,  M.,  and  Tarkowski,  A.  1998.  J.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nImmunol. 160: 5082–5087.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n6. Jose, S.M. and Samson, H.A. 1993. Clin. Rev. Allergy 11:\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n471–490.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n7. Kasamatsu,  M.,  Tsuji,  T.,  and  Miura,  M.  1993.  Arerugi.\n**BLOCK**fs== 8.7**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n42: 878–882.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n8. Laberge, S., Ghaffar, O., Boguniewicz, M., Center, D.M.,\nLeung, D.Y., and Hamid, Q. 1998. J. Allergy Clin. Immunol.\n102: 645–650.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n9. Leung,  D.Y.  2000.  J.  Allergy  Clin.  Immunol.  105:  860–\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n876.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n10. Leung, D.Y. 2003. J. Invest. Dermatol. 302: 180–192.\n11. Leyden, J.J., Marples, R.R., and Kligman, A.M. 1974. Br.\n**BLOCK**fs== 8.7**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nJ. Dermatol. 90: 525–530.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n12. Nakazawa,  M.,  Sugi,  N.,  Kawaguchi,  H.,  Ishii,  N.,\nNakajima,  H.,  and  Minami,  M.  1997.  J.  Allergy  Clin.\nImmunol. 99: 673–682.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n13. Pene,  J.,  Rousset,  F.,  Briere,  F.,  Chretien,  I.,  Bonnefoy,\nJ.Y.,  Spits,  H.,  Yokota,  T.,  Arai,  N.,  Arai,  K.,  and\nBanchereau, J. 1988. Proc. Natl. Acad. Sci. USA 85: 6880–\n6884.\n**BLOCK**fs== 8.7**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n14. Roosje, P.J., Dean, G.A., Willemse, T., Rutten, V.P., and\n**BLOCK**fs== 8.7**b== 0.3**t== 0.6**l== 0.5**r== 0.2**\nThepen, T. 2002. Vet. Pathol. 39: 228–233.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n15. Sampson, H.A. 1990. Clin. Exp. Allergy 20: 459–467.\n16. Cho, S.H., Strickland, I., Tomkinson, A., Fehringer, A.P.,\nGelfand, E.W., and Leung, D.Y. 2001. J. Invest. Dermatol.\n116: 658–663.\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n17. Tani,  S.,  Noguchi,  M.,  Hosoda,  Y.,  Sugibayasi,  K.,  and\n**BLOCK**fs== 8.7**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nMorimoto, Y. 1998. Exp. Anim. 47: 253–256.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n18. Yamamoto, S., Hamasaki, Y., Ishii, E., Ichimaru, T., and\nMiyazaki,  S.  1997.  Ann.  Allergy  Asthma.  Immunol.  78:\n517–523.\n**BLOCK**fs== 8.7**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n19. Yoshioka,  T.,  Hikita,  I.,  Matsutani,  T.,  Yoshida,  R.,\nAsakawa, M., Toyosaki-Maeda, T., Hirasawa, T., Suzuki,\nR.,  Arimura,  A.,  and  Horikawa,  T.  2003.  Immunol.  108:\n562–569.",
         "extracted",
         "Role of the Nh (Non-hair) mutation in the development of dermatitis and hyperproduction of IgE in DS-Nh mice.;DS–Nh as an experimental model of atopic dermatitis induced by Staphylococcus aureus producing staphylococcal enterotoxin C;Characterization of dermatitis arising spontaneously in DS-Nh mice maintained under conventional conditions: another possible model for atopic dermatitis.;Atopic dermatitis: new insights and opportunities for therapeutic intervention.;Locus of dominant hairless gene (Ht) causing abnormal hair and keratinization maps to rat chromosome 10.;Characteristics of spontaneous erythema appeared in hairless rats.;IgE production by normal human lymphocytes is induced by interleukin 4 and suppressed by interferons gamma and alpha and prostaglandin E2.",
         "WBN/Kob-Ht rats spontaneously develop dermatitis under conventional conditions: another possible model for atopic dermatitis.",
         "None"
        ],
        [
         "34",
         "002bc0bcbf3b5f2dc32d0f9690afddeeadebe2f3",
         "None",
         "I. Krylova,E. Sablin,Jamie M. R. Moore,R. Xu,G. Waitt,J. A. MacKay,D. Juzumiene,J. Bynum,K. Madauss,V. Montana,L. Lebedeva,M. Suzawa,Jon D. Williams,Shawn P. Williams,R. Guy,J. W. Thornton,R. Fletterick,T. Willson,H. Ingraham",
         "http://www.cell.com/article/S009286740500098X/pdf",
         "\n**BLOCK**fs== 6.6**b== 0.9**t== 0.1**l== 0.1**r== 0.3**\nCell, Vol. 120, 343–355, February 11, 2005, Copyright ©2005 by Elsevier Inc. DOI 10.1016/j.cell.2005.01.024\n**BLOCK**fs== 17.9**b== 0.8**t== 0.1**l== 0.1**r== 0.3**\nStructural Analyses Reveal Phosphatidyl\nInositols as Ligands for the NR5 Orphan\nReceptors SF-1 and LRH-1\n**BLOCK**fs== 8.5**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nIrina N. Krylova,1,7 Elena P. Sablin,2,7 Jamie Moore,3\nRobert X. Xu,5 Gregory M. Waitt,5\nJ. Andrew MacKay,4 Dalia Juzumiene,5\nJane M. Bynum,5 Kevin Madauss,5\nValerie Montana,5 Lioudmila Lebedeva,1\nMiyuki Suzawa,1 Jon D. Williams,5\nShawn P. Williams,5 Rodney K. Guy,3\nJoseph W. Thornton,6 Robert J. Fletterick,2\nTimothy M. Willson,5 and Holly A. Ingraham1,*\n1Department of Physiology\n2 Department of Biochemistry and Biophysics\n3 Department of Pharmaceutical Chemistry\n4 Graduate Group in Bioengineering\n1550 4th Street\nMission Bay Campus\nUniversity of California\nSan Francisco, California 94143\n5 Discovery Research\nGlaxoSmithKline\n5 Moore Drive\nResearch Triangle Park, North Carolina 27709\n6 Center for Ecology and Evolutionary Biology\n321 Pacific Hall\nUniversity of Oregon\nEugene, Oregon 97403\n**BLOCK**fs== 8.5**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nSummary\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nVertebrate members of the nuclear receptor NR5A sub-\nfamily, which includes steroidogenic factor 1 (SF-1) and\nliver receptor homolog 1 (LRH-1), regulate crucial as-\npects of development, endocrine homeostasis, and\nmetabolism. Mouse LRH-1 is believed to be a ligand-\nindependent transcription factor with a large and\nempty hydrophobic pocket. Here we present struc-\ntural and biochemical data for three other NR5A mem-\nbers—mouse and human SF-1 and human LRH-1—\nwhich reveal that these receptors bind phosphatidyl\ninositol second messengers and that ligand binding\nis required for maximal activity. Evolutionary analysis\nof structure-function relationships across the SF-1/\nLRH-1 subfamily indicates that ligand binding is the\nancestral state of NR5A receptors and was uniquely\ndiminished or altered in the rodent LRH-1 lineage. We\npropose that phospholipids regulate gene expression\nby directly binding to NR5A nuclear receptors.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nIntroduction\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nUnlike most nuclear receptors, which are regulated by\nsteroid, retinoid, and other nonpolar ligands, NR sub-\nfamily 5A are constitutively active in many cell types,\nand obvious ligand candidates have not been forth-\ncoming. SF-1 (NR5A1) orchestrates endocrine tissue\ndevelopment and male sexual differentiation and con-\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.1**r== 0.6**\n*Correspondence: hollyi@itsa.ucsf.edu\n7 These authors contributed equally to this work.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.2**l== 0.5**r== 0.1**\ntrols steroid biosynthesis (Ingraham et al., 1994; Luo et\nal., 1994). LRH-1 (NR5A2) is essential in early develop-\nment and regulates bile acid synthesis, cholesterol\ntransport, and ovarian function (Fayard et al., 2004; Lu\net al., 2000). This subfamily is distinct in that members\nare able to bind as monomers to DNA, and they pos-\nsess an unusually large hinge domain that is subject to\nposttranslational modifications, including phosphoryla-\ntion and sumoylation (Desclozeaux et al., 2002; Lee et\nal., 2005). Although physiological roles for these recep-\ntors have emerged, ligand candidates have remained\ncontroversial. The proposed SF-1 ligand 25-hydro-\nxycholesterol, while appealing because of the recep-\ntor's regulation of steroid synthesis, appears to be in-\ncorrect because oxysterols fail to activate SF-1 in most\ncell lines (Christenson et al., 1998; Lala et al., 1997; Mel-\nlon and Bair, 1998). Further, the notion that ligands\nmight be dispensable for this subfamily of receptors\nwas bolstered by the crystal structure of mouse LRH-1\nligand binding domain (LBD), showing an active confor-\nmation in the absence of either ligand or coregulator\npeptide. Unlike other orphan nuclear receptors, which\neither lack a ligand binding pocket (e.g., Nurr1, DHR38,\nor ERRα) or have a constitutively bound lipid (e.g.,\nHNF4α, HNF4γ, or USP), mLRH-1 was found to possess\na large unoccupied hydrophobic pocket of w830 Å (re-\nviewed in Li et al. [2003]). Remarkably, mutations that\nfilled the mLRH-1 pocket failed to reduce transcrip-\ntional activity (Sablin et al., 2003).\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nWhereas an elegant structural paradigm has devel-\noped to explain the activation of ligand-regulated re-\nceptors, no uniform mechanism has been proposed to\naccount for the modulation of orphan nuclear receptor\nactivity (Steinmetz et al., 2001; Weatherman et al.,\n1999). The presence of fortuitous lipid molecules re-\nvealed in crystallization experiments can be associated\nwith an inactive LBD conformation as noted for the\ndipterian USP LBD (Billas et al., 2001). On the other\nhand, lipid components have been proposed to stabi-\nlize an active LBD conformation in constitutively active\nnuclear receptors, with HNF4 binding fatty acids (Dhe-\nPaganon et al., 2002; Wisely et al., 2002) and RORα\nbinding cholesterol (Kallen et al., 2002). In this context,\nthe structure of the murine LRH-1 with its empty ligand\nbinding pocket remains a puzzle. To further explore\nhow NR5A orphan receptors are regulated, we obtained\nthe LBD crystal structures of other members of this\nsubfamily, including mSF-1, hSF-1, and hLRH-1. Here\nwe provide structural and biochemical data supporting\nligand regulation for NR5A orphan nuclear receptors.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nResults and Discussion\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nNR5A Structures Reveal a Phospholipid\nin the Ligand Binding Pocket\nThe structure of mSF-1 was determined by the molecu-\nlar replacement method using the atomic coordinates\nfrom Protein Data Bank entry 1PK5 for the mLRH-1 LBD\ncoordinates (Sablin et al., 2003). Two hSF-1 structures\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nTable 1. Data Collection and Refinement Statistics\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nCrystallization\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nmSF-1/mSHP-1\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nhSF-1/No Pep.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nhSF-1/TIF-2\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nhLRH-1/TIF-2\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nUnit Cell Dimensions\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\na (Å)\nb (Å)\nc (Å)\nSpace group\nMolecules per asymmetric unit\nResolution (Å)\nNumber of unique reflections\nData redundancy\nCompleteness (%)\nRsymm (%)\n< I/s(I) >\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nRefinement\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nR\nRfree\nRms deviation from ideality\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nBond length (Å)\nBond angle (0)\nAverage B factor (A2)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nAll atoms\nProtein atoms\nWater molecules\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\n73.9\n73.9\n117.0\nP41212\n\n25–1.2\n100,431\n13.1\n99.9\n6.1\n39.2\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n19.7\n21.1\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n0.012\n1.483\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\n22.8\n21.3\n25.7\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n128.4\n66.0\n141.0\nP21\n\n20–2.5\n71,698\n\n97.9\n4.5\n28.0\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n24.9\n28.3\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n0.009\n1.20\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n73.1\n73.1\n139.4\nP41212\n\n20–2.9\n15,614\n\n99.8\n5.8\n30.0\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n24.6\n29.0\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n0.011\n1.34\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.7**r== 0.2**\n59.9\n67.2\n79.6\nP212121\n\n40–2.5\n10,787\n\n91.9\n3.7\n44.0\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n23.2\n28.3\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n0.008\n1.349\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n64.4\n63.9\n60.5\n**BLOCK**fs== 8.5**b== 0.2**t== 0.5**l== 0.1**r== 0.6**\nwere obtained, with and without TIF-2 peptide, and\nwere determined by MAD and SAD methods using sele-\nnomethionine substituted proteins, respectively (Mad-\nauss et al., 2004). The structure of hLRH-1 was det-\nermined by molecular replacement using the atomic\ncoordinates for hSF-1. The current best structures are\nrefined to 1.2 Å, 2.5 Å, and 2.5 Å for mSF-1, hLRH-1, and\nhSF-1, respectively. Relevant features of data collection\nand refinement statistics for each protein crystal are\nprovided in Table 1. All three LBDs exhibit a character-\nistic protein fold noted for other LBDs, with 12 α heli-\nces, two β strands, and helix H12 positioned in an\nactive conformation (Figure 1). A fourth layer is pro-\nvided by a structured helix H2, as previously shown for\nmLRH-1, although the position and length of this helix\nare the most varied features as judged by comparison\nof mSF-1 with hSF-1 LBD (Figure 1C) and mLRH-1 with\nhLRH-1 (Figure 1D). Upon refinement of these three\nLBD structures, additional electron density was present\nin the ligand binding pockets consistent with the pres-\nence of a phospholipid; the presence of ligand con-\ntrasts the reported structure of mLRH-1 showing no\ndensity inside or at the opening of the pocket.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.6**\nThe nature of this phospholipid was further charac-\nterized by nondenaturing mass spectrometry of LBD\nproteins used for crystallization. Spectra were consis-\ntent with two principle series of ions consisting of both\nligand bound and apo-proteins. Comparison of all four\nLBD protein solutions showed the presence of phos-\npholipid in SF-1 and hLRH-1 but with a much lower\nconcentration in mLRH-1 where only w10% is bound\nby ligand (Figures 2A and 2B and Supplemental Figure\nS1 available with this article online). Quantitation of\nhLRH-1 and mSF-1 bound by phospholipid showed a\n1:1 ligand/receptor complex with nearly all of the pro-\ntein bound by a phospholipid. The lower levels of phos-\npholipid bound to bacterially expressed mLRH-1 LBD\nare consistent with the absence of electron density in\n**BLOCK**fs== 8.5**b== 0.0**t== 0.5**l== 0.5**r== 0.2**\nthe pocket of the mLRH-1 LBD structure. Of possible\nligands ranging from 680–780 Da, the most prominent\nspecies were found to be 747 and 721 Da, consistent\nwith two common bacterial phospholipids, phosphati-\ndyl glycerol (PG) and phosphatidyl ethanolamine (PE).\nDenaturing negative electrospray ionization assigned the\nprominent ligand species as PG (Figure 2C); confirmation\nof this assignment was provided by the atomic crystal\nstructure of mSF-1, where the head group of the phos-\npholipid was determined to be glycerol rather than eth-\nanolamine (Figure 2D). Mutating Ala270 to Trp in helix\nH3 of mSF-1, which would fill the pocket and disfavor\nligand binding, eliminates all detectable PG ligand from\nthe pocket by mass spectrometry (A270W, Figure 2B).\nElectron densities maps of mSF-1 or hSF-1 (± TIF pep-\ntide) showed nearly identical conformation of a phospho-\nlipid containing two fatty acid chains (C16 and C18) occu-\npying nearly the entire hydrophobic pocket; only\nhydrophobic interactions are observed between the\nC16 and C18 chains and surrounding residues (Figures\n2D, 2E, and Supplemental Table S1). Atomic resolution\nstructure of mSF-1 LBD confirmed the presence of\ndouble bonds at the C9 position in both chains within\nthe bound phospholipid. The large pocket sizes of hSF-1\nand hLRH-1 are similar to that reported for the mLRH-1\nLBD, easily accommodating two tails of the phospho-\nlipid. In hLRH-1, the ligand adopts a similar conforma-\ntion or path as found for SF-1; however, the density is\nless ordered and the lower resolution prevented abso-\nlute assignment of the head group (data not shown).\nNonetheless, we find that all phosphates are positioned\nsimilarly in the mouth of the pocket for mSF-1, hSF-1,\nand hLRH-1. Although a phospholipid ligand is present\nin both USP and NR5A, the consequences of ligand\nbinding differ greatly, with ligand proposed to drive USP\ninto an antagonist conformation, rather than the agonist\nstate observed for NR5A receptors (Billas et al., 2001).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nFigure 1. NR5A LBD Receptors Structures Reveal a Ligand\n(A and B) Two views of mouse SF-1 (mSF-1) LBD protein are shown (red, orange, and brown ribbons) with all twelve α helices and the single\nβ turn indicated in panels. Helix H2 is highlighted in bright red (H2). The corepressor peptide form the N-terminal portion of mSHP is shown\nin the AF2 cleft just adjacent to helix H12 (SHP-1, turquoise ribbon). The presence of a ligand determined to be phosphatidyl glycerol (PG) is\nalso shown in the hydrophobic pocket (blue stick).\n(C) The model for human SF-1 (hSF-1, dark red ribbons) superposed with the mSF-1 (light red) is shown.\n(D) The human LRH-1 LBD (hLRH-1, orange ribbons) superposed with the mouse LRH-1 LBD (yellow ribbons) structure is shown. Helices H2,\nH9, and H12 are indicated in both structures, with H2 highlighted in red for all LBDs. For both hSF-1 and hLRH-1, the TIF-2 peptide bound\nstructures are shown (refer to Table 1), and the phospholipid is indicated (PG, blue stick).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nPhosphatidyl Inositol Phosphates Are Selectively\nBound by SF-1 and hLRH-1\nWhile our structural studies indicate that bacterially ex-\npressed SF-1 and LRH-1 can bind to phospholipids, the\nnature of a bona fide ligand in eukaryotic cells remains\nunknown. We tested the ability of mSF-1 to bind\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\neukaryotic phospholipids, including phosphatidyl inosi-\ntols (PIs), using immobilized lipids on nitrocellulose, as\ndescribed in Experimental Procedures. Of the common\neukaryotic phospholipids, mSF-1 and hLRH-1 selec-\ntively bound the PIP2 and PIP3 phosphatidyl\ninositol\nspecies 2- to 3-fold better than the pocket mutant pro-\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nFigure 2. Binding of Phospholipids by SF-1s and hLRH-1 Is Greater than for mLRH-1 or for the mSF-1 A270W Pocket Mutant\n**BLOCK**fs== 7.0**b== 0.1**t== 0.7**l== 0.1**r== 0.2**\n(A) Nondenaturing mass spectrometry was carried out on all four LBD proteins used for crystallization (mSF-1, hSF-1 and mLRH-1, hLRH-1).\nSpectra are shown for mLRH-1 (left panel) and mSF-1 (right panel), with the apo [A], and the bound [B] species indicated. The molecular\nweight difference between the two peaks is also shown (arrow).\n(B) The concentrations of all bound LBD species, including the mSF-1 A270W pocket mutant, were determined as described and are listed\nfor equal amounts of LBD protein (w30 µM), with the exception of hSF-1, where more protein was used due to the poorer quality of protein\n(50 µM).\n(C) Product ions following collision-induced decomposition using denaturing electrospray tandem mass spectrometry (ESI MS/MS) on m/z\n745 (blue diamond) are shown for hSF-1 and reveal a phosphatidyl glycerol (chemical structure) ligand with two acyl chains of C16 and C18.\n(D) Electron density for mSF-1 is from a 2Fo−Fc map contoured at 1.2 sigma that shows the phospholipid ligand flanked by surrounding\nresidues (sticks). Three residues are labeled: Gly342, Tyr437, and Ala270, and the phosphate in the PG ligand is indicated (P).\n(E) Electron density found for both mSF-1 (blue) and hSF-1 ± TIF peptide (white or gray) ligands are superposed showing an identical position\nin the hydrophobic ligand binding pocket.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.6**\ntein (A270W, Figure 3A), with signals being especially\nstrong for both PI(3,5)P2 and PI(3,4,5)P3. Both wild-type\nand mutant mSF-1 protein bound equally to PA or the\nsingle phosphorylated PIPs, suggesting that these neg-\natively charged lipids bound to the solvent-exposed\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.2**\nsurface of the LBD. More importantly, equal amounts\nof incubated apo human PPARγ-LBD displayed much\nweaker binding with a different profile (Figure 3A, right\npanel), and no binding was detected with liganded\nhTRβ LBD (data not shown). The capacity to bind such\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nFigure 3. Selective Binding to Phosphorylated Phosphatidyl Inositols by mSF-1 and Loss of Activity with mSF-1 Pocket Mutant\n(A) Binding patterns to immobilized phospholipids for mSF-1, the mSF-1 A270W pocket mutant, hLRH-1, and a control protein apo-hPPARγ\nare shown with the identity of each phospholipid indicated. Levels of all four proteins incubated were equivalent (data not shown). Quantifica-\ntion of binding by wild-type mutant mSF-1 LBD protein (A270W) showed selective binding to all PIP2 and PIP3 species by 1.85- to 3.1-fold.\nAbbreviations are as follows: lysophosphatidic acid (LPA), lysophosphocholine (LPC), sphingosine-1-phosphate (SIP), phosphatidic acid (PA),\nphosphatidyl choline (PC), serine (PS), ethanolamine (PE), phosphatidyl inositol (PI), phosphatidyl (3), (4), or (5) inositiol monophosphate (P3I),\n(P4I), or (P5I), phosphatidyl (3,4), (3,5), or (4,5) inositol bisphosphate (P3,4I), (P3,5I) (P4,5I), and finally phosphatidyl (3,4,5) inositol triphosphate\n(PIP3). Chemical structures are shown for PA and P3,4,5I.\n(B) Activities of pocket mutants made in comparable positions in helix H3 for mSF-1 (A270W mSF-1) are shown following transfections in\nHepG2 human cells with two luciferase reporters, the aromatase, and mouse SHP promoters. Relative luciferase activities are shown as fold\nactivation for each receptor at increasing concentrations (ng per 24 well plate), as indicated on the X axis.\n(C) The luciferase activity is shown for the corresponding helix H3 A368W pocket mutant for mLRH-1 on the aromatase promoter in HepG2\ncells.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nligands appears critical for mSF-1 activity as evidenced\nby the diminished activity exhibited by A270W mSF-1\npocket mutant (Figure 3C, refer back to Figure 2B). This\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nsame mutation when made in mLRH-1 (A368W, H3)\ncaused little or no loss of activity in cellular assays, as\npreviously observed (Figure 3C and Sablin et al. [2003]).\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nFigure 4. Selective Binding of Phosphatidyl Inositols to mSF-1\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.2**\nIn the upper left corner, a flow diagram is shown for ligand analyses of mSF-1 LBD protein following incubation with different liposomes, as\ndescribed in Experimental Procedures. Mass spectra profiles (ESI-LC/MS) are shown for the basal state (No Liposomes, panel A), and after\nincubations with the neutral liposomes of phosphatidyl choline (PC, panel B), with phosphatidic acid liposomes (PC:PA [9:1], panel C), and\nwith phosphatidyl inositol liposomes PC:PI(3,4)P2 [9:1] (PIP2, panel D) or PC:PI(3,4,5)P3 [9:1] (PIP3, panel E). In (B), (D), and (E), the small\npeaks at mass 28730 Da are PC liganded receptor (PC), and in (C), the peak at mass 28646 is PA liganded receptor (PA). Direct mass\nspectrometry analysis (without gel filtration) on the PC:PA-receptor sample revealed a strong peak of free phosphatidic acid (data not shown),\nshowing that ample PA was present in the PC:PA liposomes.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.6**l== 0.1**r== 0.6**\nThe binding of phospholipids was further explored\nby mass spectrometry analysis of purified LBD proteins\nafter incubation with liposomes containing phosphati-\ndyl choline (PC), mixed with either phosphatidic acid\n(PA), PI(3,4)P2, or PI(3,4,5)P3 in a 9:1 ratio. Remarkably,\nincubation of the mSF-1 LBD with the neutral PC lipo-\nsome showed an almost complete loss of the bacterial\nPG phospholipid with very little exchange or binding by\nPC; similar results were found with the PC:PA liposome\n(Figures 4B and 4C) with both profiles differing greatly\nfrom spectra observed with the PG bound state of the\nmSF-1 LBD (Figure 4A, No Liposomes). In contrast, effi-\ncient binding was observed for PI(3,4)P2 or PI(3,4,5)P3\nas evidenced by the increase mass of 970 and 1050 Da,\nrespectively, with only a small uptake of PC observed\n(Figures 4D and 4E). A similar experiment carried out\nwith mLRH-1 LBD revealed less selectivity compared\nto that of mSF-1, with equal exchange of PC, PI(3,4)P2,\nor PI(3,4,5)P3, as well as some liganded PA detected\n(see Supplemental Figure S2). These data taken to-\ngether with the PIP strip binding data suggest that SF-\n1s and hLRH-1 bind much more robustly and selec-\ntively to phosphorylated PIs than does mLRH-1. The\nstructural basis for this species difference in selective\nphospholipid binding is discussed below.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.6**\nRegulation of NR5 Receptors Activity\nNR5A receptors are coexpressed with and repressed\nby the orphan nuclear receptors, Dax-1 and SHP (Fig-\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.5**r== 0.2**\nure 5A). These orphan nuclear receptors contain a pu-\ntative LBD but no DBD; their capacity to repress SF-1,\nLRH-1, and steroid hormone receptors is mediated by\nregions within the very N- and C-terminal LBD (Holter\net al., 2002; Ito et al., 1997; Nachtigal et al., 1998). Un-\nlike NR4 orphan receptors (Nurr1, DHR38) that have\nlost their coactivator cleft (Baker et al., 2003; Wang et\nal., 2003), all four LBD structures of SF-1 and LRH-1\nexhibit a well-formed docking site for coregulators.\nPeptide binding assays were carried out\nfor both\nmLRH-1 and mSF-1 LBD proteins using C-terminal flu-\norescently tagged SHP and Dax-1 peptides and fluo-\nrescence anisotropy, as described (Moore et al., 2004).\nSelective high-affinity binding was found for both\nmLRH-1 and mSF-1, with SF-1 binding more tightly to\npeptides from Dax-1; only the N-terminal LXXLL motif\nin the Dax-1 LBD region is able to bind both mSF-1 and\nmLRH-1 (Figures 5A and 5B).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.2**\nAs observed with NR/coactivator complexes, bound\nSHP-1 peptide in the coactivator cleft is positioned by\nhydrophobic contacts between the peptide leucines\nand LBD surface residues, and by hydrogen bonds be-\ntween the backbone carbonyl and the conserved argi-\nnine (mArg282, hArg281), and the backbone amide with\nthe conserved glutamate (mGlu455, hGlu454) found in\nhelices H4 and H12, respectively (Figure 5C, left panel).\nFurther, superposition of the mSF-1/SHP-1 structure\nwith the apo-structure of mLRH-1 shows little change\nin the positions of helices H3, H4, and H12 contacting\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.7**r== 0.1**\n(A) Schematics of SHP and Dax-1 are shown\nwith colored boxes corresponding to LXXLL\npeptides used in the fluorescence anisot-\nropy assay, as depicted and listed in the\ntable.\n(B) Relative binding affinities for labeled pep-\ntides are indicated by color-coded affinities\n(Kd values) ranging from 0.5–10.0, 10.1–30.0,\nand >30.0 µM and were determined as de-\nscribed in Experimental Procedures. The\nmSHP-1 peptide used in crystallization of\nmSF-1 is RPTILYALLSPSPR (mSHP-1).\n(C) Ribbon depiction of the mSHP-1 peptide\nin the coactivator cleft of mSF-1 (orange rib-\nbon) or superposed with the mLRH-1 struc-\nture (beige ribbon). Residues are numbered\naccording to the database entry Q62227 for\nmSHP.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nthe peptide (Figure 5C, right panel). Similarly, confor-\nmations of LBDs and ligands were found to be nearly\nidentical when hSF-1 was crystallized with or without\nTIF-2 (Figures 1C and 2E). Taken together, these data\nsuggest that selectivity of corepressor (Dax-1 and SHP)\nor coactivator binding to NR5A receptors is mediated\nby flanking amino acids. Moreover, NR5A receptors\nadopt a seemingly “active conformation” at all times,\nas opposed to classic ligand-dependent\nreceptors\n(Darimont et al., 1998). Activity levels achieved by this\nclass of receptor would then depend on ligand, post-\ntranslational modifications, and levels of coregulators.\nThus, simply increasing corepressor levels, such as\nDax-1, may override ligand-dependent activation and\nshift the receptor into an inactive state. This notion is\npartially supported by genetic data showing dosage of\nDax-1 to be critical in mammalian sexual development,\nwhere SF-1 regulates multiple genes (reviewed in Lud-\nbrook and Harley, 2004), and by the fact that SF-1 and\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nLRH-1 regulate Dax-1 and SHP in a negative feedback\nregulatory mechanism (Lu et al., 2000).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.7**l== 0.5**r== 0.1**\nCompared with LRH-1, SF-1 exhibits a shorter helix\nH2 and a more flexible region connecting helices H2\nand H3 (Figure 6A). The mutation R314M, predicted to\ndisrupt a 2.7 Å salt bridge between Arg314 (H5) and\nGlu238 (H2) in mSF-1 but not to interfere with ligand\nbinding, sharply diminished receptor activity (Figure\n6B), suggesting that the integrity of the NR5 receptors\nis mediated in part by packing interaction between he-\nlix H2 and other helices in the body of receptor. Inter-\nestingly, this same mutation when made in mLRH-1\n(R412M) showed a much more modest decrease than\nobserved with the analogous SF-1 R314M mutant—in\nmLRH-1 this interaction with corresponding shorter\nside chain of Asp236 is predicted to be less important,\nsuggesting that additional interactions at the more ex-\ntended H2-H3 interface in LRH-1 help to stabilize the\nreceptor structure.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nFigure 6. The Role of H2/H3 Structural Elements in Maintaining an\nActive Conformation of mLRH-1 and mSF-1\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nIn the top panel, ribbon diagrams of partial structures of mLRH-1\nand mSF-1 are shown. Helices H2 (red) and H3 (pink) of mSF-1 are\nsuperposed with the helices H2 and H3 of mLRH-1 (light yellow),\nand the phosphatidyl glycerol ligand is shown (blue stick). The H2\nof mSF-1 is approximately half the length of helix H2 in mLRH-1\nH2. The bottom panel shows reporter activity in HepG2 cells with\nincreasing concentrations of wild-type and helix H5 mutants for\nboth mLRH-1 and mSF-1. Relative luciferase activities are shown.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.6**\nSpecies Difference in the Ligand Binding Pocket\nThe structures of NR5A receptors allow us to identify\nspecific elements that facilitate phospholipid binding in\nthe mouse and human SF-1s and human LRH-1 but di-\nminish it in mouse LRH-1. For all receptors, only hy-\ndrophobic interactions are made between the 33 residues\ncontacting the PG phospholipid; residues contacting the\nligand are listed for all four LBDs (Supplemental Table\nS1). Direct comparison of these residues shows con-\nserved changes between SF-1 and LRH-1, with most\ndifferences adding mass to unoccupied space in the\nmLRH-1 ligand-free pocket. However, when comparing\nspecies, a cluster of nonconserved residue changes\nthat span helices H6-H7 is noted between human and\nmouse LRH-1 (Figure 7A); on the other hand, only one\nresidue difference occurs between mouse and human\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.5**r== 0.2**\nSF-1. Further analysis shows residues within this clus-\nter to be crucial for phospholipid binding. Indeed, bind-\ning of this ligand is optimized by coordination of the\nphosphate head group at the base of the pocket, with\nhydrogen bonding provided by Gly342 at the beginning\nof H6, and Tyr437 and Lys441 at the end of H11, which\nare 2.8 Å, 2.6 Å, and 2.8 Å from the ligand, respectively\n(Figure 7B). These interactions effectively seal the bot-\ntom of the ligand binding pocket with the phosphate\ngroup from the ligand contributing to this closure (Fig-\nure 7C).\n**BLOCK**fs== 8.5**b== 0.4**t== 0.3**l== 0.5**r== 0.2**\nA different scenario occurs in mLRH-1, where Glu440\nforms a strong ionic interaction with the opposing\nLys539 in helix H11 (analogous to Lys441 in mSF-1),\nsealing the pocket and stabilizing the mLRH-1 LBD in\nthe absence of the ligand (Figure 7C). The other unique\nresidues spanning helices H6-H7 in mLRH-1 (Thr439\nand Phe443, Figure 7A) are also predicted to minimize\nligand binding by clashing with both the phosphate\ngroup and aliphatic chains, respectively. To test the hy-\npothesis that key residues interacting with the lipid’s\nphosphate group are critical to ligand binding, mLRH-1\nwas “humanized” by an E440G mutation. Wild-type and\nmutant receptors were assayed before and after reduc-\ning cellular pools of phosphatidic acid and subsequent\nphosphatidyl\ninositols with a competitive inhibitor of\nphospholipase D (Fang et al., 2001). N-butanol treat-\nment had a marginal effect on the activity of mLRH-1\nbut substantially reduced activation by the humanized\nE440G mLRH-1, and to a lesser extent wild-type mSF-1\n(Figure 7D). Collectively, these data are consistent with\nthe lowered selectively observed for mLRH-1 phospho-\nlipid binding and are predicted by structures of the\nLRH-1 and SF-1 orthologs.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.6**l== 0.5**r== 0.2**\nRecent Divergence in the Rodent LRH-1\nEvolutionary analysis of structure-function relations in\nthe LRH-1/SF-1 subfamily indicates that ligand binding\nis the ancestral state for these proteins where ligand\nbinding was subsequently reduced or altered in the lin-\neage leading to the rodent LRH-1. We coded the pres-\nence or absence of ligand binding as a phylogenetic\ncharacter and reconstructed its evolution on the phy-\nlogeny of the NR5A gene family (Figure 8). As described\nin Experimental Procedures, both maximum parsimony\nand maximum likelihoods were used to evaluate two\nhypotheses concerning the function of the common an-\ncestor of the entire NR5A subfamily—that the ancestor\nwas liganded (H1) and that it was unliganded (H0). Un-\nder H1, a single loss of ligand binding is required on the\nbranch leading to the rodent LRH-1, whereas H0 re-\nquires ligand binding to have been gained indepen-\ndently in the SF-1 and human LRH-1 lineages. H1 is\ntherefore the most parsimonious hypothesis. A maxi-\nmum likelihood analysis assuming that ligand binding\ncan be gained or lost under a simple reversible Markov\nprocess also supports H1, albeit nonsignificantly, indi-\ncating a loss of ligand binding in rodent LRH-1s (likeli-\nhood ratio = 3.05).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.2**\nTo increase the power of our analysis, we analyzed\nthe evolution of the key sequence sites that have been\nstructurally identified as critical for ligand binding in\nNR5A receptors. Although functional data from structures\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.1**\nFigure 7. Species Differences in the Closure\nof the Ligand Binding Pocket\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.7**r== 0.1**\n(A) Sequence alignment of the region span-\nning helices H6 and H7 is shown for a limited\nnumber of species. Rodent LRH-1-specific\nchanges are highlighted in red.\n(B) Electron density showing coordination of\nthe phosphate group by three residues in\nmSF-1. These include Gly342 (H7) and Tyr437\nand Lys441 (H11) with hydrogen bonds indi-\ncated (dashed line). The phosphate is la-\nbeled (P).\n(C) Interactions between helices H7 and H11\nare depicted for mSF-1, with phospholipid li-\ngand (white stick) and for mLRH-1 without\nligand. The salt bridge between Glu440 and\nLys539 in mLRH-1 is depicted.\n(D) Reporter activity is shown after depletion\nof phosphatidic acid by butanol with wild-type\nmLRH-1, mSF-1, and “humanized” mLRH-1\nE440G mutant transfected into HepG2 cells.\nActivity is expressed as fold activation using\nthe aromatase luciferase reporter.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nare known for only four family members, sequences are\navailable for dozens. We used maximum likelihood\n(Yang et al., 1995) to determine ancestral amino acid\nstates and evolutionary changes in the H6-H7 residues\ncorresponding to Thr439, Glu440, and Phe443 in rodent\nLRH-1 (TEF, Figure 8). These sites were chosen be-\ncause, as discussed above, structural and experimen-\ntal evidence indicates that they exclude or diminish\nphospholipid binding in mLRH-1, but the homologous\nresidues, Ala, Gly, and Leu, confer ligand binding on\nhLRH-1 and the SF-1s (AGL, Figure 8). The NR5A an-\ncestral sequence has all three ligand-associated states\n(AGL), and this reconstruction has extremely high confi-\ndence (posterior probability 100% for each site). The\nderived TEF residues that close the pocket in the ab-\nsence of ligand all evolved much later on than the\nbranch leading to the rodent LRH-1 (posterior prob-\nability 100% for each site). Examination of the genetic\ncode indicates that each of these replacements can be\nconferred by a single base-pair mutation.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nThese findings strongly corroborate the hypothesis\nthat ligand binding was ancestral\nin the SF-1/LRH-1\nfamily and was later lost or radically diminished in ro-\ndent LRH-1s due to a small number of specific amino\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nacid replacements. The lost or altered ligand binding\nstate in rodent LRH-1 is the second example of a taxon-\nspecific nuclear receptor losing the capacity to bind li-\ngand from a liganded ancestor; the other is the estro-\ngen receptor ortholog of the mollusc Aplysia californica\n(Thornton et al., 2003). These findings cast doubt on\nthe hypothesis that ligand binding in the nuclear recep-\ntor superfamily was independently gained numerous\ntimes from an ancestral orphan receptor (Escriva et al.,\n2000). If the loss of ligand dependence is common and\nrequires only a few sequence changes, the opposite\nscenario—repeated loss of ligand binding from a li-\nganded ancestor—may better characterize NR evo-\nlution.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nFinally, the finding that the rodent LRH-1 has di-\nverged from other members of the family is of interest\nin light of differences noted in both bile acid and cho-\nlesterol transport for humans and mice (Dietschy and\nTurley, 2002). While species differences can be attrib-\nuted to the absence of the murine cholesterol transport\nprotein (CETP) (Lie et al., 2002), there are notable differ-\nences in the regulation of LRH-1 targets (Cyp7A1,\nCyp8B, and SHP) involved in cholesterol homeostasis\n(Delerive et al., 2004; Fayard et al., 2004).\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nFigure 8. Amino Acids that Coordinate Ligand in SF-1 Are Ancestral\n**BLOCK**fs== 7.0**b== 0.2**t== 0.6**l== 0.1**r== 0.6**\nThe maximum posterior probability phylogeny of the NR5 gene\nfamily is shown, including LRH-1s (red), SF-1s (blue), and outgroup\nsequences (black, with number of sequences analyzed shown in\nparentheses). Residues homologous to mLRH-1 positions 439, 440,\nand 443 are shown for each LRH-1/SF-1 sequence (aa). Maximum\nlikelihood reconstructions of ancestral amino acid states are shown\nfor SF-1 and LRH-1 proteins in the last common ancestor of verte-\nbrates (squares), of tetrapods (triangles), and of mammals (circles),\nas well as for the common ancestral gene preceding the duplica-\ntion that split the SF-1 and LRH-1 lineages (star). All ancestral re-\nconstructions have 100% posterior probability. Green arrowhead\nshows acquisition of amino acids that exclude ligand in the rodent\nLRH-1 lineage and loss of ligand binding on the same branch. Spe-\ncies names and accession numbers are provided in Supplemental\nTable S2.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.6**\nDo LRH-1/SF-1 Function as Nuclear\nPhospholipid Sensors?\nThat the bacterial phospholipid (PG) adopts an iden-\ntical conformation, filling both hLRH-1 and SF-1 pock-\nets equally well, offers a strong structural argument that\nthis “fortuitous” ligand resembles closely the native li-\ngand. Furthermore, phosphatidyl inositols shown here\nto bind NR5A receptors are predicted to adopt a similar\nconformation. Thus, our collective structural and bio-\nchemical data support phospholipids as ligands for\nboth SF-1 and hLRH-1; this result is unexpected given\nthe role that each plays in cholesterol transport or cho-\nlesterol metabolism in sterol synthesis. This proposal\n**BLOCK**fs== 8.5**b== 0.3**t== 0.2**l== 0.5**r== 0.2**\nimplies that a distinct pool of cellular lipids signal via\nmodulation of the NR5A nuclear receptors. Indeed, the\nliterature suggests that nuclear phosphatidyl\ninositol\nlipids exist as well as enzymes affecting their metabo-\nlism, such as phosphatidyl inositol 3-kinase (PI-3K) (Ir-\nvine, 2002; Ledeen and Wu, 2004; Martelli et al., 2003).\nAdditionally, both PI-3 and PI-5 phosphatases exhibit\nprominent nuclear staining (Bacqueville et al., 2001; De-\nleris et al., 2003), thus providing a mechanism to down-\nregulate PI nuclear signaling. Although we found that\nWortmannin (PI-3 kinase inhibitor) and the PI-3 phos-\nphatase PTEN affected the activities of wild-type and\npocket mutant SF-1 receptors differently (I.N.K., unpub-\nlished data), further studies and additional pharmaco-\nlogical tools are needed to selectively disrupt nuclear\nphospholipid signaling pathways. Nonetheless, the link\nbetween PI-3 kinase signaling and NR5A receptor ac-\ntivity is intriguing given the roles that both SF-1 and\nLRH-1 play in cell proliferation. Indeed, haploinsuffi-\nciency of both sf-1 and lrh-1 leads to decreased cell\nproliferation (Bland et al., 2000; Botrugno et al., 2004),\nwhile increased gene dosage of SF-1 is reported to be\nassociated with human adrenal cortical tumors (de Fi-\ngueiredo et al., 2004). Because PI-3 kinase pathways\nare known to regulate cell cycle progression and cell\ngrowth, it is tempting to speculate that extracellular\nevents that trigger this pathway might also elevate\nnuclear PIs, and thus modulate SF-1/LRH-1 activities.\nFinally, it is worth considering how phospholipids are\ndelivered to nuclear receptors. Cytoplasmic shuttling of\nPIs to the plasma membrane is carried out by phospha-\ntidyl inositol transport proteins (PITP) in an ATP-inde-\npendent manner; these proteins are also present in the\nnucleus. Crystal structures of phospholipid binding\nproteins show a large hydrophobic pocket or tunnel\nable to bind lipids (Yoder et al., 2001; Hsuan and Cock-\ncroft, 2001; Soccio and Breslow, 2003). Whether PIs are\ndirectly taken up by SF-1/LRH-1 from membranes (due\nto transient receptor shuttling between the cytoplasm\nand nucleus) or are delivered by nuclear phospholipid\nbinding proteins remains unclear. While many ques-\ntions are still to be answered, our current study leads\nus to hypothesize phosphatidyl inositols as bona fide\nligands for NR5A receptors and, as such, provide a di-\nrect link between transcription and phospholipid sig-\nnaling.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nExperimental Procedures\n**BLOCK**fs== 7.0**b== 0.0**t== 0.8**l== 0.5**r== 0.2**\nProtein Purification and Crystallization\nThe mSF-1 LBD (219–462) was engineered by mutating cysteines at\n302, 408, 413, and 423 to prevent oxidation (mSF-1 residue number\nfollows accession NP_620639). Modified HIS6-tagged mSF-1 LBD\nwas expressed and purified by Ni-Trap affinity purification followed\nby cleavage of HIS-tag with recombinant tobacco etch viral prote-\nase, as previously described for mLRH-1 LBD (Sablin et al., 2003).\nHIS6-tagged hSF-1 (222–461) and hLRH-1 (300–542) were ex-\npressed using the pRSETa vector, purified by Ni-NTA chromatogra-\nphy, followed by cleavage of the HIS6-tag using thrombin. Crystalli-\nzation conditions for hSF-1 with and without\nthe coactivator\npeptide TIF-2 are described in detail in Madauss et al. (2004), and\ncrystals were obtained in 0.1 M TRIS (pH 7.8), 0.4 M MgCl2, 13%\nPEG6000, 5 mM DTT. For mSF-1, equal volumes of mSF-1 LBD\nwere added to a 6-fold molar excess of mSHP-1 peptide (RPTILY-\nALLSPSPR) in 14% PEG8K, 0.1 M TRIS (pH 7.5), 5% isopropanol,\nand 15% xylitol to produce in 5 days large (200 µm) crystals that\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\ndiffracted to 1.1–1.2 Å. Data were integrated using DENZO and\nscaled with SCALEPACK. For hLRH-1, the protein and TIF-2 pep-\ntide were mixed in equal volumes with 1.2 M sodium potassium\nphosphate (pH 4.9), 4% (v/v) glycerol, 4% (v/v) ethylene glycol.\nCrystals were grown at 22°C and appeared in one week and\nreached 100–200 m in each dimension over a period of 3 weeks.\nPrior to data collection, hLRH-1 crystals were dipped in cryopro-\ntectant (reservoir solution containing 11.5% glycerol and 11.5%\nethylene glycol) and flash-frozen in liquid nitrogen. The TIF-2 pep-\ntide sequence is KENALLRYLLDKDD and was used for both hLRH-1\nand hSF-1 crystallization. X-ray data were collected at IMCA 17-ID\nline (Argonne, Illinois), and data were processed and scaled with\nHKL2000. The hLRH-1 model was built by molecular replacement\nwith Amore program using hSF-1 as a search model and refined\nwith CNS. In this model, a second TIF-2 peptide is found in the\nasymmetric unit and forms loose contacts with the tail of the first\nTIF-2 peptide positioned in the coactivator cleft formed by H3, H4,\nand H12 of the hLRH-1 receptor. Rebuilding between rounds of\nrefinement was performed with the graphics program QUANTA.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nCell Culture and Transfections\nHuman HepG2 liver carcinoma cells were plated the day of trans-\nfection in 24 well plates and transfected with FUGENE 6 with\namounts of DNA indicated in figures and legends. For all experi-\nments, 200 ng/well of luciferase reporter were used and pCI-Neo-\nβ-gal (50 ng/well) were transfected along with wild-type or mutant\nreceptors. Luciferase activity was measured after 48 hr, as pre-\nviously described (Sablin et al., 2003). Expression of all mutants\nwas tested by Western blot analyses using anti-HA antibody (Co-\nvance) directed against an N-terminal tag. In all cases, mutant re-\nceptors were expressed at equivalent or higher levels than wild-\ntype protein. For N-butanol experiments, 0.1% 1-butanol was\nadded for 2 hr. All transfection experiments were done in triplicates\nand repeated at least three times; representative experiments are\nshown with standard errors indicated by bars.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nLigand Analyses and Phospholipid Binding Assays\nProtein samples were desalted and injected onto a column packed\nwith P6 gel (BioRad) to exchange nonvolatile buffers with 20 mM\nammonium acetate (pH 6.8). Buffer-exchanged protein was col-\nlected and back flushed at a flow rate of 0.75–1 µl/min into a na-\nnoflow electrospray ionization source fitted on a Waters/Micromass\nQTOF-1 (Beverly, Massachusetts) mass spectrometer. Data were col-\nlected in positive ion mode over a mass range of 1000–4000 m/z. For\nstructural determination of ligands, a similar in-gel filtration appara-\ntus was connected to an orthogonal acceleration time-of-flight\nmass spectrometer (LCMSD-TOF, Agilent) and a quadrupole ion\ntrap mass spectrometer (LCMSD Trap SL, Agilent). Samples were\nexamined utilizing negative ion electrospray using conditions that\ndenatured the protein-ligand complex.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nPIP StripsTM (Echelon) were first blotted for 1 hr with Casein hy-\ndrolysate (Sigma) dissolved in TBST buffer. HIS6-tagged LBD pro-\nteins were added at a concentration of 1.5 M and incubated at\n4°C overnight, followed by incubation at 25°C for 4 hr. Proteins were\ndetected with rabbit polyclonal anti-His antibody (Immunology Con-\nsultants Laboratory). For liposome preparations, lipids (4 µM) were\nmixed in chloroform/methanol and dried under reduced pressure\nto form a lipid film. Lipids were as follows: 1-palmitoyl-2-oleoyl-sn-\nglycero-3-phosphate (PA) and 1-palmitoyl-2-oleoyl-sn-glycero-\n3-phosphocholine (PC) were from Avanti Polar Lipids (Alabaster,\nAlabama), and 1,2-dipalymitoyl-sn-glycero-3-phosphoinositol 3,4-\nbisphosphate (PI(3,4)P2) and 1,2-dipalymitoyl-sn-glycero-3-phos-\nphoinositol 3,4,5-trisphosphate (PI(3,4,5)P3) were from Echelon\nBiosciences Inc (Salt Lake City, Utah). Liposomes were formed at\npH 7.6 TRIS buffered saline by vortexing and immersed in a bath\nsonicator for 1–2 hr under argon at room temperature to facilitate\nformation of uniformly mixed small unilamellar vesicles. Particle\nzeta average size was determined using a Malvern (Southborough,\nMassachusetts) Zeta3000 Dynamic Light Scattering Instrument.\nNeutral liposomes were prepared with 100% PC and found to be\n70.6 ± 0.4 nm. All other liposomes were prepared at a lipid ratio of\nPC:nonPC [9:1] and found to be 46.4 ± 1.4 nm for PC:PA, 37.1 ±\n3.8 nm for PC:PI(3,4)P2, and 31.2 ± 1.4 nm for PC:PI(3,4,5)P3. Under\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nthese conditions, PI and PA lipids should mix completely into sta-\nble bilayers and consist primarily of PC. LBD proteins (2 mg/ml)\nwere exchanged into liposome buffer and combined with the lipo-\nsomes to produce a 25:1 molar ratio of PC to protein and a 2.5:1\nmolar ratio of PA, PI(3,4)P2, or PI(3,4,5)P3 to protein. Mixtures were\nincubated for 20 hr at room temperature, and protein was sepa-\nrated from liposomes by gel-filtration on Superdex 75 (Pharmacia),\nconcentrated, subjected to another round of gel filtration, and ana-\nlyzed by mass spectrometry as described above. The amount of\nSF-1 LBD protein associated with free liposomes (void volume) fol-\nliposome\nlowing gel filtration was determined to be <1% for all\nspecies.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nPeptide Binding Assays\nLBD proteins were serially diluted from 150 µM to 0.002 µM in bind-\ning buffer (50 mM NaPO4, 150 mM NaCl, pH 7.2, 1 mM DTT, 1 mM\nEDTA, 0.01% NP40, 10% glycerol). Diluted protein was added to\nfluorescent coregulator peptide (20 nM) in 384 well plates yielding\nfinal protein concentrations of 75–0.001 µM and 10 nM of fluores-\ncent peptide. Samples were analyzed as previously described\n(Moore et al., 2004). Construction of 20 aa coregulator peptides\nwith the general motif of CXXXXXXXLXXL/AL/AXXXXXXX were as\nfollows: peptides were synthesized in parallel using standard fluor-\nenylmethoxycarbonyl (Fmoc) chemistry in 48 well synthesis blocks\n(FlexChem System, Robbins) (Wellings and Atherton, 1997). Cou-\npling efficiency was monitored and the completed peptides were\ncleaved from the resin, followed by reversed-phase chromatogra-\nphy and mass spectrometry (MALDI-TOF/ESI) to purify the pep-\ntides, and are described in Moore et al. (2004).\n**BLOCK**fs== 7.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nEvolutionary Analysis\nProtein sequences of 48 SF-1/LRH-1/FtzF1 family members with\nfull-length ligand binding domains were obtained from GenBank by\nBLAST search using Homo sapiens SF-1 as a query. Twenty addi-\ntional sequences from the closely related ERR, GCNF/GRF, and\nRXR groups (Thornton and DeSalle, 2000) were also obtained to\nserve as outgroup sequences. (See Supplemental Table S2 for ac-\ncession numbers and species names). These sequences were\naligned using ClustalX 1.83 software (Thompson et al., 1994) as-\nsuming a gap:change cost ratio of 10. The phylogeny of the gene\nfamily was then inferred from the aligned sequences using the\nBayesian Markov Chain Monte Carlo method (Ronquist and\nHuelsenbeck, 2003) implemented in MrBayes v. 3.0 software. Tree\nand parameter space were explored with four simultaneous chains\n(three heated) for 300,000 generations each, using the Jones model\nof protein evolution with a four-category discrete γ model of\namong-site rate variation. Priors were equal for tree topologies,\nuniform (0,10) for branch lengths, and uniform (0.1, 5) for the α\nshape parameter of the γ distribution. The initial 100,000 trees, a\npoint well past stationarity, were discarded as burn-in.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nPhylogenetic reconstruction of the ancestral states at internal\nnodes for ligand binding or ligand independence, coded as a binary\ncharacter, were conducted by parsimony and maximum likelihood.\nFor the former, we used Fitch’s parsimony algorithm (Fitch, 1971)\non a simple tree consisting of the four taxa for which ligand binding\nstates are known (Sablin et al., 2003; this paper). For maximum\nlikelihood reconstruction, we calculated the likelihood of each an-\ncestral state 0 (no ligand binding) and 1 (ligand binding) as the\nprobability of evolving the terminal states in mLRH-1, hLRH-1,\nmSF-1, and hSF-1, given the phylogeny that relates these genes,\nand a simple reversible Markov process for transitions between\nstates 0 and 1 (Pagel, 1999). The probability Pij of observing a state\nchange on a tree branch of length k is Pij =\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\ne−k.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nBranch lengths were assumed equal a priori and scaled to their\nmaximum likelihood values by hill climbing; unknown states at in-\nternal nodes other than the NR5A ancestor were treated as nui-\nsance parameters and optimized by maximum likelihood.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nAncestral amino acid states for positions in the sequence align-\nment corresponding to 439, 440, and 443 in mLRH-1 were recon-\nstructed using the marginal Bayesian maximum likelihood method\nof Yang (Yang et al., 1995), as implemented in PAML 3.14 software,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\nassuming a Jones model of amino acid evolution and a four-cate-\ngory γ model of rate variation among sites (α optimized by ML). The\ntree for ancestral state reconstruction was the maximum posterior\nprobability tree from the Bayesian analysis, adjusted at three nodes\nthat had posterior probability <0.95 to make them consistent with\nknown taxonomic relationships among species (see Supplemental\nFigure S3). Branch lengths for the ancestral reconstruction were\noptimized by maximum likelihood using PAML software.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nSupplemental Data\nSupplemental Data include three figures and two tables and can\nbe found with this article online at http://www.cell.com/cgi/\ncontent/full/120/3/343/DC1/.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nAcknowledgments\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nWe would like to thank Drs. D. Julius, K. Shokat, D. Stokoe, S.\nWadekar, N. Jouravel, D. Moore, and F. Szoka for critical com-\nments, experimental suggestions, and reagents. The IMCA-CAT\nbeamline 17-ID (or 17-BM) at the Advanced Photon Source was\nsupported by the companies of the Industrial Macromolecular\nCrystallography Association through a contract with Illinois Insti-\ntute of Technology. We would like to thank Drs. Cheng Yang and\nJoseph Ferrara from Rigaku/MSC Inc. for an invitation to this facil-\nity. This work was funded by NIH 1R21-GM70792-1 and the Oregon\nMedical Research Foundation to J.W.T., a HHMI predoctoral fellow-\nship to J.A.M., and a NIDDK-PO1 support to R.J.F., R.K.G., and\nH.A.I.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nReceived: November 3, 2004\nRevised: January 13, 2005\nAccepted: January 27, 2005\nPublished: February 10, 2005\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nReferences\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nBacqueville, D., Deleris, P., Mendre, C., Pieraggi, M.T., Chap, H.,\nGuillon, G., Perret, B., and Breton-Douillon, M. (2001). Characteriza-\ntion of a G protein-activated phosphoinositide 3-kinase in vascular\nsmooth muscle cell nuclei. J. Biol. Chem. 276, 22170–22176.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nBaker, K.D., Shewchuk, L.M., Kozlova, T., Makishima, M., Hassell,\nA., Wisely, B., Caravella, J.A., Lambert, M.H., Reinking, J.L., Krause,\nH., et al. (2003). The Drosophila orphan nuclear receptor DHR38\nmediates an atypical ecdysteroid signaling pathway. Cell 113,\n731–742.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nBillas, I.M., Moulinier, L., Rochel, N., and Moras, D. (2001). Crystal\nstructure of the ligand-binding domain of the ultraspiracle protein\nUSP, the ortholog of retinoid X receptors in insects. J. Biol. Chem.\n276, 7465–7474.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.6**\nBland, M.L., Jamieson, C.A., Akana, S.F., Bornstein, S.R., Eisen-\nhofer, G., Dallman, M.F., and Ingraham, H.A. (2000). Haploinsuffi-\nciency of steroidogenic factor-1 in mice disrupts adrenal develop-\nment leading to an impaired stress response. Proc. Natl. Acad. Sci.\nUSA 97, 14488–14493.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nBotrugno, O.A., Fayard, E., Annicotte, J.S., Haby, C., Brennan, T.,\nWendling, O., Tanaka, T., Kodama, T., Thomas, W., Auwerx, J., and\nSchoonjans, K. (2004). Synergy between LRH-1 and beta-catenin\ninduces G1 cyclin-mediated cell proliferation. Mol. Cell 15, 499–\n509.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nChristenson, L.K., McAllister, J.M., Martin, K.O., Javitt, N.B., Os-\nborne, T.F., and Strauss, J.F., 3rd. (1998). Oxysterol regulation of\nsteroidogenic acute regulatory protein gene expression. Structural\nspecificity and transcriptional and posttranscriptional actions. J.\nBiol. Chem. 273, 30729–30735.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nDarimont, B.D., Wagner, R.L., Apriletti, J.W., Stallcup, M.R., Kush-\nner, P.J., Baxter, J.D., Fletterick, R.J., and Yamamoto, K.R. (1998).\nStructure and specificity of nuclear receptor-coactivator interac-\ntions. Genes Dev. 273, 30729–30735.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.1**r== 0.6**\nde Figueiredo, B.C., Cavalli, L.R., Pianovski, M.A., Lalli, E., Sandrini,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nR., Ribeiro, R.C., Zambetti, G., Delacerda, L., Rodrigues, G.A., and\nHaddad, B.R. (2004). Amplification of the steroidogenic factor 1\n(SF-1) gene in childhood adrenocortical tumors. J. Clin. Endocrinol.\nMetab. Published online November 2004. 10.1210/jc.2004-0942\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.2**\nDeleris, P., Bacqueville, D., Gayral, S., Carrez, L., Salles, J.P., Perret,\nB., and Breton-Douillon, M. (2003). SHIP-2 and PTEN are expressed\nand active in vascular smooth muscle cell nuclei, but only SHIP-2 is\nassociated with nuclear speckles. J. Biol. Chem. 278, 38884–38891.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nDelerive, P., Galardi, C.M., Bisi, J.E., Nicodeme, E., and Goodwin,\nB. (2004). Identification of liver receptor homolog-1 as a novel regu-\nlator of apolipoprotein AI gene transcription. Mol. Endocrinol. 18,\n2378–2387.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nDesclozeaux, M., Krylova, I.N., Horn, F., Fletterick, R.J., and Ingra-\nham, H.A. (2002). Phosphorylation and intramolecular stabilization\nof the ligand binding domain in the nuclear receptor steroidogenic\nfactor 1. Mol. Cell. Biol. 22, 7193–7203.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.2**\nDhe-Paganon, S., Duda, K., Iwamoto, M., Chi, Y.I., and Shoelson,\nS.E. (2002). Crystal structure of the HNF4 alpha ligand binding do-\nmain in complex with endogenous fatty acid ligand. J. Biol. Chem.\n277, 37973–37976.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nDietschy, J.M., and Turley, S.D. (2002). Control of cholesterol turn-\nover in the mouse. J. Biol. Chem. 277, 3801–3804.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nEscriva, H., Delaunay, F., and Laudet, V. (2000). Ligand binding and\nnuclear receptor evolution. Bioessays 22, 717–727.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.2**\nFang, Y., Vilella-Bach, M., Bachmann, R., Flanigan, A., and Chen, J.\n(2001). Phosphatidic acid-mediated mitogenic activation of mTOR\nsignaling. Science 294, 1942–1945.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nFayard, E., Auwerx, J., and Schoonjans, K. (2004). LRH-1: an or-\nphan nuclear receptor involved in development, metabolism and\nsteroidogenesis. Trends Cell Biol. 14, 250–260.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nFitch, W.M. (1971). Toward defining the course of evolution: minimal\nchange for a specific tree topology. Syst. Zool. 20, 406–416.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.2**\nHolter, E., Kotaja, N., Makela, S., Strauss, L., Kietz, S., Janne, O.A.,\nGustafsson, J.A., Palvimo, J.J., and Treuter, E. (2002). Inhibition of\nandrogen receptor\nfunction by the reproductive orphan\nnuclear receptor DAX-1. Mol. Endocrinol. 16, 515–528.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n(AR)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nHsuan, J., and Cockcroft, S. (2001). The PITP family of phosphati-\ndylinositol transfer proteins. Genome Biol. 2, REVIEWS3011.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nIngraham, H.A., Lala, D.S., Ikeda, Y., Luo, X., Shen, W.H., Nachtigal,\nM.W., Abbud, R., Nilson, J.H., and Parker, K.L. (1994). The nuclear\nreceptor steroidogenic factor 1 acts at multiple levels of the repro-\nductive axis. Genes Dev. 8, 2302–2312.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.2**\nIrvine, R.F. (2002). Nuclear lipid signaling. Sci. STKE, http://stke.\nsciencemag.org/cgi/content/full/OC_sigtrans;2002/150/re13.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nIto, M., Yu, R., and Jameson, J.L. (1997). DAX-1 inhibits SF-1-medi-\nated transactivation via a carboxy-terminal domain that is deleted\nin adrenal hypoplasia congenita. Mol. Cell. Biol. 17, 1476–1483.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.2**\nKallen, J.A., Schlaeppi, J.M., Bitsch, F., Geisse, S., Geiser, M., Del-\nhon, I., and Fournier, B. (2002). X-ray structure of the hRORalpha\nLBD at 1.63 A. Structural and functional data that cholesterol or a\ncholesterol derivative is the natural ligand of RORalpha. Structure\n(Camb.) 10, 1697–1707.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nLala, D.S., Syka, P.M., Lazarchik, S.B., Mangelsdorf, D.J., Parker,\nK.L., and Heyman, R.A. (1997). Activation of the orphan nuclear\nreceptor steroidogenic factor 1 by oxysterols. Proc. Natl. Acad. Sci.\nUSA 94, 4895–4900.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nLedeen, R.W., and Wu, G. (2004). Nuclear lipids: key signaling ef-\nfectors in the nervous system and other tissues. J. Lipid Res. 45, –.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.2**\nLee, M.K., Lebedeva, L., Suzawa, M., Wadekar, S., Desclozeaux,\nM., and Ingraham, H.A. (2005). Repression of orphan nuclear recep-\ntor activity via SUMO-modification and the Dead-Box Protein\nDP103 (Ddx20, Gemin-3). Mol. Biol. Cell 25, 1879–1890.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\nLi, Y., Lambert, M.H., and Xu, H.E. (2003). Activation of nuclear re-\nceptors: a perspective from structural genomics. Structure (Camb.)\n11, 741–746.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.5**r== 0.2**\nLie, J., de Crom, R., van Gent, T., van Haperen, R., Scheek, L.,\nLankhuizen, I., and van Tol, A. (2002). Elevation of plasma phospho-\nlipid transfer protein in transgenic mice increases VLDL secretion.\nJ. Lipid Res. 43, 1875–1880.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nL.R., and Helmkamp, G.M., Jr. (2001). Structure of a multifunctional\nprotein. Mammalian phosphatidylinositol\ntransfer protein com-\nplexed with phosphatidylcholine. J. Biol. Chem. 276, 9246–9252.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nAccession Numbers\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nCoordinates have been deposited in the Protein Data Bank as\nfollows: mSF-1 PDB ID 1YMT, RCSB code RCSB031696, hLRH-1\nPDB ID 1YOK and RCSB031753, and hSF-1 PDB ID 1YOW and\nRCSB031764.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nLu, T.T., Makishima, M., Repa, J.J., Schoonjans, K., Kerr, T.A., Au-\nwerx, J., and Mangelsdorf, D.J. (2000). Molecular basis for feed-\nback regulation of bile acid synthesis by nuclear receptors. Mol.\nCell 6, 507–515.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nLudbrook, L.M., and Harley, V.R. (2004). Sex determination: a ‘win-\ndow’ of DAX1 activity. Trends Endocrinol. Metab. 15, 116–121.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nLuo, X., Ikeda, Y., and Parker, K.L. (1994). A cell-specific nuclear\nreceptor is essential for adrenal and gonadal development and sex-\nual differentiation. Cell 77, 481–490.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nMadauss, K., Juzumiene, D., Waitt, G., Williams, J.P., and Williams,\nS.P. (2004). Generation and characterization of human steroido-\ngenic factor 1 LBD crystals with and without bound co-factor pep-\ntide. Endocr. Res. 30, 775–785.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nMartelli, A.M., Tabellini, G., Borgatti, P., Bortul, R., Capitani, S., and\nNeri, L.M. (2003). Nuclear lipids: new functions for old molecules?\nJ. Cell. Biochem. 88, 455–461.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nMellon, S.H., and Bair, S.R. (1998). 25-Hydroxycholesterol is not a\nligand for the orphan nuclear receptor steroidogenic factor-1 (SF-\n1). Endocrinology 139, 3026–3029.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nMoore, J.M., Galicia, S.J., McReynolds, A.C., Nguyen, N.H., Scan-\nlan, T.S., and Guy, R.K. (2004). Quantitative proteomics of the thy-\nroid hormone receptor-coregulator interactions. J. Biol. Chem. 279,\n27584–27590.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nNachtigal, M.W., Hirokawa, Y., Enyeart-VanHouten, D.L., Flanagan,\nJ.N., Hammer, G.D., and Ingraham, H.A. (1998). Wilms’ tumor 1 and\nDax-1 modulate the orphan nuclear receptor SF-1 in sex-specific\ngene expression. Cell 93, 445–454.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nPagel, M. (1999). Inferring the historical patterns of biological evo-\nlution. Nature 401, 877–884.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nRonquist, F., and Huelsenbeck, J.P. (2003). MrBayes 3: Bayesian\nphylogenetic inference under mixed models. Bioinformatics 19,\n1572–1574.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nSablin, E.P., Krylova,\nI.N., Fletterick, R.J., and Ingraham, H.A.\n(2003). Structural basis for ligand-independent activation of the or-\nphan nuclear receptor LRH-1. Mol. Cell 11, 1575–1585.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nSoccio, R.E., and Breslow, J.L. (2003). StAR-related lipid transfer\n(START) proteins: mediators of intracellular lipid metabolism. J.\nBiol. Chem. 278, 22183–22186.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nSteinmetz, A.C., Renaud, J.P., and Moras, D. (2001). Binding of li-\ngands and activation of transcription by nuclear receptors. Annu.\nRev. Biophys. Biomol. Struct. 30, 329–359.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nThompson, J.D., Higgins, D.G., and Gibson, T.J. (1994). CLUSTALW:\nImproving the sensitivity of progressive multiple sequence align-\nment through sequence weighting, position-specific gap penalties\nand weight matrix choice. Nucleic Acids Res. 22, 4673–4680.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nThornton, J.W., and DeSalle, R. (2000). A new method to localize\nand test the significance of incongruence: detecting domain shuf-\nfling in the nuclear receptor superfamily. Syst. Biol. 49, 183–201.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nThornton, J.W., Need, E., and Crews, D. (2003). Resurrecting the\nancestral steroid receptor: ancient origin of estrogen signaling. Sci-\nence 301, 1714–1717.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nWang, Z., Benoit, G., Liu, J., Prasad, S., Aarnisalo, P., Liu, X., Xu,\nH., Walker, N.P., and Perlmann, T. (2003). Structure and function of\nNurr1 identifies a class of ligand-independent nuclear receptors.\nNature 423, 555–560.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nWeatherman, R.V., Fletterick, R.J., and Scanlan, T.S.\n(1999).\nNuclear-receptor ligands and ligand-binding domains. Annu. Rev.\nBiochem. 68, 559–581.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nWellings, D.A., and Atherton, E. (1997). Standard Fmoc protocols.\nMethods Enzymol. 289, 44–67.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nWisely, G.B., Miller, A.B., Davis, R.G., Thornquest, A.D., Jr., John-\nson, R., Spitzer, T., Sefler, A., Shearer, B., Moore, J.T., Willson, T.M.,\nand Williams, S.P. (2002). Hepatocyte nuclear factor 4 is a transcrip-\ntion factor that constitutively binds fatty acids. Structure (Camb.)\n10, 1225–1234.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nYang, Z., Kumar, S., and Nei, M. (1995). A new method of inference\nof ancestral nucleotide and amino acid sequences. Genetics 141,\n1641–1650.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nYoder, M.D., Thomas, L.M., Tremblay, J.M., Oliver, R.L., Yarbrough,",
         "extracted",
         "The DEAD-Box Protein DP103 (Ddx20 or Gemin-3) Represses Orphan Nuclear Receptor Activity via SUMO Modification;Amplification of the steroidogenic factor 1 gene in childhood adrenocortical tumors.;Identification of liver receptor homolog-1 as a novel regulator of apolipoprotein AI gene transcription.;Synergy between LRH-1 and beta-catenin induces G1 cyclin-mediated cell proliferation.;Quantitative Proteomics of the Thyroid Hormone Receptor-Coregulator Interactions*[boxs];LRH-1: an orphan nuclear receptor involved in development, metabolism and steroidogenesis.;Sex determination: a ‘window’ of DAX1 activity;Generation and Characterization of Human Steroidogenic Factor 1 LBD Crystals with and without Bound Cofactor Peptide;SHIP-2 and PTEN Are Expressed and Active in Vascular Smooth Muscle Cell Nuclei, but Only SHIP-2 Is Associated with Nuclear Speckles*;Resurrecting the Ancestral Steroid Receptor: Ancient Origin of Estrogen Signaling;MrBayes 3: Bayesian phylogenetic inference under mixed models;Activation of nuclear receptors: a perspective from structural genomics.;StAR-related Lipid Transfer (START) Proteins: Mediators of Intracellular Lipid Metabolism*;The Drosophila Orphan Nuclear Receptor DHR38 Mediates an Atypical Ecdysteroid Signaling Pathway;Structural basis for ligand-independent activation of the orphan nuclear receptor LRH-1.;Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors;Nuclear lipid signalling;Nuclear lipids: New functions for old molecules?;X-Ray Structure of the hRORα LBD at 1.63 Å;Elevation of plasma phospholipid transfer protein in transgenic mice increases VLDL secretion Published, JLR Papers in Press, August 16, 2002. DOI 10.1194/jlr.M200166-JLR200;Crystal Structure of the HNF4α Ligand Binding Domain in Complex with Endogenous Fatty Acid Ligand*;Phosphorylation and Intramolecular Stabilization of the Ligand Binding Domain in the Nuclear Receptor Steroidogenic Factor 1;Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids.;Inhibition of androgen receptor (AR) function by the reproductive orphan nuclear receptor DAX-1.;Control of Cholesterol Turnover in the Mouse*;Phosphatidic Acid-Mediated Mitogenic Activation of mTOR Signaling;The PITP family of phosphatidylinositol transfer proteins;Characterization of a G Protein-activated Phosphoinositide 3-Kinase in Vascular Smooth Muscle Cell Nuclei*;Binding of ligands and activation of transcription by nuclear receptors.;Structure of a Multifunctional Protein;Crystal Structure of the Ligand-binding Domain of the Ultraspiracle Protein USP, the Ortholog of Retinoid X Receptors in Insects*;Haploinsufficiency of steroidogenic factor-1 in mice disrupts adrenal development leading to an impaired stress response.;Nuclear Lipid Signaling;Molecular basis for feedback regulation of bile acid synthesis by nuclear receptors.;Ligand binding and nuclear receptor evolution;A new method to localize and test the significance of incongruence: detecting domain shuffling in the nuclear receptor superfamily.;Inferring the historical patterns of biological evolution;Oxysterol Regulation of Steroidogenic Acute Regulatory Protein Gene Expression;Structure and specificity of nuclear receptor-coactivator interactions.;25-Hydroxycholesterol is not a ligand for the orphan nuclear receptor steroidogenic factor-1 (SF-1).;Wilms' Tumor 1 and Dax-1 Modulate the Orphan Nuclear Receptor SF-1 in Sex-Specific Gene Expression;Activation of the orphan nuclear receptor steroidogenic factor 1 by oxysterols.;DAX-1 inhibits SF-1-mediated transactivation via a carboxy-terminal domain that is deleted in adrenal hypoplasia congenita;A new method of inference of ancestral nucleotide and amino acid sequences.;CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;The nuclear receptor steroidogenic factor 1 acts at multiple levels of the reproductive axis.;A cell-specific nuclear receptor is essential for adrenal and gonadal development and sexual differentiation;Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology;Repression of orphan nuclear recep - or activity via SUMO - modification and the Dead - Box Protein P 103 ( Ddx 20 , Gemin - 3 );Nuclear lipids: key signaling effectors in the nervous system and other tissues;X-ray structure of the hRORalpha LBD at 1.63 A: structural and functional data that cholesterol or a cholesterol derivative is the natural ligand of RORalpha.;Nuclear-receptor ligands and ligand-binding domains.;Standard Fmoc protocols.;Methods Enzymol",
         "Structural Analyses Reveal Phosphatidyl Inositols as Ligands for the NR5 Orphan Receptors SF-1 and LRH-1",
         "None"
        ],
        [
         "35",
         "002c2130f79dcbd4ddd5866fd302a424fb8914e8",
         "The dopaminergic and renin-angiotensin systems regulate blood pressure, in part, by affecting sodium transport in renal proximal tubules (RPTs). We have reported that activation of a D1-like receptor decreases AT1 receptor expression in the mouse kidney and in immortalized RPT cells from Wistar-Kyoto (WKY) rats. The current studies were designed to test the hypothesis that activation of the AT1 receptor can also regulate the D1 receptor in RPT cells, and this regulation is aberrant in spontaneously hypertensive rats (SHRs). Long-term (24 hours) stimulation of RPT cells with angiotensin II, via AT1 receptors increased total cellular D1 receptor protein in a time- and concentration-dependent manner in WKY but not in SHR cells. Short-term stimulation (15 minutes) with angiotensin II did not affect total cellular D1 receptor protein in either rat strain. However, in the short-term experiments, angiotensin II decreased cell surface membrane D1 receptor protein in WKY but not in SHR cells. D1 and AT1 receptors colocalized (confocal microscopy) and their coimmunoprecipitation was greater in WKY than in SHRs. However, AT1/D1 receptor coimmunoprecipitation was decreased by angiotensin II (10−8M/24 hours) to a similar extent in WKY (−22±8%) and SHRs (−22±12%). In summary, these studies show that AT1 and D1 receptors interact differently in RPT cells from WKY and SHRs. It is possible that an angiotensin II-mediated increase in D1 receptors and dissociation of AT1 from D1 receptors serve to counter regulate the long-term action of angiotensin II in WKY rats; different effects are seen in SHRs.",
         "C. Zeng,Z. Wang,U. Hopfer,L. Asico,G. Eisner,R. Felder,P. Jose",
         "https://www.ahajournals.org/doi/pdf/10.1161/01.HYP.0000184251.01159.72",
         "\n**BLOCK**fs== 18.5**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nRat Strain Effects of AT1 Receptor Activation on D1\nDopamine Receptors in Immortalized Renal Proximal\nTubule Cells\n**BLOCK**fs== 11.5**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nChunyu Zeng, Zheng Wang, Ulrich Hopfer, Laureano D. Asico, Gilbert M. Eisner,\nRobin A. Felder, Pedro A. Jose\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.1**\nAbstract—The dopaminergic and renin-angiotensin systems regulate blood pressure, in part, by affecting sodium transport\nin renal proximal tubules (RPTs). We have reported that activation of a D1-like receptor decreases AT1 receptor\nexpression in the mouse kidney and in immortalized RPT cells from Wistar-Kyoto (WKY) rats. The current studies were\ndesigned to test the hypothesis that activation of the AT1 receptor can also regulate the D1 receptor in RPT cells, and\nthis regulation is aberrant in spontaneously hypertensive rats (SHRs). Long-term (24 hours) stimulation of RPT cells\nwith angiotensin II, via AT1 receptors increased total cellular D1 receptor protein in a time- and concentration-dependent\nmanner in WKY but not in SHR cells. Short-term stimulation (15 minutes) with angiotensin II did not affect total\ncellular D1 receptor protein in either rat strain. However, in the short-term experiments, angiotensin II decreased cell\nsurface membrane D1 receptor protein in WKY but not in SHR cells. D1 and AT1 receptors colocalized (confocal\nmicroscopy) and their coimmunoprecipitation was greater in WKY than in SHRs. However, AT1/D1 receptor\ncoimmunoprecipitation was decreased by angiotensin II (108M/24 hours) to a similar extent in WKY (228%) and\nSHRs (2212%). In summary, these studies show that AT1 and D1 receptors interact differently in RPT cells from\nWKY and SHRs. It is possible that an angiotensin II-mediated increase in D1 receptors and dissociation of AT1 from\nD1 receptors serve to counter regulate the long-term action of angiotensin II in WKY rats; different effects are seen in\nSHRs. (Hypertension. 2005;46:799-805.)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nKey Words: dopamine  kidney  receptors, angiotensin II  rats, spontaneously hypertensive\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nA ngiotensin II and dopamine are important regulators of\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nsodium and water transport across the renal proximal\ntubule (RPT).1– 8 Angiotensin II receptor subtypes (AT1, AT2,\nand AT4) are expressed in brush border and basolateral\nmembranes of RPTs.1–5 The activation of AT1 receptors by\nlow concentrations of angiotensin II (picomolar) causes an\nincrease in sodium reabsorption in RPTs.1,3,6 All the dopa-\nmine receptor subtypes, D1-like (D1 and D5) and D2-like (D2,\nD3, and D4) are also expressed in brush border and basolateral\nmembranes of RPTs.7–10 In contrast to the stimulatory effect\nof AT1 receptors on renal sodium transport, activation of D1\nand D3 receptors decreases renal sodium reabsorption.7,8,11–13\nSeveral reports have shown an interaction between the\ndopamine and renin-angiotensin system. Intrarenally pro-\nduced angiotensin II opposes the natriuretic action of the D1-\nlike dopamine receptor agonist, fenoldopam, in rats.14 D2-like\nreceptor agonists also antagonize the stimulatory effect of\nangiotensin II, acting via AT1 receptors, on renal proximal\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\ntubular luminal sodium transport.15,16 When angiotensin II\ngeneration is inhibited or AT1 receptors are blocked, the\nnatriuretic effect of dopaminergic drugs is enhanced.17\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nThe AT1 receptor mediates the anti-natriuretic effect of\nangiotensin II, whereas the D1 dopamine receptor is respon-\nsible for 80% of D1-like receptor activity in RPTs.18 We\nhave reported that in RPT cells of Wistar-Kyoto (WKY) rats,\nthe D1-like agonist, fenoldopam, increased D1 receptor but\ndecreased AT1 receptor protein expression. In contrast, in\nRPT cells of spontaneously hypertensive rats (SHRs),\nfenoldopam also decreased AT1 receptor expression but no\nlonger stimulated D1 receptor expression. We also reported\nthat AT1 and D1 receptors coimmunoprecipitate in RPT cells;\nfenoldopam increased this coimmunoprecipitation in WKY\nRPT cells but decreased it in SHR RPT cells.19 We hypoth-\nesize that the AT1 receptor may also regulate the D1 receptor,\nincluding its expression. Therefore, we studied the effect of\nangiotensin II on D1 receptor expression and AT1 and D1\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nReceived June 23, 2005; first decision July 11, 2005; revision accepted August 18, 2005.\nFrom the Department of Cardiology (C.Z.), Daping Hospital, Third Military Medical University, Chongqing, P.R. China; Departments of Pediatrics\n(C.Z., Z.W., L.D.A., G.M.E., P.A.J.) and Physiology and Biophysics (P.A.J.), and Internal Medicine (G.M.E.), Georgetown University Medical Center,\nWashington, DC; Department of Physiology and Biophysics (U.H.), Case Western Reserve School of Medicine, Cleveland, Ohio; Department of\nPathology (R.A.F.), University of Virginia Health Sciences Center, Charlottesville, Va.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nCorrespondence to Chunyu Zeng, Department of Pediatrics, PHC-2, Georgetown University Medical Center, 3800 Reservoir Rd, NW, Washington,\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nDC 20007. E-mail cyzeng1@hotmail.com\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\n© 2005 American Heart Association, Inc.\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.6**\nHypertension is available at http://www.hypertensionaha.org\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nreceptor interaction in immortalized rat RPT cells. We have\npreviously shown that\nthe D1 receptor effects on signal\ntransduction and sodium transporters in immortalized RPT\ncells are similar to those obtained in freshly obtained renal\nproximal tubules.20 –22 Parenti et al have also reported that the\nAT1 receptor behaves similarly in freshly obtained renal\ntubules and immortalized RPT cells.23\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nWe now report that AT1 and D1 receptors colocalize in RPT\ncells. D1 and AT1\nreceptor coimmunoprecipitation was\nin agreement with our\ngreater in WKY than in SHRs,\nprevious report.19 In this earlier study, long-term fenoldopam\nstimulation (24 hours) increased the coimmunoprecipitation\nof D1 and AT1 receptors in WKY but decreased it in SHRs.19\nWe now show that angiotensin II decreases D1 and AT1\ncoimmunoprecipitation similarly in WKY and SHRs. How-\never, D1 receptor expression is increased by angiotensin II in\nWKY but not in SHR. In contrast, short-term angiotensin II,\nvia AT1 receptors (15 minutes), decreases cell surface mem-\nbrane expression of D1 receptors in WKY but not in SHRs.\nThese studies show that angiotensin II differentially affects\ntotal cellular and surface membrane D1 receptor expression in\nRPT cells from WKY and SHRs.\n**BLOCK**fs== 11.5**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nMethods\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nCell Culture\nImmortalized RPT cells from 4- to 8-week-old WKY and SHRs were\ncultured at 37°C in 95% air/5% CO2 atmosphere in DMEM/F-12\nculture media, as previously described.21–26 The cells (80% conflu-\nence) were extracted in ice-cold lysis buffer (phosphate-buffered\nsaline with 1% NP40, 0.5% sodium deoxycholate, 0.1% SDS,\n1 mmol/L EDTA, 1 mmol/L EGTA, 1 mmol/L PMSF, 10 g/mL\naprotinin, and 10 g/mL leupeptin), sonicated, kept on ice for 1 hour,\nand centrifuged at 16 000g for 30 minutes. The supernatants were\nstored at 70°C until use\nimmunoblotting and/or\nimmunoprecipitation.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nfor\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nImmunoblotting\nThe antibodies are polyclonal purified antipeptides. The amino acid\nsequence for the immunogenic peptide (rabbit anti-human AT1\nreceptor antibody) is QDDCPKAGRHC, amino acids 15 to 24 of the\nAT1 receptor. The specificity of this antibody to the AT1 receptor is\nindicated as previously reported.27,28 The amino acid sequence for rat\nD1\nreceptor antibody corresponds to positions 299 to 307\n(GSEETQPFC) on D1 receptor (Research Genetics). The specificity\nof this D1 receptor antibody has also been reported.19,29 Rat RPT\ncells were treated with vehicle (dH2O), angiotensin II or an AT1\nreceptor antagonist (losartan) at the indicated concentrations and\ntimes. Immunoblotting was performed as previously reported,22,26\nexcept that the transblots were probed with the D1 (1:800) or the AT1\nreceptor antibody (1:400). The amount of protein transferred onto the\nmembranes was determined by Ponceau-S staining and immunoblot-\nting for -actin.19\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nImmunofluorescence Confocal Microscopy of\nDouble-Stained RPT Cells\nRPT cells, grown on coverslips, were fixed with 3% paraformalde-\nhyde (30 minutes) and permeabilized with 0.1% Triton X-100 in\nphosphate-buffered saline (15 minutes). Reactions with antibodies\nwere performed as described previously.22,30 The D1 and AT1\nreceptors were visualized by using a rabbit antipeptide polyclonal\nIgG affinity-purified rat D1 receptor antibody followed by fluores-\ncein isothiocyanate-conjugated anti-rabbit secondary antibody (Mo-\nlecular Probes) or by a mouse anti-human AT1 receptor monoclonal\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nantibody followed by the secondary goat anti-mouse antibody\nlabeled with Alexa 568 (Molecular Probes).22 Cells on coverslips\nwere mounted with the ProLong Antifade Kit (Molecular Probes).\nThe immunofluorescence densities and images were acquired\n(Olympus AX70) at an excitation wavelength of 488 nm and 568 nm;\nemission was detected at 535 and 645 nm.22,30\n**BLOCK**fs== 8.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nImmunoprecipitation\nRPT cells were incubated with vehicle or angiotensin II (108 M) for\n24 hours, as described. The cells were lysed with ice-cold lysis buffer\nfor 1 hour and centrifuged at 16 000g for 30 minutes. Equal amounts\nof lysates (500 g protein/mL supernatant for RPT cells from both\nSHR and WKY rats) were incubated with affinity-purified anti-D1\nreceptor antibody (2 L/mL) for 1 hour and protein-G agarose at 4°C\nfor 12 hours. The immunoprecipitates were pelleted and washed 4\ntimes with lysis buffer. The pellets were suspended in sample buffer,\nboiled for 10 minutes, and subjected to immunoblotting with the AT1\nreceptor antibody. To determine the specificity of the bands, pre-\nimmune serum of D1 receptor antibody (negative control) and AT1\nreceptor antibody (positive control) were used as immunoprecipi-\ntants, instead of the D1 receptor antibody. The densities of the bands\nwere quantified by densitometry using Quantiscan (Ferguson, Mo) as\npreviously reported.19,22,26\n**BLOCK**fs== 8.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nCell Surface D1 Receptor Expression\nCultured RPT cells were starved in serum-free medium for 2 hours,\nand then treated with angiotensin II (108 M) for 15 minutes. Surface\nmembrane proteins were biotinylated by adding sulfo-NHS-LC-\nbiotin (final concentration 250 g/mL) into the medium 5 minutes\nbefore adding angiotensin II. The cells were washed 3 times with\nwash buffer. Then, the cells were lysed with lysis buffer, sonicated\nand placed on ice for 1 hour. The supernatant from the cell lysate was\nimmunoprecipitated with the anti-rat D1 receptor antibody, followed\nby immunoblotting. The membrane sheets were blocked with 5%\nmilk overnight and after washing (3 times), the sheets were incu-\nbated with peroxidase-conjugated streptavidin (Jackson ImmunoRe-\nsearch Laboratory, Inc, in wash buffer at 1/5000 dilution for 30\nminutes. The biotinylated protein bands were visualized by enhanced\nchemiluminescence (Western Blotting Detection Kit; Amersham,\nArlington Heights, Ill).31\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nMaterials\nRabbit anti-human AT1 receptor antibodies were purchased from\nSanta Cruz Biotechnology, Inc (Santa Cruz, Calif). Rabbit anti-rat D1\nreceptor antibody was produced against a synthetic oligopeptide\nfrom the amino acid sequence of rat D1 receptor (amino acids 299 to\n307) (Research Genetics). Monoclonal mouse anti-human AT1 re-\nceptor was purchased from Abcam Limited (Cambridge, UK).\nAngiotensin II was purchased from Peninsula Laboratory, Inc (St\nLouis, Mo). Losartan was a gift from Merck & Co (Philadelphia, Pa).\nPeroxidase-conjugated streptavidin was purchased from Jackson\nImmunoResearch Laboratory, Inc (West Grove, Pa). Sulfo-NHS-LC-\nbiotin was purchased from PIERCE (Rockford, Ill). Other chemicals\nfor various buffers were of the highest purity available and purchased\neither from Sigma or Gibco.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nStatistical Analysis\nThe data are expressed as meanSEM. Comparison within groups\nwas made by ANOVA for repeated measures, and comparison\namong groups was made by factorial ANOVA with Duncan’s test.\nP0.05 was considered significant.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nResults\nActivation of AT1 Receptors Increases D1 Receptor\nProtein in RPT Cells From WKY Rats, But Not\nFrom SHRs\nIn RPT cells from WKY rats, angiotensin II increased D1\nreceptor expression in a concentration- and time-dependent\n**BLOCK**fs== 8.0**b== 0.4**t== 0.2**l== 0.7**r== 0.1**\nFigure 1. Effect of angiotensin II on D1\nreceptor protein in RPT cells from WKY\nand SHRs. A, Concentration-response of\nD1 receptor protein in RPT cells from\nWKY rats treated with angiotensin II.\nImmunoreactive D1 receptor protein was\ndetermined after 24 hour incubation with\nthe indicated concentrations of angioten-\nsin II. Results are expressed as relative\ndensity units (DU) (n8, *P0.05 vs con-\ntrol (C), ANOVA, Duncan’s test). B, Time-\ncourse of D1 receptor protein in RPT\ncells from WKY rats treated with angio-\ntensin II. The cells were incubated for the\nindicated times with 108 M angiotensin\nII. Results are expressed as relative den-\nsity units (DU) (n8, *P0.05 vs control\n[0 time], ANOVA, Duncan’s test). C,\nEffect of angiotensin (108 M/24 hours)\non D1 receptor protein in RPT cells from\nWKY and SHRs. Results are expressed\nas the ratio of D1 receptor to –actin\ndensities (n7, *P0.05 vs control,\nANOVA, Duncan’s test). D, Effect of an-\ngiotensin II (Ang) and an AT1 receptor\nantagonist (losartan) on D1 receptor pro-\ntein in WKY rat RPT cells. The cells were\nincubated with the indicated reagents\n(angiotensin II, 108 M; losartan, 108 M)\nfor 24 hour. Results are expressed as\nthe ratio of D1 receptor to –actin densi-\nties (n8, *P0.05 vs others, ANOVA,\nDuncan’s test). E, Concentration-\nresponse of D1 receptor protein in SHR\nRPT cells treated with angiotensin II.\nImmunoreactive D1 receptor protein was\ndetermined after a 24 hour-incubation\nwith the indicated concentrations of an-\ngiotensin II. Results are expressed as\nrelative density units (DU) (n6, PNS\nvs control (C), ANOVA, Duncan’s test).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nM to 105 M) for 24 hours. Consistent with the results in\nFigure 1C, angiotensin II had no effect on D1 receptor\nexpression in SHRs (Figure 1E).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.5**r== 0.1**\nAT1 Receptor Colocalizes With the D1 Receptor in\nRat RPT Cells\nImmunofluorescence laser confocal microscopy revealed a co-\nlocalization of D1 and AT1 receptors in WKY rat RPT cells\n(Figure 2). To determine whether there is a physical interaction\nbetween the D1 and the AT1 receptor, additional experiments\nwere performed. As shown in Figure 3, in the basal state, the\nband of 45 kDa, representing the coimmunoprecipitated D1 and\nAT1 receptors, was greater in RPT cells from WKY rats (371\nDU) than in those from SHRs (233 DU, n7, P0.05),\nsimilar to the findings in our previous report.19 A 24-hour\nincubation with angiotensin II (108 M) decreased D1/AT1\nreceptor coimmunoprecipitation to a similar degree in RPT cells\nfrom WKY rats and SHRs (WKY: control371 density units\n(DU), angiotensin II282 DU; SHRs: control233 DU,\nangiotensin II173 DU; n7, P0.05).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nmanner. The stimulatory effect was evident at 109 M (Figure\n1A) as early as 16 hours and maintained for at least 30 hours\n(Figure 1B). Consistent with the stimulatory effect of angio-\ntensin II on D1 receptor expression (Figure 1A and 1B), a 24\nhour-incubation with angiotensin II (108 M) increased D1\nreceptors in RPT cells from WKY rats. In contrast, this effect\nwas not observed in RPT cells from SHRs (WKY: con-\ntrol0.770.15, angiotensin II1.330.19; SHR: con-\ntrol0.870.13, angiotensin II1.040.17; n7) (Figure 1C).\nTo determine the specificity of angiotensin II action on\nAT1 receptors, we studied the effect of the AT1 receptor\nantagonist, losartan. Consistent with the results in WKY rats\nin Figure 1A, 1B, and 1C, angiotensin II (108 M/24 hours)\nincreased D1 receptor expression (control0.830.15, angio-\ntensin II1.480.15; n8). The AT1 receptor antagonist, losar-\ntan (108 M), by itself, had no effect on D1 receptor expression\n(0.80.13) but reversed the stimulatory effect of angiotensin II\non D1 receptor expression (0.70.08) (Figure 1D).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nTo investigate whether\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nthe\nconcentration-response curve in SHRs, their RPT cells were\nincubated with varying concentrations of angiotensin II (109\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nthere is a right-shift of\n**BLOCK**fs== 8.0**b== 0.7**t== 0.1**l== 0.7**r== 0.1**\nFigure 2. D1 and AT1 receptor colocaliza-\ntion in rat RPT cells. Colocalization\nappears as yellow after merging the\nimages of ﬂuorescein isothiocyanate-\nconjugated anti-rabbit secondary anti-\nbody (green), directed against the anti-\nrat D1 receptor and Alexa 568-\nconjugated goal anti-mouse secondary\nantibody (red), directed against mouse\nanti-human AT1 receptor (red).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nActivation of AT1 Receptors Decreases the D1\nReceptor Expression in Surface Membranes of\nRPT Cells From WKY Rats But Not From SHRs\nThe effect of short-term stimulation with angiotensin II on\ncell surface D1 receptors is shown in Figure 4. Angiotensin II\n(108 M) decreased the quantity of D1 receptors on the cell\nsurface membranes in WKY RPT cells, but not in SHR RPT\ncells (WKY: control459 DU, angiotensin II175 DU;\nSHR: control193 DU, angiotensin II195 DU, n6).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\nin renal proximal\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nDiscussion\nPolygenic essential hypertension is associated with increased\ntubules.32,33 As men-\nsodium transport\ntioned, dopamine and angiotensin II are important regulators\nof sodium and water reabsorption in the kidney, serving\ncontrasting functions in the proximal tubule.1,5,17,34,35 Both\ndopamine and renin-angiotensin systems exist in the RPT,\nand the components of the renin-angiotensin system, includ-\ning angiotensinogen mRNA,\nrenin, and angiotensin-\nconverting enzyme, have been localized to the proximal\ntubule.1,6,36,37 The proximal tubule is also the site of local\ndopamine production.38 – 40 Urinary dopamine and angiotensin\nII concentrations exceed circulating levels, suggesting both\nsystems may have autocrine and/or paracrine effects in\nRPTs.1,6,38,41 Angiotensin II stimulates sodium reabsorption\nvia activation of sodium transporting proteins that are inhib-\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nited by dopamine (NHE3, Cl/HCO3 exchanger, Na/HCO3\ncotransporter, and Na/K ATPase).34,35 Dopamine and D1-\nlike and D2-like receptor agonists have been reported to\nantagonize the actions of angiotensin II on NHE3 and\nNaK-ATPase activities in brush border and basolateral\nmembranes of renal proximal tubules.15,16 Dopamine, via\nD1-like receptors, has also been reported to decrease the AT1\nreceptor mRNA and protein, as well as angiotensin II binding\nsites in renal proximal tubules.19,42 The effect of angiotensin\nII on D1-like receptors has not been reported.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nIn WKY RPT cells, stimulation of the AT1 receptor with\nangiotensin II for 2 different periods (short-term or long-\nterm) yields opposite results; short-term stimulation de-\ncreases cell surface D1 receptor expression, whereas long-\nterm stimulation increases whole cell D1 receptor expression.\nIt is unlikely that the change in the short-term could account\nfor the long-term result. A decrease of cell surface D1 receptor\nexpression would remove a restraint allowing AT1 receptor\nactivation to increase sodium reabsorption to a greater extent.\nHowever, the increase in D1 receptor expression would tend\nto counterbalance the prolonged angiotensin II effect.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nCell surface membrane localization of G protein-coupled\nreceptors is decreased by phosphorylation of the receptor.8,43\nWe have reported that G protein-coupled receptor kinases\n(GRKs), GRK2 and GRK4, are involved in D1 receptor\nphosphorylation.44 Among the GRKs, GRK2 is most impor-\n**BLOCK**fs== 8.0**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nFigure 3. Effect of angiotensin II on the coimmunoprecipitation\nof D1 and AT1 receptors in rat RPT cells. The cells were incu-\nbated with angiotensin II (108 M) for 24 hour. Results are\nexpressed as relative density units (DU) (* P0.05 vs control,\n#P0.05, vs WKY, n7, ANOVA, Duncan’s test). One immuno-\nblot (45 kDa) is depicted in the inset: (lane 1positive control,\nlane 2negative control, lane 3vehicle-treated RPT cells from\nWKY rats, lane 4angiotensin II-treated RPT cells from WKY\nrats, lane 5vehicle-treated RPT cells from SHRs, lane 6an-\ngiotensin II-treated RPT cells from SHRs).\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nFigure 4. Effect of angiotensin II on cell surface D1 receptors\nin rat RPT cells. The cells were incubated with angiotensin II\n(108 M) for 15 minutes. Results are expressed as relative den-\nsity (DU) (*P0.05 vs control, #P0.05 vs WKY, n6, ANOVA,\nDuncan’s test). One immunoblot is depicted on top of the bar\ngraphs.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nis\nin promoting AT1 receptor phosphorylation.45 It\ntant\npossible that the GRK2 activation subsequent to AT1 receptor\nstimulation also results in the alteration of the phosphoryla-\ntion and membrane localization of the D1 receptor. Such a\nscenario can explain the ability of AT1 receptors to decrease\nthe membrane localization of the D1 receptor in WKY. This\ndoes not occur in SHRs because the D1 receptor is already\nhyperphosphorylated and fewer D1 receptors are found on\nrenal cell surface membranes in SHR.46 Although GRK4 is\nthe major GRK regulating D1 receptor function,43,44 GRK4 is\nnot involved in the desensitization of the AT1 receptor.47 The\npossibility exists that the AT1 receptor can cause heterologous\ndesensitization of the D1 receptor via stimulation of effector\nenzymes, but it is not clear which one.48\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nThe dephosphorylation of D1 receptors in renal proximal\ntubules is also different in WKY and SHRs. We have reported\nthat short-term D1-like receptor stimulation increases protein\nphosphatase 2A activity in renal proximal tubules in WKY\nbut not in SHRs49. It is possible that in the WKY, the AT1\nreceptor inhibits protein phosphatase 2A activity and thus\nprevents dephosphorylation and recycling of the D1 receptor\nto the cell surface membrane. Although there is no report to\nsupport this mechanism, AT2 receptor has been reported to\nstimulate protein phosphatase 2A activity.50,51\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nLong-term stimulation of AT1 receptors increases total\ncellular D1 receptor expression in WKY. This finding in vitro\nwas also found in vivo; chronic angiotensin II infusion in\nmice also caused an increase in D1 receptor expression\n(unpublished data). Presumably, cell surface D1 receptor\nexpression is also increased. Although the mechanism by\nwhich AT1 receptor stimulation increases D1 receptor expres-\nsion remains to be determined, such an effect could serve as\na physiological negative feedback under normal circum-\nstances (in WKY). However, in SHR cells, this upregulation\nof D1 receptors by stimulation of AT1 receptors is lost because\nof decreased responsiveness of the hyperphosphorylated D1\nreceptor.46,52,53 We have reported that the renal D1 receptor is\nhyperphosphorylated in SHRs as a consequence of increased\nactivity of GRK454 and decreased protein phosphatase 2A\nactivity.49\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nLong-term incubation with angiotensin II decreases D1/AT1\nreceptor coimmunoprecipitation to the same degree in cells\nfrom SHR and WKY rats. Our previous study showed that\nlong-term exposure to angiotensin II decreases AT1 receptor\nprotein expression in RPT cells from WKY rats but increases\nit in SHRs.26 The decreased interaction between D1 and AT1\nreceptors in renal proximal tubules in WKY rats is, therefore,\nnot the result of decreased AT1 receptor expression, per se,\nbecause a similar response is seen in SHRs where AT1\nexpression is increased by angiotensin II. Thus, angiotensin\nII, via the AT1 receptor, probably causes a decrease in the\nphysical interaction of AT1 and D1 receptors in both WKY\nand SHR cells. Further studies are needed to determine\nwhether the decreased interaction between these 2 receptors\nis direct or indirect, possibly by the alteration of an adaptor\ngene or adaptor proteins.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nIn both SHR and Dahl salt-sensitive rats, dopamine and\nD1-like receptor agonist-mediated natriuretic and diuretic\nresponses are impaired.8,43 The impaired D1-like receptor\n**BLOCK**fs== 9.5**b== 0.6**t== 0.1**l== 0.5**r== 0.1**\nfunction in hypertension is not caused by abnormalities in G\nproteins, effectors, or ion transporting proteins, such as\nadenylyl cyclase, NHE3 or Na-KATPase. Rather, the renal\nD1-like receptor is uncoupled from G protein subunits,\nleading to decreased D1-like receptor interaction with G\nprotein subunits, and resulting in decreased production of\nsecond messengers and decreased interaction between G\nprotein subunits, effector enzymes, and ion transporters.8,43,53\nTo determine whether the impaired D1 receptor function\noccurs at the receptor level, we investigated the expression of\nD1 receptor expression in cell surface membranes and whole\ncells. We found that cell surface membrane expression of D1\nreceptors is lower and D1 receptor phosphorylation is higher\nin SHR cells than in WKY cells, although total and D1\nreceptor expression are not different in the 2 cell lines.46,54\nRenal D1 receptor phosphorylation and GRK4 activity impact\non the regulation of blood pressure. Inhibition of renal GRK4\nactivity by antisense oligonucleotides attenuates the increase\nin blood pressure with age in SHRs without affecting blood\npressure in WKY rats.55\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nIn summary, we have demonstrated that long-term stimu-\nlation of AT1 receptors positively regulates the expression of\nD1 receptors in RPT cells from WKY but not from SHRs. AT1\nand D1 receptors colocalize in RPT cells from both rat strains,\nbut the basal level of cell surface membrane D1 receptor\nexpression is higher in WKY than in SHR RPT cells. It\nappears that a greater proportion of D1 receptors are located\non the cell surface membrane in WKY cells than in SHR\ncells. Short-term AT1 receptor stimulation decreases cell\nsurface membrane D1 receptor expression in WKY rats but\nnot in SHRs. It is possible that differences in AT1 receptor\nregulation of D1 receptor expression and cell surface expres-\nsion may participate in the abnormal regulation of renal\nproximal sodium transport in genetic hypertension.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nPerspectives\nThe dopaminergic and renin-angiotensin system are 2 impor-\ntant systems that regulate blood pressure.1– 8 Dopamine pro-\nmotes natriuresis, whereas angiotensin II decreases sodium\nexcretion.1– 8 The major D1-like receptor subtype mediating\nthe increase in sodium excretion is probably the D1 receptor,\nwhereas the major angiotensin II receptor mediating the\ndecrease in renal sodium excretion is the AT1 receptor.1– 8 In\nSHRs, renal proximal tubular D1 receptor function is im-\npaired.7,8 Regardless of the mechanism involved in the\ninteraction between D1 and AT1 receptors, the dissociation of\nD1 from AT1 receptors after angiotensin II stimulation may\nallow the AT1 and D1 receptor to exert\ntheir functions\nseparately. Renal proximal tubular D1 receptors are functional\nin WKY but not in SHRs. In contrast, AT1 receptor function\nis enhanced in SHRs.32 The inability of AT1 receptors to\nincrease D1 receptor expression in renal proximal tubules in\nSHRs and a decreased D1 and AT1 receptor interaction could\nlead to enhanced AT1 receptor function.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nAcknowledgments\nThese studies were supported in part by grants from the National\nInstitutes of Health, HL23081, DK39308, HL68686, DK52612,\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nHL074940, HL41618, and National Natural Science Foundation of\nChina, 30470728.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nReferences\n1. Navar LG, Lewis L, Hymel A, Braam B, Mitchell KD. Tubular fluid\nconcentrations and kidney contents of angiotensins I and II in anesthe-\ntized rats. J Am Soc Nephrol. 1994;5:1153–1158.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n2. Haithcock D, Jiao H, Cui XL, Hopfer U, Douglas JG. Renal proximal\ntubular AT2 receptor: signaling and transport. J Am Soc Nephrol. 1999;10\nSuppl 11:S69 –S74.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n3. Inagami T, Kambayashi Y, Ichiki T, Tsuzuki S, Eguchi S, Yamakawa T.\nAngiotensin receptors: molecular biology and signalling. Clin Exp\nPharmacol Physiol. 1999;26:544 –549.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n4. Carey RM, Jin XH, Siragy HM. Role of the angiotensin AT2 receptor in\nblood pressure regulation and therapeutic implications. Am J Hypertens.\n2001;14:98S–102S.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n5. Handa RK, Krebs LT, Harding JW, Handa SE. Angiotensin IV AT4-\nreceptor system in the rat kidney. Am J Physiol. 1998;274:F290 –F299.\n6. Navar LG, Harrison-Bernard LM, Imig JD, Wang CT, Cervenka L,\nMitchell KD. Intrarenal angiotensin II generation and renal effects of AT1\nreceptor blockade. J Am Soc Nephrol. 1999;10(Suppl 12):S266 –S272.\n7. Hussain T, Lokhandwala MF. Renal dopamine receptor function in\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nhypertension. Hypertension. 1998;32:187–197.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n8. Jose PA, Eisner GM, Felder RA. Renal dopamine receptors in health and\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nhypertension. Pharmacol Ther. 1998;80:149 –182.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n9. Narkar VA, Hussain T, Pedemonte C, Lokhandwala MF. Dopamine D2\nreceptor activation causes mitogenesis via p44/42 mitogen-activated\nprotein kinase in opossum kidney cells. J Am Soc Nephrol. 2001;12:\n1844 –1852.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n10. Ricci A, Marchal-Victorion S, Bronzetti E, Parini A, Amenta F, Tayebati\nSK. Dopamine D4 receptor expression in rat kidney: evidence for pre- and\npostjunctional localization. J Histochem Cytochem. 2002;50:1091–1096.\n11. Luippold G, Zimmermann C, Mai M, Kloor D, Starck D, Gross G,\nMuhlbauer B. Dopamine D3 receptors and salt-dependent hypertension.\nJ Am Soc Nephrol. 2001;12:2272–2279.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n12. Asico LD, Ladines C, Fuchs S, Accili D, Carey RM, Semeraro C,\nPocchiari F, Felder RA, Eisner GM, Jose PA. Disruption of the dopamine\nD3 receptor gene produces renin-dependent hypertension. J Clin Invest.\n1998;102:493– 498.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n13. Jose PA, Asico LD, Eisner GM, Pocchiari F, Semeraro C, Felder RA.\nEffects of costimulation of dopamine D1- and D2-like receptors on renal\nfunction. Am J Physiol. 1998;275:R986 –R994.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n14. Chen CJ, Apparsundaram S, Lokhandwala MF. Intrarenally produced\nangiotensin II opposes the natriuretic action of the dopamine-1 receptor\nagonist fenoldopam in rats. J Pharmacol Exp Ther. 1991;256:486 – 491.\n15. Sheikh-Hamad D, Wang YP, Jo OD, Yanagawa N. Dopamine antag-\nonizes the actions of angiotensin II in renal brush-border membrane. Am J\nPhysiol. 1993;264:F737–F743.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n16. Hussain T, Abdul-Wahab R, Kotak DK, Lokhandwala MF. Bro-\nmocriptine regulates angiotensin II response on sodium pump in proximal\ntubules. Hypertension. 1998;32:1054 –1059.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n17. Chen C, Lokhandwala MF. Potentiation by enalaprilat of fenoldopam-\nevoked natriuresis is due to blockade of intrarenal production of angio-\ntensin-II in rats. Naunyn Schmiedebergs Arch Pharmacol. 1995;352:\n194 –200.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n18. Sanada H, Xu J, Watanabe H, Jose PA, Felder RA. Differential\nexpression and regulation of dopamine-1 (D-1) and dopamine-5 (D-5)\nreceptor function in human kidney. Am J Hypertens. 2000;13:156A\n(abstract).\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n19. Zeng C, Luo Y, Asico LD, Hopfer U, Eisner GM, Felder RA, Jose PA.\nPerturbation of D1 dopamine and AT1 receptor interaction in sponta-\nneously hypertensive rats. Hypertension. 2003;42:787–792.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n20. Albrecht FE, Xu J, Moe OW, Hopfer U, Simonds WF, Orlowski J, Jose\nPA. Regulation of NHE3 activity by G protein subunits in renal brush-\nborder membranes. Am J Physiol Regul Integr Comp Physiol. 2000;278:\nR1064 –R1073.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n21. Xu J, Li XX, Albrecht FE, Hopfer U, Carey RM, Jose PA. D1 receptor,\nGs, and Na/H exchanger interactions in the kidney in hypertension.\nHypertension. 2000;36:395–399.\n**BLOCK**fs== 7.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\n22. Zeng C, Yang Z, Wang Z, Jones JE, Wang X, Altea J, Mangrum AJ,\nHoper U, Sibley DR, Eisner GM, Felder RA, Jose PA. Interaction of AT1\nand D5 dopamine receptors in renal proximal tubule cells. Hypertension.\n2005;45:804 – 810.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n23. Parenti A, Cui XL, Hopfer U, Ziche M, Douglas JG. Activation of\nMAPKs in proximal tubule cells from spontaneously hypertensive and\ncontrol Wistar-Kyoto rats. Hypertension. 2000;35:1160 –1166.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n24. Woost PG, Orosz DE, Jin W, Frisa PS, Jacobberger JW, Douglas JG,\nHopfer U. Immortalization and characterization of proximal tubule cells\nderived from kidneys of spontaneously hypertensive and normotensive\nrats. Kidney Int. 1996;50:125–134.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n25. Pedrosa R, Jose PA, Soares-da-Silva P. Defective D1-like receptor-\nmediated inhibition of the Cl-/HCO3- exchanger in immortalized SHR\nproximal tubular epithelial cells. Am J Physiol Renal Physiol. 2004;286:\nF1120 –1126.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n26. Zeng C, Asico LD, Wang X, Hopfer U, Eisner GM, Felder RA, Jose PA.\nAngiotensin II regulation of AT1 and D3 dopamine receptors in renal\nproximal tubule cells of SHR. Hypertension. 2003;41:724 –729.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n27. Park SH, Han HJ. The mechanism of angiotensin II binding downregu-\nlation by high glucose in primary renal proximal tubule cells. Am J\nPhysiol Renal Physiol. 2002;282:F228 –F237.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n28. Zeng C, Asico LD, Hopfer U, Eisner GM, Felder RA, Jose PA. Aberrant\nETB receptor regulation of AT1 receptors in renal proximal tubule cells of\nspontaneously hypertensive rats. Kidney Int. 2005;68:623– 631.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n29. O’Connell DP, Botkin SJ, Ramos SI, Sibley DR, Ariano MA, Felder RA,\nCarey RM. Localization of dopamine D1A receptor protein in rat kidneys.\nAm J Physiol. 1995;268:F1185–F1197.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n30. Zheng S, Yu P, Zeng C, Yang Z, Andrews PM, Felder RA, Jose PA. G12-\nand G13-protein subunit linkage of D5 dopamine receptors in the nephron.\nHypertension. 2003;41:604 – 610.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n31. Levy-Toledano R, Caro LH, Hindman N, Taylor SI. Streptavidin blotting:\na sensitive technique to study cell surface proteins; application to inves-\ntigate autophosphorylation and endocytosis of biotin-labeled insulin\nreceptors. Endocrinology. 1993;133:1803–1808.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n32. Ortiz PA, Garvin JL. Intrarenal transport and vasoactive substances in\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nhypertension. Hypertension. 2001;38:621– 624.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n33. Doris PA. Renal proximal tubule sodium transport and genetic mech-\nanisms of essential hypertension. J Hypertens. 2000;18:509 –519.\n34. Felder CC, Campbell T, Albrecht F, Jose PA. Dopamine inhibits Na-H\nexchanger activity in renal BBMV by stimulation of adenylate cyclase.\nAm J Physiol. 1990;259:F297–F303.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n35. Hu MC, Fan L, Crowder LA, Karim-Jimenez Z, Murer H, Moe OW.\nDopamine acutely stimulates Na/H exchanger (NHE3) endocytosis via\nclathrin-coated vesicles: dependence on protein kinase A-mediated NHE3\nphosphorylation. J Biol Chem. 2001;276:26906 –26915.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n36. Harris RC, Becker BN, Cheng HF. Acute and chronic mechanisms for\nregulating proximal tubule angiotensin II receptor expression. J Am Soc\nNephrol. 1997;8:306 –313.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n37. Ingelfinger JR, Zuo WM, Fon EA, Ellison KE, Dzau VJ. In situ hybrid-\nization evidence for angiotensinogen messenger RNA in the rat proximal\ntubule. An hypothesis for the intrarenal renin angiotensin system. J Clin\nInvest. 1990;85:417– 423.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n38. Baines AD, Chan W. Production of urine free dopamine from DOPA; a\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nmicropuncture study. Life Sci. 1980;26:253–259.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n39. Soares-da-Silva P. Source and handling of renal dopamine: its physio-\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nlogical importance. News Physiol Sci. 1994;9:128 –134.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n40. Wang Z-Q, Siragy HM, Felder RA, Carey RM. Intrarenal dopamine\nproduction and distribution in the rat. Physiological control of sodium\nexcretion. Hypertension. 1997;29:228 –234.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n41. Seikaly MG, Arant BS Jr, Seney FD Jr. Endogenous angiotensin concen-\ntrations in specific intrarenal fluid compartments of the rat. J Clin Invest.\n1990;86:1352–1357.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n42. Cheng HF, Becker BN, Harris RC. Dopamine decreases expression of\ntype-1 angiotensin II receptors in renal proximal tubule. J Clin Invest.\n1996;97:2745–2752.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n43. Zeng C, Sanada H, Watanabe H, Eisner GM, Felder RA, Jose PA.\nFunctional genomics of the dopaminergic system in hypertension. Physiol\nGenomics. 2004;19:233–246.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n44. Watanabe H, Xu J, Bengra C, Jose PA, Felder RA. Desensitization of\nhuman renal D1 dopamine receptors by G protein-coupled receptor kinase\n4. Kidney Int. 2002;62:790 –799.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n45. Kim J, Ahn S, Ren XR, Whalen EJ, Reiter E, Wei H, Lefkowitz RJ.\nFunctional antagonism of different G protein-coupled receptor kinases for\nbeta-arrestin-mediated angiotensin II receptor signaling. Proc Natl Acad\nSci U S A. 2005;102:1442–1447.\n**BLOCK**fs== 7.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\n46. Yu PY, Hopfer U, Felder RA, Jose PA. Increased serine-phosphorylation\nof the D1 receptor in renal proximal tubule cells in hypertension. Am J\nHypertens. 2000;13:12A–13A (abstract).\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n47. Oppermann M, Diverse-Pierluissi M, Drazner MH, Dyer SL, Freedman\nNJ, Peppel KC, Lefkowitz RJ. Monoclonal antibodies reveal receptor\nspecificity among G-protein-coupled receptor kinases. Proc Natl Acad Sci\nU S A. 1996;93:7649 –7654.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n48. Gardner B, Liu ZF, Jiang D, Sibley DR. The role of phosphorylation/\ndephosphorylation in agonist-induced desensitization of D1 dopamine\nreceptor function: evidence for a novel pathway for receptor dephosphor-\nylation. Mol Pharmacol. 2001;59:310 –321.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n49. Yu P, Asico LD, Eisner GM, Hopfer U, Felder RA, Jose PA. Renal\nprotein phosphatase 2A activity and spontaneous hypertension in rats.\nHypertension. 2000;36:1053–1058.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n50. Huang XC, Sumners C, Richards EM. Angiotensin II stimulates protein\nphosphatase 2A activity in cultured neuronal cells via type 2 receptors in\na pertussis toxin sensitive fashion. Adv Exp Med Biol. 1996;396:209 –215.\n51. Gelband CH, Sumners C, Lu D, Raizada MK. Angiotensin receptors and\nnorepinephrine neuromodulation: implications of functional coupling.\nRegul Pept. 1998;73:141–147.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n52. Sanada H, Jose PA, Hazen-Martin D, Yu PY, Xu J, Bruns DE, Phipps J,\nCarey RM, Felder RA. Dopamine-1 receptor coupling defect in renal\ntubule cells in hypertension. Hypertension. 1999;33:\nproximal\n1036 –1042.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n53. Felder RA, Sanada H, Xu J, Yu PY, Wang Z, Watanabe H, Asico LD,\nWang W, Zheng S, Yamaguchi I, Williams SM, Gainer J, Brown NJ,\nHazen-Martin D, Wong LJ, Robillard JE, Carey RM, Eisner GM, Jose\nPA. G protein-coupled receptor kinase 4 gene variants in human essential\nhypertension. Proc Natl Acad Sci U S A. 2002;99:3872–3877.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n54. Felder RA, Kinoshita S, Ohbu K, Mouradian MM, Sibley DR, Monsma\nFJ Jr, Minowa T, Minowa MT, Canessa LM, Jose PA. Organ specificity\nof the dopamine1 receptor/adenylyl cyclase coupling defect in sponta-\nneously hypertensive rats. Am J Physiol. 1993;264:R726 –R732.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n55. Sanada H, Yatabe J, Yoneda M, Hashimoto S, Midorikawa S, Watanabe\nT, Felder RA, Jose PA In vivo targeting of the renal G protein-coupled\nreceptor kinase type 4 (GRK4) with antisense oligonucleotides induces\nnatriuresis in spontaneously hypertensive rats. Circulation. 2002;106\n(Suppl II):II-234 (abstract).",
         "extracted",
         "",
         "Rat Strain Effects of AT1 Receptor Activation on D1 Dopamine Receptors in Immortalized Renal Proximal Tubule Cells",
         "None"
        ],
        [
         "36",
         "002c412eb8a1ec7a73278e2f447f5a0a914e35c8",
         "The expression of cytokeratins in gastrointestinal stromal tumours (GISTs) is rare and may lead to diagnostic confusion when it occurs. This report describes a metastatic GIST that stained strongly for cytokeratins, CD117, and CD34 in a patient who was previously diagnosed with gastric epithelioid angiosarcoma. A review of both tumours showed the same histological and immunohistochemical profiles, and c-kit molecular analysis revealed an insertional mutation at codon 558 of exon 11 in both tumours. Thus, pathologists should be aware that GISTs can occasionally express cytokeratins, and that c-kit mutational investigations may have a key diagnostic role and may prevent diagnostic mistakes that could have important clinical implications.",
         "Giulio Rossi,G. Sartori,Riccardo Valli,F. Bertolini,Nazzarena Bigiani,L. Schirosi,A. Cavazza,G. Luppi",
         "https://jcp.bmj.com/content/jclinpath/58/9/991.full.pdf",
         "\n**BLOCK**fs== 13.3**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\nCASE REPORT\n**BLOCK**fs== 7.6**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nThe value of c-kit mutational analysis in a cytokeratin\npositive gastrointestinal stromal tumour\nG Rossi, G Sartori, R Valli, F Bertolini, N Bigiani, L Schirosi, A Cavazza, G Luppi\n. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nThe expression of cytokeratins in gastrointestinal stromal\ntumours (GISTs) is rare and may lead to diagnostic confusion\nwhen it occurs. This report describes a metastatic GIST that\nstained strongly for cytokeratins, CD117, and CD34 in a\npatient who was previously diagnosed with gastric epithe-\nlioid angiosarcoma. A review of both tumours showed the\nsame histological and immunohistochemical profiles, and\nc-kit molecular analysis revealed an insertional mutation at\ncodon 558 of exon 11 in both tumours. Thus, pathologists\nshould be aware that GISTs can occasionally express\ncytokeratins, and that c-kit mutational\ninvestigations may\nhave a key diagnostic role and may prevent diagnostic\nmistakes that could have important clinical implications.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nG astrointestinal stromal tumours (GISTs) are the most\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\ncommon mesenchymal neoplasms of the digestive\ntract.1 The identification of KIT (CD117) as a specific\nimmunohistochemical marker,2 3 together with the discovery\nof gain of function mutations in the protooncogene c-kit in\nmost GISTs4–6 represent crucial steps in the definition of this\ntumour as a biologically distinctive malignancy and, most\nimportantly, in the development of an alternative therapeutic\napproach using targeted treatment with a specific KIT\ninhibitor,\nimatinib mesylate (formerly known as STI571,\nGlivecH).7\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n‘‘Gastrointestinal stromal tumours are the most common\nmesenchymal neoplasms of the digestive tract’’\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nIn routine practice, when faced with a soft tissue tumour of\nthe gastrointestinal tract, surgical pathologists usually use a\nsmall immunohistochemical panel of antibodies to support\nthe morphological picture, including CD117, CD34, smooth\nmuscle actin, desmin, and S-100 protein.8 In fact, almost all\nGISTs are CD117 positive and about 60–70% are CD34\nimmunopositive, whereas the other markers are usually\nnegative.8 In this context, immunostaining for cytokeratins\nmay be helpful to rule out the possibility of a sarcomatoid\ncarcinoma, because GISTs are usually negative for epithelial\nmarkers.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nHere, we describe a gastric GIST initially diagnosed as\nepithelioid angiosarcoma which metastasised to the pelvis\nand strongly expressed cytokeratins and CD117. Molecular\nanalysis revealed the presence of a c-kit mutation in exon 11\nin both tumours, thus confirming the diagnosis of GIST. The\nexceptionally rare occurrence of cytokeratin immunostaining\nin GISTs is also discussed.\n**BLOCK**fs== 7.6**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nJ Clin Pathol 2005;58:991–993. doi: 10.1136/jcp.2004.024364\n**BLOCK**fs== 8.5**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nand abdominal discomfort resulting from a painful mass in\nthe right iliac fossa. Her medical history revealed that four\nyears previously she had undergone total gastrectomy plus\nomentectomy at another hospital for a gastric malignancy\nfor which she\ndiagnosed as epithelioid angiosarcoma,\nreceived adjuvant chemotherapy (doxorubicin and ifosfa-\nmide). Based on her previous diagnosis, metastatic disease\nwas clinically suspected. Abdominal ultrasonography and a\ncomputerised tomography scan confirmed the presence of a\nlarge (8 cm across) solid mass in the right iliac region and\na partially cystic mass (6 cm) in the left one. Radiological\ninvestigations revealed no significant signs in the other\nabdominal organs or regional lymph nodes. A chest x ray was\nunremarkable. Tumour markers (CA125 and carcinoembryo-\nnic antigen) and laboratory studies were negative. The\npatient then underwent complete surgical excision of the\npelvic masses and right adnexectomy. The surgical specimen,\nroutinely formalin fixed and paraffin wax embedded, grossly\nshowed a grey/whitish cut surface with bloody areas. At\nthe tumour appeared as a proliferation of\nhistology,\npolygonal to round epithelioid cells, with abundant lightly\neosinophilic cytoplasm and centrally located nucleus (fig 1A).\nClear perinuclear vacuoles were seen. Mitotic activity was\nvery high (18 mitoses/650 high power field) and foci of\npunctate necrosis were found.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nImmunohistochemistry revealed that the neoplastic ele-\nments were strongly positive for CD117 (polyclonal antibody,\nA4502; 1/200 dilution; no antigen retrieval; Dako, Glostrup,\nDenmark) (fig 1B), CD34 (monoclonal QB-END/10; 1/40\ndilution; antigen retrieval, microwave; Novocastra, Newcastle\nupon Tyne, UK) bcl-2 (monoclonal; 1/10 dilution; antigen\nretrieval, microwave; Ventana, Tucson, Arizona, USA), AE1/\nAE3 (monoclonal; prediluted; antigen retrieval, protease;\nVentana), MNF116 (monoclonal; 1/1500 dilution; antigen\nretrieval, protease; Dako), CAM5.2 (monoclonal; 1/50 dilu-\ntion; no antigen retrieval; Becton Dickinson, San Jose`,\nCalifornia, USA) (fig 1C), but did not stain for high molecular\nweight cytokeratins (monoclonal 34bE12; 1/500 dilution;\nantigen retrieval, microwave; Dako), CD31 (monoclonal JC/\n70A; prediluted; antigen retrieval, protease; Ventana),\nepithelial membrane antigen (monoclonal E29; 1/300 dilu-\ntion; no antigen retrieval; Dako), desmin (monoclonal D33;\n1/10 dilution; antigen retrieval, microwave; Dako), smooth\nmuscle actin (monoclonal 1A4; 1/20 dilution; no antigen\nretrieval; Biogenex, San Ramon, California, USA), S-100\n(polyclonal; 1/5 dilution; no antigen retrieval; NeoMarkers,\nFreemont, California, USA), chromogranin (monoclonal\nDAK-A3; 1/100 dilution; antigen retrieval, microwave;\nDako), HMB-45 (monoclonal HMB-45; prediluted; antigen\nretrieval, microwave; Ventana), or h-caldesmon (monoclonal\nhCD; 1/100 dilution; antigen retrieval, microwave; Dako).\n**BLOCK**fs== 8.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nCASE REPORT\nIn July 2003, a 32 year old woman originating from Ghana\nwas admitted to our hospital as a result of a lack of appetite\n**BLOCK**fs== 5.5**b== 0.6**t== 0.2**l== 1.0**r== 0.0**\nP\nr\no\nt\ne\nc\nt\ne\nd\nb\ny\nc\no\np\ny\nr\ni\ng\nh\nt\n,\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\nl\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 5.5**b== 0.5**t== 0.4**l== 1.0**r== 0.0**\nn\nc\nu\nd\nn\ng\nf\no\nr\nu\ns\ne\ns\nr\ne\nl\na\nt\ne\nd\nt\no\nt\ne\nx\nt\n**BLOCK**fs== 5.5**b== 0.4**t== 0.5**l== 1.0**r== 0.0**\na\nn\nd\nd\na\nt\na\nm\nn\nn\ng\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\n,\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\nA\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\nI\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\n(\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\nU\nP\nM\nC\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\n)\n**BLOCK**fs== 2.5**b== 0.4**t== 0.5**l== 1.0**r== 0.0**\n.\n**BLOCK**fs== 5.5**b== 0.3**t== 0.6**l== 1.0**r== 0.0**\nt\nr\na\ni\nn\nn\ng\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\n,\na\nn\nd\ns\ni\nm\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\ni\nl\na\nr\n**BLOCK**fs== 5.5**b== 0.2**t== 0.7**l== 1.0**r== 0.0**\nt\ne\nc\nh\nn\no\no\ng\ni\ne\ns\n.\n**BLOCK**fs== 2.5**b== 0.2**t== 0.8**l== 1.0**r== 0.0**\nl\n**BLOCK**fs== 8.5**b== 0.6**t== 0.1**l== 0.5**r== 0.1**\neach dNTP, 20pM of each primer, and 0.25 U Platinum Taq\npolymerase. The PCR reaction was carried out on a Uno II\nThermoblock (Biometra, Gottingen, Germany). Initial dena-\nturation at 94˚C for three minutes was followed by 41 cycles\nof denaturation at 95˚C for one minute, annealing at 53–58˚C\nfor 40 seconds, and extension at 72˚C for 35 seconds, with a\nfinal extension step of five minutes at 72˚C. The amplified\nDNA was eletrophoresed on a 1% low melt agarose gel for one\nhour. The amplification products were then excised from the\ngel and purified using Wizard PCR Preps-DNA purification\nsystem (Promega Corp, Madison, Wisconsin, USA), as\nindicated by the manufacturer. The PCR products were then\nsequenced in both directions with the BigDye Terminator\n(Applied Biosystems, Weiterstadt, Germany) sequencing kit,\nusing the same primers as those used for PCR. PCR products\nwere finally purified by Centri-Sep spin columns and\nsubsequently analysed using the ABIPrism 310 sequence\nanalyser (Applied Biosystems). The following oligonucleotide\nprimers were used to amplify c-kit exons 9 and 11: exon 9\n(forward, 59-ATG CTC TGC TTC TGT ACT GCC-39; reverse, 59-\nCAG AGC CTA AAC ATC CCC TTA-39; 238 bp; 58˚C annealing\ntemperature), exon 11 (forward, 59-CTA TTT TTC CCT TTC\nTCC CC-39; reverse, 59-TAC CCA AAA AGG TGA CAT GG-39;\n193 bp; 53˚C annealing temperature).\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nreview of\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nMolecular analysis revealed a c-kit mutation consisting of\nan insertion of the TCC nucleotides at codon 558 (fig 2). The\nhistological\nthe previously diagnosed gastric\ntumour showed a similar epithelioid malignancy with\nimmunohistochemical features identical to those seen in\nthe pelvic metastatic deposits. Most importantly, the original\ngastric tumour revealed the same insertion mutation found\nin the metastatic deposits. Thus, a diagnosis of metastatic\nGIST was rendered and the patient started treatment with\nGlivecH (400 mg/day). She is alive and well at 22 months\nfollow up.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\ntyrosine kinase KIT,\n**BLOCK**fs== 8.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nDISCUSSION\nGISTs are well defined clinicopathological tumours and are\nthe most frequent mesenchymal malignancies found in the\ndigestive tract.1 9 They are characterised by the expression of\nthe type III\nthe\nprotooncogene c-kit, recognised by the antibody CD117.1–6 8 9\nMost importantly, c-kit activating gain of function mutations\noccur in most GISTs, whereas a small subset of GISTs are KIT\nnegative and lack c-kit mutations.9 These KIT negative\ntumours show activating mutations of the PDGFRA gene,\nencoding the type III tyrosine kinase PDGFR-a.10\n**BLOCK**fs== 8.5**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\nthe product of\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nT T\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nT A\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nC C C\nT\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nC\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nA\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nC\nAA\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nA\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nC C\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nT T\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nC C C G\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nC C\nT\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nG\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nT T\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nC\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nA\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nC\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nA\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nN\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\nC\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nG\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nT\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nN\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nG\n**BLOCK**fs== 5.7**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nNN\n**BLOCK**fs== 5.5**b== 0.6**t== 0.2**l== 1.0**r== 0.0**\nP\nr\no\nt\ne\nc\nt\ne\nd\nb\ny\nc\no\np\ny\nr\ni\ng\nh\nt\n,\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\nl\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 5.5**b== 0.5**t== 0.4**l== 1.0**r== 0.0**\nn\nc\nu\nd\nn\ng\nf\no\nr\nu\ns\ne\ns\nr\ne\nl\na\nt\ne\nd\nt\no\nt\ne\nx\nt\n**BLOCK**fs== 5.5**b== 0.4**t== 0.5**l== 1.0**r== 0.0**\na\nn\nd\nd\na\nt\na\nm\nn\nn\ng\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\n,\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\nA\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\nI\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\n(\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\nU\nP\nM\nC\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\n)\n**BLOCK**fs== 2.5**b== 0.4**t== 0.5**l== 1.0**r== 0.0**\n.\n**BLOCK**fs== 5.5**b== 0.3**t== 0.6**l== 1.0**r== 0.0**\nt\nr\na\ni\nn\nn\ng\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\n,\na\nn\nd\ns\ni\nm\n**BLOCK**fs== 7.6**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nFigure 1 (A) The tumour appeared as an epithelioid cell proliferation\nconsistent with gastrointestinal stromal tumour and was strongly\nimmunopositive for (B) CD117 and (C) cytokeratin.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSeveral 5 mm thick sections obtained from a representative\nparaffin wax embedded block were dewaxed by xylene and\ntumour DNA was extracted using a laser capture micro-\ndissection method (LaserScissor-PRO300; Olympus, Tokyo,\nJapan). Microdissected tumour cells were subjected to\nproteinase K treatment in extraction buffer (50mM Tris/HCl\n(pH 8.0), 1mM EDTA, 0.5% Tween 20) and then incubated\novernight at 37˚C. Polymerase chain reaction (PCR) was\nperformed in 10 ml reactions containing 1.0 ml DNA, 10mM\nTris/HCl (pH 8.3), 40mM KCl, 1.0–1.5mM MgCl2, 200mM\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nCC\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nC\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nAA\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nC\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nAA\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nCC\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nTT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nCCC\nA\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nG\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nA\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nC\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nC\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nA\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\nT\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\nA\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\nC\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\nA\n**BLOCK**fs== 5.7**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\nT\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\ni\nl\na\nr\n**BLOCK**fs== 5.5**b== 0.2**t== 0.7**l== 1.0**r== 0.0**\nt\ne\nc\nh\nn\no\no\ng\ni\ne\ns\n.\n**BLOCK**fs== 2.5**b== 0.2**t== 0.8**l== 1.0**r== 0.0**\nl\n**BLOCK**fs== 7.6**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nFigure 2 Molecular analysis revealed the presence of an insertion\nmutation on codon 558 of the c-kit exon 11 (arrow).\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nTake home messages\nN We describe a metastatic gastrointestinal stromal\ntumour (GIST) that stained strongly for cytokeratins,\nCD117, and CD34 in a patient previously diagnosed\nwith gastric epithelioid angiosarcoma\nN Both tumours showed the same histological and\nimmunohistochemical profiles, and c-kit molecular\nanalysis revealed the same mutation in both tumours\nN Thus, pathologists should be aware that GISTs can\noccasionally express cytokeratins and that c-kit muta-\ntional analysis may help in diagnosis and prevent\nmistakes that could have important clinical implications\n**BLOCK**fs== 8.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nMost GISTs display a spindle, epithelioid, or mixed\n(spindle/epithelioid) cell morphology, but they may show a\nbroad spectrum of histological growth patterns that make the\ndifferential diagnosis difficult, and may be confused with\nsmooth muscle tumours, desmoids, neural tumours, neuro-\nendocrine tumours,\ninflammatory myofibroblastic tumour\nand fibroid polyp, synovial sarcoma, sarcomatoid carcinoma,\nand angiosarcoma.11 For this reason,\nimmunostaining is\nmandatory, not only to achieve the diagnosis, but also\nbecause of the clinical implications of a diagnosis of GIST. As\nsuggested by Fletcher et al,8 the diagnosis of GIST should be\nbased not only on morphological grounds, but staining with a\nsmall panel of antibodies should also be performed to rule\nout other mimicking tumours. Here, we report a rare case of\nmetastatic GIST strongly expressing cytokeratins, in addition\nto CD34 and CD117, and originally thought to be a gastric\nepithelioid angiosarcoma.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nNga and colleagues12 recently reported a case of cytokeratin\npositive pleural metastatic GIST initially suspected to be lung\ncarcinoma and found two other GISTs showing focal, dot-like\nimmunoreactivity for CAM5.2 in a series of 41 GISTs. Of note,\nall these cytokeratin positive GISTs had a gastric origin.\nAlthough extremely rare, this has been described previously\nby Miettinen et al,13 who found two keratin 18 positive\ntumours in a series of 57 anorectal GISTs. In both studies, no\nGIST showed immunostaining for other epithelial markers\n(such as AE1/AE3 and epithelial membrane antigen).\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n‘‘Insertional mutation in codon 558 of c-kit exon 11 has\nbeen reported previously in gastrointestinal stromal\ntumours, and seems to be related to a good clinical\nresponse rate with STI571’’\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nApart from GIST, CD117 expression has been reported in\nseveral other tumours, mainly as a result of the presence of\ndifferent commercially available antibodies against KIT and\nthe use of inappropriate technical protocols.14 Among tumours\ncoexpressing CD34 and cytokeratins, CD117 immunoreactivity\nhas been reported in angiosarcoma only.15 Even though\nangiosarcomas may primarily arise from the stomach, they\nlack c-kit mutations and generally carry a worse prognosis than\nthat of GIST.15\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nMutational analysis of c-kit appears to be extremely\nhelpful, if not mandatory, in confirming the diagnosis of\nGIST when this tumour shows an unusual presentation\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\non clinical, morphological, and/or\nimmunohistochemical\ngrounds, as in our case. In addition, insertional mutation in\ncodon 558 of c-kit exon 11 has been reported previously in\nGIST, and seems to be related to a good clinical response rate\nwith STI571.6 16\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nFinally, pathologists should be aware that, although\nunusual, GISTs may express cytokeratins, and this should\nbe kept in mind when dealing with small biopsies to prevent\nan erroneous diagnosis.\n**BLOCK**fs== 7.6**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n. . . . . . . . . . . . . . . . . . . . .\nAuthors’ affiliations\nG Rossi, G Sartori, R Valli, N Bigiani, L Schirosi, Integrated Department\nof Diagnostic and Laboratory Services and Legal Medicine, Section of\nPathologic Anatomy, University of Modena and Reggio Emilia, via del\nPozzo 71, 41100 Modena, Italy\nF Bertolini, G Luppi, Department of Oncology and Haematology, Section\nof Oncology, University of Modena and Reggio Emilia\nA Cavazza, Department of Pathology, S. Maria Nuova Hospital, Viale\nRisorgimento 80, 42100 Reggio Emilia, Italy\n**BLOCK**fs== 7.6**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nThe patient gave her full permission for this case report to be published\n**BLOCK**fs== 7.6**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nCorrespondence to: Dr G Rossi, Dipartimento Integrato di Servizi\nDiagnostici, di Laboratorio e di Medicina Legale, Sezione di Anatomia\nPatologica, Universita` di Modena e Reggio Emilia, via del Pozzo 71,\n41100 Modena, Italy; rossi.giulio@unimo.it\n**BLOCK**fs== 7.6**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nAccepted for publication 17 January 2005\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nREFERENCES\n**BLOCK**fs== 6.6**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n1 Miettinen M, Lasota J. Gastrointestinal stromal tumors—definition, clinical,\nhistological, immunohistochemical, and molecular genetic features and\ndifferential diagnosis. Virchows Arch 2001;438:1–12.\n**BLOCK**fs== 6.6**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n2 Kindblom LG, Remotti HE, Aldenborg F, et al. Gastrointestinal pacemaker cell\n**BLOCK**fs== 6.6**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\ntumor (GIPACT). Gastrointestinal stromal tumors show phenotypic\ncharacteristics of the interstitial cells of Cajal. Am J Pathol\n1998;152:1259–69.\n**BLOCK**fs== 6.6**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n3 Sarlomo-Rikala M, Kovatich AJ, Barusevicius A, et al. CD117: a sensitive\nmarker for gastrointestinal stromal tumors that is more specific than CD34.\nMod Pathol 1998;11:728–34.\n**BLOCK**fs== 6.6**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n4 Hirota S, Isozaki K, Moriyama Y, et al. Gain-of-function mutations of c-kit in\n**BLOCK**fs== 6.6**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nhuman gastrointestinal stromal tumors. Science 1998;279:577–80.\n**BLOCK**fs== 6.6**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n5 Rubin BP, Singer S, Tsao C, et al. KIT activation is a ubiquitous feature of\n**BLOCK**fs== 6.6**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\ngastrointestinal stromal tumors. Cancer Res 2001;61:8118–21.\n**BLOCK**fs== 6.6**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n6 Heinrich MC, Corless CL, Demetri GD, et al. Kinase mutations and imatinib\nresponse in patients with metastatic gastrointestinal stromal tumor. J Clin\nOncol 2003;21:4342–9.\n**BLOCK**fs== 6.6**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n7 Joensuu H, Roberts PJ, Sarlomo-Rikala M, et al. Effect of the tyrosine kinase\ninhibitor STI571 in a patient with metastatic gastrointestinal stromal tumor.\nN Engl J Med 2001;344:1052–6.\n**BLOCK**fs== 6.6**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n8 Fletcher CDM, Berman JJ, Corless C, et al. Diagnosis of gastrointestinal\nstromal tumors: a consensus approach. Hum Pathol 2002;33:459–65.\n9 Corless C, Fletcher JA, Heinrich MC. Biology of gastrointestinal stromal\n**BLOCK**fs== 6.6**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\ntumors. J Clin Oncol 2004;22:3813–25.\n**BLOCK**fs== 6.6**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n10 Heinrich MC, Corless C, Duensing A, et al. PDGFRA activating mutations in\n**BLOCK**fs== 6.6**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\ngastrointestinal stromal tumors. Science 2003;299:708–10.\n11 Suster S. Gastrointestinal stromal tumors. Semin Diagn Pathol\n**BLOCK**fs== 6.6**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1996;13:297–313.\n**BLOCK**fs== 6.6**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n12 Nga ME, Wong ASC, Wee A, et al. Cytokeratin expression in gastrointestinal\n**BLOCK**fs== 6.6**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nstromal tumours: a word of caution. Histopathology 2002;40:480–1.\n13 Miettinen M, Furlong M, Sarlomo-Rikala M, et al. Gastrointestinal stromal\n**BLOCK**fs== 6.6**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\ntumors, intramural leiomyomas, and leiomyosarcomas in the rectum and\nanus. A clinicopathologic, immunohistochemical, and molecular genetic study\nof 144 cases. Am J Surg Pathol 2001;25:1121–33.\n**BLOCK**fs== 6.6**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n14 Hornick JL, Fletcher CDM. Immunohistochemical staining for KIT (CD117) in\n**BLOCK**fs== 6.6**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nsoft tissue sarcomas is very limited in distribution. Am J Clin Pathol\n2002;117:188–93.\n**BLOCK**fs== 6.6**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n15 Miettinen M, Sarlomo-Rikala M, Lasota J. KIT expression in angiosarcomas\nand fetal endothelial cells: lack of mutations of exon 11 and exon 17 of c-kit.\nMod Pathol 2000;13:536–41.\n**BLOCK**fs== 6.6**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n16 Iijima S, Maesawa C, Sato N, et al. Gastrointestinal stromal tumour of the\n**BLOCK**fs== 6.6**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\noesophagus: significance of immunohistochemical and genetic analyses of the\nc-kit gene. Eur J Gastroenterol Hepatol 2002;14:445–8.\n**BLOCK**fs== 5.5**b== 0.6**t== 0.2**l== 1.0**r== 0.0**\nP\nr\no\nt\ne\nc\nt\ne\nd\nb\ny\nc\no\np\ny\nr\ni\ng\nh\nt\n,\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\nl\n**BLOCK**fs== 2.5**b== 0.6**t== 0.4**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 5.5**b== 0.5**t== 0.4**l== 1.0**r== 0.0**\nn\nc\nu\nd\nn\ng\nf\no\nr\nu\ns\ne\ns\nr\ne\nl\na\nt\ne\nd\nt\no\nt\ne\nx\nt\n**BLOCK**fs== 5.5**b== 0.4**t== 0.5**l== 1.0**r== 0.0**\na\nn\nd\nd\na\nt\na\nm\nn\nn\ng\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\n,\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\nA\n**BLOCK**fs== 2.5**b== 0.4**t== 0.6**l== 1.0**r== 0.0**\nI\n**BLOCK**fs== 5.5**b== 0.3**t== 0.6**l== 1.0**r== 0.0**\nt\nr\na\ni\nn\nn\ng\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\ni\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\n,\na\nn\nd\ns\ni\nm\n**BLOCK**fs== 2.5**b== 0.3**t== 0.7**l== 1.0**r== 0.0**\ni\nl\na\nr\n**BLOCK**fs== 5.5**b== 0.2**t== 0.7**l== 1.0**r== 0.0**\nt\ne\nc\nh\nn\no\no\ng\ni\ne\ns\n.\n**BLOCK**fs== 2.5**b== 0.2**t== 0.8**l== 1.0**r== 0.0**\nl\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\n(\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\nU\nP\nM\nC\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 1.0**r== 0.0**\n)\n**BLOCK**fs== 2.5**b== 0.4**t== 0.5**l== 1.0**r== 0.0**\n.",
         "extracted",
         "",
         "The value of c-kit mutational analysis in a cytokeratin positive gastrointestinal stromal tumour",
         "None"
        ],
        [
         "37",
         "002cc193a7a313a08711d45ac3bea51509e0edc0",
         "The present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated rat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes were placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-dependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15-45 min rather than 46-120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity transiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-phase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed oocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat oocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent level of aged (120 min) or enucleated (+60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2 kinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were treated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated oocytes. A significantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control oocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus cell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage. These results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for their inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemicals such as MG132 is in part effective for rat oocytes to promote PCC and further development.",
         "J. Ito,M. Hirabayashi,Megumi Kato,A. Takeuchi,Mayumi Ito,M. Shimada,S. Hochi",
         "https://rep.bioscientifica.com/downloadpdf/journals/rep/129/2/1290171.pdf",
         "\n**BLOCK**fs== 17.9**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\nContribution of high p34cdc2 kinase activity to premature\nchromosome condensation of injected somatic cell nuclei\nin rat oocytes\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\nJunya Ito1, Masumi Hirabayashi1,2, Megumi Kato1,3, Ayumu Takeuchi4, Mayumi Ito4,\nMasayuki Shimada5 and Shinichi Hochi4\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\n1National Institute for Physiological Sciences, Okazaki 444-8787, Japan, 2The Graduate University of\nAdvanced Studies, Okazaki 444-8787, Japan, 3CREST of Japan Science and Technology Agency, Kawaguchi\n332-0012, Japan, 4Faculty of Textile Science and Technology, Shinshu University, Ueda 386-8567, Japan and\n5Graduate School of Biosphere Science, Hiroshima University, Higashi-Hiroshima 739-8528, Japan\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nCorrespondence should be addressed to S Hochi; Email: shochi@shinshu-u.ac.jp\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nAbstract\n**BLOCK**fs== 9.0**b== 0.3**t== 0.4**l== 0.1**r== 0.1**\nThe present study was undertaken to clarify the relationship between the p34cdc2 kinase activity of in vitro-aged or enucleated\nrat oocytes and the premature chromosome condensation (PCC) of microinjected cumulus cell nuclei. Wistar rat oocytes\nwere placed in vitro up to 120 min after the animal was killed. The p34cdc2 kinase activity of the oocytes decreased in a time-\ndependent manner. The incidence of PCC was higher when nuclear injection into intact oocytes was completed in 15– 45 min\nrather than 46– 120 min. When rat oocytes were enucleated for subsequent nuclear injection, the p34cdc2 kinase activity tran-\nsiently increased soon after enucleation but drastically decreased after 30 min. Removal of the cytoplasm instead of the meta-\nphase-plate did not affect the p34cdc2 kinase activity even after 60 min. PCC occurred in intact and cytoplasm-removed\noocytes but not in enucleated oocytes. In contrast, oocytes from BDF1 mice exhibited a p34cdc2 kinase level twice that of rat\noocytes and supported PCC despite enucleation. The p34cdc2 kinase level of intact rat oocytes was reduced to the equivalent\nlevel of aged (120 min) or enucleated (1 60 min) oocytes by a 45 min treatment with roscovitine, an inhibitor of p34cdc2\nkinase. None of the roscovitine-treated oocytes supported PCC while half of the control oocytes did. When rat oocytes were\ntreated with MG132, a proteasome inhibitor, delayed inactivation of the p34cdc2 kinase was observed in the MG132-treated\noocytes. A signiﬁcantly higher proportion of the MG132-treated oocytes supported PCC when compared with the control\noocytes. Moreover, a higher proportion of MG132-treated and enucleated oocytes carried two pseudo-pronuclei after cumulus\ncell injection and developed to the two-cell stage when compared with the enucleated oocytes at the telophase-II stage.\nThese results suggest that the decreased level of p34cdc2 kinase activity in aged or enucleated rat oocytes is responsible for\ntheir inability to support PCC of microinjected donor cell nuclei and that inhibition of p34cdc2 kinase inactivation by chemi-\ncals such as MG132 is in part effective for rat oocytes to promote PCC and further development.\nReproduction (2005) 129 171–180\n**BLOCK**fs== 10.5**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nIntroduction\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSuccessful animal cloning by somatic cell nuclear transfer\n(NT) has been reported in large domestic species (Wilmut\net al. 1997, Kato et al. 1998, Baguisi et al. 1999, Onishi\net al. 2000, Galli et al. 2003). As for small rodents,\nWakayama et al. (1998) were the ﬁrst to produce cloned\nmice using G0/G1-phase cumulus cells, and very recently\nZhou et al. (2003) reported the production of fertile cloned\nrats using M-phase fetal ﬁbroblast cells. However, the suc-\ncess rates of producing cloned rodents by NT are still low.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nIn the mouse, nuclei injected into enucleated oocyte\ncytoplasm are directly exposed to reprogramming factors,\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nresulting in premature chromosome condensation (PCC),\nwith subsequent multiple pseudo-pronuclei formation fol-\nlowing activation (Wakayama et al. 1998, 1999, Ogura\net al. 2000). It is considered in rodents, at least, that PCC\nis a critical aspect contributing to successful cloning\n(Wakayama et al. 1999, Ogura et al. 2000). However,\nPCC is not essential for somatic cell cloning in all species,\nsince telophase-II\n(TII) oocytes have promoted embryo\ndevelopment in goats (Baguisi et al. 1999) and cattle (Kur-\nosaka et al. 2002).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThere is a major difference between mouse and rat, in\nthe behavior of somatic nuclei upon exposure to oocyte\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.1**r== 0.7**\nq 2005 Society for Reproduction and Fertility\nISSN 1470–1626 (paper) 1741–7899 (online)\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.7**r== 0.1**\nDOI: 10.1530/rep.1.00431\nOnline version via www.reproduction-online.org\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\ncytoplasm. We have reported that the PCC of both mouse\nand rat cumulus cell nuclei occurred in enucleated mouse\noocytes but not at all\nin enucleated rat oocytes\n(Hirabayashi et al. 2003a). Even if cumulus nuclei were\ninjected into the rat oocytes without enucleation, the rate\nof PCC formation was less than half (Hirabayashi et al.\n2003b).\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nWe also reported that\n**BLOCK**fs== 9.5**b== 0.1**t== 0.2**l== 0.1**r== 0.5**\nthe ability of rat oocytes to\npromote PCC of the injected nuclei was dependent on\nthe characteristics of the oocytes, such as age (4- to 5-\nweek-old, superior to .10-week-old rats) or strain of the\ndonor rats (Wistar and LEW, superior to the Donryu and\nF344 strains) and the timing of oocyte recovery (14 h post-\nhuman chorionic gonadotropin (hCG), superior to 17 h\npost hCG) (Hirabayashi et al. 2003b). In the same study,\nthe time management of NT completion (less than 45 min\nafter donor animal killing) and addition to the oocyte-\nhandling media\nof N-acetylleucylleucylnorleucinal\n(ALLN), a neutral cysteine protease inhibitor, were also\nfound important for intact rat oocytes to promote PCC.\nThe successful production of cloned rats (Zhou et al.\n2003) may be attributed to injecting the metaphase-\narrested fetal ﬁbroblasts into oocytes treated with MG132,\na proteasome inhibitor, followed by quick removal of the\nrecipient metaphase-plate. Therefore, it is a possibility that\nrecipient oocytes arrested at the metaphase-II (MII) stage\nby the inhibition of protein degradation have essential\nroles in nuclear reprogramming including PCC formation.\nOvulated rat oocytes are known to activate spon-\ntaneously but abortively during in vitro culture (Zernicka-\nGoetz 1991). We also conﬁrmed that more than half of\nthe oocytes from Wistar rats progressed to anaphase-II\n(AII)/TII stages at 70 min after killing of\nthe donor rats\n(Kato et al. 2001). In vertebrate oocytes, it has been con-\nsidered that high activity of maturation promoting factor\n(MPF) affects the chromosome condensation and the\nmaintenance of the meiotic spindle, both of which are\ninvolved in the meiotic arrest at\nthe MII stage (Jones\n2004). MPF is a heterodimer composed of p34cdc2 kinase\nand cyclin B1, controlling the cell cycle. When the\nactivity was inactivated by the degradation of cyclin B\n(Nurse 1990), oocytes were released from the arrest at the\nMII stage. The p34cdc2 kinase in oocytes was maintained\nat a high level until ovulation in many species (Kubiak\net al. 1993, Naito et al. 1995, Abrieu et al. 2001, Ito &\nShimada 2005), suggesting that p34cdc2 kinase activity in\nrat ovulated oocytes was rapidly decreased in vitro. More-\nover, destruction of\nthe nuclear envelope in mitotic\nsomatic cells occurred during the G2/M-phase transition\nby p34cdc2 kinase-related phosphorylation of lamin, which\nis one of\nthe nuclear membrane proteins (Peter et al.\n1990). It has been reported that ALLN, which was used in\nour previous study (Hirabayashi et al. 2003b), has an\neffect on the inhibition of cyclin B destruction (Sherwood\net al. 1993). Therefore, it is possible that the low rate of\nPCC of injected somatic cell nuclei resulted from rapidly\ndecreasing p34cdc2 kinase activity in rat ovulated oocytes,\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nwhile direct evidence that p34cdc2 kinase activity is\ninvolved in the PCC incidence has not yet been shown.\nClarifying the relationship between p34cdc2 kinase activity\nand PCC incidence would contribute to understanding the\nreprogramming mechanism in mammalian\nnuclear\noocytes.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nIn the present study, two experiments were conducted\nto examine the effects of in vitro aging and enucleation of\nrat oocytes on the kinetics of p34cdc2 kinase and the inci-\ndence of PCC of microinjected cumulus cell nuclei, with\nsome comparisons with mouse oocytes. In addition, two\nmore experiments were conducted to explain a possible\nrole of p34cdc2 kinase for promoting PCC, by treatment of\nthe rat oocytes with roscovitine, an inhibitor of p34cdc2\nkinase (Lazar et al. 2002), and MG132, a proteasome\ninhibitor which has an effect on the inhibition of cyclin\nB1 degradation (Josefsberg et al. 2000).\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nMaterials and Methods\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nOocyte preparation\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nSpeciﬁc-pathogen-free Crj:Wistar female rats (3 –5 weeks\nold) or Crj:BDF1 female mice (7 weeks old) were pur-\nchased from Charles River Japan, Inc. (Kanagawa, Japan).\nThey were housed under controlled lighting (lights on at\ntemperature (23 ^ 2 8C) and humidity\n0600 –1800 h),\n(55 ^ 5%), with free access\n(MF;\nOriental Yeast Co., Tokyo, Japan) and water. Rats were\nsuperovulated by i.p. injections of 30 IU equine chorio-\nnic gonadotropin (eCG) (Nippon Zenyaku, Co., Fukush-\nima, Japan) and 30 IU hCG (Sankyo Yell Yakuhin, Co.,\nTokyo, Japan) at 48 h intervals (Hirabayashi et al. 2001).\nMice were superovulated as well with 5 IU eCG and 5\nIU hCG.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.7**r== 0.1**\nto laboratory diet\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nFourteen hours after the hCG injection, cumulus –oocyte\ncomplexes (COCs) were collected from the oviductal\nampullae of donor rats or mice with modiﬁed Krebs–\nRinger bicarbonate solution (mKRB) in Experiments 1 and\n3 (Toyoda & Chang 1974) and modiﬁed rat one-cell\nembryo culture medium (mR1ECM) in Experiments 2, 4\nand 5 (Oh et al. 1998) supplemented with 0.1% hyaluroni-\ndase (Sigma-Aldrich Corp., St Louis, MO, USA) respect-\nively. Two to ﬁve minutes later, the denuded oocytes were\nwashed three times with fresh mKRB or R1ECM and kept in\nthe same medium at 37 8C until being subjected to the\ntreatments. The time of killing of the donor animals was\ndeﬁned as 0 min, and it usually took 10 –15 min from the\nanimal killing to the preparation of denuded oocytes.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nMeasurement of p34cdc2 kinase activity\nThe p34cdc2 kinase assay was performed according to the\nmethod as described by Ito et al. (2001) with some modiﬁ-\ncations. In brief, the oocytes were washed several times\nwith PBS and placed into cell lysis buffer composed of\n20 mM Tris, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1%\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nwww.reproduction-online.org\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTriton X-100, 2.5 mM sodium pyrophosphate, 1 mM b-gly-\ncerophosphate, 1 mM Na3VO4, 1 mg/ml\nleupeptin (all\nchemicals from Cell Signaling Technology, Beverly, MA,\nUSA) and 1 mM phenylmethylsulfonyl ﬂuoride (Sigma-\nAldrich). The oocyte lysates were stored at 2 80 8C until\nthe ELISA assay.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n(pH 7.5)\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n(MBL, Nagoya,\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nThe lysate (ten oocytes/5 ml cell lysis buffer) was mixed\nwith 45 ml of kinase assay buffer-A composed of 25 mM\nHepes buffer\nJapan), 10 mM\nMgCl2 (MBL), 10% (v/v) mouse vimentin peptide solution\n(SLYSSPGGAYC) (MBL) and 0.1 mM ATP (Sigma-Aldrich),\nand the mixture was incubated for 30 min at 30 8C. The\nreaction was terminated by the addition of 200 ml PBS\ncontaining 50 mM EGTA (MBL). The phosphorylation of\nmouse vimentin peptides was detected using an ELISA\nanalysis (MESACUP cdc2 kinase assay kit\n(code no.\n5234); MBL). Data were expressed in terms of the strength\nof p34cdc2 kinase activity in denuded rat oocytes 15 min\n(Experiments 1, 3 and 4) or 45 min (Experiment 2) after\nanimal killing.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nAssessment of nuclear conﬁguration\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nRat oocytes immediately after recovery and 30, 60, 90\nand 120 min after culture in mKRB were stained with\n5 mg/ml Hoechst 33342 (Sigma-Aldrich). Their nuclear\nconﬁguration was then assessed under UV light at 330–\n380 nm, according to the classiﬁcations reported pre-\nviously (Kato et al. 2001): MII, characterized by a meta-\nphase spindle with the ﬁrst polar body; AII, characterized\nby elongated spindle without the second polar body; and\nTII, characterized by elongated spindle with extrusion of\nthe second polar body.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nNuclear injection and PCC evaluation\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nCumulus cells dispersed from COCs by the hyaluronidase\ntreatment were placed in mKRB (Experiments 1 and 3) or\nR1ECM (Experiments 2, 4 and 5) containing 22 mM Hepes\nand 5 mM NaHCO3 (abbreviated hereafter as Hepes-\nmKRB and Hepes-R1ECM respectively). An aliquot (2 ml)\nof the cumulus cell suspension was transferred to 10 ml of\nHepes-mKRB or Hepes-R1ECM supplemented with 12%\nw/v 360 kDa polyvinylpyrrolidone (ICN Pharmaceuticals,\nInc., Costa Mesa, CA, USA). Nuclei were removed from\ncumulus cells and gently aspirated in and out of the injec-\ntion pipette (7 –10 mm) until\ntheir nuclei were largely\ndevoid of visible cytoplasmic materials. Each nucleus was\ninjected into a separate enucleated oocyte within 5 min of\nits isolation, as described by Kimura & Yanagimachi\n(1995). One hour after the nuclear injection, the oocytes\nwere stained with 5 mg/ml Hoechst 33342 and evaluated\nfor incidence of PCC under UV light at 330–380 nm, as\ndescribed previously (Hirabayashi et al. 2003a,b). The\ninjected somatic cell nucleus was distinguished from the\noocyte nucleus by the position of the opening in the zona\npellucida at the nuclear microinjection site.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.1**l== 0.5**r== 0.3**\nExperimental design\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nExperiment 1: effect of in vitro aging\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nThe effect of in vitro aging on the p34cdc2 kinase activity\nand the PCC incidence was investigated using rat oocytes.\nTen oocytes for each test were sampled for\nin vitro\np34cdc2 kinase assay immediately after recovery (15 min\ngroup), and after culture for 30 min (45 min group), 60 min\n(75 min group) or 105 min (120 min group) in mKRB at\n37 8C in 5% CO2 in air. Nuclear conﬁguration of\nthe\noocytes was also examined immediately after recovery\nand every subsequent 30 min up to 120 min. The other\noocytes were placed in Hepes-mKRB, and the cumulus\ncell nuclei were then injected into the oocytes up to\n120 min. The injected oocytes were divided into two\ngroups based on the time from the animal killing to NT\ncompletion (15 –45 min group and 46 – 120 min group).\nThe incidence of PCC was evaluated 1 h after the last\ninjection among each group.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nExperiment 2: effect of enucleation\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nThe effect of oocyte enucleation on the p34cdc2 kinase\nactivity and the PCC incidence was investigated using\nboth rat and mouse oocytes. For the in vitro p34cdc2\nkinase assay, oocytes derived from the same donor rats\nand mice were allocated to the non-enucleation group\nand the enucleation group, and the non-enucleated and\nenucleated oocytes were sampled 45 min after the animal\nkilling. Enucleation was performed by aspiration of the\nmetaphase-plate with a part of the volume (,10–20%) of\ncytoplasm in Hepes-R1ECM containing\nsurrounding\n5 mg/ml cytochalasin B (Sigma-Aldrich). In addition to the\nnon-enucleation and enucleation groups, denuded rat\noocytes were allocated to a cytoplasm-removal group\n(a part of the cytoplasm was mechanically aspirated as a\nsham control.). All the treatments were completed within\n45 min. The oocyte samples were harvested at 45, 75, 90\nand 105 min (0, 30, 45 and 60 min after enucleation\nrespectively). For PCC evaluation, nuclear injection into\nnon-enucleated and enucleated mouse and rat oocytes,\nand into cytoplasm-removed rat oocytes, was completed\nwithin 45 min after animal killing.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.2**\nExperiment 3: effect of roscovitine treatment\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nWe investigated whether the decreased level of p34cdc2\nkinase activity in non-enucleated rat oocytes negatively\naffected the PCC incidence of injected cell nuclei. Ros-\ncovitine (Sigma-Aldrich) dissolved at 10 mM in DMSO\nand stored at 2 80 8C, a speciﬁc inhibitor of p34cdc2\nkinase, was added to all media throughout the process of\noocyte recovery and the in vitro culture at the ﬁnal con-\ncentration of 150 mM, as reported by Lazar et al. (2002).\nOocytes were sampled for in vitro p34cdc2 kinase assay at\n15 (immediately after recovery), 45, 75 and 120 min as in\nExperiment 1. Because the 45 min treatment of rat oocytes\nwith 150 mM roscovitine decreased their p34cdc2 kinase\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwww.reproduction-online.org\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nactivity to an equivalent level of oocytes 120 min after in\nvitro aging or 60 min after enucleation, nuclear injection\ninto the roscovitine-treated oocytes was performed within\n45 min, and the incidence of PCC was then evaluated\nafter a further 1 h.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nExperiment 4: effect of MG132 treatment\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nWe investigated whether the treatment of rat oocytes with a\nproteasome inhibitor, by which the p34cdc2 kinase activity\nwould be maintained, could support the PCC incidence of\nthe injected donor cell nuclei. MG132 (Sigma-Aldrich), an\ninhibitor which can block cyclin B1 degradation, was used\nat the ﬁnal concentration of 5 mM, as reported by Josefsberg\net al. (2000). Non-enucleated oocytes were exposed to\nMG132,\nthe process of oocyte recovery to\nin vitro culture or nuclear injection. Oocyte sampling for\nin vitro p34cdc2 kinase assay was performed at 15 min\n(immediately after recovery), 45, 75 and 120 min. Nuclear\ninjection into the MG132-treated oocytes and PCC evalu-\nation were performed as in Experiment 3.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\nthroughout\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nExperiment 5: effect of PCC on NT embryo\ndevelopment\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nIn the ﬁrst group, oocytes exposed to MG132 from oocyte\nrecovery to just before enucleation were used as recipi-\nents. From the result of Experiment 4, MG132 treatment is\nexpected to support PCC even in enucleated oocytes,\nbecause the p34cdc2 kinase activity was maintained at a\nhigh level during the treatment. The procedure of enuclea-\ntion was as described above. The cytoplasts were then\ninjected with cumulus cell nuclei. One hour later, a high\nincidence of PCC was expected.\nIn the second group,\noocytes with the extrusion of a second polar body were\nselected from 2 h cultures of non-treated oocytes, and\nthen enucleated and nuclear injected as well. The reduced\nincidence of PCC under a lowered MPF was expected. For\nboth groups, the oocytes 1 h after nuclear injection were\nactivated by two direct current pulses (100 V/mm, 99 ms)\nand 4 h treatment with 2 mM 6-dimethylaminopurine\n(6-DMAP)\nin mR1ECM medium. The\nreconstructed oocytes were cultured in 100 ml microdrops\nof mR1ECM at 37 8C in 5% CO2 in air, and number of\noocytes forming one or two pseudo-pronuclei and cleav-\ning to the two-cell stage was recorded at 6 and 24 h of\nculture respectively.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n(Sigma-Aldrich)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nStatistical analysis\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nEach experiment had at least three replicates. Data on\np34cdc2 kinase activity were compared by ANOVA and\nFisher’s protected least signiﬁcant difference test using the\nStatView program (Abacus Concepts, Inc., Berkeley, CA,\nUSA). Data on PCC incidence and NT embryo develop-\nment were compared by Fisher’s exact probability test. A\nvalue of P , 0.05 was chosen as an indication of statisti-\ncal signiﬁcance.\n**BLOCK**fs== 10.5**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nResults\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nExperiment 1: effect of in vitro aging\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nThe time-dependent change in p34cdc2 kinase activity in\nrat oocytes is shown in Fig. 1. The p34cdc2 kinase activity\nof\nrat oocytes decreased signiﬁcantly at 45 min (after\n30 min cultivation) and more signiﬁcantly at 75 min (after\n60 min cultivation). The decreased activity was maintained\nuntil 120 min (after 105 min cultivation). Probably due to\nthe decrease of p34cdc2 kinase activity, the release from\nmeiotic arrest of ovulated oocytes occurred in a time-\ndependent manner, as shown in Fig. 2. Within 75 min\nafter animal killing, more than half of the oocytes pro-\ngressed beyond the MII stage. At 135 min, most of the\noocytes were at either the AII or the TII stage. The pro-\nportion of in vitro-aged rat oocytes promoting PCC of the\ninjected cumulus cell nuclei is shown in Table 1. When\nnuclear injection was completed within 45 min from ani-\nmal killing, a positive sign of PCC was observed in 32.5%\nof the NT oocytes. On the other hand, PCC occurred in\nonly 2.6% of the oocytes into which cumulus cell nuclei\nwere injected between 46 and 120 min.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nExperiment 2: effect of oocyte enucleation\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe activities of p34cdc2 kinase in non-enucleated and\nenucleated oocytes within 45 min were compared in rats\nand mice, as shown in Fig. 3. In both species, there was\na slight decrease in the p34cdc2 kinase activity after enu-\ncleation in both rat and mouse oocytes. A considerable\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nFigure 1 Kinetics of p34cdc2 kinase activity in rat oocytes during in\nvitro culture. 1Data are expressed as relative percentage of the level\nof p34cdc2 kinase activity in rat oocytes immediately after recovery\n(15 min after the animal killing; deﬁned as 100%). a – cDifferent super-\nscripts on S.E.M. bars denote signiﬁcant differences among the culture\nperiods (P , 0.05).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nwww.reproduction-online.org\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nFigure 3 Changes in p34cdc2 kinase activity by enucleation in rat and\nmouse oocytes. 1Data are expressed as relative percentage of the\nlevel of p34cdc2 kinase activity in non-enucleated rat oocytes har-\nvested at 45 min from the animal killing (deﬁned as 100%). *denotes\nsigniﬁcant difference from the corresponding value in the rat\n(P , 0.05).\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\ntreatment (enucleation or cytoplasm removal) is shown in\nTable 2. In rats, despite little reduction of p34cdc2 kinase\nactivity, few of the oocytes that either had a small portion\nof cytoplasm-removed (17.1%) or the non-treated controls\n(28.2%) could support PCC at 105 min after animal\nin the\nkilling. However, PCC was not observed at all\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nFigure 2 Time-dependent changes of nuclear conﬁguration during\nin vitro culture of ovulated rat oocytes. MII, metaphase-II; AII, ana-\nphase-II; TII, telophase-II. 1Data are expressed as relative percentage\nof the level of p34cdc2 kinase activity in non-enucleated rat oocytes\nharvested at 45 min from the animal killing (deﬁned as 100%).\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nfound in the p34cdc2 kinase activity\ndifference was\nbetween rat and mouse oocytes, because the mouse\noocytes carried a p34cdc2 kinase activity 2.5 times that of\nrat oocytes either immediately or up to 60 min after enu-\ncleation or cytoplasm removal and compared with non-\ntreated, age-matched controls, as shown in Fig. 4. The\nactivities of p34cdc2 kinase were measured in rat oocytes\nimmediately after or 60 min after one of three treatments\n(enucleated, cytoplasm-removed and non-treated), as\nshown in Fig. 4. Surprisingly, a higher p34cdc2 kinase\nactivity was detected in the oocytes immediately after\nenucleation, although the p34cdc2 kinase activity of\noocytes immediately after cytoplasm removal was not\nfrom that of non-treated oocytes. The p34cdc2\ndifferent\nin enucleated oocytes dramatically\nactivity\nkinase\ndecreased within 30 min after enucleation. At 45 min\nafter enucleation, the activity was further decreased and\nthereafter maintained until 60 min. However, the activity\nin non-treated or cytoplasm-removed oocytes did not sig-\nniﬁcantly differ by 60 min. The proportion of rat and\nmouse oocytes promoting PCC in relation to mechanical\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nTable 1 PCC of rat cumulus cell nuclei injected into rat oocytes kept\nin vitro for different periods.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nElapsed time until NT completiona\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nInjectedb\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nPCC-positive (%)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n15–45 min\n46–120 min\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n13 (32.5)c\n1 (2.6)d\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\na Time from donor animal killing to NT completion.\nb Number of oocytes injected, survived and evaluated.\nc – d Different superscripts denote signiﬁcant difference (P , 0.05).\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nFigure 4 Effect of enucleation and cytoplasm removal on the kinetics\nof p34cdc2 kinase activity in rat oocytes. 1Data are expressed as rela-\ntive percentage of the level of p34cdc2 kinase activity in non-enu-\ncleated rat oocytes harvested at 45 min from the animal killing\n(deﬁned as 100%). a, b; c, d; e, f; g, hDifferent superscripts on S.E.M.\nbars denote signiﬁcant differences between the experimental groups\nin each culture period (P , 0.05). *An asterisk shows a signiﬁcant\ndifference compared with the activity in enucleated oocytes at 0 min\n(P , 0.05). NE, not examined.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwww.reproduction-online.org\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nTable 2 PCC of rat and mouse cumulus cell nuclei injected into rat\nand mouse oocytes.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nRat\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nMouse\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nInjecteda PCC-positive Injecteda PCC-positive\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nTreatments\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n(%)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n(%)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nNon-treated\nCytoplasm-removed\nEnucleated\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n13 (28.2)b\n7 (17.1)b\n0 (0)c\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nNEd\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n29 (96.7)\n—\n27 (93.1)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\na Number of oocytes injected, survived and evaluated.\nb – c Different superscripts denote signiﬁcant difference (P , 0.05).\nd Not examined.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nenucleated oocytes where p34cdc2 kinase activity was sig-\nniﬁcantly decreased at 60 min after completion of enu-\ncleation. In contrast, PCC of mouse cumulus cell nuclei\noccurred in most of the mouse oocytes before enucleation\n(96.7%) and even after enucleation (93.1%).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nExperiment 3: effect of roscovitine treatment\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nThe time-dependent decrease of p34cdc2 kinase activity in\nroscovitine-treated rat oocytes is shown in Fig. 5. The\nactivity\nlinearly\nin roscovitine-treated oocytes was\ndecreased until 75 min and maintained until 120 min.\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nAt 45, 75 and 120 min, each activity in the roscovitine-\ntreated oocytes was signiﬁcantly lower than that of non-\ntreated control oocytes. The result of PCC incidence in the\noocytes treated with or without roscovitine for 45 min is\nshown in Table 3. None of the roscovitine-treated oocytes\npromoted PCC, although 41.7% of non-treated oocytes\npromoted it.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nExperiment 4: effect of MG132 treatment\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nThe activities of p34cdc2 kinase in MG132-treated rat\noocytes are shown in Fig. 6. The level of p34cdc2 kinase\nactivity still decreased in the MG132-treated oocytes\nafter 30 min of culture, but were signiﬁcantly greater\nthan in control oocytes. The p34cdc2 kinase activity was\nmaintained during the further incubation up to 105 min.\nThe kinase activities of MG132-treated oocytes at 60\nand 105 min cultivation were signiﬁcantly greater than\nthose of control oocytes. As shown in Table 4, the pro-\nportion of MG132-treated oocytes promoting PCC\n(66.0%) was higher than that of the non-treated oocytes\n(46.5%) when the nuclear\ninjection was completed\nwithin 45 min.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nExperiment 5: effect of PCC on NT embryo\ndevelopment\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nthe reconstructed embryos\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nAfter oocyte activation with the electrical stimulus plus 6-\nDMAP,\nfrom both groups\n(MG132-treated MII oocytes and non-treated TII oocytes\nat the enucleation) formed one or two pseudo-pronuclei\n(91.7 and 87.5% in total respectively). After activation, a\ngreater proportion of oocytes enucleated at the MII stage\ntended to form two pseudo-pronuclei compared with\nthose enucleated at the TII stage. A signiﬁcantly higher\nproportion of the NT embryos in the MG132-treated group\nvs the non-treated group (59.1 vs 7.1%) developed to the\ntwo-cell stage.\n**BLOCK**fs== 10.5**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nDiscussion\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nRat oocytes have poor potential\nfor promoting PCC of\ninjected somatic cell nuclei, and the rate was markedly\ndecreased in aged or enucleated oocytes (Hirabayashi\nIn the present study, rat oocytes into\net al. 2003a,b).\nwhich cumulus cell nuclei were injected within 45 min\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nTable 3 PCC of rat cumulus cell nuclei injected into roscovitine-\ntreated rat oocytes.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nFigure 5 Effect of roscovitine treatment on the kinetics of p34cdc2\nkinase activity in rat oocytes. 1Data are expressed as relative percen-\ntage of the level of p34cdc2 kinase activity in non-treated rat oocytes\nimmediately after recovery (15 min after the animal killing; deﬁned as\n100%). a – c; d – fDifferent superscripts on S.E.M. bars denote signiﬁcant\ndifferences among different culture periods within each treatment\ngroup (P , 0.05). *Asterisk denotes a signiﬁcant difference between\nthe treatment groups (P , 0.05).\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nRoscovtine treatmenta\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nInjectedb\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nPCC-positive (%)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\nþ\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.8**r== 0.1**\n15 (41.7)c\n0 (0)d\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\na Oocytes were treated with 150 mM roscovitine for 45 min, and\nnuclear injection was completed during the treatment.\nb Number of oocytes injected, survived and evaluated.\nc – d Different superscripts denote signiﬁcant difference (P , 0.05).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nwww.reproduction-online.org\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 9.5**b== 0.4**t== 0.1**l== 0.5**r== 0.1**\nthe non-treated oocytes during the same period, may\ninﬂuence the outcome of PCC. The high concentration of\nroscovitine affects the other cyclin-dependent protein\nkinases such as cdk2 (De Azevado et al. 1997). The\nunknown pathway and/or chemical toxicity of roscovitine\ncannot be ruled out\nfor complete suppression of PCC\ninduction. Moreover, the level of activity in rat oocytes\nwas less than half compared with that in mouse oocytes,\nwhich had a high potential for promoting PCC. Judging\nfrom these results, we estimated that the high activity of\np34cdc2 kinase is associated with promoting PCC of\ninjected somatic cell nuclei, and the low level of p34cdc2\nkinase activity is one of the reasons for the poor potential\nto promote PCC in rat oocytes compared with that of\nmouse oocytes. The reason why the potential\nlevel of\np34cdc2 kinase activity is different between mouse and rat\noocytes remains unclear. Our previous study indicated, at\nleast, that the PCC-inducing potential is determined by\nthe origin of the recipient cytoplasm, not by the origin of\nin the interspecies NT experiment\nthe donor nucleus,\nbetween mouse and rat\nIn\n(Hirabayashi et al. 2003a).\nsomatic cell division, dissolution of the nuclear envelope\noccurs during G2/M transition in mitosis (Nurse 1990) and\nis induced by phosphorylation of lamin, one of the major\nnuclear membrane proteins (Peter et al. 1990). In oocytes,\np34cdc2 kinase had the potential to phosphorylate lamins,\ninducing meiotic breakdown of the nuclear envelope and\nthe meiotic resumption (Dessev & Goldman 1988). There-\nfore, it is possible that p34cdc2 kinase plays an important\nrole in the destruction of the nuclear membrane of the\ninjected somatic cell through phosphorylation of lamin.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nOne possible pathway known for inactivation of p34cdc2\nkinase is the calmodulin-dependent pathway via Ca2þ.\nWe have reported that the Ca2þ-dependent pathway trig-\ngers the degradation of cyclin B and results in inactivation\nof p34cdc2 kinase in porcine oocytes (Ito et al. 2003,\n2004a). In Xenopus oocytes (Lorca et al. 1994, Morin et al.\n1994), activation of calmodulin-dependent protein kinase\nII induces the degradation of cyclin B via the ubiquitin\nproteasome pathway. Based on these literature sources\nand preliminary data, we have investigated the effect of a\nproteasome inhibitor MG132 on p34cdc2 kinase activity\nand PCC incidence. The PCC incidence in rat oocytes was\nsigniﬁcantly improved by MG132 treatment (Table 4). This\nresult matched well with the report by Josefsberg et al.\n(2000), who demonstrated that MG132-treated rat oocytes\ncontained a high amount of cyclin B1.\nIn the present\nstudy, MG132 also provided a suppressive effect on inacti-\nvation of p34cdc2 kinase (Fig. 6). Use of ALLN, a neutral\ncysteine protease inhibitor, was also slightly effective for\nimproving the PCC incidence in rat oocytes (Hirabayashi\net al. 2003b). Thus, the Ca2þ-dependent degradation path-\nway, which may induce a decrease in the cyclin B1 level,\nis activated in aged rat oocytes, which results in a low\nlevel of p34cdc2 kinase activity, and a low potential\nto promote PCC. Our unpublished data (M Hirabayashi\nand J Ito, unpublished data) showing that the decreased\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nFigure 6 Effect of MG132 treatment on the kinetics of p34cdc2 kinase\nactivity in rat oocytes. 1Data are expressed as relative percentage of\nthe level of p34cdc2 kinase activity in non-treated rat oocytes immedi-\nately after recovery (15 min after the animal killing; deﬁned as\n100%). a, b; c – eDifferent superscripts on S.E.M. bars denote signiﬁcant\ndifferences among culture periods within each treatment group\n(P , 0.05). *Asterisk denotes a signiﬁcant difference between the\ntreatment groups (P , 0.05).\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nIn mouse oocytes, PCC was initiated at\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\npromoted PCC (frequency 28–47% in Tables 1 –4), but\nthe incidence of PCC decreased with oocyte aging with a\nsigniﬁcant decrease in p34cdc2 kinase activity (Table 1,\nFig. 1).\nleast\n30 min after nuclear injection and lasted until 2 h (Chen\net al. 2004), therefore the timing of PCC observation in\nthe present study (60 min after nuclear injection) might be\noptimal. When oocytes are treated with roscovitine,\nwhich decreases the level of p34cdc2 kinase activity, PCC\nformation was never observed even though nuclear injec-\ntion is completed within 45 min. Roscovitine is widely\nused as cyclin-dependent kinase inhibitor\nto suppress\np34cdc2 kinase in rodent oocytes (Phillips et al. 2002).\nTherefore, the rapid decrease of p34cdc2 kinase activity in\nthe roscovitine-treated oocytes found between 45 and\n75 min (Fig. 5), when compared with the slow decrease in\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nTable 4 PCC of rat cumulus cell nuclei injected into MG 132-treated\nrat oocytes.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nMG 132 treatmenta\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nInjectedb\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nPCC-positive (%)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.9**\nþ\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\n20 (46.5)c\n31 (66.0)d\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\na Oocytes were treated with 5 mM MG 132 for 45 min, and nuclear\ninjection was completed during the treatment.\nb Number of oocytes injected, survived and evaluated.\nc – d Different superscripts denote signiﬁcant difference (P , 0.05).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwww.reproduction-online.org\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.2**\nTable 5 The effect of PCC induction on the pseudo-pronuclei (PN) formation and the ﬁrst cleavage of reconstructed rat embryos.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nRecipient oocytes\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nNuclear injected\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nMG-treated MIIa\nNon-treated TIIb\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n1PNc\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n8 (36.4%)\n11 (78.6%)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nPN\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n2PNc\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n14 (63.6%)\n3 (21.4%)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nTotal\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n22 (91.7%)\n14 (87.5%)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nTwo-cell\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n13 (59.1%)d\n1 (7.1%)e\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\na Oocytes were treated with 5 mM MG132 until just before enucleation after oocyte recovery.\nb Oocytes were cultured for about 2 h and only oocytes with a completely extruded polar body (the TII stage) were used.\nc Percentage of reconstructed embryos with 1 or 2 pronuclei.\nd – e Different superscripts denote signiﬁcant difference (P , 0.05).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nconcentration of extracellular free calcium in the handling\nmedium partially inhibits spontaneous activation of rat\noocytes, support this hypothesis.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nThe other pathway for inactivation by phosphorylation\nof 14Thr and 15Tyr of p34cdc2 kinase has been reported in\naged porcine oocytes (Kikuchi et al. 2000). In the same\nstudy, it has been reported that caffeine suppressed the\nfrom MPF to pre-MPF (14Thr/15Tyr-phosphorylated\nshift\np34cdc2 kinase þ cyclin B1) in the aged porcine oocytes.\nOkadaic acid is also known as a protein phosphatase\ninhibitor which prevents phosphorylation of 14Thr and\n15Tyr of p34cdc2 kinase (Rime et al. 1995). In our prelimi-\nnary study, PCC incidence of injected cumulus cell nuclei\nwas slightly improved in rat oocytes treated with okadaic\nacid (M Hirabayashi and J Ito, unpublished data). There-\nfore, the phosphorylation of p34cdc2 kinase as well as the\ncyclin degradation may be involved in spontaneous acti-\nvation of rat oocytes during in vitro culture.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nThe level of p34cdc2 kinase activity in non-treated rat\noocytes was similar to that of cytoplasm-removed or enu-\ncleated oocytes in the case where the oocytes were\nsampled immediately after the treatment\n(Fig. 4). The\nmechanical puncture of the oocyte membrane and the\naspiration of the cytoplasm or karyoplast did not inﬂuence\nthe p34cdc2 kinase activity immediately after the enuclea-\ntion procedure beyond the level expected from the rela-\ntive volume of aspirated cytoplasm (, 10–20%). These\nresults strongly suggest that active p34cdc2 kinase is not\naccumulated near the metaphase spindles in rat oocytes.\nThere was signiﬁcant difference in the p34cdc2 kinase\nactivity between 45 and 75 min in both non-treated and\nenucleated groups (Fig. 4). A dramatic decrease in the\nactivity of p34cdc2 kinase was observed in the rat oocytes\nenucleated and cultured for 30 min compared with the\nnon-treated group (Fig. 4), suggesting that\nthe oocyte\nmetaphases per se are involved in the maintenance of\np34cdc2 kinase activity. We have previously reported in\nporcine oocytes that activation of p34cdc2 kinase was\ndependent on the synthesis of cyclin B and that the syn-\nthesis was induced by the activation of mitogen-activated\nprotein (MAP) kinase (Ito et al. 2004b). The MAP kinase\nsuppressed Myt1 activity that phosphorylated p34cdc2\nkinase at 14Thr and 15Tyr (Mueller et al. 1995, Palmer et al.\n1998). Because activated MAP kinase is localized on the\nmeiotic spindle in porcine oocytes (Lee et al. 2000), it is\n**BLOCK**fs== 9.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\npossible that enucleation results in the loss of MAP kinase\nfrom the recipient oocytes.\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nNuclear\n**BLOCK**fs== 9.5**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\ninjection into the MG132-treated, MII-enu-\ncleated rat oocytes resulted in a higher development to\nthe two-cell stage when compared with non-treated, TII-\nenucleated rat oocytes after 24 h cultivation (Table 5).\nThe favorable circumstance of recipient cytoplasm with\na high MPF activity must induce PCC and promote sub-\nsequent development of NT embryos. Wakayama &\nYanagimachi\nthe injected\n(2001) reported in mice that\ncumulus cell nuclei undergoing PCC transformation to\nthe two pseudo-pronuclei and such NT embryos are able\nto develop to full-term. In contrast, a different situation\nhas been reported in the cloning of large domestic ani-\nmals. The somatic cell NT in the livestock species is suc-\ncessfully reproducible when enucleated and activated\noocytes with a low MPF activity are used as recipient\ncytoplasm (Baguisi et al. 1999, Kurosaka et al. 2002,\nShin et al. 2002). Such a difference between rodents\nand large domestic animals remains unclear, but our\nthat\npresent data suggest\ninduction of PCC in recon-\nstructed embryos\nrelated to the high activity of\nis\np34cdc2 kinase, which seems to play a key role in the\nsuccessful rat cloning.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nIn conclusion,\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nthe decreased level of p34cdc2 kinase\nactivity in aged or enucleated rat oocytes is responsible\nfor their inability to promote PCC of microinjected donor\ncell nuclei. Inhibition of p34cdc2 kinase inactivation by\nchemicals such as MG132 is in part effective for rat\noocytes to promote PCC and cleavage. These data may\nexpand our existing knowledge on characteristics of rat\noocytes and contribute to the development of a rat NT\nprotocol using G0/G1-stage donor cells.\n**BLOCK**fs== 10.5**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nAcknowledgements\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nThe authors thank Mrs A Ishikawa and Miss A Takata for their\ntechnical assistance. This work was supported in part by\nGrant-in-Aids for the Scientiﬁc Research (No. 15082211, No.\n16300139; to M H and S H), and the 21st Century COE Pro-\ngram (to S H) from the Ministry of Education, Culture, Sports,\nScience and Technology of Japan, and a Research Fellowship\nfor Young Scientists (No. 8254; to J I.) from the Japan Society\nfor the Promotion of Science.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nwww.reproduction-online.org\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 10.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nReferences\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nAbrieu A, Doree M & Fisher D 2001 The interplay between\ncyclin-B-Cdc2 kinase (MPF) and MAP kinase during maturation of\noocytes. Journal of Cell Science 114 257–267.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nBaguisi A, Behboodi E, Melican DT, Pollock JS, Destrempes MM,\nCammuso C et al. 1999 Production of goats by somatic cell NT.\nNature Biotechnology 5 456–461.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nChen SU, Chang CY, Lu CC, Hsieh FJ, Ho HN & Yang YS 2004\nMicrotubular spindle dynamics and chromosome complements\nfrom somatic cell nuclei haploidization in mature mouse oocytes\nthe derived embryos. Human\nand developmental potential of\nReproduction 19 1181–1188.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nDe Azevedo WF, Leclerc S, Meijer L, Havlicek L, Strnad M & Kim SH\n1997 Inhibition of cyclin-dependent kinases by purine analogues:\ncrystal structure of human cdk2 complexed with roscovitine. Euro-\npean Journal of Biochemistry 243 518 –526.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nDessev G & Goldman R 1988 Meiotic breakdown of nuclear envel-\nope in oocytes of Spisula solidissima involves phosphorylation and\nrelease of nuclear lamin. Developmental Biology 130 543–550.\nGalli C, Lagutina I, Crotti G, Colleoni S, Turini P, Ponderato N et al.\n2003 Pregnancy: a cloned horse born to its dam twin. Nature 424\n635.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nHirabayashi M, Ito K, Sekimoto A, Hochi S & Ueda M 2001 Pro-\nduction of transgenic rats using young Sprague–Dawley females\ntreated with PMSG and hCG. Experimental Animals 50 365–369.\nHirabayashi M, Kato M, Ishikawa A & Hochi S 2003a Factors inﬂuen-\ncing chromosome condensation and development of cloned rat\nembryos. Cloning and Stem Cells 5 35–42.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nHirabayashi M, Kato M, Takeuchi A, Ishikawa A & Hochi S 2003b\nFactors affecting premature chromosome condensation of cumulus\ncell nuclei injected into rat oocytes. Journal of Reproduction and\nDevelopment 49 121–126.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nIto J & Shimada M 2005 Timing of MAP kinase inactivation effects\non emission of polar body in porcine oocytes activated by Ca2þ\nionophore. Molecular Reproduction and Development 70 64– 69.\nIto J, Shimada M & Terada T 2001 Progression of nuclear maturation\nand p34cdc2 kinase activity in porcine oocytes during in vitro cul-\nture in different media. Journal of Mammalian Ova Research 18\n39– 43.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nIto J, Shimada M & Terada T 2003 Effect of protein kinase C activator\non mitogen-activated protein kinase and p34cdc2 kinase activity\nduring parthenogenetic activation of porcine oocytes by calcium\nionophore. Biology of Reproduction 69 1675– 1682.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nIto J, Kawano N, Hirabayashi M & Shimada M 2004a The role of cal-\ncium/calmodulin-dependent protein kinase II (CaMKII) on inacti-\nvation of MAP kinase and p34cdc2 kinase during fertilization and\nactivation in pig oocytes. Reproduction 128 409 –415.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nIto J, Shimada M & Terada T 2004b Mitogen-activated protein kinase\nkinase inhibitor suppresses cyclin B1 synthesis and reactivation of\np34cdc2 kinase, which improves pronuclear\nformation rate in\nmatured porcine oocytes activated by Ca2þ ionophore. Biology of\nReproduction 70 797–804.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nJones KT 2004 Turning it on and off: M-phase promoting factor\nduring meiotic maturation and fertilization. Molecular Human\nReproduction 10 1– 5.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nJosefsberg LB, Galiani D, Dantes A, Amsterdam A & Dekel N 2000\nThe proteasome is involved in the ﬁrst metaphase-to-anaphase\ntransition of meiosis in rat oocytes. Biology of Reproduction 62\n1270–1277.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nKato M, Hirabayashi M, Aoto T, Ito K, Ueda M & Hochi S 2001\nStrontium-induced activation regimen for rat oocytes in somatic\ncell nuclear transplantation. Journal of Reproduction and Develop-\nment 47 407–413.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nKato Y, Tani T, Sotomaru Y, Kurokawa K, Kato J & Doguchi H et al.\n1998 Eight calves cloned from somatic cells of a single adult.\nScience 282 2095–2098.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nKikuchi K, Naito K, Noguchi J, Shimada A, Kaneko H, Yamashita M\net al. 2000 Maturation/M-phase promoting factor: a regulator of\naging in porcine oocytes. Biology of Reproduction 63 715– 722.\nKimura Y & Yanagimachi R 1995 Mouse oocytes injected with testi-\ncular spermatozoa or round spermatids can develop into normal\noffspring. Development 121 2397–2405.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nKubiak JZ, Weber M, de Pennart H, Winston NJ, Maro B 1993 The\nmetaphase II arrest in mouse oocytes is controlled through micro-\ntubule-dependent destruction of cyclin B in the presence of CSF.\nEMBO Journal 12 3773–3778.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nKurosaka S, Nagao Y, Minami N, Yamada M, Imai H 2002 Depen-\ndence of DNA synthesis and in vitro development of bovine NT\nembryos on the stage of the cell cycle of donor cells and recipient\ncytoplasts. Biology of Reproduction 67 643–647.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nLazar S, Galiani D & Dekel N 2002 cAMP-dependent PKA negatively\nregulates polyadenylation of c-mos mRNA in rat oocytes. Molecu-\nlar Endocrinology 16 331 –341.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nLee J, Miyano T & Moor RM 2000 Localisation of phosphorylated\nMAP kinase during the transition from meiosis I to meiosis II in pig\noocytes. Zygote 8 119–125.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nLorca T, Abrieu A, Means A & Doree M 1994 Ca2þ is involved\nthrough type II calmodulin-dependent protein kinase in cyclin\ndegradation and exit from metaphase. Biochimica et Biophysica\nActa 1223 325 –332.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nMorin N, Abrieu A, Lorca T, Martin F & Doree M 1994 The proteol-\nysis-dependent metaphase to anaphase transition: calcium/calmo-\ndulin-dependent protein kinase II mediates onset of anaphase in\nextracts prepared from unfertilized Xenopus eggs. EMBO Journal\n13 4343–4352.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nMueller PR, Coleman TR, Kumagai A & Dunphy WG 1995 Myt1: a\nmembrane-associated inhibitory kinase that phosphorylates Cdc2\non both threonine-14 and tyrosine-15. Science 270 86–90.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nNaito K, Hawkins C, Yamashita M, Nagahama Y, Aoki F & Kohmoto\nK et al. 1995 Association of p34cdc2 and cyclin B1 during meiotic\nmaturation in porcine oocytes. Developmental Biology 168\n627 –634.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nNurse P 1990 Universal control mechanism regulating onset of\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nM-phase. Nature 344 503–508.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nOgura A, Inoue K, Takano K, Wakayama T & Yanagimachi R 2000\nBirth of mice after NT by electrofusion using tail tip cells. Molecu-\nlar Reproduction and Development 57 55–59.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nOh SH, Miyoshi K & Funahashi H 1998 Rat oocytes fertilized in\nmodiﬁed rat 1-cell embryo culture medium containing a high\nsodium chloride concentration and bovine serum albumin main-\ntain developmental ability to the blastocyst stage. Biology of\nReproduction 59 884–889.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nOnishi A, Iwamoto M, Akita T, Mikawa S, Takeda K, Awata T et al.\nfetal ﬁbroblast nuclei.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\n2000 Pig cloning by microinjection of\nScience 289 1188–1190.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nPalmer A, Gavin AC & Nebreda AR 1998 A link between MAP\nkinase and p34cdc2/cyclin B during oocyte maturation: p90rsk phos-\nphorylates and inactivates the p34cdc2 inhibitory kinase Myt1.\nEMBO Journal 17 5037–5047.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nPeter M, Nakagawa J, Doree M, Labbe JC & Nigg EA 1990 In vitro\ndisassembly of the nuclear lamina and M phase-speciﬁc phos-\nphorylation of lamins by cdc2 kinase. Cell 61 591–602.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nPhillips KP, Petrunewich MA, Collins JL, Booth RA, Liu XJ & Baltz\nJM 2002 Inhibition of MEK or cdc2 kinase parthenogenetically\nthe same phenotypes as\nactivates mouse eggs and yields\nMos(2/2)\n\n210 –223.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nparthenogenotes. Developmental\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nBiology\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nRime H, Jessus C & Ozon R 1995 Tyrosine phosphorylation of\np34cdc2 is regulated by protein phosphatase 2A in growing imma-\nture Xenopus oocytes. Experimental Cell Research 219 29– 38.\nSherwood SW, Kung AL, Roitelman J, Simoni RD & Schimke RT\n1993 In vivo inhibition of cyclin B degradation and induction\nof cell-cycle arrest in mammalian cells by the neutral cysteine\nprotease\ninhibitor N-acetylleucylleucylnorleucinal. PNAS 90\n3353–3357.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwww.reproduction-online.org\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.1**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nShin MR, Park SW, Shim H & Kim NH 2002 Nuclear and microtu-\nembryos.\nreorganization in nuclear-transferred bovine\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nbule\nMolecular Reproduction and Development 62 74–82.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nToyoda Y & Chang MC 1974 Fertilization of rat eggs in vitro by\nepididymal spermatozoa and the development of eggs following\ntransfer. Journal of Reproduction and Fertilization 36 9 –22.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nWakayama T & Yanagimachi R 2001 Effect of cytokinesis inhibitors,\nDMSO and the timing of oocyte activation on mouse cloning\nusing cumulus cell nuclei. Reproduction 122 49–60.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nWakayama T, Perry AC, Zuccotti M, Johnson KR & Yanagimachi R\n1998 Full-term development of mice from enucleated oocytes\ninjected with cumulus cell nuclei. Nature 394 369–374.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nWakayama T, Rodriguez I, Perry AC, Yanagimachi R & Mombaerts P\n1999 Mice cloned from embryonic stem cells. PNAS 96\n14984–14989.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nWilmut I, Schnieke AE, McWhir J, Kind AJ & Campbell KH 1997\nViable offspring derived from fetal and adult mammalian cells.\nNature 385 810– 813.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nZernicka-Goetz M 1991 Spontaneous and induced activation of\nrat oocytes. Molecular Reproduction and Development 28\n169–176.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nZhou Q, Renard JP, Le Friec G, Brochard V, Beaujean N & Cheriﬁ Y\net al. 2003 Generation of fertile cloned rats by regulating oocyte\nactivation. Science 302 1179.\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nReceived 20 July 2004\nFirst decision 16 August 2004\nRevised manuscript received 8 September 2004\nAccepted 12 October 2004\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nReproduction (2005) 129 171–180\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nwww.reproduction-online.org\n**BLOCK**fs== 6.0**b== 0.0**t== 1.0**l== 0.7**r== 0.0**\nDownloaded from Bioscientifica.com at 03/31/2025 03:49:14PM\nvia IBPS / Pierre and Marie Curie Universityu Biologie Rech",
         "extracted",
         "",
         "Contribution of high p34cdc2 kinase activity to premature chromosome condensation of injected somatic cell nuclei in rat oocytes.",
         "None"
        ],
        [
         "38",
         "002d29c3ca9c915040b454c1572a89378eec6a72",
         "Explicit computational formulas for the coefficients of the periodic normal forms for all codim 1 bifurcations of limit cycles in generic autonomous ODEs are derived. They include second-order coefficients for the fold (limit point) bifurcation, as well as third-order coefficients for the flip (period-doubling) and Neimark--Sacker (torus) bifurcations. The formulas are independent of the dimension of the phase space and involve solutions of certain boundary-value problems on the interval [0,T], where T is the period of the critical cycle, as well as multilinear functions from the Taylor expansion of the right-hand sides near the cycle. The formulas allow us to distinguish between sub- and supercritical bifurcations, in agreement with earlier asymptotic expansions of the bifurcating solutions. Our formulation makes it possible to use robust numerical boundary-value algorithms based on orthogonal collocation, rather than shooting techniques, which greatly expands its applicability. The actual implementation is described in detail. We include three numerical examples, in which codim 2 singularities are detected along branches of codim 1 bifurcations of limit cycles as zeros of the periodic normal form coefficients.",
         "Y. Kuznetsov,W. Govaerts,E. Doedel,Annick Dhooge",
         "http://arxiv.org/pdf/1111.4445.pdf",
         "\n**BLOCK**fs== 17.2**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nNumerical Periodic Normalization for Codim 2 Bifurcations of\nLimit Cycles\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nF. Della Rossa‡∗\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nV. De Witte †\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nW. Govaerts †\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nYu. A. Kuznetsov‡\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nOctober 30, 2018\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nAbstract\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nPeriodic normal forms for the codim 2 bifurcations of limit cycles up to a 3-dimensional center\nmanifold in generic autonomous ODEs and computational formulas for their coeﬃcients are derived.\nThe formulas are independent of the dimension of the phase space and involve solutions of certain\nboundary-value problems on the interval [0, T ], where T is the period of the critical cycle, as well as\nmultilinear functions from the Taylor expansion of the right-hand sides near the cycle. The formulas\nallow us to distinguish between various bifurcation scenarios near codim 2 bifurcations. Our formulation\nmakes it possible to use robust numerical boundary-value algorithms based on orthogonal collocation,\nrather than shooting techniques, which greatly expands its applicability. The actual implementation is\ndescribed in detail with numerical examples.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nKeywords: normal forms, limit cycles, bifurcations, codimension two\n**BLOCK**fs== 14.3**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 14.3**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nIntroduction\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nIsolated periodic orbits (limit cycles) of smooth diﬀerential equations\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n(1)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n˙u = f (u, p), u\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nRn, p\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nRm,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.1**\nplay an important role in applications. In generic systems of the form (1) depending on one control parameter\n(i.e. with m = 1) a hyperbolic limit cycle exists for an open interval of parameter values p. At a boundary of\nsuch an interval, the limit cycle may become non-hyperbolic, so that either a cycle limit point (saddle-node),\nor a period-doubling (ﬂip), or a torus (Neimark-Sacker) bifurcation occurs. In two-parameter generic systems\n(1) (i.e. with m = 2) these bifurcations happen at certain curves in the parameter plane. These curves of\ncodim 1 bifurcations can meet tangentially or intersect transversally at some codim 2 points characterized\nby a double degeneracy of the limit cycle, which play the role of organizing centers for local dynamics, i.e.\nnear the critical cycle and for nearby parameter values. In some cases, such codim 2 bifurcations imply the\nappearance of “chaotic motions”.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nThe codim 2 bifurcations of limit cycles in generic systems (1) are well understood with the help of\nthe corresponding Poincar´e maps and their normal forms (see for example [22, 20, 3, 29, 18]). However,\napplications of these results to the analysis of concrete systems (1) are exceptional, since they require\naccurate higher-order derivatives of the Poincar´e map which are hardly available numerically [42, 19, 21, 32].\nWe note that there exists software, e.g. CAPD [1], TIDES [2, 5] that allows to compute up to any\nprecision level the solution of an ODE using a Taylor series method in a variable stepsize - variable order\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n∗DEI, Politecnico di Milano, V. Ponzio 35/5, 20133 Milano, Italy (dellarossa@elet.polimi.it).\n†Department of Applied Mathematics and Computer Science, Ghent University, Krijgslaan 281-S9, B-9000, Gent, Belgium\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n(Virginie.DeWitte@UGent.be, Willy.Govaerts@UGent.be).\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n‡Department of Mathematics, Utrecht University, Budapestlaan 6, P.O. Box 80010, 3508 TA Utrecht, The Netherlands\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\n(I.A.Kouznetsov@uu.nl).\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nformulation. It can also compute, up to any order, the partial derivatives of the solution with respect to the\ninitial conditions. When applied to compute a periodic orbit by a shooting method, this will also provide\nthe derivatives of the Poincar´e map.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.1**r== 0.1**\nThough this is a valuable approach in the case of a single periodic orbit, it is neither practical nor eﬃcient\nin a continuation context since the periodic orbits are not obtained as the solution to a set of equations. Also,\nthe shooting method does not have the high order convergence properties of the method of approximation by\npiecewise polynomials with collocation in the Gauss points that is routinely used in standard software such\nas auto [14], content [31], and matcont [11, 10]. Moreover, the number of derivatives of the Poincar´e\nmap to be computed is O(nk) if derivatives up to order k are needed (in several cases k = 5). Even for\nmoderate values of n this involves a great deal of unnecessary work since in our situation the normal form\nitself is known in advance and we need only compute its coeﬃcients. We will show that this can be done\nwithout computing the derivatives of the Poincar´e map.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nIndeed, recently an alternative numerical method to analyse codim 1 limit cycle bifurcations has been\nIt is based on the periodic normalization proposed in [25, 23, 24]\ndeveloped and implemented in [30].\nInstead, the\nand completely avoids the numerical computation of Poincar´e maps and their derivatives.\ncomputation of the normal form coeﬀcients is reduced to solving certain linear boundary value problems\n(BVP), where only the partial derivatives of the RHS of (1) are used [17, 16, 28]. In our implementation in\nMATCONT, we discretize these BVPs by orthogonal collocation with piecewise-polynomial functions.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nIn the present paper, we apply the approach developed in [30] to codim 2 bifurcations of limit cycles.\nIt should be noted that already in [7] normal forms for some codim 2 bifurcations of cycles in (1) were\nderived, while [23] contains the periodic normal forms for many codim 2 bifurcations of cycles, as well as\na general normalization technique applicable at any codimension. However, in neither of these publications\nexplicit formulas for the normal form coeﬃcients were given. The derivation of such formulas is the primary\ncontribution of this paper.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nThe paper is organized as follows. In Section 2 we ﬁx notation and list the periodic normal forms for\ncodim 2 bifurcations of limit cycles. Then we derive explicit formulas to compute the critical normal form\ncoeﬃcients for these bifurcations, which we order by the dimension nc of the cycle center manifold (i.e. the\n= 1). We restrict to the cases nc = 2 and 3. The formulas are\ntotal number of critical multipliers with\nindependent of the dimension of the phase space and involve solutions of certain BVPs on the interval [0, T ],\nwhere T is the period of the critical cycle, as well as multilinear functions from the Taylor expansion of the\nright-hand sides of (1) near the cycle. A derivation of the critical periodic normal forms based on [23] is\ngiven in Appendix A, while their relationships with the normal forms of the Poincar´e maps are discussed in\nAppendix B.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nµ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 14.3**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\n2 Periodic normal forms on the center manifold\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nWrite (1) at the critical parameter values as\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\n(2)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n˙u = F (u),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nand suppose that there is a limit cycle Γ corresponding to a periodic solution u0(t) = u0(t + T ), where T > 0\nis its (minimal) period. Develop F (u0(t) + v) into the Taylor series\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.5**\nF (u0(t) + v) = F (u0(t))+\n\n\nD(t; v, v, v, v) +\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\nA(t)v +\n\n4!\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nB(t; v, v) +\n\n5!\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nC(t; v, v, v)+\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n3!\nE(t; v, v, v, v, v) + O(\nk\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\n6),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\nv\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\nk\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\n(3)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwhere\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nA(t)v = Fu(u0(t))v, B(t; v1, v2) = Fuu(u0(t))[v1, v2], C(t; v1, v2, v3) = Fuuu(u0(t))[v1, v2, v3],\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\netc. The multilinear forms A, B, C, D, and E are periodic in t with period T but this dependence will often\nnot be indicated explicitly.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nConsider the initial-value problem for the fundamental matrix solution Y (t), namely,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.9**\n(4)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\ndY\ndt\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n= A(t)Y, Y (0) = In,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n×\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nn identity matrix. The eigenvalues of the monodromy matrix M = Y (T ) are called\nwhere In is the n\n(Floquet) multipliers of the limit cycle. The multipliers with\n= 1 are called critical. There is always a\nµ\n“trivial” critical multiplier µn = 1. We denote the total number of critical multipliers by nc and assume that\nthe limit cycle is non-hyperbolic, i.e. nc > 1. In this case, there exists an invariant nc-dimensional critical\ncenter manifold W c(Γ)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nRn near Γ1.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.1**\nIt is well known [3, 29], that in generic two-parameter systems (1) only eleven codim 2 bifurcations\noccur. To describe the normal forms of (2) on the critical center manifold W c(Γ) for these codim 2 cases, we\nparameterize W c(Γ) near Γ by (τ, ξ), where τ\nand ξ is a real or complex vector,\n[0, kT ] for k\ndepending on the bifurcation. It follows from [23] that it is possible to select the τ - and ξ-coordinates so\nthat the restriction of (2) to the corresponding critical center manifold W c(Γ) with nc = 2 or nc = 3 will\ntake one of the following periodic normal forms (for derivation see Appendix A). In Section 3 we will see\nthat in two cases, namely the cusp of cycles bifurcation and the fold-ﬂip bifurcation, a further simpliﬁcation\nis possible in the normal form, and more speciﬁcally in the transformation of time.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n1, 2, 3, 4\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈ {\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n}\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n⊂\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\n2.1 Bifurcations with a 2D center manifold\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nWe list here the critical periodic normal forms with nc = 2 and brieﬂy describe bifurcations in their generic\nunfoldings (see [3, 29] for more details).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\n2.1.1 Cusp of cycles bifurcation\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe cycle has a cusp of cycles bifurcation (CPC) if the eigenvalue µ1 = µn = 1 of Y (T ) corresponds to a two-\ndimensional Jordan block and the monodromy matrix has no other critical multipliers. The two-dimensional\nperiodic normal form at the CPC bifurcation is\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.9**\n(5)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n= 1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nξ + α1ξ2 + α2ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n−\n= cξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.3**\n[0, T ], ξ is a real coordinate on W c(Γ) that is transverse to Γ, α1, α2, c\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nR and the dots denote the\nwhere τ\nO(ξ4)-terms, which are T -periodic in τ . If c\n= 0, the limit cycle Γ is triple. In generic two-parameter systems\n(1), three hyperbolic limit cycles exist in a cuspidal wedge approaching the codim 2 point and delimited by\ntwo bifurcations curves, where two cycles collide and disappear via the saddle-node bifurcation.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n2.1.2 Generalized period-doubling bifurcation\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nThe cycle has a generalized period-doubling bifurcation (GPD) if the trivial eigenvalue µn = 1 of the monodromy\n1. The two-dimensional\nmatrix Y (T ) is simple and there is only another critical simple eigenvalue µ1 =\nperiodic normal form at the GPD bifurcation is\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\n(6)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\n= 1 + α1ξ2 + α2ξ4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n= eξ5 + . . . ,\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n1This manifold should not be confused with the (nc − 1)-dimensional center manifold of the corresponding Poincar´e map.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\n[0, 2T ], ξ is a real coordinate on W c(Γ) that is transverse to Γ, α1, α2, e\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nR and the dots denote\nwhere τ\nthe O(ξ6)-terms, which are 2T -periodic in τ . If e\n= 0, at most two period doubled limit cycles can bifurcate\nfrom the critical limit cycle Γ. In generic two-parameter systems (1), the GPD-point in the period-doubling\nbifurcation curve separates its sub- and super-critical branches and is the origin of a unique saddle-node\nbifurcation curve, where two period doubled cycles collide and disappear.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\n2.2 Bifurcations with a 3D center manifold\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nWe now list the critical periodic normal forms with nc = 3 and brieﬂy describe bifurcations in their generic\nunfoldings (see [3, 29]). In all cases, “chaotic motions” are possible.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\n2.2.1 Chenciner bifurcation\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nThe cycle has a Chenciner bifurcation (CH) if the trivial critical eigenvalue µn = 1 is simple and there are only\ntwo more critical simple multipliers µ1,2 = e±iθ with θ\n= 2π\nj , for j = 1, 2, 3, 4, 5, 6. The three-dimensional\nperiodic normal form at the CH bifurcation can be written as\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\n= 1 + α1|\n= iωξ + icξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n2 + α2|\n|\n2 + eξ\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\n[0, T ], ω = θ/T , ξ is a complex coordinate on W c(Γ) transverse to Γ, α1, α2, c\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nC and\nwhere τ\nthe dots denote the O(\n)-terms, which are T -periodic in τ . In generic two-parameter systems (1), at the\nk\nCH-point the Neimark-Sacker bifurcation changes its criticality (i.e. the bifurcating invariant torus changes\nits stability). A complicated bifurcation set responsible for “collision” and destruction of two tori of opposite\nstability is rooted at this codim 2 point.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nR, e\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nξ6\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nk\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n2.2.2 Strong resonance 1:1 bifurcation\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe cycle has a strong resonance 1 : 1 bifurcation (R1) if the trivial critical eigenvalue µn = 1 corresponds\nto a three-dimensional Jordan block. The three-dimensional periodic normal form at the R1 bifurcation is\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n(7)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\n(8)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\ndτ\ndt\ndξ1\ndt\ndξ2\ndt\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n= 1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nξ1 + αξ2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n= ξ2 + ξ1ξ2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n= aξ2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1 + bξ1ξ2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nk\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\n[0, T ], (ξ1, ξ2) are real coordinates on W c(Γ) transverse to Γ, α, a, b\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nR and the dots denote the\nwhere τ\nξ3\n)-terms, which are T -periodic in τ . In generic two-parameter systems (1), in the R1-point is located\nO(\nk\non the saddle-node of cycles curve. At this point, a torus bifurcation curve is rooted together with global\nhomoclinic bifurcation curves, along which the stable and the unstable invariant manifolds of a saddle cycle\nare tangent. The intersection of the invariant manifolds generates a Poincar´e homoclinic structure with the\nassociated periodic and “chaotic motions”.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n2.2.3 Strong resonance 1:2 bifurcation\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nThe cycle has a strong resonance 1 : 2 bifurcation (R2) if the trivial critical eigenvalue µn = 1 is simple and\n1 corresponding to a two-dimensional Jordan block.\nthere are only two more critical multipliers µ1 = µ2 =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nThe three-dimensional periodic normal form at the R2 bifurcation is\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.9**\n(9)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\ndτ\ndt\ndξ1\ndt\ndξ2\ndt\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n= 1 + αξ2\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1 + . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n= ξ2 + αξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n1 ξ2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n= aξ3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n1 + bξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1 ξ2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈\nξ4\nk\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\n[0, 2T ], (ξ1, ξ2) are real coordinates on W c(Γ) transverse to Γ, α, a, b\nR and the dots denote\nwhere τ\n)-terms, which are 2T -periodic in τ . In generic two-parameter systems (1), the R2-point is the\nthe O(\nk\nend-point of a torus bifurcation curve. The period-doubling bifurcation curve passes through this point, and\n(depending on the normal form coeﬀcients) a torus bifurcation curve of the period doubled limit cycle can\noriginate there. As in the R1-case, global bifurcation curves related to homoclinic tangencies can be present.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n2.2.4 Strong resonance 1:3 bifurcation\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nThe cycle has a strong resonance 1 : 3 bifurcation (R3) if the trivial critical eigenvalue µn = 1 is simple and\nthere are only two more critical simple multipliers µ1,2 = e±i 2π\n3 . The three-dimensional periodic normal\nform at the R3 bifurcation can be written as\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(10)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n= 1 + α1|\n= b ¯ξ2 + cξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2 + α2ξ3 + α3 ¯ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.3**\n[0, 3T ], ξ is a complex coordinate on W c(Γ) transverse to Γ, α1 ∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\n\nC with α3 = ¯α2\nwhere τ\nand the dots denote the O(\n)-terms, which are 3T -periodic in τ . In generic two-parameter systems (1),\nk\nnear the R3-point a homoclinic Poincar´e structure of the 3T -periodic limit cycle destroys the torus that is\nborn at the Neimark-Sacker bifurcation curve passing through this point. Homoclinic tangencies are rooted\nthere.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\nR, α2, α3, b, c\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nξ4\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nk\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n2.2.5 Strong resonance 1:4 bifurcation\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nThe cycle has a strong resonance 1 : 4 bifurcation (R4) if the trivial critical eigenvalue µn = 1 is simple and\nthere are only two more critical simple multipliers µ1,2 = e±i π\n2 . The three-dimensional periodic normal form\nat the R4 bifurcation can be written as\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n2 + α2ξ4 + α3 ¯ξ4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\n= 1 + α1|\nξ\n= cξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(11)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\n2 + d ¯ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\n\n\n\nC with α3 = ¯α2\nwhere τ\nand the dots denote the O(\n)-terms, which are 4T -periodic in τ . In generic two-parameter systems (1),\nk\nat the R4-point there can be eight diﬀerent situations, depending upon the values taken by the parameter c\nand d. In the simplest case a homoclinic structure associated to a 4T -periodic cycle destroys an invariant\ntorus that is born at the Neimark-Sacker bifurcation curve that passes trough this point.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\n[0, 4T ], ξ is a complex coordinate on W c(Γ) transverse to Γ, α1 ∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nR, α2, α3, c, d\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nξ5\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nk\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\n2.2.6 Fold-ﬂip bifurcation\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nThe cycle has a fold ﬂip bifurcation (LPPD) if the trivial critical eigenvalue µ1 = µn = 1 is double non\n1. The three-dimensional periodic normal\nsemi-simple and there is only one more critical multiplier µ2 =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nform at the LPPD bifurcation is\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nξ1 + α20ξ2\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n1 + α02ξ2\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n2 + α30ξ3\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n1 + α12ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(12)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n1 + a02ξ2\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n2 + a30ξ3\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1 + a12ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\n= b11ξ1ξ2 + b21ξ2\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n1 ξ2 + b03ξ3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n= 1\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n−\n= a20ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\ndτ\ndt\ndξ1\ndt\ndξ2\ndt\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\n[0, 2T ], (ξ1, ξ2) are real coordinates on W c(Γ) transverse to Γ, all the coeﬃcients are real and\nwhere τ\nthe dots denote the O(\n)-terms, which are 2T -periodic in τ . In generic two-parameter systems (1), the\nk\nperiod-doubling and saddle-node of cycles bifurcation curves are tangent at the LPPD-point, where (depending\non the normal form coeﬃcients) a Neimark-Sacker bifurcation curve of the 2T -periodic cycle can be rooted.\nGlobal bifurcations of heteroclinic structures and invariant tori are also possible.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nξ4\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nk\n**BLOCK**fs== 14.3**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\n3 Computation of critical coeﬃcients\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.1**r== 0.1**\nIn view of the above, we can assume that a parametrization of the center manifold W c(Γ) has been selected\nso that the restriction of (2) to this manifold has one of the normal forms given in Section 2. We then\napply the so-called homological equation approach [6]: the Taylor expansions of T -, 2T -, 3T - or 4T -periodic\nunknown functions involved in these parametrizations can be found by solving appropriate BVPs on [0, T ]\nso that (2) restricted to W c(Γ) has the corresponding periodic normal form. The coeﬃcients of the normal\nforms arise from the solvability conditions for the BVPs as integrals of scalar products over [0, T ], involving\nnonlinear terms of (2) near the periodic solution u0, as well as the critical eigenfunctions and higher order\nexpansion terms of the center manifold. The Taylor expansion coeﬃcient functions are usually unique up to\nthe addition of a multiple of a known eigenfunction. This has to be ﬁxed by adding an integral condition.\nAmong other things this leads to the fact that normal form coeﬃcients are not unique but implications\nfor the underlying dynamical systems are independent of this. We also remark that the solvability of all\nthe equations up to the maximal order of the normal form has to be checked. Finally, we note that the\ncoeﬃcients related to the transformation of time will only be computed when needed in the computation of\nthe critical coeﬃcients in the normal form for the state variables.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\n3.1 Cusp of cycles bifurcation\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nThe two-dimensional critical center manifold W c(Γ) at the CPC bifurcation can be parametrized locally by\n(τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(13)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nu = u0 + ξv + H(τ, ξ), τ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n[0, T ], ξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nR,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nwhere H satisﬁes H(T, ξ) = H(0, ξ) and has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\n(14)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\nH(τ, ξ) =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\nh2ξ2 +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nh3ξ3 + O(ξ4),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nwhere u0 and all hj are functions of τ , with hj(T ) = hj(0), for j = 2, 3, while the generalized eigenfunction\nv is given by\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(15)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nA(τ )v\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n˙v\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nF (u0) = 0, τ\n−\nv(0) = 0,\nv(T )\ndτ = 0.\nv, F (u0)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nThe function v exists due to Lemma 2 of [23]. Let ϕ∗ be a nontrivial solution of the adjoint eigenvalue\nproblem\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(16)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(17)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\n˙v∗ + AT(τ )v∗ + ϕ∗ = 0, τ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nv∗(T )\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nv∗(0) = 0,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nwhich is now deﬁned up to the addition of a multiple of ϕ∗. Note that the ﬁrst equation of (15) implies\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(18)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nfor ϕ∗ satisfying (16). Indeed,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nϕ∗, F (u0)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\ndτ = 0\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\nϕ∗, F (u0)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n0 h\nZ\ndue to (16).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\ndτ =\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nϕ∗,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nA(τ )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\ndτ =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nv\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n+ AT (τ )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\nϕ∗, v\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nMoreover, due to spectral assumptions at the CPC-point, we can also assume\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(19)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nZ\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n∗\nϕ\nh\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n, v\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\ndτ = 1.\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nNotice that this assumption gives us another normalization for free, since because of (15) and (17) we have\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(20)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nv∗, F (u0)\ndτ =\ni\nh\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nv∗,\nh\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nA(τ )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nv\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\ndτ\ni\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\nh\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n+ AT (τ )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nv∗, v\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\ndτ =\ni\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.8**r== 0.2**\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nϕ∗, v\nh\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\ndτ = 1,\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\ni.e. we have normalized the eigenfunction of the adjoint problem with the generalized one of the original\nproblem and the generalized eigenfunction of the adjoint problem with the eigenfunction of the original\nproblem. So ϕ∗ is the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(21)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\n˙ϕ∗ + AT(τ )ϕ∗ = 0, τ\nϕ∗(T )\nϕ∗(0) = 0,\n−\nT\nϕ∗, v\n1 = 0.\ndτ\n0 h\ni\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nNow, we still need an integral condition for the adjoint generalized eigenfunction v∗.\nIn all cases, for\nthe computation of an adjoint generalized eigenfunction we will require the inproduct with an original\neigenfunction to be zero. Here, the inproduct with v is appropriate. Therefore we obtain\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(22)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nNow, we substitute (13) into (2), using (3), (5) and (14), as well as\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\n˙v∗ + AT(τ )v∗ + ϕ∗ = 0, τ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n−\nv∗, v\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nv∗(0) = 0,\ndτ = 0.\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\nv∗(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\ndu\ndt\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n∂u\n∂ξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n+\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n∂u\n∂τ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nThis gives\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n˙u0 + ξ ( ˙v\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n˙u0) + ξ2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n˙v +\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nh2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n+ ξ3\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nα2 ˙u0 + α1 ˙v\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nα1 ˙u0 −\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.5**\n\nξ2 (A(τ )h2 + B(τ ; v, v)) +\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n˙h2 +\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n˙h3 + cv\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n+ O(ξ4)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n= F (u0) + ξA(τ )v +\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nξ3 (A(τ )h3 + 3B(τ ; h2, v) + C(τ ; v, v, v)) + O(ξ4),\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nwhere dots denote the derivatives with respect to τ .\nCollecting the ξ0-terms we get the identity\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n˙u0 = F (u0),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\nsince u0 is the periodic solution of (2).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nThe ξ1-terms provide another identity, namely\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\ndue to (15).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nFrom collecting the ξ2-terms we obtain an equation for h2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n˙u0 = A(τ )v,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n˙v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\n(23)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n˙h2 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nA(τ )h2 = B(τ ; v, v) + 2 ˙v\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n2α1 ˙u0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nNow, we project the left-hand side of this equation on the adjoint null-eigenfunction and obtain\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nϕ∗,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nA(τ )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\ndτ =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nh2i\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n+ AT (τ )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nϕ∗, h2i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\n0 h\nZ\ndue to (16). We can use this result to impose the so-called Fredholm solvability condition, i.e. also project\nthe right-hand side of (23) on ϕ∗ which then has to be equal to 0\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nϕ∗, B(τ ; v, v) + 2 ˙v\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n2α1 ˙u0i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\ndτ =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nϕ∗, B(τ ; v, v) + 2A(τ )v\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nNotice that this condition is actually trivially satisﬁed, due to the fact that we are at a cusp of cycles point,\nso that the second order normal form coeﬃcient\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nb =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n, B(τ ; v, v) + 2A(τ )v\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\ndτ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\n(see [30]) vanishes. Hence equation (23) is solvable, independent of the value of α1. So, for any value of α1\nwe get an equation for h2 to be solved in the space of vector-functions on [0, T ] satisfying h2(T ) = h2(0).\nNotice that if h2 satisﬁes (23), h2 + εF (u0) also satisﬁes (23), due to the fact that F (u0) is in the kernel\nA(τ ) and to the linearity of this operator. Now, the orthogonality condition with v∗\nof the operator d\ndetermines the value of ε such that we can deﬁne h2 as the unique solution of\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.2**r== 0.7**\ndτ −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n˙h2 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nA(τ )h2 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nB(τ ; v, v)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n2Av\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(24)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\n2F (u0) + 2α1F (u0) = 0, τ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nh2(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nh2(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nv∗, h2i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nCollecting the ξ3-terms we ﬁnally obtain an equation in h3 which allows us to determine the normal form\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.7**\n\ncoeﬃcient c of (5)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n6α1 ˙v + 3 ˙h2 −\nAs before, the null-eigenfunction of the adjoint operator\ncondition implies that\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n6α2 ˙u0 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nA(τ )h3 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n˙h3 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\n6cv + 3B(τ ; h2, v) + C(τ ; v, v, v).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nAT(τ ) is ϕ∗. Thus, the Fredholm solvability\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n6α1 ˙v + 3 ˙h2 −\nUsing (15), (19) and (18), we get the expression\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\n6α2 ˙u0 −\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.3**\n6cv + 3B(τ ; h2, v) + C(τ ; v, v, v)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(25)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\nc =\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nϕ∗,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.2**\n6α1A(τ )v + 3A(τ )h2 + 3B(τ ; v, v) + 6A(τ )v + 3B(τ ; h2, v) + C(τ ; v, v, v)\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\ndτ,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nwhere v and ϕ∗ are deﬁned by (15) and (21), while h2 satisﬁes (24).\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nIn what follows we will prove that the choice of α1 does not inﬂuence the value of the critical normal\nh(1)\n2 =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nform coeﬃcient c. Two solutions h2 diﬀer by h(2)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nα(1)\n1 )v, from which\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n2(α(2)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n2 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n1 −\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nc(2)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\nc(1) =\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n0 h\nZ\n= (α(2)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1 −\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n6(α(2)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nα(1)\n1 )\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n1 −\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nϕ∗,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0 h\nT\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nϕ∗,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nα(1)\n1 )A(τ )v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n6A(τ )(α(2)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n1 −\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\nα(1)\n1 )v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n6(α(2)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n1 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.1**\nα(1)\n1 )B(τ ; v, v)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n2A(τ )v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nB(τ ; v, v)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n2A(τ )v + 2A(τ )v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n= (α(2)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1 −\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nα(1)\n1 )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\n= 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nsince b vanishes. So, for simplicity we take α1 = 0 which further simpliﬁes normal form (5).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nTherefore, the critical coeﬃcient c in the periodic normal form (5) has been computed. The bifurcation\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nis nondegenerate if c\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n= 0.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\n3.2 Generalized period-doubling bifurcation\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nThe two-dimensional critical center manifold W c(Γ) at the GPD bifurcation can be parametrized locally by\n(τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(26)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nu = u0 + ξv + H(τ, ξ), τ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n[0, 2T ], ξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nR,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nwhere the function H satisﬁes H(2T, ξ) = H(0, ξ). It has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(27)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nH(τ, ξ) =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nh2ξ2 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nh3ξ3 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nh4ξ4 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nh5ξ5 + O(ξ6),\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nwith hj(2T ) = hj(0), with\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n˙v\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(28)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nA(τ )v = 0, τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\nv(T ) + v(0) = 0,\n1 = 0,\nv, v\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\ndτ\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nT\n0 h\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nThe function v exists due to Lemma 5 of [23].\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nv(τ + T ) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nv(τ ) for τ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n[0, T ].\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nThe functions hi, i = 1 . . . 5, can be found by solving appropriate BVPs, assuming that (2) restricted to\nW c(Γ) has the periodic normal form (6). From (26) and (27) it follows that hi(τ + T ) = hi(τ ) for i even\nand hi(τ + T ) =\n[0, T ]. Indeed, since we are at a generalized period-doubling point\n−\nu(τ, ξ) = u(τ + T,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nhi(τ ) for i odd, for τ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nξ), so\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nand thus\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nhi(τ )ξi =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\ni!\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\ni\nX\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ni\nX\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ni!\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nhi(τ + T )(\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n1)iξi,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\nhi(τ ) = (\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\n1)ihi(τ + T ),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nfrom which the stated follows. This makes it possible to restrict our considerations to the interval [0, T ]\ninstead of [0, 2T ].\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nThe coeﬃcients α1, α2 and e arise from the solvability conditions for the BVPs as integrals of scalar\nproducts over the interval [0, T ]. Speciﬁcally, these scalar products involve among other things the terms\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nup to the ﬁfth order of (1) near the periodic solution u0, the eigenfunction v, the adjoint eigenfunction ϕ∗\nsatisfying\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(29)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.4**\n˙ϕ∗ + AT(τ )ϕ∗ = 0, τ\nϕ∗(T )\n−\nϕ∗, F (u0)\ni\nand a similar adjoint eigenfunction v∗ satisfying\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nϕ∗(0) = 0,\n1 = 0,\ndτ\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(30)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n˙v∗ + AT(τ )v∗ = 0, τ\nv∗(T ) + v∗(0) = 0,\nT\n1 = 0.\n0 h\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nv∗, v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\ndτ\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nTo derive the normal form coeﬃcient, we proceed as in Section 3.1, namely, we substitute (26) into (2)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\n\nand use (3), as well as (6) and (27).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nCollecting the ξ0-terms in the resulting equation gives us the identity\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nsince u0 is the T -periodic solution of (2).\nThe ξ1-terms provide another identity\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n˙u0 = F (u0),\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n˙v = A(τ )v,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\ndue to (28) and (3.2).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nBy collecting the ξ2-terms, we obtain the equation for h2,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(31)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n˙h2 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nA(τ )h2 = B(τ ; v, v)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n2α1 ˙u0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.4**\nto be solved in the space of functions satisfying h2(T ) = h2(0).\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.1**\nA(τ ) is singular with null-function ˙u0. Thus, the following Fredholm solvability condition is involved\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.1**\nIn this space, the diﬀerential operator\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n, B(τ ; v, v)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n2α1 ˙u0i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nwhich leads to the expression\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n(32)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nα1 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n0 h\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nϕ∗, B(τ ; v, v)\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\ndτ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nwhere v and ϕ∗ are deﬁned by (28) and (29), respectively.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nWith α1 deﬁned in this way, let h2 be a solution of (31) in the space of functions satisfying h2(0) = h2(T ).\nNotice also here that if h2 is a solution of (31), then also h2 + ε1F (u0) satisﬁes (31), since F (u0) is in the\nkernel of the operator d\nIn order to obtain a unique solution (without projection on the null\neigenspace) we impose the following orthogonality condition which determines the value of ε1\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\ndτ −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nA(τ ).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nThus h2 is the unique solution of the BVP\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nϕ∗, h2i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n˙h2 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nA(τ )h2 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.3**\nB(τ ; v, v) + 2α1F (u0) = 0, τ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\n(33)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(34)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nh2(T )\nT\n0 h\nR\nCollecting the ξ3-terms, we get the equation for h3,\n˙h3 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nh2(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nϕ∗, h2i\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nA(τ )h3 = C(τ ; v, v, v) + 3B(τ ; v, h2)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n6α1 ˙v,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nIn this space the diﬀerential operator\nA(τ ) has a one-dimensional nullspace, spanned by v, and (34) is solvable only if the RHS of this\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h3(T ) =\nd\ndτ −\nequation lies in the reachable space of that operator. Using (28), we can rewrite the right-hand side as\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nh3(0).\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nBut the Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nC(τ ; v, v, v) + 3B(τ ; v, h2)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n6α1A(τ )v.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(35)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nv∗, C(τ ; v, v, v) + 3B(τ ; v, h2)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n6α1A(τ )v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nis trivially satisﬁed due to the fact that we are in a generalized period-doubling point and so the cubic\ncoeﬃcient of the normal form\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nc =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n∗\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\n, C(τ ; v, v, v) + 3B(τ ; v, h2)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n6α1A(τ )v\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\n(for it’s deﬁnition see [30]) vanishes. Since the RHS of (34) is in the range space of the operator d\nwe can solve the equation in order to ﬁnd h3 as the unique solution of the BVP\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\ndτ −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\nA(τ ),\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n˙h3 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nA(τ )h3 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nC(τ ; v, v, v)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(36)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nBy collecting the ξ4-terms, we get the equation for h4,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\n3B(τ ; v, h2) + 6α1A(τ )v = 0, τ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nh3(T ) + h3(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nv∗, h3i\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\n˙h4 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nA(τ )h4 = D(τ ; v, v, v, v) + 6C(τ ; v, v, h2) + 3B(τ ; h2, h2)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n+ 4B(τ ; v, h3)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.8**r== 0.2**\n12α1 ˙h2 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n24α2 ˙u0,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h4(T ) = h4(0). Formulation of the Fredholm solvability\ncondition\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\nϕ∗, D(τ ; v, v, v, v) + 6C(τ ; v, v, h2) + 3B(τ ; h2, h2) + 4B(τ ; v, h3)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n12α1 ˙h2 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n24α2 ˙u0i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\ngives us an equation for the parameter α2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nα2 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\n, D(τ ; v, v, v, v) + 6C(τ ; v, v, h2) + 3B(τ ; h2, h2) + 4B(τ ; v, h3)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n12α1 ˙h2i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\ndτ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nwhich can be simpliﬁed considering (31) into\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nα2 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.4**\n, D(τ ; v, v, v, v) + 6C(τ ; v, v, h2) + 3B(τ ; h2, h2)+\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n4B(τ ; v, h3)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\n12α1(A(τ )h2 + B(τ ; v, v))\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\ndτ + α2\n1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nwhere α1 is given by (32), and h2, h3, v and ϕ∗ are the solutions of the BVPs (33), (36), (28) and (29),\nrespectively.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nUsing this value of α2 we can ﬁnd h4 by\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n˙h4 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nA(τ )h4 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.5**\n4B(τ ; v, h3) + 12α1(A(τ )h2 + B(τ ; v, v)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\n2α1F (u0)) + 24α2F (u0) = 0, τ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nD(τ ; v, v, v, v)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n6C(τ ; v, v, h2)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n3B(τ ; h2, h2)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(37)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nFinally, by collecting the ξ5-terms, we get the equation for h5,\n˙h5 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nA(τ )h5 = E(τ ; v, v, v, v, v) + 10D(τ ; v, v, v, h2) + 15C(τ ; v, h2, h2)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\n+ 10C(τ ; v, v, h3) + 10B(τ ; h2, h3) + 5B(τ ; v, h4)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nwhich has to be solved in the space of functions satisfying h5(T ) =\nhas a one-dimensional null-space, we can write\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n120α2 ˙v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.2**\n20α1 ˙h3 −\nh5(0). Since the operator d\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n120ev,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nA(τ )\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\ndτ −\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\n, E(τ ; v, v, v, v, v) + 10D(τ ; v, v, v, h2) + 15C(τ ; v, h2, h2) + 10C(τ ; v, v, h3) +\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\n10B(τ ; h2, h3) + 5B(τ ; v, h4)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n120α2 ˙v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n20α1 ˙h3 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n120ev\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nwhich makes it possible to compute the parameter e of the normal form (6). Using the normalization of\n(30), (36) and (35) gives\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(38)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\ne =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\nv∗, E(τ ; v, v, v, v, v) + 10D(τ ; v, v, v, h2) + 15C(τ ; v, h2, h2)+\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.4**\n10C(τ ; v, v, h3) + 10B(τ ; h2, h3) + 5B(τ ; v, h4)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n120α2A(τ )v\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n20α1A(τ )h3i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\ndτ.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nA check that this quantity doesn’t vanish, guarantees us that the codim-2 bifurcation is non-degenerate.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\n3.3 Chenciner bifurcation\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nThe three-dimensional critical center manifold W c(Γ) at the CH bifurcation can be parametrized locally by\n(τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.2**\n(39)\nwhere the real function H satisﬁes H(T, ξ, ¯ξ) = H(0, ξ, ¯ξ), and has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.4**\nu = u0 + ξv + ¯ξ¯v(τ ) + H(τ, ξ, ¯ξ), τ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n[0, T ], ξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nC,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nH(τ, ξ, ¯ξ) =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n+\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n+\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n+\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n+\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(40)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nh12ξ ¯ξ2 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nh02 ¯ξ2\n\n\nh31(τ )ξ3 ¯ξ +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\nh41(τ )ξ4 ¯ξ +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nh30ξ3 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nh20ξ2 + h11ξ ¯ξ +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\nh21ξ2 ¯ξ +\n\n\nh50(τ )ξ5 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nh40(τ )ξ4 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nh05(τ ) ¯ξ5 + O(\n|\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n6),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nh03 ¯ξ3\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nh22(τ )ξ2 ¯ξ2 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nh32(τ )ξ3 ¯ξ2 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nh13(τ )ξ ¯ξ3 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nh04(τ ) ¯ξ4\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nh23(τ )ξ2 ¯ξ3 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nh14(τ )ξ ¯ξ4\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nwith hij(T ) = hij(0) and hij = ¯hji so that hii is real, while v and its conjugate ¯v are deﬁned as\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(41)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nThese functions exist due to Lemma 2 of [23].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n˙v(τ )\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.4**\nA(τ )v + iω v = 0, τ\nv(T )\nv, v\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nv(0) = 0,\n1 = 0.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\ndτ\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nIf we assume that (2) restricted to W c(Γ) has the periodic normal form (7), as in the previous cases, we\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\ncan ﬁnd the functions hij(τ ) by solving appropriate BVPs.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nFirst we introduce the two needed adjoint eigenfunctions. The ﬁrst one, namely ϕ∗, satisﬁes (29), and\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nthe second one, namely v∗, satisﬁes\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(42)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\n˙v∗(τ ) + AT(τ )v∗ + iω v∗ = 0, τ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nAs usual, we substitute (39) into (2), use (3), (7), and (40), as well as the homological equation\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\ndu\ndt\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n=\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n∂u\n∂ξ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n+\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n∂u\n∂ ¯ξ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nd ¯ξ\ndt\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n+\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∂u\n∂τ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.3**\nand collect the corresponding terms in order to ﬁnd the needed coeﬃcients of (7).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nThe ξ-independent and the linear terms give rise to the usual identities\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n˙u0 = F (u0),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n˙v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nA(τ )v + iωv = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n˙¯v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nA(τ )¯v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\niω¯v = 0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nCollecting the coeﬃcients of the ξ2- or ¯ξ2-terms leads to the equation\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nA(τ )h20 + 2iωh20 = B(τ ; v, v)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nor its complex-conjugate. This equation has a unique solution h20 satisfying h20(T ) = h20(0), since due to\nthe spectral assumptions e2iωT is not a multiplier of the critical cycle. Thus, h20 can be found by solving\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(43)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nA(τ )h20 + 2iωh20 −\nh20(T )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nB(τ ; v, v) = 0, τ\nh20(0) = 0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.4**\nBy collecting the ξ ¯ξ-terms we obtain an equation for h11, namely\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nA(τ )h11 = B(τ ; v, ¯v)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nα1 ˙u0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nto be solved in the space of the functions satisfying h11(T ) = h11(0). In this space the operator d\nA(τ ) has\nAT(τ )\na range space with codimension one. As before, the null-eigenfunction of the adjoint operator\nis ϕ∗, given by (29), and thus because of the Fredholm solvability condition, we can easily obtain the needed\nvalue for α1\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\ndτ −\nd\ndτ −\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(44)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nα1 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nZ\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n0 h\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nϕ∗, B(τ ; v, ¯v)\ndτ.\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nWith α1 deﬁned in this way, let h11 be the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n(45)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nA(τ )h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nB(τ ; v, ¯v) + α1 ˙u0 = 0, τ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nh11(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nϕ∗, h11i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\nh11(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nThe coeﬃcient in front of the third order terms in (7) is purly imaginary since the ﬁrst Lyapunov\ncoeﬃcient vanishes at a Chenciner point. We are now ready to compute this coeﬃcient. In fact, if we collect\nthe ξ2 ¯ξ-terms we obtain\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.7**\n˙h21 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.3**\nA(τ )h21 + iωh21 = C(τ ; v, v, ¯v) + 2B(τ ; v, h11) + B(τ ; ¯v, h20)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\n2icv\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.8**r== 0.2**\n2α1 ˙v,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h21(T ) = h21(0). In this space the operator d\nA(τ ) + iω\nis singular, since eiωT is a multiplier of the critical cycle. So we can impose the usual Fredholm solvability\ncondition, taking (42) into account\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\ndτ −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(46)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.4**\n, C(τ ; v, v, ¯v) + 2B(τ ; v, h11) + B(τ ; ¯v, h20)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n2icv\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n2α1 ˙v\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\ndτ = 0.\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nFrom this equation we can ﬁnd the value of the coeﬃcient c of the normal form (7)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(47)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\nc =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.4**\nv∗, C(τ ; v, v, ¯v) + 2B(τ ; v, h11) + B(τ ; ¯v, h20)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n2α1A(τ )v\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.3**\nand, with c deﬁned in this way, we can ﬁnd h21 as the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(48)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n˙h21 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nA(τ )h21 + iωh21 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nC(τ ; v, v, ¯v)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n2B(τ ; v, h11)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nB(τ ; ¯v, h20) + 2icv + 2α1(A(τ )v\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nh21(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\niωv) = 0, τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n−\nh21(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nv∗, h21i\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nCollecting the ξ3-terms gives us an equation for h30\n˙h30 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\nA(τ )h30 + 3iωh30 = C(τ ; v, v, v) + 3B(τ ; v, h20),\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nwhich has a unique solution h30 satisfying h30(T ) = h30(0), since e3iωT is not a multiplier of the critical\ncycle by the spectral assumptions. Thus, h30 is the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(49)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n˙h30 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nA(τ )h30 + 3iωh30 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nC(τ ; v, v, v)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n3B(τ ; v, h20) = 0, τ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n−\nh30(T )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nh30(0) = 0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nBy collecting the ξ3 ¯ξ-terms we obtain an equation for h31\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n˙h31 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.1**\nA(τ )h31 + 2iωh31 = D(τ ; v, v, v, ¯v) + 3C(τ ; v, v, h11) + 3C(τ ; v, ¯v, h20) + 3B(τ ; h11, h20)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n+3B(τ ; v, h21) + B(τ ; ¯v, h30)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n6ich20 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n3α1 ˙h20\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nwhich has a unique solution h31 satisfying h31(T ) = h31(0), since e2iωT is not a multiplier of the critical\ncycle by the spectral assumptions. Thus, h31 is the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(50)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.5**\n˙h31 −\nA(τ )h31 + 2iωh31 −\n3C(τ ; v, ¯v, h20)\n−\nB(τ ; ¯v, h30) + 6ich20 + 3α1(A(τ )h20 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nD(τ ; v, v, v, ¯v)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\n−\n3B(τ ; h11, h20)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n3C(τ ; v, v, h11)\n3B(τ ; v, h21)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\n2iωh20 + B(τ ; v, v)) = 0, τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nh31(T )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nh31(0) = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nTaking into account the\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nξ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.4**\n4-terms gives an equation for h22\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n|\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\n˙h22 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\nA(τ )h22 = D(τ ; v, v, ¯v, ¯v) + C(τ ; v, v, h02)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.2**\n+4C(τ ; v, ¯v, h11) + C(τ ; ¯v, ¯v, h20) + 2B(τ ; h11, h11) + 2B(τ ; v, h12)\n+B(τ ; h02, h20) + 2B(τ ; ¯v, h21)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n4α2 ˙u0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n4α1 ˙h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h22(T ) = h22(0). In this space the operator d\nA(τ ) has\na range space with codimension one which is orthogonal to ϕ∗. So one Fredholm solvability condition is\ninvolved, namely\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\ndτ −\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n∗\nϕ\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\n, D(τ ; v, v, ¯v, ¯v) + C(τ ; v, v, h02) + 4C(τ ; v, ¯v, h11) + C(τ ; ¯v, ¯v, h20) + 2B(τ ; h11, h11)\n4α1 ˙h11 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.3**\n+ 2B(τ ; v, h12) + B(τ ; h02, h20) + 2B(τ ; ¯v, h21)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n4α2 ˙u0i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nwhich allows us to compute the value of the coeﬃcient α2 of our normal form\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(51) α2 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.4**\nϕ∗, D(τ ; v, v, ¯v, ¯v) + C(τ ; v, v, h02) + 4C(τ ; v, ¯v, h11)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.2**\n+ C(τ ; ¯v, ¯v, h20) + 2B(τ ; h11, h11) + 2B(τ ; v, h12) + B(τ ; h02, h20)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n+ 2B(τ ; ¯v, h21)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.3**\nUsing this value for α2 we can ﬁnd h22 as the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n4C(τ ; v, ¯v, h11)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.5**\n˙h22 −\nC(τ ; ¯v, ¯v, h20)\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nA(τ )h22 −\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nD(τ ; v, v, ¯v, ¯v)\n2B(τ ; h11, h11)\nB(τ ; h02, h20)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.2**\nC(τ ; v, v, h02)\n2B(τ ; v, h12)\n2B(τ ; ¯v, h21)\nα1F (u0)) + 4α2F (u0) = 0, τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n−\n−\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.5**\n+4α1(A(τ )h11 + B(τ ; v, ¯v)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nh22(T )\nT\n0 h\nFinally, by collecting the ξ3 ¯ξ2-terms we get an equation for h32\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.2**\nh22(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nϕ∗, h22i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n(52)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n˙h32 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.2**\nA(τ )h32 + iωh32 = E(τ ; v, v, v, ¯v, ¯v) + D(τ ; v, v, v, h02) + 6D(τ ; v, v, ¯v, h11)\n+3D(τ ; v, ¯v, ¯v, h20) + 6C(τ ; v, h11, h11) + 3C(τ ; v, v, h12) + 3C(τ ; v, h02, h20)\n+6C(τ ; ¯v, h11, h20) + 6C(τ ; v, ¯v, h21) + C(τ ; ¯v, ¯v, h30) + 3B(τ ; h12, h20) + 6B(τ ; h11, h21)\n+3B(τ ; v, h22) + B(τ ; h02, h30) + 2B(τ ; ¯v, h31)\n6α1 ˙h21\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n12α2 ˙v\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n12ev\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n6ich21 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nthat, since the operator is singular, allows us, using the ﬁrst of (41) as well as the ﬁrst and the last of (48)\nand (46), to compute the critical coeﬃcient e of (7) imposing the Fredholm solvability condition, obtaining\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\ne =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\nv∗\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.3**\n, E(τ ; v, v, v, ¯v, ¯v) + D(τ ; v, v, v, h02) + 6D(τ ; v, v, ¯v, h11)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.2**\n+3D(τ ; v, ¯v, ¯v, h20) + 6C(τ ; v, h11, h11) + 3C(τ ; v, v, h12)\n+3C(τ ; v, h02, h20) + 6C(τ ; ¯v, h11, h20) + 6C(τ ; v, ¯v, h21) + C(τ ; ¯v, ¯v, h30)\n+3B(τ ; h12, h20) + 6B(τ ; h11, h21) + 3B(τ ; v, h22) + B(τ ; h02, h30)\n+2B(τ ; ¯v, h31)\n+B(τ ; ¯v, h20)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n6α1(A(τ )h21 + 2B(τ ; v, h11) + C(τ ; v, v, ¯v)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\nα2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n1iω.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\n(53)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.4**\n12α2A(τ )v\n−\ndτ + iωα2 + icα1 −\n2α1Av)\ni\nAs stated in Appendix B, we deﬁne the second Lyapunov coeﬃcient as\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nIf this coeﬃcient does not vanish, no more degeneracies happen at this codim 2 point.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nL2(0) =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n(e).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nℜ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nSince we have to check all equations up to the ﬁfth order, we still have to look at the ξ4-terms, the\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n˙h40 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.2**\nA(τ )h40 + 4iωh40 = D(τ ; v, v, v, v) + 6C(τ ; v, v, h20) + 3B(τ ; h20, h20)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nB(τ ; v, h30),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n˙h50 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n˙h41 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nA(τ )h50 + 5iωh50 = E(τ ; v, v, v, v, v) + 10D(τ ; v, v, v, h20) + 10C(τ ; v, v, h30)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.2**\n+15C(τ ; v, h20, h20) + 10B(τ ; h20, h30) + 5B(τ ; v, h40)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\nA(τ )h41 + 3iωh41 = E(τ ; v, v, v, v, ¯v) + 6D(τ ; v, v, ¯v, h20) + 4D(τ ; v, v, v, h11)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.2**\n+4C(τ ; v, ¯v, h30) + 8C(τ ; v, h20, h11) + 6C(τ ; v, v, h21)\n+C(τ ; ¯v, h20, h20) + 6B(τ ; h20, h21) + B(τ ; ¯v, h40)\n+4B(τ ; v, h31) + 4B(τ ; h30, h11)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n12ich30.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n3α1 ˙h30 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nSince we are in a complex eigenvalues case, v is determined up to a factor γ, for which γHγ = 1. Then\nv∗, h20, h21, h30, h31 are replaced by γv∗, γ2h20, γh21, γ3h30, γ2h31 respectively, but α1, α2, c and e are not\naﬀected by this factor.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThe three-dimensional critical center manifold W c(Γ) at the R1 bifurcation can be parameterized locally by\n(τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n(54)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.5**\nu = u0 + ξ1v1 + ξ2v2 + H(τ, ξ), τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n[0, T ], ξ = (ξ1, ξ2)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nR2,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nwhere H satisﬁes H(T, ξ) = H(0, ξ) and has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(55)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nH(τ, ξ) =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nh20ξ2\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1 + h11ξ1ξ2 +\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nh02ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n2 + O(\nk\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n3),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nk\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nwhere the functions h20, h11 and h02 are T -periodic in τ , where v1 and v2 are the generalized eigenfunctions\nassociated with the trivial multiplier and deﬁned as the unique solutions of the BVP\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(56)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n˙v1 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nA(τ )v1 −\nF (u0) = 0, τ\nv1(T )\nv1(0) = 0,\n−\ndτ = 0,\nv1, F (u0)\ni\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n˙v2 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nA(τ )v2 + v1 = 0, τ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(57)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nv2(0) = 0,\nv2(T )\ndτ = 0,\nv2, F (u0)\ni\nrespectively. The functions v1 and v2 exist and are diﬀerent due to Lemma 2 of [23]. Following our approach\nto ﬁnd the value of the normal form constants, we deﬁne ϕ∗ as a solution of the adjoint eigenfunction problem\n(16), v∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.3**\n1 as a solution of the adjoint generalized eigenfunction problem (17) and v∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\n2 as a solution of\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n˙v2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n(τ ) + AT(τ )v∗\nv∗\n2 (T )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n1 = 0, τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n2 + v∗\nv∗\n2(0) = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFirst, notice that the Fredholm solvability condition gives us immediately the following scalar products\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n(58)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\nϕ∗, F (u0)\ndτ =\ni\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\nϕ∗, v1i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\ndτ =\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\nF (u0), v∗\n1 i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nDue to the spectral assumptions at the R1 point we are free to assume that\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(59)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nϕ∗, v2i\nAppending this condition to the eigenproblem, we can ﬁnd the eigenfunctions ϕ∗ as the unique solutions of\nthe BVP\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\ndτ = 1.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n0 h\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nZ\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(60)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\n˙ϕ∗ + AT(τ )ϕ∗ = 0, τ\nϕ∗(T )\nϕ∗, v2i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nϕ∗(0) = 0,\n1 = 0.\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nAs already mentioned in the cusp of cycles case, we will choose adjoint generalized eigenfunctions orthog-\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nonal to an original eigenfunction. Therefore, v∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n1 and v∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.2**\n2 are obtained as the solution of\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(61)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n˙v1\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nϕ∗ = 0, τ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\n+ AT(τ )v∗\n1 −\nv∗\nv∗\n1(0) = 0,\n1 (T )\n−\nT\nv∗\ndτ = 0,\n1 , v2i\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(62)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n˙v2\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1 = 0, τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\n(τ ) + AT(τ )v∗\n2 + v∗\nv∗\nv∗\n2 (0) = 0,\n2(T )\n−\nT\nv∗\ndτ = 0,\n2, v2i\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n\nrespectively. Notice that, as in the cusp of cycles case, we have normalized in (59) the adjoint eigenfunction\nwith the last generalized eigenfunction, which gives us in addition\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n1 , v1i\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\ndτ =\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\n∗\ndτ = 1.\n2, F (u0)\nv\ni\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nAs usual, to derive the value of the normal form coeﬃcients we substitute (54) into (2), we use (3) as\nwell as (8) and (55) and get diﬀerent equalities for every degree of ξ. Remark that in fact the solvability of\nall the equations up to the maximal order of the normal form has to be checked. We will pay extra attention\nto it in this section.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nBy collecting the ξ0-terms we get the identity\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nThe linear terms provide two other identities, namely\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n˙u0 = F (u0).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n˙v1 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nA(τ )v1 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nF (u0) = 0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n˙v2 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nAv2 + v1 = 0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\ncf. (56) and (57).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nBy collecting the ξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.5**\n1-terms we ﬁnd an equation for h20\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(63)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nA(τ )h20 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2α ˙u0 + 2 ˙v1 + B(τ ; v1, v1)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n2av2,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nto be solved in the space of periodic functions on [0, T ]. In this space, the diﬀerential operator d\nsingular with a range orthogonal to ϕ∗. Thus a Fredholm solvability condition is involved, namely\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\ndτ −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\nA(τ ) is\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nϕ∗,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n2α ˙u0 + 2 ˙v1 + B(τ ; v1, v1)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n2av2i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nThe equations (58), (59), and (56) let us obtain the following value for a\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(64)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\na =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nϕ∗, 2A(τ )v1 + B(τ ; v1, v1)\ndτ.\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nNotice that in the RHS of (63) we have no freedom which could change the value of the coeﬃcient a. This\nconﬁrms the theoretically proved fact that the ξ2\n1 -term of normal form (8) is resonant. Notice moreover that\nparameter α is undetermined, which gives us two degrees of freedom for h20. In fact, if h20 is a solution of\n(63), then also ˜h20 = h20 + εI\n20v1 is a solution, due to the fact that F (u0) spans the nullspace of\nthe operator d\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.4**\nA(τ ) and that we can tune α as desirable:\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.6**\n20F (u0) + εII\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\ndt −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(65)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nd˜h20\ndt −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nA(τ )˜h20 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\ndh20\ndt −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nA(τ )h20 + εII\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\ndv1\ndt −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nA(τ )v1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\ndh20\ndt −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nA(τ )h20 + εII\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n20 ˙u0.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.4**\nBy collecting the ξ1ξ2-terms we ﬁnd an equation for h11\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nA(τ )h11 = B(τ ; v1, v2) + ˙v2 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\n(66)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\nh20 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\nbv2 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\nv1,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nto be solved in the space of T -periodic functions. As in the previous case, a solvability condition is involved\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nEquation (59) as well as (57) and (58) let us rewrite this condition as\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nb =\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nϕ∗, B(τ ; v1, v2) + A(τ )v2i\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\ndτ\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nZ\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n0 h\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nϕ∗, h20i\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\ndτ.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\nUsing (61), (63), (58) and (61) we can rewrite the second term of the right-hand side as\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n+ AT (τ )\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nv∗\n1, h20i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\ndτ =\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nv∗\n1 ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n+ A(τ )\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ndτ\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nh20i\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nv∗\n1 ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n2α ˙u0 + 2 ˙v1 + B(τ ; v1, v1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n2av2i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ndτ =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\nv∗\ndτ\n1 , 2Av1 + B(τ ; v1, v1)\ni\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nobtaining an equation for b which involves only the original and adjoint eigenfunctions\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(67)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nb =\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.5**\nϕ∗, B(τ ; v1, v2) + A(τ )v2i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\ndτ +\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\nv∗\ndτ.\n1 , 2Av1 + B(τ ; v1, v1)\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nNotice that the freedom that we have on h20 can not be used to change the value to coeﬃcient b (and so\nthe ξ1ξ2-term of the normal form (8) is resonant). Indeed, h20 is deﬁned up to a multiple of F (u0) and\nv1, but both vectors are orthogonal to ϕ∗, see the ﬁrst two orthogonality conditions in (58). However the\npresence of h20 in the RHS gives us three degrees of freedom for h11. In fact, if h11 is a solution of (66), also\n˜h11 = h11 + εI\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n20v2 is a solution, since\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n20v1 + εII\nεI\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n11F (u0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nd˜h11\ndt −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nA(τ )˜h11 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\ndh11\ndt −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nA(τ )h11 −\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nεI\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\ndv1\ndt −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nA(τ )v1\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n+ εII\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nA(τ )v2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\ndv2\ndt −\ndh11\ndt −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\n\nA(τ )h11 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nεI\n20F (u0)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\nεII\n20v1.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nCollecting the ξ2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.4**\n2-terms gives us the following equation for h02\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n˙h02 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nA(τ )h02 = B(τ, v2, v2)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n2h11,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nto be solved in the space of T -periodic functions. This equation should be solvable, so the RHS should lay\nin the reachable space of the operator d\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nA(τ ):\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\ndt −\nT\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n∗\nϕ\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n, B(τ, v2, v2)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n2h11i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nThis condition can be satisﬁed by correctly tuning h11. In fact, εII\nprojection on v2. Due to(59) v2 does not lay in the reachable space of the d\nwe can impose that\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\ndt −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.1**\n20 is not yet determined, so h11 can have a\nA(τ ) operator, and therefore\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nϕ∗, h11i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\ndτ =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nϕ∗, B(τ, v2, v2)\ndτ.\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n20 uniquely, and since εII\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nThis last solvability condition determines εII\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\n20 determines the value of α, see (63)\nand (65), also α is now uniquely determined. So the center manifold expansion (8) has now become unique.\nNote that in fact the value of α is not needed since, as shown in Appendix B, it does not aﬀect the bifurcation\nscenario. Remark also that in order to compute the necessary coeﬃcients a and b by equations (64) and (67),\nthe second order expansion of the center manifold is not needed. Indeed, we have rewritten the formulas of\nthe normal form coeﬃcients in terms of the original and adjoint eigenfunctions. h20 or h11 are not needed,\ntherefore we don’t write down the BVPs for their unique solutions.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThe three-dimensional critical center manifold W c(Γ) at the R2 bifurcation can be parametrized locally by\n(τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n(68)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.5**\nu = u0 + ξ1v1 + ξ2v2 + H(τ, ξ), τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n[0, 2T ], ξ = (ξ1, ξ2)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nR2,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nwhere H satisﬁes H(2T, ξ) = H(0, ξ) and has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(69) H(τ, ξ) =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nh20ξ2\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n1 + h11ξ1ξ2 +\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nh02ξ2\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n2+\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nwhere all functions hij are 2T -periodic, the eigenfunction corresponding to eigenvalue\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nh30ξ3\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n1 +\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\nh21ξ2\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n1 ξ2 +\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\nh12ξ1ξ2\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\n2 +\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\nh03ξ3\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\n2 + O(\nk\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\n4),\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\nξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\nk\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n1 is given by\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(70)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nand the generalized eigenfunction by\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nA(τ )v1 = 0, τ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\n˙v1 −\nv1(T ) + v1(0) = 0,\n1 = 0,\nv1, v1i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n˙v2 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nA(τ )v2 + v1 = 0, τ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nv2(T ) + v2(0) = 0,\ndτ = 0,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nT\n0 h\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nv2, v1i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(71)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(72)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nv1(τ + T ) :=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nv1(τ ) and v2(τ + T ) :=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nv2(τ ) for τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n[0, T ].\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe functions v1 and v2 exist due to Lemma 5 of [23]. The functions hij of (69) can be found by solving\nappropriate BVPs, assuming that (2) restricted to W c(Γ) has normal form (9). As in the generalized period-\ndoubling case, we ﬁrst deduce a property for these functions hij. More general than in the GPD case, we\nξ2). This implies that\nhere have that u(τ, ξ1, ξ2) = u(τ + T,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nξ1,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\ni!j!\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\ni,j\nX\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nand thus\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nhij(τ )ξi\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n1ξj\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n2 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\ni!j!\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nhij (τ + T )(\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n1)i+jξi\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n1ξj\n2,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\ni,j\nX\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nhij(τ ) = (\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1)i+jhij(τ + T ),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.1**\nhij(τ ) for i + j odd, for\n[0, T ]. Taking these periodicity properties into account, we can reduce our observations to the discussion\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nfrom which follows that hij(τ + T ) = hij (τ ) for i + j even and hij(τ + T ) =\nτ\nof the interval [0, T ] instead of [0, 2T ].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nThe coeﬃcients α, a and b arise from the solvability conditions for the BVPs as integrals of scalar products\nover the interval [0, T ]. Speciﬁcally, those scalar products involve among other things the quadratic and cubic\nterms of (3) near the periodic solution u0, the eigenfunction v1. The adjoint eigenfunction ϕ∗ associated to\nthe trivial multiplier is the T -periodic solution of (29). The adjoint eigenfunction v∗\n1 is the unique solution\nof the problem\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(73)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n1 = 0, τ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nNote that we can indeed require this normalization since v2 is the last generalized eigenfunction of the\noriginal problem and therefore not orthogonal to all the eigenfunctions of the adjoint problem. We further\ndeﬁne the generalized adjoint eigenfunction v∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n2 as the unique solution of\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(74)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nsince, as above, v∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\n1 is not orthogonal to v2. Moreover, we have\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n2 (τ ) + AT(τ )v∗\n˙v∗\n2 −\n2(T ) + v∗\nv∗\nv2, v∗\n2 i\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nv∗\n1 = 0, τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n2 (0) = 0\ndτ = 0,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\nv∗\n2 , v1i\nh\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\ndτ =\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\nv∗\n2,\nh\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nA(τ )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\nv2i\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nh\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n+ AT (τ )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n2, v2i\nv\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\ndτ =\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nZ\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n1 , v2i\nv\nh\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\ndτ = 1.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nNote that\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(75)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\nv2, v1i\nh\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\ndτ =\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\nv∗\n1 , v1i\nh\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\ndτ =\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\nv∗\n2 , v2i\nh\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nTo derive the normal form coeﬃcients, we proceed as in the previous sections, namely, we substitute (68)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\ninto (2), and use (3) as well as (9) and (69).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nBy collecting the ξ0-terms we get the identity\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe linear terms provide two other identities, namely\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n˙u0 = F (u0).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nin correspondance with (70) and (71).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n˙v1 = A(τ )v1,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nv1 + ˙v2 = A(τ )v2,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nCollecting the ξ2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.5**\n2-terms gives us an equation for h02\n˙h02 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nA(τ )h02 = B(τ ; v2, v2)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n2h11,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h02(T ) = h02(0). In this space, the diﬀerential operator\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nA(τ ) is singular and its null-space is spanned by ˙u0. The Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\ngives us a normalization condition for function h11, i.e.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nϕ∗, B(τ ; v2, v2)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n2h11i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\ndτ = 0\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nϕ∗, h11i\nBy collecting the ξ1ξ2-terms we obtain the diﬀerential equation for h11\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nϕ∗, B(τ ; v2, v2)\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nA(τ )h11 = B(τ ; v1, v2)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\ndτ =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\nh20,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\ndτ.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nwhich must be solved in the space of functions satisfying h11(T ) = h11(0). The orthogonality condition\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\nϕ∗, B(τ ; v1, v2)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\ngives us a normalization condition for h20, i.e.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(76)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nϕ∗, h20i\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\ndτ =\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nϕ∗, B(τ ; v1, v2)\ni\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\ndτ.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nBy collecting the ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(77)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.5**\n1-terms we ﬁnd an equation for h20\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nA(τ )h20 = B(τ ; v1, v1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n2α ˙u0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h20(T ) = h20(0). In this space, the diﬀerential operator\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nA(τ ) is singular and its null-space is spanned by ˙u0. The Fredholm solvability condition\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n∗\nϕ\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n, B(τ ; v1, v1)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n2α ˙u0i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nleads to the expression\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(78)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nwhere v1 is deﬁned in (70).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nα =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nϕ∗, B(τ ; v1, v1)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\ndτ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nWith α deﬁned in this way we have to ﬁnd a normalization condition which makes the solution of (77)\nunique. Indeed, if h20 is a solution of (77) with h20(T ) = h20(0), also ˜h20 = h20 + ε1 ˙u0 is a solution, since\n˙u0 spans the kernel of the operator d\nA(τ ) in the space of T -periodic functions. The projection along the\nspace generated by ˙u0 is ﬁxed by solvability condition (76). So h20 can be found as the unique solution of\nthe BVP\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\ndt −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\nA(τ )h20 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(79)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\nB(τ ; v1, v1) + 2αF (u0) = 0, τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nh20(0) = 0,\nT\ndτ =\n0 h\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nϕ∗, h20i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n∈\nϕ∗, B(τ ; v1, v2)\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\ndτ.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nh20(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nIn the line of the previous observations, we can deﬁne h11 as the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n(80)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nA(τ )h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nwith h20 deﬁned in (79).\nBy collecting the ξ3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nB(τ ; v1, v2) + h20 = 0, τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n[0, T ]\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nh11(0) = 0,\ndτ = 1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nϕ∗, h11i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\nh11(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\n∈\nϕ∗, B(τ ; v2, v2)\ni\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nT\n0 h\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\ndτ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.5**\n1-terms we get an equation for h30\n˙h30 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.4**\nA(τ )h30 = C(τ ; v1, v1, v1) + 3B(τ ; v1, h20)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n6av2 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n6α ˙v1,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(81)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(82)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nwhich again must be solved in the space of functions satisfying h30(T ) =\ncondition of (73) into account, we obtain\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nh30(0). Taking the integral\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\na =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nv∗\n1 , C(τ ; v1, v1, v1) + 3B(τ ; v1, h20)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n6αA(τ )v1i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\ndτ,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\n0 h\nZ\nwhere α is deﬁned by (78), h20 is the solution of (79) and v1 and v∗\n1 are deﬁned in (70) and (71), respectively.\nAs remarked before, it is important to note that if h30 is a solution of (81) with h30(T ) = h30(0), also\n˜h30 = h30 + ǫI\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.3**\n30v1 is a solution, since v1 spans the nullspace of the operator d\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nA(τ ).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\ndt −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.5**\n1ξ2-terms we get the equation for h21\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\nA(τ )h21 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n2αv1 + C(τ ; v1, v1, v2) + B(τ ; h20, v2) + 2B(τ ; h11, v1),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(83)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nto be solved in the space of functions satisfying h21(T ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\nh21(0). The solvability of this equation implies\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nv∗\n1,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nh30 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n2bv2 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n2α ˙v2 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\n0 h\nZ\nNotice that the bξ2\nthe normal form parameter b zero since\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.2**\ndτ = 0.\n2αv1 + C(τ ; v1, v1, v2) + B(τ ; h20, v2) + 2B(τ ; h11, v1)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.1**\n1ξ2-term in normal form (9) is resonant: in fact we cannot use the freedom on h30 to make\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nv∗\n1, ˜h30i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\ndτ =\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nv∗\n1 , h30 + εI\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n30v1i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ndτ =\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nv∗\n1 , h30i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\ndτ.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nUsing the normalization from (73) and (75) gives us the following expression for b\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nb =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n∗\n1 ,\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\ndτ\n2αA(τ )v2 + C(τ ; v1, v1, v2) + B(τ ; h20, v2) + 2B(τ ; h11, v1)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n1 , h30i\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\ndτ.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nThere is no need to compute explicitly the cubic expansion of the center manifold since the last term of this\nsum can be rewritten as\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\n+ AT (τ )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\nv∗\n2, h30i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\ndτ =\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\nv∗\n2 ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n+ A(τ )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\ndτ\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nh30i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.4**\n∗\nv\n2, C(τ ; v1, v1, v1) + 3B(τ ; v1, h20)\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n6av2 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n6αAv1i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\n∗\nv\n2 , C(τ ; v1, v1, v1) + 3B(τ ; v1, h20)\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n6αAv1i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\ndτ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nobtaining\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(84)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nb =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nv∗\n1,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.3**\ndτ\n2αA(τ )v2 + C(τ ; v1, v1, v2) + B(τ ; h20, v2) + 2B(τ ; h11, v1)\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\n0 h\nZ\nwhere h20 is deﬁned in (79) and α calculated in (78). Notice that, since h30 appears on the RHS of equation\n(83), we have two degrees of freedom on h21. In fact, if h21 is a solution of (83), also ˜h21 = h21 + εI\n30v2\nis a solution since\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n21v1 + εI\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nv∗\n2 , C(τ ; v1, v1, v1) + 3B(τ ; v1, h20)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n6αAv1i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\ndτ,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n+\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nd˜h21\ndt −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nA(τ )˜h21 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\ndh21\ndt −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nA(τ )h21 + εI\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\ndv2\ndt −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nA(τ )v2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\ndh21\ndt −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nA(τ )h21 −\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nεI\n30v1.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.7**\nBy collecting the ξ1ξ2\n˙h12 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.4**\n2 -terms we get the equation for h12\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nA(τ )h12 = C(τ, v1, v2, v2) + B(τ, v1, h02) + 2B(τ, v2, h11)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n2h21,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nto be solved in the space of functions satisfying h12(T ) =\nimplies that\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nh12(0). The Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nv∗\n1 , C(τ, v1, v2, v2) + B(τ, v1, h02) + 2B(τ, v2, h11)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n2h21i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nAs mentioned before, h21 has a component in the direction of v2, which is not orthogonal to the adjoint\neigenfunction v∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.6**\n1, so it is possible to impose\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\ndτ =\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.2**\nv∗\ndτ.\n1, C(τ, v1, v2, v2) + B(τ, v1, h02) + 2B(τ, v2, h11)\ni\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n21 gives us as usual another freedom on h12 in the\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nFinally, collecting the ξ3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n2 -terms gives\n˙h03 −\nto be solved in the space of functions satisfying h03(T ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.3**\nA(τ )h03 = C(τ, v2, v2, v2) + 3B(v2, h02)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n3h12,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\nh03(0). The Fredholm solvability condition is\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nv∗\n1 , C(τ, v2, v2, v2) + 3B(v2, h02)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n3h12i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nwhich can be satisﬁed imposing\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nv∗\n1 , h12i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\ndτ =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nv∗\ndτ.\n1, C(τ, v2, v2, v2) + 3B(v2, h02)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n0 h\nZ\nThis last condition determines the value of εI\n21 and thus the third order center manifold expansion is uniquely\ndetermined. However, since this third order expansion of the center manifold is not needed for the compu-\ntation of the critical coeﬃcients, we don’t write down those conditions.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n3.6 Strong resonance 1:3 bifurcation\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nThe three-dimensional critical center manifold W c(Γ) at the R3 bifurcation can be parametrized locally by\n(τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(85)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.4**\nu = u0 + ξv + ¯ξ¯v(τ ) + H(τ, ξ, ¯ξ), τ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n[0, 3T ], ξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nC,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.2**\nwhere the real function H satisﬁes H(3T, ξ, ¯ξ) = H(0, ξ, ¯ξ) and has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(86)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nH(τ, ξ, ¯ξ) =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n+\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nh20ξ2 + h11ξ ¯ξ +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nh02 ¯ξ2 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nh03 ¯ξ3 + O(\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n4),\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nh12ξ ¯ξ2 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nh30ξ3 +\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nh21ξ2 ¯ξ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nwith hij(3T ) = hij (0) and hij = ¯hji so that hii is real. The eigenfunction v is deﬁned as the unique solution\nof the BVP\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(87)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nA(τ )v = 0, τ\n−\nei 2π\n3 v(0) = 0,\n1 = 0,\ndτ\ni\nand extended on the interval [0, 3T ] using the equivariance property of the normal form, i.e.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nv(T )\nT\n0 h\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n−\nv, v\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n˙v(τ )\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.6**\nv(τ + T ) := ei 2π\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.4**\n3 v(τ ) and v(τ + 2T ) := ei 4π\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n3 v(τ ) for τ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n[0, T ].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nThe deﬁnition of the conjugate eigenfunction ¯v follows immediately. These functions exist due to Lemma 2\nof [23].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nAs usual the functions hij can be found by solving appropriate BVPs, assuming that (2) restricted to\nW c(Γ) has the periodic normal form (10). Also here we can deduce a property for the functions hij. The\ndeﬁnition of v(τ ) in [0, 3T ] states that u(τ, ξ, ¯ξ) = u(τ + T, e−i2π/3ξ, ei2π/3 ¯ξ). Therefore,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nhkl(τ )ξk ¯ξl =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\nk!l!\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nXk,l\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\nk!l!\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\nXk,l\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.3**\nhkl(τ + T )(e−i2π/3)kξk(ei2π/3)l ¯ξl,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nand thus\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\n[0, T ]. This for example implies that hkk is T-periodic. These periodicity properties allow us to just\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nfor τ\nconcentrate on the interval [0, T ]\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThe adjoint eigenfunction ϕ∗ corresponding to the trivial multiplier is the unique T -periodic solution of\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nBVP (29). The adjoint eigenfunction v∗ satisﬁes\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(88)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nSimilarly, we obtain ¯v∗.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\n˙v∗(τ ) + AT(τ )v∗ = 0, τ\nv∗(T )\nT\n0 h\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nei 2π\n3 v∗(0) = 0,\n1 = 0.\ndτ\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n−\nv∗, v\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nWe now write down the homological equation and compare term by term. The constant and linear terms\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\ngive us as usual\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\n˙v\n−\nFrom the ξ2- or ¯ξ2-terms we obtain the following equation (or its complex conjugate)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n˙u0 = F (u0),\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nA(τ )v = 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nA(τ )¯v = 0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n˙¯v\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nA(τ )h20 = B(τ ; v, v)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n2¯b¯v,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h20(T ) = ei 4π\nA(τ )\nhas a range space with codimension one which is orthogonal to ¯v∗. So one Fredholm solvability condition is\ninvolved, namely\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\n3 h20(0). In this space the operator d\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\ndτ −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nwhich makes it possible to obtain the value of the coeﬃcient b. In fact,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n¯v∗, B(τ ; v, v)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2¯b¯v\ndτ = 0,\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\n(89)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\nb =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nv∗, B(τ ; ¯v, ¯v)\ndτ.\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nUsing this value for b we can ﬁnd h20 as the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nA(τ )h20 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nB(τ ; v, v) + 2¯b¯v = 0, τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(90)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.4**\nei 4π\n3 h20(0) = 0,\n¯v∗, h20i\ndτ = 0.\nBy collecting the ξ ¯ξ-terms we obtain an equation for h11\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n−\nT\n0 h\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nh20(T )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h11(T ) = h11(0). The Fredholm solvability condition with\nϕ∗ gives us the value of α1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nA(τ )h11 = B(τ ; v, ¯v)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nα1 ˙u0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(91)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nα1 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nZ\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n0 h\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nϕ∗, B(τ ; v, ¯v)\ndτ.\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nWith α1 deﬁned in this way, let h11 be the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nA(τ )h11 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nB(τ ; v, ¯v) + α1 ˙u0 = 0, τ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(92)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.4**\nϕ∗, h11i\nFinally, collecting the ξ2 ¯ξ-terms gives an equation for h21\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\nh11(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nh11(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.4**\nA(τ )h21 = C(τ ; v, v, ¯v) + 2B(τ ; v, h11) + B(τ ; ¯v, h20)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n2cv\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n2α1 ˙v,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nto be solved in the space of the functions satisfying h21(T ) = ei 2π\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\n3 h21(0). Therefore, there must hold that\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.4**\nv∗, C(τ ; v, v, ¯v) + 2B(τ ; v, h11) + B(τ ; ¯v, h20)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n2cv\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n2¯bh02 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n2α1 ˙v\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\ndτ = 0,\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nsuch that parameter c of (10) is determined by\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nc =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\n, C(τ ; v, v, ¯v) + 2B(τ ; v, h11) + B(τ ; ¯v, h20)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n2α1Av\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\ndτ,\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nwhere α1 and b are deﬁned by (91) and (89), respectively, and v, h11 and h20 are the unique solutions of the\nBVPs (87), (92) and (90).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nSince we have to check the solvability of all the equations up to the maximal order of the normal form,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nwe also collect the ξ3-terms and obtain\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n˙h30 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.4**\nA(τ )h30 = C(τ ; v, v, v) + 3B(τ ; v, h20)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n6¯bh11 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n6α2 ˙u0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nto be solved in the space of the functions satisfying h30(T ) = h30(0). Therefore, there must hold that\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.4**\nϕ∗, C(τ ; v, v, v) + 3B(τ ; v, h20)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n6¯bh11 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n6α2 ˙u0i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nwhich determines the value of α2, namely\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nα2 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\nZ\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.3**\nϕ∗, C(τ ; v, v, v) + 3B(τ ; v, h20)\ndτ.\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nRemark that as in the Chenciner case v is not uniquely determined. Indeed, when v is a solution of (87)\nC with γHγ = 1, then γv is also a solution. Then the adjoint function is given by γv∗, and b and\nand γ\nh20 are replaced by ¯γ3b and γ2h20, respectively. The normal form coeﬃcient c stays the same. However, the\nnormal form coeﬃcient b is multiplied with ¯γ3. This doesn’t aﬀect the bifurcation analysis since this normal\nform coeﬃcient only has to be diﬀerent from zero, and obviously γ\n= 0. Moreover, the analysis around the\nbifurcation point is independent from the sign of b.\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n3.7 Strong resonance 1:4 bifurcation\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nThe three-dimensional critical center manifold W c(Γ) at the R4 bifurcation can be parametrized locally by\n(τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(93)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.4**\nu = u0 + ξv + ¯ξ¯v(τ ) + H(τ, ξ, ¯ξ), τ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n[0, 4T ], ξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nC,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.2**\nwhere the real function H satisﬁes H(4T, ξ, ¯ξ) = H(0, ξ, ¯ξ) and has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nH(τ, ξ, ¯ξ) =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nwith hij(4T ) = hij (0) and hij = ¯hji so that hii is real, while v is deﬁned by\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nh03 ¯ξ3 + O(\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nh20ξ2 + h11ξ ¯ξ +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n4),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nh12ξ ¯ξ2 +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nh02 ¯ξ2 +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nh30ξ3 +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(94)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n+\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nh21ξ2 ¯ξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(95)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n˙v\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nA(τ )v = 0, τ\n−\nei π\n2 v(0) = 0,\n1 = 0,\ndτ\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.3**\nextended on [0, 4T ] using the equivariance property of the normal form, i.e.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nv(τ + T )\nv(τ + 2T )\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nv(τ + 3T )\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n[0, T ].\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n2 v(τ ) = iv(τ ),\nv(τ ),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n:= ei π\n:= eiπv(τ ) =\n:= ei 3π\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n2 v(τ ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\niv(τ ),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nfor τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nThe deﬁnition of the conjugate ¯v follows from this. These functions exist due to Lemma 2 of [23]. As\nusual the functions hij can be found by solving appropriate BVPs, assuming that (2) restricted to W c(Γ)\nhas the periodic normal form (11). Similar to the R1:3 case, there holds that\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nhkl(τ ) = hkl(τ + T )(e−iπ/2)k(eiπ/2)l,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nfor τ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n[0, T ].\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nThe adjoint eigenfunction ϕ∗ is deﬁned by the T -periodic solution of (29) and v∗ satisﬁes\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(96)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nSimilarly, we obtain ¯v∗.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\n˙v∗(τ ) + AT(τ )v∗ = 0, τ\nv∗(T )\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nei π\n2 v∗(0) = 0,\n1 = 0.\ndτ\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n−\nv∗, v\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nThe constant and the linear terms give the identities\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.2**\nFrom the ξ2- or ¯ξ2-terms the following equation (or its complex conjugate) is obtained\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n˙u0 = F (u0),\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nA(τ ) = 0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n˙v\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n˙¯v\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nA(τ )¯v = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\n(97)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\nA(τ )h20 = B(τ ; v, v).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nNotice that this equation is non-singular in the space of functions satisfying h20(T ) =\ngiven as the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\nh20(0). So h20 is\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(98)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nA(τ )h20 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nB(τ ; v, v) = 0, τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nh20(T ) + h20(0) = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nBy collecting the ξ ¯ξ-terms we obtain an equation for h11\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nA(τ )h11 = B(τ ; v, ¯v)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nα1 ˙u0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h11(T ) = h11(0). The Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\nϕ∗, B(τ ; v, ¯v)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nα1 ˙u0i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\ngives us the possibility to obtain the value of α1, namely given by (91). With this value of α1, h11 is the\nunique solution of BVP (92).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nThe ξ ¯ξ2-terms give an equation for h12\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n˙h12 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nA(τ )h12 = C(τ ; v, ¯v, ¯v) + B(τ ; v, h02) + 2B(τ ; ¯v, h11)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n2¯c¯v\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n2α1 ˙¯v,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n−\nih12(0). The Fredholm solvability condition leads\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nto be solved in the space of functions satisfying h12(T ) =\nus to the value of ¯c, namely\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(99)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n¯c =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\n¯v∗, C(τ ; v, ¯v, ¯v) + B(τ ; v, h02) + 2B(τ ; ¯v, h11)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nwhere α1 is deﬁned in (91), and v, h11 and h02 are the unique solutions of the BVPs (95), (92) and the\ncomplex conjugate of (98). Taking the complex conjugate gives us the critical coeﬃcient c.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nFinally, by collecting the ¯ξ3-terms we obtain an equation for h03\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\n˙h03 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nA(τ )h03 = C(τ ; ¯v, ¯v, ¯v) + 3B(τ ; ¯v, h02)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n6dv,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nto be solved in the space of the functions satisfying h03(T ) = ih03(0). The non-trivial Fredholm solvability\ncondition\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\n0 h\nZ\ngives us the value of the critical coeﬃcient d of (11), namely\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nv∗, C(τ ; ¯v, ¯v, ¯v) + 3B(τ ; ¯v, h02)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n6dv\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\ndτ = 0\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(100)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nd =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n∗\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\ndτ.\n, C(τ ; ¯v, ¯v, ¯v) + 3B(τ ; ¯v, h02)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nSo we ﬁnally obtain the value of\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\na =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nc\nd\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nwhich makes it possible to understand which bifurcation scenario of the R4 resonance we have.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nAlso in this case v is not uniquely determined, since for every γ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nC with γHγ = 1, γv is a solution.\nThen the adjoint eigenfunction is given by γv∗, and h20 is replaced by γ2h20. The normal form coeﬃcient c\nstays the same, but instead of d we get ¯γ4d. However, this again doesn’t inﬂuence the bifurcation analysis\nsince the study is determined by the above deﬁned a for which we need only\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nd\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n|\n**BLOCK**fs== 12.0**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\n3.8 Fold-Flip bifurcation\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe three-dimensional critical center manifold W c(Γ) at the LPPD bifurcation can be parametrized locally\nby (τ, ξ) as\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(101)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.5**\nu = u0 + ξ1v1 + ξ2v2 + H(τ, ξ), τ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n[0, 2T ], ξ = (ξ1, ξ2)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nR2,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nwhere H satisﬁes H(2T, ξ) = H(0, ξ) and has the Taylor expansion\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(102) H(τ, ξ) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nh20ξ2\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n1 + h11ξ1ξ2 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nh02ξ2\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n2 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nwhile the eigenfunctions v1 and v2 are given by\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nh30ξ3\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n1 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nh21ξ2\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n1 ξ2 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nh12ξ1ξ2\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n2 +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nh03ξ3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n2 + O(\nk\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n4),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nk\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(103)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\n(104)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(105)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n˙v1 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.4**\nA(τ )v1 −\nF (u0) = 0, τ\nv1(0) = 0,\nv1(T )\n−\ndτ = 0,\nv1, F (u0)\ni\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nA(τ )v2 = 0, τ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\n˙v2 −\nv2(T ) + v2(0) = 0,\n1 = 0\nv2, v2i\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\ndτ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.4**\nv1(τ + T ) := v1(τ ) and v2(τ + T ) :=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n[0, T ].\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThe functions v1 and v2 exist because of Lemma 2 and Lemma 5 of [23]. The functions hij can be found\nby solving appropriate BVPs, assuming that (2) restricted to W c(Γ) has the periodic normal form (12).\nMoreover, similar as before, u(τ, ξ1, ξ2) = u(τ + T, ξ1,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nξ2) such that\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n−\n1)jhij(τ + T ),\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nhij(τ ) = (\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nfor τ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\n[0, T ]. As before, we will reduce all computations to the interval [0, T ].\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nThe coeﬃcients of the normal form arise from the solvability conditions for the BVPs as integrals of scalar\nproducts over the interval [0, T ]. Speciﬁcally, those scalar products involve among other things the quadratic\nand cubic terms of (3) near the periodic solution u0, the generalized eigenfunction v1 and eigenfunction v2,\nand the adjoint eigenfunctions ϕ∗, v∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\n2 as solution of the problems\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n1 and v∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(106)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(107)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(108)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\n˙ϕ∗ + AT(τ )ϕ∗ = 0, τ\nϕ∗(T )\nϕ∗, v1i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nϕ∗(0) = 0,\n1 = 0,\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\nT\n0 h\nR\n1 + ϕ∗ = 0, τ\n1 + AT(τ )v∗\n˙v∗\nv∗\nv∗\n1(0) = 0,\n1 (T )\n−\nT\nv∗\ndτ = 0,\n1 , v1i\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2 = 0, τ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n2 + AT(τ )v∗\n˙v∗\n2(T ) + v∗\nv∗\nv∗\ndτ\n2, v2i\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nT\n0 h\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2(0) = 0,\n1 = 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\n−\nNote that the integral conditions are possible due to the spectral assumptions at the LPPD point. The\nfollowing orthogonality conditions hold automatically\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(109)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nϕ∗, F (u0)\ndτ =\ni\nh\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n=\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nZ\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nϕ∗, v2i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nh\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\ndτ =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\nv∗\n2 , v1i\nh\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\ndτ =\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nZ\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nv∗\n1 , v2i\nh\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\ndτ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\nZ\nv∗\ndτ\n2, F (u0)\ni\nh\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n= 0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nand since we have normalized the adjoint eigenfunction associated to multiplier 1 with the last generalized\neigenfunction, we have for free that\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(110) 1 =\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nϕ∗, v1i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nh\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\ndτ =\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nh\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n+ AT (τ )\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nv∗\n1, v1i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\ndτ\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nZ\nAs usual, to derive the normal form coeﬃcients we write down the homological equation and compare\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n∗\n1 ,\nv\nh\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nA(τ )\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\ndτ =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nv1i\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.8**r== 0.1**\n∗\ndτ .\n1 , F (u0)\nv\ni\nh\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nterm by term.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nBy collecting the constant and linear terms we get the identities\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\n˙u0 = F (u0),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\n˙v1 = A(τ )v1 + F (u0),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n˙v2 = A(τ )v2,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nand the complex conjugate of the last equation.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nBy collecting the ξ2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.5**\n1-terms we ﬁnd an equation for h20\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(111)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\nA(τ )h20 = B(τ ; v1, v1)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n2α20 ˙u0 + 2 ˙v1,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nto be solved in the space of functions satisfying h20(T ) = h20(0). In this space, the diﬀerential operator\nd\ndτ −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nA(τ ) is singular and its null-space is spanned by ˙u0. The Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nϕ∗, B(τ ; v1, v1)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n2a20v1 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n2α20 ˙u0 + 2 ˙v1i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\ngives us the possibility to calculate parameter a20 of our normal form, i.e.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(112)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\na20 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nϕ∗, B(τ ; v1, v1) + 2A(τ )v1i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\ndτ.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nWith a20 tuned in this way equation (111) is solvable, for any value of parameter α20. As in the Cusp\nof cycles case, we are free to choose parameter α20 as we want, and we take α20 = 0. This choice will not\ninﬂuence our ﬁnal conclusion about the kind of situation we are in.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nIn order to make the solution of (111) unique, we have to ﬁx the projection on the null-space of the\noperator, more speciﬁcally in the direction of F (u0). Therefore we impose the orthogonality with the adjoint\ngeneralized eigenfunction v∗\n1, and obtain h20 as the unique solution of the BVP\n˙h20 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.4**\nB(τ ; v1, v1) + 2a20v1 + 2α20F (u0)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nA(τ )h20 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.2**\n2F (u0) = 0, τ\nh20(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\n2A(τ )v1 −\nh20(T )\n−\nT\nv∗\n1, h20i\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nBy collecting the ξ1ξ2-terms we obtain a singular equation for h11\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\n−\nto be solved in the space of the functions that satisfy h11(T ) =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nA(τ )h11 = B(τ ; v1, v2)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nb11v2 + ˙v2,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\nh11(0). The Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n(113)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n∗\nv\n2 , B(τ ; v1, v2)\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nb11v2 + ˙v2i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\ngives us the possibility, using (104) and (108), to calculate coeﬃcient b11 of our normal form\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(114)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nb11 =\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\nv∗\n2, B(τ ; v1, v2) + A(τ )v2i\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\ndτ.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nWith b11 deﬁned in this way we can compute h11 as the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(115)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n˙h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nA(τ )h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nA(τ )v2 = 0, τ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.3**\nB(τ ; v1, v2) + b11v2 −\nh11(T ) + h11(0) = 0,\nv∗\ndτ = 0.\n2 , h11i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.4**\n\nCollecting the ξ2\n2-terms gives a singular equation for h02\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\nT\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(116)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n˙h02 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nA(τ )h02 = B(τ ; v2, v2)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n2a02v1 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n2α02 ˙u0\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.2**\nand, since this equation has to be solvable, the following Fredholm solvability condition is involved\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nϕ∗, B(τ ; v2, v2)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\n2a02v1 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n2α02 ˙u0i\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nfrom which we obtain an equation for a02\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\na02 =\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\ndτ.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nSo (116) is solvable, for any value of the parameter α02. For simplicity, we take α02 = 0.\nNotice that also here, the solution of (116) is orthogonal to the adjoint eigenfunction ϕ∗. Since we have to\nﬁx the projection in the direction of eigenfunction ˙u0, we deﬁne h02 as the unique solution of\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(118)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n˙h02 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\nA(τ )h02 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.3**\nB(τ ; v2, v2) + 2a02v1 + 2α02F (u0) = 0, τ\nh02(T )\n−\nT\nv∗\n1 , h02i\n0 h\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nh02(0) = 0,\ndτ = 0.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nBy collecting the ξ3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.4**\n1-terms we get a singular equation for h30\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n˙h30 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.5**\nA(τ )h30 = C(τ ; v1, v1, v1) + 3B(τ ; h20, v1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n6a20h20 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n6a30v1 + 3 ˙h20 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n6α30 ˙u0 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n6α20 ˙v1,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nwhere the Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nϕ∗, C(τ ; v1, v1, v1) + 3B(τ ; h20, v1)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n6a20h20 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n6a30v1 + 3 ˙h20 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n6α30 ˙u0 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n6α20 ˙v1i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\ngives us the value of a30\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n(119)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\na30 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nZ\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n0 h\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n, C(τ ; v1, v1, v1) + 3B(τ ; h20, v1)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n−\n+3(A(τ )h20 + B(τ ; v1, v1)) + 6(1\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n6a20h20\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nα20)A(τ )v1i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\na20.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nSimilarly, by collecting the ξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.3**\n1ξ2-terms we get a singular equation for h21\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\n˙h21 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.3**\nA(τ )h21 = C(τ ; v1, v1, v2) + B(τ ; h20, v2) + 2B(τ ; h11, v1)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n2b11h11 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\n2b21v2 + 2 ˙h11 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n2α20 ˙v2,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n2a20h11\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nwhere the Fredholm solvability condition\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nv∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.3**\n, C(τ ; v1, v1, v2) + B(τ ; h20, v2) + 2B(τ ; h11, v1)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n2a20h11 −\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n2b11h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\n2b21v2 + 2 ˙h11 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\n2α20 ˙v2i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.2**\ndτ = 0\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nproduces the value of b21\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(120)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nb21 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nBy collecting the ξ1ξ2\n˙h12 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\n∗\n2 , C(τ ; v1, v1, v2) + B(τ ; h20, v2) + 2B(τ ; h11, v1)\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.4**\n2b11h11 + 2(A(τ )h11 + B(τ ; v1, v2)) + 2(1\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n2a20h11\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\nα20)A(τ )v2i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.8**r== 0.2**\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.8**r== 0.2**\nb11.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.4**\n2 -terms we obtain a singular equation for h12\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n2a02h20 −\nwhere its solvability requires the following equality\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n2b11h02 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nA(τ )h12 = C(τ ; v1, v2, v2) + B(τ ; h02, v1) + 2B(τ ; h11, v2)\n2α02 ˙v1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n2a12v1 + ˙h02 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n2α12 ˙u0 −\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nϕ∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\n, C(τ ; v1, v2, v2) + B(τ ; h02, v1) + 2B(τ ; h11, v2)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nsuch that\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(121)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\na12 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n∗\nϕ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\n, C(τ ; v1, v2, v2) + B(τ ; h02, v1) + 2B(τ ; h11, v2)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n2b11h02\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n2a02h20 + A(τ )h02 + B(τ ; v2, v2)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n2α02A(τ )v1i\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\na02.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nFinally, the ξ3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n2 -terms give us the value of the last needed critical coeﬃcient. The equation for h03 is\n˙h03 −\nwith Fredholm solvability condition\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.4**\nA(τ )h03 = C(τ ; v2, v2, v2) + 3B(τ ; h02, v2)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n6a02h11 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n6b03v2 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n6α02 ˙v2,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\n∗\n2 , C(τ ; v2, v2, v2) + 3B(τ ; h02, v2)\nv\n0 h\nZ\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n6a02h11 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n6b03v2 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n6α02 ˙v2i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nand thus\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(122)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nb03 =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\nZ\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n0 h\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.4**\nv∗\n2, C(τ ; v2, v2, v2) + 3B(τ ; h02, v2)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n6a02h11 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n6α02A(τ )v2i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\ndτ.\n**BLOCK**fs== 14.3**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 14.3**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nImplementation issues\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nNumerical implementation of the formulas derived in the preceding sections requires the evaluation of in-\ntegrals of scalar functions over [0, T ] and the solution of nonsingular linear BVPs with integral constraints.\nSuch tasks can be carried out with continuation software such as auto [14], content [31], and matcont\n[11, 10]. In these software packages, periodic solutions to (1) are computed with the method of orthogonal\ncollocation with piecewise polynomials applied to properly formulated BVPs.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe standard BVP for the periodic solutions is formulated on the unit interval [0, 1], so that the period\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nT becomes a parameter, and it involves an integral phase condition:\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(123)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\nT f (x(τ ), α) = 0, τ\n−\nx(0)\n−\nx(τ ), ˙ξ(τ )\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nx(1) = 0,\ndτ = 0,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n˙x(τ )\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nwhere ξ is a previously calculated periodic solution to a nearby problem, rescaled to [0, 1].\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nIn the orthogonal collocation method [4], problem (123) is replaced by the following discretization:\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\n(124)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nm\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nj=0\nX\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nxi,j ˙ℓi,j(ζi,k)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nT f\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nm\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nj=0\nX\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nxi,j ℓi,j(ζi,k), α\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n= 0,\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nx0,0 −\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\nxN −1,m = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nN −1\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nm−1\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\ni=0\nX\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nj=0\nX\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nσi,jh\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nxi,j , ˙ξi,j i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n+ σN,0h\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nxN,0, ˙ξN,0i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n= 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nThe points xi,j form the approximation of x(τ ) with m + 1 equidistant mesh points\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nτi,j = τi +\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\nj\nm\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\n(τi+1 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\nτi),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.3**\nj = 0, 1, . . . , m,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nin each of the N intervals [τi, τi+1], where\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n0 = τ0 < τ1 <\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThe ℓi,j(τ )’s are the Lagrange basis polynomials, while the points ζi,j (j = 1, . . . , m) are Gauss points\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\n[9], i.e. the roots of the Legendre polynomial of degree m, all relative to the interval [τi, τi+1].\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nWith this choice of collocation points ζi,j , the approximation error at the mesh points has order of\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\naccuracy m,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nx(τi,j )\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nk\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\nxi,j k\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nO\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n(hm),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nwhere h = maxi=1,2,...,N {\norder of accuracy 2m,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nti}\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n, ti = τi −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.1**\nτi−1 (i = 1, . . . , N ), while for the coarse mesh points τi the error has\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\n(“superconvergence”).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nx(τi)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nk\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nxi,0k\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nO\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n(h2m)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nThe integration weight σi,j of τi,j is given by wj+1ti+1 for 0\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\n1 and 0 < j < m. For\ni = 0, . . . , N\n2, the integration weight of τi,m (τi,m = τi+1,0) is given by σi,m = wm+1ti+1 + w1ti+2, and\nthe integration weights of τ0 and τN are given by w1t1 and wm+1tN , respectively. In the above expressions,\nwj+1 is the Lagrange quadrature coeﬃcient.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nThe numerical continuation of solutions of (124) leads to structured, sparse linear systems, which in auto\n[14] and content [31] are solved by an eﬃcient, specially adapted elimination algorithm that computes the\nmultipliers as a by-product, without explicitly using the Poincar´e map. To detect codim 1 bifurcations, one\ncan specify test functions that are based on computing multipliers [13, 14] or on solving appropriate bordered\nlinear BVPs [15].\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n≤\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n≤\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ni\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\n4.1 Discretization symbols\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nIt is convenient to discretize all computed functions using the same mesh as in (124). For a given vector\nfunction η\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\n1([0, 1], Rn) we consider three diﬀerent discretizations:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.3**\n∈ C\nR(N m+1)n, the vector of the function values at the mesh points;\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\nRN mn, the vector of the function values at the collocation points;\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\nηM ∈\nηC ∈\nRn, where ηW1 is the vector of the function values at the collocation points\nηW = [\nmultiplied by the Gauss-Legendre weights and the lengths of the corresponding mesh intervals, and\nηW2 = η(0).\n**BLOCK**fs== 5.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nηW1\nηW2 ]\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nRN mn\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n×\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nFormally we also introduce the structured sparse matrix LC×M that converts a vector ηM of function\nvalues at the mesh points into a vector ηC of its values at the collocation points, namely, ηC = LC×M ηM .\nThis matrix is never formed explicitly; its entries are approximated by the ℓi,j(ζi,k)-coeﬃcients in (124).\nWe also need a matrix AC×M such that AC×M ηM = (A(t)η(t))C . Again this matrix need not be formed\nexplicitly. On the other hand, we do need the matrix (D\nT A(t))C×M ηM = ( ˙η(t)\nBC×M×M η1M η2M = (B(t; η1(t), η2(t)))C and\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\n−\nT A(t)η(t))C . Finally, let the tensors BC×M×M and CC×M×M×M be deﬁned by\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\nT A(t))C×M explicitly; it is deﬁned by (D\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.3**\nCC×M×M×M η1M η2M η3M = (C(t; η1(t), η2(t), η3(t)))C\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nP\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\n1([0, 1], Rn). (These tensors are not formed explicitly.)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nfor all ηi ∈ C\nLet f (t), g(t)\n∈ C\nN −1\nm\nj=1 ωj(fC)i,j ti+1 =\ni=0\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\n0([0, 1], R) be two scalar functions. Then the integral\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.1**\n0 f (t)dt is represented by\nm\nj=1(fW1 )i,j , where (fC )i,j = f (ζi,j) and ωj is the Gauss-Legendre\nR\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nN −1\ni=0\nquadrature coeﬃcient. The integral\nP\nwhere equality holds if g(t) is a piecewise polynomial of degree m or less on the given mesh. For vector\nfunctions f (t), g(t)\ndt is formally approximated by the same ex-\n∈ C\npression: f T\nf T\nW1 gC ≈\nW1 LC×M gM , where again we have equality if g(t) is a piecewise polynomial of degree m\nor less on the given mesh. Concerning the accuracy of the quadrature formulas, we ﬁrst note that accuracy\nis not an important issue for the phase integral in (123), as this equation only selects a speciﬁc solution\nfrom the continuum of solutions obtained by phase shifts. Similarly, the discretization of the normalization\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.2**\n0 f (t)g(t)dt is approximated with Gauss-Legendre by f T\nP\nR\n0([0, 1], Rn), the integral\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nf (t), g(t)\ni\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nW1gC ≈\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nP\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nintegrals does not aﬀect the inherent accuracy, including superconvergence at the main mesh points τi of the\nsolution of the discretized BVP. Discretization of integrals, as speciﬁed above, follows the standard Gauss\nquadrature error, which has order of accuracy 2m if, as mentioned, the function g(t) is a piecewise polynomial\nof degree m or less on the given mesh and if f (t) is suﬃciently smooth (in a piecewise sense). Otherwise,\nstill assuming suﬃcient piecewise smoothness, the order of accuracy of the numerical integrals is m + 1 if m\nis odd, and m + 2 if m is even. In particular, for the often used choice m = 4, the integrals would then have\norder of accuracy 6.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\n4.2 Cusp of cycles bifurcation\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nThe ﬁrst task is to rescale the computed functions to the interval [0, 1]. We start by deﬁning u1(t) = u0(T t) =\nu0(τ ) for t\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\n[0, 1]. The linear BVP’s (15) and (21) are replaced by\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(125)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nwith v(τ ) = v1(τ /T ) and\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n˙v1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nT A(t)v1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nT F (u1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\n−\nv1(1)\nv1(0) = 0,\ndt = 0,\nv1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\n1(t) + T AT(t)ϕ∗\n˙ϕ∗\nϕ∗\nϕ∗\n1(1)\n−\ndt\n1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nϕ∗\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n1(t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\n1(τ /T )/T , respectively. We also need to rescale the adjoint generalized eigenfunction deﬁned\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nwhere ϕ∗(τ ) = ϕ∗\nby (22):\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nwith v∗(τ ) = v∗\n(126)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.4**\n1 (t) + T ϕ∗\n1 (t) + T AT(t)v∗\n˙v∗\n1(t) = 0, t\nv∗\nv∗\n1(0) = 0,\n1(1)\n−\nv∗\ndt = 0,\n1(t), v1(t)\ni\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n0 h\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\n1 (τ /T )/T . Now, α1 = 0 and h2,1 is the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n˙h2,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nT A(t)h2,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.5**\nT B(t; v1(t), v1(t))\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n2T A(t)v1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n\nwhere h2(τ ) = h2,1(τ /T ). Therefore, we obtain\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\n2T F (u1(t)) + 2α1T F (u1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nh2,1(1)\n−\nv∗\n1 (t), h2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nh2,1(0) = 0,\ndt = 0,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(127)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nc =\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nϕ∗\n1(t),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\n+6A(t)v1(t) + 3B(t; h2,1(t), v1(t)) + C(t; v1(t), v1(t), v1(t))\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.2**\n6α1A(t)v1(t) + 3A(t)h2,1(t) + 3B(t; v1(t), v1(t))\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\ndt.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nWe now determine the matrix solutions for the several functions. We compute v1M by solving the\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\ndiscretization of (125)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(128)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nT A(t))C×M\n−\nδ1\nδ0 −\ngT\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nwhere a equals zero since the M\nobtained by solving the following system\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n×\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nv1M\na\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nT fC\n0n×1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n0 \n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.1**\nM upper left part of the big matrix is singular, g(t) = F (u1(t)), and p is\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\npT b\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n0M×1\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nwith r1 and r2 random vectors. In the solution of this system b = 0; in (128) we then use the normalized p.\nThis technique guarantees that we are working with well-deﬁned systems.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nWe will compute ϕ∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n1W instead of ϕ∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n1M since ϕ∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nthe matrix from (128). Formally, the computation of ϕ∗\nsince\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n1W can be computed with a matrix which is very similar to\n1W is based on Proposition C.1 from the appendix\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nwe have that\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nϕ∗\n\nϕ∗\n1(0)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\nis orthogonal to the range of\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nTherefore, ϕ∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\n1W can be obtained by solving\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n∗\n1)T\n(ϕ\nW\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nϕ∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n1(t) + T AT(t)ϕ∗\n˙ϕ∗\n1(0)\nD\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n1(1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nϕ∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n= 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\nT A(t)\nδ1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\n−\nδ0 −\nT A(t))C×M\nδ1\nδ0 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n= 0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.2**\n. By discretization we obtain\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n1)T\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nT A(t))C×M\nδ1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\np\n0 \n\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.2**\n\nT A(t))C×M\nδ1\nδ0 −\n1W is then rescaled to ensure that I1 = 1.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nwhere a equals zero and q is the normalized right null-vector of\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.1**\n. We then approximate\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nϕ∗\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nI =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\ndt by I1 = (ϕ∗\n1)T\n1(t), v1(t)\n0 h\ni\nIt is more eﬃcient to compute v∗\nR\n1 (t), ζ(t) > dt. Proposition C.5 learns us how to determine v∗\n0 < v∗\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nW1 LC×M v1M . ϕ∗\n1W than v∗\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n1M , since v∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nT Ah\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\nv∗\n\nv∗\n1(0)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n˙h\nh(0)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\nh(1)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n\n1 can be obtained by solving\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nfor all appropriate functions h, thus v∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n1 . Indeed,\nT ϕ∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\nh\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n1 will be used only to compute integrals of the form\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n1 )T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nW a\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\nT A(t))C×M v1C\n0n×1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nδ1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n(T ϕ∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n1)T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nW1 LC×M 0\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n0 \n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\n\nwhere a equals zero and p is deﬁned above.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nNext, (h2,1)M is found by solving the discretization of (126), namely,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n(D\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nT A(t))C×M\n−\nδ0 −\nδ1\n(v∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\np\n0 \n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nh2,1M\na\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nT BC×M×M v1M v1M + 2T AC×M v1M + 2T gC\n0n×1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nwith a = 0 and p deﬁned above.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.6**\nFinally, (127) is approximated by\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nc =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n1)T\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.4**\nW1 (3AC×M h2,1M + 3BC×M×M v1M v1M\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.2**\n+6AC×M v1M + 3BC×M×M h2,1M v1M + CC×M×M×M v1M v1M v1M ).\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\n4.3 Generalized period-doubling bifurcation\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nAs done in Section 4.2, we ﬁrst rescale the computed quantities to the interval [0, 1]. The linear BVPs (28)\nand (29) are replaced by\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n˙v1(t)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nT A(t)v1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n1(t) + T AT(t)ϕ∗\n˙ϕ∗\nϕ∗\nϕ∗\n1(1)\n−\ndt\n1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1(t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.5**\n0 h\nR\n1(τ /T )/T . This leads to the expression\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nϕ∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nwith ϕ∗(τ ) = ϕ∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nα1,1 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n∗\n1(t), B(t; v1(t), v1(t))\nϕ\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\ndt,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nwith α1,1 = T α1.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nThe adjoint eigenfunction, deﬁned by (30), determines the following rescaled v∗\n1 :\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(129)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nwith v∗(τ ) = v∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n1 (τ /T )/√T .\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n1 (t) + T AT(t)v∗\n˙v∗\n1(1) + v∗\nv∗\nv∗\ndt\n1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n1 (t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nLet h2,1 be the unique solution of the BVP\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\n˙h2,1(t)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nT A(t)h2,1(t)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.3**\nT B(t; v1(t), v1(t)) + 2α1,1T F (u1(t)) = 0, t\nh2,1(1)\nϕ∗\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nh2,1(0) = 0,\ndt = 0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\n−\n1(t), h2,1(t)\ni\nwhere h2(τ ) = h2,1(τ /T )/T , and h3,1 be the unique solution of the BVP\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n˙h3,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nT A(t)h3,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.5**\nT C(t; v1(t), v1(t), v1(t))\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nwhere h3(τ ) = h3,1(τ /T )/(√T T ).\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nTherefore, we obtain\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n3T B(t; v1(t), h2,1(t)) + 6α1,1T A(t)v1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.1**\nh3,1(1) + h3,1(0) = 0,\nv∗\ndt = 0,\n1(t), h3,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.9**r== 0.1**\n∈\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nα2,1 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nϕ∗\n1(t), D(t; v1(t), v1(t), v1(t), v1(t)) + 6C(t; v1(t), v1(t), h2,1(t)) + 3B(t; h2,1(t), h2,1(t))+\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n4B(t; v1(t), h3,1(t))\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\n12α1,1(A(t)h2,1(t) + B(t; v1(t), v1(t)))\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\ndt + α2\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n1,1,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nwith α2,1 = T 2α2.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nNow, h4,1 is obtained as unique solution of the following BVP\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n˙h4,1(t)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nT A(t)h4,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.7**\n−\n3T B(t; h2,1(t), h2,1(t))\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\n\n\nwith h4(τ ) = h4,1(τ /T )/T 2.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nT D(t; v1(t), v1(t), v1(t), v1(t))\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.2**\n6T C(t; v1(t), v1(t), h2,1(t))\n−\n4T B(t; v1(t), h3,1(t)) + 12α1,1T (A(t)h2,1(t) + B(t; v1(t), v1(t))\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n2α1,1F (u1(t))) + 24α2,1T F (u1(t)) = 0, t\nh4,1(1)\nϕ∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nh4,1(0) = 0,\ndt = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n−\n1(t), h4,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n0 h\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nFinally, we can write down the critical normal form coeﬃcient\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\ne =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n120 T 2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.2**\nv∗\n1 (t), E(t; v1(t), v1(t), v1(t), v1(t), v1(t)) + 10D(t; v1(t), v1(t), v1(t), h2,1(t))\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\n0 h\nZ\n+ 15C(t; v1(t), h2,1(t), h2,1(t)) + 10C(t; v1(t), v1(t), h3,1(t)) + 10B(t; h2,1(t), h3,1(t))\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\n+ 5B(t; v1(t), h4,1(t))\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n120α2,1A(t)v1(t)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\ndt.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nWe now come to the implementation details in MatCont. We compute v1M by solving\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nqT\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nwith p1 and q1 the rescaled unit vectors of\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nv1M\na1 \n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\np1\n0 \n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nrT\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nq1\na2 \n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nr1\n0 \n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n0C×1\n0n×1\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n0C×1\n0n×1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nrT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\npT\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\na3\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nr1\n0 \n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.2**\nwhere r1 and r2 are random vectors. Every ai is equal to zero. We normalize v1M by requiring\n1, where σj is the Lagrange quadrature coeﬃcient.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nj=0 σjh\nThe discretization of (129) can be computed with the same matrix, see Proposition C.2 of the appendix,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\nP\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.8**r== 0.1**\nP\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nN −1\ni=0\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\nm\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n1 )T\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nqT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\np1\n0 \n\ndt by I1 = (v∗\nv∗\n1 (t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\nwhere a = 0. We then approximate I =\nto ensure that I1 = 1.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nAn analogous matrix is used to compute ϕ∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n1W :\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n0 h\nR\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n1 )T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.6**r== 0.2**\nW1 LC×M v1M . v∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.8**r== 0.1**\n1W is then rescaled\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n1)T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nT A(t))C×M\nδ1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nwhere q is the normalized right null-vector and p the normalized left null-vector of\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n(D\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n\nand a is equal to zero. In what follows we will use these deﬁnitions for p, q, p1 and q1. We then approximate\nI =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.2**\n1W to ensure that I1 = 1.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\ndt by I1 = (ϕ∗\n1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\n0 h\nThis then leads to the discretization of the expression for α1,1:\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nW1 gC and normalize ϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n1)T\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nϕ∗\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\nT A(t))C×M\nδ1\nδ0 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\n(130)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\nα1,1 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n1)T\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nW1 BC×M×M v1M v1M .\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nNow, h2,1 and h3,1 are found by solving the following systems:\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nT A(t))C×M\n−\nδ0 −\nδ1\n(ϕ∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nh2,1M\na\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nT BC×M×M v1M v1M −\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n0n×1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n2α1,1T gC\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nh3,1M\nb\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nrespectively, with\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nrhs = T CC×M×M×M v1M v1M v1M + 3T BC×M×M v1M h2,1M −\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n6α1,1T AC×M v1M .\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\na and b will be zero.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nThus,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(131) α2,1 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1)T\nW1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nThen, h4,1 is found by\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.2**\nDC×M×M×M×M v1M v1M v1M v1M + 6CC×M×M×M v1M v1M h2,1M\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.3**\n+ 3BC×M×M h2,1M h2,1M + 4BC×M×M v1M h3,1M\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\n12α1,1(AC×M h2,1M + BC×M×M v1M v1M )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n!\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\n+ α2\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n1,1.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nT A(t))C×M\n−\nδ0 −\nδ1\n(ϕ∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nh4,1M\na\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\nrhs\n0n×1\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nrhs = T DC×M×M×M×M v1M v1M v1M v1M + 6T CC×M×M×M v1M v1M h2,1M\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.3**r== 0.3**\n+3T BC×M×M h2,1M h2,1M + 4T BC×M×M v1M h3,1M\n12α1,1T (AC×M h2,1M + BC×M×M v1M v1M −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n2α1,1gC )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n24α2,1T gC\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nand a = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nNow, we have all ingredients for the computation of the normal form coeﬃcient\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(132)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\ne =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n120T 2 (v∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n1 )T\nW1\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.1**\nEC×M×M×M×M×M v1M v1M v1M v1M v1M + 10DC×M×M×M×M v1M v1M v1M h2,1M\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\n+ 15CC×M×M×M v1M h2,1M h2,1M + 10CC×M×M×M v1M v1M h3,1M + 10BC×M×M h2,1M h3,1M\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\n+ 5BC×M×M v1M h4,1M −\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n120α2,1AC×M v1M −\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.1**\n20α1,1AC×M h3,1M\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n!\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n4.4 Chenciner bifurcation\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nAs before, we rescale the computed quantities to the interval [0, 1]. The linear BVPs (41), (29) and (42) are\nreplaced by\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(133)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nwith v(τ ) = v1(τ /T )/√T ,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n˙v1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nT A(t)v1(t) + iωT v1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nv1(1)\nv1(0) = 0,\n−\n1 = 0,\ndt\nv1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n1(τ /T )/T , and\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.5**\n1 (t) + T AT(t)v∗\n˙v∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\n1 (t) + iωT v∗\n1(t) = 0, t\nv∗\nv∗\n1(0) = 0,\n1(1)\n−\nv∗\n1 = 0,\ndt\n1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nwhere v∗(τ ) = v∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nR\n\n1 (τ /T )/√T , respectively.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nThen h20 is approximated by\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(134)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n˙h20,1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\nT A(t)h20,1(t) + 2 iωT h20,1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nwhere h20(τ ) = h20,1(τ /T )/T .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nT B(t; v1(t), v1(t)) = 0, t\nh20,1(1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nh20,1(0) = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nBefore being able to compute the approximation to h11 we need α1, deﬁned by (44):\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nα1,1 =\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n∗\nϕ\ndt,\n1(t), B(t; v1(t), ¯v1(t))\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nwith α1,1 = T α1. Therefore, we get\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n˙h11,1(t)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nT A(t)h11,1(t)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nwith h11(τ ) = h11,1(τ /T )/T .\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nNow, we can compute normal form coeﬃcient c:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.2**\nT B(t; v1(t), ¯v1(t)) + α1,1T F (u1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nh11,1(0) = 0,\n−\ndt = 0,\n1(t), h11,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nh11,1(1)\n\nϕ∗\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nc1 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n˙h21,1(t)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\nv∗\n1 (t), C(t; v1(t), v1(t), ¯v1(t))+2B(t; v1(t), h11,1(t))+B(t; ¯v1(t), h20,1(t))\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nwith c1 = T c. With c1 deﬁned in this way, h21M can be computed as follows\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.6**\nT A(t)h21,1(t) + iωT h21,1(t)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\nT C(t; v1(t), v1(t), ¯v1(t))\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.2**\n2T B(t; v1(t), h11,1(t))\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.1**\nT B(t; h20,1(t), ¯v1(t)) + 2ic1T v1(t) + 2α1,1T (A(t)v1(t)\niωv1(t)) = 0, t\n−\nh21,1(1)\nh21,1(0) = 0,\n−\n\nv∗\ndt = 0,\n1 (t), h21,1(t)\n0 h\ni\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n\nwhere h21(τ ) = h21,1(τ /T )/(√T T ).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nNext, the rescaling of h30 gives us\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n˙h30,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nT A(t)h30,1(t) + 3iωT h30,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nT C(t; v1(t), v1(t), v1(t))\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\n3T B(t; v1(t), h20,1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nh30,1(1)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nh30,1(0) = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nwith h30(τ ) = h30,1(τ /T )/(√T T ).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nNow, we need the rescaled h31,1 before being able to compute coeﬃcient α2,1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(135)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n˙h31,1(t)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.5**\nT A(t)h31,1(t) + 2iωT h31,1(t)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.6**\n3T C(t; v1(t), v1(t), h11,1(t))\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\n3T C(t; v1(t), ¯v1(t), h20,1(t))\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\n3T B(t; v1(t), h21,1(t))\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.2**\nT D(t; v1(t), v1(t), v1(t), ¯v1(t))\n3T B(t; h11,1(t), h20,1(t))\nT B(t; ¯v1(t), h30,1(t))\n2iωh20,1(t) + B(t; v1(t), v1(t))) = 0, t\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.5**\n+6ic1T h20,1(t) + 3α1,1T (A(t)h20,1(t)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nα2,1 =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nϕ∗\n1(t), D(t; v1(t), v1(t), ¯v1(t), ¯v1(t)) + C(t; v1(t), v1(t), h02,1(t)) + 4C(t; v1(t), ¯v1(t), h11,1(t))\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\n+ C(t; ¯v1(t), ¯v1(t), h20,1(t)) + 2B(t; h11,1(t), h11,1(t)) + 2B(t; v1(t), h12,1(t)) + B(t; h02,1(t), h20,1(t))\ndt + α2\n4α1,1(A(t)h11,1(t) + B(t; v1(t), ¯v1(t)))\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n+ 2B(t; ¯v1(t), h21,1(t))\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n1,1,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nwith α2,1 = T 2α2.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nNow, we still need h22,1(t):\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n˙h22,1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nT A(t)h22,1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.6**\n4T C(t; v1(t), ¯v1(t), h11,1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.4**\nT D(t; v1(t), v1(t), ¯v1(t), ¯v1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\n−\nT C(t; ¯v1(t), ¯v1(t), h20,1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\n2T B(t; v1(t), h12,1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\n−\n+4α1,1T (A(t)h11,1(t) + B(t; v1(t), ¯v1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\n−\nT B(t; h02,1(t), h20,1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.2**\nT C(t; v1(t), v1(t), h02,1(t))\n2T B(t; h11,1(t), h11,1(t))\n2T B(t; ¯v1(t), h21,1(t))\nα1,1F (u1(t))) + 4α2,1T F (u1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nh22,1(0) = 0,\n−\ndt = 0,\n1(t), h22,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nh22,1(1)\n\nϕ∗\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nwhere h22(τ ) = h22,1(τ /T )/T 2.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nTherefore, we can compute the critical coeﬃcient e:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\ne =\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.2**\nv∗\n1 (t), E(t; v1(t), v1(t), v1(t), ¯v1(t), ¯v1(t)) + D(t; v1(t), v1(t), v1(t), h02,1(t))\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.1**\n12T 2\n+6D(t; v1(t), v1(t), ¯v1(t), h11,1(t)) + 3D(t; v1(t), ¯v1(t), ¯v1(t), h20,1(t))\n+6C(t; v1(t), h11,1(t), h11,1(t)) + 3C(t; v1(t), v1(t), h12,1(t))\n+3C(t; v1(t), h02,1(t), h20,1(t)) + 6C(t; ¯v1(t), h11,1(t), h20,1(t)) + 6C(t; v1(t), ¯v1(t), h21,1(t))\n+C(t; ¯v1(t), ¯v1(t), h30,1(t))\n+3B(t; h12,1(t), h20,1(t)) + 6B(t; h11,1(t), h21,1(t)) + 3B(t; v1(t), h22,1(t))\n+B(t; h02,1(t), h30,1(t)) + 2B(t; ¯v1(t), h31,1(t))\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n12α2,1A(t)v1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\n6α1,1(A(t)h21,1(t) + 2B(t; v1(t), h11,1(t)) + C(t; v1(t), v1(t), ¯v1(t))\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\n−\n+B(t; h20,1(t), ¯v1(t))\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\ndt + α2,1i\n2α1,1A(t)v1(t))\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\nω\nT 2 + α1,1i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\nc1\nT 2 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nα2\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n1,1i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.2**\nω\nT 2 .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nWe now impose the matrix solutions for the functions. We compute v1M by solving the discretization of\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(133)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.5**\nT A(t) + iωT L)C×M\nδ1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nδ0 −\nqH\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nv1,1M\na\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\np2\n0 \n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n0C×1\n0n×1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nwith a = 0, where q2 is the normalized right null-vector of the complex matrix K =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nand p2 the normalized right null-vector of K H. This vector is then rescaled so that\nFor the computation of ϕ1M , we use Proposition C.1 to obtain\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nδ0 −\n\n0 < v1(t), v1(t) > dt = 1.\nR\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n1)T\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\nT A(t))C×M\nδ1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.3**\np\n0 \n\n\ndt by I1 = (ϕ∗\nϕ∗\n1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nwhere a equals zero. We then approximate I =\nrescale ϕ∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n1)T\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nW1 LC×M gM and we\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.1**\n1 lies in the kernel of the\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nover there deﬁned φ2, we have that\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nby solving\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nv∗\n\nv∗\n1(0)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\n1([0, 1], Cn)). The eigenfunction v∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n1 is thus computed\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nφ1(\nC\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1 )H\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nW a\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\nT A(t) + iωT L)C×M\nδ1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nδ0 −\nqH\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\np2\n0 \n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\n\n\ndt by I1 = (v∗\nv∗\n1 (t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\n\nW1 LC×M v1M and we rescale v∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n1 )H\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n1W so that I1 = 1.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nWe then approximate I =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n0 h\n(134) is approximated by\nR\nT A(t) + 2 iωT L)C×M\nδ1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nδ0 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nh20,1M =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\nT BC×M×M v1M v1M\n0n×1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nThe coeﬃcient α1,1 can be approximated as\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n∗\n1)T\nα1,1 = (ϕ\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nW1 BC×M×M v1M ¯v1M\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nwhich gives then all the information to determine the real function h11,1:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nT A(t))C×M\n−\nδ0 −\nδ1\n(ϕ∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nwhere a equals zero.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nh11,1M\na\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nT BC×M×M v1M ¯v1M −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n0n×1\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nα1,1T gC\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nThen an approximation for the rescaled normal form coeﬃcient c1 is given by\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nc1 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n(v∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n1 )H\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.2**r== 0.2**\nW1 (CC×M×M×M v1M v1M ¯v1M +2BC×M×M v1M h11,1M +BC×M×M ¯v1M h20,1M −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nNext, we determine the third order coeﬃcients of the center manifold expansion, namely\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.5**\nT A(t) + iωT L)C×M\nδ0 −\nδ1\n1)H\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n(v∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nh21,1M\na\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\np2\n0 \n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nrhs\n0n×1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\n0 \n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nwhere\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nrhs = T CC×M×M×M v1M v1M ¯v1M + 2T BC×M×M v1M h11,1M + T BC×M×M h20,1M ¯v1M −\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\n2α1,1T (AC×M v1M −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\niωLC×M v1M )\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n2ic1T LC×M v1M\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nand a = 0, and\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.7**\nT A(t) + 3iωT L)C×M\nδ1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nδ0 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\n\nThe approximation to (135) is given by\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nh30,1M =\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.2**\nT CC×M×M×M v1M v1M v1M + 3T BC×M×M v1M h20,1M\n0n×1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nT A(t) + 2iωT L)C×M\nδ1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nδ0 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nh31,1M =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nrhs\n0n×1 \n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nrhs = T DC×M×M×M×M v1M v1M v1M ¯v1M + 3T CC×M×M×M v1M v1M h11,1M + 3T CC×M×M×M v1M ¯v1M h20,1M\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\n+3T BC×M×M h11,1M h20,1M + 3T BC×M×M v1M h21,1M + T BC×M×M ¯v1M h30,1M\n6ic1T LC×M h20,1M −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nwhile\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nα2,1 =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n∗\n1)T\n(ϕ\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.1**\nW1 (DC×M×M×M×M v1M v1M ¯v1M ¯v1M +CC×M×M×M v1M v1M h02,1M +4CC×M×M×M v1M ¯v1M h11,1M\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\n+ CC×M×M×M ¯v1M ¯v1M h20,1M + 2BC×M×M h11,1M h11,1M + 2BC×M×M v1M h12,1M + BC×M×M h02,1M h20,1M\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\n+ 2BC×M×M ¯v1M h21,1M −\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n4α1,1(AC×M h11,1M + BC×M×M v1M ¯v1M )) + α2\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n1,1.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nNow, we are able to compute h22,1 by\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.5**\nT A(t))C×M\n−\nδ0 −\nδ1\n(ϕ∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nh22,1M\na\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nrhs\n0n×1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n0 \n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.3**\nrhs = T DC×M×M×M×M v1M v1M ¯v1M ¯v1M + T CC×M×M×M v1M v1M h02,1M −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n4α2,1T gC\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\n+4T CC×M×M×M v1M ¯v1M h11,1M + 2T BC×M×M h11,1M h11,1M + 2T BC×M×M v1M h12,1M\n+T CC×M×M×M ¯v1M ¯v1M h20,1M + T BC×M×M h02,1M h20,1M + 2T BC×M×M ¯v1M h21,1M\n4α1,1T (AC×M h11,1M + BC×M×M v1M ¯v1M −\nand a = 0. For the ﬁfth order coeﬃcient of the normal form, we then obtain\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nα1gC)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\ne =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n1 )H\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nW1 (EC×M×M×M×M×M v1M v1M v1M ¯v1M ¯v1M + DC×M×M×M×M v1M v1M v1M h02,1M\n**BLOCK**fs== 7.0**b== 0.3**t== 0.5**l== 0.2**r== 0.1**\n12T 2 (v∗\n+6DC×M×M×M×M v1M v1M ¯v1M h11,1M + 3DC×M×M×M×M v1M ¯v1M ¯v1M h20,1M\n+6CC×M×M×M v1M h11,1M h11,1M + 3CC×M×M×M v1M v1M h12,1M\n+3CC×M×M×M v1M h02,1M h20,1M + 6CC×M×M×M ¯v1M h11,1M h20,1M + 6CC×M×M×M v1M ¯v1M h21,1M\n+CC×M×M×M ¯v1M ¯v1M h30,1M + 3BC×M×M h12,1M h20,1M\n+6BC×M×M h11,1M h21,1M + 3BC×M×M v1M h22,1M\n+BC×M×M h02,1M h30,1M + 2BC×M×M ¯v1M h31,1M −\n6α1,1(AC×M h21,1M + 2BC×M×M v1M h11,1M + CC×M×M×M v1M v1M ¯v1M\n−\nω\n+BC×M×M h20,1M ¯v1M −\nT 2 .\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n2α1,1AC×M v1M )) + α2,1i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nω\nT 2 + α1,1i\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n12α2,1AC×M v1M\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\nc1\nT 2 −\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n1,1i\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nα2\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n4.5 Strong resonance 1:1 bifurcation\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nAgain, we rescale the computed quantities to the interval [0, 1]. The linear BVPs (56) and (57) are replaced\nby\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n˙v1,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nwhere v1(τ ) = v1,1(τ /T ), and\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nT A(t)v1,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n−\nv1,1(1)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nT F (u1(t)) = 0, t\nv1,1(0) = 0,\ndt = 0,\nv1,1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\n˙v2,1(t)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nT A(t)v2,1(t) + T v1,1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\nv2,1(1)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nv2,1(0) = 0,\ndt = 0,\nv2,1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.4**\nThe rescaling of the adjoint eigenfunction, deﬁned by (60), gives us\n1(t) + T AT(t)ϕ∗\n˙ϕ∗\nϕ∗\nϕ∗\n1(1)\n−\nϕ∗\ndt\n1(t), v2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1(t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.5**\n0 h\n−\n1(τ /T )/T . This leads to the expression\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nwhere ϕ∗(τ ) = ϕ∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\na =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nZ\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\nϕ∗\ndt.\n1(t), 2A(t)v1,1(t) + B(t; v1,1(t), v1,1(t))\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nDeﬁnition (61) of the ﬁrst generalized adjoint eigenfunction is rescaled as\nT ϕ∗\n1,1(t) + T AT(t)v∗\n˙v∗\n1,1(t)\n−\nv∗\nv∗\n1,1(0) = 0,\n1,1(1)\n−\nv∗\ndt = 0,\n1,1(t), v2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\n1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nwith v∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n1(τ ) = v∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.1**\n1,1(τ /T )/T , which then gives all the information we need to compute the critical coeﬃcient b\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(136) b =\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nϕ∗\ndt +\n1(t), B(t; v1,1(t), v2,1(t)) + A(t)v2,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n0 h\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.1**\nv∗\ndt.\n1,1(t), 2A(t)v1,1(t) + B(t; v1,1(t), v1,1(t))\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nThe generalized eigenfunctions are computed as\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nT A(t))C×M\n−\nδ0 −\nδ1\ngT\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nv1,1M\na\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nT gC\n0n×1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\nT A(t))C×M\n−\nδ0 −\nδ1\ngT\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nwith p the left null-vector, as before.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nv2,1M\na\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\nT v1,1C\n0n×1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nAs always, we compute the adjoint eigenfunctions by means of the transposed of the usual matrix, and\n1W is based on Proposition C.1 from the\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.4**\n1M . Formally, the computation of ϕ∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nobtain here ϕ∗\nappendix and as in the cusp of cycles case we obtain then\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n1W instead of ϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n1)T\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.1**\nW1 LC×M v2,1M and rescale ϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n1W\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n1)T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nT A(t))C×M\nδ1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\np\n0 \n\ndt by I1 = (ϕ∗\n1(t), v2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nwhere a equals zero. We approximate I =\nto ensure that I1 = 1.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n0 h\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.2**\nHaving found v1,1M and ϕ∗\n1W , the ﬁrst normal form coeﬃcient of interest can be computed as\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.3**\nW1 (2AC×M v1,1M + BC×M×M v1,1M v1,1M ).\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n1)T\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n(ϕ∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\na =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nNow we still need v∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\n1,1. The technique of Proposition C.5 from the appendix is used to obtain (v∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n1,1)W ,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nnamely\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwhere a = 0.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n1,1)T\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nW a\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.4**\nT A(t))C×M v2,1C\n0n×1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nδ1\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nT (ϕ∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n1)T\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nW1 LC×M 0\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nFinally (136) is approximated by\nb = (ϕ∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n1)T\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.2**r== 0.4**\nW1 (BC×M×M v1,1M v2,1M + AC×M v2,1M ) + (v∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n1,1)T\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.1**\nW1 (2AC×M v1,1M + BC×M×M v1,1M v1,1M ).\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nAs before, we rescale the computed quantities to the interval [0, 1]. The linear BVPs (70) and (71) are\nreplaced by\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(137)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nwith v1(τ ) = v1,1(τ /T )/√T , and\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n˙v1,1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nT A(t)v1,1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nv1,1(1) + v1,1(0) = 0,\n1 = 0,\ndt\nv1,1(t), v1,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n˙v2,1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nwhere v2(τ ) = v2,1(τ /T )/√T .\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\nT A(t)v2,1(t) + T v1,1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nv2,1(1) + v2,1(0) = 0,\ndt = 0,\nv2,1(t), v1,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nThe rescaling of the adjoint eigenfunction gives us\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\n1(t) + T AT(t)ϕ∗\n˙ϕ∗\n1(t) = 0, t\nϕ∗\nϕ∗\n1(1)\n1(0) = 0,\n−\n1 = 0,\ndt\n1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\nϕ∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nwhere ϕ∗(τ ) = ϕ∗\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\n\n1(τ /T )/T , so we can compute\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nα1 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nϕ∗\n1(t), B(t; v1,1(t), v1,1(t))\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\ndt,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.2**\nwith α1 = T α. With α1 deﬁned in this way, let h20,1 be the unique solution of the BVP\n(138)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n˙h20,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nT A(t)h20,1(t)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nh20,1(0) = 0,\n\ndt =\n0 h\nR\nThe rescaling of the adjoint eigenfunction corresponding with multiplier\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.3**\nT B(t; v1,1(t), v1,1(t)) + 2α1T F (u1(t)) = 0, t\nh20,1(1)\n\nϕ∗\n0 h\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n−\n1(t), h20,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nwhere h20(τ ) = h20,1(τ /T )/T .\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n∈\nϕ∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.1**\n1(t), B(v1,1(t), v2,1(t))\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\ndt,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.1**\n1 and the adjoint generalized\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\neigenfunction gives us\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\nwith v∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.8**\n1(τ ) = v∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.6**\n1,1(τ /T )/√T , and\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\n1,1(t) + T AT(t)v∗\n˙v∗\n1,1(t) = 0, t\n1,1(1) + v∗\nv∗\n1,1(0) = 0,\nv∗\n1 = 0,\ndt\n1,1(t), v2,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.4**\n2,1(t) + T AT(t)v∗\nT v∗\n˙v∗\n2,1(t)\n−\n2,1(1) + v∗\nv∗\nv2,1(t), v∗\n2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\n1,1(t) = 0, t\n2,1(0) = 0,\ndt = 0,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n0 h\nR\n2,1(τ /T )/√T , respectively.\nThe rescaled critical coeﬃcient is then\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n2(τ ) = v∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nwhere v∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(139)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\na1 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwith a1 = T a.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.3**\n∗\n1,1(t), C(t; v1,1(t), v1,1(t), v1,1(t)) + 3B(t; v1,1(t), h20,1(t))\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n6α1A(t)v1,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\ndt,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nWe replace (80) by\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n˙h11,1(t)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nT A(t)h11,1(t)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.3**\nT B(t; v1,1(t), v2,1(t)) + T h20,1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.3**\nh11,1(0) = 0,\ndt = 1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n−\n1(t), h11,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nh11,1(1)\n\nϕ∗\n0 h\nR\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n∈\nϕ∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.1**\n1(t), B(v2,1(t), v2,1(t))\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\ndt,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nwith h11(τ ) = h11,1(τ /T )/T , to ﬁnally obtain that\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nb =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n∗\n1,1(t),\nv\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n+\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n2T\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.4**\n2α1A(t)v2,1(t) + C(t; v1,1(t), v1,1(t), v2,1(t))\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.3**\ndt\n+ B(t; h20,1(t), v2,1(t)) + 2B(t; h11,1(t), v1,1(t))\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\nv∗\n2,1(t), C(t; v1,1(t), v1,1(t), v1,1(t)) + 3B(t; v1,1(t), h20,1(t))\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\ndt.\n6α1A(t)v1,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nWe now come to the second part of the implementation details and deﬁne the matrix solutions for the\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nseveral functions. We compute v1,1M by solving the discretization of (137)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n(140)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nqT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nv1,1M\na\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\n0C×1\n0n×1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\np1\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n\nwhere a = 0. We then normalize v1,1M by requiring\nthe Lagrange quadrature coeﬃcient.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nN −1\ni=0\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nm\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\nj=0 σj h\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\n(v1,1M )i,j, (v1,1M )i,j i\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n= 1, where σj is\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nP\nWe have now obtained the value of v1,1 in the mesh points. However, since v1,1 is used in the integral\ncondition for v2,1, we have to transfer this vector to the collocation points and multiply it with the Gauss-\nLegendre weights and the lenghts of the corresponding intervals, giving us vector (v1,1)W1 . Then, v2,1M can\nbe found by solving the following system\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nwhere a equals zero.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nT A(t))C×M\nδ0 + δ1\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nW1 LC×M\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n−\n(v1,1)T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nv2,1M\na\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\np1\n0 \n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nT v1,1C\n0n×1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nThe adjoint eigenfunction corresponding to the trivial eigenvalue is computed with an analogous matrix\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nas in (140):\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.4**\nwhile the adjoint eigenfunction corresponding to eigenvalue\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n1 can be found by\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nT A(t))C×M\nδ1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n1)T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nW a1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n1,1)T\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nW a2\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nqT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\np1\n0 \n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\n\n\nwhere a1 and a2 are equal to zero. ϕ∗\n1W is then rescaled to make sure that (ϕ∗\n1)T\nso that (v∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.6**\nW1 LC×M v2,1M = 1.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n1,1)T\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\nMaking use of Proposition C.6, we obtain the adjoint generalized eigenfunction by solving\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nW1 LC×M gM = 1 and v∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n1,1W\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n2,1)T\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nW a\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\nT (v∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n1,1)T\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nW1 LC×M 0\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\n1W1 and v1,1M , α1 can be computed as\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nα1 =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n∗\n1)T\n(ϕ\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nW1 BC×M×M v1,1M v1,1M .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.4**\nNow, h20,1M is found by solving the discretization of (138), namely,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n(D\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nT A(t))C×M\n−\nδ0 −\nδ1\n(ϕ∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nand (139) is approximated by\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nh20,1M\na\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nT BC×M×M v1,1M v1,1M −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n2α1T gC\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n0n×1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n1)T\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.2**\nW1 BC×M×M v1,1M v2,1M\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\na1 =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n(v\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n∗\n1,1)T\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.3**\nW1 (CC×M×M×M v1,1M v1,1M v1,1M + 3BC×M×M v1,1M h20,1M −\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\n6α1AC×M v1,1M ) .\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nNext, h11,1M is found by\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n(D\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\nT A(t))C×M\n−\nδ0 −\nδ1\n(ϕ∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nFinally, we obtain\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\nh11,1M\na\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nb =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n(v∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n1,1)T\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nW1 (\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n2α1AC×M v2,1M\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nT BC×M×M v1,1M v2,1M −\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nT h20,1C\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0n×1\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n2 (ϕ∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n1)T\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.2**\nW1 BC×M×M v2,1M v2,1M\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\n+ CC×M×M×M v1,1M v1,1M v2,1M + BC×M×M h20,1M v2,1M + 2BC×M×M h11,1M v1,1M )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n+\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\n2T\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n2,1)T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\nW1 (CC×M×M×M v1,1M v1,1M v1,1M + 3BC×M×M v1,1M h20,1M −\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n6α1AC×M v1,1M ).\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n4.7 Strong resonance 1:3 bifurcation\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nAs before, we rescale the computed quantities to the interval [0, 1]. The BVPs for the eigenfunction and its\nadjoint belonging to eigenvalue ei 2π\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n3 are replaced by\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(141)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nwith v(τ ) = v1(τ /T )/√T , and\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nT A(t)v1(t) = 0, t\n˙v1(t)\n−\nei 2π\n3 v1(0) = 0,\nv1(1)\n1 = 0,\ndt\nv1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\n1(t) + T AT(t)v∗\n˙v∗\nei 2π\n3 v∗\nv∗\n1 (1)\n−\nv∗\ndt\n1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\n1 (t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nwhere v∗(τ ) = v∗\ngives\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\n1(τ /T )/√T . The rescaling of the adjoint eigenfunction corresponding to the trivial multiplier\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(142)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nwith ϕ∗(τ ) = ϕ∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\n1(t) + T AT(t)ϕ∗\n˙ϕ∗\nϕ∗\nϕ∗\n1(1)\n−\nϕ∗\ndt\n1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\n1(t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThese eigenfunctions make it already possible to compute the following 2 rescaled normal form coeﬃcients\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(143)\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nα1,1 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.7**\nwhere α1,1 = T α1, and\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n∗\ndt,\n1(t), B(v1(t), ¯v1(t))\nϕ\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nb1 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nv∗\ndt,\n1 (t), B(¯v1(t), ¯v1(t))\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nwith b1 = √T b.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\nThe rescaled second order functions in the center manifold expansion are solutions of\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n˙h20,1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\nT A(t)h20,1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.3**\nT B(v1(t), v1(t)) + 2¯b1T ¯v1(t) = 0, t\nei 4π\n3 h20,1(0) = 0,\nh20,1(1)\n−\n\n¯v∗\ndt = 0,\n1(t), h20,1(t)\n0 h\ni\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\n[0, T ],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\n\nwith h20(τ ) = h20,1(τ /T )/T , and\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n˙h11,1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nT A(t)h11,1 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n(144)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nwith h11(τ ) = h11,1(τ /T )/T . This all results then in\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nT B(v1(t), ¯v1(t)) + α1,1T F (u1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\nh11,1(0) = 0,\n−\ndt = 0,\n1(t), h11,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nh11,1(1)\n\nϕ∗\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nc =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nZ\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n0 h\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.3**\nv∗\n1(t), C(v1(t), v1(t), ¯v1(t)) + 2B(v1(t), h11,1(t)) + B(¯v1(t), h20,1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\ndt.\n2α1,1Av1(t)\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nWe now come to the implementation details in MatCont. Eigenfunction v1, determined by (141), is\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\ncomputed by\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n−\nδ0 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nT A(t))C×M\ne−i 2π\nqH\n**BLOCK**fs== 5.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n3 δ1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\np3\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nv1M\na\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n0C×1\n0n×1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n\nwith a = 0. We then normalize v1M by requiring\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\nN −1\ni=0\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nLagrange quadrature coeﬃcient. q3 is the normalized right null-vector of K =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nP\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nP\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\nm\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.6**r== 0.2**\n\n\n(v1M )i,j, (v1M )i,j i\nj=0 σjh\n−\nδ0 −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n(D\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n= 1, where σj is the\nT A(t))C×M\ne−iθδ1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\nand p3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nthe normalized right null-vector of K H, with θ = 2π\n3 .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nTo compute the adjoint eigenfunction v∗\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nKer(φ2), this function can be obtained by solving\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nSince v∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n1 ∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.1**\n1 , we apply Proposition C.4 from the appendix with θ = 2π\n3 .\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n(145)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n1 )H\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\n1W is rescaled such that (v∗\nv∗\ndiscretized by\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n1 )H\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n−\nδ0 −\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nT A(t))C×M\ne−i 2π\nqH\n**BLOCK**fs== 5.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n3 δ1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\np3\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.1**\nW1 LC×M v1M = 1. The adjoint eigenfunction corresponding to eigenvalue 1 is\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n(146)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n1)T\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\nT A(t))C×M\nδ1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nwhere a equals zero. ϕ∗\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nThe normal form coeﬃcients α1,1 and b1 become\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(147)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nα1,1 = (ϕ∗\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n1)T\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nW1 BC×M×M v1M ¯v1M\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nb1 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n(v∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n1)H\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nW1 BC×M×M ¯v1M ¯v1M .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nBy computing ﬁrst the complex conjugate of h20,1 we can use the same matrix as in (145), except for the\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nlast line which represents the integral condition, to get\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n(D\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\nT A(t))C×M\ne−i 2π\n3 δ1\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\n−\nδ0 −\n(v∗\n1 )H\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\np3\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n¯h20,1M\na\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nT BC×M×M ¯v1M ¯v1M −\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n0n×1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n2b1T v1C\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nTo obtain the discretization of h11,1, the following system is solved\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(148)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nT A(t))C×M\n−\nδ0 −\nδ1\n(ϕ∗\n1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nwhich gives us\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nh11,1M\na\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nT BC×M×M v1M ¯v1M −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0n×1\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nα1,1T gC\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nc =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n(v∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n1 )H\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nW1 (CC×M×M×M v1M v1M ¯v1M + 2BC×M×M v1M h11,1M + BC×M×M ¯v1M h20,1M −\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n2α1,1AC×M v1M ).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\n4.8 Strong resonance 1:4 bifurcation\nThe eigenfunction and the adjoint eigenfunction corresponding to multiplier ei π\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.1**\n2 are given by the solution of\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nwith v(τ ) = v1(τ /T )/√T and\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n˙v1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nT A(t)v1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n−\nei π\n2 v1(0) = 0,\nv1(1)\n1 = 0,\ndt\nv1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n1(t) + T AT(t)v∗\n˙v∗\nei π\n2 v∗\nv∗\n1(1)\nv∗\ndt\n1(t), v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\n1 (t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nwhere v∗(τ ) = v∗\n(144) and (143), respectively.\nThe rescaling of (98) gives\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.3**\n1 (τ /T )/√T , respectively. ϕ∗, h11 and α1 are replaced by ϕ∗\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.1**\n1, h11,1 and α1,1, deﬁned by (142),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n˙h20,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nT A(t)h20,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nT B(v1(t), v1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\n−\nh20,1(1) + h20,1(0) = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nwith h20(τ ) = h20,1(τ /T )/T .\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nThe critical coeﬃcients are then given by\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n¯c =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.3**\n¯v∗\n1 (t), C(v1(t), ¯v1(t), ¯v1(t)) + B(v1(t), h02,1(t)) + 2B(¯v1(t), h11,1(t))\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.1**\ndt\n2α11A(t)¯v1(t)\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nd =\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nThe eigenfunction and its adjoint, corresponding to the complex eigenvalue, are discretized by\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\nδ0 −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nT A(t))C×M\ne−i π\nqH\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n2 δ1\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nv1M\na\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\np3\n0 \n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n0C×1\n0n×1\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n−\nδ0 −\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nT A(t))C×M\ne−i π\nqH\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n2 δ1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1 )H\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nW a\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\np3\n0 \n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nrespectively. For the computation of the adjoint function we have applied Proposition C.4 with θ = π\nthen normalize v1M by requiring\ncoeﬃcient. v∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.1**\n2 . We\n= 1, where σj is the Lagrange quadrature\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\nN −1\ni=0\n1W is rescaled so that (v∗\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\n1 )H\nW1 LC×M v1M = 1.\nP\n(146) and the corresponding normalization, (148) and (147) determine ϕ∗\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n(v1M )i,j , (v1M )i,j i\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\n1, h11,1 and α1,1, respectively.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nj=0 σj h\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nP\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nm\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nAn approximation to h20 is obtained by\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nT A(t))C×M\nδ0 + δ1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nh20,1M =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\nT BC×M×M v1M v1M\n0n×1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nWe are now able to compute the two needed normal form coeﬃcients:\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n¯c =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n(v\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n∗\n1 )T\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.2**\nW1 (CC×M×M×M v1M ¯v1M ¯v1M + BC×M×M v1M h02,1M + 2BC×M×M ¯v1M h11,1M −\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n2α1,1AC×M ¯v1M )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\nd =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n6T\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\n1 )H\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.2**\nW1 (CC×M×M×M ¯v1M ¯v1M ¯v1M + 3BC×M×M ¯v1M h02,1M ).\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\n4.9 Fold-Flip bifurcation\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nThe rescaling of the eigenfunctions (103) and (104) gives us\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n(149)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(150)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n˙v1,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nT A(t)v1,1(t)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n−\nv1,1(1)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.3**\nT F (u1(t)) = 0, t\nv1,1(0) = 0,\ndt = 0,\nv1,1(t), F (u1(t))\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n˙v2,1(t)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nT A(t)v2,1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.4**\nv2,1(1) + v2,1(0) = 0,\n1 = 0,\ndt\nv2,1(t), v2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nrespectively, with v1(τ ) = v1,1(τ /T ) and v2(τ ) = v2,1(τ /T )/√T .\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nThe rescaled adjoint eigenfunctions can be obtained by solving\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\n1(t) + T AT(t)ϕ∗\n˙ϕ∗\nϕ∗\nϕ∗\n1(1)\n−\ndt\n1(t), v1,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nϕ∗\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n1(t) = 0, t\n1(0) = 0,\n1 = 0,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nR\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\n\n1,1(t) + T ϕ∗\n1,1(t) + T AT(t)v∗\n˙v∗\nv∗\nv∗\n1,1(0) = 0,\n1,1(1)\n−\nv∗\ndt = 0,\n1,1(t), v1,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\n2,1(t) + T AT(t)v∗\n˙v∗\n2,1(t) = 0, t\n2,1(1) + v∗\nv∗\n2,1(0) = 0,\nv∗\n1 = 0,\ndt\n2,1(t), v2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.5**\n0 h\nR\n1,1(τ /T )/T and v∗\n1(τ ) = v∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nwith ϕ∗(τ ) = ϕ∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.6**\n1(τ /T )/T, v∗\nThe two coeﬃcients in front of the ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n1-terms are given by\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n2(τ ) = v∗\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n2,1(τ /T )/√T .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\na20 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\n∗\n1(t), B(v1,1(t), v1,1(t)) + 2A(t)v1,1(t)\nϕ\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\ndt\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nand α20 = 0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.2**\nThe second order functions of the center manifolfd expansion are deﬁned by the BVPs\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n˙h20,1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nT A(t)h20,1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.7**\nwith h20(τ ) = h20,1(τ /T ),\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\nT B(v1,1(t), v1,1(t)) + 2a20T v1,1(t) + 2α20T F (u1(t))\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n2T A(t)v1,1(t)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\n2T F (u1(t)) = 0, t\nh20,1(0) = 0,\nh20,1(1)\n−\nv∗\ndt = 0,\n1,1(t), h20,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n˙h11,1(t)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nT A(t)h11,1(t)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nwith h11(τ ) = h11,1(τ /T )/√T and\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nT B(v1,1(t), v2,1(t)) + T b11v2,1(t)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nT A(t)v2,1(t) = 0, t\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nh11,1(1) + h11,1(0) = 0,\nv∗\ndt = 0,\n2,1(t), h11,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n˙h02,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nT A(t)h02,1(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nwith h02(τ ) = h02,1(τ /T )/T , where\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.2**\nT B(v2,1(t), v2,1(t)) + 2a02,1T v1,1(t) + 2α02,1T F (u1(t)) = 0, t\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n[0, 1],\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\nh02,1(0) = 0,\nh02,1(1)\n−\nv∗\ndt = 0,\n1,1(t), h02,1(t)\ni\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n0 h\nR\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nb11 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.3**\nv∗\n2,1(t), B(v1,1(t), v2,1(t)) + A(t)v2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\ndt,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\na02,1 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n∗\n1(t), B(v2,1(t), v2,1(t))\nϕ\ni\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\ndt\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nwith a02,1 = T a02 and α02 = 0.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nThe rescaling of the last four normal form coeﬃcients of interest gives\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\na30 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n∗\n1(t)\nϕ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\n, C(v1,1(t), v1,1(t), v1,1(t)) + 3B(h20,1, v1,1(t))\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\n+3(A(t)h20,1(t) + B(v1,1(t), v1,1(t))) + 6(1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n6a20h20,1(t)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n−\nα20)A(t)v1,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\ndt\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\na20,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nb21 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\nv∗\n2,1(t), C(v1,1(t), v1,1(t), v2,1(t)) + B(h20,1(t), v2,1(t)) + 2B(h11,1(t), v1,1(t))\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\n2a20h11,1(t)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\n∗\n1(t), C(v1,1(t), v2,1(t), v2,1(t)) + B(h02,1(t), v1,1(t)) + 2B(h11,1(t), v2,1(t))\nϕ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n2b11h02,1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\n2a02,1h20,1(t) + A(t)h02,1(t) + B(v2,1(t), v2,1(t))\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\n2α02,1A(t)v1,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\na12 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nb03 =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n6T\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nv∗\n2,1(t), C(v2,1(t), v2,1(t), v2,1(t)) + 3B(h02,1(t), v2,1(t))\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n0 h\nZ\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n6a02,1h11,1(t)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n6α02,1A(t)v2,1(t)\ni\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\ndt.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nThe discretization of the functions determined in (149) and (150) is given by\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nT A(t))C×M\n−\nδ0 −\nδ1\ngT\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nv1,1M\na1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nT gC\n0n×1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0 \n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nqT\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nv2,1M\na2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n0C×1\n0n×1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\np1\n0 \n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.2**\n\nwith a1 = a2 = 0. We normalize v21M by requiring\nthe Lagrange quadrature coeﬃcient. The implementation of the adjoint eigenfunctions is done by\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\n(v2,1M )i,j, (v2,1M )i,j i\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\nj=0 σjh\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nN −1\ni=0\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nm\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n= 1, where σj is\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\nP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\nP\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n1)T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nW a1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nT A(t))C×M\nδ1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n1,1)T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\nW a2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.4**\nT A(t))C×M (v1,1)C\n0n×1\nδ0 −\nqT\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\nδ1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\nT (ϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\n1)T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\nW1 LC×M 0\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n2,1)T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nW a3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.6**\n\n1W and v∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n(D\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nT A(t))C×M\nδ0 + δ1\nqT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\np1\n0 \n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n0M×1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.3**\n2,1W are then rescaled to ensure that (ϕ∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n1)T\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\nW1 LC×M v1,1M = 1 and\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nwith a1 = a2 = a3 = 0. ϕ∗\n(v∗\nW1 LC×M v2,1M = 1.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n2,1)T\nThe ﬁrst needed normal form coeﬃcient is given by\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\na20 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n∗\n1)T\n(ϕ\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.3**\nW1 (BC×M×M v1,1M v1,1M + 2AC×M v1,1M ) .\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nThe second order functions can be obtained by solving\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nh20,1M\na\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nrhs\n0n×1\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.5**\nrhs = T BC×M×M v1,1M v1,1M −\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\n2a20T v1,1C + 2T AC×M v1,1M + 2T gC,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nT A(t))C×M\nδ0 + δ1\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nW1 LC×M\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\n−\n(v∗\n2,1)T\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nh11,1M\na\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\np1\n0 \n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\nrhs\n0n×1\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n0 \n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\nrhs = T BC×M×M v1,1M v2,1M −\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nT b11v2,1C + T AC×M v2,1M ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n(D\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.3**r== 0.5**\nT A(t))C×M\n−\nδ1\nδ0 −\n(v∗\n1,1)T\nW1 LC×M\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nh02,1M\na\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\np\n0 \n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\nrhs\n0n×1\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0 \n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nrhs = T BC×M×M v2,1M v2,1M −\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n2a02,1T v1,1C.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nThe in these functions needed coeﬃcients are given by\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nb11 = (v∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n2,1)T\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.3**\nW1 (BC×M×M v1,1M v2,1M + AC×M v2,1M )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nAt last, we obtain\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\na02,1 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n1)T\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nW1 BC×M×M v2,1M v2,1M .\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\na30 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n1)T\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.3**r== 0.3**\nW1 (CC×M×M×M v1,1M v1,1M v1,1M + 3BC×M×M h20,1M v1,1M −\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\n6a20h20,1C\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.2**\n+ 3(AC×M h20,1M + BC×M×M v1,1M v1,1M ) + 6AC×M v1,1M )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\na20,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nb21 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n(v∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\n2,1)T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nW1 (CC×M×M×M v1,1M v1,1M v2,1M + BC×M×M h20,1M v2,1M + 2BC×M×M h11,1M v1,1M −\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.1**\n2b11h11,1C + 2(AC×M h11,1M + BC×M×M v1,1M v2,1M ) + 2AC×M v2,1M )\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\na12 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n2T\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n(ϕ∗\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n1)T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nW1 (CC×M×M×M v1,1M v2,1M v2,1M + BC×M×M h02,1M v1,1M + 2BC×M×M h11,1M v2,1M −\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n2a02,1h20,1C + AC×M h02,1M + BC×M×M v2,1M v2,1M )\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nb03 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n6T\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n(v∗\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n2,1)T\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\nW1 (CC×M×M×M v2,1M v2,1M v2,1M + 3BC×M×M h02,1M v2,1M −\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n6a02,1h11,1C ) .\n**BLOCK**fs== 14.3**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\n5 Examples\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nThe computations in this section are done with matcont [10]. In particular, the bordering methods from\n[15] are used to continue the codim 1 bifurcations of limit cycles in two parameters. The algorithms described\nabove for computing the normal form coeﬃcients are also implemented in the current version of matcont.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nOur ﬁrst model is a periodically forced predator-prey model, studied in [34], and described by the following\ndiﬀerential equations\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n(151)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nx\n˙x = r\nK\n˙y = ep(x, t)y\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n(\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\np(x, t)y,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nx\n−\ndy,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nwhere x and y are the numbers of individuals respectively of prey and predator populations or suitable (but\nequivalent) measures of density or biomass. The parameters present in system (151) are the intrinsic growth\nrate r, the carrying capacity K, the eﬃciency e and the death rate d of the predator. The function p(x, t) is a\nfunctional response, for which the Holling type II is choosen, with constant attack rate a and half saturation\nb(t) that varies periodically with as period one year, i.e.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\np(x, t) =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nax\nb(t) + x\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\nb(t) = b0(1 + ε cos 2πt).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nNotice that this system can be made autonomous by adding the following two diﬀerential equations\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n˙α = µα\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nωβ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n(α2 + β2)α,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n˙β = ωα + µβ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n(α2 + β2)β.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nIn fact, if µ is positive then this system has as asymptotic behavior a stable circular limit cycle of radius µ\nwith angular velocity ω. Therefore, we can set µ = 1 and ω = 2π in order to obtain the forcing function\nsin 2πt as the ﬂow of one of the variables of this subsystem with a particular phase shift that depends on the\ninitial conditions. The system becomes\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\n(152)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nx\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nx\nK\n−\naxy\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n˙x = r\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n˙y = e\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n−\n\nb0(1+ǫα)+x −\n2πβ\n˙α = α\n−\n˙β = 2πα + β\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\naxy\nb0(1+ǫα)+x ,\ndy,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n(α2 + β2)α,\n(α2 + β2)β.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nWe have chosen this system as ﬁrst example since it allows us to check if the computation of the αi\nnormal form coeﬃcients is correct.\nIn fact, in a periodically forced system the time of the normal form\nshould not depend on the coordinate, i.e. dτ /dt = 1, and so all the αi coeﬃcients must vanish. With ﬁxed\nr = 2π, K = e = 1, a = 4π and d = 2π we perform a bifurcation analysis in the remaining parameters\n(ε, b0) obtaining the bifurcation diagram reported in Figure 1. Since the system is periodically forced,\nno equilibria are present. The blue curve, with label LPC2, is a limit point of cycles bifurcation curve of\nthe second iterate, the magenta curves are supercritical Neimark-Sacker bifurcations (of the ﬁrst or of the\nsecond iterate, respectively labeled with NS1 and NS2) while the brown and green curves are period-doubling\nbifurcations, brown when subcritical and green when supercritical (with notation PD1, PD2, PD4 and PD8).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nWe now analyze in detail all the detected codimension two points, reporting the scalar computed coeﬃ-\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\ncients explained in Section 4.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\n5.1.1 The two GPD points\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nIn Figure 1 the LPC2 curve is tangent to the PD1 curve in two diﬀerent GPD points.\nIn the ﬁrst one, in\n(ǫ, b0) = (0.319, 0.412), the limit point of cycles curve is tangent to the subcritical period-doubling curve\n(type presented in Figure 12-(b)), while in the second one, in (ǫ, b0) = (1.09, 0.218), the LPC2 curve is tangent\nto the supercritical part of the PD bifurcation curve (i.e. the type presented in Figure 12-(a)).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\nPerforming the computation of the GPD normal form coeﬃcients at the ﬁrst point we obtain:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.1**\nfor the ﬁrst equation of normal form (6) the two coeﬃcients α1 and α2, up to a scaling term T and T 2\ncomputed through the formula (130) and (131) are zero, up to the accuracy of the computation.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 6.1**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n0.45\n**BLOCK**fs== 6.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n0.4\n**BLOCK**fs== 6.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n0.35\n**BLOCK**fs== 7.6**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nPD1\n**BLOCK**fs== 7.6**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nGPD\n**BLOCK**fs== 7.6**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nPD2\n**BLOCK**fs== 7.6**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nPD8\n**BLOCK**fs== 7.6**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\nNS1\n**BLOCK**fs== 7.6**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nR2\nR2\n**BLOCK**fs== 7.6**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nPD4\n**BLOCK**fs== 3.7**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nb\n**BLOCK**fs== 6.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n0.3\n**BLOCK**fs== 7.6**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nLPC2\n**BLOCK**fs== 7.6**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nLPC2\n**BLOCK**fs== 6.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n0.25\n**BLOCK**fs== 6.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n0.2\n**BLOCK**fs== 7.6**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nNS2\n**BLOCK**fs== 7.6**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\nR1\n**BLOCK**fs== 7.6**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\nR4\n**BLOCK**fs== 7.6**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nR2   R2\n**BLOCK**fs== 7.6**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\nR3\n**BLOCK**fs== 7.6**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\nR2\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n0.2\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n0.4\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n0.6\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nε\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n0.8\n**BLOCK**fs== 6.1**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nFigure 1: Bifurcation diagram of limit cycles in model (152). Blue are limit point of cycles bifurcations, green\nperiod doubling bifurcations and magenta Neimark-Sacker birucations. Continue/dotted curves correspond\nto supercritical/subcritical bifurcations.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nthe normal form coeﬃcient of the second equation, computed through formula (132) equals e =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n58.2867.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nNotice that these results are in agreement with what we expected, i.e. that since the system is periodically\nforced the time doesn’t depend on the distance from the critical limit cycle, and since we are in the case\npresented in Figure 12-(b) the normal form coeﬃcient e is negative.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nFrom the computation of the GPD normal form coeﬃcients at the second critical point we obtain:\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nfor the ﬁrst equation of normal form (6) the two coeﬃcients equal zero.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nthe normal form coeﬃcient of the second equation has value e = 41.5442.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nAlso in this case the obtained results are in agreement with the theory.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n5.1.2 The 1:1 and 1:2 resonance points\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nWe divide the 1:1 and 1:2 resonance points present in this model into two groups, namely the R2 point at\n(ǫ, b0) = (0.337, 0.34) and the cascade of resonance points in the lower part of the graph.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nThe isolated R2 point forms the intersection of the NS1 curve, the supercritical Neimark-Sacker curve of a\nlimit cycle with period approximately equal to 1, and PD1. The situation is thus the one depicted in Figure\n15-(a). Performing the normal form coeﬃcient computation we obtain:\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.4**\nfor the ﬁrst equation of normal form (9) holds that α = 0.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\nfor the last equation of the normal form (9) we have (a, b) = (3.401426,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n12.90745).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nNotice that the obtained results are in accordance with the theory (no secondary Neimark-Sacker curve\nimplies that a > 0 and supercritical Neimark-Sacker curve implies that b < 0).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nIn the lower part of the bifurcation diagram a resonance cascade is present, which accumulates on the\nsequence of period-doubling curves. A zoom of this part is shown in Figure 2. Each resonance point of this\n**BLOCK**fs== 8.1**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n0.19\n**BLOCK**fs== 8.1**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n0.185\n**BLOCK**fs== 3.7**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nb\n**BLOCK**fs== 8.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n0.18\n**BLOCK**fs== 8.1**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n0.175\n**BLOCK**fs== 10.2**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nR1\n**BLOCK**fs== 10.2**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nNS2’\n**BLOCK**fs== 10.2**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nR4\n**BLOCK**fs== 10.2**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nNS8\nR2\n**BLOCK**fs== 10.2**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\nNS4\n**BLOCK**fs== 10.2**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nR2\n**BLOCK**fs== 10.2**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nR3\nR4\n**BLOCK**fs== 10.2**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nR2\n**BLOCK**fs== 10.2**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nR3\n**BLOCK**fs== 10.2**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\nPD8’\n**BLOCK**fs== 10.2**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nPD4’\n**BLOCK**fs== 10.2**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\nPD2’\n**BLOCK**fs== 10.2**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\nLPC2\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n0.17\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n0.6\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n0.65\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n0.7\nε\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n0.75\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n0.8\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nIn blue are limit point of cycles bifurcations, green\nFigure 2: The resonance cascade in model (152).\nperiod doubling bifurcations and magenta Neimark-Sacker birucations. Continue/dotted curves correspond\nto supercritical/subcritical bifurcations.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\ncascade (except for the ﬁrst R1 point on the LPC2 curve) can be seen in two ways: the ﬁrst (and more natural\nway) is to see them as R2 points of the type represented in Figure 15-(b) (so with a < 0 and the sign of\nb dependent on the criticality of the incoming Neimark-Sacker curve), the second way is to see them as R1\npoints represented in Figure 14-(b). Notice that the criticality of each NS curve of the cascade changes at\nthe R2 point (as depicted in Figure 15-(b)).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nAs ﬁrst general result we see that for the ﬁrst equation of normal form (9) there holds that α = 0 for\nall points (as expected since the system is periodically forced). We remark that for the computation of the\nnormal form coeﬃcients of the second equation, the tolerances have to be strong enough. The results are\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n(on LPC2) The R1 point is in (ǫ, b0) = (0.6044021, 0.1769608). The period 2 limit cycle Neimark-Sacker curve\nNS2 starts tangentially to the LPC2 curve and is subcritical. In this situation we therefore expect the\nproduct of the two normal form coeﬃcients of the last equation of (8) to be positive; the computed\ncoeﬃcients are (a, b) = (2.005489e\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.2**\n6, 6.4806e + 9), such that ab = 1.299679e + 4.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.1**r== 0.1**\n(on PD2) The R2 point is in (ǫ, b0) = (0.7437713, 0.1835935). On the left side of the R2 point the PD2 curve is\nsupercritical, on the right side it is subcritical. The NS2 curve incoming in the R2 point is subcritical,\nwhile the NS4 curve outgoing at the R2 point is supercritical. We are thus in the case depicted in\nFigure 15-(b) with time reversed. So we expect b > 0 (subcritical incoming Neimarck-Sacker curve)\nand a < 0 (there is an outgoing secondary Neimarck-Sacker curve). The computed coeﬃcients at the\nR2 point are (a, b) = (\n65.76676, 16.26708). Notice that this point is also a degenerate R1 point for\nthe NS4 curve. In fact, when we compute the normal form coeﬃcient at this 1:1 resonance bifurcation\npoint, we obtain (a, b) = (\n4, 6.116846870167980e+12). In this case the product\n6.812790e + 8 is negative, in accordance with the fact that the NS4 curve (for which we have an\nab =\nR1 bifurcation) is supercritical.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\n1.113774823354237e\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\n(on PD4) The R2 point is in (ǫ, b0) = (0.7427991, 0.1861098). On the left side of the R2 point the PD4 curve is\nsupercritical, on the right side it is subcritical. The NS4 curve incoming in the R2 point is supercritical,\nwhile the NS8 curve outgoing at the R2 point is subcritical. We are therefore in the case depicted\nin Figure 15-(b). We expect b < 0 (supercritical incoming Neimarck-Sacker curve) and a < 0 (there\nis an outgoing secondary Neimarck-Sacker curve). The computed coeﬃcients at the R2 point are\n(a, b) = (\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n18.15061).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n269.3681,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\n(on PD8) The R2 point is in (ǫ, b0) = (0.7439079, 0.1864190). On the left side of the R2 point the PD8 curve is su-\npercritical, on the right side it is subcritical. The NS8 curve incoming in the R2 point is subcritical: we\nare thus in the case depicted in Figure 15-(b) with time reversed. Thus, we expect b > 0 (subcritical in-\ncoming Neimarck-Sacker curve) and a < 0 (there is an outgoing secondary Neimarck-Sacker curve, since\nthe cascade continues). The computed coeﬃcients of the R2 point are (a, b) = (\n921.7011, 16.58059).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nAll the obtained results are in agreement with the theory.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\n5.1.3 The 1:3 resonance points\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nThere are two 1:3 resonance points, one on NS2, the other one on NS4, as can be seen in Figure 2. These\ntwo points behave in a diﬀerent way. The Neimark-Sacker curve corresponding with the ﬁrst point at\n(c) to be positive. The Neimark-Sacker curve of the\n(ǫ, b0) = (0.709, 0.179) is subcritical, so we expect\nsecond point at (ǫ, b0) = (0.743, 0.185) is supercritical, so\n(c) should be negative. To check whether we\nare in a non degenerate case, we also have to look at b, however, as mentioned before, the sign of b is not\nrelevant. We obtain\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nℜ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nℜ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nfor the ﬁrst R3 point we have that (b,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.5**\nℜ\nfor the second R3 point we have that (b,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nℜ\nThese results are in accordance with the theory.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n(c)) = (4.5567\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n4.4567i, 9.155003).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.3**\n−\n(c)) = (0.4049 + 12.1425i,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\n8.819864).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\n5.1.4 The 1:4 resonance points\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nThere are two 1:4 resonance points, one on NS2, the other one on NS4, as can be seen in Figure 2. Also\nthese two points behave in the same way as the 1:3 resonance bifurcation points. The Neimark-Sacker curve\n(A) to be\ncorresponding with the ﬁrst point at (ǫ, b0) = (0.675, 0.177) is subcritical, so here we expect\n(A)\npositive. The Neimark-Sacker curve of the second point at (ǫ, b0) = (0.743, 0.185) is supercritical, so\nshould be negative. Moreover, since those points are part of a resonance cascade, we should not have limit\npoint bifurcations of non trivial equilibria, so we are in region I of Figure 17. In order to assure that we are\nnot in a degenerate case, we also need to check that d\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n= 0. We obtain\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\nℜ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\nℜ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nfor the ﬁrst R4 point we have that (c, d) = (11.624\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\n84.897i, 65 + 92.254i), so A = 0.102999\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nfor the second R4 point we have that (c, d) = (\n0.645406i.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n414.71i,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n416.64\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n489.17i), so A =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\n0.013352\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n−\n8.5796\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nSo for both bifurcation points the value of A belongs to region I, and thus the results are in accordance with\nthe theory.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n5.2 The Steinmetz-Larter model\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\nThe following model of the peroxidase-oxidase reaction was studied by Steinmetz and Larter [41]:\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(153)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\nk1ABX\nk1ABX\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.5**\n˙A =\n−\n˙B =\n−\n˙X = k1ABX\n˙Y =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nk3ABY + k7 −\n−\nk3ABY + k8,\n−\n2k2X 2 + 2k3ABY\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\nk−7A,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nk4X + k6,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n−\nk3ABY + 2k2X 2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nk5Y,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nwhere A, B, X, Y are state variables and k1, k2, k3, k4, k5, k6, k7, k8, and k−7 are parameters. We ﬁx all\nparameters as reported in the following table\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nPar. Value\nk1\nk5\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n0.1631021\n1.104\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\nPar. Value Par. Value\nk2\nk6\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n0.001\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n0.046875\n0.1175\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nk3\nk−7\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.2**\nPar. Value\nk4\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nand we perform a bifurcation analysis in the parameter space (k7, k8). A few curves are reported in Figure\n3.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.6**\n5.2.1 The 1:1 resonance points\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe two 1:1 resonance points have diﬀerent nature, since in one the Neimark-Sacker curve rooted at the\nbifurcation point is supercritical, while in the other one it is subcritical.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nWe obtain:\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.1**\nfor the R1 point in (k7, k8) = (1.179554, 0.7239571), the two coeﬃcients of the last equation of (8) are\nequal to (a, b) = (\n3 is\n0.003654362200739, 0.735048055230916). Their product ab =\nnegative, which corresponds with the fact that the NS curve rooted at the R1 point is supercritical.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n2.686132e\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.9**r== 0.1**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.1**\nfor the R1 point in (k7, k8) = (1.857676, 0.9304220), the two coeﬃcients of the last equation of (8)\n2.156596806473489). Their product ab = 0.1432622 is\n0.066429738171756,\nare equal to (a, b) = (\npositive, and indeed the NS curve rooted at the R1 point is subcritical.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nSo we can conclude that the results are in accordance with the theory.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\n5.2.2 The Chenciner points\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nAs can be seen in Figure 3 we have detected a CH point at (k7, k8) = (1.757356, 0.9125773). The normal\nform coeﬃcient at that bifurcation point equals\n(e) = 1.391931, hence positive so the unfolding is the one\ndepicted in Figure 13-(b). In order to verify if the normal form computation is correct, one should use tori\ncontinuation techniques [37], either recurring to Poincar´e maps [26, 8] or to the so-called invariance equation\n[12, 39, 35, 38]. However, these techniques are not stable, especially in critical cases like the one we have.\nIn order to validate our result we thus have to do simulations.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nℜ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nThe obtained result is shown in Figure 4. The indicated regions correspond with the regions from Figure\n13. The green curve between regions 2 and 3 is the supercritical Neimark-Sacker curve, the red one between\nregion 1 and 2 is the subcritical Neimark-Sacker curve. For each point of the grid, we have started time\nintegration from a point close to the orginal limit cycle (a 1 % perturbation) until an attractor was found.\n**BLOCK**fs== 11.1**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nR1\n**BLOCK**fs== 11.1**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nLPC\n**BLOCK**fs== 11.1**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nCH\n**BLOCK**fs== 3.7**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nk\n**BLOCK**fs== 8.9**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n0.9\n**BLOCK**fs== 8.9**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n0.85\n**BLOCK**fs== 8.9**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n0.8\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n0.75\n**BLOCK**fs== 11.1**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\nNS1\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n0.7\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n1.1\n**BLOCK**fs== 11.1**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nR1\n**BLOCK**fs== 11.1**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nNS2\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n1.2\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n1.3\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n1.4\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n1.5\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n1.6\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n1.7\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n1.8\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n1.9\n**BLOCK**fs== 8.9**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\nk7\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFigure 3: Bifurcation diagram of a limit cycle in model (153).\nIn blue are limit point of cycles bifurca-\ntions, green period doubling bifurcations and magenta Neimark-Sacker birucations. Continue/dotted curves\ncorrespond to supercritical/subcritical bifurcations.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.6**l== 0.1**r== 0.1**\nThe 1-norm of the X-coordinate of an orbit with time length 1000 along the attractor is shown in the\ncolormap. In region 2 this attractor is the original limit cycle, in region 3 it is the inner torus arisen through\nthe supercritical Neimark-Sacker curve. In region 1 the original limit cycle is unstable, and so the trajectory\nwhich starts nearby converges to another attractor. Between region 1 and 3 and region 1 and 2 happens\na catastrophic bifurcation, i.e. a drastic change of the attractor, identiﬁed from the change of color which\nvaries from blue to red. Right above the Chenciner point, the catastrophic bifurcation is the subcritical NS\ncurve, while left below it is the limit point of tori (Tc) curve. Figure 4 shows that we obtain the scenario\nwhich corresponds with a positive second Lyapunov coeﬃcient.\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\n5.3 The Lorenz1984 system\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nThis model, taken from [36], is a meteorological model written by Lorenz in 1984 in order to describe the\natmosphere. The equations of the model are\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\n(154)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ny2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.5**\nz2\n˙x =\n−\n−\n˙y = xy\nbxz\n˙z = bxy + xz\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nax + aF,\ny + G,\nz,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n−\n−\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nwhere (a, b, F, G) are parameters, with a = 0.25, b = 4. This model, as depicted in [40], has most of the\nanalyzed codimension two bifurcations of limit cycles. We report in Figure 5 a bifurcation diagram obtained\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nFigure 4: Simulations on a parameter grid (black points) of system (153). The magenta continue/dotted\nline is the supercritical/subbritical Neimark-Sacker curve. The color represents the value of the maximum\nof the ﬁrst coordinate of the attractor reached through simulation from a point close to the limit cycle. A\nsketch of the state portrait is reported in Figure 13-(b).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nwith matcont in which bifurcations of equilibria (LP stands for limit point, H for Andronov-Hopf) are thicker\nand limit cycle bifurcations are thin. In particular, the blue curve is a limit point of cycles (LPC) bifurcation\ncurve, the green ones are period-doubling (PD) bifurcations curves and the magenta ones are Neimark-Sacker\n(NS) curves. The codimension two points are marked with a red dot, and, as can be seen in the ﬁgure, almost\nall cases (except the Chenciner bifurcation and the fold-ﬂip bifurcation) discussed in Section 3 are present in\nthis model. In the sequel of this section we will investigate the normal form coeﬃcients of each bifurcation.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n5.3.1 The Swallowtail bifurcation\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nThe ﬁrst degeneracy we want to analyze is the vanishing of the coeﬃcient c in the cusp of cycles normal\nform (5). This bifurcation, named Swallowtail bifurcation, is characterized, in our case, by the collision and\ndisappearance of two cusp points of limit cycle. In order to get this codimension three bifurcation we analyze\npart of the blue dashed curve in Figure 5 for diﬀerent parameter values of b. The result is shown in Figure\n6: in the graph part of the limit point of cycles manifold is plotted in the (F, G)-plane for diﬀerent values of\n[2.91, 2.95] (from blue to red). In the table we can see the behavior of the critical normal form\nparameter b\ncoeﬃcient c, where it exists (the colors of the lines correspond with the bifurcation diagram). Notice how\nthe behavior of this codim 3 bifurcation is captured by a smooth vanishing of the normal form coeﬃcient.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.4**\n5.3.2 The degenerate generalized period doubling bifurcation\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nOn the green curve PD4 of Figure 5 there are two generalized period-doubling (GPD) points. In the ﬁrst one\nthe ﬂip bifurcation curve passes from subcritical to supercritical, in the second one the opposite happens.\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.4**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nF\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\nPD2f’\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nCPC\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nCPC\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nGPD\n**BLOCK**fs== 8.3**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\nCPC\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nGPD\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nR1\nR1\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nR4\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nCPC\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nR3\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nNS2\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nR4\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nR3\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nR2\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nR3\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nR4\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nNS\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nZH\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nLPC\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nCPC\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nR1\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nPD4\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nPD2\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nPD1\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nH’\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nH\n**BLOCK**fs== 8.3**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nLP\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nCP\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nG\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.3**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nFigure 5: Bifurcation diagram of model (154). The thicker curves are bifurcation curves of equilibria, the\nthin curves are bifurcation curves of limit cycles and invariant tori (blue, limit point of cycles, green period\ndoubling and magenta Neiamrk-Sacker). Continue/dotted curves correspond to supercritical/subcritical\nbifurcations.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\n3 < 0, therefore there is a\nComputing the normal form coeﬃcient in the ﬁrst case gives e =\nlimit point of cycles bifurcation curve that starts rightward tangent to the supercritical part of the period-\ndoubling manifold, while in the second case e = 2.895460e\n3 > 0, and so the limit point of cycles bifurcation\nstarts leftward tangent to the subcritical part of the PD curve. These conclusions can be seen in Figure 7,\nwhere the period-doubling curve is black, dotted when supercritical; in the upper panels are sketched the\nPoincar´e map of the limit cycle involved in the bifurcation. On the yellow curve the limit cycles sketched in\ngreen and red collide and disappear, while on the violet curve the two involved limit cycles are sketched in\nred and blue.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n1.317656e\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\n5.3.3 The 1:1 resonance points\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nTwo R1 points are located on the LPC curve. Those two points should have diﬀerent product of the normal\nform coeﬃcients. In fact, in the ﬁrst one (in (F, G) = (10.72, 0.522)) the Neimark-Sacker curve rooted at the\nbifurcation point is supercritical (i.e. the system has the behavior depicted in 14-(a)), while in the second\none (in (F, G) = (9.81, 2.22)) the NS curve is subcritical (behavior similar to Figure 14-(b)). If we apply our\nanalysis we obtain:\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nfor the ﬁrst R1 point (a, b) = (2.577,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nfor the second R1 point (a, b) = (\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.3**\n1.2659), so the product ab =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n3.26237 is negative.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\n−\n2.005), so the product ab = 19.81858e is positive.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n−\n9.887,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nThese results are in accordance with the theory. The blue curve on Figure 8 is the limit point of cycles curve.\nThe violet curves are the Neimark-Sacker curves of ﬁrst iterate, the green curve is the Neimark-Sacker curve\nof second iterate.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\n5.3.4 The 1:2 resonance points\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nAt the unique R2 point at (F, G) = (10.72, 0.522) shown in Figure 5, the incoming Neimark-Sacker curve,\nnamely NS, is subcritical (and so b > 0), while the outgoing curve (which exists and so a < 0), namely NS2, is\n**BLOCK**fs== 8.1**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n12.82\n**BLOCK**fs== 8.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n12.8\n**BLOCK**fs== 8.1**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n12.78\n**BLOCK**fs== 8.1**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n12.76\n**BLOCK**fs== 8.1**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n12.74\n**BLOCK**fs== 4.9**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nF\n**BLOCK**fs== 8.1**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n12.72\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n12.7\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n12.68\n**BLOCK**fs== 8.1**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n12.66\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n12.64\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n0.394\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n0.396\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n0.398\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n0.4\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n0.402\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n0.404\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n0.406\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\n0.408\n**BLOCK**fs== 8.1**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nG\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.3**r== 0.5**\n(F, G)1\n(12.76893,0.406176)\n(12.74929,0.402934)\n(12.72891,0.399746)\n(12.70759,0.396624)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\nc1\n16.4570\n13.3315\n10.0534\n6.30503\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.3**\n(F, G)2\n(12.79664,0.40445)\n(12.76712,0.40183)\n(12.73840,0.39917)\n(12.71071,0.39643)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\nc2\n-8.83567\n-7.77721\n-6.47271\n-4.67510\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\nb\n2.95\n2.94\n2.93\n2.92\n2.91\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nFigure 6: Diﬀerent limit point of cycles bifurcation curves in the (F, G)-plane for diﬀerent values of the third\nparameter b. The parameter values are reported in the table.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.4**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nF\n**BLOCK**fs== 8.2**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.2**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.2**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.2**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.2**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 8.2**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nPD4\n**BLOCK**fs== 8.2**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nGPD\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nGPD\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n0.8\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n0.9\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n1.1\nG\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n1.2\n**BLOCK**fs== 8.2**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n1.3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nFigure 7: Two generalized period-doubling points with diﬀerent normal form coeﬃcients on the period-\ndoubling bifurcation curve PD4 of Figure 5.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nsupercritical, i.e. we are in the case reported in Figure 15-(b) with time reversed. The coeﬃcients computed\nat the 1:2 resonance point are (a, b) = (\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.2**\n0.6329965, 0.1785283), in accordance with the theory.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nAt the R1 point located at (F, G) = (9.81, 2.22) starts a resonance cascade, as shown in Figure 8-(b). On\nthat cascade we can ﬁnd many resonance points which we will analyze in what follows. In particular, since\nthe R2 points belong to a cascade they are of the same type as presented in Figure 15-(b) (so a < 0), with\nat each step a change of criticality of the incoming NS curve. As mentioned before, the ﬁrst NS curve, born\nat the R1 point, is subcritical, so for the ﬁrst R2 point we expect that b > 0, while for the second one b < 0.\nThe obtained numerical results are\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.3**\nfor the ﬁrst R2 point at (F, G) = (9.9158, 2.2978) we have that (a, b) = (\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\n1.3157, 0.11076).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.3**\nfor the second R2 point at (F, G) = (9.9197, 2.2978) we have that (a, b) = (\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n2.6228,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n0.0564).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nResults are in accordance with the theory.\n**BLOCK**fs== 5.8**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n10.5\n**BLOCK**fs== 5.8**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.4**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\nF\n**BLOCK**fs== 7.2**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nCPC\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n9.5\n**BLOCK**fs== 7.2**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nR1\nR1\n**BLOCK**fs== 7.2**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nNS2\n**BLOCK**fs== 7.2**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nR4\n**BLOCK**fs== 7.2**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nPD4’\n**BLOCK**fs== 7.2**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nR2\n**BLOCK**fs== 7.2**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nR1\n**BLOCK**fs== 7.2**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nLPC\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.2**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nR3\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n0.4\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n0.6\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n0.8\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n1.2\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n1.4\nG\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n1.6\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n1.8\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n2.2\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n2.4\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nR2\n**BLOCK**fs== 9.6**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nR2\n**BLOCK**fs== 9.6**b== 0.3**t== 0.6**l== 0.8**r== 0.2**\nLPC\n**BLOCK**fs== 9.6**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nR3\n**BLOCK**fs== 9.6**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nR4\n**BLOCK**fs== 7.7**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n9.92\n**BLOCK**fs== 7.7**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n9.9\n**BLOCK**fs== 7.7**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n9.88\n**BLOCK**fs== 5.4**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nF\n**BLOCK**fs== 7.7**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n9.86\n**BLOCK**fs== 7.7**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n9.84\n**BLOCK**fs== 9.6**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nPD7\n**BLOCK**fs== 9.6**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nNS3\n**BLOCK**fs== 9.6**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nPD6\n**BLOCK**fs== 7.7**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n9.82\n**BLOCK**fs== 10.0**b==-0.0**t== 1.0**l== 0.1**r== 0.1**\nFigure 8: (a) Two 1:1 resonance points with diﬀerent behavior on the LPC curve of Figure 5. (b) Zoom\n**BLOCK**fs== 10.0**b==-0.0**t== 1.0**l== 0.1**r== 0.1**\non the Lorenz cascade that starts at the right R1 point. In blue are the limit point of cycles bifurcation\n**BLOCK**fs== 10.0**b==-0.0**t== 1.0**l== 0.1**r== 0.1**\ncurves, in green the period-doubling curves, in violet the Neimark-Sacker curves. Continue/dotted curves\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\n5.3.5 The 1:3 resonance points\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nWe have several 1:3 resonance points at which we can have a closer look. There is one R3 point located\non the NS curve and two R3 points on the NS2 curve. The R3 point corresponding to the ﬁrst iterate\nis at (F, G) = (4.628, 1.624), with a positive normal form coeﬃcient of the Neimark-Sacker curve such\nthat we are in the case represented in Figure 16-(b). The R3 points corresponding to the second iterate\nare at (F, G) = (7.072, 1.235) and (F, G) = (8.989, 0.7394), where the Neimark-Sacker curve is both times\nsupercritical, so we are in the situation depicted in Figure 16-(a).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nfor the R3 point at (F, G) = (4.628, 1.624) we have that (b,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n(c)) = (0.1913\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nfor the R3 point at (F, G) = (7.072, 1.235) we have that (b,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n(c)) = (\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n0.4461\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\n0.5464i, 6.185900e\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n2)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n0.1901i,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\n3.612302e\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nfor the R3 point at (F, G) = (8.989, 0.7394) we have that (b,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n(c)) = (\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nℜ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n0.1285+0.0168i,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n1.950822e\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nThese results are in accordance with the theory.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nThere are also R3 points on the cascade (see Figure 8-(b)). The ﬁrst one corresponds with a subcritical\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nNS curve, while the second one corresponds with a supercritical NS curve.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nfor the ﬁrst R3 point, in (F, G) = (9.8888, 2.2798) we have that (b,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n(c)) = (\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n2.9582\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\nℜ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\nfor the second R3 point, in (F, G) = (9.9187, 2.2978) we have that (b,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\n(c)) = (2.7447+3.5391i,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nℜ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nAlso in this case all results are in accordance with the theory.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nThere are 5 1:4 resonance points at which we will we have a look. One is located on the NS curve, two others\non the NS2 curve and the last two lie on the resonance cascade (see Figure 8-(b)). We obtain\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.3**\nfor the R4 point at (F, G) = (3.376, 1.647) we have that (c, d) = (0.0501\nand so A = 0.0918\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.4**\n0.1368i (subcritical NS curve, case I).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n0.0746i,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n0.0577\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.4**\nfor the R4 point at (F, G) = (9.777, 0.595) we have that (c, d) = (\nand so A =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.3**\n2.752998i (supercritical NS curve, case VIII).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n0.307829\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n0.0151\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nfor the R4 point at (F, G) = (6.620, 1.390) we have that (c, d) = (\nand so A =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.3**\n−\n0.851653i (supercritical NS curve, case I).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n0.035841\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n0.0417\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n0.1348i,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n0.9915i,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n0.0266\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n0.4283\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nFor the ﬁrst and the last point no further bifurcation analysis is possible to conﬁrm the correctness of the\nresults (present curves rooted at the point are global bifurcations of limit cycles).\nInstead it is possible\nto continue all local bifurcations of limit cycles rooted at the second R4 point, obtaining the result shown\nin Figure 9. Note that we haven’t made the distinction between region VII and VIII since we have not\ncomputed the fold of torus curve typical for region VIII.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nFor the cascade (see Figure 8-(b)) we have that the ﬁrst point is on a subcritical NS curve, while the\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nsecond one is on a supercritical NS curve. Moreover, since they lie on a cascade, they should be of type I.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nfor the ﬁrst R4 point at (F, G) = (9.9159, 2.2978) we have that (c, d) = (0.0518\n0.4546i) and so A = 0.025102\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.3**\n0.853919i (subcritical NS curve, case I).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n1.7633i,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\n2.0143 +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.3**\nfor the ﬁrst R4 point at (F, G) = (9.9197, 2.2978) we have that (c, d) = (\n2.1458i) and so A =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.3**\n0.616491i (supercritical NS curve, case I).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n0.002550\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n0.0282\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n6.8152i,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\n10.8446+\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.4**\nAlso in this case the results are in accordance with the theory.\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n9.81\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n9.8\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n9.79\n**BLOCK**fs== 8.3**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n9.78\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n9.77\n**BLOCK**fs== 5.4**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nF\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n9.76\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n9.75\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n9.74\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n9.73\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n9.72\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n9.71\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nR4\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nTin\n**BLOCK**fs== 8.3**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nNS4\n**BLOCK**fs== 8.3**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nNS\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n0.594\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n0.595\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n0.596\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n0.597\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n0.598\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0.599\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n0.6\n**BLOCK**fs== 8.3**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n0.601\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nG\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nFigure 9: Bifurcation diagram at the R4 point at (F, G) = (9.777, 0.595).\nIn blue are the limit point\nof cycles bifurcation curves, in violet the Neimark-Sacker curves. Continue/dotted curves correspond to\nsupercritical/subcritical curves.\n**BLOCK**fs== 12.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n5.4 The Extended Lorenz1984 system\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nAs done in [33], it is possible to extend the Lorenz1984 system (154) by adding a fourth variable which takes\nthe inﬂuence on the jet stream and the baroclinic waves of external parameters like the temperature of the\nsea surface into account. The obtained system is\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n(155)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nγu2,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\ny2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\nz2\n˙x =\n−\n−\n˙y = xy\nbxz\n˙z = bxy + xz\n˙u =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n−\n−\n−\nδu + γux + K.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nax + aF\ny + G,\nz,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nWe use the parametervalues mentioned in [33], i.e.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.2**\na = 0.25, b = 1, G = 0.2, δ = 1.04, γ = 0.987, F = 1.75, K = 0.0003.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nSimulating this system from the trivial initial condition leads to a limit cycle. In a continuation in K that\nlimit cycle undergoes a subcritical period-doubling bifurcation. Now, we can do a two parameter continuation\nin (F, K) and draw the bifurcation diagram reported in Figure 10.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\n5.4.1 The fold-ﬂip point\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\n10−3). Since there is\nAs can be seen in Figure 10, a fold-ﬂip point is detected for (F, K) = (1.7620, 0.2806\na Neimark-Sacker curve of the period doubled limit cycle rooted at the bifurcation point and the NS curve\nand the LPC curve lie on diﬀerent sides of the PD curve, we are in the case represented in Figure 19-(a), i.e.\nwe have a20b11 < 0 and a02b11 < 0. Moreover, since the NS curve is supercritical, LN S should be negative.\nNumerically, we obtain that b11 = 562.2215, a20 =\n178.8596. Hence,\nthese results are in accordance with the theory.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n0.5756, a02 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n3, LN S =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n0.1036e\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\n×\n**BLOCK**fs== 10.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n4 x 10\n**BLOCK**fs== 10.5**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nLPC\n**BLOCK**fs== 10.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nPD\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nT\nT\n**BLOCK**fs== 10.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nLPPD\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nPSfrag replacements\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n1.75\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nNS2\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n1.76\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nF\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n1.77\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nFigure 10: Bifurcation diagram of a limit cycle in model (155). The blue curve is a limit point of cycles\nbifurcation, the green curve is a period-doubling curve (continue/dotted curve correspond to supercriti-\ncal/subcritical curves) and the magenta curve is a supercritical Neimark-Sacker bifurcation curve of the\nperiod doubled limit cycle.\n**BLOCK**fs== 14.3**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nA Derivation of the normal forms\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nA.1 Notation\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nRn×n be the monodromy matrix. In all codimension 2 cases all critical multipliers, i.e. all multipliers\nLet M\nwith modulus 1, have non-degenerate Jordan blocks. Let M0 be the critical Jordan structure, i.e. the block\ndiagonal matrix consisting of the critical Jordan blocks, starting with the block of the trivial multiplier 1.\nRmk×mk is\nLet µk = eiθk (0\ndeﬁned as\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.2**r== 0.7**\n≤\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.2**\nθk < 2π) be a critical multiplier with multiplicity mk. The matrix Lk ∈\n. . .\n. . .\n. . .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nσk\n. . .\n. . .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nσk\n\n...\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nσk\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nLk = \n\n\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nwhere σk is the Floquet exponent of the multiplier µk, with σk = iθk/T in the case of a positive real multiplier\nor a complex multiplier µk and σk = 0 for µk =\n1. The matrix L0 is the block diagonal matrix formed from\n= 1, starting with the block that corresponds with multiplier 1. The matrix ˜L0\nthe blocks Lk for which\nis the matrix L0 without the ﬁrst row and the ﬁrst column.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nµk|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n|\n**BLOCK**fs== 12.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nA.2 Bifurcations with 2 critical eigenvalues\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nA.2.1 CPC\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nAt the CPC bifurcation the monodromy matrix has the critical Jordan structure\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nM0 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\ni.e. the multiplier µ = 1 is double non semi-simple. According to the proposed notation σ = 0 and thus\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nL0 =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n0 1\n0 0\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n˜L0 = 0.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\nWe are in a situation where we can apply Theorem 2 from [23]. In particular, we have that\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n= 1 + ξ + p(τ, ξ),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n= ˜L0ξ + P (τ, ξ),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nwhere τ plays the role of phase coordinate along the orbit. The polynomials p and P are T -periodic in τ\nand at least quadratic in ξ such that\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\np(τ, ξ)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\np(τ, ξ) ˜L\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n∗\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nP (τ, ξ) + ˜L\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n∗\n0P (τ, ξ)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nP (τ, ξ) ˜L\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n∗\n0ξ = 0\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nfor all τ and ξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nR. Putting ˜L0 = 0 we obtain\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\np(τ, ξ) = 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nP (τ, ξ) = 0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\ni.e. the two polynomials are independent from τ . So, by a Taylor expansion of the two polynomials the\nnormal form becomes\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.7**\n\nApplying the chain rule gives\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\ndτ\ndt\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n= 1 + ξ + p(ξ) = 1 + ξ + α1ξ2 + α′\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n2ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n= P (ξ) = bξ2 + cξ3 + . . . .\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n= (bξ2 + cξ3 + . . . )(1 + ξ + α1ξ2 + α′\n= bξ2 + bξ3 + cξ3 + . . .\n= cξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n2ξ3 + . . . )\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nbecause of the cusp degeneracy condition.\nsubstitution ξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\nξ and ﬁnd\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n7→ −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nIn order to obtain the proposed normal form (5), we do the\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n= 1\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nξ + α1ξ2 + α2ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n−\n= cξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nwith α2 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nα′\n2.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nA.2.2 GPD\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nAt the GPD bifurcation the matrices described in Section A.1 are\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nM0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n, L0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n˜L0 = 0.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nWe are in a case in which we can apply Theorem 3 from [23]. So we have the following 2T -periodic normal\nform (using the formula of Theorem 1 from [23])\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n= 1 + p(τ, ξ),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\n= ˜L0ξ + P (τ, ξ),\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.3**\nwith polynomials p and P 2T -periodic in τ and at least quadratic in ξ such that\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\np(τ, ξ)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\np(τ, ξ) ˜L\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n∗\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\np(τ + T, ξ) = p(τ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nξ),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nP (τ, ξ) + ˜L\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n∗\n0P (τ, ξ)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nP (τ, ξ) ˜L\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n∗\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nd\ndτ\nP (τ + T,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n−\nP (τ, ξ).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nξ) =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\nPutting ˜L0 = 0 in the ﬁrst two formulas brings us in the same situation as of the previous case\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\np(τ, ξ) = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nP (τ, ξ) = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ni.e. the two polynomials are independent of τ . This makes it possible to rewrite the last two formulas as\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nso polynomial p is even (p = φ(ξ2)) and polynomial P is odd (P = ξψ(ξ2)). Therefore, taking the GPD\ndegenerate condition into account, we can write down the ﬁrst approximation of our normal form\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\np(ξ) = p(\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nξ), P (\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nξ) =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nP (ξ),\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.3**\n= 1 + φ(ξ2) = 1 + α1ξ2 + α2ξ4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n= ξψ(ξ2) = cξ3 + eξ5 + . . . .\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nApplying the chain rules gives\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.3**\n= (cξ3 + eξ5 + . . . )(1 + α1ξ2 + α2ξ4 + . . . )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\n= cξ3 + α1cξ5 + eξ5 + . . .\n= eξ5 + . . . ,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.2**\nbecause of the GPD degeneracy condition. So we obtain the normal form presented in (6), namely\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n= 1 + α1ξ2 + α2ξ4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n= eξ5 + . . . .\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nA.3 Bifurcations with 3 critical eigenvalues\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nA.3.1 CH\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.2**\nIn the CH case the Jordan block associated to the trivial multiplier is one-dimensional. We have\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nM0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n0 eiωT\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\ne−iωT \n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n, L0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\niω\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\niω\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n˜L0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\niω\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\niω\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.2**\nThis puts us in a situation in which we can apply Theorem 1 from [23]. If we assume that ωT\n2π 6∈\nfollows immediately from the results of Example III.9 from [24] that the normal form is given by\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nQ, then it\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n= 1 + φ(\n|\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n2),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n2),\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nwhere the polynomials φ and ψ are at least linear in their argument. φ is real, while ψ takes values in C.\nWe epand the polynomials up to the ﬁfth order, namely\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nsuch that the chain rule gives\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n= 1 + α1|\n|\n= iωξ + c′ξ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n2 + α2|\n|\n2 + e′ξ\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n= (iωξ + c′ξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\n|\n= iωξ + iωα1ξ\n= iωξ + icξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n2 + e′ξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\nξ\n|\n|\n2 + c′ξ\nξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nξ\n|\n|\n2 + eξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n|\n|\n4 + . . . .\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n4 + . . . )(1 + α1|\n2 + iωα2ξ\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n2 + α2|\nξ\n|\n4 + α1c′ξ\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n4 + . . . )\n|\n4 + e′ξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n4 + . . .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nHere iωα1 + c′ = ic since the Lyapunov coeﬃcient of the Neimark-Sacker bifurcation is purely imaginary\nand iωα2 + α1c′ + e′ = e. This gives us normal form (7).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nA.3.2 R1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nAt the R1 bifurcation the matrices described in Section A.1 are\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n\n\n\nWe are in a case in which we can apply Theorem 2 from [23]. So we can deﬁne a T -periodic normal form\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n1 0\n0 1\n0 0\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n1 0\n1 1\n0 1\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n0 1\n0 0\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n, L0 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n˜L0 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nM0 =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n= 1 + ξ1 + p(τ, ξ),\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n= ˜L0ξ + P (τ, ξ),\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nwhere ξ = (ξ1, ξ2). The polynomials p and P are T -periodic in τ and at least quadratic in (ξ1, ξ2) such that\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\np(τ, ξ)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\np(τ, ξ) ˜L∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nP (τ, ξ) + ˜L∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n0P (τ, ξ)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nP (τ, ξ) ˜L∗\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\n0ξ = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nIf we write the polynomials in a Fourier expansion, namely\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nwe obtain for any l\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\np(τ, ξ) =\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n∞\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nXl=−∞\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\npl(ξ)ei 2πlτ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nT , P (τ, ξ) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nPl(ξ)ei 2πlτ\nT ,\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∞\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nXl=−∞\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.4**\nZ the following diﬀerential equations\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\nd\ndξ\nPl(ξ) ˜L∗\n0ξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\npl(ξ) ˜L∗\n0ξ\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\ni 2πl\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nT pl(ξ) = 0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ni 2πl\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nT Pl(ξ)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n˜L∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n0Pl(ξ) = 0.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nPutting our ˜L0 into the equations and writing Pl(ξ1, ξ2) = (P (1)\na set of diﬀerential equations in variable ξ2\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nl\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n(ξ1, ξ2), P (2)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nl\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.1**\n(ξ1, ξ2)) we can rewrite them as\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\nd\ndξ2\nP (1)\nl\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.5**\npl(ξ1, ξ2) = i\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\npl(ξ1, ξ2),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n(ξ1, ξ2) = i\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nP (1)\nl\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n(ξ1, ξ2),\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nP (2)\nl\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n(ξ1, ξ2) =\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nP (2)\nl\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n(ξ1, ξ2) + P (1)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\nl\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n(ξ1, ξ2)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nSince pl(ξ1, ξ2), P (1)\nl\nl equals zero and thus the polynomials are τ independent. We obtain\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\n(ξ1, ξ2) are polynomials, if l\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nl\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.1**\n= 0 the only solution is the trivial one. So\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nd\ndξ2\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\np0(ξ1, ξ2) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nd\ndξ2\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nP (1)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n(ξ1, ξ2) = 0,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nd\ndξ2\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nP (2)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n(ξ1, ξ2) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nξ1\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nP (1)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n(ξ1, ξ2).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nThe ﬁrst two equations show that p0 and P (1)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nare independent from ξ2, thus\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\np0(ξ1) = φ0(ξ1),\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nP (1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n(ξ1) = ξ1χ(ξ1).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nIntegrating the last diﬀerential equation gives\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nP (2)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\n(ξ1, ξ2) = ξ2χ(ξ1) + ψ(ξ1).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nNow we can further simplify our normal form. In fact, since the homological operator associated with ˜L0\nhas a two-dimensional null-space, we can make a change of variables such that polynomial P (1)\nvanishes (see\n[24]). So we can write\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n˜P (1)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n(ξ1) = 0,\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n˜P (2)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.3**\n(ξ1, ξ2) = ξ2φ1(ξ1) + φ2(ξ1),\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.4**\nwhere φ1 and φ2 are polynomials satisfying φ1(0) = φ2(0) =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\n= 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nξ1=0\nAssembling all the information gives us the following normal form\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.4**\ndφ2\ndξ1 \n\n\n\n= 1 + ξ1 + φ0(ξ1) = 1 + ξ1 + αξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n1 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n= ξ2,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n= ξ2φ1(ξ1) + φ2(ξ1) = aξ2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n1 + bξ1ξ2 + . . . .\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\ndτ\ndt\ndξ1\ndτ\ndξ2\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nNote that the polynomials φ0 and φ2 are at least quadratic in ξ1, while φ1 is at least linear in its argument.\nIn order to obtain the normal form presented in (8) we apply the chain rule which gives\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\ndξ1\ndt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n= ξ2(1 + ξ1 + αξ2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n1 + . . . )\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\n= ξ2 + ξ1ξ2 + . . .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n= (aξ2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\n1 + bξ1ξ2 + . . . )(1 + ξ1 + αξ2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n1 + . . . )\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n= aξ2\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\n1 + bξ1ξ2 + . . . .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nand\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nA.3.3 R2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\ndξ2\ndt\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nAt the R2 bifurcation the matrices described in Section A.1 are\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nM0 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n1\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n, L0 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\n0 0\n0 0\n0 0\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n0\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n˜L0 =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nWe are in a case in which we can apply Theorem 3 from [23]. So we have a 2T -periodic normal form\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n= 1 + p(τ, ξ),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\n= ˜L0ξ + P (τ, ξ),\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nwhere ξ = (ξ1, ξ2). The polynomials p and P are 2T -periodic in τ and at least quadratic in their argument\nwith\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(156)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n(157)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\np(τ, ξ)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\np(τ, ξ) ˜L∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\np(τ + T, ξ) = p(τ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nξ),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nP (τ, ξ) + ˜L∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n0P (τ, ξ)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nP (τ, ξ) ˜L∗\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nd\ndτ\nP (τ + T,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n−\nP (τ, ξ).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nξ) =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nSimilar as in the R1 case (since the ˜L0 matrix is the same) we obtain that all polynomials are independent\nfrom τ , l has to be equal to zero and the ﬁrst two polynomials p and P (1) are independent from ξ2.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nSince the polynomials obtained are independent from τ , we can rewrite (157) as:\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\np(ξ) = p(\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nξ), P (\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nξ) =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nP (ξ),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nobtaining that polynomial p is even (p(ξ1) = φ0(ξ2\nξ1 ˜φ1(ξ2\nas discussed in the previous section. So we can write\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\n1 ) and P (2)(ξ1, ξ2) = ξ2 ˜φ1(ξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n1)+ξ1 ˜φ2(ξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.1**\n1)) and the polynomials P (ξ1, ξ2) are odd (P (1)(ξ1) =\n1 )). Now we can simplify our normal form by changing variables\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n˜P (1)(ξ1) = 0,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n˜P (2)(ξ1, ξ2) = ξ2φ1(ξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n1 ) + ξ1φ2(ξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n1).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nPutting all information in the normal form equations gives the system\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\ndτ\ndt\ndξ1\ndτ\ndξ2\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n= 1 + φ0(ξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n1 ) = 1 + αξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n1 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n= ξ2,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n= ξ2φ1(ξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n1 ) + ξ1φ2(ξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n1) = aξ3\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n1 + bξ2\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n1ξ2 + . . . .\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nIn order to obtain normal form (9) we apply the chain rule\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\ndξ1\ndt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\ndξ2\ndt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n= ξ2(1 + αξ2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n1 + . . . )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n= ξ2 + αξ2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n1 ξ2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n1ξ2 + . . . )(1 + αξ2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n1 + . . . )\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n= (aξ3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n= aξ3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n1 + bξ2\n1 + bξ2\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1ξ2 + . . . .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nA.3.4 R3\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nThis is a simple case, since the Jordan block associated with the trivial multiplier is one-dimensional and -1\nis not a multiplier of the critical limit cycle. So we can write\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nM0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\nei 2π\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\ne−i 2π\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n3 \n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n, L0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\ni 2π\n3T\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\ni 2π\n3T\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n˜L0 =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\ni 2π\n3T\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\ni 2π\n3T \n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nWe are in a case in which we can apply Theorem 1 from [23]. So we can deﬁne the following T -periodic\nnormal form\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\ndz\ndτ\nwhere z = (z1, ¯z1). The polynomials p and P are T -periodic in τ and at least quadratic in their argument\nsuch that\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\n= ˜L0z + P (τ, z),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n= 1 + p(τ, z),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\np(τ, z)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\np(τ, z) ˜L\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n∗\n0z = 0,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nP (τ, z) + ˜L\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n∗\n0P (τ, z)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\nP (τ, z) ˜L\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n∗\n0z = 0.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nWe apply the results obtained in Example III.9 from [24] with ωT /2π = 1/3 to obtain\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\ndτ\ndt\ndz1\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n= 1 + P0(\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\ni2π\n3T\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n2, ¯z3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\nz1|\nz1 + z1Q0(\n|\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\n1ei2πτ /T , z3\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n1e−i2πτ /T ),\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n2, z3\n1e\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nz1|\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\n−i2πτ /T ) + ¯z2\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n1ei2πτ /T Q1(\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n2, ¯z3\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.7**r== 0.2**\n1ei2πτ /T ).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nz1|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\n\nDeﬁning a new variable ξ = e−i 2πτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n3T z1, this system can be rewritten as\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n2, ¯ξ3, ξ3),\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\n= 1 + φ0(\n|\n2, ξ3) + ¯ξ2φ2(\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\n= ξφ1(\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n2, ¯ξ3),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.2**\nwith polynomials φ0 and φ1 at least linear in their arguments, while φ2(0)\nautonomous and equivariant under the rotations of angle 2π/3. Expanding the polynomials gives\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.1**\n= 0. Notice that this system is\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nand thus\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n2 + α2ξ3 + α3 ¯ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\n= 1 + α1|\n= b ¯ξ2 + cξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n= (b ¯ξ2 + cξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n= b ¯ξ2 + cξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n|\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n2 + . . . )(1 + α1|\n|\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n2 + α2ξ3 + α3 ¯ξ3 + . . . )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nso normal form (10) is obtained.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nA.3.5 R4\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nAs in the previous case the Jordan block associated with the trivial multiplier is one-dimensional. The\nmatrices in Section A.1 are\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nM0 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\nei π\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\ne−i π\n**BLOCK**fs== 5.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n2 \n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n, L0 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n0 i π\n2T\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\ni π\n2T\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n˜L0 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\ni π\n2T\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\ni π\n2T \n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nWe can apply Theorem 1 from [23] and deﬁne a T -periodic normal form\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n= 1 + p(τ, z),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\ndz\ndτ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\n= ˜L0z + P (τ, z),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nwhere z = (z1, ¯z1). The polynomials p and P are T -periodic in τ and at least quadratic in their argument\nsuch that\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\np(τ, z)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nd\ndz\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\np(τ, z) ˜L∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n0z = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nP (τ, z) + ˜L∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n0P (τ, z)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nd\ndz\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nP (τ, z) ˜L∗\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n0z = 0.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nAgain, we use Example III.9 from [24] with ωT /2π = 1/4 and obtain\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n= 1 + P0(\n|\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n2, ¯z4\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n=\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n1ei2πτ /T , z4\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n1e−i2πτ /T ),\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n2, z4\n1e\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n−i2πτ /T ) + ¯z3\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n2, ¯z4\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\n1ei2πτ /T ).\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\n2T z1, the system can be rewritten as\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n2, ¯ξ4, ξ4),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\n= 1 + φ0(\n|\n2, ξ4) + ¯ξ3φ2(\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n= ξφ1(\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n2, ¯ξ4),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\nwith polynomials φ0 and φ1 at least linear in their arguments, while φ2(0)\nautonomous and equivariant under the rotations of angle π/2. Expanding the polynomials gives\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.1**\n= 0. Notice that this system is\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\ndτ\ndt\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\n= 1 + α1|\nξ\n= cξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n2 + α2ξ4 + α3 ¯ξ4 + . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\n2 + d ¯ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nWe still have to transform the second equation, namely\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ndξ\ndt\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n= (cξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\n2 + d ¯ξ3 + . . . )(1 + α1|\n|\n2 + d ¯ξ3 + . . . ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n|\nξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.6**\nso normal form (11) is obtained.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n= cξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nA.3.6 LPPD\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n2 + α2ξ4 + α3 ¯ξ4 + . . . )\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nAt the LPPD bifurcation the matrices from Section A.1 are\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nM0 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n1 1\n0 1\n0 0\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n1\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n, L0 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\n0 1\n0 0\n0 0\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n0\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n˜L0 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nWe are in a case in which we can apply Theorem 3 from [23]. So we can deﬁne a 2T -periodic normal\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nform\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n= 1 + ξ1 + p(τ, ξ),\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n= ˜L0ξ + P (τ, ξ),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nwhere ξ = (ξ1, ξ2). The polynomials p and P are 2T -periodic in τ and at least quadratic in their argument\nsuch that\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nP (τ, ξ) + ˜L∗\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n0P (τ, ξ)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nd\ndξ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nP (τ, ξ) ˜L∗\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(158)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(159)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(160)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\np(τ, ξ)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\np(τ, ξ) ˜L∗\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.6**\nd\nd\ndξ\ndτ\np(τ + T, ξ1, ξ2) = p(τ, ξ1,\nP (1)(τ + T, ξ1,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n0ξ = 0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nξ2),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\nξ2) = P (1)(τ, ξ1, ξ2), P (2)(τ + T, ξ1,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nξ2) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nP (2)(τ, ξ1, ξ2).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nBy putting ˜L0 into (158), we obtain\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\np(τ, ξ1, ξ2) =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nP (1)(τ, ξ1, ξ2) =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nd\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nP (2)(τ, ξ1, ξ2) = 0,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nso our polynomials are independent from τ . Then, using (159) and (160), there holds\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\np(ξ1, ξ2) = p(ξ1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nξ2), P (1)(ξ1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nξ2) = P (1)(ξ1, ξ2), P (2)(ξ1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nξ2) =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nP (2)(ξ1, ξ2),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nso the polynomials are of the following form\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n2 )(1 + χ3(ξ1)),\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\np = χ1(ξ1) + χ2(ξ2\nP (1) = ψ1(ξ1) + ψ2(ξ2\nP (2) = ξ2ϕ1(ξ1) + ξ2ϕ2(ξ2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n2)(1 + ψ2(ξ1)),\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nwith χ1 and ψ1 at least quadratic in their argument and all the other polynomials at least linear in their\nargument.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nAssembling all the information gives the following system\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\ndτ\ndt\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\ndξ1\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\ndξ2\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\n= 1 + ξ1 + χ1(ξ1) + χ2(ξ2\n2 )(1 + χ3(ξ1))\n1 + α′\n2 + α′\n30ξ3\n1 + α02ξ2\n= 1 + ξ1 + α20ξ2\n= ψ1(ξ1) + ψ2(ξ2\n2)(1 + ψ2(ξ1))\n= a′\n1 + a′\n2 + a′\n1 + a′\n30ξ3\n12ξ1ξ2\n02ξ2\n= ξ2ϕ1(ξ1) + ξ2ϕ2(ξ2\n2 )(1 + ϕ3(ξ1))\n11ξ1ξ2 + b′\n= b′\n2 + . . . .\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1ξ2 + b03ξ3\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n20ξ2\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n21ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n2 + . . . ,\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n12ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ndξ1\ndt\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nApplying the chain rule gives\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n1 + a′\n1 + a′\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n02ξ2\n02ξ2\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n2 + a′\n30ξ3\n2 + a30ξ3\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n1 + a′\n20ξ3\n1 + a12ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n1 + a′\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n12ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n2 + a′\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n02ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n2 + . . .\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n= a′\n= a′\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\n20ξ2\n20ξ2\n12 + a′\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nwith a30 = a′\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\n30 + a′\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n20, a12 = a′\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n02, and\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\ndξ2\ndt\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n= b\n= b′\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n′\n′\n21ξ2\n11ξ1ξ2 + b\n11ξ1ξ2 + b21ξ2\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n′\n11ξ2\n1ξ2 + b\n1ξ2 + b03ξ3\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n2 + . . . ,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n1 ξ2 + b03ξ3\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n2 + . . .\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nwith b21 = b′\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n21 + b′\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.5**\n11. Now, we do the substitution ξ1 7→ −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nξ1 which gives\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\n= 1\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\nξ1 + α20ξ2\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n1 + α02ξ2\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nα′\n30ξ3\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\nα′\n12ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n2 + . . . ,\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n1 −\n2 −\n2 + a30ξ3\n1 + a12ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n2 + . . . ,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\na\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n′\n20ξ2\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n1 −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\na\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n′\n02ξ2\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nb′\n11ξ1ξ2 + b21ξ2\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n1ξ2 + b03ξ3\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n2 + . . . .\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\ndτ\ndt\ndξ1\ndt\ndξ2\ndt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n=\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\nα′\n12, a20 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nBy putting α30 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nα′\n30, α12 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\na′\n20, a02 =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\na′\n02, b11 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\nb′\n11, we obtain (12).\n**BLOCK**fs== 14.3**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nB Poincar´e maps of the periodic normal forms\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.4**\nB.1 Bifurcations with 2 critical eigenvalues\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nB.1.1 CPC\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nIf we reparametrize time, we obtain the system\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\ndτ\ndτ\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n= 1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ncξ3\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nξ + α1ξ2 + α2ξ3 +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n· · ·\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nNote that we can deﬁne a Poincar´e map out of system (5) evaluating the solution of this system starting\nfrom a point (0, η) at time t = T . Since the two equations are uncoupled we can consider only the second\none. Making one Picard iteration [29], we can construct an approximation of this map as follows\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nξ0 = η,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\nξ1 = ξ0 +\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\ncξ3\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\ndt\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nand expanding ξ1 in a Taylor series we obtain\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n(161)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.4**\nη + cT η3 + O(η4).\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nη\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n7→\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nFurther iterations do not change this expansion and so (161) is the Poincar´e map of system (5). Note that\nthis Poincar´e map is similar to the one for the cusp point of ﬁxed points [29]. Therefore, we can conclude\nthat the behavior of the system in the neighborhood of the bifurcation is the same. In particular, referring\nalways to [29], we can draw the bifurcation diagram of the Poincar´e map and obtain Figure 11. On the two\ndrawn curves, with label T1 and T2, two limit cycles collide and disappear. The output given by MatCont\nis the normal form coeﬃcient c.\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nβ2\n**BLOCK**fs== 7.7**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nT2\n**BLOCK**fs== 7.7**b== 0.6**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n˜η\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n˜η\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nT1\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nη\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nη\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT1\n**BLOCK**fs== 7.7**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nβ1\n**BLOCK**fs== 7.7**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n˜η\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\nT2\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n˜η\n**BLOCK**fs== 7.8**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 8.6**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n˜\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nη\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nη\n**BLOCK**fs== 8.6**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nT1\n**BLOCK**fs== 8.6**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n˜\n**BLOCK**fs== 8.6**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.6**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.6**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nT1\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.6**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\nT2\n**BLOCK**fs== 8.6**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n˜\n**BLOCK**fs== 7.8**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.6**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nT2\n**BLOCK**fs== 7.8**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.6**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n˜\n**BLOCK**fs== 5.9**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n(a) c < 0\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n(b) c > 0\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.2**\nFigure 11: Bifurcation diagram of the cusp bifurcation of the ﬁxed point normal form.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nB.1.2 GPD\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.1**\nFirst, we have to calculate the second iterate of the normal form of the Generalized ﬂip bifurcation of ﬁxed\npoints:\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(162)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\ng(v) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nv +\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n˜ev5 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n⇒\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\ng2(ω) = v\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n˜ev5.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nReparametrizing the time of (6) and doing Taylor expansion up to the ﬁfth order gives\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\ndτ\ndτ\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n= 1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n= eξ5 + . . . ,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nwhere the dots are O(ξ6) terms 2T -periodic in τ . Doing one Picard iteration of the second equation up to\n2T we obtain the Poincar´e map of our normal form\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nand see that it is the same map as the second iterate of the generalized period-doubling normal form of ﬁxed\npoints. In particular, since the coeﬃcient of the ﬁfth order term of the second iterate of the normal form of\nﬁxed points has opposite sign than the one of limit cycles, we can conclude that the behavior of the system\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\nξ + 2T eξ5 + . . .\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n7→\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nat the bifurcation is the same but with opposite sign of the normal form coeﬃcient. Moreover, since T is\nnonzero, the non-degeneracy condition is e\n= 0. In fact, if e > 0 we obtain the bifurcation diagram reported\nin Figure 12-(a), in which the limit point bifurcation of the period doubled limit cycles (T (2)) is tangent to\nthe supercritical period-doubling branch (the one in which the normal form coeﬃcient is positive, so labeled\nas F (1)\n+ ), while if e is negative we are in the opposite situation, reported in Figure 12-(b). The output given\nby MatCont is the normal form coeﬃcient e.\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nT (2)\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\nF (1)\n+\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 6.3**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nF (1)\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n, F (1)\n+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n,F (1)\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT (2)\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.3**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\nF (1)\n+\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.3**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nT (2)\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nF (1)\n**BLOCK**fs== 6.4**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n, F (1)\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 6.4**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 5.3**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.3**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.3**b== 0.5**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.3**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nT (2)\n**BLOCK**fs== 6.4**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 5.3**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n+\n,F (1)\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n(a) e > 0\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n(b) e < 0\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nFigure 12: Bifurcation diagram of the degenerate period-doubling point bifurcation of the ﬁxed point normal\nform.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nRemark that the normal form coeﬃcient of the period-doubling bifurcation of limit cycles, computed\nthrough periodic normalization as done in [30], has opposite sign of the one for ﬁxed points. Also in this\ncase, the normal form coeﬃcient of the GPD bifurcation of limit cycles has opposite sign of the one for ﬁxed\npoints.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nB.2 Bifurcations with 3 critical eigenvalues\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nB.2.1 CH\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nIn this section we will show how the periodic normal form (7) is related to the normal form of the Chenciner\nbifurcation of ﬁxed points, and how the non-degeneracy condition of the Chenciner bifurcation of limit cycles\nis related to the Chenciner bifurcation of maps. First note that, if we scale the time, we can rewrite system\n(7) as\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\ndτ\ndτ\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n= 1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n= iωξ + i(c\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nα1ω)ξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n2 + (e\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\ni(α1c\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nα2\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n1ω + α2ω))ξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n4 + . . . .\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nWe need to make a change of variables in order to obtain a quasi-identity ﬂow. Introducing the new\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\ncomplex variable z = e−iωτ ξ, the second equation can be rewritten as\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\ndz\ndτ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n= i(c\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nα1ω)z2¯z +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ne\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ni(α1c\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nα2\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n1ω + α2ω)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nz3¯z2 + . . . .\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\nDoing two Picard iterations up to time T , we obtain the rotating Poincar´e map of the system\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\nξ + iT (c\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\nξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\nα1ω)ξ2 ¯ξ + T\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n+ α1cT ω\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n1T ω2 + i\nα2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nα2\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n1ω\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\nα1c\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nα2ω\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.8**r== 0.1**\nξ3 ¯ξ2 + . . . .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\nℜ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.2**l== 0.1**r== 0.1**\nNote that this system has the same Poincar´e map as the Chenciner bifurcation in the ﬁxed point case [29], so\nwe expect the same bifurcation scenario on the Poincar´e map of the system. Notice that the real part of the\nﬁrst Lyapunov coeﬃcient is 0, since the Neimark-Sacker bifurcation is degenerate. The sign of the second\nLyapunov coeﬃcient L2 (as deﬁned on page 420 of [29]) determines the bifurcation scenario. However, from\n(e) > 0 with an unstable\n(7) we can derive that\n(e) < 0 corresponds with a stable critical limit cycle and\ncritical limit cycle. Therefore, the case\n(e) > 0 corresponds\n(e) < 0 corresponds with the case L2 < 0 and\n(e) and the second Lyapunov coeﬃcient L2 as deﬁned in [29] have the same sign and\nwith L2 > 0. So\nvanish at the same time. Since both coeﬃcients have the same eﬀect and L2 requires more computations, we\ncompute\n(e) the second Lyapunov\ncoeﬃcient. The bifurcation diagram in the neighborhood of this codim 2 point can be found by looking at\n(e) < 0 the outer invariant curve is stable and the limit point of tori curve\nthe sign of this value. When\nis tangent to the subcritical Neimark-Sacker branch, as shown in Figure 13-(a). When\n(e) > 0 the outer\ninvariant curve is unstable and the limit point of tori curve is tangent to the supercritical Neimark-Sacker\nbranch, see Figure 13-(b). The output given by MatCont is\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\n(e) to determine the bifurcation scenario, and in this paper we will call\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n(e).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nℜ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nℜ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\nℜ\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nℜ\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 9.2**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nTc\n**BLOCK**fs== 7.7**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.7**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 9.2**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nTc\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 9.2**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nβ2\nN+\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.7**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 9.2**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nN−\n**BLOCK**fs== 9.2**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\nβ1\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 9.2**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nTc\n**BLOCK**fs== 6.4**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nN+\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 7.7**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 9.2**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 9.2**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nTc\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.4**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\n(a) ℜ(e) < 0\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\n(b) ℜ(e) > 0\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nFigure 13: Bifurcation diagram of the generalized Neimark-Sacker bifurcation of the ﬁxed point normal form.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\nB.2.2 R1\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\nIn this section we will show how the periodic normal form (8) is related to the normal form for the 1:1\nresonance of ﬁxed points, which allows us to formulate the non-degeneracy condition. First note that, if we\nscale the time, we can rewrite system (8) as\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n= 1,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nξ2 + ξ1ξ2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n+ . . . ,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n+ . . . .\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nDoing Picard iterations up to time T , it’s possible to show that the generated map is the same, up to second\norder terms, of the one generated by Picard iterations up to time 1 of the truncated system\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\n˙ξ1 = T (ξ2 + 2ξ1ξ2),\n˙ξ2 = T (aξ2\n1 + bξ1ξ2).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n(\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nThis last system is topological equivalent to the one-time shift ODE which represents the ﬁrst iteration of\nthe normal form of 1:1 resonance of ﬁxed points, i.e. the system\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nsince we can transform one in the other using the following change of variables\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n˙ζ1 = ζ2,\n˙ζ2 = a1ζ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1 + b1ζ1ζ2,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n(\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nζ1 = ξ1 −\nζ2 = T ξ2.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n(\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nξ2\n1 ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nThe bifurcation phenomena in the 1:1 resonance of cycles are the same of those which appear in the 1:1\nresonance of ﬁxed points. Notice that\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nso the non-degeneracy conditions are\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\na1 = T 2a,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nb1 = T b,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\na\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n= 0,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nb\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n= 0,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nand the cases depend on the sign of the product of a and b. In particular, as shown in ﬁgure 14, if the two\ncoeﬃcients have diﬀerent sign the Neimark-Sacker bifurcation (labeled H) is supercritical (with negative\nnormal form coeﬃcient), while in the other case it is subcritical. The output given by MatCont is the\nproduct of the coeﬃcients a and b.\n**BLOCK**fs== 6.9**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 6.9**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nT+\n**BLOCK**fs== 6.2**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 6.9**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nP\n**BLOCK**fs== 6.2**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 6.2**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.9**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\nβ2\n**BLOCK**fs== 6.9**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nT+\n**BLOCK**fs== 7.6**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.9**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.9**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nβ1\n**BLOCK**fs== 6.9**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nT−\n**BLOCK**fs== 7.3**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nP\n**BLOCK**fs== 6.9**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nT−\n**BLOCK**fs== 7.6**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.6**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.9**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nH\n**BLOCK**fs== 6.2**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.9**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n, H\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.6**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 6.9**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nP\n**BLOCK**fs== 7.3**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.3**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nT\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.3**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nP\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.3**b== 0.3**t== 0.6**l== 0.7**r== 0.2**\nT\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 7.3**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.3**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nH\n**BLOCK**fs== 7.3**b== 0.2**t== 0.7**l== 0.7**r== 0.2**\nT+\n**BLOCK**fs== 7.3**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n2 , H\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 7.3**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nT+\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\n(a) a > 0, ab < 0\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\n(b) a > 0, ab > 0\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nFigure 14: Bifurcation diagram of the 1:1 resonance bifurcation of the ﬁxed point normal form. The other\ntwo cases in which a < 0 can be obtained by a reﬂection around the origin of the state portraits and a\nhorizontal ﬂip of the bifurcation diagrams.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nB.2.3 R2\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nIn this section we will show how the periodic normal form (9) is related with the normal form for the 1:2\nresonance of ﬁxed points, which allows us to formulate the non-degeneracy condition. First note that, if we\nscale the time, we can rewrite system (9) as\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\ndτ\ndτ\ndξ1\ndτ\ndξ2\ndτ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n= 1,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n1 ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nξ2 + αξ2\n1 + αξ2\n\n1 + bξ2\n1ξ2\n1 + αξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\naξ3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n+ . . . ,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n+ . . . .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nDoing Picard iterations up to time 2T , it’s possible to show that the generated map is the same, up to third\norder terms, of the one generated by Picard iterations up to time 1 of the truncated system\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n˙ξ1 = 2T ξ2,\n˙ξ2 = 2T (aξ3\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n1 + bξ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n1ξ2).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n(\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nThis last system is topological equivalent to the one-time shift ODE which represents the second iteration\nof the normal form of 1:2 resonance of ﬁxed points, i.e. the system\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n˙ζ1 = ζ2,\n˙ζ2 = a1ζ3\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n1 + b1ζ2\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n1 ζ2,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n(\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nsince we can transform one in the other using the following change of variables\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.4**\nζ1 = ξ1,\nζ2 = 2T ξ2.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n(\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe bifurcation phenomena in the 1:2 resonance of cycles are the same that appear in the corresponding\nbifurcation of ﬁxed points. Notice that in this case\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nso the non-degeneracy conditions are\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\na1 = 4T 2a,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nb1 = 2T b.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\na\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n= 0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nb\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n= 0.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nWe have four diﬀerent unfoldings as possible bifurcation diagrams, determined by the signs of the coeﬃcients.\nThe ones with negative b are reported in Figure 15. The other two cases can be obtained by reversing the\narrows of the phase portraits and making a vertical ﬂip both of the state portraits and of the bifurcation\ndiagrams. The primary Neimark-Sacker bifurcation (labeled H (1)) is supercritical (with negative normal form\ncoeﬃcient) if our coeﬃcient b is negative, subcritical otherwise. Moreover if a < 0 a secondary Neimark-\nSacker bifurcation (H (2)) is rooted at the 1:2 resonance point with opposite criticality of the primary one.\nThe output given by MatCont is (a, b).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nB.2.4 R3\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nIn this section we will show how the periodic normal form (10) is related with the normal form for the 1:3\nresonance of ﬁxed points, which allows us to formulate the non-degeneracy condition. First note that, if we\nscale the time, we can rewrite system (10) as\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n= 1,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n= b ¯ξ2 + cξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n2 + . . . .\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nE1\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nE0\n**BLOCK**fs== 6.4**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nE2\n**BLOCK**fs== 6.4**b== 0.7**t== 0.2**l== 0.1**r== 0.9**\nC\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nF (1)\n+\n**BLOCK**fs== 6.1**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 6.1**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nF (1)\n**BLOCK**fs== 5.3**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 4.5**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 6.4**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\nC\n**BLOCK**fs== 4.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nH (1)\n**BLOCK**fs== 6.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.4**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.7**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n, H (1)\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.7**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n, H (2)\n**BLOCK**fs== 4.4**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.3**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nE1\n**BLOCK**fs== 5.3**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nE2\n**BLOCK**fs== 5.3**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nE0\n**BLOCK**fs== 6.3**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nε2\n**BLOCK**fs== 4.4**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nF (1)\n+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.4**b== 0.7**t== 0.2**l== 0.7**r== 0.2**\nH (2)\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.3**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nP\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.3**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nK\nε1\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.4**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nF (1)\n−\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nK\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.4**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nH (1)\n**BLOCK**fs== 4.4**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.3**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nP\n**BLOCK**fs== 4.4**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n(a) b < 0, a > 0\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n(b) b < 0, a < 0\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nFigure 15: Bifurcation diagram of the 1:2 resonance bifurcation of the ﬁxed point normal form. The other\ntwo possible cases in which b > 0 can be obtained by reversing time and making a vertical ﬂip both of the\nstate portraits and of the bifurcation diagrams.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nDoing Picard iterations up to time 3T , it’s possible to show that the generated map is the same, up to cubic\nterms, of the one generated by Picard iterations up to time 1 of the truncated system\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\nThis system is topological equivalent to the one-time shift ODE which represents the third iteration of the\nnormal form of 1:3 resonance of ﬁxed points, i.e. the system\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\n˙ξ = 3T b ¯ξ2 + 3T cξ\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n2.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nξ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nThe bifurcation phenomena in the 1:3 resonance of cycles are the same of the ones which appear in the 1:3\nresonance of ﬁxed points, if the non-degeneracy conditions are satisﬁed, i.e.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.4**\n˙ζ = b1 ¯ζ2 + c1ζ\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n2.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nζ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n|\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nb\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n= 0,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\n(c)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n= 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\nℜ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\nAs can be seen in Figure 16, if\n(c) < 0 the Neimark-Sacker bifurcation (labeled N ) is supercritical (with\nnegative normal form coeﬃcient), while in the other case it is subcritical. The output given by MatCont is\n(b,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n(c)).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nℜ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nℜ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nIn this section we will show how the periodic normal form (11) is related with the normal form for the 1:4\nresonance of ﬁxed points, which allows us to formulate the non-degeneracy condition. First note that, if we\nscale the time, we can rewrite system (11) as\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\ndτ\ndτ\ndξ\ndτ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n= 1,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n= cξ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n2 + d ¯ξ3 + . . . .\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nDoing Picard iterations up to time 4T , it’s possible to show that the generated map is the same, up to cubic\nterms, of the one generated by Picard iterations up to time 1 of the truncated system\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n˙ξ = 4T cξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nξ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n2 + 4T d ¯ξ3.\n**BLOCK**fs== 4.8**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\n1+\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nE1\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n1−\n**BLOCK**fs== 5.9**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n+\n**BLOCK**fs== 4.8**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n2+\n**BLOCK**fs== 5.9**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nE3\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nE0\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nE2\n**BLOCK**fs== 4.8**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n3+\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nβ2\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n2+\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nH+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1−\n**BLOCK**fs== 4.9**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n3+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n3−\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nβ1\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n3−\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n2−\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nN\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nH−\n**BLOCK**fs== 4.8**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n3+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n2−\n**BLOCK**fs== 5.9**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nH−\n**BLOCK**fs== 5.9**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nH\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nH\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n2+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n3+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nH+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n1+\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.8**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.8**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n1+\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 4.8**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\nE3\n**BLOCK**fs== 5.9**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nE1\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nE0\n**BLOCK**fs== 5.9**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\nE2\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n(a) ℜ(c) < 0\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n(b) ℜ(c) > 0\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.1**\nFigure 16: Bifurcation diagram of the 1:3 resonance bifurcation of the ﬁxed point normal form.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nThis system is topologically equivalent to the one-time shift ODE which represents the fourth iteration of\nthe normal form of 1:4 resonance of ﬁxed points, i.e. the system\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\n|\nThe bifurcation phenomena in the 1:4 resonance of cycles are the ones which appear in the corresponding\nbifurcation of ﬁxed points, if the non-degeneracy condition is satisﬁed, i.e.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.5**\n|\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n˙ζ = c1ζ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nζ\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2 + d1 ¯ζ3.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nd\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n= 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nAfter deﬁning\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\nA =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\nc\nd\n|\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n|\nwe can determine which bifurcations occur by looking at the place of A in the Gauss plane (see Figure 17).\nWe here report the diﬀerent possible bifurcation diagrams. A complete investigation can be found in the\nliterature [27, 29].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nMany topologically diﬀerent bifurcation diagrams can be found near a 1:4 resonance point. The analysis,\nif one excludes higher codimension situations, can be reduced to 22 diﬀerent cases, which, as mentioned\nbefore, depend on the value of A. First of all, analyzing the normal form, one can divide the Gauss plane\n(A) < 0 the primary Neimark-Sacker bifurcation is supercritical, in\ninto two big regions: in the semiplane\nthe semiplane\n(A) > 0 can therefore be obtained\nby inverting the direction of the vector ﬁelds. We can further reduce the analysis to the third quadrant of\nthe Gauss plane, since the 12 possible cases are topologically equivalent paired through the transformation\n¯ζ. The diﬀerent regions are reported in Figure 17, in which only some curves (the continuous lines) are\nζ\nknown analytically, the dashed curves are computed numerically.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\n(A) > 0 it is subcritical. What happens in the semiplane\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n7→\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nℜ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nℜ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nℜ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nFigure 18 shows the possible bifurcation diagrams with the sketches of the phase portraits for the Poincar´e\n(A) < 0. Many local and global bifurcations are involved in the diﬀerent scenarios.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.4**\nmaps in the case that\nWe use the following notation, consistent with the rest of the text:\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nℜ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nN: Neimark-Sacker bifurcation. In regions VII and VIII we also have a Neimark-Sacker bifurcation of the\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\nperiod-4 limit cycle.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nℑ(A)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nII\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\nd\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nc1\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nc2\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nd\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nV\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\nVb\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\ng\nVII\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nVII\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\ne\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nVIII\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ng\nVIII\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\ne\nV\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nf\nVb\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nIII\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nIII\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nVI\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nVa\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nIIIa\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nIVa\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nIV\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nI\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nIVa\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nIIIa\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nVa\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nVI\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\ng\nIVa\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\ne\nI\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\ng\nIVa\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nf\nVI\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\nf\nVa\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\ng\nIIIa\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\ng\nIIIa\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nf\nVa\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nf\nVI\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nf\nIV\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\nf\nIII\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nf\nIII\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nVb\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nV\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nVIII\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\ng\nVIII\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\ne\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nVII\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\ng\nVII\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nℜ(A)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nf\nVb\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\ne\nV\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\ne\nII\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.2**\nFigure 17: Partitioning of the A plane into topologically diﬀerent regions.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nT: Fold bifurcation of the period-4 limit cycles. There are three possibilities. Superscript in, on or out\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.3**\nmeans that the bifurcation happens inside, on or outside the invariant curve.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nH: Homoclinic connection of the period-4 saddle limit cycle. Superscript S means that the born invariant\ncurve is smaller than the limit cycle (a square looking homoclinic connection), C that it is bigger (a\nclover looking homoclinic connection), and L means that the born invariant curve is around the period-\n4 limit cycle; subscript + (-) means that the saddle quantity is positive (negative), so the borning\ninvariant curve is repelling (attracting).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nF: Fold bﬁrucation of the tori.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nThe output given by MatCont is (A, d).\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.1**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nN\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n3+\n**BLOCK**fs== 3.5**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\non\n**BLOCK**fs== 5.1**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n1+\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.2**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.6**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n1+\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nII\n**BLOCK**fs== 3.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n2+\n**BLOCK**fs== 3.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n1+\n**BLOCK**fs== 3.6**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nE0\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nIV\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n2+\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n1+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 3.6**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nE0\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n2+\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nout\n**BLOCK**fs== 5.1**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n3+\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n4+\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\ns\nH\n**BLOCK**fs== 3.2**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\ns\nH\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nout\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nN\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n4+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n4_\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nI\n**BLOCK**fs== 2.8**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\ns\nH_\n**BLOCK**fs== 3.6**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nE0\n**BLOCK**fs== 3.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n2+\n**BLOCK**fs== 3.6**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nE3\n**BLOCK**fs== 3.6**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nE2\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nIIIa\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n2+\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n1+\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.1**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n2+\n**BLOCK**fs== 3.2**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nN\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.6**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nHs\n**BLOCK**fs== 3.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n3+\n**BLOCK**fs== 3.2**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\ns\nH\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n3+\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\ns\nH\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n3+\n**BLOCK**fs== 3.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n1+\n**BLOCK**fs== 3.6**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nE4\n**BLOCK**fs== 3.6**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nE1\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\nT\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n1+\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ns\nH\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n3+\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\ns\nH\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n4+\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\non\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.2**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n4+\n**BLOCK**fs== 3.2**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n2+\n**BLOCK**fs== 4.6**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nN\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\n1+\n**BLOCK**fs== 5.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 3.6**b== 0.5**t== 0.4**l== 0.2**r== 0.8**\nE0\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nV\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n2+\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n1+\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 3.6**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nE0\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 3.5**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nin\nT _\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n3+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n4+\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nVa,Vb\n**BLOCK**fs== 3.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n2+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n3+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n4+\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n1+\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 3.2**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.6**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n4_\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n2+\n**BLOCK**fs== 3.5**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nout\nT\n_\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n3+\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n4+\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\ns\nH\n**BLOCK**fs== 3.2**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nc\nH\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nHc\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nN\n**BLOCK**fs== 4.1**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nin\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 4.1**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n3_\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n1+\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 3.6**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nE0\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 3.5**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nin\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 3.2**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n2+\n**BLOCK**fs== 4.6**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n1+\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nN\n**BLOCK**fs== 4.1**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n3+\n**BLOCK**fs== 3.5**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\ns\nH\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n4+\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.2**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.5**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nc\nH\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nHc\n**BLOCK**fs== 3.5**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nin\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nT\n**BLOCK**fs== 4.1**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n3_\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nIII\n**BLOCK**fs== 3.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nout\n**BLOCK**fs== 5.1**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nT\n**BLOCK**fs== 3.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n3+\n**BLOCK**fs== 3.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n3+\n**BLOCK**fs== 3.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n2+\n**BLOCK**fs== 3.2**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 4.1**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n2+\n**BLOCK**fs== 3.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\non\n**BLOCK**fs== 5.1**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nT\n**BLOCK**fs== 3.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n1+\n**BLOCK**fs== 3.2**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n2+\n**BLOCK**fs== 3.5**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nout\n**BLOCK**fs== 5.1**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 3.5**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\ns\nH\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n3+\n**BLOCK**fs== 3.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\non\n**BLOCK**fs== 5.1**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nT\n**BLOCK**fs== 3.2**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\non\n**BLOCK**fs== 5.1**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nT\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.6**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nE0\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n3+\n**BLOCK**fs== 3.5**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\ns\nH\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n4+\n**BLOCK**fs== 3.5**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\non\n**BLOCK**fs== 5.1**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.2**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n4+\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n1+\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 3.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.6**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.1**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nIVa\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n2+\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n3+\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n1+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.6**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nE0\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 3.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n4+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.1**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nT\n**BLOCK**fs== 3.2**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n2+\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n3+\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n1+\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.6**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n4+\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.5**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\ns\nH\n**BLOCK**fs== 3.2**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\ns\nH\n**BLOCK**fs== 4.1**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nout\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nVI\n**BLOCK**fs== 3.0**b== 0.4**t== 0.5**l== 0.6**r== 0.3**\n2+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n3+\n**BLOCK**fs== 3.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n4+\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n1+\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.6**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nE0\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n1+\n**BLOCK**fs== 3.2**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nT\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n2+\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n3+\n**BLOCK**fs== 4.6**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 4.1**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n3_\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nF\n**BLOCK**fs== 3.5**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\ns\nH\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n4+\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nF\n**BLOCK**fs== 3.5**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nc\nH\n+\n**BLOCK**fs== 3.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.2**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 4.1**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.5**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nin\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nT\n**BLOCK**fs== 5.1**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nHc\n**BLOCK**fs== 3.2**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n+\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nVII\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n2+\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n3+\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n6+\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n6_\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nVIII\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n2+\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n3+\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n4+\n**BLOCK**fs== 3.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n6+\n**BLOCK**fs== 3.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n1+\n**BLOCK**fs== 3.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 3.6**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nE0\n**BLOCK**fs== 3.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n1+\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n3+\n**BLOCK**fs== 3.2**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n2+\n**BLOCK**fs== 4.6**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nin\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nT\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\nN\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\nin\n**BLOCK**fs== 5.1**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\nT\n**BLOCK**fs== 3.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.5**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\ns\nH\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n6+\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n6_\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n5_\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n4_\n**BLOCK**fs== 3.2**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nHc\nL\nH+\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nN’\n**BLOCK**fs== 3.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 3.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n5_\n**BLOCK**fs== 3.5**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nL\nH+\n**BLOCK**fs== 3.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n1+\n**BLOCK**fs== 3.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.6**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nE0\n**BLOCK**fs== 3.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 3.2**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n2+\n**BLOCK**fs== 4.6**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n1+\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nT\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nN\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 3.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n3+\n**BLOCK**fs== 3.5**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\ns\nH\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n4+\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n6+\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nF\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 3.5**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nin\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nT\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n6_\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n5_\n**BLOCK**fs== 4.1**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n4_\n**BLOCK**fs== 3.2**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 3.5**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nc\nH+\nL\nH+\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nN’\n**BLOCK**fs== 3.0**b== 0.1**t== 0.8**l== 0.8**r== 0.2**\n5_\n**BLOCK**fs== 3.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n6_\n**BLOCK**fs== 5.1**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\nHc\n**BLOCK**fs== 3.2**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n+\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nFigure 18: Bifurcation diagrams locally to the 1:4 resonance bifurcation in the diﬀerent regions of ﬁgure 17.\nThe cases in which Re(A) > 0 can be obtained with the transformation t\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nB.2.6 LPPD\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nIn this section we will show how the periodic normal form (12) is related to the normal form for the fold-ﬂip\nof ﬁxed points, which allows us to formulate the non-degeneracy condition. As for the R2 case we show the\ntopological equivalence of the 2T -shift map of our system with the 1-shift map of the approximating vector\nﬁeld of the fold-ﬂip normal form for ﬁxed points. First of all we can prove, by scaling the time and using\nPicard iterations, that the Poincar´e map of our normal form is the same, up to cubic terms, as the one-shift\nmap of the truncated system\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\n˙ξ1 = 2T a20ξ2\n1 + 2T a02ξ2\n˙ξ2 = 2T b11ξ1ξ2 + 2T (b21 + b11)ξ2\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n(\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1 ξ2 + 2T b03ξ3\n2.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.4**\n2 + 2T (a30 + a20)ξ3\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n1 + 2T (a12 + a02)ξ1ξ2\n2,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\nThis system is topologically equivalent with the one-time shift ODE which represents the second iteration\nof the normal form of the fold-ﬂip bifurcation of ﬁxed points, i.e. the system\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n(163)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n˙ζ2 =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n˙ζ1 = a1ζ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n1 + b1ζ2\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n2 + (c1 −\n1)ζ2\n(a1 −\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nζ1ζ2 +\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n1)ζ3\na2\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n1 ζ2 +\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n1 + (d1 −\n\nb1ζ3\n2 ,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\na1b1 + b1)ζ1ζ2\n2 ,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\nsince this second system can be obtained (neglecting higher order terms) from the ﬁrst one using the trans-\nformation\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\nζ1 =\n−\nζ2 = 2b11T ξ2.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n2b11T ξ1 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n(\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n2T\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.4**\nb11 + b21 + a20b11T + b2\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n11T\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\n2T (b03 + a02b11T )ξ2\n2,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nξ2\n1 −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.2**\nThis transformation should be invertible, so one non-degeneracy condition is involved, namely\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nIf this condition is satisﬁed, then the system can be put in the form (163), where the constants are deﬁned\nas\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n= 0.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nb11 6\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\na1 =\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\na20\nb11\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\nb1 =\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\na02\nb11\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.5**r== 0.4**\nc1 =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\na20 + a30 + 2a2\n2b2\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n11T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\n20T\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nd1 = −\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.2**\n2a20b03 + 3a02b11 + a12b11 + 2b03b11 + 2a02b21 + 2a02a20b11T + 6a02b2\n2b3\n11T\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n11T\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nand from those values we can understand which types of bifurcation the system has.\n[29, 33] for more details) three more non-degeneracy conditions are involved\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\nIn particular (see\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\n= 0 there are two limit cycles that collide and disappear\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.4**\n= 0 a period doubled limit cycle born in this point.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nif a20 6\nif a02 6\nMoreover if a02\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n•\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.1**\nb11 < 0 a torus bifurcation occurs on the period doubled orbit, with Lyapunov coeﬃcient\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\nLN S =\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n2a2\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.4**r== 0.6**\n20b03 −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n−\nand so, in order to avoid degeneracy, we also assume LN S 6\nThe output given by MatCont is (b11, a20, a02, LN S).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n= 0.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.4**r== 0.3**\n3a02a30b11 + a20(a12b11 + 2b03b11 + 2a02b21)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nIn Figure 19 the four possible scenarios are reported depending on the sign of the normal form coeﬃcients.\n**BLOCK**fs== 4.7**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n4+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.2**l== 0.1**r== 0.9**\nJ+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nJ\n**BLOCK**fs== 5.3**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nF+\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nF+\n**BLOCK**fs== 3.7**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nF\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nF\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nP+\n**BLOCK**fs== 3.7**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nN S+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n4+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nJ+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nP\nLN S > 0\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nP+\n**BLOCK**fs== 3.7**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nF+\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 3.7**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nJ\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nF\n**BLOCK**fs== 4.4**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nN S\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.3**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\nP\nLN S < 0\n**BLOCK**fs== 5.0**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.6**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n(a) a20b11 < 0, a02b11 < 0\n**BLOCK**fs== 5.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nF+\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n4+\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nF+\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nP+\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nP+\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nF+\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nB\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nP\n**BLOCK**fs== 5.0**b== 0.4**t== 0.5**l== 0.3**r== 0.7**\nLN S > 0\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\nN S\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nF\n**BLOCK**fs== 5.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nF\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\nP\n**BLOCK**fs== 5.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\nLN S < 0\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n4+\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nN S+\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nB +\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\nF\n**BLOCK**fs== 5.1**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 3.5**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 5.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 5.9**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 6.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nP+\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 5.9**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nF+\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 4.1**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nP+\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nF+\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 4.1**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nP\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nF\n**BLOCK**fs== 5.9**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.9**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nF\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\n(b) a20b11 < 0, a02b11 > 0\n**BLOCK**fs== 5.8**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nF+\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.0**b== 0.4**t== 0.5**l== 0.7**r== 0.3**\nF\n**BLOCK**fs== 6.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nP\n**BLOCK**fs== 5.8**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 4.2**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 4.2**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nP+\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 5.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nP\n**BLOCK**fs== 5.8**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 6.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n(c) a20b11 > 0, a02b11 < 0\n**BLOCK**fs== 6.0**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\n(d) a20b11 > 0, a02b11 > 0\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.2**\nFigure 19: Bifurcation diagrams of a fold-ﬂip bifurcation of the ﬁxed point normal form.\n**BLOCK**fs== 14.3**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nC Kernels of some diﬀerential-diﬀerence operators\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.2**\nIn Section 4 we used the orthogonality with respect to the following inner product: if ζ1, ζ2 ∈ C\nand η1, η2 ∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nCn, then\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.8**r== 0.1**\n0([0, 1], Cn)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\nζ1\nη1 \n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.7**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.6**\nζ2\nη2 \n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\nζ1(t), ζ2(t)\ni\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n0 h\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\ndt +\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\nη1, η2i\nh\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n=\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nZ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.2**\n1 (t)ζ2(t)dt + ηH\nζH\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.8**r== 0.2**\n1 η2.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.2**\nIf this inner product vanishes, then we say that the corresponding vectors are orthogonal and write\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nζ1\nη1 \n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nζ2\nη2 \n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.2**\nIn Section 4 we used some propositions, from which we will give the proof in this appendix.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.4**\nProposition C.1. Consider two diﬀerential-diﬀerence operators\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n1([0, 1], Rn)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n0([0, 1], Rn)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nRn,\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nwith\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nIf ζ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n1([0, 1], Rn), then ζ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n∈\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n˙ζ\n−\nζ(0)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nT A(t)ζ\nζ(1)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n, φ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n˙ζ + T AT(t)ζ\nζ(1)\nζ(0)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.5**\n\nKer(φ1) if and only if\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nand ζ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nKer(φ2) if and only if\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1([0, 1], Rn)),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nφ2(\nC\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1([0, 1], Rn)).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nφ1(\nC\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nProof. We will focus on the ﬁrst assertion. If ζ is in the kernel of φ1, then ˙ζ\nFor all g\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\n1([0, 1], Rn) we have\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\nT A(t)ζ = 0 and ζ(0)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.8**r== 0.1**\nζ(1) = 0.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.4**\n0 gT(t) ˙ζ(t)dt\n0 T gT(t)A(t)ζ(t)dt = 0\n−\n\n0 ˙gT(t)ζ(t)dt\n\ngT(t)ζ(t)\n0 −\nR\nR\n|\ngT(1)ζ(1)\ngT(0)ζ(0)\nR\n−\ng(1))Tζ(0)\n(g(0)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nR\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\n0 T gT(t)A(t)ζ(t)dt = 0\n0 ( ˙g(t) + T AT(t)g(t))Tζ(t)dt = 0\n−\n\n0 ( ˙g(t) + T AT(t)g(t))Tζ(t)dt = 0\nR\nζ\nR\nζ(0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n= 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\n˙g + T AT(t)g\ng(1)\ng(0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n⇒\n⇒\n⇒ −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n⇒\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nConversely, assume that\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\nh\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n= 0 for all g\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\ni\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n∈ C\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\n1([0, 1], Rn). Then,\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.4**\n0 ζT(t)( ˙g(t) + T AT(t)g(t))dt + ζT(0)(g(0)\nζT(1)g(1)\nR\n(ζ(0)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n−\nζ(1))Tg(1)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.6**\nζT(0)g(0)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\n−\nT A(t)ζ(t))Tg(t)dt = 0.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.5**r== 0.3**\ng(1)) = 0\nT A(t)ζ(t))Tg(t)dt + ζT(0)(g(0)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\ng(1)) = 0\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n⇒\n⇒ −\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n\n˙g + T AT(t)g\ng(1)\ng(0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.5**\n0 ( ˙ζ(t)\n−\n\n0 ( ˙ζ(t)\nR\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nIf ˙ζ(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\nT A(t)ζ(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.4**\n= 0, then there exists a g(t) with g(1) = 0 such that\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n( ˙ζ(t)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nZ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nT A(t)ζ(t))Tg(t)dt\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n= 0.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nThis is impossible, so ˙ζ(t)\nthat ζ(0)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nζ(1) = 0. From both observations it follows that ζ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nThe proof of the second assertion is similar.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nKer(φ1).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.5**\nT A(t)ζ(t) = 0. Hence (ζ(0)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nζ(1))Tg(1) = 0 for all g; and thus there must hold\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nProposition C.2. Consider φ1,2 :\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n1([0, 1], Rn)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nC\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n→ C\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n0([0, 1], Rn)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.3**r== 0.6**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\n˙ζ\nT A(t)ζ\nζ(0) + ζ(1)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.5**\n\nKer(φ1) if and only if\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nIf ζ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\n1([0, 1], Rn), then ζ\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n∈\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.5**r== 0.4**\n, φ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nRn, where\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.6**r== 0.3**\n×\n˙ζ + T AT(t)ζ\nζ(0) + ζ(1)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nand ζ\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.2**r== 0.7**\nKer(φ2) if and only if\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n1([0, 1], Rn)),\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nφ2(\nC\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n1([0, 1], Rn)).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nφ1(\nC\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.4**\nProof. The proof is similar to the proof of Proposition C.1.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nProposition C.3. Consider φ1,2 :\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nC\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n→ C\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n˙ζ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nT A(t)ζ + iθζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nζ(1)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n, φ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n×\n˙ζ + T AT(t)ζ + iθζ\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nζ(0)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nζ(1)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nIf ζ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n1([0, 1], Cn), then ζ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.5**\nKer(φ1) if and only if\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nand ζ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nKer(φ2) if and only if\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n1([0, 1], Cn)),\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nφ2(\nC\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.4**\n1([0, 1], Cn)).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nφ1(\nC\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nProof. If ζ is in the kernel of φ1, then ˙ζ\nwe have\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.4**\nT A(t)ζ + iθζ = 0 and ζ(0)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nζ(1) = 0. For all g\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\n1([0, 1], Cn)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n∈ C\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n0 iθgH(t)ζ(t) = 0\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.3**r== 0.4**\n0 gH(t) ˙ζ(t)dt\n0 T gH(t)A(t)ζ(t)dt +\n−\n\n0 ˙gH(t)ζ(t)dt\ngH(t)ζ(t)\n\n0 −\nR\nR\n|\ngH(1)ζ(1)\ngH(0)ζ(0)\nR\n−\ng(1))Hζ(0)\n(g(0)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n−\n˙g + T AT(t)g + iθg\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.7**r== 0.2**\n0 iθgH(t)ζ(t) = 0\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.3**\n0 T gH(t)A(t)ζ(t)dt +\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\n0 ( ˙g(t) + T AT(t)g(t) + iθg(t))Hζ(t)dt = 0\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.3**\n−\n\n0 ( ˙g(t) + T AT(t)g(t) + iθg(t))Hζ(t)dt = 0\nR\n,\nR\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n= 0.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n⇒\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\ng(0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\ng(1)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n⇒\n⇒\n⇒ −\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nThe proofs of the reverse implication and the second assertion are similar.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nProposition C.4. Consider φ1,2 :\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\n1([0, 1], Cn)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\nC\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n→ C\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n0([0, 1], Cn)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n˙ζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.5**\nT A(t)ζ\ne−iθζ(1)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n, φ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nCn, where\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n×\n˙ζ + T AT(t)ζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\ne−iθζ(1)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nIf ζ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n1([0, 1], Cn), then ζ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n∈\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.5**\nKer(φ1) if and only if\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nand ζ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n∈\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\nKer(φ2) if and only if\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1([0, 1], Cn)),\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nφ2(\nC\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n1([0, 1], Cn)).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nφ1(\nC\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n⊥\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nProof. If ζ is in the kernel of φ1, then ˙ζ\nhave\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\ne−iθζ(1) = 0. For all g\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\n1([0, 1], Cn) we\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n∈ C\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nT A(t)ζ = 0 and ζ(0)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.4**\n−\n0 gH(t) ˙ζ(t)dt\n0 T gH(t)A(t)ζ(t)dt = 0\n\ngH(t)ζ(t)\n0 −\nR\nR\n|\ngH(1)ζ(1)\ngH(0)ζ(0)\nR\n−\n−\ne−iθg(1))Hζ(0)\n(g(0)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n−\n\n0 ˙gH(t)ζ(t)dt\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.5**\n˙g + T AT(t)g\ne−iθg(1)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\ng(0)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.3**\n0 T gH(t)A(t)ζ(t)dt = 0\n0 ( ˙g(t) + T AT(t)g(t))Hζ(t)dt = 0\nR\nR\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.2**\n0 ( ˙g(t) + T AT(t)g(t))Hζ(t)dt = 0\nζ\nR\nζ(0)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n= 0.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.3**r== 0.7**\n⇒\n⇒\n⇒ −\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n⇒\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nThe proofs of the reverse implication and the second assertion are similar.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.4**\nProposition C.5. Consider two diﬀerential-diﬀerence operators φ1,2 :\nwhere\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nC\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n×\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\n→ C\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.9**\nIf ζ\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n1([0, 1], Rn), then\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nif and only if\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.6**\n\n1([0, 1], Rn). Furthermore\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nfor all h\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n˙ζ\n−\nζ(0)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nT A(t)ζ\nζ(1)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\n, φ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n˙ζ + T AT(t)ζ\nζ(1)\nζ(0)\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\ng\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n˙h + T AT(t)h\nh(1)\nh(0)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ng\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nh\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nφ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\ng\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nif and only if\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nfor all h\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.2**r== 0.7**\n1([0, 1], Rn).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\n˙h\n−\nh(0)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nT A(t)h\nh(1)\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\ng\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nh\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nProof. We focus on the ﬁrst assertion. Suppose that ˙ζ(t)\nh\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n1([0, 1], Rn) we have\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nT A(t)ζ(t) = g(t) and ζ(0)\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.1**\nζ(1) = 0. For all\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n−\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n−\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.4**r== 0.4**\n0 T hT(t)A(t)ζ(t)dt =\n˙hT(t)ζ(t)dt\nR\n\nhT(0)ζ(0)\nR\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.3**r== 0.3**\n0 hT(t) ˙ζ(t)dt\n0 hT(t)g(t)dt\n0 T hT(t)A(t)ζ(t)dt =\nhT(t)ζ(t)\n\n0 −\nR\nR\n|\n0 ( ˙h(t) + T AT(t)h(t))Tζ(t)dt =\nhT(1)ζ(1)\nR\n−\n−\n0 ζT(t)( ˙h(t) + T AT(t)h(t))dt + ζT(0)(h(0)\nR\n˙h + T AT(t)h\nR\nh(1)\nh(0)\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n−\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nh\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\ng\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.6**r== 0.2**\n0 hT(t)g(t)dt\nR\nh(1)) =\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.7**r== 0.2**\n0 hT(t)g(t)dt\n\n0 gT(t)h(t)dt\nR\nR\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.2**r== 0.7**\n⇒\n⇒\n⇒\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n⇒\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\n\nThe proofs of the reverse implication and the second assertion are similar.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.4**\nProposition C.6. Consider two diﬀerential-diﬀerence operators φ1,2 :\nwhere\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n1([0, 1], Rn)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.6**r== 0.3**\nC\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\n0([0, 1], Rn)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\nRn,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.9**r== 0.1**\n×\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.7**r== 0.2**\n→ C\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n˙ζ\nT A(t)ζ\nζ(0) + ζ(1)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\n, φ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n˙ζ + T AT(t)ζ\nζ(0) + ζ(1)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nIf ζ\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n1([0, 1], Rn), then\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.8**\nif and only if\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nφ1(ζ) =\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\ng\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n˙h + T AT(t)h\nh(0) + h(1)\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\ng\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nh\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nh\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\n∀\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n∈ C\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\n1([0, 1], Rn). Furthermore\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nφ2(ζ) =\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\ng\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nif and only if\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\nh\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\n1([0, 1], Rn).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n˙h\nT A(t)h\nh(0) + h(1)\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\n=\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n−\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\ng\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\nh\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\nT A(t)ζ(t) = g(t) and ζ(0) + ζ(1) = 0. For all h\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n∈ C\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.4**\n0 T hT(t)A(t)ζ(t)dt =\n˙hT(t)ζ(t)dt\nR\n\nhT(0)ζ(0)\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.3**r== 0.2**\n−\n0 hT(t) ˙ζ(t)dt\n0 hT(t)g(t)dt\n\nhT(t)ζ(t)\n0 −\nR\n|\nhT(1)ζ(1)\nR\n−\n0 ζT(t)( ˙h(t) + T AT(t)h(t))dt + ζT(0)(h(0) + h(1)) =\n.\nR\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.3**\n0 hT(t)g(t)dt\n0 T hT(t)A(t)ζ(t)dt =\nR\n0 ( ˙h(t) + T AT(t)h(t))Tζ(t)dt =\nR\nR\n˙h + T AT(t)h\nh(0) + h(1)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.7**r== 0.2**\n0 hT(t)g(t)dt\n\n0 gT(t)h(t)dt\nR\nR\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\nζ\nζ(0)\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nh\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\ng\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n=\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n−\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\n,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n⇒\n⇒\n⇒\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n⇒\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\n\nThe proofs of the reverse implication and the second assertion are similar.\n**BLOCK**fs== 14.3**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nReferences\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n[1] CAPD: Computer assisted proofs in dynamics.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n[2] TIDES: A taylor integrator for diﬀerential equations.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n[3] V. I. Arnol’d. Geometrical Methods in the Theory of Ordinary Diﬀerential Equations. Springer-Verlag,\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nNew York, 1983.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n[4] U. M. Ascher, R. M. M. Mattheij, and R. D. Russell. Numerical solution of boundary value problems\nfor ordinary diﬀerential equations, volume 13 of Classics in Applied Mathematics. Society for Industrial\nand Applied Mathematics (SIAM), Philadelphia, PA, 1995. Corrected reprint of the 1988 original.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n[5] R. Barrio. Sensitivity analysis of ode’s/dae’s using the taylor series method. SIAM J. Sci. Comput.,\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n27(6):1929–1947, 2006.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.1**\n[6] W. Beyn, A. Champneys, E. J. Doedel, W. Govaerts, Yu. A. Kuznetsov, and B. Sandstede. Numerical\ncontinuation and computation of normal forms. In B. Fiedler, editor, Handbook of Dynamical Systems,\nvolume 2, pages 149–219. American Mathematical Society, Burlington, MA, 3rd edition, 2002.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n[7] S.-N. Chow and D. Wang. Normal forms of bifurcating periodic orbits. In Multiparameter bifurcation\ntheory (Arcata, Calif., 1985), volume 56 of Contemp. Math., pages 9–18. Amer. Math. Soc., Providence,\nRI, 1986.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n[8] H. Dankowicz and G. Thakur. A Newton method for locating invariant tori of maps. Internat. J. Bifur.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\nChaos Appl. Sci. Engrg., 16(5):1491–1503, 2006.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n[9] C. De Boor and B. Swartz. Collocation at gaussian points. SIAM J. Numer. Anal., 10(4):582–606,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n1973.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n[10] A. Dhooge, W. Govaerts, and Yu. A. Kuznetsov. MATCONT: A MATLAB package for numerical\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.2**r== 0.3**\nbifurcation analysis of ODEs. ACM Trans. Math. Software, 29(2):141–164, 2003.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n[11] A. Dhooge, W. Govaerts, Yu. A. Kuznetsov, W. Mestrom, and A. M. Riet. CL matcont: A continuation\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\ntoolbox in Matlab. In Symposium on Applied Computing, Melbourne, Florida, 2003. ACM.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n[12] L. Dieci, J. Lorenz, and R. D. Russell. Numerical calculation of invariant tori. SIAM J. Sci. Statist.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nComput., 12(3):607–647, 1991.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n[13] E. Doedel, H. B. Keller, and J. P. Kernevez. Numerical analysis and control of bifurcation problems\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.2**\n(II): Bifurcation in inﬁnite dimensions. Int. J. Bifurcation and Chaos, 1(4):745–772, 1991.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n[14] E. J. Doedel, A. R. Champneys, T. F. Fairgrieve, Yu. A. Kuznetsov, B. Sandstede, and X. J. Wang.\nauto97: Continuation and bifurcation software for ordinary diﬀerential equations (with HomCont),\n1997.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n[15] E. J. Doedel, W. Govaerts, and Yu. A. Kuznetsov. Computation of periodic solution bifurcations in\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nODEs using bordered systems. SIAM J. Numer. Anal., 41(2):401–435.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n[16] C. Elphick, G. Iooss, and E. Tirapegui. Normal form reduction for time-periodically driven diﬀerential\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.5**\nequations. Phys. Lett. A, 120(9):459–463, 1987.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n[17] C. Elphick, E. Tirapegui, M. E. Brachet, P. Coullet, and G. Iooss. A simple global characterization for\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.3**\nnormal forms of singular vector ﬁelds. Phys. D, 29(1-2):95–127, 1987.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n[18] W. Govaerts, R. Khoshsiar Ghaziani, Yu. A. Kuznetsov, and H. G. E. Meijer. Numerical methods for\ntwo-parameter local bifurcation analysis of maps. SIAM J. Sci. Comput., 29(6):2644–2667, 2007.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n[19] A. Griewank, D. Juedes, and J. Utke. Algorithm 755: ADOL-C: A package for the automatic diﬀeren-\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.2**r== 0.2**\ntiation of algorithms written in C/C++. ACM Trans. Math. Software, 22(2):131–167, 1996.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n[20] J. Guckenheimer and P. Holmes. Nonlinear Oscillations, Dynamical Systems and Bifurcations of Vector\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nFields. Springer-Verlag, New York, 1983.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n[21] J. Guckenheimer and B. Meloon. Computing periodic orbits and their bifurcations with automatic\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\ndiﬀerentiation. SIAM J. Sci. Comput., 22(3):951–985, 2000.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\n[22] G. Iooss. Bifurcation of maps and applications, volume 36 of North-Holland Mathematics Studies.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nNorth-Holland Pub. Co., Amsterdam, 1979.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n[23] G. Iooss. Global characterization of the normal form for a vector ﬁeld near a closed orbit. J. Diﬀerential\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nEquations, 76(1):47–76, 1988.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.1**\n[24] G. Iooss and M. Adelmeyer. Topics in bifurcation theory and applications, volume 3 of Advanced Series\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.2**r== 0.3**\nin Nonlinear Dynamics. World Sci. Pub. Co. Inc., River Edge, New York, 1992.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n[25] G. Iooss and D. D. Joseph. Elementary stability and bifurcation theory. Springer-Verlag, New York,\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n1990.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n[26] I. G. Kevrekidis, R. Aris, L. D. Schmidt, and S. Pelikan. Numerical computation of invariant circles of\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nmaps. Phys. D, 16(2):243–251, 1985.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\n[27] B. Krauskopf. Bifurcation sequences at 1 : 4 resonance: an inventory. Nonlinearity, 7(3):1073–1091,\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n1994.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n[28] Yu. A. Kuznetsov. Numerical normalization techniques for all codim 2 bifurcations of equilibria in\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.4**\nODEs. SIAM J. Numer. Anal., 36(4):1104–1124, 1999.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n[29] Yu. A. Kuznetsov. Elements of Applied Bifurcation Theory. Springer-Verlag, New York, 2004. 3rd ed.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.1**\n[30] Yu. A. Kuznetsov, W. Govaerts, E. J. Doedel, and A. Dhooge. Numerical periodic normalization for\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\ncodim 1 bifurcations of limit cycles. SIAM J. Numer. Anal., 43(4):1407–1435, 2005.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n[31] Yu. A. Kuznetsov and V. V. Levitin. Content: A multiplatform environment for analyzing dynamical\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.7**\nsystems, 1997.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.1**\n[32] Yu. A. Kuznetsov and H. G. E. Meijer. Numerical normal forms for codim 2 bifurcations of ﬁxed points\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.2**r== 0.2**\nwith at most two critical eigenvalues. SIAM J. Sci. Comput., 26(6):1932–1954, 2005.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n[33] Yu. A. Kuznetsov, H. G. E. Meijer, and L. van Veen. The fold-ﬂip bifurcation. Int. J. Bifurcation and\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.6**\nChaos, 14(7):2253–2282, 2004.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\n[34] Yu. A. Kuznetsov, S. Muratori, and S. Rinaldi. Bifurcations and chaos in a periodic predator-prey\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.3**\nmodel. Internat. J. Bifur. Chaos Appl. Sci. Engrg., 2(1):117–128, 1992.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n[35] Y. Lan, C. Chandre, and P. Cvitanovic. Newton’s descent method for the determination of invariant\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\ntori. Phys. Rev. E, 74(4, Part 2), 2006.\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\n[36] E. N. Lorenz. Irregularity: a fundamental property of the atmosphere. Tellus, 36(A):98–110, 1984.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\n[37] B. Rasmussen. Numerical methods for the continuation of invariant tori. ProQuest LLC, Ann Arbor,\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nMI, 2003. Thesis (Ph.D.)–Georgia Institute of Technology.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n[38] B. Rasmussen and L. Dieci. A geometrical method for the approximation of invariant tori. J. Comput.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.6**\nAppl. Math., 216(2):388–412, 2008.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n[39] F. Schilder, H. M. Osinga, and W. Vogt. Continuation of quasi-periodic invariant tori. SIAM J. Appl.\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.2**r== 0.5**\nDyn. Syst., 4(3):459–488 (electronic), 2005.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n[40] A. Shilnikov, G. Nicolis, and C. Nicolis. Bifurcation and predictability analysis of a low-order atmo-\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.3**\nspheric circulation model. Int. J. Bifurcation and Chaos, 5(6):1701–1711, 1995.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.1**\n[41] C. G. Steinmetz and R. Larter. The quasiperiodic route to chaos in a model of the peroxidase-oxidase\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nreaction. J. Chem. Phys., 94(2):1388–1396, 1991.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\n[42] G. Torrini, R. Genesio, and A. Tesi. On the computation of characteristic multipliers for predicting\nlimit cycle bifurcations. Chaos Solitons Fractals, 9(1-2):121–133, 1998. Applications of chaos (Inow lodz,\n1996).",
         "extracted",
         "Numerical Methods for Two-Parameter Local Bifurcation Analysis of Maps;Newton's descent method for the determination of invariant tori.;A Newton Method for Locating Invariant Tori of Maps;Sensitivity Analysis of ODES/DAES Using the Taylor Series Method;Numerical Normal Forms for Codim 2 Bifurcations of Fixed Points with at Most Two Critical Eigenvalues;Numerical Methods for the Continuation of Invariant Tori;MATCONT: A MATLAB package for numerical bifurcation analysis of ODEs;Cl_matcont: a continuation toolbox in Matlab;Computation of Periodic Solution Bifurcations in ODEs Using Bordered Systems;Chapter 4 – Numerical Continuation, and Computation of Normal Forms;Computing Periodic Orbits and their Bifurcations with Automatic Differentiation;Numerical Normalization Techniques for All Codim 2 Bifurcations of Equilibria in ODE's;Topics in bifurcation theory and applications;Elements of applied bifurcation theory (2nd ed.);Harmonic balance analysis of period-doubling bifurcations with implications for control of nonlinear dynamics;Algorithm 755: ADOL-C: a package for the automatic differentiation of algorithms written in C/C++;Bifurcation and predictability analysis of a low-order atmospheric circulation model;Models of complex dynamics in nonlinear systems;Bifurcation sequences at 1:4 resonance: an inventory;Harmonic balance methods for the analysis of chaotic dynamics in nonlinear systems;BIFURCATIONS AND CHAOS IN A PERIODIC PREDATOR-PREY MODEL;NUMERICAL ANALYSIS AND CONTROL OF BIFURCATION PROBLEMS (II): BIFURCATION IN INFINITE DIMENSIONS;Numerical Calculation of Invariant Tori;THE QUASIPERIODIC ROUTE TO CHAOS IN A MODEL OF THE PEROXIDASE-OXIDASE REACTION;Global Characterization of the Normal Form for a Vector Field Near a Closed Orbit;A simple global characterization for normal forms of singular vector fields;Normal form reduction for time-periodically driven differential equations;Numerical computation of invariant circles of maps;NONLINEAR OSCILLATIONS, DYNAMICAL SYSTEMS, AND BIFURCATIONS OF VECTOR FIELDS (Applied Mathematical Sciences, 42);Irregularity: a fundamental property of the atmosphere*;Nonlinear Oscillations, Dynamical Systems, and Bifurcations of Vector Fields;Collocation at Gaussian Points;Continuation of Quasi-periodic Invariant Tori;The fold-flip bifurcation;On the computation of characteristic multipliers for predicting limit cycle bifurcations;Elements of applied bifurcation theory;AUTO 2000 : CONTINUATION AND BIFURCATION SOFTWARE FOR ORDINARY DIFFERENTIAL EQUATIONS (with HomCont);CONTENT: A multiplatform environment for analyzing dynamical systems;Normal forms of bifurcating periodic orbits;Numerical computation of periodic solution branches and oscillatory dynamics of the stirred tank reactor with A → B → C reactions;Geometrical Methods in the Theory of Ordinary Differential Equations;Elementary stability and bifurcation theory;Bifurcation of maps and applications;Topics in bifurcation theory;Numerical solution of boundary value problems in ordinary differential equations;Numerical Continuation, And Computation Of Normal Forms;CAPD: Computer assisted proofs in dynamics",
         "Numerical Periodic Normalization for Codim 1 Bifurcations of Limit Cycles",
         "None"
        ],
        [
         "39",
         "002dea70291555c99aa2d88b87db7e8d3ccad66a",
         "None",
         "E. Petroulakis,Y. Mamane,O. Bacquer,D. Shahbazian,Nahum Sonenberg",
         "https://www.nature.com/articles/6602902.pdf",
         "\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nBritish Journal of Cancer (2006) 94, 195 – 199\n& 2006 Cancer Research UK All rights reserved 0007 – 0920/06 $30.00\n**BLOCK**fs== 19.9**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nMinireview\nmTOR signaling: implications for cancer and anticancer therapy\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nE Petroulakis1,2, Y Mamane1,2, O Le Bacquer1, D Shahbazian1 and N Sonenberg*,1\n1Department of Biochemistry, McGill Cancer Centre, McGill University, 3655 Promenade Sir William Osler, Montreal, QUE, Canada H3G 1Y6\n**BLOCK**fs== 9.2**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nMounting evidence links deregulated protein synthesis to tumorigenesis via the translation initiation factor complex eIF4F.\nComponents of this complex are often overexpressed in a large number of cancers and promote malignant transformation in\nexperimental systems. mTOR affects the activity of the eIF4F complex by phosphorylating repressors of the eIF4F complex, the eIF4E\nbinding proteins. The immunosuppressant rapamycin specifically inhibits mTOR activity and retards cancer growth.\nImportantly,\nmutations in upstream negative regulators of mTOR cause hamartomas, haemangiomas, and cancers that are sensitive to rapamycin\ntreatment. Such mutations lead to increased eIF4F formation and consequently to enhanced translation initiation and cell growth.\nThus, inhibition of translation initiation through targeting the mTOR-signalling pathway is emerging as a promising therapeutic option.\nBritish Journal of Cancer (2006) 94, 195 – 199. doi:10.1038/sj.bjc.6602902 www.bjcancer.com\nPublished online 13 December 2005\n& 2006 Cancer Research UK\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nKeywords: translational control; eIF4F; eIF4E binding proteins; rapamycin; mTOR; malignant transformation\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nRapamycin (Rapamunes, Wyeth Ayerst) is a specific inhibitor of\nthe target of rapamycin (TOR) (for review see Hay and Sonenberg,\n2004). The protein kinase TOR genes (TOR1 and TOR2) were first\nidentified in the early 1990s in a screen for rapamycin-resistant\nyeast mutants (Heitman et al, 1991). The mammalian orthologue\n(mTOR),\nis also referred to as FKBP12-rapamycin associated\nprotein (FRAP), rapamycin and FKBP12 target (RAFT), rapamycin\ntarget (RAPT), or sirolimus effector protein (SEP). Rapamycin\nforms a complex with the immunophilin FK506 binding protein-12\n(FKBP12), which binds to the FKBP12-rapamycin binding (FRB)\ndomain of mTOR and inhibits its kinase activity. Rapamycin\ninhibits the growth of a broad spectrum of cancers including\nrhabdomyosarcoma, neuroblastoma, glioblastoma, small cell lung\ncarcinoma, osteosarcoma, pancreatic cancer,\nleukaemia, B-cell\nlymphoma, and breast and colon cancer-derived cells (Huang and\nHoughton, 2002). The rapamycin analogues, CCI-779 (Wyeth-\nAyerst, PA, USA), RAD001 (Novartis, Switzerland), and AP23573\n(Ariad Pharmaceuticals, MA, USA) have shown promise in clinical\ntrials (Panwalkar et al, 2004).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nmTOR functions by integrating extracellular signals (growth\nfactors and hormones), with amino-acid availability and intra-\ncellular energy status to control translation rates and additional\nmetabolic processes (Hay and Sonenberg, 2004). mTOR enhances\ntranslation initiation in part by phosphorylating two major targets,\nthe eIF4E binding proteins (4E-BPs) and the ribosomal protein S6\nkinases (S6K1 and S6K2) that cooperate to regulate translation\ninitiation rates.\n**BLOCK**fs== 8.7**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n*Correspondence: Dr N Sonenberg;\nE-mail: nahum.sonenberg@mcgill.ca\n2 These authors contributed equally to this article.\nReceived 12 August 2005; revised 10 November 2005; accepted 14\nNovember 2005; published online 13 December 2005\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nTranslation initiation is implicated in tumorigenesis All nuclear\ntranscribed eukaryotic mRNAs contain a ‘cap’ structure, m7GpppN\n(where ‘N’ is any nucleotide and ‘m’ is a methyl group), at the 50\nterminus. The ‘cap’ is specifically bound by the initiation factor\ninitiation factors,\neIF4E that associates with two additional\neIF4G (a large scaffolding protein) and eIF4A (a helicase which\nis believed to unwind mRNA 50 secondary structures), to form the\neIF4F complex, which facilitates the recruitment of ribosomes to\nthe mRNA (Gingras et al, 1999).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nSeveral studies demonstrated that eIF4E acts as an oncogene (for\na recent review see Mamane et al, 2004). eIF4E levels are limiting\nfor cap-dependent translation in most systems and overexpression\nof eIF4E in rodent cells causes malignant transformation (Lazaris-\nKaratzas et al, 1990; Mamane et al, 2004). Therefore, increased\neIF4E expression in cancer cells is thought to enhance eIF4F\ncomplex formation and as a consequence, the translation of a\nsubset of mRNAs that contain highly structured 50 untranslated\nregions (UTRs), such as vascular endothelial growth factor (VEGF)\nand ornithine decarboxylase (ODC). Indeed, eIF4E is elevated in\nnumerous types of cancers, including bronchioalveolar, bladder,\nhead and neck, liver, colon, and breast cancers (De Benedetti and\nGraff, 2004). Consistent with this, two important studies demon-\nstrated that overexpression of eIF4E in mice promotes lympho-\nmagenesis (Ruggero et al, 2004; Wendel et al, 2004). Furthermore,\nthe importance of eIF4E’s role in tumorigenesis is reinforced by\nthe finding that eIF4F complex is necessary for maintaining\ntumour cell growth (Avdulov et al, 2004). Importantly, siRNA\ntreatment to reduce eIF4E expression inhibits the growth of several\ncell lines including those of head and neck squamous carcinoma\ncells (Oridate et al, 2005).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nOther components of the eIF4F complex are also implicated in\nmalignant transformation. eIF4GI can transform NIH 3T3 cells,\nand is overexpressed in squamous cell lung carcinomas and breast\nlines (see Mamane et al (2004)). eIF4A is also\ncancer cell\nlines and in primary\noverexpressed in human melanoma cell\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nmTOR signaling: implications for cancer and anticancer therapy\nE Petroulakis et al\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nhepatocellular carcinomas (see Mamane et al (2004)). Hence, these\nfindings reinforce a molecular model where increased eIF4F\ncomplex promotes tumorigenesis.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nRapamycin inhibits cap-dependent translation Growth inhibition\nby rapamycin is thought to be partially mediated by the inhibition\nof cap-dependent translation (Hay and Sonenberg, 2004). The\nmammalian 4E-BPs constitute a three-member family of transla-\ntional inhibitors that bind to eIF4E, inhibit formation of the eIF4F\ncomplex and thus cap-dependent translation (Gingras et al, 1999).\nThe 4E-BPs are dephosphorylated as a consequence of rapamycin\ntreatment. Binding of 4E-BP1 (the best studied 4E-BP family\nmember) to eIF4E is regulated by its phosphorylation status: the\nhypophosphorylated form (residues Thr37 and 46) of 4E-BP1\nbinds very tightly to eIF4E and inhibits cap-dependent translation.\nIn response to stimuli such as growth factors, hormones, nutrients,\nor increased energy levels, mTOR phosphorylates 4E-BP1 primarily\nat residues Ser65 and Thr73. As a result, eIF4E is released from 4E-\nBP1 with a subsequent increase in cap-dependent translation\n(Mamane et al, 2004).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nSeveral studies addressed the role of the 4E-BPs in tumour\nsuppression. For example, 4E-BP overexpression counteracts\neIF4E-, src-, or ras-mediated cellular transformation (Mamane\net al, 2004). Also, a nonphosphorylatable mutant of 4E-BP1, which\nconstitutively binds eIF4E, inhibits breast cancer cell growth more\nstrongly than wild-type 4E-BP1 (Avdulov et al, 2004). In addition,\nthe 4E-BPs appear to mediate rapamycin-sensitivity since rhabdo-\nmyosarcoma cells expressing very low levels of 4E-BP1 are\nrefractory to growth inhibition by rapamycin (Dilling et al,\n2002). Thus, the 4E-BPs are considered to be important mTOR\ntargets that modulate cancer cell growth through a mechanism that\ninvolves cap-dependent translation.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nRibosomal protein S6 kinase The ribosomal protein S6 kinases\nare also important substrates of mTOR. S6K1 (p70S6K; the better\ncharacterised S6 kinase) is implicated in the positive regulation\nof cell growth and proliferation (Fumagalli and Thomas, 2000).\nInterestingly, S6K1 activation correlates with enhanced translation\nof a subset of mRNAs that contain a terminal 50 oligopyrimidine\ntract (TOP mRNAs) (Fumagalli and Thomas, 2000). These mRNAs\nencode ribosomal proteins, elongation factors, the poly-A binding\nprotein and other components of the translational machinery\nthat become selectively translated in response to growth factors.\nHowever, TOP mRNA translation remains intact even when both\nS6K1 and S6K2 genes are genetically disrupted in mice (Pende\net al, 2004), indicating that S6Ks are not essential for the regulation\nof TOP mRNA translation.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nS6K regulates mTOR through a negative feedback signalling\npathway that affects insulin receptor substrate-1 (IRS-1) (Figure 1).\nS6K was shown to directly phosphorylate IRS-1 to inhibit\nphospohatidylinositol-3-kinase\nactivation\n(Harrington et al, 2004). S6K activation decreases IRS-1 expres-\nIRS-1 expression\nsion, while rapamycin treatment\n(Harrington et al, 2004). Also, deletion of S6K1 in mice renders\nthem resistant to age- and high-fat diet-induced obesity while\nenhancing insulin sensitivity (Um et al, 2004). Lastly, Akt\nactivation in response to insulin is enhanced in S6K-null mice,\nthus implicating S6K1 in diabetes and obesity (Um et al, 2004).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\nand Akt\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.3**r== 0.6**\nrestores\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n(PI3K)\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nOther translational targets of mTOR Other rapamycin-sensitive\ntargets involved in translational regulation exist. The translation\ninitiation factor eIF4B, which stimulates the activity of eIF4A in\nunwinding duplex RNA, is phosphorylated by S6K1 on serine 422\n(Raught et al, 2004). mTOR also mediates the phosphorylation of\nserines 1108, 1148, and 1192 in the C-terminus of eIF4G (Raught\net al, 2000). In addition, mTOR regulates the elongation stage of\nprotein synthesis by modulating elongation factor (eEF2) kinase\n(Proud, 2004).\n**BLOCK**fs== 6.0**b== 0.8**t== 0.1**l== 0.7**r== 0.3**\nInsulin\ngrowth factors\nhormones\n**BLOCK**fs== 6.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nWortmannin\nLY294002\n**BLOCK**fs== 6.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nIRS-1\n**BLOCK**fs== 6.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nEnergy\nstatus\n**BLOCK**fs== 6.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nPI3K\n**BLOCK**fs== 6.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nPDK1\n**BLOCK**fs== 6.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nPTEN\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nLKB1\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nAkt\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.8**r== 0.1**\nmTOR\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nrictor\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\nGL\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nAMPK\n**BLOCK**fs== 6.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\nAMP/ATP\nMetformin\nAICAR\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nTSC2\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nTSC1\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nGTP\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\nRheb\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nmTOR\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nraptor\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\nGL\n**BLOCK**fs== 6.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\nAmino\nacids\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.8**r== 0.1**\nRapamycin\nCCI779\nRAD001\nAP23573\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n4E-BPs\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\neIF4G\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nS6K\n**BLOCK**fs== 6.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\neIF4E\n**BLOCK**fs== 6.0**b== 0.5**t== 0.4**l== 0.7**r== 0.2**\neIF4B\n**BLOCK**fs== 6.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\nrpS6\n**BLOCK**fs== 6.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\nCap-dependent\ntranslation\ninitiation\n**BLOCK**fs== 6.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nTranslation initiation\n**BLOCK**fs== 6.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n?\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nFigure 1 mTOR signaling to translation initiation. mTOR signaling\nregulates translation initiation by integrating several different inputs. Insulin,\nhormones and growth factors activate the PI3K/Akt signaling cascade. Akt\nreceives inputs from PDK1 and the rictor/GbL/mTOR complex. Energy\nstatus (the AMP/ATP ratio) modulates AMPK activity. These pathways\naffect TSC1/2 to regulate mTOR activity via Rheb. The raptor/GbL/mTOR\ncomplex mediates the phosphorylation of 4E-BP and S6K. Pharmacological\ninhibition of PI3K (wortmannin or LY294002) and mTOR (rapamycin or its\nanalogs), or activation of AMPK (by increased AMP/ATP, metformin or\nAICAR) are indicated.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nPI3K/Akt/mTOR signaling Growth factors, mitogens and hor-\nmones activate the PI3K signaling pathway and consequently\nmTOR (Hay and Sonenberg, 2004) (Figure 1). Nutrients (amino\nacids, glucose) also regulate mTOR activity to affect translation\n(Proud, 2004). PI3K phosphorylates phosphatidylinositol-4,5-bi-\nsphosphate (PIP2)\nto generate phosphatidylinositol-3,4,5-tri-\nphosphate (PIP3). PIP3 binds and activates pleckstrin homology\n(PH) domain-bearing proteins such as Akt and the phosphoinosi-\ntide dependent kinase (PDK) family members. PDK1 and the Ser/\nThr kinase Akt are recruited to the membrane where PDK1\nphosphorylates and activates Akt. In turn, Akt phosphorylates the\ntuberous sclerosis complex, TSC1/TSC2 (hamartin/tuberin) that\nserves as a GTPase activating protein (GAP) for the small G\nprotein, Ras homolog enriched in brain (Rheb). Rheb, in its GTP-\nbound state, can activate mTOR.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nNegative regulators of mTOR are implicated in tumour\nsuppression\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nPTEN PTEN is a lipid phosphatase that acts on the lipid substrate\nPIP3 to convert it to PIP2. PIP3 accumulation results from PTEN-\ninactivating mutations to effect cell size, adhesion, motility, and\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nangiogenesis (Inoki et al, 2005a). PTEN is frequently mutated in\nmany cancers and in a group of cancer-like syndromes including\nCowden, Lhermitte – Duclos, Bannayan – Zonana, and Proteus\nsyndromes that are characterised by the emergence of hamartomas\n(Inoki et al, 2005a). A critical outcome of PTEN inactivation is an\nincrease in mTOR activity, resulting in the phosphorylation of 4E-\nBPs and S6Ks. Strikingly, PTEN/ cells are hypersensitive to\ngrowth inhibition by rapamycin and CCI-779 (Guertin and\nSabatini, 2005).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nAMP-activated protein kinase (AMPK) mTOR activity becomes\nrepressed under conditions of energy deprivation, in which an\nelevated AMP/ATP intracellular ratio causes activation of AMPK.\nStressful conditions, such as nutrient deprivation, hypoxia, heat\nshock, and ischaemia diminish cellular energy reserves. The link\nbetween mTOR inhibition and AMPK was first demonstrated with\nthe use of AICAR (50-phosphoribosyl-5-aminoimidazole-4-carbox-\namide ribonucleoside), an AMPK activator, thus linking the amino\nacid- and energy-sensing functions of mTOR. Metformin, an\nantidiabetic drug, also activates AMPK and is hypothesised to\nreduce the risk of cancer in patients with type 2 diabetes (Evans\net al, 2005). Thus,\ntargeting AMPK may be an interesting\ntherapeutic option for cancer therapy.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nLKB1 LKB1 is a serine/threonine kinase that phosphorylates and\nactivates AMPK. LKB1 is frequently mutated in Peutz-Jeghers\nsyndrome (PJS), an autosomal dominant inherited cancer predis-\nposing patients to the development of tumours mainly throughout\nthe gastrointestinal tract. Disruption of LKB1 in mice causes\ngastrointestinal and hepatic polyps (Bardeesy et al, 2002). In\nLKB1-deficient cells, phosphorylation AMPK is defective and cells\nresist mTOR inactivation in response to AICAR (Shaw et al, 2004).\nImportantly, 4E-BP1 is less enriched in eIF4E-bound cap analogue\nprecipitates from LKB1-deficient cells (Shaw et al, 2004). This is\nindicative of enhanced eIF4F formation as a result of increased\nmTOR signaling caused by LKB1-deficiency. LKB1-mediated\ninhibition of mTOR also involves the TSC1/2 complex,\nthus\ndemonstrating the link between LKB1 and mTOR through the\nregulation of AMPK and TSC1/2 activities (Shaw et al, 2004).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nTSC1/TSC2 TSC1 and TSC2 are mutated in tuberous sclerosis\ncomplex (TSC) patients. In response to low energy levels, TSC2\nbecomes phosphorylated by AMPK, resulting in increased stability\nof the TSC1/2 complex. The stabilised TSC1/2 complex inhibits\nmTOR and protects cells\nfrom energy deprivation-induced\napoptosis (see recent reviews by (Inoki et al, 2005b; Kwiatkowski\nand Manning, 2005)). TSC2 is also phosphorylated by Akt leading\nto its inactivation and consequently to increased mTOR activity.\nFurthermore, S6K phosphorylation is elevated in TSC/ cells\nand is rapidly abrogated by rapamycin treatment. Activation of the\nextracellular signal-regulated kinase (Erk) by the Ras signaling\npathway also leads to TSC1/2 inactivation through phosphoryla-\ntion of TSC2 on Ser664 (Ma et al, 2005). Taken together, the\nTSC1/2 complex functions as a key player in the regulation of the\nmTOR pathway by receiving inputs from PI3K/PTEN/Akt and Ras\nsignalling pathways, energy levels, and amino acids to regulate\ntranslation initiation and affect cell growth and proliferation.\n**BLOCK**fs== 10.5**b== 0.1**t== 0.8**l== 0.1**r== 0.6**\nmTOR COMPLEXES REGULATE RAPAMYCIN\nSENSITIVITY\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nmTOR signaling: implications for cancer and anticancer therapy\nE Petroulakis et al\n**BLOCK**fs== 9.0**b== 0.6**t== 0.1**l== 0.5**r== 0.1**\ngrowth 1) (reviewed in Kim and Sabatini (2004), Lorberg and Hall\n(2004)). Raptor interacts with mTOR and tethers it\nto its\ndownstream effectors through a TOS (TOR signaling) motif found\nin the mTOR substrates, 4E-BP1 and S6K1 (amino acid sequences:\nFEMDI and FDIDL, respectively) (reviewed in Proud (2004)). The\nraptor/mTOR interaction is nutrient-sensitive and is dependent on\nthe presence of another partner, GbL (a mammalian ortholog of\nyeast Lst8p) (Guertin and Sabatini, 2005). GbL/mLst8 interacts\nwith the mTOR kinase domain independently of raptor and\npotentiates mTOR activity. Rictor (rapamycin insensitive compo-\nnent of TOR; the yeast AVO3 ortholog), is involved in the control\nof cytoskeleton organisation (Lorberg and Hall, 2004; Guertin and\nSabatini, 2005). Unlike raptor/GbL/mTOR (mTORC1 complex),\nthe rictor/mTOR (mTORC2 complex) interaction is insensitive to\nrapamycin treatment. Most importantly, the rictor/mTOR complex\nis required for Akt phosphorylation on Ser473 to achieve its\nfull activation (Guertin and Sabatini, 2005). Although the activity\nthe rictor/mTOR complex is insensitive to short periods\nof\nof rapamycin treatment, it has been suggested that long-term\nexposure to rapamycin could prevent newly-synthesised mTOR\nmolecules from associating with rictor, thereby preventing rictor/\nmTOR-mediated Akt phosphorylation (Guertin and Sabatini,\n2005). This model predicts that rictor could serve as a new\ntherapeutic target in mTOR-linked hamartoma syndromes (such\nas tuberous sclerosis) and PTEN-mutated cancers, where rapa-\nmycin therapy has already proven beneficial.\n**BLOCK**fs== 10.5**b== 0.5**t== 0.5**l== 0.5**r== 0.2**\nRAPAMYCIN IN COMBINATION THERAPY\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\ninhibitor,\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nIn Peutz Jeghers syndrome, Tuberous Sclerosis, and other diseases\nwhere PTEN is inactivated, the use of rapamycin as a clinical\nmeans to reverse the effect of elevated mTOR activity is an\nattractive option (Inoki et al, 2005a). These diseases are distinct\nfrom other hamartoma-associated disorders\n(such as VHL\nsyndrome) since they have an established molecular link to mTOR\n(Inoki et al, 2005a). Earlier studies demonstrated that PTEN-\ninactivated tumour cells exhibit enhanced sensitivity to the\nrapamycin analog CCI-779 (Guertin and Sabatini, 2005). More\nrecently, several studies have shown that rapamycin treatment,\nin combination with other chemotherapeutic drugs, can lead to\nenhanced selective killing of cancer cells. In particular, the protein\ntyrosine kinase (PTK)\nImatinib (Gleevec, STI571)\nsynergises with rapamycin to inhibit BCR/ABL transformed cells\n(Mohi et al, 2004). The effect of rapamycin may be enhanced as a\nresult of Imatinib-induced Akt/mTOR signaling, a complication\nthat is thought to lead to Imatinib resistance (Burchert et al, 2005).\nRapamycin can also synergise with paclitaxel, carboplatin, and\nvinorelbine to induce apoptosis in breast cancer cells (Mondesire\net al, 2004). Cisplatin-induced apoptosis of A549 lung cancer cells\nis also significantly enhanced when combined with RAD001\n(Beuvink et al, 2005). This could be in part due to reduced\ntranslation of p53-activated p21 mRNA in A549 and MCF7 cells\ntreated with RAD001, thereby allowing the dosage of cisplatin to be\nreduced (Beuvink et al, 2005). Also, the use of the EGFR/VEGFR\ninhibitor, AEE788, in combination with RAD001 greatly decreased\ntumour growth in glioma xenografts (Goudar et al, 2005).\nFurthermore, targeting the glycolytic pathway in combination\nwith mTOR inhibition may also be useful in cases where DNA-\ndamaging agents are less efficient\nin inhibiting growth and\npromoting apoptosis of cancer cells (Xu et al, 2005).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nRecent findings shed new light on the mechanisms of mTOR\nsensitivity to rapamycin. Three key interacting partners of mTOR\n(raptor, GbL, and rictor) modulate many of mTOR’s rapamycin-\nsensitive and insensitive functions. These evolutionarily conserved\nproteins have been characterised in yeast and mammalian cells.\nRaptor is the mammalian ortholog of yeast Kog1 (Kontroller of\n**BLOCK**fs== 10.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nTARGETING THE TRANSLATIONAL APPARATUS AS\nA THERAPEUTIC APPROACH\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nA critical outcome of mTOR activation is the phosphorylation of\nseveral components of the translational apparatus, which mediate\ntranslation initiation and cell proliferation. A major rate-limiting\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nmTOR signaling: implications for cancer and anticancer therapy\nE Petroulakis et al\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nstep in translation initiation is the formation of the eIF4F complex\nat the 50 mRNA cap structure. Rapamycin inhibits eIF4F formation\nlargely in part through 4E-BP dephosphorylation. Several studies\nhave implicated the eIF4E/4E-BP pathway as a putative down-\nstream target of mTOR in tumorigenesis (Avdulov et al, 2004;\nRuggero et al, 2004; Wendel et al, 2004). Inhibition of mTOR-\nmediated cap-dependent translation, through the use of rapamycin\nto elicit a complete inhibition of cancer cell\nis insufficient\nproliferation. This may be in part due to chemoresistance that\narises as a consequence of Akt-mediated activation. Hence, the\nbenefits of rapamycin treatment may not be realised in all cases\nof aberrant mTOR activity. Since experimental models implicate\ntranslation initiation directly in cancer cell growth, it would be of\ninterest to develop molecules that specifically target downstream\ncomponents of the mTOR signaling pathways, which control eIF4F\nformation.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nRecent work has implicated microRNAs (miRNAs) in the\nregulation of translation initiation by inhibiting cap-dependent\ntranslation (Pillai et al, 2005).\nImportantly, many miRNAs\nfunction as oncogenes or tumour suppressors (Chen, 2005).\nThus, it is possible that the mTOR pathway could control the\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nactivity of miRNAs in cancer. It will be important to continue\nto study the PI3K/Akt/mTOR pathway and its downstream\nfor anticancer\neffectors of\ntherapies.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\ntranslation as molecular\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\ntargets\n**BLOCK**fs== 10.5**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nACKNOWLEDGEMENTS\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nWe apologize to those authors whose work is not cited due to\neditorial space limitations. We thank Colin Lister and Pamela Kirk\nfor their assistance. This research was supported by grants from\nthe National Cancer Institute of Canada (NCIC), the Howard\nHughes Medical Institute (HHMI) and Canadian Institutes of\nHealth Research (CIHR) of Canada to NS. EP was supported by a\nFellowship from the Cancer Research Society and a Chemical\nBiology Fellowship from McGill University. YM is a recipient of an\nNCIC fellowship. DS is supported by a McGill Majors Fellowship\nfrom McGill University. OLB is supported by a Chemical Biology\nFellowship. NS is a James McGill Professor, a CIHR distinguished\nscientist and an HHMI International Scholar.\n**BLOCK**fs== 10.5**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nREFERENCES\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nAvdulov S, Li S, Michalek V, Burrichter D, Peterson M, Perlman DM,\nManivel JC, Sonenberg N, Yee D, Bitterman PB, Polunovsky VA (2004)\nActivation of translation complex eIF4F is essential for the genesis and\nmaintenance of the malignant phenotype in human mammary epithelial\ncells. Cancer Cell 5: 553 – 563\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nBardeesy N, Sinha M, Hezel AF, Signoretti S, Hathaway NA, Sharpless NE,\nLoda M, Carrasco DR, DePinho RA (2002) Loss of the Lkb1 tumour\nsuppressor provokes intestinal polyposis but resistance to transforma-\ntion. Nature 419: 162 – 167\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nBeuvink I, Boulay A, Fumagalli S, Zilbermann F, Ruetz S, O’Reilly T, Natt F,\nHall J, Lane HA, Thomas G (2005) The mTOR inhibitor RAD001\nsensitizes tumor cells to DNA-damaged induced apoptosis through\ninhibition of p21 translation. Cell 120: 747 – 759\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nBurchert A, Wang Y, Cai D, von Bubnoff N, Paschka P, Muller-Brusselbach\nS, Ottmann OG, Duyster J, Hochhaus A, Neubauer A (2005) Compensa-\ntory PI3-kinase/Akt/mTor activation regulates\nimatinib resistance\ndevelopment. Leukemia 19: 1774 – 1782\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nChen CZ (2005) MicroRNAs as oncogenes and tumor suppressors. N Engl J\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nMed 353: 1768 – 1771\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nDe Benedetti A, Graff JR (2004) eIF-4E expression and its role in\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\nmalignancies and metastases. Oncogene 23: 3189 – 3199\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nDilling MB, Germain GS, Dudkin L, Jayaraman AL, Zhang X, Harwood FC,\nHoughton PJ (2002) 4E-binding proteins, the suppressors of eukaryotic\ninitiation factor 4E, are down-regulated in cells with acquired or intrinsic\nresistance to rapamycin. J Biol Chem 277: 13907 – 13917\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nEvans JM, Donnelly LA, Emslie-Smith AM, Alessi DR, Morris AD (2005)\nMetformin and reduced risk of cancer in diabetic patients. BMJ 330:\n1304 – 1305\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nFumagalli S, Thomas G (2000) S6 phosphorylation and signal transduction.\nIn Translational Control of Gene Expression, Sonenberg N, Hershey JWB\nand Mathews MB (eds) pp 695 – 717. Cold Spring Harbor Laboratory\nPress: Cold Spring Harbor, New York\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nGingras AC, Raught B, Sonenberg N (1999) eIF4 initiation factors: effectors\nof mRNA recruitment to ribosomes and regulators of translation. Annu\nRev Biochem 68: 913 – 963\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nGoudar RK, Shi Q, Hjelmeland MD, Keir ST, McLendon RE, Wikstrand CJ,\nReese ED, Conrad CA, Traxler P, Lane HA, Reardon DA, Cavenee WK,\nWang XF, Bigner DD, Friedman HS, Rich JN (2005) Combination\ntherapy of\ninhibitors of epidermal growth factor receptor/vascular\nendothelial growth factor receptor 2 (AEE788) and the mammalian target\nof rapamycin (RAD001) offers improved glioblastoma tumor growth\ninhibition. Mol Cancer Ther 4: 101 – 112\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nGuertin DA, Sabatini DM (2005) An expanding role for mTOR in cancer.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nTrends Mol Med 11: 353 – 361\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nHarrington LS, Findlay GM, Gray A, Tolkacheva T, Wigfield S, Rebholz H,\nBarnett J, Leslie NR, Cheng S, Shepherd PR, Gout I, Downes CP, Lamb RF\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n(2004) The TSC1-2 tumor suppressor controls insulin-PI3K signaling via\nregulation of IRS proteins. J Cell Biol 166: 213 – 223\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nHay N, Sonenberg N (2004) Upstream and downstream of mTOR. Genes\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nDev 18: 1926 – 1945\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nHeitman J, Movva NR, Hall MN (1991) Targets for cell cycle arrest by the\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nimmunosuppressant rapamycin in yeast. Science 253: 905 – 909\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nHuang S, Houghton PJ (2002) Inhibitors of mammalian target of rapamycin\nas novel antitumor agents: from bench to clinic. Curr Opin Investig Drugs\n3: 295 – 304\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nInoki K, Corradetti MN, Guan KL (2005a) Dysregulation of the TSC-mTOR\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.2**\npathway in human disease. Nat Genet 37: 19 – 24\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nInoki K, Ouyang H, Li Y, Guan KL (2005b) Signaling by target of rapamycin\nproteins in cell growth control. Microbiol Mol Biol Rev 69: 79 – 100\nKim DH, Sabatini DM (2004) Raptor and mTOR: subunits of a nutrient-\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nsensitive complex. Curr Top Microbiol Immunol 279: 259 – 270\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nKwiatkowski DJ, Manning BD (2005) Tuberous sclerosis: a GAP at the\ncrossroads of multiple signaling pathways. Hum Mol Genet 14(Suppl 2):\nR251 – R258\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nLazaris-Karatzas A, Montine KS, Sonenberg N (1990) Malignant transfor-\nmation by a eukaryotic initiation factor subunit that binds to mRNA 50\ncap. Nature 345: 544 – 547\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nLorberg A, Hall MN (2004) TOR: the first 10 years. Curr Top Microbiol\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nImmunol 279: 1 – 18\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nMa L, Chen Z, Erdjument-Bromage H, Tempst P, Pandolfi PP (2005)\nPhosphorylation and functional inactivation of TSC2 by Erk implications\nfor tuberous sclerosis and cancer pathogenesis. Cell 121: 179 – 193\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nMamane Y, Petroulakis E, Rong L, Yoshida K, Ler LW, Sonenberg N (2004)\neIF4E – from translation to transformation. Oncogene 23: 3172 – 3179\nMohi MG, Boulton C, Gu TL, Sternberg DW, Neuberg D, Griffin JD,\nGilliland DG, Neel BG (2004) Combination of rapamycin and protein\ntyrosine kinase (PTK) inhibitors for the treatment of leukemias caused\nby oncogenic PTKs. Proc Natl Acad Sci USA 101: 3130 – 3135\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nMondesire WH, Jian W, Zhang H, Ensor J, Hung MC, Mills GB, Meric-\nBernstam F (2004) Targeting mammalian target of rapamycin synergis-\ntically enhances chemotherapy-induced cytotoxicity in breast cancer\ncells. Clin Cancer Res 10: 7031 – 7042\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nOridate N, Kim HJ, Xu X, Lotan R (2005) Growth inhibition of head and\nneck squamous carcinoma cells by small interfering RNAs targeting\neIF4E or cyclin D1 alone or combined with cisplatin. Cancer Biol Ther 4:\n318 – 323\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\nPanwalkar A, Verstovsek S, Giles FJ\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n(2004) Mammalian target of\nrapamycin inhibition as therapy for hematologic malignancies. Cancer\n100: 657 – 666\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nPende M, Um SH, Mieulet V, Sticker M, Goss VL, Mestan J, Mueller M,\nFumagalli S, Kozma SC, Thomas G (2004) S6K1(/)/S6K2(/)\nlethality and rapamycin-sensitive 50-terminal\nmice exhibit perinatal\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\noligopyrimidine mRNA translation and reveal a mitogen-activated\nprotein kinase-dependent S6 kinase pathway. Mol Cell Biol 24:\n3112 – 3124\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nPillai RS, Bhattacharyya\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nSN, Artus CG, Zoller T, Cougot N,\nBasyuk\nof\ntranslational initiation by Let-7 MicroRNA in human cells. Science 309:\n1573 – 1576\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.6**\nFilipowicz W (2005)\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nE, Bertrand\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nInhibition\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nE,\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nProud CG (2004) mTOR-mediated regulation of translation factors by\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\namino acids. Biochem Biophys Res Commun 313: 429 – 436\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nRaught B, Gingras AC, Gygi SP, Imataka H, Morino S, Gradi A, Aebersold\nR, Sonenberg N (2000) Serum-stimulated, rapamycin-sensitive phos-\nphorylation sites in the eukaryotic translation initiation factor 4GI.\nEMBO J 19: 434 – 444\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nRaught B, Peiretti F, Gingras AC, Livingstone M, Shahbazian D, Mayeur GL,\nPolakiewicz RD, Sonenberg N, Hershey JW (2004) Phosphorylation of\neucaryotic translation initiation factor 4B Ser422 is modulated by S6\nkinases. EMBO J 23: 1761 – 1769\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nmTOR signaling: implications for cancer and anticancer therapy\nE Petroulakis et al\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nRuggero D, Montanaro L, Ma L, Xu W, Londei P, Cordon-Cardo C, Pandolfi\nPP (2004) The translation factor eIF-4E promotes tumor formation and\ncooperates with c-Myc in lymphomagenesis. Nat Med 10: 484 – 486\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nShaw RJ, Bardeesy N, Manning BD, Lopez L, Kosmatka M, DePinho RA,\nCantley LC (2004) The LKB1 tumor suppressor negatively regulates\nmTOR signaling. Cancer Cell 6: 91 – 99\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nUm SH, Frigerio F, Watanabe M, Picard F, Joaquin M, Sticker M, Fumagalli\nS, Allegrini PR, Kozma SC, Auwerx J, Thomas G (2004) Absence of S6K1\nprotects against age- and diet-induced obesity while enhancing insulin\nsensitivity. Nature 431: 200 – 205\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nWendel HG, De Stanchina E, Fridman JS, Malina A, Ray S, Kogan S,\nCordon-Cardo C, Pelletier J, Lowe SW (2004) Survival signalling by Akt\nand eIF4E in oncogenesis and cancer therapy. Nature 428: 332 – 337\nXu RH, Pelicano H, Zhang H, Giles FJ, Keating MJ, Huang P (2005)\nSynergistic effect of targeting mTOR by rapamycin and depleting ATP by\ninhibition of glycolysis in lymphoma and leukemia cells. Leukemia 19:\n2153 – 2158",
         "extracted",
         "",
         "mTOR signaling: implications for cancer and anticancer therapy",
         "None"
        ],
        [
         "40",
         "002fbca2bbbe668326f59226c20ed85af17d375a",
         "None",
         "J. Otten",
         "https://link.springer.com/content/pdf/10.1007%2F11554554_19.pdf",
         "\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\nJens Otten\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nInstitut f¨ur Informatik, University of Potsdam,\nAugust-Bebel-Str. 89, 14482 Potsdam-Babelsberg, Germany\njeotten@cs.uni-potsdam.de\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.4**\nAbstract. We present a clausal connection calculus for ﬁrst-order in-\ntuitionistic logic. It extends the classical connection calculus by adding\npreﬁxes that encode the characteristics of intuitionistic logic. Our calcu-\nlus is based on a clausal matrix characterisation for intuitionistic logic,\nwhich we prove correct and complete. The calculus was implemented by\nextending the classical prover leanCoP. We present some details of the\nimplementation, called ileanCoP, and experimental results.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n1 Introduction\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nAutomated reasoning in intuitionistic ﬁrst-order logic is an important task within\nthe formal approach of constructing veriﬁable correct software. Interactive proof\nassistants, like NuPRL [5] and Coq [2], use constructive type theory to formalise\nthe notion of computation and would greatly beneﬁt from a higher degree of\nautomation. Automated theorem proving in intuitionistic logic is considerably\nmore diﬃcult than in classical logic, because additional non-permutabilities in\nthe intuitionistic sequent calculus [8] increase the search space. In classical logic\n(disjunctive or conjunctive) clausal forms are commonly used to simplify the\nproblem of copying appropriate subformulae (so-called multiplicities). Once a\nformula is converted into clausal form multiplicities can be restricted to clauses.\nFor intuitionistic logic a validity-preserving clausal form does not exist.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nAn elegant way of encoding the intuitionistic non-permutabilities was given\nby Wallen [30] by extending Bibel’s (non-clausal) characterisation of logical va-\nlidity [3]. The development of proof calculi and implementations based on this\ncharacterisation (e.g. [11,21,22,25]) were restricted to non-clausal procedures,\nmaking it diﬃcult to use more established clausal methods (e.g. [3,4,13,14]).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nIn this paper we present a clausal matrix characterisation for intuitionistic\nlogic. Extending the usual Skolemization technique makes it possible to adapt a\nconnection calculus that works on preﬁxed matrices in clausal form. It simpliﬁes\nthe notation of multiplicities and the use of existing clausal connection-based\nimplementations for classical logic. Only a few changes are required to adapt the\nclassical prover leanCoP to deal with preﬁxed matrices.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nThe paper is organised as follows. In Section 2 the standard (non-clausal)\nmatrix characterisation is presented before Section 3 introduces a clausal charac-\nterisation using Skolemization and a preﬁxed connection calculus. The ileanCoP\n**BLOCK**fs== 7.0**b== 0.0**t== 1.0**l== 0.1**r== 0.5**\nB. Beckert (Ed): TABLEAUX 2005, LNAI 3702, pp. 245–261, 2005.\nc Springer-Verlag Berlin Heidelberg 2005\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nimplementation with experimental results is described in Section 4. We conclude\nwith a summary and a brief outlook on further research in Section 5.\n**BLOCK**fs== 12.0**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\n2 Preliminaries\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nWe assume the reader to be familiar with the language of classical ﬁrst-order\nlogic (see, e.g., [7]). We start by deﬁning some basic concepts before brieﬂy\ndescribing the matrix characterisation for classical and intuitionistic logic.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n2.1 Formula Trees, Types and Multiplicities\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nSome basic concepts are formula trees, types and multiplicities (see [4,7,30]).\nMultiplicities encode the contraction rule in the sequent calculus [8].\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\nDeﬁnition 1 (Formula Tree). A formula tree is the graphical representation\nof a formula F as a tree. Each node is labeled with a connective/quantiﬁer or\natomic subformula of F and marked with a unique name, its position, denoted\nby a0, a1, . . . . The set of all positions is denoted by Pos. The tree ordering\n< ⊆ Pos × Pos is the (partial) ordering on the positions in the formula tree,\ni.e. ai<aj iﬀ position ai is below position aj in the formula tree.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nExample 1. Figure 1 shows the formula tree for F1 = ∀xP x ⇒ P b ∧ P c with\nPos = {a1, . . . , a5}. It is, e.g., a1<a2 and a0<a4. Note that each subformula of a\ngiven formula corresponds to exactly one position in its formula tree, e.g. ∀xP x\ncorresponds to the position a1.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nP x a2\n\n∀x a1\n\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\na5 P c\n\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\nP b a4\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.2**r== 0.8**\n\n∧ a3\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n⇒ a0\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\na2 P 1a1 (φ)\n\na1 ∀1x (γ,φ)\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\na7 P 1a6 (φ)\n\na6 ∀1x (γ,φ)\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\n \n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\nα\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\na5 P 0c (ψ)\na4 P 0b (ψ)\n\n\n\na3 ∧0 (β)\n\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.4**r== 0.5**\na0 ⇒0 (α,ψ)\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\nFig. 1. Formula tree for F1 and F µ\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.4**r== 0.5**\n1 with µ(a1) = 2\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nDeﬁnition 2 (Types, Polarity). The principal and intuitionistic type of a\nposition is determined by its label and polarity according to Table 1 (e.g. a for-\nmula/position A ∧ B with polarity 1 has type α). Atomic positions have no prin-\ncipal type and some positions have no intuitionistic type. We denote the sets of\npositions of type γ, δ, φ, and ψ by Γ , ∆, Φ, and Ψ , respectively. The polarity\n(0 or 1) of a position is determined by the label and polarity of its predecessor\nin the formula tree according to table 1 (e.g. if A ∧ B has polarity 1 then both\nsubformula A and B have polarity 1 as well). The root position has polarity 0.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nprincipal type α\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nsuccessor polarity\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nprincipal type β\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nsuccessor polarity\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nprincipal type γ\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nsuccessor polarity\nintuitionistic type φ\nintuitionistic type ψ\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\n(A ∧ B)1\nA1, B1\n(A ∧ B)0\nA0, B0\n(∀xA)1 (∃xA)0\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.3**r== 0.6**\n(A ∨ B)0\nA0, B0\n(A ∨ B)1\nA1, B1\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\n(A⇒B)0\nA1, B0\n(A⇒B)1\nA0, B1\nprincipal type δ\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nA0\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nA1\n(¬A)1\n(¬A)0\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.4**r== 0.5**\nsuccessor polarity\n(∀xA)1\n(∀xA)0\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\n(A⇒B)1\n(A⇒B)0\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n(¬A)1\nA0\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n(¬A)0\nA1\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n(∀xA)0 (∃xA)1\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nA0\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nA1\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nP 1 (P is atomic)\nP 0 (P is atomic)\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\nExample 2. In the left formula tree for F1 in Figure 1 (ignore the additional\nbranch for now) each node/position is marked with its types and its polarity.\nThe positions a0, a1, a3 have principal type α, γ and β, respectively; the positions\na1, a2 and a0, a4, a5 have intuitionistic type φ and ψ, respectively.\nDeﬁnition 3 (Multiplicity). The multiplicity µ : Γ ∪ Φ → IN assigns each\nposition of type γ and φ a natural number. By F µ we denote an indexed formula.\nIn the formula tree of F µ multiple instances of subformulae — according to the\nmultiplicity of its positions — have been considered. The branch instances of a\nsubformula have an implicit node/position of type α as predecessor.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nExample 3. The formula tree for F1 with µ(a1)=2 is shown in Figure 2. Here and\nin the following we will replace variables in atomic formulae by their quantiﬁer\npositions. Thus positions of type γ and δ appear in atomic formulae.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n2.2 Matrix Characterisation for Classical Logic\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nTo resume the characterisation for classical logic [3,4] we introduce the concepts\nof matrices, paths and connections. A path corresponds to a sequent and a\nconnection to an axiom in the sequent calculus [8]. For the ﬁrst-order case we\nalso need the notation of ﬁrst-order substitution and reduction ordering.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.6**\nDeﬁnition 4 (Matrix,Path,Connection).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\n1. In the matrix(-representation) of an indexed formula F µ we place the sub-\nformulae of a formula of principal type α side by side whereas subformulae of\na subformula of principal type β are placed one upon the other. Furthermore,\nwe omit all connectives and quantiﬁers.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\n2. A path through an indexed formula F µ is a subset of the atomic formulae of\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.4**\nits formula tree; it is a horizontal path through the matrix of F µ.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\n3. A connection is a pair of atomic formulae with the same predicate symbol\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nbut with diﬀerent polarities.\nExample 4. The matrix for F µ\n1 is given in Figure 2. There are two paths through\nF µ\n1 : {P 1a1, P 1a6, P 0b} and {P 1a1, P 1a6, P 0c}. They contain the connections\n{P 1a1, P 0b}, {P 1a6, P 0b} and {P 1a1, P 0c}, {P 1a6, P 0c}, respectively. We will\npresent a more formal deﬁnition of matrices and paths in Section 3.\n**BLOCK**fs== 0.1**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nP 1a1\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nP 1a6\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nP 0b\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\nP 0c\n**BLOCK**fs== 0.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.7**t== 0.3**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.3**r== 0.6**\nΓ,A \nΓ  ¬A,∆\n¬-right\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\nΓ,A  B\nΓ  A⇒B,∆\n⇒-right\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\nΓ  A[x\\a]\nΓ  ∀xA,∆\n∀-right\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.2**r== 0.7**\nFig. 2. Matrix for F µ\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.3**r== 0.4**\n1 and special intuitionistic sequent rules\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.3**\nDeﬁnition 5 (First-order Substitution, Reduction Ordering).\n1. A ﬁrst-oder substitution σQ : Γ → T assigns to each position of type γ\na term t ∈ T . T is the set of terms made up of constants, functions and\nelements of Γ and ∆, which are called term variables and term constants,\nrespectively. A connection {P 0s, P 1t} is said to be σQ-complementary iﬀ\nσQ(s)=σQ(t). σQ induces a relation Q ⊆ ∆ × Γ in the following way: for\nall u ∈ Γ vQu holds for all v ∈ ∆ occurring in σQ(u).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\n2. The reduction ordering  := (< ∪ Q)+ is the transitive closure of the tree\nordering < and the relation Q. A ﬁrst order substitution σQ is said to be\nadmissible iﬀ the reduction ordering  is irreﬂexive.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\nTheorem 1 (Characterisation for Classical Logic [4]). A (ﬁrst-order) for-\nmula F is classically valid, iﬀ there is a multiplicity µ, an admissible ﬁrst-order\nsubstitution σQ and a set of σQ-complementary connections such that every path\nthrough F µ contains a connection from this set.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nExample 5. Let µ(a1)=2 and σQ={a1\\b, a6\\c}, i.e. σQ(a1)=b, σQ(a6)=c. σQ is\nadmissible, since the induced reduction ordering (= tree ordering) is irreﬂexive.\n{{P 1a1, P 0b}, {P 1a6, P 0c}} is a σQ-complementary set of connections and every\npath through F µ\n1 contains a connection from this set. Thus F1 is classically valid.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n2.3 Matrix Characterisation for Intuitionistic Logic\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nA few extensions are necessary to adapt the characterisation of classical valid-\nity to intuitionistic logic. These are preﬁxes and an intuitionistic substitution.\nPreﬁxes encode the characteristics of the special rules (see Figure 2) in the\nintuitionistic sequent calculus (see [30]). Alternatively they can be seen as an\nencoding of the possible world semantics of intuitionistic logic (see [30]).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nDeﬁnition 6 (Preﬁx). Let u1<u2< . . . <un≤u be the positions of type φ or\nψ that dominate the position/formula u in the formula tree. The preﬁx of u,\ndenoted pre(u), is a string over Φ ∪ Ψ and deﬁned as pre(u) := u1u2 . . . un.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.4**\nExample 6. For the formula F µ\npre(a4)=a0a4 and pre(a5)=a0a5. Positions of type φ are written in capitals.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.3**r== 0.3**\n1 we obtain pre(a2)=a0A1A2, pre(a7)= a0A6A7,\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nDeﬁnition 7 (Intuitionistic/Combined Substitution, Admissibility).\n1. An intuitionistic substitution σJ : Φ → (Φ ∪ Ψ )∗ assigns to each position of\ntype φ a string over the alphabet (Φ∪Ψ ). Elements of Φ and Ψ are called preﬁx\nvariables and preﬁx constants, respectively. σJ induces a relation J ⊆ Ψ ×Φ\nin the following way: for all u∈Φ vJ u holds for all v∈Ψ occurring in σJ (u).\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\n2. A combined substitution σ:=(σQ, σJ ) consists of a ﬁrst-order and an in-\ntuitionistic substitution. A connection {P 0s, P 1t} is σ-complementary iﬀ\nσQ(s)=σQ(t) and σJ (pre(P 0s))=σJ (pre(P 1t)). The reduction ordering  :=\n(< ∪ Q ∪ J )+ is the transitive closure of <, Q and J .\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\n3. A combined substitution σ=(σQ, σJ ) is said to be admissible iﬀ the reduction\nordering  is irreﬂexive and the following condition holds for all u ∈ Γ and\nall v ∈ ∆ occurring in σQ(u): σJ (pre(u))=σJ (pre(v))◦q for some q ∈ (Φ∪Ψ )∗.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.3**\nTheorem 2 (Characterisation for Intuitionistic Logic [30]). A formula\nF is intuitionistically valid, iﬀ there is a multiplicity µ, an admissible combined\nsubstitution σ = (σQ, σJ ), and a set of σ-complementary connections such that\nevery path through F µ contains a connection from this set.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nExample 7. Let µ(a1)=2 and σ = (σQ, σJ ) with σQ={a1\\b, a6\\c} and σJ =\n{A1\\ε, A2\\a4, A6\\ε, A7\\a5} in which ε denotes the empty string. Then σ is ad-\nmissible and {{P 1a1, P 0b}, {P 1a6, P 0c}} is a σQ-complementary set of connec-\ntions and every path through F µ\n1 contains a connection from this set. Therefore\nF1 is intuitionistically valid.\n**BLOCK**fs== 12.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n3 Clausal Connection-Based Theorem Proving\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nThis section presents a clausal version of the matrix characterisation for intu-\nitionistic logic and a preﬁxed connection calculus based on this characterisation.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.6**\n3.1 Preﬁxed Matrix and Skolemization\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nIn order to develop a preﬁxed connection calculus more formal deﬁnitions of\nthe concepts of a matrix and paths (as introduced in Deﬁnition 4) are required.\nAfterwards we will also introduce a Skolemization technique, which extends the\nSkolemization used for classical logic to intuitionistic logic.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nA (non-clausal, i.e. nested) matrix is a set of clauses and contains α-related\nsubformulae. A clause is a set of matrices and contains β-related subformulae. A\n(non-clausal) matrix is a compact representation of a (classical) formula and can\nbe represented in the usual two dimensional graphical way. Adding preﬁxes will\nmake it suitable to represent intuitionistic formulae as well. The formal deﬁnition\nof paths through a (matrix of a) formula is adapted accordingly.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nDeﬁnition 8 (Preﬁxed Matrix).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\n1. A preﬁx p is a string with p ∈ (Φ ∪ Ψ )∗. Elements of Φ and Ψ play the role\nof variables and constants, respectively. A preﬁxed (signed) formula F pol:p\nis a formula F marked with a polarity pol ∈ {0, 1} and a preﬁx p.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\n2. A preﬁxed matrix M is a set of clauses in which a clause is a set of preﬁxed\nmatrices and preﬁxed atomic formulae. The preﬁxed matrix of a preﬁxed\nformula F pol:p is inductively deﬁned according to Table 2. In Table 2 A, B\nare formulae, P s is an atomic formula, p ∈ (Φ ∪ Ψ )∗ is a preﬁx, Z ∈ Φ, a ∈ Ψ\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\nFormula Preﬁxed matrix MA, MB is\nof F pol:p\nF pol:p\nmatrix of\n{{P 1s:p◦Z}}\n(P 1s):p\n{{P 0s:p ◦a}}\n(P 0s):p\n(A ∧ B)1:p {{MA}, {MB}} A1:p , B1:p\n(A ∨ B)0:p {{MA}, {MB}} A0:p , B0:p\n(A⇒B)0:p {{MA}, {MB}} A1:p , B0:p\n(∀yA)1:p\nMA\n(∃yA)0:p\nMA\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nx:p ◦Z\nx:p\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\nA1\nA0\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.4**r== 0.4**\nFormula Preﬁxed matrix MA, MB is\nof F pol:p\nF pol:p\nmatrix of\nA0:p◦Z\n(¬A)1:p\nMA\n(¬A)0:p\nA1:p ◦a\nMA\n(A ∧ B)0:p\n(A ∨ B)1:p\n(A⇒B)1:p\n(∀yA)0:p\n(∃yA)1:p\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n{{MA, MB}} A0:p , B0:p\n{{MA, MB}} A1:p , B1:p\n{{MA, MB}} A0:p , B1:p\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nc:p ◦a\nc:p\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nMA\nMA\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nA0\nA1\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\nare new (unused) elements. Ax and Ac is the formula A in which y is replaced\nby new (unused) elements x ∈ Γ and c ∈ ∆, respectively. The preﬁxed matrix\nof a formula F , denoted matrix(F ), is the preﬁxed matrix of F 0:ε.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\nDeﬁnition 9 (Path). A path through a (preﬁxed) matrix M or clause C is a\nset of preﬁxed atomic formulae and deﬁned as follows: If M ={{P pols:p}} is a\nmatrix and P s an atomic formula then {P pols:p} is the only path through M .\nOtherwise if M ={C1, .., Cn} is a matrix and pathi is a path through the clause\ni=1 pathi is a path through M . If C={M1, .., Mn} is a clause and pathi\nCi then\nis a path through the matrix Mi then pathi (for 1≤i≤n) is a path through C.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n\nn\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nExample 8. The preﬁxed matrix of formula F1 (see examples in Section 2) is\nM1=matrix(F1)= {{P 1x1:a2Z3Z4}, {P 0b:a2a5, P 0c:a2a6}}, in which submatri-\nces of the form {{M1, .., Mn}} have been simpliﬁed to M1, .., Mn. The paths\nthrough M1 are {P 1x1:a2Z3Z4, P 0b:a2a5} and {P 1x1:a2Z3Z4, P 0c:a2a6}.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nAn indexed formula F µ is deﬁned in the usual way, i.e. each subformula\nF  of type γ or φ is assigned its multiplicity µ(F ) ∈ IN encoding the number\nof instances to be considered. Before F µ is translated into a matrix, each F \nis replaced with (F \ni is a copy of F . The notation\nµ(F )) in which F \n\nof σ-complementary is slightly modiﬁed, i.e. a connection {P 0s:p, P 1t:q} is σ-\ncomplementary iﬀ σQ(s)=σQ(t) and σJ (p)=σJ (q).\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n∧ . . . ∧ F \n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nFor a combined substitution σ to be admissible (see Deﬁnition 7) the re-\nduction ordering  induced by σ has to be irreﬂexive. In classical logic this\nrestriction encodes the Eigenvariable condition in the classical sequent calculus\n[8]. It is usually integrated into the σ-complementary test by using the well-\nknown Skolemization technique together with the occurs-check of term uniﬁca-\ntion. We extend this concept to the intuitionistic substitution σJ . To this end\nwe introduce a new substitution σ<, which is induced by the tree ordering <.\nσ< assigns to each constant (elements of ∆ and Ψ ) a Skolemterm containing all\nvariables (elements of Γ and Φ) occurring below this constant in the formula\ntree. It is suﬃcient to restrict σ< to these elements, since J ⊆ Ψ × Φ and\nQ ⊆ ∆ × Γ .\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nDeﬁnition 10 (σ<-Skolemization). A tree ordering substitution σ< : (∆∪Ψ )\n→ T assigns a Skolemterm to every element of ∆ ∪ Ψ . It is induced by the tree\nordering < in the following way: σ< := { c\\c(x1, ...xn) | c ∈ ∆ ∪ Ψ and for all\nx ∈ Γ ∪ Φ: (x ∈ {x1, . . . xn} iﬀ x < c)}.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\nExample 9. The tree ordering of formula F1 induces the substitution σ< =\n{a2\\a2(), a5\\a5(), a6\\a6()}, since no variables occur before any constant.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.3**\nNote that we follow a purely proof-theoretical view on Skolemization, i.e. as a\nway to integrate the irreﬂexivity test of the reduction ordering into the condition\nof σ-complementary. For classical logic this close relationship was pointed out by\nBibel [4]. Like for classical logic the use of Skolemization simpliﬁes proof calculi\nand implementations. There is no need for an explicit irreﬂexivity check and\nsubformulae/-matrices can be copied by just renaming all their variables.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\nLemma 1 (Admissibility Using σ<-Skolemization). Let F be a formula\nand < be its tree ordering. A combined substitution σ = (σQ, σJ ) is admissible\niﬀ (1) σ = σ<∪σQ∪σJ is idempotent, i.e. σ(σ) = σ, and (2) the following\nholds for all u ∈ Γ and all v ∈ ∆ occurring in σQ(u): σJ (pre(u))=σJ (pre(v))◦q\nfor some q ∈ (Φ ∪ Ψ )∗.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nProof. It is suﬃcient to show that  is reﬂexive iﬀ σ<∪σQ∪σJ is not idempotent.\n”⇒”: Let  be reﬂexive. Then there are positions with a1 . . . ana1. For each\naiaj there is ai<aj, aiQaj or aiJ aj. According to Deﬁnition 10, 5 and 7\nthere is some substitution with {aj\\t} in σ<, σQ or σJ , respectively, in which\nai occurs in t. Then σ<∪σQ∪σJ is not idempotent. ”⇐”: Let σ<∪σQ∪σJ be not\nidempotent. Then there is σ={a1\\..an.., an\\..an−1.., . . . , a2\\..a1..}⊆σ<∪σQ∪σJ .\nEach {aj\\..ai..} ∈ σ is part of σ<, σQ or σJ . According to Deﬁnition 10, 5 and\n7 it is ai<aj, aiQaj or aiJ aj. Therefore :=(<∪Q∪J )+ is reﬂexive. \n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n3.2 A Clausal Matrix Characterisation\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nWe will now deﬁne preﬁxed matrices in clausal form and adapt the notation of\nmultiplicities to clausal matrices before presenting a clausal matrix characteri-\nsation for intuitionistic logic. The restriction of multiplicities to clauses makes it\npossible to use existing clausal calculi for which multiplicities can be increased\nduring the proof search in an easy way. The transformation of a preﬁxed matrix\nto clausal form is done like for classical logic. Note that we apply the substitution\nσ< to the clausal matrix, i.e. to terms and preﬁxes of atomic formulae.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nDeﬁnition 11 (Preﬁxed Clausal Matrix). Let < be a tree ordering and M\nbe a preﬁxed matrix. The (preﬁxed) clausal matrix of M , denoted clausal(M ),\nis a set of clauses in which each clause is a set of preﬁxed atomic formu-\nlae. It is inductively deﬁned as follows: If M has the form {{P pols:p}} then\ni=1 ci} | ci ∈ clausal(Mi)\nclausal(M ):=M ; otherwise clausal(M ) :=\nand C={M1, . . . , Mn}}. The (preﬁxed) clausal matrix of a formula F (with tree\nordering <), denoted Mc = matrixc(F ), is σ<(clausal(matrix(F ))).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\nC ∈ M\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\n\nn\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.5**\n{{\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nDeﬁnition 12 (Clausal Multiplicity). The clausal multiplicity µc : Mc → IN\nassigns each clause in Mc a natural number, specifying the number of clause\ninstances to be considered. matrixµ\nc (F ) denotes the clausal matrix of F in which\nclause instances/copies speciﬁed by µc have been considered. Term and preﬁx\nvariables in clause copies are renamed (Skolemfunctions are not renamed).\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\nExample 10. For F1 let µc({P 1x1:a2()Z3()Z4()}) = 2. Then matrixµ\n{{P 1x1:a2()Z3Z4}, {P 1x7:a2()Z8Z9}, {P 0b:a2()a5(), P 0c:a2()a6()}}.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nc (F1) =\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.3**\nNote that we use the same Skolemfunction (symbol) for instances of the\nsame subformula/clause. This is an optimisation, which has a similar eﬀect like\nthe liberalised δ+-rule for (classical) semantic tableaux [9]. Since we apply the\nsubstitution σ< to the clausal matrix, term and preﬁx constants can now both\ncontain term and preﬁx variables. Therefore the ﬁrst-order and intuitionistic\nsubstitutions σQ and σJ are extended so that they assign terms over Γ ∪∆∪Φ∪Ψ\nto term and preﬁx variables (elements of Γ and Φ).\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\nTheorem 3 (Clausal Characterisation for Intuitionistic Logic). A for-\nmula F is intuitionistically valid, iﬀ there is a multiplicity µc, an admissible\ncombined substitution σ = (σQ, σJ ), and a set of σ-complementary connections\nsuch that every path through matrixµ\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.3**r== 0.4**\nc (F ) contains a connection from this set.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\nProof. Follows directly from the following Lemma 2 and Theorem 2.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nLemma 2 (Equivalence of Matrix and Clausal Matrix Proofs). There\nis a clausal matrix proof for F iﬀ there is a (non-clausal) matrix proof for F .\n**BLOCK**fs== 10.0**b== 0.1**t== 0.6**l== 0.1**r== 0.3**\nProof. Main idea: paths through clausal matrix and (non-clausal) matrix of F\nare identical and substitutions and multiplicities µc/µ can be transfered into each\nother. Note that in the following proof ”variable” refers to term and preﬁx vari-\nables. We assume that σ< is applied to the non-clausal matrix as well.\n”⇒”: Let (µc, σ, S) be a clausal matrix proof of F in which S is the connection\nset. We will construct a matrix proof (µ, σ, S) for F . Let Mc=matrixµ\nc (F ) be\nthe clausal matrix of F (with clause instances according to µc). We construct a\nmatrix M =matrix(F µ) in the following way: We start with the original matrix\nM . For each clause Ci ∈ Mc we identify the vertical path through M which rep-\nresents Ci (modulo Skolemfunction names); see matrices on left side of Figure\n3. If this vertical path shares variables with already identiﬁed clauses in M we\n1, .., x\ncopy the smallest subclause of M , so that the new path contains copies x\nn\nof all (term and preﬁx) variables x1, .., xn in Ci with σ(xi)=σ(x\ni). Note that\nSkolemfunctions in copied subclauses of M are renamed and therefore unique.\nThe constructed matrix M determines µ. Let σ:=σ and S:=S. We identify ev-\nery connection {P 0s:p, P 1t:q} ∈ S from Mc as {P 0s:p, P 1t:q} in M . If s, t, p\nor q contain a Skolemfunction fi (which is unique in M ), we rename Skolem-\nfunctions in σ so that σ(s, p)=σ(t, q). There can be no renaming conﬂict for\nthe set S: Let f (x1), f (x2), .. be the same Skolemfunctions in Mc, which are\nrepresented by diﬀerent Skolemfunctions f1(x1), f2(x2), .. in M . If σ(xi)=σ(xj )\nthen diﬀerent Skolemfunctions do not matter, since fi(xi) and fj(xj) are never\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nassigned to the same variable. The case σ(xi)=σ(xj ) does not occur according\nto the construction of M . If copies of the same variable are substituted by the\nsame term, the branches in the formula tree can be folded up (see right side of\nFigure 3); otherwise the branches (and Skolemterms) diﬀer. Every path through\nM contains a path from Mc as a subset. Therefore (µ, σ, S) is a proof for F .\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nMC =matrixµ\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nC (F )=\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.6**\nM=matrix(F µ)=\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\n.\n.\n.\n..f (x)..\n.\n.\n.\n.\n.\n.\nCi\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n..\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n..\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n..\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n..f1(x)..\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\n. . .\n..f2(x)..\n.\n.\n.\n.\n.\n.\nCi\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n..\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.6**t== 0.4**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 0.1**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\n\nσ(x) =\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\nσ(x)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nx\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nσ(f (x))=\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\nσ(f (x))\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.4**\nFig. 3. Clause Ci in (non-)clausal matrix and folding up branches in formula tree\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.1**r== 0.3**\n”⇐”: Let (µ, σ, S) be a (non-clausal) matrix proof for F . We will construct\na clausal matrix proof (µc, σ, S). Let M =matrix(F µ) be the matrix of F µ. We\nconstruct a clausal matrix Mc=matrixµ\nc (F ) in the following way: We start with\nthe clausal form of the original formula F , i.e. Mc=matrixc(F ). We extend Mc\nby using an appropriate µc, so that it is the clausal form of M (modulo Skolem-\nfunction and variable names). Let σ:=σ and S:=S. We extend σ by unifying all\nvariables x1, x2, .. in Mc which have the same image in M . Since copied Skolem-\nfunction names in M diﬀer, but are identical in Mc we can simply replace them\nwith a unique name in σ. Then all connections in S are σ-complementary. By\ninduction we can also show that every path through Mc contains a connection\nfrom S. Therefore (µc, σ, S) is a clausal proof for F .\n\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n3.3 A Preﬁxed Connection Calculus\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.3**\nThe clausal characterisation of intuitionistic validity in Theorem 3 serves as a\nbasis for a proof calculus. Calculating the ﬁrst-order substitution σQ is done by\nthe well-known algorithms for term uniﬁcation (see, e.g., [16]). Checking that\nall paths contain a connection can be done by, e.g., sequent calculi [8] tableau\ncalculi [7] or connection-driven calculi [3,4,13,14]. The clausal connection calcu-\nlus is successfully used for theorem proving in classical logic (see, e.g., [12]). A\nbasic version of the connection calculus for ﬁrst-order classical logic is presented\nin [19]. The calculus uses a connection-driven search strategy, i.e. in each step a\nconnection is identiﬁed along an active (sub-)path and only paths not containing\nthe active path and this connection will be investigated afterwards. The clausal\nmultiplicity µc is increased dynamically during the path checking process. We\nadapt this calculus to deal with intuitionistic logic (based on the clausal charac-\nterisation) by adding preﬁxes to the atomic formulae of each clause as speciﬁed\nin Deﬁnition 8 and 11.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nDeﬁnition 13 (Preﬁxed Connection Calculus). The axiom and the rules\nof the preﬁxed connection calculus are given in Figure 3. M is a preﬁxed clausal\nmatrix, C, Cp, C1, C\n2 are clauses of M and Cp is a positive clause (i.e. contains\nno atomic formulae with polarity 1), C1∪{L} contains no (term or preﬁx) vari-\nables, C\n∪{L} contains at least one variable and C2/L are copies of C\n2/L in\n\nwhich all variables have been renamed. {L, L} and {L, L} are σ-complementary\nconnections, and P ath is the active path, which is a subset of some path trough\nM . A formula F is valid iﬀ there is an admissible substitution σ = (σQ, σJ ) and\na derivation for matrixc(F ) in which all leaves are axioms.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.9**\nAxiom:\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n({}, M, P ath)\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nExtension Rule:\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.6**\nStart Rule:\n(Cp, M ∪{Cp}, {})\nM ∪{Cp}\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.4**\nReduction Rule:\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n(C, M, P ath∪{L})\n(C∪{L}, M, P ath∪{L})\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\n(C1, M, P ath∪{L})\n(C,M,P ath)\n(C∪{L}, M ∪{C1∪{L}}, P ath)\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.5**\nExtension∗ Rule:\n(C,M,P ath)\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n(C2, M∪{C \n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2∪{L}}, P ath∪{L})\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.4**r== 0.5**\n(C∪{L}, M ∪{C \n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n2∪{L}}, P ath)\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.2**r== 0.5**\nFig. 4. The connection calculus for ﬁrst-order logic\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nExample 11. Let matrixc(F1)={{P 1x1:a2()Z3Z4},{P 0b:a2()a5(), P 0c:a2()a6()}}\n=M be the clausal matrix of F1. The following is a derivation for M .\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n({}, M, {})\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\nAxiom\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.2**r== 0.6**\n({}, M, {P 0c:a2()a6()})\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\n({P 0c:a2()a6()}, M, {})\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.6**\nAxiom\nExtension∗\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.2**r== 0.5**\n({P 0b:a2()a5(), P 0c:a2()a6()}, M, {})\n{{P 1x1:a2()Z3Z4}, {P 0b:a2()a5(), P 0c:a2()a6()}}\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nStart\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\n({}, M, {P 0b:a2()a5()})\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nAxiom\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nExtension∗\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\nThis derivation is a proof for F1 with the admissible substitution σ=(σQ, σJ ) and\n\\a2()a5(), Z \nσQ={x\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.4**\n\\a2()a6()}, i.e. F1 is valid.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.2**r== 0.7**\n\\c}, σJ ={Z \n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.4**r== 0.6**\n\\ε, Z \n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\n\\ε, Z \n**BLOCK**fs== 10.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n\\b, x\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\nThe intuitionistic substitution σJ can be calculated using the preﬁx uniﬁca-\ntion algorithm in [20]. It calculates a minimal set of most general uniﬁers for\na given set of preﬁxes {s1=s1, .., tn=tn} by using a small set of rewriting rules\n(similar to the algorithm in [16] for term uniﬁcation). The following deﬁnition\nbrieﬂy describes this algorithm (see [20] for a detailed introduction).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nDeﬁnition 14 (Preﬁx Uniﬁcation). The preﬁx uniﬁcation of two preﬁxes s\nand t is done by applying the rewriting rules deﬁned in Table 3. The rewriting\nrules replace a tuple (E, σJ ) by a modiﬁed tuple (E, σJ\n) in which E and E\n are intuitionistic substitutions. Rules are\nrepresent a preﬁx equation and σJ , σJ\napplied non-deterministically. We start with the tuple ({s = ε|t}, {}), for technical\nreasons we divide the right part of t, and stop when the tuple ({}, σJ ) is derived.\nIn this case σJ represents a most general uniﬁer. There can be more than one\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nmost general uniﬁer (see [20]). The algorithm can also be used to calculate a\nuniﬁer for a set of preﬁx equations {s1=t1, .., sn=tn} in a stepwise manner.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nWe use these symbols: s, t, z ∈ (Φ∪Ψ )∗, s+, t+, z+∈ (Φ∪Ψ )+, X ∈ (Φ∪Ψ ), V, V1\nV  ∈ Φ and C, C1, C2 ∈ Ψ . V  is a new variable which does not refer to a position\nin the formula tree and does not occur in the substitution σJ computed so far.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nTable 3. Rewriting rules for preﬁx uniﬁcation\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.3**r== 0.5**\n→ {}, σJ\n→ {t+ = ε|ε}, σJ\n→ {s = ε|t}, σJ\n→ {V t = ε|Cs}, σJ\n→ {s = ε|ε}, {V \\z}∪σJ\n→ {s = ε|C1t}, {V \\ε}∪σJ\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n{ε = ε|ε}, σJ\n{ε = ε|t+}, σJ\n{Xs = ε|Xt}, σJ\n{Cs = ε|V t}, σJ\n{V s = z|ε}, σJ\n{V s = ε|C1t}, σJ\n{V s = z|C1C2t}, σJ → {s = ε|C2t}, {V \\zC1}∪σJ\n{V s+ = ε|V1t}, σJ → {V1t = V |s+}, σJ\n{V s+ = z+|V1t}, σJ → {V1t = V |s+}, {V \\z+V }∪σJ\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.8**\nR1.\nR2.\nR3.\nR4.\nR5.\nR6.\nR7.\nR8.\nR9.\nR10. {V s = z|Xt}, σJ\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.3**r== 0.6**\n→ {V s = zX|t}, σJ\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n(V =V1)\n(V  ∈ Φ)\n(s=ε or t=ε or X ∈ Ψ )\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n3Z \n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nExample 12. In the ﬁrst step/connection of the proof for F1 we need to unify the\npreﬁxes s = a2Z \n4 and t = a2a5. This is done as follows: ({a2Z \n4=ε|a2a5}, {})\n\\ε}) R10−→ ({Z \nR3−→ ({Z \n\\ε}) R5−→\n4=ε|a5}, {Z \n3Z \n\n\\ε, Z \n({ε=ε|ε}, {Z \n\\a5}). Note that the second (most\n\n\ngeneral) uniﬁer can be calculated by using rule R10 instead of R6.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\n3Z \n4=a5|ε}, {Z \n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\n4=ε|a5}, {}) R6−→ ({Z \n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.7**\n\\a5}) R1−→ ({}, {Z \n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\n\\ε, Z \n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 12.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\n4 An Implementation: ileanCoP\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nThe calculus presented in Section 3 was implemented in Prolog. We will present\ndetails of the implementation and performance results. The source code (and\nmore information) can be obtained at http://www.leancop.de/ileancop/ .\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n4.1 The Code\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nleanCoP is an implementation of the clausal connection calculus presented in\nSection 3 for classical ﬁrst-order logic [19]. The reduction rule is applied before\nextension rules are applied and open branches are selected in a depth-ﬁrst way.\nWe adapt leanCoP to intuitionistic logic by adding\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\na. preﬁxes to atomic formulae and a preﬁx uniﬁcation procedure,\nb. to each clause the set of term variables contained in it and an admissibility\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.4**\ncheck in which these sets are used to check condition (2) of Lemma 1.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nThe main part of the ileanCoP code is depicted in Figure 5. The underlined text\nwas added to leanCoP; no other changes were done. A preﬁx Pre is added to\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.9**\n(1)\n(2)\n(3)\n(4)\n(5)\n(6)\n(7)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.4**l== 0.1**r== 0.9**\n(8)\n(9)\n(10)\n(11)\n(12)\n(13)\n(14)\n(15)\n(16)\n(17)\n(18)\n(19)\n(20)\n(21)\n(22)\n(23)\n(24)\n(25)\n(26)\n(27)\n(28)\n(29)\n(30)\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nprove(Mat,PathLim) :-\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.4**\nappend(MatA,[FV:Cla|MatB],Mat), \\+ member(-( ): ,Cla),\nappend(MatA,MatB,Mat1),\nprove([!:[]],[FV:[-(!):(-[])|Cla]|Mat1],[],PathLim,[PreSet,FreeV]),\ncheck addco(FreeV), prefix unify(PreSet).\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\nprove(Mat,PathLim) :-\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.2**r== 0.4**\n\\+ ground(Mat), PathLim1 is PathLim+1, prove(Mat,PathLim1).\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.5**\nprove([], , , ,[[],[]]).\nprove([Lit:Pre|Cla],Mat,Path,PathLim,[PreSet,FreeV]) :-\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.6**\n(-NegLit=Lit;-Lit=NegLit) ->\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\n( member(NegL:PreN,Path), unify with occurs check(NegL,NegLit),\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.2**r== 0.4**\n\\+ \\+ prefix unify([Pre=PreN]), PreSet1=[], FreeV3=[]\n;\nappend(MatA,[Cla1|MatB],Mat), copy term(Cla1,FV:Cla2),\nappend(ClaA,[NegL:PreN|ClaB],Cla2),\nunify with occurs check(NegL,NegLit),\n\\+ \\+ prefix unify([Pre=PreN]),\nappend(ClaA,ClaB,Cla3),\n( Cla1==FV:Cla2 ->\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.6**\nappend(MatB,MatA,Mat1)\n;\nlength(Path,K), K<PathLim,\nappend(MatB,[Cla1|MatA],Mat1)\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.2**r== 0.4**\n),\nprove(Cla3,Mat1,[Lit:Pre|Path],PathLim,[PreSet1,FreeV1]),\nappend(FreeV1,FV,FreeV3)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.2**r== 0.5**\n),\nprove(Cla,Mat,Path,PathLim,[PreSet2,FreeV2]),\nappend([Pre=PreN|PreSet1],PreSet2,PreSet),\nappend(FreeV2,FreeV3,FreeV).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.2**r== 0.5**\nFig. 5. Main part of the ileanCoP source code\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\natomic formulae q:Pre in which Pre is a list of preﬁx variables and constants.\nThe set of term variables Var is added to each clause Var:Cla in which Var is a\nlist containing pairs [V,Pre] in which V is a term variable and Pre its preﬁx.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nCondition (2) of the admissibility check, check addco, and the uniﬁcation\nof preﬁxes, prefix unify, are done after a classical proof, i.e. a set of con-\nnections, has been found. Therefore the set of preﬁx equations PreSet and the\nclause-variable sets FreeV are collected in an additional argument of prove.\ncheck addco and prefix unify require 5 and 26 more lines of code, respec-\ntively (see [20,22]). Condition (1) of the admissibility check of Lemma 1 is real-\nized by the occurs-check of Prolog during term uniﬁcation. Two preﬁx constants\nare considered equal if they can be uniﬁed by term uniﬁcation. Note that we per-\nform a weak preﬁx uniﬁcation (line 12 and 17) for each connection already during\nthe path checking process (double negation prevents any variable bindings).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nTo prove the formula F1 from Section 2 and 3 we call prove(M,1). in which\nM= [ [[X,[1^[],Z]]]:[-(p(X)): -([1^[],Z,Y])], []:[p(b):\\[1^[],2^[]],p(c):\n[1^[],3^[]]] ] is the preﬁxed matrix of F1 (with clause-variable sets added).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\n4.2 Refuting Some First-Order Formulae\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nIn order to obtain completeness ileanCoP performs iterative deepening on the\nproof depth, i.e. the size of the active path. The limit for this size, PathLim, is\n**BLOCK**fs== 10.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\nincreased after the proof search for a given path limit has failed (line 7 in Figure\n5). If the matrix contains no variables, the given matrix is refuted.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\nWe will integrate a more restrictive method in which the path limit will\nonly be increased if the current path limit was actually reached. In this case\nthe predicate pathlim is written into Prolog’s database indicating the need to\nincrease the path limit if the proof search fails. It can be realized by modifying\nthe following lines of the code in Figure 5:\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n(7)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\n(22)\n(23a)\n(23b)\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.4**\nretract(pathlim), PathLim1 is PathLim+1, prove(Mat,PathLim1).\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.3**r== 0.5**\nlength(Path,K), ( K<PathLim ->\nappend(MatB,[Cla1|MatA],Mat1) ;\n\\+ pathlim -> assert(pathlim), fail )\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.3**\nThe resulting implementation is able to refute a large set of ﬁrst-order formulae\n(but it does not result in a decision procedure for propositional formulae).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\n4.3 Performance Results\n**BLOCK**fs== 10.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\nWe have tested ileanCoP on all 1337 non-clausal (so-called FOF) problems in\nthe TPTP library [27] of version 2.7.0 that are classically valid or are not known\nto be valid/invalid. The tests were performed on a 3 GHz Xeon system running\nLinux and ECLiPSe Prolog version 5.7 (”nodbgcomp.” was used to generate code\nwithout debug information). The time limit for all proof attempts was 300 s.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nThe results of ileanCoP are listed in the last column of Table 4. We have\nalso included the results of the following ﬁve theorem provers for intuitionistic\nﬁrst-order logic: JProver [25] (implemented in ML using a non-clausal connec-\ntion calculus and preﬁxes), the Prolog and C versions of ft [24] (using an intu-\nitionistic tableau calculus with many additional optimisation techniques and a\ncontraction-free calculus [6] for propositional formulae), ileanSeP (using an in-\ntuitionistic tableau calculus; see http://www.leancop.de/ileansep) and ileanTAP\n[22] (using a tableau calculus and preﬁxes). The timings of the classical provers\nOtter [17], leanTAP [1] and leanCoP [19] are provided as well.\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nOf the 234 problems solved by ileanCoP 112 problems could not be solved by\nany other intuitionistic prover. The rating (see rows eight to twelve) expresses\nthe relative diﬃculty of the problems from 0.0 (easy) to 1.0 (very diﬃcult). The\nintuitionistic rating is taken from the ILTP library [23].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\n4.4 Analysing Test Runs\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nThe proof search process of ileanCoP can be divided into the following four\nsections: transformation into clausal form, search for a ”classical” proof (using\nweak preﬁx uniﬁcation), admissibility check and preﬁx uniﬁcation. We mark the\ntransition to these sections with the letters a, b and c as follows:\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nclausal transformation\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.3**r== 0.6**\n”classical”-proof\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.3**r== 0.7**\na\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.4**r== 0.4**\nb\nadmissibility check preﬁx uniﬁcation\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.5**\nc\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.4**\nTable 4. Performance results of ileanCoP and other provers on the TPTP library\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nOtter leanTAP leanCoP ileanSeP JProver ftP rolog\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.2**r== 0.8**\n-\n-\n-\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.3**r== 0.6**\n\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.3**r== 0.7**\n-\n-\n-\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.2**r== 0.8**\n-\n-\n-\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.9**\nsolved\nproved\nrefuted\n0 to <1s\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nto <10s\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nto <100s\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nto <300s\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.9**\nintui. 0.0\nto ≤0.7\nto ≤1.0\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nclass. 0.0\nto ≤1.0\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\ntime out\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nerror\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nftC\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.4**\nileanTAP ileanCoP\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.2**r== 0.5**\nTable 5. Analysis of the TPTP test runs\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.4**\na=b=c=1 a=b=1; c>1 a=1; b,c>1 a=1; b≥1; c=0 a=1; b=c=0 a=b=c=0\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nproved\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nvar(ø,max)\npre(ø,max)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nrefuted\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nvar(ø,max)\npre(ø,max)\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\ntime out\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\nvar(ø,max)\npre(ø,max)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.2**r== 0.8**\n(13,216)\n(29,1191)\n\n(2,2)\n(3,3)\n\n(26,34)\n(15,19)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.3**r== 0.7**\n(21,62)\n(22,119)\n-\n-\n-\n\n(25,106)\n(21,99)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.3**r== 0.6**\n(11,23)\n(13,26)\n\n(3,5)\n(4,4)\n\n(41,146)\n(40,112)\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.4**r== 0.5**\n-\n-\n-\n\n(5,5)\n-\n\n(47,70)\n-\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.5**\n-\n-\n-\n\n-\n-\n\n-\n-\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.6**r== 0.4**\n-\n-\n-\n-\n-\n-\n\n-\n-\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nTable 5 contains information about how often each section is entered when run-\nning ileanCoP on the 1337 TPTP problems. The numbers assigned to the letters\nexpress the number of times this section is entered. For example the column\na=b=c=1 means that each section is entered just once. The rows var(ø,max)\ncontain the average and maximum number of variables collected when enter-\ning section b, whereas the rows pre(ø,max) contain the average and maximum\nnumber of preﬁx equations collected when entering section c.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\nMost of the problems (178 formulae) have been proved without backtrack-\ning in sections a/b (1st column) or section a (2nd column). This means that\nthe corresponding classical sequent proofs can be converted into intuitionistic\nsequent proofs by simply reordering the rules. Most of the refuted problems (40\nformulae) failed because of weak preﬁx uniﬁcation in section a (5th column).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nFor 987 problems no ”classical” proof could be found (5th column). For 62\nproblems (6th column) the size of the clausal form is too big and the trans-\nformation results in a stack overﬂow (”error” in Table 4). For 31 problems\nsections b and c did not ﬁnish after a ”classical” proof has been found (2nd\ncolumn).\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n5 Conclusion\n**BLOCK**fs== 10.0**b== 0.5**t== 0.3**l== 0.1**r== 0.3**\nWe have presented a clausal matrix characterisation for intuitionistic logic based\non the standard characterisation [30] (see also [29]). Encoding the tree-ordering\nby an additional substitution replaces the reﬂexivity test of the reduction or-\ndering by checking if the (extended) combined substitution is idempotent. Due\nto this Skolemization technique and the transformation into clausal form, mul-\ntiplicities can be restricted to clauses and clause instances can be generated\nby just renaming the term and preﬁx variables of the original clauses. This al-\nlows the use of existing clausal calculi and implementations for classical logic\n(in contrast to specialised calculi [28]), without the redundancy caused by (re-\nlational) translations [18] into classical ﬁrst-order logic. Instead an additional\npreﬁx uniﬁcation is added, which captures the speciﬁc restrictions of intuition-\nistic logic, i.e.\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nﬁrst-order logic\nintuitionistic logic = classical logic\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.3**r== 0.4**\n= propositional logic + term uniﬁcation ,\n+ preﬁx uniﬁcation .\n**BLOCK**fs== 10.0**b== 0.3**t== 0.6**l== 0.1**r== 0.3**\nWe have adapted a clausal connection calculus and transformed the classical\nprover leanCoP into the intuitionistic prover ileanCoP with only minor addi-\ntions. Experimental results on the TPTP library showed that the performance\nof ileanCoP is signiﬁcantly better than any other (published) automated theorem\nprover for intuitionistic ﬁrst-order logic available today. The correctness proof\nprovides a way to convert the clausal matrix proofs back into non-clausal matrix\nproofs, which can then be converted into sequent-style proofs [26].\n**BLOCK**fs== 10.0**b== 0.2**t== 0.7**l== 0.1**r== 0.3**\nWhile a clausal form technically simpliﬁes the proof search procedure, it has\na negative eﬀect on the size of the resulting clausal form and the search space.\nA non-clausal connection-based proof search, which increases multiplicities in a\ndemand-driven way, would not suﬀer from this weakness. Whereas the used preﬁx\nuniﬁcation algorithm is very straightforward to implement, it can be improved\nby solving the set of preﬁx equations in a more simultaneous way. The clausal\ncharacterisation, the clausal connection calculus and the implementation can\nbe adapted to all modal logics considered within the matrix characterisation\nframework [30,21,11] by changing the uniﬁcation rules [20]. By adding tests for\nthe linear and relevant constraints within the complementary condition it can\neven be adapted to fragments of linear logic [10,11,15].\n**BLOCK**fs== 12.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nAcknowledgements\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\nThe author would like to thank Thomas Raths for providing the performance\nresults for the listed classical and intuitionistic theorem proving systems.\n**BLOCK**fs== 12.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nReferences\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\n1. B. Beckert, J. Posegga.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.3**r== 0.3**\nleanTAP : lean, tableau-based theorem proving. 12th\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nCADE , LNAI 814, pp. 793–797, Springer, 1994.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\n2. Y. Bertot, P. Cast´eran. Interactive Theorem Proving and Program Develop-\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nment. Texts in Theoretical Computer Science, Springer, 2004.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\n3. W. Bibel. Matings in matrices. Communications of the ACM , 26:844–852, 1983.\n4. W. Bibel. Automated Theorem Proving. Vieweg, second edition, 1987.\n5. R. L. Constable et. al. Implementing Mathematics with the NuPRL proof de-\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nvelopment system. Prentice Hall, 1986.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\n6. R. Dyckhoff. Contraction-free sequent calculi for intuitionistic logic. Journal of\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\nSymbolic Logic, 57:795–807, 1992.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.3**\n7. M. C. Fitting. First-Order Logic and Automated Theorem Proving. Springer,\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\n1990.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\n8. G. Gentzen. Untersuchungen ¨uber das logische Schließen. Mathematische\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nZeitschrift, 39:176–210, 405–431, 1935.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.3**\n9. R. H¨ahnle, P. Schmitt The Liberalized δ-rule in free variable semantic tableaux.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nJournal of Automated Reasoning, 13:211–221, 1994.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.3**\n10. C. Kreitz, H. Mantel, J. Otten, S. Schmitt Connection-based proof con-\nstruction in linear logic. 14th CADE , LNAI 1249, pp. 207–221, Springer, 1997.\n11. C. Kreitz, J. Otten. Connection-based theorem proving in classical and non-\nclassical logics. Journal of Universal Computer Science, 5:88–112, Springer, 1999.\n12. R. Letz, J. Schumann, S. Bayerl, W. Bibel. Setheo: A high-performance\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.4**\ntheorem prover. Journal of Automated Reasoning, 8:183–212, 1992.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\n13. R. Letz, G. Stenz Model elimination and connection tableau procedures. Hand-\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nbook of Automated Reasoning, pp. 2015–2114, Elsevier, 2001.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\n14. D. Loveland. Mechanical theorem proving by model elimination. JACM , 15:236–\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\n251, 1968.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\n15. H. Mantel, J. Otten. linTAP: A tableau prover for linear logic. 8th TABLEAUX\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nConference, LNAI 1617, pp. 217–231, Springer, 1999.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\n16. A. Martelli, U. Montanari. An eﬃcient uniﬁcation algorithm. ACM Transac-\ntions on Programming Languages and Systems (TOPLAS), 4:258–282, 1982.\n17. W. McCune. Otter 3.0 reference manual and guide. Technical Report ANL-94/6,\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nArgonne National Laboratory, 1994.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.3**\n18. H. J. Ohlbach et al. Encoding two-valued nonclassical logics in classical logic.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nHandbook of Automated Reasoning, pp. 1403–1486, Elsevier, 2001.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\n19. J. Otten, W. Bibel. leanCoP: lean connection-based theorem proving. Journal\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nof Symbolic Computation, 36:139–161, 2003.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.3**\n20. J. Otten, C. Kreitz. T-string-uniﬁcation: unifying preﬁxes in non-classical proof\nmethods. 5th TABLEAUX Workshop, LNAI 1071, pp. 244–260, Springer, 1996.\n21. J. Otten, C. Kreitz. A uniform proof procedure for classical and non-classical\nlogics. KI-96: Advances in Artiﬁcial Intelligence, LNAI 1137, pp. 307–319,\nSpringer, 1996.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.3**\n22. J. Otten. ileanTAP: An intuitionistic theorem prover. 6th TABLEAUX Confer-\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nence, LNAI 1227, pp. 307–312, Springer, 1997.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\n23. T. Raths, J. Otten, C. Kreitz. The ILTP library: benchmarking automated\ntheorem provers for intuitionistic logic. TABLEAUX 2005 , this volume, 2005.\n(http://www.iltp.de)\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.3**\n24. D. Sahlin, T. Franzen, S. Haridi. An intuitionistic predicate logic theorem\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nprover. Journal of Logic and Computation, 2:619–656, 1992.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.3**\n25. S. Schmitt et. al. JProver: Integrating connection-based theorem proving into\ninteractive proof assistants. IJCAR-2001 , LNAI 2083, pp. 421–426, Springer, 2001.\n26. S. Schmitt, C. Kreitz Converting non-classical matrix proofs into sequent-style\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nsystems. 13th CADE , LNAI 1104, pp. 2–16, Springer, 1996.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.3**\n27. G. Sutcliﬀe, C. Suttner. The TPTP problem library - CNF release v1.2.1. Journal\nof Automated Reasoning, 21: 177–203, 1998. (http://www.cs.miami.edu/∼tptp)\n28. T. Tammet. A resolution theorem prover for intuitionistic logic. 13th CADE ,\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nLNAI 1104, pp. 2–16, Springer, 1996.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.3**\n29. A. Waaler Connections in nonclassical logics. Handbook of Automated Reasoning,\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.7**\npp. 1487–1578, Elsevier, 2001.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.4**\n30. L. Wallen. Automated deduction in nonclassical logic. MIT Press, 1990.",
         "extracted",
         "",
         "Clausal Connection-Based Theorem Proving in Intuitionistic First-Order Logic",
         "None"
        ],
        [
         "41",
         "002fe46a688bf95a086a415278e9bff1496107b5",
         "None",
         "S. Moody,D. Perez,Tien-Chi Pan,C. J. Sarkisian,C. P. Portocarrero,Christopher J. Sterner,Kathleen L Notorfrancesco,R. Cardiff,L. Chodosh",
         "http://www.cell.com/article/S153561080500231X/pdf",
         "\n**BLOCK**fs== 16.9**b== 0.8**t== 0.1**l== 0.1**r== 0.2**\nThe transcriptional repressor Snail promotes mammary\ntumor recurrence\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nSusan E. Moody,1 Denise Perez,1 Tien-chi Pan,1 Christopher J. Sarkisian,1 Carla P. Portocarrero,1\nChristopher J. Sterner,1 Kathleen L. Notorfrancesco,1 Robert D. Cardiff,2 and Lewis A. Chodosh1,*\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\n1Department of Cancer Biology\nDepartment of Cell and Developmental Biology\nDepartment of Medicine\nAbramson Family Cancer Research Institute, University of Pennsylvania School of Medicine, Philadelphia, Pennsylvania 19104\n2 University of California Davis Center for Comparative Medicine, University of California, Davis, Davis, California 95616\n*Correspondence: chodosh@mail.med.upenn.edu\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nSummary\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nBreast cancer recurrence is a fundamental clinical manifestation of tumor progression and represents the principal cause\nof death from this disease. Using a conditional transgenic mouse model for the recurrence of HER2/neu-induced mam-\nmary tumors, we demonstrate that the transcriptional repressor Snail is spontaneously upregulated in recurrent tumors\nin vivo and that recurrence is accompanied by epithelial-to-mesenchymal transition (EMT). Consistent with a causal role\nfor Snail in these processes, we show that Snail is sufficient to induce EMT in primary tumor cells, that Snail is sufficient\nto promote mammary tumor recurrence in vivo, and that high levels of Snail predict decreased relapse-free survival in\nwomen with breast cancer. In aggregate, our observations strongly implicate Snail in the process of breast cancer recur-\nrence.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.9**\nIntroduction\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nBreast cancer is the most common malignancy diagnosed\namong women worldwide and is the leading cause of cancer\nmortality (Parkin et al., 2005). In 2005, more than 1.1 million\nwomen will be diagnosed with breast cancer, and over 400,000\nwomen will die from this disease. While breast cancer inci-\ndence has steadily increased in Western countries, breast can-\ncer mortality rates have declined for more than a decade such\nthat nearly 90% of women diagnosed with breast cancer sur-\nvive for at least 5 years. As a consequence of its high incidence\nand favorable prognosis, breast cancer is the most prevalent\ncancer in the world today (Parkin et al., 2005). Among these\nwomen, tumor dormancy followed by recurrence—local, re-\ngional, or distant—represents the most common cause of\nbreast cancer mortality.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nAlthough recurrence generally occurs relatively early, be-\ntween 1 and 2 years after surgery (Demicheli et al., 2004;\nSaphner et al., 1996), a considerable proportion of breast can-\ncers that appear cured resurface as local or distant tumor re-\ncurrences 10 or 20 years after surgery (Fisher et al., 2004;\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nWeiss et al., 2003). This suggests that in many cases tumor\ncells have already disseminated to distant sites by the time that\nbreast cancers are diagnosed. Indeed, analysis of bone mar-\nrow specimens indicates that residual cancer cells are detecta-\nble in up to 40% of primary breast cancer patients who lack\nclinical or histopathological signs of metastasis (Pantel et al.,\n2003). These and other observations argue that breast cancer\nrecurrence represents a major obstacle to curing this disease.\nDespite the central role of recurrence in breast cancer mor-\ntality, little is known about the cellular or molecular events re-\nsponsible. A limited number of clinical and molecular charac-\nteristics of breast cancers have been shown to correlate with\nrelapse-free survival. In women with breast cancer, the most\nimportant factors for predicting recurrence are tumor size and\nthe extent of lymph node involvement (Carter et al., 1989; Vala-\ngussa et al., 1978). In addition, a number of molecular markers\nfor aggressive tumor behavior can be used to identify breast\ncancer patients at high risk for recurrence, including HER2/neu\nexpression, c-myc amplification, and estrogen receptor (ER)\nnegativity (Esteva and Hortobagyi, 2004; Schlotter et al., 2003).\nHowever, neither these nor other molecular prognostic markers\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.4**r== 0.4**\nS I G N I F I C A N C E\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.1**\nBreast cancer is the most common malignancy diagnosed among women worldwide and is the leading cause of cancer mortality.\nAmong the more than 5 million women currently living with a diagnosis of breast cancer, recurrence represents the most common\ncause of death from this disease. Nevertheless, while recurrence constitutes a problem of unrivaled clinical importance, little is\nknown about the mechanisms underlying it. Our findings demonstrate that Snail is sufficient to promote mammary tumor recurrence\nin mice and that its expression is associated with rapid tumor recurrence in women independently of established prognostic factors.\nThese observations provide in vivo genetic evidence for a molecular pathway contributing to mammary tumor recurrence and\nsuggest that Snail may represent a target for cancer therapy.\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.1**r== 0.3**\nCANCER CELL : SEPTEMBER 2005 · VOL. 8 · COPYRIGHT © 2005 ELSEVIER INC. DOI 10.1016/j.ccr.2005.07.009\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nhave been shown to play a causal role in breast cancer recur-\nrence. As such, their mechanistic relationship to this process\nremains speculative.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nElucidating the molecular mechanisms that allow tumors to\nevade primary therapy and recur is a critical goal of breast can-\ncer research. In particular, understanding the biology of tumor\nlatency and recurrence would permit improvements in the pre-\ndiction, prevention, and treatment of breast cancer recurrence.\nAchieving this goal, however, has been hampered by the lack\nof animal models that faithfully recapitulate this fundamental\nstep in breast cancer progression. Such models are essential\nfor the rational development and testing of therapeutics tar-\ngeted against the residual population of neoplastic cells re-\nsponsible for the majority of breast cancer deaths.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nFurther compounding difficulties in understanding recur-\nrence is the limited availability of clinical material for analysis.\nWhile molecular profiles for primary human breast cancers\nhave become widely available, no comprehensive molecular\nanalysis of recurrent human breast cancers currently exists.\nConsequently, not only do we lack information on pathways\ncausally involved in recurrence, but we also lack a basic under-\nstanding of the specific molecular features that distinguish re-\ncurrent breast cancers from the primary tumors from which\nthey arose.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nOne molecular prognostic marker for poor clinical outcome\nin breast cancer patients is the proto-oncogene HER2/neu.\nAmplification and overexpression of this receptor tyrosine ki-\nnase occurs in 15%–30% of primary human breast cancers\nand is associated with aggressive tumor behavior, high rates\nof relapse, and poor prognosis (Berger et al., 1988; Slamon et\nal., 1987). In recent years trastuzumab (Herceptin), a neutraliz-\ning antibody that inhibits the activity of HER2/neu, has been\ntested in clinical trials for patients with HER2/neu-amplified\nbreast cancers. The efficacy of this agent in slowing disease\nprogression and prolonging survival, even in advanced stages\nof disease, has been demonstrated in multiple studies (Baselga\net al., 1996; Slamon et al., 2001; Vogel et al., 2002; Wang et\nal., 2001). However, even in cases in which trastuzumab is\ncombined with standard chemotherapeutic regimens, breast\ncancers typically become resistant to therapy and recur (Horto-\nbagyi, 2001). As with cancer recurrence in general, the mecha-\nnisms by which HER2/neu-amplified breast tumor cells evade\nthe blockade of this pathway are poorly understood.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nGiven the importance of the proto-oncogene HER2/neu in\nhuman breast cancers, we have used a mammary-specific,\nconditional transgenic model to investigate the effects of neu\npathway downregulation on the regression and recurrence of\nmammary tumors induced by this oncogene. Specifically, we\nhave used the tetracycline-regulatory system to inducibly ex-\npress an activated form of neu in the mammary epithelium of\ntransgenic mice (Moody et al., 2002). When treated with doxy-\ncycline, these mice develop multiple invasive mammary adeno-\ncarcinomas that regress to a nonpalpable state upon targeted\ndownregulation of the neu pathway. However, consistent with\nthe behavior of human malignancies, the vast majority of mice\nharboring fully regressed tumors ultimately develop recur-\nrences in the absence of doxycycline treatment and neu ex-\npression. As such, this model recapitulates key features of the\nnatural history of human breast cancers relevant to tumor re-\ncurrence.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nWe have employed this conditional transgenic mouse model\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nto investigate secondary pathways involved in breast cancer\nprogression and escape. We now report the identification of a\nmolecular pathway involved in spontaneous mammary cancer\nrecurrence. We show that recurrent neu-induced mammary tu-\nmors display phenotypic and molecular alterations characteris-\ntic of the epithelial-to-mesenchymal transition (EMT) and spon-\ntaneously upregulate expression of Snail, a transcriptional\nrepressor previously implicated in EMT. We further show that\nSnail is sufficient both to induce EMT in neu-induced primary\ntumor cells and to promote rapid tumor recurrence in vivo fol-\nlowing downregulation of the neu pathway.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nFinally, analysis of four independent microarray expression\ndata sets derived from prospectively harvested human breast\ncancer samples revealed that high levels of Snail expression\nstrongly predict decreased relapse-free survival in women with\nbreast cancer independently of most currently used prognostic\nindicators for breast cancer. These studies identify a molecular\npathway involved in mammary tumor recurrence in vivo and\nsuggest that Snail may play a role in the progression of human\nbreast cancers.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nResults\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nFully regressed neu-induced tumors recur spontaneously\nfollowing a period of dormancy\nWe previously generated a doxycycline-inducible bitransgenic\nmouse model for HER2/neu-induced mammary carcinogene-\nsis, referred to as MMTV-rtTA/TetO-NeuNT (MTB/TAN) (Moody\net al., 2002). When expression of this activated form of neu\nis induced with doxycycline, MTB/TAN mice develop multiple\ninvasive mammary adenocarcinomas, many of which metasta-\nsize to the lung (Moody et al., 2002). Downregulation of neu\nexpression in the vast majority of these fully formed mammary\ntumors results in their regression to a nonpalpable state. How-\never, following a latent period, some mice bearing fully re-\ngressed tumors develop spontaneous tumor recurrences in the\nabsence of neu expression.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nTo investigate the mechanisms responsible for the recur-\nrence of neu-induced mammary tumors, we first generated a\nlarge cohort of tumor-bearing mice. Of 507 neu-induced pri-\nmary mammary tumors monitored in 62 MTB/TAN mice, 493\n(97%) regressed to a nonpalpable state following doxycycline\nwithdrawal and downregulation of the neu pathway. Following\ndoxycycline withdrawal, two neu-induced primary tumors failed\nto regress, whereas an additional 11 primary tumors regressed\npartially and then rapidly resumed neu-independent growth.\nThus, in nearly all cases the vast majority of cells within neu-\ninduced primary tumors remain dependent upon neu for main-\ntenance of the transformed state.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nFifty mice in which all tumors had regressed to a nonpalpa-\nble state following doxycycline withdrawal were monitored for\nextended periods of time in the absence of doxycycline. Over\na 1 year period, 43 (86%) of these mice spontaneously devel-\noped recurrent tumors with a mean latency of 117 days (Figure\n1A). Formally, doxycycline-independent tumors arising in mice\nthat had previously harbored mammary tumors could represent\neither genuine recurrences of neu-initiated tumors or the de novo\nformation of tumors in the absence of doxycycline. Our re-\npeated observations that uninduced MTB/TAN animals do not\ndevelop tumors, even over periods exceeding 18 months, and\nthat recurrent tumors always appear at a site at which a pri-\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.7**r== 0.1**\nFigure 1. Fully regressed neu-induced tumors re-\ncur spontaneously following a latent period\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.7**r== 0.1**\nA: Recurrence-free survival for never-induced\nMTB/TAN mice (n = 13) and MTB/TAN mice dein-\nduced after developing primary tumors on dox-\nycycline (n = 62) (SD 48, range 27–222).\nB: Timing of tumor regression and recurrence for\nindividual neu-induced tumors. The time of dox-\nycycline withdrawal (day 0) from tumor-bearing\nmice is indicated by an arrow. A bracket indi-\ncates the range of times (relative to doxycy-\ncline withdrawal) at which neu was initially in-\nduced by doxycycline treatment.\nC: Luciferase imaging of tumor-bearing MTB/\nTAN mice demonstrating that recurrent tumors\ndo not express the TAN transgene in the ab-\nsence of doxycycline. A mouse with multiple pri-\nmary tumors (arrows) on doxycycline is shown\nwith (B) and without (A) luciferase activity over-\nlay. The same mouse after full regression of tu-\nmors off doxycycline is shown with luciferase ac-\ntivity overlay (C). A second mouse with a\nrecurrent tumor in mammary gland 3R (arrow)\nlacking luciferase activity is shown in D.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nmary tumor had previously existed, suggest that these doxycy-\ncline-independent tumors represent bona fide recurrences.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nTo confirm that doxycycline-independent recurrent tumors\narise from cells within the original primary tumor, we implanted\nsmall fragments of primary tumors from MTB/TAN mice onto\nthe flanks of wild-type mice. Grafted hosts were initially main-\ntained on doxycycline to permit tumor outgrowth, after which\ntime doxycycline was withdrawn. Similar to the behavior of\nprimary mammary tumors in MTB/TAN mice, grafted tumors\nregressed to a nonpalpable state following doxycycline with-\ndrawal, and a subset of these regressed tumor grafts eventu-\nally recurred at the original site following a latent period. Since\ntumors in nontransgenic hosts could only arise from grafted\ncells, these data confirm that doxycycline-independent tumors\nthat arise in MTB/TAN mice harboring fully regressed tumors\nare in fact recurrences derived from cells within the primary\ntumor. This conclusion is further strengthened by our subse-\nquent demonstration that epithelial cell cultures derived from\nprimary neu-induced mammary tumors also give rise to recur-\nrences when subjected to a similar grafting protocol\n(see\nbelow).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nAnalysis of mammary tumor behavior in intact MTB/TAN\nmice revealed that recurrent tumors arose stochastically over\nan 8 month period, frequently following a prolonged latency\nperiod (Figure 1B). The concept of tumor latency in breast can-\ncer patients arises in part from the inability of continuous\ngrowth models to explain the kinetics of tumor recurrence in\npatients, as well as the lack of a relationship between the\nlength of time from surgery to tumor recurrence and the growth\nrate of tumors once they reappear. Analogous to the situation\nin humans, the latency for tumor recurrence in MTB/TAN mice\nwas considerably longer than that observed for primary tumor\ndevelopment following neu induction (mean latency, 117 ver-\nsus 42 days, respectively; Figure 1A and Moody et al., 2002).\nSimilarly, the growth rate of recurrent tumors once they reap-\npeared was unrelated to the length of time between tumor re-\ngression and tumor recurrence (Figure 1B). Thus, as with breast\ncancer patients, the timing of tumor recurrence and the kinetics\nof recurrent tumor growth cannot be explained by a model\npostulating constant growth of residual tumor cells. In aggre-\ngate, our observations strongly suggest that deinduced MTB/\nTAN mice harbor residual neoplastic cells at the sites of their\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\noriginal tumors that persist in a latent state for variable periods\nof time before reemerging as recurrent disease.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nFully regressed neu-induced tumors recur\nin the absence of neu expression\nIn theory, doxycycline-independent tumor recurrences could\nresult from doxycycline-independent activation of the NeuNT\ntransgene, compensatory upregulation of endogenous ErbB2,\nor activation of neu-independent growth and/or survival path-\nways. Since the TetO-NeuNT transgene contains a bicistronic\nIRES-firefly luciferase cassette, transgene expression can be\nmonitored longitudinally in mice by noninvasive in vivo imaging\nof luciferase. As predicted, luciferase activity was readily visu-\nalized within primary tumors in MTB/TAN mice maintained on\ndoxycycline, whereas mice maintained off doxycycline harbor-\ning either fully regressed or recurrent tumors did not express\ndetectable luciferase activity (Figure 1C). In addition, Northern\nanalysis as well as anti-ErbB2 immunohistochemistry failed to\ndetect upregulation of endogenous ErbB2 in recurrent tumors\n(Figure 2C and data not shown). These observations indicate\nthat spontaneous recurrence of neu-induced mammary tumors\noccurs by a process other than the doxycycline-independent\nreactivation of the NeuNT transgene or compensatory upregu-\nlation of endogenous ErbB2.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nTaken together, these results suggest that a subset of cells\nin neu-induced primary tumors are able to progress to a state\nthat is independent of neu overexpression for survival and\ngrowth. Moreover, the fact that MTB/TAN mice bearing fully\nregressed tumors typically relapse following a latency of up to\n8 months indicates that these mice harbor viable residual neo-\nplastic cells that persist in the mammary gland for extended\nperiods of time.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nRecurrent tumors display characteristics of EMT\nNeu-induced mammary adenocarcinomas in MTB/TAN mice\nmaintained on doxycycline display a characteristic epithelial\nmorphology and stain positively for\nthe luminal epithelial\nmarker cytokeratin 8 (CK8) (Moody et al., 2002). Since mam-\nmary tumors induced in mice by different oncogenic pathways\nhave been shown to exhibit distinct histopathological “signa-\ntures,” we reasoned that doxycycline-independent tumor re-\ncurrences that no longer expressed neu might not exhibit the\nclassic neu phenotype (Cardiff et al., 1991). In agreement with\nthis prediction, histopathological examination of multiple neu-\nnegative, doxycycline-independent recurrent tumors revealed\nthat the vast majority of these tumors were composed of spin-\ndle-shaped cells with mesenchymal morphology (Figure 2A).\nFurthermore,\nimmunohistochemical analysis demonstrated\nthat recurrent tumors had downregulated CK8, although occa-\nsional CK8-positive cells were detected (Figure 2A).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThese findings led us to hypothesize that neu-induced mam-\nmary tumors undergo EMT prior to their reemergence as neu-\nindependent recurrent tumors. Consistent with this hypothesis,\nNorthern analysis confirmed the absence of NeuNT transgene\nexpression in recurrent tumors and further revealed that ex-\npression of the mesenchymal markers vimentin and fibronectin\nwere upregulated (Figure 2C). Conversely, a striking reduction\nin expression of the epithelial marker E-cadherin was observed\nin virtually every neu-independent recurrent tumor (Figure 2C).\nE-cadherin is an essential component of adherens junctions in\nepithelial cells, and its expression is commonly lost in EMT.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nImmunofluorescence studies confirmed that recurrent tumors\ndownregulate expression of CK8 and E-cadherin and upregu-\nlate expression of the mesenchymal marker S100A4 (fibro-\nblast-specific protein [Fsp1]) (Figure 2D). These findings de-\nmonstrate that\ntumors display multiple features\ncharacteristic of cells that have undergone EMT.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nrecurrent\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nMesenchymal recurrences arise from primary\nepithelial tumors\nSince primary neu-dependent mammary tumors and doxycy-\ncline-independent tumor recurrences exhibit strikingly different\nmorphological phenotypes, we wished to confirm that mes-\nenchymal recurrences arise from primary epithelial tumors. Two\nlines of evidence indicate that this is indeed the case. First, we\nperformed both Northern and histological analysis on the small\nnumber of neu-induced primary mammary tumors that re-\ngressed only partially following doxycycline withdrawal before\nresuming growth. This analysis revealed that incompletely re-\ngressing doxycycline-independent primary tumors, which remain\nclinically apparent throughout their period of partial regression\nand regrowth, display the same spindle cell phenotype and\nidentical molecular characteristics with respect to markers of\nEMT as doxycycline-independent recurrences (Figure 2C and\ndata not shown). This suggests that the same cellular pro-\ncesses that lead to tumor recurrence also contribute to the de-\nvelopment of neu-independent primary tumors.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nSecond, primary MTB/TAN tumors grafted onto the flanks of\nwild-type mice were allowed to engraft and grow on doxycy-\ncline before being induced to regress to a nonpalpable state\nby withdrawal of doxycycline treatment. Grafted mice bearing\nfully regressed tumors were monitored for spontaneous recur-\nrence in the absence of doxycycline. Primary tumor grafts were\nbiopsied prior to doxycycline withdrawal, and doxycycline-\nindependent tumors that eventually recurred at graft sites were\nharvested. Histological analysis of these matched sets of pri-\nmary and recurrent tumor grafts demonstrated that primary tu-\nmor grafts displayed the classic neu epithelial phenotype,\nwhereas doxycycline-independent recurrences that arose at\nthe same sites had acquired a mesenchymal phenotype (Figure\n2B). Taken together, these two lines of evidence indicate that\nmesenchymal-appearing, doxycycline-independent tumors that\narise at the site of fully regressed neu-induced neoplasms consti-\ntute recurrences of epithelial tumors that have undergone EMT.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nSnail, an inducer of EMT, is spontaneously upregulated\nin recurrent mammary tumors\nA major regulator of EMT during embryonic mesoderm and\nneural crest development is the zinc finger transcription factor\nSnail. First identified in Drosophila mutant embryos exhibiting\ndefective mesoderm invagination, Snail is required for normal\nmesoderm development in mice (Grau et al., 1984; Carver et\nal., 2001). In addition, Snail directly represses E-cadherin tran-\nscription in both mouse and human epithelial cell lines (Batlle\net al., 2000; Cano et al., 2000). Similarly, both Snail and its\nclosely related family member Slug can repress endogenous\nE-cadherin expression in human breast cancer cell lines (Hajra\net al., 2002). Consistent with this, invasive ductal carcinomas\nof the breast have been reported to express Snail in a manner\nthat is inversely correlated with E-cadherin expression (Cheng\net al., 2001).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nIn light of the association of Snail with EMT, we wished to\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.7**r== 0.1**\nA: H&E- and anti-CK8-stained sections from re-\npresentative MTB/TAN primary (neu-expressing)\nand MTB/TAN recurrent (neu-negative) tumors\ndemonstrating the transition from epithelial to\nmesenchymal morphology and marker expres-\nsion. Magnification, 400×.\nB: H&E-stained sections of two primary tumor\ngrafts allowed to engraft on doxycycline and\nbiopsied while on doxycycline. Tumors either\npartially regressed and then resumed growth in\nthe absence of doxycycline (T1) or completely\nregressed and recurred at the same site (T2).\nMagnification, 400×.\nC: Northern analysis of primary and recurrent\nMTB/TAN tumors for expression of epithelial and\nmesenchymal markers. rRNA (18S) is shown as a\nloading control.\nD:\nImmunofluorescence (IF) analysis of CK8,\nE-cadherin, and S100A4 expression in represen-\ntative primary and recurrent tumors. Magnifica-\ntion, 400×.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\ndetermine whether Snail was upregulated in recurrent, neu-\ninduced mammary tumors that exhibited features of EMT.\nMicroarray analysis of doxycycline-independent MTB/TAN tu-\nmors demonstrated a 9-fold increase in Snail expression com-\npared to primary MTB/TAN tumors (p = 8 × 10−6), whereas only\na 1.8-fold change in expression was detected in its closely re-\nlated family member, Slug (p = 10−4)\n(Figure 3A). Northern\nanalysis confirmed that recurrent tumors express increased\nlevels of Snail compared to primary tumors (Figure 3B). Simi-\nlarly, consistent with their spindle cell appearance, primary\nmammary tumors that regressed only partially prior to resum-\ning growth in a neu- and doxycycline-independent manner also\ndisplayed increased levels of Snail (Figure 3B).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nSnail induces EMT in primary neu-induced\ntumor cells in vitro\nSince we had observed a correlation between Snail expression\nand acquisition of a mesenchymal phenotype in doxycycline-\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nindependent tumors, we wished to test whether Snail could\ninduce EMT in primary neu-induced mammary tumor cells. Al-\nthough several studies in epithelial cell\nlines have demon-\nstrated that Snail expression can induce EMT, some normal\nand malignant epithelial cell lines have proven to be resistant\nto this phenomenon in vitro (Batlle et al., 2000; Cano et al.,\n2000; Zhou et al., 2004). To determine whether Snail is capable\nof inducing EMT in neu-expressing tumor cells, we transduced\nprimary tumor cells maintained on doxycycline with a retroviral\nvector encoding Snail or with the control retroviral vector pk1.\nInfected cells were selected in puromycin-containing media,\nand immunoblotting was performed to assess Snail expression\nin transduced cells (Figure 3C). Within two weeks of transduc-\ntion, pk1-Snail-infected cells acquired a fibroblastic spindle cell\nphenotype accompanied by loss of cell-to-cell contacts (Figure\n3D). In contrast, pk1-infected control cells retained their epithe-\nlial morphology. These results demonstrate that Snail expres-\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.7**r== 0.1**\nFigure 3. Snail\nis spontaneously upregulated in\nrecurrent mammary tumors and induces EMT in\nprimary neu-induced tumor cells\n**BLOCK**fs== 7.0**b== 0.6**t== 0.2**l== 0.7**r== 0.1**\nA: Microarray analysis of mean Snail and Slug\nexpression levels in primary and recurrent tu-\nmors. Expression levels were significantly dif-\nferent between primary tumors and recurrent\ntumors for both genes (p = 8 × 10−6 for Snail; p =\n0.0001 for Slug). Error bars indicate standard er-\nror of the mean.\nB: Northern analysis of Snail expression in pri-\nmary,\nrecurrent, and incompletely regressing\n(Inc. Reg.) tumors. rRNA (28S) is shown as a load-\ning control.\nC: Western analysis of Snail expression in pk1-\nand pk1-Snail-infected primary tumor cells.\nD: Photomicrographs of pk1- and pk1-Snail-\ninfected primary tumor cells showing spindle\ncell morphology of Snail-expressing cells.\nE: IF analysis of uninfected primary tumor cells,\nuninfected recurrent tumor cells, and primary\ntumor cells infected with pk1 or pk1-Snail. Mag-\nnification, 400× (D and E).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nsion in neu-induced primary tumor cells is sufficient to induce\nthe acquisition of a spindle-shaped morphology.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSince the transition to a spindle-shaped morphology is con-\nsistent with EMT, we next asked whether alterations associated\nwith EMT also occurred at the molecular level. Primary tumor\ncells transduced with pk1- or pk1-Snail-expressing retroviral\nvectors, as well as nontransduced primary and recurrent tumor\ncells, were immunostained with antibodies directed against\nepithelial and mesenchymal markers. As expected, CK8 and\nE-cadherin were expressed in both uninfected and pk1-infected\nprimary tumor cells, whereas expression of these epithelial\nmarkers was lost in cells cultured from recurrent tumors (Figure\n3E). Conversely, the mesenchymal markers S100A4 and fibro-\nnectin were highly upregulated in the recurrent tumor cell line\ncompared to uninfected or pk1-infected primary tumor cells.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nNotably, while complete repression of E-cadherin expression\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nwas observed in primary tumor cells transduced with pk1-Snail,\nCK8 expression was lost in only a fraction of Snail-expressing\nprimary tumor cells (Figure 3E). Similarly, a marked increase in\nS100A4 and fibronectin expression was evident in a subset of\npk1-Snail-infected primary tumor cells. The upregulation of\nmesenchymal markers in only a subset of Snail-transduced\ncells contrasts with the ubiquitous upregulation of mesenchy-\nmal markers in recurrent tumor cells. This differential behavior\nmay be due to the concurrent expression of neu in tumor cell\nlines maintained on doxycycline, since neu may impede Snail’s\nability to induce the full phenotypic manifestations of EMT. Al-\nternately, full manifestation of the EMT phenotype may require\nadditional genetic or epigenetic alterations besides Snail ex-\npression. Nonetheless, complete transition to a mesenchymal\nmorphology occurs in pk1-Snail-infected cells. In aggregate,\nthese observations demonstrate that Snail expression triggers\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nSnail promotes the recurrence of neu-induced\nprimary tumors\nSince virtually all transgene-independent MTB/TAN recurrent\ntumors display a mesenchymal phenotype and express high\nlevels of Snail, we hypothesized that Snail may directly pro-\nmote tumor\nrecurrence. To investigate this possibility, we\ngrafted pk1-Snail- or pk1-transduced primary tumor cells onto\nthe flanks of nude mice maintained on doxycycline. Both sets\nof cells formed tumors with equal efficiency. As predicted, tu-\nmor grafts derived from pk1-transduced primary tumor cells\ndisplayed an epithelial morphology, whereas tumor grafts aris-\ning from pk1-Snail-transduced cells exhibited a mesenchymal\nmorphology despite continued expression of the neu onco-\ngene (Figure 4A). To confirm that tumors derived from pk1-\nSnail-transduced cells do not have altered NeuNT transgene\nexpression prior to the withdrawal of doxycycline, we assayed\nluciferase activity levels in pk1- and pk1-Snail-transduced tu-\nmor grafts harvested from mice maintained on doxycycline. As\nexpected, in mice maintained on doxycycline both pk1-Snail\nand pk1 tumor grafts expressed high, comparable levels of the\nNeuNT-IRES-Luciferase transgene (Figure 4B).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nWhen pk1- and pk1-Snail-transduced tumor grafts reached\na size of 27 mm3, doxycycline was withdrawn. All tumors re-\ngressed to a nonpalpable state following doxycycline with-\ndrawal regardless of Snail expression status. Strikingly, within\n60 days of neu downregulation, 17 of 20 (85%) pk1-Snail-\ntransduced tumors recurred, whereas only 1 of 20 (5%) pk1-\ntransduced tumors recurred (p < 0.0001) (Figure 4C). Moreover,\nsimilar to recurrent tumors that arose in tumor-bearing MTB/\nTAN mice withdrawn from doxycycline, regressed pk1-Snail tu-\nmor grafts that recurred in the absence of doxycycline lacked\ndetectable NeuNT transgene expression (Figure 4B).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nNotably, pk1-Snail-induced recurrent tumors displayed a fi-\nbroblastic phenotype similar both to doxycycline-independent\ntumor recurrences arising in MTB/TAN mice and to pk1-Snail-\ntransduced primary tumor grafts arising in mice maintained on\ndoxycycline (Figure 4A). Consistent with this, immunofluores-\ncence analysis revealed the downregulation of E-cadherin\nexpression and upregulation of S100A4 in both doxycycline-\ndependent pk1-Snail primary tumor grafts and doxycycline-\nindependent pk1-Snail recurrent tumor grafts (Figure 4D). In\ncontrast, high levels of E-cadherin and low levels of S100A4\nwere observed in epithelial-appearing pk1 control tumor grafts\ngrowing in doxycycline-treated mice. Thus, Snail-induced re-\ncurrent tumors are morphologically and molecularly similar to\nMTB/TAN recurrent tumors that arise in intact mice in the ab-\nsence of NeuNT transgene expression. In aggregate, these\ndata demonstrate that Snail promotes the rapid recurrence of\nprimary tumor cells in vivo following downregulation of the\nneu pathway.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSnail expression predicts decreased relapse-free survival\nin women with breast cancer\nOur observation that enforced Snail expression in primary tu-\nmor cells promotes mammary tumor recurrence in mice raised\nthe question of whether Snail might play a similar role in the\nrecurrence of breast cancers in humans. If this were the case,\nwe reasoned that women with primary breast cancers express-\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nFigure 4. Snail promotes the recurrence of neu-induced primary tumors\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nA: H&E-stained sections from tumors formed by pk1- and pk1-Snail-infected\ntumor cells harvested from mice on doxycycline, and from doxycycline-\nindependent, pk1-Snail-transduced recurrent tumors. Magnification, 400×.\nB: Luciferase activity levels in pk1 and pk1-Snail tumors in mice on doxycy-\ncline, and in pk1-Snail recurrent tumors (Rec) arising after doxycycline with-\ndrawal. n = 6 for each tumor type. Error bars indicate standard error of\nthe mean.\nC: Recurrence-free survival for mice engrafted with pk1-infected and pk1-\nSnail-infected primary tumor cells. Mice were removed from doxycycline\non day 0 after the injected tumor cells had formed a tumor w27 mm3 in\nsize. Recurrence rates were significantly higher in pk1-Snail-infected cells\n(p < 0.0001).\nD: IF analysis of tumors formed by pk1- and pk1-Snail-infected tumor cells\nharvested from mice on doxycycline, and of doxycycline-independent,\npk1-Snail-transduced recurrent tumors. Magnification, 400×.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\ning high levels of Snail might relapse at a faster rate than\nwomen whose breast cancers expressed lower levels of Snail.\nTherefore, we examined four microarray expression data sets\nderived from the analysis of primary human breast cancers in\nwhich both Snail expression levels and clinical outcome were\nreported. For each data set, tumor samples were classified as\neither high or low Snail expressers based on array hybridization\nlevels (for two-color arrays) or absolute expression analysis p\nvalues (for Affymetrix arrays). The relationship between Snail\nexpression level and 5 year relapse-free survival was then de-\ntermined.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nIn the first series of samples examined, Sorlie et al. profiled\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nFigure 5. Snail expression predicts decreased relapse-free survival\nwomen with breast cancer\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nin\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFive year relapse-free survival for human tumors with low versus high Snail\nexpression in microarray data sets representing the following. A: Locally\nadvanced human breast cancers (Sorlie et al., 2003); B: lymph node-nega-\ntive human breast cancers (van’t Veer et al., 2002); C: lymph node-nega-\ntive, ER-positive, and ER-negative human breast cancers (Wang et al.,\n2005); and D: hormone receptor-positive human breast cancers (Ma et\nal., 2004).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n74 locally advanced ER-positive and ER-negative primary\nbreast cancers from patients without distant metastases at the\ntime of presentation (Sorlie et al., 2003). Microarrays containing\nprobe sets capable of detecting Snail had been used in analyz-\ning 22 of these 74 samples. Applying the Kaplan-Meier estima-\ntor to high and low Snail-expressing groups revealed a signifi-\ncant correlation between breast cancers expressing high levels\nof Snail and decreased relapse-free survival (p = 0.009) (Fig-\nure 5A).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nA similar analysis of relapse-free survival was performed\nusing microarray data derived from 97 ER-positive and ER-\nnegative breast cancers that were lymph node-negative at pre-\nsentation (van’t Veer et al., 2002). Similar to locally advanced\ntumors, high Snail expression in node-negative human primary\nbreast cancers was significantly associated with decreased\ndistant relapse-free survival (p = 0.021) (Figure 5B). A third\nanalysis of the relationship between Snail expression and the\nrate of distant relapse in 286 patients with lymph node-nega-\ntive breast cancers (Wang et al., 2005) showed a similar signifi-\ncant association (p = 0.029) between breast cancers express-\ning high levels of Snail and decreased relapse-free survival\n(Figure 5C). Finally, we analyzed a fourth set of 60 hormone\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nreceptor-positive breast cancers from patients without distant\nmetastases at the time of tumor resection (Ma et al., 2004).\nOnce again, a trend between high Snail expression and rapid\ntumor recurrence was observed within this patient group, al-\nthough the association did not reach statistical significance\n(p = 0.085) (Figure 5D).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nTaken together, these data demonstrate that high levels of\nSnail expression in human primary breast cancers consistently\npredict decreased relapse-free survival. The demonstration that\nfour different Snail probes on four different microarray platforms\napplied to four different patient sample sets representing 464\npatients each yield the same result strongly suggests that the\nassociation between Snail expression and relapse-free survival\nis not specific to the particular probes used to detect Snail,\nthe particular platforms on which the microarray studies were\nperformed, or the particular characteristics of the patient pop-\nulations that were represented in these studies. Moreover,\nsince a significant association between Snail expression and\nrate of relapse was observed for both locally advanced and\nnode-negative breast cancers, as well as among ER+ and ER−\nbreast cancers, this analysis suggests that Snail expression\nmay be an important prognostic indicator for breast cancer in\na variety of clinical contexts.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe prognostic significance of Snail expression\nis comparable to classic prognostic indices\nHaving determined that Snail expression levels predict relapse-\nfree survival in women with breast cancer, we wished to com-\npare the magnitude of this association with currently used clin-\nical prognostic markers. Accordingly, the Cox proportional\nhazards regression was used to calculate hazard ratios (HR) for\nrelapse based on Snail expression, tumor size, tumor grade,\nlymph node status, ER status, and HER2 status (Table 1). This\nanalysis revealed that the risk of breast cancer relapse associ-\nated with high levels of Snail expression (HR >2-fold) is compa-\nrable to those associated with currently used clinical prognos-\ntic indicators, including that of ER status, tumor grade, and\nlymph node status.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nSnail predicts relapse-free survival independently\nof other prognostic markers\nSince several prognostic markers have previously been estab-\nlished for human breast cancer, we asked whether the prog-\nnostic significance of Snail was attributable to its correlation\nwith other markers of aggressive tumor behavior. Two charac-\nteristics of breast cancers in women that have been found to\nbe strong predictors of relapse-free survival are tumor size at\ndiagnosis and lymph node status (Carter et al., 1989; Vala-\ngussa et al., 1978). Therefore, we analyzed the association be-\ntween Snail expression and tumor size by four methods: con-\ntingency table analysis of binned Snail expression levels versus\nbinned tumor size; Snail expression level as a continuous vari-\nable versus binned tumor size; binned Snail expression versus\ntumor size as a continuous variable; and the correlation be-\ntween Snail expression and tumor size, in which each is ana-\nlyzed as a continuous variable. None of these methods re-\nvealed a significant correlation between Snail expression and\ntumor size in any of the three human breast cancer data sets\nanalyzed (Table 2).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nSimilarly, we attempted to correlate Snail expression with\nlymph node status using both contingency table analysis and\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nTable 1. Hazard ratios for relapse based on clinical prognostic indicators\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nSnail status\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\nTumor size\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nTumor grade\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nSorlie\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nvan’t Veer\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.9**\nWang\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nMa\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.2**r== 0.8**\nHR\n95% CI\np value\nHR\n95% CI\np value\nHR\n95% CI\np value\nHR\n95% CI\np value\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n**\n**\n**\n2.24*\n1.11–4.53\n0.025\n2.04*\n1.06–3.92\n0.034\n2.21\n0.87–5.57\n0.094\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n1.11\n0.49–0.51\n0.810\n1.73*\n1.19–2.53\n0.004\nN/A\nN/A\nN/A\n1.33\n0.80–2.23\n0.269\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n2.02*\n1.13–3.59\n0.017\n3.01*\n1.60–5.68\n0.001\nN/A\nN/A\nN/A\n2.01\n0.94–4.30\n0.073\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nLN status\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n1.13\n0.59–3.03\n0.494\nN/A\nN/A\nN/A\nN/A\nN/A\nN/A\n0.97\n0.44–2.14\n0.947\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.2**\nER status\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.7**r== 0.2**\n2.59*\n1.16–5.79\n0.021\n2.32*\n1.28–4.19\n0.005\n1.18\n0.76–1.85\n0.460\nN/A\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nHER2 status\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.8**r== 0.1**\n3.04*\n1.04–8.86\n0.042\n4.10*\n1.72–9.80\n0.002\n1.06\n0.63–1.80\n0.820\n1.90\n0.45–8.13\n0.385\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nHazard ratios, 95% confidence intervals, and p values for breast cancer relapse within 5 years of surgery, based on Snail expression grouping or previously described\nprognostic factors. Data are presented for each of the four human breast cancer data sets analyzed. HR, hazard ratio; CI, confidence interval; LN, lymph node.\nStatistically significant hazard ratios (p < 0.05) are indicated by asterisks. The hazard ratio is mathematically infinite, since no tumors with low Snail expression recurred.\nNo confidence interval or p value can be generated.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nSnail expression level as a continuous variable versus lymph\nnode status. Again, neither of these methods revealed a signifi-\ncant correlation between Snail expression and lymph node\nstatus in either of the data sets analyzed (Table 2). These find-\nings indicate that Snail expression predicts relapse-free sur-\nvival independently of two of the most commonly used prog-\nnostic markers for breast cancer relapse.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nWe next examined the association between Snail expression\nand other prognostic indicators, including histological tumor\ngrade, ER status, and HER2/neu expression. Using analytical\nmethods similar to those described above, no consistent cor-\nrelation between Snail expression and histological tumor grade\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nwas detected (Table 2). Similarly, a correlation between Snail\nexpression and HER2 expression was observed in only one of\n16 such analyses. We were, however, able to detect a statisti-\ncally significant negative correlation between Snail expression\nand ER expression in two of the four data sets.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nWe then asked whether Snail expression might correlate with\na specific cellular subtype of breast cancer associated with\npoor outcome. Sorlie et al. have previously described array-\nthe molecular classification of human\nbased methods for\nbreast cancers into five subtypes: luminal A, luminal B + C,\nbasal, ERBB2+, and normal breast-like (Sorlie et al., 2001,\n2003). Of these, the basal and ERBB2+ subtypes have been\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nTable 2. p values for association between Snail expression and 5 year relapse-free survival or prognostic indicators\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.8**\nRelapse-free survival\nTumor size\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nTumor grade\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nLymph node status\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.9**\nER status\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nHER2 status\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nDuctal versus lobular\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\nSorlie subtype\nclassification\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.2**r== 0.8**\nAnalysis\n**BLOCK**fs== 7.0**b== 0.1**t== 0.6**l== 0.2**r== 0.5**\nLog-rank test based on Snail status\nSnail status versus size status\nSnail expression versus size status\nSnail status versus size\nSnail expression versus size\nSnail status versus grade status\nSnail expression versus grade status\nSnail status and lymph node status\nSnail expression versus lymph node status\nSnail status versus ER status\nSnail expression versus ER status\nSnail status versus ESR1 expression\nSnail expression versus ESR1 expression\nSnail expression versus ESR1 expression in ER+ tumors\nSnail expression versus ESR1 expression in ER− tumors\nSnail status versus HER2 status\nSnail expression versus HER2 status\nSnail status versus HER2 expression\nSnail expression versus HER2 expression\nSnail status versus histological status\nSnail expression versus histological status\nSnail status versus Sorlie subtype status\nSnail expression versus Sorlie subtype status\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\nSorlie\n**BLOCK**fs== 7.0**b== 0.1**t== 0.6**l== 0.6**r== 0.4**\n0.009*\n0.193\n0.397\n0.335\n0.456\n0.104\n0.010*\n0.648\n0.375\n1.000\n0.601\n0.438\n0.146\n0.166\n0.784\n1.000\n0.505\n0.217\n0.373\n1.000\n0.293\n1.000\n0.334\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\nvan’t Veer\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\nWang\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.8**r== 0.1**\nMa\n**BLOCK**fs== 7.0**b== 0.1**t== 0.6**l== 0.7**r== 0.3**\n0.021*\n0.793\n0.686\n0.908\n0.737\n0.454\n0.965\nN/A\nN/A\n0.636\n0.901\n0.518\n0.806\n0.258\n0.475\n0.660\n0.455\n0.843\n0.933\nN/A\nN/A\n0.520\n0.976\n**BLOCK**fs== 7.0**b== 0.1**t== 0.6**l== 0.8**r== 0.2**\n0.029*\nN/A\nN/A\nN/A\nN/A\nN/A\nN/A\nN/A\nN/A\n<0.0001*\n0.0002*\n<0.0001*\n<0.0001*\n0.0621\n0.0584\n0.7482\n0.3850\n0.0353*\n0.1387\nN/A\nN/A\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.1**t== 0.6**l== 0.8**r== 0.1**\n0.085\n0.367\n0.533\n0.986\n0.522\n0.644\n0.287\n1.000\n0.162\nN/A\nN/A\n0.024*\n0.001*\n<0.001*\nN/A\n0.272\n0.095\n0.707\n0.467\n1.000\n0.994\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nCorrelation between Snail expression and 5 year relapse-free survival, and between Snail expression and previously described prognostic factors for breast cancer. p\nvalues are shown for each of the four human breast cancer data sets analyzed. “Status” designation refers to groupings of samples as described in the Experimental\nProcedures. All other designations refer to the variable as continuous, based either on microarray-determined expression levels or on size. Contingency table analyses\n(group versus group) were tested by Fisher’s exact test. Group versus continuous variable analyses were tested by ANOVA. Correlations between two continuous\nvariables were tested by the p value of the Pearson correlation coefficient. Asterisks indicate statistically significant correlations (p < 0.05).\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nTable 3. Proportional hazards analyses of Snail status and selected covariates\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nSnail status\nTumor grade\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.9**\nSnail status\nTumor size\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nSnail status\nER status\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nSnail status\nHER2 status\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nvan’t Veer\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nHR\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\n2.09*\n1.51*\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.2**r== 0.7**\n2.12*\n1.70*\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n2.18*\n2.15*\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.2**r== 0.7**\n2.06*\n3.68*\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n95% CI\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.4**r== 0.6**\n1.03–4.24\n1.51–5.33\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n1.05–4.29\n1.16–2.49\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n1.08–4.42\n1.18–3.94\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.4**r== 0.6**\n1.01–4.19\n1.53–8.86\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\np value\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\n0.040\n0.001\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n0.037\n0.006\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n0.030\n0.013\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n0.046\n0.004\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nWang\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nHR\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n2.00\n1.04\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n2.07*\n1.12\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n95% CI\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\np value\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n0.99–4.02\n0.65–1.67\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\n1.07–4.00\n0.66–1.89\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.8**r== 0.2**\nN/A\nN/A\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n0.053\n0.876\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n0.031\n0.687\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nProportional hazard ratios, confidence intervals, and p values for 5 year relapse-free survival in the van’t Veer et al. (2002) and Wang et al. (2005) data sets based on\nSnail expression groups controlled for tumor grade, tumor size, ER status, and HER2 status, and for each of these factors controlled for Snail expression. Asterisks\nindicate statistically significant hazard ratios (p < 0.05).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nshown to be associated with the worst prognosis, whereas the\nluminal A subtype has been associated with the best progno-\nsis, in both the Sorlie and van’t Veer data sets. Neither contin-\ngency table analysis nor continuous variable analysis of Snail\nexpression differences between these subtypes revealed any\ncorrelation between Snail expression and tumor subtype clas-\nsification (Table 2). Similarly, for the Sorlie and Ma data sets,\ninformation was provided regarding whether tumors were of\nductal or lobular origin. Again, Snail expression levels were not\ncorrelated with ductal versus lobular tumor type in either data\nset (Table 2). In aggregate, our results indicate that the ability\nof Snail expression levels to predict survival is not attributable\nto its preferential expression in a previously described aggres-\nsive breast cancer subtype.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nFinally, although Snail expression did not correlate with most\npreviously described prognostic markers, we examined whether\nadjusting for any of these factors would alter the ability of Snail\nexpression to predict relapse-free survival. Notably, the associ-\nation between Snail expression and decreased relapse-free\nsurvival remained statistically significant even when adjusted\nindividually for tumor grade, tumor size, ER status, or HER2\nstatus (Table 3). Therefore, although limited correlations were\nobserved between Snail expression and either tumor grade or\nHER2 status in one data set each, and between Snail and ER\nstatus in two data sets, the correlation between elevated Snail\nexpression and decreased relapse-free survival is not attribut-\nable to any of these associations. Overall, these findings de-\nmonstrate that Snail expression strongly predicts relapse-free\nsurvival in women with breast cancer and that this association\nis largely independent of previously described prognostic\nfactors.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nDiscussion\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nTumor recurrence is a cardinal manifestation of breast cancer\nprogression and represents the principal cause of death from\nthis disease. Preventing relapse by eradicating residual neo-\nplastic cells will\nlikely require simultaneous targeting of the\nmultiple pathways that contribute to tumor survival and pro-\ngression. Moreover, with the increasing use of molecularly tar-\ngeted cancer therapies—many of which are cytostatic—the\nidentification of molecular pathways that permit tumor cells to\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nescape blockade of a dominant oncogenic pathway, survive in\na latent state, and eventually reestablish malignant growth has\nbecome even more important. While many animal models de-\nveloped over the past two decades have provided insight into\nthe molecular events leading to tumor development, few have\naddressed the molecular and cellular events that lead to recur-\nrent disease.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nWe now present in vivo genetic evidence for a molecular\nmechanism that contributes to mammary tumor recurrence.\nOur data demonstrate that Snail is spontaneously upregulated\nduring the process of tumor recurrence in intact animals and\nthat tumor recurrence is accompanied by EMT, a cellular transi-\ntion that has been linked to breast cancer progression and in\nwhich Snail has been implicated. Consistent with a causal role\nfor Snail in EMT and breast cancer progression, we have fur-\nther shown that Snail\nis sufficient to induce EMT in primary\ntumor cells and to promote mammary tumor recurrence in vivo\nand that high levels of Snail expression strongly predict de-\ncreased relapse-free survival\nin women with breast cancer.\nThese observations strongly implicate Snail in the process of\nbreast cancer recurrence.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nAlthough we identified Snail on the basis of its role in the\nrecurrence of neu-induced mammary tumors in mice, we have\nalso shown that the ability of Snail to predict relapse-free sur-\nvival in women is not due to its preferential expression within\nany currently recognized aggressive subtype of breast cancer.\nMoreover, its prognostic significance is independent of the\nmost robust currently recognized molecular and cellular mark-\ners. If Snail’s role in human tumor recurrence were restricted to\nHER2/neu-amplified breast cancers, we would not have ex-\npected to observe a general relationship between Snail expres-\nsion and breast cancer recurrence, since HER2/neu-amplified\ntumors represent only a small proportion of the cancers in the\ndata sets examined. Consistent with a more general role for\nSnail\nin human breast cancer recurrence, we have demon-\nstrated that Snail is not preferentially expressed in HER2/neu-\namplified tumors or in other breast cancer subtypes. Since a\nsignificant association between Snail expression and the likeli-\nhood of relapse was observed for both ER-positive and ER-\nnegative breast cancers, HER2/neu-amplified and unamplified\nbreast cancers, and breast cancers with and without lymphatic\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nspread, our data suggest that Snail may be linked to breast\ncancer recurrence in a wide variety of clinical contexts.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nImportantly, the association that we detected between Snail\nexpression and disease relapse in humans was observed in\nprimary tumors, implying that Snail activation within primary\ntumors may play a role in promoting tumor recurrence. Consis-\ntent with this, we have demonstrated that enforced Snail\nupregulation in murine primary tumors is sufficient to promote\ntumor recurrence. Moreover, in the small subset of primary\nneu-induced tumors that did not regress fully when neu was\ndownregulated, we identified high levels of Snail expression\ncomparable to those found in recurrent tumors (Figure 3B).\nNevertheless, it is worth noting that the association between\nbreast cancer relapse and Snail expression in primary human\ntumors is not inconsistent with our observation that spontane-\nous Snail upregulation in this mouse model occurs in recurrent\ntumors. First, Snail is expressed at detectable levels in primary\nneu-induced tumor cells in mice—albeit at lower levels than in\nrecurrent tumors. Second, Snail expression levels in recurrent\nhuman breast cancers are at present unknown, since these tu-\nmors are generally unavailable for molecular analysis. As such,\nbased upon our findings in mice, we predict that analysis of\nrecurrent human breast cancers will reveal elevated levels of\nSnail expression compared to primary tumors.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nThe association between EMT and breast cancer progres-\nsion has significant support within the literature, although no\nupstream regulator of both processes has previously been\ncorrelated with relapse-free survival. A subset of human breast\ncancers and human breast cancer cell lines display features of\nEMT (Hajra et al., 2002; Yang et al., 2004); in addition, suppres-\nsion of CK8 and expression of vimentin have each been corre-\nlated with decreased survival of breast cancer patients (Fuchs\nloss of\net al., 2002; Niveditha and Bajaj, 2003). Moreover,\nE-cadherin expression in breast cancers has been shown to be\npositively correlated with advanced histological grade, metas-\ntasis, and decreased disease-free as well as overall survival\n(Gamallo et al., 1993; Guriec et al., 1996; Heimann et al., 2000).\nGiven the strong repression of E-cadherin observed in Snail-\nexpressing tumor samples, the well-described association of\nE-cadherin with poor prognosis may in some cases reflect\nSnail activity. Indeed, an inverse association between Snail and\nE-cadherin expression has been reported in some human\nbreast cancers (Zhou et al., 2004). Nevertheless, our data do\nnot address which of Snail’s many functions are most relevant\nto recurrence or the maintenance of residual disease.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nSnail expression has previously been correlated with histo-\nlogical grade and lymph node metastasis in a panel of 17 hu-\nman breast cancers (Blanco et al., 2002). While we were able\nto detect a significant correlation between Snail expression and\nhistological grade in one breast cancer data set by one method\nof analysis, we did not find consistent correlations between\nSnail expression and histological grade across most data sets,\nnor between Snail expression and lymph node status in any of\nthe data sets examined. As such, it is possible that in larger,\nmore diverse tumor sets Snail expression levels may not corre-\nlate significantly with tumor grade or lymph node status, yet\nmay remain a significant predictor of relapse-free survival.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nOf note, we demonstrated that Snail expression does not\nconsistently correlate with tumor size, HER2 status, ductal ver-\nsus lobular subtype, or luminal versus basal subtype, and that\nits prognostic significance is comparable to currently used\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nprognostic markers. This suggests that the association be-\ntween Snail and relapse-free survival is not simply due to its\nselective expression in a presently recognized subset of ag-\ngressive breast cancers. Thus, independent of most commonly\nused prognostic indicators, high Snail expression identifies a\nsubset of human breast cancer patients at high risk for recur-\nrence. Moreover, the HR associated with increased Snail ex-\npression is comparable to that observed for ER status, which\nrepresents one of the most commonly used clinical prognostic\nvariables as well as the most commonly used target for breast\ncancer prevention and treatment.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nNotably, another molecule involved in mesoderm develop-\nment, Twist, has been reported to be a positive regulator of\nSnail expression in Drosophila. However, in experiments impli-\ncating Twist in tumor metastasis, Yang et al. were unable to\ndetect induction of Snail\nin human mammary epithelial cells\ninduced to undergo EMT by forced Twist expression (Yang et\nal., 2004). We were also unable to detect a significant correla-\ntion between Twist expression and Snail expression in the Ma,\nSorlie, or Wang human breast cancer data sets (Twist expres-\nsion was not available for the van’t Veer data set). This is in\nagreement with data suggesting that Snail and Twist function\nindependently in mice (Carver et al., 2001; Soo et al., 2002).\nAs such, the observed associations between Twist and metas-\ntasis and between Snail and recurrence are likely to represent\nindependent manifestations of—and mechanisms for—mam-\nmary tumor progression.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\nIt is important to note that the findings presented in this\nmanuscript most directly address the process of local recur-\nrence. Isolated local recurrence following breast conservation\nsurgery with radiation treatment occurs in w10%–20% of\nwomen at 10 years and typically comprises up to one-third of\nall recurrences (Doyle et al., 2001; Fisher et al., 1991; Fortin et\nal., 1999; Schmoor et al., 2000). As such, local recurrence is in\nand of itself a critical problem to understand. However, there\nare additional reasons to believe that the findings presented\nhere may be relevant to distant as well as local recurrence. For\nexample, a priori considerations alone would suggest that the\nprocesses by which tumor cells—whether local or distant—\nsurvive in a latent state and ultimately reestablish malignant\ngrowth are likely to be related mechanistically. Consistent with\nthis presumption, multiple studies have demonstrated that lo-\ncal recurrence is strongly associated with an increased risk of\nboth distant relapse (relative risk [RR] = 5.1) and mortality (RR =\n3.6) (Doyle et al., 2001; Fisher et al., 1991; Fortin et al., 1999;\nSchmoor et al., 2000). Moreover, even the timing of local recur-\nrences after surgery is similar to that of distant recurrences\n(Demicheli et al., 2004). Together, these observations suggest\nthat local and distant recurrence may result from similar pro-\ncesses. Thus, beyond the recognition that local recurrence is\nitself important to understand, the associations between local\nand distant recurrence suggest that the mechanisms responsi-\nble for one may be informative for the other. As such, our find-\nings that Snail promotes local recurrence in mice and predicts\nlocal and distant recurrence in humans raise the possibility that\nSnail may contribute to both processes.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nSimilarities between local and distant recurrences notwith-\nstanding, distinctions between the processes of recurrence\nand metastasis must be considered since the mechanisms that\nallow latent tumor cells to survive and ultimately reestablish\nmalignant growth are likely to differ from those that permit\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nactively growing tumor cells to invade and spread to distant\nsites. While metastasis is a sine qua non of distant spread, it\nis clearly not required for local recurrence. Moreover, even in\nthe case of distant recurrence, the mechanisms that permit\nmetastatic cells to remain in a quiescent state and reemerge at\na later time are not the same as those that permit distant\nspread in the first place. As such, while the processes of recur-\nrence and metastasis likely share a number of similar features,\ndistinct mechanisms almost certainly contribute to each pro-\ncess. It is therefore essential to distinguish these processes\nexperimentally to permit elucidation of these separate mecha-\nnisms.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.3**l== 0.1**r== 0.5**\nGiven the enormous number of women at risk for breast can-\ncer recurrence, Snail’s ability to predict relapse-free survival in\nwomen, coupled with evidence that Snail promotes the recur-\nrence of mammary tumors in mice, suggest that further studies\nof this transcriptional repressor may be clinically relevant. Be-\nyond the ability to identify women at high risk for relapse, our\nfindings raise the important possibility that breast cancer recur-\nrence might be prevented by therapeutic agents that inhibit\nSnail, either by preventing its upregulation or by inhibiting its\nfunction. For example, GSK-3β inhibits Snail transcription and\nalso promotes its nuclear export and proteasome-dependent\ndegradation by a phosphorylation-dependent mechanism (Zhou\net al., 2004; Bachelder et al., 2005). Accordingly, inhibition of\nGSK-3β activity by the PI3K/AKT, MAPK, and Wnt signaling\npathways promotes Snail stabilization and activation. In addi-\ntion, phosphorylation of Snail by Pak1—which is activated by\nPI3K via AKT and the small GTPase Rac (Tang et al., 2000)—\nenhances Snail’s repressor activity by promoting its nuclear ac-\ncumulation (Yang et al., 2005). As such, the development of\ncancer therapeutics that target these upstream pathways may\nprovide a dual mechanism for the inhibition of Snail activity\nboth by activating GSK-3β and by inhibiting Pak1. Thus, while\nit is not possible to confirm a causal role for Snail in human\nbreast cancer recurrence until drugs are available to inhibit this\npathway, we hypothesize that treatment of patients with phar-\nmacologic agents that block Snail expression or function may\nbe effective for preventing breast cancer relapse. Snail may\nthereby represent an important new target for a generation of\ncancer therapeutics directed against specific molecules in-\nvolved in breast cancer recurrence.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nExperimental procedures\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nAnimals, tissues, immunostaining, and molecular analyses\nMMTV-rtTA/TetO-NeuNT mice were engineered, housed, induced with 2\nmg/ml doxycycline, monitored for tumor development, and sacrificed as\npreviously described (Gunther et al., 2002; Moody et al., 2002). Details for\ntissue fixation, immunofluorescence, immunohistochemistry, Northern blot-\nting, and microarray analyses are provided in the Supplemental Experimen-\ntal Procedures.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nTumor grafting and retroviral transduction\nDetails for culture of tumor cells harvested from chronically induced MTB/\nTAN mice, retroviral transduction, and tumor grafting are provided in the\nSupplemental Experimental Procedures.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nHuman breast cancer data sets\nDescriptions of human breast cancer microarray data sets that were interro-\ngated and methods used for data analysis, including survival analysis, cal-\nculation of HRs, and estimation of the association between Snail expression\nand established prognostic factors, are described in detail in the Supple-\nmental Experimental Procedures.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nSupplemental data\nThe Supplemental Data include Supplemental Experimental Procedures\nand can be found with this article online at http://www.cancercell.org/cgi/\ncontent/full/8/3/197/DC1/.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nAcknowledgments\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nThe authors wish to thank Warren Pear for the pk1, pCGP, and pHIT123\nplasmids. This research was supported in part by NIH grants CA92910,\nCA93719, CA98371, and CA105490 from the National Cancer Institute, and\nby U.S. Army Breast Cancer Research Program grants DAMD17-02-1-0728\nand W81XWH-04-1-0353.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.3**\nReceived: March 28, 2005\nRevised: June 21, 2005\nAccepted: July 28, 2005\nPublished: September 19, 2005\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nReferences\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nBachelder, R.E., Yoon, S.O., Franci, C., de Herreros, A.G., and Mercurio,\nA.M. (2005). Glycogen synthase kinase-3 is an endogenous inhibitor of Snail\ntranscription: implication for the epithelial-mesenchymal transition. J. Cell\nBiol. 168, 29–33.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nBaselga, J., Tripathy, D., Mendelsohn, J., Baughman, S., Benz, C.C., Dantis,\nL., Sklarin, N.T., Seidman, A.D., Hudis, C.A., Moore, J., et al. (1996). Phase\nII study of weekly intravenous recombinant humanized anti-p185HER2\nmonoclonal antibody in patients with HER2/neu-overexpressing metastatic\nbreast cancer. J. Clin. Oncol. 14, 737–744.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nBatlle, E., Sancho, E., Franci, C., Dominguez, D., Monfar, M., Baulida, J.,\nand Garcia De Herreros, A. (2000). The transcription factor snail is a repres-\nsor of E-cadherin gene expression in epithelial tumour cells. Nat. Cell Biol.\n2, 84–89.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nBerger, M.S., Locher, G.W., Saurer, S., Gullick, W.J., Waterfield, M.D.,\nGroner, B., and Hynes, N.E. (1988). Correlation of c-erbB-2 gene amplifica-\ntion and protein expression in human breast carcinoma with nodal status\nand nuclear grading. Cancer Res. 48, 1238–1243.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nBlanco, M.J., Moreno-Bueno, G., Sarrio, D., Locascio, A., Cano, A., Pala-\ncios, J., and Nieto, M.A. (2002). Correlation of Snail expression with histo-\nlogical grade and lymph node status in breast carcinomas. Oncogene 21,\n3241–3246.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nCano, A., Perez-Moreno, M.A., Rodrigo, I., Locascio, A., Blanco, M.J., del\nBarrio, M.G., Portillo, F., and Nieto, M.A. (2000). The transcription factor\nsnail controls epithelial-mesenchymal transitions by repressing E-cadherin\nexpression. Nat. Cell Biol. 2, 76–83.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nCardiff, R.D., Sinn, E., Muller, W., and Leder, P. (1991). Transgenic oncogene\nmice. Tumor phenotype predicts genotype. Am. J. Pathol. 139, 495–501.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nCarter, C.L., Allen, C., and Henson, D.E. (1989). Relation of tumor size,\nlymph node status, and survival in 24,740 breast cancer cases. Cancer 63,\n181–187.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nCarver, E.A., Jiang, R., Lan, Y., Oram, K.F., and Gridley, T. (2001). The mouse\nsnail gene encodes a key regulator of the epithelial-mesenchymal transition.\nMol. Cell. Biol. 21, 8184–8188.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nCheng, C.W., Wu, P.E., Yu, J.C., Huang, C.S., Yue, C.T., Wu, C.W., and\nShen, C.Y. (2001). Mechanisms of inactivation of E-cadherin in breast carci-\nnoma: modification of the two-hit hypothesis of tumor suppressor gene.\nOncogene 20, 3814–3823.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nDemicheli, R., Bonadonna, G., Hrushesky, W.J., Retsky, M.W., and Vala-\ngussa, P. (2004). Menopausal status dependence of the timing of breast\ncancer recurrence after surgical removal of the primary tumour. Breast Can-\ncer Res. 6, R689–R696.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nDoyle, T., Schultz, D.J., Peters, C., Harris, E., and Solin, L.J. (2001). Long-\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nterm results of local recurrence after breast conservation treatment for inva-\nsive breast cancer. Int. J. Radiat. Oncol. Biol. Phys. 51, 74–80.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nc-myc, not HER-2/neu, can predict recurrence and mortality of patients with\nnode-negative breast cancer. Breast Cancer Res. 5, R30–R36.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nEsteva, F.J., and Hortobagyi, G.N. (2004). Prognostic molecular markers in\nearly breast cancer. Breast Cancer Res. 6, 109–118.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nFisher, B., Anderson, S., Fisher, E.R., Redmond, C., Wickerham, D.L.,\nWolmark, N., Mamounas, E.P., Deutsch, M., and Margolese, R. (1991). Sig-\nnificance of ipsilateral breast tumour recurrence after lumpectomy. Lancet\n338, 327–331.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nFisher, B., Jeong, J.H., Bryant, J., Anderson, S., Dignam, J., Fisher, E.R.,\nand Wolmark, N. (2004). Treatment of lymph-node-negative, oestrogen-\nreceptor-positive breast cancer: long-term findings from National Surgical\nAdjuvant Breast and Bowel Project randomised clinical trials. Lancet 364,\n858–868.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nFortin, A., Larochelle, M., Laverdiere, J., Lavertu, S., and Tremblay, D.\n(1999). Local failure is responsible for the decrease in survival for patients\nwith breast cancer treated with conservative surgery and postoperative ra-\ndiotherapy. J. Clin. Oncol. 17, 101–109.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nFuchs, I.B., Lichtenegger, W., Buehler, H., Henrich, W., Stein, H., Kleine-\nTebbe, A., and Schaller, G. (2002). The prognostic significance of epithelial-\nmesenchymal transition in breast cancer. Anticancer Res. 22, 3415–3419.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nGamallo, C., Palacios, J., Suarez, A., Pizarro, A., Navarro, P., Quintanilla,\nM., and Cano, A. (1993). Correlation of E-cadherin expression with differen-\ntiation grade and histological type in breast carcinoma. Am. J. Pathol. 142,\n987–993.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nGrau, Y., Carteret, C., and Simpson, P. (1984). Mutations and chromosomal\nrearrangements affecting the expression of snail, a gene involved in embry-\nonic patterning in Drosophila melanogaster. Genetics 108, 347–360.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nGunther, E.J., Belka, G.K., Wertheim, G.B., Wang, J., Hartman, J.L., Boxer,\nR.B., and Chodosh, L.A. (2002). A novel doxycycline-inducible system for\nthe transgenic analysis of mammary gland biology. FASEB J. 16, 283–292.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nGuriec, N., Marcellin, L., Gairard, B., Calderoli, H., Wilk, A., Renaud, R.,\nBergerat, J.P., and Oberling, F. (1996). E-cadherin mRNA expression in\nbreast carcinomas correlates with overall and disease-free survival. Inva-\nsion Metastasis 16, 19–26.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nHajra, K.M., Chen, D.Y., and Fearon, E.R. (2002). The SLUG zinc-finger pro-\ntein represses E-cadherin in breast cancer. Cancer Res. 62, 1613–1618.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nHeimann, R., Lan, F., McBride, R., and Hellman, S. (2000). Separating favor-\nable from unfavorable prognostic markers in breast cancer: the role of\nE-cadherin. Cancer Res. 60, 298–304.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nHortobagyi, G.N. (2001). Overview of treatment results with trastuzumab\n(Herceptin) in metastatic breast cancer. Semin. Oncol. 28, 43–47.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nMa, X.J., Wang, Z., Ryan, P.D., Isakoff, S.J., Barmettler, A., Fuller, A., Muir,\nB., Mohapatra, G., Salunga, R., Tuggle, J.T., et al. (2004). A two-gene ex-\npression ratio predicts clinical outcome in breast cancer patients treated\nwith tamoxifen. Cancer Cell 5, 607–616.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nMoody, S.E., Sarkisian, C.J., Hahn, K.T., Gunther, E.J., Pickup, S., Dugan,\nK.D., Innocent, N., Cardiff, R.D., Schnall, M.D., and Chodosh, L.A. (2002).\nConditional activation of Neu in the mammary epithelium of transgenic mice\nresults in reversible pulmonary metastasis. Cancer Cell 2, 451–461.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nNiveditha, S.R., and Bajaj, P. (2003). Vimentin expression in breast carcino-\nmas. Indian J. Pathol. Microbiol. 46, 579–584.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nPantel, K., Muller, V., Auer, M., Nusser, N., Harbeck, N., and Braun, S. (2003).\nDetection and clinical implications of early systemic tumor cell dissemina-\ntion in breast cancer. Clin. Cancer Res. 9, 6326–6334.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nParkin, D.M., Bray, F., Ferlay, J., and Pisani, P. (2005). Global cancer statis-\ntics, 2002. CA Cancer J. Clin. 55, 74–108.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nSaphner, T., Tormey, D.C., and Gray, R. (1996). Annual hazard rates of recur-\nrence for breast cancer after primary therapy. J. Clin. Oncol. 14, 2738–2746.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nSchlotter, C.M., Vogt, U., Bosse, U., Mersch, B., and Wassmann, K. (2003).\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nSchmoor, C., Sauerbrei, W., Bastert, G., and Schumacher, M. (2000). Role\nof isolated locoregional recurrence of breast cancer: results of four pros-\npective studies. J. Clin. Oncol. 18, 1696–1708.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nSlamon, D.J., Clark, G.M., and Wong, S.G. (1987). Human breast cancer:\ncorrelation of relapse and survival with amplification of the HER-2/neu on-\ncogene. Science 235, 177–182.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nSlamon, D.J., Leyland-Jones, B., Shak, S., Fuchs, H., Paton, V., Bajamonde,\nA., Fleming, T., Eiermann, W., Wolter, J., Pegram, M., et al. (2001). Use of\nchemotherapy plus a monoclonal antibody against HER2 for metastatic\nbreast cancer that overexpresses HER2. N. Engl. J. Med. 344, 783–792.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nSoo, K., O’Rourke, M.P., Khoo, P.L., Steiner, K.A., Wong, N., Behringer, R.R.,\nand Tam, P.P. (2002). Twist function is required for the morphogenesis of\nthe cephalic neural tube and the differentiation of the cranial neural crest\ncells in the mouse embryo. Dev. Biol. 247, 251–270.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nSorlie, T., Perou, C.M., Tibshirani, R., Aas, T., Geisler, S., Johnsen, H., Has-\ntie, T., Eisen, M.B., van de Rijn, M., Jeffrey, S.S., et al. (2001). Gene expres-\nsion patterns of breast carcinomas distinguish tumor subclasses with clin-\nical implications. Proc. Natl. Acad. Sci. USA 98, 10869–10874.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nSorlie, T., Tibshirani, R., Parker, J., Hastie, T., Marron, J.S., Nobel, A., Deng,\nS., Johnsen, H., Pesich, R., Geisler, S., et al. (2003). Repeated observation\nof breast tumor subtypes in independent gene expression data sets. Proc.\nNatl. Acad. Sci. USA 100, 8418–8423.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nTang, Y., Zhou, H., Chen, A., Pittman, R.N., and Field, J. (2000). The Akt\nproto-oncogene links Ras to Pak and cell survival signals. J. Biol. Chem.\n275, 9106–9109.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nValagussa, P., Bonadonna, G., and Veronesi, U. (1978). Patterns of relapse\nand survival following radical mastectomy. Analysis of 716 consecutive pa-\ntients. Cancer 41, 1170–1178.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nvan’t Veer, L.J., Dai, H., van de Vijver, M.J., He, Y.D., Hart, A.A., Mao, M.,\nPeterse, H.L., van der Kooy, K., Marton, M.J., Witteveen, A.T., et al. (2002).\nGene expression profiling predicts clinical outcome of breast cancer. Nature\n415, 530–536.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nVogel, C.L., Cobleigh, M.A., Tripathy, D., Gutheil, J.C., Harris, L.N., Fehren-\nbacher, L., Slamon, D.J., Murphy, M., Novotny, W.F., Burchmore, M., et al.\n(2002). Efficacy and safety of trastuzumab as a single agent in first-line\ntreatment of HER2-overexpressing metastatic breast cancer. J. Clin. Oncol.\n20, 719–726.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nWang, S.C., Zhang, L., Hortobagyi, G.N., and Hung, M.C. (2001). Targeting\nHER2: recent developments and future directions for breast cancer pa-\ntients. Semin. Oncol. 28, 21–29.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nWang, Y., Klijn, J.G., Zhang, Y., Sieuwerts, A.M., Look, M.P., Yang, F., Talan-\ntov, D., Timmermans, M., Meijer-van Gelder, M.E., Yu, J., et al. (2005). Gene-\nexpression profiles to predict distant metastasis of lymph-node-negative\nprimary breast cancer. Lancet 365, 671–679.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nWeiss, R.B., Woolf, S.H., Demakos, E., Holland, J.F., Berry, D.A., Falkson,\nG., Cirrincione, C.T., Robbins, A., Bothun, S., Henderson, I.C., and Norton,\nL. (2003). Natural history of more than 20 years of node-positive primary\nbreast carcinoma treated with cyclophosphamide, methotrexate, and fluo-\nrouracil-based adjuvant chemotherapy: a study by the Cancer and Leuke-\nmia Group B. J. Clin. Oncol. 21, 1825–1835.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nYang, J., Mani, S.A., Donaher, J.L., Ramaswamy, S., Itzykson, R.A., Come,\nC., Savagner, P., Gitelman, I., Richardson, A., and Weinberg, R.A. (2004).\nTwist, a master regulator of morphogenesis, plays an essential role in tumor\nmetastasis. Cell 117, 927–939.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nYang, Z., Rayala, S., Nguyen, D., Vadlamudi, R.K., Chen, S., and Kumar, R.\n(2005). Pak1 phosphorylation of Snail, a master regulator of epithelial-to-\nmesenchymal\nlocalization and\nfunction. Cancer Res. 65, 3179–3184.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.6**r== 0.2**\ntransition, modulates Snail’s subcellular\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nZhou, B.P., Deng, J., Xia, W., Xu, J., Li, Y.M., Gunduz, M., and Hung, M.C.\n(2004). Dual regulation of Snail by GSK-3β-mediated phosphorylation in\ncontrol of epithelial-mesenchymal transition. Nat. Cell Biol. 6, 931–940.",
         "extracted",
         "Pak1 phosphorylation of snail, a master regulator of epithelial-to-mesenchyme transition, modulates snail's subcellular localization and functions.;Global Cancer Statistics, 2002;Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer;Glycogen synthase kinase-3 is an endogenous inhibitor of Snail transcription;Menopausal status dependence of the timing of breast cancer recurrence after surgical removal of the primary tumour;Dual regulation of Snail by GSK-3β-mediated phosphorylation in control of epithelial–mesenchymal transition;Treatment of lymph-node-negative, oestrogen-receptor-positive breast cancer: long-term findings from National Surgical Adjuvant Breast and Bowel Project randomised clinical trials;Twist, a Master Regulator of Morphogenesis, Plays an Essential Role in Tumor Metastasis;A two-gene expression ratio predicts clinical outcome in breast cancer patients treated with tamoxifen.;Prognostic molecular markers in early breast cancer;Detection and clinical implications of early systemic tumor cell dissemination in breast cancer.;Vimentin expression in breast carcinomas.;Repeated observation of breast tumor subtypes in independent gene expression data sets;Natural history of more than 20 years of node-positive primary breast carcinoma treated with cyclophosphamide, methotrexate, and fluorouracil-based adjuvant chemotherapy: a study by the Cancer and Leukemia Group B.;C-myc, not HER-2/neu, can predict recurrence and mortality of patients with node-negative breast cancer;Conditional activation of Neu in the mammary epithelium of transgenic mice results in reversible pulmonary metastasis.;The prognostic significance of epithelial-mesenchymal transition in breast cancer.;Twist function is required for the morphogenesis of the cephalic neural tube and the differentiation of the cranial neural crest cells in the mouse embryo.;Correlation of Snail expression with histological grade and lymph node status in breast carcinomas;The SLUG zinc-finger protein represses E-cadherin in breast cancer.;A novel doxycycline‐inducible system for the transgenic analysis of mammary gland biology;Efficacy and safety of trastuzumab as a single agent in first-line treatment of HER2-overexpressing metastatic breast cancer.;Gene expression profiling predicts clinical outcome of breast cancer;Overview of treatment results with trastuzumab (Herceptin) in metastatic breast cancer.;The Mouse Snail Gene Encodes a Key Regulator of the Epithelial-Mesenchymal Transition;Targeting HER2: recent developments and future directions for breast cancer patients.;Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications;Long-term results of local recurrence after breast conservation treatment for invasive breast cancer.;Mechanisms of inactivation of E-cadherin in breast carcinoma: modification of the two-hit hypothesis of tumor suppressor gene;Role of isolated locoregional recurrence of breast cancer: results of four prospective studies.;The Akt Proto-oncogene Links Ras to Pak and Cell Survival Signals*;The transcription factor Snail is a repressor of E-cadherin gene expression in epithelial tumour cells;The transcription factor Snail controls epithelial–mesenchymal transitions by repressing E-cadherin expression;Separating favorable from unfavorable prognostic markers in breast cancer: the role of E-cadherin.;Annual hazard rates of recurrence for breast cancer after primary therapy.;Phase II study of weekly intravenous recombinant humanized anti-p185HER2 monoclonal antibody in patients with HER2/neu-overexpressing metastatic breast cancer.;Correlation of E-cadherin expression with differentiation grade and histological type in breast carcinoma.;Transgenic oncogene mice. Tumor phenotype predicts genotype.;Significance of ipsilateral breast tumour recurrence after lumpectomy;Relation of tumor size, lymph node status, and survival in 24,740 breast cancer cases;Correlation of c-erbB-2 gene amplification and protein expression in human breast carcinoma with nodal status and nuclear grading.;Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene.;Mutations and Chromosomal Rearrangements Affecting the Expression of Snail, a Gene Involved in Embryonic Patterning in DROSOPHILA MELANOGASTER.;Patterns of relapse and survival following radical mastectomy. Analysis of 716 consecutive patients;Local failure is responsible for the decrease in survival for patients with breast cancer treated with conservative surgery and postoperative radiotherapy.;E-cadherin mRNA expression in breast carcinomas correlates with overall and disease-free survival.",
         "The transcriptional repressor Snail promotes mammary tumor recurrence.",
         "None"
        ],
        [
         "42",
         "00309d5539a3db7b5bf065351c3b085e87a3b1f3",
         "ABSTRACT A Bacillus subtilis sigM null mutant, lacking the extracytoplasmic function σM protein, was sensitive to paraquat (PQ), a superoxide-generating reagent, but not to the redox stress-inducing compounds hydrogen peroxide, cumene hydroperoxide, t-butyl hydroperoxide, or diamide. Surprisingly, a sigM mutant was only sensitive to superoxide-generating compounds with a dipyridyl ring such as PQ, ethyl viologen, benzyl viologen, and diquat but not to menadione, plumbagin, pyrogallol, or nitrofurantoin. Mutational analysis of candidate σM-regulated genes revealed that both YqjL, a putative hydrolase, and BcrC, a bacitracin resistance protein, were involved in PQ resistance. Expression of yqjL, but not bcrC, from a xylose-inducible promoter restored PQ resistance to the sigM mutant.",
         "M. Cao,Charles M. Moore,J. Helmann",
         "https://jb.asm.org/content/jb/187/9/2948.full.pdf",
         "\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nJOURNAL OF BACTERIOLOGY, May 2005, p. 2948–2956\n0021-9193/05/$08.000 doi:10.1128/JB.187.9.2948–2956.2005\nCopyright © 2005, American Society for Microbiology. All Rights Reserved.\n**BLOCK**fs== 16.0**b== 0.7**t== 0.2**l== 0.1**r== 0.2**\nBacillus subtilis Paraquat Resistance Is Directed by M, an\nExtracytoplasmic Function Sigma Factor, and Is\nConferred by YqjL and BcrC\nMin Cao, Charles M. Moore, and John D. Helmann*\nDepartment of Microbiology, Cornell University, Ithaca, New York 14853-8101\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.3**r== 0.4**\nReceived 23 August 2004/Accepted 22 January 2005\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.2**\nA Bacillus subtilis sigM null mutant, lacking the extracytoplasmic function M protein, was sensitive to\nparaquat (PQ), a superoxide-generating reagent, but not to the redox stress-inducing compounds hydrogen\nperoxide, cumene hydroperoxide, t-butyl hydroperoxide, or diamide. Surprisingly, a sigM mutant was only\nsensitive to superoxide-generating compounds with a dipyridyl ring such as PQ, ethyl viologen, benzyl viologen,\nand diquat but not to menadione, plumbagin, pyrogallol, or nitrofurantoin. Mutational analysis of candidate\nM-regulated genes revealed that both YqjL, a putative hydrolase, and BcrC, a bacitracin resistance protein,\nwere involved in PQ resistance. Expression of yqjL, but not bcrC, from a xylose-inducible promoter restored PQ\nresistance to the sigM mutant.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nBacillus subtilis responds to the presence of oxidizing agents\nby activating complex adaptation mechanisms under the con-\ntrol of redox-sensitive regulatory proteins. Transcriptional pro-\nﬁling indicates that hydrogen peroxide activates a large stimu-\nlon, including genes controlled by the general stress  factor\nB, the peroxide sensor PerR, and the organic peroxide sensor\nOhrR (15). Under some conditions, members of the Fur regu-\nlon are also induced by hydrogen peroxide (30). Many of these\nsame regulons are induced by the superoxide generator para-\nquat (PQ), although it is difﬁcult to know whether these re-\nsponses are due to superoxide or subsequently produced hy-\ndrogen peroxide (30). Nitric oxide induces both the PerR and\nFur regulons, particularly under anaerobic conditions (28),\nconsistent with a direct nitrosylation of the Fe(II) corepressors\n(10). B. subtilis mounts a distinct, but overlapping, response to\ndisulﬁde stress elicited by thiol-oxidizing agents such as dia-\nmide (23). The diamide stimulon includes both the PerR regu-\nlon and genes regulated by the novel thiol stress sensor Spx\n(31, 49).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nExtracytoplasmic function (ECF)  factors have also been\nimplicated in the regulation of oxidative stress responses. ECF\n factors comprise a unique subgroup within the 70 family\n(14, 24) which, in general, controls genes related to cell enve-\nlope functions such as cell surface stress (Escherichia coli E\nand B. subtilis W and M), secretion (Pseudomonas aeruginosa\nE), and metal ion homeostasis (E. coli FecI, Pseudomonas\nﬂuorescens PbrA, and Ralstonia eutropha CnrH). However,\nthere are exceptions: Streptomyces coelicolor R and Mycobac-\nterium tuberculosis H respond to cytoplasmic disulﬁde stress\n(25, 34, 35, 43). In both cases, the  factor is negatively regu-\nlated by an anti- factor (RsrA for R and RshA for H)\ncontaining a Zn(II)-binding motif (His-X-X-X-Cys-X-X-Cys,\nwhere X stands for any amino acid) near the N terminus. In\nRsrA, cysteines within this motif are part of a thiol-disulﬁde\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n* Corresponding author. Mailing address: Department of Microbi-\nology, Wing Hall, Cornell University, Ithaca, NY 14853-8101. Phone:\n(607) 255-6570. Fax: (607) 255-3904. E-mail: jdh9@cornell.edu.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nredox switch (34). Upon oxidation, RsrA releases R, which in\nturn switches on the transcription of the thioredoxin operon\ntrxBA. RsrA and related anti- factors comprise the Zn-bind-\ning anti- (ZAS) family (34).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nB. subtilis encodes two members of the ZAS anti- factors:\nRsiW and YlaD. RsiW serves as the anti- factor for W (40),\nwhile YlaD is a transmembrane protein of unknown function\nencoded by the same operon as the ECF sigma factor (YlaC),\nalthough it does not appear to bind directly to YlaC (47). The\nW regulon has been extensively characterized and includes\napproximately 60 genes that function in a cell envelope stress\nresponse (5). There is no evidence, as yet, to link either W or\nYlaC to the regulation of oxidative stress response genes.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nHere, we demonstrate that only sigM of the seven ECF \nfactors in B. subtilis contributes to oxidative stress resistance. A\nsigM null mutant had a dramatically increased sensitivity to PQ\nand structurally related superoxide-generating agents. This is\nconsistent with previous results, demonstrating that the expres-\nsion of M is induced by PQ (44). Although PQ can increase\nthe intracellular production of superoxide in many cell types,\nthe mechanism and subcellular location of PQ stress in B.\nsubtilis is largely unknown. We identify two M-regulated pro-\nteins, YqjL (a putative hydrolase) and BcrC (a bacitracin re-\nsistance protein) (4), as major contributors to the M-depen-\ndent resistance to PQ and structurally related superoxide-\ngenerating compounds.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nMATERIALS AND METHODS\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nBacterial strains, growth conditions, and chemicals. All B. subtilis strains used\nin this study are listed in Table 1. Bacteria were grown in Luria-Bertani (LB)\nmedium or low-salt LB medium (LS) lacking added NaCl at 37°C with vigorous\nshaking. Antibiotics were added to the growth medium when appropriate at 100\ng/ml of ampicillin or 200 g/ml of spectinomycin for E. coli and 100 g/ml of\nspectinomycin, 10 g/ml of kanamycin, 10 g/ml of tetracycline, 10 g/ml of\nchloramphenicol, 8 g/ml of neomycin, or 1 g/ml of erythromycin plus 25 g/ml\nof lincomycin (for macrolide-lincomycin-streptogramin B [MLS] resistance) for\nB. subtilis.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nAll chemicals used in this study were purchased from Sigma Chemical Co. (St.\nLouis, Mo.). Paraquat dichloride (PQ), diquat dibromide monohydrate (DQ),\nethyl viologen dibromide (EV), benzyl viologen (BV), pyrogallol, and mitomycin\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nTABLE 1. Bacterial strains used in this study\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nStrain\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nGenotype\n**BLOCK**fs== 7.0**b== 0.4**t== 0.2**l== 0.1**r== 0.6**\nW168 attSP trpC2\nW168 SPc22::Tn917::pSK106 (MLSr)\nPS832 sodA::cat\nCU1065 sigX::spc\nCU1065 sigY::MLS\nCU1065 sigW::MLS\nCU1065 sigV::kan\nCU1065 ylaC::kan\nCU1065 sigM::kan\nCU1065 sigZ::kan\nCU1065 SP(PM-cat-lacZ)\nCU1065 sigM::kan SP(PM-cat-lacZ)\nZB307A SP(PyqjL-cat-lacZ)\nCU1065 sigW::MLS sigM::kan\nCU1065 sigX::spc sigM::kan\nCU1065 sigX::spc sigW::MLS sigM::kan\nCU1065 bcrC::pMUTIN\nCU1065 sigW::MLS ylaC::kan\nCU1065 sigM::kan PxylA-sigM\nCU1065 sodA::cat\nCU1065 sigM::kan sodA::cat\nCU1065 sodA::cat PxylA-sigM\nCU1065 sodA::cat PxylA-sodA\nCU1065 sigM::kan PxylA-sodA\nCU1065 yrhJ::cat\nCU1065 ydaH::cat\nCU1065 ypbG::cat\nCU1065 yraA::cat\nCU1065 radA::cat\nCU1065 yacK::cat\nCU1065 yqjL::tet\nCU1065 radC::tet\nCU1065 yqjL::tet bcrC::pMUTIN\nCU1065 sigM::kan PxylA-yqjL\nCU1065 yqjL::tetPxylA-yqjL\nCU1065 sigM::kan PxylA-bcrC\nCU1065 bcrC::pMUTIN PxylA-bcrC\nCU1065 SP(PyqjL-cat-lacZ)\nCU1065 sigM::kan SP(PyqjL-cat-lacZ)\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nReference\n**BLOCK**fs== 7.0**b== 0.4**t== 0.2**l== 0.4**r== 0.5**\nLaboratory stock\n\n\n\n\n\nThis work\nThis work\n\nThis work\n\n\n\nThis work\n\nThis work\n\nThis work\n\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\nThis work\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nC were dissolved in distilled water. Plumbagin, menadione, and dipyridyl were\ndissolved in ethanol. Nitrofurantoin (7.6 mg) was dissolved in 14.9 ml of 95%\nethanol. One gram of 4-nitroquinoline N-oxide (4NQO) was dissolved in 52.5 ml\nof acetone. One gram of 4, 4-dimethylbiphenyl was dissolved in 5 ml of chlo-\nroform. Benzidine was dissolved in 10% acetic acid.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nConstruction of sigV, sigZ, ylaC, sigW ylaC, and sigX sigW sigM mutants.\nPrimers 335 and 336 (all oligonucleotide primers are listed in Table 1S in the\nsupplemental material; see “Supplemental ﬁgures and material” below) were\nused to amplify the sigV gene from B. subtilis strain CU1065 chromosomal DNA.\nThe PCR fragment (about 900 bp, including the coding region of sigV as well as\n200 bp of upstream sequence and 190 bp of downstream sequence) was digested\nwith HindIII and EcoRI and ligated into pUC18 to generate plasmid pMC61.\npMC61 was digested with SacI and HincII and ligated with a Kanr cassette (from\npJM114 [36], which was digested with SacI and HincII) to generate pMC65.\nThus, an internal fragment (320 bp) of sigV was replaced by the Kanr cassette.\npMC65 was linearized with ScaI and moved into B. subtilis strain CU1065 by\ntransformation, selecting for Kanr, to generate strain HB0028. The Kanr cassette\nis oriented in the same direction as sigV.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nPrimers 325 and 327 were used to amplify the sigZ gene from B. subtilis strain\nCU1065 chromosomal DNA. The PCR fragment (about 950 bp, including the\ncoding region of sigZ as well as 220 bp of upstream sequence and 200 bp of\ndownstream sequence) was digested with HindIII and BamHI and ligated into\npUC18 to generate plasmid pMC60. pMC60 was digested with StyI and XhoI (to\nrelease a 70-bp internal fragment) and ligated with the pJM114 Kanr cassette\n(digested with XbaI and XhoI) to generate pMC64. pMC64 was linearized with\nScaI and moved into B. subtilis strain CU1065 by transformation, selecting for\nKanr, to generate strain HB0032. The Kanr cassette is oriented in the same\ndirection as sigZ.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nPrimers 328 and 330 were used to amplify the ylaC gene from B. subtilis strain\nCU1065 chromosomal DNA. The PCR fragment (about 800 bp, including the\ncoding region of ylaC as well as 200 bp of upstream sequence and 80 bp of\ndownstream sequence) was digested with HindIII and BamHI and ligated into\npUC18 to generate plasmid pMC62. pMC62 was digested with StyI and HincII\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n(releasing a 170-bp internal gene fragment) and ligated with the same Kanr\ncassette (digested with XbaI and HincII) to generate pMC66. pMC66 was lin-\nearized with ScaI and moved into B. subtilis strain CU1065 by transformation,\nselecting for Kanr, to generate strain HB0029. The Kanr cassette is oriented in\nthe same direction as ylaC.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nThe sigW ylaC double mutant HB0139 was constructed by transforming chro-\nmosomal DNA from HB0029 (ylaC::kan) into HB0020 (sigW::MLS) and select-\ning for Kanr and MLS resistance. The sigX sigW sigM triple mutant HB0099 was\nconstructed by moving chromosomal DNA from HB0020 (sigW::MLS) into\nHB0097 (sigXsigM) by transformation and selecting for Spcr, Kanr, and MLS\nresistance.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nConstruction of null mutants in yrhJ, ydaH, ypbG, yraA, radA, yacK, radC, and\nyqjL. Long-ﬂanking homology PCR was used as described previously (45) to\ngenerate allelic replacement mutants for each gene or group of genes. In brief,\napproximately 1,000-bp genomic regions ﬂanking the gene(s) to be deleted were\nampliﬁed from CU1065 chromosomal DNA by PCR. Drug resistance cassettes\nwere ampliﬁed by PCR from pGEM-cat-3Zf() (48) (cat) or pDG1513 (tet) (12).\nFor each mutant construction, equal amounts (approximately 200 to 300 ng) of\npuriﬁed upstream ﬂanking fragment, downstream ﬂanking fragment, and the\ncorresponding drug resistance cassette were used in a joint PCR procedure using the\nHotStarTaq master mix kit (QIAGEN). The resulting PCR products were puriﬁed,\nand B. subtilis wild-type strain CU1065 was transformed selecting for the corre-\nsponding antibiotic resistance. The generated mutant strains are listed in Table 1. A\nyqjL bcrC double mutant strain (HB0154) was constructed by transformation of the\nbcrC mutant strain (HB0106) to Tetr using yqjL chromosomal DNA.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nOverproduction of SodA, BcrC, and YqjL in B. subtilis using a xylose-inducible\nsystem for complementation tests. The sodA open reading frame was PCR\nampliﬁed with primers 900 and 901, digested with BamHI and EcoRI, and cloned\ninto pXT, a derivative of pDG1731 that places the inserted gene under control\nof the inducible PxylA promoter and can be integrated into the thrC locus (3). The\nresulting plasmid was designated pMC121 (PxylA-sodA). pMC121 was linearized\nwith ScaI and ectopically integrated into B. subtilis strains HB0141 (sodA::cat)\nand HB0031 (sigM::kan) at the thrC locus by transformation with selection for\nSpcr. The transformants were screened for MLS sensitivity (indicative of a\ndouble-crossover event) and threonine auxotrophy. The resulting strains were\ndesignated HB0144 (sodA, with PxylA-sodA at the thrC locus) and HB0145 (sigM,\nwith PxylA-sodA at the thrC locus).\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nPrimer sets 732-919 and 1668-1669 were used to amplify the bcrC and yqjL\nopen reading frames, respectively. The fragments were ﬁrst cloned into pCR2.1-\nTOPO vector (Invitrogen) and digested with EcoRI. The resulting fragments\nwere cloned into pXT at the EcoRI site. Plasmids with correct orientation were\nselected and designated pMC122 (PxylA-yqjL) and pMC123 (PxylA-bcrC). The\nsame procedures as described above were used to introduce the plasmid into the\nthrC locus of corresponding mutant strains. For overproduction, xylose was\nadded to the growth medium to a ﬁnal concentration of 1%.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nConstruction of PyqjL-cat-lacZ transcriptional fusion. The yqjL promoter re-\ngion was ampliﬁed from B. subtilis chromosomal DNA by PCR using primers 552\nand 553. The resulting fragment was digested with HindIII and BamHI and\ncloned into pJPM122 (42) to generate plasmid pMC88 (PyqjL-cat-lacZ). The\nsequence of the promoter region was veriﬁed by DNA sequencing (Cornell\nUniversity BioResources facility). Plasmid pMC88, carrying the promoter fusion\nwas linearized with ScaI and introduced into the modiﬁed SP prophage of B.\nsubtilis strain ZB307A (50) by a double-crossover recombination with selection\nfor Neo resistance. This SP prophage is a temperature-sensitive lysogen, and\nplasmid integration results in the generation of promoter region cat-lacZ operon\nfusions in which both chloramphenicol resistance and -galactosidase can be\nused as reporters for promoter activity (42). SP lysates were prepared by heat\ninduction as described previously (42) and used to transduce CU1065 and\nHB0031 (sigM::kan) to generate strains HB0183 and HB0184.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nZone of inhibition assay. B. subtilis strains were grown at 37°C overnight in LB\nor LS medium, diluted 1:100 into fresh LB or LS medium, and grown at 37°C\nwith shaking. At mid-logarithmic phase, 20 l of culture was mixed with 2 ml of\n0.7% soft LB or LS medium agar and poured onto the bottom agar. After\ncooling, ﬁlter paper disks (6-mm diameter) carrying chemicals to be tested were\nplaced on the top of the agar and the plates were incubated at 37°C overnight.\nThe next day, the diameters of the inhibition zones were measured.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nGradient plate assays. B. subtilis strains were grown in LB or LS medium at\n37°C overnight, diluted 1:100 into LB or LS medium, and grown at 37°C with\nshaking. Mid-logarithmic-growth-phase cultures were adjusted to an optical density\nat 600 nm (OD600) of 0.3 by dilution with prewarmed LB or LS and streaked onto\ngradient agar plates using a sterile Dacron swab applicator. Plates were prepared\nusing Omnitray single-well microtiter plates (Nalge Nunc International) and poured\nto a thickness of 0.8 cm. The initially poured agar wedge (34 ml) contained the test\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\ncompound and was allowed to solidify for 30 min prior to the addition of a second\nagar wedge (34 ml). Plates were allowed to equilibrate for 2 h at room temperature\nprior to inoculation and overnight incubation (37°C).\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nSOD activity assay. Overnight B. subtilis cultures (wild type, sigM, and sodA\nmutants) were diluted 1:100 into LB medium and grown at 37°C. Cell samples were\ncollected at early, mid-, and late log growth phase and washed in Tris-HCl buffer (50\nmM Tris-HCl, pH 9.0, 5% glycerol, 30 M dithiothreitol). Cell pellets were resus-\npended in Tris-HCl buffer and lysed using sonication and lysozyme. The total protein\nconcentration was determined by the Bradford dye-binding assay (Bio-Rad). Total\nprotein (30 g) was applied onto a 10% native polyacrylamide gel electrophoresis\ngel. Bands representing superoxide dismutase (SOD) activity were visualized by\nnegative staining using the nitroblue tetrazolium method (2).\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nParaquat induction and -galactosidase assay. Cells from overnight cultures\nwere inoculated into fresh LB medium and grown to an OD600 of 0.2. The culture\nwas split into two aliquots, one of which was challenged by the addition of\nparaquat to a ﬁnal concentration of 80 M. The culture was returned to 37°C,\nand cell samples were collected every 15 min. The -galactosidase activity of each\nsample was measured according to the method of Miller (26).\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nSupplemental ﬁgures and material. A complete listing of the primers used\nduring strain construction (Table 1S) and supplemental Fig. 1S are available at\nhttp://www.micro.cornell.edu/faculty.JHelmann.html.\n**BLOCK**fs== 9.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nRESULTS\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nThe M ECF  factor contributes to oxidative stress resis-\ntance. We tested the sensitivity of mutant strains altered in\neach of the seven ECF  factors to hydrogen peroxide (8.8 M),\ntert-butyl hydroperoxide (7.77 M), cumene hydroperoxide (0.53\nM), diamide (1 M), and PQ (0.5 M) using zone of inhibition\nexperiments. Only sigM affected oxidative stress resistance and\nonly to PQ (Fig. 1A). Moreover, induction of sigM from a\nxylose-inducible promoter (PxylA) completely restored PQ re-\nsistance to the sigM mutant (Fig. 1A). We conclude that M\ncontributes speciﬁcally to superoxide resistance and not to\nredox stress resistance in general. In contrast, no increased\nsensitivity to any of these reagents was observed for the sigW,\nylaC, or sigW ylaC mutant despite the fact that it is these \nfactors that are regulated by ZAS-type anti- factors.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nPrevious studies have demonstrated that the M regulon has\nsome overlap with regulons controlled by X and W (4, 5, 27,\n33). However, analysis of sigX sigM and sigW sigM dou-\nble mutants and a sigX sigW sigM triple mutant did not re-\nveal an increase in PQ sensitivity relative to the sigM single\nmutant.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nThe sigM mutant is only sensitive to superoxide-generating\nreagents with a dipyridyl ring. The sigM mutant, while sensi-\ntive to PQ, did not have an increased sensitivity to the struc-\nturally distinct superoxide generators menadione, pyrogallol,\nor plumbagin (Fig. 1B). Whereas PQ is a dication with a 4,\n4-dipyridyl ring, menadione, pyrogallol, and plumbagin are\nstructurally distinct (Fig. 1C). The correlation with chemical\nproperties was reinforced by the ﬁnding that the sigM mutant\nwas more sensitive than the wild type to EV and BV (both with\na 4, 4-dipyridyl ring) and DQ (containing a 2, 2-dipyridyl\nring), all of which are also cationic compounds (Table 2). In\ncontrast, the sigM mutant was not more sensitive to compounds\nwith pyridine or dipyridyl rings that do not generate superoxide\nradicals (pyridine, 2, 2-dipyridyl, and 4, 4-dipyridyl), benzi-\ndine and 4, 4-bimethylbiphenyl, which are structurally similar\nto PQ but have a biphenyl ring and do not generate superoxide\nradicals, or nitrofurantoin, a structurally distinct superoxide\ngenerator (Table 2).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nSince PQ, EV, BV, and DQ are all superoxide generators,\nwe hypothesized that their toxicity was linked to the production\n**BLOCK**fs== 8.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nFIG. 1. Role of M in resistance to paraquat. (A) Sensitivity of the\nsigM mutant to PQ but not to diamide (left). Lawns of wild type (wt) and\nsigW were generated by overlaying LB agar plates with 2 ml of soft LB\nagar containing 200 l of mid-log-phase (OD600\n 0.4) cultures. Five\nmicroliters of 0.5 M PQ or 1 M diamide was spotted on the ﬁlter paper\ndisk and added to the plate. The plates were scanned after overnight\nincubation at 37°C. PQ resistance of the sigM mutant was completely\nrestored by overexpression of sigM under a xylose-inducible system\n(right). Xylose (1% [wt/vol]) was added to the cell culture and was present\nin the soft LB agar for induction. (B) The sigM mutant was sensitive to PQ\nbut not to structurally different redox cycling compounds. Five microliters\nof 0.5 M PQ, 1% menadione, 400 mg/ml pyrogallol, or 100 mM plumbagin\nwas spotted on the ﬁlter disk, respectively. (C) The chemical structures of\nparaquat, menadione, pyrogallol, and plumbagin.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nof reactive oxygen species. Consistent with this hypothesis, the\nsigM mutant and wild type displayed greatly reduced, but\nequal, sensitivity to these compounds in zone of inhibition\nassays under anaerobic conditions (data not shown). This in-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.3**r== 0.3**\nTABLE 2. Chemical sensitivity of wild-type and mutant B. subtilis strains\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nCompound (concn)a\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.3**r== 0.5**\nCompound structureb\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nMedium\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.9**\nPQ (0.5 M)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nEV (1.5 M)\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nBV (50 mg/ml)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nDQ (0.5 g/ml)\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nPyridine\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n2,2-dipyridyl (50 mM)\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n4,4-dipyridyl (1.0 M)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nBenzidine (4.0 mM)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n4,4-bimethylbiphenyl (1.0 M)\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nNitrofurantoin (2.0 mM)\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\nMitomycin C (1 mg/ml)\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n4NQO (100 mM)\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nMethyl methancsulfonate (11.0 M)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\na Concentration added to ﬁlter disk (10 l).\nb Structure added where appropriate to data interpretation.\nc Filters were 6 mm in diameter.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.5**\nLB\nLS\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.5**\nLB\nLS\nLB\nLS\nLB\nLS\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.1**\nDiam of inhibition zone (mm) forc:\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nsigM\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nMutant\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\nyqjL\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\nbcrC\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.7**r== 0.3**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.8**r== 0.2**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.9**r== 0.1**\n\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nWild\ntype\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\ndicates that one role of M is to protect cells against the ability\nof these compounds to trigger oxidative stress.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSuperoxide radicals cause damage to proteins, DNA, and\nlipids. As the M regulon includes several putative DNA repair\nproteins (e.g., RadA, RadC, and YpbG), we tested whether the\nsigM mutant was also sensitive to other DNA-damaging agents.\nHowever, no signiﬁcant differences were observed between the\nwild type and the sigM mutant in sensitivity to mitomycin C,\n4NQO, or methyl methanesulfonate (Table 2).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nPQ sensitivity of the sigM mutant is not due to a decrease in\nsuperoxide dismutase activity. In B. subtilis, only one detect-\nable superoxide dismutase (SodA) is found in both vegetative\ncells and spores (21). The factors controlling expression of\nsodA are not well deﬁned, but sodA is not thought to be part of\nthe M regulon. Moreover, a sodA mutant was sensitive not\nonly to PQ but also to menadione, pyrogallol, and plumbagin\n(data not shown). This suggests that the effect of the M mu-\ntation is unlikely to be due to effects on sodA expression.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nFIG. 2. M does not affect SOD activity in B. subtilis. Wild-type\n(lanes 1), sigM (lanes 2), and sodA (lanes 3) mutant cells were har-\nvested during early log, mid-log, and transition (late log/onset of sta-\ntionary) phases. Total protein (30 g) was applied to a 10% native\npolyacrylamide gel electrophoresis gel. Bands representing SOD ac-\ntivity were visualized by negative staining using the nitroblue tetrazo-\nlium method (2).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nIndeed, induction of sigM from the xylose-inducible PxylA pro-\nmoter completely restored PQ resistance to the sigM mutant\n(Fig. 1A) but not to the sodA mutant (data not shown). Con-\nversely, induction of sodA from PxylA could fully restore PQ\nresistance to the sodA mutant but not to the sigM mutant (data\nnot shown). In addition, we directly measured SOD activity on\na native polyacrylamide gel by negative staining using the ni-\ntroblue tetrazolium method (2). Only one active band, in ap-\nproximately equal intensities, was seen in both the wild-type\nand the sigM mutant strains at all growth phases (Fig. 2). This\nband corresponds to SodA, as it disappeared in the sodA mu-\ntant.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nYqjL and BcrC are implicated in paraquat resistance. To\nidentify genes responsible for resistance to PQ, we generated\nnull mutants in several M-regulated genes including radA,\nyacK, radC, ypbG, ydaH, yqjL, yrhJ, and yraA (1, 44). In addi-\ntion, we previously generated a null mutant for bcrC, a gene\ncontrolled by both M and X (4). Three of these genes (radA,\nradC, and ypbG) encode putative DNA repair proteins, yacK\nencodes a DNA-binding protein, ydaH encodes a membrane\nprotein with unknown function, and yqjL encodes a member of\nthe / hydrolase family. YrhJ is a cytochrome P450/NADPH\nreductase, and YraA is 61% identical to the B. subtilis general\nstress protein YfkM. Of these nine mutant strains, only the\nyqjL mutant displayed an increased sensitivity comparable to\nthat seen in the sigM mutant on LB plates containing 30 M\nPQ (Fig. 3 and data not shown). Although a bcrC mutant did\nnot display a signiﬁcantly increased PQ sensitivity, the yqjL\nbcrC double mutant was more sensitive than either single mu-\ntant and comparable to the sigM mutant (Fig. 3 and 4). Over-\nexpression of yqjL, under the control of a xylose-inducible\npromoter (PxylA), restored PQ resistance to both the yqjL and\nthe sigM mutants (Fig. 4), but overexpression of bcrC could not\nrestore the PQ resistance of the sigM mutant. These results\nsupport the notion that YqjL is the primary M-dependent\ndeterminant of PQ resistance.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSince the disk diffusion assay requires growth in the pres-\nence of the superoxide generator, it does not distinguish be-\ntween bactericidal and bacteriostatic effects. To monitor bac-\ntericidal effects, we added PQ to mid-logarithmic-phase cells\nand monitored both growth and viability. PQ resulted in a\ncessation of growth after approximately 20 min for both the\nwild type and the single and double yqjL bcrC mutants (Fig.\n5A). Moreover, the rate of loss of viability (as measured by\nspotting dilutions of the culture onto LB plates) was indistin-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nFIG. 3. The yqjL and bcrC genes affect growth on plates containing\nPQ. Dilutions (1, 101, 102, 103, 104, and 105) of log-phase cell\n 0.4) were spotted on LB and LB plus 30 M PQ\ncultures (OD600\nplates and incubated at 37°C overnight. This experiment was repeated\nseveral times with various concentrations of PQ, and similar results\nwere observed. wt, wild type.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nguishable between the four tested strains (Fig. 5B and data not\nshown). In all cases, there was approximately 99% killing by 80\nmin after addition. The difference between the PQ-sensitive\nphenotype observed in the plate-based growth assay and the\nlack of effect noted in these viability measurements may indi-\ncate that PQ has a bacteriostatic effect that is monitored on\nsolid media, that the surface- and liquid-grown cells are phys-\niological different, or both.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe toxicity of PQ and EV is exacerbated by low NaCl\nconcentrations. In E. coli, high-salt concentrations, such as\nfound in LB (over 170 mM), decrease PQ sensitivity relative to\nlow-salt culture medium, perhaps by inhibition of uptake (22).\nSimilarly, B. subtilis and the sigM mutant were more sensitive\nto PQ and EV (but not to BV or DQ) on LS medium than on\nLB (Table 2). Using the disk diffusion assay, we demonstrated\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nFIG. 4. Both YqjL and BcrC contribute to protection against PQ.\nThe sensitivity of each indicated strain was measured by zone of inhi-\nbition assays. The PQ resistance of the sigM mutant and the yqjL\nmutant was restored by overexpression of yqjL under control of a\nxylose-inducible promoter (PxylA). Xylose (1% [wt/vol]) was added ()\nto the culture and in the soft LB agar for induction. The error bars\nindicate the standard deviations of the results from replicate samples.\nwt, wild type.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nthat both yqjL and bcrC contribute to resistance to PQ, EV,\nBV, and DQ (Table 2). Like the sigM mutant, the sensitivity of\nthese mutants to PQ and EV was higher on LS than on LB.\nUnder these low-salt conditions, a PQ-sensitive phenotype for\nthe bcrC single mutant is also readily apparent (Table 2).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nYqjL and BcrC together account for the sensitivity of the\nsigM mutant. The PQ sensitivity of the yqjL bcrC double mu-\ntant was greater than either single mutant as measured by both\ndisk diffusion (Fig. 4) and cell plating assays (Fig. 3). To extend\nthese results to other stress agents and take advantage of the\nobservation that the apparent role of bcrC is enhanced under\nlow-salt conditions, we used a gradient plate analysis to com-\npare yqjL and bcrC single and double mutants to the wild type\n(Table 3). Although the effect of the bcrC mutation alone is\nonly apparent with PQ (and only under low-salt conditions),\nthe additive effect of the bcrC and yqjL mutations is apparent\nfor all four tested compounds, at least under low-salt condi-\ntions. For example, both the sigM and bcrC yqjL double mutant\ngrow across 2.6 cm of the gradient plate, whereas the bcrC and\nyqjL single mutants grow 9.0 and 4.7 cm, respectively.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nExpression of yqjL is M dependent and is inducible by PQ.\nWe used a PyqjL-cat-lacZ reporter fusion, integrated ectopically\nat SP, to measure expression in both wild-type and sigM\nmutant strains. Expression was generally low in the wild-type\nstrain, with maximum activity during early to mid-log growth\nphase and then decreasing during late log phase (Fig. 6A), a\npattern similar to the expression of sigM (18). The expression\nwas reduced to the background level in the sigM mutant strain\n(Fig. 6A). Addition of 80 M PQ to the growth medium led to\na strong induction of yqjL expression (Fig. 5A) but only after a\n30-min delay. A similar delayed response to PQ is observed for\nthe autoregulatory sigM promoter (Fig. 6B) (PM-cat-lacZ), as\noriginally reported by Thackray and Moir (44). Note that both\nthe basal and induced expression from these promoters is\ncompletely dependent on M. Although the cloned sigM reg-\nulatory region also contains a A-dependent promoter de-\nscribed previously (18), expression is entirely M dependent in\nour construct (4).\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nDISCUSSION\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nWe have demonstrated that a M null mutant is more sen-\nsitive to a subfamily of superoxide-generating reagents related\nto PQ but not to other superoxide generators such as mena-\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nFIG. 5. YqjL does not protect against the bactericidal effects of\nPQ. (A) The sensitivity of CU1065 and yqjL mutant cells was moni-\ntored in LB broth in the presence and absence of 1 mM PQ added at\nan OD600 of 0.3. PQ quickly resulted in bacteriostasis of both CU1065\nand the yqjL mutant. Growth was monitored by OD600. (B) Viability\nwas monitored by spotting 5 l of a 10-fold dilution series on LB plates\n20, 80, and 120 min after PQ addition to monitor cell killing. PQ\naddition resulted in signiﬁcant killing in both the CU1065 and yqjL\ncultures as observed in the 80- and 120-min samples.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.2**r== 0.2**\nTABLE 3. Resistance to superoxide-generating compounds as measured by a solid medium gradient plate assay\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nCompound\n(concn[M])a\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.2**r== 0.8**\nMediumb\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nPQ (50)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.9**\nEV (50)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.9**\nBV (100)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\nDQ (50)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.2**r== 0.8**\nLB\nLS\nLB\nLS\nLB\nLS\nLB\nLS\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.3**r== 0.6**\nCU1065 (wild type)\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\nsigM mutant\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.6**r== 0.4**\nyqjL mutant\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nbcrC mutant\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nyqjL bcrC mutant\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\nLength (cm) (% of wild type) forc:\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.3**r== 0.6**\n12.0 (100)\n12.0 (100)\n12.0 (100)\n12.0 (100)\n12.0 (100)\n12.0 (100)\n12.0 (100)\n9.0 (100)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\n4.5 (38)\n1.5 (13)\n6.4 (53)\n3.0 (25)\n3.8 (32)\n3.4 (28)\n3.1 (26)\n2.6 (29)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.6**r== 0.4**\n5.5 (46)\n1.8 (15)\n9.2 (77)\n3.1 (26)\n6.3 (53)\n6.3 (53)\n5.0 (42)\n4.7 (52)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.2**\n12.0 (100)\n6.9 (57.5)\n12.0 (100)\n12.0 (100)\n12.0 (100)\n12.0 (100)\n12.0 (100)\n9.0 (100)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.8**r== 0.1**\n5.5 (46)\n1.5 (13)\n6.3 (55)\n2.3 (19)\n5.3 (44)\n4.2 (35)\n4.6 (38)\n2.6 (29)\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\na Maximum concentration of PQ, EV, BV, and DQ in growth medium.\nb Luria-Bertani medium (LB) or low-salt Luria-Bertani medium (LS).\nc Results reported as length (in cm) of growth band from agar with no compound into zone of increasing concentrations and as the percentage of this length compared\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nwith the wild-type strain from the same culture plate.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nboth required for efﬁcient export of PQ (39). SmvA is similar\nto the MvrA protein of E. coli (17, 29). A similar efﬂux system\nhas been identiﬁed in S. coelicolor encoded by the pqrAB\noperon (9). The PqrA protein is a predicted transcriptional\nrepressor, while PqrB is an MvrA/SmvA ortholog. B. subtilis\ncontains a similarly organized operon (yvkBA) that may also\nplay a role in PQ resistance (9); however, the yvkBA operon\nwas not identiﬁed in global surveys of B. subtilis genes induc-\nible by PQ or other oxidative stress agents (15, 23, 30), nor has\nexpression of yvkBA been linked to M (1, 44).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nTo identify genes that may play a role in PQ resistance, we\nanalyzed null mutants defective for each of nine different M-\ndependent genes. These genes, chosen from the list of genes\npreviously shown to be M dependent (4, 44) or found to be\nstrongly induced upon induction of M in microarray studies\n(1), included bcrC, radA, yacK, radC, ypbG, ydaH, yrhJ, yraA,\nand yqjL. Among these genes, only yqjL and bcrC signiﬁcantly\naffected PQ resistance. Expression of YqjL from a xylose-\ninducible promoter complemented the PQ sensitivity of the\nsigM mutant, suggesting that this protein is the primary, M-\ndependent defense against PQ. However, the BcrC bacitracin\nresistance protein (4) also plays a role in PQ resistance.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nThe yqjL gene was originally thought to be transcribed by\neither or both W and X, two other B. subtilis ECF  factors\n(5). A GAAAC-N17-CGTC sequence 34 bp upstream of yqjL\nfunctions as a promoter for both the W holoenzyme (as seen\nin in vitro runoff transcription experiments) and X holoen-\nzyme (observed in runoff transcription/macroarray analysis\n[ROMA] experiments) (5). Induction of yqjL was also noted\nduring the characterization of the vancomycin stimulon (7).\nHowever, induction was independent of W, consistent with\nthe notion that yqjL may be regulated primarily by M, which\ncontrols several genes inducible by cell wall biosynthesis inhib-\nitors (4, 7, 44).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nThe mechanism by which YqjL confers paraquat resistance\nremains unclear. YqjL has similarity to members of the /\nhydrolase family protein, suggesting a possible role in the deg-\nradation of paraquat. The / hydrolase fold is common to a\nnumber of hydrolytic enzymes of widely differing phylogenetic\norigin and catalytic function (16). One example is TodF of\nPseudomonas putida, which is involved in toluene degradation\n(41). This suggests that YqjL may participate in the degrada-\ntion of dipyridyl-containing compounds. Indeed, biodegrada-\ntion of PQ by other soil bacteria, including Streptomyces and\nNocardia spp., has been observed, but the molecular mecha-\nnism has not been characterized (32). Using a spectrophoto-\nmetric assay, we observed little if any degradation of 50 M\nPQ present in the culture medium over a 24-h period of growth\nof either wild-type or sigM mutant strains (see Fig. 1S in the\nsupplemental material). Moreover, incubation with puriﬁed\nYqjL protein in vitro did not reduce the toxicity of PQ. De-\ntermining the mechanism of action of YqjL will clearly require\nfurther study. Similarly, the role of BcrC in PQ resistance is not\nyet clear. BcrC is a predicted membrane permease (although it\nalso has similarity to the PAP2 family of acid phosphatases)\nand contributes to bacitracin resistance (4, 13, 33, 37, 38). BcrC\ncould be involved in the efﬂux of either PQ or a PQ degrada-\ntion product.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nInterestingly, yqjM, encoding a ﬂavoprotein xenobiotic re-\nductase of the Old Yellow Enzyme family (46), is encoded\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nFIG. 6. PQ elicits induction of both yqjL and sigM. Induction of\nPyqjL-cat-lacZ (A) and PM-cat-lacZ (B) in LB medium by 80 M PQ.\n-Galactosidase activity (ﬁlled symbols) and cell density (OD600, open\nsymbols) are shown for the wild-type strains in the absence (squares)\nand presence (diamonds) of PQ. Expression is greatly reduced when\nthe fusions are in the sigM mutant background in either the absence\n(circles) or presence (triangles) of PQ (symbols along the x axis).\nWhere indicated, PQ was applied when the cell density (OD600)\nreached 0.2 (0 min). The results shown are representative of several\nreplicates, although in some cases, the magnitude of induction was\nsomewhat less (2- to 3-fold). The lysis observed in this experiment at\nthe late time points in panel A was not seen in other replicates.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\ndione, pyrogallol, or plumbagin. This sensitivity was not due to\na decrease in SodA activity because, unlike the sigM mutant,\nthe sodA mutant is sensitive to all tested superoxide generators\nand the same level of SOD activity was detected in both the\nwild type and the sigM mutant.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nPQ resistance is most frequently mediated by an efﬂux mech-\nanism. For example, Salmonella enterica serovar Typhimurium\nhas two efﬂux pumps (SmvA and YddG plus OmpD) that are\n**BLOCK**fs== 9.0**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nabout 113 bp upstream of yqjL. Expression of yqjM is activated\nby hydrogen peroxide and paraquat, as determined using both\nDNA microarray technology (30) and immunological analysis\nof YqjM protein levels (11), indicating its potential role in\ndefending against oxidative stress. Members of this enzyme\nfamily reduce a wide range of compounds containing /-\nunsaturated aldehydes and ketones (46). There is a transcrip-\ntion terminator between these two genes; thus, it is unlikely\nthat expression of yqjL is due to readthrough from yqjM. More-\nover, yqjL was not identiﬁed as PQ inducible under the con-\nditions used for the DNA microarray studies (10-min expo-\nsure). Conversely, only yqjL (and not yqjM) was found to be\ninduced by expression of M (1). The independent regulation\nof these two enzymes suggests that they may function in dis-\ntinct pathways for the protection against redox active com-\npounds.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nACKNOWLEDGMENTS\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nWe thank Ahmed Gaballa and Jin-Won Lee for helpful discussions\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nduring this study.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n17. Hongo, E., M. Morimyo, K. Mita, I. Machida, H. Hama-Inaba, H. Tsuji, S.\nIchimura, and Y. Noda. 1994. The methyl viologen-resistance-encoding gene\nsmvA of Salmonella typhimurium. Gene 148:173–174.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n18. Horsburgh, M. J., and A. Moir. 1999. Sigma M, an ECF RNA polymerase\nsigma factor of Bacillus subtilis 168, is essential for growth and survival in\nhigh concentrations of salt. Mol. Microbiol. 32:41–50.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n19. Huang, X., A. Decatur, A. Sorokin, and J. D. Helmann. 1997. The Bacillus\nsubtilis X protein is an extracytoplasmic function sigma factor contributing\nto the survival of high temperature stress. J. Bacteriol. 179:2915–2921.\n20. Huang, X., and J. D. Helmann. 1998. Identiﬁcation of target promoters for\nthe Bacillus subtilis X factor using a consensus-directed search. J. Mol. Biol.\n279:165–173.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n21. Inaoka, T., Y. Matsumura, and T. Tsuchido. 1998. Molecular cloning and\nnucleotide sequence of the superoxide dismutase gene and characterization\nof its product from Bacillus subtilis. J. Bacteriol. 180:3697–3703.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n22. Kitzler, J., and I. Fridovich. 1986. Effects of salts on the lethality of paraquat.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nJ. Bacteriol. 167:346–349.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n23. Leichert, L. I., C. Scharf, and M. Hecker. 2003. Global characterization of\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\ndisulﬁde stress in Bacillus subtilis. J. Bacteriol. 185:1967–1975.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n24. Lonetto, M. A., K. L. Brown, K. E. Rudd, and M. J. Buttner. 1994. Analysis\nof the Streptomyces coelicolor sigE gene reveals the existence of a subfamily\nof eubacterial  factors involved in the regulation of extracytoplasmic func-\ntions. Proc. Natl. Acad. Sci. USA 91:7573–7577.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n25. Manganelli, R., R. Provvedi, S. Rodrigue, J. Beaucher, L. Gaudreau, and I.\nSmith. 2004. Sigma factors and global gene regulation in Mycobacterium\ntuberculosis. J. Bacteriol. 186:895–902.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n26. Miller, J. H. 1972. Experiments in molecular genetics, p. 352–355. Cold\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nThis work was supported by grant GM47446 from the NIH.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.2**\nSpring Harbor Laboratory, Cold Spring Harbor, N.Y.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n1. Asai, K., H. Yamaguchi, C. M. Kang, K. Yoshida, Y. Fujita, and Y. Sadaie.\n2003. DNA microarray analysis of Bacillus subtilis sigma factors of extracy-\ntoplasmic function family. FEMS Microbiol. Lett. 220:155–160.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n2. Beauchamp, C., and I. Fridovich. 1971. Superoxide dismutase: improved\nassays and an assay applicable to acrylamide gels. Anal. Biochem. 44:276–\n287.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n3. Cao, M., B. A. Bernat, Z. Wang, R. N. Armstrong, and J. D. Helmann. 2001.\nFosB, a cysteine-dependent fosfomycin resistance protein under the control\nof W, an extracytoplasmic-function sigma factor in Bacillus subtilis. J. Bac-\nteriol. 183:2380–2383.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n4. Cao, M., and J. D. Helmann. 2002. Regulation of the Bacillus subtilis bcrC\nbacitracin resistance gene by two extracytoplasmic function sigma factors. J.\nBacteriol. 184:6123–6129.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n5. Cao, M., P. A. Kobel, M. M. Morshedi, M. F. Wu, C. Paddon, and J. D.\nHelmann. 2002. Deﬁning the Bacillus subtilis W regulon: a comparative\nanalysis of promoter consensus search, run-off transcription/macroarray\nanalysis (ROMA), and transcriptional proﬁling approaches. J. Mol. Biol.\n316:443–457.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n6. Cao, M., L. Salzberg, C. S. Tsai, T. Mascher, C. Bonilla, T. Wang, R. W. Ye,\nL. Marquez-Magana, and J. D. Helmann. 2003. Regulation of the Bacillus\nsubtilis extracytoplasmic function protein Y and its target promoters. J.\nBacteriol. 185:4883–4890.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n7. Cao, M., T. Wang, R. Ye, and J. D. Helmann. 2002. Antibiotics that inhibit\ncell wall biosynthesis induce expression of the Bacillus subtilis W and M\nregulons. Mol. Microbiol. 45:1267–1276.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n8. Casillas-Martinez, L., and P. Setlow. 1997. Alkyl hydroperoxide reductase,\ncatalase, MrgA, and superoxide dismutase are not involved in resistance of\nBacillus subtilis spores to heat or oxidizing agents. J. Bacteriol. 179:7420–\n7425.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n9. Cho, Y. H., E. J. Kim, H. J. Chung, J. H. Choi, K. F. Chater, B. E. Ahn, J. H.\nShin, and J. H. Roe. 2003. The pqrAB operon is responsible for paraquat\nresistance in Streptomyces coelicolor. J. Bacteriol. 185:6756–6763.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n10. D’Autreaux, B., O. Horner, J. L. Oddou, C. Jeandey, S. Gambarelli, C.\nBerthomieu, J. M. Latour, and I. Michaud-Soret. 2004. Spectroscopic de-\nscription of the two nitrosyl-iron complexes responsible for fur inhibition by\nnitric oxide. J. Am. Chem. Soc. 126:6005–6016.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n11. Fitzpatrick, T. B., N. Amrhein, and P. Macheroux. 2003. Characterization of\nYqjM, an Old Yellow Enzyme homolog from Bacillus subtilis involved in the\noxidative stress response. J. Biol. Chem. 278:19891–19897.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n12. Guerout-Fleury, A. M., N. Frandsen, and P. Stragier. 1996. Plasmids for\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nectopic integration in Bacillus subtilis. Gene 180:57–61.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n13. Harel, Y. M., A. Bailone, and E. Bibi. 1999. Resistance to bacitracin as\nmodulated by an Escherichia coli homologue of the bacitracin ABC trans-\nporter BcrC subunit from Bacillus licheniformis. J. Bacteriol. 181:6176–6178.\n14. Helmann, J. D. 2002. The extracytoplasmic function (ECF) sigma factors.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nAdv. Microb. Physiol. 46:47–110.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n15. Helmann, J. D., M. F. Wu, A. Gaballa, P. A. Kobel, M. M. Morshedi, P.\nFawcett, and C. Paddon. 2003. The global transcriptional response of Bacil-\nlus subtilis to peroxide stress is coordinated by three transcription factors. J.\nBacteriol. 185:243–253.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n16. Holmquist, M. 2000. Alpha/beta-hydrolase fold enzymes: structures, func-\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\ntions and mechanisms. Curr. Protein Pept. Sci. 1:209–235.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n27. Minnig, K., J. L. Barblan, S. Kehl, S. B. Moller, and C. Mauel. 2003. In\nBacillus subtilis W23, the duet XM, two sigma factors of the extracytoplas-\nmic function subfamily, are required for septum and wall synthesis under\nbatch culture conditions. Mol. Microbiol. 49:1435–1447.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n28. Moore, C. M., M. M. Nakano, T. Wang, R. W. Ye, and J. D. Helmann. 2004.\nResponse of Bacillus subtilis to nitric oxide and the nitrosating agent sodium\nnitroprusside. J. Bacteriol. 186:4655–4664.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n29. Morimyo, M. 1988. Isolation and characterization of methyl viologen-sensi-\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\ntive mutants of Escherichia coli K-12. J. Bacteriol. 170:2136–2142.\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n30. Mostertz, J., C. Scharf, M. Hecker, and G. Homuth. 2004. Transcriptome\nand proteome analysis of Bacillus subtilis gene expression in response to\nsuperoxide and peroxide stress. Microbiology 150:497–512.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n31. Nakano, S., E. Kuster-Schock, A. D. Grossman, and P. Zuber. 2003. Spx-\ndependent global transcriptional control is induced by thiol-speciﬁc oxidative\nstress in Bacillus subtilis. Proc. Natl. Acad. Sci. USA 100:13603–13608.\n32. Namdeo, K. N. 1972. Biodegradation of paraquat dichloride. Indian J. Exp.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nBiol. 10:133–135.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n33. Ohki, R., K. Tateno, Y. Okada, H. Okajima, K. Asai, Y. Sadaie, M. Murata,\nand T. Aiso. 2003. A bacitracin-resistant Bacillus subtilis gene encodes a\nhomologue of the membrane-spanning subunit of the Bacillus licheniformis\nABC transporter. J. Bacteriol. 185:51–59.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n34. Paget, M. S., J. B. Bae, M. Y. Hahn, W. Li, C. Kleanthous, J. H. Roe, and\nM. J. Buttner. 2001. Mutational analysis of RsrA, a zinc-binding anti-sigma\nfactor with a thiol-disulphide redox switch. Mol. Microbiol. 39:1036–1047.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n35. Paget, M. S., J. G. Kang, J. H. Roe, and M. J. Buttner. 1998. sigmaR, an\nRNA polymerase sigma factor that modulates expression of the thioredoxin\nsystem in response to oxidative stress in Streptomyces coelicolor A3(2).\nEMBO J. 17:5776–5782.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n36. Perego, M. 1993. Integrational vectors for genetic manipulation in Bacillus\nsubtilis, p. 615–624. In A. L. Sonenshein, J. A. Hoch, and R. Losick (ed.),\nBacillus subtilis and other gram-positive bacteria: biochemistry, physiology,\nand molecular genetics. American Society for Microbiology, Washington,\nD.C.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n37. Podlesek, Z., A. Comino, B. Herzog-Velikonja, and M. Grabnar. 2000. The\nrole of the bacitracin ABC transporter in bacitracin resistance and collateral\ndetergent sensitivity. FEMS Microbiol. Lett. 188:103–106.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n38. Podlesek, Z., A. Comino, B. Herzog-Velikonja, D. Zgur-Bertok, R. Komel,\nand M. Grabnar. 1995. Bacillus licheniformis bacitracin-resistance ABC\ntransporter: relationship to mammalian multidrug resistance. Mol. Micro-\nbiol. 16:969–976.\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n39. Santiviago, C. A., J. A. Fuentes, S. M. Bueno, A. N. Trombert, A. A. Hildago,\nL. T. Socias, P. Youderian, and G. C. Mora. 2002. The Salmonella enterica sv.\nTyphimurium smvA, yddG and ompD (porin) genes are required for the\nefﬁcient efﬂux of methyl viologen. Mol. Microbiol. 46:687–698.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n40. Schobel, S., S. Zellmeier, W. Schumann, and T. Wiegert. 2004. The Bacillus\nsubtilis W anti-sigma factor RsiW is degraded by intramembrane proteolysis\nthrough YluC. Mol. Microbiol. 52:1091–1105.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n41. Seah, S. Y., G. Terracina, J. T. Bolin, P. Riebel, V. Snieckus, and L. D. Eltis.\n1998. Puriﬁcation and preliminary characterization of a serine hydrolase\ninvolved in the microbial degradation of polychlorinated biphenyls. J. Biol.\nChem. 273:22943–22949.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n42. Slack, F. J., J. P. Mueller, and A. L. Sonenshein. 1993. Mutations that relieve\nnutritional repression of the Bacillus subtilis dipeptide permease operon. J.\nBacteriol. 175:4605–4614.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n43. Song, T., S. L. Dove, K. H. Lee, and R. N. Husson. 2003. RshA, an anti-sigma\nfactor that regulates the activity of the mycobacterial stress response sigma\nfactor SigH. Mol. Microbiol. 50:949–959.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n44. Thackray, P. D., and A. Moir. 2003. SigM, an extracytoplasmic function\nsigma factor of Bacillus subtilis, is activated in response to cell wall antibi-\notics, ethanol, heat, acid, and superoxide stress. J. Bacteriol. 185:3491–3498.\n45. Wach, A. 1996. PCR-synthesis of marker cassettes with long ﬂanking homol-\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nBacillus subtilis extracytoplasmic function (ECF) sigma factors with the N-ter-\nminal regions of their potential anti-sigma factors. Microbiology 150:591–599.\n48. Youngman, P. 1990. Use of transposons and integrational vectors for mu-\ntagenesis and construction of gene fusions in Bacillus species, p. 221–266. In\nC. R. Harwood and S. M. Cutting (ed.), Molecular biological methods for\nBacillus. John Wiley and Sons, Ltd., Chichester, United Kingdom.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n49. Zuber, P. 2004. Spx-RNA polymerase interaction and global transcriptional\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nogy regions for gene disruptions in S. cerevisiae. Yeast 12:259–265.\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\ncontrol during oxidative stress. J. Bacteriol. 186:1911–1918.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n46. Williams, R. E., and N. C. Bruce. 2002. ‘New uses for an Old Enzyme’–the\nOld Yellow Enzyme family of ﬂavoenzymes. Microbiology 148:1607–1614.\n47. Yoshimura, M., K. Asai, Y. Sadaie, and H. Yoshikawa. 2004. Interaction of\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n50. Zuber, P., and R. Losick. 1987. Role of AbrB and SpoOA- and SpoOB-\ndependent utilization of a sporulation promoter in Bacillus subtilis. J. Bac-\nteriol. 169:2223–2230.",
         "extracted",
         "Response of Bacillus subtilis to Nitric Oxide and the Nitrosating Agent Sodium Nitroprusside;The Bacillus subtilisσW anti‐sigma factor RsiW is degraded by intramembrane proteolysis through YluC;Spectroscopic description of the two nitrosyl-iron complexes responsible for fur inhibition by nitric oxide.;Spx-RNA Polymerase Interaction and Global Transcriptional Control during Oxidative Stress;Interaction of Bacillus subtilis extracytoplasmic function (ECF) sigma factors with the N-terminal regions of their potential anti-sigma factors.;σ Factors and Global Gene Regulation in Mycobacterium tuberculosis;Transcriptome and proteome analysis of Bacillus subtilis gene expression in response to superoxide and peroxide stress.;The pqrAB Operon Is Responsible for Paraquat Resistance in Streptomyces coelicolor;Spx-dependent global transcriptional control is induced by thiol-specific oxidative stress in Bacillus subtilis;RshA, an anti‐sigma factor that regulates the activity of the mycobacterial stress response sigma factor SigH;In Bacillus subtilis W23, the duet σXσM, two sigma factors of the extracytoplasmic function subfamily, are required for septum and wall synthesis under batch culture conditions;Regulation of the Bacillus subtilis Extracytoplasmic Function Protein σY and Its Target Promoters;SigM, an Extracytoplasmic Function Sigma Factor of Bacillus subtilis, Is Activated in Response to Cell Wall Antibiotics, Ethanol, Heat, Acid, and Superoxide Stress;Characterization of YqjM, an Old Yellow Enzyme Homolog from Bacillus subtilis Involved in the Oxidative Stress Response*;Global Characterization of Disulfide Stress in Bacillus subtilis;DNA microarray analysis of Bacillus subtilis sigma factors of extracytoplasmic function family.;A Bacitracin-Resistant Bacillus subtilis Gene Encodes a Homologue of the Membrane-Spanning Subunit of the Bacillus licheniformis ABC Transporter;The Global Transcriptional Response of Bacillus subtilis to Peroxide Stress Is Coordinated by Three Transcription Factors;Regulation of the Bacillus subtilis bcrC Bacitracin Resistance Gene by Two Extracytoplasmic Function σ Factors;The Salmonella enterica sv. Typhimurium smvA, yddG and ompD (porin) genes are required for the efficient efflux of methyl viologen;Antibiotics that inhibit cell wall biosynthesis induce expression of the Bacillus subtilisσW and σM regulons;'New uses for an Old Enzyme'--the Old Yellow Enzyme family of flavoenzymes.;Defining the Bacillus subtilis sigma(W) regulon: a comparative analysis of promoter consensus search, run-off transcription/macroarray analysis (ROMA), and transcriptional profiling approaches.;FosB, a Cysteine-Dependent Fosfomycin Resistance Protein under the Control of ςW, an Extracytoplasmic-Function ς Factor in Bacillus subtilis;Mutational analysis of RsrA, a zinc‐binding anti‐sigma factor with a thiol–disulphide redox switch;Alpha/Beta-hydrolase fold enzymes: structures, functions and mechanisms.;The role of the bacitracin ABC transporter in bacitracin resistance and collateral detergent sensitivity.;Resistance to Bacitracin as Modulated by an Escherichia coli Homologue of the Bacitracin ABC Transporter BcrC Subunit from Bacillus licheniformis;σM, an ECF RNA polymerase sigma factor of Bacillus subtilis 168, is essential for growth and survival in high concentrations of salt;σR, an RNA polymerase sigma factor that modulates expression of the thioredoxin system in response to oxidative stress in Streptomyces coelicolor A3(2);Purification and Preliminary Characterization of a Serine Hydrolase Involved in the Microbial Degradation of Polychlorinated Biphenyls*;Identification of target promoters for the Bacillus subtilis sigma X factor using a consensus-directed search.;Alkyl hydroperoxide reductase, catalase, MrgA, and superoxide dismutase are not involved in resistance of Bacillus subtilis spores to heat or oxidizing agents;The Bacillus subtilis sigma(X) protein is an extracytoplasmic function sigma factor contributing to survival at high temperature;PCR‐synthesis of marker cassettes with long flanking homology regions for gene disruptions in S. cerevisiae;Bacillus licheniformis bacitracin‐resistance ABC transporter: relationship to mammalian multidrug resistance;The methyl viologen-resistance-encoding gene smvA of Salmonella typhimurium;Analysis of the Streptomyces coelicolor sigE gene reveals the existence of a subfamily of eubacterial RNA polymerase sigma factors involved in the regulation of extracytoplasmic functions.;Mutations that relieve nutritional repression of the Bacillus subtilis dipeptide permease operon;Isolation and characterization of methyl viologen-sensitive mutants of Escherichia coli K-12;Role of AbrB in Spo0A- and Spo0B-dependent utilization of a sporulation promoter in Bacillus subtilis;Effects of salts on the lethality of paraquat;Biodegradation of paraquat dichloride.;Superoxide dismutase: improved assays and an assay applicable to acrylamide gels.;The extracytoplasmic function (ECF) sigma factors.;Antibiotics that inhibit cell wall biosynthesis induce expression of the Bacillus subtilis sigma(W) and sigma(M) regulons.;FosB, a cysteine-dependent fosfomycin resistance protein under the control of sigma(W), an extracytoplasmic-function sigma factor in Bacillus subtilis.;sigmaR, an RNA polymerase sigma factor that modulates expression of the thioredoxin system in response to oxidative stress in Streptomyces coelicolor A3(2).;Molecular cloning and nucleotide sequence of the superoxide dismutase gene and characterization of its product from Bacillus subtilis;Plasmids for ectopic integration in Bacillus subtilis.;The methyl viologen-resistance-encoding gene smvA of Salmonella typhimurium.;Integrational Vectors for Genetic Manipulation in Bacillus subtilis;Bacillus Subtilis and Other Gram-Positive Bacteria: Biochemistry, Physiology, and Molecular Genetics;Molecular biological methods for Bacillus;Use of transposons and integrational vectors for mutagenesis and construction of gene fusions in Bacillus species;Biodegradation of paraquat dichloride.;Experiments in molecular genetics",
         "Bacillus subtilis Paraquat Resistance Is Directed by σM, an Extracytoplasmic Function Sigma Factor, and Is Conferred by YqjL and BcrC",
         "None"
        ],
        [
         "43",
         "003437ce9cbb8b1b43b45bed926c2e1a54a3d389",
         "Chorioamnionitis and funisitis are associated with neonatal morbidity and mortality. We hypothesized that chorioamnionitis may stress fetal endothelium, activate proinflammatory gene transcription. and affect angiogenic homeostasis in fetal capillaries. Placentas from preterm infants were stained for heat-shock protein 70, nuclear factor-κB, hypoxia-inducible factor-1α, and vascular endothelial growth factor (VEGF). VEGF receptors (VEGF-R) 1 and 2 as well as the receptor tyrosine kinase with immunoglobulin and epidermal growth factor homology domains (TIE-2), which is involved in vascular remodeling, were quantified. Immunohistochemistry was analyzed by counting positive capillaries in placental terminal villi. Staining intensity was quantified by a three-step semiquantitative scale. The samples were divided into three matched groups according to histology: chorioamnionitis with funisitis (“funisitis”), chorioamnionitis without funisitis (“chorioamnionitis”), and control group with no inflammation. In tissues from the funisitis or chorioamnionitis group, heat-shock protein 70 expression was increased over the control group. More nuclear factor-κB–positive nuclei of endothelial cells in capillaries were counted in the funisitis and chorioamnionitis groups. Expression of VEGF and VEGF-R1 and -R2 were reduced in cases of funisitis or chorioamnionitis in comparison with controls. Hypoxia-inducible factor-1α expression tended to be slightly lower in the funisitis and chorioamnionitis groups but did not reach statistical significance. We speculate that cellular stress and changes in angiogenic homeostasis induced by proinflammatory activation of fetal endothelium in chorioamnionitis may not be limited to the placenta but may also involve other fetal organs.",
         "B. Kramer,U. Kaemmerer,M. Kapp,Daniele Herbst,A. Marx,D. Berg,P. Groneck,C. Speer",
         "https://www.nature.com/articles/pr2005710.pdf",
         "\n**BLOCK**fs== 18.0**b== 0.8**t== 0.2**l== 0.2**r== 0.2**\nDecreased Expression of Angiogenic Factors in\nPlacentas with Chorioamnionitis after Preterm\nBirth\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nBORIS W. KRAMER, ULRIKE KAEMMERER, MICHAELA KAPP, DANIELE HERBST,\nALEXANDER MARX, DANIEL BERG, PETER A. GRONECK, AND CHRISTIAN P. SPEER\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nUniversity Children’s Hospital [B.W.K., D.H., C.P.S.], 97080 Würzburg, Germany; Department of Obstetrics\nand Gynecology [U.K., M.K.], Institute of Pathology [A.M.] University of Würzburg, 97080 Würzburg,\nGermany; Children’s Hospital of the City of Cologne [D.B.], 50735 Cologne, Germany; and Children’s\nHospital of the City of Leverkusen [P.A.G.], 51375 Leverkusen, Germany\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.4**\nABSTRACT\n**BLOCK**fs== 9.0**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nChorioamnionitis and funisitis are associated with neonatal mor-\nbidity and mortality. We hypothesized that chorioamnionitis may\nstress fetal endothelium, activate proinﬂammatory gene transcrip-\ntion. and affect angiogenic homeostasis in fetal capillaries. Placentas\nfrom preterm infants were stained for heat-shock protein 70, nuclear\nfactor-B, hypoxia-inducible factor-1, and vascular endothelial\ngrowth factor (VEGF). VEGF receptors (VEGF-R) 1 and 2 as well\nas the receptor tyrosine kinase with immunoglobulin and epidermal\ngrowth factor homology domains (TIE-2), which is involved in\nvascular remodeling, were quantiﬁed. Immunohistochemistry was\nanalyzed by counting positive capillaries in placental terminal villi.\nStaining intensity was quantiﬁed by a three-step semiquantitative\nscale. The samples were divided into three matched groups accord-\ning to histology: chorioamnionitis with funisitis (“funisitis”), cho-\nrioamnionitis without funisitis (“chorioamnionitis”), and control\ngroup with no inﬂammation. In tissues from the funisitis or chorio-\namnionitis group, heat-shock protein 70 expression was increased\nover the control group. More nuclear factor-B–positive nuclei of\nendothelial cells in capillaries were counted in the funisitis and\nchorioamnionitis groups. Expression of VEGF and VEGF-R1 and\n-R2 were reduced in cases of funisitis or chorioamnionitis in com-\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nparison with controls. Hypoxia-inducible factor-1 expression\ntended to be slightly lower in the funisitis and chorioamnionitis\ngroups but did not reach statistical signiﬁcance. We speculate that\ncellular stress and changes in angiogenic homeostasis induced by\nproinﬂammatory activation of fetal endothelium in chorioamnionitis\nmay not be limited to the placenta but may also involve other fetal\norgans. (Pediatr Res 58: 607–612, 2005)\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nAbbreviations\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nBPD, bronchopulmonary dysplasia\nCI, conﬁdence interval\nHIF-1 hypoxia-inducible factor-1\nHRP, horseradish peroxidase\nHSP70, heat-shock protein 70\nIL, interleukin\nNF-B, nuclear factor-B\nTIE-2, tyrosine kinase with immunoglobulin and epidermal\ngrowth factor homology domains-2\nVEGF, vascular endothelial growth factor\nVEGF-R, vascular endothelial growth factor receptor\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nChorioamnionitis is very common in preterm birth (1).\nMicrobial infection of the amnion and chorion are detected in\n60% of all preterm deliveries (2). Funisitis is diagnosed as an\ninﬁltration of neutrophils in the umbilical cord vessels (3) and\ncan be considered as a progression of chorioamnionitis with\nsystemic inﬂammatory response syndrome of the fetus (4).\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nReceived November 10, 2004; accepted January 19, 2005.\nCorrespondence: Boris W. Kramer, M.D., University Children’s Hospital, Josef-Schneider-\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nStrasse 2, 97080 Würzburg, Germany; e-mail: Kramer_b@kinderklinik.uni-wuerzburg.de.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nThis study was supported by a grant of the Interdisciplinary Center for Clinical\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nResearch (A-27) of the University of Würzburg, Germany.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.1**r== 0.7**\nDOI: 10.1203/01.PDR.0000175641.39056.7A\n**BLOCK**fs== 10.0**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nAlthough fetal inﬂammation reduces the incidence of respi-\nratory distress syndrome (5,6) and increases the survival of pre-\nterm infants (7), the fetal inﬂammatory response has been asso-\nciated with increased neonatal morbidity. The mechanisms by\nwhich fetal inﬂammation increases the incidence of bronchopul-\nmonary dysplasia (BPD) (8), retinopathy of prematurity (9), in-\ntracranial hemorrhage (10), cerebral palsy, and adverse neurode-\nvelopmental outcome (11–13) are not fully understood. Recently,\na new potential mechanism in the pathogenesis of increased\nmorbidity was put forward. The importance of impaired angio-\ngenesis in the development of organs and BPD was shown by a\nreduced expression of proangiogenic factors and their receptors in\nlung tissue from preterm infants with BPD (14,15).\n**BLOCK**fs== 10.0**b== 0.2**t== 0.1**l== 0.1**r== 0.5**\nDe novo synthesis of blood vessels (vasculogenesis) ac-\ncounts for the major part of new blood vessel formation in the\nplacenta during the ﬁrst trimester, which is followed by angio-\ngenesis (growth of existing vessels) in the next trimesters (16).\nAngiogenesis is a complex process in which the balance of\nagonists and antagonists drive growth, differentiation, survival,\nand remodeling of endothelial cells and their precursors (17).\nDuring the complete gestational period, fetal capillaries grow\nin the placenta toward the decidua (18). It is unclear whether a\nbacterial infection of the placenta—like in chorioamnionitis–\naffects the angiogenesis during gestation. In this study, we\nfocused on the inﬂammatory changes on endothelial cell ho-\nmeostasis and on selected angiogenic factors expressed by\ncapillaries of the placenta of preterm infants that were exposed\nto chorioamnionitis or funisitis. We used chorioamnionitis and\nfunisitis as criteria to characterize the prenatal inﬂammation to\nwhich the fetus was exposed. These histologic diagnoses are\nreadily available in clinical practice and are important for\noutcome (19). We hypothesized that inﬂammation in chorio-\namnionitis and/or funisitis may impair cellular homeostasis in\nendothelial cells that may activate heat-shock protein 70\n(HSP70). HSP70 is an unspeciﬁc marker of cellular stress that\ncan be induced by hypoxic, oxidative, or osmotic stress (20). In\naddition, we evaluated the expression of hypoxia-inducible\nfactor-1 (HIF-1), which can be induced by hypoxia or\nischemia (21,22). HIF-1 and HSP70 are expressed in placen-\ntal endothelial cells throughout pregnancy (23) and interfere\nwith proinﬂammatory gene transcription (24). Proinﬂamma-\ntory gene transcription can be induced by activation of nuclear\nfactor-B (NF-B), which is a fundamental feature in inﬂam-\nmatory events in monocytes and endothelial cells (25–27). We\nevaluated the expression of NF-B in placental endothelial\ncells that can induce proinﬂammatory gene transcription for\ninterleukin (IL)-1 and tumor necrosis factor-, which are\nknown to be up-regulated in chorioamnionitis (4). The effects\nof inﬂammation on angiogenesis were evaluated by staining\nthe placenta tissue for the expression of vascular endothelial\ngrowth factor (VEGF), which induces endothelial cell prolif-\neration, sprouting, and tubulogenesis (28). VEGF can be in-\nduced by HIF-1 or many oncogenes (21,22,29) and is closely\nrelated to the regulation of NF-B by inhibiting IB kinase\nactivity (26). The effects of VEGF are mediated in part by the\nVEGF receptors (VEGF-R) 1 and 2 (30), which were detected\nin the placenta throughout gestation (31). In addition, the\nreceptor tyrosine kinase with immunoglobulin and epidermal\ngrowth factor homology domains-2 (TIE-2) was quantiﬁed in\nthe placenta. TIE-2 stimulates angiogenesis and vascular re-\nmodeling by endothelial cell migration and stabilizes newly\nformed vascular networks by recruiting periendothelial support\ncells and formation of extracellular matrix (32).\n**BLOCK**fs== 10.0**b== 0.1**t== 0.9**l== 0.2**r== 0.7**\nMETHODS\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nPatients and study design. Umbilical cord blood, umbilical cord, and\nplacenta samples from 32 preterm infants [13 female, 19 male; gestational age\nmedian 26 wk (95% conﬁdence interval [CI] 23–31 wk); birth weight median\n925 g (95% CI 568 –1752 g)] of a collection of 57 samples that were obtained\nfrom the Children’s Hospital of the City of Cologne in 1998 –2001 were\nanalyzed (33). In the study of D’Alquen et al. (33), umbilical cord sections and\numbilical blood samples of the same patients were analyzed for the expression\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nof adhesion molecules. In this study, only funisitis resulted in increased\nexpression and shedding of endothelial adhesion molecules, which correlated\nwith increased concentrations of proinﬂammatory cytokines in umbilical blood\nsamples (33).\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nPrenatal steroid prophylaxis was deﬁned as an inclusion criterion. The\npatients were grouped according to histologic diagnosis of placenta and\numbilical cord. Hematoxylin-eosin–stained sections of placenta and umbilical\ncord were reviewed by a pathologist who was unaware of clinical data.\nChorioamnionitis was deﬁned as inﬁltration of neutrophils in the extraplacental\nmembranes or chorionic plate (34). Funisitis was diagnosed when neutrophils\ninﬁltrated the wall of umbilical cord vessels or Wharton’s jelly (35). The ﬁrst\ngroup of patients with chorioamnionitis and funisitis was referred to as funisitis\ngroup (n  11). The second group had a diagnosis of chorioamnionitis without\ninﬂammation of the umbilical cord and therefore was referred to as chorio-\namnionitis group (n  9). The control group showed no inﬂammation of\nplacenta and umbilical cord (n  12).\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nCharacteristics of patients are shown in Table 1. There was no difference in\ndistribution of gender, gestational age, birth weight, Apgar score after 5 min,\nand umbilical cord artery pH between groups. Almost all mothers in all three\ngroups had uncontrollable preterm labor with no difference between groups.\nThe funisitis group had the highest incidence of premature rupture of mem-\nbranes, and the control group had the lowest (p  0.05). The incidence of\npremature rupture of membranes was not different between the funisitis and\nchorioamnionitis groups. Pregnancies with pathologic changes in the placenta,\nsuch as gestational diabetes or preeclampsia, were excluded. The study was\napproved by the hospital ethics committee, and patients’ parents gave informed\nconsent.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nImmunohistochemistry. Immunohistochemical staining was performed\nfollowing standard procedures. In brief, sections were cut from formalin-ﬁxed,\nparafﬁn-embedded placenta samples at 2 m and mounted on 3-amino-\npropyltriethoxy-silane– coated slides (Roth, Karlsruhe, Germany). After dewaxing\nand rehydration in graded ethanol solutions, antigen retrieval was carried out with\n10 min of boiling in citric acid buffer (pH 6.0), using a microwave (750 W).\nEndogenous peroxidase activity was inhibited by 3% hydrogen peroxide in methyl\nalcohol. Nonspeciﬁc binding was blocked by incubation in PBS with 1% goat-\nserum and 1% BSA. Slides were incubated overnight at 4°C in a humidiﬁed\nchamber with primary antibodies at appropriate dilution in PBS. Primary antibod-\nies were anti-human NF-B (diluted 1:2000; sc-226; Santa Cruz Biotechnology,\nSanta Cruz, CA) and anti-human HSP70 (diluted 1:75; clone 289M; Biogenex,\nSan Ramon, CA), anti-human VEGF (diluted 1:50; clone sc-152; Santa Cruz\nBiotechnology), anti-human VEGF-R1 (diluted 1:500; clone RP 077; Zytomed,\nBerlin, Germany), anti-human VEGF-R2 (diluted 1:500; clone AF 357; R&D\nSystems, Minneapolis, MN), anti-human TIE-2 (diluted 1:500; clone AF 313;\nR&D Systems), and anti-human HIF-1 (diluted 1:100; clone H167; Novus\nBiologicals, Littleton, CO). Detection was done with a goat anti-mouse secondary\nantibody conjugated to horseradish peroxidase (HRP; Dako, Glostrup, Denmark).\nThe enzymatic reaction was developed with 3,3' diaminobenzidine (brown). After\ncounterstaining with hematoxylin, samples were dehydrated and mounted. A\nnegative control without primary antibody was included in each staining.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nDouble staining for NF-B and PECAM-1 (CD31), a speciﬁc endothelial\nsurface protein, was performed as described previously (36). In brief, after\nincubation with rabbit anti-human NF-B, an HRP-conjugated goat-anti-rabbit\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nTable 1. Clinical characteristics of patients according to the\nhistologic ﬁnding of chorioamnionitis and funisitis in placenta and\numbilical cord\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\nControl\ngroup\n(n  12)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.7**r== 0.3**\n(23–31)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.7**r== 0.3**\n(581–1596)\n8.00\n(6.93– 8.57)\n7.3\n(7.26 –7.35)\n12/12\n3/12\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nMedian (95% CI)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.1**\nChorioamnionitis\ngroup\n(n  9)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.8**r== 0.2**\n(24 –31)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.8**r== 0.2**\n(632–1704)\n8.00\n(6.91–9.09)\n7.33\n(7.29 –7.36)\n9/9\n6/9*\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\nFunisitis\ngroup\n(n  11)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.9**r== 0.1**\n(22–31)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.9**r== 0.1**\n(588 –1516)\n8.00\n(6.72– 8.55)\n7.29\n(7.24 –7.33)\n11/11\n8/11*\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nGestational age (wk)\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nBirth weight (g)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nApgar score 5 min\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\npH (umbilical cord artery)\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nPrenatal steroids\nPROM\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.2**\nPROM, premature rupture of membranes.\n* p  0.05 vs control group.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nantibody (Dako) was used as second antibody. Detection reaction was devel-\noped with 3,3' diaminobenzidine followed by 10 min of incubation in 0.3%\nhydrogen peroxide in methanol. In the second step, the antibody against CD31\n(diluted 1:20; clone M 0823; Dako) was mounted and detected with a goat\nanti-mouse HRP antibody (Dako). The second color reaction was developed\nwith the HistoGreen substrate kit (green; Linaris, Wertheim, Germany). No\ncounterstaining was performed.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nAnalysis of immunohistochemical staining. For HSP70 and NF-B, a total\nof 150 capillaries per sample in placental terminal villi were analyzed. The\npercentage of positive capillaries was recorded at a magniﬁcation of 200.\nCapillaries that were stained along the circumference were recorded as posi-\ntive. VEGF, VEGF-R1, VEGF-R2, TIE-2, and HIF-1 were expressed on all\nendothelial cells (22,37–39). Therefore, the intensity of staining was measured\nwith a three-step semiquantitative scale (magniﬁcation 200) as weak (1),\nmoderate (2), or strong (3) (40). The average for each sample was\ncalculated out of ﬁve analyzed areas that were chosen at random.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nStatistical analysis. Differences between groups were tested with the\nKruskal-Wallis H test (for gestational age, birth weight, umbilical cord artery\npH, and Apgar score after 5 min), the 2 test (for gender), and the Mann-\nWhitney U test for all other comparisons. A p  0.05 was considered\nsigniﬁcant.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.2**r== 0.7**\nRESULTS\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nFetal capillaries were stressed by chorioamnionitis and\nfunisitis. The percentage of HSP70-positive capillaries was\nhigher in placenta tissue of the chorioamnionitis group (median\n24%; 95% CI 20 –27%) and the funisitis group (median 25%;\n95% CI 22–27%) in comparison with the control group (me-\ndian 3%; 95% CI 2–7%; p  0.05). The result of each analyzed\nsample is shown in Fig. 1. The percentage of HSP70-positive\ncells was not signiﬁcantly different between the funisitis group\nand the chorioamnionitis group.\n**BLOCK**fs== 10.0**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nChorioamnionitis and funisitis activated NF-B in endo-\nthelial cells. The percentage of NF-B–positive capillaries was\nincreased in the chorioamnionitis group (median 40%; 95% CI\n33– 48%) versus the control group (median 16%; 95% CI\n9 –20%; p  0.05). The percentage in the funisitis group\n(median 47%; 95% CI 41–53%) was increased in comparison\nwith the control group but not different from the chorioamnio-\nnitis group (p  0.05). The result of each analyzed sample is\nsummarized in Fig. 2. Representative samples are shown in\nFig. 3A and B. The staining was localized to the inﬂammatory\ncells in the capillary lumen and to the nuclei of the endothelial\ncells. A double-staining PECAM-1 (CD31), which is an endo-\nthelial marker, and NF-B were performed to demonstrate the\nco-localization (Fig. 3C).\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nFigure 1. HSP70 expression in fetal capillaries. The percentage of positive\ncapillaries is indicated. The median of each group is marked. The percentage\nof HSP70-positive capillaries is increased in the chorioamnionitis group and\nthe funisitis group.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nFigure 2. The percentage of NF-B–positive capillaries was higher in the\nchorioamnionitis group and in the funisitis group. The median of each group\nis marked.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nImmunohistochemistry. (A and B) NF-B–positive capillaries were\nFigure 3.\ndetected in placenta tissue with chorioamnionitis but not in controls. (C) Nuclei\nof endothelial cells were positive for NF-B in the chorioamnionitis group,\nwhich was shown by staining for CD31 (brown) and NF-B (green). (D and E)\nVEGF expression was decreased in placenta tissue from chorioamnionitis.\nMagniﬁcation: 200 in A, B, D, and E; 1000 in C.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nVEGF was decreased in chorioamnionitis and funisitis.\nThe intensity of VEGF expression in fetal capillaries was\ndecreased in the chorioamnionitis and in the funisitis group.\nRepresentative samples are shown in Fig. 3D and E. The\nintensity of all analyzed samples is shown in Fig. 4. The\nintensity in the control group (median 2.3; 95% CI 1.9 –2.6)\nwas higher than in the chorioamnionitis group (median 0.7;\n95% CI 0.4 –1.1) or in the funisitis group (median 1.1; 95% CI\n0.8 –1.4; p  0.05).\n**BLOCK**fs== 10.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nVEGF-R1 and -R2 were decreased in chorioamnionitis\nand funisitis. The intensity of expression of VEGF-R1 and -R2\nare demonstrated in Fig. 5A and B, respectively. The intensity\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nFigure 4. The intensity of VEGF staining was rated on a three-step semi-\nquantitative scale. The intensity of VEGF expression was lower in the cho-\nrioamnionitis group and in the funisitis group. The median of each group is\nmarked.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nFigure 6. The intensity of TIE-2 staining was rated on a three-step scale. The\nintensity of TIE-2 expression was higher in the chorioamnionitis group and in\nthe funisitis group. The median of each group is marked.\n**BLOCK**fs== 10.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nalmost 3-fold in the chorioamnionitis group (median 2.4; 95%\nCI 2.2–2.7) and in the funisitis group (median 2.5; 95% CI\n2.2–2.7) over the control group (median 0.8; 95% CI 0.5–1.0;\np  0.05).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nHIF-1  was not changed in chorioamnionitis and funisi-\ntis. There was no difference in the staining pattern and only\nminor differences in the intensity of the expression of HIF-1\nin all three groups: chorioamnionitis group (median 1.6; 95%\nCI 1.1–1.8), funisitis group (median 1.8; 95% CI 1.6 –2.2), and\ncontrol group (median 1.9; 95% CI 1.5–2.2; Fig. 7).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\nDISCUSSION\n**BLOCK**fs== 10.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nWe found in this study that inﬂammation in the placenta\naffected the homeostasis of endothelial cells in the placental\ncapillaries. Expression of HSP70 was increased in placentas\nfrom patients with funisitis or chorioamnionitis. Proinﬂamma-\ntory gene transcription was activated in endothelial cells. Ex-\npression of proangiogenic VEGF and VEGF-R1 and -R2 were\ndecreased in cases of funisitis and chorioamnionitis, whereas\nTIE-2 expression was increased. The changes reﬂected the\ninvolvement of placental capillaries in the inﬂammatory pro-\ncess with possible impact on placental function. Remarkably,\nthe changes were similar in the chorioamnionitis and the\nfunisitis groups and do not reﬂect the idea of funisitis as a\nprogression of chorioamnionitis. However, we focused on\nangiogenic changes in capillaries of the placental compartment\nthat may not be involved in the progression of chorioamnionitis\nto funisitis. In contrast, the analysis of adhesion molecules in\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nFigure 7. The intensity of HIF-1 staining was rated on a three-step scale.\nThe intensity of HIF-1  was not different in patients with chorioamnionitis or\nfunisitis. The median of each group is marked.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nFigure 5. The intensity of VEGF-R1 and -R2 staining was rated on a\nthree-step scale. (A) The intensity of VEGF-R1 expression was reduced in the\nchorioamnionitis group and the funisitis group. (B) Similar results were\nobtained for VEGF-R2 expression. The median of each group is marked.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nof VEGF-R1 was decreased in the control group (median 2.6;\n95% CI 2.4 –2.8; p  0.05) compared with the chorioamnio-\nnitis group (median 0.5; 95% CI 0.4 – 0.8) and the funisitis\ngroup (median 0.7, 95% CI 0.5– 0.9; Fig. 5A). Similar effects\nwere seen with the expression of VEGF-R2 (Fig. 5B). The\nintensity was reduced in the chorioamnionitis group (median\n0.5; 95% CI 0.4 – 0.8) and the funisitis group (median 0.7; 95%\nCI 0.5– 0.9) in comparison with the control group (median 2.6;\n95% CI 2.4 –2.8; p  0.05).\n**BLOCK**fs== 10.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nTIE-2 was increased in chorioamnionitis and funisitis. The\nstaining pattern for TIE-2 was different from the patterns seen\nwith VEGF and its receptors. The intensity of expression of\nTIE-2 was markedly increased in the chorioamnionitis group\nand in the funisitis group (Fig. 6). The intensity was increased\n**BLOCK**fs== 10.0**b== 0.3**t== 0.1**l== 0.1**r== 0.5**\nthe umbilical cord arteries and veins in the study of D’Alquen\net al. (33) supported the idea of funisitis as a progression of\ninﬂammation to large blood vessels in a different compartment.\nThe impact of inﬂammatory and angiogenic changes on\nplacental function remain unclear. However, Satosar et al. (41)\nrecently reported the correlation of inﬂammation of the villi by\nbacterial or viral infection with neonatal mortality and mor-\nbidity. NF-B can be induced by many different proinﬂamma-\ntory agents, such as Gram-negative and Gram-positive bacteria\nor proinﬂammatory cytokines. The effects on NF-B activation\non angiogenesis in the placenta are not well understood. HIF-\n1, VEGF, VEGF receptors, and TIE-2 have been detected on\ntrophoblasts and endothelial cells (22,37–39). The mRNA\nlevels of VEGF and VEGF receptors were dependent on\ngestational age and mode of delivery, but no analysis of\nchorioamnionitis was included (38). The expression of VEGF\nwas affected by pathologic conditions such as preterm prema-\nture rupture of membranes (43), recurrent miscarriages (44),\nhypoxia/ischemia (45), fetal growth retardation, fetal alcohol\nsyndrome, or diabetes (31). Marvin et al. (42) showed a\nreduction of VEGF-R2 mRNA in preterm deliveries with\ninfection in comparison with preterm deliveries without infec-\ntion. Protein was not quantiﬁed in that study. The regulation of\nVEGF and its receptors occurs at transcriptional, translational,\nand posttranslational levels (46). In our study, protein expres-\nsion but not mRNA was analyzed. Therefore, the comparison\nof the results is limited to the lack of information on respective\ndata on protein or mRNA. The reduction of expression of\nVEGF and VEGF receptors and the increase in TIE-2 in\nchorioamnionitis may reﬂect injury of the placental vessels.\nThe angiogenesis may be impaired, which is reﬂected by a\ndecrease in VEGF and its receptors. The increase in TIE-2\nexpression may be part of the repair after inﬂammation-\nmediated injury in placenta capillaries because TIE-2 mediates\nvascular stabilization and remodeling (32). This assumption\nremains speculative because there was no morphologic differ-\nence in the analyzed sections.\n**BLOCK**fs== 10.0**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nThe expression of HIF-1 was not changed in patients with\nchorioamnionitis or funisitis compared with the control group.\nOne possible explanation for this ﬁnding is that the changes of\nHIF-1 have been transient in the course of inﬂammation and\ninfection. The half-life of HIF-1is extremely short at ~5 min.\nHypoxia and ischemia acutely extend the half-life of HIF-1\nby inhibiting protein degradation (46). We have no information\non the onset or the duration of inﬂammation in these patients.\nThe early changes of inﬂammation may not have been avail-\nable for analysis in our study. Another possibility is that\nendothelial VEGF and VEGF-receptor gene expression in cho-\nrioamnionitis may be regulated in a HIF-1–independent man-\nner for example by transmembrane tyrosine kinases such as the\nepidermal growth factor receptor (47,48).\n**BLOCK**fs== 10.0**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nThe implications of the present study may not be limited to\nthe placenta. Recent ﬁndings in preterm infants with pulmo-\nnary morbidity highlight effects of infection and chorioamnio-\nnitis on neonatal organs other than the placenta (49,50). The\nrole of angiogenesis in the development of BPD was shown in\nlungs of preterm infants who developed BPD (14,15). The\nexpression of VEGF and its receptors was decreased in infants\n**BLOCK**fs== 10.0**b== 0.6**t== 0.1**l== 0.5**r== 0.1**\nwho developed BPD. Therefore, the interaction of alveogenesis\nand vascular development was impaired in patients with BPD\n(51). Apparently, the development of BPD can be primed by\nexposure to prenatal pulmonary inﬂammation, which can be\naggravated by postnatal events (8,52,53). Structural changes by\napoptosis of parenchymal cells were induced in the course of\ninjury and inﬂammation in the airways and lungs (54). Appar-\nently, alveolar inﬂammation affects endothelial function and\nangiogenesis (8). This concept was demonstrated recently by\nreduced expression of endothelial cell proteins in the lungs of\nlamb fetuses within days after alveolar inﬂammation in an\nendotoxin-induced chorioamnionitis model with little systemic\ninﬂammation (55,56). However, lung function can be affected\nby vascular proinﬂammatory stimuli as well (51). Our study\nsupports the concept that angiogenesis-related genes may be\ninteresting new therapeutic targets for the prevention and\ntreatment of chorioamnionitis-associated neonatal morbidity\nand mortality (51).\n**BLOCK**fs== 10.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nOur study showed that fetal capillaries are stressed by\nchorioamnionitis. The expression of VEGF and VEGF-R1 and\n-R2 was reduced, whereas TIE-2 was increased. The impor-\ntance of this disturbance of angiogenic homeostasis for placen-\ntal function and neonatal morbidity remains to be determined.\nOur ﬁndings suggest for the ﬁrst time that impaired fetal\nangiogenesis in the placenta might be a pathogenetic mecha-\nnism contributing to increased morbidity and mortality of\nfetuses and newborns who are exposed to chorioamnionitis and\nfunisitis.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nREFERENCES\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n1. Lahra MM, Jeffery HE 2004 A fetal response to chorioamnionitis is associated with\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nearly survival after preterm birth. Am J Obstet Gynecol 190:147–151\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n2. Goldenberg RL, Hauth JC, Andrews WW 2000 Intrauterine infection and preterm\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\ndelivery. N Engl J Med 342:1500 –1507\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n3. Sampson JE, Theve RP, Blatman RN, Shipp TD, Bianchi DW, Ward BE, Jack RM\n1997 Fetal origin of amniotic ﬂuid polymorphonuclear leukocytes. Am J Obstet\nGynecol 176:77– 81\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n4. Yoon BH, Romero R, Park JS, Kim M, Oh SY, Kim CJ, Jun JK 2000 The relationship\namong inﬂammatory lesions of the umbilical cord (funisitis), umbilical cord plasma\ninterleukin 6 concentration, amniotic ﬂuid infection, and neonatal sepsis. Am J Obstet\nGynecol 183:1124 –1129\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n5. Watterberg KL, Scott SM, Naeye RL 1997 Chorioamnionitis, cortisol, and acute lung\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\ndisease in very low birth weight infants. Pediatrics 99:E6\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n6. Shimoya K, Taniguchi T, Matsuzaki N, Moriyama A, Murata Y, Kitajima H,\nFujimura M, Nakayama M 2000 Chorioamnionitis decreased incidence of respiratory\ndistress syndrome by elevating fetal interleukin-6 serum concentration. Hum Reprod\n15:2234 –2240\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n7. Costeloe K, Hennessy E, Gibson AT, Marlow N, Wilkinson AR 2000 The EPICure\nstudy: outcomes to discharge from hospital for infants born at the threshold of\nviability. Pediatrics 106:659 – 671\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n8. Speer CP 2003 Inﬂammation and bronchopulmonary dysplasia. Semin Neonatol\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n8:29 –38\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n9. Smith LE 2003 Pathogenesis of retinopathy of prematurity. Semin Neonatol 8:469 –\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n\n**BLOCK**fs== 7.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n10. DiSalvo D, The Developmental Epidemiology Network Investigators 1998 The\ncorrelation between placental pathology and intraventricular hemorrhage in the\npreterm infant. Pediatr Res 43: 9 –15\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n11. Leviton A, Paneth N, Reuss ML, Susser M, Allred EN, Dammann O, Kuban K, Van\nMarter LJ, Pagano M, Hegyi T, Hiatt M, Sanocka U, Shahrivar F, Abiri M, Disalvo\nD, Doubilet P, Kairam R, Kazam E, Kirpekar M, Rosenfeld D, Schonfeld S, Share J,\nCollins M, Genest D, Shen-Schwarz S, et al. 1999 Maternal infection, fetal inﬂam-\nmatory response, and brain damage in very low birth weight infants. Developmental\nEpidemiology Network Investigators. Pediatr Res 46:566 –575\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n12. Nelson KB, Dambrosia JM, Grether JK, Phillips TM 1998 Neonatal cytokines and\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\ncoagulation factors in children with cerebral palsy. Ann Neurol 44:665– 675\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n13. Yoon BH, Romero R, Park JS, Kim CJ, Kim SH, Choi JH, Han TR 2000 Fetal\nexposure to an intra-amniotic inﬂammation and the development of cerebral palsy at\nthe age of three years. Am J Obstet Gynecol 182:675– 681\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\n14. Lassus P, Turanlahti M, Heikkila P, Andersson LC, Nupponen I, Sarnesto A,\nAndersson S 2001 Pulmonary vascular endothelial growth factor and Flt-1 in fetuses,\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nin acute and chronic lung disease, and in persistent pulmonary hypertension of the\nnewborn. Am J Respir Crit Care Med 164:1981–1987\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n15. Bhatt AJ, Pryhuber GS, Huyck H, Watkins RH, Metlay LA, Maniscalco WM 2001\nDisrupted pulmonary vasculature and decreased vascular endothelial growth factor,\nFlt-1, and TIE-2 in human infants dying with bronchopulmonary dysplasia. Am J\nRespir Crit Care Med 164:1971–1980\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n16. Demir R, Kaufmann P, Castellucci M, Erbengi T, Kotowski A 1989 Fetal vasculo-\ngenesis and angiogenesis in human placental villi. Acta Anat (Basel) 136:190 –203\n17. Bergers G, Benjamin LE 2003 Tumorigenesis and the angiogenic switch. Nat Rev\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nCancer 3:401– 410\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n18. Georgiades P, Ferguson-Smith AC, Burton GJ 2002 Comparative developmental\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nanatomy of the murine and human deﬁnitive placentae. Placenta 23:3–19\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n19. Matsuda T, Nakajima T, Hattori S, Hanatani K, Fukazawa Y, Kobayashi K, Fujimoto\nS 1997 Necrotizing funisitis: clinical signiﬁcance and association with chronic lung\ndisease in premature infants. Am J Obstet Gynecol 177:1402–1407\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n20. Morimoto RI, Kroeger PE, Cotto JJ 1996 The transcriptional regulation of heat shock\ngenes: a plethora of heat shock factors and regulatory conditions. In: Feige U,\nMorimoto RI, Yahara I, Polla B (eds) Stress-Inducible Cellular Responses.\nBirkhäuser Verlag, Basel, pp 38 –55\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n21. Kelly BD, Hackett SF, Hirota K, Oshima Y, Cai Z, Berg-Dixon S, Rowan A, Yan Z,\nCampochiaro PA, Semenza GL 2003 Cell type-speciﬁc regulation of angiogenic\ngrowth factor gene expression and induction of angiogenesis in nonischemic tissue by\na constitutively active form of hypoxia-inducible factor 1. Circ Res 93:1074 –1081\n22. Rajakumar A, Conrad KP 2000 Expression, ontogeny, and regulation of hypoxia-\ninducible transcription factors in the human placenta. Biol Reprod 63:559 –569\n23. Shah M, Stanek J, Handwerger S 1998 Differential localization of heat shock proteins\n90, 70, 60 and 27 in human decidua and placenta during pregnancy. Histochem J\n30:509 –518\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n24. Feinstein DL, Galea E, Aquino DA, Li GC, Xu H, Reis DJ 1996 Heat shock protein\n70 suppresses astroglial-inducible nitric-oxide synthase expression by decreasing\nNFB activation. J Biol Chem 271:17724 –17732\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n25. Baur FM, Brenner B, Goetze-Speer B, Neu S, Speer CP 1998 Natural porcine\nsurfactant (Curosurf) down-regulates mRNA of tumor necrosis factor- (TNF-) and\nTNF- type II receptor in lipopolysaccharide-stimulated monocytes. Pediatr Res\n44:32–36\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n26. Dikov MM, Oyama T, Cheng P, Takahashi T, Takahashi K, Sepetavec T, Edwards B,\nAdachi Y, Nadaf S, Daniel T, Gabrilovich DI, Carbone DP 2001 Vascular endothelial\ngrowth factor effects on nuclear factor-B activation in hematopoietic progenitor\ncells. Cancer Res 61:2015–2021\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n27. Cao L, Liu C, Cai B, Jia X, Kang L, Speer CP, Sun B 2004 Nuclear factor-kappa B\nexpression in alveolar macrophages of mechanically ventilated neonates with respi-\nratory distress syndrome. Biol Neonate 86:116 –123\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n28. Ferrara N, Gerber HP, LeCouter J 2003 The biology of VEGF and its receptors. Nat\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.1**r== 0.8**\nMed 9:669 – 676\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n29. Okada F, Rak JW, Croix BS, Lieubeau B, Kaya M, Roncari L, Shirasawa S, Sasazuki\nT, Kerbel RS 1998 Impact of oncogenes in tumor angiogenesis: mutant K-ras\nup-regulation of vascular endothelial growth factor/vascular permeability factor is\nnecessary, but not sufﬁcient for tumorigenicity of human colorectal carcinoma cells.\nProc Natl Acad Sci USA 95:3609 –3614\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n30. Zachary I 2003 VEGF signalling: integration and multi-tasking in endothelial cell\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.7**\nbiology. Biochem Soc Trans 31:1171–1177\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n31. Helske S, Vuorela P, Carpen O, Hornig C, Weich H, Halmesmaki E 2001 Expression\nof vascular endothelial growth factor receptors 1, 2 and 3 in placentas from normal\nand complicated pregnancies. Mol Hum Reprod 7:205–210\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n32. Sato TN, Tozawa Y, Deutsch U, Wolburg-Buchholz K, Fujiwara Y, Gendron-Maguire\nM, Gridley T, Wolburg H, Risau W, Qin Y 1995 Distinct roles of the receptor tyrosine\nkinases Tie-1 and Tie-2 in blood vessel formation. Nature 376:70 –74\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n33. d’Alquen D, Kramer BW, Seidenspinner S, Marx A, Berg D, Groneck P, Speer CP\n2005 Activation of umbilical cord endothelial cells and fetal inﬂammatory response\nin preterm infants with chorioamnionitis and funisitis. Pediatr Res 57:263–269\n34. Naeye RL 1987 Functionally important disorders of the placenta, umbilical cord, and\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nfetal membranes. Hum Pathol 18:680 – 691\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n35. Salaﬁa CM, Weigl C, Silberman L 1989 The prevalence and distribution of acute\nplacental inﬂammation in uncomplicated term pregnancies. Obstet Gynecol 73:383–\n**BLOCK**fs== 7.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n36. Kammerer U, Eggert AO, Kapp M, McLellan AD, Geijtenbeek TB, Dietl J, van\nKooyk Y, Kampgen E 2003 Unique appearance of proliferating antigen-presenting\ncells expressing DC-SIGN (CD209) in the decidua of early human pregnancy. Am J\nPathol 162:887– 896\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n37. Banks RE, Forbes MA, Searles J, Pappin D, Canas B, Rahman D, Kaufmann S,\nWalters CE, Jackson A, Eves P, Linton G, Keen J, Walker JJ, Selby PJ 1998 Evidence\nfor the existence of a novel pregnancy-associated soluble variant of the vascular\nendothelial growth factor receptor, Flt-1. Mol Hum Reprod 4:377–386\n**BLOCK**fs== 7.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n38. Ahmed A, Li XF, Dunk C, Whittle MJ, Rushton DI, Rollason T 1995 Colocalisation\nof vascular endothelial growth factor and its Flt-1 receptor in human placenta. Growth\nFactors 12:235–243\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n39. Goldman-Wohl DS, Ariel I, Greenﬁeld C, Lavy Y, Yagel S 2000 Tie-2 and angio-\npoietin-2 expression at the fetal-maternal interface: a receptor ligand model for\nvascular remodelling. Mol Hum Reprod 6:81– 87\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n40. Kramer BW, Kramer S, Ikegami M, Jobe AH 2002 Injury, inﬂammation, and\nremodeling in fetal sheep lung after intra-amniotic endotoxin. Am J Physiol\n283:L452–L459\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n41. Satosar A, Ramirez NC, Bartholomew D, Davis J, Nuovo GJ 2004 Histologic\ncorrelates of viral and bacterial infection of the placenta associated with severe\nmorbidity and mortality in the newborn. Hum Pathol 35:536 –545\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n42. Marvin KW, Keelan JA, Eykholt RL, Sato TA, Mitchell MD 2002 Expression of\nangiogenic and neurotrophic factors in the human amnion and choriodecidua. Am J\nObstet Gynecol 187:728 –734\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n43. Daneshmand SS, Chmait RH, Moore TR, Bogic L 2002 Preterm premature rupture of\nmembranes: vascular endothelial growth factor and its association with histologic\nchorioamnionitis. Am J Obstet Gynecol 187:1131–1136\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n44. Vuorela P, Carpen O, Tulppala M, Halmesmaki E 2000 VEGF, its receptors and the\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\ntie receptors in recurrent miscarriage. Mol Hum Reprod 6:276 –282\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n45. Kumazaki K, Nakayama M, Suehara N, Wada Y 2002 Expression of vascular\nendothelial growth factor, placental growth factor, and their receptors Flt-1 and KDR\nin human placenta under pathologic conditions. Hum Pathol 33:1069 –1077\n**BLOCK**fs== 7.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n46. Xie K, Wei D, Shi Q, Huang S 2004 Constitutive and inducible expression and\nregulation of vascular endothelial growth factor. Cytokine Growth Factor Rev\n15:297–324\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n47. Slomiany MG, Rosenzweig SA 2004 IGF-1-induced VEGF and IGFBP-3 secretion\ncorrelates with increased HIF-1 alpha expression and activity in retinal pigment\nepithelial cell line D407. Invest Ophthalmol Vis Sci 45:2838 –2847\n**BLOCK**fs== 7.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n48. Petit AM, Rak J, Hung MC, Rockwell P, Goldstein N, Fendly B, Kerbel RS 1997\nNeutralizing antibodies against epidermal growth factor and ErbB-2/neu receptor\ntyrosine kinases down-regulate vascular endothelial growth factor production by\ntumor cells in vitro and in vivo: angiogenic implications for signal transduction\ntherapy of solid tumors. Am J Pathol 151:1523–1530\n**BLOCK**fs== 7.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n49. Schmidt B, Cao L, Mackensen-Haen S, Kendziorra H, Klingel K, Speer CP 2001\nChorioamnionitis and inﬂammation of the fetal lung. Am J Obstet Gynecol 185:173–\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n50. May M, Marx A, Seidenspinner S, Speer CP 2004 Apoptosis and proliferation in\nlungs of human fetuses exposed to chorioamnionitis. Histopathology 45:283–290\n51. Abman SH 2001 Bronchopulmonary dysplasia: “a vascular hypothesis.” Am J Respir\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nCrit Care Med 164:1755–1756\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n52. Speer CP 2001 New insights into the pathogenesis of pulmonary inﬂammation in\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\npreterm infants. Biol Neonate 79:205–209\n**BLOCK**fs== 7.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n53. Speer CP, Ruess D, Harms K, Herting E, Gefeller O 1993 Neutrophil elastase and\nacute pulmonary damage in neonates with severe respiratory distress syndrome.\nPediatrics 91:794 –799\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n54. May M, Strobel P, Preisshofen T, Seidenspinner S, Marx A, Speer CP 2004 Apoptosis\nand proliferation in lungs of ventilated and oxygen-treated preterm infants. Eur Respir\nJ 23:113–121\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n55. Kallapur SG, Bachurski CJ, Le Cras TD, Joshi SN, Ikegami M, Jobe AH 2004\nVascular changes after intra-amniotic endotoxin in preterm lamb lungs. Am J Physiol\n287:L1178 –L1185\n**BLOCK**fs== 7.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n56. Kramer BW, Moss TJ, Willet KE, Newnham JP, Sly PD, Kallapur SG, Ikegami M,\nJobe AH 2001 Dose and time response after Intramniotic endotoxin in preterm lambs.\nAm J Respir Crit Med 164:982–988",
         "extracted",
         "",
         "Decreased Expression of Angiogenic Factors in Placentas with Chorioamnionitis after Preterm Birth",
         "None"
        ],
        [
         "44",
         "00357f6991440ffb57022cdc2b7ab210c1ed63ec",
         "Ovarian carcinoma is the fourth most common cause of cancer death in women. The cause and pathogenesis of this disease has remained obscure. Galactose, the hydrolyzing product of the milk sugar lactose, has been hypothesized to be toxic to ovarian epithelial cells and consumption of dairy products and lactase persistence has been suggested to be a risk factor for ovarian carcinoma. In adults, downregulation of lactase depends on a variant C/T−13910 at the 5′ end of the lactase gene. To explore whether lactase persistence is related to the risk of ovarian carcinoma we determined the C/T−13910 genotype in a cohort of 782 women with ovarian carcinoma. The C/T−13910 genotype was defined by solid phase minisequencing from 327 Finnish, 303 Polish, 152 Swedish patients and 938 Finnish, 296 Polish and 97 Swedish healthy individuals served as controls. Lactase persistence did not associate significantly with increased risk for ovarian carcinoma in the Finnish (odds ratio [OR] = 0.77, 95% confidence interval [CI] = 0.57–1.05, p = 0.097), in the Polish (OR = 0.95, 95% CI = 0.68–1.33, p = 0.75), or in the Swedish populations (OR = 1.63, 95% CI = 0.65–4.08, p = 0.29). Our results do not support the hypothesis that lactase persistence increases the ovarian carcinoma risk. On the contrary, lactase persistence may decrease the ovarian carcinoma risk at least in the Finnish population. © 2005 Wiley‐Liss, Inc.",
         "M. Kuokkanen,R. Butzow,H. Rasinperä,K. Mȩdrek,M. Nilbert,S. Malander,J. Lubiński,I. Järvelä",
         "https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/ijc.21130",
         "\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nLactase persistence and ovarian carcinoma risk in Finland, Poland and Sweden\nMikko Kuokkanen1,2*, Ralf Butzow3,6, Heli Rasinper€a2, Krzysztof Medrek4, Mef Nilbert5, Susanne Malander5, Jan Lubinski4\nand Irma J€arvel€a2,6\n1National Public Health Institute, Department of Molecular Medicine, Helsinki, Finland\n2Department of Medical Genetics, University of Helsinki, Finland\n3Department of Obstetrics and Gynecology, Helsinki University Central Hospital, Helsinki, Finland\n4Hereditary Cancer Center-Department of Genetics and Pathology, Pomeranian Academy of Medicine, Szczecin, Poland\n5Department of Oncology, Lund University Hospital, Lund, Sweden\n6Helsinki University Central Hospital, Laboratory Services HUSLAB, Laboratory of Molecular Genetics, Finland\n**BLOCK**fs== 8.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nOvarian carcinoma is the fourth most common cause of cancer\ndeath in women. The cause and pathogenesis of this disease has\nremained obscure. Galactose, the hydrolyzing product of the milk\nsugar lactose, has been hypothesized to be toxic to ovarian epithe-\nlial cells and consumption of dairy products and lactase persis-\ntence has been suggested to be a risk factor for ovarian carcinoma.\nIn adults, downregulation of\nlactase depends on a variant\nC/T213910 at the 50 end of the lactase gene. To explore whether lac-\ntase persistence is related to the risk of ovarian carcinoma we\ndetermined the C/T213910 genotype in a cohort of 782 women with\novarian carcinoma. The C/T213910 genotype was deﬁned by solid\nphase minisequencing from 327 Finnish, 303 Polish, 152 Swedish\npatients and 938 Finnish, 296 Polish and 97 Swedish healthy indi-\nviduals served as controls. Lactase persistence did not associate\nsigniﬁcantly with increased risk for ovarian carcinoma in the Fin-\nnish (odds ratio [OR] 5 0.77, 95% conﬁdence interval [CI] 5\n0.57–1.05, p 5 0.097), in the Polish (OR 5 0.95, 95% CI 5 0.68–\n1.33, p 5 0.75), or in the Swedish populations (OR 5 1.63, 95% CI\n5 0.65–4.08, p 5 0.29). Our results do not support the hypothesis\nthat lactase persistence increases the ovarian carcinoma risk. On\nthe contrary, lactase persistence may decrease the ovarian carci-\nnoma risk at least in the Finnish population.\n' 2005 Wiley-Liss, Inc.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nKey words: lactase persistence; epithelial ovarian cancer; C/T213910\npolymorphism; Finland; Poland; Sweden\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nOvarian cancer is the sixth most common cancer among women\nworldwide with the second highest mortality rate among gyneco-\nlogical malignancies.1 The highest ovarian cancer incidence rates\nare found in Northern and Western Europe.2 More than 90% of\nthe cases of ovarian cancer develop sporadically.3 The molecular\npathogenesis is poorly known but inactivation of p53 tumor sup-\npressor gene has been reported frequently.4 Depending on the pop-\nulation, 3.3–13.5% of epithelial ovarian cancer cases are explained\nby germline mutations of the BRCA1 gene, the BRCA2 gene prob-\nably contributes to a lesser extent.5–13 Multiparity, lactation, use\nof oral contraceptives, tubal ligation and hysterectomy are associ-\nated with decreased risk of ovarian carcinoma.14,15 The etiology\nof ovarian carcinoma is unknown but frequent ovulation and hor-\nmonal factors have been suggested to play a role in its pathogene-\nsis.16 Also, dietary factors have been implicated as causal or con-\ntributing factors.17\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nDisaccharide lactose is found in milk and other dairy products.\nIt is hydrolyzed to galactose and glucose by lactase-phlorizin\nhydrolase (LPH) in the intestinal mucosa. Women with galactose-\nmia suffer from early ovarian failure.18 Galactosemia is an autoso-\nmal recessive disorder with almost total absence of galactose-1-\nphosphate uridyltransferase (GALT) activity leading to the metab-\nolites of galactose accumulating in the ovary and other tissues.\nGALT belongs to the Leloir pathway together with several\nenzymes that are responsible for the conversion of galactose to\nglucose.19,20 Premature ovarian failure has been suggested to be\ndue to direct toxicity of galactose, its metabolites or elevated\ngonadotropin levels.21–23 Additionally, animal studies have shown\nthat galactose is poisonous to oocytes.24,25 Lactose, a main source\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nof galactose, was then hypothesized to play a role in ovarian\ntumorigenesis.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nLactase activity is high in infants but may decrease remarkably\nduring childhood. This state is known as adult-type hypolactasia\nor lactase non-persistence and represents a normal physiological\ncondition after weaning. The majority of northern Europeans have\nthe ability to maintain lactase activity and digest lactose through-\nout life (lactase persistence).26 Based on epidemiological observa-\ntions there is higher incidence of ovarian cancer in populations\nwhere lactase persistence is common.27 This was supported by a\ncase-control study where the consumption of dairy products and\nlactase persistence were observed to elevate ovarian carcinoma\nrisk28 but 2 studies reported no association.29,30 It is noteworthy\nthat the effect of lactase persistence on ovarian carcinoma has\nbeen based on indirect lactose tolerance tests or self-reports whose\naccuracy has been challenged.31,32\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nA variant C/T213910, locating 13.9 kb upstream of the LPH\ngene, has been shown to associate with the adult-type hypolactasia\ntrait. The CC213910 genotype is associated with lactase non-persis-\ntence whereas CT213910 and TT213910 genotypes are associated\nwith lactase persistence.33 The developmental downregulation of\nlactase activity operates at the transcriptional level. The T213910\nallele allows lactase gene expression and maintains lactase activity\nwhereas the C213910 allele lets the downregulation operate nor-\nmally.34 In addition, in vitro studies have shown that the T213910\nallele increases the lactase promoter activity.35,36 The frequency\nof the CC213910 genotype is in agreement with the reported pre-\nvalence of adult-type hypolactasia in several populations includ-\ning Finns, French, North Americans, African-Americans and\nSomalians.32,33\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nTo assess whether lactase persistence is a risk factor for ovarian\ncarcinoma we have determined the C/T213910 variant in Finnish,\nPolish and Swedish women with ovarian carcinoma and deﬁned\nits relation to their corresponding control populations by solid\nphase minisequencing.37\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nMaterial and methods\nSubjects\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nThe Finnish sample set consisted of 327 patients with epithelial\novarian cancer treated at the Department of Obstetrics and Gynae-\ncology, Helsinki University Central Hospital between 1989–1998.\nThe clinical diagnosis of ovarian carcinoma in the Finnish patients\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nGrant sponsor: Finnish Cultural Foundation; Grant sponsor: Sigrid\nJuselius Foundation, Helsinki; Grant\nsponsor: Helsinki University\nResearch Funding; Grant sponsor: Helsinki University Science Founda-\ntion; Grant sponsor: Swedish Cancer Fund.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n*Correspondence to: National Public Health Institute, Biomedicum\nHelsinki, Department of Molecular Medicine, PL104, 00251 Helsinki,\nFinland. Fax: 1358-9-47448480. E-mail: mikko.kuokkanen@ktl.ﬁ\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nReceived 15 November 2004; Accepted after revision 16 February 2005\nDOI 10.1002/ijc.21130\nPublished online 4 May 2005 in Wiley InterScience (www.interscience.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nwiley.com).\n**BLOCK**fs== 6.5**b== 0.9**t== 0.1**l== 0.1**r== 0.1**\nTABLE I – LACTASE PERSISTENCE, DEFINED THE C/T213910 VARIANT, AND ITS RISK FOR OVARIAN CARCINOMA PATHOGENESIS IN THE FINNISH,\nPOLISH AND SWEDISH POPULATIONS AND HISTOPATHOLOGICAL CHARACTERISTICS\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nFinnish patients\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nn\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nLactase persistence (%)\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nGenotype for C/T213910\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.5**\nCC\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nCT\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nTT\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nOR\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n95% CI\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\np-trend\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nHistology\nSerous\nMucinous\nEndometrioid\nOther\nTotal\nControl population\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nPolish patients\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nSerous\nMucinous\nEndometrioid\nOther\nTotal\nControl population\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nSwedish patients\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.3**r== 0.7**\n179 (55%)\n56 (17%)\n34 (10%)\n58 (17%)\n327 (100%)\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.3**r== 0.7**\n138 (45.5%)\n23 (8%)\n13 (4%)\n129 (42.5%)\n303 (100%)\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\n77.7 (139/179)\n80.4 (45/56)\n79.4 (27/34)\n74.1 (43/58)\n77.7 (254/327)\n81.9 (768/938)\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.4**r== 0.5**\n63.0 (87/138)\n56.5 (13/23)\n61.5 (8/13)\n70.5 (91/129)\n65.7 (199/303)\n66.9 (198/296)\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.1**r== 0.8**\nSerous\nMucinous\nEndometrioid\nOther\nTotal\nControl population\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.3**r== 0.7**\n85 (56%)\n8 (5%)\n20 (13%)\n39 (26%)\n152 (100%)\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.3**r== 0.7**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.4**r== 0.5**\n90.6 (77/85)\n100 (8/8)\n100 (20/20)\n94.9 (37/39)\n93.4 (142/152)\n89.7 (87/97)\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\n0.77\n0.91\n0.85\n0.64\n0.77\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.7**r== 0.3**\n0.84\n0.64\n0.79\n1.19\n0.95\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.7**r== 0.3**\n1.11\n0.90\n0.90\n2.13\n1.63\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.8**r== 0.2**\n0.52–1.14\n0.46–1.79\n0.37–1.99\n0.35–1.17\n0.57–1.05\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.8**r== 0.2**\n0.55–1.29\n0.27–1.52\n0.25–2.48\n0.76–1.86\n0.68–1.33\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.8**r== 0.2**\n0.42–2.95\n0.84–0.96\n0.84–0.96\n0.44–10.18\n0.65–4.08\n**BLOCK**fs== 8.5**b== 0.8**t== 0.2**l== 0.9**r== 0.1**\n0.185\n0.78\n0.72\n0.14\n0.097\n**BLOCK**fs== 8.5**b== 0.7**t== 0.3**l== 0.9**r== 0.1**\n0.43\n0.31\n0.77b\n0.46\n0.75\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.9**r== 0.1**\n0.84\n1.00b\n0.21b\n0.51b\n0.29\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nAll patientsa\nSerous\nMucinous\nEndometrioid\nOther\nTotal\nAll controls\na Mantel-Haenszel common OR estimates, CI.–b Fisher’s exact test, otherwise Pearson’s v2 test, CC213910, lactase non-persistence, CT213910\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.4**r== 0.5**\n75.4 (303/402)\n75.9 (66/87)\n82.1 (55/67)\n75.7 (171/226)\n76.1 (595/782)\n79.1 (1053/1331)\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.3**r== 0.7**\n402 (51%)\n87 (11%)\n67 (9%)\n226 (29%)\n782 (100%)\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.8**r== 0.2**\n0.63–1.09\n0.50–1.43\n0.53–1.99\n0.71–1.45\n0.71–1.10\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.6**r== 0.3**\n\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.6**r== 0.4**\n\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.7**r== 0.3**\n0.83\n0.85\n1.03\n1.02\n0.88\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.9**r== 0.1**\n0.17\n0.53\n0.94\n0.93\n0.26\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.3**r== 0.7**\n1,331\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nand TT213910, lactase persistence.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nwas made at the mean age of 56 years 6 14. About a quarter of\nthe Finns reside in the Helsinki region and due to recent migration\npatterns in Finland the patients have ancestral origins throughout\nFinland from early and late settlements regions. Finnish controls\nconsisted of 938 anonymous blood donors from Western and East-\nern Finland reﬂecting Finnish population mixture in early and late\nsettlements regions. The analysis of the Finnish control material\nwas reported by Enattah et al.33\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nPolish samples were obtained from 303 consecutive patients\nwith ovarian carcinoma treated at the Department of Chemother-\napy, Regional Oncology Centre, Szczecin, between 2001–2004.\nThe mean age of diagnosis was 54 (range 5 21–80) years. The\ncontrol material consisted of 296 unselected adults in the region of\nSzczecin representing the same population as the cases. The mean\nage of controls was 70.8 (range 5 22–80) years.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nThe 152 Swedish patients lived in the southern Swedish health\ncare region and were diagnosed with ovarian carcinoma between\n1998–2000. The mean age of diagnosis was 59 (range 5 22–82).\nThis patient material has previously been analyzed for germline\nmutations in BRCA1 and BRCA2.7 The control samples were\nobtained from healthy individuals and from patients without\nknown malignancy operated on at the Dept. of Orthopaedics at the\nUniversity Hospital in Lund, representing the southern Swedish\npopulation.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nHistological classiﬁcation\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nOvarian tumors were classiﬁed as serous, mucinous, endome-\ntrioid or other (includes undifferentiated, mixed epithelial tumors).\nThe histopathological characteristics of these cases are illustrated\nin Table I. All patient materials were approved by the local ethical\ncommittees and written informed consent was obtained from the\npatients and controls.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.7**\nPCR and solid phase minisequencing\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nPCR reactions were carried out in a total volume of 50 ll con-\ntaining PCR buffer (Dynazyme; Finnzymes, Espoo, Finland) con-\n**BLOCK**fs== 9.0**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\ntaining 10 mmol/l Tris-HCl (pH 5 8.8 at 25C), 1.5 mmol/l\nMgCl2, 50 mmol/l KCl, and 0.1% Triton X-100, 10 nmol deoxy-\nnucleotide triphosphates (dNTP) (Amersham Pharmacia Biotech,\nLittle Chalfont, Buckinghamshire, UK), 50 pmol\nforward\n(50-CCTCGTTAATACCCACTGACCTA-30) and 5 pmol biotiny-\nlated reverse (50-GTCACTTTGATATGATGAGAGCA-30) pri-\nmers amplifying the C/T213910 variant. All described PCR-reac-\ntions were initiated by a hot start with 0.8 U of Taq polymerase\nand carried out in the following conditions: after denaturation at\n194C for 4 min the PCR was initiated by a hot start, the samples\nwere cycled 34 times with a 30-sec annealing step at 153C, a 75-\nsec elongation step at 172C and a 30-sec denaturing step at\n194C followed by the last 30-sec annealing step at 153C and\n10 min extension at 172C. PCR reactions were electrophoresed\nwith 1 kb DNA ladder size standards (GeneRuler, Vilnius, Lithua-\nnia) on a 1.5% agarose gel with ethidium bromide.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nTen microliters of the biotinylated PCR products were immobi-\nlized on a streptavidin-coated solid support by incubation with\n40 ll of 0.15 mol/l NaCl, 20 mmol/l Na2SO4 (pH 7.5), and 0.1%\nTween 20 in streptavidin coated microtiter wells for 90 min at\n37C (Titramax 1000, Heidolph, Schwabach, Germany). The\nwells were washed 6 times with 350 ll of 40 mmol/l Tris-HCl,\npH 5 8.8, 1 mmol/l EDTA, 50 mmol/l NaCl and 0.1% Tween 20\nusing an automated washer\n(BW50, Biohit Plc, Helsinki,\nFinland). To remove the unbiotinylated complementary strand of\nthe immobilized PCR product, the wells were treated with 100 ll\nof 50 mmol/l NaOH for 5 min and washed 6 times as described\nearlier. Fifty microliters of minisequencing reaction mix contain-\ning 20 pmol of the detection primer (50-GGCAATACAGATAAG-\nATAATGTAG-30), 0.5 U of Taq polymerase, 2 pmol tritium\nlabeled dNTP: 3H-dTTP (111 Ci/mmol) or 3H-dCTP (64 Ci/mmol)\n(Amersham Pharmacia Biotech) in PCR buffer (Dynazyme, Finn-\nzymes, Espoo, Finland). The minisequencing reaction was allowed\nto continue for 15 min at 56C. The wells were washed 6 times as\ndescribed above and treated with 60 ll of 50 mmol/l NaOH for\n5 min to release tritium labeled detection primers. Fifty microliters\nof NaOH solution containing eluted detection primers were added\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nto 100 ll of scintillation liquid (Perkin Elmer, Wellesley, MA).\nThe incorporated radioactivity of 3H-labeled nucleotides was meas-\nured using a scintillation counter (1450 Microbeta, Perkin Elmer).\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nStatistical analysis\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nLactase persistence and its relation to ovarian carcinoma were\nevaluated by unconditional logistic regression and expressed as\nodds ratio (OR) with 95% conﬁdence interval (CI). Common OR\nwere estimated by Mantel-Haenszel statistics. Furthermore, the\nrisk was evaluated by 2-sided Pearson’s v2 or Fischer exact tests.\nStatistical analyses were carried out by SPSS for Windows,\nRelease 11.5.1 (SPSS Inc., Chicago, IL).\n**BLOCK**fs== 9.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nResults\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nThe distribution of lactase genotypes in ovarian carcinoma\npatients and their corresponding controls according to histological\nsubtype is shown in Table I. In the Finnish patients, lactase persis-\ntence had a modestly decreased effect on ovarian cancer risk (OR 5\n0.77, 95% CI 5 0.57–1.05). The probability for the trend (0.097),\nhowever, was not statistically signiﬁcant.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nLactase persistence had no signiﬁcant effect on the risk of ovar-\nian carcinoma in the Polish patients (OR 5 0.95, 95% CI 5 0.68–\n1.33, p 5 0.75) or the Swedish patients (OR 5 1.63, 95% CI 5\n0.65–4.08, p 5 0.29). When OR was evaluated for all 3 popula-\ntions lactase persistence showed a tendency of decreased, although\nnot signiﬁcant, risk for ovarian carcinoma (OR 5 0.88, 95% CI 5\n0.71–1.10, p 5 0.26) The results were not dependent on tumor his-\ntology (Table I).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nThe frequency of\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nlactase non-persistence, deﬁned by the\nCC213910 genotype, in the Polish controls was observed to be\n33.1%. This frequency ﬁgure is a somewhat lower than the previ-\nously published (37.5%) in Poland by breath hydrogen assay.38\nThe frequency of lactase non-persistence was 10.3% in the Swed-\nish control population, which is in agreement with the frequency\nobserved recently (9.6%) in Swedish Caucasian children (T.K.\nNilsson, personal communication).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nDiscussion\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nOur results indicate that lactase persistence, deﬁned by the\nC/T213910 variant associated with the lactase persistence/non-per-\nsistence trait,32–34,39 may decrease the ovarian carcinoma risk in\nthe Finnish population. Lactase persistence did not have effect on\nthe ovarian carcinoma risk in the Polish or Swedish populations.\nEarlier, results on lactase persistence and ovarian carcinoma risk\nthat have been obtained using lactose tolerance tests, self-reports\nand milk products consumption data have been conﬂicting.\nLactase persistence and lactose consumption were shown to asso-\nciate with increased ovarian carcinoma risk in Sardinia, Italy28\nand lactose intolerance has been suggested to indicate protective\neffect for ovarian carcinoma at an early age in eastern United\nStates40 but in studies with subjects from Ontario, Canada29 and\nwith Caucasians from Washington (United States)30 no such asso-\nciation was found.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nOur results are in agreement with previous studies in which lac-\ntose consumption had a protective effect on ovarian carcinoma.41–44\nThere is evidence that milk consumption is more abundant among\nsubjects with lactase persistence than in subjects with lactase non-\npersistence.32,45 In Finland with the highest milk consumption rate\n(177.9 kg/year/person, which equals to 23.4 g of lactose/day/person)\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nthe association between lactase persistence and ovarian carcinoma\nwas negative. The corresponding ﬁgures of milk consumption are\n142.4 kg/year/person in Sweden and 62.7 kg/year/person in\nPoland.46 In 2 cohort studies lactose consumption was associated\nwith increased ovarian carcinoma risk47,48 especially for that of the\nserous subtype.48 Several studies, however, have failed to ﬁnd a\nclear relation between lactose consumption and ovarian carci-\nnoma.29,30,49–53\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nMilk is an important source of vitamin D (0.5 lg/100 g of milk).\nAbout a half of the recommended daily intake of vitamin D\n(5 lg),54 added in non- or low-fat milk, is satisﬁed by the use of\nﬂuid milk in Finland. It is of interest that vitamin D has been shown\nto inhibit the growth of a human ovarian cancer cell lines.55,56 There\nare studies that dietary vitamin D42 with the highest quintile57 and\nsunlight-induced vitamin D formation58 are protective factors for\novarian carcinoma. Also, calcium intake is closely related to milk\nconsumption (120 mg/100 g of milk). Recommended daily intake\nof calcium is 800–900 mg in women but increasing the calcium\nintake to 500–1,000 mg per day may prevent osteoporosis.54\nCalcium consumption has been detected to inversely associate\nwith ovarian carcinoma risk.41,57 Not all studies have conﬁrmed\nthe possible protective effect of vitamin D or calcium.41,42,47,50,59\nCalcium as well as dietary vitamin D and endogenous vitamin D\nformation downregulate the production of parathyroid hormone\n(PTH).60,61 PTH participates in dietary calcium absorption stimu-\nlating 1,25 dihydroxyvitamin D formation. It has been speculated\nthat PTH can promote cancer indirectly, increasing IGF-1 synthe-\nsis or directly, acting as a co-mitogen in preneoplastic lesions\nexpressing the PTH receptor.62 Lactase persistence and high con-\nsumption of milk might partly explain our observed protective\ninﬂuence for ovarian carcinoma in the Finnish population.\nUnfortunately, due to lack of data for milk consumed by the study\nsubjects we could not draw further conclusions.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nGALT activity has been observed to be lower in patients with\novarian carcinoma than in controls. Furthermore, reduced activity\nof GALT together with lactose consumption has been shown to\nassociate with ovarian carcinoma.63 Later studies have not been\nable to conﬁrm these observations30,40 except among women with\nabnormally low GALT activity.52 Polymorphism N314D of\nGALT, can reduce the activity to 74–43% from normal GALT\nactivity.64,65 N314D frequency was higher in women with serous\nsubtype40 and endometrioid or clear cell ovarian cancers than con-\ntrols.66 The N314D polymorphism showed little evidence for\novarian carcinoma risk44 together with galactose intake.67\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nLactase persistence allows the efﬁcient use of milk as nutrition\nafter childhood. It has been thought to have been a beneﬁt since\ndairying began around 9,000 years ago especially in Northern\nEurope where the amount of sunlight is limited. We found that\nwomen with lactase persistence may have a moderately decreased\nrisk for the development of ovarian carcinoma in the Finnish pop-\nulation. We detected no association in the Polish or Swedish popu-\nlations. The results do not support the hypothesis that lactase per-\nsistence increases ovarian carcinoma risk.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nAcknowledgements\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nH. Komu is acknowledged for her technical assistance. The\nﬁnancial support of the Finnish Cultural Foundation, the Sigrid\nJuselius Foundation, Helsinki, Helsinki University Research Fund-\ning, the Helsinki University Science Foundation, and the Swedish\nCancer Fund are gratefully acknowledged.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\n1.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nParkin DM, Bray F, Ferlay J, Pisani P. Estimating the world cancer\nburden: Globocan 2000. Int J Cancer 2001;94:153–6.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n2. World Cancer Research Fund, American Institute for Cancer Research.\nOvary, food , nutrition and the prevention of cancer: a global perspec-\ntive. Washington, DC. American Institute for Cancer Research. 1997,\np 288–93.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n3. Runnebaum IB, Stickeler E. Epidemiological and molecular aspects\nof ovarian cancer risk. J Cancer Res Clin Oncol 2001;127:73–9.\n4. Wen WH, Reles A, Runnebaum IB, Sullivan-Halley J, Bernstein L,\nJones LA, Felix JC, Kreienberg R, el-Naggar A, Press MF. p53 muta-\ntions and expression in ovarian cancers: correlation with overall sur-\nvival. Int J Gynecol Pathol 1999;18:29–41.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.5**\nReferences\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n5. Risch HA, McLaughlin JR, Cole DE, Rosen B, Bradley L, Kwan E,\nJack E, Vesprini DJ, Kuperstein G, Abrahamson JL, Fan I, Wong B,\net al. Prevalence and penetrance of germline BRCA1 and BRCA2\nmutations in a population series of 649 women with ovarian cancer.\nAm J Hum Genet 2001;68:700–10.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n6. Van Der Looij M, Szabo C, Besznyak I, Liszka G, Csokay B, Pulay\nT, Toth J, Devilee P, King MC, Olah E. Prevalence of founder\nBRCA1 and BRCA2 mutations among breast and ovarian cancer\npatients in Hungary. Int J Cancer 2000;86:737–40.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n7. Malander S, Ridderheim M, Masback A, Loman N, Kristoffersson U,\nOlsson H, Nilbert M, Borg A. One in 10 ovarian cancer patients carry\ngerm line BRCA1 or BRCA2 mutations: results of a prospective study\nin Southern Sweden. Eur J Cancer 2004;40:422–8.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n8. Rubin SC, Blackwood MA, Bandera C, Behbakht K, Benjamin I,\nRebbeck TR, Boyd J. BRCA1, BRCA2, and hereditary nonpolyposis\ncolorectal cancer gene mutations in an unselected ovarian cancer pop-\nulation: relationship to family history and implications for genetic\ntesting. Am J Obstet Gynecol 1998;178:670–7.\nSarantaus L, Vahteristo P, Bloom E, Tamminen A, Unkila-Kallio L,\nButzow R, Nevanlinna H. BRCA1 and BRCA2 mutations among 233\nunselected Finnish ovarian carcinoma patients. Eur J Hum Genet\n2001;9:424–30.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\n9.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n10. Matsushima M, Kobayashi K, Emi M, Saito H, Saito J, Suzumori K,\nNakamura Y. Mutation analysis of the BRCA1 gene in 76 Japanese\novarian cancer patients: four germline mutations, but no evidence of\nsomatic mutation. Hum Mol Genet 1995;4:1953–6.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n11. Stratton JF, Gayther SA, Russell P, Dearden J, Gore M, Blake P,\nEaston D, Ponder BA. Contribution of BRCA1 mutations to ovarian\ncancer. N Engl J Med 1997;336:1125–30.\n12. Møller P, Heimdal K, Apold J, Fredriksen ˚A, Borg ˚A, Hovig E, Hagen\nA, Hagen B, Pedersen JC, Maehle L. Genetic epidemiology of\nBRCA1 mutations in Norway. Eur J Cancer 2001;37:2428–34.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n13. Menkiszak J, Gronwald J, Gorski B, Jakubowska A, Huzarski T,\nByrski T, Foszczynska-Kloda M, Haus O, Janiszewska H, Perkowska\net al. Hereditary ovarian cancer in\nM, Brozek I, Grzybowska E,\nPoland. Int J Cancer 2003;106:942–5.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n14. Whittemore AS, Harris R, Itnyre J. Characteristics relating to ovarian\ncancer risk: collaborative analysis of 12 US case-control studies. IV.\nThe pathogenesis of epithelial ovarian cancer. Collaborative Ovarian\nCancer Group. Am J Epidemiol 1992;136:1212–20.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n15. Hankinson SE, Hunter DJ, Colditz GA, Willett WC, Stampfer MJ,\nRosner B, Hennekens CH, Speizer FE. Tubal ligation, hysterectomy,\nand risk of ovarian cancer. A prospective study. JAMA 1993;270:\n2813–8.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n16. Risch HA. Hormonal etiology of epithelial ovarian cancer, with a\nhypothesis concerning the role of androgens and progesterone. J Natl\nCancer Inst 1998;90:1774–86.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n17. Haenszel W, Kurihara M. Studies of Japanese migrants. I. Mortality\nfrom cancer and other diseases among Japanese in the United States.\nJ Natl Cancer Inst 1968;40:43–68.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n18. Kaufman FR, Kogut MD, Donnell GN, Goebelsmann U, March C,\nKoch R. Hypergonadotropic hypogonadism in female patients with\ngalactosemia. N Engl J Med 1981;304:994–8.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n19. Xu YK, Ng WG, Kaufman FR, Lobo RA, Donnell GN. Galactose\nmetabolism in human ovarian tissue. Pediatr Res 1989;25:151–5.\n20. Reichardt JK, Woo SL. Molecular basis of galactosemia: mutations\nand polymorphisms in the gene encoding human galactose-1-phos-\nphate uridylyltransferase. Proc Natl Acad Sci USA 1991;88:2633–7.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n21. Harlow BL, Cramer DW, Geller J, Willett WC, Bell DA, Welch WR.\nThe inﬂuence of lactose consumption on the association of oral con-\ntraceptive use and ovarian cancer risk. Am J Epidemiol 1991;134:\n445–53.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n22. Cramer DW, Welch WR. Determinants of ovarian cancer risk. II.\nInferences regarding pathogenesis. J Natl Cancer Inst 1983;71:717–21.\n23. Leslie ND. Insights into the pathogenesis of galactosemia. Annu Rev\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\nNutr 2003;23:59–80.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n24. Swartz WJ, Mattison DR. Galactose inhibition of ovulation in mice.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.7**\nFertil Steril 1988;49:522–6.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n25. Chen YT, Mattison DR, Feigenbaum L, Fukui H, Schulman JD.\nReduction in oocyte number following prenatal exposure to a diet\nhigh in galactose. Science 1981;214:1145–7.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n26. Simoons FJ. The geographic hypothesis and lactose malabsorption. A\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nweighing of the evidence. Am J Dig Dis 1978;23:963–80.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n27. Cramer DW. Lactase persistence and milk consumption as determi-\nnants of ovarian cancer risk. Am J Epidemiol 1989;130:904–10.\n28. Meloni GF, Colombo C, La Vecchia C, Ruggiu G, Mannazzu MC,\nAmbrosini G, Cherchi PL. Lactose absorption in patients with ovarian\ncancer. Am J Epidemiol 1999;150:183–6.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n29. Risch HA, Jain M, Marrett LD, Howe GR. Dietary lactose intake, lac-\ntose intolerance, and the risk of epithelial ovarian cancer in southern\nOntario (Canada). Cancer Causes Control 1994;5:540–8.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n30. Herrinton LJ, Weiss NS, Beresford SA, Stanford JL, Wolﬂa DM,\nFeng Z, Scott CR. Lactose and galactose intake and metabolism in\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nrelation to the risk of epithelial ovarian cancer. Am J Epidemiol\n1995;141:407–16.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n31. Arola H. Diagnosis of hypolactasia and lactose malabsorption. Scand\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.3**\nJ Gastroenterol Suppl 1994;202:26–35.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n32. Rasinper€a H, Savilahti E, Enattah NS, Kuokkanen M, T€otterman N,\nLindahl H, J€arvel€a I, Kolho KL. A genetic test which can be used to\ndiagnose adult-type hypolactasia in children. Gut 2004;53:1571–6.\n33. Enattah NS, Sahi T, Savilahti E, Terwilliger JD, Peltonen L, J€arvel€a I.\nIdentiﬁcation of a variant associated with adult-type hypolactasia. Nat\nGenet 2002;30:233–7.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n34. Kuokkanen M, Enattah NS, Oksanen A, Savilahti E, Orpana A,\nJ€arvel€a I. Transcriptional regulation of the lactase-phlorizin hydrolase\ngene by polymorphisms associated with adult-type hypolactasia. Gut\n2003;52:647–52.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n35. Olds LC, Sibley E. Lactase persistence DNA variant enhances lactase\npromoter activity in vitro: functional role as a cis regulatory element.\nHum Mol Genet 2003;12:2333–40.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n36. Troelsen JT, Olsen J, Moller J, Sjostrom H. An upstream polymor-\nphism associated with lactase persistence has increased enhancer\nactivity. Gastroenterology 2003;125:1686–94.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n37. Syv€anen AC, Aalto-Set€al€a K, Harju L, Kontula K, S€oderlund H. A\nprimer-guided nucleotide incorporation assay in the genotyping of\napolipoprotein E. Genomics 1990;8:684–92.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n38. Socha J, Ksiazyk J, Flatz G, Flatz SD. Prevalence of primary adult\nlactose malabsorption in Poland. Ann Hum Biol 1984;11:311–6.\n39. Nilsson TK, Johansson CA. A novel method for diagnosis of adult\nhypolactasia by genotyping of the 213910 C/T polymorphism with\nPyrosequencing technology. Scand J Gastroenterol 2004;39:287–90.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n40. Cramer DW, Greenberg ER, Titus-Ernstoff L, Liberman RF, Welch\nWR, Li E, Ng WG. A case-control study of galactose consumption\nand metabolism in relation to ovarian cancer. Cancer Epidemiol Bio-\nmarkers Prev 2000;9:95–101.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n41. Goodman MT, Wu AH, Tung KH, McDufﬁe K, Kolonel LN, Nomura\nAM, Terada K, Wilkens LR, Murphy S, Hankin JH. Association of\nlactose, and calcium with the risk of ovarian cancer.\ndairy products,\nAm J Epidemiol 2002;156:148–57.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n42. Salazar-Martinez E, Lazcano-Ponce EC, Gonzalez Lira-Lira G, Escu-\ndero-De los Rios P, Hernandez-Avila M. Nutritional determinants of\nepithelial ovarian cancer risk: a case-control study in Mexico. Oncol-\nogy 2002;63:151–7.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n43. Yen ML, Yen BL, Bai CH, Lin RS. Risk factors for ovarian cancer in\nTaiwan: a case-control study in a low-incidence population. Gynecol\nOncol 2003;89:318–24.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n44. Goodman MT, Wu AH, Tung KH, McDufﬁe K, Cramer DW, Wilkens\nLR, Terada K, Reichardt JK, Ng WG. Association of galactose-1-\nphosphate uridyltransferase activity and N314D genotype with the\nrisk of ovarian cancer. Am J Epidemiol 2002;156:693–701.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n45. Obermayer-Pietsch BM, Bonelli CM, Walter DE, Kuhn RJ, Fahrleit-\nner-Pammer A, Berghold A, Goessler W, Stepan V, Dobnig H, Leb\nG, Renner W. Genetic predisposition for adult lactose intolerance and\nrelation to diet, bone density, and bone fractures. J Bone Miner Res\n2004;19:42–7.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n46. The World Dairy Situation. Bulletin of the International Dairy Feder-\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nation 384/2003, 2003.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n47. Kushi LH, Mink PJ, Folsom AR, Anderson KE, Zheng W, Lazovich\nD, Sellers TA. Prospective study of diet and ovarian cancer. Am J\nEpidemiol 1999;149:21–31.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n48. Fairﬁeld KM, Hunter DJ, Colditz GA, Fuchs CS, Cramer DW, Speizer\nFE, Willett WC, Hankinson SE. A prospective study of dietary lactose\nand ovarian cancer. Int J Cancer 2004;110:271–7.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n49. Mettlin CJ, Piver MS. A case-control study of milk-drinking and ovar-\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nian cancer risk. Am J Epidemiol 1990;132:871–6.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n50. Pan SY, Ugnat AM, Mao Y, Wen SW, Johnson KC. A case-control\nstudy of diet and the risk of ovarian cancer. Cancer Epidemiol Bio-\nmarkers Prev 2004;13:1521–7.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n51. Engle A, Muscat JE, Harris RE. Nutritional risk factors and ovarian\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.3**\ncancer. Nutr Cancer 1991;15:239–47.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n52. Webb PM, Bain CJ, Purdie DM, Harvey PW, Green A. Milk\nconsumption, galactose metabolism and ovarian cancer (Australia).\nCancer Causes Control 1998;9:637–44.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n53. Britton JA, Westhoff C, Howe GR, Gammon MD. Lactose and benign\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\novarian tumors in a case-control study. Br J Cancer 2000;83:1552–5.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n54. National Nutrition Council. Finnish Nutrition Recommendations. Hel-\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nsinki: Ministry of Agriculture and Forestry, 1999.32 p.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n55. Miettinen S, Ahonen MH, Lou YR, Manninen T, Tuohimaa P, Syv€al€a\nH, Ylikomi T. Role of 24-hydroxylase in vitamin D3 growth response\nof OVCAR-3 ovarian cancer cells. Int J Cancer 2004;108:367–73.\n56. Ahonen MH, Zhuang YH, Aine R, Ylikomi T, Tuohimaa P. Androgen\nreceptor and vitamin D receptor in human ovarian cancer: growth\nstimulation and inhibition by ligands. Int J Cancer 2000;86:40–6.\n57. Bidoli E, La Vecchia C, Talamini R, Negri E, Parpinel M, Conti E,\nMontella M, Carbone MA, Franceschi S. Micronutrients and ovarian\ncancer: a case-control study in Italy. Ann Oncol 2001;12:1589–93.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n58. Lefkowitz ES, Garland CF. Sunlight, vitamin D, and ovarian cancer\nmortality rates in US women. Int J Epidemiol 1994;23:1133–6.\n59. Tzonou A, Hsieh CC, Polychronopoulou A, Kaprinis G, Toupadaki N,\nTrichopoulou A, Karakatsani A, Trichopoulos D. Diet and ovarian\ncancer: a case-control study in Greece. Int J Cancer 1993;55:411–4.\n60. Takeuchi A, Okano T, Ishida Y, Kobayashi T. Effects of dietary vita-\nmin D intake on plasma levels of parathyroid hormone and vitamin D\nmetabolites in healthy Japanese. Miner Electrolyte Metab 1995;21:\n217–22.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n61. Malabanan A, Veronikis IE, Holick MF. Redeﬁning vitamin D insufﬁ-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nciency. Lancet 1998;351:805–6.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n62. McCarty MF. Parathyroid hormone may be a cancer promoter—an\nexplanation for the decrease in cancer risk associated with ultraviolet\nlight, calcium, and vitamin D. Med Hypotheses 2000;54:475–82.\n63. Cramer DW, Harlow BL, Willett WC, Welch WR, Bell DA,\nScully RE, Ng WG, Knapp RC. Galactose consumption and metab-\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\nolism in relation to the risk of ovarian cancer. Lancet 1989;2:\n66–71.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n64. Elsas LJ, Dembure PP, Langley S, Paulk EM, Hjelm LN, Fridovich-\nKeil J. A common mutation associated with the Duarte galactosemia\nallele. Am J Hum Genet 1994;54:1030–6.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n65. Langley SD, Lai K, Dembure PP, Hjelm LN, Elsas LJ. Molecular\nbasis for Duarte and Los Angeles variant galactosemia. Am J Hum\nGenet 1997;60:366–72.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n66. Morland SJ, Jiang X, Hitchcock A, Thomas EJ, Campbell IG. Muta-\ntion of galactose-1-phosphate uridyl transferase and its association\nwith ovarian cancer and endometriosis. Int J Cancer 1998;77:825–7.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n67. Cozen W, Peters R, Reichardt JK, Ng W, Felix JC, Wan P, Pike MC.\nGalactose-1-phosphate uridyl\ntransferase (GALT) genotype and\nphenotype, galactose consumption, and the risk of borderline and\ninvasive ovarian cancer (United States). Cancer Causes Control\n2002;13:113–20.",
         "extracted",
         "",
         "Lactase persistence and ovarian carcinoma risk in Finland, Poland and Sweden",
         "None"
        ],
        [
         "45",
         "00376fbb6ade96f6daf3b760897fc030dfb1a7aa",
         "We have created a stable transgenic rag2-EGFP-mMyc zebrafish line that develops GFP-labeled T cell acute lymphoblastic leukemia (T-ALL), allowing visualization of the onset and spread of this disease. Here, we show that leukemias from this transgenic line are highly penetrant and render animals moribund by 80.7 ± 17.6 days of life (±1 SD, range = 50-158 days). These T cell leukemias are clonally aneuploid, can be transplanted into irradiated recipient fish, and express the zebrafish orthologues of the human T-ALL oncogenes tal1/scl and lmo2, thus providing an animal model for the most prevalent molecular subgroup of human T-ALL. Because T-ALL develops very rapidly in rag2-EGFP-mMyc transgenic fish (in which “mMyc” represents mouse c-Myc), this line can only be maintained by in vitro fertilization. Thus, we have created a conditional transgene in which the EGFP-mMyc oncogene is preceded by a loxed dsRED2 gene and have generated stable rag2-loxP-dsRED2-loxP-EGFP-mMyc transgenic zebrafish lines, which have red fluorescent thymocytes and do not develop leukemia. Transgenic progeny from one of these lines can be induced to develop T-ALL by injecting Cre RNA into one-cell-stage embryos, demonstrating the utility of the Cre/lox system in the zebrafish and providing an essential step in preparing this model for chemical and genetic screens designed to identify modifiers of Myc-induced T-ALL.",
         "D. Langenau,H. Feng,S. Berghmans,J. Kanki,J. Kutok,A. Look",
         "https://europepmc.org/articles/pmc1087915?pdf=render",
         "\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.1**\nDavid M. Langenau*†, Hui Feng*, Stephane Berghmans*, John P. Kanki*, Jeffery L. Kutok‡, and A. Thomas Look*§\n**BLOCK**fs== 7.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\n*Department of Pediatric Oncology, Dana–Farber Cancer Institute, Harvard Medical School, Boston, MA 02115; †Department of HematologyOncology,\nChildren’s Hospital, Boston, MA 02115; and ‡Department of Pathology, Brigham and Women’s Hospital, Boston, MA 02115\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nEdited by Max D. Cooper, University of Alabama at Birmingham, Birmingham, AL, and approved March 15, 2005 (received for review November 22, 2004)\n**BLOCK**fs== 8.0**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\nWe have created a stable transgenic rag2-EGFP-mMyc zebraﬁsh\nline that develops GFP-labeled T cell acute lymphoblastic leukemia\n(T-ALL), allowing visualization of the onset and spread of this\ndisease. Here, we show that leukemias from this transgenic line are\nhighly penetrant and render animals moribund by 80.7  17.6 days\nof life (1 SD, range  50 –158 days). These T cell leukemias are\nclonally aneuploid, can be transplanted into irradiated recipient\nﬁsh, and express the zebraﬁsh orthologues of the human T-ALL\noncogenes tal1scl and lmo2, thus providing an animal model for\nthe most prevalent molecular subgroup of human T-ALL. Because\nT-ALL develops very rapidly in rag2-EGFP-mMyc transgenic ﬁsh (in\nwhich ‘‘mMyc’’ represents mouse c-Myc), this line can only be\nmaintained by in vitro fertilization. Thus, we have created a\nconditional transgene in which the EGFP-mMyc oncogene is pre-\nceded by a loxed dsRED2 gene and have generated stable rag2-\nloxP-dsRED2-loxP-EGFP-mMyc transgenic zebraﬁsh lines, which\nhave red ﬂuorescent thymocytes and do not develop leukemia.\nTransgenic progeny from one of these lines can be induced to\ndevelop T-ALL by injecting Cre RNA into one-cell-stage embryos,\ndemonstrating the utility of the Crelox system in the zebraﬁsh\nand providing an essential step in preparing this model for chem-\nical and genetic screens designed to identify modiﬁers of Myc-\ninduced T-ALL.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nlymphoma  tal1scl  lmo2\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nT cell lineage acute lymphoblastic leukemia (T-ALL) is an\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\naggressive hematologic malignancy arising when immature T\ncells acquire mutations that cause differentiation arrest, rapid\nproliferation, and suppression of apoptosis within developing T\nlymphocytes (1). Our current understanding of the molecular basis\nof T cell malignancies has emerged largely from the analysis of\nrecurrent chromosomal translocations, which typically juxtapose T\ncell oncogenes with strong promoter elements responsible for high\nexpression levels of the T cell receptor (2, 3). These T cell oncogenes\nencode transcription factors, including (i) basic helix–loop–helix\n(bHLH) family members such as TAL1SCL, TAL2, LYL1, and\nBHLHB1, (ii) LIM-only (LMO) domain genes such as LMO1 and\nLMO2, and (iii) the orphan homeobox genes HOX11TLX1 and\nHOX11L2TLX3 (3–5). Up-regulation of T-ALL oncogene tran-\nscription factors can also occur in the absence of chromosomal\ntranslocations (6–8), presumably due to mutations that cause either\nmonoallelic activation, through cis-acting mutations or deletions, or\nbiallelic activation, from the disruption of upstream factors that\nnormally suppress the expression of these genes in developing\nthymoctyes (8).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nIn human T-ALL, we have identified five distinct multistep\nmolecular pathways based on the overexpression of (i) TAL1SCL\nplus LMO1 or LMO2, (ii) LYL1 plus LMO2, (iii) HOX11, (iv)\nHOX11L2, and (v) MLL-ENL (1, 9), each of which is characterized\nby distinct molecular signatures (1, 10, 11). These subgroups are\nclinically relevant, with event-free survival differing among patient\ngroups (1, 11, 12). For example, patients expressing both TAL1\nSCL or LYL1 and a LMO family member (LMO1 or LMO2) have\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\na worse prognosis than those expressing HOX11 (1, 11, 12).\nActivation of these T cell oncogenes appears to be critical for\nthymocyte transformation, possibly by causing stage-specific arrest\nof T cell maturation (1). However, additional mutations are also\nfound in leukemic cells from patients in each of the major sub-\ngroups, including those that affect pathways that control apoptosis,\nproliferation, and genomic instability. Recently, we have identified\nactivating mutations in NOTCH1 that result in increased NOTCH\nsignaling and increased proliferation of developing thymocytes\n(13). In addition, four of five subgroups of human T-ALL express\nhigh levels of either MYC or MYCN (1), suggesting that MYC may\nbe a central regulator of proliferation andor genomic instability in\nthis malignancy. Finally, most human T-ALLs biallelically delete\nthe CDKN2A locus, which encodes both the p16(INK4A) and\np14(ARF) tumor suppressors, thereby disrupting both the RB and\np53 pathways and contributing to aberrant control of both cell cycle\nprogression and programmed cell death (1, 14).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nThe zebrafish has recently emerged as an important vertebrate\nmodel of Myc-induced T-ALL (15); however, studies have not been\nperformed to assess how closely zebrafish T-ALL mimics the\nhuman disease. In addition, the promise of the zebrafish T-ALL\nmodel lies in its utility for chemical (16–18) and genetic modifier\nscreens (19–21), marking the emergence of the zebrafish as a\nunique vertebrate model with which to identify enhancers that\naccelerate disease or suppressors that curb tumor growth. Here, we\nshow that transgenic rag2-EGFP-mMyc zebrafish (in which ‘‘mMyc’’\nrepresents mouse c-Myc) develop T-ALLs that faithfully model the\nmost common and most treatment-resistant subtype of human\nT-ALL, in which SCL and LMO12 are coexpressed. However,\nthese rag2-EGFP-mMyc transgenic fish are often severely diseased\nby the time they reach reproductive maturity, making this line\ndifficult to breed and maintain. Thus, conditional transgenic ap-\nproaches are needed to establish zebrafish leukemia models that are\namenable to forward genetic and small molecule suppressor\nscreens. Our current results indicate that the molecular mechanisms\nunderlying zebrafish T-ALL are remarkably similar to those found\nin the human disease and establish Crelox strategies in transgenic\nzebrafish that provide a general means to develop conditional\nmodels of cancer for genetic analysis in this model organism.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nMaterials and Methods\nIsolation of lmo1 and p16. RNA was obtained from 1- to 5-day-old\nembryos and made into cDNA, and degenerate PCR primers were\nused to amplify a fragment of the lmo1 and p16 gene. RACE PCR\nwas used to isolate the full-length lmo1 (GenBank accession no.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\nThis paper was submitted directly (Track II) to the PNAS ofﬁce.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nAbbreviations: T-ALL, T cell acute lymphoblastic leukemia; mMyc, mouse c-Myc; LMO,\nLIM-only; TCR, T cell receptor.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nData deposition: The sequence reported in this paper has been deposited in the GenBank\ndatabase (accession no. AF398514).\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n§To whom correspondence should be addressed. E-mail: thomaslook@dfci.harvard.edu.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.2**\n© 2005 by The National Academy of Sciences of the USA\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.1**r== 0.6**\n6068 – 6073  PNAS  April 26, 2005  vol. 102  no. 17\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nAF398514). By contrast, the p16 sequence fragment was used to\nsearch the zebrafish genome (www.sanger.ac.ukProjectsDrerio)\nand identify a putative full-length ORF for the p16 locus (R. Stewart\nand A.T.L., unpublished data).\n**BLOCK**fs== 9.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nPenetrance of Disease in rag2-EGFP-mMyc Stable Transgenic Ze-\nbrafish. Stable transgenic rag2-EGFP-mMyc fish have been gener-\nated previously and develop GFP-labeled T-ALL (15). To deter-\nmine the penetrance of disease in stable transgenic rag2-EGFP-\nmMyc fish, sperm was harvested from 10- to 20-week-old leukemic\nmale fish and used for in vitro fertilization of AB WT eggs. The\nresulting progeny were scored for leukemia onset at 30–60 days of\nlife as determined by infiltration of GFP-labeled leukemic cells into\nsites adjacent to the thymus. At 3 months of age, nonleukemic\nsibling fish were analyzed for the presence of the mMyc transgene\nas determined by PCR of genomic DNA isolated from the tail fin\nas described in ref. 15.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nCollection of Leukemias and Determination of DNA Content. Leuke-\nmic rag2-EGFP-mMyc fish were killed, and the heads were fixed in\n4% paraformaldehyde, embedded in paraffin, and sectioned. The\nremaining portion of fish was diced over 5 ml of ice-cold 0.9  PBS\nplus 5% FBS. The suspension was filtered over a 40-m filter,\nwashed, and subsequently (i) transplanted into irradiated recipient\nfish (1  106 cells per fish, 2–3 days after receiving 23 Gy of total\nbody irradiation from a 137Cs source), (ii) analyzed for DNA\ncontent as determined by DNA flow cytometry, (iii) frozen in 10\nmillion-cell aliquots, andor (iv) analyzed by FACS to determine\nthe percentage of GFP-labeled leukemic cells contained within\neach sample. DNA flow cytometric analysis was completed essen-\ntially as described in ref. 15. Specifically, tumor cells and WT\nnucleated red blood cells were stained with propidium iodide in\nhypotonic sodium-citrate buffer and analyzed for cellular DNA\ncontent by flow cytometry alone and as a mixture with normal\ncontrol cells.\n**BLOCK**fs== 9.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nRT-PCR Analysis. RNA was isolated (with TRIzol, GIBCOBRL)\nfrom leukemia cells and control FACS-sorted, GFP-positive thy-\nmocytes from rag2-GFP and lck-GFP transgenic fish (15, 22, 23).\nRNA was treated with DNaseI before reverse transcription, and\nRT-PCR was performed. (PCR primers and thermocycling condi-\ntions are described in detail in the Supporting Text and Table 1,\nwhich are published as supporting information on the PNAS web\nsite.)\n**BLOCK**fs== 9.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nTo confirm that expression of the scl and lmo2 transcripts was\nconfined to the leukemic lymphoblasts, RNA in situ hybridization\nwas completed on paraffin-embedded sections from transgenic\nrag2-EGFP-mMyc fish essentially as described in refs. 15 and 22.\nPCR primers used to generate probes for in situ hybridization\nanalysis are described in the Supporting Text.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nTo determine whether scl and lmo2 expression resulted from\ntranscription of one or both alleles, PCR was used to amplify the 3\nuntranslated regions of scl and lmo2 (Table 1). PCR products were\npurified (QIAquick PCR Purification Kit, Qiagen, Valencia, CA)\nand sequenced. cDNA was obtained from zebrafish T-ALL samples\nhaving polymorphic alleles for either scl or lmo2 and subjected to\nPCR. PCR fragments were purified and sequenced.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nSouthern Blot Analysis to Determine Clonality and to Assess Loss of\nthe p16Locus. Southern blot analysis was used to determine whether\nleukemia cells have T cell receptor (TCR)- or IgM receptor\nrearrangements and whether the p16 genomic locus is lost. South-\nern blot analysis was performed as described in refs. 15 and 24.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nSequencing of the p53 Locus. Genomic DNA was isolated from\nzebrafish leukemic samples and subjected to PCR amplification of\nexons 4–9 of the zebrafish p53 gene. Fragments were purified and\nsequenced as described in ref. 25 and Table 1.\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nCrelox Strategies. To verify that Crelox-mediated strategies work\nin the zebrafish, an expression vector was created that contains loxP\nsites flanking the dsRED2 transgene and the polyadenylation site\ncontained within the dsRED2-N1 vector (Clontech). The loxed\ndsRED2 transgene was cloned into the EGFP-N1 expression vector\nupstream of the EGFP ORF (Fig. 3A). One-cell-stage embryos\nwere injected with either the CMV-loxP-dsRED2-loxP-EGFP plas-\nmid (50 ngl) alone or in combination with Cre RNA (25 ngl).\nCre RNA was made by in vitro transcription by using the pCS2Cre\nvector and SP6 RNA polymerase. Transiently injected embryos\nwere analyzed 26 h postfertilization for GFP and dsRED2 expres-\nsion as determined by fluorescent microscopy.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nDeveloping Conditional Transgenic Zebrafish. The loxed dsRED2\ncoding sequence was cloned into the rag2-EGFP-mMyc plasmid\nupstream of the EGFP-mMyc transgene by using BamHI restriction\nenzyme sites (Fig. 4A). The resulting rag2-loxP-dsRED2-loxP-\nEGFP-mMyc plasmid was linearized with XhoI, phenol\nchloroform-extracted, and ethanol-precipitated. Linearized DNA\nwas injected into one-cell-stage AB strain embryos [100 ngl DNA\nin 0.5 TE buffer (10 mM Tris1 mM EDTA, pH  7.0) containing\n100 mM KCl. Primary injected adult fish were screened for the\nability to produce offspring that contained the transgene as deter-\nmined by detection of dsRED2 fluorescence within developing\nthymocytes at 6 days postfertilization. Two stable transgenic rag2-\nloxP-dsRED2-loxP-EGFP-mMyc zebrafish lines were identified\n(lines G7 and G16, AB strain). F1 and F2 rag2-loxP-dsRED2-loxP-\nEGFP-mMyc transgenic fish were injected with 25 ngl Cre RNA\nand analyzed for leukemia onset.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nAnalysis of Leukemias from Cre-Injected rag2-loxP-dsRED2-loxP-EGFP-\nmMyc Fish. Leukemic cells from Cre-injected stable transgenic\nrag2-loxP-dsRED2-loxP-EGFP-mMyc fish were harvested and (i)\ntransplanted into irradiated adult fish, (ii) analyzed by cytospin and\nGiemsaMay–Grunwald staining to confirm lymphoblast morphol-\nogy, (iii) subjected to FACS analysis to assess levels of GFP and\ndsRED2 expression within lymphoblasts, andor (iv) extracted for\ngenomic DNA and analyzed for CRE recombination as detected by\nPCR (forward primer specific to the rag2 promoter region, AT-\nGCTAATTTGAAGCACTAGCA; reverse primer specific to the\nEGFP coding sequence, GTGCAGATGAACTTCAGGGT).\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nResults\nComplete Penetrance of T-ALL in a rag2-EGFP-mMycStable Transgenic\nLine. We have previously described a rag2-EGFP-mMyc stable\ntransgenic zebrafish line in which the onset and progression of T cell\nmalignancy can be monitored by fluorescence microscopy (15). The\ndisease is first detected as an expansion of GFP-labeled T cells in\nthe thymus, and, subsequently, malignant cells infiltrate regions\nadjacent to the thymus in a phase comparable to human T cell\nlymphoblastic lymphoma. Then, transformed T cells rapidly spread\nthroughout the skeletal musculature, visceral organs, and kidney\nmarrow, leading to widely disseminated T-ALL (Fig. 1 and Fig. 4\nof ref. 15). In a series of 106 stable transgenic rag2-EGFP-mMyc\nfish, all animals developed T-ALL and succumbed to death by\n80.7  17.6 days of life (1 SD, range  50–158 days). Sixty-four\nnonleukemic siblings were raised until 3 months of age, and none\nharbored the mMyc transgene in somatic DNA. Taken together,\nthese results indicate that Myc-induced leukemias are fully pene-\ntrant in our rag2-EGFP-mMyc transgenic line.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nMyc-Induced Leukemias Are of T Cell Origin and Oligoclonal. Because\nrag2 is expressed in both immature T and B cells (22), we wanted\nto assess whether leukemias arising in our stable transgenic ze-\nbrafish were of T cell or B cell origin. Several lines of evidence\nindicated that all of the leukemias arising in this transgenic line were\nof T cell origin. (i) Each malignancy developed as a GFP-labeled\nlymphoma in the thymus (n  106). (ii) Southern blot analysis\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.6**r== 0.1**\nPNAS  April 26, 2005  vol. 102  no. 17  6069\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\ncancer, so we tested whether leukemic cells from rag2-EGFP-mMyc\nfish could be transplanted by i.p. injection into irradiated adult\nrecipient fish. Lymphoblasts were isolated from primary leukemic\nfish, and FACS analysis confirmed that samples were highly en-\nriched for leukemic lymphoblasts, containing 93.8  1.4% (SD)\nGFP-labeled leukemic cells (n  10). GFP-labeled cells multiplied\nand generated leukemia in irradiated recipient fish within 1 month\nafter injection (n  11 independently arising leukemias). These\nresults are similar to those reported previously for leukemia cells\narising in F0 primary injected fish (15).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nZebrafish Myc-Induced T-ALLs Coexpress both tal1scl and lmo2.\nBecause cMYC expression is up-regulated in three of five molecular\nsubgroups of human T-ALL (1, 9), including those misexpressing (i)\nTAL1SCL plus LMO1 or LMO2, (ii) HOX11, and (iii) HOX11L2\nTLX3, we predicted that we might observe several distinct molec-\nular subgroups of zebrafish T-ALL. However, RT-PCR (Fig. 2 A\nand Table 3) and in situ hybridization analyses (Fig. 2 B–E) showed\nthat all Myc-induced leukemias coexpress tal1scl and lmo2 (n \n20) but not lmo1, hox11, or the zebrafish orthologues of HOX11L2\nTLX3, tlx3a, or tlx3b (26). By contrast, RT-PCR analysis revealed\nthat normal thymocytes from lck-GFP and rag2-GFP stable trans-\ngenic fish express lower levels of scl and similar levels of lmo2 when\ncompared with Myc-induced T-ALLs (Fig. 2 A). Finally, RNA in\nsitu hybridization of paraffin-embedded sections from 70-day-old\nlck-GFP transgenic animals showed that scl and lmo2 are expressed\nin only a subset of cortical thymocytes (Fig. 2 F–M).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nTo investigate the mechanisms by which tal1scl and lmo2 are\noverexpressed in zebrafish T-ALL, we asked whether expression\nwas monoallelic or biallelic at the chromosomal level. We identified\nfour zebrafish leukemia sample DNAs that harbored polymor-\nphisms in the 3 untranslated region of scl and three with polymor-\nphisms in the lmo2 gene (n  20). RT-PCR analysis of these cDNAs\nrevealed that in each case, scl and lmo2 transcripts were up-\nregulated equally from both chromosomal alleles (Table 2; see also\nFig. 9, which is published as supporting information on the PNAS\nweb site), indicating that expression is biallelic and does not result\nfrom chromosomal translocations or other allele-specific deletions\nor mutations.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nFig. 1.\nStable transgenic rag2-EGFP-mMyc zebraﬁsh develop GFP-labeled\nthymic lymphoma, which progresses to T-ALL. Fluorescence microscopic anal-\nysis at 50 days of life showing the thymus of control rag2-GFP transgenic ﬁsh\n(A) and massive GFP-labeled cellular dissemination of leukemic lymphoblasts\nin rag2-EGFP-mMyc transgenic ﬁsh (B). Fish are oriented with anterior to the\nleft and dorsal to the top. Arrowheads mark location of the thymus (T).\n**BLOCK**fs== 9.0**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nshowed that Myc-induced leukemias contained oligoclonal TCR-\ngene rearrangements (18 of 36), whereas none of the leukemias\ncontained Ig heavy chain receptor gene (IgM) rearrangements (n \n21; Figs. 6 and 7 and Table 2, which are published as supporting\ninformation on the PNAS web site). (iii) RT-PCR analysis showed\nthat leukemic lymphoblasts expressed high levels of the T cell-\nspecific tyrosine kinase gene (lck) (22) (Fig. 2A) and TCR- (24)\n(n  30; Table 3, which is published as supporting information on\nthe PNAS web site).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nDNA flow cytometry was also used to determine clonality and to\nassess whether leukemias acquired chromosomal abnormalities\nduring disease progression (Table 2; see also Fig. 8, which is\npublished as supporting information on the PNAS web site). Ten of\n63 leukemias analyzed were hyperdiploid, as indicated by clonal\nincreases in DNA content (range  1.01–1.45). Although most of\nthese leukemias contained monoclonal populations of cells with\nincreased DNA content (9 of 10), one T-ALL sample contained\ntwo distinct populations of hyperdiploid cells but failed to show\nTCR- gene rearrangement by Southern analysis (Fig. 8B).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nMyc-Induced Leukemias Are Transplantable. Transplantation and\npropagation of disease into secondary recipients is a hallmark of\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nApoptotic Pathways in Zebrafish Myc-Induced T-ALL. Myc-induced\ntransformation in mammals collaborates with mutations that dis-\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nLeukemic lymphoblasts express T cell markers and both scl and lmo2. (A) Semiquantitative RT-PCR showing that Myc-induced leukemias are arrested\nFig. 2.\nat a stage in which scl and lmo2 are coexpressed. RT-PCR analysis of FACS-sorted, GFP-labeled thymocytes from 70-day-old transgenic lck-GFP (lck) and rag2-GFP\n(rag2) ﬁsh or leukemic lymphoblasts isolated from diseased ﬁsh (denoted by numbers). RT, reverse transcription reactions; No RT, no reverse transcription\ncontrols. (B–M) RNA in situ hybridization of parafﬁn-embedded sections conﬁrms that scl and lmo2 are coexpressed in lymphoblasts from rag2-EGFP-mMyc\ntransgenic ﬁsh (shown here inﬁltrating the skeletal musculature) and that scl and lmo2 are expressed in a subset of thymocytes in 70-day-old, nonleukemic,\nlck-GFP transgenic zebraﬁsh. Images are photographed at 400 (B–E), 50 (F–I), and 200 (J–M). Images in J–M are higher-magniﬁcation images of the respective\nregions in F–I (indicated by the boxed region in F).\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nable components of the cellular apoptotic machinery (1, 14, 27, 28).\nHowever, given the rapidity of leukemia onset in our transgenic\nmodel, we questioned whether apoptosis was suppressed in ze-\nbrafish T-ALL and, if so, whether we could document the mech-\nanisms that deregulate cell death in these tumors. Semiquantitative\nRT-PCR analysis showed that each of the zebrafish T-ALLs\nexpressed similar levels of p53, mdm2, bcl-xL, bcl-2, and bax RNAs\n(n  12; Fig. 2 A and Table 3). When compared with thymocyte\ncontrols, Myc-induced T-ALLs expressed higher levels of bcl-xL,\nbax, p53, and mdm2; however, bcl-2 RNA expression was decreased,\nlikely reflecting that T-ALLs are arrested at a stage of development\nmarked by this gene expression profile (8).\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nBecause human T-ALLs have either biallelic deletions of the\nCDKN2A locus (14) or, less frequently, mutational inactivation of\np53 (29, 30), we asked whether zebrafish Myc-induced leukemias\nharbor abnormalities in orthologues of these loci. The p16 copy\nnumber was not decreased in these tumors as detected by Southern\nblot analysis (n  21) (Table 2; see also Fig. 10, which is published\nas supporting information on the PNAS web site). Because 90%\nof mutations in p53 occur within exons 4–8 (the region that encodes\nthe DNA-binding domain) (31, 32), we analyzed zebrafish T-ALL\nsamples for mutations in p53 in these corresponding exons. We\nsequenced genomic DNA from zebrafish T-ALL leukemia cells and\nfailed to identify any mutations in exons 4–9 of the p53 genomic\nlocus (n  12; Table 2).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.1**r== 0.5**\nCrelox Conditional Transgenic Strategies in the Zebrafish. T-ALL\ndevelops in 100% of stable transgenic rag2-EGFP-mMyc fish and\nprogresses to widespread disease before reproductive maturity,\nnecessitating harvesting sperm from diseased males to maintain the\ntransgenic line by in vitro fertilization (IVF) (15). Because IVF\nprocedures are cumbersome and not amenable to forward genetic\napproaches, we sought to develop a conditional transgenic ap-\nproach, allowing identification and maintenance of zebrafish lines\nthat did not develop leukemia until the investigator selectively\ninduced T cell-specific expression of the Myc oncogene. For this\npurpose, we developed Crelox-mediated transgenic approaches,\nwhich have been reported in mice (33, 34) and Xenopus (35, 36) but\nnot zebrafish.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nWe created a vector in which the CMV promoter drives the\nubiquitous expression of a dsRED2 transgene that is followed by\nmultiple transcription stop sites and flanked by loxP sites. The\nEGFP coding sequence was cloned downstream of this cassette\n(CMV-loxP-dsRED2-loxP-EGFP vector; Fig. 3A). Transient injec-\ntion of the CMV-loxP-dsRED2-loxP-EGFP construct into embryos\nwithout Cre recombinase results in dsRED2 fluorescence and no\nEGFP expression (50 ngl; Fig. 3 C and D). By contrast, coinjec-\ntion of the CMV-loxP-dsRED2-loxP-EGFP plasmid (50 ngl) with\nCre RNA (25 ngl) resulted in the excision of the dsRED2 allele\nand juxtaposition of the EGFP transgene next to the CMV pro-\nmoter, leading to embryos that express EGFP and no red fluores-\ncence (Fig. 3 F and G). Excision was extremely efficient in embryos\ninjected with 25 ngl Cre RNA because single cells with red\nfluorescence were observed in 5% of injected zebrafish embryos\n(n  100).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nApplying this strategy to our transgenic models of T cell malig-\nnancy, the rag2-EGFP-mMyc transgene was modified by inserting\nthe loxP-dsRED2-loxP cassette between the rag2 promoter and the\nEGFP-mMyc oncogene (Fig. 4A), and two stable transgenic lines\nwere generated (G7 and G16). In the absence of Cre-mediated\nrecombination, rag2-loxP-dsRED2-loxP-EGFP-mMyc transgenic\nfish exhibited high levels dsRED2 expression in the developing\nthymocytes but failed to express the EGFP-mMyc transgene or\ndevelop lymphoma or leukemia (Fig. 4B). After the injection of Cre\nRNA into G7 and G16 one-cell embryos, the dsRED2 allele was\nexcised in both lines (Fig. 4C); however, only transgenic zebrafish\nfrom the G7 line developed T-ALL. Most leukemias arising in the\nG7 line expressed both dsRED2 and EGFP-mMyc (Fig. 4D),\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nFig. 3.\nCre-mediated recombination in transiently injected embryos. (A)\nDiagram of the CMV-loxP-dsRED2-loxP-EGFP construct. (B–G) One-cell-stage\nembryos were injected with the CMV-loxP-dsRED2-loxP-EGFP vector in the\nabsence of Cre RNA ( Cre RNA) (B–D) or with Cre RNA ( Cre RNA, 25 ngl)\n(E–G). Shown are bright-ﬁeld (B and E), red ﬂuorescence (dsRED2) (C and F),\nand green ﬂuorescence (EGFP) images (D and G) of embryos at 26 h postfer-\ntilization. Anterior is to the left, and dorsal is toward the top.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nsuggesting that Cre recombination was incomplete in embryos\ninjected with 25 ngl Cre RNA. Some of the G7 line leukemias\nexhibited complete recombination and expressed only the EGFP-\nmMyc transgene, indicated by green fluorescence without any\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nFig. 4.\nCre RNA injection into stable transgenic rag2-loxP-dsRED2-loxP-\nEGFP-mMyc ﬁsh leads to transgene recombination and rapid onset of Myc-\ninduced T-ALL. (A) Diagram of rag2-loxP-dsRED2-loxP-EGFP-mMyc construct.\n(B) Thymocytes from a 73-day-old rag2-lox-dsRED2-EGFP-mMyc transgenic\nﬁsh are red-ﬂuorescent-labeled in the absence of Cre expression. (C) PCR of\ngenomic DNA isolated from blood cells of WT control or transgenic rag2-loxP-\ndsRED2-loxP-EGFP-mMyc (LDL-EMyc) ﬁsh. rag2 primers amplify genomic DNA,\nmMyc primers amplify the mMyc transgene, and Lox primers amplify either a\n1.7-kb nonrecombined fragment (Lox-NR) or a 0.4-kb fragment, which results\nwhen Cre recombination has occurred (Lox-Rec). (D and E) One-cell-stage\nrag2-loxP-dsRED2-loxP-EGFP-mMyc embryos were injected with the Cre RNA\n(25 ngl) and grown to 51 days of development, at which time they had both\nGFP- and dsRED2-labeled (D) or GFP-positive alone (E) leukemias. The same\nﬁsh is shown, right (D) and left (E) side. Images are composites of dsRED2 and\nGFP ﬂuorescence and bright-ﬁeld images.\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.6**r== 0.1**\nPNAS  April 26, 2005  vol. 102  no. 17  6071\n**BLOCK**fs== 7.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nFig. 5.\nLeukemias developing in Cre-injected rag2-loxP-dsRED2-loxP-EGFP-mMyc transgenic ﬁsh are transplantable, express low levels of GFP ﬂuorescence, and\nhave typical lymphoblast morphology. WT control (A–C) and irradiated WT ﬁsh transplanted with lymphoblasts from rag2-loxP-dsRED2-loxP-EGFP-mMyc (D–F)\nor rag2-EGFP-mMyc transgenic (G–I) ﬁsh. (A, D, and G) Fluorescent microscopic analysis with ﬁsh oriented with anterior to the left and dorsal to the top. Images\nare a composite of GFP ﬂuorescence and bright-ﬁeld images. (B, E, and H) FACS analysis based on GFP ﬂuorescence. (C, F, and I) GiemsaMay–Grunwald staining\nof blood cells obtained from cytospin analysis. Original images were photographed at 1,000.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\ndetectable red fluorescence (Fig. 4E). Heterozygous rag2-loxP-\ndsRED2-loxP-EGFP-mMyc fish (G7 line) were bred to AB WT fish\nand injected with Cre RNA at the one-cell stage of development.\nIn total, 12 of 186 CRE-injected progeny developed disease in the\nG7 line by 151  61 days (range  52–192 days; n  5).\n**BLOCK**fs== 9.0**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nT-ALLs in Cre-Injected Fish Are Similar to Those from rag2-EGFP-mMyc\nLeukemias. Fluorescent microscopic analysis revealed that rag2-\nloxP-dsRED2-loxP-EGFP-mMyc transgenic zebrafish injected with\nCre RNA at the one-cell stage develop thymic leukemias as adults,\nindicating that these tumors are of T cell origin. Additionally,\nfluorescent leukemia cells can be transplanted into irradiated\nrecipients (Fig. 5 D and G; n  3) and have lymphoblast morphol-\nogy as determined by GiemsaMay–Grunwald staining (Fig. 5 F\nand I; n  2). Taken together, these results indicate that leukemias\narising in the conditional rag2-loxP-dsRED2-loxP-EGFP-mMyc\ntransgenic line (G7) are similar to those observed in the rag2-\nEGFP-mMyc stable transgenic line.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nDiscussion\nWe have previously shown that transgenic zebrafish develop Myc-\ninduced T cell lineage leukemia (15); however, a detailed molecular\ncharacterization of these tumors has not been described. Leukemias\ndeveloping in rag2-EGFP-mMyc transgenic fish are remarkably\nsimilar to those found in human patients with T-ALL. For example,\nzebrafish Myc-induced T-ALLs arise after a defined latency period\nand have clonal TCR- gene rearrangements, suggesting that\nadditional mutations are required for malignant transformation of\nthe T cell. Additionally, these zebrafish T-ALLs coexpress both scl\nand lmo2, resembling the most common and most treatment-\nresistant molecular subtype of this disease in humans (1, 11). Both\nscl and lmo2 are biallelically activated in zebrafish Myc-induced\nT-ALL, in a pattern similar to that of a subset of patients who have\noverexpression of SCL and LMO2 in leukemic lymphoblasts,\nindicating that malignant transformation likely results from the\ndisruption of upstream regulatory mechanisms that normally turn\noff the expression of these transcription factors during double-\nnegative thymocyte development (8).\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nAlthough zebrafish Myc-induced T-ALLs are similar to human\nleukemias, there are also key differences. First, zebrafish Myc-\ninduced T-ALLs resemble only one subclass of human T-ALLs,\nthose that coexpress SCL and LMO2. Remarkably, we never\nobserve expression of hox11tlx1 or the hox11L2tlx3 family mem-\n**BLOCK**fs== 9.0**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nbers in Myc-induced leukemias in the zebrafish (26). It is possible\nthat timing of transgene expression during thymocyte development\naffects the subtype of T-ALL that is observed. For example, rag2\nexpression is tightly regulated during T cell development and is only\ninduced in cells undergoing active TCR- and TCR- gene rear-\nrangement (37, 38), providing two waves of Myc transgene expres-\nsion during thymocyte development. Another possibility is that the\nmechanisms that regulate HOX11- and HOX11L2-induced trans-\nformation in humans are not found in the zebrafish. rag2-hox11 and\nrag2-tlx3 transgenic zebrafish will need to be developed to deter-\nmine whether overexpression of a hox11 family member can\nsynergize with Myc in the genesis of zebrafish T-ALL.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nHuman and murine leukemias escape cell death by inactivation\nof multiple gene products involved in regulating the apoptotic\npathways. For example, leukemias developing in Emu-Myc mice\nharbor mutations that curb apoptosis,\nincluding the loss of\np19(ARF), mutation of p53, or up-regulation of Mdm2 (27), and\nmost human T-ALLs have biallelic deletion of the CDKN2A locus,\nwhich encodes the P14(ARF) gene (14). Because of these findings,\nwe expected to identify abnormalities that down-regulate apoptotic\npathways in our zebrafish leukemias; however, we did not find\nvariable expression levels of mdm2, bcl-xL, bcl-2, p53, or bax, nor did\nwe find deletions in the p16 gene locus or mutations in the p53\nDNA-binding domain. Thus, we conclude that either suppression of\napoptosis is mediated by a currently unidentified mechanism in\nzebrafish T-ALLs or Myc-induced transformation in zebrafish\nlymphoid cells does not require an associated mutational inactiva-\ntion of the apoptotic machinery. Given that teleost fishes apparently\nlack an ARF gene (39), it is likely that overexpression of myc may\nnot activate the cell death machinery through the p53 pathway in\nzebrafish, as has been documented in mammalian cells (40).\nFurther experiments using rag2-EGFP-bcl-2 transgenic (41) and\np53-deficient fish (25) will likely resolve whether suppression of\napoptosis is required for malignant transformation of the T cell in\nzebrafish.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nLeukemias developed in 100% of stable transgenic rag2-EGFP-\nmMyc fish, which is optimal for performing genetic screens de-\nsigned to uncover mutations that enhance or suppress leukemo-\ngenesis. However, the transgenic rag2-EGFP-mMyc zebrafish line\nhas been difficult to maintain, because these fish develop disease\nbefore reaching full reproductive maturity. To resolve this problem,\nwe developed conditional transgenic zebrafish by using Crelox\ntechnology (33–36). Two transgenic zebrafish lines were identified\n**BLOCK**fs== 9.0**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nthat contained the rag2-loxP-dsRED2-loxP-EGFP-mMyc transgene.\nIn the absence of Cre RNA expression, both lines had strong\nexpression of dsRED2 within the developing T cells, and no fish\ndeveloped disease, indicating that the mMyc oncogene was not\nexpressed in the absence of Cre recombination. After injection of\nCre RNA into one-cell-stage embryos, both transgenic lines exhib-\nited recombination at the loxP sites; however, only the G7 stable\nline produced offspring that developed T-ALL. Lack of leukemia\nonset in the second transgenic line (G16) reinforces the need to\ndevelop multiple transgenic lines and suggests that positional effects\nof integration andor concatamer orientation may significantly\naffect Cre-mediated excision and subsequent expression of the\nsecond transgene.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\nBecause only 13% of Cre-injected transgenic fish developed\ndisease, it is likely that injection of Cre RNA results in suboptimal\nrecombination and mosaic activation of the EGFP-mMyc transgene\nafter recombination. This interpretation is supported by the fact\nthat most leukemias arising in Cre-injected rag2-loxP-dsRED2-loxP-\nEGFP-mMyc transgenic fish are both dsRED2- and GFP-labeled,\nindicating that leukemic clones had partial Cre-mediated recom-\nbination at the locus containing transgene concatamers. Similarly,\nindividual fish had a leukemic clone that expressed both dsRED2\nand GFP arising in one thymus and a second leukemia clone that\nexpressed only GFP in the other thymus, indicating that Cre\nrecombination occurs in a mosaic fashion within hematopoietic\nprogenitors of individual fish. Similar results have been observed in\nXenopus (36). For example, transient injection of Cre RNA into\nstable transgenic frogs harboring a CMV-loxP-ECFP-loxP-EYFP\ntransgene resulted in mosaic expression of the second ORF.\nFurthermore, in some animals, neither blue (ECFP) nor yellow\n(EYFP) fluorescence was detected in transgenic frogs after Cre\nRNA injection, leading the researchers to conclude that the copy\nnumber of the reporter had been reduced by recombination without\ngenerating an active EYFP expression cassette (36). By contrast,\nbreeding these CMV-loxP-ECFP-loxP-EYFP transgenic frogs to\nstable transgenic Cre-expressing animals resulted in 100% of\ndoubly transgenic offspring having homogenous expression of the\nsecond EYFP ORF. Thus, developing transgenic zebrafish lines that\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nspecifically express Cre in the developing T cells will likely aid in the\nestablishment of more penetrant models of disease.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nAlthough zebrafish models of T-ALL exhibit key differences\nwhen compared with the human disease, analysis of the conserved\nmechanisms underlying transformation will likely lead to insights\ninto the pathogenesis of human disease. For example, the molecular\nmechanisms responsible for regulating biallelic activation of scl and\nlmo2 are unknown, and the downstream targets of Myc that are\nresponsible for oncogenic transformation and genomic instability\nhave yet to be identified. Because the zebrafish affords the unique\nopportunity to perform forward genetic screens, it should be\npossible to dissect the pathways that regulate Myc-induced disease\nin a genetically tractable vertebrate. The proven feasibility of\nCrelox-mediated strategies in the zebrafish will aid the develop-\nment of models of leukemia, lymphoma, and other cancers, which\nwill provide opportunities for both genetic and chemical modifier\nscreens designed to identify suppressors and enhancers in carcino-\ngenesis. For example, a dominant modifier genetic screen could be\nconducted by breeding N-ethyl-N-nitrosourea-mutagenized fish\nwith homozygous rag2-loxP-dsRED2-loxP-EGFP-mMyc fish and\nthen analyzing the progeny for the time of leukemia onset after\nCRE recombination and expression of the Myc transgene. Muta-\ntions that modify the time of leukemia onset could be either\nenhancers that result in more rapid onset of leukemia due to the\ninactivation of one allele of a tumor suppressor gene or suppressors\nthat delay or prevent the onset of Myc-induced transformation.\nFinally, use of the Crelox technology in the zebrafish will provide\ntools for assessing cell lineage commitment and plasticity of stem\ncells and generating conditional knockouts in developing embryos.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nWe thank Y. Yang, N. Campisi, and E. Ronan for expert technical\nassistance; L. I. Zon, B. Paw, and B. E. H. Langenau for critical review\nof the manuscript; and J. Vinokur, G. Kourkoulis, and W. Saganic for fish\ncare and husbandry. This work was supported by National Institutes of\nHealth Grants CA-68484 (to A.T.L.) and CA-06516 (to J.L.K.). D.M.L.\nwas a National Science Foundation Predoctoral Fellow and is now the\nEdmond J. Safra Foundation–Irvington Institute Fellow.\n**BLOCK**fs== 6.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n1. Ferrando, A. A., Neuberg, D. S., Staunton, J., Loh, M. L., Huard, C., Raimondi, S. C., Behm,\nF. G., Pui, C. H., Downing, J. R., Gilliland, D. G., et al. (2002) Cancer Cell 1, 75–87.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n2. Look, A. T. (1997) Science 278, 1059–1064.\n3. Ferrando, A. A. & Look, A. T. (2000) Semin. Hematol. 37, 381–395.\n4. Bernard, O. A., Busson-LeConiat, M., Ballerini, P., Mauchauffe, M., Della Valle, V., Monni,\nR., Nguyen Khac, F., Mercher, T., Penard-Lacronique, V., Pasturaud, P., et al. (2001)\nLeukemia 15, 1495–1504.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n5. Wang, J., Jani-Sait, S. N., Escalon, E. A., Carroll, A. J., de Jong, P. J., Kirsch, I. R. & Aplan,\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\nP. D. (2000) Proc. Natl. Acad. Sci. USA 97, 3497–3502.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n6. Kees, U. R., Heerema, N. A., Kumar, R., Watt, P. M., Baker, D. L., La, M. K., Uckun, F. M.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.1**r== 0.7**\n& Sather, H. N. (2003) Leukemia 17, 887–893.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n7. Watt, P. M., Kumar, R. & Kees, U. R. (2000) Genes Chromosomes Cancer 29, 371–377.\n8. Ferrando, A. A., Herblot, S., Palomero, T., Hansen, M., Hoang, T., Fox, E. A. & Look, A. T.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.1**r== 0.8**\n(2004) Blood 103, 1909–1911.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n9. Ferrando, A. A. & Look, A. T. (2003) Semin. Hematol. 40, 274–280.\n10. Yeoh, E. J., Ross, M. E., Shurtleff, S. A., Williams, W. K., Patel, D., Mahfouz, R., Behm,\nF. G., Raimondi, S. C., Relling, M. V., Patel, A., et al. (2002) Cancer Cell 1, 133–143.\n11. Ferrando, A. A., Neuberg, D. S., Dodge, R. K., Paietta, E., Larson, R. A., Wiernik, P. H.,\nRowe, J. M., Caligiuri, M. A., Bloomfield, C. D. & Look, A. T. (2004) Lancet 363,\n535–536.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n12. Ballerini, P., Blaise, A., Busson-Le Coniat, M., Su, X. Y., Zucman-Rossi, J., Adam, M.,\nvan den Akker, J., Perot, C., Pellegrino, B., Landman-Parker, J., et al. (2002) Blood 100,\n991–997.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n13. Weng, A. P., Ferrando, A. A., Lee, W., Morris, J. P. T., Silverman, L. B., Sanchez-Irizarry,\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nC., Blacklow, S. C., Look, A. T. & Aster, J. C. (2004) Science 306, 269–271.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n14. Okuda, T., Shurtleff, S. A., Valentine, M. B., Raimondi, S. C., Head, D. R., Behm, F.,\nCurcio-Brint, A. M., Liu, Q., Pui, C. H. & Sherr, C. J. (1995) Blood 85, 2321–2330.\n15. Langenau, D. M., Traver, D., Ferrando, A. A., Kutok, J. L., Aster, J. C., Kanki, J. P., Lin,\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nS., Prochownik, E., Trede, N. S., Zon, L. I. & Look, A. T. (2003) Science 299, 887–890.\n**BLOCK**fs== 6.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n20. Haffter, P., Granato, M., Brand, M., Mullins, M. C., Hammerschmidt, M., Kane, D. A.,\nOdenthal, J., van Eeden, F. J., Jiang, Y. J., Heisenberg, C. P., et al. (1996) Development\n(Cambridge, U.K.) 123, 1–36.\n**BLOCK**fs== 6.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n21. Golling, G., Amsterdam, A., Sun, Z., Antonelli, M., Maldonado, E., Chen, W., Burgess, S.,\n**BLOCK**fs== 6.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nHaldi, M., Artzt, K., Farrington, S., et al. (2002) Nat. Genet. 31, 135–140.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n22. Langenau, D. M., Ferrando, A. A., Traver, D., Kutok, J. L., Hezel, J. P., Kanki, J. P., Zon,\nL. I., Look, A. T. & Trede, N. S. (2004) Proc. Natl. Acad. Sci. USA 101, 7369–7374.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n23. Jessen, J. R., Jessen, T. N., Vogel, S. S. & Lin, S. (2001) Genesis 29, 156–162.\n24. Haire, R. N., Rast, J. P., Litman, R. T. & Litman, G. W. (2000) Immunogenetics 51, 915–923.\n25. Berghmans, S., Murphey, R. D., Wienholds, E., Neuberg, D., Kutok, J. L., Fletcher, C. D.,\nMorris, J. P., Liu, T. X., Schulte-Merker, S., Kanki, J. P., et al. (2005) Proc. Natl. Acad. Sci.\nUSA 102, 407–412.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n26. Langenau, D. M., Palomero, T., Kanki, J. P., Ferrando, A. A., Zhou, Y., Zon, L. I. & Look,\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nA. T. (2002) Mech. Dev. 117, 243–248.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n27. Eischen, C. M., Weber, J. D., Roussel, M. F., Sherr, C. J. & Cleveland, J. L. (1999) Genes\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nDev. 13, 2658–2669.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n28. Yunis, J. J., Frizzera, G., Oken, M. M., McKenna, J., Theologides, A. & Arnesen, M. (1987)\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.5**r== 0.3**\nN. Engl. J. Med. 316, 79–84.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n29. Diccianni, M. B., Yu, J., Hsiao, M., Mukherjee, S., Shao, L. E. & Yu, A. L. (1994) Blood\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n84, 3105–3112.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n30. Jonveaux, P. & Berger, R. (1991) Leukemia 5, 839–840.\n31. Olivier, M., Eeles, R., Hollstein, M., Khan, M. A., Harris, C. C. & Hainaut, P. (2002) Hum.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nMutat. 19, 607–614.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n32. Beroud, C. & Soussi, T. (2003) Hum. Mutat. 21, 176–181.\n33. Gu, H., Marth, J. D., Orban, P. C., Mossmann, H. & Rajewsky, K. (1994) Science 265,\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\n103–106.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n34. Kuhn, R., Schwenk, F., Aguet, M. & Rajewsky, K. (1995) Science 269, 1427–1429.\n35. Werdien, D., Peiler, G. & Ryffel, G. U. (2001) Nucleic Acids Res. 29, E53-3.\n36. Ryffel, G. U., Werdien, D., Turan, G., Gerhards, A., Goosses, S. & Senkel, S. (2003) Nucleic\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n16. Peterson, R. T., Link, B. A., Dowling, J. E. & Schreiber, S. L. (2000) Proc. Natl. Acad. Sci.\n**BLOCK**fs== 6.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nAcids Res. 31, e44.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.8**l== 0.1**r== 0.8**\nUSA 97, 12965–12969.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n17. Peterson, R. T., Mably, J. D., Chen, J. N. & Fishman, M. C. (2001) Curr. Biol. 11, 1481–1491.\n18. Peterson, R. T., Shaw, S. Y., Peterson, T. A., Milan, D. J., Zhong, T. P., Schreiber, S. L.,\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nMacRae, C. A. & Fishman, M. C. (2004) Nat. Biotechnol. 22, 595–599.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n19. Driever, W., Solnica-Krezel, L., Schier, A. F., Neuhauss, S. C., Malicki, J., Stemple, D. L.,\nStainier, D. Y., Zwartkruis, F., Abdelilah, S., Rangini, Z., et al. (1996) Development\n(Cambridge, U.K.) 123, 37–46.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n37. Oettinger, M. A., Schatz, D. G., Gorka, C. & Baltimore, D. (1990) Science 248, 1517–1523.\n38. Schatz, D. G., Oettinger, M. A. & Baltimore, D. (1989) Cell 59, 1035–1048.\n39. Gilley, J. & Fried, M. (2001) Oncogene 20, 7447–7452.\n40. Zindy, F., Eischen, C. M., Randle, D. H., Kamijo, T., Cleveland, J. L., Sherr, C. J. & Roussel,\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nM. F. (1998) Genes Dev. 12, 2424–2433.\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n41. Langenau, D. M., Jette, C., Berghmans, S., Palomero, T., Kanki, J. P., Kutok, J. L. & Look,\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nA. T. (2005) Blood 105, 3278–3285.\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.6**r== 0.1**\nPNAS  April 26, 2005  vol. 102  no. 17  6073",
         "extracted",
         "Suppression of apoptosis by bcl-2 overexpression in lymphoid cells of transgenic zebrafish.;tp53 mutant zebrafish develop malignant peripheral nerve sheath tumors.;Activating Mutations of NOTCH1 in Human T Cell Acute Lymphoblastic Leukemia;In vivo tracking of T cell development, ablation, and engraftment in transgenic zebrafish.;Chemical suppression of a genetic mutation in a zebrafish model of aortic coarctation;Biallelic transcriptional activation of oncogenic transcription factors in T-cell acute lymphoblastic leukemia.;Prognostic importance of TLX1 (HOX11) oncogene expression in adults with T-cell acute lymphoblastic leukaemia;Gene expression profiling in T-cell acute lymphoblastic leukemia.;Expression of HOX11 in childhood T-lineage acute lymphoblastic leukaemia can occur in the absence of cytogenetic aberration at 10q24: a study from the Children's Cancer Group (CCG);Tagging muscle cell lineages in development and tail regeneration using Cre recombinase in transgenic Xenopus.;The UMD‐p53 database: New mutations and analysis tools;Myc-Induced T Cell Leukemia in Transgenic Zebrafish;Molecular cloning and developmental expression of Tlx (Hox11) genes in zebrafish (Danio rerio);HOX11L2 expression defines a clinical subtype of pediatric T-ALL associated with poor prognosis.;The IARC TP53 database: New online mutation analysis and recommendations to users;Insertional mutagenesis in zebrafish rapidly identifies genes essential for early vertebrate development;Classification, subtype discovery, and prediction of outcome in pediatric acute lymphoblastic leukemia by gene expression profiling.;Gene expression signatures define novel oncogenic pathways in T cell acute lymphoblastic leukemia.;One INK4 gene and no ARF at the Fugu equivalent of the human INK4A/ARF/INK4B tumour suppressor locus;Convergence of distinct pathways to heart patterning revealed by the small molecule concentramide and the mutation heart-and-soul;A new recurrent and specific cryptic translocation, t(5,14)(q35,q32), is associated with expression of the Hox11L2 gene in T acute lymphoblastic leukemia;FLP and Cre recombinase function in Xenopus embryos.;Concurrent expression of recombination activating genes 1 and 2 in zebrafish olfactory sensory neurons;Promoter demethylation accompanies reactivation of the HOX11 proto‐oncogene in leukemia;Small molecule developmental screens reveal the logic and timing of vertebrate development.;Clinical implications of recurring chromosomal and associated molecular abnormalities in acute lymphoblastic leukemia.;Characterization of three isotypes of immunoglobulin light chains and T-cell antigen receptor α in zebrafish;The t(14,21)(q11.2,q22) chromosomal translocation associated with T-cell acute lymphoblastic leukemia activates the BHLHB1 gene.;Disruption of the ARF-Mdm2-p53 tumor suppressor pathway in Myc-induced lymphomagenesis.;Myc signaling via the ARF tumor suppressor regulates p53-dependent apoptosis and immortalization.;Oncogenic transcription factors in the human acute leukemias.;The identification of genes with unique and essential functions in the development of the zebrafish, Danio rerio.;A genetic screen for mutations affecting embryogenesis in zebrafish.;Inducible gene targeting in mice;Frequent deletion of p16INK4a/MTS1 and p15INK4b/MTS2 in pediatric acute lymphoblastic leukemia.;Clinical significance of p53 mutations in relapsed T-cell acute lymphoblastic leukemia.;Deletion of a DNA polymerase beta gene segment in T cells using cell type-specific gene targeting.;Infrequent mutations in the P53 gene in primary human T-cell acute lymphoblastic leukemia.;RAG-1 and RAG-2, adjacent genes that synergistically activate V(D)J recombination.;The V(D)J recombination activating gene, RAG-1;Multiple recurrent genomic defects in follicular lymphoma. A possible model for cancer.",
         "Cre/lox-regulated transgenic zebrafish model with conditional myc-induced T cell acute lymphoblastic leukemia",
         "None"
        ],
        [
         "46",
         "003800d8d89dfba1f08640cd3aa49c844956a22f",
         "Neurexins constitute a large family of highly variable cell-surface molecules that may function in synaptic transmission and/or synapse formation. Each of the three known neurexin genes encodes two major neurexin variants, α- and β-neurexins, that are composed of distinct extracellular domains linked to identical intracellular sequences. Deletions of one, two, or all three α-neurexins in mice recently demonstrated their essential role at synapses. In multiple α-neurexin knock-outs, neurotransmitter release from excitatory and inhibitory synapses was severely reduced, primarily probably because voltage-dependent Ca2+ channels were impaired. It remained unclear, however, which neurexin variants actually influence exocytosis and Ca2+ channels, which domain of neurexins is required for this function, and which Ca2+-channel subtypes are regulated. Here, we show by electrophysiological recordings that transgenic neurexin 1α rescues the release and Ca2+-current phenotypes, whereas transgenic neurexin 1β has no effect, indicating the importance of the extracellular sequences for the function of neurexins. Because neurexin 1α rescued the knock-out phenotype independent of the α-neurexin gene deleted, these data are consistent with a redundant function among different α-neurexins. In both knock-out and transgenically rescued mice, α-neurexins selectively affected the component of neurotransmitter release that depended on activation of N- and P/Q-type Ca2+ channels, but left L-type Ca2+ channels unscathed. Our findings indicate that α-neurexins represent organizer molecules in neurotransmission that regulate N- and P/Q-type Ca2+ channels, constituting an essential role at synapses that critically involves the extracellular domains of neurexins.",
         "Weiqi Zhang,A. Rohlmann,V. Sargsyan,G. Aramuni,R. Hammer,T. Südhof,M. Missler",
         "https://www.jneurosci.org/content/jneuro/25/17/4330.full.pdf",
         "\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.6**\n4330 • The Journal of Neuroscience, April 27, 2005 • 25(17):4330 – 4342\n**BLOCK**fs== 11.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nCellular/Molecular\n**BLOCK**fs== 22.0**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nExtracellular Domains of -Neurexins Participate in\nRegulating Synaptic Transmission by Selectively Affecting N-\nand P/Q-Type Ca2 Channels\n**BLOCK**fs== 9.0**b== 0.6**t== 0.3**l== 0.1**r== 0.1**\nWeiqi Zhang,1* Astrid Rohlmann,1* Vardanush Sargsyan,1 Gayane Aramuni,1 Robert E. Hammer,2,5\nThomas C. Su¨dhof,3,4,5 and Markus Missler1,6\n1Center for Physiology and Pathophysiology, Georg-August University, D-37073 Go¨ttingen, Germany, Departments of 2Biochemistry and 3Molecular\nGenetics, 4Center for Basic Neuroscience, and 5Howard Hughes Medical Institute, The University of Texas Southwestern Medical Center, Dallas, Texas\n75390, and 6Department of Genetics and Molecular Neurobiology, Otto-von-Guericke-University, D-39106 Magdeburg, Germany\n**BLOCK**fs== 10.0**b== 0.4**t== 0.4**l== 0.1**r== 0.1**\nNeurexins constitute a large family of highly variable cell-surface molecules that may function in synaptic transmission and/or synapse\nformation. Each of the three known neurexin genes encodes two major neurexin variants, - and -neurexins, that are composed of\ndistinct extracellular domains linked to identical intracellular sequences. Deletions of one, two, or all three -neurexins in mice recently\ndemonstrated their essential role at synapses. In multiple -neurexin knock-outs, neurotransmitter release from excitatory and inhibi-\ntory synapses was severely reduced, primarily probably because voltage-dependent Ca 2 channels were impaired. It remained unclear,\nhowever, which neurexin variants actually influence exocytosis and Ca 2 channels, which domain of neurexins is required for this\nfunction, and which Ca 2-channel subtypes are regulated. Here, we show by electrophysiological recordings that transgenic neurexin 1\nrescues the release and Ca 2-current phenotypes, whereas transgenic neurexin 1 has no effect, indicating the importance of the\nextracellular sequences for the function of neurexins. Because neurexin 1 rescued the knock-out phenotype independent of the\n-neurexin gene deleted, these data are consistent with a redundant function among different -neurexins. In both knock-out and\ntransgenically rescued mice, -neurexins selectively affected the component of neurotransmitter release that depended on activation of\nN- and P/Q-type Ca 2 channels, but left L-type Ca 2 channels unscathed. Our findings indicate that -neurexins represent organizer\nmolecules in neurotransmission that regulate N- and P/Q-type Ca 2 channels, constituting an essential role at synapses that critically\ninvolves the extracellular domains of neurexins.\n**BLOCK**fs== 10.0**b== 0.4**t== 0.6**l== 0.1**r== 0.3**\nKey words: transgenic mouse; synapse; transmission; calcium channels; cell adhesion; brainstem\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nIntroduction\nNeurotransmission requires formation and differentiation of\npresynaptic and postsynaptic compartments, including the as-\nsembly of an extensive molecular machinery for exocytosis and\nsignal transduction. The number, ultrastructure, and physiolog-\nical properties of synapses vary considerably between individual\nneurons and brain regions and can change dramatically during\nsynaptogenesis and synaptic plasticity. Neurexins were proposed\nas candidate molecules to mediate some of the diversity (Missler\nand Sudhof, 1998) because (1) their structure indicates a\nreceptor-like transmembrane protein (Ushkaryov et al., 1992,\n**BLOCK**fs== 7.0**b== 0.1**t== 0.8**l== 0.1**r== 0.7**\nReceived Sept. 8, 2004; revised March 7, 2005; accepted March 16, 2005.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nThis work was supported by the Deutsche Forschungsgemeinschaft (Sonderforschungsbereich 406 and Center\nfor Molecular Physiology of the Brain grants to M.M. and W.Z.) and by the National Institutes of Health (T.C.S.). We\nthank R. W. Tsien and R. Schneggenburger for discussion, M. Goedert (Medical Research Council, Cambridge, UK) and\nH. Kramer (University of Texas Southwestern Medical Center) for plasmids, and S. Gerke for excellent technical\nassistance.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n*W.Z. and A.R. contributed equally to this work.\nCorrespondence should be addressed to Markus Missler, Center for Physiology and Pathophysiology, Georg-\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nAugust University, Humboldtallee 23, D-37073 Go¨ttingen, Germany. E-mail: mmissle1@gwdg.de.\n**BLOCK**fs== 7.0**b== 0.0**t== 0.9**l== 0.1**r== 0.8**\nDOI:10.1523/JNEUROSCI.0497-05.2005\nCopyright © 2005 Society for Neuroscience\n**BLOCK**fs== 7.0**b== 0.0**t== 1.0**l== 0.2**r== 0.6**\n0270-6474/05/254330-13$15.00/0\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.5**r== 0.1**\n1994; Ushkaryov and Sudhof, 1993), (2) they bind to postsynap-\ntic cell-surface proteins (Ichtchenko et al., 1995; Sugita et al.,\n2001), (3) they are receptors for -latrotoxin, which induces neu-\nrotransmitter release (Geppert et al., 1998; Sugita et al., 1999),\nand (4) they are polymorphic in their extracellular sequences\nbecause of significant alternative splicing (Ichtchenko et al., 1995;\nMissler et al., 1998; Sugita et al., 1999, 2001). Mammalian neur-\nexins are encoded by three genes that contain independent pro-\nmoters for - and -neurexins (Rowen et al., 2002; Tabuchi and\nSudhof, 2002). -Neurexins contain substantially longer extra-\ncellular sequences than the much shorter -neurexins, but share\nwith -neurexins the same C terminus (Missler and Sudhof,\n1998). Consequently, available biochemical evidence has re-\nvealed overlapping C-terminal (intracellular) binding partners\n(Hata et al., 1996; Butz et al., 1998; Biederer and Sudhof, 2000),\nbut distinct extracellular interactions for - and -neurexins\n(Ichtchenko et al., 1995; Missler et al., 1998; Sugita et al., 2001).\nTo determine the role of neurexins, we previously produced\nknock-out (KO) mice that lack one, two, or all three -neurexins\n(Missler et al., 2003). Mutants with different combinations of\nknock-out alleles mostly died prematurely as a result of respira-\ntory problems. Analyses of newborn mice deficient for multiple\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nJ. Neurosci., April 27, 2005 • 25(17):4330 – 4342 • 4331\n**BLOCK**fs== 9.5**b== 0.5**t== 0.1**l== 0.1**r== 0.5**\n-neurexins revealed almost no changes in brain architecture or\nsynapse structure but uncovered a severe reduction in neuro-\ntransmitter release. As the presumptive cause of the inefficient\nexocytosis in mutant mice, we suggested that voltage-dependent\nCa 2 channels (VDCCs) were impaired because the response\npattern of synaptic transmission to specific Ca 2-channel block-\ners was altered, and whole-cell Ca 2 currents were reduced\n(Missler et al., 2003). Although no knock-out mice are available\nyet for -neurexins, their putative function was addressed re-\ncently by cell culture assays: overexpression of neurexin 1 in-\nduced differentiation of postsynaptic receptors (Graf et al., 2004),\nand its trans-synaptic partner neuroligin stimulated de novo for-\nmation and differentiation of presynaptic terminals in vitro\n(Scheiffele et al., 2000; Dean et al., 2003; Sara et al., 2005). These\nstudies established a prominent role for neurexins at synapses but\nalso raised important questions: is the phenotype of reduced neu-\nrotransmitter release in knock-out mice specific for the deletion\nof -neurexins? Are there functional differences between various\nneurexin genes? Is the highly variable extracellular domain of neur-\nexins required for regulating VDCCs? Why were the -neurexins\nthat are still abundantly expressed in -neurexin KO mice appar-\nently unable to compensate for the loss of -neurexins? Which sub-\ntypes of high voltage-activated Ca 2 channels are impaired in\n-neurexin KO mice?\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nHere, we show that (1) a single transgenically expressed neur-\nexin 1splice variant rescued the impaired neurotransmission at\nexcitatory and inhibitory synapses in all deletions of -neurexins\ntested; (2) comparable transgenic expression of -neurexins had\nno effect, indicating that extracellular sequences of -neurexins\nare important for regulating VDCCs; and (3) -neurexins selec-\ntively altered N- and P/Q-type Ca 2-channel function but left\nL-type Ca 2 channels unchanged.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nMaterials and Methods\nGeneration and breeding of transgenic mice. Transgenic vectors for neur-\nexin 1 (Nrx::HRP)\n(Missler et al., 2003) and neurexin 1\n(Nrx::HRP) (generated de novo for this study) were constructed using\ncDNA plasmids pCMVL2 and pCMVL13, respectively. Plasmid pBS-\nHRP served as the source of the horseradish peroxidase (a gift from Dr.\nH. Kraemer, University of Texas Southwestern Medical Center). Two\ninternal restriction sites (the 5 NotI site in pCMVL2 and the PvuI site in\npBS-HRP) had to be destroyed by silent mutagenesis. A 980 bp HRP\nfragment was cloned into the 3 NotI site of the modified pCMVL2 and of\npCMVL13 [resulting junctional amino acid sequence: neurexin . . .\nGRQPAMQLT.HRP.NSNSGQPRGR . . . neurexin (italicized residues\nrepresent the HRP cDNA)]. The chimeric neurexin–HRP cDNAs were\nblunt-end cloned into the unique XhoI site of the mouse Thy1.2 expres-\nsion vector pEX21 (a gift from Dr. M. Goedert, Medical Research Coun-\ncil, Cambridge, UK) to generate the transgenic vectors. The Thy1.2 pro-\nmoter was chosen because of its developmentally early widespread\nneuronal expression (Andra et al., 1996). Transgenic mice were pro-\nduced by pronucleus injection, and founder mice and progeny were\ngenotyped using dot/Southern blotting with EagI/EagI-isolated 32P-\nlabeled HRP cDNA probes. Because our rescue experiments required\ncrossing mice from FVB strains (transgenics) into the mixed 129Sv \nC57BL/6 background (-neurexin KOs), each set of experiments was\ndone on littermate mice with and without additional transgene. Geno-\ntyping a large number of newborn offspring [n  265 pups killed at\npostnatal day 1 (P1)] confirmed a normal mendelian ratio for all combi-\nnations of mutated neurexin alleles (data not shown). In addition, 52%\nof pups (n  96 of 183) inherited the Nrx::HRP transgene, and 51%\ninherited the Nrx::HRP transgene (n  42 of 82). All analyses were\nperformed with hemizygous transgenic mice on wild-type or -neurexin\nKO backgrounds, as described below.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nHRP antibodies and biochemical procedures. Expression of neurexin–\nHRP transgenic fusion proteins was tested in brains of hemizygous mice\n**BLOCK**fs== 8.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nusing antibodies against HRP epitopes. Most commercial antibodies\nwere generated against the plant HRP and cross-react with endogenous\nmouse brain proteins (data not shown). Therefore, we produced a re-\ncombinant HRP– glutathione S-transferase (GST) fusion protein (from\npET28-HRP) in BL21 cells to affinity purify a commercially available\npolyclonal serum (Nordic) using nitrocellulose membrane stripes con-\ntaining HRP separated by SDS-PAGE (Olmsted, 1981). Immunoprecipi-\ntations and GST pull-downs were performed with P2 brain membrane\nproteins solubilized in 1% 3-[(3-cholamidopropyl)dimethylammonio]-\n1-propanesulfonate with affinity-purified HRP antibodies (anti-HRPaff)\nor recombinant calcium/calmodulin-dependent serine kinase (CASK)\nimmobilized on protein A and glutathione agarose beads, respectively\n(Hata et al., 1996). To measure the enzymatic activity of transgenic neur-\nexin–HRP fusion proteins, mouse brain membrane fractions (nonsolu-\nbilized P2 fractions) were incubated for 15 min with diaminobenzidine (1\nmg/ml) and H2O2 (0.04%), and the absorbance at 595 nm was deter-\nmined with reactions from nontransgenic mice serving as controls.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.3**l== 0.5**r== 0.1**\nMorphological studies. To analyze the distribution of transgenic ex-\npression of both Nrx::HRP and Nrx::HRP proteins in the brainstem of\nnewborn mice, brains were freshly dissected in PBS, and brainstem prep-\narations were immersion fixed in 2% paraformaldehyde in 0.1 M phos-\nphate buffer for 2 h, postfixed for 2 h in the same solution, and cryopro-\ntected in 25% sucrose. For light microscopy of transgenic mice, 12 m\ncryosections were thaw mounted on poly-L-lysine-coated glass slides and\nlabeled overnight with anti-HRPaff antibodies in blocking buffer (0.1%\nTriton X-100, 50% normal goat serum, and PBS). Secondary goat anti-\nrabbit antibody and PAP antibodies (rabbit PAP) were incubated with\nsections in blocking solution (both antibodies were from Sternberger\nMonoclonals, Lutherville, MD) and visualized with diaminobenzidine\n(0.05% w/v) plus H2O2 (0.005% v/v) and heavy metal enhancement\n(NiCl2, 0.15% w/v). To compare tissues from different genotypes, sec-\ntions from several mice were usually processed in parallel. Overview\npanels of newborn brainstem (see Fig. 2) are montages from up to 15\nindividual pictures captured via an AxioCam HR digital camera system\nmounted on an Axioscope 2 microscope (Zeiss, Oberkochen, Germany)\nand composed using Adobe Photoshop 6.0 software (Adobe Systems, San\nJose, CA).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.6**l== 0.5**r== 0.1**\nElectrophysiological recordings. All electrophysiological analyses were\nperformed on brainstem neurons of mice whose genotype was unknown\nto the experimenter, essentially as described previously (Zhang et al.,\n1999; Missler et al., 2003). Acute slices containing the pre-Botzinger\ncomplex (PBC) and hypoglossal motor nucleus from newborn (P1) lit-\ntermate mice were used for whole-cell recordings. The bath solution in all\nexperiments consisted of the following (in mM): 118 NaCl, 3 KCl, 1.5\nCaCl2, 1 MgCl2, 25 NaHCO3, 1 NaH2PO4, and 20 glucose, pH 7.4 (aer-\nated with 95% O2 and 5% CO2 and kept at 28 –30°C). Evoked EPSCs\n(eEPSCs) and evoked IPSCs (eIPSCs) were recorded from hypoglossal\nmotor neurons in the presence of 1 M strychnine and 1 M bicuculline\nor from PBC neurons in the presence of 10 M CNQX, respectively.\nEPSCs were evoked by 0.1 Hz field stimulations of axons of interneurons\nclose to the PBC using a custom-built bipolar platinum electrode. An\nisolation unit, IsoFlex (A.M.P.I., Jerusalem, Israel), with a custom-built\npower supply, was used to apply currents of supramaximal stimulation\nstrength (1 mA actual current near the slice). During the long time\ncourse of investigations represented in this study (extending over 3\nyears), different stimulation setups and types of electrodes had to be\nused, preventing the direct comparison of absolute EPSC amplitudes\nbetween the Nrx::HRP tg/ and Nrx::HRP tg/ experiments. This,\nhowever, does not affect our conclusions regarding their respective res-\ncue potential. The pipette solution for eEPSC and eIPSC measurements\ncontained the following (in mM): 140 K-gluconate (eEPSCs) or 140 KCl\n(eIPSCs), 1 CaCl2, 10 EGTA, 2 MgCl2, 4 Na3ATP, 0.5 Na3GTP, and 10\nHEPES, pH 7.3. Peak amplitudes were averaged from 25 consecutive\nresponses. To monitor changes in input resistance, current responses to a\n10 mV voltage step (20 ms) from a holding potential of 70 mV were\nrecorded before every fifth stimulus. In all experiments, the distance\nbetween the stimulation and recording electrodes was similar between\nslices of different genotypes, and we could not find any significant differ-\nences in latency of evoked responses between the genotypes. The supra-\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\n4332 • J. Neurosci., April 27, 2005 • 25(17):4330 – 4342\n**BLOCK**fs== 8.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nmaximal stimulation protocol using constant current amplitudes (see\nabove) differs slightly from the one applied in our previous paper for the\nKO analysis (Missler et al., 2003), in which we adjusted the stimulation\nstrength according to the failure rate obtained. The current approach was\nchosen to emphasize differences between mice that are homozygous for\nthe same -neurexin mutation and that either lack or contain the neur-\nexin transgene. Using the supramaximal stimulation strength in all cases\nproduced a higher resolution of rescue effect even in the case of multiple\n-neurexin KOs. However, using supramaximal stimulation strength\nobscured comparison of absolute responses between the different geno-\ntypes in these experiments, explaining why the differences between KO\ncombinations are less pronounced than in our previous study.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.3**l== 0.1**r== 0.5**\nMiniature postsynaptic current analysis. Spontaneous miniature IPSCs\n(mIPSCs) and miniature EPSCs (mEPSCs) were recorded from neurons\nof the PBC and hypoglossal nucleus at a Cl  reversal potential of 0 mV\nin 10 M CNQX or 1 M strychnine and 1 M bicuculline, respectively;\nsignals with amplitudes of at least 2.5 times above background noise were\nselected, and statistical significance was tested in each experiment. Based\non our selection criteria, miniature postsynaptic current (mini) events\nclose to the detection level could be missed; however, the noise levels and\namplitudes were similar between genotypes. Voltage-activated Ca 2\ncurrents were measured from neurons of the PBC with electrodes con-\ntaining (in mM) 110 CsCl2, 30 TEA-Cl, 1 CaCl2, 10 EGTA, 2 MgCl2, 4\nNa3ATP, 0.5 Na3GTP, and 10 HEPES, pH 7.3, and with 0.5 M tetrodo-\ntoxin in the bath solution. Serial and membrane resistances were esti-\nmated from current transients induced by 20 mV hyperpolarization volt-\nage commands from a holding potential of 70 mV. The serial resistance\nwas compensated by 80%, and patches with a serial resistance of 20\nM, a membrane resistance of 0.8 G, or leak currents of 150 pA\nwere excluded. The membrane currents were filtered by a four-pole\nBessel filter at a corner frequency of 2 kHz and digitized at a sampling\nrate of 5 kHz using the DigiData 1200B interface (Axon Instruments,\nUnion City, CA). Currents were recorded in response to voltage step\nchanges from the 70 mV holding potential to test potentials be-\ntween 80 and 30 mV and quantified as peak currents in response\nto voltage steps from 70 to 0 mV. Ca 2-current measurements were\ncorrected using the P/4 protocol that subtracts leak currents mea-\nsured during four leak-subtraction prepulses applied immediately\nbefore each voltage step. Blocking experiments using specific VDCC\ninhibitors were always performed in the same sequential order, ap-\nplying 10 M nifedipine for L-type channel contributions, 0.5 M\n-conotoxin MVIIA for N-type channel contributions, 0.2 M aga-\ntoxin IVA for P/Q-type channel contributions, and 100 M Cd 2 to\nprobe for residual Ca 2 channel activities. Data acquisition and anal-\nysis were performed using commercially available software [pClamp\n6.0 and AxoGraph 4.6 (Axon Instruments); Prism 3 software (Graph-\nPad Software, San Diego, CA)].\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nMiscellaneous. Primary antibodies to neurexins (A473), neuroligin\n(4C12), and synaptotagmin 1 (W855) were characterized previously\n(Ushkaryov et al., 1992; Butz et al., 1998; Song et al., 1999) or obtained\ncommercially: Sy38, anti-synaptophysin (DakoCytomation, Hamburg,\nGermany), anti-P/Q Ca 2-channel\nsubunit and anti-NMDA R1\n(Chemicon, Temecula, CA), and heat shock protein 70 (Affinity BioRe-\nagents, Golden, CO). Statistical significance was tested with a two-tailed\nStudent’s t test in Excel (Microsoft, Redmond, WA) spreadsheets or with\nInStat (GraphPad Software) and, in case of mini events, with the Kol-\nmogorov–Smirnov test integrated in MiniAnalysis (Synaptosoft, Deca-\ntur, GA).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nResults\nTransgenic mice expressing neurexins that differ solely in\ntheir extracellular sequences\nTo elucidate the molecular determinants that link neurexins to\nCa 2 channels, we used a comparative transgenic rescue analysis.\nThis analysis allows us to explore the role of structurally distinct\n- and -neurexins in synaptic transmission. In our initial char-\nacterization of -neurexin KO mice, we showed that the decrease\nin whole-cell Ca 2 currents caused by deletion of -neurexins\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\ncan be partially reversed by transgenic neurexin 1expressed as a\nfusion protein with HRP under control of the Thy1 promoter\n(Missler et al., 2003). This finding raised questions as to whether\nthe reduced neurotransmitter release was also reversed by the\ntransgene and whether the partial rescue was a specific conse-\nquence of transgenic neurexin 1 expression (or could be ex-\ntended to -neurexins). To address these questions, here we have\ncompared the physiological effects mediated by two sets of trans-\ngenic mice that express two different neurexins (Fig. 1A): (1) the\noriginal neurexin 1 transgene (Nrx::HRP) (Missler et al., 2003)\nand (2) a newly generated neurexin 1transgene (Nrx::HRP).\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nBoth sets of transgenic mice were produced with identical\nvectors in which an HRP cDNA was inserted at the same position\ninto neurexin 1or 1to monitor expression (Fig. 1 A). HRP was\nchosen as an internal epitope tag, because studies in Drosophila\ndemonstrated that HRP did not interfere with membrane traffic\nwhen inserted into the extracellular sequences of a cell-surface\nprotein (Sunio et al., 1999; Dubois et al., 2001). The major differ-\nence between the two sets of transgenic mice arises from varia-\ntions in neurexin 1and 1sequences, which are identical apart\nfrom their extracellular domains, the former containing a large\nextracellular sequence consisting of multiple domains and the\nlatter consisting of a shorter extracellular region (Ushkaryov et\nal., 1992; Missler and Sudhof, 1998) (Fig. 1 A). To ensure that the\ndifferences observed between the activity of these transgenes were\nnot attributable to insertion effects, we generated multiple lines\nof both transgenes, which, as described below, produced similar\ntransgene protein levels and expression patterns and thus allowed\na direct comparison of the functions of the two types of\nneurexins.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.5**l== 0.5**r== 0.1**\nHeterozygous (Nrx::HRP tg/ and Nrx::HRP tg/) and ho-\nmozygous (Nrx::HRP tg/tg and Nrx::HRP tg/tg) transgenic mice\nwere viable and fertile (data not shown), indicating that the trans-\ngenic neurexins did not strongly interfere with normal neurexin\nfunction or the role of an unidentified essential gene close to the\ninsertion site of the transgenes. To examine protein expression,\nwe affinity-purified commercially available HRP antibodies on\nimmobilized recombinant HRP and used anti-HRPaff for immu-\nnoblotting. The anti-HRPaff reacted with a single protein on\nimmunoblots from adult transgenic mice and detected no cross-\nreactive bands in nontransgenic littermates (Fig. 1 B). Compara-\nble immunoblots of -neurexin transgenic mice (data not\nshown) indicated that expression levels of\nthe transgenic\nNrx::HRP were approximately the same as those for transgenic\nNrx::HRP (for a direct comparison of their expression in new-\nborn mice, see Fig. 2). We next investigated whether the HRP-\ntagged neurexins expressed in transgenic mice were capable of\nnormal protein–protein interactions typical for endogenous\nneurexins. Pull-down experiments using a GST fusion protein\ncontaining\nlarge/zona\nthe postsynaptic density-95/Discs\noccludens-1 (PDZ) domain of CASK, a protein that binds to the\nC terminus of neurexins (Hata et al., 1996), revealed that both\ntransgenic neurexins bound CASK efficiently (Fig. 1C). This ob-\nservation suggested that endogenous (Fig. 1C, asterisks) and\ntransgenic (Fig. 1C, arrows) neurexins engage in similar intracel-\nlular binding activities, as predicted from their identical C termi-\nnus. To explore the integrity of specific extracellular interactions,\nwe performed immunoprecipitations of brain proteins from\ntransgenic mice with anti-HRPaff antibodies. These experiments\nshowed that the Nrx::HRP fusion protein is present in the brain\nin a complex with neuroligin 1, as predicted for this particular\nsplice variant (Fig. 1 D) (Ichtchenko et al., 1995). In contrast,\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nJ. Neurosci., April 27, 2005 • 25(17):4330 – 4342 • 4333\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.6**r== 0.1**\nand also (in 50% of the cases) a single copy of\nthe respective transgene.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.2**l== 0.6**r== 0.1**\nTransgenic neurexin 1enhances\nsynaptic transmission and Ca 2\ncurrents in -neurexin KO mice\nTo explore the potential of one -neurexin\nisoform to rescue the synaptic phenotype\nof multiple -neurexin KO mice, we first\nasked whether Nrx::HRP tg/ was able to\nalter the amplitude of evoked synaptic re-\nsponses. Because -neurexin KO mice ho-\nmozygous for two or three mutant alleles\ndie early postnatally, we analyzed all mu-\ntant mice immediately after birth (P1) in\nacute brainstem slices, because the syn-\naptic connections in this area mature rel-\natively early and are involved in breath-\ning control\n[normal breathing was\ndisrupted in -neurexin KO mice\n(Missler et al., 2003)]. Synaptic re-\nsponses were monitored in hypoglossal\nmotor neurons after afferent fiber stim-\nulation near the PBC by measuring eEP-\nSCs in littermate -neurexin mutant\nmice with and without the Nrx::HRP\ntransgene.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.5**l== 0.1**r== 0.4**\nFigure 1.\nTransgenic mice expressing two distinct neurexins under control of a heterologous promoter. A, Domain model of\ntransgenically introduced Nrx::HRP and Nrx::HRP proteins with an internal HRP epitope to monitor expression. The transgenic\n- and -neurexins under control of the Thy1.2 promoter differ exclusively in their extracellular sequences. Gray sheet, trans-\nmembrane domain (Missler and Sudhof, 1998). B, Comparative immunoblot analysis of several transgenic (Tg) mouse lines\nexpressing Nrx::HRP. Brain membrane proteins (20 g/lane) were probed with anti-HRPaff. Loading control blots were probed\nwith antibodies against the NMDA receptor subunit R1 (anti-NR1). C, Bound material from pull-down experiments with the\nprotein CASK was analyzed by immunoblotting with affinity-purified HRP antibodies (top) or pan-neurexin antibodies (middle).\nBoth transgenically expressed (arrows) and endogenous (asterisks) neurexins interact with their intracellular ligand CASK.\nEquivalent amounts of recombinantly expressed GST–CASK were used for the pull-down procedures (bottom; Coomassie\nstain). D, Immunoprecipitation (IP) of brain membrane proteins from transgenic mice expressing Nrx::HRP and\nNrx::HRP and from control mice (non-Tg). Transgenic neurexin–HRP fusion proteins were precipitated with anti-HRPaff\nand analyzed by immunoblotting with antibodies to neurexins (top) or neuroligins (middle; bottom shows control samples\nbefore immunoprecipitation). Only Nrx::HRP binds to neuroligin, confirming that isoform-specific interactions are intact\nin brains of transgenic animals.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nno neuroligin 1 coimmunoprecipitated with Nrx::HRP was\ndetectable.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nTo ensure a meaningful comparison of the effects of the trans-\ngenic neurexins (see below), we confirmed by immunohisto-\nchemistry that transgenic neurexin 1 and 1 are expressed in\nsimilar patterns in the brainstem of newborn mice in areas used\nfor the electrophysiological rescue analysis (Fig. 2 A–C, RVLN,\nXII). Although activity measurements revealed that the HRP\nmoiety of both transgenic proteins was enzymatically active (Fig.\n2 D), the peroxidase activity conferred by HRP could not be used\nfor these morphological studies because of the low sensitivity of\nthe technique (data not shown). For functional analyses, we bred\nmice that are homozygous for the neurexin 2 KO allele\n(Nrx2/) and heterozygous for the two other neurexin KO alleles\n(Nrx1/ and Nrx3/) and, in addition, either lack or contain\ntransgene (nontransgenic, Nrx::HRP tg/, or\na hemizygous\nNrx::HRP tg/) (Fig. 2E). Mating of this parent generation resulted\nin littermate homozygous neurexin 2KO offspring that addition-\nally carried various combinations of the neurexin 1and 3alleles\n**BLOCK**fs== 9.5**b== 0.5**t== 0.5**l== 0.7**r== 0.2**\ntransgenic\n**BLOCK**fs== 9.5**b== 0.0**t== 0.5**l== 0.5**r== 0.1**\nRepresentative EPSC traces in Figure\n3A and their quantification in Figure 3B\nshow that\nexpression of\nNrx::HRP caused a dramatic increase in\nevoked synaptic responses in -neurexin\nKO mice (Fig. 3B). In single neurexin 2\n(SKO2) and double neurexin 2/3\n(DKO2/3) or 1/2 KO (DKO1/2) mice,\nNrx::HRP doubled the mean EPSC to\n198 –223% compared with their non-\ntransgenic littermates. Higher absolute\nEPSC amplitudes in SKO mice expressing\nNrx::HRP, compared with DKO and tri-\nple KO (TKO) mice\ncontaining\nNrx::HRP, suggested that a single trans-\ngene was not sufficient to overcome dele-\ntion of multiple -neurexins (Fig. 3B).\nOnly a limited number of TKO mice could be analyzed in these\nexperiments because TKO mice are difficult to obtain by breed-\ning (the probability of obtaining a TKO plus Nrx::HRP tg/ was\n3.125%). Consistent with the increased amplitude of eEPSCs, the\nNrx::HRP tg/ also decreased the failure rate in all genotypes\n(from 19.2 to 7.2% in SKO2 mice when the Nrx::HRP transgene\nwas expressed and from 30.0 to 7.3% in DKO1/2 mice; data not\nshown). To test whether this rescue also extended to evoked re-\nsponses at inhibitory terminals, eIPSCs were recorded from PBC\nneurons after field stimulations of axons close to the PBC in the\npresence of 10 M CNQX. For these experiments, because of the\ndifficulty of breeding a sufficient number of mice with the correct\ngenotype, quantifications were restricted to littermate mice of one\n-neurexin genotype (DKO2/3) with and without Nrx::HRP tg/.\nAs for excitatory synapses, eIPSCs increased significantly when an\n-neurexin transgene was present (DKO2/3, 165.4  38.11 pA, n \n6; DKO2/3 and Nrx::HRP tg/, 347.7  68.55 pA, n  5; p  0.05).\nTo analyze whether the increase in evoked synaptic responses\nby the neurexin 1 transgene reflects a general improvement of\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\n4334 • J. Neurosci., April 27, 2005 • 25(17):4330 – 4342\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nthat\n**BLOCK**fs== 9.5**b== 0.8**t== 0.2**l== 0.2**r== 0.7**\nobserved\n**BLOCK**fs== 9.5**b== 0.0**t== 0.1**l== 0.1**r== 0.5**\nvesicle release, we next measured sponta-\nneous frequencies and amplitudes of minis\nin PBC neurons (Fig. 3C,D). Confirming\nthe results from the evoked release ex-\nperiments, we\nthe\nNrx::HRP tg/ transgene caused a signifi-\ncant enhancement of\nthe spontaneous\nneurotransmitter release in all -neurexin\nKO combinations studied. Mini frequen-\ncies at inhibitory synapses increased to\n160% in SKO2 and Nrx::HRP tg/ neu-\nto 174% in DKO2/3\nrons,\nand\nNrx::HRP tg/ neurons, to 233% in\nDKO1/2 and Nrx::HRP tg/ neurons, and\nto 274% in TKO and Nrx::HRP tg/\nneurons compared with their respective\nnontransgenic littermate controls (Fig.\n3D). Although this proportional increase\nfrequency by the Nrx::HRP\nin mini\nthe more\nstronger\ntransgene appears\n-neurexin alleles are missing, the lower\nabsolute mIPSC frequencies in rescued\nDKO and TKO versus SKO2 neurons are\nconsistent with our previous notion that a\nsingle transgene is not enough to replace\nmultiple -neurexins (see above). In con-\ntrast to the frequency, no changes in mini\namplitudes were observed in rescued neu-\nrons (Fig. 3C and data not shown), sup-\nporting the idea that neurexins primarily\nfunction presynaptically. To test whether\nthe rescue of mini release also extends to\nexcitatory synapses, we measured minis in\nhypoglossal motor neurons in the pres-\nence of 1 M strychnine and 1 M bicucul-\nline, again by restricting quantitative anal-\nysis to littermate mice of one particular\n-neurexin genotype (DKO2/3) with and without Nrx::HRP tg/.\nFrequencies of mEPSCs increased from 0.7  0.013  10 1 Hz\n(DKO2/3; n  4) to 3.9  0.24  10 1 Hz (DKO2/3 and\nNrx::HRP tg/; n  4), confirming that transgenic -neurexin po-\ntently enhances spontaneous and evoked synaptic neurotransmitter\nrelease in both excitatory and inhibitory synapses of knock-out mice.\nInvestigation of -neurexin KO mice had prompted the hy-\npothesis that the impairment in neurotransmission is essentially\na result of an impairment of Ca 2-channel function (Missler et\nal., 2003). Here, we performed whole-cell recordings of total high\nvoltage-activated Ca 2 currents in KO neurons of the PBC with\nand without the transgenically expressed Nrx::HRP. Consistent\nwith our previous results, we found that Ca 2 currents were low\nin -neurexin KO mice, but transgenic expression of -neurexin\n(Nrx::HRP tg/) robustly increased peak amplitudes (Fig. 4A). To\ncompare the current data with our previous study, we quantified the\nwhole-cell Ca 2 current as current densities (Fig. 4B), which con-\nfirmed and extended our previous observation that the Nrx::HRP\ntransgene can increase Ca 2-current densities approximately two-\nfold in all -neurexin KO combinations. Although we noted that the\nincrease in whole-cell Ca 2 currents paralleled the increase in the\nevoked and spontaneous responses (shown above), it remains an\nopen question to what extent these presumably somatodendritic\ncurrents reflect the activity of presynaptic Ca 2 channels. Even the\nbest rescue by Nrx::HRP in single and double KO mice (to 17–18\npA/pF) (Fig. 4B), however, did not completely reach the Ca 2-\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.4**r== 0.1**\nTransgenically expressed - and -neurexin can be explored for their potential to rescue the knock-out phenotype.\nFigure 2.\nA–C, Overviews of brainstem areas from newborn Nrx::HRP tg/ (A) or Nrx::HRP tg/ (B) mice or from nontransgenic (non-Tg)\nlittermates (C), assembled from sections at the indicated level (see inset). Transgenic neurexins visualized by immunohistochem-\nistry with anti-HRPaff antibodies are present in areas used for functional analyses [PBC, part of the rostroventrolateral nucleus\n(RVLN), shadowed in C; and the nucleus of the hypoglossal nerve, nucleus of the 12th cranial nerve (XII), shadowed in C]. NA,\nNucleus ambiguus; IO, inferior olive complex; Sp5I, spinal trigeminal nucleus interpolar part; 4V, fourth ventricle; ECu, external\ncuneate nucleus. Scale bar: A–C, 250 m. D, A calorimetric enzyme assay of membrane fractions from adult nontransgenic mice\nand mice expressing Nrx::HRP or Nrx::HRP was used to assess the integrity of the HRP moiety. Peroxidase activity is depicted\nbelow the corresponding Western blot lanes and expressed as extinction at OD595 (means from n  3 experiments). E, Mouse\nbreeding strategy for electrophysiological experiments. Animals homozygous for the neurexin 2and double heterozygous for\nneurexin 1and 3genes (Nrx1/2/ 3/) were bred to mice of the same KO genotype but carrying an additional\nNrx::HRP tg/ or Nrx::HRP tg/ or no transgene. All functional analyses were done on their newborn offspring (P1).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\ncurrent densities observed in corresponding neurons of genetically\nmatching wild-type mice (20 pA/pF) (Missler et al., 2003).\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nTransgenic neurexin 1has no effect on the -neurexin\nKO phenotype\nthe neurexin 1 transgene\nWe next examined whether\n(Nrx::HRP tg/) caused rescue activity similar to that seen for\nthe neurexin 1 transgene. A successful rescue effect with\nNrx::HRP tg/ would suggest that the difference in extracellular\nsequences between neurexin 1and 1is unimportant with re-\nspect to synaptic transmission and that the strong phenotype in\n-neurexin KO mice in the continued presence of -neurexins is\nattributable to the differential expression or localization of - and\n-neurexins. Conversely, if the neurexin 1 transgene is unable\nto rescue the phenotype despite similar expression levels and pat-\nterns, an essential function of the -neurexin-specific extracellu-\nlar domains in affecting presynaptic neurotransmitter release\nseems likely.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nWe found that the Nrx::HRP tg/ transgene had no effect on\nthe size of evoked synaptic responses or on the failure rate in any\nof the -neurexin KO mice tested (Fig. 5A,B and data not shown).\nIn these experiments, we compared littermates that carry the\nsame KO combinations with or without an additional\nNrx::HRP tg/ transgene and measured the amplitudes in hypo-\nglossal motor neurons after field stimulation of their inputs from\nthe PBC. However, no significant differences in mean currents\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nJ. Neurosci., April 27, 2005 • 25(17):4330 – 4342 • 4335\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nThe Nrx::HRP tg/ transgene improves Ca 2 currents in all combinations of\nFigure 4.\n-neurexin KO mice. A, Individual sample traces (left) and current–voltage relationships (right)\nof whole-cell Ca 2-current recordings from PBC neurons in DKO1/2 mice without [nontrans-\ngenic (non-Tg)] and with additional Nrx::HRP tg/ transgene. High voltage-activated Ca 2\ncurrents were evoked by 150-ms-long voltage steps (10 mV each) from a holding potential of\n70 mV to various test potentials ranging from 70 to 30 mV. B, The mean current den-\nsities show quantification of maximal depolarization steps to 0 mV (as shown in A, left). Whole-\ncell Ca 2 current increases significantly in neurons of all combinations of -neurexin KO mice\nwhen an Nrx::HRP tg/ transgene is present (filled bars). Data shown are means  SEM.\nStatistical significance is indicated above the bars. Error bars represent SEM.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.6**l== 0.5**r== 0.1**\nwere observed (Fig. 5B). To test whether Nrx::HRP had a de-\ntectable influence on spontaneous release at inhibitory synapses,\nwe measured inhibitory mini events in PBC neurons with and\nwithout a Nrx::HRP tg/ transgene. However, mini frequencies\nwere unchanged between all KO mice with or without the\nNrx::HRP tg/ transgene present (Fig. 5C). In addition, trans-\ngenically expressed -neurexin caused no alterations of postsyn-\naptic receptor function, as evidenced by unchanged mini ampli-\ntudes in all KO combinations tested (SKO, 57.3  5.98 pA, n  7;\nSKO and Nrx::HRP tg/, 55.6  11.1 pA, n  6; DKO2/3, 52.1 \n10.9 pA, n  4; DKO2/3 and Nrx::HRP tg/, 49.7  8.8 pA, n \n4; DKO1/2, 48.1  6.9, n  7; DKO1/2 and Nrx::HRP tg/,\n45.9  4.2 pA, n  6; TKO, 47.2  12.9 pA, n  3; TKO and\nNrx::HRP tg/, 48.3  11.2 pA, n  4). Finally, to exclude the\npossibility that a potential rescue effect by the extracellularly\nshort -neurexin was restricted to increasing the VDCC activity\nwithout significantly improving presynaptic function, we also\nmeasured whole-cell Ca 2 currents (Fig. 5D,E). No difference in\ntraces or current–voltage curves, however, was apparent from\nlittermate mice with or without the Nrx::HRP tg/ transgene\n(Fig. 5D), and the respective current densities were unchanged\n(Fig. 5E). The experiments with Nrx::HRP established that ex-\ntracellular sequences of -neurexins are required for their role in\nneurotransmission. It should be noted, however, that the failure\nof Nrx::HRP to improve any of the KO parameters does not\nexclude an important function of the C-terminal domains of\nneurexins but demonstrates that they are not sufficient to regu-\nlate Ca 2 channels.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nTransgenically expressed Nrx::HRP increases evoked and spontaneous neuro-\nFigure 3.\ntransmitter release in neurons of KO mice. A, Sample traces of eEPSCs, representing an average\nfrom 25 consecutive responses to field stimulations recorded under 1 M bicuculline and 1 M\nstrychnine. Recordings were obtained in hypoglossal motor neurons from newborn SKO2 and\nDKO1/2 mice without [nontransgenic (non-Tg)] and with an additional Nrx::HRP tg/ trans-\ngene. B, Corresponding mean EPSC amplitudes of multiple -neurexin KO littermate mice\n(SKO2, DKO2/3, DKO1/2, and TKO) with and without the Nrx::HRP tg/ transgene. All record-\nings were performed on newborn mice (P1) before genotyping in this and all subsequent ex-\nperiments. n.s., Not significant. C, Cumulative histogram of miniature frequency (left) and\namplitudes (right) of spontaneous inhibitory neurotransmitter release (mIPSCs) in brainstem\nneurons, recorded from double KO (DKO1/2) neurons with (filled symbols) and without (open\nsymbols) an additional Nrx::HRP tg/ transgene. D, Corresponding mean frequency of mIPSCs\nin brainstem neurons recorded in the presence of TTX and CNQX from multiple -neurexin KO\nlittermate mice (SKO2, DKO2/3, DKO1/2, and TKO) without (open bars) and with (filled bars) the\nNrx::HRP tg/ transgene. In all graphs and panels, statistical significance is indicated above\nthe bars. Data shown are means  SEM. Error bars represent SEM.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\n4336 • J. Neurosci., April 27, 2005 • 25(17):4330 – 4342\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nTransgenic - and -neurexins do not induce a general gain-of-function in wild-\nFigure 6.\ntype neurons. A, B, Sample traces (A) and averaged evoked excitatory postsynaptic responses\n(B) from wild-type neurons (carrying no neurexin deletions) without [nontransgenic (non-Tg);\nopen bar] and with (filled bar) an additional Nrx::HRP tg/ transgene, exhibiting no effect of\ntransgenically expressed -neurexin on wild-type levels of synaptic transmission. C, Represen-\ntative traces of high voltage-activated Ca 2 currents from wild-type neurons without (non-Tg)\nand with an additional Nrx::HRP tg/ or Nrx::HRP tg/ transgene. The depolarization pro-\ntocol was identical to the one used in Figures 4 A and 5D. D, Quantitative comparison of current\ndensities in littermate wild-type mice without (non-Tg, 100%) and with an additional\nNrx::HRP tg/ or Nrx::HRP tg/ transgene, indicating that the expression of neurexin trans-\ngenes did not increase Ca 2 currents beyond wild-type levels. Data shown are means  SEM.\nE, Binding of 125I-labeled -conotoxin GVIA at 30 pM to brain membranes from wild-type\ncontrol (non-Tg) and transgenic mice expressing Nrx::HRP tg/ or Nrx::HRP tg/. All exper-\niments were done with and without a 100-fold excess of unlabeled (“cold”) toxin to account for\nunspecific binding. Data shown are means  SEM. Error bars represent SEM.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nThe robust rescue mediated by transgenic Nrx::HRP raises\nthe question of whether the Nrx::HRP acts as a general enhancer\nof synaptic transmission that would also increase neurotransmit-\nter release within a wild-type genetic background. We therefore\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nTransgenically expressed Nrx::HRP is not sufficient to improve synaptic function\nFigure 5.\nand Ca 2 currents. Experiments similar to those described in Figures 3 and 4, with the excep-\ntion that a different neurexin variant (-neurexin) with distinct extracellular but identical\nC-terminal sequences was transgenically expressed and tested for its rescue potential\n(Nrx::HRP; see Fig. 1 A). A–E, Evoked EPSC amplitudes (A, B), spontaneous IPSC frequencies\n(C), and high voltage-activated Ca 2 currents (D, E) are unchanged in KO mice (SKO2, DKO2/3,\nDKO1/2, and TKO) without [nontransgenic (non-Tg); open bars] and with (filled bars) the\nNrx::HRP tg/ transgene present. In all graphs and panels, statistical significance is indicated\nabove the bars. Data shown are means  SEM. Error bars represent SEM.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nJ. Neurosci., April 27, 2005 • 25(17):4330 – 4342 • 4337\n**BLOCK**fs== 9.5**b== 0.6**t== 0.1**l== 0.5**r== 0.1**\ninvestigated whether transgenic neurexins potentiate evoked\npostsynaptic responses or Ca 2-channel function even in wild-\ntype neurons that express endogenous -neurexins normally.\nFor this purpose, we measured eEPSCs in neurons from wild-\ntype littermate mice that carried or lacked a transgenic\nNrx::HRP but found no differences in release (Fig. 6 A,B). We\nalso tested whether the normal Ca 2-channel activity could be\nfacilitated (Nrx::HRP) or inhibited (Nrx::HRP) by recording\nhigh voltage-activated Ca 2 currents in neurons from wild-type\nlittermate mice that carried or lacked an Nrx::HRP tg/ or\nNrx::HRP tg/ transgene (Fig. 6C,D). However, current traces\nand quantitations revealed no changes (Fig. 6C,D), consistent\nwith observations that expression levels of VDCC pore-forming\nsubunits were neither reduced in -neurexin KO mice (Missler et\nal., 2003) nor increased in the transgenic mice shown here (Fig.\n6 E). Thus, expression of additional transgenic neurexins by itself\ndoes not produce a gain-of-function or a dominant-negative\nphenotype.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\n-Neurexins specifically affect non-L-type Ca 2 channels\nAlthough transgenic neurexin 1 increased synaptic transmis-\nsion in all KO combinations, evoked postsynaptic responses are\nnot completely abolished in KO neurons, even in neurons lacking\nall three -neurexins (Figs. 3B, 5B). The residual synaptic trans-\nmission that is independent of -neurexins could be explained by\nat least two hypotheses: that -neurexins only partially modulate\nthe activity of all VDCCs or that -neurexins modulate only spe-\ncific subtypes of VDCCs. To test which Ca 2 channels are af-\nfected by -neurexins, we first analyzed the contribution of dif-\nferent VDCC subtypes to evoked neurotransmitter release in\nhypoglossal motor neurons from wild-type mice by sequentially\nblocking specific channels pharmacologically (Fig. 7A,B). Addi-\ntion of nifedipine, -conotoxin, and agatoxin (blockers of L-, N-,\nand P/Q-type Ca 2 channels, respectively) reduced the EPSC to\nalmost zero (Fig. 5A). The small residual EPSC after these addi-\ntions was blocked by Cd 2, suggesting that it most likely was\nmediated by R-type Ca 2 channels (Fig. 5A). Therefore, we fo-\ncused our quantitative analysis on L-, N-, and P/Q-type VDCCs\nthat appear to be dominant at this synapse.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nThe amplitude of EPSCs in wild-type neurons depended\n60% on an N-type Ca 2 channel-dependent component, sup-\nplemented by smaller but approximately equal contributions\nfrom L-type and P/Q-type Ca 2 channels (Fig. 7B). The contri-\nbutions of these three VDCC subtypes to neurotransmission\nmake the neonatal brainstem particularly suitable to study regu-\nlatory preferences. Control experiments in wild-type neurons us-\ning different mixtures of blockers instead of adding them sequen-\ntially resulted in a similar contribution of Ca 2-channel subtypes\n(data not shown). Because the participation of L-type channels\n(15% of the total amplitude) (Fig. 7B) reported here is larger\n**BLOCK**fs== 8.5**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nto probe for residual Ca 2-channel activities (presumably mediated by R-type). B, Relative\ncontributions of different VDCC subtypes to total EPSC amplitudes calculated from blocking\nexperiments, as shown in A. The evoked synaptic response in the neonatal brainstem prepara-\ntion depends mostly on N-type Ca 2 channels but also has significant contributions from L- and\nP/Q-type VDCCs. C, Comparison of L-type-mediated EPSC components in wild-type (WT) and\nmultiple -neurexin KO mice, uncovering a significantly increased proportion of the L-type\nVDCC component in mutant neurons, which is already evident in single KO (SKO2) mutant mice.\nD, Absolute currents of evoked postsynaptic responses (EPSCs; in picoamperes) during field stimula-\ntions of hypoglossal neurons. Total (open bars) and L-type-mediated (filled bars) amplitudes were\nanalyzedinwild-typecontrolsandinmultiple-neurexinKOmice.Statisticalsignificanceisindicated\nabove the bars. Data shown are means  SEM. Error bars represent SEM.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nL-type Ca 2-channel-mediated EPSC components are unaffected by\nFigure 7.\n-neurexins. A, Averaged traces of glutamatergic EPSCs monitored in wild-type hypoglossal\nmotor neurons before (control) and after sequential application of 10 M nifedipine (Nif) for\nL-type channel contributions, 0.5 M -conotoxin MVIIA (-Ctx) for N-type channel contribu-\ntions, 0.2 M agatoxin IVA (-Aga) for P/Q-type channel contributions, and 100 M Cd 2\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\n4338 • J. Neurosci., April 27, 2005 • 25(17):4330 – 4342\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nthan the L-type-dependent component observed in other central\nsynapses (Wu et al., 1999), we tested how this L-type dependence\nwas affected in -neurexin KO neurons. Analyzing the relative\nproportion of L-type-mediated EPSCs to the total postsynaptic\nresponse revealed that the L-type component increases from\n15% in wild-type neurons to 50% in TKO neurons (Fig. 7C).\nIn contrast to most other parameters studied thus far in\n-neurexin KO mice (survival rates, ventilation activity, evoked\nand spontaneous release, and Ca 2 currents) (Missler et al.,\n2003), the relative L-type contribution increases dramatically in\nSKO2 mice (Fig. 7C).\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nThe augmented L-type dependence of EPSCs in KO neurons\ncould reflect a true compensation caused by upregulated activity\nof L-type Ca 2 channels or merely constitute a relative increase\ncaused by the dramatically reduced total EPSC. Evaluation of\nabsolute amplitudes demonstrated that the L-type-mediated\nEPSC amplitudes did not differ significantly between -neurexin\ngenotypes (Fig. 7D, filled bars). This result suggests that the ac-\ntivity of L-type VDCCs is not dependent on -neurexins, indi-\ncating that the impairment of N- and P/Q-type Ca 2 channels is\nprimarily responsible for the phenotype.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nTo obtain direct evidence for a positive regulation of non-L-type\nCa 2 channels by -neurexins, we applied the sequential blocking\nprotocol described above to two different -neurexin KO combina-\ntions (SKO2 and DKO1/2) with and without the Nrx::HRP tg/\ntransgene (Fig. 8A,B). The presence of Nrx::HRP tg/ increased the\ncontributions of N- and P/Q-type VDCCs to the total EPSC twofold\nto threefold (Fig. 8C,D). Consistent with the unchanged L-type-\nmediated EPSC in KO neurons (Fig. 7D), expression of an additional\nNrx::HRP tg/ transgene did not change the size of L-type-\ndependent amplitudes in rescued neurons (Fig. 8C,D).\n**BLOCK**fs== 9.5**b== 0.0**t== 0.6**l== 0.5**r== 0.1**\nOur finding of a specific regulation of N- and P/Q-type VDCC\nactivity by -neurexins documents the molecular specificity of\nthe neurexin–Ca 2-channel link. To further validate this result,\nwe performed experiments that tested the effect of Ca 2-channel\ninhibitors on the frequency of mIPSCs. Recording mini frequen-\ncies in PBC neurons to study Ca 2-channel activity is meaning-\nful, because in these neurons approximately one-half of the mini\nevents depend on VDCCs and can be blocked by Cd 2 (45–55%)\n(Missler et al., 2003) (data not shown). We first recorded minis\nmediated by different VDCC subtypes in wild-type neurons (Fig.\n9A) after a sequential blocking protocol described above (Figs. 7,\n8). Consistent with the data on evoked responses, we found that\nN-type Ca 2 channels mediated most of the Ca 2 channel-\ndependent minis (60%). However, the P/Q-type-dependent\ncomponent of minis was slightly higher (26%) than the L-type-\ndependent component (6%) (Fig. 9B). Despite the small contri-\nbution of L-type VDCCs to wild-type minis, their relative pro-\nportion increased greatly in the different -neurexin KO mice,\nreaching proportions similar to the EPSC data shown above (Fig.\n9C). Corresponding to the EPSC analysis, however, this apparent\ncompensation was not attributable to an actual increase in\nL-type-dependent minis, because the overall mini frequency is\nreduced in KO neurons (Missler et al., 2003) (Figs. 3D, 5C), sup-\nporting the notion that the function of L-type VDCCs is mostly\nunaffected by the absence of -neurexins. We then deter-\nmined how the VDCC subtype contributions to mini frequen-\ncies changed when an Nrx::HRP tg/ transgene was crossed into\nthe same -neurexin KO combinations used above (SKO2 and\nDKO1/2) (Fig. 10 A,B). Consistent with the results from evoked\nrelease, the presence of the Nrx::HRP tg/ transgene signifi-\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nTransgenic -neurexin increases evoked responses by selectively activating N-\nFigure 8.\nand P/Q-type Ca 2 channels. A, B, Averaged traces representing glutamatergic EPSCs from\nSKO2 and DKO1/2 mice without [nontransgenic (non-Tg)] and with additional Nrx::HRP tg/\ntransgene. Evoked responses were recorded before (control) and after sequential application of\nspecific VDCC blockers (same protocol as in Fig. 7A). Nif, Nifedipine; -Ctx, -conotoxin;\n-Aga, agatoxin. C, D, Quantitative comparison of the contributions from different VDCC sub-\ntypes to EPSCs in single KO and double KO mice without (open bars) and with (filled bars)\nadditional Nrx::HRP tg/ transgene shows an increased amplitude mediated by N- and P/Q-\ntype calcium channels when a transgenic -neurexin is present. The proportion of L-type-\ndependent current components is unchanged in rescued neurons. n.s., Not significant. In all\ngraphs and panels, statistical significance is indicated above the bars. Data shown are means \nSEM. Error bars represent SEM.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nJ. Neurosci., April 27, 2005 • 25(17):4330 – 4342 • 4339\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nCa 2-dependent minis at inhibitory synapses of wild-type and -neurexin KO\nFigure 9.\nmice. A, Representative traces of spontaneous release events (mIPSCs) recorded under TTX and\nCNQX from neonatal PBC neurons of wild-type mice before (control) and after the sequential\nblocking of Ca 2 channels, as described in Figures 7 and 8 for evoked release. Nif, Nifedipine;\n-Ctx, -conotoxin; -Aga, agatoxin. B, Dependence of wild-type mIPSCs on different VDCC\nsubtypes is expressed as a percentage of the respective Ca 2-dependent mini frequency, quan-\ntitated from blocking experiments, as shown in A. In wild-type neurons of this brainstem area,\napproximately one-half of all minis depend on Ca 2 influx (data not shown). C, In knock-out\nmice lacking one, two, or all three -neurexins (SKO2, DKO2/3, DKO1/2, and TKO), the propor-\ntion of L-type-mediated minis appears significantly larger because of the strongly reduced\noverall mIPSC frequencies (compare with Figs. 3D and 5C). WT, Wild type. Statistical significance\nis indicated above the bars. Data shown are means  SEM. Error bars represent SEM.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\ncantly increased the N- and P/Q-type-mediated mini frequen-\ncies, whereas L-type-dependent mIPSC events were unchanged\n(Fig. 10C,D), indicating that L-type channels are also excluded\nfrom -neurexin regulation at inhibitory synapses.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nTransgenically expressed -neurexin specifically increases N- and P/Q-type-\nFigure 10.\nmediated mini frequencies. A, B, Sample traces of mini events (mIPSCs) recorded under TTX and\nCNQX from SKO2 and DKO1/2 mice, both with an additional Nrx::HRP tg/ transgene present.\nSpontaneous postsynaptic responses were recorded before (control) and after sequential appli-\ncation of Ca 2-channel inhibitors (same as in Fig. 9A). To show representative changes in mini\nevents after drug treatment, recordings with above-average frequencies were chosen for all\nconditions (compare with absolute frequencies in C, D). Nif, Nifedipine; -Ctx, -conotoxin;\n-Aga, agatoxin; n.s., not significant. C, D, Because transgenic -neurexin rescued mini fre-\nquencies in all -neurexin KO combinations studied (Fig. 3D), the contributions of different\nVDCC subtypes were compared in single KO (C) and double KO (D) mice without [nontransgenic\n(non-Tg); open bars] and with (filled bars) additional Nrx::HRP tg/ transgene. Increased mini\nfrequencies reveal an augmented activity of N- and P/Q-type VDCCs but not of L-type channels.\nIn all graphs and panels, statistical significance is indicated above the bars. Data shown are\nmeans  SEM. Error bars represent SEM.\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\n4340 • J. Neurosci., April 27, 2005 • 25(17):4330 – 4342\n**BLOCK**fs== 9.5**b== 0.6**t== 0.1**l== 0.1**r== 0.5**\nDiscussion\nNeurexins are among several neuronal cell-adhesion molecules\nthat were proposed to contribute to synaptic neurotransmitter\nrelease and/or synapse formation (for review, see Sudhof, 2001;\nMissler, 2003; Scheiffele, 2003). For many of these candidate cell-\nadhesion molecules, studies in knock-out mice have produced\nilluminative phenotypes, although their suggested functions are\noften only indirectly related to synapses (Tomasiewicz et al.,\n1993; Cremer et al., 1994; Evers et al., 2002; Togashi et al., 2002;\nWang et al., 2002). The -neurexin KO phenotype, in contrast,\nstrongly points toward a specifically synaptic function, because\ntheir deletion severely impairs both excitatory and inhibitory\nneurotransmitter release, presumably as a result of impaired\nCa 2-channel function, but does not affect brain architecture\nand synaptic ultrastructure (Missler et al., 2003). Despite the\npresent inavailability of -neurexin knock-out mice, recent in\nvitro work on the -neurexin/neuroligin complex has suggested\nthat it may have a role in promoting synapse formation and dif-\nferentiation (Scheiffele et al., 2000; Dean et al., 2003; Graf et al.,\n2004).\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.1**r== 0.5**\nWe have now directly compared the putative roles of - and\n-neurexins and investigated the molecular specificity of the\nneurexin–Ca 2-channel regulation using a comparative rescue\nstrategy in transgenic mice. Such a strategy is rarely used because\nof the amount of work involved, although a similar approach\nhas been followed, for example, to analyze the functional redun-\ndancy of the closely related growth cone proteins GAP43 and\ncytoskeleton-associated protein 23 (Frey et al., 2000). For ques-\ntions relating to the normal regulation of Ca 2 channels by a\ncell-adhesion molecule in an intact brain, a transgenic strategy\nmay be advantageous, because cell-adhesion events are best stud-\nied in acute slices of an intact neuronal network. The brainstem\nsystem was chosen because multiple -neurexin KO mutants die\nearly postnatally as a result of respiratory problems arising from\nbrainstem defects, and data from the rescue analysis could be\ntested for consistency compared with the previous KO study.\ntransgenic\nFurthermore,\nneurexin::HRP proteins allows an unbiased estimation of expres-\nsion levels and distribution patterns. In addition, choosing trans-\ngenic lines that express neurexin::HRP fusion proteins within the\nrange of endogenous neurexins reduces the problems of\ndominant-negative effects and high variability, frequently associ-\nated, for example, with overexpression approaches in vitro. Our\ncurrent study extends the previous analysis of neurexins by re-\nporting three novel findings.\n**BLOCK**fs== 9.5**b== 0.4**t== 0.6**l== 0.2**r== 0.6**\nthe background-free detection of\n**BLOCK**fs== 9.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nFirst, transgenic neurexin 1, but not transgenic neurexin 1,\npartly rescued the well characterized properties of the -neurexin\nKO phenotype. Because transgenic neurexin 1 and 1 differ\nsolely in their extracellular domains, but contain identical trans-\nmembrane regions and cytoplasmic sequences, we conclude that\n- and -neurexins have nonredundant functions that involve\ntheir distinct N-terminal domains. However, if the extracellular\nsequences of -neurexin define their role as organizer molecules\nof presynaptic release, what then is the respective role of\n-neurexins? The work by the Scheiffele and Craig laboratories\nsuggests, as originally proposed when neurexins were first iden-\ntified (Ushkaryov et al., 1992), that neurexins can promote the\nformation of de novo synapses and differentiation of postsynaptic\nreceptors (Scheiffele et al., 2000; Dean et al., 2003; Graf et al.,\n2004). Because these reports were confined to the in vitro activi-\nties of -neurexins, whereas we study the effects of transgenes in\nvivo, it is at present difficult to compare their results with ours.\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nHowever, we did not detect an effect of transgenic expression of\nneurexin 1on mini amplitudes or evoked responses (as would\nbe expected if postsynaptic receptors were recruited or activated)\nand did not find that transgenic neurexin 1 augmented mini\nfrequencies (as would be expected from a synaptogenetic activ-\nity). A possible interpretation of this discrepancy could be that\nthe synapse formation and differentiation observed by overex-\npression in cultures uncovers an additional (rather than essen-\ntial) functional activity of neurexins that is redundant with other\nmolecules and thus is not present in intact tissue. Alternatively, it\nis possible that a synaptogenetic function for neurexins is devel-\nopmentally compensated, although this raises the question of\nwhy the essential function for -neurexins in executing neuro-\ntransmitter release, as documented here and previously (Missler\net al., 2003), is not compensated.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nSecond, the neurexin 1transgene ameliorated all facets of the\n-neurexin KO phenotype, independent of which neurexin genes\nwere mutated, demonstrating that the various -neurexins do\nnot exhibit fundamental functional differences. The fact that a\nsingle transgene with only one set of splice variants was active in\nreversing the various phenotypes suggests that -neurexins me-\ndiate a single integrated activity. The increase in neurotransmit-\nter release and the regulation of VDCCs mediated by -neurexin\nwere not simple gain-of-function activities, because the trans-\ngenic proteins did not further facilitate their activity in wild-type\nmice. This result suggests that neurexins may regulate VDCC\nactivity only within their normal range but not beyond.\n**BLOCK**fs== 9.5**b== 0.0**t== 0.5**l== 0.5**r== 0.1**\nThird, our data now show that deletion of -neurexins selec-\ntively impairs neurotransmitter release that is dependent on non-\nL-type Ca 2 channels. Other classes of extracellular molecules\nsuch as the matrix component tenascin-C may modulate L-type-\ndependent synaptic plasticity (Evers et al., 2002), but L-type-\nmediated EPSC amplitudes and mini frequencies in brainstem\nsynapses were unaffected by -neurexins, as shown here. The fact\nthat a preference of -neurexins for N- and P/Q-type Ca 2 chan-\nnels exists is interesting, because the VDCCs involved in presyn-\naptic Ca 2 entry are usually of the N- and P/Q-type (Dunlap et\nal., 1995; Reid et al., 1998; Wu et al., 1999), but R- and L-type\nchannels may be involved at particular synapses as well (Wu et al.,\n1998; Brandt et al., 2003). Although all presynaptic terminals\ncontain VDCCs, the release probability and Ca 2 dynamics of\npresynaptic terminals vary significantly (Hessler et al., 1993;\nRosenmund et al., 1993; Koester and Sakmann, 2000), even at\nsynaptic terminals of the same neuron (Maccaferri et al., 1998;\nMarkram et al., 1998; Reyes et al., 1998; Scanziani et al., 1998).\nBecause of the high Ca 2 cooperativity and Ca 2 sensitivity of\nneurotransmitter release (Bollmann et al., 2000; Schneggen-\nburger and Neher, 2000), regulating VDCCs efficiently controls\nsynaptic strength, and small changes in Ca 2 influx lead to mag-\nnified effects on synaptic release (Felmy et al., 2003). It comes as\nno surprise, therefore, that VDCCs are extensively regulated. In\naddition to the intracellular free calcium concentration itself, sev-\neral helix–loop– helix Ca2-binding motifs containing proteins\ncalled EF-hand proteins (e.g., calmodulin, normal calf serum,\ncalcium-binding protein) (for review, see Weiss and Burgoyne,\n2002), G-protein-coupled receptor pathways (such as GABAB or\n5-HT) (for review, see Catterall, 2000; Zamponi, 2001), compo-\nnents of\nsyntaxin 1 or\nsynaptosomal-associated protein of 25 kDa) (for review, see Cat-\nterall, 1999; Atlas, 2001), or even signaling events induced by lipid\nbilayer components (Wu et al., 2002) facilitate or inhibit VDCCs.\nOur findings on -neurexins now put the first synaptic cell–\nadhesion molecule on the list of important VDCC regulators.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.6**r== 0.2**\nthe exocytotic machinery (e.g.,\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nJ. Neurosci., April 27, 2005 • 25(17):4330 – 4342 • 4341\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nThese results, however, raise additional questions. Why are there\nmultiple -neurexin genes with hundreds of splice variants if a\nsingle isoform, expressed in a single splice variant, is at least partly\nable to rescue different phenotypic effects produced by the dele-\ntions of multiple -neurexin genes? What are the extracellular\ninteractions that mediate the essential functions of -neurexins,\nand how are these extracellular interactions coupled to intracel-\nlular effector pathways? What is the actual role of -neurexins,\nincluding their splice variants that do not bind to neuroligin in a\ntrans-synaptic complex? Addressing these questions will signifi-\ncantly promote our understanding of the organization of presyn-\naptic terminals but will require new experimental approaches.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nReferences\nAndra K, Abramowski D, Duke M, Probst A, Wiederhold KH, Burki K, Goe-\ndert M, Sommer B, Staufenbiel M (1996) Expression of APP in trans-\ngenic mice: a comparison of neuron-specific promoters. Neurobiol Aging\n17:183–190.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nAtlas D (2001) Functional and physical coupling of voltage-sensitive cal-\ncium channels with exocytotic proteins: ramifications for the secretion\nmechanism. J Neurochem 77:972–985.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nBiederer T, Sudhof TC (2000) Mints as adaptors. Direct binding to neurex-\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nins and recruitment of munc18. J Biol Chem 275:39803–39806.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\nBollmann JH, Sakmann B, Borst JG (2000) Calcium sensitivity of glutamate\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nrelease in a calyx-type terminal. Science 289:953–957.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nBrandt A, Striessnig J, Moser T (2003) CaV1.3 channels are essential for\ndevelopment and presynaptic activity of cochlear inner hair cells. J Neu-\nrosci 23:10832–10840.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nButz S, Okamoto M, Sudhof TC (1998) A tripartite protein complex with\nthe potential to couple synaptic vesicle exocytosis to cell adhesion in\nbrain. Cell 94:773–782.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nCatterall WA (1999) Interactions of presynaptic Ca 2 channels and snare\nproteins in neurotransmitter release. Ann NY Acad Sci 868:144 –159.\nCatterall WA (2000) Structure and regulation of voltage-gated Ca 2 chan-\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.1**r== 0.7**\nnels. Annu Rev Cell Dev Biol 16:521–555.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nCremer H, Lange R, Christoph A, Plomann M, Vopper G, Roes J, Brown R,\nBaldwin S, Kraemer P, Scheff S (1994) Inactivation of the N-CAM gene\nin mice results in size reduction of the olfactory bulb and deficits in spatial\nlearning. Nature 367:455– 459.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nDean C, Scholl FG, Choih J, DeMaria S, Berger J, Isacoff E, Scheiffele P\n(2003) Neurexin mediates the assembly of presynaptic terminals. Nat\nNeurosci 6:708 –716.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nDubois L, Lecourtois M, Alexandre C, Hirst E, Vincent JP (2001) Regulated\nendocytic routing modulates wingless signaling in Drosophila embryos.\nCell 105:613– 624.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nDunlap K, Luebke JI, Turner TJ (1995) Exocytotic Ca 2 channels in mam-\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.1**r== 0.6**\nmalian central neurons. Trends Neurosci 18:89 –98.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nEvers MR, Salmen B, Bukalo O, Rollenhagen A, Bosl MR, Morellini F, Bartsch\nU, Dityatev A, Schachner M (2002) Impairment of L-type Ca 2\nchannel-dependent forms of hippocampal synaptic plasticity in mice de-\nficient in the extracellular matrix glycoprotein tenascin-C. J Neurosci\n22:7177–7194.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nFelmy F, Neher E, Schneggenburger R (2003) Probing the intracellular cal-\ncium sensitivity of transmitter release during synaptic facilitation. Neu-\nron 37:801– 811.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\nFrey D, Laux T, Xu L, Schneider C, Caroni P (2000) Shared and unique roles\nof CAP23 and GAP43 in actin regulation, neurite outgrowth, and ana-\ntomical plasticity. J Cell Biol 149:1443–1454.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nGeppert M, Khvotchev M, Krasnoperov V, Goda Y, Missler M, Hammer RE,\nIchtchenko K, Petrenko AG, Sudhof TC (1998) Neurexin I alpha is a\nmajor alpha-latrotoxin receptor that cooperates in alpha-latrotoxin ac-\ntion. J Biol Chem 273:1705–1710.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nGraf ER, Zhang X, Jin SX, Linhoff MW, Craig AM (2004) Neurexins induce\ndifferentiation of GABA and glutamate postsynaptic specializations via\nneuroligins. Cell 119:1013–1026.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nHata Y, Butz S, Sudhof TC (1996) CASK: a novel dlg/PSD95 homolog with\nan N-terminal calmodulin-dependent protein kinase domain identified\nby interaction with neurexins. J Neurosci 16:2488 –2494.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nHessler NA, Shirke AM, Malinow R (1993) The probability of transmitter\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.1**r== 0.6**\nrelease at a mammalian central synapse. Nature 366:569 –572.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nTC (1995) Neuroligin 1: a splice site-specific ligand for beta-neurexins.\nCell 81:435– 443.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nKoester HJ, Sakmann B (2000) Calcium dynamics associated with action\npotentials in single nerve terminals of pyramidal cells in layer 2/3 of the\nyoung rat neocortex. J Physiol (Lond) 529:625– 646.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nMaccaferri G, Toth K, McBain CJ (1998) Target-specific expression of pre-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nsynaptic mossy fiber plasticity. Science 279:1368 –1370.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nMarkram H, Wang Y, Tsodyks M (1998) Differential signaling via the same\naxon of neocortical pyramidal neurons. Proc Natl Acad Sci USA\n95:5323–5328.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nMissler M (2003) Synaptic cell adhesion goes functional. Trends Neurosci\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\n26:176 –178.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nMissler M, Sudhof TC (1998) Neurexins: three genes and 1001 products.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.3**\nTrends Genet 14:20 –26.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nMissler M, Hammer RE, Sudhof TC (1998) Neurexophilin binding to\nalpha-neurexins. A single lns domain functions as an independently fold-\ning ligand-binding unit. J Biol Chem 273:34716 –34723.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nMissler M, Zhang W, Rohlmann A, Kattenstroth G, Hammer RE, Gottmann\nK, Sudhof TC (2003) Alpha-neurexins couple Ca 2 channels to synap-\ntic vesicle exocytosis. Nature 424:939 –948.\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nOlmsted JB (1981) Affinity purification of antibodies from diazotized paper\nblots of heterogeneous protein samples. J Biol Chem 256:11955–11957.\nReid CA, Bekkers JM, Clements JD (1998) N- and P/Q-type Ca 2 channels\nmediate transmitter release with a similar cooperativity at rat hippocam-\npal autapses. J Neurosci 18:2849 –2855.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nReyes A, Lujan R, Rozov A, Burnashev N, Somogyi P, Sakmann B (1998)\nTarget-cell-specific facilitation and depression in neocortical circuits. Nat\nNeurosci 1:279 –285.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nRosenmund C, Clements JD, Westbrook GL (1993) Nonuniform probabil-\nity of glutamate release at a hippocampal synapse. Science 262:754 –757.\nRowen L, Young J, Birditt B, Kaur A, Madan A, Philipps DL, Qin S, Minx P,\nWilson RK, Hood L, Graveley BR (2002) Analysis of the human neur-\nexin genes: alternative splicing and the generation of protein diversity.\nGenomics 79:587–597.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nSara Y, Biederer T, Atasoy D, Chubykin A, Mozhayeva MG, Sudhof TC,\nKavalali ET (2005) Selective capability of SynCAM and neuroligin for\nfunctional synapse assembly. J Neurosci 25:260 –270.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nScanziani M, Gahwiler BH, Charpak S (1998) Target cell-specific modula-\ntion of transmitter release at terminals from a single axon. Proc Natl Acad\nSci USA 95:12004 –12009.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nScheiffele P (2003) Cell-cell signaling during synapse formation in the CNS.\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.5**r== 0.3**\nAnnu Rev Neurosci 26:485–508.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nScheiffele P, Fan J, Choih J, Fetter R, Serafini T (2000) Neuroligin expressed\nin nonneuronal cells triggers presynaptic development in contacting ax-\nons. Cell 101:657– 669.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nSchneggenburger R, Neher E (2000) Intracellular calcium dependence of\ntransmitter release rates at a fast central synapse. Nature 406:889 – 893.\nSong JY, Ichtchenko K, Sudhof TC, Brose N (1999) Neuroligin 1 is a\npostsynaptic cell-adhesion molecule of excitatory synapses. Proc Natl\nAcad Sci USA 96:1100 –1105.\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nSugita S, Khvochtev M, Sudhof TC (1999) Neurexins are functional alpha-\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.2**\nlatrotoxin receptors. Neuron 22:489 – 496.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nSugita S, Saito F, Tang J, Satz J, Campbell K, Sudhof TC (2001) A stoichio-\nmetric complex of neurexins and dystroglycan in brain. J Cell Biol\n154:435– 445.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nSunio A, Metcalf AB, Kramer H (1999) Genetic dissection of endocytic traf-\nficking in Drosophila using a horseradish peroxidase-bride of sevenless\nchimera: hook is required for normal maturation of multivesicular endo-\nsomes. Mol Biol Cell 10:847– 859.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nSudhof TC (2001) Alpha-latrotoxin and its receptors: neurexins and CIRL/\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.2**\nlatrophilins. Annu Rev Neurosci 24:933–962.\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nTabuchi K, Sudhof TC (2002) Structure and evolution of neurexin genes:\nsplicing. Genomics\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\ninto the mechanism of\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nalternative\n**BLOCK**fs== 8.0**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\ninsight\n79:849 – 859.\n**BLOCK**fs== 8.0**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nTogashi H, Abe K, Mizoguchi A, Takaoka K, Chisaka O, Takeichi M (2002)\nCadherin regulates dendritic spine morphogenesis. Neuron 35:77– 89.\nTomasiewicz H, Ono K, Yee D, Thompson C, Goridis C, Rutishauser U,\nMagnuson T (1993) Genetic deletion of a neural cell adhesion molecule\nvariant (N-CAM-180) produces distinct defects in the central nervous\nsystem. Neuron 11:1163–1174.\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.1**r== 0.5**\nIchtchenko K, Hata Y, Nguyen T, Ullrich B, Missler M, Moomaw C, Sudhof\n**BLOCK**fs== 8.0**b== 0.0**t== 1.0**l== 0.5**r== 0.1**\nUshkaryov YA, Sudhof TC (1993) Neurexin III alpha: extensive alternative\n**BLOCK**fs== 8.0**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\n4342 • J. Neurosci., April 27, 2005 • 25(17):4330 – 4342\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nsplicing generates membrane-bound and soluble forms. Proc Natl Acad\nSci USA 90:6410 – 6414.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nUshkaryov YA, Petrenko AG, Geppert M, Sudhof TC (1992) Neurexins:\nsynaptic cell surface proteins related to the alpha-latrotoxin receptor and\nlaminin. Science 257:50 –56.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nUshkaryov YA, Hata Y, Ichtchenko K, Moomaw C, Afendis S, Slaughter CA,\nSudhof TC (1994) Conserved domain structure of beta-neurexins. Un-\nusual cleaved signal sequences in receptor-like neuronal cell-surface pro-\nteins. J Biol Chem 269:11987–11992.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\nWang X, Weiner JA, Levi S, Craig AM, Bradley A, Sanes JR (2002) Gamma\nprotocadherins are required for survival of spinal interneurons. Neuron\n36:843– 854.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nWeiss JL, Burgoyne RD (2002) Sense and sensibility in the regulation of\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nvoltage-gated Ca2 channels. Trends Neurosci 25:489 – 491.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nWu L, Bauer CS, Zhen XG, Xie C, Yang J (2002) Dual regulation of voltage-\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\ngated calcium channels by PtdIns(4,5)P2. Nature 419:947–952.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nWu LG, Borst JG, Sakmann B (1998) R-type Ca 2 currents evoke trans-\nmitter release at a rat central synapse. Proc Natl Acad Sci USA\n95:4720 – 4725.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nWu LG, Westenbroek RE, Borst JG, Catterall WA, Sakmann B (1999) Cal-\ncium channel types with distinct presynaptic localization couple differen-\ntially to transmitter release in single calyx-type synapses. J Neurosci\n19:726 –736.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nZamponi GW (2001) Determinants of G protein inhibition of presynaptic\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\ncalcium channels. Cell Biochem Biophys 34:79 –94.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nZhang W, Elsen F, Barnbrock A, Richter DW (1999) Postnatal development\nof GABAB receptor-mediated modulation of voltage-activated Ca 2 cur-\nrents in mouse brain-stem neurons. Eur J Neurosci 11:2332–2342.",
         "extracted",
         "Selective Capability of SynCAM and Neuroligin for Functional Synapse Assembly;Neurexins Induce Differentiation of GABA and Glutamate Postsynaptic Specializations via Neuroligins;Cell-cell signaling during synapse formation in the CNS.;CaV1.3 Channels Are Essential for Development and Presynaptic Activity of Cochlear Inner Hair Cells;Neurexin mediates the assembly of presynaptic terminals;α-Neurexins couple Ca2+ channels to synaptic vesicle exocytosis;Synaptic cell adhesion goes functional;Probing the Intracellular Calcium Sensitivity of Transmitter Release during Synaptic Facilitation;Gamma Protocadherins Are Required for Survival of Spinal Interneurons;Dual regulation of voltage-gated calcium channels by PtdIns(4,5)P2;Sense and sensibility in the regulation of voltage-gated Ca2+ channels;Impairment of L-type Ca2+ Channel-Dependent Forms of Hippocampal Synaptic Plasticity in Mice Deficient in the Extracellular Matrix Glycoprotein Tenascin-C;Cadherin Regulates Dendritic Spine Morphogenesis;Structure and evolution of neurexin genes: insight into the mechanism of alternative splicing.;Analysis of the human neurexin genes: alternative splicing and the generation of protein diversity.;A stoichiometric complex of neurexins and dystroglycan in brain;Regulated Endocytic Routing Modulates Wingless Signaling in Drosophila Embryos;Functional and physical coupling of voltage‐sensitive calcium channels with exocytotic proteins: ramifications for the secretion mechanism;Calcium dynamics associated with action potentials in single nerve terminals of pyramidal cells in layer 2/3 of the young rat neocortex;Intracellular calcium dependence of transmitter release rates at a fast central synapse;Calcium sensitivity of glutamate release in a calyx-type terminal.;Shared and Unique Roles of Cap23 and Gap43 in Actin Regulation, Neurite Outgrowth, and Anatomical Plasticity;Neuroligin Expressed in Nonneuronal Cells Triggers Presynaptic Development in Contacting Axons;Postnatal development of GABAB receptor‐mediated modulation of voltage‐activated Ca2+ currents in mouse brain‐stem neurons;Genetic dissection of endocytic trafficking in Drosophila using a horseradish peroxidase-bride of sevenless chimera: hook is required for normal maturation of multivesicular endosomes.;Interactions of Presynaptic Ca2+ Channels and Snare Proteins in Neurotransmitter Release;Neurexins Are Functional α-Latrotoxin Receptors;Neuroligin 1 is a postsynaptic cell-adhesion molecule of excitatory synapses.;Calcium Channel Types with Distinct Presynaptic Localization Couple Differentially to Transmitter Release in Single Calyx-Type Synapses;Neurexophilin binding to alpha-neurexins. A single LNS domain functions as an independently folding ligand-binding unit.;Target cell-specific modulation of transmitter release at terminals from a single axon.;A Tripartite Protein Complex with the Potential to Couple Synaptic Vesicle Exocytosis to Cell Adhesion in Brain;Target-cell-specific facilitation and depression in neocortical circuits;Differential signaling via the same axon of neocortical pyramidal neurons.;N- and P/Q-Type Ca2+ Channels Mediate Transmitter Release with a Similar Cooperativity at Rat Hippocampal Autapses;R-type Ca2+ currents evoke transmitter release at a rat central synapse.;Target-specific expression of presynaptic mossy fiber plasticity.;Neurexin Iα Is a Major α-Latrotoxin Receptor That Cooperates in α-Latrotoxin Action*;CASK: a novel dlg/PSD95 homolog with an N-terminal calmodulin-dependent protein kinase domain identified by interaction with neurexins;Expression of APP in transgenic mice: a Comparison of neuron-specific promoters;Neuroligin 1: A splice site-specific ligand for β-neurexins;Exocytotic Ca2+ channels in mammalian central neurons;Conserved domain structure of beta-neurexins. Unusual cleaved signal sequences in receptor-like neuronal cell-surface proteins.;Inactivation of the N-CAM gene in mice results in size reduction of the olfactory bulb and deficits in spatial learning;The probability of transmitter release at a mammalian central synapse;Genetic deletion of a neural cell adhesion molecule variant (N-CAM-180) produces distinct defects in the central nervous system;Nonuniform probability of glutamate release at a hippocampal synapse.;Neurexin III alpha: extensive alternative splicing generates membrane-bound and soluble forms.;Neurexins: synaptic cell surface proteins related to the alpha-latrotoxin receptor and laminin.;Affinity purification of antibodies from diazotized paper blots of heterogeneous protein samples.;Determinants of G protein inhibition of presynaptic calcium channels;alpha-Latrotoxin and its receptors: neurexins and CIRL/latrophilins.;Structure and regulation of voltage-gated Ca2+ channels.;Neurexins: three genes and 1001 products.;Neurexin I alpha is a major alpha-latrotoxin receptor that cooperates in alpha-latrotoxin action.;Neuroligin 1: a splice site-specific ligand for beta-neurexins.;Mints as adaptors. Direct binding to neurexins and recruitment of munc18",
         "Extracellular Domains of α-Neurexins Participate in Regulating Synaptic Transmission by Selectively Affecting N- and P/Q-Type Ca2+ Channels",
         "None"
        ],
        [
         "47",
         "003bffead36371a8f49d13dbbc128130d1d9bd90",
         "The histone chaperone NAP1 is a carrier of histones during nuclear import, nucleosome assembly, and chromatin remodeling. Analytical ultracentrifugation was used to determine the association states of NAP1 alone and in complexes with core histones. In addition, the concentration dependence of the association was quantified by determining the equilibrium dissociation constant between different NAP1 species. At physiological protein and salt concentrations the prevalent species were the NAP1 dimer and octamer. These were also the association states found to interact with histones in a stoichiometry of one NAP1 monomer per histone. Based on these results a model for a cell cycle-dependent shift of the NAP1 dimer-octamer equilibrium is proposed that reflects different biological functions of NAP1.",
         "K. Toth,J. Mazurkiewicz,K. Rippe",
         "http://www.jbc.org/article/S0021925820692419/pdf",
         "\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nTHE JOURNAL OF BIOLOGICAL CHEMISTRY\n© 2005 by The American Society for Biochemistry and Molecular Biology, Inc.\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nVol. 280, No. 16, Issue of April 22, pp. 15690 –15699, 2005\nPrinted in U.S.A.\n**BLOCK**fs== 14.0**b== 0.8**t== 0.1**l== 0.1**r== 0.2**\nAssociation States of Nucleosome Assembly Protein 1 and Its\nComplexes with Histones*\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.1**\nReceived for publication, November 26, 2004, and in revised form, January 10, 2005\nPublished, JBC Papers in Press, January 31, 2005, DOI 10.1074/jbc.M413329200\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.2**r== 0.3**\nKatalin Fejes To´ th‡§, Jacek Mazurkiewicz‡, and Karsten Rippe‡¶\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.2**r== 0.2**\nFrom the ‡Kirchhoff-Institut fu¨ r Physik, AG Molekulare Biophysik, Ruprecht-Karls-Universita¨ t Heidelberg,\nIm Neuenheimer Feld 227 and the §Deutsches Krebsforschungszentrum, Abt. Molekulare Genetik (B060),\nIm Neuenheimer Feld 280, D-69120 Heidelberg, Germany\n**BLOCK**fs== 8.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nThe histone chaperone NAP1 is a carrier of histones\nduring nuclear import, nucleosome assembly, and chro-\nmatin remodeling. Analytical ultracentrifugation was\nused to determine the association states of NAP1 alone\nand in complexes with core histones. In addition, the\nconcentration dependence of the association was quan-\ntified by determining the equilibrium dissociation con-\nstant between different NAP1 species. At physiological\nprotein and salt concentrations the prevalent species\nwere the NAP1 dimer and octamer. These were also the\nassociation states found to interact with histones in a\nstoichiometry of one NAP1 monomer per histone. Based\non these results a model for a cell cycle-dependent shift\nof the NAP1 dimer-octamer equilibrium is proposed that\nreflects different biological functions of NAP1.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nIn eukaryotes the DNA is packed into nucleosomes consist-\ning of 147 bp of DNA wrapped around an octamer of small basic\nhistone proteins (1– 4). This octamer consists of two copies of\nhistones H2A, H2B, H3, and H4, respectively. The nucleosome\nconstitutes the basic repeating unit of chromatin, and forma-\ntion of a regularly spaced nucleosome chain during chromatin\nassembly is a prerequisite to maintain the biological activity of\nchromatin. This requires ATP-dependent chromatin assembly\nfactors as well as so-called histone chaperones (5–10). Histone\nchaperones bind histone proteins and prevent nonspecific,\ncharge-based interaction with nucleic acids. This activity ap-\npears essential to cell viability, because simple mixing of his-\ntones and DNA in vitro at physiological salt concentrations\nleads to the rapid formation of poorly defined insoluble aggre-\ngates (11, 12). One of these chaperones, the nucleosome assem-\nbly protein 1 (NAP1)1 (13, 14), is involved in the transport of\nthe histone H2AH2B dimer from the cytoplasm to the nucleus\nand the deposition of histones onto the DNA as described in\nseveral reviews (5–7, 10). NAP1 is highly acidic, which is likely\nto mediate its interaction with the positively charged histone\nproteins. It is thought to function in large chromosomal do-\nmains rather than at local restricted sites (5), and in yeast loss\nof NAP1 leads to an altered gene expression of about 10% of the\ngenome (15). NAP1 is present at micromolar concentrations\nthroughout the cell cycle, and an increase of the NAP1 concen-\ntration during S-phase has been reported for some NAP1 ho-\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n* The costs of publication of this article were defrayed in part by the\npayment of page charges. This article must therefore be hereby marked\n“advertisement” in accordance with 18 U.S.C. Section 1734 solely to\nindicate this fact.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n¶ To whom correspondence should be addressed. Tel.: 49-6221-54-9270;\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nFax: 49-6221-54-9112; E-mail: Karsten.Rippe@kip.uni-heidelberg.de.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n1 The abbreviations used are: NAP1, nucleosome assembly protein 1;\nAUC, analytical ultracentrifugation; s, sedimentation coefficient; D,\ndiffusion coefficient.\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nmologues in higher eukaryotes (16 –19). Various lines of evi-\ndence indicate that NAP1 and the related NAP2 protein are\nmainly cytoplasmic during G1 and G2 phase with only a small\nfraction in the nucleus and translocate into the nucleus during\nS phase (14, 19 –21).\n**BLOCK**fs== 8.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nIn vivo experiments show coimmunoprecipitation of NAP1\nwith the histones H2A and H2B but not with H3 and H4 (11,\n22). In contrast, NAP1 is capable of binding all four core his-\ntones in vitro (13, 14, 23–25). Several studies focused on the\ndetermination of the association state of NAP1, which in yeast\nhas a monomer molecular mass of 47.9 kDa. By gel filtration or\ngradient centrifugation very different molecular masses rang-\ning from 120 to 600 kDa and sedimentation coefficients from\n1.5 S up to 7 S were reported. In a recent study it was concluded\nbased on analytical ultracentrifugation experiments that\nyNAP1 exists as complex mixture of species with s values\nbetween 4.5 and 12 S at physiological salt concentrations (75\nand 150 mM NaCl). In the presence of 500 mM NaCl concentra-\ntion a 4.5 S NAP1 dimer was identified (26). It appears that\nupon mixing NAP1 with histones distinct complexes are\nformed, which sediment around 5– 6 S, 8 S, and 10 –12 S,\nrespectively (13, 23, 24). Within the 12 S complex all four\nhistones cosedimented in nearly equal amounts, and upon ad-\ndition of DNA nucleosomes were formed. The 5– 6 S complex\nwas assigned to a NAP1 complex with the H2AH2B dimer and\nthe 8 S complex to a complex that contains NAP1 and the\n(H3H4)2 tetramer (13, 23, 24). Hence, the biologically highly\nrelevant interaction of NAP1 with histones appears to be\ntightly connected to the NAP1 association state. In particular,\nthe nature of the different NAP1 complexes observed previ-\nously at physiological ionic strength and their relation to spe-\ncific NAP1 activities remain to be elucidated.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nHere, we have used analytical ultracentrifugation (AUC) to\ndetermine the different association states of NAP1 on its own\nand in complexes with histones. AUC is very useful for the\nidentification of the oligomeric state and the stoichiometry of\nproteins, and to characterize their thermodynamic and hydro-\ndynamic properties in solution (27, 28). The concentration de-\npendence of the equilibrium between different NAP1 species\nwas quantified in terms of the corresponding dissociation con-\nstants. At physiological protein and salt concentrations the\ndominating species were the NAP1 dimer and octamer, which\nwere also the association states found to interact with histones.\nBased on these results and estimates of the intracellular con-\ncentration of NAP1 we propose a model, in which the transport/\nexchange of H2AH2B dimers in G1 and G2 phase is mediated\nby NAP1 dimers. The accumulation of NAP1 in the nucleus\nduring S phase induces the association of NAP1 into an octa-\nmer complex. This species has eight histone binding sites so\nthat it could act as a carrier for multiple histone dimers as\nrequired for the de novo assembly of nucleosomes.\n**BLOCK**fs== 8.0**b==-0.0**t== 1.0**l== 0.1**r== 0.6**\nThis is an Open Access article under the CC BY license.\n**BLOCK**fs== 7.0**b== 0.9**t== 0.1**l== 0.2**r== 0.6**\nEXPERIMENTAL PROCEDURES\n**BLOCK**fs== 4.5**b== 0.5**t== 0.5**l== 0.2**r== 0.8**\n\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.2**r== 0.7**\n 37,000 M\n**BLOCK**fs== 7.5**b== 0.5**t== 0.1**l== 0.1**r== 0.5**\nExpression and Purification of NAP1—Yeast NAP1 was expressed\nfrom plasmid pET28-yNAP1 from Toshio Tsukiyama with a N-terminal\nHis tag. The plasmid corresponds to Saccharomyces cerevisiae NAP1\nwith a T7 tag cloned into the NdeI site of pET28a. The His tag is\nfollowed by a thrombin cleavage site. The expressed NAP1 protein has\na calculated molecular mass of 52.4 kDa with the His tag and 49.5 kDa\nafter thrombin cleavage. Protein overexpression from pET28-yNAP1\nwas in Escherichia coli BL21(DE3) carrying the pLysS plasmid. For\npurification, cells were suspended in lysis buffer (20 mM Tris-HCl, pH\n7.5, 500 mM KCl, 20 mM imidazole, 5 mM mercaptoethanol, 15% glyc-\nerol, 1 mM EDTA, 0. 1% Nonidet P-40, 1 mM Pefabloc), sonicated, and\ncentrifuged. Nickel-agarose beads were equilibrated with lysis buffer\nand added to the supernatant after supplementing MgCl2 to a final\nconcentration of 2 mM. Binding was allowed for 3– 4 h at 4 °C, and beads\nwere washed twice with lysis buffer and twice with washing buffer (20\nmM Tris-HCl, pH 7.5, 100 mM KCl, 20 mM imidazole, 5 mM mercapto-\nethanol, 15% glycerol, 0.01% Nonidet P-40, 1 mM Pefabloc). Elution was\nconducted with washing buffer supplemented with 0.5 M imidazole. The\neluate was dialyzed against buffer A (20 mM Tris-HCl, pH 7.5, 100 mM\nM KCl, 1 mM dithiothreitol, 10% glycerol, 0.1 mM EDTA). The protein\nwas further purified on a Mono-Q column and eluted with a KCl\ngradient from 0.1 to 1 M in buffer A. Fractions were checked on SDS-\nPAGE, and NAP1-containing fractions were pooled and dialyzed\nagainst buffer A. DNase activity was tested by incubating NAP1 with\nsupercoiled plasmid DNA over 2 days. For some experiments the His\ntag was removed by binding purified NAP1 to nickel-agarose and incu-\nbating with biotin-tagged thrombin for 2 h at 20 °Cusing the thrombin\ncleavage kit according to the specifications given by the manufacturer\n(Novagen, Madison, WI). Thrombin was subsequently removed with\nstreptavidin beads. Using the program SEDNTERP2 (29) the monomer\nextinction coefficient of NAP1 was calculated from the amino acid\n1cm1. From the absorbance spectrum\nsequence to be \nof the protein we derived the corresponding value of \n 293,000\nM\n**BLOCK**fs== 7.5**b== 0.1**t== 0.5**l== 0.1**r== 0.5**\n1cm1.\nPreparation and Gel Electrophoresis Analysis of Fluorescent Histone\nComplexes—Recombinant Xenopus laevis histones H2A12C, H2B,\nH3C110A, and H4K5C were overexpressed from pET plasmids in E. coli\nas described previously (30, 31). The H2A12C and H4K5C proteins\ncontain a cysteine substitution, allowing specific labeling with thiol-\nreactive probes (32, 33). In Histone H3C110A the cysteine is substituted\nagainst alanine to hinder modification of this residue, which could lead\nto functional impairment. Histones H2A12C and H2B or H3C110A and\nH4K5C were dissolved in unfolding buffer containing 7 M guanidinium\nchloride, 20 mM Tris-HCl, pH 7.5, 1 mM EDTA, and 5 mM mercaptoeth-\nanol at equimolar ratios and a total protein concentration of 1 mg/ml.\nRefolding was by dialysis into the same buffer but without guanidinium\nchloride. A reducing agent was removed by repeated concentration and\ndilution of histone proteins in freshly degassed labeling buffer (20 mM\nTris-HCl, pH 7.2, 0.1 mM EDTA) using Vivaspin 20 concentrators (Viva-\nscience, Hannover, Germany). Labeling was carried out at a histone\nmonomer concentration of 60 M with a 1.5 times molar excess of\nAlexa Fluor 488 C5 maleimide (Molecular Probes Europe BV, Leiden,\nNetherlands) for 1 h. The reaction was stopped by adding 10 mM\ndithiothreitol and incubating for 30 min on ice. Labeled histone com-\nplexes were then purified from misfolded histone and free dye using\nBioRex 70 resin (Bio-Rad Laboratories GmbH) as described before (32).\nConcentration of labeled histones and labeling efficiency were assessed\nby absorbance spectroscopy using an excitation coefficient of \n\n1cm1 for the Alexa 488 label (34) and the published values\n72,000 M\nfor the histones at a wavelength of 276 nm (30) after subtracting the\ncontribution of Alexa 488 at this wavelength. A labeling efficiency of\n70% was routinely reached for H2A12C and 50% for H4K5C, and the\nlabeled histones behaved identically to the wild type in terms of asso-\nciation state. Agarose gel electrophoresis of NAP1 complexes with la-\nbeled histones was conducted on 1% agarose gels in 0.11 TBE (10 mM\nTris, 10 mM boric acid, 0.22 mM EDTA) supplemented with 10 mM KCl.\nNAP1 and histones were mixed prior to loading in 10 mM Tris-HCl, pH\n7.5, 150 mM KCl and incubated for at least 10 min at room temperature.\nBands were visualized by illumination with a UV-light box at 302 nm\nand detection by using a charge-coupled device camera.\n**BLOCK**fs== 4.5**b== 0.5**t== 0.5**l== 0.4**r== 0.6**\n\n**BLOCK**fs== 4.5**b== 0.2**t== 0.8**l== 0.5**r== 0.5**\n\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nAnalytical Ultracentrifugation—Analytical sedimentation equilib-\nrium ultracentrifugation was carried out at 20 °C on a Beckman Op-\ntima XL-A analytical ultracentrifuge equipped with absorbance optics\n**BLOCK**fs== 7.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\n2 J. Philo, D. Hayes, and T. Laue, available at www.jphilo.\n**BLOCK**fs== 7.5**b== 0.0**t== 1.0**l== 0.1**r== 0.7**\nmailway.com/download.htm.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nand an An60 Ti rotor. The partial specific volume v at this temperature\nwas determined from the amino acid composition to be 0.726 mlg1 for\nNAP1, 0.746 mlg1 for the histones, and 0.730 mlg1 for the NAP1-\nhistone complexes using the program SEDNTERP version 1.052 (29).\nHydration values of 0.44 g of H2O per g of NAP1, 0.39 g of H2O per g of\nhistone, and 0.43 g of H2O per g of NAP1histone complex were calcu-\nlated as described previously (35). Proteins were dialyzed against buffer\ncontaining 10 mM Tris-HCl, pH 7.5, 5% (v/v) glycerol and 10 or 100 mM\nKCl prior to analysis. The density  of the buffer was 1.0133 gml1 (10\nmM KCl) and 1.0176 gml1 (100 mM KCl), and the viscosity  was\n1.1680 mPas (10 mM KCl) and 1.1667 mPas (100 mM KCl) at 20 °C as\ncalculated with SEDNTERP.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nSedimentation equilibrium ultracentrifugation runs were performed\nwith NAP1 concentrations ranging between 0.5 and 32 M and at\nspeeds of 5,000, 7,000, and 10,000 rpm and for some runs also at 3,000\nand 15,000 rpm. The NAP1-H2AfH2B complexes were studied with 3\nM H2AfH2B dimer (6 M histone monomer) and 6 M NAP1. In\naddition, experiments were conducted, in which the NAP1 concentra-\ntion was varied from 3 to 12 M. NAP1 and H3H4f were mixed at a 1:1\nmonomer ratio using 8.6 M H3H4f dimer and 17.3 M NAP1. Equilib-\nrium was obtained after 24 –30 h as judged from a comparison of scans\nrecorded at intervals of 4 h. Absorbance data were collected at 230 and\n280 nm by averaging 10 scans with radial increments of 0.001 cm in\nstep mode. Absorbance values above 1.3 were not included in the anal-\nysis. The sedimentation equilibrium data were analyzed by global fit-\nting of all data sets of a given sample with the software UltraScan\nversion 6.2 (www.ultrascan.uthscsa.edu) from Borries Demeler. A mon-\nomer-dimer, dimer-octamer, or dimer-octamer-hexadecamer model was\nused as described in the text. Other models that involved a NAP1 dimer\nas basic building block and included tetramers or hexamers were also\ntested but did not lead to good fits. All fits derived according to Reac-\ntions 1– 4 had a fit quality of 35% or higher, with a value of 30% being\nconsidered as an adequate model for the data.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nAnalytical sedimentation velocity studies were conducted at 20 °C\nand at 35,000 and 42,000 rpm in epon double-sector cells and in the\nsame buffer used for the equilibrium runs. The NAP1 concentration was\nbetween 1 and 4 M for the analysis of NAP1 complexes. For experi-\nments with histones 3 M H2AH2B dimer (6 M histone monomer) and\n6 M NAP1 monomer or 20 M NAP1 and 10 M H3H4f dimer (20 M\nhistone monomer) were used. Data were collected at 230, 280, or 496 nm\nusing a spacing of 0.03 cm in a continuous scan mode. A global analysis\nof the sedimentation velocity experiments was conducted with the pro-\ngram SEDPHAT version 2.0 (www.analyticalultracentrifugation.com)\n(36). The local residual mean square deviation was between 0.006 and\n0.011 for single and global fits. The same models as in the equilibrium\nruns were used with an instantaneous equilibrium between different\nassociation states corresponding to kinetic off-rate constants that are\nequal or faster than 0.1/s.\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.7**r== 0.2**\nRESULTS\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nNAP1 Associates into Dimer, Octamer, and Hexadecamer—\nYeast NAP1 was expressed in Escherichia coli and purified by\nHis tag affinity and ion exchange chromatography (Fig. 1A). A\ncontaminating DNase activity present in the eluate from the\nnickel-chelating resin was removed by a second chromatogra-\nphy step on a Mono-Q ion exchange column. After the final\npurification step the protein was more than 95% pure and the\npreparation had no detectable DNase activity as tested by\nincubation with supercoiled plasmid substrate at 37 °C over\nseveral days and analysis by agarose gel electrophoresis (data\nnot shown). For some AUC experiments the His tag of NAP1\nwas removed by thrombin digestion to test if the tag effects the\nassociation state. The His tag was cut off quantitatively under\nappropriate conditions with hardly any additional degradation\nof the protein (Fig. 1B). NAP1 preparations equivalent to those\nobtained after 6 –22 h of digestion were used in subsequent\nAUC experiments.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nBinding of NAP1 to histones was studied with histone\nH2AH2B and H3H4 complexes that were reconstituted from\noverexpressed and purified recombinant histones (30). The het-\nerodimer/tetramer complexes were prepared with the fluores-\ncent dye Alexa 488 covalently attached to either H2A or to H4\nin a site-specific manner (see “Experimental Procedures”).\nThese labeled H2Af or H4f histones displayed an additional\n**BLOCK**fs== 8.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nbinant histones H2A und H2B from X. laevis have a very\nsimilar mobility under standard electrophoretic conditions and\nare hardly separated (37).\n**BLOCK**fs== 8.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\nThe association states of NAP1 in absence of histones were\ndetermined by sedimentation equilibrium ultracentrifugation.\nTypical data sets are displayed in Fig. 2(A and B), and the\nresults are summarized in Table I. Initial runs at approxi-\nmately physiological salt concentrations (100 mM KCl, 2 mM\nMgCl2) revealed a complex multispecies equilibrium. To iden-\ntify the basic building blocks of the NAP1 multimers the salt\nconcentration was varied from 10 mM to 1 M KCl. At 10 mM KCl\nan excellent fit of the data was obtained over a concentration\nrange from 0.5 to 13.4 M NAP1 with a monomer-dimer model\nand a monomer molecular mass constrained to 51.3 kDa as\ncalculated from the amino acid composition (Fig. 2A) according\nto Reaction 1.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.7**r== 0.2**\n2 NAP1^ NAP12\n**BLOCK**fs== 5.5**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nREACTION 1\n**BLOCK**fs== 8.5**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\n 0.2  106\n**BLOCK**fs== 8.5**b== 0.4**t== 0.4**l== 0.5**r== 0.1**\nA dissociation constant of Kd1–2\nM was deter-\nmined. At NAP1 concentrations well above the dissociation\nconstant (measurements at 6.7, 8.1, and 13.4 M) the data\ncould be described well with a single component fit. A molecu-\nlar mass of 104  4 kDa was obtained with the molecular mass\nbeing the only fit parameter and using the calculated values for\nthe partial specific volume v  0.726 mlg1 of NAP1 and a\nbuffer density of   1.0133 gml1 (see “Experimental Proce-\ndures”). This result is in excellent agreement with the calcu-\nlated value of 102.7 kDa for the NAP1 dimer. Thus, NAP1\nforms a dimer in low salt buffer, which is the building block for\nthe oligomers formed at physiological salt concentrations. This\nfinding confirms a previous study, in which the NAP1 dimer\nwas identified as the basic association state (26). Accordingly,\nthe molecular mass of the NAP1 dimer of 102.7 kDa was used\nin further experiments as a fixed parameter to determine the\nhigher association states at physiological buffer conditions and\nin complexes with histones.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nAt 100 mM KCl concentration the association of NAP1 into\nlarger complexes was evident from a comparison of the molec-\nular weight averages. For example at 2 M protein concen-\ntration average values of 81  4 kDa (10 mM KCl) and 169  9\nkDa (100 mM KCl) were determined when fitting the data to a\nsingle component model. However, in these experiments sys-\ntematic deviations from a one or two component model were\nobserved in the residuals of the fits, indicating a more complex\nequilibrium state. A detailed analysis revealed that the data at\n100 mM KCl were best described by a dimer-octamer-hexa-\ndecamer equilibrium (Reaction 2).\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\n8NAP12 ^ 2NAP18 ^ NAP116\n**BLOCK**fs== 5.5**b== 0.2**t== 0.8**l== 0.7**r== 0.2**\nREACTION 2\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nOther models that involve the NAP1 dimer as the smallest unit\n(e.g. formation of tetramer or hexamer) did not lead to good fits.\nFig. 2B shows a representative data set at 100 mM KCl with a\nfit to a dimer-octamer-hexadecamer equilibrium and the dimer\nmolecular mass constrained to 102.7 kDa. Very good fits to this\nmodel were obtained over a concentration range from 0.5 M to\n32 M NAP1 monomer with data recorded at 230 and 280 nm.\n (5.8  3.9) \nValues for the dissociation constant of Kd2– 8\n1018\n (1.2  0.3)  106\nM were determined.\nThe concentration at which the ratio of NAP1 in the dimeric\nand octameric state is 1:1 is (Kd2– 8)1/3  (1.8  0.4)  106\nM.\nAt salt concentrations of 150 mM KCl, 100 mM KCl plus 2 mM\nMgCl2, or 10 mM KCl plus 2 mM MgCl2 (Table I) the data were\nalso best described with a dimer-octamer-hexadecamer equilib-\n**BLOCK**fs== 5.5**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n3 and Kd8 –16\n**BLOCK**fs== 6.0**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\nM\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nFIG. 1. A, purification of NAP1 protein from E. coli with nickel-\naffinity and ion exchange chromatography. Due to its high number of\nnegative charges the NAP1 protein displays a somewhat reduced mo-\nbility on SDS gels with an apparent molecular mass of 60 kDa. L,\nlysate; S, supernatant; P, pellet; E(Ni), eluate after affinity purification\nwith nickel-agarose beads; E(Q), eluate from Mono-Q ion exchange\ncolumn. B, cleavage of His tag from NAP1. NAP1 was bound to nickel-\nagarose beads, and thrombin cleavage was conducted for the times\nindicated. Increasing amounts of tag-free NAP1 were released to the\nsupernatant. At extended incubation times unspecific cleavage by\nthrombin resulted in faster moving bands. C, gel electrophoretic anal-\nysis of reconstituted and purified H2AfH2B dimer and (H3H4f)2 tet-\nramer. An SDS-gel is shown with visualization of the protein by Coo-\nmassie staining in comparison to the H2Af and H4f fluorescence signal\nof the same lanes. As a reference, a histone octamer is included that has\nbeen reconstituted from purified recombinant histones as described\npreviously (31). Histones H2Af and H2B are not separated on the gel.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nabsorbance maximum at 496 nm and a fluorescence maximum\nat about 520 nm allowing specific detection of NAP1-histone\ncomplexes by absorbance and fluorescence measurements. Fig.\n1C shows a Coomassie-stained SDS-gel with the purified his-\ntone H2AfH2B and H3H4f complexes and a reconstituted hi-\nstone octamer preparation as a reference. The SDS-gel demon-\nstrates that the histone preparations are highly purified and\nthat H2Af and the H4f displayed the expected fluorescence\nsignal upon excitation of the Alexa 488 fluorophore. The recom-\n**BLOCK**fs== 7.5**b== 0.0**t== 0.9**l== 0.1**r== 0.1**\nFIG. 2. Analytical ultracentrifugation of NAP1. The left panel shows the experimental data and the fitted curves in dependence of the radial\nposition. On the right the residuals of the fits are plotted. Equilibrium centrifugation experiments were carried out at 20 °C and at 5,000, 7,000,\nand 10,000 rpm as shown in A and B. Representative sedimentation velocity runs conducted at 35,000 rpm (C) and 42,000 rpm (D) are shown with\nevery fifth scan being included in the plot. A and C, NAP1 in buffer containing 10 mM KCl was fitted with a monomer-dimer equilibrium (Reaction\n1) with the molecular mass of the monomer fixed at 51.3 kDa. B and D, NAP1 in buffer containing 100 mM KCl fitted with a dimer-octamer-\nhexadecamer equilibrium (Reaction 2).\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.2**r== 0.1**\nTABLE I\nAssociation states of NAP1, histones, and NAP1-histone complexes determined by sedimentation equilibrium analysis\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nKCl (mM)\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nBasic association state\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.4**\nAbbreviation\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nc\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\n10  MgCl2\n100  MgCl2\n10/100\n10/100\n10/100\n10/100\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.2**r== 0.8**\nc\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.2**r== 0.6**\nNAP1 monomer\nNAP1 dimer\nNAP1 dimer\nNAP1 dimer\nH2AH2B dimer\nNAP1 dimer with H2AH2B dimer\nH3H4 dimer\nNAP1 dimers with H3H4 dimer\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.4**\nNAP1\n(NAP1)2\n(NAP1)2\n(NAP1)2\n(H2AfH2B)1\n(NAP1)2-(H2AfH2B)1\n(H3H4f)1\n(NAP1)2-(H3H4f)1\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\na\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nMfit\n(kDa)\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\n51.3b\n102.7\n102.7\n102.7\n28.5d\n131.2\n27.2e\n129.8\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nEquilibrium complexes\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.7**r== 0.1**\nMonomerdimerb\nDimer-octamer-hexadecamer\nDimer-octamer-hexadecamer\nDimer-octamer-hexadecamer\nDimerd\nDimer-octamer\nDimere\nDimer-tetramer-octamer\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\na Molecular masses were fixed in the analysis with equilibrium association models at these values calculated from the amino acid sequence.\nb A molecular mass of 103.8  1 kDa demonstrating the formation of a dimer was determined from a one component fit for NAP1 at\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.7**\nconcentrations of 6.7 M and higher.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\nc MgCl2 was added at a concentration of 2 mM.\nd A molecular mass of 27.5  4 kDa was determined for H2AfH2B dimer from measurements at 10 mM and 100 mM KCl with a one-component\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.9**\nfit.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.1**\ne Under the conditions of the experiments most of the H3H4 was present in the dimer state. A molecular mass of 29.7  5 kDa was determined\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nin equilibrium runs at 10 and 100 mM KCl from a one-component fit.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nrium with no apparent changes of the dissociation constants.\nRaising the KCl concentration above 0.5 M led to the dissocia-\ntion of the complex into increasing amounts of monomeric\nNAP1 indicating that the formation of the NAP1 dimer in-\nvolves predominantly electrostatic interactions that are weak-\nened by high salt concentrations. At 500 mM KCl NAP1 was\npresent almost completely as a dimer in agreement with re-\nsults reported recently (26).\n**BLOCK**fs== 8.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nThe possible effect of the His tag on the NAP1 association\nwas examined by cleavage of the tag with thrombin (Fig. 1B).\nThis construct was studied by equilibrium analytical ultracen-\ntrifugation experiments at 100 mM KCl and at NAP1 monomer\nconcentrations of 2, 4, 8, 12, 16, and 32 M. Within the accuracy\nof measurement no differences were observed in the average\nmolecular weight or multimer equilibrium distribution (data\nnot shown). Best fits were also obtained with a dimer-octamer-\nhexadecamer equilibrium and similar dissociation constants as\nthose determined for the His-tagged NAP1. Thus, the His tag\nhad no detectable effect on the association properties of NAP1.\nBecause a significant fraction of the protein was lost during the\nadditional purification after thrombin digestion of NAP1, the\nsubsequent experiments were conducted with His-tagged\nNAP1.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nNAP1 complexes were also studied by sedimentation velocity\nanalysis to determine the sedimentation coefficient and hydro-\ndynamic shape of the complexes. The salt-dependent associa-\ntion of NAP1 was clearly evident in the sedimentation velocity\nanalysis when computing the weight average sedimentation\ncoefficient. At a protein concentration of 2 M values of 3.8 \n0.3 S at 10 mM KCl and 6.5  0.6 S at 100 mM KCl were\ndetermined. Again the data were indicative of the presence of\nseveral species. The sedimentation coefficients of the single\ncomponents in the NAP1 multispecies equilibrium system ac-\ncording to Reaction 1 or Reaction 2 were determined with the\nprogram SEDPHAT from Peter Schuck (36). The resulting val-\nues of the sedimentation coefficients s measured at 20 °C are\ngiven in Table II. Representative sedimentation velocity plots\nare shown for 10 mM KCl (Fig. 2C) and 100 mM KCl (Fig. 2D).\nThe sedimentation coefficients of the monomer and dimer were\ndetermined at 10 mM KCl using a monomer-dimer equilibrium\nmodel (Reaction 1). The monomer had an s value of 2.4  0.2 S,\nwhereas the dimer yielded 4.4  0.2 S. At 100 mM KCl a\ndimer-octamer-hexadecamer equilibrium (Reaction 2) was\nused and sedimentation coefficients of 11.4  0.8 S (octamer)\nand 21.3  0.5 S (hexadecamer) were measured.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nNAP1 Binds Histones in a 1:1 Stoichiometry—Initial at-\ntempts to analyze the association of NAP1 with recombinant\nhistones H2AH2B and H3H4 by analytical ultracentrifugation\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nyielded no clear results. The complex dimer-octamer-hexa-\ndecamer equilibrium of NAP1 precluded the identification of\nadditional species with bound histones. Therefore, recombi-\nnant histone complexes were prepared with the fluorescent dye\nAlexa 488 covalently attached to either H2A or to H4 in a\nsite-specific manner. These labeled histones H2Af or H4f al-\nlowed a specific detection of NAP1histone complexes both by\nabsorbance and fluorescence measurements.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\nThe initial\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\ncharacterization of\n**BLOCK**fs== 8.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\ncomplex formation of\nH2AfH2B and H3H4f with NAP1 was conducted using elec-\ntrophoresis mobility shift assays. Increasing amounts of NAP1\nwere mixed with labeled histone complexes (H2AfH2B or\nH3H4f) and analyzed on agarose gels exploiting the H2Af or\nH4f fluorescence signal (Fig. 3). Free histones migrate into the\nopposite direction due to their highly positive charge. Accord-\ningly, they were not visible in the gel, except for nonspecific,\nhardly migrating aggregates in samples, in which an excess of\nhistones was present (Fig. 3, lanes with 0, 0.4, and 0.8 M\nNAP1). Upon addition of NAP1 to histones, a specific complex\nformed and reached saturation at a ratio of approximately one\nNAP1 monomer per histone, both for H2AfH2B (Fig. 3A) and\nH3H4f (Fig. 3B). Thus, one NAP1 protein binds one histone in\ngood agreement with previous studies (25). It is further noted\nthat NAP1-H2AfH2B and NAP1-H3H4f complexes migrated\nat an apparent identical height, indicative for the formation of\ncomplexes of similar size and charge.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nNAP1 Dimer and Octamer Form Complexes with H2AH2B\nDimers—The AUC analysis was conducted with absorbance\ndata recorded at 496 nm. Thus, only the species containing\nH2AfH2B dimer was detected. For the H2AfH2B dimer alone\na molecular mass of 29  4 kDa was determined in equilibrium\nruns, which is in very good agreement with the 28.5 kDa\nderived from the amino acid sequence. NAP1 and histone dimer\nwere mixed at different molar ratios. Only at a ratio of one\nNAP1 monomer per histone monomer a good fit could be ob-\ntained with a dimer-octamer model for sedimentation equilib-\nrium and velocity experiments at both 10 mM and 100 mM KCl\naccording to Reaction 3 (Fig. 4).\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.6**r== 0.1**\n4NAP12-H2AH2B1 ^ NAP18-H2AH2B4\n**BLOCK**fs== 5.5**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nREACTION 3\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nA NAP1 hexadecamer species with H2AfH2B was not detected\nin the experiments. Using a dimer-octamer-hexadecamer\nmodel did not improve the fit, and the Kd8 –16 values indicated\nnegligible amounts of hexadecamer. Interestingly, already at\n10 mM KCl the octamer complex was present, suggesting a\nstabilization of the NAP1 octamer upon histone binding at this\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.4**r== 0.4**\nTABLE II\nHydrodynamic parameters of NAP1 complexes\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.3**r== 0.7**\nsa\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.4**r== 0.5**\nMolecular massb\n**BLOCK**fs== 4.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nc\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\nf/fo\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.7**r== 0.3**\nD  107d\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.8**r== 0.1**\nModel fit Reactiona\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.1**\nNAP1\n(NAP1)2\n(NAP1)8\n(NAP1)16\n(NAP1)2-(H2AH2B)1\n(NAP1)8-(H2AH2B)4\n(NAP1)2-(H3H4)1\n(NAP1)4-(H3H4)2\n(NAP1)8-(H3H4)4\na The sedimentation coefficient at standard conditions (20 °C, H2O) as determined from single and global fits to models described by Reactions\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.6**r== 0.4**\n1.80  0.16\n1.56  0.07\n1.52  0.10\n1.29  0.05\n1.45  0.16\n1.21  0.14\n1.44  0.16\n1.15  0.14\n1.19  0.14\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.9**r== 0.1**\n1, 2\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\nS\n2.4  0.2\n4.4  0.2\n11.4  0.8\n21.3  0.5\n5.5  0.7\n16.6  2.2\n5.5  0.7\n10.9  1.6\n16.6  2.2\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.7**r== 0.3**\ncm2 s1\n4.1  0.4\n3.8  0.4\n2.5  0.2\n2.3  0.1\n3.8  0.4\n2.8  0.4\n3.8  0.4\n3.8  0.5\n2.9  0.4\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.4**r== 0.5**\nkDa\n51.3\n102.6\n410.4\n820.8\n131.1\n524.4\n129.8\n259.6\n519.2\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.2**\n1– 4 as indicated using SEDPHAT. Standard deviations were determined from averaging results from single fits.\nb Calculated molecular masses are included as a fixed parameter in the analysis of sedimentation velocity runs.\nc Ratio of the measured friction coefficient f to the friction coefficient fo of a sphere with the same volume, including hydration.\nd Diffusion constants at standard conditions (20 °C, H2O) determined from the friction coefficient (35).\n**BLOCK**fs== 8.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nspecies. For H3H4f alone a molecular mass of 29.7  5 kDa was\ndetermined in equilibrium runs at 10 and 100 mM KCl with a\ngood fit to a one component model. This value is very close to\nthe molecular mass of 27.2 kDa calculated for the dimer, and\nthe fit to a monomer-dimer equilibrium indicated that under\nthese experimental conditions only a small fraction (15%)\nassociates into the (H3H4f)2 tetramer. This is consistent with\nprevious studies that reported the dissociation of (H3H4)2\ntetramer into dimers at low ionic strength and protein concen-\ntration in the micromolar range (39, 40). The formation of a\nstable (H3H4)2 tetramer would require higher protein and salt\nconcentrations than those used here (40).\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nNAP1 and histones were mixed in equimolar amounts and\nexamined by sedimentation equilibrium (Fig. 5A) and velocity\ncentrifugation (Fig. 5B) at 10 mM and 100 mM KCl. All data sets\nshowed a good fit to a dimer-tetramer-octamer model with a\nfixed molecular mass of 129.8 kDa for the (NAP1)2-(H3H4f)1\ndimer complex as described in Reaction 4.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n4NAP12-H3H41 ^ 2NAP14-H3H42 ^ NAP18-(H3H4)4\n**BLOCK**fs== 5.5**b== 0.4**t== 0.6**l== 0.7**r== 0.2**\nREACTION 4\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThus, H3H4f behaves very similar to H2AfH2B in its interac-\ntion with NAP1 but in addition to the dimer and octamer\ncomplex an association state forms, in which two NAP1 dimers\nstabilize the (H3-H4f)2 tetramer complex. At 10 mM KCl this\ncomplex was the prevalent species, and little octamer complex\nwas present, whereas at 100 mM KCl the concentration of the\n(NAP1)4-(H3H4f)2 complex was low and the (NAP1)8-(H3H4f)4\n 2 \ncomplex formed with a dissociation constant of Kd2– 8\n1017\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.6**r== 0.2**\n3 and (Kd2– 8)1/3  3  106\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nSedimentation coefficients for the (NAP1)2-(H3H4f) and\n(NAP1)8-(H3H4f)4 complexes were found to be equivalent to\nthose of the corresponding complexes with H2AfH2B within\nthe accuracy of the measurement. Accordingly, the values for\nthese association states were averaged resulting in s  5.5 \n0.7 S (dimer) and s  16.6  2.2 S (octamer). The sedimentation\ncoefficient of the (NAP1)4-(H3H4f)2 complex was determined to\nbe 10.9  1.6 S (Table II).\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.8**r== 0.2**\nM.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\nM\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nConformations of NAP1 Complexes Are Derived from Hydro-\ndynamic Measurements—From the sedimentation coefficients\nmeasured for the NAP1 complexes the ratio of the friction\ncoefficient f to that of a sphere with the same volume and\nfriction coefficient fo was calculated (Table II). The relatively\nhigh f/fo ratios indicate that the conformation of NAP1 dimer\nand octamer differ significantly from a sphere for which f/fo\nwould equal one. Upon binding of histones to NAP1 dimer only\na small decrease of the friction coefficient ratio was observed\nfrom 1.52  0.10 to 1.45  0.16 for the (NAP1)2-(H2AH2B)1\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nFIG. 3. Gel shift analysis of NAP1 with the H2AH2B dimer and\nthe (H3H4)2 tetramer. A, increasing concentrations of NAP1 were\nmixed with 1.0 M H2AfH2B dimer. NAP1 was completely saturated\nwith histones at a ratio of 1 NAP1 monomer per histone. B, increasing\nconcentrations of NAP1 were mixed with 1.0 M H3H4f dimer. At a\nratio of 1 NAP1 monomer per histone NAP1 was completely saturated\nwith histones.\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nM\n**BLOCK**fs== 6.0**b== 0.3**t== 0.7**l== 0.4**r== 0.5**\nM\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n 5  1016\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.3**r== 0.6**\n 3  1018\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\n3 and\nsalt concentration. A value of Kd2– 8\n(Kd2– 8)1/3  8  106\nM was measured. At 100 mM KCl the\n3 and\nmultimerization occurred with Kd2– 8\n(Kd2– 8)1/3  1.4  106\nM. This is very similar to the NAP1\ndimer-octamer equilibrium in the absence of histones showing\nthat the binding of H2AfH2B dimer had little effect on the\nformation of the NAP1 octamer at 100 mM KCl. The Histone\nH2AfH2B dimer interacted with NAP1 with high affinity, be-\ncause no indication for the presence of free H2AfH2B dimer\nwas found at a 1:1 stoichiometry of NAP1 and histone. This is\nconsistent with a NAP1histone dissociation constant of 20 nM\nat 0.1 M ionic strength as estimated from affinity blotting\nexperiments (38).\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nTo determine the sedimentation coefficients of the NAP1-\nH2AfH2B complexes sedimentation velocity runs were per-\nformed and analyzed according to Reaction 3. A representative\nfit at 10 mM KCl is shown in Fig. 4C. The sedimentation\ncoefficient of the (NAP1)2-(H2AfH2B)1 dimer complex was\nmeasured to be 5.5  0.7 S and for the (NAP1)8-(H2AfH2B)4\noctamer complex a value of s  16.6  2.2 S was determined\n(Table II).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nNAP1 Dimer and Octamer Form Complexes with H3H4 —\nThe analysis of NAP1 complexes in the presence of H3H4 was\nalso conducted at 496 nm to detect only histone-containing\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.1**\nFIG. 4. Analytical ultracentrifugation of NAP1 complexes with histone H2AfH2B dimer. Experiments were recorded at 496 nm. The left\npanel shows the experimental data and the fitted curves in dependence of the normalized radial position. On the right the residuals of the fits are\nplotted. Runs were fitted to the model given in Reaction 3 with the molecular mass of the (NAP1)2-(H2AH2B)1 species constrained to 131.2 kDa.\nData acquired at 10 mM KCl (A) and 100 mM KCl (B) are shown. C, sedimentation velocity run at 10 mM and a NAP1/histone ratio of 1:1. Only every\nfifth scan is shown.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nand to 1.44  0.16 for the (NAP1)2-(H3H4)1 species. In con-\ntrast the NAP1 octamer complex adopted a more globular\nshape when associated with histones, because the value of f/fo\ndecreased significantly. In analogy to the stacked annular pen-\ntamer-decamer structure formed by the histone chaperone\nnucleoplasmin (9, 10, 41) we propose the model depicted in Fig.\n6A. It is consistent with the hydrodynamic parameters derived\nfrom the sedimentation velocity analysis, but it is noted also\nthat other arrangements of NAP1 subunits would be compati-\nble with the measured friction coefficient ratios. In our model\nthe NAP1 octamer forms an annular structure, which can stack\nto form a hexadecamer. The disk-shaped structure of the de-\npicted octamer has a relatively high friction coefficient, and\nstacking of two octamers would lead to a reduction of the f/fo\nratio as observed in the sedimentation velocity runs, which\n 1.29 \nyielded values of f/fo\n0.05 (hexadecamer) (Table II). Upon histone binding the NAP1\noctamer undergoes a transition to a more compact association\nstate as indicated in Fig. 6A by an accompanying decrease of\nthe central cavity.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.8**l== 0.2**r== 0.6**\n 1.52  0.10 (octamer) and f/fo\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nThe friction coefficients and the known molecular weights\ncan be used to calculate the diffusion constant D. The values for\nD in pure water are listed in Table II. From mobility measure-\nments of other proteins like GFP it is estimated that in the cell\nthe corresponding in vivo value would be 3 to 4 times lower due\nto the higher viscosity in the absence of any topological con-\nstraints to the mobility imposed by chromatin or other cellular\nstructures (42).\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nNAP1 Octamer Formation Could Be Induced in Vivo during\nS Phase by NAP1 Accumulation in the Nucleus—The number of\nNAP1 monomers in haploid yeast cells has been determined to\nbe around 8070 molecules in microarray experiments (43). Dur-\ning G1 phase NAP1 is mostly excluded from the nucleus (21,\n44). A haploid yeast cell has an average cell volume of 32 m3\n(45). Estimating that about half of the cell volume is occupied\nby organelles like nucleus, vacuole, mitochondria, and the\nGolgi apparatus that are inaccessible to NAP1 this would cor-\nrespond to a concentration of 1 M NAP1 monomer or 0.5\nM NAP1 dimer. From the dissociation constants Kd2– 8 and\nKd8 –16 determined in the AUC experiments it can be concluded\nthat during G1 phase NAP1 is present in the cytoplasm mostly\nas a dimer. As plotted in Fig. 6B the fraction of NAP1 present\nin the dimer, octamer, and hexadecamer state would be\n94:6:0.1%.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nDuring S phase dephosphorylation of NAP1 leads to its ac-\ncumulation in the nucleus (14, 19) with some increase in the\nexpression level (18) resulting in a significantly higher NAP1\nconcentration. The yeast nucleus has a total volume of 3.6\nm3 of which not more than 2.9 m3 are estimated to be\naccessible for NAP1 (46, 47). Without considering any increase\nin the expression level a concentration of 4 M NAP1 monomer\ncould be reached if 80% of NAP1 would be localized in the\nnucleus. This would favor the association of NAP1 dimer into\noctamer and hexadecamer. For a 2 M NAP1 dimer concentra-\ntion the ratio of dimer:octamer:hexadecamer is 50:38:12% (Fig.\n6B). Taking into consideration that upon histone binding the\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.1**\nFIG. 5. Analytical ultracentrifugation of NAP1 complexes with histone (H3H4f)2 tetramer. Experiments were recorded at 496 nm. The\nleft panel shows the experimental data and the fitted curves in dependence of the normalized radial position. On the right the residuals of the fits\nare plotted. Equilibrium runs were fitted to the model given in Reaction 4 with the molecular mass of the (NAP1)2-(H3H4f)1 species constrained\nto 129.8 kDa. The data sets shown were acquired at 100 mM KCl by equilibrium sedimentation (A) and velocity sedimentation (B), where only every\nfifth scan is shown.\n**BLOCK**fs== 8.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\nhexadecamer was destabilized and not detected in the AUC\nexperiments the fraction of NAP1 in dimeric and octameric\ncomplexes would be 53:47%.\n**BLOCK**fs== 7.0**b== 0.4**t== 0.6**l== 0.3**r== 0.7**\nDISCUSSION\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nNAP1 mediates various important biological activities like\nthe rearrangement of nucleosomes during transcription, the\nshuttling of the histone H2AH2B dimer from the cytoplasm to\nthe nucleus, and the assembly of newly synthesized DNA into\nchromatin (see Refs. 5–7, 10 for reviews). These diverse func-\ntions of NAP1 are likely to differ with respect to the number\nand type of histones found in complexes with NAP1. An under-\nstanding of the different NAP1 association states is therefore\nan essential prerequisite for any mechanistic studies of NAP1\nactivities. Considerable efforts have been made to characterize\nthe multimer equilibrium of NAP1 alone and in complexes with\nhistones (13, 14, 23, 25, 26). However, the physiologically rel-\nevant association states adopted by NAP1 remained to be elu-\ncidated. Here analytical ultracentrifugation was used to iden-\ntify the complexes of NAP1 alone and with core histones, and to\ncharacterize these with respect to their thermodynamic stabil-\nity and hydrodynamic shape.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nThe NAP1 monomer was only observed at unphysiologically\nlow ionic strength and associates into a dimer with high affin-\nity if the salt concentration is raised. At physiological ionic\nstrength an equilibrium between NAP1 dimer, octamer, and\nhexadecamer was present (Table I). The measured sedimenta-\ntion coefficients of the monomer (2.4  0.2 S) and dimer (4.4 \n0.2 S) determined in our analysis are in good agreement with\nthe values measured recently in the presence of 500 mM KCl\nand 1.8 M guanidinium hydrochloride (26). In addition, a NAP1\noctamer and hexadecamer were identified here with s values of\n11.4  0.8 and 21.3  0.5 (Table II). The data summarized in\nTables I and II lead to the model shown in Fig. 6A, in which the\nNAP1 dimer associates into a disk-shaped annular octamer.\nThe hexadecamer is formed by stacking two octamers on top of\neach other. This conformation is consistent with the large re-\n**BLOCK**fs== 8.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n 1.52  0.10\nduction of the friction coefficient ratio from f/fo\n 1.29  0.05 (hexadecamer). Because these\n(octamer) to f/fo\nconformations are similar to the pentamer-decamer structure\nformed by the histone chaperone nucleoplasmin (9, 10, 41),\nthey appear the most likely, although other shapes and ar-\nrangements of NAP1 subunits would also be compatible with\nthe observed friction coefficient ratios.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nThe interactions between NAP1 and core histones were in-\nvestigated first in gel shift experiments, and a stoichiometry of\n1:1 for NAP1-histone complexes was measured, which confirms\nthe results reported previously (25). The NAP1-histone com-\nplexes have been described to sediment between 5 and 12 S (13,\n14, 23, 24). Here, the species that predominantly form under\nphysiological salt and protein concentrations were identified as\na NAP1 dimer bound to a histone dimer (5.5  0.7 S) and a\nNAP1 octamer-histone complex (16.6  2.2 S) (Table II). No\nsignificant shifts in the equilibrium between the NAP1 dimer\nand the NAP1 octamer were observed upon histone binding\nunder these conditions.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nBased on AUC results and estimates of the intracellular\nconcentration of NAP1 the model shown in Fig. 7 was devised\nfor the cell cycle-dependent formation of different NAP1 asso-\nciation states. During transcription chromatin regions have to\nadopt a more open conformation. Removal of one H2AH2B\ndimer from the histone octamer seems to be essential for tran-\nscription elongation through nucleosomes (48, 49), and NAP1\nand other histone chaperones stimulate the binding of tran-\nscription factors to chromatin templates (50, 51). In addition,\nthe histone H2AH2B dimer exchanges more rapidly than the\nH3H4 tetramer in vivo (52, 53). It has been shown that NAP1\nis present in complexes with SWR1 that catalyzed the ex-\nchange of H2AH2B dimer to a dimer variant in yeast (54).\nThus, as shown in Fig. 7, mediating the dissociation and re-\nbinding of single H2AH2B dimers during transcription in G1\nphase is a likely function of NAP1 dimers that could explain its\neffect on gene expression (15). During DNA replication in S\n**BLOCK**fs== 7.5**b== 0.5**t== 0.3**l== 0.5**r== 0.1**\nFIG. 7. Hypothetical model for the cell cycle-dependent asso-\nciation states of NAP1. During G1 phase NAP1 is mainly phospho-\nrylated and thus cytoplasmic. Low concentrations of NAP1 present as\ndimers in the nucleus participate in chromatin rearrangement during\ntranscription regulation and bind free H2AH2B dimers. For this func-\ntion a dimer that can carry the H2AH2B dimer would be sufficient.\nDuring transition into S phase NAP1 is dephosphorylated and accumu-\nlates in the nucleus leading to higher concentrations. This induces the\nassociation of NAP1 into the octameric state, which is able to carry\nmultiple H2AH2B dimers and possibly also H3H4 dimers to the rep-\nlication sites. At the end of S phase NAP1 gets phosphorylated again, is\nexported to the cytoplasm, and dissociates to a dimer.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\n(Figs. 6 and 7). This complex has eight histone binding sites,\nwhich could facilitate the de novo assembly of nucleosomes by\nbinding two H2AH2B dimers and two H3H4 dimers, sufficient\nfor the assembly of a complete histone octamer. It is noted that\nthe in vitro experiments presented here and in the literature\nshow high affinity binding of NAP1 to all core histones (13, 14,\n23–25). However, an in vivo interaction of NAP1 with histones\nH3H4 has not been demonstrated and only NAP1-H2AH2B\ncomplexes have been isolated by immunoprecipitation experi-\nments (11, 14, 22). This suggests that the NAP1 interaction\nwith H3H4 is prevented by the association of H3H4 dimers\nwith histone chaperone CAF-1 (5, 39, 59) and/or DNA. How-\never, NAP1 is sufficient for the in vitro assembly of nucleo-\nsomes and can serve as a carrier for all four core histones in the\nreaction (60). This process is of considerable interest for the\nreconstitution of chromatin with defined histone composition.\nIt is also conceivable that the NAP1-mediated assembly of\ncomplete nucleosomes is taking place in vivo under specific\nconditions that remain to be identified.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nIn summary, NAP1 has various important activities like the\nimport of H2AH2B into the nucleus and the assembly of two\nH2AH2B dimers and possibly also H3H4 into nucleosomes\nduring replication. Furthermore, it interacts with the histone\nH2AH2B dimer during transcription and histone exchange.\nThese processes differ with respect to the number of histones\ninvolved. Thus, the quantitative description of the association\nstates of NAP1 alone and in complexes with histones presented\nhere provides new insights into the mechanisms by which\nNAP1 can exert its different biological functions.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nAcknowledgments—The support of Peter Lichter is gratefully ac-\nknowledged. We thank Felix Kepert, Malte Wachsmuth, Lutz\nEhrhardt, and Borries Demeler for help and Tamas Fischer for valuable\ndiscussions. We are grateful to Karolin Luger, Gernot Laengst, and\nTom Owen-Hughes for providing plasmid vectors. The project was sup-\nported by the Volkswagen Foundation in the program “Junior Research\nGroups at German Universities.”\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.7**r== 0.2**\nREFERENCES\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n1. Arents, G., Burlingame, R. W., Wang, B.-C., Love, W. E., and Moudrianakis,\n**BLOCK**fs== 6.5**b== 0.0**t== 0.9**l== 0.6**r== 0.1**\nE. N. (1991) Proc. Natl. Acad. Sci. U. S. A. 88, 10148 –10152\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.5**r== 0.1**\n2. Luger, K., Ma¨ der, A. W., Richmond, R. K., Sargent, D. F., and Richmond, T. J.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nFIG. 6. A, model of association states of NAP1. At low salt concentra-\ntion (10 mM KCl) NAP1 is present as a monomer and a dimer. At higher\nsalt concentration (100 mM) the smallest association state is the NAP1\ndimer. This associates further to an octamer and a hexadecamer with\nthe indicated Kd values. The values in parentheses refer to (Kd2– 8)1/3, the\nconcentration at which the ratio of NAP1 in the dimeric and octameric\nstate is 1:1. The Kd values for multimerization of complexes with his-\ntones are similar to those of free NAP1. However, no hexadecamer was\ndetected in the presence of histones. The NAP1 octamer-histone com-\nplex adopts a more compact conformation than the free NAP1 octamer\nas inferred from the hydrodynamic analysis. H3H4 behaves essentially\nas H2AH2B except for an additional intermediate complex formed\npredominantly at 10 mM KCl as described in Reaction 4. B, dependence\nof the three NAP1 species on total NAP1 concentration. At 1 M NAP1\nmonomer concentration the protein is mostly present as a dimer with a\nratio of dimer:octamer:hexadecamer of 94:6:0.1 (dotted line). In con-\ntrast, at around 4 M NAP1 monomer half of the protein is in larger\ncomplexes with relative dimer:octamer:hexadecamer fractions of 50:\n38:12 (dashed line).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nphase histones are needed in large quantities in the nucleus.\nUpon dephosphorylation during the G1/S transition NAP1\nshuttles histones into the nucleus (14, 19 –21, 55). In some\ncases also a higher expression of NAP1 was observed during\nS-phase (16, 18). Due to the accumulation of NAP1 in the\nnucleus its local concentration increases, leading to a signifi-\ncant increase of the NAP1 octamer complex fraction (Fig. 6B).\nBased on these results, it is estimated that for a concentration\nof 2 M NAP1 dimer in the nucleus 50% would be present in\nthe octamer complex. The actual fraction of the NAP1 octamers\nmight be higher in vivo due to excluded volume effects that are\nreferred to as macromolecular crowding (56). Furthermore, the\nNAP1 dimer-octamer equilibrium could be directly affected by\npost-translational modifications like polyglutamylation (57),\nphosphorylation, or acetylation. Acetylation of NAP1 by p300\nwas shown to promote its ability to assemble chromatin and\nfacilitates the transfer of H2AH2B from nucleosomes to NAP1.\nAcetylation occurs in a cell cycle-dependent manner (19, 58)\nsuggesting a possible regulatory function in histone binding\nand complex assembly. Here it is proposed that during S-phase\nthe nuclear NAP1 complex is present mostly as an octamer\n**BLOCK**fs== 6.5**b== 0.9**t== 0.1**l== 0.1**r== 0.7**\n(1997) Nature 389, 251–260\n**BLOCK**fs== 6.5**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\n31. Kepert, J. F., Fejes To´th, K., Caudron, M., Mu¨ cke, N., Langowski, J., and\n**BLOCK**fs== 6.5**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\n3. Davey, C. A., Sargent, D. F., Luger, K., Maeder, A. W., and Richmond, T. J.\n**BLOCK**fs== 6.5**b== 0.9**t== 0.1**l== 0.6**r== 0.2**\nRippe, K. (2003) Biophys. J. 85, 4012– 4022\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\n(2002) J. Mol. Biol. 319, 1097–1113\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n32. Lee, K. M., and Hayes, J. J. (1997) Proc. Natl. Acad. Sci. U. S. A. 94,\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n4. Harp, J. M., Hanson, B. L., Timm, D. E., and Bunick, G. J. (2000) Acta\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n8959 – 8964\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nCrystallogr. D Biol. Crystallogr. 56, 1513–1534\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n5. Loyola, A., and Almouzni, G. (2004) Biochim. Biophys. Acta 1677, 3–11\n6. Tyler, J. K. (2002) Eur. J. Biochem. 269, 2268 –2274\n7. Adams, C. R., and Kamakaka, R. T. (1999) Curr. Opin. Genet. Dev. 9, 185–190\n8. Haushalter, K. A., and Kadonaga, J. T. (2003) Nat. Rev. Mol. Cell. Biol. 4,\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\n613– 620\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n9. Prado, A., Ramos, I., Frehlick, L. J., Muga, A., and Ausio, J. (2004) Biochem.\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nCell Biol. 82, 437– 445\n**BLOCK**fs== 6.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n10. Akey, C. W., and Luger, K. (2003) Curr. Opin. Struct. Biol. 13, 6 –14\n11. Nakagawa, T., Bulger, M., Muramatsu, M., and Ito, T. (2001) J. Biol. Chem.\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n276, 27384 –27391\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n12. van Holde, K. E. (1989) Chromatin, p. 416, Springer, Heidelberg\n13. Ishimi, Y., Hirosumi, J., Sato, W., Sugasawa, K., Yokota, S., Hanaoka, F., and\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nYamada, M. (1984) Eur. J. Biochem. 142, 431– 439\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n14. Ito, T., Bulger, M., Kobayashi, R., and Kadonaga, J. T. (1996) Mol. Cell. Biol.\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n16, 3112–3124\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n15. Ohkuni, K., Shirahige, K., and Kikuchi, A. (2003) Biochem. Biophys. Res.\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\nCommun. 306, 5–9\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n16. Dong, A., Zhu, Y., Yu, Y., Cao, K., Sun, C., and Shen, W. H. (2003) Planta 216,\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.1**r== 0.8**\n561–570\n**BLOCK**fs== 6.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\n17. Zimmerman, Z. A., and Kellogg, D. R. (2001) Mol. Biol. Cell 12, 201–219\n18. Mortensen, E. M., McDonald, H., Yates, J., 3rd, and Kellogg, D. R. (2002) Mol.\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\nBiol. Cell 13, 2091–2105\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n19. Asahara, H., Tartare-Deckert, S., Nakagawa, T., Ikehara, T., Hirose, F.,\nHunter, T., Ito, T., and Montminy, M. (2002) Mol. Cell. Biol. 22, 2974 –2983\n20. Rodriguez, P., Munroe, D., Prawitt, D., Chu, L. L., Bric, E., Kim, J., Reid, L. H.,\nDavies, C., Nakagama, H., Loebbert, R., Winterpacht, A., Petruzzi, M. J.,\nHiggins, M. J., Nowak, N., Evans, G., Shows, T., Weissman, B. E., Zabel, B.,\nHousman, D. E., and Pelletier, J. (1997) Genomics 44, 253–265\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n21. Miyaji-Yamaguchi, M., Kato, K., Nakano, R., Akashi, T., Kikuchi, A., and\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nNagata, K. (2003) Mol. Cell. Biol. 23, 6672– 6684\n**BLOCK**fs== 6.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n22. Chang, L., Loranger, S. S., Mizzen, C., Ernst, S. G., Allis, C. D., and Annun-\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\nziato, A. T. (1997) Biochemistry 36, 469 – 480\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n23. Ishimi, Y., Sato, W., Kojima, M., Sugasawa, K., Hanaoka, F., and Yamada, M.\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.7**\n(1985) Cell Struct. Funct. 10, 373–382\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n24. Fujii-Nakata, T., Ishimi, Y., Okuda, A., and Kikuchi, A. (1992) J. Biol. Chem.\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\n267, 20980 –20986\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n25. McBryant, S. J., Park, Y. J., Abernathy, S. M., Laybourn, P. J., Nyborg, J. K.,\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.6**\nand Luger, K. (2003) J. Biol. Chem. 278, 44574 – 44583\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n26. McBryant, S. J., and Peersen, O. B. (2004) Biochemistry 43, 10592–10599\n27. Laue, T. M., and Stafford, W. F., 3rd. (1999) Annu. Rev. Biophys. Biomol.\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nStruct. 28, 75–100\n**BLOCK**fs== 6.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n28. Lebowitz, J., Lewis, M. S., and Schuck, P. (2002) Protein Sci. 11, 2067–2079\n29. Laue, T. M., Shah, B. D., Ridgeway, T. M., and Pelletier, S. L. (1992) in\nAnalytical Ultracentrifugation in Biochemistry and Polymer Science (Har-\nding, S. E., Rowe, A. J., and Horton, J. C., eds) pp. 90 –125, Royal Society of\nChemistry, Cambridge\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n30. Luger, K., Rechsteiner, T. J., and Richmond, T. J. (1999) Methods Enzymol.\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\n304, 3–19\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n33. Bruno, M., Flaus, A., Stockdale, C., Rencurel, C., Ferreira, H., and Owen-\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nHughes, T. (2003) Mol. Cell 12, 1599 –1606\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n34. Haugland, R. P. (2004) Handbook of Fluorescent Probes and Research Chem-\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nicals, 9th Ed., pp. 20 –36, Molecular Probes, Eugene, OR\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n35. Rippe, K., Mu¨ cke, N., and Schulz, A. (1998) J. Mol. Biol. 278, 915–933\n36. Schuck, P. (2003) Anal. Biochem. 320, 104 –124\n37. Luger, K., Rechsteiner, T. J., Flaus, A. J., Waye, M. M., and Richmond, T. J.\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n(1997) J. Mol. Biol. 272, 301–311\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n38. McQuibban, G. A., Commisso-Cappelli, C. N., and Lewis, P. N. (1998) J. Biol.\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.6**r== 0.3**\nChem. 273, 6582– 6590\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n39. Tagami, H., Ray-Gallet, D., Almouzni, G., and Nakatani, Y. (2004) Cell 116,\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n51– 61\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n40. Baxevanis, A. D., Godfrey, J. E., and Moudrianakis, E. N. (1991) Biochemistry\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n30, 8817– 8823\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n41. Dutta, S., Akey, I. V., Dingwall, C., Hartman, K. L., Laue, T., Nolte, R. T.,\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.6**r== 0.2**\nHead, J. F., and Akey, C. W. (2001) Mol. Cell 8, 841– 853\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n42. Wachsmuth, M., Waldeck, W., and Langowski, J. (2000) J. Mol. Biol. 298,\n**BLOCK**fs== 6.5**b== 0.7**t== 0.3**l== 0.6**r== 0.4**\n677– 689\n**BLOCK**fs== 6.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n43. Ghaemmaghami, S., Huh, W. K., Bower, K., Howson, R. W., Belle, A., De-\nphoure, N., O’Shea, E. K., and Weissman, J. S. (2003) Nature 425, 737–741\n44. Rodriguez, P., Pelletier, J., Price, G. B., and Zannis-Hadjopoulos, M. (2000) J.\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.6**r== 0.3**\nMol. Biol. 298, 225–238\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n45. Johnston, G. C., Ehrhardt, C. W., Lorincz, A., and Carter, B. L. (1979) J.\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\nBacteriol. 137, 1–5\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n46. Ostashevsky, J. (2002) Mol. Biol. Cell 13, 2157–2169\n47. Weidemann, T., Wachsmuth, M., Knoch, T. A., Muller, G., Waldeck, W., and\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\nLangowski, J. (2003) J. Mol. Biol. 334, 229 –240\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n48. Studitsky, V. M., Walter, W., Kireeva, M., Kashlev, M., and Felsenfeld, G.\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\n(2004) Trends Biochem. Sci. 29, 127–135\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n49. Belotserkovskaya, R., Oh, S., Bondarenko, V. A., Orphanides, G., Studitsky,\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.6**r== 0.2**\nV. M., and Reinberg, D. (2003) Science 301, 1090 –1093\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n50. Walter, P. P., Owen-Hughes, T. A., Cote, J., and Workman, J. L. (1995) Mol.\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nCell. Biol. 15, 6178 – 6187\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n51. Chen, H., Li, B., and Workman, J. L. (1994) EMBO J. 13, 380 –390\n52. Kimura, H., and Cook, P. R. (2001) J. Cell Biol. 153, 1341–1353\n53. Levchenko, V., and Jackson, V. (2004) Biochemistry 43, 2359 –2372\n54. Mizuguchi, G., Shen, X., Landry, J., Wu, W. H., Sen, S., and Wu, C. (2004)\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.6**r== 0.3**\nScience 303, 343–348\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n55. Li, M., Strand, D., Krehan, A., Pyerin, W., Heid, H., Neumann, B., and\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.6**r== 0.2**\nMechler, B. M. (1999) J. Mol. Biol. 293, 1067–1084\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n56. Ellis, R. J. (2001) Trends Biochem. Sci 26, 597– 604\n57. Regnard, C., Desbruyeres, E., Huet, J. C., Beauvallet, C., Pernollet, J. C., and\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\nEdde, B. (2000) J. Biol. Chem. 275, 15969 –15976\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n58. Ito, T., Ikehara, T., Nakagawa, T., Kraus, W. L., and Muramatsu, M. (2000)\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nGenes Dev. 14, 1899 –1907\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n59. Smith, S., and Stillman, B. (1989) Cell 58, 15–25\n60. Ito, T., Levenstein, M. E., Fyodorov, D. V., Kutach, A. K., Kobayashi, R., and\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.6**r== 0.2**\nKadonaga, J. T. (1999) Genes Dev. 13, 1529 –1539",
         "extracted",
         "Self-association of the yeast nucleosome assembly protein 1.;Nucleoplasmin: a nuclear chaperone.;Histone chaperones, a supporting role in the limelight.;Histone release during transcription: NAP1 forms a complex with H2A and H2B and facilitates a topologically dependent release of H3 and H4 from the nucleosome.;Chromatin remodeling by RNA polymerases.;ATP-Driven Exchange of Histone H2AZ Variant Catalyzed by SWR1 Chromatin Remodeling Complex;Histone H3.1 and H3.3 Complexes Mediate Nucleosome Assembly Pathways Dependent or Independent of DNA Synthesis;Conformation of reconstituted mononucleosomes and effect of linker histone H1 binding studied by scanning force microscopy.;Histone H2A/H2B dimer exchange by ATP-dependent chromatin remodeling activities.;Counting nucleosomes in living cells with a combination of fluorescence correlation spectroscopy and confocal imaging.;Preferential Binding of the Histone (H3-H4)2 Tetramer by NAP1 Is Mediated by the Amino-terminal Histone Tails*;Global analysis of protein expression in yeast;On the analysis of protein self-association by sedimentation velocity analytical ultracentrifugation.;Involvement of Nucleocytoplasmic Shuttling of Yeast Nap1 in Mitotic Progression;FACT Facilitates Transcription-Dependent Nucleosome Alteration;Chromatin assembly by DNA-translocating motors;Genome-wide expression analysis of NAP1 in Saccharomyces cerevisiae.;Histone chaperones and nucleosome assembly.;Regulation of biosynthesis and intracellular localization of rice and tobacco homologues of nucleosome assembly protein 1;Modern analytical ultracentrifugation in protein science: A tutorial review;Solvent mediated interactions in the structure of the nucleosome core particle at 1.9 a resolution.;Cell cycle-dependent assembly of a Gin4-septin complex.;A polymer model for large-scale chromatin organization in lower eukaryotes.;Dual Roles of p300 in Chromatin Assembly and Transcriptional Activation in Cooperation with Nucleosome Assembly Protein 1 In Vitro;Chromatin assembly. Cooperation between histone chaperones and ATP-dependent nucleosome remodeling machines.;The crystal structure of nucleoplasmin-core: implications for histone binding and nucleosome assembly.;Multistep Chromatin Assembly on Supercoiled Plasmid DNA by Nucleosome Assembly Protein-1 and ATP-utilizing Chromatin Assembly and Remodeling Factor*;Kinetics of Core Histones in Living Human Cells;Asymmetries in the nucleosome core particle at 2.5 A resolution.;p300-mediated acetylation facilitates the transfer of histone H2A-H2B dimers from nucleosomes to a histone chaperone.;Polyglutamylation of Nucleosome Assembly Proteins*;Anomalous diffusion of fluorescent probes inside living cell nuclei investigated by spatially-resolved fluorescence correlation spectroscopy.;NAP-2: histone chaperone function and phosphorylation state through the cell cycle.;ACF consists of two subunits, Acf1 and ISWI, that function cooperatively in the ATP-dependent catalysis of chromatin assembly.;Chromatin assembly: biochemical identities and genetic redundancy.;Association states of the transcription activator protein NtrC from E. coli determined by analytical ultracentrifugation.;Assembly, Remodeling, and Histone Binding Capabilities of Yeast Nucleosome Assembly Protein 1*;Characterization of nucleosome core particles containing histone proteins made in bacteria.;Crystal structure of the nucleosome core particle at 2.8 Å resolution;Functional characterization of human nucleosome assembly protein-2 (NAP1L4) suggests a role as a histone chaperone.;The N-terminal tail of histone H2A binds to two distinct sites within the nucleosome core.;Histones in transit: cytosolic histone complexes and diacetylation of H4 during nucleosome assembly in human cells.;Drosophila NAP-1 is a core histone chaperone that functions in ATP-facilitated assembly of regularly spaced nucleosomal arrays;Stimulation of transcription factor binding and histone displacement by nucleosome assembly protein 1 and nucleoplasmin requires disruption of the histone octamer;A histone‐binding protein, nucleoplasmin, stimulates transcription factor binding to nucleosomes and factor‐induced nucleosome disassembly.;Functional analysis of nucleosome assembly protein, NAP-1. The negatively charged COOH-terminal region is not necessary for the intrinsic assembly activity.;The nucleosomal core histone octamer at 3.1 A resolution: a tripartite protein assembly and a left-handed superhelix.;Associative behavior of the histone (H3-H4)2 tetramer: dependence on ionic environment.;Purification and characterization of CAF-I, a human cell factor required for chromatin assembly during DNA replication in vitro;Rapid purification of nucleosome assembly protein (AP-I) and production of monoclonal antibodies against it.;Purification and initial characterization of a protein which facilitates assembly of nucleosome-like structure from mammalian cells.;Regulation of cell size in the yeast Saccharomyces cerevisiae;The Sda1 protein is required for passage through start.;Preparation of nucleosome core particle from recombinant histones.;Modern applications of analytical ultracentrifugation.;Handbook of fluorescent probes and research chemicals;Analytical ultracentrifugation in biochemistry and polymer science;Macromolecular crowding : obvious but underappreciated;hp://w w w .jb.org/ D",
         "Association States of Nucleosome Assembly Protein 1 and Its Complexes with Histones*",
         "None"
        ],
        [
         "48",
         "003e9dc9c9aadce5d8504e376cd2d60a37563b17",
         "The prodomain of TACE [TNFalpha (tumour necrosis factor alpha)-converting enzyme] is essential for the secretion of the functional enzyme. Previously, we showed that a TACE truncate was not secreted in the absence of the prodomain and that it was subjected to intracellular degradation. In the present study, we show that full-length TACE was also degraded when expressed without the prodomain. We demonstrate that the prodomain can rescue TACE's secretion in trans, suggesting an intramolecular chaperone function. We addressed the question whether a cysteine switch consensus motif is needed for the secretion of active TACE. The cysteine switch mutants [C184A (Cys184-->Ala)] of TACE resembled the wild-type functionally and in their sensitivity to inhibitors. Interestingly, TACE zymogen forms expressed in the context of the C184A mutation were susceptible to intracellular degradation, suggesting that the prodomain-bound TACE zymogen may be more accessible to intracellular proteinases when compared with mature TACE. Two independent findings confirmed that the catalytic domain of TACE is in a more open state when bound to its prodomain: (i) core tryptophan residues were exposed to the solvent in the procatalytic domain complex and (ii) LysC rapidly proteolysed the procatalytic domain complex but not mature TACE. Therefore the prodomain of TACE is a specific intramolecular chaperone that aids in the secretion of this enzyme, while keeping the catalytic domain in a relatively open conformation. The cysteine switch of TACE is not essential for the secretion of the functional enzyme, but may prevent intracellular degradation of the TACE zymogen.",
         "J. D. Leonard,F. Lin,M. Milla",
         "https://europepmc.org/articles/pmc1135011?pdf=render",
         "\n**BLOCK**fs== 16.4**b== 0.8**t== 0.1**l== 0.1**r== 0.1**\nChaperone-like properties of the prodomain of TNFα-converting enzyme\n(TACE) and the functional role of its cysteine switch\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.1**r== 0.2**\nJennifer D. LEONARD1, Frank LIN and Marcos E. MILLA2\nDepartment of Biochemistry and Biophysics and Johnson Research Foundation, University of Pennsylvania School of Medicine, Philadelphia, PA 19104, U.S.A.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nThe prodomain of TACE [TNFα (tumour necrosis factor α)-con-\nverting enzyme] is essential for the secretion of the functional\nenzyme. Previously, we showed that a TACE truncate was not\nsecreted in the absence of the prodomain and that it was subjected\nto intracellular degradation. In the present study, we show that\nfull-length TACE was also degraded when expressed without the\nprodomain. We demonstrate that\nthe prodomain can rescue\nTACE’s secretion in trans, suggesting an intramolecular chaper-\none function. We addressed the question whether a cysteine switch\nconsensus motif is needed for the secretion of active TACE. The\ncysteine switch mutants [C184A (Cys184 → Ala)] of TACE re-\nsembled the wild-type functionally and in their sensitivity to in-\nhibitors. Interestingly, TACE zymogen forms expressed in the\ncontext of the C184A mutation were susceptible to intracellular\ndegradation, suggesting that the prodomain-bound TACE zymo-\ngen may be more accessible to intracellular proteinases when\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\ncompared with mature TACE. Two independent ﬁndings con-\nﬁrmed that the catalytic domain of TACE is in a more open state\nwhen bound to its prodomain: (i) core tryptophan residues were\nexposed to the solvent in the procatalytic domain complex and\n(ii) LysC rapidly proteolysed the procatalytic domain complex\nbut not mature TACE. Therefore the prodomain of TACE is a\nspeciﬁc intramolecular chaperone that aids in the secretion of this\nenzyme, while keeping the catalytic domain in a relatively open\nconformation. The cysteine switch of TACE is not essential for the\nsecretion of the functional enzyme, but may prevent intracellular\ndegradation of the TACE zymogen.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\nKey words: a disintegrin and metalloproteinase (ADAM), cysteine\nswitch, molecular chaperone, TNFα-converting enzyme (TACE),\ntumour necrosis factor (TNF), zymogen.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nINTRODUCTION\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nTACE [TNFα (tumour necrosis factor α)-converting enzyme] is\nemerging as a centrepiece of mammalian cell signal transduction.\nOriginally, it was described as the processing proteinase of pre-\ncursor TNFα [1,2] whose release mediates inﬂammatory cell re-\ncruitment systemically or locally [3,4]. Later work by Black,\nPeschon and several groups of investigators using TACE−/− cells\nderived from knockout mice demonstrated its role in the shedding\nof a growing number of ectodomains including transforming\ngrowth factor-α [5], amyloid precursor protein [6], L-selectin\n[5], TNFα receptors 1 and 2 [5,7], growth hormone receptor [8],\nfractalkine [9,10], the epidermal growth factor receptor erbB4/\nHER4 [11], interleukin-6 receptor [12], Notch1 receptor [13] and\nthe cellular prion protein [14], among many others (see [15],\nfor a recent review). Thus the function of TACE in regulating the\nsecretion of both receptors and ligands goes well beyond its origin-\nally described role in TNFα release and inﬂammation.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nDespite its importance in mammalian cell biology, surprisingly,\nlittle is known about TACE’s biogenesis and mechanism of activ-\nation. TACE is a multidomain, membrane-bound zinc metallo-\nproteinase belonging to the disintegrin family of metalloprotein-\nases known as the ADAM (a disintegrin and metalloproteinase)\nfamily (see [16] for a review). At its N-terminus, TACE contains a\nsignal peptide followed by an approx. 200-residue prodomain that\nincludes a consensus cysteine switch box (PKVCGY186; see Fig-\nure 3) towards its C-terminal end [1,2]. The cysteine switch was\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\npreviously observed in other zinc metalloproteinases such as the\nMMPs (matrix metalloproteinases) [17–19]. By analogy to those\nproteinases, it was hypothesized that the cysteine residue at posi-\ntion 184 co-ordinates the zinc atom in the active site of TACE’s\ncatalytic domain, holding the enzyme in an inactive, precursor\nform [20]. When this co-ordination is present, the procatalytic\ndomain complex is said to be in the ‘closed’, inhibited position. In\nthis way, TACE is prevented from becoming proteolytically active\nuntil it has reached its resident compartment. More recently, we\nhave shown that an intact cysteine switch is not necessary for\nTACE’s prodomain function as an inhibitor of this enzyme [21].\nHowever, the ‘closed’ position may still allow the prodomain to\nact as a chaperone during folding and/or intracellular sorting of\nthe catalytic domain. This stems from our previous observation\nthat this enzyme is made as an inactive zymogen containing the\nprodomain [20].\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThe biosynthesis of TACE requires an intact prodomain, but\nactivation occurs only after the removal of that prodomain, since\nit acts as an inhibitor of this enzyme [20–22]. Proteolytic removal\nof the prodomain seems to be mediated by a member of the furin-\nlike proprotein convertase family [23,24]. In vivo and in vitro, pro-\ndomain dissociation after furin cleavage requires the presence\nof the disintegrin/cysteine-rich region comprising residues 475–\n671 [20,21]. This region is followed by a single transmembrane\ndomain and a 146-residue cytoplasmic tail that includes Src homo-\nlogy 2 and 3 consensus binding domains [1,2]. Induction of TACE\nactivity in many cells requires exogenous stimulation with phorbol\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nAbbreviations used: ADAM, a disintegrin and metalloproteinase; Dnp, 2,4-dinitrophenyl; HFBA, heptaﬂuorobutyric acid; MMP, matrix metalloproteinase;\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.1**\nMOI, multiplicity of infection; NP40, Nonidet P40; TNFα, tumour necrosis factor α; TACE, TNFα-converting enzyme.\n1 Present address: Department of Biological Sciences, Carnegie Mellon University, Pittsburgh, PA 15213, U.S.A.\n2 To whom correspondence should be addressed, at 242 Anatomy-Chemistry Bldg, 3620 Hamilton Walk, Philadelphia, PA 19104, U.S.A. (email\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.8**\nmmilla@mail.med.upenn.edu).\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nesters [5,6,25]. Although these ﬁndings suggest the involvement\nof a kinase cascade, the identity of the kinase(s) involved and the\nmechanism of activation are hitherto unknown. Additionally, it\nis not clear whether changes in the phosphorylation status of the\ncytoplasmic domain of TACE affect furin processing or removal\nof its inhibitory prodomain [26–28].\n**BLOCK**fs== 9.5**b== 0.5**t== 0.2**l== 0.1**r== 0.5**\nOur aim was to determine the nature of the association be-\ntween TACE’s pro- and catalytic domains and elucidate what\ndeterminants within the prodomain are responsible for its dual role\nin maintaining the zymogen state and in the secretion of functional\nTACE. In the present study, we report studies demonstrating that\nthe prodomain of TACE acts as an intramolecular chaperone,\naiding in the secretion of both the full-length membrane-bound\nenzyme and engineered soluble truncates. We show that the cata-\nlytic domain of TACE is held by its prodomain in a more open\nconformation relative to the mature, active enzyme. Because of\nthis, we hypothesize that the prodomain of TACE behaves as a\ntrue intramolecular chaperone, holding the catalytic domain of\nTACE in a relatively open conformation that is inactive. More-\nover, we show that the cysteine switch consensus sequence of\nthe prodomain is not essential for the secretion of functional en-\nzyme, but has a role in protecting the non-native TACE zymogen\nfrom intracellular degradation during its secretion. These results\nindicate that the prodomain of TACE functions in a way that is\nmechanistically different from the one proposed for secreted\nserine proteinase zymogens from several bacteria [29,30]. They\nare discussed in the context of existing knowledge on the\nprodomain’s function in the biosynthesis and secretion of ADAM\nand other metalloproteinases.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nMATERIALS AND METHODS\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.6**\nProduction of recombinant human TACE viruses\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nThe construction of human TACE truncates, hR651 (Met1 to\nArg651) and hR473 (Met1 to Arg473), in the baculoviral expression\nshuttle vector pFastBac1 has been described previously ([20] and\nFigure 3). Two variants of these truncates were constructed:\nhR473/C184A (Cys184 → Ala) and hR651/C184A. Both were\nconstructed by cassette mutagenesis. The cassettes contained a\nC184A substitution and were engineered with EcoRV and AccI\ncompatible ends for insertion into hR473 and hR651. The TACE\ngene has naturally occurring unique EcoRV and AccI sites brack-\neting the cysteine switch. All recombinant plasmids were trans-\nformed into DH10Bac cells (Invitrogen). All truncates were\ncloned into the pFastBac1 vector and recombinant plasmids\nwere transformed into DH10Bac cells. Recombinant bac-\nmids were isolated, puriﬁed and used to generate baculovirus in\nSf 9 (Spodoptera frugiperda) insect cells as described elsewhere\n[31].\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nProduction of recombinant mouse full-length TACE viruses\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe full-length mouse TACE clone (mTACE; a gift from Dr R. A.\nBlack, Amgen, Seattle, WA, U.S.A.) was inserted by digestion\nwith KpnI and XhoI into the pFastBac1 (pFB) vector containing\nan inverted multiple cloning site. In addition to the wild-type\nclone, the following variants were constructed: mTACEPro and\nmTACE/C184A. The variant mTACE/C184A was constructed by\nStratagene’s Quik Change PCR methodology using divergent\nprimers containing a C184A substitution and pFB mTACE as\nthe template. The variant mTACEPro was constructed by PCR\nusing a forward primer containing Glu22 fused to Arg215. The\nPCR product was inserted into pCDNA3 mTACE using unique\nBspEI and EcoRI restriction sites. The variant mTACEPro was\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\ncloned into the pFastBac1-Inverse vector by digestion with KpnI\nand XhoI. Recombinant bacmids and viruses were generated as\ndescribed above.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.5**r== 0.2**\nExpression of TACE truncates and mutants\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nExponentially growing Trichoplusia ni cells were infected with\nrecombinant TACE baculovirus strains at an MOI (multiplicity\nof infection) of 1. Cultures were harvested 24 and 48 h post-\ninfection. The cells were separated from the media by low-speed\ncentrifugation at 1200 g for 10 min. The cell-associated protein\nfraction was extracted by resuspending the cell pellet in 20 mM\nTris (pH 7.5) and 1 % NP40 (Nonidet P40) followed by centri-\nfugation at 20 000 g for 30 min. The supernatants were saved\nand pellets containing highly insoluble material were discarded.\nMedia and cell-associated fractions were mixed with Laemmli\nsample buffer [32], subjected to SDS/PAGE analysis and blotted\non to Hybond ECL® (enhanced chemiluminescence) nitrocellu-\nlose membranes. Blots were probed with either a mouse mono-\nclonal antibody that recognizes the catalytic domain of TACE or\na rabbit antiserum recognizing the prodomain of TACE. Blots\nwere developed using the Pierce Supersignal West Pico ECL® kit\nafter incubation with a sheep anti-mouse or donkey anti-rabbit\nhorseradish peroxidase conjugate.\n**BLOCK**fs== 9.0**b== 0.5**t== 0.4**l== 0.5**r== 0.2**\nPuriﬁcation of TACE truncates and mutants\n**BLOCK**fs== 9.5**b== 0.1**t== 0.5**l== 0.5**r== 0.1**\nExponentially growing T. ni cells (3 litres) were infected with\nbaculovirus strains encoding hR473, hR473/C184A, hR651 or\nhR651/C184A at an MOI of 1. Cultures were harvested 48 h\npost-infection. Cells were separated from media by low-speed\ncentrifugation (2000 rev./min) for 30 min. Media supernatants\nwere incubated with concanavalin A–Sepharose (50 ml of bed\nvolume; Amersham Biosciences) in a batchwise manner, with\novernight stirring at 4 ◦C. The concanavalin A beads were washed\nthree times with 20 mM Tris/HCl (pH 8.0) containing 150 mM\nNaCl and twice with 20 mM Tris/HCl (pH 8.0) without salt. TACE\nwas eluted from the concanavalin A beads with three washes\n(100 ml each) with the same buffer containing 0.4 M methyl-\nmannoside. Eluates were combined and applied to a 20 ml Bio-\nScale DEAE FPLC column (Bio-Rad) and washed with 400 ml of\n20 mM Tris/HCl (pH 8.0). TACE was eluted with a 400 ml linear\ngradient of 0–0.4 M NaCl. The fractions were monitored by\nSDS/PAGE analysis. TACE-containing fractions were pooled and\nconcentrated by ultraﬁltration using Centriprep-30 concentrators\n(Amicon/Millipore, Billerica, MA, U.S.A.). The concentrates\nwere applied on to a 16 mm × 600 mm Superdex S200 size-exclu-\nsion chromatography column (Amersham Biosciences) at the ﬂow\nrate of 1 ml/min, using 20 mM Tris/HCl (pH 8.0) and 150 mM\nNaCl as mobile phase. Fractions containing TACE were deter-\nmined by SDS/PAGE analysis, pooled and concentrated using\ncentriprep-30 concentrators (Amicon). Both for the wild-type\nh473 and the hC184A variant, TACE was eluted from this column\nas two peaks, an earlier peak corresponding to the procatalytic do-\nmain zymogen complex [Rf = 0.187, where Rf stands for elution\nratio from the column relative to the marker for the internal volume\nof the column (Phenol Red)] and a later peak corresponding to the\nfree catalytic domain (Rf = 0.267). This behaviour has been re-\nported previously [20]. The hTACE procatalytic domain zymogen\ncomplex was stored at − 20 ◦C and used without further modiﬁ-\ncation for spectroscopic and proteolysis protection studies (see the\nResults section). TACE polypeptides were determined to be more\nthan 95 % pure by SDS/PAGE analysis after visualization with the\nCoomasie-based stain Simply Blue Safestain (Invitrogen). Protein\nconcentrations were determined with Bradford’s reagent (Bio-\nRad) or based on TACE’s UV light absorbance at 280 nm [33].\n**BLOCK**fs== 9.0**b== 0.8**t== 0.1**l== 0.1**r== 0.8**\nActivity assays\n**BLOCK**fs== 9.5**b== 0.6**t== 0.2**l== 0.1**r== 0.5**\nProteolytic activities of hTACE truncates and variants were deter-\nmined in an HPLC-based assay using the synthetic peptide\nDnp-SPLAQAVRSSSR-NH2 (where DNp stands for 2,4-dinitro-\nphenyl) as the substrate [1]. The sequence of this peptide cor-\nresponds to the cleavage site of TACE on pro-TNFα. The reaction\ncocktail contained 25 µM Dnp-serine as a standard, 20 µM pep-\ntide substrate and a 1:1000 dilution of mammalian proteinase in-\nhibitors without EDTA (Sigma P 8340) in a reaction buffer\n(10 mM Hepes, pH 7.5, and 0.05 % NP40). TACE was added\nin a tenth of the total reaction volume, at a dilution such that the\nturnover was between 25 and 50 %. We have previously observed\nthat the cleavage reaction is linear up to a turnover of 50 %. Re-\naction mixtures were incubated at 37 ◦C for 30 min, quenched\nwith an equal volume of 1 % HFBA (heptaﬂuorobutyric acid)\nand cleared by ultraﬁltration through PVDF membrane ﬁlters. The\nproduct to substrate + product ratio was determined by integrating\nthe absorbance at 350 nm of the corresponding peaks after HPLC\nseparation of the reaction mixture using a 150 mm C18 column\n(Vydac) resolved with a discontinuous gradient of 0.1 % HFBA in\nwater and 0.1 % HFBA in acetonitrile. Non-speciﬁc activity was\ndetermined by measuring the product formation in the presence\nof 10 µM GW9901, a hydroxamic acid competitive inhibitor that\ncompletely blocks TACE activity at that concentration [34].\n**BLOCK**fs== 9.0**b== 0.5**t== 0.5**l== 0.1**r== 0.8**\nInhibitor proﬁles\n**BLOCK**fs== 9.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nThe effects of various hydroxamic acid inhibitors on the activities\nof hR473/C184A and hR651/C184A versus hR473 and hR651\nrespectively were determined using the HPLC assay described\nabove. The reactions were incubated at room temperature (23 ◦C)\nfor 1 h. The inhibitors screened were GW9471 [35], GW4023\n([35a,35b]) and GW1988 [34]. Each inhibitor was screened by the\naddition of the following concentrations to the reaction cocktail:\n10, 1 µM, 100, 10, 1 nM and 100 pM.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nFluorescence spectroscopy\n**BLOCK**fs== 9.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nIsolated hTACE catalytic domain, prodomain or procatalytic do-\nmain complexes were diluted to a ﬁnal concentration of 3 µM\nwith 20 mM Tris/HCl (pH 8.0) containing 150 mM NaCl. In-\ntrinsic ﬂuorescence spectra were collected between 300 and\n400 nm after excitation at 390 nm using an Aviv ATF-105 spectro-\nﬂuorimeter. For each sample, the detector was adjusted to 50 %\nsensitivity at the wavelength of maximum emission (λmax). Scans\nwere background-subtracted by placing the same buffer in the\nreference cell.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nIsolated hTACE catalytic or procatalytic domain complexes at\na ﬁnal concentration of 40 µg/ml were incubated with highly\npuriﬁed LysC (Roche Molecular Biochemicals) at a ﬁnal con-\ncentration of 1 µg/ml, in 20 mM Tris/HCl (pH 8.0) containing\n150 mM NaCl and 10 µM GW9471. After the addition of LysC,\nsamples were incubated at room temperature. Reactions were\nquenched by adding 5× Laemmli sample buffer [32] and heating\nto 100 ◦C for 3 min. Samples were resolved by SDS/PAGE fol-\nlowed by staining with Simply Blue Safestain.\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nFigure 1\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\nExpression of full-length mouse TACE wild-type and variants\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nWestern blot of mTACEPro (lanes 1, 2 and 3), wild-type (lanes 4, 5 and 6) and C184A (lanes 7,\n8 and 9) forms. Cell-associated fractions were analysed at 24 h (lanes 1, 4 and 7), 48 h (lanes 2\nand 5) and 72 h (lanes 3, 6 and 9) post-infection. Cell associated fractions were obtained\nafter protein extraction with 1 % NP40, followed by centrifugation, SDS/PAGE separation and\nWestern blotting using ECL® for TACE polypeptide visualization (see the Materials and methods\nsection). The mTACEPro signal was weaker and the gel was overdeveloped, resulting in higher\nnoise levels.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.4**l== 0.5**r== 0.1**\nnot secreted and was extensively degraded intracellularly [20]. To\nconﬁrm that the prodomain of TACE serves an essential role in the\nbiogenesis and maturation of this enzyme, we made an equivalent\nprodomain deletion in full-length mouse TACE (mTACEPro)\nand compared its expression with that of the wild-type counterpart\n(mTACE). Owing to the human clone’s extreme instability and\nresistance to genetic manipulation, this experiment was performed\nwith the mouse clone. It has been shown previously by several\ngroups that TACE’s mouse orthologue is biochemically indis-\ntinguishable from its human counterpart [5,23,26,27]. Insect cells\nbearing the prodomain deletion mutant expressed full-length\nmTACE to minimal levels relative to the wild-type polypeptide\n(Figure 1, lanes 1–3 versus lanes 4–6). mTACEPro was rapidly\ndegraded and was nearly undetectable by 48 h post-infection. This\nfull-length enzyme result supports previous observations with\nthe catalytic and extracellular domain truncates and conﬁrms the\nessential role of the prodomain in the biogenesis and secretion of\nTACE [20]. This result also veriﬁes that soluble truncated forms\nof TACE behave similarly to full-length TACE in terms of their\nbiosynthesis and maturation.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nThis result suggested to us that the prodomain of TACE must\nbe acting as an intramolecular chaperone, necessary for the bio-\nsynthesis of this proteinase and its targeting to the late secretory\npathway, where its activating enzyme, furin or another PC\nfamily proprotein convertase [23,24], localizes. We sought to test\nthis hypothesis by reconstituting the expression and secretion\nof the prodomain-deﬁcient catalytic domain truncate, TACE\nhR473Pro [20], by co-expression of its prodomain in trans. T. ni\ncells were doubly infected with a constant titre of the hR473Pro\nvirus and increasing titres of a virus encoding the signal pep-\ntide followed by the prodomain (hR214, Figure 3). We observed\nthat greater prodomain expression and secretion correlated with\nincreased secretion of the catalytic domain into culture media\n(Figure 2). Thus the prodomain can rescue the secretion of TACE\nin trans. This suggested that the prodomain must be playing an\nessential role within TACE by affecting its assembly, its stability\nagainst intracellular proteinases or its trafﬁcking through the\nsecretory pathway.\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.9**\nRESULTS\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.1**r== 0.6**\nRequirement of the prodomain for TACE expression\n**BLOCK**fs== 9.0**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nCys184 in the ‘cysteine switch’ box of TACE’s prodomain is not\nessential for the secretion of functional TACE\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nWe have previously reported that a form of hTACE consisting\nof the signal peptide fused to the mature catalytic domain was\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nTACE fails to be secreted in the absence of its prodomain (the\npresent study and [20]). The prodomain of this enzyme contains\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nFigure 2 Reconstitution of TACE’s catalytic domain secretion by co-\nexpression of the hR473Pro virus with a second virus encoding hTACE’s\nprodomain\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nT. ni cells were simultaneously infected with 5 MOI of the hR473Pro virus and 0, 1, 2 or\n10 MOI of the prodomain virus. Culture media were harvested at 24 and 48 h post-infection and\nanalysed by SDS/PAGE followed by Western blotting. Upper panel: anti-TACE catalytic domain\nmouse antibody [20]; lower panel: anti-TACE prodomain antiserum (rabbit).\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.9**\nFigure 3\n**BLOCK**fs== 8.0**b== 0.4**t== 0.6**l== 0.1**r== 0.8**\nhTACE truncates\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nSchematic representation of the domain arrangement of human hTACE truncates. SP, signal\npeptide; Pro, prodomain; Disint/Cys-rich, disintegrin/cysteine-rich domain; TM, transmembrane\ndomain.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\na consensus cysteine switch box (PKVCGY186). It has been\ndemonstrated for several zinc metalloproteinases that this motif\nmediates the inhibition of the zymogen through a ligation of the\nzinc ion in the active site by the cysteinyl thiol within the cysteine\nswitch motif [17–19]. The role of this motif as an inhibitor of the\ncatalytic domain within the ADAM family of metalloproteinases\nis less clear and our own results show that it is not important for this\nfunction [21]. Since the cysteine switch was shown to be important\nfor the secretion of several MMPs, we constructed variants of the\nsoluble TACE truncates hR473 and hR651 (Figure 3), in which\nthe Cys184 was replaced with an alanine residue. Insect cells were\ninfected with baculovirus strains harbouring wild-type hTACE\nor the C184A variants. The level of secretion of TACE into the\nculture media relative to the wild-type was monitored by Western\nblotting.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nThe C184A variants were secreted into media with comparable\nefﬁciency as the wild-type TACE truncates hR473 and hR651\n(Figure 4). The amount of immature wild-type protein produced\n24 h post-infection was somewhat higher compared with the\nC184A variant for both TACE hR473 (Figure 4B, lane 3 versus\nlane 1) and hR651 (Figure 4D, lane 3 versus lane 1); however, by\n48 h, secretion of mature enzyme into culture media was similar\nfor the mutant relative to wild-type for both hR473 (Figure 4A,\nlane 4 versus lane 2) and hR651 (Figure 4C, lane 4 versus lane 2).\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.5**r== 0.4**\nFigure 4\n**BLOCK**fs== 8.0**b== 0.7**t== 0.3**l== 0.6**r== 0.1**\nExpression of hTACE truncates and C184A variants\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n(A) Western blot of TACE hR473 (lanes 1 and 2) and hR473/C184A (lanes 3 and 4) culture\nmedium supernatants at 24 h (lanes 1 and 3) and 48 h (lanes 2 and 4) post-infection; M, mature\nTACE; I, TACE zymogen. (B) Western blot of hR473 (lanes 1 and 2) and hR473/C184A (lanes 3\nand 4) cell-associated fractions obtained after protein extraction with 1 % NP40. (C) Western\nblot of hR651 (lanes 1 and 2) and hR651/C184A (lanes 3 and 4) culture medium supernatants\n24 h (lanes 1 and 3) and 48 h (lanes 2 and 4) post-infection. (D) Western blot of hR651 (lanes 1\nand 2) and hR651/C184A (lanes 3 and 4) cell-associated fractions obtained after protein\nextraction with 1 % NP40.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.5**r== 0.1**\nThis suggested that, although TACE secretion is completely de-\npendent on the presence of its prodomain, it does not require the\nCys184 within the cysteine switch box. Other determinants within\nTACE’s prodomain, yet to be identiﬁed, must be mediating the\nformation of a secretion-competent complex between the pro-\nand catalytic domains. Intriguingly, the only noticeable differ-\nence between the C184A variants and wild-type enzymes was\nthe presence of proteolytic degradation products in the medium\nof C184A cultures 48 h post-infection. These fragments were\nabsent from the wild-type sample. The observed differences in\nTACE’s intracellular degradation did affect the ﬁnal puriﬁcation\nyields noticeably: 9.87 nmol for hR473 expressed in the context\nof a wild-type prodomain versus 1.79 nmol for the C184A cys-\nteine switch mutant; and 16.4 nmol for wild-type hR651 versus\n3.69 nmol for the C184A variant. This may indicate that a ‘closed’\ncysteine-switch is essential to prevent intracellular proteolytic\ndegradation of the TACE zymogen.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\nBoth TACE hR473 and hR651 truncates lack the transmem-\nbrane and cytoplasmic domains. Therefore it could be argued that\nthe biogenesis and maturation of these forms do not reﬂect the\nbehaviour of the full-length enzyme. Under this view, mutational\neffects on the cysteine switch may only be readable in the context\nof intact full-length TACE. To address this issue, we constructed\na full-length mTACE variant bearing the C184A cysteine-switch\nmutation and compared its expression and maturation with those\nof the wild-type proteinase. Expressions of the enzyme by T. ni\ncells infected with either wild-type or C184A full-length mTACE-\nencoding viruses were similar (Figure 1). The only form present\n24 h post-infection was immature mTACE (Figure 1, lanes 4 and\n7). By 48 h post-infection, immature mTACE, its mature form\nand degradation products were observed for both wild-type and\nC184A full-length mTACE (Figure 1, lanes 5 and 8). Inter-\nestingly, degradation products were more evident with the C184A\nvariant. By 72 h post-infection, this variant was completely de-\ngraded, whereas the wild-type polypeptide was still detected in\nsubstantial amounts (Figure 1, lanes 6 and 9). This result obtained\nwith the entire membrane-bound enzyme strongly indicates that\nthe cysteine switch is not essential for TACE’s expression and\nconversion from the zymogen into the mature form. The rela-\ntive susceptibility of the defective cysteine-switch variants to\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.6**\nTable 1 Peptide substrate cleavage by recombinant hTACE forms\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\nhTACE activity was measured using a Dnp-labelled peptide containing the cleavage site\nof pro-TNFα (SPLAQAVRSSR). Dnp-labelled substrate and product were separated using\nreversed-phase HPLC and monitored at 350 nm.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nTACE form\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.8**\nhR473 wild-type\nhR473 C184A\nhR651 wild-type\nhR651 C184A\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.3**r== 0.6**\nkcat/Km peptide (M−1 · s−1)\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.3**r== 0.6**\n1.09 × 106 +\n1.78 × 105 +\n1.69 × 105 +\n9.93 × 104 +\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.4**r== 0.6**\n− 0.54 × 105\n− 0.20 × 105\n− 0.18 × 105\n− 0.54 × 104\n**BLOCK**fs== 9.5**b== 0.6**t== 0.3**l== 0.1**r== 0.5**\nintracellular proteolysis mirrored our previous observation using\nthe complete extracellular domain of TACE (hR651). There-\nfore the ‘closed’ form of the cysteine switch seems to be important\nfor protecting the enzyme from intracellular degradation during\nits biogenesis and maturation.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nWe puriﬁed to homogeneity hTACE forms expressed in the\npresence of a wild-type prodomain or the C184A counterpart and\ntested whether the variant-expressed forms were functional by\nassaying their catalytic activity against a synthetic peptide con-\ntaining the pro-TNFα cleavage site. Both hR473/C184A and\nhR651/C184A cleaved this substrate only at the expected position\nbetween the alanine and valine residues and their speciﬁcity con-\nstants (kcat/Km) were similar to their respective wild-type forms\n(Table 1). This demonstrates that secretion of functional TACE\ndoes not depend on the presence of the ‘closed switch’ complex\nbetween the pro- and catalytic domains made by the co-ordination\nof the cysteinyl thiol at position 184 of the prodomain and the\ncatalytic zinc ion within the active site of TACE.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nTACE C184A variants are equally sensitive to hydroxamic acid\ninhibitors relative to their wild-type counterparts\n**BLOCK**fs== 9.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nAlthough both hR473/C184A and hR651/C184A variants were\nsecreted as functional enzymes, we observed modest differences\nin kcat and Km values (approx. 5-fold lower for hR473/C184A\nrelative to hR473 and 2-fold lower for hR651/C184A relative to\nhR651). To assess further whether these variants were enzymatic-\nally equivalent to wild-type TACE hR473 and hR651, we ex-\namined their sensitivity to three hydroxamic acid inhibitors exhi-\nbiting different potencies against TACE: GW9471, GW4023 and\nGW1988 [34–35b]. The inhibition proﬁles for wild-type hTACE\nand the C184A variants were virtually identical for all inhibitors\n(Figure 5). The IC50 values calculated for wild-type hTACE and\nthe C184A variants were also comparable (Table 2). This remark-\nable similarity in sensitivity to inhibitors indicates that the overall\nfold and substrate-binding cleft features of the C184A variants\nmust have been preserved relative to the wild-type.\n**BLOCK**fs== 9.0**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\nThe catalytic domain of TACE is in a more open conformation\nwhen bound to its prodomain\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nAn intriguing possibility is that the pro-bound catalytic domain\nof TACE may be in a form different from the native, active form\nobserved by X-ray crystallography [36]. This form may display\nan increased sensitivity to intracellular proteolysis, explaining\nthe requirement for an intact, ‘closed’ cysteine switch to block\naccess to one or more proteolysis-susceptible sites. We used two\nindependent, complementary strategies to assess conformational\ndifferences between the catalytic domain of TACE in its zymogen\n(pro-bound) and mature (pro-free) forms. The pro- and catalytic\ndomains of hTACE contain two tryptophan residues each [1].\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.5**r== 0.4**\nFigure 5\nvariants\n**BLOCK**fs== 8.0**b== 0.3**t== 0.7**l== 0.6**r== 0.1**\nEffect of hydroxamic acid inhibitors on hTACE truncates and\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nAssays were run using hR473 (), hR473/C184A (), hR651 () and hR651/C184A ().\nInhibitor curves are the averages of three or four replicate runs. The inhibitors were added at the\nstart of the 1 h incubation: (A) GW9471, (B) GW4023 and (C) GW1988.\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.5**r== 0.4**\nTable 2\n**BLOCK**fs== 8.0**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nIC50 values for hTACE truncates and C184A mutants\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nhTACE activity was measured using a Dnp-labelled peptide containing the cleavage site of\npro-TNFα in the presence and absence of three hydroxamic acid inhibitors, GW9471, GW4023\nand GW1988. Each inhibitor was screened by the addition of the following concentrations to\nthe reaction cocktail before a 1 h incubation at room temperature: 10, 1 µM, 100, 10, 1 nM and\n100 pM. Dnp-labelled substrate and product were separated using reversed-phase HPLC\nand monitored at 350 nm.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nTACE form\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.4**\nhR473 wild-type\nhR473 C184A\nhR651 wild-type\nhR651 C184A\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.6**r== 0.3**\nIC50 (nM)\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nGW9471\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\n46.0 +\n44.6 +\n48.2 +\n48.7 +\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.7**r== 0.3**\n− 6.2\n− 12.0\n− 5.6\n− 5.3\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\nGW4023\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\nGW1988\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n736 +\n767 +\n851 +\n836 +\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.8**r== 0.2**\n− 203\n− 179\n− 94\n− 407\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\n33.4 +\n31.8 +\n34.4 +\n43.2 +\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.9**r== 0.1**\n− 12.6\n− 23.5\n− 13.0\n− 13.5\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.9**\nFigure 6\n**BLOCK**fs== 8.0**b== 0.6**t== 0.4**l== 0.1**r== 0.6**\nFluorescence emission spectra of hTACE forms\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\nhTACE’s catalytic domain, prodomain or the procatalytic domain zymogen complex isolated\nfrom cell culture media was diluted to a ﬁnal concentration of 3 µM in 20 mM Tris/HCl (pH 8.0)\ncontaining 150 mM NaCl. The excitation wavelength was 290 nm and emission was recorded\nbetween 300 and 400 nm. Dark grey line, free prodomain; light grey line, free catalytic domain;\nblack line, procatalytic domain complex. The background-subtracted ﬂuorescence intensity is\nshown in arbitrary units.\n**BLOCK**fs== 9.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nThe ﬂuorescence emission spectra of the tryptophan residues\nfor hTACE’s isolated prodomain (Figure 6, dark grey line)\nand catalytic domain (light grey line) exhibited the λmax values\nexpected for folded polypeptides with well-deﬁned cores burying\nhydrophobic amino acids, including tryptophan residues (316 nm\nfor the catalytic domain and 314 nm for the prodomain). In stark\ncontrast, the procatalytic domain zymogen complex isolated from\ncell culture media exhibited a shift in the ﬂuorescence emission\nλmax to 331 nm (Figure 6, black line), approaching the λmax ob-\nserved for the pro- or catalytic domain fully denatured with 6 M\nguanidinium chloride (340 nm; results not shown). This indicates\nthat the conformations of the catalytic domains and prodomains of\nTACE are more ‘open’ when complexed together in the zymogen\nform relative to their free forms. This results in the exposure of\ncore tryptophan residues to the solvent environment as indicated\nby the change in their ﬂuorescence properties.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nUsually, unfolded or partially folded polypeptides are better\nsubstrates for proteolytic degradation, since sites that are other-\nwise protected in the folded form become accessible to protein-\nases. A simple method to assess conformational differences con-\nsists of exposing a protein to trypsin or a trypsin-like activity\nand observing its degradation relative to solution conditions. We\nincubated the native, the isolated hTACE catalytic domain and\nthe procatalytic domain zymogen complex with small amounts\nof LysC (1:40, w/w), a proteinase we have used previously for\npeptide digestion/Edman analysis of native hTACE [1]. The cata-\nlytic domain of native full-length hTACE is resistant to LysC\n(results not shown). As expected from this previous observation,\nthe isolated catalytic domain was resistant to LysC digestion\nwith minimal degradation visible over the time course of incub-\nation (4 h; Figure 7A). In contrast, the catalytic domain was ex-\ntremely sensitive to LysC when complexed with the prodomain;\nsubstantial to complete degradation was observed even at the\nshortest incubation times (5–15 min; Figure 7B). We conclude\nfrom these results that the prodomain renders the catalytic domain\nof TACE in a more open conformation when bound to it, probably\nresembling the interaction of molecular chaperones with their\ntarget polypeptides.\n**BLOCK**fs== 8.0**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\nFigure 7 Degradation of the catalytic domain of mature hTACE and its\nprodomain-bound form by the endoproteinase LysC\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nhTACE’s catalytic domain complex or procatalytic domain complex (8 µg each) was incubated\nwith 0.2 µg of LysC in 200 µl of Tris/HCl containing 150 mM NaCl and 10 µM GW9471 to\ninhibit the enzymatic activity of TACE. At the speciﬁed times, 18 µl of each incubation was\nprocessed as speciﬁed in the Materials and methods section, then resolved by SDS/PAGE\n(4–20 % gradient gels). (A) hTACE catalytic domain degradation by LysC versus time (min).\n(B) Degradation of hTACE procatalytic domain complex by LysC versus time (min). The initial\npositions of undigested pro- and catalytic domains are shown next to the molecular-mass\nstandards.\n**BLOCK**fs== 9.0**b== 0.4**t== 0.6**l== 0.5**r== 0.4**\nDISCUSSION\n**BLOCK**fs== 9.5**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nThe role of the prodomain in the biogenesis and activity regul-\nation of TACE and other members of the ADAM family is under-\nstood only at an incipient level. It has been shown previously that\nthe prodomain is essential for the secretion of TACE, and in its\nabsence, the catalytic domain is rapidly degraded intracellularly\n[20,27]. This observation extends to other ADAM family mem-\nbers (see below). We have also shown that the prodomain of TACE\nis a potent inhibitor of this enzyme, and its removal is required\nfor the activation of this proteinase [20,21]. In the present study,\nwe demonstrate that the prodomain of TACE is essential for the\nbiogenesis and maturation of both full-length native TACE and\nsoluble TACE truncates generated by protein engineering. We\nshow that hTACE Pro alone can reconstitute the secretion of a\nTACE prodomain deletion mutant when expressed in trans. This is\nnot due to an increase in the expression of TACE hR473pro,\nsince the intracellular levels of this polypeptide remained un-\nchanged under conditions where peak recovery of secretion was\nobserved (results not shown). Therefore the observed reconsti-\ntution of TACE’s secretion on a prodomain-dependent basis\noccurred after translation of both protein products. This strongly\nsuggests that the prodomain must be serving as a chaperone,\naiding in the secretion of this enzyme.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nThe consensus cysteine switch box within the prodomain has\nbeen demonstrated to be important for both metalloproteinase\ninhibition and secretion [17]. The folding or secretion of TACE\nmay require the interaction of the thiol function of Cys184 with\n**BLOCK**fs== 9.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nthe catalytic zinc ion in the active site of TACE. To test this\nhypothesis, hTACE truncates were expressed in the context of the\nC184A mutation within the prodomain. They were secreted at\nlevels comparable with those observed with the wild-type forms.\nIt is noteworthy that the secreted hR651/C184A and full-length\nmTACE/C184A variants were susceptible to proteolysis, whereas\ntheir wild-type counterparts exhibited no proteolytic fragments.\nThis degradation most probably occurred intracellularly, not in the\nculture media, since it was also observed with full-length, mem-\nbrane-bound mTACE. This increased susceptibility to degrad-\nation was more evident at 72 h post-infection, where all of the\nC184A zymogen form was degraded intracellularly, whereas\nthe wild-type form was still present at signiﬁcant levels. Therefore\neven though a ‘closed’ complex is not essential for secretion, the\nCys184–Zn2+ interaction seems to play a role by stabilizing TACE\nagainst intracellular proteolysis.\n**BLOCK**fs== 9.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nIt has been suggested that the prodomain acts as a chaperone\nduring the biosynthesis and secretion of the catalytic domain in\nADAM family members [20,22,37] and the related membrane-\nbound MMP, MT1-MMP [38]. It is possible that the prodomain\nhas a role in either the folding of the enzyme or its secretion\n(trafﬁcking through the secretory pathway). These two models,\nalthough not mutually exclusive, are mechanistically different. In\nthe ﬁrst one, enzymes depending on the prodomain for folding\nreach their native state in its presence. Examples of this class of\nproteinases are subtilisin BPN and the α-lytic proteinase. For\nthese, the prodomain effectively lowers the energy barrier between\nthe unfolded and folded states (folding under kinetic control\n[39,40]). Once collapse into the native, functional state occurs,\nthese enzymes remove the prodomain by themselves.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nThe second model may be more appropriate for TACE and\nother ADAM family members: the prodomain acts as a true intra-\nmolecular chaperone, associating with the catalytic domain in\na way that actually prevents access to its native conformation.\nExamples of molecular chaperones affecting the conformation of\ntheir target proteins are SecB [41], Hsp90 (heat-shock protein\n90) [42], rap [43,44] and Grp94 [45]. These proteins bind to\npartially folded intermediates of their target proteins. Although the\nprodomain may still be involved in the catalytic domain assembly\npathway, such a process cannot be completed before its release\nfrom the catalytic domain. We have found that the tryptophan\nﬂuorescence emission of TACE hR473 was signiﬁcantly different\nfor the free, active form of this enzyme, relative to the zymogen\nform complexed with the prodomain. The tryptophan residues\nbecame more exposed to solvent on binding of the catalytic do-\nmain to the prodomain. Thus the prodomain of TACE may indeed\nhold this enzyme in a more open conformation, functionally re-\nsembling a molecular chaperone. Our discovery that the procata-\nlytic domain complex is remarkably more sensitive to LysC\ndigestion relative to the free enzyme strongly supports this hypo-\nthesis. Both of these results were obtained with the hTACE zymo-\ngen isolated from cell-culture media, arguing against the possi-\nbility of an artifact occurring on mixing both domains in the test\ntube.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nHaving established that an intact cysteine switch is not essential\nfor the secretion of TACE, we sought to determine whether it was\nessential for the production of the active enzyme. It is important to\nnote that the speciﬁcity constant, obtained using our methods, for\nthe wild-type TACE hR473 truncate is higher than the previously\nreported value: 1.09 × 106 compared with 1.3 × 105 [20]. The\nprodomain of TACE inhibits the catalytic domain of this enzyme\npotently, with an IC50 value of 70 nM [21]. Previous puriﬁcations\nhave used 4-aminophenylmercuric acetate in their methods to\nremove the prodomain from the catalytic domain. Since the time\nof those initial experiments, we have discovered that it is possible\n**BLOCK**fs== 9.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nto remove the prodomain from the catalytic domain simply by\nincubating procatalytic domain complexes at room temperature.\nThe free prodomain is completely degraded by the activated\nenzyme during this incubation. The previously reported constant\nmay be an underestimate stemming from the ability of 4-amino-\nphenylmercuric acetate to inhibit TACE itself [20].\n**BLOCK**fs== 9.5**b== 0.5**t== 0.2**l== 0.5**r== 0.1**\nSecreted hTACE truncates expressed in the context of the\nC184A mutation and puriﬁed from culture media showed activity\nlevels similar to their wild-type counterparts. However, we did\nobserve modest differences in speciﬁcity constants between wild-\ntype and the C184A variants (< 2-fold for hR651 and approx.\n5-fold for hR473). These minor differences may be due to changes\nin the thermodynamic stability of the catalytic domain caused by\nthe C184A mutation or a lower efﬁciency in loading the catalytic\nzinc ion required for enzymatic activity due to the absence of the\nthiol at position 184. The second hypothesis is a strong possi-\nbility, since zinc ligation by the thiol of Cys184 has been observed\nin the zymogen form of TACE [21]. These results do suggest that\nformation of a ‘closed’ TACE zymogen, although not essential\nfor the secretion of mature enzyme, may factor into the efﬁciency\nof this process. Despite the observed differences in kcat/Km values,\nboth hR473/C184A and hR651/C184A show identical sensitivity\nwith a series of hydroxamic acid inhibitors with different poten-\ncies against TACE. The inhibitor screen is a more stringent evalu-\nation of the functionality of these variants, for the ﬁne architecture\nof the substrate-binding cleft within the active site of TACE\nmust be preserved for these inhibitors to interact with similar\npotencies.\n**BLOCK**fs== 9.5**b== 0.2**t== 0.5**l== 0.5**r== 0.1**\nWe have shown that the prodomain of TACE is essential for\nthe secretion of this enzyme [20]. Several groups have reported\nsimilar observations with ADAM 9 [22], ADAM 10 [37], ADAM\n12 [46,47] and ADAM 19 [48]. Of these studies, only those on\nADAM 9 and ADAM 12 [22,46] speciﬁcally addressed whether\nan intact cysteine switch was required for secretion. Intriguingly,\nthe equivalent C175A mutation did not affect ADAM 12 [46], but\nhad a profound adverse effect on the secretion of ADAM 9 [22].\nGiven these results, the most pressing question becomes, what\nis the true role of the cysteine switch consensus sequence within\nthe ADAM family? We have reported previously that a functional\nswitch is not required for inhibition of TACE. Mutational analysis\nof Cys184 and vicinal residues failed to demonstrate a loss of in-\nhibitory potency with a non-functional switch [21]. On the basis\nof our results, it is possible that the true physiological role of\nthe cysteine switch, at least for TACE, is to prevent proteolytic\ndegradation of the zymogen during its biogenesis and maturation.\nWe have shown by two independent experiments that the TACE\nPro appears to distort the conformation of the catalytic domain of\nTACE, making its core solvent-exposed. The open conformation\nresulting from prodomain binding probably exposes sites within\nthe catalytic domain of TACE to intracellular proteinases. The\npresence of a functional cysteine switch in the ‘closed’ position\nmay be necessary to prevent such an exposure in vivo, explaining\nthe remarkable conservation of the cysteine switch motif through-\nout the ADAM family.\n**BLOCK**fs== 9.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nA second question is, which are the key residues determining\nthe secretion competency of TACE and other ADAM family mem-\nbers? Owing to the lack of structural information on prodomains\nof ADAM family proteinases, we were not able to build models of\nTACE’s prodomain or its interaction with the catalytic domain. In\na search for those determinants, we are currently performing the\ngenetic analysis of regions of TACE’s prodomain that appear to\nbe unique to this ADAM. Furthermore, efforts to solve the three-\ndimensional structure of TACE’s zymogen forms are ongoing and\nshowing promise. These studies may prove useful for the design\nof novel TACE inhibitors acting as prodomain mimetics.\n**BLOCK**fs== 8.0**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\nWe thank Dr D. Becherer from GlaxoSmithKline for discussions and his generous\ncontribution of the TNFα peptide substrate, hydroxamic acid inhibitors and a monoclonal\nantibody that recognizes the catalytic domain of TACE. We also thank Dr R. A. Black\n(Amgen) for generously providing the mouse full-length mTACE clone. This work was\nsupported by grant no. AR45949 from the National Institutes of Health.\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1**r== 0.8**\nREFERENCES\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.1**r== 0.5**\n1 Moss, M. L., Jin, S.-L. C., Milla, M. E., Bickett, D. M., Burkhart, W., Carter, H. L., Chen,\nW.-J., Clay, W. C., Didsbury, J. R., Hassler, D. et al. (1997) Cloning of a disintegrin\nmetalloproteinase that processes tumour necrosis factor-α. Nature (London) 385,\n733–736\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n2 Black, R. A., Rauch, C. T., Kozlosky, C. J., Peschon, J. J., Slack, J. L., Wolfson, M. F.,\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\nCastner, B. J., Stocking, K. L., Reddy, P., Srinivasan, S. et al. (1997) A metalloproteinase\ndisintegrin that releases tumour necrosis factor-α from cells. Nature (London) 385,\n729–733\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.5**\n3 Beutler, B. and Cerami, A. (1988) Tumor necrosis, cachexia, shock and inﬂammation:\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.1**r== 0.7**\na common mediator. Annu. Rev. Biochem. 57, 505–518\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n4 Vassalli, P. (1992) The pathophysiology of tumor necrosis factors. Annu. Rev. Immunol.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.8**\n10, 411–452\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.1**r== 0.5**\n5 Peschon, J. J., Slack, J. L., Reddy, P., Stocking, K. L., Sunnarborg, S. W., Lee, D. C.,\nRussell, W. E., Castner, B. J., Johnson, R. S., Fitzner, J. N. et al. (1998) An essential\nrole for ectodomain shedding in mammalian development. Science 282,\n1281–1284\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.1**r== 0.5**\n6 Buxbaum, J. D., Liu, K. N., Luo, Y., Slack, J. L., Stocking, K. L., Peshon, J. J.,\nJohnson, R. S., Castner, B. J., Cerretti, D. P. and Black, R. A. (1998) Evidence\nthat tumor necrosis factor alpha converting enzyme is involved in regulated alpha-\nsecretase cleavage of the Alzheimer amyloid protein precursor. J. Biol. Chem. 273,\n27765–27767\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.1**r== 0.5**\n7 Solomon, K. A., Pesti, N., Wu, G. and Newton, R. C. (1999) Cutting edge: a dominant\nnegative form of TNF-alpha converting enzyme inhibits proTNF and TNFRII secretion.\nJ. Immunol. 163, 4105–4108\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\n8 Zhang, Y., Jiang, J., Black, R. A., Baumann, G. and Frank, S. J. (2000) Tumor necrosis\nfactor-alpha converting enzyme (TACE) is a growth hormone binding protein (GHBP)\nsheddase: the metalloprotease TACE/ADAM-17 is critical for (PMA-induced) GH\nreceptor proteolysis and GHBP generation. Endocrinology 141, 4342–4348\n9 Tsou, C. L., Haskell, C. A. and Charo, I. F. (2001) Tumor necrosis factor-alpha-\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\nconverting enzyme mediates the inducible cleavage of fractalkine. J. Biol. Chem. 276,\n44622–44626\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.1**r== 0.5**\n10 Garton, K. J., Gough, P. J., Blobel, C. P., Murphy, G., Greaves, D. R., Dempsey, P. J. and\nRaines, E. W. (2001) Tumor necrosis factor-alpha-converting enzyme (ADAM17) mediates\nthe cleavage and shedding of fractalkine (CX3CL1). J. Biol. Chem. 276, 37993–38001\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\n11 Rio, C., Buxbaum, J. D., Peschon, J. J. and Corfas, G. (2000) Tumor necrosis factor-\nalpha-converting enzyme is required for cleavage of erbB4/HER4. J. Biol. Chem. 275,\n10379–10387\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n12 Althoff, K., Reddy, P., Voltz, N., Rose-John, S. and Mullberg, J. (2000) Shedding of\ninterleukin-6 receptor and tumor necrosis factor alpha. Contribution of the stalk\nsequence to the cleavage pattern of transmembrane proteins. Eur. J. Biochem. 267,\n2624–2631\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n13 Brou, C., Logeat, F., Gupta, N., Bessia, C., LeBail, O., Doedens, J. R., Cumano, A.,\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nRoux, P., Black, R. A. and Israel, A. (2000) A novel proteolytic cleavage involved in Notch\nsignaling: the role of the disintegrin-metalloprotease TACE. Mol. Cell 5, 207–216\n14 Vincent, B., Paitel, E., Saftig, P., Fobert, Y., Hartmann, D., De Strooper, B., Grassi, J.,\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.5**\nLopez-Perez, E. and Checler, F. (2001) The disintegrins ADAM10 and TACE contribute to\nthe constitutive and phorbol ester-regulated normal cleavage of the cellular prion\nprotein. J. Biol. Chem. 276, 37743–37746\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.5**\n15 Arribas, J. and Borroto, A. (2002) Protein ectodomain shedding. Chem. Rev. 102,\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.8**\n4627–4638\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.1**r== 0.6**\n16 Becherer, J. D. and Blobel, C. P. (2003) Biochemical properties and functions of\nmembrane-anchored metalloprotease-disintegrin proteins (ADAMs). Curr. Top.\nDev. Biol. 54, 101–123\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n17 van Wart, H. E. and Birkedal-Hansen, B. (1990) The cysteine switch: a principle of\nregulation of metalloprotease activity with potential applicability to the entire matrix\nmetalloprotease gene family. Proc. Natl. Acad. Sci. U.S.A. 87, 5578–5581\n18 Springman, E. B., Angleton, E. L., Birkedal-Hansen, H. and Van Wart, H. E. (1990)\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n20 Milla, M. E., Leesnitzer, M. A., Moss, M. L., Clay, W. C., Carter, H. L., Miller, A. B., Su,\nJ.-L., Lambert, M. H., Willard, D. H., Sheeley, D. M. et al. (1999) Speciﬁc sequence\ndeterminants are required for the expression of functional tumor necrosis factor-α\nconverting enzyme (TACE). J. Biol. Chem. 274, 30563–30570\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n21 Gonzales, P., Solomon, A., Leesnitzer, M. A., Miller, A. B., Sagi, I. and Milla, M. E. (2004)\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nInhibition of TACE by its prodomain. J. Biol. Chem. 279, 31638–31645\n**BLOCK**fs== 7.5**b== 0.7**t== 0.2**l== 0.5**r== 0.1**\n22 Roghani, M., Becherer, J. D., Moss, M. L., Atherton, R. E., Erdjument-Bromage, H.,\nArribas, J., Blackburn, R. K., Weskamp, G., Tempst, P. and Blobel, C. P. (1999)\nMetalloprotease-disintegrin MDC9: intracellular maturation and catalytic activity.\nJ. Biol. Chem. 274, 3531–3540\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n23 Srour, N., Lebel, A., McMahon, S., Fournier, I., Fugere, M., Day, R. and Dubois, C. M.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n(2003) TACE/ADAM-17 maturation and activation of sheddase activity require proprotein\nconvertase activity. FEBS Lett. 554, 275–283\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n24 Endres, K., Anders, A., Kojro, E., Gilbert, S., Fahrenholz, F. and Postina, R. (2003) Tumor\nnecrosis factor-alpha converting enzyme is processed by proprotein-convertases to its\nmature form which is degraded upon phorbol ester stimulation. Eur. J. Biochem. 270,\n2386–2393\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\n25 Pradines-Figueres, A. and Raetz, C. R. H. (1992) Processing and secretion of tumor\n**BLOCK**fs== 7.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nnecrosis factor alpha in endotoxin-treated Mono Mac 6 cells are dependent on phorbol\nmyristate acetate. J. Biol. Chem. 267, 23261–23268\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n26 Doedens, J. R. and Black, R. A. (2000) Stimulation-induced down-regulation of tumor\n**BLOCK**fs== 7.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nnecrosis factor-alpha converting enzyme. J. Biol. Chem. 275, 14598–14607\n27 Schl¨ondorff, J., Becherer, J. D. and Blobel, C. P. (2000) Intracellular maturation and\nlocalization of the tumour necrosis factor alpha convertase (TACE). Biochem. J. 347,\n131–138\n**BLOCK**fs== 7.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n28 Fan, H., Turck, C. W. and Derynck, R. (2003) Characterization of growth factor-induced\nserine phosphorylation of tumor necrosis factor-alpha converting enzyme and of an\nalternatively translated polypeptide. J. Biol. Chem. 278, 18617–18627\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n29 Baker, D., Shiau, A. K. and Agard, D. A. (1993) The role of pro regions in protein folding.\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.3**\nCurr. Opin. Cell Biol. 5, 966–970\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n30 Bryan, P. N. (2002) Prodomains and protein folding catalysis. Chem. Rev. 102,\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.4**\n4805–4816\n**BLOCK**fs== 7.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n31 Luckow, V. A., Lee, S. C., Barry, G. F. and Olins, P. O. (1993) Efﬁcient generation of\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\nrecombinant baculoviruses by site-speciﬁc transposon-mediated insertion of foreign\ngenes into a baculovirus genome propagated in Escherichia coli. J. Virol. 67, 4566–4579\n32 Laemmli, U. K. (1970) Cleavage of structural proteins during the assembly of the head of\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.2**\nbacteriophage T4. Nature (London) 227, 680–685\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n33 Pace, C. N., Vajdos, F., Fee, L., Grimsley, G. and Gray, T. (1995) How to measure and\npredict the molar absorption coefﬁcient of a protein. Protein Sci. 4, 2411–2423\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n34 Moss, M., Becherer, J. D., Milla, M., Pahel, G., Lambert, M., Andrews, R.,\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\nFrye, S., Haffner, C., Cowan, D., Maloney, P. et al. (1999) TNF-α convertase. In\nMetalloproteinases as Targets for Anti-inﬂammatory Drugs (Bradshaw, D.,\nNixon, J. S. and Bottomley, K., eds.), pp. 187–204, Birkhauser, Basel\n**BLOCK**fs== 7.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n35 McGeehan, G. M., Becherer, J. D., Bast, Jr, R. C., Boyer, C. M., Champion, B., Connolly,\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nK. M., Conway, J. G., Furdon, P., Karp, S., Kidao, S. et al. (1994) Regulation of tumour\nnecrosis factor-α processing by a metalloproteinase inhibitor. Nature (London) 370,\n558–561\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n35a Hundhausen, C., Misztela, D., Berkhout, T. A., Broadway, N., Saftig, P., Reiss, K.,\n**BLOCK**fs== 7.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\nHartmann, D., Fahrenholz, F., Postina, R., Matthews, V. et al. (2003) The disintegrin-like\nmetalloproteinase ADAM10 is involved in constitutive cleavage of CX3CL1 (fractalkine)\nand regulates CX3CL1-mediated cell–cell adhesion. Blood 102, 1186–1195\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n35b Hussain, I., Hawkins, J., Shikotra, A., Riddell, D. R., Falter, A. and Dingwall, C. (2003)\nCharacterization of the ectodomain shedding of the α-site amyloid precursor\nprotein-cleaving enzyme 1 (BACE1). J. Biol. Chem. 278, 36264–36268\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n36 Maskos, K., Fernandez-Catalan, C., Huber, R., Bourenkov, G. P., Bartunik, H., Ellestad,\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\nG. A., Reddy, P., Wolfson, M. F., Rauch, C. T., Castner, B. J. et al. (1998) Crystal structure\nof the catalytic domain of human tumor necrosis factor-α converting enzyme.\nProc. Natl. Acad. Sci. U.S.A. 95, 3408–3412\n**BLOCK**fs== 7.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n37 Anders, A., Gilbert, S., Garten, W., Postina, R. and Fahrenholz, F. (2001) Regulation of\nthe alpha-secretase ADAM10 by its prodomain and proprotein convertases. FASEB J.\n15, 1837–1839\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\n38 Cao, J., Hymowitz, M., Conner, C., Bahou, W. F. and Zucker, S. (2000) The propeptide\ndomain of membrane type 1-matrix metalloproteinase acts as an intramolecular\nchaperone when expressed in trans with the mature sequence in COS-1 cells.\nJ. Biol. Chem. 275, 29648–29653\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\nMultiple modes of activation of latent human ﬁbroblast collagenase: evidence for the role\nof a Cys73 active-site zinc complex in latency and a ‘cysteine switch’ mechanism for\nactivation. Proc. Natl. Acad. Sci. U.S.A. 87, 364–368\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n39 Bryan, P., Alexander, P., Strausberg, S., Schwarz, F., Wang, L., Gilliland, G. and Gallagher,\nD. T. (1992) Energetics of folding of subtilysin BPN. Biochemistry 31, 4937–4945\n40 Baker, D., Sohl, J. L. and Agard, D. A. (1992) A protein-folding reaction under kinetic\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n19 Grams, F., Huber, R., Kress, L. F., Moroder, L. and Bode, W. (1993) Activation of snake\nvenom metalloproteinases by a cysteine switch-like mechanism. FEBS Lett. 335,\n76–80\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\ncontrol. Nature (London) 356, 263–265\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n41 Randall, L. L. and Hardy, S. J. (2002) SecB, one small chaperone in the complex milieu\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.3**\nof the cell. Cell. Mol. Life Sci. 59, 1617–1623\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.1**r== 0.5**\n42 Kimmins, S. and MacRae, T. H. (2000) Maturation of steroid receptors: an example of\nfunctional cooperation among molecular chaperones and their associated proteins.\nCell Stress Chaperones 5, 76–86\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\n46 Loechel, F., Gilpin, B. J., Engvall, E., Albrechtsen, R. and Wewer, U. M. (1998) Human\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nADAM 12 (meltrin alpha) is an active metalloprotease. J. Biol. Chem. 273,\n16993–16997\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n43 Bu, G. and Marzolo, M. P. (2000) Role of rap in the biogenesis of lipoprotein receptors.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n47 Loechel, F., Overgaard, M. T., Oxvig, C., Albrechtsen, R. and Wewer, U. M. (1999)\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nTrends Cardiovasc. Med. 10, 148–155\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n44 Willnow, T. E. (1998) Receptor-associated protein (RAP): a specialized chaperone for\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nendocytic receptors. Biol. Chem. 379, 1025–1031\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n45 Argon, Y. and Simen, B. B. (1999) GRP94, an ER chaperone with protein and peptide\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\nbinding properties. Semin. Cell Dev. Biol. 10, 495–505\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nRegulation of human ADAM 12 protease by the prodomain, Evidence for a functional\ncysteine switch. J. Biol. Chem. 274, 13427–13433\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n48 Kang, T., Zhao, Y. G., Pei, D., Sucic, J. F. and Sang, Q. X. A. (2002) Intracellular\n**BLOCK**fs== 7.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\nactivation of human adamalysin 19/disintegrin and metalloproteinase 19 by furin occurs\nvia one of the two consecutive recognition sites. J. Biol. Chem. 277, 25583–25591\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.1**r== 0.6**\nReceived 12 October 2004/16 November 2004; accepted 7 December 2004\nPublished as BJ Immediate Publication 7 December 2004, DOI 10.1042/BJ20041727",
         "extracted",
         "",
         "Chaperone-like properties of the prodomain of TNFalpha-converting enzyme (TACE) and the functional role of its cysteine switch.",
         "None"
        ],
        [
         "49",
         "004051ae2ab180bb719c695daae098eabb8a2afd",
         "Tie2 is an endothelium-specific receptor tyrosine kinase required for normal blood vessel maturation. We report that Tie2 mRNA translation is maintained under hypoxic conditions. To identify the mechanism responsible for this, we undertook structure/function analysis of the Tie2 5′-untranslated region (UTR). Transcription start site mapping indicates the existence of a several mRNA isoforms containing unusually long 5′-UTRs (>350 nucleotides) with five upstream open reading frames. We find internal ribosome binding activity that allows the Tie2 mRNA to initiate in a cap-independent fashion. Our data provide a framework for understanding how Tie2 mRNA is translated despite a cumbersome structured 5′-UTR and how its production is secured under unfavorable environmental conditions.",
         "E. Park,Joseph M. Lee,Jaime D. Blais,J. Bell,J. Pelletier",
         "http://www.jbc.org/article/S0021925820615307/pdf",
         "\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\nTHE JOURNAL OF BIOLOGICAL CHEMISTRY\n© 2005 by The American Society for Biochemistry and Molecular Biology, Inc.\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.7**r== 0.1**\nVol. 280, No. 22, Issue of June 3, pp. 20945–20953, 2005\nPrinted in U.S.A.\n**BLOCK**fs== 14.0**b== 0.8**t== 0.1**l== 0.1**r== 0.2**\nInternal Translation Initiation Mediated by the Angiogenic\nFactor Tie2*\n**BLOCK**fs== 8.0**b== 0.8**t== 0.2**l== 0.4**r== 0.1**\nReceived for publication, November 10, 2004, and in revised form, March 8, 2005\nPublished, JBC Papers in Press, March 31, 2005, DOI 10.1074/jbc.M412744200\n**BLOCK**fs== 8.0**b== 0.7**t== 0.2**l== 0.2**r== 0.1**\nEun-Hee Park‡§, Joseph M. Lee‡, Jaime D. Blais¶, John C. Bell¶, and Jerry Pelletier‡**\nFrom the ‡Department of Biochemistry and McGill Cancer Center, McGill University, Montreal, Quebec H3G 1Y6,\nCanada and the ¶Ottawa Regional Cancer Centre Research Laboratories, Ottawa, Ontario K1H 8L6, Canada\n**BLOCK**fs== 8.5**b== 0.5**t== 0.3**l== 0.1**r== 0.5**\nTie2 is an endothelium-specific receptor tyrosine ki-\nnase required for normal blood vessel maturation. We\nreport that Tie2 mRNA translation is maintained under\nhypoxic conditions. To identify the mechanism respon-\nsible for this, we undertook structure/function analysis\nof the Tie2 5-untranslated region (UTR). Transcription\nstart site mapping indicates the existence of a several\nmRNA isoforms containing unusually long 5-UTRs\n(>350 nucleotides) with five upstream open reading\nframes. We find internal ribosome binding activity that\nallows the Tie2 mRNA to initiate in a cap-independent\nfashion. Our data provide a framework for understand-\ning how Tie2 mRNA is translated despite a cumbersome\nstructured 5-UTR and how its production is secured\nunder unfavorable environmental conditions.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.5**l== 0.1**r== 0.5**\nAngiogenesis is an essential step in allowing tumors to grow\nbeyond 1–2 mm in diameter (1). Although the vasculature of\nmost adult tissues is quiescent, during embryogenesis or in\npathological conditions such as cancer, a pro-angiogenic setting\nis established. At least two families of growth factors, vascular\nendothelial growth factor (VEGF)1 and the angiopoietins, are\nrequired for this process. VEGF mediates its effects through\nVEGFR-1/Flt-1 and VEGFR-2/Flk-1/KDR, two endothelial re-\nceptors implicated as central regulators of the vascular system\nunder both normal and abnormal physiological conditions (2–\n5). VEGF expression is stimulated under hypoxic conditions by\nthe transcription factor hypoxia-inducible factor-1 (HIF-1),\nwhich regulates a large range of physiological responses as a\nconsequence of reduced oxygen availability (6, 7).\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\nThe angiopoietins belong to a second family of angiogenic\nfactors essential for blood vessel maturation. Angiopoietin-1\n(Ang-1) is an agonist of endothelial cell tyrosine kinase receptor\nTie2/tek (8). Studies with Tie2 null mice indicate that the\nangiopoietin/Tie2 signaling system plays a role in the later\n**BLOCK**fs== 7.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\n* This work was supported by Grant MOP-11354 from the Canadian\nInstitutes of Health Research (CIHR) (to J. P.). The costs of publication\nof this article were defrayed in part by the payment of page charges.\nThis article must therefore be hereby marked “advertisement” in ac-\ncordance with 18 U.S.C. Section 1734 solely to indicate this fact.\n§ Supported by a K. M. Hunter/CIHR doctoral research award.\n** CIHR Senior Investigator. To whom correspondence should be\naddressed: McIntyre Medical Sciences Bldg., Rm. 810, 3655 Promenade\nSir William Osler, McGill University, Montreal, Quebec H3G 1Y6,\nCanada. Tel.: 514-398-2323; Fax: 514-398-7384; E-mail:\njerry.\npelletier@mcgill.ca.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.1**r== 0.5**\n1 The abbreviations used are: VEGF, vascular endothelial growth\nfactor; HIF, hypoxia-inducible factor; Ang, angiopoietin; IRES, internal\nribosome entry site; UTR, untranslated region; HUVEC, human umbil-\nical vein endothelial cell; EMCV, encephalomyocarditis virus; ORF,\nopen reading frame; uORF, upstream ORF; RT, reverse transcriptase;\nnt, nucleotide; ODC, ornithine decarboxylase; RACE, rapid amplifica-\ntion of cDNA ends; RPA, RNase protection assay.\n**BLOCK**fs== 8.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\nsteps of angiogenesis (9). Consistent with a central role for Tie2\nin angiogenesis, germ line-activating mutations in humans are\nassociated with vascular dysmorphogenesis (10). Additionally,\nthe naturally occurring Tie2 antagonist, Ang-2, disrupts angio-\ngenesis in vivo (11). Blocking Tie2 activation with recombinant\nAng-2 significantly inhibits tumor growth (12) and is associ-\nated with activation of apoptosis (13), possibly due to disrup-\ntion of Akt signaling (14). Hence, studies aimed at better de-\nfining the regulation of Tie2 expression are important in\nassessing anti-angiogenic therapeutic avenues.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.4**l== 0.5**r== 0.1**\nIn addition to profound transcriptional effects (6, 7), expo-\nsure of cells to hypoxia attenuates protein synthesis by decreas-\ning translation initiation (15, 16). Two main steps of initiation\nare targets for regulation, either 43 S ribosomal complex for-\nmation (by affecting eIF2 phosphorylation status (17)) or the\nmRNA/ribosome binding step (18). Ribosome recruitment in\neukaryotes can occur by two mechanisms, a cap-dependent\nprocess and by internal ribosome recruitment. Cap-dependent\nrecruitment occurs through eIF4F-mediated recognition of the\nm7G-cap structure and involves binding of the 43 S preinitia-\ntion complex near the mRNA 5-end, followed by scanning to\nthe appropriate AUG start codon (18). Barriers to the scanning\nprocess, such as mRNA secondary structure and uORFs, im-\npinge in a negative fashion on the translational efficiency of a\nmRNA species. The cap dependence of an mRNA is thought to\nbe a function of 5-cap-proximal secondary structure; hence\nmRNAs with reduced secondary structure will have reduced\ndependence on eIF4F for initiation, whereas those with in-\ncreased secondary structure are more dependent on eIF4F for\ninitiation (19, 20). Internal ribosome recruitment allows an\nmRNA to bypass the cap-dependent initiation requirement for\neIF4F. These different mechanisms of initiation provide a layer\nof gene regulation by which expression of specific mRNAs can\nbe maintained or altered independent of others. Although hy-\npoxia exerts inhibitory effects on cap dependent ribosome bind-\ning (16), several mRNAs implicated in angiogenesis remain\nefficiently translated, including VEGF (21–24), HIF-1 (25),\nand Tie2 (this report). Whereas VEGF and HIF-1achieve this\nby recruiting ribosomes internally to an IRES, the issue of how\nTie2 is able to circumvent this translational block has not been\naddressed previously. Herein, we present functional studies\nthat identify the presence of an IRES element within the Tie2\n5-UTR, highlighting the complexities of Tie2 expression at the\nlevel of translation.\n**BLOCK**fs== 7.0**b== 0.1**t== 0.9**l== 0.6**r== 0.2**\nEXPERIMENTAL PROCEDURES\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nCell Lines, Transfections, and Hypoxia Treatment—Primary human\numbilical vein endothelial cells (HUVECs) were purchased from Clo-\nnetics (Walkersville, MD) and maintained in endothelial growth medi-\num-2 supplemented with growth factors, 2% fetal bovine serum, 50\ng/ml gentamicin, and 50 g/ml amphotericin according to the manu-\nfacturer’s instructions.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nDNA transfections into primary HUVECs were performed using\n**BLOCK**fs== 8.0**b==-0.0**t== 1.0**l== 0.1**r== 0.6**\nThis is an Open Access article under the CC BY license.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.1**l== 0.1**r== 0.5**\nLipofectin (Invitrogen), as specified by the manufacturer. Briefly, 1.5–\n1.8  105 cells were seeded per 10-cm2 plate and grown in endothelial\ngrowth medium-2 supplemented with 2% fetal bovine serum. Cells were\nharvested 24 h post-transfection and luciferase activities measured\nusing the dual-luciferase reporter assay system (Promega, Madison,\nWI). Transient RNA transfections were performed using the cationic\nlipid reagent, DMRIE-C (Invitrogen). Approximately 1.8  106 HU-\nVECs were seeded per 10-cm2 plate 24 h prior to transfection. Cells\nwere then transfected with 20 g of capped and polyadenylated mRNA\nand harvested 8 h post-transfection. Transfections were performed in\nduplicate and repeated three times. For polysome analysis and RNA\nisolation, extracts prepared from HUVECs grown under hypoxia\n(0.01% O2) and normoxia (20% O2) for 16 h were fractionated by\nvelocity sedimentation on 10 –50% sucrose gradients. All experiments\nwith HUVECs were performed between passages two and five.\n**BLOCK**fs== 7.5**b== 0.4**t== 0.3**l== 0.1**r== 0.5**\nPrimer Extension—Human placental poly(A) or total RNA was ob-\ntained from Clontech (Palo Alto, CA). Total RNA from HUVECs was\nisolated using TRIzol (Invitrogen) according to the manufacturer’s sug-\ngested protocol. Oligonucleotides were end-labeled with [-32P]ATP\n(3000 Ci/mmol; PerkinElmer Life Sciences) using T4 polynucleotide\nkinase and purified after separation on 8% denaturing polyacrylamide\ngels. A mixture of 2.5 g of mRNA or 25 g of total RNA and 2  105\ncpm of the primer was annealed at 65 °C for 15 min. The extension\nreaction was performed using SuperScript II (Invitrogen) according to\nthe manufacturer’s instructions. Extension products were separated on\nan 8% denaturing gel and the products visualized by autoradiography.\nRNase Protection Assay—Templates for riboprobe synthesis were\nprepared by subcloning a 244-bp fragment containing the hTie2 5-\nflanking sequence (467 to 224) into the pBluescript II KS() plas-\nmid (Stratagene, La Jolla, CA) to create hTie2UP-Ribo15/pKSII.\n[-32P]UTP (800 Ci/mmol) was used with the MAXIscript in vitro tran-\nscription kit (Ambion, Austin, TX) to generate riboprobes from the\nXhoI-linearized hTie2UP-Ribo15/pKSII. The radiolabeled cRNA was\npurified by 8% denaturing gel electrophoresis, followed by elution for\n12 h at 37 °C in 0.5 M ammonium acetate, 1 mM EDTA, 0.2% SDS. The\nefficiency of labeling was determined by scintillation counting, and 1 \n105 cpm of each labeled riboprobe were used per reaction. RNase pro-\ntection assays were performed using the RPAIII system (Ambion) ac-\ncording to the manufacturer’s instructions. Briefly, the labeled ribo-\nprobe was hybridized to 5 g of human placental poly(A) for 16 h at\n42 °C. Following hybridization, unprotected single-stranded RNA was\ndigested with a mixture of RNase A (2.5 units/ml) plus RNase T1 (100\nunits/ml). The resulting protected products were separated on an 8%\ndenaturing polyacrylamide gel and visualized by autoradiography.\n**BLOCK**fs== 7.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nSucrose Gradients and Polysome Analysis—Polysome analyses were\nperformed on HUVECs at 70% confluence. Cells were treated with 0.1\nmg/ml cyclohexamide for 3 min and harvested in lysis buffer (15 mM\nTris-HCl (pH 7.4), 300 mM NaCl, 15 mM MgCl2, 1% Triton X-100, 0.1\nmg/ml cyclohexamide, 333.3 units/ml RNase inhibitor (Ambion)) at\n4 °C. After a brief centrifugation, the extract was layered on 10 –50%\nsucrose gradients in buffer (15 mM Tris-HCl (pH 7.4), 300 mM NaCl, 15\nmM MgCl2). Gradients were centrifuged at 39,000 rpm for 1.5 h at 4 °C,\nand fractions were collected across the gradient and snap-frozen in\nliquid nitrogen. Total RNA or RNA from the indicated pooled fractions\nfollowing sucrose gradient centrifugation was isolated using TRIzol.\n**BLOCK**fs== 7.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nGeneration of Monocistronic and Bicistronic Constructs—The gener-\nation of bicistronic constructs was based on pRDEF (26). First, a\nfragment corresponding to the hTie2 5-UTR was generated by PCR\namplification of the hTie2 5-UTR cDNA and subcloned into pGL3-\nBASIC (Promega). In the second step, the fragments containing hTie2\n5-UTR and amino-terminal coding region of firefly luciferase were\nintroduced into pRDEF by replacing the EMCV 5-UTR and the first\n121-bp of firefly luciferase. Bicistronic constructs containing a stable\nstem-loop structure (TAR(III) element (27)) were generated by insert-\ning the stem-loop element 232 bp upstream of the Renilla coding region\nof the above-mentioned bicistronic constructs. For all of the recombi-\nnant constructs generated in this study in which a reporter was placed\ndownstream of uORF5, its AUG codon was placed in-frame with the\npredicted Tie2 AUG codon embedded in uORF5. The accuracy of all\nconstructs was confirmed by sequencing.\n**BLOCK**fs== 7.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nReal-time Reverse Transcriptase (RT)-PCR Analysis—Relative ex-\npression levels were assessed using a 200 –300-ng RNA sample by\nreal-time, one-step RT-PCR using the Roche Diagnostics LightCycler\ninstrument and the Roche Diagnostics LightCycler RNA master SYBR\ngreen I kit according to the manufacturer’s instruction. The level of 28\nS rRNA was used as a reference value for quantitation within samples\nto determine the relative amounts of specific mRNA. The sequences of\noligonucleotides used for RT-PCR are available on request.\n**BLOCK**fs== 7.5**b== 0.8**t== 0.1**l== 0.5**r== 0.1**\nNorthern Blot Analysis—Total RNA was isolated following transfec-\ntion with bicistronic reporters. Following extraction of RNA with\nTRIzol, 21 g of total RNA was loaded onto a 1% formaldehyde-agarose\ngel, separated by electrophoresis, and transferred to a nylon membrane.\nHybridization was in ExpressHyb hybridization solution (BD Bio-\nsciences) at 68 °C with a 400-nt firefly luciferase-specific 32P-labeled\nprobe. Autoradiography of the membrane was performed at 80 °C\nwith film (Kodak X-Omat) to visualize the radioactive signal.\n**BLOCK**fs== 7.5**b== 0.6**t== 0.2**l== 0.5**r== 0.1**\nMetabolic Labeling with [35S]Methionine—HUVECs were plated at\n2.7  105 cells per 150-mm plate and incubated for 24 h, followed by a\nfurther 12–36-h incubation under either normoxic or hypoxic condi-\ntions. Cells were then washed with phosphate-buffered saline and in-\ncubated in methionine-free RPMI 1640 medium (Sigma) supplemented\nwith 10% fetal calf serum and [35S]methionine (PerkinElmer Life Sci-\nences) 4 h prior to harvesting. Cells were washed three times with\nphosphate-buffered saline and harvested in lysis buffer (25 mM HEPES\n(pH 7.4), 137 mM NaCl, 10% glycerol, 2.5 mM EDTA, 2.5 mM EGTA,\n0.5% Triton X-100, 5 mM -mercaptoethanol, 2 g/ml aprotinin, 2 g/ml\nleupeptin, 2 g/ml pepstatin A). The cell lysates (500 g of total protein)\nwere precleared with protein G-Sepharose and immunoprecipitations\nperformed with anti-Tie2 (Santa Cruz Biotechnology, Inc., Santa Cruz,\nCA), anti--actin antibodies (Abcam Inc., Cambridge, MA), and anti-\nornithine decarboxylase (ODC) (Sigma) antibodies, respectively. The\nimmunoprecipitates were then subjected to SDS-PAGE, followed by\nfluorography. Experiments were performed twice at least in triplicate.\n**BLOCK**fs== 7.0**b== 0.6**t== 0.4**l== 0.7**r== 0.2**\nRESULTS\n**BLOCK**fs== 8.5**b== 0.1**t== 0.4**l== 0.5**r== 0.1**\nTie2 mRNA Is Translated under Hypoxic Conditions—To\nassess the translational behavior of Tie2 mRNA under hypoxic\nconditions, we monitored its distribution across polysomes\nfrom normoxic and hypoxic treated primary HUVECs (Fig. 1).\nUnder hypoxic conditions, there is a reduction in polysomes\naccompanied by an apparent increase in amount of free ribo-\nsomal subunits (Fig. 1A), as reported previously (23, 25) and\nconsistent with a reduction in protein synthesis associated with\nthis physiological treatment. Quantitative RT-PCR from frac-\ntions collected from the bottom or top half of the polysomes\n(N/H-4 and N/H-3, respectively), as well as from the top of the\ngradient (N/H-1) and the region containing the ribosomal sub-\nunits (N/H-2), revealed that the distribution of Tie2 and VEGR\nmRNAs remain unchanged when cells are transferred from\nnormoxic to hypoxic conditions (Fig. 1B). In contrast, the vast\nmajority of -actin mRNA appears in heavy polysomes under\nnormoxic conditions (N-4) and redistributes to lighter fractions\n(H-3, H-2, H-1) when HUVECs are transferred to hypoxia (Fig.\n1B). This is similar to what has been reported previously for\n-actin mRNA redistribution when cells are exposed to hypoxia\n(25, 28). We also assessed the change in polysome distribution\nof a second mRNA encoding ODC. Translation of this transcript\nis generally cap-dependent (29, 30), except during mitosis (31).\nAs the majority of cells in our experiment were not in mitosis\n(5%), we expect ODC to behave as a cap-dependent tran-\nscript. The majority of ODC mRNA from HUVECs under nor-\nmoxic growth conditions was in N-2 or heavy polysomes (N-4).\nUpon exposure of HUVECs to hypoxia, the majority of the\nmRNA in the N-4 fraction shifted to lighter polysomes (H-3).\nThese results are consistent with a reduction in cap-dependent\nprotein synthesis associated with hypoxia (23, 25). Like other\nreports documenting mRNA redistribution among polysomes\nduring hypoxia, we do not observe a complete disaggregation of\n-actin (25, 28) or ODC mRNA into free mRNA (Fig. 1B),\ndespite an apparent flattening of the polysome peak (Fig. 1A).\nWe attribute this to the relative insensitivity of using spectro-\nphotometry to monitor polysomes and suspect that there is\nsome residual polysomes in the hypoxia-treated samples.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nTo assess Tie2 protein levels under hypoxic conditions, we\nperformed Western blot analysis on total protein isolated from\nHUVECs (Fig. 2A). HIF-2 protein was detected only in ex-\ntracts prepared from hypoxic cells confirming activation of the\nhypoxic response (Fig. 2A). In this Western blot analysis, an-\n**BLOCK**fs== 7.5**b== 0.2**t== 0.7**l== 0.1**r== 0.1**\nFIG. 1. Effect of hypoxia on Tie2 expression in HUVECs. A, A254 profile of polysomes obtained from HUVECs under normoxic or hypoxic\nconditions. B, RNA was purified from the indicated pooled fractions in A and quantified using real-time RT-PCR. For each gene, the relative\namount of each pooled fraction was calculated as a percentage of its total in the polysomes. The results are an average of three independent\nexperiments, and the standard deviations are shown. Primers used for quantitative RT-PCR of the hTie2 transcript were hTie2-(443– 423)\n(5-GATGAATTGCGAGATGGATAGGGCTTGAG-3) (targeting nts 257 to 285) and hTie2UP-52 (5-ACCTTCCACAGTTCCAGAAAGGAG-3)\n(targeting nts 43 to 66).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nti-H3 histone was used as a loading control for HIF-2 (Fig.\n2A). Consistent with the polysome profiling results (Fig. 1),\nTie2 protein was present in both normoxic and hypoxic treated\ncells (Fig. 2A). (In this experiment, a nonspecific band labeled\nwith an asterisk is used as a loading control for Tie2.) Tie2\nprotein was actively translated under both normoxic and hy-\npoxic conditions as assessed by metabolic labeling of HUVECs\nfollowed by immunoprecipitation with anti-Tie2 antibodies\n(Fig. 2B). In contrast, synthesis of -actin and ODC was sig-\nnificantly decreased after exposure of HUVECs to hypoxic\nstress (Fig. 2B). To gauge the extent to which translation in\nHUVECs is suppressed by hypoxia, we measured the rate of\n[35S]methionine incorporation into newly synthesized proteins\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.5**r== 0.1**\nin HUVECs under normoxic or hypoxic conditions (Fig. 2C).\nUnder hypoxic conditions, protein synthesis is reduced by 85%\nin HUVECs, in comparison with HUVECs under normoxic or\nreoxygenated conditions following hypoxic stress (Fig. 2C).\nTaken together these results indicate that translation of Tie2\nmRNA is resistant to the reduction in protein synthesis accom-\npanied by hypoxic treatement of HUVECs.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nThe levels of Tie2, VEGF, and -actin mRNAs in total RNA\nisolated from HUVECs were also measured by quantitative\nRT-PCR (Fig. 2D). VEGF mRNA levels increased 70-fold in\nresponse to hypoxia, whereas those of -actin decreased 4-\nfold (Fig. 2D). These results are consistent with previously\nreported effects of hypoxia on VEGF and -actin gene expres-\n**BLOCK**fs== 8.5**b== 0.4**t== 0.1**l== 0.5**r== 0.1**\ntion sites situated from 269 to 414 nucleotides upstream of the\ninitiator AUG codon (37). However from this report, it is not\npossible to assess which sites were detected by primer exten-\nsion versus those identified by 5-RACE. We re-mapped the\ntranscription initiation sites in the human Tie2 mRNA to iden-\ntify transcription start sites. Primer extension analysis was\nperformed with three oligonucleotides (hTie2-UP-6, hTie2-\nUP-7, hTie2-UP-1), which together span the entire length of\nthe Tie2 5-UTR (Fig. 3A). No specific products were visualized\nwith hTie2-UP-6 or hTie2-UP-7, indicating that these likely\nreside too far downstream of the transcription start site(s).\nHowever, two distinct extension products were obtained with\nhTie2-UP-1 (Fig. 3B). These fragments map to nts 372 and\n377 of the Tie2 5-UTR (Fig. 3D, encased within gray boxes).\nTo complement the primer extension results and determine\nwhether we could detect the same start sites, we performed\nRNase protection experiments. The probe used extended from\n467 to 224 (Fig. 3D, generated by primers hTie2-UP-4 and\nhTie2-UP-5). Two major protected fragments of 127 and 149\nnucleotides were observed (Fig. 3C). These mapped to positions\n372 and 350 of the Tie2 5-UTR (Fig. 3D). We note that the\nstart site at nucleotide 350 may have been missed in the\nprimer extension analysis, due to high background in this size\nrange of the gel (Fig. 3C). Taken together, the RNase protection\nand primer extension results indicate that a major Tie2 tran-\nscription site lies at nucleotide 372. This site also lies within\n2 nucleotides of the identified murine Tie2 transcription initi-\nation site (37). Sequence analysis of over 300 5-RACE PCR\nproducts demonstrated sequence contiguity with the Tie2\ngenomic DNA sequence, indicating the absence of a frequent\nsplicing event within the Tie2 5-UTR.2 Hence, the Tie2 mRNA\nisoforms identified here contain several features predicted to be\ninhibitory to a cap-dependent initiation mechanism, notably a\nlengthy 5-UTR (372 nts), five uORFs, and an AUG codon\nembedded with the last uORF (Fig. 3D). In this report, we\nthe\nfurther\n372 isoform.\n**BLOCK**fs== 8.5**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\ncharacterize the translational properties of\n**BLOCK**fs== 8.5**b== 0.1**t== 0.6**l== 0.5**r== 0.1**\nThe Tie2 5-UTR Mediates Internal Initiation—One mecha-\nnism by which Tie2 could be efficiently translated under hy-\npoxic conditions (Fig. 2B), a situation that reduces cap-depend-\nent translation initiation (16, 26), would be if Tie2 translation\nproceeded by internal ribosome recruitment to the 5-UTR. To\ntest this, the Tie2 5-UTR was positioned within the intercis-\ntronic spacer between the firefly and Renilla luciferase coding\nregions (Fig. 4A). We also generated a construct in which all\nuAUGs had been removed by mutating AUG to UUG, to assess\ntheir potential contribution to IRES activity (Fig. 4A). As a\nnegative control for these experiments, we utilized a deletion\nmutant of the EMCV 5-UTR that does not support internal\nribosome binding (Fig. 4A, construct Ren/EMC/FF). The\nlength of the EMC sequence is 440 bp, similar to the size of\nthe Tie2 5-UTR used in these experiments (372 bp). In addi-\ntion, we generated constructs in which the human immunode-\nficiency virus TAR element (TAR(III)) was positioned up-\nstream of Renilla luciferase. This allowed us to probe whether\nfirefly luciferase expression was independent of Renilla expres-\nsion and not a consequence of ribosome reinitation (Fig. 4A).\nThe stable secondary structure of this element (G  49.9\nkcal/mol) is expected to act as a physical barrier to 5-end-\nmediated initiation and decrease expression of the Renilla lu-\nciferase protein (27).\n**BLOCK**fs== 8.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\nTransfection experiments with HUVECs revealed that ex-\npression vectors containing the TAR(III) element upstream of\nthe Renilla luciferase initiation codon displayed a 5-fold re-\n**BLOCK**fs== 7.5**b== 0.0**t== 1.0**l== 0.5**r== 0.2**\n2 E.-H. Park and J. Pelletier, unpublished data.\n**BLOCK**fs== 7.5**b== 0.2**t== 0.5**l== 0.1**r== 0.5**\nFIG. 2. Tie2 protein production is maintained under hypoxic\nconditions. A, Tie2 protein expression in HUVECs exposed to hypoxia\n(H; 1% O2) or normoxia (N; 20% O2) for 16 and 40 h. Upper panel,\nWestern blot analysis performed using polyclonal anti-HIF-2 (Novus\nBiologicals, Inc., Littleton, CO) antibodies. Middle panel, Western blot\nanalysis performed using histone H3 antisera (Cell Signaling, Beverly,\nMA). Lower panel, Western blot analysis performed using anti-Tie2\n(Santa Cruz Biotechnology, Inc.) antibodies. A nonspecific band (indi-\ncated by an asterisk) was used as a loading control. B, metabolic label-\ning of Tie2 under hypoxic conditions. HUVECs were metabolically la-\nbeled with [35S]methionine under normoxia (N) or hypoxia (H) and\nimmunoprecipitated with anti-Tie2 or anti--actin antisera. Immuno-\nprecipiates were resolved by SDS-PAGE and visualized by fluorogra-\nphy. C, metabolic labeling using [35S]methionine in HUVECs under\nconditions of normoxia, hypoxia, or reoxygenation. HUVEC cells were\npreincubated for 40 h under hypoxia or normoxia or incubated for 40 h\nunder hypoxia, followed by 1 h ofnormoxia (reoxygenated). Radiola-\nbeled methionine incorporation under normoxia is also presented in the\npresence of 20 g/ml cycloheximide. Cells were incubated in the pres-\nence of [35S]methionine for 10, 30, or 60 min, then immediately har-\nvested. Trichloroacetic acid precipitation of cell lysates was performed\nin duplicate and used to determine the incorporation of [35S]methionine\ninto protein. Each measurement was normalized with respect to total\nprotein as determined by the Lowry protein assay (51). The results\nrepresent the average of three independent experiments performed,\nand the error bars denote the error of the mean. D, total RNA was\nisolated from cells and used as template for real-time RT-PCR. The\nrelative mRNA amounts were normalized to the abundance of the 28 S\nrRNA and averaged for three independent experiments.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.8**l== 0.1**r== 0.5**\nsions (32–34). Under hypoxic conditions, Tie2 mRNA levels\ndecreased 4-fold, consistent with the results reported by\nMandriota and Pepper (35).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nMapping the Human Tie2 5-Untranslated Region—In an\nattempt to understand how Tie2 translation is maintained\nunder hypoxic stress, during a time when overall cap-depend-\nent protein synthesis is attenuated, we undertook a structure/\nfunction approach to mapping of the Tie2 5-UTR. The tran-\nscription start sites of the human Tie2 mRNA have been\npreviously defined using 5-RACE and primer extension (36).\nThe results suggested the presence of 10 transcription initia-\n**BLOCK**fs== 7.5**b== 0.4**t== 0.2**l== 0.1**r== 0.7**\nFIG. 3. Determination of the tran-\nscription start site(s) of human Tie2\nmRNA. A, schematic representation of\nthe human Tie2 5-UTR. The uORFs are\ndepicted as well as their relative reading\nframes by different shades of gray. Aster-\nisks above an uORF indicates the pres-\nence of an internal AUG, in-frame with\nthe uORF initiator AUG. B, mapping of\nthe transcription start sites of the human\nTie2 gene by primer extension. Total RNA\nfrom HUVECs, human placental poly(A)\nRNA, and yeast tRNA were analyzed us-\ning oligonucleotide hTie2-UP-1 as a\nprimer. Sequencing reactions were used\nas molecular markers. The two major\nproducts (arrows) terminate at adenosine\nresidues and are encased in the gray\nboxes in D. The smaller products observed\nfrom the human placenta mRNA prepara-\ntions were not reproducibly seen among\ndifferent experiments. C, RNase protec-\ntion mapping of the human Tie2 mRNA.\nThe RNase protection assay (RPA) of hu-\nman placental poly(A) RNA or yeast\nRNA was performed as described under\n“Experimental Procedures.” Sequencing\nreactions were used as molecular mark-\ners. The arrows indicate the size and po-\nsition of migration of the protected frag-\nments. D, nucleotide sequence upstream\nof the human Tie2 initiation codon. The\nTie2 translation start codon is designated\nas 1, and five uORFs are indicated in\nbold and italics. The locations of the oli-\ngonucleotides used for primer extension\nand RPA are shown. The transcription\nstart sites determined by primer exten-\nsion and RPA are indicated by gray boxes\nand gray circles above the nucleotide,\nrespectively.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.1**r== 0.5**\nduction in Renilla luciferase activity relative to the constructs\nlacking this stem-loop barrier (Fig. 4B, compare lanes 1–3 to\nlanes 4 – 6). Very low levels of firefly luciferase activity were\ndetectable from cells transfected with Ren/EMC/FF or\nTAR(III)/Ren/EMC/FF, since the EMC 5-UTR does not\nallow for efficient reinitiation or for internal ribosome recruit-\nment (Fig. 4B, lanes 1 and 4). However, significant and com-\nparable firefly luciferase activities were detected in extracts\nfrom cells transfected with Ren/Tie2/FF and TAR(III)/Ren/\nTie2/FF (Fig. 4B, compare lane 2 with lane 5) or with Ren/\nTie2uAUG/FF and TAR(III)/Ren/Tie2uAUG/FF (Fig. 4B,\ncompare lane 3 with lane 6). These results indicate that the\nTie2 5-UTR mediates the production of firefly luciferase inde-\npendent of Renilla luciferase, as would be expected if an IRES\nwas present.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.1**r== 0.5**\nIRES Activity Is Maintained during Hypoxia—One function\nof the Tie2 IRES might be to maintain efficient translation\nduring hypoxia (Fig. 1). To assess whether the putative Tie2\nIRES could function in a bicistronic context under hypoxic\nconditions, we transfected HUVECs with Ren/EMC/FF or\nRen/Tie2AUG/FF and placed these cells under normoxic or\nhypoxic conditions (Fig. 4C). Luciferase activities were then\nmeasured in cell extracts. We observed that the relative Renilla\nluciferase activity decreased 2-fold when cells were exposed\nto hypoxia (Fig. 4C). This effect is likely an underestimation of\nthe true decrease in protein synthesis, since it relies on meas-\nuring enzyme activity, some of which will be already present\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\nprior to the start of hypoxic treatment. Expression of the firefly\nluciferase cistron was increased slightly under hypoxic condi-\ntions in Ren/Tie2uAUG/FF (Fig. 4C). The hypoxic/normoxic\nratio indicates that the potential Tie2 IRES activity was even\nslightly stimulated under hypoxic conditions (Fig. 4C). These\nresults indicate that the putative Tie2 IRES is functional dur-\ning hypoxic treatment.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\nNorthern blot analysis indicates that bicistronic transcripts\nproduced in vivo from Ren/EMC/FF, Ren/Tie2/FF, and Ren/\nTie2AUG/FF were of comparable quality (Fig. 4D). No major\ndegradation product was visible that could account for the\ndifferential expression of firefly luciferase from Ren/Tie2/FF\nand Ren/Tie2AUG/FF, relative to Ren/EMC/FF (Fig. 4B).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nAnother alternative interpretation for the observed results\nwould be the existence of cryptic splice sites upstream of the\nfirefly luciferase coding region that could eliminate a portion,\nor all of, the upstream Renilla sequences. Subsequent transla-\ntion of a spliced transcript could then lead to increased firefly\nluciferase levels, without the need to invoke the existence of an\nIRES. As a test for this, we performed RT-PCR on total RNA\nisolated from HUVECs that had been transfected with Ren/\nTie2/FF or Ren/Tie2uAUG/FF (Fig. 5A). PCR primers were\ndesigned that targeted the 5-UTR of Renilla and within the\nfirefly luciferase coding region (Fig. 5A, left panel). A single\nDNA product was obtained following RT-PCR of RNA from\ntransfected cells and corresponds in size to the expected full-\nlength PCR product encompassing the Renilla ORF and Tie2\n**BLOCK**fs== 7.5**b== 0.3**t== 0.5**l== 0.1**r== 0.1**\nFIG. 4. The Tie2 5-UTR mediates internal initiation in vivo. A, schematic representation of bicistronic constructs containing the Tie2\n5-UTR in the intercistronic region. The Renilla and firefly luciferase ORFs are denoted by a white and black box, respectively. The stem-loop\nstructure upstream of the Renilla ORF in the TAR(III) construct series represents the human immunodeficiency virus TAR element. B,\nquantitation of luciferase activities following DNA transfection into HUVECs. The relative light units per microgram of protein extract are denoted\nfor both the Renilla and firefly luciferase proteins. The construct number is indicated below the x axis. The results represent the average of three\nindependent experiments performed, and the error bars denote the error of the mean. C, quantitation of Renilla luciferase activities obtained from\ntransfected cells exposed to normoxia or hypoxia. Luciferase activities were determined, normalized to the protein content of each extract, then set\nrelative to the value obtained for Ren/Tie2AUG/FF under normoxic conditions. Ratios of the hypoxic to normoxic values are also calculated and\nplotted. Values plotted are the average of three independent experiments with the standard error presented. D, Northern blot of RNA isolated from\ntransfected HUVECs and probed for firefly luciferase. The nature of the construct is indicated above the panel and corresponds to the numbering\nscheme used in A. The arrowhead indicates the position of migration of the bicistronic mRNA, and the bottom panel is the same blot reprobed with\nglyceraldehyde-3-phosphate dehydrogenase (GAPDH), to account for possible variations in loading.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.7**l== 0.1**r== 0.5**\n5-UTR (1.7 kbp) (Fig. 5A). PCR products were not observed\nwhen reverse transcriptase was omitted from the RT-PCR (Fig.\n5A). From this experiment, no evidence of splicing was ob-\ntained as assessed by the lack of smaller PCR products (whose\namplification should be favored during the PCR) (Fig. 5A).\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.1**r== 0.5**\nTo ensure that the results obtained in Fig. 4 were not due to\nthe presence of a cryptic promoter within the Tie2 5-UTR, we\ngenerated expression vectors in which the Tie2 5-UTR was\nplaced upstream of the firefly luciferase gene in a promoterless\nplasmid backbone (Fig. 5B). A firefly reporter construct con-\ntaining SV40 promoter (SV40/Luc) was used as a positive con-\ntrol. Transfection of these reporter constructs into HUVECs\nshowed that pGL-Basic (negative control) and hTie2/Luc con-\nstructs produced similar levels of low luciferase activity (Fig.\n5C). On the other hand, SV40/Luc yielded over 100 times\nhigher luciferase activity compared with hTie2/Luc and pGL-\nBasic (Fig. 5C). These results indicate the absence of a detect-\nable cryptic promoter activity within the Tie2 5-UTR.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nTo confirm and extend these results, we performed a series of\nmRNA transfection experiments. We generated capped and\npolyadenylated mRNAs from bicistronic constructs containing:\n(i) the EMCV 5-UTR with the deleted IRES (negative control),\n(ii) the poliovirus IRES (positive control), and (iii) the Tie2\n**BLOCK**fs== 8.5**b== 0.1**t== 0.7**l== 0.5**r== 0.1**\n5-UTR containing or lacking all uAUGs (Fig. 6A). Following\ntransfection of mRNAs into HUVECs, firefly and Renilla pro-\nduction was assessed by Western blotting (Fig. 6B) or by meas-\nuring luciferase activities (Fig. 6C). Western blotting experi-\nments revealed that production of Renilla luciferase was\nsimilar among the four transfected mRNA species, whereas the\nlevels of firefly luciferase differed among expression constructs\n(Fig. 6B). No firefly luciferase was observed from cells trans-\nfected with TAR(III)/Ren/EMC/FF (lane 3), consistent with\nthe absence of an IRES in this construct. In contrast, transfec-\ntion of TAR(III)/Ren/Polio/FF mRNA into HUVECs produced\nhigh levels of firefly luciferase (lane 4). Firefly luciferase was\nalso detectable from cells transfected with TAR(III)/Ren/\nTie2/FF and TAR(III)/Ren/Tie2uAUG/FF mRNAs, with\nslightly more being produced from the construct lacking\nuAUGs (Fig. 6B, compare lane 1 with lane 2).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\nThese results were confirmed by monitoring Renilla and\nfirefly luciferase activities (Fig. 6C). The FF/Ren ratio obtained\ntransfected with TAR(III)/Ren/Polio/FF\nfrom HUVECs\nmRNA was 100-fold greater than that obtained from cells\ntransfected with TAR(III)/Ren/EMC/FF mRNA, consistent\nwith the presence of an IRES in the former transcript (Fig. 6C).\nThe FF/Ren ratio of TAR(III)/Ren/Tie2uAUG/FF trans-\n**BLOCK**fs== 7.5**b== 0.4**t== 0.2**l== 0.1**r== 0.7**\nFIG. 5. Absence of cryptic splicing\nwithin the Tie2 5-UTR. A, RT-PCR\nanalysis of total RNA isolated from trans-\nfected HUVECs. PCRs were performed\nfrom plasmid DNA (lanes labeled D) as\npositive controls. RT-PCRs were per-\nformed from RNA in which RT was added\n() or omitted (). The expected size of\nthe PCR products is indicated. Molecular\nsize markers (M) are 1-kbp ladders from\nNew England Biolabs. The location of the\nPCR primers used in this amplification\nare schematically shown. B, assessing\npromoter activity of the Tie2 5-UTR.\nSchematic representation of constructs\nused to test for cryptic promoter activity\nwithin the Tie2 5-UTR is shown. The\nfirefly luciferase ORF is denoted by a\nblack box. The pGL-Basic vector (Pro-\nmega) lacks a defined active promoter.\nThe Tie2 5-UTR (nts 372 to 69) was\nplaced upstream of the firefly luciferase\ngene. SV40/Luc contains the SV40 pro-\nmoter driving transcription of the firefly\nluciferase gene and was used as a positive\ncontrol. C, HUVECs were transfected\nwith pGL-Basic, hTie2/Luc, or SV40/Luc\nin conjunction with the plasmid pRL-\nCMV (Promega). Lysates were prepared\nfrom the cells following transfection, and\nluciferase activities were measured. The\nrelative light units per microgram of pro-\ntein extract are shown for the firefly lu-\nciferase protein. To account for variations\nin transfection efficiency, firefly lucifer-\nase activity was normalized to that of Re-\nnilla luciferase. The results represent the\naverage of\nthree independent experi-\nments performed in duplicate, and the er-\nror bars denote the error of the mean.\n**BLOCK**fs== 8.5**b== 0.3**t== 0.6**l== 0.1**r== 0.5**\nfected cells was 8-fold higher than that of the negative con-\ntrol. These results indicate that the observed Tie2 IRES activ-\nity is RNA-dependent and not due to the presence of a cryptic\npromoter. TAR(III)/Ren/Tie2/FF exhibited 3-fold lower FF/\nRen ratio than that detected with TAR(III)/Ren/Tie2uAUG/\nFF, further suggesting that the uORFs within the Tie2 5-UTR\nnegatively affect IRES-mediated translation (see “Discussion”).\n**BLOCK**fs== 7.0**b== 0.3**t== 0.7**l== 0.3**r== 0.7**\nDISCUSSION\n**BLOCK**fs== 8.5**b== 0.0**t== 0.7**l== 0.1**r== 0.5**\nThe angiopoietin-Tie2 signaling pathway plays an important\nrole in the development of normal and tumor vasculature. Tie2\nexpression is elevated in human cancers (breast, ovarian, hep-\natocellular, glioblastoma) with expression being localized in\n“vascular hot spots” at the leading edge of invasive tumors\n(reviewed by Peters et al. (38)). Conflicting reports have been\npresented regarding changes in Tie2 protein and mRNA levels\nduring hypoxia. Increases in Tie2 protein expression have been\nreported to parallel increases in Tie2 mRNA levels (39 – 41). In\ncontrast, Mandriota and Pepper (35) have reported decreased\nTie2 mRNA levels during hypoxia, and Lin et al. (42) indicated\nunchanged Tie2 protein levels under hypoxia. The reasons for\nthese discrepancies are not clear but may reflect response\ndifferences to hypoxia among the different cell lines. In our\nhands, Tie2 mRNA levels decrease as a consequence of hypoxia\ntreatment (Fig. 2D) with protein production continuing at lev-\nels comparable with those observed during normoxia (Fig. 2B).\nOur study indicates that Tie2 protein production is secured\nunder hypoxic conditions (Fig. 2B), when general translation\nis reduced.\n**BLOCK**fs== 8.5**b== 0.2**t== 0.6**l== 0.5**r== 0.1**\nThe presence of an IRES within the Tie2 5-UTR provides an\nexplanation for how this mRNA is able to escape the reduction\nin translation imposed by hypoxia. Exposure of cells to hypoxic\nconditions has been shown to be associated with a decrease in\ncap-dependent translation as a result of increased formation of\nthe eIF4E/4E-BP1 inhibitory complex (16, 43). In addition,\nincreased phosphorylation of eIF2 associated with hypoxia\nalso contributes to decreased general protein synthetic rates\nbut increases the production of a specific set of mRNAs (15).\nThe idea that the Tie2 IRES allows an escape from translation\nrepression during hypoxia is supported by our experiments in\nwhich the Tie2 IRES activity was maintained, and even\nslightly stimulated, under hypoxic conditions (Fig. 4C).\n**BLOCK**fs== 8.5**b== 0.0**t== 0.8**l== 0.5**r== 0.1**\nThe Tie2 mRNA 5-UTR contains five uORFs, an unusually\nhigh number for a eukaryotic transcript (Fig. 3D). The primer\ncombination used to assess the polysome distribution of the\nhTie2 mRNA in Fig. 1B spanned five uORFs and the first 60\nnucleotides following the initiation codon (Fig. 3D). Our quan-\ntitative RT-PCR results demonstrate that the Tie2 transcript\nobtained using these primers is associated with polysomes\nunder both normoxic and hypoxic conditions, indicating that\ntranslating Tie2 mRNA contains this complex 5-UTR with its\nfive uORFs. We note that in both DNA and RNA transfections,\nexpression of firefly luciferase is increased when all of the Tie2\nuAUGs are deleted (Figs. 4B and 6). A simple interpretation of\nthese results is that some (or all) of the uORFs reside between\nthe IRES and the initiator ATG and are encountered by the\ninternally initiated ribosome. This could have the consequence\n**BLOCK**fs== 8.5**b== 0.7**t== 0.1**l== 0.5**r== 0.1**\nallow specific gene expression during a time when global trans-\nlation is reduced. We would expect that additional genes whose\nexpressions are maintained during hypoxia represent excellent\ncandidates for harboring IRESs. Additionally, the Tie2/angio-\npoietin-1 signaling system has been implicated in regulating\nquiescence of hematopoietic stem cells in the bone marrow\nniche (49). Tie2 expression needs to be maintained during\nquiescence, a physiological state associated with reduced pro-\ntein synthesis (50). Hence, the Tie2 IRES described herein may\nserve to circumvent the translation block imposed by quies-\ncence to keep Tie2 protein levels constant.\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.1**\nAcknowledgments—We thank the anonymous reviewers for their\n**BLOCK**fs== 7.5**b== 0.7**t== 0.3**l== 0.5**r== 0.2**\nhelpful comments in improving the manuscript.\n**BLOCK**fs== 7.0**b== 0.7**t== 0.3**l== 0.7**r== 0.2**\nREFERENCES\n**BLOCK**fs== 6.5**b== 0.6**t== 0.3**l== 0.5**r== 0.1**\n1. Risau, W. (1997) Nature 386, 671– 674\n2. Carmeliet, P., Ferreira, V., Breier, G., Pollefeyt, S., Kieckens, L., Gertsenstein,\nM., Fahrig, M., Vandenhoeck, A., Harpal, K., Eberhardt, C., Declercq, C.,\nPawling, J., Moons, L., Collen, D., Risau, W., and Nagy, A. (1996) Nature\n380, 435– 439\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n3. Ferrara, N., Carver-Moore, K., Chen, H., Dowd, M., Lu, L., O’Shea, K. S.,\nPowell-Braxton, L., Hillan, K. J., and Moore, M. W. (1996) Nature 380,\n439 – 442\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\n4. Shalaby, F., Rossant, J., Yamaguchi, T. P., Gertsenstein, M., Wu, X. F.,\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.5**r== 0.1**\nBreitman, M. L., and Schuh, A. C. (1995) Nature 376, 62– 66\n5. Griffioen, A. W., and Molema, G. (2000) Pharmacol. Rev. 52, 237–268\n6. Maxwell, P. H., Pugh, C. W., and Ratcliffe, P. J. (2001) Curr. Opin. Genet. Dev.\n**BLOCK**fs== 6.5**b== 0.6**t== 0.4**l== 0.6**r== 0.4**\n11, 293–299\n**BLOCK**fs== 6.5**b== 0.5**t== 0.4**l== 0.5**r== 0.1**\n7. Semenza, G. L. (2002) Trends Mol. Med. 8, S62–S67\n8. Suri, C., Jones, P. F., Patan, S., Bartunkova, S., Maisonpierre, P. C., Davis, S.,\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.6**r== 0.1**\nSato, T. N., and Yancopoulos, G. D. (1996) Cell 87, 1171–1180\n**BLOCK**fs== 6.5**b== 0.5**t== 0.5**l== 0.5**r== 0.1**\n9. Dumont, D. J., Gradwohl, G., Fong, G. H., Puri, M. C., Gertsenstein, M.,\nAuerbach, A., and Breitman, M. L. (1994) Genes Dev. 8, 1897–1909\n10. Vikkula, M., Boon, L. M., Carraway, K. L., 3rd, Calvert, J. T., Diamonti, A. J.,\nGoumnerov, B., Pasyk, K. A., Marchuk, D. A., Warman, M. L., Cantley,\nL. C., Mulliken, J. B., and Olsen, B. R. (1996) Cell 87, 1181–1190\n11. Maisonpierre, P. C., Suri, C., Jones, P. F., Bartunkova, S., Wiegand, S. J.,\nRadziejewski, C., Compton, D., McClain, J., Aldrich, T. H., Papadopoulos,\nN., Daly, T. J., Davis, S., Sato, T. N., and Yancopoulos, G. D. (1997) Science\n277, 55– 60\n**BLOCK**fs== 6.5**b== 0.4**t== 0.5**l== 0.5**r== 0.1**\n12. Lin, P., Buxton, J. A., Acheson, A., Radziejewski, C., Maisonpierre, P. C.,\nYancopoulos, G. D., Channon, K. M., Hale, L. P., Dewhirst, M. W., George,\nS. E., and Peters, K. G. (1998) Proc. Natl. Acad. Sci. U. S. A. 95, 8829 – 8834\n13. Jones, N., Voskas, D., Master, Z., Sarao, R., Jones, J., and Dumont, D. J. (2001)\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.6**r== 0.3**\nEMBO Rep. 2, 438 – 445\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n14. Papapetropoulos, A., Fulton, D., Mahboubi, K., Kalb, R. G., O’Connor, D. S., Li,\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.6**r== 0.1**\nF., Altieri, D. C., and Sessa, W. C. (2000) J. Biol. Chem. 275, 9102–9105\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.5**r== 0.1**\n15. Koumenis, C., Naczki, C., Koritzinsky, M., Rastani, S., Diehl, A., Sonenberg,\nN., Koromilas, A., and Wouters, B. G. (2002) Mol. Cell. Biol. 22, 7405–7416\n16. Arsham, A. M., Howell, J. J., and Simon, M. C. (2003) J. Biol. Chem. 278,\n**BLOCK**fs== 6.5**b== 0.4**t== 0.6**l== 0.6**r== 0.4**\n29655–29660\n**BLOCK**fs== 6.5**b== 0.3**t== 0.6**l== 0.5**r== 0.1**\n17. Clemens, M. J. (2001) J. Cell. Mol. Med. 5, 221–239\n18. Gingras, A. C., Raught, B., and Sonenberg, N. (1999) Annu. Rev. Biochem. 68,\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.6**r== 0.4**\n913–963\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n19. Gehrke, L., Auron, P. E., Quigley, G. J., Rich, A., and Sonenberg, N. (1983)\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\nBiochemistry 22, 5157–5164\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n20. Svitkin, Y. V., Pause, A., Haghighat, A., Pyronnet, S., Witherell, G., Belsham,\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.6**r== 0.2**\nG. J., and Sonenberg, N. (2001) RNA (N. Y.) 7, 382–394\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n21. Huez, I., Creancier, L., Audigier, S., Gensac, M. C., Prats, A. C., and Prats, H.\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n(1998) Mol. Cell. Biol. 18, 6178 – 6190\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.5**r== 0.1**\n22. Akiri, G., Nahari, D., Finkelstein, Y., Le, S. Y., Elroy-Stein, O., and Levi, B. Z.\n**BLOCK**fs== 6.5**b== 0.3**t== 0.7**l== 0.6**r== 0.3**\n(1998) Oncogene 17, 227–236\n**BLOCK**fs== 6.5**b== 0.2**t== 0.7**l== 0.5**r== 0.1**\n23. Stein, I., Itin, A., Einat, P., Skaliter, R., Grossman, Z., and Keshet, E. (1998)\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nMol. Cell. Biol. 18, 3112–3119\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n24. Miller, D. L., Dibbens, J. A., Damert, A., Risau, W., Vadas, M. A., and Goodall,\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.6**r== 0.3**\nG. J. (1998) FEBS Lett. 434, 417– 420\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n25. Lang, K. J., Kappel, A., and Goodall, G. J. (2002) Mol. Biol. Cell 13, 1792–1801\n26. Johannes, G., Carter, M. S., Eisen, M. B., Brown, P. O., and Sarnow, P. (1999)\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nProc. Natl. Acad. Sci. U. S. A. 96, 13118 –13123\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n27. Parkin, N. T., Cohen, E. A., Darveau, A., Rosen, C., Haseltine, W., and\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.6**r== 0.2**\nSonenberg, N. (1988) EMBO J. 7, 2831–2837\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n28. Gorlach, A., Camenisch, G., Kvietikova, I., Vogt, L., Wenger, R. H., and\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.6**r== 0.1**\nGassmann, M. (2000) Biochim. Biophys. Acta 1493, 125–134\n**BLOCK**fs== 6.5**b== 0.2**t== 0.8**l== 0.5**r== 0.1**\n29. Rousseau, D., Kaspar, R., Rosenwald, I., Gehrke, L., and Sonenberg, N. (1996)\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.6**r== 0.2**\nProc. Natl. Acad. Sci. U. S. A. 93, 1065–1070\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n30. Seidel, E. R., and Ragan, V. L. (1997) Br. J. Pharmacol. 120, 571–574\n31. Pyronnet, S., Pradayrol, L., and Sonenberg, N. (2000) Mol. Cell 5, 607– 616\n32. Shweiki, D., Itin, A., Soffer, D., and Keshet, E. (1992) Nature 359, 843– 845\n33. Forsythe, J. A., Jiang, B. H., Iyer, N. V., Agani, F., Leung, S. W., Koos, R. D.,\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.6**r== 0.2**\nand Semenza, G. L. (1996) Mol. Cell. Biol. 16, 4604 – 4613\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n34. Levy, A. P., Levy, N. S., Wegner, S., and Goldberg, M. A. (1995) J. Biol. Chem.\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.6**r== 0.4**\n270, 13333–13340\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.5**r== 0.1**\n35. Mandriota, S. J., and Pepper, M. S. (1998) Circ. Res. 83, 852– 859\n36. Hewett, P. W., Daft, E. L., and Murray, J. C. (1998) Biochem. Biophys. Res.\n**BLOCK**fs== 6.5**b== 0.1**t== 0.9**l== 0.6**r== 0.3**\nCommun. 252, 546 –551\n**BLOCK**fs== 6.5**b== 0.0**t== 0.9**l== 0.5**r== 0.1**\n37. Fadel, B. M., Boutet, S. C., and Quertermous, T. (1998) Biochem. J. 330,\n**BLOCK**fs== 6.5**b== 0.0**t== 1.0**l== 0.6**r== 0.4**\n335–343\n**BLOCK**fs== 7.5**b== 0.4**t== 0.5**l== 0.1**r== 0.5**\nFIG. 6. Messenger RNA transfection into HUVEC cells. A, sche-\nmatic representation of mRNA species used in mRNA transfections. B,\nWestern blot analysis of extracts from HUVECs transfected with bicis-\ntronic mRNAs shown in A. Cell lysates were analyzed by immunoblot-\nting with anti-Renilla luciferase and anti-firefly luciferase antisera.\nThe nature of the transfected mRNA is indicated above the lanes. C,\nluciferase activities were measured from HUVEC lysates following\nRNA transfection. The relative FF/Ren ratio obtained for each mRNA\ncontruct is shown. The ratio of the negative control, TAR(III)/Ren/\nEMC/FF, was arbitrarily set to 1. The results represent the average of\nthree independent experiments performed in duplicate, and the error\nbars denote the error of the mean.\n**BLOCK**fs== 8.5**b== 0.1**t== 0.6**l== 0.1**r== 0.5**\nof reducing translation initiation, due to ribosomes dropping off\nduring re-initiation, or bypassing the initiation codon of the\nmajor ORF, due to an inability to re-acquire a ternary complex.\nAnother possibility for the role of the Tie2 uORFs in trans-\nlation initiation could be to simply prevent the majority of\nribosomes that have initiated by a cap-dependent mechanism\nfrom reaching the Tie2 initiation codon, thereby acting as a\ndamper to prevent 5-end-mediated initiation from interfering\nwith IRES activity on the Tie2 mRNA template. Alternatively,\nthe structure of the Tie2 IRES may be dynamically affected by\nscanning ribosomes. In this situation, ribosomes initiated in a\ncap-dependent fashion could affect Tie2 5-UTR secondary\nstructure and impact on IRES function. Such a model has been\nimplicated in the function of the cat-1 IRES, where disruption\nof RNA-RNA interactions between the 5- and 3-end of an\nIRES prevents inducible internal initiation. The cat-1 IRES\nactivity is restored upon inhibition of 5-end mediation initia-\ntion (44). Whether a similar scenario occurs on the Tie2 IRES\nneeds to be experimentally addressed and may indicate that\ntranslation initiation on the Tie2 mRNA is more dynamic and\ncomplex than expected.\n**BLOCK**fs== 8.5**b== 0.0**t== 0.9**l== 0.1**r== 0.5**\nOur findings indicate that Tie2 belongs to a class of mRNA\ntranscripts harboring IRES activity to circumvent the reduc-\ntion of protein synthesis imposed during hypoxic stress. Other\nmRNAs in this category include c-myc (45– 48), HIF-1 (25),\nand VEGF (21–24). The presence of IRESs in the 5-UTRs of\ntwo mRNAs encoding major angiogenic factors (VEGF and\nTie2) suggest that this translation initiation mechanism plays\nan important role in the control of angiogenesis. The advantage\nof IRES-dependent expression during this process might be to\n**BLOCK**fs== 6.5**b== 0.9**t== 0.1**l== 0.1**r== 0.5**\n38. Peters, K. G., Kontos, C. D., Lin, P. C., Wong, A. L., Rao, P., Huang, L.,\nDewhirst, M. W., and Sankar, S. (2004) Recent Prog. Horm. Res. 59, 51–71\n**BLOCK**fs== 6.5**b== 0.9**t== 0.1**l== 0.5**r== 0.1**\n45. Johannes, G., and Sarnow, P. (1998) RNA (N. Y.) 4, 1500 –1513\n46. Nanbru, C., Prats, A. C., Droogmans, L., Defrance, P., Huez, G., and Kruys, V.\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n39. Park, Y. S., Kim, N. H., and Jo, I. (2003) Microvasc. Res. 65, 125–131\n40. Willam, C., Koehne, P., Jurgensen, J. S., Grafe, M., Wagner, K. D., Bachmann,\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n(2001) Oncogene 20, 4270 – 4280\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n47. Stoneley, M., Paulin, F. E., Le Quesne, J. P., Chappell, S. A., and Willis, A. E.\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.6**\nS., Frei, U., and Eckardt, K.-U. (2000) Circ. Res. 87, 370 –377\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\n(1998) Oncogene 16, 423– 428\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n41. Christensen, R. A., Fujikawa, K., Madore, R., Oettgen, P., and Varticovski, L.\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n48. Nanbru, C., Lafon, I., Audigier, S., Gensac, M. C., Vagner, S., Huez, G., and\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.7**\n(2002) J. Cell. Biochem. 85, 505–515\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n42. Lin, S., Shyu, K. G., Lee, C. C., Wang, B. W., Chang, C. C., Liu, Y. C., Huang,\nF. Y., and Chang, H. (2002) Biochem. Biophys. Res. Commun. 296, 710 –715\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.1**r== 0.5**\n43. Tinton, S. A., and Buc-Calderon, P. M. (1999) FEBS Lett. 446, 55–59\n44. Yaman, I., Fernandez, J., Liu, H., Caprara, M., Komar, A. A., Koromilas, A. E.,\nZhou, L., Snider, M. D., Scheuner, D., Kaufman, R. J., and Hatzoglou, M.\n(2003) Cell 113, 519 –531\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nPrats, A. C. (1997) J. Biol. Chem. 272, 32061–32066\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n49. Arai, F., Hirao, A., Ohmura, M., Sato, H., Matsuoka, S., Takubo, K., Ito, K.,\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.2**\nKoh, G. Y., and Suda, T. (2004) Cell 118, 149 –161\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n50. Hofmann, G. E., and Hand, S. C. (1994) Proc. Natl. Acad. Sci. U. S. A. 91,\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.4**\n8492– 8496\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.5**r== 0.1**\n51. Lowry, O. H., Rosebrough, N. J., Farr, A. L., and Randall, R. J. (1951) J. Biol.\n**BLOCK**fs== 6.5**b== 0.8**t== 0.2**l== 0.6**r== 0.3**\nChem. 193, 265–275",
         "extracted",
         "",
         "Internal Translation Initiation Mediated by the Angiogenic Factor Tie2*",
         "None"
        ]
       ],
       "shape": {
        "columns": 9,
        "rows": 2117
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>abstract</th>\n",
       "      <th>authorName</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>pdf_blocks</th>\n",
       "      <th>pdf_status</th>\n",
       "      <th>referenceTitles</th>\n",
       "      <th>title</th>\n",
       "      <th>valid_pages</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001d5fbff6f7c763a78dd5d141292416dcfae59</td>\n",
       "      <td>Purpose: Synovial sarcoma is a soft tissue sar...</td>\n",
       "      <td>T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...</td>\n",
       "      <td>https://aacrjournals.org/clincancerres/article...</td>\n",
       "      <td>\\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0....</td>\n",
       "      <td>extracted</td>\n",
       "      <td></td>\n",
       "      <td>Disruption of Fibroblast Growth Factor Signal ...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>000372290caae5482dcdd7954feb895ccbab921d</td>\n",
       "      <td>Shuffled versions of iterative decoding of low...</td>\n",
       "      <td>Juntan Zhang,M. Fossorier</td>\n",
       "      <td>http://www.merl.com/publications/docs/TR2005-0...</td>\n",
       "      <td>\\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0....</td>\n",
       "      <td>extracted</td>\n",
       "      <td>Low-Density Parity-Check Codes;Turbo decoder a...</td>\n",
       "      <td>Shuffled iterative decoding</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>000376ec6f6ade6261fc12df9eddd25a88c1e9ca</td>\n",
       "      <td>This study was designed to investigate the eff...</td>\n",
       "      <td>Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...</td>\n",
       "      <td>https://www.jstage.jst.go.jp/article/bpb/28/8/...</td>\n",
       "      <td>\\n**BLOCK**fs== 14.0**b== 0.8**t== 0.1**l== 0....</td>\n",
       "      <td>extracted</td>\n",
       "      <td></td>\n",
       "      <td>Diphasic effects of Astragalus membranaceus BU...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0003a5747ee3676be1b6d2c72c13a04e41811274</td>\n",
       "      <td>In this report, we use zebrafish as a model sy...</td>\n",
       "      <td>Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....</td>\n",
       "      <td>https://onlinelibrary.wiley.com/doi/pdfdirect/...</td>\n",
       "      <td>\\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1...</td>\n",
       "      <td>extracted</td>\n",
       "      <td></td>\n",
       "      <td>Critical roles of CD146 in zebrafish vascular ...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>00043e28cffe744909bf69cde70e6dac0bce0a10</td>\n",
       "      <td>Cardiac troponin I is a phosphorylation target...</td>\n",
       "      <td>M. Westfall,A. M. Lee,D. Robinson</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820589394/pdf</td>\n",
       "      <td>\\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.5...</td>\n",
       "      <td>extracted</td>\n",
       "      <td></td>\n",
       "      <td>Differential Contribution of Troponin I Phosph...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2112</th>\n",
       "      <td>0a32f6f23c05827e466580647467a322b7db9f7d</td>\n",
       "      <td>This paper presents a probabilistic part-based...</td>\n",
       "      <td>Svetlana Lazebnik,C. Schmid,J. Ponce</td>\n",
       "      <td>https://hal.inria.fr/inria-00548510/file/lana_...</td>\n",
       "      <td>\\n**BLOCK**fs== 17.2**b== 0.6**t== 0.3**l== 0....</td>\n",
       "      <td>extracted</td>\n",
       "      <td>A sparse texture representation using local af...</td>\n",
       "      <td>A maximum entropy framework for part-based tex...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2113</th>\n",
       "      <td>0a3370922b04f1789a8ccfa57ae8f11188b0d378</td>\n",
       "      <td>Although the finding of severe acute respirato...</td>\n",
       "      <td>S. Lau,P. Woo,K. S. Li,Yi Huang,H. Tsoi,Beatri...</td>\n",
       "      <td>https://www.pnas.org/doi/pdf/10.1073/pnas.0506...</td>\n",
       "      <td>\\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1...</td>\n",
       "      <td>extracted</td>\n",
       "      <td>Antigenic Cross-Reactivity between Severe Acut...</td>\n",
       "      <td>Severe acute respiratory syndrome coronavirus-...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2114</th>\n",
       "      <td>0a37fc79e27f8f4a15cf83d126bd1c1f01b5d573</td>\n",
       "      <td>Aims: To study the validity of using a cross-s...</td>\n",
       "      <td>Roel C. H. Vermeulen,H. Kromhout</td>\n",
       "      <td>https://oem.bmj.com/content/oemed/62/11/793.fu...</td>\n",
       "      <td>\\n**BLOCK**fs== 13.3**b== 0.9**t== 0.1**l== 0....</td>\n",
       "      <td>extracted</td>\n",
       "      <td></td>\n",
       "      <td>Historical limitations of determinant based ex...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2115</th>\n",
       "      <td>0a38af2050b6e1ce5f20cc9df4d6e64b4af05c95</td>\n",
       "      <td>Biological methylation reactions and homocyste...</td>\n",
       "      <td>R. Jacobs,L. M. Stead,C. Devlin,I. Tabas,M. Br...</td>\n",
       "      <td>http://www.jbc.org/article/S0021925820565265/pdf</td>\n",
       "      <td>\\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.1...</td>\n",
       "      <td>extracted</td>\n",
       "      <td>Targeted Deletion of Hepatic CTP:phosphocholin...</td>\n",
       "      <td>Physiological Regulation of Phospholipid Methy...</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2116</th>\n",
       "      <td>0a3a0a1df691f2d0d3646bedd9dc48f208a3843a</td>\n",
       "      <td>None</td>\n",
       "      <td>H. Berg</td>\n",
       "      <td>http://www.cell.com/article/S0960982205008250/pdf</td>\n",
       "      <td>\\n**BLOCK**fs== 8.8**b== 0.8**t== 0.2**l== 0.1...</td>\n",
       "      <td>extracted</td>\n",
       "      <td>Species Diversity and Genetic Diversity: Paral...</td>\n",
       "      <td>Swarming Motility: It Better Be Wet</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2117 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                       paperId  \\\n",
       "0     0001d5fbff6f7c763a78dd5d141292416dcfae59   \n",
       "1     000372290caae5482dcdd7954feb895ccbab921d   \n",
       "2     000376ec6f6ade6261fc12df9eddd25a88c1e9ca   \n",
       "3     0003a5747ee3676be1b6d2c72c13a04e41811274   \n",
       "4     00043e28cffe744909bf69cde70e6dac0bce0a10   \n",
       "...                                        ...   \n",
       "2112  0a32f6f23c05827e466580647467a322b7db9f7d   \n",
       "2113  0a3370922b04f1789a8ccfa57ae8f11188b0d378   \n",
       "2114  0a37fc79e27f8f4a15cf83d126bd1c1f01b5d573   \n",
       "2115  0a38af2050b6e1ce5f20cc9df4d6e64b4af05c95   \n",
       "2116  0a3a0a1df691f2d0d3646bedd9dc48f208a3843a   \n",
       "\n",
       "                                               abstract  \\\n",
       "0     Purpose: Synovial sarcoma is a soft tissue sar...   \n",
       "1     Shuffled versions of iterative decoding of low...   \n",
       "2     This study was designed to investigate the eff...   \n",
       "3     In this report, we use zebrafish as a model sy...   \n",
       "4     Cardiac troponin I is a phosphorylation target...   \n",
       "...                                                 ...   \n",
       "2112  This paper presents a probabilistic part-based...   \n",
       "2113  Although the finding of severe acute respirato...   \n",
       "2114  Aims: To study the validity of using a cross-s...   \n",
       "2115  Biological methylation reactions and homocyste...   \n",
       "2116                                               None   \n",
       "\n",
       "                                             authorName  \\\n",
       "0     T. Ishibe,T. Nakayama,T. Okamoto,T. Aoyama,K. ...   \n",
       "1                             Juntan Zhang,M. Fossorier   \n",
       "2     Bi-qi Zhang,Shen-Jiang Hu,Lihong Qiu,Q. Shan,J...   \n",
       "3     Barden Chan,S. Sinha,Dan Cho,R. Ramchandran,V....   \n",
       "4                     M. Westfall,A. M. Lee,D. Robinson   \n",
       "...                                                 ...   \n",
       "2112               Svetlana Lazebnik,C. Schmid,J. Ponce   \n",
       "2113  S. Lau,P. Woo,K. S. Li,Yi Huang,H. Tsoi,Beatri...   \n",
       "2114                   Roel C. H. Vermeulen,H. Kromhout   \n",
       "2115  R. Jacobs,L. M. Stead,C. Devlin,I. Tabas,M. Br...   \n",
       "2116                                            H. Berg   \n",
       "\n",
       "                                          openAccessPdf  \\\n",
       "0     https://aacrjournals.org/clincancerres/article...   \n",
       "1     http://www.merl.com/publications/docs/TR2005-0...   \n",
       "2     https://www.jstage.jst.go.jp/article/bpb/28/8/...   \n",
       "3     https://onlinelibrary.wiley.com/doi/pdfdirect/...   \n",
       "4      http://www.jbc.org/article/S0021925820589394/pdf   \n",
       "...                                                 ...   \n",
       "2112  https://hal.inria.fr/inria-00548510/file/lana_...   \n",
       "2113  https://www.pnas.org/doi/pdf/10.1073/pnas.0506...   \n",
       "2114  https://oem.bmj.com/content/oemed/62/11/793.fu...   \n",
       "2115   http://www.jbc.org/article/S0021925820565265/pdf   \n",
       "2116  http://www.cell.com/article/S0960982205008250/pdf   \n",
       "\n",
       "                                             pdf_blocks pdf_status  \\\n",
       "0     \\n**BLOCK**fs== 10.0**b== 0.7**t== 0.2**l== 0....  extracted   \n",
       "1     \\n**BLOCK**fs== 10.0**b== 0.8**t== 0.2**l== 0....  extracted   \n",
       "2     \\n**BLOCK**fs== 14.0**b== 0.8**t== 0.1**l== 0....  extracted   \n",
       "3     \\n**BLOCK**fs== 9.5**b== 0.7**t== 0.2**l== 0.1...  extracted   \n",
       "4     \\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.5...  extracted   \n",
       "...                                                 ...        ...   \n",
       "2112  \\n**BLOCK**fs== 17.2**b== 0.6**t== 0.3**l== 0....  extracted   \n",
       "2113  \\n**BLOCK**fs== 9.0**b== 0.8**t== 0.2**l== 0.1...  extracted   \n",
       "2114  \\n**BLOCK**fs== 13.3**b== 0.9**t== 0.1**l== 0....  extracted   \n",
       "2115  \\n**BLOCK**fs== 6.0**b== 0.9**t== 0.1**l== 0.1...  extracted   \n",
       "2116  \\n**BLOCK**fs== 8.8**b== 0.8**t== 0.2**l== 0.1...  extracted   \n",
       "\n",
       "                                        referenceTitles  \\\n",
       "0                                                         \n",
       "1     Low-Density Parity-Check Codes;Turbo decoder a...   \n",
       "2                                                         \n",
       "3                                                         \n",
       "4                                                         \n",
       "...                                                 ...   \n",
       "2112  A sparse texture representation using local af...   \n",
       "2113  Antigenic Cross-Reactivity between Severe Acut...   \n",
       "2114                                                      \n",
       "2115  Targeted Deletion of Hepatic CTP:phosphocholin...   \n",
       "2116  Species Diversity and Genetic Diversity: Paral...   \n",
       "\n",
       "                                                  title valid_pages  \n",
       "0     Disruption of Fibroblast Growth Factor Signal ...        None  \n",
       "1                           Shuffled iterative decoding        None  \n",
       "2     Diphasic effects of Astragalus membranaceus BU...        None  \n",
       "3     Critical roles of CD146 in zebrafish vascular ...        None  \n",
       "4     Differential Contribution of Troponin I Phosph...        None  \n",
       "...                                                 ...         ...  \n",
       "2112  A maximum entropy framework for part-based tex...        None  \n",
       "2113  Severe acute respiratory syndrome coronavirus-...        None  \n",
       "2114  Historical limitations of determinant based ex...        None  \n",
       "2115  Physiological Regulation of Phospholipid Methy...        None  \n",
       "2116                Swarming Motility: It Better Be Wet        None  \n",
       "\n",
       "[2117 rows x 9 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "paperdata = load_data(subsets=2005,\n",
    "                        data_types=['text', 'metadata'],\n",
    "                        filters=[('pdf_blocks', '!=', 'None')],\n",
    "                        n_jobs=20)\n",
    "display(paperdata[\"metadata\"])\n",
    "display(paperdata[\"text\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "2025-04-01 00:15:15,444 - vectorDB.push_qdrant - INFO - Qdrant container already up and running\n"
     ]
    }
   ],
   "source": [
    "from vectorDB import QdrantManager\n",
    "qdrant = QdrantManager()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "recoRAG",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
